{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "Extract-BioMedBERT-LG-Embedding-bioasq-8b.ipynb",
      "provenance": [],
      "collapsed_sections": [
        "VfFsHIUx_HXx"
      ],
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "accelerator": "GPU"
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/aivscovid19/covid-19_research_collaboration/blob/master/notebooks/Extract_BioMedBERT_LG_Embedding_bioasq_8b.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "YxbM9_dWmq59",
        "colab_type": "text"
      },
      "source": [
        "# MRR for BERT and BERT + RESCORE\n",
        "\n",
        "[Evaluation of Ranking for Medical Papers](https://github.com/aivscovid19/covid-19_research_collaboration/issues/9)\n",
        "\n",
        "This notebook shows how to perform 3 Experiments (as of May 24 2020) on ranking using the dataset from Bioasq.\n",
        "\n",
        "- Experiment 1: score: 0.9124373431039607\n",
        "    This uses the vanilla elastic search ranking algorithm \n",
        "\n",
        "- Experiment 2: BioMedBERT LG @ 770K train step: 0.5496368744211725\n",
        "\n",
        "- Experiment 3: 0.38342058777179133 using [universal-sentence-encoder-lite](https://tfhub.dev/google/universal-sentence-encoder-lite/2)\n",
        "\n",
        "\n",
        "Right now Elastic search algorithm is the best.\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "VfFsHIUx_HXx",
        "colab_type": "text"
      },
      "source": [
        "# Setup Environment"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "VHS8WEWgTsfU",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "from google.colab.auth import authenticate_user\n",
        "authenticate_user()"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "meQeF0pr850p",
        "colab_type": "code",
        "outputId": "6af3cf50-7769-4797-fc6f-6688606727c2",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 50
        }
      },
      "source": [
        "%tensorflow_version 1.x\n",
        "\n",
        "import tensorflow as tf\n",
        "tf.__version__"
      ],
      "execution_count": 2,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "TensorFlow 1.x selected.\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "'1.15.2'"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 2
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "9q3Rzui3ugE9",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "#@title Load extensions\n",
        "%load_ext google.colab.data_table"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "RWB-_4sg1ni5",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "try:\n",
        "    %pip install --user --upgrade --quiet elasticsearch\n",
        "    import elasticsearch\n",
        "except ModuleNotFoundError:\n",
        "    import os\n",
        "    os.kill(os.getpid(), 9) "
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ucrme9vJ9mrz",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "import csv\n",
        "import os\n",
        "import time\n",
        "import re\n",
        "import sys\n",
        "import json\n",
        "\n",
        "import numpy as np\n",
        "import pandas as pd\n",
        "\n",
        "import nltk\n",
        "#from nltk.corpus import stopwords \n",
        "#from nltk.tokenize import word_tokenize\n",
        "\n",
        "from tqdm import tqdm\n",
        "\n",
        "from elasticsearch import helpers, Elasticsearch"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "E0iMu8hd-kIc",
        "colab_type": "text"
      },
      "source": [
        "## Setup Elastic Search Account\n",
        "\n",
        "Login to [Kibana](https://d7e0e807713c441295ed9707b13a089f.us-central1.gcp.cloud.es.io:9243/app/kibana#/dev_tools/console)\n",
        "Create a access token than expries in 30 days\n",
        "\n",
        "```\n",
        "POST /_security/api_key\n",
        "{\n",
        "  \"name\": \"username\",\n",
        "  \"expiration\": \"30d\", \n",
        "  \"role_descriptors\": { \n",
        "    \"role-a\": {\n",
        "      \"cluster\": [\"all\"],\n",
        "      \"index\": [\n",
        "        {\n",
        "          \"names\": [\"bioasq-8b-baseline\", \"ncbi\"],\n",
        "          \"privileges\": [\"read\"]\n",
        "        }\n",
        "      ]\n",
        "    }\n",
        "  }\n",
        "}\n",
        "```\n"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "d_w6n4Ff_miS",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "# paste the token here\n",
        "ES_ACCOUNT = {\n",
        "  \"id\" : \"bR3YinIBcfscfBmK1I1B\",\n",
        "  \"name\" : \"fabrizio\",\n",
        "  \"expiration\" : 1594059365426,\n",
        "  \"api_key\" : \"CEmtFQR4Tnq2dapnWS0tkw\"\n",
        "}"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "8NS9Vg0W1v0e",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "ES_ENDPOINT='https://d1f43211bd5c4fc29e56a232832b7b17.us-central1.gcp.cloud.es.io:9243'"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "qAkEljHg1tYC",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 235
        },
        "outputId": "e99a97b3-79f9-42a0-a1b4-d11328bb49c0"
      },
      "source": [
        "es = Elasticsearch([ES_ENDPOINT], api_key=(ES_ACCOUNT['id'], ES_ACCOUNT['api_key']))\n",
        "es.info()"
      ],
      "execution_count": 8,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "{'cluster_name': 'd1f43211bd5c4fc29e56a232832b7b17',\n",
              " 'cluster_uuid': 'gCEe5S4fTBCM_w0uejAP9w',\n",
              " 'name': 'instance-0000000006',\n",
              " 'tagline': 'You Know, for Search',\n",
              " 'version': {'build_date': '2020-03-26T06:34:37.794943Z',\n",
              "  'build_flavor': 'default',\n",
              "  'build_hash': 'ef48eb35cf30adf4db14086e8aabd07ef6fb113f',\n",
              "  'build_snapshot': False,\n",
              "  'build_type': 'tar',\n",
              "  'lucene_version': '8.4.0',\n",
              "  'minimum_index_compatibility_version': '6.0.0-beta1',\n",
              "  'minimum_wire_compatibility_version': '6.8.0',\n",
              "  'number': '7.6.2'}}"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 8
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "V6O8Hf8Q3XKO",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "#@title Elastic SEARCH\n",
        "def SEARCH(text, index, key, limit=31):\n",
        "    res = es.search(index=index,\n",
        "                    body={\n",
        "                        \"query\": {\n",
        "                            \"match\": {\n",
        "                                'context': {\n",
        "                                    \"query\": text,\n",
        "                                    \"operator\": \"or\",\n",
        "                                    \"fuzziness\": \"0\"\n",
        "                                }\n",
        "                            }\n",
        "                        },\n",
        "                        \"min_score\": -1,\n",
        "                    },\n",
        "                    size=limit)\n",
        "\n",
        "    return ([(x.get('_source'), x.get('_score')) for x in res['hits']['hits']])\n"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "1ADyCJyE342f",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "q_results = SEARCH(\"what is a virus?\", 'bioasq-8b-baseline', 'context', limit=1)"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "wKhWO9TDc5TZ",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "13aedfb6-d0b1-4675-d9be-d276d43d5cbe"
      },
      "source": [
        "len(q_results)"
      ],
      "execution_count": 11,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "1"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 11
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "PzaojXm3dGDX",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "hit, score = q_results[0]"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "-xfkcqwM3jFU",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 54
        },
        "outputId": "dcccf6c3-b24f-4265-8536-a5a27b4067be"
      },
      "source": [
        " hit['context']"
      ],
      "execution_count": 13,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "'the five key questions of human performance modeling: 1) Why we build models of human performance; 2) What the expectations of a good human performance model are; 3) What the procedures and requirements in building and verifying a human performance model are; 4) How we integrate a human performance model with system design; and 5) What the possible future directions of human performance modeling research are. This paper describes and summarizes the five key questions of human performance modeling: 1) Why we build models of human performance; 2) What the expectations of a good human performance model are; 3) What the procedures and requirements in building and verifying a human performance model are; 4) How we integrate a human performance model with system design; and 5) What the possible future directions of human performance modeling research are.We conducted a systematic review of the literature to address 5 key questions specific to this population: 1) What are the current national clinical practice recommendations and guidelines for depression screening; 2) What are the prevalence and predictors of screening; 3) How well do screening tools detect depression; 4) Does screening lead to diagnosis, treatment, and improved outcomes; and 5) What are the most effective treatment methods?This paper describes and summarizes the five key questions of human performance modeling: 1) Why we build models of human performance; 2) What the expectations of a good human performance model are; 3) What the procedures and requirements in building and verifying a human performance model are; 4) How we integrate a human performance model with system design; and 5) What the possible future directions of human performance modeling research are. We conducted a systematic review of the literature to address 5 key questions specific to this population: 1) What are the current national clinical practice recommendations and guidelines for depression screening; 2) What are the prevalence and predictors of screening; 3) How well do screening tools detect depression; 4) Does screening lead to diagnosis, treatment, and improved outcomes; and 5) What are the most effective treatment methods?<br><b>METHODS</b>: We searched bibliographic databases for full-length articles published in English between 1990 and 2010 that addressed at least 1 of our key questions.<br><b>RESULTS</b>: We identified 5 clinical practice guidelines pertinent to question 1, and 12 systematic reviews or post-hoc analyses of pooled data that addressed questions 3 through 5.'"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 13
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "mqJzF-j86mWw",
        "colab_type": "text"
      },
      "source": [
        "# Load BERT-BREATHE"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "qNdrX9Nx6la-",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "#TODO: substitude these with the latest release?\n",
        "bucket_name = 'ekaba-assets'\n",
        "model_dir = 'COPY_biomedbert_base_bert_weights_and_vocab'"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ci3jW3m88klm",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "%%capture --no-stderr\n",
        "![ -d bert ] || git clone https://github.com/google-research/bert"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "WwNDRRPQeaJs",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "sys.path.append('/content/bert')"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Po2H7pyfRMds",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "#@title read_examples\n",
        "import modeling\n",
        "import tokenization\n",
        "from extract_features import InputExample\n",
        "\n",
        "def read_examples(text_lines=[]):\n",
        "    \"\"\"Read a list of `InputExample`s from an input file.\"\"\"\n",
        "    examples = []\n",
        "    unique_id = 0\n",
        "    for line in text_lines:\n",
        "        line = line.strip()\n",
        "        linet = tokenization.convert_to_unicode(line)\n",
        "        text_a = None\n",
        "        text_b = None\n",
        "        m = re.match(r\"^(.*) \\|\\|\\| (.*)$\", line)\n",
        "        if m is None:\n",
        "            text_a = line\n",
        "        else:\n",
        "            text_a = m.group(1)\n",
        "            text_b = m.group(2)\n",
        "        examples.append(\n",
        "            InputExample(unique_id=unique_id, \n",
        "                            text_a=text_a, \n",
        "                            text_b=text_b))\n",
        "        unique_id += 1\n",
        "    return examples"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "IJVmT0VbaRtD",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "#@title input function builder \n",
        "\n",
        "def input_fn_builder():\n",
        "  all_unique_ids = []\n",
        "  all_input_ids = []\n",
        "  all_input_mask = []\n",
        "  all_input_type_ids = []\n",
        "\n",
        "  for feature in features:\n",
        "    all_unique_ids.append(feature.unique_id)\n",
        "    all_input_ids.append(feature.input_ids)\n",
        "    all_input_mask.append(feature.input_mask)\n",
        "    all_input_type_ids.append(feature.input_type_ids)\n",
        "\n",
        "  def input_fn(params):\n",
        "    \"\"\"The actual input function.\"\"\"\n",
        "    batch_size = params[\"batch_size\"]\n",
        "\n",
        "    num_examples = len(features)\n",
        "\n",
        "    # This is for demo purposes and does NOT scale to large data sets. We do\n",
        "    # not use Dataset.from_generator() because that uses tf.py_func which is\n",
        "    # not TPU compatible. The right way to load data is with TFRecordReader.\n",
        "    d = tf.data.Dataset.from_tensor_slices({\n",
        "        \"unique_ids\":\n",
        "            tf.constant(all_unique_ids, shape=[num_examples], dtype=tf.int32),\n",
        "        \"input_ids\":\n",
        "            tf.constant(\n",
        "                all_input_ids, shape=[num_examples, seq_length],\n",
        "                dtype=tf.int32),\n",
        "        \"input_mask\":\n",
        "            tf.constant(\n",
        "                all_input_mask,\n",
        "                shape=[num_examples, seq_length],\n",
        "                dtype=tf.int32),\n",
        "        \"input_type_ids\":\n",
        "            tf.constant(\n",
        "                all_input_type_ids,\n",
        "                shape=[num_examples, seq_length],\n",
        "                dtype=tf.int32),\n",
        "    })\n",
        "\n",
        "    d = d.batch(batch_size=batch_size, drop_remainder=False)\n",
        "    return d\n",
        "\n",
        "  return input_fn\n"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Q6NrWMdaP_2o",
        "colab_type": "code",
        "cellView": "code",
        "colab": {}
      },
      "source": [
        "#@title load_model\n",
        "from extract_features import convert_examples_to_features, model_fn_builder, input_fn_builder\n",
        "\n",
        "import collections\n",
        "\n",
        "def load_model(model_path, batch_size=8):\n",
        "\n",
        "    init_checkpoint = tf.train.latest_checkpoint(model_path)\n",
        "    vocab_file = os.path.join(model_path, 'vocab.txt')\n",
        "    bert_config_file = os.path.join(model_path, 'bert_config.json')\n",
        "    # TODO: assert the files exists\n",
        "\n",
        "    # we only pick the last layer\n",
        "    layer_indexes = [-1]\n",
        "    \n",
        "    bert_config = modeling.BertConfig.from_json_file(bert_config_file)\n",
        "   \n",
        "    do_lower_case = True\n",
        "\n",
        "    tokenizer = tokenization.FullTokenizer(\n",
        "        vocab_file=vocab_file, do_lower_case=do_lower_case)\n",
        "    \n",
        "    is_per_host = tf.contrib.tpu.InputPipelineConfig.PER_HOST_V2\n",
        "    master = None\n",
        "    num_tpu_cores = 8\n",
        "    run_config = tf.contrib.tpu.RunConfig(\n",
        "        master=master,\n",
        "        tpu_config=tf.contrib.tpu.TPUConfig(\n",
        "            num_shards=num_tpu_cores,\n",
        "            per_host_input_for_training=is_per_host))\n",
        "   \n",
        "    use_tpu = False\n",
        "    use_one_hot_embeddings = False\n",
        "    # BERT MODEL\n",
        "    model_fn = model_fn_builder(\n",
        "        bert_config=bert_config,\n",
        "        init_checkpoint=init_checkpoint,\n",
        "        layer_indexes=layer_indexes,\n",
        "        use_tpu=use_tpu,\n",
        "        use_one_hot_embeddings=use_one_hot_embeddings)\n",
        "    \n",
        "    # If TPU is not available, this will fall back to normal Estimator on CPU\n",
        "    # or GPU.\n",
        "    estimator = tf.contrib.tpu.TPUEstimator(\n",
        "        use_tpu=use_tpu,\n",
        "        model_fn=model_fn,\n",
        "        config=run_config,\n",
        "        predict_batch_size=batch_size)\n",
        "    \n",
        "    return estimator, tokenizer\n",
        "\n",
        "\n",
        "def predict_fn(input_values, estimator, tokenizer, max_seq_length=128):\n",
        "    # process input \n",
        "    examples = read_examples(input_values)\n",
        "\n",
        "    # Build \n",
        "    features = convert_examples_to_features(\n",
        "        examples=examples, seq_length=max_seq_length, tokenizer=tokenizer)\n",
        "    \n",
        "    # input_fn = input_fn_builder(\n",
        "    #     features=features, seq_length=max_seq_length)\n",
        "    \n",
        "    all_unique_ids = []\n",
        "    all_input_ids = []\n",
        "    all_input_mask = []\n",
        "    all_input_type_ids = []\n",
        "\n",
        "    for feature in features:\n",
        "        all_unique_ids.append(feature.unique_id)\n",
        "        all_input_ids.append(feature.input_ids)\n",
        "        all_input_mask.append(feature.input_mask)\n",
        "        all_input_type_ids.append(feature.input_type_ids)\n",
        "\n",
        "    def input_fn(params):\n",
        "        \"\"\"The actual input function.\"\"\"\n",
        "        batch_size = params[\"batch_size\"]\n",
        "    \n",
        "        num_examples = len(features)\n",
        "    \n",
        "        # This is for demo purposes and does NOT scale to large data sets. We do\n",
        "        # not use Dataset.from_generator() because that uses tf.py_func which is\n",
        "        # not TPU compatible. The right way to load data is with TFRecordReader.\n",
        "        d = tf.data.Dataset.from_tensor_slices({\n",
        "            \"unique_ids\":\n",
        "                tf.constant(all_unique_ids, shape=[num_examples], dtype=tf.int32),\n",
        "            \"input_ids\":\n",
        "                tf.constant(\n",
        "                    all_input_ids, shape=[num_examples, max_seq_length],\n",
        "                    dtype=tf.int32),\n",
        "            \"input_mask\":\n",
        "                tf.constant(\n",
        "                    all_input_mask,\n",
        "                    shape=[num_examples, max_seq_length],\n",
        "                    dtype=tf.int32),\n",
        "            \"input_type_ids\":\n",
        "                tf.constant(\n",
        "                    all_input_type_ids,\n",
        "                    shape=[num_examples, max_seq_length],\n",
        "                    dtype=tf.int32),\n",
        "        })\n",
        "\n",
        "        d = d.batch(batch_size=batch_size)\n",
        "        return d\n",
        "\n",
        "    unique_id_to_feature = {}\n",
        "    for feature in features:\n",
        "        unique_id_to_feature[feature.unique_id] = feature\n",
        "\n",
        "    layer_indexes = [-1]\n",
        "    result_list = []\n",
        "    for result in estimator.predict(input_fn, yield_single_examples=True):\n",
        "        unique_id = int(result[\"unique_id\"])\n",
        "        feature = unique_id_to_feature[unique_id]\n",
        "        output_json = collections.OrderedDict()\n",
        "        output_json[\"linex_index\"] = unique_id\n",
        "        all_features = []\n",
        "        for (i, token) in enumerate(feature.tokens):\n",
        "            all_layers = []\n",
        "            for (j, layer_index) in enumerate(layer_indexes):\n",
        "                layer_output = result[\"layer_output_%d\" % j]\n",
        "                layers = collections.OrderedDict()\n",
        "                layers[\"index\"] = layer_index\n",
        "                layers[\"values\"] = [\n",
        "                    round(float(x), 6) for x in layer_output[i:(i + 1)].flat\n",
        "                ]\n",
        "                all_layers.append(layers)\n",
        "            features = collections.OrderedDict()\n",
        "            features[\"token\"] = token\n",
        "            features[\"layers\"] = all_layers\n",
        "            all_features.append(features)\n",
        "        output_json[\"features\"] = all_features\n",
        "        result_list.append(output_json)\n",
        "    return result_list"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "3HFIpRYeAJ5u",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "170b3dda-a727-4e13-a008-5fb519591523"
      },
      "source": [
        "MODEL_LOCATION=f'gs://{bucket_name}/{model_dir}/'\n",
        "MODEL_LOCATION"
      ],
      "execution_count": 20,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "'gs://ekaba-assets/COPY_biomedbert_base_bert_weights_and_vocab/'"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 20
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "joHeQfLqYHg2",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 390
        },
        "outputId": "974bea73-3de3-4acd-a1f4-f43d72ee891c"
      },
      "source": [
        "estimator, tokenizer = load_model(MODEL_LOCATION, batch_size=64)"
      ],
      "execution_count": 21,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "WARNING:tensorflow:From /content/bert/modeling.py:93: The name tf.gfile.GFile is deprecated. Please use tf.io.gfile.GFile instead.\n",
            "\n",
            "WARNING:tensorflow:\n",
            "The TensorFlow contrib module will not be included in TensorFlow 2.0.\n",
            "For more information, please see:\n",
            "  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md\n",
            "  * https://github.com/tensorflow/addons\n",
            "  * https://github.com/tensorflow/io (for I/O related ops)\n",
            "If you depend on functionality not listed there, please file an issue.\n",
            "\n",
            "WARNING:tensorflow:Estimator's model_fn (<function model_fn_builder.<locals>.model_fn at 0x7efde2ab3598>) includes params argument, but params are not passed to Estimator.\n",
            "WARNING:tensorflow:Using temporary folder as model directory: /tmp/tmphc2waln9\n",
            "INFO:tensorflow:Using config: {'_model_dir': '/tmp/tmphc2waln9', '_tf_random_seed': None, '_save_summary_steps': 100, '_save_checkpoints_steps': None, '_save_checkpoints_secs': 600, '_session_config': allow_soft_placement: true\n",
            "graph_options {\n",
            "  rewrite_options {\n",
            "    meta_optimizer_iterations: ONE\n",
            "  }\n",
            "}\n",
            ", '_keep_checkpoint_max': 5, '_keep_checkpoint_every_n_hours': 10000, '_log_step_count_steps': None, '_train_distribute': None, '_device_fn': None, '_protocol': None, '_eval_distribute': None, '_experimental_distribute': None, '_experimental_max_worker_delay_secs': None, '_session_creation_timeout_secs': 7200, '_service': None, '_cluster_spec': <tensorflow.python.training.server_lib.ClusterSpec object at 0x7efde2ae39e8>, '_task_type': 'worker', '_task_id': 0, '_global_id_in_cluster': 0, '_master': '', '_evaluation_master': '', '_is_chief': True, '_num_ps_replicas': 0, '_num_worker_replicas': 1, '_tpu_config': TPUConfig(iterations_per_loop=2, num_shards=8, num_cores_per_replica=None, per_host_input_for_training=3, tpu_job_name=None, initial_infeed_sleep_secs=None, input_partition_dims=None, eval_training_input_configuration=2, experimental_host_call_every_n_steps=1), '_cluster': None}\n",
            "INFO:tensorflow:_TPUContext: eval_on_tpu True\n",
            "WARNING:tensorflow:eval_on_tpu ignored because use_tpu is False.\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "2toPXqc2YAB6",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "def extract_embeddings(question_context_list):\n",
        "    # 1. extract context\n",
        "    context_list = [ q['context'] for q in question_context_list]\n",
        "    query_list = [ q['question'] for q in question_context_list ] \n",
        "\n",
        "    # 2. compute the embeddings\n",
        "    embedding_list = predict_fn(query_list + context_list, estimator, tokenizer)\n",
        "\n",
        "    query_embeddings = embedding_list[:len(query_list)] \n",
        "    context_embeddings = embedding_list[len(query_list):] \n",
        "    assert len(query_embeddings) == len(context_embeddings)\n",
        "\n",
        "    query_embeddings = get_sent_embed(query_embeddings)\n",
        "    context_embeddings = get_sent_embed(context_embeddings)\n",
        "    \n",
        "    return query_embeddings, context_embeddings"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "cphoNIthWKWq",
        "colab_type": "text"
      },
      "source": [
        "# Compute MRR"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "EEPFCkhfWp7M",
        "colab_type": "text"
      },
      "source": [
        "The '''mean reciprocal rank''' is a statistic measure for evaluating any process that produces a list of possible responses to a sample of queries, ordered by probability of correctness.\n",
        "\n",
        "The reciprocal rank of a query response is the multiplicative inverse of the rank of the first correct answer: \n",
        "- 1 for first place, \n",
        "- $\\frac12$ for second place, \n",
        "- $\\frac13$ for third place and so on.\n",
        "\n",
        "The mean reciprocal rank is the average of the reciprocal ranks of results for a sample of queries \n",
        "\n",
        "$$\n",
        " \\text{MRR} = \\frac{1}{|Q|} \\sum_{i=1}^{|Q|} \\frac{1}{\\text{rank}_i}\n",
        "$$\n",
        "\n",
        "where $\\text{rank}_i$ refers to the rank position of the ''first'' relevant document for the ''i''-th query.\n",
        "\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "IUscCJMsSlTA",
        "colab_type": "text"
      },
      "source": [
        "# Read BIOASQ 8b question json  "
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "SAIELMffTGhQ",
        "colab_type": "code",
        "outputId": "786023aa-e6b6-45c7-f527-136dae6a9d62",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 67
        }
      },
      "source": [
        "!gsutil cp gs://bioasq/DATA/training8b_list.json ."
      ],
      "execution_count": 23,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Copying gs://bioasq/DATA/training8b_list.json...\n",
            "/ [0 files][    0.0 B/  2.2 MiB]                                                \r/ [1 files][  2.2 MiB/  2.2 MiB]                                                \r\n",
            "Operation completed over 1 objects/2.2 MiB.                                      \n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "fFquycc1T6wy",
        "colab_type": "code",
        "outputId": "32e1bf37-6e1c-4d0e-beb2-b732125feca3",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        }
      },
      "source": [
        "!ls ."
      ],
      "execution_count": 24,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "adc.json  bert\tsample_data  training8b_list.json\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "H0k-xayHT3Aj",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "import json\n",
        "with open('./training8b_list.json') as fd:\n",
        "    dataset = json.load(fd)"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "i8HboFAJUAtm",
        "colab_type": "code",
        "outputId": "e0433174-26f2-4e01-b018-08647107f868",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 54
        }
      },
      "source": [
        "!head training8b_list.json"
      ],
      "execution_count": 26,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "{\"questions\": [{\"question\": \"List signaling molecules (ligands) that interact with the receptor EGFR?\", \"ideal_answer\": [\"The 7 known EGFR ligands  are: epidermal growth factor (EGF), betacellulin (BTC), epiregulin (EPR), heparin-binding EGF (HB-EGF), transforming growth factor-\\u03b1 [TGF-\\u03b1], amphiregulin (AREG) and epigen (EPG).\"], \"exact_answer\": [[\"epidermal growth factor\"], [\"betacellulin\"], [\"epiregulin\"], [\"heparin-binding epidermal growth factor\"], [\"transforming growth factor-\\u03b1\"], [\"amphiregulin\"], [\"epigen\"]], \"context\": \"the epidermal growth factor receptor (EGFR) ligands, such as epidermal growth factor (EGF) and amphiregulin (AREG) EGFR ligands epidermal growth factor (EGF), amphiregulin (AREG) and transforming growth factor alpha (TGF\\u03b1) EGFR and its ligand EGF Among EGFR ligands, heparin-binding EGF-like growth factor, TGF-\\u03b1 and Betacellulin (BTC) are produced in the tumor microenvironment of FDC-S at RNA level. . Plasma amphiregulin (AR), epidermal growth factor (EGF), transforming growth factor-\\u03b1, and heparin binding-EGF were assessed by ELISA in 45 chemorefractory mCRC patientsAmong EGFR ligands, heparin-binding epidermal growth factor (HB-EGF) Of the six known EGFR ligands, transforming growth factor alpha (TGF\\u03b1) was expressed more highly in triple-negative breast tumors than in tumors of other subtypes.the 7 known EGFR ligands (EGF, betacellulin, epiregulin, heparin-binding EGF, transforming growth factor-\\u03b1 [TGF-\\u03b1], amphiregulin, and epigen) EGFR ligands based on the two affinity classes: EGF>HB-EGF>TGF-\\u03b1>BTC>EPR>EPG>ARIn this article, however, we demonstrate that PEPD directly binds to and activates epidermal growth factor receptor (EGFR),four EGFR ligands (AR, HB-EGF, TGF-\\u03b1, and EREG) Epidermal growth factor (EGF) family peptides are ligands for the EGF receptor (EGFR). oluble amphiregulin (AR), transforming growth factor alpha (TGF\\u03b1), neuregulin 2 beta, and epigen stimulate greater EGFR coupling to cell proliferation and DNA synthesis than do EGF, betacellulin, heparin-binding EGF-like growth factor, and epiregulinHere, we demonstrate that histamine releases 2 EGFR ligands, amphiregulin and heparin-binding epidermal growth factor-like growth factor (HB-EGF), from airway epithelial cells.mammalian EGFR ligands including EGF, TGF-\\u03b1 (TGF\\u03b1), amphiregulin (AREG), heparin-binding EGF-like growth factor (HB-EGF), betacellulin, epiregulin, and epigen.\"}, {\"question\": \"Which miRNAs could be used as potential biomarkers for epithelial ovarian cancer?\", \"ideal_answer\": [\"miR-200a, miR-100, miR-141, miR-200b, miR-200c, miR-203, miR-510, miR-509-5p, miR-132, miR-26a, let-7b, miR-145, miR-182, miR-152, miR-148a, let-7a, let-7i, miR-21, miR-92 and miR-93 could be used as potential biomarkers for epithelial ovarian cancer.\"], \"exact_answer\": [[\"let-7a\"], [\"let-7b\"], [\"let-7i\"], [\"miR-21\"], [\"miR-26a\"], [\"miR-92\"], [\"miR-93\"], [\"miR-100\"], [\"miR-132\"], [\"miR-141\"], [\"miR-145\"], [\"miR-148a\"], [\"miR-152\"], [\"miR-182\"], [\"miR-200a\"], [\"miR-200b\"], [\"miR-200c\"], [\"miR-203\"], [\"miR-509-5p\"], [\"miR-510\"]], \"context\": \"Finally, five promising differentially miRNAs (miR-200a, miR-100, miR-141, miR-200b, and miR-200c) were reported with the consistent direction in four or more studies. MiR-200a, miR-200b, miR-200c, and miR-141, all of them belong to miR-200 family, were reported with consistently up-regulated in at least 4 studies, whereas miR-100 was reported with down-regulated in 4 studiesUpregulation of microRNA-203 is associated with advanced tumor progression and poor prognosis in epithelial ovarian cancermultivariate analysis showed that the status of miR-203 expression was an independent predictor for both overall survival and progression-free survival in EOC. These findings provide the convincing evidence for the first time that the upregulation of miR-203 may serve as a novel molecular marker to predict the aggressive tumor progression and unfavorable prognosis of EOC patientsSome, but not all, of the data indicated that the miR-200 family was dysregulated in a variety of malignancies. In this study, we demonstrated that miR-200a and E-cadherin were significantly upregulated in EOC compared to benign epithelial ovarian cysts and normal ovarian tissuesThere was a significantly positive correlation between miR-200a and E-cadherin in EOC. The biphasic expression pattern suggested that miR-200a levels may serve as novel biomarkers for the early detection of EOC, and miR-200a and E-cadherin are candidate targets for the development of new treatment modalities against ovarian cancerKaplan-Meier analysis showed low miR-510 expression, low miR-509-5p expression, and advanced FIGO stage, and chemotherapy resistance were significantly associated with poorer overall survival (P < 0.05). Our results suggest that miRNAs may play a role in the progression of OSC, and miR-510 and miR-509-5p may be considered novel-candidate clinical biomarkers for predicting OSC outcomeThe qRT-PCR results showed that miR-510, miR-509-5p, and miR-508-3p were significantly downregulated and that miR-483-5p was upregulated in stage III OSC compared with stage I, which was consistent with the microarray resultsIn this study, we examined serum miR-21 levels in epithelial ovarian cancer (EOC) patients, and explored its association with clinicopathological factors and prognosis. The results showed significantly higher serum miR-21 levels in EOC patients than in healthy controls. In addition, increased serum miR-21 expression was correlated with advanced FIGO stage, high tumor grade, and shortened overall survival. These findings indicate that serum miR-21 may serve as a novel diagnostic and prognostic marker, and be used as a therapeutic target for the treatment of EOCIdentification of serum microRNA-21 as a biomarker for early detection and prognosis in human epithelial ovarian cancerSerum miR-132, miR-26a, let-7b, and miR-145 could be considered as potential candidates as novel biomarkers in serous ovarian cancer.Among the miRNAs that showed a consistent regulation tendency through all specimens and showed more than a 2-fold difference in serum, 5 miRNAs (miR-132, miR-26a, let-7b, miR-145, and miR-143) were determined as the 5 most markedly down-regulated miRNAs in the serum from ovarian cancer patients with respect to those of controls. Four miRNAs (miR-132, miR-26a, let-7b, and miR-145) out of 5 selected miRNAs were significantly underexpressed in the serum of ovarian cancer patients in qRT-PCR microRNA (miR)-182, miR-200a, miR-200b and miR-200c were highly overexpressed in the SEOC cell lines relative to normal human ovarian surface epithelial cells and were assessed in RNA extracted from serum as candidate biomarkersTogether, these results suggest that low miR-100 expression             may be an independent poor prognostic factor and miR-100 can function as a tumor             suppressor by targeting PLK1 in human EOCsPrognostic implications of microRNA-100 and its functional roles in             human epithelial ovarian cancerTaken together, miR-152 and miR-148a may be involved             in the carcinogenesis of ovarian cancer through deregulation of cell proliferation.             They may be novel biomarkers for early detection or therapeutic targets of ovarian             cancerMicroRNA let-7a: a potential marker for selection of paclitaxel in ovarian cancer management The study suggests that the beneficial impact of the addition of paclitaxel on EOC survival was significantly linked to let-7a levels, and that miRNAs such as let-7a may be a useful marker for selection of chemotherapeutic agents in EOC managementmiR-200c has potential as a predictor of survival, and is a biomarker of relapse, in stage I EOCMicroRNA microarray identifies Let-7i as a novel biomarker and therapeutic target in human epithelial ovarian cancer our results strongly suggest that let-7i might be used as a therapeutic target to modulate platinum-based chemotherapy and as a biomarker to predict chemotherapy response and survival in patients with ovarian cancermiRNAs-21, 92 and 93 are known oncogenes with therapeutic and biomarker potentialSerum miR-132, miR-26a, let-7b, and miR-145 could be considered as potential candidates as novel biomarkers in serous ovarian cancer\"}, {\"question\": \"Which acetylcholinesterase inhibitors are used for treatment of myasthenia gravis?\", \"ideal_answer\": [\"Pyridostigmine and neostygmine are acetylcholinesterase inhibitors that are used as first-line therapy for symptomatic treatment of myasthenia gravis. Pyridostigmine is the most widely used acetylcholinesterase inhibitor. Extended release pyridotsygmine and novel acetylcholinesterase inhibitors inhibitors with oral antisense oligonucleotides are being studied.\"], \"exact_answer\": [[\"neostigmine\"], [\"pyridostigmine\"]], \"context\": \"Pyridostigmine is the most widely used acetylcholinesterase inhibitor.For more than 50 years the acetylcholinesterase inhibitor pyridostigmine bromide has been the drug of choice in the symptomatic therapy for myasthenia gravis. The switch to SR-Pyr ameliorated the total quantified myasthenia gravis (QMG) score from 0.9 \\u00b1 0.5 to 0.6 \\u00b1 0.4 (p<0.001) in all patients and in the younger subgroup. This was accompanied by a significant improvement in the quality of life parameters. The health status valued by EuroQoL questionnaire improved from 0.626 \\u00b1 0.286 to 0.782 \\u00b1 0.186 (p<0.001). Our results support the usefulness of SR-Pyr in an individualized therapeutic regimen to improve quality of life regardless of the patient's age in myasthenia gravis.This review focuses on treatment of MG, mainly on the use of the AChE inhibitor pyridostigmine.Despite a lack of data from well controlled clinical trials to support their use, AChE inhibitors, of which pyridostigmine is the most commonly used, are recommended as first-line therapy for MG. Novel AChE inhibitors with oral antisense oligonucleotides have been developed and preliminary results appear to be promising.Except for one small and inconclusive trial of intranasal neostigmine, no randomised controlled trial has been conducted on the use of acetylcholinesterase inhibitors in myasthenia gravis. Response to acetylcholinesterase inhibitors in observational studies is so clear that a randomised controlled trial depriving participants in the placebo arm of treatment would be difficult to justify. Current guidelines and recommendations for MG treatment are based largely on clinical experience, retrospective analyses and expert consensus. Available therapies include oral acetylcholinesterase (AChE) inhibitors for symptomatic treatment, and short- and long-term disease-modifying treatments.Pyridostigmine has been used as a treatment for MG for over 50 years and is generally considered safe. It is suitable as a long-term treatment in patients with generalized non-progressive milder disease, and as an adjunctive therapy in patients with severe disease who are also receiving immunotherapy. Acetylcholinesterase inhibitors provide temporary, symptomatic treatment for all forms of myasthenia gravis. \"}, {\"question\": \"List the human genes encoding for the dishevelled proteins?\", \"ideal_answer\": [\"DVL-1\\nDVL-2\\nDVL-3\"], \"exact_answer\": [[\"DVL-1\"], [\"DVL-2\"], [\"DVL-3\"]], \"context\": \"Dishevelled (Dvl/Dsh) is a multi-module protein and a key regulator of both the canonical Wnt and the PCP pathway. In mouse, all Dvl1(-/-) ; Dvl2(-/-) double mutants display craniorachischisis, a severe form of open NTDs.  In this study, we explore the cause of HSCR by studying the expression of DVL-1 and DVL-3 genes and their proteins in the aganglionic segment and the ganglionic segment of colon in HSCR patients.Dishevelled (Dvl) proteins are key transducers of Wnt signaling encoded by members of a multi-gene family in vertebrates. We report here the divergent, tissue-specific expression patterns for all three Dvl genes in Xenopus embryos, which contrast dramatically with their expression patterns in mice.velopmental processes, including segmentation and neuroblast specification. We have isolated and characterized cDNA clones from two different human dsh-homologous genes, designated as DVL-1 and DVL-3. In the Drosophila embryo dishevelled (dsh) function is required by target cells in order to respond to wingless (wg, the homolog of Wnt-1), demonstrating a role for dsh in Wnt signal transduction. We have isolated a mouse homolog of the Drosophila dsh segment polarity gene. The 695-amino-acid protein encoded by the mouse dishevelled gene (Dvl-1) shares 50% identity (65% similarity) with dsh.The Dvl-1 gene on chromosome 1p36 belongs to a family of highly conserved secreted proteins which regulates embryonic induction, generation of cell polarity and specification of cell fate through activation of Wnt signaling pathways. Wnt signaling activates the gene encoding DVL-1;We report here that the mouse Dishevelled-1 (Dvl-1) and Dishevelled-2 genes encode proteins that are differentially localized in Wnt-overexpressing PC12 cell lines (PC12/Wnt). Recently, the DVL1 gene was identified as a middle molecule of the Wnt/beta-catenin signaling pathway. \"}, {\"question\": \"Which are the classes of anti-arrhythmic drugs according to Vaughan-Williams classification?\", \"ideal_answer\": [\"Antiarrhythmic drugs can be divided into four Vaughan Williams classes (I-IV). Class I antiarrhythmic agents have as a common action, blockade of the sodium channels. Class II agents are antisympathetic drugs, particularly the beta-adrenoceptor blockers. Class-III antiarrhythmics have as a common action the potassium-channel blockade. Class IV antiarrhythmic drugs are calcium channel blockers.\"], \"exact_answer\": [[\"Class I: sodium channel blockers\"], [\"ClassII: beta blockers\"], [\"Class III: potassium channel blockers\"], [\"Class IV: calcium channel blockers\"]], \"context\": \"Class II agents are antisympathetic drugs, particularly the beta-adrenoceptor blockersClass III antiarrhythmic agents include sotalol and amiodarone.Class IV antiarrhythmic drugs are the calcium channel blockers verapamil and diltiazem. Other agents which do not fall neatly into the Vaughan Williams classification include digoxin and perhexiline.Antiarrhythmic drugs can be divided into four Vaughan Williams classes (I-IV) according to defined electrophysiological effects on the myocardium.Thus, the Vaughan Williams classification also coincides with the main myocardial targets of the antiarrhythmics, i.e., myocardial sodium-, potassium-, and calcium-channels or beta-adrenergic receptors. The sodium-channel blockade induced by class-I substances is enhanced with increasing heart rates. Thus, class-I antiarrhythmics can be subclassified as substances showing a more exponential, an approximately linear, or rather saturated block-frequency relation.Class-III antiarrhythmics (potassium-channel blockade) can be further differentiated according to the component of the delayed rectifier potassium current (IK) which is inhibited by a drug. Class-III substances inhibiting the slowly activating IKs component are currently under investigation and are expected to show a direct rate dependence.Antiarrhythmic agents are traditionally classified according to Vaughan Williams into four classes of action. Class I antiarrhythmic agents include most of the drugs traditionally thought of as antiarrhythmics, and have as a common action, blockade of the fast-inward sodium channel on myocardium. Antiarrhythmic drugs can be divided into four Vaughan Williams classes (I-IV) according to defined electrophysiological effects on the myocardiumThe classification of antiarrhythmic agents according to Vaughan Williams is based on electrophysiological findings in isolated heart muscle and defines four classes of drug actions.Antiarrhythmic agents are traditionally classified according to Vaughan Williams into four classes of action. Class I antiarrhythmic agents include most of the drugs traditionally thought of as antiarrhythmics, and have as a common action, blockade of the fast-inward sodium channel on myocardium.Class II agents are antisympathetic drugs, particularly the b-adrenoceptor blockers.Class III antiarrhythmic agents include sotalol and amiodarone. Class IV antiarrhythmic drugs are the calcium channel blockers verapamil and diltiazem. These are classified according to their electrophysiological effects observed in isolated cardiac tissues in vitro (Vaughan Williams, 1989). Fast sodium channel blockers (class I) which reduce the upstroke velocity of the action potential are usually subclassified into three groups, class I A-C, according to their effect on the action potential duration. Beta-adrenergic antagonists (class II) exert their effects by antagonizing the electrophysiological effects of beta-adrenergic catecholamines. Class III antiarrhythmic agents (eg amiodarone) prolong the action potential and slow calcium channel blockers (class IV) suppress the calcium inward current and calcium-dependent action potentials. \"}, {\"question\": \"Which are the different isoforms of the mammalian Notch receptor?\", \"ideal_answer\": [\"Notch signaling is an evolutionarily conserved mechanism, used to regulate cell fate decisions. Four Notch receptors have been identified in man: Notch-1, Notch-2, Notch-3 and Notch-4.\"], \"exact_answer\": [[\"Notch-1\"], [\"Notch-2\"], [\"Notch-3\"], [\"Notch-4\"]], \"context\": \"Here, we investigate protein interactions between NOTCH3 and other vascular Notch isoforms and characterize the effects of elevated NOTCH3 on smooth muscle gene regulation. We demonstrate that NOTCH3 forms heterodimers with NOTCH1, NOTCH3, and NOTCH4.We conclude that CADASIL mutants of NOTCH3 complex with NOTCH1, 3, and 4, slow NOTCH3 clearance, and that overexpressed wild type and mutant NOTCH3 protein interfere with key NOTCH-mediated functions in smooth muscle cells.In the vertebrate embryo, skeletal muscle is derived from the myotome of the somites. Notch1-3 demonstrate overlapping and distinct expression patterns in mouse somites. Notch1 and Notch2 have been shown to be inhibitors of skeletal myogenesis. In this study, we analyzed the immunohistochemical staining pattern of four Notch receptors (Notch1-4) and their ligands (Delta1 and Jagged1) in 14 synovial tissues obtained from 14 RA patients. Notch signaling is an evolutionarily conserved mechanism, used to regulate cell fate decisions. Four Notch receptors have been identified in man (Notch-1 to -4).All 4 receptors were expressed in the adult liver, with no significant differences in the levels of Notch-1, -2, and -4 messenger RNA (mRNA) between normal and diseased liver. However, Notch-3 expression appeared to be increased in diseased tissue. We studied the immunohistochemical expression of NOTCH2 and its isoforms NOTCH1, NOTCH3, and NOTCH4 and the NOTCH2 primary ligand JAGGED1 in hepatoblastomas.There are four different mammalian Notch receptors that can be activated by five cell surface ligandsThere are four mammalian Notch receptors that have only partially overlapping functions despite sharing similar structures and ligands.\"}, {\"question\": \"Which are the major characteristics of cellular senescence?\", \"ideal_answer\": [\"The defining characteristics of cellular senescence are altered morphology, arrested cell-cycle progression, development of aberrant gene expression with proinflammatory behavior, and telomere shortening.\"], \"exact_answer\": [[\"altered morphology\"], [\"arrested cell-cycle progression\"], [\"development of aberrant gene expression with proinflammatory behavior\"], [\"telomere shortening\"]], \"context\": \"Cellular senescence is recognized as a critical cellular response to prolonged rounds of replication and environmental stresses. Its defining characteristics are arrested cell-cycle progression and the development of aberrant gene expression with proinflammatory behavior.telomeres are the central timing mechanism for cellular agingRecent research has shown that inserting a gene for the protein component of telomerase into senescent human cells reextends their telomeres to lengths typical of young cells, and the cells then display all the other identifiable characteristics of young, healthy cells.Our data demonstrate that Sod1 transfected cell lines that have an elevation in the ratio of Sod1 activity to Gpx1 activity produce higher levels of H2O2 and exhibit well characterised markers of cellular senescence viz. slower proliferation and altered morphology.\"}, {\"question\": \"Which are the Yamanaka factors?\", \"ideal_answer\": [\"The Yamanaka factors are the OCT4, SOX2, MYC, and KLF4 transcription factors\"], \"exact_answer\": [[\"OCT4\", \"Oct3/4\", \"Pou5f1\"], [\"SOX2\"], [\"MYC\", \"c-MYC\"], [\"KLF4\"]], \"context\": \"Yamanaka factors (OCT4, SOX2, MYC, and KLF4Yamanaka factors' (Oct4, Sox2, Klf4 and c-Myc)Yamanaka factors (Oct4, Sox2, Klf4, and c-Myc) Yamanaka factors (c-myc, KLF4, Oct3/4 and SOX2)Yamanaka factors (i.e., Oct4, Sox2, Klf4, and c-Myc) Yamanaka factors (Oct3/4, Sox2, Klf4, and c-Myc)Yamanaka factors (OCT4, SOX2, KLF4, cMYC - OSKMOct4, Sox2, Klf4 and cMyc (4TF, Yamanaka factors) Yamanaka factors (Pou5f1, Myc, Klf4, and Sox2) c-Myc, Klf4, Oct3/4, and Sox2 (the so-called \\\"Yamanaka factors\\\")Yamanaka factors, Oct3/4, Sox2, Klf4, and c-MycYamanaka factors, namely Sox2, Oct3/4 (Pou5f1), Klf4, and c-MycOct4, Sox2, Klf4 and c-Myc, also known as the Yamanaka factors. Yamanaka factors (SOX2, OCT3/4, and KLF4, with or without c-MYC)Yamanaka factors Oct4, Sox2, Klf4, and c-Mycamanaka factors (Oct3/4, Sox2, Klf4, c-Myc)Yamanaka factors (Oct3/4, Sox2, Klf4, c-Myc) are highly expressed in embryonic stem (ES) cells, and their over-expression can induce pluripotency in both mouse and human somatic cells, indicating that these factors regulate the developmental signaling network necessary for ES cell pluripotency.These protein sets have been called the Yamanaka factors, namely Sox2, Oct3/4 (Pou5f1), Klf4, and c-Myc, and the Thomson factors, namely Sox2, Oct3, Lin28, and NanogYamanaka factors (Oct3/4, Sox2, Klf4, c-Myc) are highly expressed in embryonic stem (ES) cells, and their over-expression can induce pluripotency in both mouse and human somatic cells, indicating that these factors regulate the developmental signaling network necessary for ES cell pluripotencyTranscription factors, Oct4, Sox2, Klf4 and cMyc (4TF, Yamanaka factors) are used as basal conditions to generate iPS cellsGeneration of induced pluripotent stem (iPS) cells from somatic cells has been successfully achieved by ectopic expression of four transcription factors, Oct4, Sox2, Klf4 and c-Myc, also known as the Yamanaka factors\"}, {\"question\": \"Which currently known mitochondrial diseases have been attributed to POLG mutations?\", \"ideal_answer\": [\"Mutations in the POLG gene have emerged as one of the most common causes of inherited mitochondrial disease in children and adults. They are responsible for a heterogeneous group of at least 6 major phenotypes of neurodegenerative disease that include: 1) childhood Myocerebrohepatopathy Spectrum disorders (MCHS), 2) Alpers syndrome, 3) Ataxia Neuropathy Spectrum (ANS) disorders, 4) Myoclonus Epilepsy Myopathy Sensory Ataxia (MEMSA), 5) autosomal recessive Progressive External Ophthalmoplegia (arPEO), and 6) autosomal dominant Progressive External Ophthalmoplegia (adPEO).\", \"Mutations in the POLG gene have emerged as one of the most common causes of inherited mitochondrial disease in children and adults. They are responsible for a heterogeneous group of at least 6 major phenotypes of neurodegenerative disease that include: 1) childhood Myocerebrohepatopathy Spectrum disorders (MCHS), 2) Alpers syndrome, 3) Ataxia Neuropathy Spectrum (ANS) disorders, 4) Myoclonus Epilepsy Myopathy Sensory Ataxia (MEMSA), 5) autosomal recessive Progressive External Ophthalmoplegia (arPEO), and 6) autosomal dominant Progressive External Ophthalmoplegia (adPEO)\"], \"exact_answer\": [[\"childhood Myocerebrohepatopathy Spectrum disorders (MCHS)\"], [\"Alpers syndrome\"], [\"Ataxia Neuropathy Spectrum (ANS) disorders\"], [\"Myoclonus Epilepsy Myopathy Sensory Ataxia (MEMSA)\"], [\"autosomal recessive Progressive External Ophthalmoplegia (arPEO)\"], [\"autosomal dominant Progressive External Ophthalmoplegia (adPEO)\"]], \"context\": \"Mutations in the POLG gene have emerged as one of the most common causes of inherited mitochondrial disease in children and adults. They are responsible for a heterogeneous group of at least 6 major phenotypes of neurodegenerative disease that include: 1) childhood Myocerebrohepatopathy Spectrum disorders (MCHS), 2) Alpers syndrome, 3) Ataxia Neuropathy Spectrum (ANS) disorders, 4) Myoclonus Epilepsy Myopathy Sensory Ataxia (MEMSA), 5) autosomal recessive Progressive External Ophthalmoplegia (arPEO), and 6) autosomal dominant Progressive External Ophthalmoplegia (adPEO)Mutations in the gene encoding mitochondrial DNA polymerase gamma (POLG), the enzyme that synthesises mitochondrial DNA (mtDNA), have been associated with a mitochondrial disease-autosomal dominant or recessive progressive external ophthalmoplegia-and multiple deletions of mtDNA.About 150 mutations in the human POLG have been identified in patients with mitochondrial diseases such as Alpers syndrome, progressive external ophthalmoplegia, and ataxia-neuropathy syndromes.Nineteen exhibited a cluster of three or more predefined clinical manifestations suggestive of POLG-related disease: progressive external ophthalmoplegia, seizures and/or an abnormal electroencephalogram, neuropathy, ataxia, liver function abnormalities, migraine or dysphagia/dysarthria. Novel POLG mutations in progressive external ophthalmoplegia mimicking mitochondrial neurogastrointestinal encephalomyopathy.\"}, {\"question\": \"Which proteins participate in the formation of the ryanodine receptor quaternary macromolecular complex?\", \"ideal_answer\": [\"Junctin is a major transmembrane protein in cardiac junctional sarcoplasmic reticulum, which forms a quaternary complex with the ryanodine receptor (Ca(2+) release channel), triadin, and calsequestrin.   \", \"Calsequestrin (CSQ) is a Ca(2+) storage protein that interacts with triadin (TRN), the ryanodine receptor (RyR), and junctin (JUN) to form a macromolecular tetrameric Ca(2+) signaling complex in the cardiac junctional sarcoplasmic reticulum (SR). Junctin, calsequestrin, triadin, and the ryanodine receptor form a quaternary complex that may be required for normal operation of Ca2+ release.\"], \"exact_answer\": [[\"Ryanodine receptor\", \"RyR\"], [\"Calsequestrin\", \"CASQ\", \"CSQ\"], [\"Triadin\", \"TrD\", \"TRN\"], [\"Junctin\", \"JCN\", \"JnC\", \"JUN\"]], \"context\": \"Calsequestrin (CSQ) is a Ca(2+) storage protein that interacts with triadin (TRN), the ryanodine receptor (RyR), and junctin (JUN) to form a macromolecular tetrameric Ca(2+) signaling complex in the cardiac junctional sarcoplasmic reticulum (SR).The decrease of CASQ2 is associated with a reduction in the levels of Triadin (TrD) and Junctin (JnC), two proteins that form, with CASQ2 and RyR2, a macromolecular complex devoted to control of calcium release from the sarcoplasmic reticulum.Triadin and junctin are integral sarcoplasmic reticulum membrane proteins that form a macromolecular complex with the skeletal muscle ryanodine receptor (RyR1) but their roles in skeletal muscle calcium homeostasis remain incompletely understood.Junctin, a 26 kDa intra-sarcoplasmic reticulum (SR) protein, forms a quaternary complex with triadin, calsequestrin and the ryanodine receptor (RyR) at the junctional SR membrane. In cardiac muscle, junctin forms a quaternary protein complex with the ryanodine receptor (RyR), calsequestrin, and triadin 1 at the luminal face of the junctional sarcoplasmic reticulum (jSR). By binding directly the RyR and calsequestrin, junctin may mediate the Ca(2+)-dependent regulatory interactions between both proteins.Calsequestrin, the major calcium sequestering protein in the sarcoplasmic reticulum of muscle, forms a quaternary complex with the ryanodine receptor calcium release channel and the intrinsic membrane proteins triadin and junctin. Junctin is a transmembrane protein of the cardiac junctional sarcoplasmic reticulum (SR) that binds to the ryanodine receptor, calsequestrin, and triadin 1. This quaternary protein complex is thought to facilitate SR Ca2+ release.In mammalian striated muscles, ryanodine receptor (RyR), triadin, junctin, and calsequestrin form a quaternary complex in the lumen of sarcoplasmic reticulum. Such intermolecular interactions contribute not only to the passive buffering of sarcoplasmic reticulum luminal Ca2+, but also to the active Ca2+ release process during excitation-contraction coupling. Junctin is a major transmembrane protein in cardiac junctional sarcoplasmic reticulum, which forms a quaternary complex with the ryanodine receptor (Ca(2+) release channel), triadin, and calsequestrin. Triadin 1 is a major transmembrane protein in cardiac junctional sarcoplasmic reticulum (SR), which forms a quaternary complex with the ryanodine receptor (Ca(2+) release channel), junctin, and calsequestrin.Several key proteins have been localized to junctional sarcoplasmic reticulum which are important for Ca2+ release. These include the ryanodine receptor, triadin, and calsequestrin, which may associate into a stable complex at the junctional membrane. We recently purified and cloned a fourth component of this complex, junctin, which exhibits homology with triadin and is the major 125I-calsequestrin-binding protein detected in cardiac sarcoplasmic reticulum vesiclesBy a combination of approaches including calsequestrin-affinity chromatography, filter overlay, immunoprecipitation assays, and fusion protein binding analyses, we find that junctin binds directly to calsequestrin, triadin, and the ryanodine receptor. Taken together, these results suggest that junctin, calsequestrin, triadin, and the ryanodine receptor form a quaternary complex that may be required for normal operation of Ca2+ release.Calsequestrin, the major calcium sequestering protein in the sarcoplasmic reticulum of muscle, forms a quaternary complex with the ryanodine receptor calcium release channel and the intrinsic membrane proteins triadin and junctin.Junctin, a 26 kDa intra-sarcoplasmic reticulum (SR) protein, forms a quaternary complex with triadin, calsequestrin and the ryanodine receptor (RyR) at the junctional SR membrane.Junctin (JCN), a 26-kd sarcoplasmic reticulum (SR) transmembrane protein, forms a quaternary protein complex with the ryanodine receptor, calsequestrin, and triadin in the SR lumen of cardiac muscle.Junctin (JCN), a 26-kd sarcoplasmic reticulum (SR) transmembrane protein, forms a quaternary protein complex with the ryanodine receptor, calsequestrin, and triadin in the SR lumen of cardiac muscleCalsequestrin, the major calcium sequestering protein in the sarcoplasmic reticulum of muscle, forms a quaternary complex with the ryanodine receptor calcium release channel and the intrinsic membrane proteins triadin and junctinJunctin is a major transmembrane protein in cardiac junctional sarcoplasmic reticulum, which forms a quaternary complex with the ryanodine receptor (Ca(2+) release channel), triadin, and calsequestrin.\"}, {\"question\": \"List the endoscopic diagnoses that have been reported in children with autism\", \"ideal_answer\": [\"Endoscopic examinations in autistic children have been reported to show : I or II reflux esophagitis,  Achalasia, chronic gastritis and chronic duodenitis,  mild acute and chronic inflammation of the small bowel and colorectum and Ileo-colonic lymphoid nodular hyperplasia (LNH).  \\nThe number of Paneth's cells in the duodenal crypts was found to be significantly elevated in autistic children compared with non-autistic control subjects. Low intestinal carbohydrate digestive enzyme activity was reported although there was no abnormality found in pancreatic function. Autistic children have ben reported to have an increased pancreatico-biliary fluid output after intravenous secretin administration.\"], \"exact_answer\": [[\"Reflux esophagitis\"], [\"Achalasia\"], [\"chronic gastritis\"], [\"chronic duodenitis\"], [\"inflammation of the small bowel and colorectum\"], [\"Ileo-colonic lymphoid nodular hyperplasia (LNH)\"]], \"context\": \"Autism and esophageal achalasia in childhood: a possible correlation?In the last case, a 15-year-old male underwent barium swallow and endoscopy that confirmed achalasia.Intestinal mucosal pathology, characterized by ileo-colonic lymphoid nodular hyperplasia (LNH) and mild acute and chronic inflammation of the colorectum, small bowel and stomach, has been reported in children with autistic spectrum disorder (ASD)Ileo-colonic LNH is a characteristic pathological finding in children with ASD and gastrointestinal symptoms, and is associated with mucosal inflammation.The data support the hypothesis that LNH is a significant pathological finding in ASD children.A relationship between autism and gastrointestinal (GI) immune dysregulation has been postulated based on incidence of GI complaints as well as macroscopically observed lymphonodular hyperplasia and microscopically determined enterocolitis in pediatric patients with autism.These data fail to support an association between autism and GI inflammation.Intestinal pathology, i.e., ileocolonic lymphoid nodular hyperplasia (LNH) and mucosal inflammation, has been described in children with developmental disorders.This study describes some of the endoscopic and pathological characteristics in a group of children with developmental disorders (affected children) that are associated with behavioral regression and bowel symptoms, and compares them with pediatric controls. METHODS: Ileocolonoscopy and biopsy were performed on 60 affected children (median age 6 yr, range 3-16; 53 male). Developmental diagnoses were autism (50 patients), Asperger's syndrome (five), disintegrative disorder (two), attention deficit hyperactivity disorder (ADHD) (one), schizophrenia (one), and dyslexia (one).Ileal LNH was present in 54 of 58 (93%) affected children and in five of 35 (14.3%) controls (p < 0.001). Colonic LNH was present in 18 of 60 (30%) affected children and in two of 37 (5.4%) controls (p < 0.01). Histologically, reactive follicular hyperplasia was present in 46 of 52 (88.5%) ileal biopsies from affected children and in four of 14 (29%) with UC, but not in non-IBD controls (p < 0.01). Active ileitis was present in four of 51 (8%) affected children but not in controls. Chronic colitis was identified in 53 of 60 (88%) affected children compared with one of 22 (4.5%) controls and in 20 of 20 (100%) with UCThirty-six children (age: 5.7 +/- 2 years, mean +/- SD) with autistic disorder underwent upper gastrointestinal endoscopy with biopsies, intestinal and pancreatic enzyme analyses, and bacterial and fungal culturesHistologic examination in these 36 children revealed grade I or II reflux esophagitis in 25 (69.4%), chronic gastritis in 15, and chronic duodenitis in 24. The number of Paneth's cells in the duodenal crypts was significantly elevated in autistic children compared with non-autistic control subjects. Low intestinal carbohydrate digestive enzyme activity was reported in 21 children (58.3%), although there was no abnormality found in pancreatic function. Seventy-five percent of the autistic children (27/36) had an increased pancreatico-biliary fluid output after intravenous secretin administration. Nineteen of the 21 patients with diarrhea had significantly higher fluid output than those without diarrhea.Unrecognized gastrointestinal disorders, especially reflux esophagitis and disaccharide malabsorption, may contribute to the behavioral problems of the non-verbal autistic patients. The observed increase in pancreatico-biliary secretion after secretin infusion suggests an upregulation of secretin receptors in the pancreas and liver.three children with autistic spectrum disorders who underwent upper gastrointestinal endoscopy and intravenous administration of secretin to stimulate pancreaticobiliary secretion. All three had an increased pancreaticobiliary secretory response when compared with nonautistic patients (\"}, {\"question\": \"What are the main indications of lacosamide?\", \"ideal_answer\": [\"Lacosamide is an anti-epileptic drug, licensed for refractory partial-onset seizures. In addition to this, it has demonstrated analgesic activity in various animal models. Apart from this, LCM has demonstrated potent effects in animal models for a variety of CNS disorders like schizophrenia and stress induced anxiety.\"], \"exact_answer\": [[\"epilepsy\", \"refractory epilepsy\", \"refractory partial-onset seizures\"], [\"analgesic\"], [\"CNS disorders\"]], \"context\": \"The current article presents a concise review of network theory and its application to the characterization of AED use in children with refractory epilepsy.Furthermore, first generation AEDs were often discontinued, while lacosamide and topiramate were most likely to be initiated. Overall, there is compelling evidence that salivary TDM can be usefully applied so as to optimize the treatment of epilepsy with carbamazepine, clobazam, ethosuximide, gabapentin, lacosamide, lamotrigine, levetiracetam, oxcarbazepine, phenobarbital, phenytoin, primidone, topiramate, and zonisamide.The first examples are carbamazepine, gabapentin, and lacosamide as drugs well established in epilepsy market as well as drug candidates such as valnoctamide, and other valproic acid derivatives, novel biphenyl pyrazole derivatives, etc.Two additional AEDs, lacosamide and eslicarbazepine acetate, have been licensed recently for a more traditional indication, refractory partial-onset seizures. A discussion is made of recent findings that the atypical antidepressant tianeptine increases CRMP2 expression, whereas other, neuroactive small molecules including the epilepsy drug lacosamide and the natural brain metabolite lanthionine ketimine appear to bind CRMP2 directly with concomitant affects on neural structure.Lacosamide (LCM) is a newer antiepileptic drug with a dual mode of action. It has shown potent and broad neuroprotective effects in vitro and in vivo animal models making it a potential candidate for long term treatment of epilepsy. In addition to this, it has demonstrated analgesic activity in various animal models. Apart from this, LCM has demonstrated potent effects in animal models for a variety of CNS disorders like schizophrenia and stress induced anxiety. Clinical trials have also suggested that LCM is a safe, effective, and well tolerated adjunctive treatment for reduction of seizure frequency in patients with highly refractory, partial seizures.Lacosamide (LCM), (SPM 927, (R)-2-acetamido-N-benzyl-3-methoxypropionamide, previously referred to as harkoseride or ADD 234037) is a member of a series of functionalized amino acids that were specifically synthesized as anticonvulsive drug candidates. LCM has demonstrated antiepileptic effectiveness in different rodent seizure models and antinociceptive potential in experimental animal models that reflect distinct types and symptoms of neuropathic as well as chronic inflammatory pain.Recent results suggest that LCM has a dual mode of action underlying its anticonvulsant and analgesic activity.Currently, LCM is in a late stage of clinical development as an adjunctive treatment for patients with uncontrolled partial-onset seizures, and it is being assessed as monotherapy in patients with painful diabetic neuropathy. Lacosamide was effective against sound-induced seizures in the genetically susceptible Frings mouse, against maximal electroshock test (MES)-induced seizures in rats and mice, in the rat hippocampal kindling model of partial seizures, and in the 6Hz model of psychomotor seizures in mice.Lacosamide was inactive against clonic seizures induced by subcutaneous administration of the chemoconvulsants pentylenetetrazol, bicuculline, and picrotoxin, but it did inhibit NMDA-induced seizures in mice and showed full efficacy in the homocysteine model of epilepsy.These results suggest that lacosamide may have the potential to be clinically useful for at least the treatment of generalized tonic-clonic and partial-onset epilepsies, and support ongoing clinical trials in these indications.\"}, {\"question\": \"List Hemolytic Uremic Syndrome Triad.\", \"ideal_answer\": [\"Hemolytic uremic syndrome (HUS) is a clinical syndrome characterized by the triad of anaemia, thrombocytopenia, renal failure.\"], \"exact_answer\": [[\"anaemia\"], [\"thrombocytopenia\"], [\"renal failure\"]], \"context\": \"Hemolytic uremic syndrome (HUS) is a clinical syndrome characterized by the triad of thrombotic microangiopathy, thrombocytopenia, and acute kidney injury.Atypical hemolytic uremic syndrome (aHUS) is a rare disease characterized by the triad of microangiopathic hemolytic anemia, thrombocytopenia, and acute renal failure.Atypical hemolytic uremic syndrome (aHUS) is a relatively rare disorder described by the triad of hemolytic anemia, thrombocytopenia, and renal failure.Until recently, atypical hemolytic uremic syndrome (aHUS), conventionally defined in the pediatric literature as a syndrome of the triad of renal failure, microangiopathic hemolytic anemia, and thrombocytopenia without a prodrome of hemorrhagic diarrhea, has received little attention in adult practice because the patients are commonly given the diagnosis of thrombotic thrombocytopenic purpura (TTP) or TTP/HUS and treated as TTP with plasma exchange, augmented in refractory cases with rituximab and sometimes even splenectomy. Atypical hemolytic uremic syndrome (aHUS) is rare and comprises the triad of microangiopathic hemolytic anemia, thrombocytopenia, and acute kidney injury. Hemolyitic uremic syndrome (HUS), characterized by triad of acute kidney injury, thrombocytopenia, and hemolytic anemia, has considerable morbidity and mortality and is known to be associated with diarrheal illness. Hemolytic uremic syndrome (HUS) is a rare thrombotic complication characterized by a triad of microangiopathic hemolytic anemia, thrombocytopenia, and acute renal failure.Until recently, atypical hemolytic uremic syndrome (aHUS), conventionally defined in the pediatric literature as a syndrome of the triad of renal failure, microangiopathic hemolytic anemia, and thrombocytopenia without a prodrome of hemorrhagic diarrhea, has received little attention in adult practice because the patients are commonly given the diagnosis of thrombotic thrombocytopenic purpura (TTP) or TTP/HUS and treated as TTP with plasma exchange, augmented in refractory cases with rituximab and sometimes even splenectomy.Hemolytic uremic syndrome consists of a triad of acquired hemolytic anemia, thrombocytopenia and renal failure.Hemolytic uremic syndrome (HUS) is a triad of microangiopathic hemolytic anemia, thrombocytopenia, and acute renal failure.The hemolytic-uremic syndrome consists of the triad of microangiopathic hemolytic anemia, thrombocytopenia, and renal failure.Atypical hemolytic uremic syndrome (aHUS) is characterized by the triad of microangiopathic hemolytic anemia, thrombocytopenia, and renal impairment.Hemolytic uremic syndrome is a rare condition compromising the clinical triad of acute renal failure, microangiopathic hemolytic anemia, and thrombocytopenia.Hemolytic Uremic Syndrome (HUS) consists ofa triad of acquired hemolytic anemia, thrombocytopenia, and renal failure that occurs acutely in otherwise healthy individuals.The hemolytic-uremic syndrome is a pathology characterized by a triad consisting of acute renal failure, microangiopathic hemolytic anemia and thrombocytopenia, with complications of the central nervous system arising in a considerable number of cases.Hemolytic uremic syndrome, one of the common causes of acute renal failure in children, is characterized by the triad of microangiopathy, haemolytic anemia, thrombocytopenia and acute renal failure.Hemolytic uremic syndrome (HUS) is a triad of microangiopathic hemolytic anemia, thrombocytopenia, and acute renal failureHemolytic uremic syndrome (HUS) is a rare thrombotic complication characterized by a triad of microangiopathic hemolytic anemia, thrombocytopenia, and acute renal failureHemolytic uremic syndrome (HUS) is a disorder characterized by the presence of the classic triad: microangiopathic hemolytic anemia, thrombocytopenia and acute renal injuryHemolytic uremic syndrome consists of a triad of acquired hemolytic anemia, thrombocytopenia and renal failureHemolytic uremic syndrome (HUS) is a severe disease characterized by the clinical triad of hemolytic anemia, thrombocytopenia, and acute renal failureHemolytic uremic syndrome is a rare condition compromising the clinical triad of acute renal failure, microangiopathic hemolytic anemia, and thrombocytopeniaHemolytic uremic syndrome is a triad of microangiopathic hemolytic anemia, thrombocytopenia, and acute renal failureHemolytic uremic syndrome is a rare entity in patients with carcinoma and presents with a triad of renal insufficiency, microangiopathic hemolytic anemia, and thrombocytopeniaHemolytic Uremic Syndrome (HUS) consists ofa triad of acquired hemolytic anemia, thrombocytopenia, and renal failure that occurs acutely in otherwise healthy individualsHemolytic Uremic Syndrome (HUS) consists ofa triad of acquired hemolytic anemia, thrombocytopenia, and renal failure that occurs acutely in otherwise healthy individuals. Acute renal insufficiency in the setting of hemolysis and thrombocytopenia, a triad that constitutes adult or pediatric hemolytic uremic syndrome, can be associated with or triggered by diverse conditions such as verocytotoxin-producing Escherichia coli, viral infections, pregnancy, malignant hypertension, scleroderma, renal radiation, allograft rejection, lupus erythematosus, and assorted medications such as mitomycin C, cyclosporine, and oral contraceptives.Hemolytic uremic syndrome, one of the common causes of acute renal failure in children, is characterized by the triad of microangiopathy, haemolytic anemia, thrombocytopenia and acute renal failure. The diarrhoea-associated Hemolytic uremic syndrome is usually termed as a typical Hemolytic uremic syndrome.\"}, {\"question\": \"What are the effects of depleting protein  km23-1 (DYNLRB1)  in a cell?\", \"ideal_answer\": [\"The knockdown of km23-1 results in numerous effects at the cellular level, such as decreased cell migration. Additionaly, km23-1 is involved in signalling pathways and its knockdown results in decreased RhoA activation, inhibition of TGF\\u03b2-mediated activation of ERK and JNK, phosphorylation of c-Jun, transactivation of the c-Jun promoter and decreased TGFbeta responses.\"], \"exact_answer\": [[\"inhibition of cell migration of human colon carcinoma cells (HCCCs) in wound-healing assays\"], [\"decreased RhoA activation\"], [\"inhibition of TGF\\u03b2-mediated activation of ERK and JNK\"], [\"phosphorylation of c-Jun\"], [\"transactivation of the c-Jun promoter\"], [\"decreased key TGFbeta responses\"]], \"context\": \"Finally, our results demonstrated for the first time that depletion of km23-1 inhibited cell migration of human colon carcinoma cells (HCCCs) in wound-healing assays. Overall, our findings demonstrate that km23-1 regulates RhoA and motility-associated actin modulating proteins, suggesting that km23-1 may represent a novel target for anti-metastatic therapy.Further, knockdown of km23-1 inhibited TGF\\u03b2-mediated activation of ERK and JNK, phosphorylation of c-Jun, and transactivation of the c-Jun promoter.knockdown (KD) of km23-1 decreased RhoA activation in Mv1Lu epithelial cellsWe have previously reported that the dynein light chain (DLC) km23-1 is required for Smad2-dependent TGFbeta signaling.Blockade of km23-1 using a small interfering RNA approach resulted in a reduction in both total intracellular Smad2 levels and in nuclear levels of phosphorylated Smad2 after TGFbeta treatment.Blockade of km23 using small interfering RNAs significantly decreased key TGFbeta responses, including induction of fibronectin expression and inhibition of cell growth.On the other hand, inhibiting the endogenous DYNLRB1 with gene-specific small interfering RNA or pharmacologically with a specific inhibitor (vanadate) led to a significant (P < 0.05) decrease in folate uptake.Functional studies showed that some mutations disrupt km23 function, resulting in aberrant transforming growth factor-beta signaling and presumably enhanced tumorigenicity.\"}, {\"question\": \"Which factors activate zygotic gene expression during the maternal-to-zygotic transition in zebrafish?\", \"ideal_answer\": [\"Nanog, Pou5f1 and SoxB1 activate zygotic gene expression during the maternal-to-zygotic transition. Maternal Nanog, Pou5f1 and SoxB1 are required to initiate the zygotic developmental program and induce clearance of the maternal program by activating miR-430 expression.\"], \"exact_answer\": [[\"Nanog\"], [\"Pou5f1\"], [\"SoxB1\"]], \"context\": \"Nanog, Pou5f1 and SoxB1 activate zygotic gene expression during the maternal-to-zygotic transition.Our results demonstrate that maternal Nanog, Pou5f1 and SoxB1 are required to initiate the zygotic developmental program and induce clearance of the maternal program by activating miR-430 expression.Here we show that Nanog, Pou5f1 (also called Oct4) and SoxB1 regulate zygotic gene activation in zebrafish.Here we show that Nanog, Pou5f1 (also called Oct4) and SoxB1 regulate zygotic gene activation in zebrafish. We identified several hundred genes directly activated by maternal factors, constituting the first wave of zygotic transcription.\"}, {\"question\": \"Which DNA sequences are more prone for the formation of R-loops?\", \"ideal_answer\": [\"R-loops, transcriptionally-induced RNA:DNA hybrids, occurring at repeat tracts (CTG)n, (CAG)n, (CGG)n, (CCG)n and (GAA)n, are associated with diseases including myotonic dystrophy, Huntington's disease, fragile X and Friedreich's ataxia.  Physiological R-loop formation at CpG island promoters can contribute to DNA replication origin specification at these regions, the most efficient replication initiation sites in mammalian cells. R-loops may also possess beneficial effects, as their widespread formation has been detected over CpG island promoters in human genes. R-loops are particularly enriched over G-rich terminator elements.\"], \"exact_answer\": [[\"G-rich elements\"], [\"CpG islands\"], [\"(CTG)n, (CAG)n, (CGG)n, (CCG)n and (GAA)n\"]], \"context\": \"R-loops, transcriptionally-induced RNA:DNA hybrids, occurring at repeat tracts (CTG)n, (CAG)n, (CGG)n, (CCG)n and (GAA)n, are associated with diseases including myotonic dystrophy, Huntington's disease, fragile X and Friedreich's ataxiaDouble-R-loop formation and processing to instability was extended to the expanded C9orf72 (GGGGCC)\\u00b7(GGCCCC) repeats, known to cause amyotrophic lateral sclerosis and frontotemporal dementia, providing the first suggestion through which these repeats may become unstable. Here we propose that physiological R-loop formation at CpG island promoters can contribute to DNA replication origin specification at these regions, the most efficient replication initiation sites in mammalian cells.R-loops may also possess beneficial effects, as their widespread formation has been detected over CpG island promoters in human genesFurthermore, we have previously shown that R-loops are particularly enriched over G-rich terminator elementsR-loops associated with triplet repeat expansionsPeriodic DNA patrolling underlies diverse functions of Pif1 on R-loops and G-rich DNAGinno et al. (2012) describe unusual sequence features at promoter CpG islands that can lead to formation of persistent RNA-DNA hybrids (R loops), which are proposed to prevent genomic DNA methylationR loops stimulate genetic instability of CTG.CAG repeats We demonstrate, using biochemical and genetic approaches, that the formation of stable RNA.DNA hybrids enhances the instability of CTG.CAG repeat tracts. The R-loop, previously characterized in vitro at the leading strand replication origin (OH), is isolated as a native RNA-DNA hybrid copurifying with mtDNAAll of these R-loops arise upon generation of a G-rich RNA strand by an RNA polymerase upon transcription of a C-rich DNA template strand.Finally, non-denaturing bisulfite mapping of the displaced single-stranded DNA confirmed R-loop formation at the endogenous FMR1 locus and further indicated that R-loops formed over CGG repeats may be prone to structural complexities, including hairpin formation, not commonly associated with other R-loopsWe have observed that transcription through the GC-rich FMR1 5&apos;UTR region favors R-loop formation, with the nascent (G-rich) RNA forming a stable RNA:DNA hybrid with the template DNA strand, thereby displacing the non-template DNA strandwhich is rich in AT and not prone to form R-loops,Finally, non-denaturing bisulfite mapping of the displaced single-stranded DNA confirmed R-loop formation at the endogenous FMR1 locus and further indicated that R-loops formed over CGG repeats may be prone to structural complexities, including hairpin formation, not commonly associated with other R-loops.\"}, {\"question\": \"which mutations of troponin C gene have been found to cause hypertrophic cardiomyopathy?\", \"ideal_answer\": [\"The following mutations of troponin C gene have been found to cause hypertrophic cardiomyopathy: L29Q; A8V; A31S; E134D; c.363dupG; A23Q; D145E and C84Y\"], \"exact_answer\": [[\"L29Q\"], [\"A8V\"], [\"A31S\"], [\"E134D\"], [\"c.363dupG\"], [\"A23Q\"], [\"D145E\"], [\"C84Y\"]], \"context\": \"The Ca(2+) binding properties of the FHC-associated cardiac troponin C (cTnC) mutation L29Q were examined in isolated cTnC, troponin complexes, reconstituted thin filament preparations, and skinned cardiomyocytes. Effects of calcium binding and the hypertrophic cardiomyopathy A8V mutation on the dynamic equilibrium between closed and open conformations of the regulatory N-domain of isolated cardiac troponin CWe also used PRE data to assess the structural effects of a familial hypertrophic cardiomyopathy point mutation located within the N-domain of cTnC (A8V). A mutation in TNNC1-encoded cardiac troponin C, TNNC1-A31S, predisposes to hypertrophic cardiomyopathy and ventricular fibrillation.Here, we report clinical and functional characterization of a novel TNNC1 mutation, A31S, identified in a pediatric HCM proband with multiple episodes of ventricular fibrillation and aborted sudden cardiac death.In summary, we report a novel mutation in the TNNC1 gene that is associated with HCM pathogenesis and may predispose to the pathogenesis of a fatal arrhythmogenic subtype of HCM.The objective of this work was to investigate the effect of hypertrophic cardiomyopathy-linked A8V and E134D mutations in cardiac troponin C (cTnC) on the response of reconstituted thin filaments to calcium upon phosphorylation of cardiac troponin I (cTnI) by protein kinase A.We describe a novel type of mutation (c.363dupG) in Troponin C, a rare form of hypertrophic cardiomyopathy.We report the first frameshift mutation (c.363dupG or p.Gln122AlafsX30) in Troponin C causing hypertrophic cardiomyopathy (and sudden cardiac death) in a 19-year-old male, and have demonstrated that the mutation segregates with hypertrophic cardiomyopathy within the family.One mutant (A23Q) was found with HCM-like properties (increased Ca(2+) sensitivity of force and normal levels of ATPase inhibition).In this study, we investigated the effects of hypertrophic cardiomyopathy-linked mutations A8V, E134D, and D145E in cardiac troponin C on the properties of the C-domain sites.Recently four new hypertrophic cardiomyopathy mutations in cardiac troponin C (cTnC) (A8V, C84Y, E134D, and D145E) were reported, and their effects on the Ca(2+) sensitivity of force development were evaluatedComprehensive open reading frame/splice site mutation analysis of TNNC1 performed on 1025 unrelated HCM patients enrolled over the last 10 years revealed novel missense mutations in TNNC1: A8V, C84Y, E134D, and D145E.The cardiac troponin C (cTnC) mutation, L29Q, has been found in a patient with familial hypertrophic cardiomyopathy.These results demonstrate that the L29Q mutation enhances the Ca(2+)-binding characteristics of cTnC and that when incorporated into cardiac myocytes, this mutant alters myocyte contractility.Cardiac troponin C-L29Q, related to hypertrophic cardiomyopathy, hinders the transduction of the protein kinase A dependent phosphorylation signal from cardiac troponin I to C.This mutation leads to a leucine-glutamine exchange at position 29 in the nonfunctional calcium-binding site of cTnC. \"}, {\"question\": \"Which post-translational histone modifications are characteristic of constitutive heterochromatin?\", \"ideal_answer\": [\"H3K9me3 is the major marker of constitutive heterochromatin. Other histone methylation marks usually found in constitutive heterochromatin, are H4K20me3 and H3K79me3. Classical histone modifications associated with heterochromatin include H3K9me2, H3K27me1 and H3K27me2. Histone H3 trimethylation at lysine 36 is associated with constitutive and facultative heterochromatin. H3S10 phosphorylation marks constitutive heterochromatin during interphase in early mouse embryos until the 4-cell stage\"], \"exact_answer\": [[\"H3S10p\"], [\"H3K36me3\"], [\"H4K20me3\"], [\"H3K9me3\"], [\"H3K79me3\"], [\"H3K9me2\"], [\"H3K27me1\"], [\"H3K27me2\"], [\"H3K27me3\"]], \"context\": \"H3S10 phosphorylation marks constitutive heterochromatin during interphase in early mouse embryos until the 4-cell stageWe show here that the maternally inherited Snurf-Snrpn 3-Mb region, which is silenced by a potent transcription repressive mechanism, is uniformly enriched in histone methylation marks usually found in constitutive heterochromatin, such as H4K20me3, H3K9me3, and H3K79me3.This result indicates that H3K36me3 is associated with both facultative and constitutive heterochromatin.Classical histone modifications associated with heterochromatin, including H3K9me2, H3K27me1 and H3K27me2, were distributed throughout both A and B chromosomes.In this review, available data will be evaluated concerning (1) the phylogenetic distribution of H3K9me as heterochromatin-specific histone modification and its evolutionary stability in relation to other epigenetic marks, (2) known families of H3K9 methyltransferases, (3) their responsibility for the formation of constitutive heterochromatin and (4) the evolution of Su(var)3-9-like and SUVH-like H3K9 methyltransferases While these regions were furthermore largely devoid of the constitutive heterochromatin marker H3K9-me3, we observed rapid and widespread deposition of H3K27-me3 across latent KSHV genomes, a bivalent modification which is able to repress transcription in spite of the simultaneous presence of activating marks.Histone modifications in Arabidopsis- high methylation of H3 lysine 9 is dispensable for constitutive heterochromatinThe recent discovery of the first histone Lys methyltransferase has allowed the identification of a molecular mechanism in which the specific methylation of histone H3 at Lys9 generates a binding site for heterochromatin-associated proteins.At the SUMO-1 labelled areas, the presence of HP1alpha protein, as well as of trimethylated H3-K9 and H4-K20 histone modifications, supports a role for SUMO-1 in constitutive heterochromatin organization.Surprisingly, the marsupial Xi was stably enriched for modifications associated with constitutive heterochromatin in all eukaryotes (H4K20me3, H3K9me3).Constitutive heterochromatin during mouse oogenesis: the pattern of histone H3 modifications and localization of HP1alpha and HP1beta proteins.In fission yeast, heterochromatin formation requires RNAi and the histone H3K9 methyltransferase complex CLRC, composed of Clr4, Raf1, Raf2, Cul4, and Rik1. Methylation of histone H3 at lysine 9 (H3-Lys9) by site-specific histone methyltransferases (Suv39h HMTases) marks constitutive heterochromatin. H3S10 phosphorylation marks constitutive heterochromatin during interphase in early mouse embryos until the 4-cell stage.In placental mammals non-coding XIST RNA triggers silencing of one X (Xi) and recruits a characteristic suite of epigenetic modifications, including the histone mark H3K27me3.\"}, {\"question\": \"Which interleukins are inhibited by Dupilumab?\", \"ideal_answer\": [\"Dupilumab, a fully human monoclonal antibody that blocks interleukin-4 and interleukin-13, has shown efficacy in patients with asthma and elevated eosinophil levels.\"], \"exact_answer\": [[\"interleukin-4\"], [\"interleukin-13\"]], \"context\": \"The world's first prospective controlled studies with the biologic human anti-IL4R antibody dupilumab for the indication \\\"atopic dermatitis\\\" were published in 2014. These motivated (1) to extend the studies to dupilumab and (2) to clinically test antagonization of other target molecules of TH2 polarized, atopic inflammation, e.g., IL-13, IL-31, IL-22, TSLP, and CRTH2. AAmong the recently developed antiasthma biologic drugs, the monoclonal antibody dupilumab is very promising because of its ability to inhibit the biological effects of both IL-4 and IL-13. Indeed, dupilumab prevents IL-4/13 interactions with the \\u03b1-subunit of the IL-4 receptor complex. BACKGROUND: Dupilumab, a fully human monoclonal antibody that blocks interleukin-4 and interleukin-13, has shown efficacy in patients with asthma and elevated eosinophil levels.  In early-phase trials, dupilumab, a fully human mAb targeting IL-4 receptor \\u03b1, markedly improved disease activity, but the effect of IL-4/IL-13 blockade on AD at the molecular level has not been characterized.Examples include the efficacy of omalizumab in patients with severe refractory atopic asthma characterized by raised serum total IgE, mepolizumab, reslizumab, and benralizumab in patients with recurrent eosinophilic exacerbations characterized by blood and sputum eosinophilia despite high doses of corticosteroids, and lebrikizumab, pitrakinra, dupilumab, and tralokinumab that target the IL-4/IL-13 signalling pathways in patients with eosinophilic asthma or raised serum periostin.BACKGROUND: Moderate-to-severe asthma remains poorly treated. We evaluated the efficacy and safety of dupilumab (SAR231893/REGN668), a fully human monoclonal antibody to the alpha subunit of the interleukin-4 receptor, in patients with persistent, moderate-to-severe asthma and elevated eosinophil levels.With respect to immune dysregulation, dupilumab, a fully human monoclonal antibody directed at the IL-4 receptor alpha subunit was recently shown to be effective in treating adults with moderate-to-severe AD.Dupilumab, a fully human monoclonal antibody that blocks interleukin-4 and interleukin-13, has shown efficacy in patients with asthma and elevated eosinophil levels.We evaluated the efficacy and safety of dupilumab (SAR231893/REGN668), a fully human monoclonal antibody to the alpha subunit of the interleukin-4 receptor, in patients with persistent, moderate-to-severe asthma and elevated eosinophil levels.Indeed, dupilumab prevents IL-4/13 interactions with the \\u03b1-subunit of the IL-4 receptor complex.We evaluated the efficacy and safety of dupilumab (SAR231893/REGN668), a fully human monoclonal antibody to the alpha subunit of the interleukin-4 receptor, in patients with persistent, moderate-to-severe asthma and elevated eosinophil levels. BACKGROUND: Dupilumab, a fully human monoclonal antibody that blocks interleukin-4 and interleukin-13, has shown efficacy in patients with asthma and elevated eosinophil levels. The blockade by dupilumab of these key drivers of type 2 helper T-cell (Th2)-mediated inflammation could help in the treatment of related diseases, including atopic dermatitis. Indeed, dupilumab prevents IL-4/13 interactions with the \\u03b1-subunit of the IL-4 receptor complex.We evaluated the efficacy and safety of dupilumab (SAR231893/REGN668), a fully human monoclonal antibody to the alpha subunit of the interleukin-4 receptor, in patients with persistent, moderate-to-severe asthma and elevated eosinophil levels.Dupilumab, a fully human monoclonal antibody that blocks interleukin-4 and interleukin-13, has shown efficacy in patients with asthma and elevated eosinophil levels.\"}, {\"question\": \"Which human genes are more commonly related to craniosynostosis?\", \"ideal_answer\": [\"The genes that are most commonly linked to craniosynostoses are the members of the Fibroblast Growth Factor Receptor family FGFR3 and to a lesser extent FGFR1 and FGFR2. Some variants of the disease have been associated with the triplication of the MSX2 gene and mutations in NELL-1. NELL-1 is being regulated bu RUNX2, which has also been associated to cases of craniosynostosis. Other genes reported to have a role in the development of the disease are RECQL4, TWIST, SOX6 and GNAS.\"], \"exact_answer\": [[\"FGFR3\"], [\"FGFR2\"], [\"FGFR1\"], [\"MSX2\"], [\"NELL1\"], [\"RUNX2\"], [\"RECQL4\"], [\"TWIST\"], [\"SOX6\"], [\"GNAS\"]], \"context\": \"The FGFR3 P250R mutation was the single largest contributor (24%) to the genetic groupFGFR3 P250R and FGFR2 exons IIIa/c) should be targeted to patients with coronal or multisuture synostosesGNAS, the gene for guanine nucleotide-binding protein, alpha-stimulating activity polypeptide (gene for PHP1A), identified a de novo heterozygous 3 bp in frame deletion predicting a deletion of the asparagine residue at position 377 (deltaN377craniosynostosis genes (FGFR2, FGFR3)Syndromic craniosynostosis due to complex chromosome 5 rearrangement and MSX2 gene triplicationearly fusion of cranial sutures commonly observed in the dup(5q) syndrome is caused by triplication of the MSX2 genefurther evidence that extra copy of MSX2 gene leads to craniosynostosisOur results support the previous finding that distal 5q-trisomy together with an extra copy of the MSX2 gene leads to abnormal closure of sutures and craniosynostosisCraniosynostosis-associated gene nell-1 is regulated by runx2We studied the transcriptional regulation of NELL-1, a craniosynostosis-related geneRunx2 directly binds to the OSE2 elements and transactivates the human NELL-1 promoter. These results suggest that Nell-1 is likely a downstream target of Runx2The breakpoint on chromosome 11p15 disrupts the SOX6 gene, known to be involved in skeletal growth and differentiation processesSOX6 mutation screening of another 104 craniosynostosis patients revealed one missense mutation leading to the exchange of a highly conserved amino acid (p.D68N) in a single patient and his reportedly healthy motherRevisiting the craniosynostosis-radial ray hypoplasia association: Baller-Gerold syndrome caused by mutations in the RECQL4 geneOverexpression of Nell-1, a craniosynostosis-associated geneMutations in five genes (FGFR1-, -2, -3, TWIST, and MSX2) causing craniosynostosis as the main clinical feature were described.One of the genes involved in craniosynostosis syndromes is the fibroblast growth factor receptor 2 (FGFR2) gene, a tyrosine kinase receptor geneMost mutations in Crouzon, Pfeiffer, and Apert syndromes are in the extracellular, third immunoglobulin-like domain and adjacent linker regions (exons IIIa and IIIc) of the fibroblast growth factor receptor 2 (FGFR2) geneFamilial craniosynostosis due to Pro250Arg mutation in the fibroblast growth factor receptor 3 gene.Apert (Ap) syndrome is characterized by premature cranial suture ossification caused by fibroblast growth factor receptor 2 (FGFR-2) mutationsRecently, the substitution of proline 250 by arginine in the fibroblast growth factor receptor 3 (FGFR3) gene, has been identified in patients with craniosynostosis and defines a new syndrome on a molecular basisMutations in the fibroblast growth factor receptor 1, 2 and 3 (FGFR1, -2 and -3) and TWIST genes have been identified in several syndromic forms of craniosynostosisWe describe a novel heterozygous mutation of FGFR2 (943G --> T, encoding the amino acid substitution Ala315Ser) in a girl with non-syndromic unicoronal craniosynostosis.A unique Pro250Arg mutation in fibroblast growth factor receptor 3 (FGFR3) was recently found in patients with non-syndromic craniosynostosisa possible mechanism for MSX2-mediated craniosynostosis in humansWe found previously that a single amino acid substitution in the homeodomain of the human MSX2 gene is associated with the autosomal dominant disorder craniosynostosis, Boston type.Recently, a unique Pro250Arg point mutation in fibroblast growth factor receptor 3 (FGFR3) was reported in 61 individuals with coronal craniosynostosis from 20 unrelated familiesWe identified a novel TWIST gene mutation in this patient, a Glu181Stop mutation predicting a premature termination of the protein carboxy-terminal to the helix 2 domainA unique point mutation in the fibroblast growth factor receptor 3 gene (FGFR3) defines a new craniosynostosis syndromeA mutation in the homeodomain of the human MSX2 gene in a family affected with autosomal dominant craniosynostosis\"}, {\"question\": \"List side effects of SGLT2 inhibitors?\", \"ideal_answer\": [\"SGLT2 inhibitors can be associated with urogenital infections related to the enhanced glycosuria, and low blood pressure.\"], \"exact_answer\": [[\"urinary tract infections\"], [\"genital infections\"], [\"low blood pressure\"]], \"context\": \"Due to side effects such as urinary tract and genital infections and decrease in blood pressure, proper patient selection for drug initiation and close monitoring will be important. Urogenital side-effects related to the enhanced glycosuria can be troublesome, yet seldom lead to discontinuation.Promotion of urinary loss of glucose by SGLT2 inhibitors embodies a new principle of control in type 2 diabetes that has several advantages with some urogenital side-effects, both of which are evaluated in this review. Promotion of urinary loss of glucose by SGLT2 inhibitors embodies a new principle of control in type 2 diabetes that has several advantages with some urogenital side-effects, both of which are evaluated in this reviewEffect of SGLT2 inhibitors in a murine model of urinary tract infection with Candida albicans.Promotion of urinary loss of glucose by SGLT2 inhibitors embodies a new principle of control in type 2 diabetes that has several advantages with some urogenital side-effects,Urogenital side-effects related to the enhanced glycosuria can be troublesome, yet seldom lead to discontinuation.Promotion of urinary loss of glucose by SGLT2 inhibitors embodies a new principle of control in type 2 diabetes that has several advantages with some urogenital side-effects, both of which are evaluated in this review.There are some side effects that warrant further investigation and establishing whether SGLT2 inhibition provides a renal benefit relies on future long-term studies with specific renal end-points.It should be noted that the currently approved agents have side effects that include an increased incidence of genital infections, predominantly in women.\"}, {\"question\": \"What is being measured with an accelerometer in back pain patients\", \"ideal_answer\": [\"Accelerometer assessment measuring overall physical activity (PAL), constant strain postures (CSP), standing time (ST) and lying time (LT)...\\nThe following parameters of physical activity were recorded: time upright (standing or walking), time standing, time walking, and step count.\"], \"exact_answer\": [[\"Physical activity\", \"PA\", \"PAL\"], [\"Constant Strain Postures\", \"CSP\", \"constant postures\"], [\"Standing time\", \"ST\"], [\"Lying time\", \"LT\"]], \"context\": \"accelerometer assessment measuring overall physical activity (PAL), constant strain postures (CSP), standing time (ST) and lying time (LT)Physical activity was measured for 7 days at both baseline and at 3 months with an RT3 accelerometerwearing an accelerometer to assess physical activity in daily lifeAn accelerometer was used to objectively assess their activity level objective activity data to determine whether patients with chronic lower back pain report their activity levels as accurately as controls do. DESIGN: A cross-sectional study was performed in patients and controls. SETTING: The study was carried out in the daily environment of the subjects. SUBJECTS: Thirty-two chronic lower back pain patients with symptoms more than three months and 20 healthy controls from the Netherlands, aged 18-65 years. MAIN MEASURES: A tri-axial accelerometer was worn for five weekdaysDuring 14days physical activity in daily life was measured, with both an electronic diary and an accelerometerphysical activity in daily life was measured with an accelerometerphysical activity (PA) in individuals with chronic low back pain (CLBP). Thirty-eight participants with non-specific CLBP (29=distressed; 9=non-distressed) were recruited. PA levels were measured using an accelerometer (activPAL activity monitor) over a one week period. The following parameters of physical activity were recorded: time upright (standing or walking), time standing, time walking, and step count.Physical activity levels will be measured by self report, RT3 triaxial accelerometer,to study the time spent in different static trunk postures which was recorded by a biaxial accelerometer attached to the T12 levelDaily activities were assessed by measuring body movement with a tri-axial accelerometernighttime activity data from 18 patients diagnosed with chronic back pain. The patients were followed for 6 days and 5 nights. Pain levels were collected every 90 min between 0800 hours and 2,200 hours using a computerized electronic diary. Activity levels were collected using a wrist accelerometer (Actiwatch AW-64). The Actiwatch sampled activity counts every 1 min. Patients were asked to wear the Actiwatch on their non-dominant arm. 8-h accelerometer assessment in their daily life (physical activity level (PAL), number of constant postures (CP)The activity levels were collected automatically using a wrist accelerometer and were sampled every minute.Physical activity in daily life, expressed as whole-body acceleration measured with a triaxial accelerometer (Tracmor),\"}, {\"question\": \"List the releases of JASPAR database\", \"ideal_answer\": [\"JASPAR, JASPAR CORE, JASPAR FAM, JASPAR phyloFACTS, JASPAR 2008 update, JASPAR 2010, JASPAR 2014.\"], \"exact_answer\": [[\"JASPAR\"], [\"JASPAR CORE\"], [\"JASPAR FAM\"], [\"JASPAR phyloFACTS\"], [\"JASPAR 2008 update\"], [\"JASPAR 2010\"], [\"JASPAR 2014\"]], \"context\": \"JASPAR is an open-access database of annotated, high-quality, matrix-based transcription factor binding site profiles for multicellular eukaryotes. The profiles were derived exclusively from sets of nucleotide sequences experimentally demonstrated to bind transcription factors. The database is complemented by a web interface for browsing, searching and subset selection, an online sequence analysis utility and a suite of programming tools for genome-wide and comparative genomic analysis of regulatory regions.JASPAR is the most complete open-access collection of transcription factor binding site (TFBS) matrices. In this new release, JASPAR grows into a meta-database of collections of TFBS models derived by diverse approaches. We present JASPAR CORE--an expanded version of the original, non-redundant collection of annotated, high-quality matrix-based transcription factor binding profiles, JASPAR FAM--a collection of familial TFBS models and JASPAR phyloFACTS--a set of matrices computationally derived from statistically overrepresented, evolutionarily conserved regulatory region motifs from mammalian genomes. JASPAR phyloFACTS serves as a non-redundant extension to JASPAR CORE, enhancing the overall breadth of JASPAR for promoter sequence analysisJASPAR 2010: the greatly expanded open-access database of transcription factor binding profiles.JASPAR 2014: an extensively expanded and updated open-access database of transcription factor binding profiles.The fifth major release greatly expands the heart of JASPAR-the JASPAR CORE subcollection, which contains curated, non-redundant profiles-with 135 new curated profiles (74 in vertebrates, 8 in Drosophila melanogaster, 10 in Caenorhabditis elegans and 43 in Arabidopsis thaliana; a 30% increase in total) and 43 older updated profiles (36 in vertebrates, 3 in D. melanogaster and 4 in A. thaliana; a 9% update in total). The new and updated profiles are mainly derived from published chromatin immunoprecipitation-seq experimental datasets. In addition, the web interface has been enhanced with advanced capabilities in browsing, searching and subsetting. Finally, the new JASPAR release is accompanied by a new BioPython package, a new R tool package and a new R/Bioconductor data package to facilitate access for both manual and automated methods.JASPAR, the open access database of transcription factor-binding profiles: new content and tools in the 2008 update.In this new release, JASPAR grows into a meta-database of collections of TFBS models derived by diverse approaches. We present JASPAR CORE--an expanded version of the original, non-redundant collection of annotated, high-quality matrix-based transcription factor binding profiles, JASPAR FAM--a collection of familial TFBS models and JASPAR phyloFACTS--a set of matrices computationally derived from statistically overrepresented, evolutionarily conserved regulatory region motifs from mammalian genomes.JASPAR phyloFACTS serves as a non-redundant extension to JASPAR CORE, enhancing the overall breadth of JASPAR for promoter sequence analysis. The new release of JASPAR is available at http://jaspar.genereg.net.  .JASPAR phyloFACTS serves as a non-redundant extension to JASPAR CORE, enhancing the overall breadth of JASPAR for promoter sequence analysis. The new release of JASPAR is available at http://jaspar.JASPAR phyloFACTS serves as a non-redundant extension to JASPAR CORE, enhancing the overall breadth of JASPAR for promoter sequence analysis. The new release of JASPAR is available at http://jaspar.genereg.net.\"}, {\"question\": \"List symptoms of the IFAP syndrome.\", \"ideal_answer\": [\"The IFAP syndrome is a rare X-linked genetic disorder characterized by the triad of follicular ichthyosis, atrichia, and photophobia.\"], \"exact_answer\": [[\"follicular ichthyosis\"], [\"atrichia\"], [\"photophobia\"]], \"context\": \"The IFAP syndrome is a rare X-linked genetic disorder characterized by the triad of follicular ichthyosis, atrichia, and photophobia.Mutations in MBTPS2 have been reported to cause a broad phenotypic spectrum of X-linked genodermatoses, including IFAP (ichthyosis follicularis; atrichia and photophobia) syndrome (OMIM 308205) with or without BRESHECK (brain anomalies, retardation of mentality and growth, ectodermal dysplasia, skeletal malformations, Hirschsprung disease, ear deformity and deafness, eye hypoplasia, cleft palate, cryptorchidism, and kidney dysplasia/hypoplasia) syndrome, keratosis follicularis spinulosa decalvans (KFSD; OMIM 308800) and an X-linked form of Olmsted syndrome. We report a recurrent intronic mutation in MBTPS2 (c.671-9T>G) in a Chinese patient with the typical triad of IFAP syndrome (i.e. ichthyosis, atrichia and photophobia), along with pachyonychia, palmoplantar and periorificial keratoderma, which were reminiscent of Olmsted syndrome. The ichthyosis follicular with atrichia and photophobia syndrome (IFAP) is a rare X-linked multiple congenital malformation syndrome.This patient presented with a severe IFAP/BRESHECK phenotype including ichthyosis follicular, atrichia, photophobia, brain anomalies, global developmental delay, Hirschsprung disease and kidney hypoplasia. The clinical findings include all the features that dermotrichic and ichthyosis follicularis-alopecia-photophobia (IFAP) syndrome have in common and in addition those that characterize IFAP syndrome (photophobia, recurrent respiratory infections, etc.Photoletter to the editor: A new variant of ichthyosis follicularis with alopecia and photophobia (IFAP) syndrome with coexisting psoriasiform lesions and palmoplantar keratoderma. IFAP-PPK syndrome?Ichthyosis follicularis, atrichia, and photophobia (IFAP) are typical features of a rare neuroichthyosis termed IFAP syndrome.Child with manifestations of dermotrichic syndrome and ichthyosis follicularis-alopecia-photophobia (IFAP) syndrome.These features correspond to the ichthyosis follicularis, alopecia, photophobia (IFAP) syndrome.Ichthyosis follicularis, atrichia, and photophobia (IFAP) syndrome is a rare congenital disorder.Ichthyosis follicularis with atrichia and photophobia (IFAP) syndrome in two unrelated female patients.Linear lesions reflecting lyonization in women heterozygous for IFAP syndrome (ichthyosis follicularis with atrichia and photophobia).Ichthyosis follicularis, alopecia, and photophobia (IFAP) syndrome due to mutation of the gene MBTPS2 in a large Australian kindred.Ichthyosis follicularis, alopecia and photophobia (IFAP) syndrome treated with acitretin.Atrichia, ichthyosis, follicular hyperkeratosis, chronic candidiasis, keratitis, seizures, mental retardation and inguinal hernia: a severe manifestation of IFAP syndrome?Ocular findings in ichthyosis follicularis-alopecia-photophobia (IFAP) syndrome.We describe an 18-month-old male infant suffering from the ichthyosis follicularis, atrichia, and photophobia (IFAP) syndrome and further delineate the clinical phenotype.The clinical findings include all the features that dermotrichic and ichthyosis follicularis-alopecia-photophobia (IFAP) syndrome have in common and in addition those that characterize IFAP syndrome (photophobia, recurrent respiratory infections, etc.), those that are present only in dermotrichic syndrome (nail anomalies, hypohydrosis, megacolon, vertebral defects, etc.) and additional ones (enamel dysplasia, renal anomalies, inguinal hernia, etc.).The clinical findings include all the features that dermotrichic and ichthyosis follicularis-alopecia-photophobia (IFAP) syndrome have in common and in addition those that characterize IFAP syndrome (photophobia,Ichthyosis follicularis, atrichia, and photophobia (IFAP) are typical features of a rare neuroichthyosis termed IFAP syndrome.These features correspond to the ichthyosis follicularis, alopecia, photophobia (IFAP) syndrome.Ichthyosis follicularis, atrichia, and photophobia (IFAP) syndrome is a rare congenital disorder.The ichthyosis follicular with atrichia and photophobia syndrome (IFAP) is a rare X-linked multiple congenital malformation syndrome.\"}, {\"question\": \"Which antibiotics target peptidoglycan biosynthesis?\", \"ideal_answer\": [\"Under some conditions, both ramoplanin and vancomycin probes produce helicoid staining patterns along the cylindrical walls of B. subtilis cells. This work has implications for the design of ramoplanin derivatives and may influence how other proposed substrate binding antibiotics are studied. This was confirmed by in vitro studies involving a wall-membrane particulate fraction from Gaffkya homari in which peptidoglycan synthesis from UDP-MurNAc-tetrapeptide was inhibited by ramoplanin but not by vancomycin. New results support a two-state model for septal and peripheral PG synthesis at mid-cell, involvement of essential cell division proteins in PG remodeling, and mid-cell localization of proteins that organize PG biosynthesis and that form the protein translocation apparatus.\", \"Antibiotics which inhibit bacterial peptidoglycan biosynthesis are the most widely used in current clinical practice. Cells treated with ampicillin, D-cycloserine, or fosfomycin had only one chloroplast after cell division, suggesting that the cells divided without chloroplast division. The antibiotics bacitracin and vancomycin showed no obvious effect. Colchicine inhibits Closterium cell elongation after division.  Surprisingly, also cinnamycin of Streptomyces cinnamoneus cinnamoneus), previously known to bind specifically to phosphatidylethanolamin of biological membranes, provoked strong cell wall biosynthetic stress. Other substances include fluorescent derivatives of two PG-binding antibiotics, vancomycin and ramoplanin. The muraymycins constitute a new antibiotic family whose core structure contains a glycosylated uronic acid derivative joined by an aminopropane group to a hexahydro-2-imino-4-pyrimidylglycyl residue (epicapreomycidine) containing dipeptide that is further extended by a urea-valine moiety. The muraymycins inhibited peptidoglycan biosynthesis. Ramoplanin is a cyclicdepsipeptide antibiotic that inhibits peptidoglycan biosynthesis. The lantibiotic mersacidin inhibits peptidoglycan biosynthesis at the level of transglycosylation. Ramoplanin, a new lipoglycopeptide antibiotic, inhibits cell wall peptidoglycan biosynthesis in gram-positive bacteria.\"], \"exact_answer\": [[\"colchicine\"], [\"fosfomycin\"], [\"bacitracin\"], [\"vancomycin\"], [\"D-cycloserine\", \"seromycin\"], [\"ampicillin\"], [\"cinnamycin\"], [\"ramoplanin\"], [\"muraymycin\"], [\"mersacidin\"]], \"context\": \"Treatment with antibiotics that interfere with peptidoglycan biosynthesis inhibits chloroplast division in the desmid ClosteriumTo detect cells just after division, we used colchicine, which inhibits Closterium cell elongation after divisionThe antibiotics bacitracin and vancomycin showed no obvious effect.cells treated with ampicillin, D-cycloserine, or fosfomycin had only one chloroplast after cell division, suggesting that the cells divided without chloroplast divisionWe investigated the effects of antibiotics that interfere with peptidoglycan biosynthesis on chloroplast division in the desmid Closterium peracerosum-strigosum-littorale complexRecent advances in pneumococcal peptidoglycan biosynthesis suggest new vaccine and antimicrobial targets.Of late, the peptidoglycan (PG) layer, the most important component of the bacterial cell wall has been the subject of drug targeting because, first, it is essential for the survivability of eubacteria and secondly, it is absent in humans.Antibiotics which inhibit bacterial peptidoglycan biosynthesis are the most widely used in current clinical practice.A dose-response experiment with an E. coli strain susceptible to ampicillin demonstrated a weak effect before the MIC dose. In an initial approach, the procedure accurately discriminates susceptible, intermediate and resistant strains of Escherichia coli to amoxicillin/clavulanic acid.Surprisingly, also cinnamycin of Streptomyces cinnamoneus cinnamoneus), previously known to bind specifically to phosphatidylethanolamin of biological membranes, provoked strong cell wall biosynthetic stress.Here, we compare the staining patterns observed in Bacillus subtilis using fluorescent derivatives of two PG-binding antibiotics, vancomycin and ramoplanin.Ramoplanin probes may be better imaging agents than vancomycin probes because they yield clear staining patterns at concentrations well below their minimum inhibitory concentrations.Structures of the muraymycins, novel peptidoglycan biosynthesis inhibitors.The muraymycins constitute a new antibiotic family whose core structure contains a glycosylated uronic acid derivative joined by an aminopropane group to a hexahydro-2-imino-4-pyrimidylglycyl residue (epicapreomycidine) containing dipeptide that is further extended by a urea-valine moiety.The muraymycins inhibited peptidoglycan biosynthesis. Rethinking ramoplanin: the role of substrate binding in inhibition of peptidoglycan biosynthesis.Ramoplanin is a cyclicdepsipeptide antibiotic that inhibits peptidoglycan biosynthesis.Vancomycin binds to bacterial cell-wall intermediates to achieve its antibiotic effect.The lantibiotic mersacidin inhibits peptidoglycan biosynthesis at the level of transglycosylation.The lantibiotic mersacidin has been previously reported to interfere with bacterial peptidoglycan biosynthesisHere, we focus on the target reaction and describe a mersacidin-induced accumulation of UDP-N-acetylmuramoyl-pentapeptide, indicating that inhibition of peptidoglycan synthesis occurs after the formation of cytoplasmic precursorsThe analogy to the glycopeptides may hint at an interaction of mersacidin with the peptidoglycan precursor rather than with the enzyme. Unlike vancomycin however, mersacidin inhibits peptidoglycan formation from UDP-N-acetylmuramoyl-tripeptide and is active against Enterococcus faecium expressing the vanA resistance gene cluster.Inhibition of peptidoglycan biosynthesis in vancomycin-susceptible and -resistant bacteria by a semisynthetic glycopeptide antibiotic.LY191145 is a p-chlorobenzyl derivative of LY264826 (A82846B) with activity against both vancomycin-susceptible and -resistant enterococci. Incorporation of L-[14C]lysine into peptidoglycan of intact vancomycin-susceptible and -resistant Enterococcus faecium was inhibited by LY191145 (50% inhibitory concentrations of 1 and 5 microgram/ml, respectively). Inhibition was accompanied by accumulation of UDP-muramyl-peptide precursors in the cytoplasmThe fact that inhibition of peptidoglycan biosynthesis by LY191145 was not readily antagonized by an excess of free acyl-D-alanyl-D-alanine or acyl-D-alanyl-D-lactate ligands indicates that the manner in which this compound inhibits transglycosylation may not be identical to that of vancomycin.Comparison with tunicamycin-treated cells indicated that peptidoglycan rather than teichoic acid metabolism is primarily affectedMersacidin caused the excretion of a putative cell wall precursor into the culture supernatantMode of action of the lantibiotic mersacidin: inhibition of peptidoglycan biosynthesis via a novel mechanism?Mersacidin is an antibiotic peptide produced by Bacillus sp. strain HIL Y-85,54728 that belongs to the group of lantibiotics. Its activity in vivo against methicillin-resistant Staphylococcus aureus strains compares with that of the glycopeptide antibiotic vancomycinIncubation of Staphylococcus simulans 22 with mersacidin resulted in the cessation of growth and slow lysis.n contrast to vancomycin, the activity of mersacidin was not antagonized by the tripeptide diacetyl-L-Lys-D-Ala-D-Ala, indicating that on the molecular level its mode of action differs from those of glycopeptide antibiotics.Inhibition of peptidoglycan biosynthesis by ramoplanin.Ramoplanin, a new lipoglycopeptide antibiotic, inhibits cell wall peptidoglycan biosynthesis in gram-positive bacteria.Bacitracin and other antibiotics that inhibit late stages in peptidoglycan biosynthesis induce vancomycin resistance in a high-level, inducibly vancomycin-resistant strain of Enterococcus faecium.This effect does not involve a change in the permeability of the cell wall by this drug and is consistent with the identification of D-alanine racemase as a target of D-cycloserine. CONCLUSIONS: Several pathways and genes downregulated by fosfomycin have been identified, in contrast to previously described cell wall active antibiotics, and was explained by starvation response induced by phosphoenolpyruvate accumulation. The target pathway - peptidoglycan biosynthesis - was upregulated following fosfomycin treatment. Ramoplanin, a new lipoglycopeptide antibiotic, inhibits cell wall peptidoglycan biosynthesis in gram-positive bacteria. Fosfomycin inhibited the first enzymatic step of peptidoglycan synthesis, which was followed by decreased levels of peptidoglycan precursors but enhanced levels of substrates such as UDP-GlcNAc and alanine-alanine.In contrast, vancomycin and ampicillin inhibited the last stage of peptidoglycan construction on the outer cell surface.The target pathway - peptidoglycan biosynthesis - was upregulated following fosfomycin treatment. Modulation of transport processes, cofactor biosynthesis, energy metabolism and nucleic acid biosynthesis was also observed.Ramoplanin, a new lipoglycopeptide antibiotic, inhibits cell wall peptidoglycan biosynthesis in gram-positive bacteria.\"}, {\"question\": \"List two common features of Tay syndrome.\", \"ideal_answer\": [\"Tay syndrome is a rare autosomal recessive genetic disorder characterized by congenital ichthyosis and trichothiodystrophy (abnormal brittle hair). Other less common features of this syndrome are photosensitivity, low birth weight, short stature, mental retardation, delayed neuromuscular development and other CNS anomalies, dysplasia of nails, hypoplasia of subcutaneous fatty tissue, prematurely aged facial appearance, hypogonadism, cataracts, osteosclerosis, dysphonia, and increased susceptibility to infections.\"], \"exact_answer\": [[\"ichthyosis\"], [\"trichothiodystrophy\"]], \"context\": \"TTD is part of a more broadly defined group of diseases identified as IBIDS (ichthyosis, brittle hair, impaired intelligence, decreased fertility and short stature). Photosensitive cases are also identified as PIBIDS (photosensitivity with IBIDS). Cases without manifest ichthyosis are also identified as PBIDS. These syndromes defy rigorous definition because of clinical variation between patients. The original two cases were described by Tay in oriental siblings, whose parents were first cousins; thus the disease is also known as Tay syndrome. Tay syndrome or IBIDS is a rare autosomal recessive genetic disorder characterized by congenital ichthyosis and abnormal brittle hair (trichothiodystrophy). Other features include photosensitivity, abnormal nails and multiple developmental defects affecting organs mainly derived from neuroectoderm. We report a case of trichothiodystrophy initially classified as Tay syndrome that based on clinical features, complementary exams as well as on the disease evolution was labelled as PIBIDS syndrome.Tay-syndrome is a rare monogen-inherited ektodermal dysplastic syndrome with ichtyosis, fragility of the hair and physical and mental retardation. The congenital ichtyosis is ubiquitous. Only the skin on the flexion side of the extremity joints are not involved (orthocerathosis combined with paraceratotic strings).In Tay syndrome, the trichothiodystrophy is accompanied by congenital ichthyosis, short stature, delayed physical and mental development and pyramidal tract signs with increase in muscular tone and brisk tendon reflexes. We present a case of Tay syndrome in which a cranial MRI revealed an almost total lack of myelin within the cerebral hemispheres and a patchy hypomyelination of the cerebellum. In accordance, a strongly prolonged visual evoked response pointed to a dysfunction of the white matter in Tay syndrome.For example, the brittle hair due to sulphur deficiency (trichothiodystrophy) is nowadays regarded as genetically heterogeneous; three different syndromes can be distinguished: BIDS syndrome, Tay syndrome, and PIBIDS syndrome. The Tay syndrome (congenital ichthyosis with trichothiodystrophy).The Tay syndrome is a distinct type of congenital ichthyosis characterized by a peculiar anomaly of hair growth which has been termed trichothiodystrophy. Other features of this syndrome are low birth weight, short stature, mental retardation, delayed neuromuscular development and other CNS anomalies, dysplasia of nails, hypoplasia of subcutaneous fatty tissue, prematurely aged facial appearance, hypogonadism, cataracts, osteosclerosis, dysphonia, and increased susceptibility to infections. \"}, {\"question\": \"Which cell types are known to be driving Rheumatoid Arthritis?\", \"ideal_answer\": [\"Macrophages, T cells and their respective cytokines play a pivotal role in RA. Rheumatoid arthritis synovial fibroblasts (RASFs) constitute a quite unique cell type that distinguishes RA from other inflammatory conditions of the joints. Activated synovial fibroblasts (SFs) have the ability to invade joint cartilage, actively contributing to joint destruction in RA.\"], \"exact_answer\": [[\"Macrophages\"], [\"T-cells\"], [\"Fibroblast-like synoviocytes (FLS)\"], [\"Dendritic cells\"], [\"synovial fibroblasts\"]], \"context\": \"Integration of GWAS results with cell-type specific gene expression or epigenetic marks have highlighted regulatory T cells and CD4 memory T cells as critical cell types in RAA subset of synovial DCs is important in the response to cigarette smoke we demonstrate that autoimmune patient (systemic lupus erythematosus and rheumatoid arthritis) serum activates both pDC and B cells, but IRAK1/4 kinase inhibition affects only the pDC response, underscoring the differential IRAK1/4 functional requirements in human immune cellIn rheumatoid arthritis (RA), activated synovial fibroblasts (SFs) have the ability to invade joint cartilage, actively contributing to joint destruction in RARecently, RASFs have been shown to be able to migrate to non-affected areas and joints through the blood stream and to invade distant cartilageMSC interaction with B cells provides stimuli for B-cell survival and therefore may contribute to the pathogenesis of rheumatoid arthritisFibroblast-like synoviocytes (FLS) are resident mesenchymal cells of synovial joints that have been recognized to play an increasingly important role in the pathogenesis of rheumatoid arthritis (RA)This study investigates the production of CCL18 in polymorphonuclear neutrophils (PMN), the predominant cell type recruited into synovial fluid (SF)In this regard, macrophages, T cells and their respective cytokines play a pivotal role in RAit has been understood that resident, fibroblast-like cells contribute significantly to the perpetuation of disease, and that they may even play a role in its initiationRASFs are no longer considered passive bystanders but active players in the complex intercellular network of RAThese rheumatoid arthritis synovial fibroblasts (RASFs) constitute a quite unique cell type that distinguishes RA from other inflammatory conditions of the jointsThe molecular feature that defines the myofibroblast-like phenotype was reflected as an increased proportion of myofibroblast-like cells in the heterogeneous FLS populationur findings support the notion that heterogeneity between synovial tissues is reflected in FLS as a stable trait, and provide evidence of a possible link between the behavior of FLS and the inflammation status of RA synoviumThis change was accompanied by a significant decrease in the synovial monocyte/macrophage populationIn RA patients, both etanercept and infliximab are able to induce cell type-specific apoptosis in the monocyte/macrophage populationFurthermore, fluorescent double-staining showed that the HOXD9 protein was expressed in fibroblast-like synoviocytes (FLS)AHR gene expression was demonstrated in rheumatoid synovial tissues and nodules with significantly greater expression in synovia.Twenty synovial and eighteen subcutaneous nodule tissue samples from 31 patients with RA were studied. Patient smoking status at the time of tissue collection was establishedRelease of the chemokine CCL18 has been widely attributed to antigen-presenting cells, including macrophages and dendritic cells\"}, {\"question\": \"Which drugs are utilized to treat amiodarone-induced thyroitoxicosis?\", \"ideal_answer\": [\"Amiodarone-induced thyrotoxicosis treatment includes anti-thyroid drugs and steroid therapy\\nRadio Iodine Treatment (RIT) may be a safe and useful method of AIT therapy in patients with low RAIU, in whom other treatment methods are contraindicated.\\nLithium is a useful and safe medication for treatment of iodine-induced thyrotoxicosis caused by amiodarone.\\nThyrodectomy may be necessary in presence of unresponsiveness to standard medical treatments\"], \"exact_answer\": [[\"Antithyroid drugs\"], [\"Corticosteroids\"], [\"Lithium\"], [\"Radioiodine\"]], \"context\": \"good response to anti-thyroid drugs and steroid therapyThe disease course of amiodarone-induced thyrotoxicosis is usually benign and remits with timely administration of anti-thyroid medications, with or without corticosteroids.RIT may be a safe and useful method of AIT therapy in patients with low RAIU, in whom other treatment methods are contraindicated.Treatment consists in the use of a high dose of anti-thyroid drugs and steroids in an isolated form or in combination.the addition of lithium carbonate to the two other drugs resulted in a successful and safety therapy in controlling amiodarone-induced thyrotoxicosis.lithium is a useful and safe medication for treatment of iodine-induced thyrotoxicosis caused by amiodarone.Two patients with amiodarone-induced thyrotoxicosis were treated successfully with potassium perchlorate and carbimazole while treatment with amiodarone was continued.Amiodarone-induced thyrotoxicosis seems to be a transient condition that can be treated successfully with a short course of antithyroid drugs without stopping amiodarone treatment.Both patients were successfully treated with propylthiouracil (PTU) and dexamethasone (DXT).\"}, {\"question\": \"Which are the genes thought to be regulated by EWS/FLI?\", \"ideal_answer\": [\"The EWS/FLI translocation product is the causative oncogene in Ewing sarcoma and acts as an aberrant transcription factor. EWS/FLI dysregulates gene expression during tumorigenesis by abnormally activating or repressing genes. The expression levels of a significant number of genes are affected in Ewing sarcoma, some of which are known to be directly or indirectly regulated by EWS/FLI. Such genes are BCL11B, NRoB1, GSTM4, NKX2.2 and p53.\"], \"exact_answer\": [[\"BCL11B\"], [\"NRoB1\"], [\"GSTM4\"], [\"NKX2.2\"], [\"p53\"]], \"context\": \"BCL11B is up-regulated by EWS/FLI and contributes to the transformed phenotype in Ewing sarcomaEWS/FLI dysregulates gene expression during tumorigenesis by abnormally activating or repressing genes. The expression levels of thousands of genes are affected in Ewing sarcoma, however, it is unknown which of these genes contribute to the transformed phenotype. Here we characterize BCL11B as an up-regulated EWS/FLI target that is necessary for the maintenance of transformation in patient derived Ewing sarcoma cells linesBCL11B, a zinc finger transcription factor, acts as a transcriptional repressor in Ewing's sarcoma and contributes to the EWS/FLI repressed gene signatureEWS/FLI functions as an aberrant transcription factor to regulate genes that mediate the oncogenic phenotype of Ewing's sarcoma. One of these regulated genes, NR0B1, encodes a corepressor protein, and likely plays a transcriptional role in tumorigenesisGSTM4 is a microsatellite-containing EWS/FLI target involved in Ewing's sarcoma oncogenesis and therapeutic resistanceWe recently showed that EWS/FLI regulates one of its key targets, NR0B1, through a GGAA-microsatellite in its promoterWe found that EWS/FLI directly binds the GSTM4 promoter, and regulates GSTM4 expression through a GGAA-microsatellite in its promoterThe gene that was most reproducibly up-regulated by EWS/FLI was NR0B1We validated NR0B1 as an EWS/FLI-dysregulated gene and confirmed its expression in primary human tumor samplesExpression profiling of EWS/FLI identifies NKX2.2 as a critical target gene in Ewing's sarcomaFunctional analysis revealed that NKX2.2 is an EWS/FLI-regulated gene that is necessary for oncogenic transformation in this tumorNeuroblastoma-Ews/Fli-1 infectant cell lines showed marked increases in p53 protein expression without transcriptional up-regulationWe then tested the p53 response pathway and observed that the neuroblastoma parent cells responded to genotoxic stress, whereas the neuroblastoma-Ews/Fli-1 infectants did not. These results suggest that Ews/Fli-1 can directly abrogate the p53 pathway to promote tumorigenesisWe recently showed that EWS/FLI interacts with GGAA-microsatellites to regulate some of its target genes, including NR0B1, an EWS/FLI-regulated gene that is required for the oncogenic phenotype of Ewings sarcoma.BCL11B is up-regulated by EWS/FLI and contributes to the transformed phenotype in Ewing sarcoma.The gene that was most reproducibly up-regulated by EWS/FLI was NR0B1.Here we characterize BCL11B as an up-regulated EWS/FLI target that is necessary for the maintenance of transformation in patient derived Ewing sarcoma cells lines\"}, {\"question\": \"Which pathological conditions are caused by mutations in the CYLD gene?\", \"ideal_answer\": [\"Since loss of CYLD expression can be observed in different types of human cancer, it is now well established that CYLD acts as a tumor suppressor gene. Pathogenic mutations in CYLD can be identified in patients affected with Brooke-Spiegler syndrome, (Familial) Cylindromatosis or multiple familial trichoepithelioma. CYLD expression has also been reported to be  dramatically downregulated in basal cell carcinoma (BCC), the most common cancer in humans.\"], \"exact_answer\": [[\"Brooke-Spiegler syndrome\", \"BSS\"], [\"Cylindromatosis\"], [\"Multiple Familial Trichoepithelioma\", \"MFT\"]], \"context\": \"We detected a frameshift mutation in the CYLD gene, designated 2253delG, underlying the disorder and were able to show that a single mutation can result in distinct clinical and histologic expression in familial cylindromatosis.Previously, a candidate MTF locus has been mapped to 9p21 while disease gene for familial cylindromatosis, the CYLD gene located on 16q21-13 has been identifiedHere, we show that mutations in the CYLD gene are also the genetic basis for three different Chinese families with MFTfamilial trichoepithelioma (MFT) and familial cylindromatosis are two clinically distinct cancer syndromes. MFT patients developed mostly trichoepithelioma in the face while cylindromatosis patients developed cylindromas predominantly (approximately 90%) on the head and neckphenotype of familial cylindromatosis associated with an R758X nonsense mutation in the CYLD tumour suppressor geneWe describe a family with cylindromatosis, in which affected individuals have an inherited R758X nonsense mutation of CYLDYLD mutations underlie Brooke-Spiegler, familial cylindromatosis, and multiple familial trichoepithelioma syndromesWe describe a single family with affected members exhibiting either the FC or the MFT phenotypes associated with a mutation in the CYLD gene. ndividuals with germline mutations in the tumour-suppressor gene CYLD are at high risk of developing disfiguring cutaneous appendageal tumours, the defining tumour being the highly organised cylindromaYLD is a tumor-suppressor gene mutated in the skin appendage tumors cylindromas, trichoepitheliomas, and spiradenomas. In addition, we have characterized Maspin downregulation in cylindromas, trichoepitheliomas, and spiradenomas carrying functional inactivating mutations of CYLD, also providing an evidence of the correlation between impaired CYLD function and Maspin decreased expression in vivo in human tumorsGerm\\u2011line mutation in cylindromatosis (CYLD), a tumor suppressor gene, causes familial cylindromatosis and Brook\\u2011Spiegler syndromeatients carrying heterozygous germline truncating mutations in the CYLD gene develop multiple primary hair follicle-related tumours. A highly patterned tumour, termed cylindroma, and a highly disorganized tumour, termed spiradenoma, may both develop in the same patient.ultiple trichoepitheliomas associated with a novel heterozygous mutation in the CYLD gene as an adjunct to the histopathological diagnosisultiple trichoepitheliomas (TEs), especially in the familial setting, have been associated with germline heterozygous mutations in the CYLD geneWe describe a 35-year-old woman with multiple facial TEs, in whom the molecular genetic analysis revealed a novel heterozygous c.1843delT mutation in the CYLD geneThis frameshift mutation was also present in a heterozygous state in the TE tumor cells. The demonstration of a novel CYLD mutation was used as an adjunct to the histopathological diagnosis in this casenovel germline mutation in the CYLD gene in a Slovak patient with Brooke-Spiegler syndromeThis new germline mutation in the CYLD gene of a Slovak patient with Brooke-Spiegler syndrome extends the catalogue of known CYLD germline mutations in this conditionMolecular biologic study of the CYLD gene performed from the peripheral blood identified a novel splice site c.2041+1 G>T mutationPatients with Brooke-Spiegler syndrome have various mutations in the CYLD gene, a tumor-suppressor gene located on chromosome 16qMutation and loss of heterozygosity of the CYLD gene causes development of cylindromatosis, a benign tumour originating from the skinOur study shows that CYLD expression is dramatically downregulated in basal cell carcinoma (BCC), the most common cancer in humansAlthough some early studies suggested a role for the PTCH gene on chromosome 9q22.3 in the etiopathogenesis of MFT, recent studies of occasional patients with the MFT clinical phenotype identified mutations in the CYLD gene on chromosome 16q12-q13, a gene responsible for BSSSince loss of CYLD expression can be observed in different types of human cancer, it is now well established that CYLD acts as a tumor suppressor genemutations in CYLD can be identified in patients affected with Brooke-Spiegler syndrome, (Familial) Cylindromatosis or multiple familial trichoepitheliomawe demonstrate that the expression in tumorigenic epidermal cells of a catalytically inactive form of CYLD (CYLD(C/S)) that mimics the identified mutations of cyld in human tumors and competes with the endogenous CYLD results in enhanced cell proliferation and inhibition of apoptosiIn contrast with previous studies showing the development of benign tumors by mutations in the CYLD gene, here we provide evidence that the occurrence of mutations in the CYLD gene in tumorigenic epidermal cells (carrying previous mutations) increases the aggressiveness of carcinomas, mainly through enhancement of the expression of angiogenic factors, having therefore a key role in epidermal cancer malignancyBy interphase cytogenetics, a (sub)clonal biallelic CYLD deletion was observed by interphase cytogenetics in 1 of 29 primary cHL, whereas signal patterns indicating decreased CYLD copy numbers were observed in a total of 10 of 29 primary casesmutation in the CYLD gene within a family with Brooke-Spiegler syndromerooke-Spiegler syndrome is a rare, autosomal dominant disease characterized by multiple skin appendage tumors caused by various mutations in the CYLD gene on chromosome 16q12-q13authors report a case of Brooke-Spiegler syndrome (BSS) with a novel germline CYLD mutation and various somatic mutations identified in the lesional tissueshe available histopathological material included 24 trichoepitheliomas, 2 large nodular basal cell carcinomas (BCCs), 2 spiradenomas, 1 spiradenocylindroma and 1 trichoblastoma composed of large and small nodules with prominent clear cell differentiationTo comprehensively ascertain the extent and severity of clinical features in affected individuals from 2 large families with proven heterozygous mutations in the CYLD locus and to correlate these findings with the 3 appendageal tumor predisposition syndromes (familial cylindromatosis, Brooke-Spiegler syndrome, and multiple familial trichoepitheliomas) known to be associated with such germline mutationsIn this report, we identified a novel mutation of CYLD gene in a Chinese family with MFTcase of Brooke-Spiegler syndrome with a novel germline deep intronic mutation in the CYLD gene leading to intronic exonization, diverse somatic mutations, and unusual histologypresent a case of Brooke-Spiegler syndrome with a germline deep intronic mutation in the CYLD gene leading to intronic exonizationA deep intronic mutation resulting in exonization and a somatic sequence mutations causing exon skipping are hitherto unreported genetic mechanisms involving the CYLD gene in patients with Brooke-Spiegler syndromenovel splicing mutation of the CYLD gene in a Taiwanese family with multiple familial trichoepitheliomaMutations in the CYLD gene, which is also the gene responsible for familial cylindromatosis, have been reported recentlyWe report a novel splicing mutation (IVS12 + 1 G-->A) in the CYLD gene in a Taiwanese pedigree with MFT, and discuss new developments in treatment optionsnew CYLD mutations in skin appendage tumors and evidence that aspartic acid 681 in CYLD is essential for deubiquitinase activityrooke-Spiegler syndrome, familial cylindromatosis, and familial trichoepithelioma are autosomal-dominant genetic predispositions for benign tumors of skin appendages caused by mutations in the CYLD gene localized on chromosome 16q12-q13The disease gene was mapped to 16q12-13, and mutations in the CYLD gene were identified in families with BSSBy sequence analysis, we identified a recurrent mutation 2272C>T (R758X) of the CYLD gene in the affected individuals of this family, which was previously identified in other ethnic families with familial cylindromatosis\"}, {\"question\": \"Which are the common symptoms of Cushing's syndrome?\", \"ideal_answer\": [\"Cushing syndrome is the constellation of signs and symptoms caused by protracted exposure to glucocorticoids. Presenting features commonly include weight gain, growth retardation, hirsutism, obesity, striae, acne and hypertension.\"], \"exact_answer\": [[\"weight gain\"], [\"growth retardation\"], [\"hirsutism\"], [\"obesity\"], [\"striae\"], [\"acne\"], [\"hypertension\"]], \"context\": \"Cushing syndrome is the constellation of signs and symptoms caused by protracted exposure to glucocorticoids. The most common cause of Cushing syndrome in children and adolescents is exogenous administration of glucocorticoids. Presenting features commonly include weight gain, growth retardation, hirsutism, obesity, striae, acne and hypertensionThis is a rare syndrome and its main endocrine manifestation, primary pigmented nodular adrenal disease (PPNAD), is an uncommon cause of adrenocorticotropic hormone-independent Cushing's syndrome.We report the case of a 20-year-old patient with a history of weight gain, hirsutism, acne, secondary amenorrhea and facial lentiginosis.\"}, {\"question\": \"List genes that have been found mutated in CMT1A (Charcot-Marie-Tooth disease type 1 A).\", \"ideal_answer\": [\"PMP22 is the common gene found mutated through a duplication in CMT1A. Other genes are\\nMPZ and SH3TC2\"], \"exact_answer\": [[\"PMP22\"], [\"MPZ\"], [\"SH3TC2\"]], \"context\": \"Most cases of CMT are caused by mutations in PMP22, structural myelin protein PMP22Duplication of the gene encoding the peripheral myelin protein of 22 kDa (PMP22) underlies the most common inherited neuropathy, Charcot-Marie-Tooth 1A (CMT1A), a disease without a known cure.. Two patients showed rearrangements in the PMP22 gene, which is commonly associated with CMT1Many PMP22 mutants accumulate in excess in the endoplasmic reticulum (ER) and lead to the inherited neuropathies of Charcot-Marie-Tooth (CMT) disease. The T118M mutation in PMP22 gene is associated with Charcot Marie Tooth, type 1A (CMT1A).In CMT1, PMP22 duplication was the most common mutation while the second gene in order of frequency was MPZ in familial and SH3TC2 in isolated cases. CMT1A/PMP22 duplicationhe most frequent subtype is type 1A (CMT1A) caused by duplication in chromosome 17p12 that includes PMP22.\"}, {\"question\": \"Which viruses are best known to cause myocarditis?\", \"ideal_answer\": [\"The most frequent viruses causing myocarditis are Enterovirus, Adenovirus and Coxsackie B viruses.\"], \"exact_answer\": [[\"Enterovirus\"], [\"Adenovirus\"], [\"Coxsackie B virus\"]], \"context\": \"Enteroviruses (EV) are an important cause of neonatal disease including hepatitis, meningoencephalitis, and myocarditis that can lead to death or severe long-term sequelaeEnteroviruses have been considered to be the most common cause of acute myocarditis and possible consequence of dilated cardiomyopathy.n our study the adenovirus genome was found to be the most frequent virus genome in explanted heart tissues.Coxsackie B viruses (types 1 to 5) are the most frequent reported cause of acute viral myocarditis.\"}, {\"question\": \"Which genes have been associated with Cerebral Cavernous Malformation?\", \"ideal_answer\": [\"Loss-of-function mutations in genes encoding CCM1 (also known as KRIT1), CCM2 (also known as OSM and malcavernin) or CCM3 (also known as PDCD10) cause cerebral cavernous malformations (CCMs).\"], \"exact_answer\": [[\"CCM1/KRIT1\"], [\"CCM2/OSM/Malcavernin\"], [\"CCM3/PDCD10\"]], \"context\": \"Cerebral cavernous malformations associated to meningioma: High penetrance in a novel family mutated in the PDCD10 genewe outline a consistent association between PDCD10 mutations and a syndrome of CCMs with multiple meningiomasCerebral cavernous malformation (CCM) is a disease of vascular malformations known to be caused by mutations in one of three genes: CCM1, CCM2 or CCM3To study the molecular genetic and clinical features of cerebral cavernous malformations (CCM) in a cohort of Spanish patients.METHODS: We analyzed the CCM1, CCM2, and CCM3 genesLoss-of-function mutations in genes encoding KRIT1 (also known as CCM1), CCM2 (also known as OSM and malcavernin) or PDCD10 (also known as CCM3) cause cerebral cavernous malformations (CCMs)Surveying genetic variants and molecular phylogeny of cerebral cavernous malformation gene, CCM3/PDCD10The three cerebral cavernous malformations (CCMs) genes namely CCM1/KRIT1, CCM2/MGC4607 and CCM3/PDCD10 have been identified for which mutations cause cerebral cavernous malformationsAt least 45% of families affected with cerebral cavernous malformations harbour a mutation in Krev interaction trapped-1 (Krit1) gene (cerebral cavernous malformation gene-1, CCM1).Mutations in Krev1 interaction trapped gene 1 (KRIT1) cause cerebral cavernous malformation, an autosomal dominant disease featuring malformation of cerebral capillaries resulting in cerebral hemorrhage, strokes, and seizures.KRIT1, a gene mutated in cerebral cavernous malformation, encodes a microtubule-associated protein.Germline mutations in the CCM1 gene, encoding Krit1, cause cerebral cavernous malformations.Mutations within the programmed cell death 10 gene cause cerebral cavernous malformations.Neuronal expression of the Ccm2 gene in a new mouse model of cerebral cavernous malformations.These findings document a de novo germline mutation in Kritl gene that causes cerebral cavernous malformations.Cerebral cavernous malformation is a common human vascular disease that arises due to loss-of-function mutations in genes encoding three intracellular adaptor proteins, cerebral cavernous malformations 1 protein (CCM1), CCM2, and CCM3.Cerebral cavernous malformation is a common human vascular disease that arises due to loss-of-function mutations in genes encoding three intracellular adaptor proteins, cerebral cavernous malformations 1 protein (CCM1), CCM2, and CCM3Mutation and expression analysis of the KRIT1 gene associated with cerebral cavernous malformations (CCM1).Variable expression of cerebral cavernous malformations in carriers of a premature termination codon in exon 17 of the Krit1 gene.Loss-of-function mutations in CCM1/KRIT1, CCM2/MGC4607 and CCM3/PDCD10 genes are identified in the vast majority of familial cases with multiple cerebral cavernous malformations (CCMs).Mutations in three genes are associated with CCM. These genes encode for the proteins KRIT1/CCM1 (krev interaction trapped 1/cerebral cavernous malformations 1), cerebral cavernous malformations 2, osmosensing scaffold for MEKK3 (CCM2/malcavernin/OSM), and cerebral cavernous malformations 3/programmed cell death 10 (CCM3/PDCD10).At least 45% of families affected with cerebral cavernous malformations harbour a mutation in Krev interaction trapped-1 (Krit1) gene (cerebral cavernous malformation gene-1,Exceptional aggressiveness of cerebral cavernous malformation disease associated with PDCD10 mutations.A novel CCM1 mutation associated with multiple cerebral and vertebral cavernous malformations.These genes encode for the proteins KRIT1/CCM1 (krev interaction trapped 1/cerebral cavernous malformations 1), cerebral cavernous malformations 2, osmosensing scaffold for MEKK3 (CCM2/malcavernin/OSM), and cerebral cavernous malformations 3/programmed cell death 10 (CCM3/PDCD10).CCM1 is caused by a mutation in the KRIT1 gene and CCM2 is caused by a mutation in the MGC4607 gene, while the gene for CCM3 is not yet identified.Mutation and expression analysis of the KRIT1 gene associated with cerebral cavernous malformations (CCM1).At least 45% of families affected with cerebral cavernous malformations harbour a mutation in Krev interaction trapped-1 (Krit1) gene (cerebral cavernous malformation gene-1, CCM1).KRIT1, a gene mutated in cerebral cavernous malformation, encodes a microtubule-associated protein.\"}, {\"question\": \"List symptoms of 4H leukodystrophy.\", \"ideal_answer\": [\"Hypomyelination, hypodontia, and hypogonadotropic hypogonadism are major symptoms of 4H leukodystrophy.\"], \"exact_answer\": [[\"hypomyelination\"], [\"hypodontia\"], [\"hypogonadotropic hypogonadism\"]], \"context\": \" MRI demonstrated diffuse cerebral hypomyelination, cerebellar atrophy, and thin corpus callosum; X-ray revealed persistent milk teeth and hypoplastic crowns and roots (figure), indicative of 4H syndrome (hypomyelination, hypodontia, hypogonadotropic hypogonadism).4H syndrome is a rare and distinct leukodystrophy characterized by hypomyelination, hypogonadotropic hypogonadism, and hypodontia.The 4H syndrome (hypomyelination, hypodontia, hypogonadotropic hypogonadism) is a newly recognized leukodystrophy. The five overlapping clinical phenotypes (described as distinct entities before their molecular basis was known) include: Hypomyelination, hypodontia, hypogonadotropic hypogonadism (4H syndrome); Ataxia, delayed dentition, and hypomyelination (ADDH); Tremor-ataxia with central hypomyelination (TACH); Leukodystrophy with oligodontia (LO); and Hypomyelination with cerebellar atrophy and hypoplasia of the corpus callosum (HCAHC).OBJECTIVE: To report a novel clinical and genetic presentation of a patient with 4H syndrome, which is a recently described leukodystrophy syndrome characterized by ataxia, hypomyelination, hypodontia, and hypogonadotropic hypogonadism.Because these families were mapped to the same locus as leukodystrophy with oligodontia (LO) and presented clinical and radiological overlap with individuals with hypomyelination, hypodontia and hypogonadotropic hypogonadism (4H) syndrome, we sequenced this gene in nine individuals with 4H and eight with LO.AIM: To report one patient with slowly progressive encephalopathy, ataxia, central hypomyelination, hypodontia and hypogonadotropic hypogonadism, the 4H syndrome.4H syndrome is a rare and distinct leukodystrophy characterized by hypomyelination, hypogonadotropic hypogonadism, and hypodontiaBecause these families were mapped to the same locus as leukodystrophy with oligodontia (LO) and presented clinical and radiological overlap with individuals with hypomyelination, hypodontia and hypogonadotropic hypogonadism (4H) syndrome, we sequenced this gene in nine individuals with 4H and eight with LOMRI demonstrated diffuse cerebral hypomyelination, cerebellar atrophy, and thin corpus callosum; X-ray revealed persistent milk teeth and hypoplastic crowns and roots (figure), indicative of 4H syndrome (hypomyelination, hypodontia, hypogonadotropic hypogonadism)Because these families were mapped to the same locus as leukodystrophy with oligodontia (LO) and presented clinical and radiological overlap with individuals with hypomyelination, hypodontia and hypogonadotropic hypogonadism (4H) syndrome, we sequenced this gene in nine individuals with 4H and eight with LOThe 4H syndrome (hypomyelination, hypodontia, hypogonadotropic hypogonadism) is a newly recognized leukodystrophy4H syndrome is a rare and distinct leukodystrophy characterized by hypomyelination, hypogonadotropic hypogonadism, and hypodontia\"}, {\"question\": \"Which are the plant DNA (cytosine-5) methyltransferase families?\", \"ideal_answer\": [\"The plant DNA (cytosine-5)methyltransferases are classified into the families: MET, CMT, and the de novo DRM.\"], \"exact_answer\": [[\"MET\"], [\"CMT\"], [\"DRM\"]], \"context\": \"The topologies of the trees were overall congruent: four monophyletic groups corresponding to the four plant C5-MTase families were clearly distinguished. In addition, sequence analyses of the plant C5-MTase target recognition domain sequences were performed and phylogenetic trees were reconstructed showing that there is good conservation among but not within the plant C5-MTase families.To determine the inheritance of DNA methyltransferase genes and their expression patterns we examined three major DNA methyltransferase families (MET1, CMT3 and DRM) from tobacco and the progenitor species.The comparative investigation of transcription levels of different genes of cytosine DNA methyltransferase family MET (MET1, MET2a, MET2b, MET3) and their methylation patterns shows that there may exist some mechanisms defending the most actively transcribed gene MET1 of this family from methylation mediated silencing. In contrast to DRM2 gene we could not find any adenine methylated GATC sites in the MET1 gene.Using the 1kb 3' terminal DNA fragment of the mouse methyltransferase cDNA as a probe and low stringent hybridisation conditions, a new potential methyltransferase (MTase) gene family was isolated from an Arabidopsis thaliana genomic DNA library.The Dnmt3 family of de novo DNA methyltransferases has recently been characterized in animals. Here we describe DNA methyltransferase genes from both Arabidopsis and maize that show a high level of sequence similarity to Dnmt3, suggesting that they encode plant de novo methyltransferases. Relative to all known eukaryotic methyltransferases, these plant proteins contain a novel arrangement of the motifs required for DNA methyltransferase catalytic activity.These results provide the first direct demonstration that DNA-METases of a higher eukaryote are encoded by a gene family.Here we show that Arabidopsis DNA methyltransferase2 (AtDNMT2) is localized in nucleus and associates with histone deacetylation.To explore possible relationships between DNA methylation level and accumulation of DNA-(cytosine-5) methyltransferase (DNMT) transcripts, the full-length coding sequences corresponding to three different DNMT families in oil palm, namely the MET, CMT, and DRM classes, have been isolated and characterized.In Arabidopsis a SWI2/SNF2 chromatin remodeling factor-related protein DDM1 and a cytosine methyltransferase MET1 are required for maintenance of genomic cytosine methylation.Relative to all known eukaryotic methyltransferases, these plant proteins contain a novel arrangement of the motifs required for DNA methyltransferase catalytic activity.In the present study, the isolation and characterization of two distinct cDNAs that code for carrot DNA (cytosine-5)-methyltransferase (DNA-METase) are reported.The Arabidopsis methylase has eight of the ten conserved sequence motifs found in prokaryote cytosine-5 methyltransferases and shows 50% homology to the murine enzyme in the methyltransferase domain.As deduced form the DNA sequence this protein contains all conserved sequence motifs specific for the 5m cytosine MTases.\"}, {\"question\": \"In which proteins is the chromodomain present?\", \"ideal_answer\": [\"The chromodomain (chromatin organizer modifier domain) is a highly conserved motif, 40-50 amino acids in length, present in a wide range of animal and plant proteins involved in chromatin organization. Chromodomain-containing proteins can be classified into boader families based, particularly, on the presence of other types of domains. Chromodomain is present in: the heterochromatin proteins HP1 alpha and HP1 beta, chromointgrases (e.g. Tf1 integrase) the chromodomain helicase DNA-binding proteins (CHD) and CHD 1-like (CHD1L), CReMM (chromatin-related mesenchymal modulator), dna methyltransferase 3 (cmt3), the chromointegrase of the LTR-retrotransposons, the Polycomb group (PcG) proteins, the mouse Polycomb homologs (Cbx2, Cbx4, Cbx6, Cbx7, Cbx8), the chromodomain Y chromosome (CDY) family of proteins and the CDY-like protein (CDYL), the histone acetyltransferases TgMYST-A and \\u2013B, MRG-1 and -15 (MORF4-Related Gene on chromosome 15), ADP/ATP translocase 1, MPP8, MSL3, NlMof, Chp1, Chriz, dMi-2, Corto, cpSRP43, KISMET, PICKLE (PKL), ScoHET1 and ScoHET2.\\n\"], \"exact_answer\": [[\"HP1 alpha\"], [\"CHD proteins\", \"Chromodomain helicase DNA-binding protein\"], [\"CReMM\"], [\"cmt3\"], [\"chromointegrases\"], [\"Polycomb group (PcG) proteins\"], [\"Cbx2\"], [\"TgMYST-A\"], [\"MRG15\"], [\"MSL3\"], [\"Swi6\"], [\"Chp1\"], [\"dMi-2\"], [\"Corto\"], [\"ScoHET1\"], [\"cpSRP43\"], [\"CDY\", \"chromodomain Y chromosome\"], [\"transposable elements\"], [\"like heterochromatin protein\", \"LHP1\"], [\"ADP/ATP translocase 1\"], [\"CDYL\", \"CDY-like protein\"], [\"CHD1L\", \"CHD1-like\"], [\"NlMof\"], [\"TIP60\"], [\"Kismet\"], [\"Chriz\"], [\"PICKLE\", \"PKL\"], [\"HP1 beta\"], [\"HP1 gamma\"], [\"TgMYST \\u2013B\"], [\"Cbx4\"], [\"Cbx6\"], [\"Cbx7\"], [\"Cbx8\"], [\"ScoHET2\"], [\"Cbx3\"], [\"MPP8\"], [\"MRG1\"]], \"context\": \"Chromodomain helicase DNA-binding protein 2 affects the repair of X-ray and UV-induced DNA damage.Chromodomain helicase DNA-binding proteins (CHD) belong to a group of highly conserved chromatin remodeling proteins that are implicated in regulation of transcription. In an effort to understand the physiological role of one of the CHD members in a mammalian model system, we developed a mutant mouse model for the Chd2 gene.Crystal structure of the chromodomain helicase DNA-binding protein 1 (Chd1) DNA-binding domain in complex with DNA.Because, in plants, DNA methylation can serve as a signal for H3-lysine9-dimethylation (H3K9me2), and subsequently for non-CG-context DNA methylation, SET-domain histone methyltransferase and chromodomain dna methyltransferase 3 (cmt3) mutations were introgressed. In suvh4 suvh5 suvh6 and cmt3 mutants, H3K9me2 associated with lacO repeats is diminished, but homologous pairing persists.Mutations in CHD7, the gene encoding chromodomain helicase DNA binding protein 7, are present in 60-80% of individuals with the CHARGE syndrome.Chromodomain is present in the integrase structures of blastopia and 412 subgroup LTR-retrotransposons and may facilitate the process of non-specific integration.The CHARGE syndrome is a multiple congenital malformation syndrome that usually results from deletion or heterozygous loss of function mutations of the chromodomain helicase DNA-binding protein 7 (CHD7) gene at 8q12.1.A causative mutation within the chromodomain helicase DNA-binding protein-7 gene, which plays an important role in the embryonic development, is present in 2/3 of affected patients.CHD1 is a subfamily member of the CHD family, which possesses a chromodomain, a helicase domain, and a DNA-binding domain.The chromatin organizer modifier domain (chromodomain) is present in proteins that contribute to chromatin organization and mediates their binding to methylated histone H3.Mass spectrometric analysis revealed serine-42, a conserved amino acid in the chromodomain, as a phosphorylation site of Cbx2. Phosphorylation of the chromodomain of Cbx2 on this residue in vitro resulted in a reduced level of binding to an H3 peptide containing trimethylated lysine-9 as well as an increase in the extent of binding to an H3 peptide containing trimethylated lysine-27, suggesting that such phosphorylation changes the binding specificity of Cbx2 for modified histone H3. Phosphorylation of the chromodomain of Cbx2 may therefore serve as a molecular switch that affects the reading of the histone modification code and thereby controls epigenetic cellular memory.Here we identify the ATP-dependent chromatin remodeling factor chromodomain helicase DNA-binding protein 8 (CHD8) as a novel coregulator of androgen-responsive transcription.Mutations in CHD7, a chromodomain gene, are present in a majority of individuals with CHARGE syndrome, a multiple anomaly disorder characterized by ocular Coloboma, Heart defects, Atresia of the choanae, Retarded growth and development, Genital hypoplasia and Ear anomalies.Two new chromodomain-containing proteins that associate with heterochromatin in Sciara coprophila chromosomes.Both proteins, ScoHET1 of 37 kDa and ScoHET2 of 44 kDa, display two chromodomain motifs that contain the conserved residues essential for the recognition of methylated histone H3 at lysine 9.In this study, we present a detailed histomorphologic characterization of mice containing a mutation in the chromodomain helicase DNA-binding protein 2 (Chd2).Specificity of the chromodomain Y chromosome family of chromodomains for lysine-methylated ARK(S/T) motifs.Previous studies have shown two homologous chromodomain modules in the HP1 and Polycomb proteins exhibit discriminatory binding to related methyllysine residues (embedded in ARKS motifs) of the histone H3 tail.In vertebrates, HP1-like chromodomains are also present in the chromodomain Y chromosome (CDY) family of proteins adjacent to a putative catalytic motif. The human genome encodes three CDY family proteins, CDY, CDYL, and CDYL2.The CDY chromodomain exhibits discriminatory binding to lysine-methylated ARK(S/T) motifs, whereas the CDYL2 chromodomain binds with comparable strength to multiple ARK(S/T) motifs. Interestingly, subtle amino acid changes in the CDYL chromodomain prohibit such binding interactions in vitro and in vivo.We assessed mRNA transcript abundance of seven genes that code for proteins with established roles in epigenetically-mediated gene silencing [transcriptional co-repressor SIN3A, DNA (cytosine-5-) methyltransferase 1, methyl CpG binding protein 2, chromodomain helicase DNA binding protein 4, histone binding protein rbbp4, histone deacetylase 1 and nuclear receptor co-repressor 2] using qRT-PCR.CHD7 is a member of the chromodomain helicase DNA binding domain (CHD) family of ATP-dependent chromatin remodelling enzymes.The cysteine-rich CXXC domains of MBD1 bound to Ring1b and the chromodomain of hPc2.Methylation of lysine 9 within histone H3 and the subsequent binding of the chromodomain protein heterochromatin protein 1 (HP1) are thought to initiate heterochromatin formation in vivo and to propagate a heterochromatic state lasting through several cell divisions.Here, we show that the SNF2-like chromodomain helicase protein CHD8 interacts with the insulator binding protein CTCF. CThe chromodomain (CD) of the Drosophila Polycomb protein exhibits preferential binding affinity for histone H3 when trimethylated at lysine 27.Despite a high degree of conservation, the Cbx chromodomains display significant differences in binding preferences.Three-dimensional solution structures of the chromodomains of cpSRP43.Recently, three functionally distinct chromodomains (CDs) have been identified in cpSRP43.The C-terminal helical segment typically found in the nuclear chromodomains is absent in CD1.Critical comparison of the structures of the chromodomains of cpSRP43 with those found in nuclear chromodomain proteins revealed that the diverse protein-protein interactions mediated by the CDs appear to stem from the differences that exist in the surface charge potentials of each CD.Characterization and functional analysis of CReMM, a novel chromodomain helicase DNA-binding protein.The present study describes a newly identified protein named CReMM (chromatin-related mesenchymal modulator). The protein was studied by bioinformatic means and classified as a member of the third subfamily of chromodomain helicase DNA-binding proteins (CHD). In silico translation defined CReMM as a multiple domains protein including two chromodomains, SNF2/ATPase, helicase C domain and an A/T-DNA-binding domain (DBD).Identification and analysis of chromodomain-containing proteins encoded in the mouse transcriptome.The chromodomain is 40-50 amino acids in length and is conserved in a wide range of chromatic and regulatory proteins involved in chromatin remodeling. Chromodomain-containing proteins can be classified into families based on their broader characteristics, in particular the presence of other types of domains, and which correlate with different subclasses of the chromodomains themselves.Here we show that the chromodomain of CMT3 can directly interact with the N-terminal tail of histone H3, but only when it is simultaneously methylated at both the H3K9 and H3K27 positions.In addition to the canonical MYST HAT catalytic domain, both TgMYST-A and -B possess an atypical C2HC zinc finger and a chromodomain.Clr7 and Clr8 are required for localization of the Swi6 chromodomain protein and for histone H3 lysine 9 methylation, thereby influencing not only mating-type switching but also transcriptional silencing in all previously characterized heterochromatic regions, chromosome segregation, and meiotic recombination in the mating-type region.MRG15, a novel chromodomain protein, is present in two distinct multiprotein complexes involved in transcriptional activation.MRG15 is a novel chromodomain protein that is a member of a family of genes related to MORF4. MORF4 (mortality factor on chromosome 4) induces senescence in a subset of human tumor cell lines.Analysis of deletion mutants of MRG15 indicated that the leucine zipper at the C-terminal region of MRG15 was important for the protein associations in MAF1 and that the N-terminal chromodomain was required for the assembly of the MAF2 protein complex. Consistent with these data was the fact that a histone acetyltransferase activity associated with MRG15 was lost when the chromodomain was deleted and that both mutant MRG15 proteins failed to activate the B-myb promoter. The various mechanisms by which MRG15 could activate gene transcription are discussed.The chromodomain (CD) is a highly conserved motif present in a variety of animal and plant proteins, and its probable role is to assemble a variety of macromolecular complexes in chromatin.Structure of HP1 chromodomain bound to a lysine 9-methylated histone H3 tail.The chromodomain of the HP1 family of proteins recognizes histone tails with specifically methylated lysines. Here, we present structural, energetic, and mutational analyses of the complex between the Drosophila HP1 chromodomain and the histone H3 tail with a methyllysine at residue 9, a modification associated with epigenetic silencing. The histone tail inserts as a beta strand, completing the beta-sandwich architecture of the chromodomain. The methylammonium group is caged by three aromatic side chains, whereas adjacent residues form discerning contacts with one face of the chromodomain.The MORF4-Related Gene on chromosome 15 (MRG15) is a member of a novel family of genes originally identified in studies to reveal cell senescence-inducing factors. MRG15 contains several predicted protein motifs, including a nuclear localization signal, a helix-loop-helix region, a leucine zipper, and a chromodomain.Four alleles of dMi-2 mutants were further characterized in molecular nature; dMi-2(BL1) was found to have a mutation in the ATP-binding motif of the ATPase domain, dMi-2(BL7) in the core histidine of the first plant homeodomain zinc finger and dMi-2(BL12) in a conserved serine in the chromodomain.These analyses also suggest that the msl-3/MRG15 duplication occurred after the fungus/animal split, while an independent duplication occurred in plants. The proteins encoded by these genes have similar structures, including a putative chromodomain close to their N-terminal end and a putative leucine zipper at their C-terminus.CHD1 interacts with SSRP1 and depends on both its chromodomain and its ATPase/helicase-like domain for proper association with chromatin.Interaction between an integral protein of the nuclear envelope inner membrane and human chromodomain proteins homologous to Drosophila HP1.A mammalian DNA-binding protein that contains a chromodomain and an SNF2/SWI2-like helicase domain.A Southern blot analysis indicated that this protein, which we have named CHD-1, for chromodomain-helicase-DNA-binding protein, is present in most, if not all, mammalian species.Here we show that chromodomain helicase DNA-binding protein 4 (Chd4) forms a complex with Gata3 in Th2 cells that both activates Th2 cytokine transcription and represses the Th1 cytokine IFN-\\u03b3.Chromodomain helicase DNA-binding protein 4 (CHD4), the defining subunit of the nucleosome remodeling and deacetylase (NuRD) complex, is a nucleosome-remodeling protein of the SNF2/ISWI2 family, members of which contain two chromo domains and an ATP-dependent helicase module.Among those proteins with >40% regulation were Macrophage Capping protein (CAPG) and Chromodomain Helicase DNA binding protein 4 (CHD4) proteins which were significantly upregulated by pp32r1 and pp32r1Y140H overexpression.This heterochromatin coordinates expression levels by associating with a chromodomain protein Chp1 and an antisilencing factor Epe1.Within NCCs, Brg1 partners with chromatin remodeler Chromodomain-helicase-DNA-binding protein 7 (Chd7) on the PlexinA2 promoter to activate PlexinA2, which encodes a receptor for semaphorin to guide NCCs into the OFT.Chromodomain Helicase DNA binding protein 5 (CHD5) is a tumor suppressor mapping to 1p36, a genomic region that is frequently deleted in human cancer.Here, we identify ENHANCED PHOTOMORPHOGENIC1 (EPP1), previously known as PICKLE (PKL), an ATP-dependent chromatin remodeling factor of the chromodomain/helicase/DNA binding family, as a repressor of photomorphogenesis in Arabidopsis thaliana.Chromodomain on Y-like (CDYL) is a chromodomain protein that has sequence homology to members of the enoyl CoA hydratase family.Our previous structural work demonstrated that a coiled-coil interaction between MBD2 and GATA zinc finger domain containing 2A (GATAD2A/p66\\u03b1) proteins recruits the chromodomain helicase DNA-binding protein (CHD4/Mi2\\u03b2) to the NuRD complex and is necessary for MBD2-mediated DNA methylation-dependent gene silencing in vivo (Gnanapragasam, M. N., Scarsdale, J. N., Amaya, M. L., Webb, H. D., Desai, M. A., Walavalkar, N. M., Wang, S. Z., Zu Zhu, S., Ginder, G. D., and Williams, D. C., Jr. (2011) p66\\u03b1-MBD2 coiled-coil interaction and recruitment of Mi-2 are critical for globin gene silencing by the MBD2-NuRD complex.Sequence analysis showed that NlElp3 contains GNAT-type HAT domain and Radical SAM domain, and NlMof contains chromodomain and MOZ-SAS acetyltransferase domain.We examined requirements for individual domains of chromodomain helicase DNA-binding protein 4 (CHD4), a core catalytic component of NuRD complexes, as well as the NuRD subunit methyl-binding domain protein 2 (MBD2) and methylated DNA, for NuRD function in the context of tissue-specific transcription.OBJECTIVE: Chromodomain helicase/adenosine triphosphatase DNA binding protein 1-like (CHD1L) is an SNF2-like transcription factor involved in the development of human hepatocellular carcinoma (HCC).Twelve rare copy number variants segregate with talipes equinovarus in multiplex pedigrees, and contain the developmentally expressed transcription factors and transcriptional regulators PITX1, TBX4, HOXC13, UTX, CHD (chromodomain protein)1, and RIPPLY2.OBJECTIVE: Chromodomain helicase DNA-binding protein 5 (CHD5) plays a role in normal neural development and in tumorigenesis of various human cancers.The chromodomain, helicase, DNA-binding protein 5 (CHD5) is a chromatin remodeling enzyme which is implicated in tumor suppression.The methyl-mark determines a highly flexible and very dynamic interaction of the chromodomain of hHP1\\u03b2 with the H3-tail.To test this hypothesis, we performed a comprehensive molecular dynamics study in which we analyzed a crystallographically defined complex that involves the HP1 chromodomain and an H3 tail peptide.Whereas the molecular target of the MRG15 chromodomain (CD) has been suggested to be H3K36me(2/3), the precise molecular target of the Pf1 plant homeodomain 1 (PHD1) has remained elusive.Chp1, a chromodomain (CD) protein, forms the Ago1-containing RNA-induced transcriptional silencing (RITS) complex and recruits siRNA-bound RITS to methylated histone H3 lysine 9 (H3K9me) via its CD.In addition, cells expressing S473A also displayed defective mobilization of the HP1-\\u03b2 chromodomain protein.CHD7 is a chromodomain-containing, ATP dependent helicase protein that is highly expressed in the developing ear and is required for semicircular canal development in both humans and mice.Moreover, the LTR retrotransposon fraction in BAC clones harboring genes is disproportionately composed of chromodomain-containing Gypsy LTR retrotransposons ('chromoviruses'), and the majority of the intact chromoviruses contain tandem chromodomain duplications.Stimulated by positively charged residues in the hinge region, RNA competes with methylated histone H3K9 for binding to the chromodomain of HP1(Swi6).CHD7 is one of the nine members of the chromodomain helicase DNA-binding family of ATP-dependent chromatin remodeling enzymes.Therefore, p53, ZNF237, and Chromodomain helicase DNA-binding protein 3 inhibit the function ER Ca\\u00b2\\u207a leak channels to regulate both ER and cytoplasmic Ca\\u00b2\\u207a levels and may potentially control Ca\\u00b2\\u207a-signaling function of PS1.We show that chromodomain helicase DNA-binding domain 2 (Chd2), a SNF2 chromatin remodelling enzyme family member, interacts with MyoD and myogenic gene regulatory sequences to specifically mark these loci via deposition of the histone variant H3.3 prior to cell differentiation.Analysis of the Neurospora crassa chromodomain protein CDP-2, a component of a newly characterized HP1-containing complex, reveals a second gene-silencing mechanism and provides insights into the dynamic nature of chromatin domains that possess shared components.We state that the non-tumorogenic potential of bitumen transformant in nude/SCID mice can be attributed to the downregulation of galectin-1, chromodomain helicase DNA-binding protein 1-like gene, and membrane-associated guanylate kinase 2 protein.Mutagenesis of Tf1 integrase revealed that the complete Tf1 integrase protein (excluding its chromodomain) is required for stimulating the Tf1 RT primer removal activity.About 80% of patients with a clinical diagnose, have a mutation or a deletion in the gene encoding chromodomain helicase DNA-binding protein 7 (CHD7).Owing to different polyadenylation sites and alternative splicing events, the human CBX2 locus produces two transcripts: a 5-exon transcript that encodes the 532-amino acid CBX2-1 isoform that contains the conserved chromodomain and Pc box and a 4-exon transcript encoding a shorter isoform, CBX2-2, lacking the Pc box but still possessing a chromodomain.Chromodomain helicase DNA-binding protein 4 (Chd4) is the core catalytic subunit of the nucleosome remodeling and deacetylase (NuRD) chromatin remodeling complex.Chromodomain helicase DNA-binding proteins (CHD) belong to a group of highly conserved chromatin remodeling proteins that are implicated in regulation of transcription.Here, we report that chromodomain helicase DNA-binding protein 4 (CHD4) is a novel BRIT1 binding partner that regulates the HR repair process.Here, we report that the chromodomain-containing protein MRG-1 is an important factor for genomic integrity in meiosis in Caenorhabditis elegans.The chromodomain protein, Chromator, can be divided into two main domains, a NH(2)-terminal domain (NTD) containing the chromodomain (ChD) and a COOH-terminal domain (CTD) containing a nuclear localization signal.Chromodomain helicase DNA binding protein 5 (CHD5) is a potent tumor suppressor that serves as a master regulator of a tumor-suppressive network.By high-resolution tiling array CGH, the smallest common deletion targeted just one gene, the chromatin remodeler chromodomain helicase DNA-binding protein 1 (CHD1).Here we report direct evidence for presynaptic pairing activity intrinsic to non-PC regions, which is facilitated by a conserved chromodomain protein, MRG-1.The chromodomain of MPP8 recognizes the dimethylated Dnmt3aK44me2.Members of the chromodomain helicase DNA-binding (CHD) family of proteins are thought to regulate gene expression.It has been shown that the methylation mark of vertebrate histone H1 is specifically recognized by the chromodomain of HP1.To this end, we have performed mutation studies on the Drosophila HP1\\u03b1 chromodomain, which binds H3K9Me(2)  and H3K9Me(3)  with approximately equal affinities.We are interested in defining which elements of the chromodomain helicase DNA-binding protein 1 (Chd1) remodeler are necessary and sufficient for sliding nucleosomes.Chromatin remodelers are ATP-dependent machines that dynamically alter the chromatin packaging of eukaryotic genomes by assembling, sliding, and displacing nucleosomes.Ocular coloboma, heart malformation, choanal atresia, retardation of growth and/or development, genital hypoplasia, and ear anomalies associated with deafness (CHARGE) syndrome is a rare, usually sporadic, autosomal dominant disorder, caused by mutations within the CHD7 (chromodomain helicase DNA-binding protein 7) gene, in nearly 70% of cases.Here we report that the chromodomain-containing protein CDYL specifically recognizes di- and tri-methylated H3K27 (H3K27me2 and H3K27me3) and directly interacts with EZH2, the catalytic subunit of PRC2.Murine Chd1 (chromodomain helicase DNA-binding protein 1), a chromodomain-containing chromatin remodeling protein, is necessary for embryonic stem (ES) cell pluripotency.Among those genes, zinc finger helicase (ZFH), also termed chromodomain-helicase-DNA-binding protein 3 (Chd3), was one of the highly expressed transcripts in tentative cementoblasts.UNLABELLED: Emerging evidence implicates the chromodomain helicase/ATPase DNA binding protein 1-like gene (CHD1L) as a specific oncogene in human hepatocellular carcinoma (HCC).Here, we investigate the function of chromodomain helicase DNA binding protein 7 (chd7) during zebrafish somitogenesis.Heterozygous mutations in the gene encoding chromodomain-DNA-binding-protein 7 (CHD7) cause CHARGE syndrome, a multiple anomaly condition which includes vestibular dysfunction and hearing loss.BACKGROUND: The CHD5 gene located on 1p36 encodes a protein-chromodomain helicase DNA-binding protein 5. CHD5 has been shown to be a tumor suppressor gene candidate.Because, in plants, DNA methylation can serve as a signal for H3-lysine9-dimethylation (H3K9me2), and subsequently for non-CG-context DNA methylation, SET-domain histone methyltransferase and chromodomain dna methyltransferase 3 (cmt3) mutations were introgressed.Previously, we have shown that the chromodomain protein Chriz and the zinc-finger protein Z4 are essentially required for the maintenance of polytene chromosome structure.In brain, Family with sequence similarity 174 member b (Fam174b) had increased expression in 318 females, whereas Chromodomain helicase DNA binding protein 2 (Chd2-2) had reduced expression in 318 males.RESULTS: Here we demonstrated that chromobox protein homolog 3 (Cbx3) is crucial for SMC differentiation from stem cells and that the chromodomain and chromoshadow domain of Cbx3 are responsible for Cbx3-induced SMC differentiation.Fractionation of the extract identified a single protein, chromodomain helicase DNA binding protein 1 (Chd1), capable of the remodeling activity.Chromodomain helicase DNA-binding protein 5 (CHD5) has been found to be a candidate tumor suppressor gene (TSG) in malignant neural tumors.Similarly, CenH3-GFP distribution was altered in the absence of HP1, the chromodomain protein that binds to H3K9me3.ChIP-Seq reveals that KDM5B is predominantly targeted to intragenic regions and that it is recruited to H3K36me3 via an interaction with the chromodomain protein MRG15.OBJECTIVE: Chromodomain helicase DNA-binding protein (CHD) is a regulator of the chromatin remodelling process.We determined the crystal structure of MPP8 chromodomain in complex with H3K9me3 peptide.On this gene the chromodomain protein HP1\\u03b3, frequently defined as a transcriptional repressor, facilitates inclusion of the alternative exons via a mechanism involving decreased RNA polymerase II elongation rate.Chromodomain-helicase-DNA-binding protein 4 (CHD4) is a core component of the NuRD complex and contains a nucleosome remodeling ATPase domain along with two chromodomains and two plant homeodomain (PHD) fingers.H3-Lys-9-Me2 interacts with the chromodomain of Swi6/HP1.The chromodomain (CD) of HP1 proteins specifically recognizes the methyl mark on H3 peptides, but the same extent of specificity is not observed within chromatin.We have performed simulations on models of chromodomain helicase DNA-binding protein 1 complexed with a variety of histone H3 modifications.CHARGE syndrome is an autosomal dominant multisystem disorder caused by mutation in the CHD7 gene, encoding chromodomain helicase DNA-binding protein 7. Molecular diagnostic testing for CHD7 mutation has been available in a clinical setting since 2005.In this work we identified Kismet, a chromodomain-containing protein of the SNF2-like family of ATPases, as a novel component of the hedgehog transcriptional repression mechanism in anterior compartment cells.\"}, {\"question\": \"Are there drugs for Tick-borne Encephalitis?\", \"ideal_answer\": [\"No drug therapy available today\"], \"exact_answer\": [[\"No drug therapy available today\"]], \"context\": \" tick-borne encephalitis (TBE) have been detected in Bulgaria. Considering the remarkable increase in TBE morbidity in Europe over the past two decades, we conducted a study of TBE among patients with acute viral meningitis who were hospitalised in Bulgaria during 2009 to 2012. Viruses belonging to the Flaviviridae family primarily spread through arthropod vectors, and are the major causes of illness and death around the globe. The Flaviviridae family consists of 3 genera which include the Flavivirus genus (type species, yellow fever virus) as the largest genus, the Hepacivirus (type species, hepatitis C virus) and the Pestivirus (type species, bovine virus diarrhea). The flaviviruses (Flavivirus genus) are small RNA viruses transmitted by mosquitoes and ticks that take over host cell machinery in order to propagate. However, hepaciviruses and pestiviruses are not antropod-borne. Despite the extensive research and public health concern associated with flavivirus diseases, to date, there is no specific treatment available for any flavivirus infections, though commercially available vaccines for yellow fever, Japanese encephalitis and tick-born encephalitis exist.\"}, {\"question\": \"Which are the drugs utilized for the burning mouth syndrome?\", \"ideal_answer\": [\"Dopaminergic drugs should be given in patients with BMS. \\nCatuama reduces the symptoms of BMS and may be a novel therapeutic strategy for the treatment of this disease.\\nCapsaicin, alpha-lipoic acid (ALA), and clonazepam were those that showed more reduction in symptoms of BMS.\\nTreatment with placebos produced a response that was 72% as large as the response to active drugs\"], \"exact_answer\": [[\"Dopaminergig drugs\"], [\"Catuama\"], [\"Capsaicina\"], [\"Alpha-lipoic acid\"], [\"Clonazepam\"], [\"Placebo therapy\"]], \"context\": \"On average, treatment with placebos produced a response that was 72% as large as the response to active drugs.The concomitant prescription of tongue protector and AV is effective for treating patients with BMS.We suggest that a subset of patients with BMS may be a phenotypic variant of RLS and a trial of dopaminergic drugs should be given in patients with BMS who has a history suggestive of RLS or in a patient who do not show a response to usual therapies for BMS.The systemic administration of Catuama reduces the symptoms of BMS and may be a novel therapeutic strategy for the treatment of this disease.Therapies that used capsaicin, alpha-lipoic acid (ALA), and clonazepam were those that showed more reduction in symptoms of BMS.Topical clonazepam, which previous analyses of clinical evidence have shown to be the drug of choice for BMS, also proved to be the most cost-effective of the drugs analysed for this condition.\"}, {\"question\": \"Which mutations of phopspholamban have been found in patients with cardiomyopathy?\", \"ideal_answer\": [\"PLN mutation R14del [or c.40_42delAGA(p.Arg14del)] was identified in 12 (12 %) ARVC patients and in 39 (15 %) DCM patients. Another PLN mutation is a T116G point mutation, substituting a termination codon for Leu-39 (L39stop), and it was identified in two families with hereditary heart failure. Hereditary mutants of phospholamban, such as Arg(9) to Cys, Arg(9) to Leu, Arg(9) to His, cause lethal, hereditary dilated cardiomyopathy.in specific, two patients presented a G-T missense mutation at the G26 nucleotide, which encodes an Arg-Leu substitution at codon 9 (R9L).One patient presented a G-A missense mutation at the same nucleotide, which encodes an Arg-His substitution at codon 9 (R9H). A missense mutation in PLN cytoplasmic domain (R9C) triggers dilated cardiomyopathy in humans, leading to premature death.\"], \"exact_answer\": [[\"c.40_42delAGA(p.Arg14del)\", \"PLN R14 del\"], [\"PLN L39stop\", \"PLN R39X\"], [\"Arg(9) to Cys\", \"R9C\"], [\"Arg(9) to Leu\", \"R9L\"], [\"Arg(9) to His\", \"R9H\"]], \"context\": \" In 19 of 142 patients (13%), the PLN founder mutation c.40_42delAGA (p.Arg14del) was identified. PLN mutation carriers more often had low-voltage electrocardiograms (p\\u00a0= 0.004), inverted T waves in leads V4 to V6 (p <0.001), and additional structural (p\\u00a0= 0.007) or functional (p\\u00a0= 0.017) left ventricular impairment, whereas desmosomal mutation carriers had more solitary right ventricular abnormalities.Recently, we showed that the c.40_42delAGA (p.Arg14del) mutation in the phospholamban (PLN) gene can be identified in 10-15\\u00a0% of Dutch patients with dilated cardiomyopathy or arrhythmogenic cardiomyopathy.PLN mutation R14del was identified in 12 (12 %) ARVC patients and in 39 (15 %) DCM patients. The PLN R14del founder mutation is present in a substantial number of patients clinically diagnosed with DCM or ARVC. R14del+ patients diagnosed with DCM showed an arrhythmogenic phenotype, and SCD at young age can be the presenting symptom. These findings support the concept of 'arrhythmogenic cardiomyopathy'.Herein we reveal mechanistic insight into how four hereditary mutants of phospholamban, Arg(9) to Cys, Arg(9) to Leu, Arg(9) to His, and Arg(14) deletion, alter regulation of SERCA. Deletion of Arg(14) disrupts the protein kinase A recognition motif, which abrogates phospholamban phosphorylation and results in constitutive SERCA inhibition. Mutation of Arg(9) causes more complex changes in function, where hydrophobic substitutions such as cysteine and leucine eliminate both SERCA inhibition and phospholamban phosphorylation, whereas an aromatic substitution such as histidine selectively disrupts phosphorylation. We demonstrate that the role of Arg(9) in phospholamban function is multifaceted: it is important for inhibition of SERCA, it increases the efficiency of phosphorylation, and it is critical for protein kinase A recognition in the context of the phospholamban pentamer. Given the synergistic consequences on contractility, it is not surprising that the mutants cause lethal, hereditary dilated cardiomyopathy.Phospholamban (PLN) is a key regulator of SR and cardiac function, and PLN mutations in humans have been associated with dilated cardiomyopathy (DCM). We previously reported the deletion of the highly conserved amino acid residue arginine 14 (nucleic acids 39, 40 and 41) in DCM patients.To assess the function of this mutant PLN, we introduced the PLN-R14Del in cardiac myocytes of the PLN null mouse.Thus, human PLN-R14Del is misrouted to the sarcolemma, in the absence of endogenous PLN, and alters NKA activity, leading to cardiac remodeling.Mutations in the gene encoding PLN have been associated with idiopathic dilated cardiomyopathy; however, no systematic search for PLN mutations in heart failure has been conducted.Two patients presented a G-T missense mutation at the G26 nucleotide, which encodes an Arg-Leu substitution at codon 9 (R9L).One patient presented a G-A missense mutation at the same nucleotide, which encodes an Arg-His substitution at codon 9 (R9H). The fourth affected patient presented a T-G nonsense mutation at the nucleotide 116, substituting a termination codon for Leu-39 (L39stop).The genetic analysis was focused on R9C mutation with the ability to block PLN phosphorylation leading to chronic inhibition of SERCA2a activity. Another analysed mutation causing the alteration of PLN level in cells was related to the substitution of a leucine residue at position 39 with a premature stop codon (L39X). A missense mutation in PLN cytoplasmic domain (R9C) triggers dilated cardiomyopathy in humans, leading to premature death. These effects are enhanced under oxidizing conditions, suggesting that oxidative stress may exacerbate the cardiotoxic effects of the PLN(R9C) mutant. We identified 1 family with a deletion of arginine 14 in the PLN. Interestingly, unlike other individuals reported with the identical PLN mutation, these individuals were not diagnosed with dilated cardiomyopathy until their seventh decade when they were only mildly symptomatic with congestive heart failure. Phospholamban R14 deletion results in late-onset, mild, hereditary dilated cardiomyopathy.Through genetic screening of dilated cardiomyopathy patients, we identified a previously uncharacterized deletion of arginine 14 (PLN-R14Del) in the coding region of the phospholamban (PLN) gene in a large family with hereditary heart failure. No homozygous individuals were identified. By middle age, heterozygous individuals developed left ventricular dilation, contractile dysfunction, and episodic ventricular arrhythmias, with overt heart failure in some cases.Transgenic mice overexpressing the mutant PLN-R14Del recapitulated human cardiomyopathy exhibiting similar histopathologic abnormalities and premature death.A mutation in the human phospholamban gene, deleting arginine 14, results in lethal, hereditary cardiomyopathy.Here, a T116G point mutation, substituting a termination codon for Leu-39 (L39stop), was identified in two families with hereditary heart failure. The heterozygous individuals exhibited hypertrophy without diminished contractile performance. Strikingly, both individuals homozygous for L39stop developed dilated cardiomyopathy and heart failure, requiring cardiac transplantation at ages 16 and 27. Here we report that an inherited human dilated cardiomyopathy with refractory congestive heart failure is caused by a dominant Arg --> Cys missense mutation at residue 9 (R9C) in phospholamban (PLN), a transmembrane phosphoprotein that inhibits the cardiac sarcoplasmic reticular Ca2+-adenosine triphosphatase (SERCA2a) pump. Transgenic PLN(R9C) mice recapitulated human heart failure with premature death. Cellular and biochemical studies revealed that, unlike wild-type PLN, PLN(R9C) did not directly inhibit SERCA2a. Rather, PLN(R9C) trapped protein kinase A (PKA), which blocked PKA-mediated phosphorylation of wild-type PLN and in turn delayed decay of calcium transients in myocytes. Phospholamban R14del mutation in patients diagnosed with dilated cardiomyopathy or arrhythmogenic right ventricular cardiomyopathy: evidence supporting the concept of arrhythmogenic cardiomyopathy.The PLN -42 C>G mutation was found in one patient with late onset familial apical hypertrophic cardiomyopathy\"}, {\"question\": \"Which are the supplemental antioxidant in athletes?\", \"ideal_answer\": [\"There are several antioxidant supplements belonging to different families, i.e. Vitamins, Polyphenols, alpha-lipoic acid, ubiquinones, n-3- polyunsaturated acids (PUFAs), minerals and others. Nonetheless the widespread use of these supplements, it is still debated their true usefulness, and it is not unanimously advised their use in athletes.\"], \"exact_answer\": [[\"Resveratrol\"], [\"Quercetin\"], [\"Vitamin C\"], [\"Biostimine\"], [\"Astaxanthin\"], [\"Melatonin\"], [\"coenzyme Q(10)\"], [\"Creatine\"], [\"Isoquercetin\"], [\"Epigallocatechin gallate\"], [\"n-3 fatty acids\"], [\"L. rhamnosus IMC 501\"], [\"L. paracasei IMC 502\"], [\"docosahexanoic acid (DHA)\"], [\"eicosapentanoic acid (EPA)\"], [\"Rhodiola Rosea\"], [\"Vitamin E\"], [\"Zinc\"], [\"whey and/or soy proteins\"]], \"context\": \"Although these supplementations are increasingly used by master athletes, very few data are available on their effects on oxidative stress, muscle recovery, and physical performance. The potential benefits of supplement use in athletes are thus questionable. Some studies indicate no benefits, while others highlight potential negative side effects of vitamin supplementation. These data indicate that RQ significantly reduces exercise-induced lipid peroxidation without associated changes in inflammation or plasma antioxidant status.Cr supplementation inhibited the increase of inflammation markers TNF-\\u03b1 and CRP, but not oxidative stress markers, due to acute exercise.Quercetin and vitamin C supplementation may not be beneficial in lipid profile improvement, although it may reduce induce muscle damage and body fat percent.Vitamins C and E supplementation had no significant effect on any of the studied parameters.Effects of the two treatments relative to placebo on mean performance in the incremental test and time trial were unclear, but runners faster by 1 SD of peak speed demonstrated a possible improvement on peak running speed with BC juice (1.9%; \\u00b12.5%). Training status correlates more strongly with antioxidant status than diet doesConsequently, we can conclude that Biostimine supplementation reduces the postexercise level of TBARS by increasing the antioxidant activity of plasma but has no effect on inflammatory markers.Supplementation with Asx could prevent exercise induced free radical production and depletion of non-enzymatic antioxidant defense in young soccer players.In conclusion these results indicated that treatment with melatonin in acute sports exercise reversed oxidative stress, improved defenses and lipid metabolism, which would result in an improvement in fitness.CoQ(10) supplementation before strenuous exercise decreases the oxidative stress and modulates the inflammatory signaling, reducing the subsequent muscle damage.Heretofore, Cr's positive therapeutic benefits in various oxidative stress-associated diseases have been reported in the literature and, recently, Cr has also been shown to exert direct antioxidant effects.These results indicate that Cr supplementation reduced oxidative DNA damage and lipid peroxidation induced by a single bout of RE.Nevertheless, based upon the growing evidence that many athletic populations are vitamin D deficient or insufficient, it is recommended that athletes monitor their serum vitamin D concentration and consult with their health care professional and/or nutritionist to determine if they would derive health benefits from vitamin D supplementation.This study examined the effects of 1,000 mg quercetin + 1,000 mg vitamin C (QC); 1,000 mg quercetin, 1,000 mg vitamin C, 400 mg isoquercetin, 30 mg epigallocatechin gallate, and 400 mg n-3 fatty acids (QFO)The two strains, L. rhamnosus IMC 501(\\u00ae) and L. paracasei IMC 502(\\u00ae), exert strong antioxidant activity. Athletes and all those exposed to oxidative stress may benefit from the ability of these probiotics to increase antioxidant levels and neutralize the effects of reactive oxygen species.These results indicate that supplementation with n-3 LCPUFAs significantly increased oxidative stress at rest and after a judo-training session.Rhodiola Rosea, is an adaptogen plant which has been reported to promote fatty acids utilisation, to ameliorate antioxidant function, and to improve body resistance to physical strenuous efforts. Supplementation with antioxidant vitamin E to athletes at 267 mg (400 IUs) or greater has been reported to reduce levels of LP associated with exercise.Results obtained at the end of the study indicate that zinc supplementation prevents production of free radicals by activating the antioxidant system.The significant increase in the UI of erythrocyte membranes indicates the potential for harm, because a high intake of PUFA might increase susceptibility to lipid peroxidation not counterbalanced by a higher increase in TAA. Adherence to the Mediterranean diet seems to be the better choice.A number of clinical trials have successfully been performed using whey and/or soy proteins in the treatment of many diseases. They both have antioxidant properties, which appears to be a factor in aerobic physical performance as well. The antioxidant effect of the two proteins is based on different mechanisms of action.\"}, {\"question\": \"List markers for autophagy.\", \"ideal_answer\": [\"Expression of LC3-II and BECN1 as well as SQSTM1 are used as markers of autophagy activity.\"], \"exact_answer\": [[\"LC3-II\", \"microtubule-associated protein 1 light chain 3\"], [\"BECN1\", \"beclin 1\"], [\"SQSTM1\", \"p62\", \"Sequestosome 1\"]], \"context\": \". Light chain 3/Atg8 as an autophagy marker increased conversion of microtubule-associated protein, 1A/1B-light chain 3, from its cytosolic form (LC3B-I) to its lipidated form (LC3B-II), increased Beclin-1 levels, and increased acridine orange staining as determined by flow cytometry analysis, providing further evidence of \\u03b3-tocotrienol-induced autophagy in these mammary cancer cell linesWestern blotting was used to detect hepatic microtubule-associated protein 1 light chain 3 (LC3)-II protein expression as a marker for autophagyautophagy marker LC3-IIautophagy marker LC3B the autophagy marker LC3autophagy marker molecules, p62 and LC3-II  LC3-I into LC3-II and has been confirmed in multiple mammalian cell lines with multiple autophagy markers  light chain 3 (LC3), which is an autophagy marker.LC3 (tfLC3) autophagy markerthe autophagy marker LC3,autophagy marker beclin 1 and microtubule-associated protein light chain 3 (LC3) were analyzed in control,  LC3-II levels (an autophagy marker);autophagy marker, microtubule-associated protein light chain3 (LC3)the autophagy marker LC3B-IIutophagy marker protein microtubule-associated protein 1 light chain 3 (LC3)autophagy marker LC3We used the expression of LC3-II and BECN1 as well as SQSTM1 as markers of autophagy activity. autophagy marker LC3\"}, {\"question\": \"Which syndromes are associated with mutations in the EZH2 gene?\", \"ideal_answer\": [\"EZH2 mutations that cause Weaver syndrome are primarily missense variants and the rare truncating mutations reported to date are in the last exon, suggesting that simple haploinsufficiency is unlikely to be generating the overgrowth phenotype although the exact mechanism has not yet been determined. Recent studies have shown that EZH2 mutations are often associated with RUNX1 mutations in MDS patients, although its pathological function remains to be addressed. These data show that mutations in EZH2 cause Weaver syndrome. The EZH2 gene is a homolog of the Drosophila Polycomb group (PcG) gene enhancer of zest, a crucial regulator of homeotic gene expression.\", \"Loss-of-function mutations of EZH2, a catalytic component of polycomb repressive complex 2 (PRC2), are observed in ~\\\\n10% of patients with myelodysplastic syndrome (MDS), but are rare in acute myeloid leukaemia (AML). Constitutional NSD1 and EZH2 mutations cause Sotos and Weaver syndromes respectively, overgrowth syndromes with considerable phenotypic overlap. Mutations at tyrosine 641 (Y641F, Y641N, Y641S and Y641H) in the SET domain of EZH2 have been identified in patients with certain subtypes of non-Hodgkin lymphoma (NHL). The EZH2 gene is involved in the pathogenesis of 7q35-q36 aberrations in myeloid leukaemia.\"], \"exact_answer\": [[\"myelodysplastic syndrome (MDS)\"], [\"Acquired aplastic anemia (AA)\"], [\"Weaver syndrome\"], [\"non-Hodgkin lymphoma (NHL)\"], [\"myeloid leukaemia\"]], \"context\": \"Loss-of-function mutations of EZH2, a catalytic component of polycomb repressive complex 2 (PRC2), are observed in ~\\\\n10% of patients with myelodysplastic syndrome (MDS), but are rare in acute myeloid leukaemia (AML)We describe the use of an oligo-SNP array for genomic profiling of aCNA and cnLOH, together with sequence analysis of recurrently mutated genes, in a patient with myelodysplastic syndrome (MDS) presenting with normal karyotype and FISH result Conditionally deleting Ezh2 in mature T cells dramatically reduced the production of BM-destructive Th1 cells in vivo, decreased BM-infiltrating Th1 cells, and rescued mice from BM failure.Acquired aplastic anemia (AA) is a potentially fatal bone marrow (BM) failure syndromeConstitutional NSD1 and EZH2 mutations cause Sotos and Weaver syndromes respectively, overgrowth syndromes with considerable phenotypic overlap.EZH2 mutations that cause Weaver syndrome are primarily missense variants and the rare truncating mutations reported to date are in the last exon, suggesting that simple haploinsufficiency is unlikely to be generating the overgrowth phenotype although the exact mechanism has not yet been determinedWeaver syndrome and EZH2 mutationsIn 2011, mutations in the histone methyltransferase, EZH2, were shown to cause Weaver syndromeThe identification of an EZH2 mutation can therefore provide an objective means of confirming a subtle presentation of Weaver syndrome and/or distinguishing Weaver and Sotos syndromes. Mutations in EZH2 cause Weaver syndromeThese data show that mutations in EZH2 cause Weaver syndromeThe EZH2 mutation spectrum in Weaver syndrome shows considerable overlap with the inactivating somatic EZH2 mutations recently reported in myeloid malignanciesEZH2 mutations as the cause of Weaver syndrome and provide further links between histone modifications and regulation of human growth.Mutations at tyrosine 641 (Y641F, Y641N, Y641S and Y641H) in the SET domain of EZH2 have been identified in patients with certain subtypes of non-Hodgkin lymphoma (NHL)The EZH2 gene was previously reported to be located on chromosome 21q22 and was proposed as a candidate gene for some characteristics of the Down syndrome phenotypeRecent extensive mutation analyses of the myeloid malignancies have revealed that inactivating somatic mutations in PcG genes such as EZH2 and ASXL1 occur frequently in patients with myelodysplastic disorders including myelodysplastic syndromes (MDSs) and MDS/myeloproliferative neoplasm (MPN) overlap disorders (MDS/MPN).EZH2 is frequently overexpressed and considered to be an oncogene in cancers; nevertheless, EZH2 is considered as a candidate tumor suppressor gene in MDS due to EZH2 mutations associated with poor survival.In 2011, mutations in the histone methyltransferase, EZH2, were shown to cause Weaver syndrome.Among them, Sotos and Weaver syndromes are clinically well defined and due to heterozygous mutations in NSD1 and EZH2, respectively.Weaver syndrome and EZH2 mutations: Clarifying the clinical phenotype.Constitutional NSD1 and EZH2 mutations cause Sotos and Weaver syndromes respectively, overgrowth syndromes with considerable phenotypic overlap.The identification of an EZH2 mutation can therefore provide an objective means of confirming a subtle presentation of Weaver syndrome and/or distinguishing Weaver and Sotos syndromes.We postulated that mutations in writers of these two chromatin marks could cause overgrowth conditions, resembling Sotos or Weaver syndromes, in patients with no NSD1 or EZH2 abnormalities.Recent extensive mutation analyses of the myeloid malignancies have revealed that inactivating somatic mutations in PcG genes such as EZH2 and ASXL1 occur frequently in patients with myelodysplastic disorders including myelodysplastic syndromes (MDSs) and MDS/myeloproliferative neoplasm (MPN) overlap disorders (MDS/MPN). Somatic mutations of epigenetic gene regulators are common in patients with myelodysplastic syndromes (MDS) and correlate with some clinical and laboratory features. Recently, the advent of next generation sequencing (NGS) techniques has helped identify somatic gene mutations in 75-80% of MDS, that cluster mainly in four functional groups, i.e. cytokine signaling (RAS genes), DNA methylation, (TET2, IDH1/2, DNMT3a genes) histone modifications (ASXL1 and EZH2 genes), and spliceosome (SF3B1 and SRSF2 genes) along with mutations of RUNX1 and TP 53 genes.Constitutional NSD1 and EZH2 mutations cause Sotos and Weaver syndromes respectively, overgrowth syndromes with considerable phenotypic overlap. NSD1 mutations that cause Sotos syndrome are loss-of-function, primarily truncating mutations or missense mutations at key residues in functional domains.EZH2 is considered as a candidate tumor suppressor gene in MDS due to EZH2 mutations associated with poor survival.Considerable phenotypic overlap between Sotos and Weaver syndromes is also evident. The identification of an EZH2 mutation can therefore provide an objective means of confirming a subtle presentation of Weaver syndrome and/or distinguishing Weaver and Sotos syndromes.EZH2 is frequently overexpressed and considered to be an oncogene in cancers; nevertheless, EZH2 is considered as a candidate tumor suppressor gene in MDS due to EZH2 mutations associated with poor survival.Constitutional NSD1 and EZH2 mutations cause Sotos and Weaver syndromes respectively, overgrowth syndromes with considerable phenotypic overlap.The deletion involved multiple genes, including two tumor suppressor candidate genes (CTNNA1 and HSPA9) that are associated with MDS/AML.Somatic mutations of epigenetic gene regulators are common in patients with myelodysplastic syndromes (MDS) and correlate with some clinical and laboratory features.\"}, {\"question\": \"Which databases exist for experimentally determined topologies of \\u03b1-helical transmembrane proteins ?\", \"ideal_answer\": [\"ExTopoDB and TMPDB.\"], \"exact_answer\": [[\"ExTopoDB\"], [\"TMPDB\"]], \"context\": \"ExTopoDB: a database of experimentally derived topological models of transmembrane proteinsExTopoDB is a publicly accessible database of experimentally derived topological models of transmembrane proteins. It contains information collected from studies in the literature that report the use of biochemical methods for the determination of the topology of \\u03b1-helical transmembrane proteins. Transmembrane protein topology is highly important in order to understand their function and ExTopoDB provides an up to date, complete and comprehensive dataset of experimentally determined topologies of \\u03b1-helical transmembrane proteins. Topological information is combined with transmembrane topology prediction resulting in more reliable topological models. AVAILABILITY: http://bioinformatics.biol.uoa.gr/ExTopoDBTMPDB: a database of experimentally-characterized transmembrane topologiesTMPDB is a database of experimentally-characterized transmembrane (TM) topologies. TMPDB release 6.2 contains a total of 302 TM protein sequences, in which 276 are alpha-helical sequences, 17 beta-stranded, and 9 alpha-helical sequences with short pore-forming helices buried in the membrane. The TM topologies in TMPDB were determined experimentally by means of X-ray crystallography, NMR, gene fusion technique, substituted cysteine accessibility method, N-linked glycosylation experiment and other biochemical methods. TMPDB would be useful as a test and/or training dataset in improving the proposed TM topology prediction methods or developing novel methods with higher performance, and as a guide for both the bioinformaticians and biologists to better understand TM proteins. TMPDB and its subsets are freely available at the following web site: http://bioinfo.si.hirosaki-u.ac.jp/~TMPDB/Transmembrane protein topology is highly important in order to understand their function and ExTopoDB provides an up to date, complete and comprehensive dataset of experimentally determined topologies of \\u0436\\u2534-helical transmembrane proteins.TMPDB: a database of experimentally-characterized transmembrane topologies.TMPDB is a database of experimentally-characterized transmembrane (TM) topologies.The TM topologies in TMPDB were determined experimentally by means of X-ray crystallography, NMR, gene fusion technique, substituted cysteine accessibility method, N-linked glycosylation experiment and other biochemical methods.Transmembrane protein topology is highly important in order to understand their function and ExTopoDB provides an up to date, complete and comprehensive dataset of experimentally determined topologies of \\u03b1-helical transmembrane proteins.Transmembrane protein topology is highly important in order to understand their function and ExTopoDB provides an up to date, complete and comprehensive dataset of experimentally determined topologies of \\ufffd-helical transmembrane proteins.UNLABELLED: ExTopoDB is a publicly accessible database of experimentally derived topological models of transmembrane proteins.The TM topologies in TMPDB were determined experimentally by means of X-ray crystallography, NMR, gene fusion technique, substituted cysteine accessibility method, N-linked glycosylation experiment and other biochemical methods.Transmembrane protein topology is highly important in order to understand their function and ExTopoDB provides an up to date, complete and comprehensive dataset of experimentally determined topologies of \\u03b1-helical transmembrane proteinsThe TM topologies in TMPDB were determined experimentally by means of X-ray crystallography, NMR, gene fusion technique, substituted cysteine accessibility method, N-linked glycosylation experiment and other biochemical methodsTMPDB is a database of experimentally-characterized transmembrane (TM) topologiesExTopoDB: a database of experimentally derived topological models of transmembrane proteins.ExTopoDB is a publicly accessible database of experimentally derived topological models of transmembrane proteins\"}, {\"question\": \"Which are the roles of chromatin compartments in the eukaryotic nucleus?\", \"ideal_answer\": [\"The complexity in composition and function of the eukaryotic nucleus is achieved through its organization in specialized nuclear compartments. Chromosome conformation capture approaches have shown that interphase chromatin is partitioned into spatially segregated Mb-sized compartments and sub-Mb-sized topological domains. This compartmentalization is thought to facilitate the matching of genes and regulatory elements. Cohesin-based chromatin interactions enable regulated gene expression within preexisting architectural compartments. Therefore, concentrating proteins needed to perform different steps of RNA synthesis within specialized nuclear compartments is important in orchestrating events required for efficient gene expression.\"], \"exact_answer\": [[\"providing complex regulation\"], [\"facilitating matching genes and regulatory elements\"], [\"efficient trancription\"]], \"context\": \"concentrating proteins needed to perform different steps of RNA synthesis within specialized nuclear compartments will be important in orchestrating events required for efficient gene expression.Chromatin domains and nuclear compartments: establishing sites of gene expression in eukaryotic nuclei.Using the adult bovine lens as a model system, nuclear changes accompanying denucleation are described with particular emphasis on the lamina, nucleolar and coiled body compartments in lens nuclei.Changes in the nucleolar and coiled body compartments precede lamina and chromatin reorganization during fibre cell denucleation in the bovine lensPrior to chromatin condensation, coilin redistributed to the nucleolar compartment and was absent from nuclei where chromatin had begun to condense.These constraints reflect the physical attachment of chromatin to nuclear compartments or steric impairment caused by local ultrastructureA number of these proteins accumulate in viral replication compartments in the infected cell nucleus, indicating that these proteins may have a role in viral replication. Nuclear marginalization of host cell chromatin associated with expansion of two discrete virus-induced subnuclear compartments during baculovirus infectionIn the late stage of infection, however, the peristromal region (PR), another virus-induced subnuclear compartment, was also excluded from the chromatin-localizing area.This correlation between compartmentalization and chromatin exclusion suggests the possibility that a chromatin-exclusive property of viral molecules, at least in part, supports nuclear compartmentalization of virus-infected cells.Chromatin modifications in hematopoietic multipotent and committed progenitors are independent of gene subnuclear positioning relative to repressive compartmentsTo further clarify the contribution of nuclear architecture in the regulation of gene expression patterns during differentiation of human multipotent cells, we analyzed expression status, histone modifications, and subnuclear positioning relative to repressive compartments, of hematopoietic loci in multipotent and lineage-committed primary human hematopoietic progenitors.We report here that positioning of lineage-affiliated loci relative to pericentromeric heterochromatin compartments (PCH) is identical in multipotent cells from various origins and is unchanged between multipotent and lineage-committed hematopoietic progenitors. The complexity in composition and function of the eukaryotic nucleus is achieved through its organization in specialized nuclear compartments.Cohesin-based chromatin interactions enable regulated gene expression within preexisting architectural compartments.Chromosome conformation capture approaches have shown that interphase chromatin is partitioned into spatially segregated Mb-sized compartments and sub-Mb-sized topological domains. This compartmentalization is thought to facilitate the matching of genes and regulatory elements,we find that architectural compartments are maintained in noncycling mouse thymocytes after genetic depletion of cohesin in vivoOur findings indicate that cohesin-mediated long-range interactions facilitate discrete gene expression states within preexisting chromosomal compartmentsHistone posttranslational modifications mediate establishment of structurally and functionally distinct chromatin compartments of eukaryotic nuclei.Recent studies employing chromatin conformation capture techniques indicate that Hox clusters adopt a remarkable spatial configuration, in which active and inactive genes are segregated into two distinct chromatin compartments.Recent Hi-C mapping has unveiled substructure within chromatin compartments called topologically associating domains (TADs) that are largely conserved in their positions between cell types and are similar in size to replication domains.Interaction of SP100 with HP1 proteins: a link between the promyelocytic leukemia-associated nuclear bodies and the chromatin compartment.We propose that basically three nuclear compartments exist, an &quot;open&quot; higher-order chromatin compartment with chromatin domains containing active genes, a &quot;closed&quot; chromatin compartment comprising inactive genes, and an interchromatin domain (ICD) compartment (Cremer et al., 1993; Zirbel et al., 1993) that contains macromolecular complexes for transcription, splicing, DNA replication, and repair. Genes in &quot;open,&quot; but not in &quot;closed&quot; higher-order chromatin compartments have access to transcription and splicing complexes located in the ICD compartment.Genes in &quot;open,&quot; but not in &quot;closed&quot; higher-order chromatin compartments have access to transcription and splicing complexes located in the ICD compartment. Chromatin domains that build the &quot;open&quot; chromatin compartment are organized in a way that allows the direct contact of genes and nascent RNA to transcription and splicing complexes, respectively, preformed in the ICD compartment.We propose that basically three nuclear compartments exist, an &quot;open&quot; higher-order chromatin compartment with chromatin domains containing active genes, a &quot;closed&quot; chromatin compartment comprising inactive genes, and an interchromatin domain (ICD) compartment (Cremer et al.\"}, {\"question\": \"What are the biological roles proposed for proteins containing the SPRY domain?\", \"ideal_answer\": [\"defence against retroviral infection\\ninnate and adaptative immunity\\nvesicular trafficking\\nneural differentiation\\nembryonic development\"], \"exact_answer\": [[\"defence against retroviral infection\"], [\"immunity\"], [\"vesicular trafficking\"], [\"neural differentiation\"], [\"embryonic development\"]], \"context\": \"monkey TRIM5\\u03b1 SPRY domain recognizes multiple epitopes that span several capsid monomers on the surface of the HIV-1 mature viral coreThese properties, which may enhance resistance of TRIM5\\u03b1 to capsid mutations, result in relatively low affinity of the individual SPRY domains for the capsidRIM5\\u03b1 is a retroviral restriction factor, in which the B30.2 (SPRY) and coiled-coil domains cooperate to determine the specificity of TRIM5\\u03b1-mediated capture of retroviral capsidsfunction of the SPRY/B30.2 domain proteins involved in innate immunitySPRY domain is a protein interaction module found in 77 murine and ~100 human proteins, and is implicated in important biological pathways, including those that regulate innate and adaptive immunityTRIM5\\u03b1) is a potent antiviral protein that restricts infection by HIV-1 and other retroviruses TRIM5\\u03b1 recognizes the lattice of the retrovirus capsid through its B30.2 (PRY/SPRY) domain in a species-specific mannerbinding of the PRY-SPRY domain from the TRIM50 C-terminal region to phosphatidylinositol species, suggesting that TRIM50 is involved in vesicular dynamics by sensing the phosphorylated state of phosphoinositol lipidsTRIM50 seems to play an essential role in tubulovesicular dynamicsTRIM67 regulates Ras signaling via degradation of 80K-H, leading to neural differentiation including neuritogenesisTRIM67, which has a TRIM motif, an FN3 domain and a SPRY domain, is highly expressed in the cerebellumVasa coding region is sufficient for its selective enrichment and find that gustavus, the B30.2/SPRY and SOCS box domain geneWe propose that Gustavus has a conserved, positive regulatory role in Vasa protein accumulation during embryonic developmenttwo zebrafish genes, SSB-1 and SSB-4 (SPRY domain SOCS box proteinsWe hypothesize that SSB-4 plays a role in the early development of germ cellsSPRY domain of Pyrin, mutated in familial Mediterranean fever patients, interacts with inflammasome components and inhibits proIL-1beta processingTRIM proteins may be conducive to the convergent evolution of virus-restricting factors.SPRY and B30.2 protein domains. Evolution of a component of immune defence. The combination of SPRY and PRY to produce B30.2 domains may have been selected and maintained as a component of immune defence evolution of primate TRIM5alpha, a gene restricting HIV-1 infectionThe SPRY domain of TRIM5alpha, which may be responsible for recognition of incoming viral capsids showed higher nonsynonymous/synonymous substitution ratios than the non-SPRY domain, indicating that the adaptive evolution of TRIM5alpha in primates might be an innate strategy developed in defending retrovirus infection during primate evolutioThe results are consistent with a role for TRIM5alpha in innate immunity against retrovirusesB30.2(SPRY) domain of the retroviral restriction factor TRIM5alphaprimate TRIM5alpha identifies a critical species-specific retroviral restriction domainBy using functional studies of chimeric TRIM5alpha genes, we show that this patch is generally essential for retroviral restriction and is responsible for most of the species-specific antiretroviral restriction activitySPRY protein domainHeterozygotes for gus or a deletion including gus produce embryos with fewer pole cells and posterior patterning defectsSPRY-domain and SOCS-box containing protein, GUSTAVUS.\"}, {\"question\": \"The protein NONO forms heterodimers. With which proteins?\", \"ideal_answer\": [\"The protein NONO forms heterodimers with PSPC1, SFPQ.\"], \"exact_answer\": [[\"PSPC1\"], [\"SFPQ\"]], \"context\": \"The paraspeckle component 1 (PSPC1) and non-POU-domain-containing octamer-binding protein (NONO) heterodimer is an essential structural component of paraspeckles,PSPC1-NONO heterodimer.Crystallization of a paraspeckle protein PSPC1-NONO heterodimerdifficult heterodimeric complex of two human proteins, paraspeckle component 1 (PSPC1) and non-POU domain-containing octamer-binding protein (NONO), that are involved in gene regulation and the structural integrity of nuclear bodies termed paraspeckles is described.SFPQ (PSF) and NONO (p54) are nuclear proteins that interact with each other and have diverse roles in nucleic acids metabolism. The SFPQ/NONO heterodimer was previously found to enhance DNA strand break rejoining in vitro.We identified a heterodimer, p54nrb and PSF,P54nrb forms a heterodimer with PSP1 that localizes to paraspeckles in an RNA-dependent manner.e demonstrated that the PSF heterodimer partner, p54nrb (non-POU-domain-containing, octamer binding protein), can also function as a transcription corepressor, independent of PSF.The PSPC1/NONO heterodimer has a right-handed antiparallel coiled-coilStructure of the heterodimer of human NONO and paraspeckle protein component 1\"}, {\"question\": \"List human proteins that are subject to a dimer-to-tetramer transition.\", \"ideal_answer\": [\"GAC\\nSHMT2\\nAMPAR\\nOrai1\\nOrai3\"], \"exact_answer\": [[\"GAC\"], [\"SHMT2\"], [\"AMPAR\"], [\"Orai1\"], [\"Orai3\"]], \"context\": \" the ability of GAC to undergo the dimer-to-tetramer transition necessary for enzyme activation.while SHMT2 undergoes a dimer-to-tetramer transition upon PLP binding.AMPARs that is required for the critical dimer-to-tetramer transition.We conclude that the human Orai1 and Orai3 channels undergo a dimer-to-tetramer transition to form a Ca(2+)-selective pore during store-operated activation and that Orai3 forms a dimeric nonselective cation pore upon activation by 2-APB.the conformational requirement for the dimer-to-tetramer transition during the maturation of AMPA-Rs. \"}, {\"question\": \"which mutations of phospholamban gene have been found to cause hypertrophic cardiomyopathy?\", \"ideal_answer\": [\"The following mutations of the phospholamban gene have been found to be associated with hypertrophic cardiomyopathy: PLN L39X nonsense mutation; PLN Leu39Ter; PLN -42 C>G  and PLN -77A-->G\"], \"exact_answer\": [[\"PLN L39X nonsense mutation\"], [\"PLN Leu39Ter\"], [\"PLN -42 C>G\"], [\"PLN -77A-->G\"]], \"context\": \"one L39X nonsense mutation was identified in 1 of 1,064 HCM proband cases with a family history of HCM, previously found to be negative for the current HCM genetic test panel. This mutation cosegregated with incidence of HCM in a multigenerational family.The PLN -42 C>G mutation was found in one patient with late onset familial apical hypertrophic cardiomyopathy. The PLN -42 C>G mutation is associated with a benign form of apical hypertrophic cardiomyopathy in this family, though the presence of a healthy adult carrier suggests that other genetic and environmental factors could be involved. We found a heterozygous single nucleotide transition from A to G at -77-bp upstream of the transcription start site in the phospholamban promoter region of one patient with familial hypertrophic cardiomyopathy.Using neonatal rat cardiomyocytes, the mutation, -77A-->G, increased the phospholamban promoter activity. No nucleotide change in the phospholamban coding region was found in 99 patients with cardiomyopathy. We suspect that the mutation plays an important role in the development of hypertrophic cardiomyopathy.Two novel mutations in the CALR3 gene were identified (Lys82Arg, Arg73Gln) and one truncation mutation in the PLN gene (Leu39Ter). \"}, {\"question\": \"Abnormalities in which chromosomes were linked to the Moyamoya disease?\", \"ideal_answer\": [\"chromosomes 3, 6, 8, 12, 15, 17, 21, X and Y were implicated in the Moyamoya disease.\"], \"exact_answer\": [[\"3\"], [\"6\"], [\"8\"], [\"12\"], [\"15\"], [\"17\"], [\"21\"], [\"X\"], [\"Y\"]], \"context\": \"Complex chromosome rearrangement of 6p25.3->p23 and 12q24.32->qter in a child with moyamoya.Routine karyotype demonstrated extra chromosomal material on 6p. Single nucleotide polymorphism microarray revealed a previously unreported complex de novo genetic rearrangement involving subtelomeric segments on chromosomes 6p and 12q.In terms of the genetics, recent literature suggests a low penetrance autosomal dominant or polygenic mode of transmission involving chromosomes 3, 6, 8, 12, and 17 for familial MMD. Furthermore, the current literature on familial MMD has pointed to a low penetrance autosomal dominant or polygenic mode of transmittance at loci on chromosomes 3, 6, 8, 12, and 17.Here, we report on three unrelated families affected with an X-linked moyamoya syndrome characterized by the association of a moyamoya angiopathy, short stature, and a stereotyped facial dysmorphism. We show that this syndromic moyamoya is caused by Xq28 deletions removing MTCP1/MTCP1NB and BRCC3. A genome-wide association study of 785,720 single-nucleotide polymorphisms (SNPs) was performed, comparing 72 Japanese MMD patients with 45 Japanese controls and resulting in a strong association of chromosome 17q25-ter with MMD risk. This result was further confirmed by a locus-specific association study using 335 SNPs in the 17q25-ter region. Investigations included neuroimaging, cardiologic and ophthalmologic evaluation, hormonal testing, hemoglobin electrophoresis, chromosomal karyotyping, muscle biopsy for morphology, immunohistochemistry and enzyme assays, mtDNA mutation screening, and haplotype analysis of 2 loci previously linked to moyamoya, on chromosomes 10 (ACTA2) and 17.These data strongly suggest that this family is affected by a hereditary moyamoya multisystem disorder with X-linked recessive pattern of inheritance.Her features were consistent with TS and chromosome analysis revealed mosaicism in which 17% of the cells showed a pseudoisodicentric Y chromosome: 45,X (25)/46,X psu idic (Y)(11.2) (5). The small number of cases does not allow detailed analysis other than noting patient age (two older than 40 years), karyotype (two others associated with isochrome Xq), and associated cardiac risk factors (one with BAV). Studies have shown a possible genetic association of MMD linked to chromosome 17 in Japanese cases as well as in cases found in other demographics. The gene responsible for NF1 is located on the chromosome region 17q11.2 and for familial moyamoya disease on chromosome 17q25. Autosomal dominant moyamoya disease maps to chromosome 17q25.3.Under both classifications, significant evidence of linkage was only observed on chromosome 17q25.3, with maximum multipoint logarithm of odds (lod) scores of 6.57 (under the narrow classification) and 8.07 (under the broad classification) at D17S704. Haplotype analysis revealed segregation of a disease haplotype in all families but one, and informative crossovers enabled mapping of the MMD locus to a 3.5-Mb region between D17S1806 and the telomere of 17q, encompassing 94 annotated genes. CONCLUSIONS: Our data suggest that there is a major gene locus for autosomal dominant moyamoya disease on chromosome 17q25.3.However, the mitochondrial DNA and Y chromosomal genotype showed that affected members had the same sequence of the Mitochondrial 3 portion of D-loop with Japanese patients. Smith-Magenis syndrome and Moyamoya disease in a patient with del(17)(p11.2p13.1).We have employed a combination of aCGH, FISH, and MLPA to characterize an approximately 6.3 Mb deletion spanning chromosome region 17p11.2-p13.1 in this patient, with the proximal breakpoint within the RAI1 gene. Moyamoya syndrome in a child with trisomy 12p syndrome.This report presents the first case of moyamoya syndrome with trisomy 12p with a familial pericentric inversion of chromosome 12. She had been prenatally diagnosed with trisomy 12p with a familial pericentric inversion of chromosome 12 originating from her mother. She manifested developmental delay and some dysmorphic features of the face and limbs compatible with the clinical features of trisomy 12p.The genes encoding tissue inhibitor of metalloproteinase (TIMP) 4 and TIMP2 span chromosomes 3p24.2-p26 and 17q25, respectively, which are the locations of familial moyamoya disease (FMMD) genes. We postulate that a protein encoded on chromosome 21 may be related to the pathogenesis of Down's syndrome with moyamoya syndrome.[Molecular screening for moyamoya disease by use of expressed sequence tag on chromosome 3p]. To identify the expressed sequence tags (ESTs) with monoallelic expressions on chromosome 3, we used mouse A9 hybrid cells having human chromosome 3. Two ESTs showed only maternal expression in mouse A9 hybrid cells, and four showed non-expression in the lymphocytes derived from moyamoya patients. A novel susceptibility locus for moyamoya disease on chromosome 8q23.Previous linkage studies have indicated that susceptibility loci for the disease are located on chromosomes 3p, 6q, and 17q. In the present study, we searched for loci linked to the disease in 12 Japanese families using 428 microsatellite markers and found significant evidence for linkage to 8q23 [maximum LOD score (MLS) of 3.6] and suggestive evidence for linkage to 12p12 (MLS=2.3). Considering the function, we selected nine genes as candidates from a total of 65 genes identified in the 9-cM region of D17S785-D17S836 in chromosome 17q25, and performed sequence analysis on the DNA samples obtained from a pedigree of familial moyamoya disease, which showed a complete linkage to the region by a haplotype analysis. Linkage of familial moyamoya disease (spontaneous occlusion of the circle of Willis) to chromosome 17q25.The characteristic lesions of moyamoya disease are occasionally seen in neurofibromatosis type 1, of which the causative gene (NF1) has been assigned to chromosome 17q11.2.CONCLUSIONS: A gene for familial moyamoya disease is located on chromosome 17q25.Recent genetic studies suggest some responsible genetic foci in chromosomes 3, 6 and 17.To further specify the genetic component of moyamoya disease, a linkage study of moyamoya disease using markers on chromosome 6, where the HLA gene is located, was performed. Mapping of a familial moyamoya disease gene to chromosome 3p24.2-p26.A linkage was found between the disease and markers located at 3p24.2-26.We postulate that a protein encoded on chromosome 21 may be related to the pathogenesis of moyamoya disease. The diagnosis of PWS was confirmed genetically by the method of fluorescence in situ hybridization which showed the deletion of chromosome 15. \"}, {\"question\": \"Which drugs are included in the FEC-75 regimen?\", \"ideal_answer\": [\"Fluorouracil, epirubicin, and cyclophosphamide are included in the FEC-75 regimen. This chemotherapy regiment is used for breast cancer treatment.\"], \"exact_answer\": [[\"fluorouracil\"], [\"epirubicin\"], [\"cyclophosphamide\"]], \"context\": \"Fluorouracil, epirubicin, and cyclophosphamide (FEC-75) followed by paclitaxel plus trastuzumab versus paclitaxel plus trastuzumab followed by FEC-75 plus trastuzumab as neoadjuvant treatment for patients with HER2-positive breast cancer (Z1041): a randomised, controlled, phase 3 trial.Patients randomly assigned to sequential treatment received fluorouracil 500 mg/m(2), epirubicin 75 mg/m(2), and cyclophosphamide 500 mg/m(2) (FEC-75) on day 1 of a 21-day cycle for four cycles followed by paclitaxel 80 mg/m(2) and trastuzumab 2 mg/kg (after a 4 mg/kg loading dose) once per week for 12 weeks, while those randomly assigned to the concurrent treatment group received paclitaxel and trastuzumab once per week for 12 weeks followed by four cycles of FEC-75 (on day 1 of each 21-day cycle) and once-weekly trastuzumab, in the same doses as the sequential group.In May 2008, the patient underwent preoperative chemotherapy consisting of 4 courses of FEC 75(fluororracil, epirubicin, and cyclophosphamide).Stage II-IIIC HER2-positive breast cancer patients, including inflammatory disease, were treated with weekly-trastuzumab for 24 weeks administered concurrently with all primary chemotherapy containing paclitaxel (80 mg/m(2)) for 12 weeks and 4 cycles of FEC-75 (fluorouracil 500 mg/m(2), epirubicine 75 mg/m(2), and cyclophosphamide 500 mg/m(2)) followed by surgery. Further improvements can be achieved with dose-dense regimens, but densification of fluorouracil/epirubicin/cyclophosphamide (FEC) has proved difficult, with FEC(60) providing little benefit over standard chemotherapy and FEC(100) associated with toxicity. We investigated the feasibility of two intermediate dose-dense FEC regimens. Patients were randomised to six cycles of FEC(75) or FEC(90), with all three drugs given on day 1 of each 14-day cycle. BACKGROUND: A previously published prospective randomized phase 3 trial showed that administration of 24 weeks of primary systemic chemotherapy (PST) with paclitaxel and FEC(75) (fluorouracil, epirubicin, cyclophosphamide) concurrently with trastuzumab in patients with HER2-positive primary breast cancer resulted in a 60% pathologic complete response rate (PCR) with no associated severe cardiac toxicity. Maintenance chemotherapy was FEC 75 (fluorouracil 500 mg/m(2), epirubicin 75 mg/m(2), and cyclophosphamide 500 mg/m(2) on Day 1 every 21 days for 4 cycles). The French Epirubicin Study Group carried out a randomized trial comparing epirubicin alone 75 mg/m2 with fluorouracil (5FU) 500 mg/m2, cyclophosphamide 500 mg/m2, and epirubicin 50 mg/m2 (FEC 50) and 5FU 500 mg/m2, cyclophosphamide 500 mg/m2, and epirubicin 75 mg/m2 (FEC 75) as first treatment for advanced breast cancer patients.Patients were randomised to 1 of 3 treatment groups: Group A (n = 207) received FEC 50 (fluorouracil 500 mg/m2, epirubicin 50 mg/m2 plus cyclophosphamide 500 mg/m2) every 21 days for 6 cycles; Group B (n = 193) received FEC 50 every 21 days for 3 cycles; Group C (n = 195) received FEC 75 (fluorouracil 500 mg/m2, epirubicin 75 mg/m2 plus cyclophosphamide 500 mg/m2) every 21 days for 3 cycles.Sixty chemotherapy-naive breast cancer patients of 30 to 71 years in age, P.S. = 0-1, receiving 5-fluorouracil-epirubicin-cyclophosphamide (FEC 75) q 21 days or cyclophosphamide-methotrexate-5-fluorouracil (CMF) or 120 mg/m2 epirubicin or high dose mitomycin-methotrexate-mitoxantrone (MMM) q 14 days (+ G-CSF) or 100 mg/m2 epirubicin (+ G-CSF) were randomized to receive, 15 min before chemotherapy, 8 mg i.v. bolus of ondansetron or 3 mg i.v. granisetron or 5 mg i.v. tropisetron and no further antiemetic therapy in the following days.PATIENTS AND METHODS: Four hundred seventeen anthracycline-naive MBC patients were randomized to receive one of the following regimens: arm A: 11 cycles of fluorouracil 500 mg/m(2), epirubicin 75 mg/m(2), and cyclophosphamide 500 mg/m(2) (FEC 75) every 21 days; arm B: four cycles of FEC 100 (same regimen but with epirubicin 100 mg/m(2)) then eight cycles of FEC 50 (epirubicin 50 mg/m(2)); and arm C: four cycles of FEC 100 then restart the same regimen at disease progression in case of prior response or stabilization.The aim of this study, using a Fleming single-stage design, was to explore the efficacy and safety of Taxotere 100 x mg x m(-2) docetaxel and FEC 75 cyclophosphamide 500 mg x m(-2), fluorouracil 500 x mg x m(-2) and epirubicin 75 mg x m(-2), in alternating and sequential schedules for the first-line treatment of metastatic breast cancer.PATIENTS AND METHODS: Between 1986 and 1990, 621 patients with operable breast cancer were randomly assigned to receive fluorouracil (Roche SA, Basel, Switzerland) 500 mg/m2, epirubicin (Pharmacia SA, Milan, Italy) 50 mg/m2, and cyclophosphamide (Asta Medica AG, Frankfurt, Germany) 500 mg/m2 every 21 days (FEC 50) for six cycles (6 FEC 50); FEC 50 for three cycles (3 FEC 50); or the same regimen with epirubicin 75 mg/m2 (FEC 75) for three cycles (3 FEC 75).We studied the feasibility of FEC (75) (fluorouracil: 500 mg/m(2), epirubicin: 75 mg/m(2), and cyclophosphamide:500 mg/m(2), q 3 w, 6 cycles) as adjuvant chemotherapy for 59 primary breast cancer patients.Between 1986 and 1990, 621 patients with operable breast cancer were randomly assigned to receive fluorouracil (Roche SA, Basel, Switzerland) 500 mg/m2, epirubicin (Pharmacia SA, Milan, Italy) 50 mg/m2, and cyclophosphamide (Asta Medica AG, Frankfurt, Germany) 500 mg/m2 every 21 days (FEC 50) for six cycles (6 FEC 50); FEC 50 for three cycles (3 FEC 50); or the same regimen with epirubicin 75 mg/m2 (FEC 75) for three cycles (3 FEC 75)Four hundred seventeen anthracycline-naive MBC patients were randomized to receive one of the following regimens: arm A: 11 cycles of fluorouracil 500 mg/m(2), epirubicin 75 mg/m(2), and cyclophosphamide 500 mg/m(2) (FEC 75) every 21 days; arm B: four cycles of FEC 100 (same regimen but with epirubicin 100 mg/m(2)) then eight cycles of FEC 50 (epirubicin 50 mg/m(2)); and arm C: four cycles of FEC 100 then restart the same regimen at disease progression in case of prior response or stabilization.The French Epirubicin Study Group carried out a randomized trial comparing epirubicin alone 75 mg/m2 with fluorouracil (5FU) 500 mg/m2, cyclophosphamide 500 mg/m2, and epirubicin 50 mg/m2 (FEC 50) and 5FU 500 mg/m2, cyclophosphamide 500 mg/m2, and epirubicin 75 mg/m2 (FEC 75) as first treatment for advanced breast cancer patients\"}, {\"question\": \"Between which probes does the recurrent translocation breakpoint on chromosome 22 of neuroepithelioma lie?\", \"ideal_answer\": [\"The recurrent translocation breakpoint on chromosome 22 of neuroepithelioma has been localized between two probes, D22S1 and D22S15, by both in situ hybridization and somatic cell hybrids\", \"The recurrent translocation breakpoint on chromosome 22 of neuroepithelioma has been localized between two probes, D22S1 and D22S15, by both in situ hybridization and somatic cell hybrids. \", \"The recurrent translocation breakpoint on chromosome 22 of neuroepithelioma has been localized between two probes, D22S1 and D22S15, by both in situ hybridization and somatic cell hybrids.\", \"The recurrent translocation breakpoint on chromosome 22 of neuroepithelioma has been localized between two probes, D22S1 and D22S15, by both in situ hybridization and somatic cell hybrids  \"], \"exact_answer\": [[\"D22S1\"], [\"D22S15\"]], \"context\": \"The recurrent translocation breakpoint on chromosome 22 of neuroepithelioma has been localized between two probes, D22S1 and D22S15, by both in situ hybridization and somatic cell hybridsThe recurrent translocation breakpoint on chromosome 22 of neuroepithelioma has been localized between two probes, D22S1 and D22S15, by both in situ hybridization and somatic cell hybrids.\"}, {\"question\": \"Which are the most widely used computational methods for the identification of CRMs (cis-regulatory modules)?\", \"ideal_answer\": [\"Computational methods attempting to identify instances of cis-regulatory modules (CRMs) in the genome face a challenging problem of searching for potentially interacting transcription factor binding sites while knowledge of the specific interactions involved remains limited. When discriminating CRMs from non-coding regions, those methods considering evolutionary conservation have a stronger predictive power than methods designed to be run on a single genome. Furthermore, most methods appear to be sensitive to the composition and structure of the genome to which they are applied. CisMiner allows to perform a blind search of CRMs without any prior information about target CRMs nor limitation in the number of motifs.\", \"The optimal choice of method varies depending on species and composition of the sequences in question. When discriminating CRMs from non-coding regions, those methods considering evolutionary conservation have a stronger predictive power than methods designed to be run on a single genome. Different CRM representations and search strategies rely on different CRM properties, and different methods can complement one another. For example, some favour homotypical clusters of binding sites, while others perform best on short CRMs. Furthermore, most methods appear to be sensitive to the composition and structure of the genome to which they are applied. A statistical model to describe the underlying cluster structure as well as individual motif conservation has also been proposed, accompanied with a Monte Carlo motif screening strategy for predicting novel regulatory modules in upstream sequences of coregulated genes.\"], \"exact_answer\": [[\"motif conservation\"], [\"fuzzy itemset mining\"], [\"combinatorial methods\"]], \"context\": \"Several tools allow to detect significant co-occurrences of closely located binding sites (cis-regulatory modules, CRMs). However, these tools present at least one of the following limitations: 1) scope limited to promoter or conserved regions of the genome; 2) do not allow to identify combinations involving more than two motifs; 3) require prior information about target motifs.In this work we present CisMiner, a novel methodology to detect putative CRMs by means of a fuzzy itemset mining approach able to operate at genome-wide scale. CisMiner allows to perform a blind search of CRMs without any prior information about target CRMs nor limitation in the number of motifs. CisMiner tackles the combinatorial complexity of genome-wide cis-regulatory module extraction using a natural representation of motif combinations as itemsets and applying the Top-Down Fuzzy Frequent- Pattern Tree algorithm to identify significant itemsets.To be able to handle the large datasets, the query-based setting and other specificities proper to CRM detection on ChIP-Seq based data, we developed a novel powerful CRM detection method 'CPModule'the optimal choice of method varies depending on species and composition of the sequences in question. When discriminating CRMs from non-coding regions, those methods considering evolutionary conservation have a stronger predictive power than methods designed to be run on a single genome. Different CRM representations and search strategies rely on different CRM properties, and different methods can complement one another. For example, some favour homotypical clusters of binding sites, while others perform best on short CRMs. Furthermore, most methods appear to be sensitive to the composition and structure of the genome to which they are applied. CisMiner: genome-wide in-silico cis-regulatory module prediction by fuzzy itemset miningAdditionally, we make a number of new predictions on combinatorial regulation of these five key TFs with other TFs documented in TRANSFAC.Due to degeneracy of nucleotide content among binding site instances or motifs, and intricate &apos;grammatical organization&apos; of motifs within cis-regulatory modules (CRMs), extant pattern matching-based in silico motif search methods often suffer from impractically high false positive rates, especially in the context of analyzing large genomic datasets, and noisy position weight matrices which characterize binding sitesGenome-wide identification of cis-regulatory motifs and modules underlying gene coregulation using statistics and phylogeny.We thus developed a new CRM identification method MOPAT (motif pair tree), which identifies CRMs through the identification of motif modules, groups of motifs co-occurring in multiple CRMs. It can identify &apos;orthologous&apos; CRMs without multiple alignments.These methods are problematic when binding sites are not well aligned in multiple alignments or when the number of input known motifs is large. We thus developed a new CRM identification method MOPAT (motif pair tree), which identifies CRMs through the identification of motif modules, groups of motifs co-occurring in multiple CRMs.We thus developed a new CRM identification method MOPAT (motif pair tree), which identifies CRMs through the identification of motif modules, groups of motifs co-occurring in multiple CRMs.\"}, {\"question\": \"Which diseases is microRNA 132 (miR-132) implicated in?\", \"ideal_answer\": [\"Several targets for for miR-132 have been described and it is implicated in many diseases such as:\\nneurodegenerative disease, \\nepilepsy,\\nschizophrenia,\\nHuntington's disease (HD),\\nAlzheimer's disease (AD),\\nneuroinflammation,\\nosteosarcoma,\\nchronic lymphocytic leukemia (CLL),\\nangiogenesis,\\neye disease,\\nalcoholic liver disease,\\nprogressive supranuclear palsy (PSP taupathy),\\nmild cognitive impairment.\"], \"exact_answer\": [[\"neurodegenerative disease\"], [\"epilepsy\"], [\"schizophrenia\"], [\"Huntington's disease (HD)\"], [\"Alzheimer's disease (AD)\"], [\"neuroinflammation\"], [\"osteosarcoma\"], [\"chronic lymphocytic leukemia (CLL)\"], [\"angiogenesis\"], [\"eye disease\"], [\"alcoholic liver disease\"], [\"progressive supranuclear palsy (PSP taupathy)\"], [\"mild cognitive impairment\"]], \"context\": \"Levels of six candidate miRNAs (miR-21, miR-199a-3p, miR-143, miR-34, miR-140, and miR-132) that were previously demonstrated to be regulated in osteosarcoma were examined in plasma of 40 osteosarcoma patients and 40 matched healthy controls by quantitative reverse-transcription polymerase chain reaction assays.Functional studies employing antagomirs have identified contributions from miR-34a and miR-132 to seizure-induced neuronal death whereas silencing miR-134 potently reduced status epilepticus, seizure-damage and the later occurrence of spontaneous seizures. Efforts to identify the in vivo target(s) of epilepsy-regulated miRNAs, is now a priority. Last, miRNA are stable, information-carrying (paracrine) signals. Profiling miRNA in biofluids may represent a novel source of disease biomarkers in epilepsy.Inflammation, stress signalling and neuronal excitation are among the pathways most impacted.Herpes simplex virus (HSV) causes a chronic immuno-inflammatory response in the eye that may result in corneal neovascularization during blinding immunopathological lesion stromal keratitis (SK). miR-132 is a highly conserved miRNA that is induced in endothelial cells in response to growth factors, such as vascular endothelial growth factor (VEGF).miR-132 expression was up-regulated (10- to 20-fold) after ocular infection with HSV, an event that involved the production of both VEGF-A and IL-17.The anti-angiogenic effect of antagomir-132 was reflected by a reduction in angiogenic Ras activity in corneal CD31-enriched cells (presumably blood vessel endothelial cells) during SK. To our knowledge, this is one of the first reports of miRNA involvement in an infectious ocular disease. Manipulating miRNA expression holds promise as a therapeutic approach to control an ocular lesion that is an important cause of human blindness.Role of miR-132 in angiogenesisThe cyclic AMP-responsive element binding- and NMDA-regulated microRNA miR-132 was significantly down-regulated in both the schizophrenic discovery cohort and a second, independent set of schizophrenic subjects. Analysis of miR-132 target gene expression in schizophrenia gene-expression microarrays identified 26 genes up-regulated in schizophrenia subjects.Our data suggest miR-132 dysregulation and subsequent abnormal expression of miR-132 target genes contribute to the neurodevelopmental and neuromorphological pathologies present in schizophrenia.Analysis of miRNA expression profiles from sporadic progressive supranuclear palsy (PSP) patients, a major 4R-tau tauopathy, showed that miR-132 is specifically down-regulated in disease. We demonstrate that miR-132 directly targets the neuronal splicing factor polypyrimidine tract-binding protein 2 (PTBP2), which protein levels were increased in PSP patients. miR-132 overexpression or PTBP2 knockdown similarly affected endogenous 4R:3R-tau ratios in neuronal cells. Finally, we provide evidence that miR-132 is inversely correlated with PTBP2 during post-natal brain development at the time when 4R-tau becomes expressed. Taken together, these results suggest that changes in the miR-132/PTBP2 pathway could contribute to the abnormal splicing of tau exon 10 in the brain, and sheds light into the potential role played by miRNAs in a subset of tauopathies.Huntington's disease (HD) is a genetic neurodegenerative disease caused by abnormal CAG expansion. MicroRNAs (miRNAs) are short RNA molecules regulating gene expression, and are implicated in a variety of diseases including HD.Nine miRNAs (miR-22, miR-29c, miR-128, miR-132, miR-138, miR-218, miR-222, miR-344, and miR-674*) were commonly down-regulated in both the 12-month-old YAC128 and 10-week-old R6/2 mice.Osteosarcoma is the first primary malignant bone tumor, characterized by a complex genetic and resistance to conventional treatments.Osteosarcoma of rat and human origins showed a miRNA signature, which could discriminate good from bad responders. In particular, we identified five discriminating miRNAs (miR-92a, miR-99b, miR-132, miR-193a-5p and miR-422a) in patient tumors, which could be easily transferable to diagnosis.Levels of several microRNA (miR-10a, -10b, -212, -132, -495) were significantly altered. One of them (miR-132) has been reported to be highly inducible by growth factors and to be a key regulator of neurite outgrowth. Moreover, miR-132-recognition sequences were detected in the mRNA transcripts of two differentially expressed proteins. MicroRNA may thus represent novel biomarkers for neuronal malfunction and potential therapeutic targets for human neurodegenerative diseases.The data indicate that micro-RNAs encoding miR-9, miR-124a, miR-125b, miR-128, miR-132 and miR-219 are abundantly represented in fetal hippocampus, are differentially regulated in aged brain, and an alteration in specific micro-RNA complexity occurs in Alzheimer hippocampus. These data are consistent with the idea that altered micro-RNA-mediated processing of messenger RNA populations may contribute to atypical mRNA abundance and neural dysfunction in Alzheimer's disease brain.\"}, {\"question\": \"Which are the human glutamate transporters?\", \"ideal_answer\": [\"Glutamate/aspartate transporter (GLAST) and glutamate transporter-1 (GLT-1) are the most abundant subtypes and are essential for the functioning of the mammalian CNS, but the contribution of the EAAC1 subtype in the clearance of synaptic glutamate has remained controversial, because the density of this transporter in different tissues has not been determined. Using immunofluorescence and postembedding immunogold labeling, we investigated the distributions of the glutamate-aspartate transporter (GLAST or excitatory amino acid transporter 1), vesicular glutamate transporter (VGLUT1), and the AMPA receptor glutamate receptor 4 (GluR4) along the spiral.\", \"To date, five distinct mammalian glutamate transporters have been cloned. The extracellular levels of excitatory amino acids are kept low by the action of the glutamate transporters. Glutamate/aspartate transporter (GLAST) and glutamate transporter-1 (GLT-1) are the most abundant subtypes and are essential for the functioning of the mammalian CNS, but the contribution of the EAAC1 subtype in the clearance of synaptic glutamate has remained controversial, because the density of this transporter in different tissues has not been determined. Expression of short interfering RNA-mediated knockdown of RTN2B decreases the EAAC1 protein level in neurons.\"], \"exact_answer\": [[\"EAAC1\", \"EAAT3\"], [\"GLT-1\", \"glutamate transporter-1\", \"excitatory amino acid transporter 1\"], [\"GLAST\", \"Glutamate/aspartate transporter\"], [\"GluR4\", \"AMPA receptor glutamate receptor 4\"], [\"VGLUT1\", \"vesicular glutamate transporter\"]], \"context\": \"The density of EAAC1 (EAAT3) glutamate transporters expressed by neurons in the mammalian CNS.Glutamate/aspartate transporter (GLAST) and glutamate transporter-1 (GLT-1) are the most abundant subtypes and are essential for the functioning of the mammalian CNS, but the contribution of the EAAC1 subtype in the clearance of synaptic glutamate has remained controversial, because the density of this transporter in different tissues has not been determined.Using immunofluorescence and postembedding immunogold labeling, we investigated the distributions of the glutamate-aspartate transporter (GLAST or excitatory amino acid transporter 1), vesicular glutamate transporter (VGLUT1), and the AMPA receptor glutamate receptor 4 (GluR4) along the spiral. Immunogold labeling for GluR4 was confined to synaptic sites, represented by puncta in immunofluorescence. The relative numbers of puncta changed with a gradient similar to that of GLAST labeling. VGLUT1 labeling occurred in IHCs but showed no clear cochleotopic gradient. These data suggest that both the density of innervation and the activity levels of glutamatergic synapses may be involved in modulating regional expression of GLAST.In brain, EAAC1 (excitatory amino acid carrier 1) is the primary neuronal glutamate transporter, localized on the perisynaptic membranes that are near release sites.Reticulon RTN2B regulates trafficking and function of neuronal glutamate transporter EAAC1.Previously, we identified an EAAC1-associated protein, GTRAP3-18, an ER protein that prevents ER exit of EAAC1 when inducedHere we show that RTN2B, a member of the reticulon protein family that mainly localizes in the ER and ER exit sites interacts with EAAC1 and GTRAP3-18In this study, uptake of L-[14C]cystine by three of the high affinity sodium-dependent mammalian glutamate transporters (GLT1, GLAST and EAAC1) individually expressed in HEK cells has been determinedReduction of L-[14C]cystine to L-[14C]cysteine in the presence of 1mM cysteinylglycine increases the uptake rate in HEK(GLT1), HEK(GLAST) and HEK(EAAC1) cells, but only a small proportion (<10%) of L-[14C]cysteine uptake in HEK(GLT1) and HEK(GLAST) cells occurs by the high affinity glutamate transportersPosition of the third Na+ site in the aspartate transporter GltPh and the human glutamate transporter, EAAT1.We studied the subunit stoichiometry of two distantly related glutamate transporters, the human glial glutamate transporter hEAAT2 and a bacterial glutamate transporter from Escherichia coli, ecgltP.Glutamate receptors and transporters, including T1R1 and T1R3 (taste receptor 1, subtypes 1 and 3), mGluRs (metabotropic glutamate receptors), EAAC-1 (excitatory amino acid carrier-1), GLAST-1 (glutamate-aspartate transporter-1), and GLT-1 (glutamate transporter-1), are expressed in the gastrointestinal tractThe ASCTs (alanine, serine, and cysteine transporters) belong to the solute carrier family 1 (SLC1), which also includes the human glutamate transporters (excitatory amino acid transporters, EAATs) and the prokaryotic aspartate transporter GltPhAs astrocytes protect neurons by taking up glutamate via plasma-membrane transporters, we also studied the effect of GRK2 on the localization of the GLutamate ASpartate Transporter (GLAST)GLT-1, Glutamate Transporter 1, same as excitatory amino acid transporter 2; Glu, glutamateMultiple Functions of Glutamate Uptake via Meningococcal GltT-GltM l-Glutamate ABC Transporter in Neisseria meningitidis Internalization into Human Brain Microvascular Endothelial CellsDespite the high degree of amino acid sequence identity between family members, ASCTs function quite differently from the EAATs and GltPh. First, Li(+) cannot support transport by EAAT2, whereas it can support transport by the other excitatory amino acid transporters, and second, EAAT2 is sensitive to a wider range of blockers than other subtypes. hGLT-1 and hGLuT-1 mRNAs were most abundantly expressed in the brain, while hEAAC1 mRNA expression (3.8 kb and 2,4 kb) was strongest in peripheral organs. Identification of functional domains of the human glutamate transporters EAAT1 and EAAT2.The glutamate transporter EAAT2 is different from other subtypes in two respects. DNA methylation dependent silencing of the human glutamate transporter EAAT2 gene in glial cells.We have investigated the functional impact of a naturally occurring mutation of the human glutamate transporter GLT1 (EAAT2), which had been detected in a patient with sporadic amyotrophic lateral sclerosis. We studied the subunit stoichiometry of two distantly related glutamate transporters, the human glial glutamate transporter hEAAT2 and a bacterial glutamate transporter from Escherichia coli, ecgltP. Using blue native polyacrylamide gel electrophoresis, analysis of concatenated transporters, and chemical cross-linking, we demonstrated that human and prokaryotic glutamate transporters expressed in Xenopus laevis oocytes or in mammalian cells are assembled as trimers composed of three identical subunits.Five glutamate transporters in the human brain (EAAT1-5) are present on both astroglia and neurons. We characterize the profile of three different human astroglial progenitors in vitro: human glial restricted precursors (HGRP), human astrocyte precursors (HAPC), and early-differentiated astrocytes.They are known to be multimers; however, the number of subunits forming a functional transporter is controversial. We studied the subunit stoichiometry of two distantly related glutamate transporters, the human glial glutamate transporter hEAAT2 and a bacterial glutamate transporter from Escherichia coli, ecgltP.We studied the subunit stoichiometry of two distantly related glutamate transporters, the human glial glutamate transporter hEAAT2 and a bacterial glutamate transporter from Escherichia coli, ecgltP.Position of the third Na+ site in the aspartate transporter GltPh and the human glutamate transporter, EAAT1.Immunofluorescent analysis was used to investigate the existence and location of glutamate, glutamate receptor (NR2B), and glutamate transporter (GLT1) in mouse and human sperm.\"}, {\"question\": \"List protein gel staining methods visualizing the entire protein set.\", \"ideal_answer\": [\"Several staining protocols for the visualization of proteins separated by SDS-PAGE have been described in literature: \\nfluorescence\\nSypro Ruby\\nColloidal Coomassie Blue\\nCoomassie Blue\\nSilver staining\\nCoomassie Brilliant Blue\"], \"exact_answer\": [[\"fluorochromes\", \"fluorescence\"], [\"Sypro Ruby\"], [\"Colloidal Coomassie Blue\"], [\"Coomassie Blue\"], [\"Silver staining\"], [\"Coomassie Brilliant Blue\"]], \"context\": \"DIGE requires the labelling of proteins by fluorochromes prior to their separation on 2DE gels. Stains-All (ISA) staining method for phosphoproteins in SDS-PAGE was described. Pro-Q Diamond stainQuantitative proteome analyses suggest that the well-established stain colloidal Coomassie Blue, when used as an infrared dye, may provide sensitive, post-electrophoretic in-gel protein detection that can rival even Sypro Ruby.Several new fast staining protocols for the visualization of proteins separated by SDS-PAGE utilizing Coomassie Blue staining (CBS) have been described in literature. The CyDye family of fluorescent dyes is currently the overwhelming choice for applications in proteomic analysis, using two-dimensional difference gel electrophoresis (2D-DIGE)his enables the co-detection of phosphoproteins as well as total proteins from the same gel and is accomplished by utilizing two different fluorescent stains, the ProQ-Diamond, which stains only phosphorylated proteins, and Sypro Ruby, which stains the entire subset of proteins.Detection of the resulting protein maps relies on staining (i.e. colloidal coomassie blue (CCB) or SYPRO Ruby (SR), in addition to many others). Fluorescent in-gel protein stains are generally preferred for higher sensitivity, reduced background, and wider dynamic range.Commonly used staining methods with excellent mass spectrometry compatibility are coomassie brilliant blue (CBB) or fluorescent dyes.The typical concentration of protein loaded varies from 0.13 to 1.40 \\u03bcg/\\u03bcL for a classical silver staining method in 2DE gelSilver staining is widely used to detect protein in polyacrylamide gels when high sensitivity is required.Silver staining is used to detect proteins after electrophoretic separation on polyacrylamide gels\"}, {\"question\": \"Synostosis of which cranial structures are characteristic to the Mercedes Benz syndrome?\", \"ideal_answer\": [\"Synostosis of sagittal and lambdoid structures are characteristic to the Mercedes Benz syndrome.\"], \"exact_answer\": [[\"sagittal\"], [\"lambdoid\"]], \"context\": \"Among these are a trisutural fusion, dubbed the \\\"Mercedes Benz pattern,\\\" involving the sagittal and both lambdoid sutures. CONCLUSIONS: Despite fusion of the sagittal suture, the surgical treatment for Mercedes Benz pattern craniosynostosis should include skull lengthening, not reduction.Multisutural craniosynostosis that includes bilateral lambdoid and sagittal synostosis (BLSS) results in a very characteristic head shape with frontal bossing, turribrachycephaly, biparietal narrowing, occipital concavity, and inferior displacement of the ears. This entity has been reported both in the genetics literature as craniofacial dyssynostosis and in the surgical literature as \\\"Mercedes Benz\\\" syndrome. Bilambdoid and posterior sagittal synostosis: the Mercedes Benz syndrome.A consistent pattern of craniosynostosis in the sagittal and bilateral lambdoid sutures is described in three patients. The external cranial ridging associated with fusion of these sutures produces a characteristic triradiate, or \\\"Mercedes Benz,\\\" appearance to the posterior skull.Bilambdoid and posterior sagittal synostosis: the Mercedes Benz syndrome.\"}, {\"question\": \"Which signaling pathways have been associated with medulloblastoma formation and growth?\", \"ideal_answer\": [\"Medulloblastoma comprises approximately 20% of all primary pediatric brain tumors. Multiple signaling pathways have been associated with disease formation and growth. These include the developmental pathways Hedgehog, Notch, and Wnt, as well as pathways in which the receptor tyrosine kinases (RTK) c-Met, erbB2, IGF-R and TrkC, and the oncoprotein Myc are involved.\"], \"exact_answer\": [[\"Hedgehog\"], [\"Notch\"], [\"Wnt\"], [\"c-Met\"], [\"erbB2\"], [\"IGF-R\"], [\"TrkC\"], [\"Myc\"]], \"context\": \"Hedgehog (Hh) signaling is an important factor in growth and patterning during embryonic development. A mutation in Patched, Smoothened or Gli1, which regulate the Hh signaling pathway, might lead to the onset of glioblastoma, basal cell carcinoma, medulloblastoma and rhabdomyosarcomaThese data indicate that inhibition of Aurora Kinase A inhibits cell growth in medulloblastoma through inhibition of pro-proliferative signaling pathways Wnt, Myc, and RBMultiple signaling pathways have been associated with medulloblastoma formation and growth. These include the developmental pathways Hedgehog, (Hh) Notch, and Wnt as well as the receptor tyrosine kinases (RTK) c-Met, erbB2, IGF-R and TrkC, and the oncoprotein MycActivation of neurotrophin-3 receptor TrkC induces apoptosis in medulloblastomasElevated expression of the neurotrophin-3 (NT-3) receptor TrkC by childhood medulloblastomas is associated with favorable clinical outcomeConsidered collectively, these results support the conclusion that the biological actions of TrkC activation affect medulloblastoma outcome by inhibiting tumor growth through the promotion of apoptosis.During development, proliferation of cerebellar granule neuron precursors (CGNP), candidate cells-of-origin for the pediatric brain tumor medulloblastoma, requires signaling by Sonic hedgehog (Shh) and insulin-like growth factor (IGF), the pathways of which are also implicated in medulloblastoma.Sonic hedgehog and insulin-like growth factor signaling synergize to induce medulloblastoma formation from nestin-expressing neural progenitors in mice.Ectopic expression of alpha and beta interferons in the developing brain also induces Shh-mediated medulloblastoma formation, suggesting a possible role for antiviral response in the genesis of medulloblastoma.Aberrant activation of the Hedgehog (Hh) and Wnt developmental pathways is associated with medulloblastoma, but deregulation of other molecular pathways, including insulin-like growth factor (IGF) signaling, has also been implicated in the pathogenesis of the tumor.Sunitinib induces apoptosis and growth arrest of medulloblastoma tumor cells by inhibiting STAT3 and AKT signaling pathways.Aberrant activation of the Sonic hedgehog (Shh) and insulin-like growth factor (IGF) pathways is associated with human medulloblastomas.N-myc can substitute for insulin-like growth factor signaling in a mouse model of sonic hedgehog-induced medulloblastoma.Aberrant activation of the Hedgehog (Hh) and Wnt developmental pathways is associated with medulloblastoma, but deregulation of other molecular pathways, including insulin-like growth factor (IGF) signaling, has also been implicated in the pathogenesis of the tumorAberrant expression/activation of EGFR is found in multiple human cancers, including medulloblastoma, the most prevalent pediatric brain cancer, and often has been associated with metastasis, poor prognosis, and resistance to chemotherapyWe emphasize the cancer-relevance of cilia-dependent signaling pathways and proteins including mTOR, VHL, TSC, WNT, Aurora-A, NEDD9, and Hedgehog, and highlight the emerging role of ciliary dysfunction in renal cell carcinoma, medulloblastoma, and breast cancerHere, we show that in medulloblastoma cells, both EGF and ouabain activate Erk1/2 and PI3K/Akt signalingExpression level of tumor/metastasis suppressive miRNAs in the WNT signaling associated medulloblastomas is likely to determine their response to treatment, and thus, these miRNAs would be important biomarkers for risk stratification within the WNT signaling associated medulloblastomas.Aberrant activation of the Sonic hedgehog (Shh) and insulin-like growth factor (IGF) pathways is associated with human medulloblastomasShh-dependent medulloblastoma formation is enhanced by hyperactive insulin-like growth factor (IGF) signaling and ectopic expression of Myc oncogenesPhosphatidylinositol 3'-kinase/AKT signaling is activated in medulloblastoma cell proliferation and is associated with reduced expression of PTEN.A number of miRNAs like miR-193a, miR-224/miR-452 cluster, miR-182/miR-183/miR-96 cluster, and miR-148a having potential tumor/metastasis suppressive activity were found to be overexpressed in the WNT signaling associated medulloblastomasThe requirement for SHH-Gli signaling in the growth of the mouse brain, together with the ability of inappropriate pathway activation in the cerebellum to cause medulloblastomas, and the inhibition of the growth of a number of brain tumors with cyclopamine, a SHH signaling inhibitor, underscores the critical role of the SHH-GLI pathway in brain growth and tumor formationAberrant activation of the Hedgehog (Hh) and Wnt developmental pathways is associated with medulloblastoma, but deregulation of other molecular pathways, including insulin-like growth factor (IGF) signaling, has also been implicated in the pathogenesis of the tumorDistinctive microRNA signature of medulloblastomas associated with the WNT signaling pathway.\"}, {\"question\": \"What are the symptoms of abacavir hypersensitivity?\", \"ideal_answer\": [\"Patients receiving abacavir develop an idiosyncratic hypersensitivity reaction that can include a wide range of symptoms. The most common are: fever, enathema, skin rash, nausea, vomiting, diarrhoea, cough, gastrointestinal disorders, anaphylactic shock, respiratory symptoms.\"], \"exact_answer\": [[\"fever\"], [\"enathema\"], [\"skin rash\"], [\"nausea\"], [\"vomiting\"], [\"diarrhea\"], [\"cough\"], [\"gastrointestinal disorders\"], [\"anaphylactic shock\"], [\"respiratory symptoms\"]], \"context\": \"patients receiving abacavir develop a hypersensitivity reaction, characterized by rash, fever and, occasionally, multisystemic involvementThe predictors included any one of several specific symptoms commonly found with this reaction, a claims diagnosis of adverse effect of drug, anaphylactic shock or unspecified allergy, and a discontinuation in abacavir prior to completing a 90-day course of therapy.patients who receive abacavir develop an idiosyncratic hypersensitivity reaction. The most common symptoms are fever, skin rash and gastrointestinal disorders. Respiratory symptoms occurred in approximately 20% of patients who have hypersensitivity reaction. We describe the first case, to our knowledge, of hypesensitivity reaction characterized by enanthema and fever without skin rash promptly resolved after discontinuation of abacavirAbacavir is associated with an infrequent but potentially serious hypersensitivity reaction (HSR) that can include a wide range of signs and symptoms.Rash was associated with hypersensitivity (odds ratio [OR] = 13.1, P = 0.02) as was the presence of nausea (OR = 30, P < 0.001), vomiting (OR = 17.1, P = 0.001) or diarrhoea (OR = 22, P < 0.001). The number of gastrointestinal symptoms was also predictive of hypersensitivity reaction (the number of gastrointestinal symptoms (OR = 8.6, P = 0.0032), cough (OR = 0.039, P = 0.02) and rash (OR = 16.9, P = 0.07). Abacavir hypersensitivity is strongly associated with gastrointestinal (GI) symptoms.\"}, {\"question\": \"Proteomic analyses have revealed proteins associated with the triple-negative breast cancers. List some proposed proteins.\", \"ideal_answer\": [\"Selected proteins of interest proposed from triple-negative cancer proteomic studies are CD44, PARP1, Mage-A4, LSR, RAB25, S100A14, MUC1, Hsp90, Actin, 14-3-3, vimentin, HSP70, CK18, moesin, IDH2, CRABP2, SEC14L2, beta-catenin, MUC18, Stat1 and CD74.\"], \"exact_answer\": [[\"CD44\"], [\"PARP1\"], [\"Mage-A4\"], [\"LSR\"], [\"RAB25\"], [\"S100A14\"], [\"MUC1\"], [\"Hsp90\"], [\"Actin\"], [\"14-3-3\"], [\"vimentin\"], [\"HSP70\"], [\"CK18\"], [\"moesin\"], [\"IDH2\"], [\"CRABP2\"], [\"SEC14L2\"], [\"beta-catenin\"], [\"MUC18\"], [\"Stat1\"], [\"CD74\"]], \"context\": \"Some of these proteins are of outstanding interest in the biology and clinical management of this disease, such as CD44 and PARP1.We present here our results, including a large cumulative database of proteins based on the analysis of 78 TNBCs, and the identification and validation of one specific protein, Mage-A4, which was expressed in a significant fraction of TNBC and Her2-positive/ER negative lesions.Proteomic profiling of triple-negative breast carcinomas in combination with a three-tier orthogonal technology approach identifies Mage-A4 as potential therapeutic target in estrogen receptor negative breast cancer.Further, we performed comparative quantitative proteomic and gene array analyses of these cells and identified potential novel markers of breast cancer cells with tumor-initiating features, such as lipolysis-stimulated lipoprotein receptor (LSR), RAB25, S100A14 and mucin 1 (MUC1), as well as a novel 31-gene signature capable of predicting distant metastasis in cohorts of estrogen receptor-negative human breast cancers.he results identify Hsp90 as a critical and multimodal target in this most difficult to treat breast cancer subtype and support the use of the Hsp90 inhibitor PU-H71 for clinical trials involving patients with TNBC.We obtained differential expression patterns for several glycolytic enzymes (as for example MDH2, PGK1, TKT, Aldolase1), cytokeratins (CK7, 8, 9, 14, 17, 19), further structure proteins (vimentin, fibronectin, L-plastin), for NME1-NME2, lactoferrin, and members of the Annexin family.he expression of calreticulin, cellular retinoic acid-binding protein II, chloride intracellular channel protein 1, EF-1-beta, galectin 1, peroxiredoxin-2, platelet-derived endothelial cell growth factor, protein disulfide isomerase and ubiquitin carboxyl-terminal hydrolase 5 were further validated using a tissue microarray containing 70 malignant breast carcinomas of various grades of atypia.The first is characterized by high expression of Stat1, Mx1, and CD74.This effect was accompanied with a marked increase in the membrane expression of beta-catenin, MUC18, plexins, integrins, and other proteins involved in cell adhesion and cancer metastasis. Taken together, our results show that Stat1/CD74 positive triple-negative tumors are more aggressive and suggest an approach for development of better diagnostics and more targeted therapies for triple negative breast cancer.Proteomic analysis also identified high levels of IDH2 and CRABP2 and low levels of SEC14L2 to be prognostic markers for overall breast cancer survival.Proteomic and validation immunohistochemical analyses revealed that alpha-defensins (DEFA) were overexpressed in tumors from patients with a pCR.The expression of a selected group of high abundance and/or breast cancer-specific potential biomarkers including thromobospondin 1, galectin-3 binding protein, cathepsin D, vimentin, zinc-\\u03b12-glycoprotein, CD44, and EGFR from the breast cancer cell lines and in their culture media were further validated by Western blot analysis.his provided novel evidence for PMA regulation of the enzymes glyceraldehyde-3-phosphate dehydrogenase, nucleolar RNA helicase 2 and Heterogeneous nuclear ribonucleoprotein M.Among them, we identify downregulation of components of the Ras/Raf/MAPK pathway and G(2)-M phase to contribute to its anti-proliferative effect, degradation of activated Akt and Bcl-xL to induce apoptosis, and inhibition of activated NF-kappaB, Akt, ERK2, Tyk2, and PKC to reduce TNBC invasive potential.\"}, {\"question\": \"Which are the major types of the motor speech disorder dysarthria?\", \"ideal_answer\": [\"Dysarthria is a motor speech disorder which can be classified according to the underlying neuropathology and is associated with disturbances of respiration, laryngeal function, airflow direction, and articulation resulting in difficulties of speech quality and intelligibility. There are six major types of dysarthria: \\\"flaccid dysarthria\\\" associated with lower motor neuron impairment, \\\"spastic dysarthria\\\" associated with damaged upper motor neurons linked to the motor areas of the cerebral cortex, \\\"ataxic dysarthria\\\" primarily caused by cerebellar dysfunction, and \\\"hyperkinetic dysarthria\\\" and \\\"hypokinetic dysarthria\\\", which are related to a disorder of the extrapyramidal system. The sixth is generally termed a \\\"mixed dysarthria\\\" and is associated with damage in more than one area, resulting in speech characteristics of at least two groups.\"], \"exact_answer\": [[\"flaccid dysarthria\"], [\"spastic dysarthria\"], [\"ataxic dysarthria\"], [\"hyperkinetic dysarthria\"], [\"hypokinetic dysarthria\"], [\"mixed dysarthria\"]], \"context\": \"Dysarthria is a motor speech disorder which can be classified according to the underlying neuropathology and is associated with disturbances of respiration, laryngeal function, airflow direction, and articulation resulting in difficulties of speech quality and intelligibility. There are six major types of dysarthria: flaccid dysarthria associated with lower motor neuron impairment, spastic dysarthria associated with damaged upper motor neurons linked to the motor areas of the cerebral cortex, ataxic dysarthria primarily caused by cerebellar dysfunction, and hyperkinetic dysarthria and hypokinetic dysarthria, which are related to a disorder of the extrapyramidal system. The sixth is generally termed a mixed dysarthria and is associated with damage in more than one area, resulting in speech characteristics of at least two groups.address voice problems of stroke patients with mixed dysarthriaCharacterizing intonation deficit in motor speech disorders: an autosegmental-metrical analysis of spontaneous speech in hypokinetic dysarthria, ataxic dysarthriaataxic dysarthria (AT)Acoustic analysis provides objective quantitative measures of speech that enable a comprehensive and accurate understanding of motor disorders and complement the traditional measures. This paper aims to distinguish between normal and pathological speech, more specifically between apraxia of speech and spastic dysarthriaLong-term phonatory instability in ataxic dysarthria.The objective of this study is to evaluate phonation in ataxic dysarthria and a control group of normal speakersDysprosody is a common feature in speakers with hypokinetic dysarthria.The prosodic profile of Cantonese speakers with hypokinetic dysarthria is similar to those of other languages (for example, English).Classification of dysarthria types comprises flaccid, spastic, ataxic, hypo- and hyperkinetic and mixed dysarthria.Dysarthria affects linguistic domains such as respiration, phonation, articulation, resonance and prosody due to upper motor neuron, lower motor neuron, cerebellar or extrapyramidal tract lesions.The dysarthria types were spastic (15 subjects), flaccid (10), mixed (12), hypokinetic (12), hyperkinetic (9) and ataxic (8).Dysarthria is a motor speech disorder which can be classified according to the underlying neuropathology and is associated with disturbances of respiration, laryngeal function, airflow direction, and articulation resulting in difficulties of speech quality and intelligibility.There are six major types of dysarthria: flaccid dysarthria associated with lower motor neuron impairment, spastic dysarthria associated with damaged upper motor neurons linked to the motor areas of the cerebral cortex, ataxic dysarthria primarily caused by cerebellar dysfunction, and hyperkinetic dysarthria and hypokinetic dysarthria, which are related to a disorder of the extrapyramidal system.There are six major types of dysarthria: flaccid dysarthria associated with lower motor neuron impairment, spastic dysarthria associated with damaged upper motor neurons linked to the motor areas of the cerebral cortex, ataxic dysarthria primarily caused by cerebellar dysfunction, and hyperkinetic dysarthria and hypokinetic dysarthria, which are related to a disorder of the extrapyramidal system.Motor neuron disease encompasses a group of terminal, demyelinating diseases affecting upper- and lower-motor neurons and producing muscular weakness resulting in a flaccid, spastic, or spastic-flaccid dysarthria of speech.Dysarthria is a frequently occurring motor speech disorder which can be caused by neurological trauma, cerebral palsy, or degenerative neurological diseases.Dysarthria is a motor disorder of speech characterized by abnormalities of the articulation and intelligibility of speech.There are six major types of dysarthria: flaccid dysarthria associated with lower motor neuron impairment, spastic dysarthria associated with damaged upper motor neurons linked to the motor areas of the cerebral cortex, ataxic dysarthria primarily caused by cerebellar dysfunction, and hyperkinetic dysarthria and hypokinetic dysarthria, which are related to a disorder of the extrapyramidal system\"}, {\"question\": \"Which histone modifications are associated with Polycomb group (PcG) proteins?\", \"ideal_answer\": [\"A member of the polycomb repressive complex 2 (PRC2) directly mediates the addition of K27me3 to histone H3, a modification associated with heterochromatin, and it is believed that this activity mediates transcriptional repression. At the same time PRC2 activity results in a global increase in H3K27 acetylation. Some members of the PcG display affinity towards both histone H3 trimethylated at K9 and H3K27me3, and one CD prefers K9me3.\"], \"exact_answer\": [[\"H3K27me3\"], [\"H3K27Ac\"], [\"H3K9me3\"], [\"H3K36me\"]], \"context\": \"The PcG proteins are well-conserved chromatin factors that repress transcription of numerous target genes. They bind the genome at specific sites, forming chromatin domains of associated histone modifications as well as higher-order chromatin structures.A component of Polycomb repressive complex 2 (PRC2), which mediates the addition of K27me3 to histone H3 (Suz12), was also recruited by 14 dpiWe found that a component of the PRC1 complex (Bmi1), which binds to H3K27me3, was not enriched at promoters found previously to be enriched for H3K27me3 Here we show that the HSV DNA first associates with histone H3, with later recruitment of Polycomb repressor complex 2 (PRC2) and trimethylation of the lysine 27 residue of histone H3 (H3K27me3), a modification associated with heterochromatinHere we show that direct recognition of methylated histone H3 Lys36 (H3K36me), a mark associated with activation, by the PRC2 subunit Phf19 is required for the full enzymatic activity of the PRC2 complexFurthermore, we show that Phf19 binds to a subset of PRC2 targets in mouse embryonic stem cells and that this is required for their repression and for H3K27me3 depositionThe Polycomb repressive complex 2 (PRC2) catalyzes trimethylation (me3) of histone H3 lysine 27 (K27), and it is believed that this activity mediates transcriptional repressionWe show that loss of PRC2 activity results in a global increase in H3K27 acetylationMoreover, we provide evidence that the acetylation of H3K27 is catalyzed by the acetyltransferases p300 and CBP.Taken together, these studies suggest that patterns of epigenetic modifiers and the histone code influence the propensity of a gene to become hypermethylated in cancer and that DNMT3B plays an important role in regulating PRC1 functionDuring differentiation, binding of polycomb repressive complex 2 (PRC2), which mediates trimethylation of lysine 27 on histone H3 (K27me3), is lost on developmental genes that are transcriptionally inducedchromodomain (CD) of the Drosophila Polycomb protein exhibits preferential binding affinity for histone H3 when trimethylated at lysine 27Not all CDs bind preferentially to K27me3; rather, some display affinity towards both histone H3 trimethylated at K9 and H3K27me3, and one CD prefers K9me3\"}, {\"question\": \"List phosphorylation consensus motifs for Casein Kinase 1 (CK1)?\", \"ideal_answer\": [\"The most common consensus motifs for CK1 are: pSer-Xaa-Xaa-Ser, K/R-X-K/R-X-X-S/T, SLS and acidic cluster motifs and SerP/ThrP-Xaa-Xaa-Ser/Thr.\"], \"exact_answer\": [[\"pSer-Xaa-Xaa-Ser\"], [\"K/R-X-K/R-X-X-S/T\"], [\"SLS and acidic cluster motifs\"], [\"SerP/ThrP-Xaa-Xaa-Ser/Thr\"]], \"context\": \"Members of the protein kinase CK1 family usually phosphorylate residues of serine that are close to other phosphoserine in a consensus motif of pS-X-X-S,a novel consensus phosphorylation motif (K/R-X-K/R-X-X-S/T) for CK1 CK1 acts as a 'phosphate-directed' kinase whose targeting is primed by a single phosphorylated side chain at position n-3 or n-4 relative to serine/threonineThe protein kinase CK1 phosphorylates serine residues that are located close to another phosphoserine in the consensus pSer-Xaa-Xaa-Ser.The results demonstrate that SLS and acidic cluster motifs are crucial for CK1 recognition. The common features include an SLS motif followed two to five residues downstream by a cluster of acidic residues.These data provide the clear-cut demonstration that the consensus sequence with N-terminal prephosphorylated residue(s), SerP/ThrP-Xaa-Xaa-Ser/Thr\"}, {\"question\": \"What medication were compared in the ROCKET AF Trial?\", \"ideal_answer\": [\"ROCKET-AF trial compared rivaroxaban and warfarin for for prevention of stroke and embolism.\"], \"exact_answer\": [[\"rivaroxaban\"], [\"warfarin\"]], \"context\": \"METHODS AND RESULTS: Patients randomized in the Rivaroxaban Once Daily Oral Direct Factor Xa Inhibition Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET-AF) trial (n = 14 264) were grouped by baseline AF category: paroxysmal or persistent. Multivariable adjustment was performed to compare thrombo-embolic events, bleeding, and death between groups, in high-risk subgroups, and across treatment assignment (rivaroxaban or warfarin). This review focuses on the direct Factor Xa inhibitor rivaroxaban, summarizing the properties that make rivaroxaban appropriate for anticoagulant therapy in this indication (including its predictable pharmacokinetic and pharmacodynamic profile and once-daily dosing regimen) and describing data from the Phase III ROCKET AF trial, which showed once-daily rivaroxaban to be noninferior to warfarin for the prevention of stroke in patients with nonvalvular AF.Subanalyses of ROCKET AF data showed rivaroxaban to have consistent efficacy and safety across a wide range of patients, and studies to confirm these results in real-world settings are underway. Rivaroxaban for stroke prevention in East Asian patients from the ROCKET AF trial.CONCLUSIONS: Observed relative efficacy and safety of rivaroxaban versus warfarin were similar among patients within and outside East Asia. Rivaroxaban versus warfarin in Japanese patients with nonvalvular atrial fibrillation in relation to the CHADS2 score: a subgroup analysis of the J-ROCKET AF trial.BACKGROUND: Results from a trial of rivaroxaban versus warfarin in 1280 Japanese patients with atrial fibrillation (J-ROCKET AF) revealed that rivaroxaban was noninferior to warfarin with respect to the principal safety outcome. Rivaroxaban, an oral direct factor Xa-inhibitor was non-inferior to adjusted dose warfarin in the prevention of stroke and embolism among patients with atrial fibrillation (AF) in the ROCKET-AF trial and has been approved for stroke prevention in AF.The ROCKET AF trial compared rivaroxaban (20 mg/day; 15 mg/day in patients with creatinine clearance 30-49 ml/min) with dose-adjusted warfarin (international normalized ratio 2-3) in 14,264 patients with AF and a prior history of stroke or at least two other additional risk factors for stroke.Factors associated with major bleeding events: insights from the ROCKET AF trial (rivaroxaban once-daily oral direct factor Xa inhibition compared with vitamin K antagonism for prevention of stroke and embolism trial in atrial fibrillation).In Rivaroxaban Once Daily Oral Direct Factor Xa Inhibitor Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET AF) trial, rivaroxaban was noninferior to dose-adjusted warfarin in preventing stroke or systemic embolism among patients with nonvalvular atrial fibrillation at moderate to high stroke risk.Different clinical trials have shown the benefits of new oral anticoagulants over warfarin, but patients included in the ROCKET-AF trial were found to be at a higher risk of AF-related complications.The ROCKET AF trial demonstrated the noninferiority of rivaroxaban compared with warfarin for the prevention of stroke and systemic embolism, with a similar rate of major bleeding and a substantial reduction in intracranial hemorrhage.Rivaroxaban for stroke prevention in atrial fibrillation: a critical review of the ROCKET AF trial.Outcomes after cardioversion and atrial fibrillation ablation in patients treated with rivaroxaban and warfarin in the ROCKET AF trial.The overall analysis of the rivaroxaban versus warfarin in Japanese patients with atrial fibrillation (J-ROCKET AF) trial revealed that rivaroxaban was not inferior to warfarin with respect to the primary safety outcome.Medical costs in the US of clinical events associated with oral anticoagulant (OAC) use compared to warfarin among non-valvular atrial fibrillation patients \\u226575 and <75 years of age, based on the ARISTOTLE, RE-LY, and ROCKET-AF trials.In patients at risk of stroke due to atrial fibrillation, rivaroxaban was noninferior compared to warfarin in preventing stroke/systemic embolism in the Rivaroxaban Once Daily Oral Direct Factor Xa Inhibition Compared with Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET-AF) trial and was associated with a similar risk of major bleeding; the incidence of intracranial hemorrhage was 33% lower with rivaroxabanOutcomes of discontinuing rivaroxaban compared with warfarin in patients with nonvalvular atrial fibrillation: analysis from the ROCKET AF trial (Rivaroxaban Once-Daily, Oral, Direct Factor Xa Inhibition Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation)The Phase III ROCKET AF (Rivaroxaban Once-Daily Oral Direct Factor Xa Inhibition Compared with Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation) trial showed that rivaroxaban, an oral, direct Factor Xa inhibitor, was noninferior to warfarin for the reduction of stroke or systemic embolism in patients with AFIn Rivaroxaban Once Daily Oral Direct Factor Xa Inhibitor Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET AF) trial, rivaroxaban was noninferior to dose-adjusted warfarin in preventing stroke or systemic embolism among patients with nonvalvular atrial fibrillation at moderate to high stroke riskWe investigated clinical characteristics and outcomes of patients with significant valvular disease (SVD) in the Rivaroxaban Once Daily Oral Direct Factor Xa Inhibition Compared with Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET AF) trialEfficacy and safety of rivaroxaban compared with warfarin among elderly patients with nonvalvular atrial fibrillation in the Rivaroxaban Once Daily, Oral, Direct Factor Xa Inhibition Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET AF)The oral factor Xa inhibitor rivaroxaban was noninferior to warfarin in the Rivaroxaban Once Daily, Oral, Direct Factor Xa Inhibition Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET AF)We studied the use and outcomes of digoxin in patients in the Rivaroxaban Once Daily Oral Direct Factor Xa Inhibition Compared with Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET AF)Outcomes of discontinuing rivaroxaban compared with warfarin in patients with nonvalvular atrial fibrillation: analysis from the ROCKET AF trial (Rivaroxaban Once-Daily, Oral, Direct Factor Xa Inhibition Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation).Relationship between time in therapeutic range and comparative treatment effect of rivaroxaban and warfarin: results from the ROCKET AF trial.Efficacy and safety of rivaroxaban compared with warfarin among elderly patients with nonvalvular atrial fibrillation in the Rivaroxaban Once Daily, Oral, Direct Factor Xa Inhibition Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation (ROCKET AF).Outcomes of temporary interruption of rivaroxaban compared with warfarin in patients with nonvalvular atrial fibrillation: results from the rivaroxaban once daily, oral, direct factor Xa inhibition compared with vitamin K antagonism for prevention of stroke and embolism trial in atrial fibrillation (ROCKET AF). Health care providers should consider renal function, concomitant medication, polymorbidity and age prior to prescribing rivaroxaban. Care has to be taken when prescribing rivaroxaban to patients who are different from those included in the ROCKET AF trial.Health care providers should consider renal function, concomitant medication, polymorbidity and age prior to prescribing rivaroxaban. Care has to be taken when prescribing rivaroxaban to patients who are different from those included in the ROCKET AF trial.A separate trial, J-ROCKET AF, compared the safety of a Japan-specific rivaroxaban dose with warfarin administered according to Japanese guidelines in Japanese patients with AF.J-ROCKET AF was a prospective, randomized, double-blind, phase III trial.A separate trial, J-ROCKET AF, compared the safety of a Japan-specific rivaroxaban dose with warfarin administered according to Japanese guidelines in Japanese patients with AF. J-ROCKET AF was a prospective, randomized, double-blind, phase III trial.\"}, {\"question\": \"List proteins of lipids droplets\", \"ideal_answer\": [\"perilipins\\nadipose differentiation-related protein\\nlipid storage droplet protein 5 \\ntail-interacting protein of 47 kilodaltons \\nS3-12\"], \"exact_answer\": [[\"perilipins\"], [\"adipose differentiation-related protein\", \"ADFP\"], [\"LSDP5\", \"lipid storage droplet protein 5\"], [\"tail-interacting protein of 47 kilodaltons\", \"TIP47\"], [\"S3-12\"]], \"context\": \"The perilipins are a multi-protein family that targets lipid droplet surfaces and regulates lipid storage and hydrolysis.or the abundance of the lipid droplet coat proteins (perilipins 2, 3, 4, and 5) between treatments. This study investigated the lipid droplet coat proteins perilipin 1 (PLIN1) and perilipin 2 (PLIN2) localization in pig skeletal muscle and their relationship with intramuscular fat (IMF) content.Lipid droplets of mammalian cells are coated with one or more members of the PAT protein family, which serve important functions in regulating lipolysis. In this study, we investigate the mechanisms by which PAT family members, perilipin A, adipose differentiation-related protein (ADFP), and LSDP5, control lipolysis catalyzed by hormone-sensitive lipase (HSL), a major lipase in adipocytes and several non-adipose cells.The coordination of lipid storage and utilization is regulated by the perilipin family of lipid droplet coat proteins [perilipin, adipophilin/adipocyte differentiation-related protein (ADRP), S3-12, tail-interacting protein of 47 kilodaltons (TIP47), and myocardial lipid droplet protein (MLDP)/oxidative tissues-enriched PAT protein (OXPAT)/lipid storage droplet protein 5 (LSDP5)]. Perilipin is a member of the evolutionarily related family of PAT proteins (Perilipin, Adipophilin, TIP47), which is defined by sequence similarity and association with lipid droplets. \"}, {\"question\": \"What are the indications for treatment with anti-hepcidin?\", \"ideal_answer\": [\"improving anemia control\\nanemia management in hemodialysis\\niron-restricted anemias\"], \"exact_answer\": [[\"iron-restricted anemias\"], [\"anemia of inflammation\"], [\"anemia of cancer\"], [\"anemia of chronic kidney disease\"], [\"anemia of chronic disease\"]], \"context\": \"anti-hepcidin strategies for improving anemia control.anti-hepcidin therapies may improve anemia management in hemodialysis.Anti-hepcidin therapy for iron-restricted anemiashuman anti-hepcidin antibody as a novel therapeutic for iron-restricted anemias such as anemia of inflammation, cancer, or chronic kidney disease \"}, {\"question\": \"List functions that are evaluated with the Full Outline of Unresponsiveness score?\", \"ideal_answer\": [\"The FOUR (Full Outline of UnResponsiveness) score, a new coma scale, evaluates 4 components: eye and motor responses, brainstem reflexes and respiration.\"], \"exact_answer\": [[\"eye responses\"], [\"motor responses\"], [\"brainstem reflexes\"], [\"respiration\"]], \"context\": \"The Full Outline of UnResponsiveness (FOUR) Score is a coma scale that consists of four components (eye and motor response, brainstem reflexes, and respiration). Recently, the full outline of unresponsiveness (FOUR) score was introduced, which is composed of four clinically distinct categories of evaluation: eye reaction, motor function, brainstem reflexes and respiratory pattern. The FOUR (Full Outline of UnResponsiveness) score, a new coma scale, evaluates 4 components: eye and motor responses, brainstem reflexes and respiration. The FOUR (Full Outline of UnResponsiveness) score, a new coma scale, evaluates 4 components: eye and motor responses, brainstem reflexes and respiration.To evaluate the validity of the FOUR (Full Outline of UnResponsiveness) score (ranging from 0 to 16), a new coma scale consisting of 4 components (eye response, motor response, brainstem reflexes, and respiration pattern), when used by the staff members of a medical intensive care unit (ICU).To evaluate the validity of the FOUR (Full Outline of UnResponsiveness) score (ranging from 0 to 16), a new coma scale consisting of 4 components (eye response, motor response, brainstem reflexes, and respiration pattern), when used by the staff members of a medical intensive care unit (ICU).Recently, the full outline of unresponsiveness (FOUR) score was introduced, which is composed of four clinically distinct categories of evaluation: eye reaction, motor function, brainstem reflexes and respiratory pattern.The FOUR (Full Outline of UnResponsiveness) score, a new coma scale, evaluates 4 components: eye and motor responses, brainstem reflexes and respiration.\"}, {\"question\": \"For the constructions of which organs has 3D printing been tested?\", \"ideal_answer\": [\"Nose, ear and meniscus prototypes/constructs have been produced with 3D (3-dimesional) printing.\"], \"exact_answer\": [[\"nose\"], [\"ear\"], [\"meniscus\"]], \"context\": \"To determine the potential of an integrated, image-based computer-aided design (CAD) and 3-dimensional (3D) printing approach to engineer scaffolds for head and neck cartilaginous reconstruction for auricular and nasal reconstruction.Subcutaneous in vivo implantation of auricular and nasal scaffolds was performed in a porcine model.Auricular and nasal constructs with several types of microporous architecture were rapidly manufactured with high fidelity to human patient anatomy. Subcutaneous in vivo implantation of auricular and nasal scaffolds resulted in an excellent appearance and complete soft tissue ingrowth. Histological analysis of in vitro scaffolds demonstrated native-appearing cartilaginous growth that respected the boundaries of the scaffold.CONCLUSION: Integrated, image-based CAD and 3D printing processes generated patient-specific nasal and auricular scaffolds that supported cartilage regeneration.A prototype meniscus cartilage was prepared to illustrate the potential application in bioengineering.As a proof of concept, we generated a bionic ear via 3D printing of a cell-seeded hydrogel matrix in the anatomic geometry of a human ear, along with an intertwined conducting polymer consisting of infused silver nanoparticles. This allowed for in vitro culturing of cartilage tissue around an inductive coil antenna in the ear, which subsequently enables readout of inductively-coupled signals from cochlea-shaped electrodes. The printed ear exhibits enhanced auditory sensing for radio frequency reception, and complementary left and right ears can listen to stereo audio music. In this study, we describe the construction of a hybrid inkjet printing/electrospinning system that can be used to fabricate viable tissues for cartilage tissue engineering applications.Before 3D Printing can be used routinely for the regeneration of complex tissues (e.g. bone, cartilage, muscles, vessels, nerves in the craniomaxillofacial complex), and complex organs with intricate 3D microarchitecture (e.g. liver, lymphoid organs), several technological limitations must be addressed3D Printing promises to produce complex biomedical devices according to computer design using patient-specific anatomical data3D printing of HEK 293FT cell-laden hydrogel into macroporous constructs with high cell viability and normal biological functions.3D-printing of lightweight cellular composites.3D bioprinting is being applied to regenerative medicine to address the need for tissues and organs suitable for transplantation. In this paper, we reported a novel study of 3D printing of cell lines derived from human embryonic kidney tissue into a macroporous tissue-like construct. Compared with non-biological printing, 3D bioprinting involves additional complexities, such as the choice of materials, cell types, growth and differentiation factors, and technical challenges related to the sensitivities of living cells and the construction of tissues. Organ printing, which we define as computer-aided, jet-based 3D tissue-engineering of living human organs, offers a possible solution. Organ printing involves three sequential steps: pre-processing or development of \\\"blueprints\\\" for organs; processing or actual organ printing; and postprocessing or organ conditioning and accelerated organ maturation.Organ printing involves three sequential steps: pre-processing or development of \\\"blueprints\\\" for organs; processing or actual organ printing; and postprocessing or organ conditioning and accelerated organ maturation.Bioprinting has no or little side effect to the printed mammalian cells and it can conveniently combine with gene transfection or drug delivery to the ejected living systems during the precise placement for tissue construction.Compared with non-biological printing, 3D bioprinting involves additional complexities, such as the choice of materials, cell types, growth and differentiation factors, and technical challenges related to the sensitivities of living cells and the construction of tissues.Innovations in 3D printing: a 3D overview from optics to organs.Organ printing: computer-aided jet-based 3D tissue engineering.\"}, {\"question\": \"List three major features of the CCFDN syndrome.\", \"ideal_answer\": [\"Congenital cataracts, facial dysmorphism and peripheral neuropathy are three major features of the CCFDN syndrome. Other described signs and symptoms of the CCFDN syndrome include microcornea, microphthalmos, micropupil, floppy eyelid syndrome, pseudoptosis, nystagmus, congenital esotropia, impairment of distant visual acuity, ataxia, pyramidal signs, mild chorea, short stature, muscular atrophy, delayed early motor and intellectual development, hypogonadotrop hypogonadism, hypomyelination of the peripheral nervous system, serious complications related to general anaesthesia and parainfectious rhabdomyolysis.\"], \"exact_answer\": [[\"congenital cataracts\"], [\"facial dysmorphism\"], [\"peripheral neuropathy\"]], \"context\": \"Because of our experience in neuromuscular disorders, we choose six, frequent, autosomal recessive disorders for these clinical and genetic studies: hereditary motor and sensory neuropathy type Lom (HMSNL), hereditary motor and sensory neuropathy type Russe (HMSNR), congenital cataracts facial dysmorphism syndrome (CCFDN), limb-girdle muscular dystrophy 2C (LGMD2C), congenital myasthenic syndrome (CMS) and spinal muscular atrophy (SMA).The congenital cataracts facial dysmorphism neuropathy (CCFDN) syndrome (OMIM 604168) is a recently described autosomal recessive developmental disorder. Congenital cataracts-facial dysmorphism-neuropathy syndrome (CCFDN, MIM: 604168), is a recently delineated neurogenetic disease causing recurrent episodes of rhabdomyolysis; prevention and early diagnosis of rhabdomyolysis should be part of the clinical management of the disease.Congenital Cataracts Facial Dysmorphism Neuropathy (CCFDN) syndrome is a complex developmental disorder of autosomal recessive inheritance. OBJECTIVE: To determine the nature and course of ophthalmologic abnormalities in congenital cataracts facial dysmorphism neuropathy (CCFDN) syndrome in a genetically verified group of 9 patients.Congenital cataracts facial dysmorphism neuropathy (CCFDN) syndrome (OMIM 604168) is an autosomal recessive developmental disorder that occurs in an endogamous group of Vlax Roma (Gypsies; refs. 1-3). The major differential diagnosis of MSS is the syndrome defined by congenital cataracts, facial dysmorphism and peripheral neuropathy (CCFDN), which is localised to 18qter.The CCFDN syndrome is a complex phenotype involving multiple systems, characterized by facial dysmorphism, congenital cataracts, microcorneae, delayed early motor and intellectual development, hypogonadotrop hypogonadism, hypomyelination of the peripheral nervous system, and serious complications related to general anaesthesia. Developmental abnormalities include congenital cataracts and microcorneae, primary hypomyelination of the peripheral nervous system, impaired physical growth, delayed early motor and intellectual development, mild facial dysmorphism and hypogonadism. Para-infectious rhabdomyolysis is a serious complication reported in an increasing number of patients. CONCLUSIONS: Early-onset congenital cataracts associated with microcornea, microphthalmos, and micropupil are essential ocular features of the CCFDN syndrome and are the first recognizable signs during early infancy. All patients showed a peripheral, demyelinating neuropathy and varying degrees of ataxia. In the older patients, muscular atrophy in distal muscles and facial dysmorphism was evident. Early-onset bilateral congenital cataracts associated with microcornea, microphthalmos, and micropupil could be found in all patients. All children had floppy eyelid syndrome and pseudoptosis.All patients had syndrome-associated nystagmus and congenital esotropia. Distant visual acuity could be classified as severe to moderate impairment, whereas near visual acuity was much better (mild to moderate impairment).Because these families originated from the same genetically isolated founder population as did patients with congenital cataracts facial dysmorphism neuropathy (CCFDN) syndrome, and because the two syndromes have clinical manifestations in common, we hypothesized that the two related, albeit distinct, syndromes may represent clinical variants of a single genetic disorder. Peripheral nerve abnormalities in the congenital cataracts facial dysmorphism neuropathy (CCFDN) syndrome.Congenital cataracts facial dysmorphism neuropathy (CCFDN) syndrome: a novel developmental disorder in Gypsies maps to 18qter.We propose to refer to the syndrome as congenital cataracts facial dysmorphism neuropathy (CCFDN). The disorder has been termed the congenital cataracts facial dysmorphism neuropathy (CCFDN) syndrome to emphasize its salient features. Associated neurological features are a moderate nonprogressive cognitive deficit in most affected individuals together with pyramidal signs and mild chorea in some. Accompanying nonneurological features include short stature, characteristic facial dysmorphism, and hypogonadotrophic hypogonadism. Nerve conduction studies suggest a hypomyelinating/demyelinating neuropathy, confirmed by nerve biopsy. Congenital cataracts facial dysmorphism neuropathy (CCFDN) syndrome: a rare cause of parainfectious rhabdomyolysis.CONCLUSIONS: Early-onset congenital cataracts associated with microcornea, microphthalmos, and micropupil are essential ocular features of the CCFDN syndrome and are the first recognizable signs during early infancy.The disorder has been termed the congenital cataracts facial dysmorphism neuropathy (CCFDN) syndrome to emphasize its salient features.Because these families originated from the same genetically isolated founder population as did patients with congenital cataracts facial dysmorphism neuropathy (CCFDN) syndrome, and because the two syndromes have clinical manifestations in common, we hypothesized that the two related, albeit distinct, syndromes may represent clinical variants of a single genetic disorder.Congenital cataracts-facial dysmorphism-neuropathy syndrome (CCFDN, MIM: 604168), is a recently delineated neurogenetic disease causing recurrent episodes of rhabdomyolysis; prevention and early diagnosis of rhabdomyolysis should be part of the clinical management of the diseaseThe major differential diagnosis of MSS is the syndrome defined by congenital cataracts, facial dysmorphism and peripheral neuropathy (CCFDN), which is localised to 18qter\"}, {\"question\": \"Which oncogenes are able to induce cellular senescence?\", \"ideal_answer\": [\"Cellular senescence can be induced through activation or inactivation of a number of oncogenes, such as Ras, c-Abl, Raf, Myc, Skp2, BRAF, AKT, HDAC2, p38 MAPK, Caveolin-1 and Mek1.\"], \"exact_answer\": [[\"Ras\"], [\"c-Abl\"], [\"Raf\"], [\"Myc\"], [\"Skp2\"], [\"BRAF\"], [\"AKT\"], [\"HDAC2\"], [\"p38 MAPK\"], [\"Caveolin-1\"], [\"Mek1\"]], \"context\": \"ROS-generating oxidases Nox1 and Nox4 contribute to oncogenic Ras-induced premature senescenceActivated oncogenes induce premature cellular senescence, a permanent state of proliferative arrest in primary rodent and human fibroblasts. Recent studies suggest that generation of reactive oxygen species (ROS) is involved in oncogenic Ras-induced premature senescence.these findings suggest that Nox1- and Nox4-generated ROS play an important role in Ras-induced premature senescenceA role for c-Abl in cell senescence and spontaneous immortalizationHere, we report an important role for c-Abl in replicative senescence and immortalization by regulating the expression of two tumor suppressors that induce cellular senescence, p53 and p16(INK4a).The role for c-Abl in regulating cell senescence and immortalization might explain some of the developmental defects in c-Abl (-/-) mice and how BCR-ABL transforms cells.Ras/Raf-prototypic oncogenes induce cellular senescence, a terminal cell-cycle arrest, as a default cellular safeguard programoncogene-induced senescence, Myc styleWe review and discuss here evidence for Myc-induced senescence - which is detectable to a limited degree as a cell-autonomous, direct response to Myc action, but occurs predominantly in a non-cell-autonomous fashion via crosstalk of the oncogene-driven cell population with non-neoplastic bystanders, namely cells of the host immune system, prompting them to release pro-senescent cytokines that strike back onto adjacent proliferating tumor cells.genetic Skp2 inactivation evokes cellular senescence even in oncogenic conditions in which the p19(Arf)-p53 response is impaired, whereas a Skp2-SCF complex inhibitor can trigger cellular senescence in p53/Pten-deficient cells and tumour regression in preclinical studies.Activated RAS/BRAF oncogenes induce cellular senescence as a tumor-suppressive barrier in early cancer development, at least in part, via an oncogene-evoked DNA damage response (DDR).Cdk2 suppresses cellular senescence induced by the c-myc oncogeneLoss of Cdk2 also caused sensitization to Myc-induced senescence in pancreatic beta-cells or splenic B-cells in vivoMyc also causes senescence in cells lacking the DNA repair protein Wrn.In MEFs, Myc-induced senescence was genetically dependent on the ARF-p53-p21Cip1 and p16INK4a-pRb pathways, p21Cip1 and p16INK4a being selectively induced in Cdk2-/- cells.We have found that expression of such an oncogene, Akt, causes senescence in primary mouse hepatoblasts in vitro.hyperproliferative signaling through AKT might be a driving force of the senescence in activated hepatic stellate cells.oncogenic RAS expression, which causes genotoxic stress and senescence in normal cells,Here we report that the suppression of the c-Myc (MYC) oncogene induces cellular senescence in diverse tumor types including lymphoma, osteosarcoma, and hepatocellular carcinoma.Cellular senescence is an important mechanism of tumor regression upon c-Myc inactivationMYC inactivation was associated with prototypical markers of senescenceonly after lymphomas were repaired for p53 expression did MYC inactivation induce robust senescence and sustained tumor regression.Replicative senescence due to telomere shortening can, for example, be induced by a dominant negative version of telomerase, premature senescence by the overexpression of oncogenic ras, or p16.knockdown of HDAC2 induced cellular senescence.MYC overexpression is thought to initiate tumorigenesis by inducing cellular proliferation and growth and to be restrained from causing tumorigenesis by inducing cell cycle arrest, cellular senescence, and/or apoptosis.Several conditions, including oncogenic Ras over-expression and inappropriate culture conditions, also induce senescence without telomere shortening.We demonstrate that p38 mitogen-activated protein kinase (MAPK) plays important causative roles in senescent cells following telomere shortening, Ras-Raf activation, oxidative stress or inappropriate culture conditions.Mitogen-activated protein kinase p38 defines the common senescence-signalling pathwayThese results indicate that p38 comprises the senescence-executing pathway in response to diverse stimuli.this p38-activating condition appears to be defined quantitatively as a sum of continuous and low-level stresses, and remains even after the initial stimuli are withdrawn, which may explain the well-known irreversible nature of cellular senescence. We also show that papilloma virus E7 abolishes the p38-induced growth arrest but not other senescence-associated phenotypes, indicating the differential role of pRb in the downstream of p38.Expression of caveolin-1 induces premature cellular senescence in primary cultures of murine fibroblastsHere, we demonstrate that mouse embryonic fibroblasts transgenically overexpressing caveolin-1 show: 1) a reduced proliferative lifespan; 2) senescence-like cell morphology; and 3) a senescence-associated increase in beta-galactosidase activity. These results indicate for the first time that the expression of caveolin-1 in vivo is sufficient to promote and maintain the senescent phenotype.Interestingly, premature senescence induced by hydrogen peroxide is greatly reduced in NIH 3T3 cells harboring antisense caveolin-1. Importantly, induction of premature senescence is recovered when caveolin-1 levels are restored.these results clearly indicate a central role for caveolin-1 in promoting cellular senescenceOncogenic activation of the mitogen-activated protein (MAP) kinase cascade in murine fibroblasts initiates a senescence-like cell cycle arrest that depends on the ARF/p53 tumor suppressor pathway.In contrast, coexpression of oncogenic ras or activated mek1 with p53 enhanced both p53 levels and activity relative to that observed for p53 alone and produced an irreversible cell cycle arrest that displayed features of cellular senescence.our results indicate that oncogenic activation of the MAP kinase pathway in murine fibroblasts converts p53 into a senescence inducer through both quantitative and qualitative mechanisms.Myc-evoked apoptosis serves as a signal for macrophage attraction and activation, followed by the secretion of TGF-\\u03b2 as a cytokine that is capable of terminally arresting Myc-driven lymphoma cells without causing further DNA damage and without launching a senescence-associated, pro-inflammatory, and, therefore, potentially detrimental cytokine response in the target population.The suppression of oncogenic levels of MYC is sufficient to induce sustained tumor regression associated with proliferative arrest, differentiation, cellular senescence, and/or apoptosis, a phenomenon known as oncogene addiction.\"}, {\"question\": \"Which are the different proteins/isoforms encoded but the ASPH (aspartate beta-hydroxylase) gene in humans?\", \"ideal_answer\": [\"Alternative splicing of the locus AbetaH-J-J (asparetyl-beta-hydroxylase) generates three functionally distinct proteins: an enzyme, AbetaH (aspartyl-beta-hydroxylase), a structural protein of the sarcoplasmic reticulum membrane (junctin), and an integral membrane calcium binding protein (junctate). Aspartyl (asparaginyl)-beta-hydroxylase (AAH), has also two related transcripts, Humbug and Junctin, which lack catalytic domains.  The smallest BAH-related transcript (2,789 base pairs) uses an alternative 3' terminal exon, resulting in a protein lacking a catalytic domain. Evolutionary conservation of this noncatalytic isoform of BAH (humbug) is demonstrated in mouse, man, and Drosophila. A human junctin isoform (isoform 1, 225 aa) was also identified and characterized. The isoform 1 has a 15 aa insertion at the amino acid residue 55 of the human junctin.\"], \"exact_answer\": [[\"Aspartyl-beta-hydroxylase\", \"Aspartyl (asparaginyl)-beta-hydroxylase\", \"AbetaH\", \"AAH\", \"BAH\"], [\"humbug\"], [\"junctate\"], [\"junctin\"], [\"junctin isoform\"]], \"context\": \"The single genomic locus, AbetaH-J-J, encodes three functionally distinct proteins aspartyl beta-hydroxylase, junctin and junctate by alternative splicing.The human AbetaH-J-J locus is a genomic sequence which generates three functionally distinct proteins, the enzyme aspartyl-beta-hydroxylase (AbetaH), the structural protein of sarcoplasmic reticulum junctin, and the membrane-bound calcium binding protein junctate.Aspartyl (asparaginyl)-beta-hydroxylase (AAH) hydroxylates Asp and Asn residues within EGF-like domains of Notch and Jagged, which mediate cell motility and differentiation. This study examines the expression, regulation and function of AAH, and its related transcripts, Humbug and Junctin, which lack catalytic domains, using SH-Sy5y neuroblastoma cells.Alternative splicing of the locus AbetaH-J-J generates three functionally distinct proteins: an enzyme, AbetaH (aspartyl-beta-hydroxylase), a structural protein of the sarcoplasmic reticulum membrane (junctin), and an integral membrane calcium binding protein (junctate).Junctate is a newly identified integral ER/SR membrane calcium binding protein, which is an alternative splicing form of the same gene generating aspartyl beta-hydroxylase and junctin.Screening a mouse heart cDNA library using canine junctin cDNA as a probe yielded three complete mouse heart cDNAs. One of the cDNAs is homologous to the previously reported human junctate.Screening a cDNA library from human skeletal muscle and cardiac muscle with a cDNA probe derived from junctin led to the isolation of two groups of cDNA clones. The first group displayed a deduced amino acid sequence that is 84% identical to that of dog heart junctin, whereas the second group had a single open reading frame that encoded a polypeptide with a predicted mass of 33 kDa, whose first 78 NH(2)-terminal residues are identical to junctin whereas its COOH terminus domain is identical to aspartyl beta-hydroxylase, a member of the alpha-ketoglutarate-dependent dioxygenase family. We named the latter amino acid sequence junctate.Our study shows that alternative splicing of the same gene generates the following functionally distinct proteins: an enzyme (aspartyl beta-hydroxylase), a structural protein of SR (junctin), and a membrane-bound calcium binding protein (junctate).Junctin is a calsequestrin binding protein detected in junctional sarcoplasmic reticulum of striated muscles. In the present study, the human cardiac junctin cDNA has been cloned by human heart cDNA library screening and RT-PCR, and the cDNA sequence has been determined. The deduced amino acid sequence of human junctin (210 aa) has 84% sequence identity to that of canine junctin identified previously. A human junctin isoform (isoform 1, 225 aa) was also identified and characterized. The isoform 1 has a 15 aa insertion at the amino acid residue 55 of the human junctin. Northern blot analysis revealed that the human junctin was present both in cardiac and skeletal muscles, and the sizes of the transcripts were approximately 3.0 and 4.2kb. Amino acid residues 6-78 of human junctin and 35-107 of human aspartyl beta-hydroxylase (hAspH) overlapped perfectly.The mouse aspartyl beta-hydroxylase gene (Asph, BAH) has been cloned and characterized. The mouse BAH gene spans 200 kilobase pairs of genomic DNA and contains 24 exons. Of three major BAH-related transcripts, the two largest (6,629 and 4,419 base pairs) encode full-length protein and differ only in the use of alternative polyadenylation signals. The smallest BAH-related transcript (2,789 base pairs) uses an alternative 3' terminal exon, resulting in a protein lacking a catalytic domain. Evolutionary conservation of this noncatalytic isoform of BAH (humbug) is demonstrated in mouse, man, and Drosophila.The single genomic locus, AbetaH-J-J, encodes three functionally distinct proteins aspartyl beta-hydroxylase, junctin and junctate by alternative splicingMultiple levels of control of the expression of the human A beta H-J-J locus encoding aspartyl-beta-hydroxylase, junctin, and junctate.A human junctin isoform (isoform 1, 225 aa) was also identified and characterized\"}, {\"question\": \"List processes which are under the control of the YAP protein.\", \"ideal_answer\": [\"Yes-associated protein (YAP), a transcription coactivator, is the major downstream effector of the Hippo pathway, which plays a critical role in organ size control, cell poliferation and cancer development and tissue homeostasis and differentiation.\"], \"exact_answer\": [[\"cell proliferation\"], [\"organ size control\"], [\"cancer development\"], [\"tissue homeostasis\"], [\"cell differentiation\"]], \"context\": \"in which YAP appears to regulate cell proliferation YAP and TAZ oncoproteins confer malignancy and drug resistance to various cancer types. Yes-associated protein (YAP), a transcription coactivator, is the major downstream effector of the Hippo pathway, which plays a critical role in organ size control and cancer developmentYAP (Yes-associated protein) and TAZ (transcriptional coactivator with PDZ-binding motif) are major downstream effectors of the Hippo pathway that influences tissue homeostasis, organ size, and cancer development. The Hippo signaling pathway converges on YAP to regulate growth, differentiation, and regeneration.Yes associated protein (YAP) and transcriptional co-activator with PDZ-binding motif (TAZ) are its targets and terminal effectors: inhibition of the pathway promotes YAP/TAZ translocation to the nucleus, where they interact with transcriptional enhancer associate domain (TEAD) transcription factors and coactivate the expression of target genes, promoting cell proliferation. Intracellular molecules Yap, Akt, mTOR, and Erk are signaling pathway members that regulate the proliferation of tumor cells\"}, {\"question\": \"List omics technologies comprised in system biology.\", \"ideal_answer\": [\"System biology combines various omics technologies such as genomics, transcriptomics, proteomics, metabolomics, epigenomics, glucomics, degradomics and fluxomics.\"], \"exact_answer\": [[\"genomics\"], [\"transcriptomics\"], [\"proteomics\"], [\"metabolomics\"], [\"degradomics\"], [\"fluxomics\"], [\"epigenomics\"], [\"glucomics\"]], \"context\": \"The increasing role of metabolomics in system biology is driving the development of tools for comprehensive analysis of high-resolution NMR spectral datasets.Proteomics technology, a major component of system biology, has gained comprehensive attention in the area of medical diagnosis, drug development, and mechanism research.Metabolomics complements other omics, such as transcriptomics and proteomics and since it is a 'downstream' result of gene expression, changes in the metabolome is considered to best reflect the activities of the cell at a functional level. In addition, integrated applications with genomics, transcriptomics, and proteomics provide greater understanding of global system biology.From proteomics to integrated omics towards system biology.In recent years, in the post genomic era, more and more data is being generated by biological high throughput technologies, such as proteomics and transcriptomics. Glycomics meets genomics, epigenomics and other high throughput omics for system biology studies.The current state-of-the-art in high-throughput glycomics and its integration with genomics, epigenomics and lipidomics is reviewed in this article.The resulting lists of metabolites and their steady state concentrations in combination with transcriptomics can be used in system biology approaches.all system biology aspects (genomics, transcriptomics, proteomics, metabolomics, degradomics and fluxomics). Recent development in high-throughput methods enables analysis of genome, transcriptome, proteome, and metabolome at an unprecedented scale, thus contributing to the deluge of experimental data in numerous public databases. \"}, {\"question\": \"Which gene mutations are responsible for isolated Non-compaction cardiomyopathy?\", \"ideal_answer\": [\"The gene mutations that have been shown to be the causes of isolated non-compaction cardiomyopathy are alpha-tropomyosin, alpha-tropomyosin, troponin T and desmoplakin\", \"cardiac \\u03b2-myosin heavy chain gene (MYH7)\\nc.349G>A (p.D117N) in the ZASP gene\\nmutation in the isoform-1 specific region of the DSP C-terminus\\npE96K mutation in the cardiac troponin T gene (TNNT2)\"], \"exact_answer\": [[\"alpha-tropomyosin\", \"TPM1\"], [\"\\u03b2-myosin heavy chain\", \"MYH7\"], [\"troponin T\", \"TNNT2\", \"pE96K mutation\"], [\"Desmoplakin\", \"DSP\", \"mutation in the isoform-1 specific region of the DSP C-terminus\"], [\"mindbomb homolog 1\", \"MIB1\"], [\"c.349G>A (p.D117N) in the ZASP gene\"]], \"context\": \"A novel alpha-tropomyosin mutation associates with dilated and non-compaction cardiomyopathyWe show that a mutation in TPM1 is associated with DCM and a lethal, early onset form of NCCMMutation in MYH7B causes a classical LVNC phenotypeWe report on two prenatal ultrasound diagnoses of left ventricular non-compaction cardiomyopathy (LVNC) associated with mutation of the cardiac \\u03b2-myosin heavy chain gene (MYH7)This novel homozygous truncating mutation in the isoform-1 specific region of the DSP C-terminus caused Carvajal syndrome comprising severe early-onset heart failure with features of non-compaction cardiomyopathySevere familial left ventricular non-compaction cardiomyopathy due to a novel troponin T (TNNT2) mutationwe describe a family with LVNC due to a novel missense mutation, pE96K, in the cardiac troponin T geneWe describe, in two separate autosomal-dominant NCCM families, the identification of mutations in the sarcomeric cardiac beta-myosin heavy chain gene (MYH7)the molecular classification of cardiomyopathies includes an MYH7-associated spectrum of NCCM\"}, {\"question\": \"What is the treatment of acute myocarditis?\", \"ideal_answer\": [\"Treatment of acute myocarditis includes antiinflammatory drugs like ibuoprofen and steroids, inotropic agents and mechanical support (intra-aortic ballon pump). TandemHeart percutaneous ventricular assist device may be used in some, more compromised, patients for few days.\"], \"exact_answer\": [[\"antiinflammatory steroid and non steroid drugs\"], [\"inotropic agents\"], [\"mechanical support\"]], \"context\": \"ibuprofen 400 mg twice a day as therapyAcute fulminant myocarditis commonly manifests itself as severe, rapidly progressive hemodynamic deterioration and circulatory collapse that may be resistant to high doses of inotropic agents and steroids and to mechanical support by intra-aortic balloon pumpthe TandemHeart percutaneous ventricular assist device, can enable patients to recover in a few days.he authors report a typical case of fulminating myocarditis with electromechanical dissociation, which recovered completely after a period of circulatory assistance.To clarify the effects of Astragalus Membranaceus (AM) combined with taurine and/or coenzyme Q10(CoQ10) on coxsackievirus B3 (CVB3) murine myocarditisAM, taurine and CoQ10 have some curative effects on CVB3 murine myocarditis, AM combined with taurine and CoQ10 is the best.\"}, {\"question\": \"List causative genes for autosomal recessive forms of monogenic Parkinson's disease\", \"ideal_answer\": [\"Causative genes for autosomal recessive forms of monogenic Parkinson's disease are:\\nPARK2\\nPARK7\\nPINK1\\nPARK9\\nPARK14\\nPARK15\"], \"exact_answer\": [[\"PARK2\"], [\"PARK7\", \"DJ-1\"], [\"PINK1\"], [\"PARK9\", \"ATP13A2\"], [\"PARK14\", \"PLA2G6\"], [\"PARK15\", \"FBX07\"]], \"context\": \"Mutations in the Parkin gene (PARK2) are the major cause of autosomal recessive early-onset parkinsonism.Early onset PD (EOPD) is particularly associated with autosomal recessive (AR) mutations, and three genes, PARK2, PARK7 and PINK1, have been found to carry mutations leading to AR disease. Homozygous or compound heterozygous mutations in the phosphatase and tensin homolog-induced putative kinase 1 (PINK1) gene are causative of autosomal recessive, early onset Parkinson's disease. Mutations in other genes, including ATP13A2 (PARK9), PLA2G6 (PARK14), and FBX07 (PARK15), cause more rare forms of recessive parkinsonism with very early-onset (<30 years)In three forms, caused by mutations in parkin (PARK2), PINK1 (PARK6), or DJ-1 (PARK7),oss-of-function mutations in Parkin, PINK1, DJ-1 and ATP13A2 cause autosomal recessive parkinsonism with early-onset.mutations in the parkin gene, in DJ-1, PINK1 and ATP13A2 all cause autosomal-recessive parkinsonism of early onset.mutations in the parkin gene, in DJ-1, PINK1 and ATP13A2 cause autosomal recessive parkinsonism of early onsetMutations in the parkin gene, in DJ-1 and PINK1 all cause autosomal recessive parkinsonism of early onset.PTEN-induced putative kinase 1 (PINK1) is a causative gene for autosomal recessive early onset parkinsonism. we focus on PARK7, a gene relates to an autosomal recessive form of early-onset Parkinsonism and encodes a protein named DJ-1.PINK-1 for an autosomal-recessive early-onset variant Heritable mutations in alpha-synuclein, parkin, DJ-1 and PINK1 cause familial forms of PD. we focus on PARK7, an autosomal recessive form of early-onset parkinsonism caused by mutations in the DJ-1 geneMutations in the DJ-1 gene associated with autosomal recessive early-onset parkinsonism.Autosomal recessive parkinsonism associated with mutations in the parkin gene represents a monogenic form of hereditary parkinsonism.Mutations of the parkin gene on chromosome 6 cause autosomal recessive, early onset parkinsonism.  A gene causing autosomal recessive parkinsonism of juvenile onset has been mapped to chromosome 6 (PARK 2), and the causative gene has been identified and named parkin. The gene responsible for AR-JP was recently identified and designated parkin.\"}, {\"question\": \"List symptoms of congenital toxoplasmosis triad.\", \"ideal_answer\": [\"Classic triad of toxoplasmosis include hydrocephalus, cerebral calcification and chorioretinitis.\"], \"exact_answer\": [[\"hydrocephalus\"], [\"cerebral calcification\"], [\"chorioretinitis\"]], \"context\": \"Most patients had systemic manifestations, but only 25% of diagnosed patients exhibited the classic triad of hydrocephalus, cerebral calcification and chorioretinitis.The pathogenic potential of T. gondii was recognized in the 1920s and 1930s, in congenitally infected children presenting with the classic triad of symptoms, namely hydrocephalus, retinochoroiditis and encephalitis. Intracranial calcifications within the classical triad were only found in two cases.Congenital toxoplasmosis is characterized by the classical triads: hydrocephaly, chorioretinitis and intracerebral calcifications.Most patients had systemic manifestations, but only 25% of diagnosed patients exhibited the classic triad of hydrocephalus, cerebral calcification and chorioretinitis. \"}, {\"question\": \"Which cellular processes are regulated by Nanog?\", \"ideal_answer\": [\"The pluripotency sustaining factor Nanog, controls a cascade of pathways that are intricately connected to govern pluripotency, self-renewal, genome surveillance and cell fate determination. Elevated expression of Nanog has also been reported to result in clonal expansion of murine ESCs, but it also plays a role in tumor development. A positive regulator of cell proliferation, it is essential for G1 to S transition in human embryonic stem cells while it regulates primordial germ cell migration.\"], \"exact_answer\": [[\"cell proliferation\"], [\"pluripotency\"], [\"cell fate determination\"], [\"G1 to S transition\"], [\"germ cell migration\"], [\"tumour development\"]], \"context\": \"critical for the regulation of cancer stem cells.the effect of FAK and Nanog cross-regulation on cancer cell morphology, invasion, and growth that plays a significant role in carcinogenesisNanog-like regulates endoderm formationIn mammalian embryonic stem cells, the acquisition of pluripotency is dependent on Nanogwe identified a zebrafish Nanog ortholog and found that its\\u00a0knockdown impaired endoderm formationestablishes a role for Nanog-like in regulating the formation of the extraembryonic tissue required for endoderm inductionNanog regulates molecules involved in stemness and cell cycle-signaling pathway for maintenance of pluripotencyNanog, a key transcription factor in the maintenance of pluripotency of embryonic stem (ES) and embryonal carcinoma (EC) cellsNanog, a positive regulator of ESC proliferation and G1/S transitionNanog is a stem cell transcription factor required for self-renewal and for maintaining pluripotencyNANOG regulates glioma stem cellsWe find that NANOG modulates gliomasphere clonogenicity, CD133(+) stem cell cell behavior and proliferationNANOG is essential for GBM tumourigenicityNanog regulates primordial germ cell migrationNanog mediates PGC migration by regulating Cxcr4b expressionNanog regulates proliferation during early fish development.Nanog is necessary for S-phase transition and proliferation in the developing embryoour results demonstrate that NANOG, a cell-fate regulatory molecule known to be important for ESC self-renewal, also plays a novel role in tumor developmentthe self-renewal gene NANOG regulates human tumor developmentThe emerging picture is one in which Oct4 and Nanog control a cascade of pathways that are intricately connected to govern pluripotency, self-renewal, genome surveillance and cell fate determinationthe pluripotency sustaining factor nanog\"}, {\"question\": \"Which mutations of alpha-myosin heavy chain gene are implicated in hypertrophic cardiomyopathy?\", \"ideal_answer\": [\"The following mutations of alpha-myosin heavy chain gene are implicated in hypertrophic cardiomyopathy: R403Q; Q1065H and Arg-249-->Gln\"], \"exact_answer\": [[\"R403Q\", \"Arg403Gln\"], [\"Q1065H\"], [\"Arg-249-->Gln\"]], \"context\": \"Accordingly, pressure overload by transverse aortic constriction (TAC) was induced in 2-month-old, male mice with and without a FHC (R403Q) mutation in \\u03b1-myosin heavy chain.This model (designated TnI-203/MHC-403) was generated by crossbreeding mice with the Gly203Ser cardiac troponin I (TnI-203) and Arg403Gln alpha-myosin heavy chain (MHC-403) FHC-causing mutations. Male but not female mice carrying a single R403Q missense allele for cardiac alpha-myosin heavy chain (M-alphaMHC(R403Q/+) and F-alphaMHC(R403Q/+), respectively) develop significant hypertrophic cardiomyopathy (HCM) compared with male and female wild-type mice (M-alphaMHC(+/+) and F-alphaMHC(+/+), respectively) after approximately 30 wk of age.A Q1065H mutation was detected in 1 of 21 HCM probands and was absent in 2 unaffected offspring.To understand further the pathogenesis of familial hypertrophic cardiomyopathy, we determined how the cardiomyopathy induced by an Arg403-->Gln missense mutation in the alpha-myosin heavy chain (403) is affected by chronically enhancing sympathetic drive by mating the mice with those overexpressing G(s)alpha (G(s)alpha x403). A mouse model of FHC resulting from a mutation in the alpha-myosin heavy-chain (Arg403Gln) was used to study the electrophysiologic phenotype of this disease.We used small-amplitude (0.25%) length-perturbation analysis to examine the mechanical properties of skinned left ventricular papillary muscle strips from mouse hearts bearing the R403Q mutation in the alpha-myosin heavy chain (alphaMHC403/+).Genetically-manipulated mice harboring an alpha-myosin heavy chain Arg403Gln missense mutation (alpha-MHC403/+) display a phenotype characteristic of familial hypertrophic cardiomyopathy (FHC).A new mouse cardiac electrophysiology method was used to study mice harboring an alpha-myosin heavy chain Arg403Gln missense mutation (alpha-MHC403/+), which results in histological and hemodynamic abnormalities characteristic of familial hypertrophic cardiomyopathy (FHC) and sudden death of uncertain etiology during exercise. The introduction of the mouse model for FHC (the mouse expresses predominantly alpha-MHC as opposed to the beta-isoform in larger mammals) created a new paradigm for FHC based on finding enhanced motor function for R403Q alpha-MHC.A mouse model of familial hypertrophic cardiomyopathy (FHC) was generated by the introduction of an Arg 403 --> Gln mutation into the alpha cardiac myosin heavy chain (MHC) gene.Biochemical analysis of one FHC mutant (Arg-249-->Gln) demonstrates that the structures formed by the mutant are solubilized at a lower ionic strength than those formed by wild-type MHC. We conclude that although the FHC mutant MHC is not labile, its assembly properties may be impaired.\"}, {\"question\": \"Which are the cardiac manifestations of Marfan syndrome?\", \"ideal_answer\": [\"Cardiac manifestations of Marfan syndrome include aortic root dilation,aortic regurgitation, mitral valve prolapse and mitral valve regurgitation.\"], \"exact_answer\": [[\"aortic root dilation\"], [\"mitral valve prolapse\"], [\"aortic regurgitation\"], [\"mitral valve regurgitation\"]], \"context\": \"Cardiac manifestations of Marfan syndrome include aortic root dilation and mitral valve prolapse (MVP).Marfan syndrome (MFS) is a genetic disorder of the connective tissue. Aortic root dilation is a main criterion of the Ghent Nosology. Dural ectasia and the presence of mitral valve prolapse (MVP) contribute to its systemic score.The typical cardiac manifestations of Marfan syndrome are aortic regurgitation with progressive dilatation of the aortic root, which may cause dissection and rupture of the ascending aorta, mitral valve prolapse and mitral valve regurgitation.To describe the clinical cardiac manifestations and temporal evolution of Marfan syndrome in children; to estimate the incidence of annuloaortic ectasia and mitral valve prolapsethe presence of mitral valve prolapse, aortic root diameter, mitral and aortic valves regurgitation, and aortic enlargement during beta-blocker therapyCardiovascular manifestations in Marfan syndrome.The major cardiac diagnoses were aortic dilatation (1/3) and mitral valve prolapse with severe mitral regurgitation (2/3).Marfan syndrome is a hereditable disorder of connective tissue that causes several distinct cardiovascular abnormalities, including aortic regurgitation, dissection, and aneurysm.\"}, {\"question\": \"What are the results of loss of the protein Lon1 in the plant Arabidopsis?\", \"ideal_answer\": [\"Loss of Lon1 in Arabidopsis changes the mitochondrial proteome leading to altered metabolite profiles and growth retardation. Additionaly, seedling establishment is also impaired.\"], \"exact_answer\": [[\"growth rate deficit in both roots and shoots\"], [\"lowered activity of specific mitochondrial enzymes associated with respiratory metabolism\"], [\"impaired seedling establishment\"]], \"context\": \"Knockout in Arabidopsis (Arabidopsis thaliana) leads to a significant growth rate deficit in both roots and shoots and lowered activity of specific mitochondrial enzymes associated with respiratory metabolism. loss of Lon1 significantly modifies respiratory function and plant performance by small but broad alterations in the mitochondrial proteome gained by subtly changing steady-state protein assembly, stability, and damage of a range of components that debilitate an anaplerotic role for mitochondria in cellular carbon metabolism.the lack of Lon selective proteolysis leading to growth retardation and impaired seedling establishment can be attributed to defects in the oil reserve mobilization pathway. Loss of Lon1 in Arabidopsis changes the mitochondrial proteome leading to altered metabolite profiles and growth retardation without an accumulation of oxidative damage. a genetic screen was performed that led to the identification of Arabidopsis thaliana Lon1 protease mutants that exhibit a post-embryonic growth retardation phenotype.mitochondria isolated from lon1 mutants had a lower capacity for respiration of succinate and cytochrome c via complexes II and IV, respectively. Furthermore, the activity of key enzymes of the tricarboxylic acid (TCA) cycle was significantly reduced. Additionally, mitochondria in lon1 mutants had an aberrant morphology.  AtLon1 protease in organelle biogenesis and seedling establishment.\"}, {\"question\": \"Which are the different members/isoforms of the Ras oncogenes?\", \"ideal_answer\": [\"Ras proteins are proto-oncogenes that are frequently mutated in human cancers. Three closely related isoforms, HRAS, KRAS and NRAS, are expressed in all cells and have overlapping but distinctive functions.\", \"H-ras, N-ras, and K-ras are canonical ras gene family members frequently activated by point mutation in human cancers and coding for 4 different, highly related protein isoforms (H-Ras, N-Ras, K-Ras4A, and K-Ras4B)\"], \"exact_answer\": [[\"H-Ras\"], [\"N-Ras\"], [\"K-Ras4A\"], [\"K-Ras4B\"]], \"context\": \"The major Ras isoforms (K, H, and N) Mutations in Ras isoforms such as K-Ras, N-Ras, and H-Ras contribute to roughly 85, 15, and 1% of human cancers, respectively.the Ras isoforms (H, N and K) The mutant forms of KRas, NRas and HRas lipidated Ras isoforms (H-Ras and N-Ras): H-Ras, K-Ras, and N-Ras regulate cellular growth and survival and are often activated by somatic mutation in human tumors.Human tumours frequently express Ras proteins (Ha-, Ki-, N-Ras)  There are three major ras isoforms: H-, N- and K-Ras. All mammalian cells express 3 closely related Ras proteins, termed H-Ras, K-Ras, and N-Rashree closely related isoforms, HRAS, KRAS and NRAS, H-ras, N-ras, and K-ras are canonical ras gene family members frequently activated by point mutation in human cancers and coding for 4 different, highly related protein isoforms (H-Ras, N-Ras, K-Ras4A, and K-Ras4B).he 3 Ras isoforms-H-Ras, K-Ras, and N-Ras-In this study, we compared the leukemogenic potential of activated NRAS, KRAS, and HRAS in the same bone marrow transduction/transplantation model system. H-Ras, K-Ras, and N-Ras regulate cellular growth and survival and are often activated by somatic mutation in human tumors.Ras proteins are small GTPase functioning as molecular switches that, in response to particular extracellular signalling, as growth factors, activate a diverse array of intracellular effector cascades regulating cell proliferation, differentiation and apoptosis. Human tumours frequently express Ras proteins (Ha-, Ki-, N-Ras) activated by point mutations which contribute to malignant phenotype, including invasiveness and angiogenesis.Ras small GTPases function as transducers of extracellular signals regulating cell survival, growth and differentiation. There are three major ras isoforms: H-, N- and K-Ras. All mammalian cells express 3 closely related Ras proteins, termed H-Ras, K-Ras, and N-Ras, that promote oncogenesis when they are mutationally activated at codon 12, 13, or 61.Ras proteins are proto-oncogenes that are frequently mutated in human cancers. Three closely related isoforms, HRAS, KRAS and NRAS, are expressed in all cells and have overlapping but distinctive functions.H-ras, N-ras, and K-ras are canonical ras gene family members frequently activated by point mutation in human cancers and coding for 4 different, highly related protein isoforms (H-Ras, N-Ras, K-Ras4A, and K-Ras4B)\"}, {\"question\": \"What are the names of anti-CD52 monoclonal antibody that is used for treatment of multiple sclerosis patients?\", \"ideal_answer\": [\"Alemtuzumab and Campath-1H are the names of anti-CD52 monoclonal antibody that is used for treatment of multiple sclerosis patients. It has been shown to be effective for treatment naive and treatment resistant multiple sclerosis patients.\"], \"exact_answer\": [[\"Alemtuzumab\"], [\"Campath-1H\"]], \"context\": \"Alemtuzumab offers induction strategy for very active relapsing MS patients who have failed conventional therapy, and possibly selected treatment-naive patients.Differential reconstitution of T cell subsets following immunodepleting treatment with alemtuzumab (anti-CD52 monoclonal antibody) in patients with relapsing-remitting multiple sclerosis.Alemtuzumab (anti-CD52 mAb) provides long-lasting disease activity suppression in relapsing-remitting multiple sclerosis (RRMS).Alemtuzumab, an anti-CD52 monoclonal antibody, increased the risk of thyroid dysfunction in CAMMS223, a phase 2 trial in relapsing-remitting multiple sclerosis.Since then, one phase II and two phase III trials have shown that alemtuzumab reduces the relapse rate, compared with the active comparator interferon beta-1a (IFN\\u03b2-1a), in treatment-na\\u00efve and treatment-experienced MS up to 10 years from disease onset. Furthermore, in two of these trials, alemtuzumab reduced the risk of accumulating disability compared with IFN\\u03b2-1a; indeed alemtuzumab treatment led to an improvement in disability and reduction in cerebral atrophy.Alemtuzumab is a humanized monoclonal antibody specific for the CD52 protein present at high levels on the surface of B and T lymphocytes. In clinical trials, alemtuzumab has shown a clinical benefit superior to that of interferon-\\u03b2 in relapsing-remitting multiple sclerosis patients.These results potentially help explain the level of immunocompetence observed in alemtuzumab-treated MS patients.Alemtuzumab was first used in multiple sclerosis in 1991. It is a monoclonal antibody which is directed against CD52, a protein of unknown function on lymphocytes. Against an active comparator and the current first-line therapy for relapsing-remitting multiple sclerosis (interferon-beta), alemtuzumab showed a significant reduction in annualized relapse rate as well as a significant reduction in the accumulation of disability. Alemtuzumab (previously known as Campath(\\u00ae)) is a humanized monoclonal antibody directed against the CD52 antigen on mature lymphocytes that results in lymphopenia and subsequent modification of the immune repertoire. Compared to interferon beta-1a, alemtuzumab reduced the relapse rate by 49%-74% (P < 0.0001), and in two studies it reduced the risk of sustained disability accumulation by 42%-71% (P < 0.01). In one study (Comparison of Alemtuzumab and Rebif Efficacy in Multiple Sclerosis; CARE-MS1), there was no significant difference compared to interferon, perhaps reflecting the surprisingly low frequency of disability events in the comparator group.Alemtuzumab has been submitted for licensing in relapsing-remitting multiple sclerosis in the United States and Europe.Alemtuzumab was associated with a greater chance of improved contrast sensitivity in patients with RRMS and may delay the worsening of visual function. The anti-CD52 monoclonal antibody alemtuzumab reduced disease activity in a phase 2 trial of previously untreated patients with relapsing-remitting multiple sclerosis. Alemtuzumab's consistent safety profile and benefit in terms of reductions of relapse support its use for patients with previously untreated relapsing-remitting multiple sclerosis; however, benefit in terms of disability endpoints noted in previous trials was not observed here. Alemtuzumab, formally known as Campath-1H, is a humanized monoclonal antibody directed against CD52, a protein on the surface of lymphocytes and monocytes with unknown function. A single dose of alemtuzumab leads to a rapid, profound and prolonged lymphopenia. A Phase II trial has shown that alemtuzumab reduces the risk of relapse and accumulation of disability by over 70% compared with interferon beta in patients with early relapsing-remitting multiple sclerosis (MS).Alemtuzumab, targeting CD52 expressed on T-cells, B-cells, monocytes and macrophages, has also shown to be effective in early RRMS and phase III trials are currently ongoing. monoclonal antibody therapy has provided the opportunity to rationally direct the therapeutic intervention by specifically targeting mechanisms of the immune system such as CD52 (alemtuzumab), CD25 (daclizumab), VLA-4 (natalizumab) and CD20 (rituximab)Alemtuzumab is a humanized monoclonal antibody targeting CD52, a broadly expressed cell surface molecule on immune cells.Alemtuzumab-treatment of MS patients with relapsing-remitting forms of the disease significantly reduced the risk of relapse and accumulation of disability compared to interferon \\u03b2-1a treatment in a phase II trial. Alemtuzumab (CD52-specific humanized monoclonal antibody) was found to be an effective therapy for treatment-naive patients with relapsing-remitting multiple sclerosis. Alemtuzumab effectively reduced relapse rates and improved clinical scores in patients with active relapsing-remitting multiple sclerosis not controlled by interferon therapy.Alemtuzumab is an anti-CD52 monoclonal antibody with remarkable efficacy in relapsing multiple sclerosis (MS). The anti-CD52 monoclonal antibody alemtuzumab reduces disease activity in previously untreated patients with relapsing-remitting multiple sclerosis. For patients with first-line treatment-refractory relapsing-remitting multiple sclerosis, alemtuzumab could be used to reduce relapse rates and sustained accumulation of disability. Campath-1H causes the immune response to change from the Th1 phenotype, suppressing multiple sclerosis disease activity, but permitting the generation of antibody-mediated thyroid autoimmunity.Campath-1H is a humanized monoclonal antibody directed at CD52 expressed on lymphocytes and other cells of the immune system. It has been tested extensively in lymphoid malignancies, autoimmune diseases including rheumatoid arthritis, multiple sclerosis, and organ transplantation.Campath 1-H (Alemtuzumab) is a humanised monoclonal antibody which targets the CD52 antigen, a low molecular weight glycoprotein present on the surface of most lymphocyte lineages, causing complement mediated lysis and rapid and prolonged T lymphocyte depletion. Following encouraging initial data from other centres we report our open label experience of using Campath 1-H as a treatment in aggressive relapsing multiple sclerosis in a consecutive series of 39 highly selected patients treated across three regional centres and followed for a mean of 1.89 years.Alemtuzumab, a humanized monoclonal antibody that targets CD52 on lymphocytes and monocytes, may be an effective treatment for early multiple sclerosis. In patients with early, relapsing-remitting multiple sclerosis, alemtuzumab was more effective than interferon beta-1a but was associated with autoimmunity, most seriously manifesting as immune thrombocytopenic purpura.Alemtuzumab, a monoclonal antibody targeting CD52, is a highly promising agent currently being studied in two phase III clinical trials. In this review, data from the recently published phase II clinical trial in the treatment of early relapsing remitting MS is summarized and analyzed in light of the development of alemtuzumab for MS and its potential role in treating this disease is discussed.Alemtuzumab is a humanized monoclonal antibody against CD52, an antigen found on the surface of normal and malignant lymphocytes. It is approved for the treatment of B-cell chronic lymphocytic leukaemia and is undergoing Phase III clinical trials for the treatment of multiple sclerosis. To assess the effect of systemic lymphocyte depletion on disease activity, seven patients with MS received a 10-day intravenous course of the humanised monoclonal antibody CAMPATH-1H (anti-CDw52).The effect of CAMPATH-1H on disease activity provides direct, but preliminary, evidence that disease activity in MS depends on the availability of circulating lymphocytes and can be prevented by lymphocyte depletion.The elective treatment of patients with multiple sclerosis, using a humanized anti-leukocyte (CD52) monoclonal antibody (Campath-1H), has illuminated mechanisms that underlie the clinical course of the disease. Alemtuzumab is currently under review to treat relapsing multiple sclerosis (MS) in the United States, based on positive Phase II and Phase III trials in both treatment-na\\u00efve and treated relapsing MS patients. Alemtuzumab has recently been approved by the European authorities for active relapsing MS, in essence as a first-line agent.Alemtuzumab is a humanized monoclonal antibody directed against CD52. \"}, {\"question\": \"Which brain structures have been investigated as potential targets for deep brain stimulation of patients suffering from major depression?\", \"ideal_answer\": [\"Subgenual cingulate gyrus, the anterior limb of the capsula interna, nucleus accumbens, medial forebrain bundle, habenula, and caudate nucleus have been investigated as potential targeted for the deep brain stimulation of patients suffering from major depression.\"], \"exact_answer\": [[\"subgenual cingulate gyrus\"], [\"anterior limb of the capsula interna\"], [\"nucleus accumbens\"], [\"medial forebrain bundle\"], [\"habenula\"], [\"ventral caudate nucleus\"]], \"context\": \"Deep brain stimulation (DBS) is under scientific evaluation as a new treatment option for these treatment-resistant patients. First clinical studies with small samples have been stimulated at the subgenual cingulate gyrus (Cg25/24), the anterior limb of the capsula interna (ALIC), and the nucleus accumbens (NAcc). New hypothesis-guided targets (e.g., medial forebrain bundle, habenula) are about to be assessed in clinical trials. Deep brain stimulation (DBS) to the nucleus accumbens (NAcc-DBS) has antidepressant effects in patients suffering from treatment-resistant depression (TRD). The ventral striatum, including the head of the caudate nucleus and the nucleus accumbens, is a putative target for deep brain stimulation (DBS) in the treatment of obsessive-compulsive disorder (OCD) and major depression (MD). High frequency stimulation was successfully applied in several small samples of patients with treatment-resistant depression when the stimulation focused on different areas, e.g., nucleus accumbens, the lateral habenula or cortical areas.We recorded electrophysiological activity directly from the nucleus accumbens of five patients undergoing deep brain stimulation for treatment of refractory major depression. Functional inhibition of the lateral habenula via deep brain stimulation (DBS) has antidepressive properties.Because a prominent clinical feature of depression is anhedonia--the inability to experience pleasure from previously pleasurable activities--and because there is clear evidence of dysfunctions of the reward system in depression, DBS to the nucleus accumbens might offer a new possibility to target depressive symptomatology in otherwise treatment-resistant depression.These preliminary findings suggest that DBS to the nucleus accumbens might be a hypothesis-guided approach for refractory major depression.Deep brain stimulation of the ventral caudate nucleus markedly improved symptoms of depression and anxiety until their remission, which was achieved at 6 months after the start of stimulation (HDRS < or = 7 and HARS < or = 10).No neuropsychological deterioration was observed, indicating that DBS of the ventral caudate nucleus could be a promising strategy in the treatment of refractory cases of both OCD and major depression.Comparisons of clinical outcomes and anatomical data on electrode positioning showed that caudate nucleus stimulation preferentially alleviated OCD manifestations, whereas nucleus accumbens stimulation improved depressive symptoms. \"}, {\"question\": \"Which deiodinase polymorphisms are implicated in arterial hypertension?\", \"ideal_answer\": [\"Two deiodinase polymorphisms are implicated in arterial hypertension: Ala92 type 2 deiodinase allele and rs7140952 polymorphism of DIO2\", \"At least two deiodinease polymorfisms are  implicated in arterial hypertension:\\nDIO 2 Thr92Ala\\nrs7140952\"], \"exact_answer\": [[\"rs7140952 polymorphism of DIO2\", \"rs7140952\"], [\"Ala92 type 2 deiodinase allele\", \"DIO 2 Thr92Ala\"]], \"context\": \"However, rs7140952 polymorphism is associated with components of metabolic syndrome including blood pressure and central obesity.Among euthyroid adults, the common Ala92 allele of the type 2 iodothyronine deiodinase increases risk for the development of hypertension. Ala92 type 2 deiodinase allele increases risk for the development of hypertension.This study investigates the clinical and biochemical response to L-T4 replacement therapy in hypothyroid patients in correlation with genetic variation in Deiodinase type || (DIO2) gene. \"}, {\"question\": \"Which transcription factors are involved in E-cadherin repression during EMT?\", \"ideal_answer\": [\"Downregulation of E-cadherin is a crucial event for epithelial to mesenchymal transition (EMT) in embryonic development and cancer progression. Overexpression of Snail1 (Snail), Snail2 (Slug), Zeb1, Twist, SIP1 and DeltaEF1 have been found to mediate E-cadherin repression, induce the mesenchymal markers vimentin and fibronectin, and finally promote the migratory and invasive capabilities in cancer cells.\"], \"exact_answer\": [[\"Snail1 (Snail)\"], [\"Snail2 (Slug)\"], [\"Zeb1\"], [\"Twist\"], [\"SIP1\"], [\"DeltaEF1\"]], \"context\": \"ifferential role of Snail1 and Snail2 zinc fingers in E-cadherin repression and epithelial to mesenchymal transitionnail1 (Snail) and Snail2 (Slug) are transcription factors that share a similar DNA binding structure of four and five C2H2 zinc finger motifs (ZF), respectively. Both factors bind specifically to a subset of E-box motifs (E2-box: CAGGTG/CACCTG) in target promoters like the E-cadherin promoter and are key mediators of epithelial-to-mesenchymal transition (EMT).hrombin induces slug-mediated E-cadherin transcriptional repression and the parallel up-regulation of N-cadherin by a transcription-independent mechanism in RPE cellsWe demonstrate, for the first time, that thrombin induces E-cadherin repression by stimulating snail-2 (SLUG) transcription factor expression,Mutation of the putative GSK-3\\u03b2 phosphorylation sites (S92/96A or S100/104A) enhanced the Slug/Snail2-mediated EMT properties of E-cadherin repression and vimentin induction, compared with wild-type Slug/Snail2.xpression of E-cadherin, a hallmark of epithelial-mesenchymal transition (EMT), is often lost due to promoter DNA methylation in basal-like breast cancer (BLBC),Here, we identified that Snail interacted with Suv39H1 (suppressor of variegation 3-9 homolog 1), a major methyltransferase responsible for H3K9me3 that intimately links to DNA methylation. We showed that Snail interacted with Suv39H1 and recruited it to the E-cadherin promoter for transcriptional repression.Knockdown of Suv39H1 restored E-cadherin expression by blocking H3K9me3 and DNA methylation and resulted in the inhibition of cell migration, invasion and metastasis of BLBC.9a interacts with Snail and is critical for Snail-mediated E-cadherin repression in human breast cancerHere, we have determined that methylation of histone H3 on lysine 9 (H3K9me2) is critical for promoter DNA methylation of E-cadherin in three TGF-\\u03b2-induced EMT model cell lines, as well as in CLBC cell lines.Further, Snail interacted with G9a, a major euchromatin methyltransferase responsible for H3K9me2, and recruited G9a and DNA methyltransferases to the E-cadherin promoter for DNA methylation.Knockdown of G9a restored E-cadherin expression by suppressing H3K9me2 and blocking DNA methylation. This resulted in inhibition of cell migration and invasion in vitro and suppression of tumor growth and lung colonization in in vivo models of CLBC metastasis.ZEB1, which targets E-cadherin repression, is a transcriptional regulator that has been implicated in EMT, and is associated with uterine and colorectal cancers.The introduction or knockdown of ZEB1 expression in bladder carcinoma cell lines showed enhanced or reduced migration and invasive potential, respectively.However, in vitro assays showed enhanced or reduced migration and invasion after the introduction or reduction of ZEB1, respectively, in transfected bladder cell lines.Finally, we showed aberrant IL-6 production and STAT3 activation in MCF-7 cells that constitutively express Twist, a metastatic regulator and direct transcriptional repressor of E-cadherin. To our knowledge, this is the first study that shows IL-6 as an inducer of an EMT phenotype in breast cancer cells and implicates its potential to promote breast cancer metastasis.unctional loss of the cell-cell adhesion molecule E-cadherin is an essential event for epithelial-mesenchymal transition (EMT), a process that allows cell migration during embryonic development and tumour invasion. In most carcinomas, transcriptional repression has emerged as the main mechanism responsible for E-cadherin downregulation.Here, we report the identification of class I bHLH factor E2-2 (TCF4/ITF2) as a new EMT regulator.E-cadherin repression mediated by E2-2 is indirect and independent of proximal E-boxes of the promoter.Knockdown studies indicate that E2-2 expression is dispensable for maintenance of the EMT driven by Snail1 and E47.Slug mRNA was shown to be over-expressed in adenocarcinoma and inversely correlated to E-cadherin expression.Overexpression of Slug in OE33 mediated E-cadherin repression and induced the mesenchymal markers vimentin and fibronectin.SIP1 and deltaEF1 each dramatically down-regulated the transcription of E-cadherin in NMuMG cells through direct binding to the E-cadherin promoter.Silencing of the expression of both SIP1 and deltaEF1, but not either alone, completely abolished TGF-beta-induced E-cadherin repression.eltaEF1 is a transcriptional repressor of E-cadherin and regulates epithelial plasticity in breast cancer cellsownregulation of E-cadherin is a crucial event for epithelial to mesenchymal transition (EMT) in embryonic development and cancer progression.Using the EpFosER mammary tumour model we show that during EMT, upregulation of the transcriptional regulator deltaEF1 coincided with transcriptional repression of E-cadherin.Ectopic expression of deltaEF1 in epithelial cells was sufficient to downregulate E-cadherin and to induce EMT.Analysis of E-cadherin promoter activity and chromatin immunoprecipitation identified deltaEF1 as direct transcriptional repressor of E-cadherin.In human cancer cells, transcript levels of deltaEF1 correlated directly with the extent of E-cadherin repression and loss of the epithelial phenotype.RNA interference-mediated downregulation of deltaEF1 in cancer cells was sufficient to derepress E-cadherin expression and restore cell to cell adhesion, suggesting that deltaEF1 is a key player in late stage carcinogenesis.TGF-beta-induced the expression of Ets1, which in turn activated deltaEF1 promoter activity. Moreover, up-regulation of SIP1 and deltaEF1 expression by TGF-beta was suppressed by knockdown of Ets1 expression. ETV5 modulated Zeb1 expression and E-Cadherin repression leading to a complete reorganization of cell-cell and cell-substrate contacts. ETV5-promoted EMT resulted in the acquisition of migratory and invasive capabilities in endometrial cell lines.\"}, {\"question\": \"List human diseases involving genomic imprinting.\", \"ideal_answer\": [\"Prader Willi Syndrome\\nAngelman syndrome\\nBeckwith-Wiedemann syndrome\\nHydatidiform mole\\nCancer\\nSilver-Russell syndrome\\nDiabetes\"], \"exact_answer\": [[\"Prader Willi Syndrome\"], [\"Angelman syndrome\"], [\"Beckwith-Wiedemann syndrome\"], [\"Hydatidiform mole\"], [\"Cancer\"], [\"Silver-Russell syndrome\"], [\"Diabetes\"]], \"context\": \" Here we describe the clinical case of a female patient with Prader Willi Syndrome (PWS), a genomic imprinting disorderPerturbation of the allelic DNA methylation at ICRs is causally involved in several human diseases, including the Beckwith-Wiedemann and Silver-Russell syndromes, associated with aberrant foetal growth.Perturbed imprinted gene expression is also implicated in the neuro-developmental disorders Prader-Willi syndrome and Angelman syndrome.This epigenetic \\\"life cycle\\\" of imprinting (germline erasure, germline establishment, and somatic maintenance) can be disrupted in several human diseases, including Beckwith-Wiedemann syndrome (BWS), Prader-Willi syndrome (PWS), Angelman syndrome and Hydatidiform mole.In the neurodevelopmental Rett syndrome, the way the ICR mediates imprinted expression is perturbed.eregulation of imprinted genes has been observed in a number of human diseases as gestation trophoblastic disease, Prader-Willi, Angelmann and Beckwith-Wiedemann syndromes and plays significant role in the carcinogenesis.Angelman's syndrome families, which are known to be imprinted. Deregulation of imprinted genes has been observed in a number of human diseases such as Beckwith-Wiedemann syndrome, Prader-Willi/Angelman syndromes and cancer.Genomic imprinting is the phenomenon whereby some genes preferentially produce mRNA transcripts from the gene copy derived from the parent of a specific sex. It has been implicated in a number of human diseases (most of them of endocrine interest), such as Prader-Willi/Angelman syndromes, Silver-Russell syndrome, Beckwith-Wiedemann syndrome, transient neonatal diabetes, the focal form of nesidioblastosis, and pseudohypoparathyroidism.imprinting disorders like Beckwith-Wiedemann and Prader-Willi/Angelman syndromesIt is particularly interesting from the clinical point of view that a number of human diseases, such as the Beckwith-Wiedemann and Prader-Willi/Angelman syndromes, appear to involve unbalanced parental contributions of imprinted loci. Examples include the Prader-Willi, Angelman, and Beckwith-Wiedemann syndromes [Nicholls (1994): Am J Hum Genet 54:733-740], malignancy [Sapienza (1990): Biochim Biophys Acta 1072:51-61; Feinberg (1993): Nat Genet 4:110-113], and insulin-dependent diabetes mellitus (IDDM) [Julier et al. (1994) Nature 354:155-159; Bennett et al. (1995) Nat Genet 9:284-292]. \"}, {\"question\": \"What memory problems are reported in the \\\" Gulf war syndrome\\\"?\", \"ideal_answer\": [\"Loss of memory and dysmnesia are memory problems reported in the \\\" Gulf war syndrome\\\". Patients suffering from this syndrome often have other\\nnonspecific symptoms such as fatigue, skin rash, headache, muscle and joint pain and sexual dysfunction.\"], \"exact_answer\": [[\"loss of memory\"], [\"dysmnesia\"]], \"context\": \"The strongest associations were for mood swings (OR 20.9, 95%CI 16.2-27.0), memory loss/lack of concentration (OR 19.6, 95% CI 15.5-24.8), night sweats (OR 9.9, 95% CI 6.5-15.2), general fatigue (OR 9.6, 95% CI 8.3-11.1) and sexual dysfunction (OR 4.6, 95%CI 3.2-6.6).The symptoms include incapacitating fatigue, musculoskeletel and joint pains, headaches, neuropsychiatric disorders, affect changes, confusion, visual problems, changes of gait, loss of memory, lymphadenopathies, respiratory impairment, impotence, and urinary tract morphological and functional alterations. In early 1992, U.S. troops returning from the Gulf War began reporting a variety of nonspecific symptoms such as fatigue, skin rash, headache, muscle and joint pain, and loss of memory. These reports marked the beginning of what was to be identified as the Gulf War Syndrome (GWS). We report on a 29-year-old man who suffered from dysmnesia, disturbance of orientation, cognitive impairment, and double vision. His history revealed several front-line operations in 1990 and 1991 during the Gulf War. Neuropsychological tests disclosed severe cognitive impairment especially concerning memory.A cluster of common health problems included: skin rash, cough, depression, unintentional weight loss, insomnia, and memory problems.Syndromes 1 (\\\"impaired cognition,\\\" characterized by problems with attention, memory, and reasoning, as well as insomnia, depression, daytime sleepiness, and headaches), 2 (\\\"confusion-ataxia,\\\" characterized by problems with thinking, disorientation, balance disturbances, vertigo, and impotence), and 3 (\\\"arthro-myo-neuropathy,\\\" characterized by joint and muscle pains, muscle fatigue, difficulty lifting, and extremity paresthesias) represented strongly clustered symptoms; whereas, syndromes 4 (\\\"phobia-apraxia\\\"), 5 (\\\"fever-adenopathy\\\"), and 6 (\\\"weakness-incontinence\\\") involved weaker clustering and mostly overlapped syndromes 2 and 3. \"}, {\"question\": \"Which neuroendocrine tumors are associated with specific tumor syndromes?\", \"ideal_answer\": [\"Neuroendocrine tumors are a heterogeneous group of benign and malignant neoplasias, detectable in the context of hereditary tumor syndromes in up to 30% of cases. Neuroendocrine tumors include medullary thyroid carcinoma, gastroenteropancreatic tumors, pheochromocytoma, and paraganglioma.\"], \"exact_answer\": [[\"medullary thyroid carcinoma\"], [\"gastroenteropancreatic tumors\"], [\"pheochromocytoma\"], [\"paraganglioma\"]], \"context\": \"Pheochromocytomas are neuroendocrine tumors of the adrenal medulla which can occur either sporadically or in the context of hereditary tumor syndromesPheochromocytomas (PCCs) and paragangliomas (PGLs) are rare neuroendocrine tumors of the adrenal glands and the sympathetic and parasympathetic paraganglia. They can occur sporadically or as a part of different hereditary tumor syndromesPancreatic neuroendocrine tumors (PNETs) are a characteristic feature of the tumor syndromes multiple endocrine neoplasia type 1 (MEN-1) and von Hippel-Lindau disease (VHL)This review focuses on hereditary syndromes with neuroendocrine tumors, including multiple endocrine neoplasia types 1 and 2, Von Hippel-Lindau disease, neurofibromatosis type 1, Carney complex, pheochromocytoma-paraganglioma syndrome, and familial nonmedullary thyroid carcinoma. In addition, several individual neuroendocrine tumors are described, such as medullary thyroid carcinoma, gastroenteropancreatic tumors, pheochromocytoma, and paraganglioma, emphasizing specific histopathologic characteristicsNeuroendocrine tumors (NETs) are a heterogeneous group of benign and malignant neoplasias, detectable in the context of hereditary tumor syndromes in up to 30% of casesRET gene analysis can identify individuals with a very high risk to develop familial medullary cancer (MEN2)In addition, several individual neuroendocrine tumors are described, such as medullary thyroid carcinoma, gastroenteropancreatic tumors, pheochromocytoma, and paraganglioma, emphasizing specific histopathologic characteristics\"}, {\"question\": \"List medication interfering with purine metabolism that are used for treatment of T-cell prolymphocytic leukemia?\", \"ideal_answer\": [\"Deoxycoformycin and pentostatin are purine analogs that interfere with purine metabolism and are used for treatment of T-cell prolymphocytic leukemia patients.\"], \"exact_answer\": [[\"deoxycoformycin\"], [\"pentostatin\"], [\"nelarabine\"]], \"context\": \"Treatment with purine analogues and alemtuzumab has resulted in significantly higher response rates and improved survival.  Treatment with purine analogs and the monoclonal antibody alemtuzumab has resulted in significantly higher response rates and increased survival.Nelarabine is an effective regimen against indolent leukemias, and combining it with fludarabine was most promising. Determination of tumor cell ara-GTP levels may provide a predictive test for response to nelarabine.T-cell prolymphocytic leukemia has an aggressive course with short median survival and poor response to chemotherapy. With the use of the purine analogue pentostatin more than half of patients will have a major response and a minority will have a complete remission, usually lasting months. T-cell prolymphocytic leukaemia (T-PLL) is a rare post-thymic T-cell malignancy with an aggressive clinical course. It has generally been resistant to alkylating chemotherapy, but some effect has been observed with the purine analog 2-deoxycoformicin with documented partial or complete response rates in up to 45% of patients.Fifteen patients with T-PLL, most of whom had received the purine analog deoxycoformycin (DCF), were treated with CAMPATH-1H.Pentostatin (2'-deoxycoformycin; DCF) is a purine analogue that has shown activity in treatment of chronic lymphoid malignancies.Trials using DCF or other purine analogues alone or in combination with standard chemotherapeutic agents in front-line or salvage therapy are warranted to improve the prognosis of patients with prolymphocytic leukemia.\"}, {\"question\": \"List Genes associated with adolescent idiopathic scoliosis\", \"ideal_answer\": [\"Exome Sequencing Identifies a Rare HSPG2 Variant Associated with Familial Idiopathic Scoliosis.Overall, these findings demonstrate a novel role for kif6 in spinal development and identify a new candidate gene for human idiopathic scoliosis.\", \"No genetic associations have yet been found to adolescent idiopathic scoliosis.\"], \"exact_answer\": [[\"no associations found yet\"]], \"context\": \"Exome Sequencing Identifies a Rare HSPG2 Variant Associated with Familial Idiopathic Scoliosis.Overall, these findings demonstrate a novel role for kif6 in spinal development and identify a new candidate gene for human idiopathic scoliosis.HL1 is of interest, as it encodes an axon guidance protein related to Robo3. Mutations in the Robo3 protein cause horizontal gaze palsy with progressive scoliosis (HGPPS), a rare disease marked by severe scoliosis. Other top associations in our GWAS were with SNPs in the DSCAM gene encoding an axon guidance protein in the same structural class with Chl1 and Robo3.\"}, {\"question\": \"List the off-label use of SSRIs\", \"ideal_answer\": [\"depression during childhood and adolescence\\nPremature ejaculation (PE)\\nerectile dysfunction\\nInsomnia\\npostprostatectomy established stress urinary incontinence.\\nmood and anxiety disorders during pregnancy and breast feeding\\nsymptoms of vasomotor dysregulation (hot flashes) associated with the menopausal transition and sex hormone deprivation\\n..off-label uses include the treatment of bulimia, benzodiazepine/alcohol dependence, fibromyalgia, central nervous system degenerative diseases (behavioral disorders in dementia and other organic disorders), schizophrenia, chronic pain disease and diabetic neuropathy, sexual dysfunction.\"], \"exact_answer\": [[\"depression during childhood and adolescence\"], [\"Premature ejaculation\", \"PE\"], [\"erectile dysfunction\"], [\"Insomnia\"], [\"postprostatectomy established stress urinary incontinence\", \"SUI\"], [\"mood and anxiety disorders during pregnancy and breast feeding\"], [\"hot flashes\", \"postmenopausal symptoms\"], [\"bulimia\"], [\"benzodiazepine/alcohol dependence\"], [\"fibromyalgia\"], [\"behavioral disorders in dementia\"], [\"schizophrenia\"], [\"chronic pain disease\"], [\"diabetic neuropathy\"]], \"context\": \"The present analysis evaluates the prevalence and medication use in inpatients with depression during childhood and adolescenceFluoxetine and mirtazapine were the most frequently prescribed substances. Sertraline, escitalopram, and citalopram were also prescribed. CONCLUSION: A reserved medical treatment can be observed in child and adolescence psychiatry. Off-label use seems to be nearly unavoidable due to the lack of newly authorized medicine. Moreover, the numerous prescriptions for fluoxetine, the only SSRI currently approved for this age group in Germany, lead to the question of possible unauthorized alternatives.Premature ejaculation (PE) is a common sexual dysfunction affecting 20% to 30% of men worldwideTreatment of PE typically involves pharmacotherapy, particularly when lifelong. Although there are numerous reports on the off-label use of selective serotonin reuptake inhibitors (SSRIs) and other compounds, only 2 treatments have been evaluated in randomized controlled phase 3 clinical trials: PSD502 and dapoxetine (SSRI)Drug therapy of fibromyalgia syndrome. Systematic review, meta-analysis and guideline].Amitriptyline and-in case of comorbid depressive disorder or generalized anxiety disorder-duloxetine are recommended. Off-label use of duloxetine and pregabalin can be considered in case of no comorbid mental disorder.Trazodone is an antidepressant belonging to the class of serotonin receptor antagonists and reuptake inhibitors. It is approved by the FDA for the treatment of depression. Insomnia is the most frequent reason for prescription of trazodone. It has also been proven useful in the treatment of anxiety disorders. Other off-label uses include the treatment of bulimia, benzodiazepine/alcohol dependence, fibromyalgia, central nervous system degenerative diseases (behavioral disorders in dementia and other organic disorders), schizophrenia, chronic pain disease and diabetic neuropathy, sexual dysfunction.symptoms of vasomotor dysregulation (hot flashes) associated with the menopausal transition and sex hormone deprivation. Implication of changes in central neurotransmission in the pathogenesis of hot flashes has prompted the off-label use of serotonergic and \\u03b3-aminobutyric acid-ergic drugs as a therapeutic alternative,Sertraline was the most prescribed antidepressant for the treatment of major depressive disorder, followed by fluvoxamine and tianeptine. Fluvoxamine was the most prescribed antidepressant for the treatment of anxiety disorders and mixed disorders of emotions and conduct. Off-label prescribing of antidepressants was found in 85.6% of young patients.Our results suggest that duloxetine is a possible alternative treatment of postprostatectomy established stress urinary incontinence.Pharmacological treatment of fibromyalgia.different classes of drugs with different mechanisms of action are used off-label, including tricyclic antidepressants (TCAs), selective serotonin reuptake inhibitors (SSRIs)Gynaecological cancer patients generally suffer from an earlier and more severe menopausal syndrome than the general female population. Hormone replacement therapy is often contraindicated and there are non-hormonal treatments that are proven to be more effective than placebo in randomized controlled trials, e.g., some antidepressants, gabapentine and clonidine. The main limits to the use of these drugs in controlling hot flashes are the off-label use for this purpose,ymptomatic therapy for patients with erectile dysfunction.a oral daily off label use therapy with selective serotonin re-uptake inhibitors (paroxetine, fluoxetine, sertraline) can be offered.Without the Food and Drug Administration approval, dapoxetine, as well as other SSRIs in PE, is an off-label drug for PE.the treatment of PE consists of primarily off-label use of oral selective serotonin reuptake inhibitors (SSRIs) via either on-demand or daily delivery. The results of this off-label use show that Duloxetine is effective in men with SUI after prostate surgery even if standard pelvic floor exercises have failed.Preliminary results on the off-label use of duloxetine for the treatment of stress incontinence after radical prostatectomy or cystectomy.the number of children and adolescents whose visits involved prescription of antidepressants, particularly SSRIs, has increased markedly through 2002. Although fluoxetine remained the most commonly prescribed, other SSRIs were increasingly prescribed through 2002. These trends raise concerns regarding the widespread off-label use of antidepressants lacking reliable evidence of safety and efficacy for use in children and adolescents.Given the presumed efficacy of these new compounds and the off-label use of the current SSRIs, one might conclude that psychotherapy\\\\behavior therapy for rapid ejaculation is an obsolete and antiquated interventionThe use of selective serotonin reuptake inhibitors during pregnancy and breast-feeding:The use of selective serotonin reuptake inhibitors during pregnancy or lactation is, to date, not promoted because of lack of safety documentation. However, the off-label use of these drugs has been common for several years. In the treatment of mood and anxiety disorders during pregnancy, the serotonin reuptake inhibitors are often preferred over tricyclic antidepressants because of their relatively few adverse effects and safety in overdose. The last few years have seen a remarkable rise in the off-label use of trazodone for inducing sleep in nondepressed patients, to a degree that it is prescribed for this purpose as commonly as the leading hypnoticSince the mid-1980s there has been a rapid rise in the off-label use of antidepressants, particularly trazodone, for treating insomnia.\"}, {\"question\": \"List receptors of the drug Cilengitide\", \"ideal_answer\": [\"Cilengitide binds \\u03b1v\\u03b23 and \\u03b1v\\u03b25 integrins. It inhibits attachment and invasion of malignant cells. Thus, cilengitide is being tested for treatment of cancer patients.\"], \"exact_answer\": [[\"\\u03b1v\\u03b23 integrin\"], [\"\\u03b1\\u03bd\\u03b25 integrin\"], [\"\\u03b1v\\u03b25\", \"\\u03b1v\\u03b23\"]], \"context\": \"Cilengitide inhibits attachment and invasion of malignant pleural mesothelioma cells through antagonism of integrins \\u03b1v\\u03b23 and \\u03b1v\\u03b25.Also, this dimer bound 3650-fold stronger and inhibited tumor cell migration and proliferation compared with cilengitide, an integrin-targeting peptidomimetic that performed poorly in recent clinical trials, suggesting promise for further therapeutic development.As there is evidence for expression of the integrins \\u03b1v\\u03b23 and \\u03b1v\\u03b25 in MPM, there is a rationale for investigating the effects on MPM of cilengitide, a synthetic peptide inhibitor of integrin \\u03b1v heterodimer with high specificity for \\u03b1v\\u03b23 and \\u03b1v\\u03b25. Gene knockdown experiments indicated that these effects of cilengitide were, at least partly, due to antagonism of \\u03b1v\\u03b23 and \\u03b1v\\u03b25.Abstract The RGD cyclic pentapetide, cilengitide, is a selective inhibitor of \\u03b1v\\u03b23 and \\u03b1v\\u03b25 integrins and was developed for antiangiogenic therapy. Cilengitide is an RGD-peptide of sequence cyclo[RGDfNMeV] that was was developed as a highly active and selective ligand for the \\u03b1v\\u03b23 and \\u03b1v\\u03b25 integrin receptors.RESULTS: \\u03b1v\\u03b25 was the predominantly expressed integrin heterodimer in meningiomas, whereas \\u03b1v\\u03b23 was mainly detected in tumor blood vessels. Application of up to 100 \\u03bcg/mL cilengitide resulted in only mildly reduced proliferation/survival of meningioma cell lines. UW479 cells expressed only \\u03b1v\\u03b25 integrin and were not sensitive to cilengitide, suggesting that cilengitide's action largely depends on \\u03b1v\\u03b23 inhibition.Cilengitide's action on glioma and neuroblastoma cells appears to be dependent on \\u03b1v\\u03b23 expression and sensitivity to anoikis. METHODS: For this purpose, nude rats bearing bone metastases were treated with cilengitide, a small molecule inhibitor of \\u03b1v\\u03b23 and \\u03b1v\\u03b25 integrins, from day 30 to 55 after tumor cell inoculation of MDA-MB-231 breast cancer cells (25 mg/kg, 5 days per week; n = 8 rats) and compared to control rats (n = 8). The \\u03b1V\\u03b23/\\u03b1V\\u03b25 integrin inhibitor cilengitide augments tumor response to melphalan isolated limb perfusion in a sarcoma model.Cilengitide, a cyclized Arg-Gly-Glu(RGD)-containing pentapeptide that selectively blocks activation of the \\u03b1v\\u03b23 and \\u03b1v\\u03b25 integrins has shown encouraging activity in patients with glioblastoma as single agent, and in association with standard RT and temozolomide.  Aza-amino acid scanning was performed on the cyclic RGD-peptide Cilengitide, cyclo[R-G-D-f-N(Me)V] 1, and its parent peptide cyclo(R-G-D-f-V) 2, potent antagonists of the \\u03b1v\\u03b23, \\u03b1v\\u03b25, and \\u03b15\\u03b21 integrin receptors, which play important roles in human tumor metastasis and tumor-induced angiogenesis. Cilengitide (EMD121974; Merck KGaA, Darmstadt, Germany) is a new drug targeting \\u03b1v\\u03b23 and \\u03b1v\\u03b25 integrins thanks to a specific peptide called RGD sequence. Cilengitide, a cyclic RGD-mimetic peptide of \\u03b1v\\u03b23 and \\u03b1v\\u03b25 integrins is in advanced clinical development in glioblastoma.The purpose of this study was the assessment of the feasibility of dynamic positron emission tomography (PET) studies with fluorine-18 fluorodeoxyglucose ((18)F-FDG) to quantify effects of the cyclic Arg-Gly-Asp peptide cilengitide, which targets the \\u03b1\\u03bd\\u03b2 3 and \\u03b1\\u03bd\\u03b2 5 integrin receptors in rats with breast cancer bone metastases. We conducted a phase II study of cilengitide, a selective antagonist of \\u03b1(v)\\u03b2(3) and \\u03b1(v)\\u03b2(5) integrins, in non-metastatic castration resistant prostate cancer with rising PSA. AlphaVbeta3 and alphaVbeta5 integrins are overexpressed on both glioma cells and tumor vasculature. Cilengitide, the most advanced specific integrin inhibitor in oncology, has shown antitumor activity against glioma in early clinical trials. The aim of this study was to investigate the effect of inhibiting \\u03b1v\\u03b2(3)/\\u03b1(v) \\u03b2(5) integrins by cilengitide in experimentally induced breast cancer bone metastases using noninvasive imaging techniques. For this purpose, nude rats bearing established breast cancer bone metastases were treated with cilengitide, a small molecule inhibitor of \\u03b1v\\u03b2(3) and \\u03b1v\\u03b2(5) integrins (75 mg/kg, five days per week; n = 12 rats) and compared to vehicle-treated control rats (n = 12).  In conclusion, treatment of experimental breast cancer bone metastases with cilengitide resulted in pronounced antiresorptive and antitumor effects, suggesting that \\u03b1v\\u03b2(3)/\\u03b1v\\u03b2(5) inhibition may be a promising therapeutic approach for bone metastases.Integrin antagonists, including the alphavbeta3 and alphavbeta5 inhibitor cilengitide, have shown encouraging activity in Phase II clinical trials and cilengitide is currently being tested in a Phase III trial in patients with glioblastoma. Cilengitide (cyclic peptidic alphavbeta3 and alphavbeta5 antagonist) is currently in clinical trials for anti cancer therapy.PURPOSE: Cilengitide, an inhibitor of alphavbeta3 and alphavbeta5 integrin receptors, demonstrated minimal toxicity and durable activity across a wide range of doses administered to adults with recurrent glioblastoma multiforme (GBM) in a prior phase I study. In line with this concept, peptide ligands containing the Arginine-Glycine-Aspartate (RGD) triad, which display a strong affinity and selectivity to the alpha(V)beta(3) integrin, have been developed to target the tumor-associated cells expressing the alpha (V)beta (3) receptors. Among the validated ligands, the leader compound is the cyclic pentapeptide c[-RGDf(NMe)V-] (Cilengitide) developed by kessler et al. (J. Med. Chem., 1999, 42, 3033-3040). Assessment of the biological and pharmacological effects of the alpha nu beta3 and alpha nu beta5 integrin receptor antagonist, cilengitide (EMD 121974), in patients with advanced solid tumors.BACKGROUND: Cilengitide, an antiangiogenic agent that inhibits the binding of integrins alpha(nu)beta(3) and alpha(nu)beta(5) to the extracellular matrix, was studied at two dose levels in cancer patients to determine the optimal biological dose. Cilengitide (EMD121974; NSC 707544), is a potent selective alphavbeta3 and alphavbeta5 integrin antagonist. Phase I and pharmacokinetic study of continuous twice weekly intravenous administration of Cilengitide (EMD 121974), a novel inhibitor of the integrins alphavbeta3 and alphavbeta5 in patients with advanced solid tumours.A single-agent dose escalating phase I and pharmacokinetic study with Cilengitide, an inhibitor of the integrins alphavbeta3 and alphavbeta5, was performed to determine its safety and toxicity. This study was designed to determine whether, and how, the cyclic Arg-Gly-Asp peptide Cilengitide (EMD 121974), which targets the alpha(v)beta(3) integrin receptor expressed on neovasculature, could increase systemic RIT efficacy of therapy in a human breast cancer tumor model having mutant p53 and expressing bcl-2. The purpose of this study was the assessment of the feasibility of dynamic positron emission tomography (PET) studies with fluorine-18 fluorodeoxyglucose ((18)F-FDG) to quantify effects of the cyclic Arg-Gly-Asp peptide cilengitide, which targets the \\u03b1\\u03bd\\u03b2 3 and \\u03b1\\u03bd\\u03b2 5 integrin receptors in rats with breast cancer bone metastases\"}, {\"question\": \"Which genes are more frequently affected by somatic mutations in Chronic Lymphocytic Leukemia\", \"ideal_answer\": [\"TP53, ATM, NOTCH1, XPO1, MYD88, KLHL6, SF3B1, ZMYM3, MAPK1, FBXW7 and DDX3X\"], \"exact_answer\": [[\"TP53\"], [\"ATM\"], [\"NOTCH1\"], [\"XPO1\"], [\"MYD88\"], [\"KLHL6\"], [\"SF3B1\"], [\"ZMYM3\"], [\"MAPK1\"], [\"FBXW7\"], [\"DDX3X\"]], \"context\": \"Among these genes, SF3B1, encoding a subunit of the spliceosomal U2 small nuclear ribonucleoprotein (snRNP), is somatically mutated in 9.7% of affected individuals.Further analysis of these mutations in 363 patients with CLL identified four genes that are recurrently mutated: notch 1 (NOTCH1), exportin 1 (XPO1), myeloid differentiation primary response gene 88 (MYD88) and kelch-like 6 (KLHL6).Nine genes that are mutated at significant frequencies were identified, including four with established roles in chronic lymphocytic leukemia (TP53 in 15% of patients, ATM in 9%, MYD88 in 10%, and NOTCH1 in 4%) and five with unestablished roles (SF3B1, ZMYM3, MAPK1, FBXW7, and DDX3X). SF3B1, which functions at the catalytic core of the spliceosome, was the second most frequently mutated gene (with mutations occurring in 15% of patients).\"}, {\"question\": \"List chromosomes that have been linked to Arnold Chiari syndrome in the literature.\", \"ideal_answer\": [\"Chromosomes 1, 3, 5, 6, 8, 9, 12, 13, 15, 16, 18, 22, X and Y have been reported in association with Arnold Chiari syndrome in genetic linkage studies and individual case reports.\"], \"exact_answer\": [[\"1\"], [\"3\"], [\"5\"], [\"6\"], [\"8\"], [\"9\"], [\"12\"], [\"13\"], [\"15\"], [\"16\"], [\"18\"], [\"22\"], [\"X\"], [\"Y\"]], \"context\": \"Consistent with a genetic hypothesis for CMI, much of the PF morphology was found to be heritable and multiple genomic regions were strongly implicated from OSA, including regions on Chromosomes 1 (LOD = 3.07, p = 3 \\u00d7 10(-3) ) and 22 (LOD = 3.45, p = 6 \\u00d7 10(-5) ) containing several candidates warranting further investigation.Furthermore, we report the first case of documented Arnold-Chiari malformation type I and increased factor XIII activity associated with 6p trisomy. Of particular interest were two regions (Chr8, Max LOD\\u200a=\\u200a3.04; Chr12, Max LOD\\u200a=\\u200a2.09) identified within the subset of \\\"CTD-negative\\\" families, both of which harbor growth differentiation factors (GDF6, GDF3) implicated in the development of Klippel-Feil syndrome (KFS).Moreover, the performed DNA analysis showed interstitial duplication in chromosome 5 (5q35.1). CGH microarray showed a approximately 520.7 kb microdeletion on 16p13.3 involving CREBBP, ADCY9, and SRL genes. Pentasomy 49,XXXXY associated with a Chiari type 1 malformation and cervical syrinx.A 13-year-old with pentasomy 49,XXXXY and a Chiari type 1 malformation with an associated cervical syrinx is presented.Phenotypic definition of Chiari type I malformation coupled with high-density SNP genome screen shows significant evidence for linkage to regions on chromosomes 9 and 15.Two-point LOD scores on chromosome 15 reached 3.3 and multipoint scores in this region identified a 13 cM region with LOD scores over 1 (15q21.1-22.3). Multipoint LOD scores on chromosome 9 maximized at 3.05, identifying a 40 cM region with LOD scores over 1 (9q21.33-33.1) and a tighter region with multipoint LOD scores over 2 that was only 8.5 cM. In addition, this child had an Arnold-Chiari type I malformation that required surgical decompression. FISH studies using BAC clones spanning the 5q15 to 5q22 region revealed that these were all present in both homologues. The karyotype showed 46, XY, del(1)(q23q31.2). Karyotype analysis showed inversion of Y chromosome. The clinical features and morphological findings in 31 Japanese infants with trisomy 18 are presented. The majority were small-for-date infants. There was no sex predominance in our series, as opposed to male:female ratios of 1:3 reported in the literature. The average age at death was greater in females than in males. Cardiovascular anomalies were consistently present; ventricular septar defect and patent ductus arteriosus being the most common malformations. Various other internal malformations including the Arnold-Chiari malformation were observed.An unusual heterotopia of striated skeletal muscle and glial tissue occurred in the pontine meninges of a stillborn male showing feature of trisomy 13 and an Arnold-Chiari malformation.Type II Arnold-Chiari malformation with normal spine in trisomy 18.A variety of anomalies of the central nervous system are observed in trisomy 18. The present case describes an infant having a type II Arnold-Chiari malformation without spina bifida. One previous case of an Arnold-Chiari malformation was reported in trisomy 18 but that infant also had a lumbar meningomyelocoele. Recently, a child was reported who presented with a 3p13-14.1 deletion of four genes, including FOXP1, and a constellation of deficits that included speech delay. In this study, we report the case of a patient with a single deletion of FOXP1. This patient presented with speech and motor developmental delays, a Chiari I malformation, and epileptiform discharges.\"}, {\"question\": \"List symptoms of Meigs' Syndrome.\", \"ideal_answer\": [\"Meigs' syndrome is a benign ovarian tumor associated with ascites and pleural effusion.\"], \"exact_answer\": [[\"benign ovarian tumor\"], [\"ascites\"], [\"pleural effusion\"]], \"context\": \"Meigs' syndrome is a benign ovarian tumor associated with ascites and pleural effusion.Although postmenopausal women with ovarian tumor, ascites, pleural effusion, and elevation of CA-125 levels probably have malignant ovarian tumors, Meigs' syndrome must be considered in the differential diagnosis.BACKGROUND: The Demons-Meigs syndrome should usually be evoked in case of presence of a typical triad: abdominopelvic mass, ascites and hydrothorax. CONCLUSION: Meigs' syndrome should be considered at the differential diagnosis for a patient with pelvic mass, pleural effusion and ascites with normal cytology, increased CA125 levels.BACKGROUND: The Meigs' syndrome is a rare but well-known syndrome defined as the triad of benign solid ovarian tumor, ascites, and pleural effusion.Ovarian mass, pleural effusion, and ascites: revisiting Meigs syndrome.When benign ovarian fibroma is associated with ascites and/or pleural effusion it is termed Meigs syndrome. Ovarian mass, pleural effusion, and ascites: revisiting Meigs syndrome\"}, {\"question\": \"Which medical diagnostic tests are used to test kidney function?\", \"ideal_answer\": [\"Most common tests used in diagnosing normal kidney function include blood tests such as serum creatinine levels, glomerular filtration rate (GFR) and blood urea nitrogen (BUN) levels, also medical imaging tests like ultrasound and CT Scan. Additionally kidney biopsy is used in more direct but invasive approach. Lastly, and probably the most relevant tests to kidney function are urine tests along the lines of urinalysis, urine protein levels and microalbuminuria creatinine clearance.\"], \"exact_answer\": [[\"Blood Tests\"], [\"Imaging Tests\"], [\"Kidney Biopsy\"], [\"Urine Tests\"], [\"estimated GFR (eGFR)\"], [\"Decreased tryptophan (TRP)\"]], \"context\": \"kidney dysfunction and decreased glomerular filtration rate (GFR) are diagnosed by the evaluation of changes in the serum creatinine (SCr) and blood urea nitrogen (BUN) concentrations. Renal biomarkers, especially those present in urine, may be useful for the study of both acute and chronic nephropathies.The International Renal Interest Society (IRIS) canine AKI grading system and the implementation of urinary biomarkersDecreased tryptophan (TRP) and increased kynurenine (KYN) and kynurenic acid (KYNA) in blood have been reported in patients and experimental animals with renal diseases.Acute kidney injury was defined as a decrease in estimated glomerular filtration rate of 50% or more from the beginning of vancomycin therapy.  In critically ill children, the development of reversible AKI during vancomycin therapy is associated with administration of nephrotoxic drugs and an elevated BUN: Scr ratio.The physiologic determination of renal status is the measured glomerular filtration rate (mGFR). Serum creatinine, blood urea nitrogen, cystatin C, and estimated GFR (eGFR), based on serum creatinine have failed to replace mGFR.After kidney donation, renal function measured by blood urea nitrogen (BUN) and serum creatinine of all donors returned to normal within one week, and no serious complications were noticed. In living kidney donors GFR is not significantly correlated with age or sex.Increased proteinuria would lead to a larger risk for renal failure in the long termA reduction of proteinuria in patients with non-diabetic renal disease was observed during the 4-month treatment with pioglitazone which continued for 2 months after the cessation of the treatment. However, 4 months after the cessation of the treatment, a little increase was detected in the level of proteinuria.We identify pleiotropic associations among these loci with kidney function-related traits and risk of CKD.\"}, {\"question\": \"Which are the main causes of fetal echogenic bowel?\", \"ideal_answer\": [\"Fetal echogenic bowel is mainly associated to feto-maternal, intramniotic bleeding but in several cases it is linked to cystic fibrosis, cytomegalovirus (CMV), herpes simplex virus and other viral infections and fetal aneuploidy.\", \"Fetal echogenic bowel (FEB) is a soft marker found on second trimester sonography. (PMID: 22990134) A disorder was diagnosed in 32.2% of the fetuses, cystic fibrosis being the most commonly identified (7.6%). We also found digestive malformations (7.0%), chromosomal abnormalities (3.7%), and maternofetal infections (3.7%). (PMID: 20932506) Brightly echogenic bowel in the second trimester was found to be associated with a significant risk of fetal aneuploidy. (PMID: 1415421) echogenic bowel does not uniformly herald an abnormal outcome. Echogenic bowel coexistent with other abnormalities (such as growth deficiency or structural malformations) may be a comarker for aneuploidy. (PMID: 8142051) 112 cases (57%) had a known etiology, which included chromosomal abnormality (7%), infection (4%), cystic fibrosis (1.5%), bowel abnormality (3%), bleeding or stained amniotic fluid (11%), Doppler abnormality (14%), malformation (16%) and miscellaneous (0.5%) (PMID: 12835583)\"], \"exact_answer\": [[\"Itramniotic bleeding\"], [\"CMV infection\"], [\"Cystic Fibrosis (CF)\"], [\"Fetal aneuploidy\"]], \"context\": \"In group 2 and 3, two anomalies, anorectal malformation and cystic fibrosis, were detected postnatallySix had chromosomal/genetic abnormalities, two had congenital cytomegalovirus, none had cystic fibrosisPrimary bowel pathology is rare following the finding of FEBMaternal serology for cytomegalovirus (CMV) was performed in 49 (78%) casesThirty-three pregnancies (53%) were tested for cystic fibrosis (CF) and 1 baby was confirmed to have CF postnatallyThis study reiterates the increased prevalence of aneuploidy, CMV, CF and fetal growth restriction in pregnancies complicated by the midtrimester sonographic finding of FEBPrimary outcomes were IUGR, defined as birth weight less than the 10th percentile for gestational age and intrauterine fetal demise at 20 weeks or more of gestatioAnalyses were repeated after excluding cases of aneuploidy, cytomegalovirus (CMV) infection,The presence of echogenic bowel on ultrasonography is independently associated with an increased risk for both IUGR and intrauterine fetal demiseThis study highlights the importance of pregnancy ultrasound examinations and their efficiency in detecting cystic fibrosisA disorder was diagnosed in 32.2% of the fetuses, cystic fibrosis being the most commonly identified (7.6%). We also found digestive malformations (7.0%), chromosomal abnormalities (3.7%), and maternofetal infections (3.7%).A potential association with placental abnormalities and a low prevalence of viral infections was observedOur data suggests an inverse relationship between the maternal BMI and the detection of fetal EIF and/or EBFetal echogenic bowel at 17 weeks' gestational age as the early and only sign of a very long segment of Hirschsprung disease.fetal ultrasound findings associated with intrauterine cytomegalovirus (CMV) infectionthe combination of hydrops fetalis, cerebral hemorrhage, and hyperechoic bowel should raise the possibility of a CMV infectionstrongly associated with adverse pregnancy outcome due to utero-placental insufficiency, particularly in women with elevated maternal serum alpha-fetoprotein concentration due to severe feto-maternal bleedingCongenital cytomegalovirus infection presenting with echogenic bowel and oligohydramnios.Five cases of trisomy 21 and one case of trisomy 18 were detectedBrightly echogenic bowel in the second trimester was found to be associated with a significant risk of fetal aneuploidy.echogenic bowel does not uniformly herald an abnormal outcome. Echogenic bowel coexistent with other abnormalities (such as growth deficiency or structural malformations) may be a comarker for aneuploidy.Congenital cytomegalovirus infection with oligohydramnios and echogenic bowel at 14 weeks' gestationParental CF carrier testing and amniocentesis to identify aneuploidy or fetal CF status has a high positive ascertainment rate in fetuses with echogenic bowel grades 2 and 3.Swallowing of amniotic fluid after intraamniotic bleeding seems implicated in the etiology of second-trimester echogenic bowel in both euploid and aneuploid fetusesFetal echogenic bowel and a dilated loop of bowel associated with cystic fibrosis (CF) mutations delta F508 and 2183AA-->GFifteen cases (19%) were associated with maternal vaginal bleedingFive fetuses (6.3%) had evidence of bowel obstruction or perforation not associated with cystic fibrosis (CF)Chromosomal aberrations were found in 5 fetuses (6.3%). Intrauterine infection with cytomegalovirus, herpes simplex virus, varicella-zoster virus, or parvovirus B-19 was documented in 5 patients (6.3%).Fetal echogenic bowel has been reported as a normal variant in the second trimester, and has also been associated with an adverse fetal outcome, including cystic fibrosis (CF)Intra-amniotic bleeding can lead to echogenic bowel112 cases (57%) had a known etiology, which included chromosomal abnormality (7%), infection (4%), cystic fibrosis (1.5%), bowel abnormality (3%), bleeding or stained amniotic fluid (11%), Doppler abnormality (14%), malformation (16%) and miscellaneous (0.5%)\"}, {\"question\": \"What are the advantages of the top down mass spectrometric analysis of histones?\", \"ideal_answer\": [\"Top down mass spectrometry is a way to analyze intact proteins thus enabling: isoform characteriztion and analysis of post-translational modifications.\"], \"exact_answer\": [[\"isoform characteriztion\"], [\"determination of the type and location of post translational modifications\"]], \"context\": \"recent advances in MS technology, such as high-field Fourier transform ion cyclotron resonance (FTICR)-mass spectrometry, have permitted the study of intact proteins and their modifications. On-line and off-line protein separation instruments coupled to FTICR-MS allow the characterization of PTMs previously undetectable with bottom-up approaches.With just \\u223c24\\u2009\\u03bcg on-column of core histones (H4, H2B, H2A, and H3) purified from human fibroblasts, 41 H4 isoforms were identified, with the type and location of PTMs unambiguously mapped for 20 of these variants.Our results indicated that dH1 is extensively modified by phosphorylation, methylation, acetylation and ubiquitination, with most PTMs falling in the N-terminal domainOptimized for the intricacies of whole proteins, new software modules visualize proteome-scale data based on the LC retention time and intensity of intact masses and enable selective detection of PTMs to automatically screen for acetylation, phosphorylation, and methylation.our UHPLC-MS approach for histone profiling offers a sensitive and reproducible tool that will be of great value for exploring PTMs and variantsThis work demonstrates the power of top-down MS for a detailed structural confirmation of recombinant proteins, even without prior information on aa substitutions or modifications.Top and Middle Down MS were utilized to identify the most commonly occurring combinatorially modified forms.In this paper we investigate the application of spectral alignment to the problem of identifying protein forms in top-down mass spectra (i.e., identifying the modifications, mutations, insertions, and deletions).This work represents the most comprehensive analysis of histone H4 forms present in human cells reported to date.Metabolic labeling and top-down mass spectrometry reveal that newly synthesized H4 is progressively methylated at K20 during the G(2), M, and G(1) phases of the cell cycle in a process that is largely inescapable and irreversible.The \\\"top down\\\" mass spectrometry approach detected dramatic decreases in acetylation on H3 and H2B in gcn5Delta cells versus wild type.In sum, we have defined the \\\"basis set\\\" of histone forms present in yeast chromatin using a current mass spectrometric approach that both quickly profiles global changes and directly probes the connectivity of modifications on the same histone.Recent developments in top down mass spectrometry have enabled closely related histone variants and their modified forms to be identified and quantitated with unprecedented precision, facilitating efforts to better understand how histones contribute to the epigenetic regulation of gene transcription and other nuclear processes.This unequivocal identification of H2A forms illustrates the advantages of Top Down Mass Spectrometry and provides a global perspective of H2A regulation through the cell cycle.The modification of H3 in asynchronous HeLa cells was profiled using Top Down Mass Spectrometry.The basis set of protein forms expressed by human cells from the H2B gene family was determined by Top Down Mass Spectrometry.To help crack this code more efficiently, we demonstrate a new strategy for protein characterization wherein complete PTM descriptions are obtained by database retrieval instead of manual interpretation of information-rich data from high-resolution tandem mass spectrometry (MS/MS).Targeted analysis and discovery of posttranslational modifications in proteins from methanogenic archaea by top-down MS.This study illustrates a significant evolutionary step for the MS tools available for characterization of WT proteins from complex proteomes without proteolysis.\"}, {\"question\": \"List the main proteases used for sample digestion in proteomics.\", \"ideal_answer\": [\"Trypsin is the main protease used in proteomics followed by Asp-N, chymotrypsin, LysC, GluC and thermolysin.\"], \"exact_answer\": [[\"trypsin\"], [\"Asp-N\"], [\"chymotrypsin\"], [\"LysC\"], [\"Glu-C\"], [\"thermolysin\"]], \"context\": \"the consecutive use of endoproteinases LysC and trypsin Asp-N to trypsin  tryptic digestiontryptic digestenzymatic digestion is most commonly performed using trypsinchymotrypsin/trypsin digestionIn this method, chymotrypsin, single or in combination with trypsin, was used, which enabled to obtain proteolytic peptides from the hydrophobic regions and to identify new oil bodies' proteins.proteins were digested by thermolysintryptic digestionprotein digestion by trypsinTrypsin is an endoprotease commonly used for sample preparation in proteomics experimentstryptic digestion direct tryptic digestion trypsin digestionTryptic digestion is an important component of most proteomics experimentsWe evaluated nine trypsin-based digestion protocols, based on standard in-solution or on spin filter-aided digestionGetting intimate with trypsin, the leading protease in proteomics.sample preparation via trypsin digestion in-gel digested with trypsin, chymotrypsin, Asp-N, or trypsin plus Asp-N in triplicate. In this study, we examined the use of multiple proteases (trypsin, LysC, tandem LysC/trypsin) on both protein identification and quantification digested with LysC and trypsin, In-gel trypsin digestionMagnetic bead cellulose activated with divinyl sulfone was used for the immobilization of Staphylococcus aureus endoproteinase Glu-C (EC 3.4.21.19).protease digestion with AspN or trypsin\"}, {\"question\": \"Which proteins act as factors that promote transcription-coupled repair in bacteria?\", \"ideal_answer\": [\"Transcription coupled nucleotide excision repair (TC-NER or TCR) is a cellular process by which UV-induced damage and other road-blocks encountered in the transcribed strand are restored. Bacterial transcription-coupled repair is initiated when RNA polymerase stalled at a DNA lesion is removed by Mfd (Mutation frequency decline), an ATP-dependent DNA translocase. Mfd is the major transcription repair coupling factor in bacteria. Also, the transcription elongation factor NusA, in addition to its role in recruiting translesion synthesis (TLS) DNA polymerases to gaps encountered during transcription,  promotes an alternative class of TCR involved in the identification and removal of a class of lesion, such as the N(2)-f-dG lesion.\"], \"exact_answer\": [[\"Mfd\"], [\"NusA\"]], \"context\": \"Transcription-coupled repair (TCR) is a cellular process by which some forms of DNA damage are repaired more rapidly from transcribed strands of active genes than from nontranscribed strands or the overall genome. In humans, the TCR coupling factor, CSB, plays a critical role in restoring transcription following both UV-induced and oxidative DNA damage. It also contributes indirectly to the global repair of some forms of oxidative DNA damage. The Escherichia coli homolog, Mfd, is similarly required for TCR of UV-induced lesions.Transcription coupled nucleotide excision repair (TC-NER) is involved in correcting UV-induced damage and other road-blocks encountered in the transcribed strand. Mutation frequency decline (Mfd) is a transcription repair coupling factor, involved in repair of template strand during transcription.the transcription-repair coupling factor, Mfd, promotes direct restart of the fork following the collision by facilitating displacement of the RNAP.We report observations suggesting that the transcription elongation factor NusA promotes a previously unrecognized class of transcription-coupled repair (TCR) in addition to its previously proposed role in recruiting translesion synthesis (TLS) DNA polymerases to gaps encountered during transcription.NusA participates in an alternative class of TCR involved in the identification and removal of a class of lesion, such as the N(2)-f-dG lesionBacterial transcription-coupled repair is initiated when RNA polymerase stalled at a DNA lesion is removed by Mfd, an ATP-dependent DNA translocase.Transcription-coupled repair, the targeted repair of the transcribed strands of active genes, is defective in bacteria, yeast, and human cells carrying mutations in mfd, RAD26 and ERCC6, respectively.The effect of the bacterial transcription-repair coupling factor, Mfd, at such lesions is not known: it has been suggested that Mfd may promote mutagenesis by increasing the efficiency with which RNA polymerase bypasses non-bulky lesions, but it has also been reported that 8-oxoguanine, a major product of oxidative DNA damage that is efficiently bypassed by RNA polymerase, is subject to Mfd-dependent transcription-coupled repair in Escherichia coli.The transcription-repair coupling factor (TRCF, the product of the mfd gene) is a widely conserved bacterial protein that mediates transcription-coupled DNA repair.Recent structural studies of the bacterial transcription-repair coupling factor, Mfd, have revealed a modular architecture in which an ATP-dependent DNA-based motor is coupled to protein-protein interaction domains that can attach the motor to RNA polymerase and the DNA repair protein UvrA.The bacterial Mfd protein is a transcription-repair coupling factor that performs two key functions during transcription-coupled DNA repair.Mfd remains bound to the DNA in a long-lived complex that could act as a marker for sites of DNA damage, directing assembly of subsequent DNA repair factors.Transcription and DNA repair are coupled in E. coli by the Mfd protein, which dissociates transcription elongation complexes blocked at nonpairing lesions and mediates recruitment of DNA repair proteins.The first is to remove RNA polymerase (RNAP) complexes that have been stalled by a DNA lesion from the site of damage, and the second is to mediate the recruitment of DNA repair proteins.Coupling of transcription and DNA repair in bacteria is mediated by transcription-repair coupling factor (TRCF, the product of the mfd gene), which removes transcription elongation complexes stalled at DNA lesions and recruits the nucleotide excision repair machinery to the site.Mfd remains bound to the DNA in a long-lived complex that could act as a marker for sites of DNA damage, directing assembly of subsequent DNA repair factorsMfd may act through a translocase activity that rewinds upstream DNA, leading either to translocation or to release of RNA polymerase when the enzyme active site cannot continue elongation.Recent structural studies of the bacterial transcription-repair coupling factor, Mfd, have revealed a modular architecture in which an ATP-dependent DNA-based motor is coupled to protein-protein interaction domains that can attach the motor to RNA polymerase and the DNA repair protein UvrAE. coli Transcription repair coupling factor (Mfd protein) rescues arrested complexes by promoting forward translocation.We also find that the transcription-repair coupling factor, Mfd, promotes direct restart of the fork following the collision by facilitating displacement of the RNAPWe also found that the transcription-repair coupling factor Mfd promotes direct restart of the fork after the collision by facilitating displacement of the RNAPWe show that Mfd-dependent TCR in bacteria involves the formation of a damage search complex that can detect lesions downstream of a stalled RNAP, and that the strand specificity of the accelerated repair pathway is independent of the requirement for a lesion to stall RNAPcoli by the Mfd protein, which dissociates transcription elongation complexes blocked at nonpairing lesions and mediates recruitment of DNA repair proteins\"}, {\"question\": \"Which genes are affected in ROMANO-WARD syndrome?\", \"ideal_answer\": [\"The genes involved in ROMANO-WARD syndrome are KCNQ1, KCNE1, KCNE2, KCNH2,  SCN5A, CAV3, SCN4B, AKAP9, SNTA1, KCNJ5 and  Ankyrin-B.\"], \"exact_answer\": [[\"KCNQ1\", \"KVLQT1\", \"KCNA9\"], [\"KCNE1\"], [\"KCNE2\"], [\"KCNH2\", \"HERG\"], [\"SCN5A\"], [\"CAV3\"], [\"SCN4B\"], [\"AKAP9\"], [\"SNTA1\"], [\"KCNJ5\"], [\"Ankyrin-B gene\"]], \"context\": \"Long QT syndrome (LQTS) 1 is the most common type of inherited LQTS and is linked to mutations in the KCNQ1 gene. This report describes a three-generation family with a severe phenotype of long-QT syndrome-1 (LQTS-1) caused by a single nucleotide mutation in the KQT-like, voltage-gated potassium channel-1 gene (KCNQ1; MIM 607542).Mutations in a cardiac voltage-gated potassium channel, KCNQ1, account for the most common form of LQTS, LQTS1. Nine patients were diagnosed with LQT1 and nine with LQT2. The other six individuals were healthy, with no symptoms characteristic for prolonged QT syndrome, but came from families with confirmed disease occurrence. The study was conducted on members of four families. In order to search for mutations (using mSSCP and sequencing), genomic DNA was obtained from patients to determine the expression levels of the genes KCNQ1 and KCNH2 (HERG), involved in the occurrence of clinical signs of disease.Diagnosis of RWS is established by prolongation of the QTc interval in the absence of specific conditions known to lengthen it (for example, QT-prolonging drugs) and/or molecular genetic testing of the genes known to be associated with RWS, of which KCNQ1 (locus name LQT1), KCNH2 (locus name LQT2) and SCN5A (locus name LQT3) are the most common. Other, less frequently involved genes are KCNE1 (locus name LQT5), KCNE2 (locus name LQT6), CAV3 (locus name LQT9), SCN4B (locus name LQT10), AKAP9 (locus name LQT11), SNTA1 (locus name LQT12) and KCNJ5 (locus name LQT13).Type-1 long-QT syndrome (LQT1) is caused by mutations in the KCNQ1 gene.Congenital LQTS is most frequently caused by mutations in KCNQ1 (Kv7.1), whereas drug-induced LQTS is a consequence of HERG (human ether-a-go-go-related gene) channel inhibition. Mutations in the voltage-gated potassium channel subunit KCNQ1 induce the most common form of LQTS.Genetic studies have identified mutations in six ion channel genes, KCNQ1, KCNH2, SCN5A, KCNE1 and KCNE2 and the accessory protein Ankyrin-B gene, to be responsible for this disorder. Seven family members were carriers of two amino acid alterations in cis (V254M-V417M) in the cardiac potassium channel gene KCNQ1. KCNQ1 (formerly called KVLQT1) is a Shaker-like voltage-gated potassium channel gene responsible for the LQT1 sub-type of LQTS.Five genes have been implicated in Romano-Ward syndrome, the autosomal dominant form of LQTS: KVLQT1, HERG, SCN5A, KCNE1, and KCNE2.It usually is inherited as an autosomal dominant trait (Romano-Ward syndrome). The primary defect in LQT1 is a mutation in KVLQT1, a gene that encodes the pore-forming alpha-subunit of a K+ channel. Romano-Ward syndrome is an autosomal dominant long-QT syndrome (LQTS) that predisposes affected individuals to sudden death from tachyarrhythmias. We investigated the molecular basis of LQTS in a Taiwanese kindred. Clinical and genetic analyses revealed that a mutation was linked to the human ether-a-go-go-related gene (HERG).We describe a Swedish family with the proband and his brother suffering from severe Romano-Ward syndome (RWS) associated with compound heterozygosity for two mutations in the KVLQT1 (also known as KCNQ1 and KCNA9)The cardiac sodium channel gene, SCN5A, is also mutated in some Romano-Ward cases to produce defects in INa, the cardiac inward Na+ current. These findings provide the first evidence for a recessive form of the Romano-Ward long-QT syndrome and indicate that homozygous mutations on KVLQT1 do not invariably produce the Jervell and Lange-Nielsen syndrome. \"}, {\"question\": \"What are the major clinical Villefranche criteria for classic Ehlers-Danlos syndrome?\", \"ideal_answer\": [\"The major clinical Villefranche criteria for classic Ehlers-Danlos syndrome are skin hyperextensibility, dystrophic scarring, and joint hypermobility.\"], \"exact_answer\": [[\"skin hyperextensibility\"], [\"dystrophic scarring\"], [\"joint hypermobility\"]], \"context\": \"All type V collagen defects were identified within a group of 102 patients fulfilling all major clinical Villefranche criteria, that is, skin hyperextensibility, dystrophic scarring and joint hypermobilityAll type V collagen defects were identified within a group of 102 patients fulfilling all major clinical Villefranche criteria, that is, skin hyperextensibility, dystrophic scarring and joint hypermobility\"}, {\"question\": \"What is the treatment of neuropathic pain in children?\", \"ideal_answer\": [\"It is unclear if any treatment is registered for pediatric use. The reported treatments are:\\nOxcarbazepine \\nOpioids alone, in rotations  or with Analgesics (e.g. Ketamine and Lidocaine infusion)\\nOpioids and Benzodiazepines\\nPregabalin -  is one of the first drugs registered for the treatment of neuropathic pain. It is unclear if Pregabalin is registered for the treatment of neuropathic pain in children specifically but it is being used in practice.\\nTricyclic Antidepressants\\nLidocaine 5% patches for chronic localized neuropathic pain\"], \"exact_answer\": [[\"Oxcarbazepine\"], [\"Opioids alone, in rotations or with Analgesics (e.g. Ketamine and Lidocaine infusion)\"], [\"Opioids and Benzodiazepines - for terminal care\"], [\"Pregabalin\"], [\"Tricyclic Antidepressants\"], [\"Lidocaine 5% patches for chronic localized neuropathic pain\"], [\"Ketamine\"]], \"context\": \"Oxcarbazepine, a metabolite of carbamazepine, is used as an antiepileptic, analgesic for neuropathic pain and in the treatment of affective disorders. It has been approved by the Food and Drug Administration for partial seizures in adults as both adjunctive and monotherapy, and as adjunctive therapy in children aged from 2 to 16 yearsFor difficult to treat neuropathic pain from cancer, adjuvant analgesics are often used with opioidsKetamine and lidocaine can be safely infused together with concomitant opioids for the treatment of refractory neuropathic pain caused by cancer.Patient-controlled analgesia with morphine as continuous subcutaneous or intravenous infusions and the possibility of a bolus injection is suited for children aged 6 years.In neuropathic pain or phantom pain coanalgetics should be used to effectively treat pain in young patients.Pregabalin is one of the first drugs registered for the treatment of neuropathic pain. It is also indicated as adjuvant therapy in the treatment of epilepsy and for generalized anxiety disorder. Pregabalin is a GABA analogueTreatment was typically initiated with a tricyclic antidepressant, and 5 of the 6 girls noted improvement in their symptoms, including 2 who had marked improvement, and another 3 with substantial improvement who were able to discontinue therapy without a recurrence. CONCLUSIONS: Vulvodynia does occur among young girls and, when treated as a neuropathic pain disorder, was found to dramatically improve or remit in the majority of those treated in this small case series.We describe a case series of five adolescents who were managed with lidocaine 5% patches for chronic localized neuropathic pain from a variety of causes with minimal adverse effects. Treatment was effective in four of five patients with only one patient complaining of minimal pain reliefTwenty-two children or 14% of children on opioid therapy underwent 30 opioid rotations. Mucositis was the cause of pain in 19 (70%) children, bone pain in 3 (11%) children, and postoperative, visceral, or neuropathic pain in the remainder. The opioid was rotated either for excessive side effects with adequate analgesia (70%), excessive side effects with inadequate analgesia (16.7%), or tolerance (6.7%). Five (23%) children required two rotations, 3 during the same admission. The favored rotations were morphine to fentanyl in 20 (67%) children and fentanyl to hydromorphone in 6 (20%). Adverse opioid effects were resolved in 90% of cases, all failures occurred when morphine was rotated to fentanyl.To test the hypothesis that children with terminal cancer and neuropathic pain require rapid increases of opioids and benzodiazepines immediately before death, we compared drug usage in the last 72 hours of life in children with and without neuropathic painDying children with cancer and neuropathic pain have higher baseline requirements of morphine and benzodiazepines and require rapid increases of both drugs in the last 72 hours of life than dying children without neuropathic pain.ketamine treatment may be effective in children with severe neuropathic pain not responsive to other analgesics. This patient also demonstrates the feasibility of long-term ketamine treatment in pediatric oncology and that such treatment can be administered in a home care setting.Oral amitriptyline has been used as an analgesic in a wide range of pain settings. Despite long-term availability of a parenteral form, the few reports about this formulation have been limited to pharmacokinetic studies in normal volunteers, trials in depressed patients, and analyses of electroencephalogram (EEG) activation. We retrospectively reviewed our experience using intravenous (IV) amitriptyline at Children's Hospital, Boston and at Children's Hospital at Stanford. Eight children (aged 5-16.6 years), who were unable to tolerate medications by the oral route, received IV amitriptyline for a variety of indications, including neuropathic pain, depression, sleep disturbance, and as an adjuvant agent for opioid analgesia.\"}, {\"question\": \"Which deiodinases are present in skeletal  muscle?\", \"ideal_answer\": [\"Type 2 and Type 3 deiodinases are expressed in skeletal muscle and their expression is modulated by disease state and fasting.\"], \"exact_answer\": [[\"Type 2 deiodinase\"], [\"Tipe 3 deiodinase\"]], \"context\": \"The iodothyronine deiodinases D1, D2, and D3 enable tissue-specific adaptation of thyroid hormone levels in response to various conditions, such as hypothyroidism or fasting. The possible expression of D2 mRNA in skeletal muscle is intriguing because this enzyme could play a role in systemic as well as local T3 productionHuman skeletal muscle D2 mRNA expression is modulated by fasting and insulin, but not by hypothyroidism.SM had very low D2 activity and again no differences were found between groups; D3 activity in SM was higher in NTIS than controlsDeiodinase activities were then assayed in cell sonicates. The ratio of T3 production in cell sonicates (catalytic efficiency) was multiplied by the tissue activities reported in human liver (D1) and skeletal muscle (D2)\"}, {\"question\": \"Which genes are involved in patient response to warfarin?\", \"ideal_answer\": [\"The following genes have been associated with patient response to warfarin: CYP2C9, VKORC1, ORM1, CYP4F2, EPHX1, CYP2C18, CYP2C19, CYP3A5, protein S, clotting factor V, PROC, GGCX.\"], \"exact_answer\": [[\"CYP2C9\"], [\"VKORC1\"], [\"ORM1\"], [\"CYP4F2\"], [\"EPHX1\"], [\"CYP2C18\"], [\"CYP2C19\"], [\"CYP3A5\"], [\"protein S\"], [\"clotting factor V\"], [\"PROC\"], [\"GGCX\"]], \"context\": \"We identified ORM1 as another polymorphic gene affecting warfarin dose requirements. ORM1 *S carriers require lower maintenance doses to achieve and maintain an optimal level of anticoagulation.Detecting genetic polymorphism of CYP2C9 and VKORC1 could guide clinical use of warfarin to reduce the risk of adverse reactions including bleeding in patients receiving chronic anticoagulation therapyThe genes encoding for cytochrome P450 (CYP) 2C9 (CYP2C9) and vitamin K epoxide reductase complex subunit 1 (VKORC1) are the major genetic determinants of warfarin pharmacokinetics and pharmacodynamics, respectively.Numerous studies have demonstrated significant contributions of these genes to warfarin dose requirements. The CYP2C9 gene has also been associated with bleeding risk with warfarin. The CYP4F2 gene influences vitamin K availability and makes minor contributions to warfarin dose requirementsThere are other genetic polymorphisms that may further explain the response to warfarin. The VKORC1 genotype is an important determinant of response to warfarin in Chinese, but some genetic variants found in other ethnic groups that have a large effect on warfarin response and dosing are not commonly found in Chinesethe alleles rs1799853 (*2) and rs1057910 (*3) of the CYP2C9 gene, as well as rs9923231 of the VKORC1 gene were associated with warfarin dose required to achieve anticoagulation with INR of 2-3.Polymorphisms in the genes encoding the cytochrome P450 2C9 enzyme (CYP2C9) and the vitamin K epoxide reductase (VKORC1) are known to contribute to variability in sensitivity to coumarins.VKORC1 and CYP2C9 polymorphisms are important factors that influence warfarin dose response in Sudanese patientshe response to the anticoagulant drug warfarin is greatly affected by genetic polymorphisms in the VKORC1 and CYP2C9 genes.The HDA-based assays demonstrated a clinically acceptable performance for genotyping the VKORC1 -1639G>A SNP and two SNPs (430C>T and 1075A>C) for the CYP2C9 enzyme (CYP2C9*2 and CYP2C9*3), all of which are relevant in warfarin pharmacogenentics.Genetic variability in the VKORC1 and CYP2C9 genes is associated with increased warfarin sensitivityCYP2C9 and VKORC1 polymorphisms known to affect warfarin responsegenetic variants of VKORC1, CYP2C9, CYP4F2, and EPHX1 were found to be significant predictor variables for the maintenance dose of warfarin, explaining 26.6% of dose variability.CYP2C9 was the most important gene determining initial anticoagulant control, whereas VKORC1 was more important for stable anticoagulation. Novel associations with some clinical outcomes were found with single nucleotide polymorphisms in the cytochrome 450 genes CYP2C18 and CYP2C19, which were independent of the associations observed with CYP2C9 and in genes encoding CYP3A5, protein S and clotting factor V, although the variability explained by these genes was small.VKORC1 -1639 G>A polymorphism, body weight, age, and serum albumin were found to affect the inter-individual variabilitthe CYP2C9 and VKORC1 genes have been demonstrated to be determinants of warfarin response.On the basis of these observations, the Food and Drug Administration (FDA) approved a labeling change for warfarin that includes the genetic information of VKORC1 and CYP2C9 as factors influencing interindividual variability in warfarin dosing. Tthe two key genes of interest, the cytochrome P450 2C9 gene, CYP2C9, and the vitamin K epoxide reductase complex 1 gene, VKORC1, on warfarin responseThe influence of CYP2C9 and VKORC1 genotypes on warfarin dose requirements has been consistently demonstrated in diverse racial and ethnic patient groups in observational studies and randomized clinical trials.genetic factors influencing drug pharmacokinetics (CYP2C9) and pharmacodynamic response (VKORC1). In particular, the discovery of polymorphisms in the VKORC1 gene that strongly impact oral anticoagulant dose has heightened expectations that genetic testingOur data suggest that CYP2C9 genotype, age and body size are important determinants of warfarin dose requirements in African-Americans. Our data further suggest that the VKORC1 G6853C polymorphism alone may not be useful for predicting warfarin dose requirements in this racial group.A strong association was found between genetic polymorphisms in six genes, including VKORC1, CYP2C9, PROC, EPHX1, GGCX, and ORM1, and interindividual variability in the anticoagulant effect of warfarin; the strongest predictors were VKORC1 and CYP2C9.Three of six VKORC1 SNPs were found to be very strongly associated with the average warfarin dose required to achieve the target international normalised ratio (INR; p<0.0001)These results are of considerable clinical interest and confirm recently published results regarding the role of these two genes in modifying warfarin metabolism and maintenance dosage. The consistent findings regarding the role of VKORC1 and CYP2C9 in warfarin metabolism and maintenance dosage represent a clinically useful proof of principal for the use of pharmacogenomic information in medicine and may lead to improved understanding of warfarin's actions.Clinical studies showed an increased plasma level of S-warfarin, decreased clearance of S-warfarin, increased frequency of bleeding, and prolongation of hospitalization in patients with variant CYP2C9 alleles.\"}, {\"question\": \"List sodium glucose co-transporter-2 (SGLT2) inhibitors that have been FDA approved for type 2 diabetes mellitus treatment.\", \"ideal_answer\": [\"Canagliflozin, along with dapagliflozin and empagliflozin, are SGLT2 inhibitors approved by the US FDA for use in the treatment of type 2 diabetes.\"], \"exact_answer\": [[\"Dapagliflozin\"], [\"Empagliflozin\"], [\"Canagliflozin\"]], \"context\": \"Canagliflozin was the first SGLT2 inhibitor to be approved by the US FDA for use in the treatment of type 2 diabetes, and recently dapagliflozin has also been approved.To review available studies of empagliflozin, a sodium glucose co-transporter-2 (SGLT2) inhibitor approved in 2014 by the European Commission and the United States Food and Drug Administration for the treatment of type 2 diabetes mellitus (T2DM). Empagliflozin is a new once-daily oral SGLT2 inhibitor with a mechanism of action that is independent of \\u03b2-cell function and the insulin pathway.Sodium-glucose cotransporter type 2 (SGLT2) inhibitors such as canagliflozin and dapagliflozin have been approved for the treatment of type 2 diabetes mellitusTo review available studies of empagliflozin, a sodium glucose co-transporter-2 (SGLT2) inhibitor approved in 2014 by the European Commission and the United States Food and Drug Administration for the treatment of type 2 diabetes mellitus (T2DM)The present study emphasizes the molecular interactions between a new Food and Drug Administration (FDA) approved antidiabetic drug 'Invokana' (chemically known as Canagliflozin) with AChE and SGLT2 to establish a link between the treatment of T2DM and Alzheimer's Disease (AD). Invokana (Canagliflozin) as a dual inhibitor of acetylcholinesterase and sodium glucose co-transporter 2: advancement in Alzheimer's disease- diabetes type 2 linkage via an enzoinformatics study.This analysis assessed the effects of the SGLT2 inhibitor, canagliflozin, on model-based measures of beta cell function in patients with type 2 diabetes.  Inhibitors of the sodium-glucose co-transporter 2 (SGLT2) promote the excretion of glucose to reduce glycated hemoglobin (HbA1c) levels. Canagliflozin was the first SGLT2 inhibitor to be approved by the US FDA for use in the treatment of type 2 diabetes, and recently dapagliflozin has also been approved.a sodium glucose co-transporter-2 (SGLT2) inhibitor approved in 2014 by the European Commission and the United States Food and Drug Administration for the treatment of type 2 diabetes mellitus (T2DM).To review available studies of empagliflozin, a sodium glucose co-transporter-2 (SGLT2) inhibitor approved in 2014 by the European Commission and the United States Food and Drug Administration for the treatment of type 2 diabetes mellitus (T2DM).Sodium glucose co-transporter 2 (SGLT2) inhibition with canagliflozin in type 2 diabetes mellitus.Canagliflozin (Invokana\\u2122), an oral selective sodium-glucose co-transporter 2 (SGLT2) inhibitor, is under global development with Mitsubishi Tanabe Pharma and Janssen Pharmaceuticals, a subsidiary of Johnson and Johnson, for the treatment of type 2 diabetes mellitus.Invokana (Canagliflozin) as a dual inhibitor of acetylcholinesterase and sodium glucose co-transporter 2: advancement in Alzheimer's disease- diabetes type 2 linkage via an enzoinformatics study.Canagliflozin was the first SGLT2 inhibitor to be approved by the US FDA for use in the treatment of type 2 diabetes, and recently dapagliflozin has also been approved.\"}, {\"question\": \"For the treatment of which conditions can atypical neuroleptic drugs be used?\", \"ideal_answer\": [\"Atypical neuroloeptic drugs are antipsychotics used in patients with schizophrenia, schizoaffective disorder, delusional disorder, psychotic disorders, psychotic relapse in neuroleptic malignant syndrome and attention deficit hyperactivity disorder when presenting with negativism and conduct disorder.\"], \"exact_answer\": [[\"schizophrenia\", \"schizophrenia psychosis\"], [\"schizoaffective disorder\"], [\"delusional disorder\"], [\"psychotic relapse in neuroleptic malignant syndrome\"], [\"attention deficit hyperactivity disorder (ADHD) with negativism or conduct disorders\"], [\"psychotic disorders\"]], \"context\": \"Further clinical implications are described (capability of learning the therapeutic strategies, deliverability in broader clinical settings, acceptability by patients, combination with atypical neuroleptic drugs,and treatment of choice in risk populations).Intervention strategies are a multimodal psychological programme for the intervention in early prodromal stages and a combination of psychotherapy with atypical neuroleptic drugs in the late prodromal stages. Atypical neuroleptic drugs have enriched our treatment programs, especially in childhood and adolescent schizophrenia. Reviewed here is the use of atypical neuroleptics in children and adolescents with a schizophrenic disorder. o prove whether weight gain is a relevant side effect of atypical neuroleptics, the charts of all patients admitted with DSM-III-R diagnoses of schizophrenia, schizoaffective disorder, or delusional disorder in the years 1991 to 1995 were evaluated.The handwriting of 18 schizophrenic patients before and during treatment with typical (haloperidol, haloperidol decanoate) and atypical (clozapine, risperidone) neuroleptic drugs was examined.Atypical and typical neuroleptics, when administered chronically, can bring about profound but contrasting changes in schizophrenic symptoms and motor activation and dramatically modulate brain neurochemistryRecent studies suggest that clozapine is more effective than typical neuroleptics for patients with treatment-resistant schizophrenia.Acute and late onset movement disorders frequently complicate the treatment of psychosis with typical neuroleptic drugs like haloperidol, but not with atypical neuroleptic drugs like clozapine. Only the group of boys presented other comorbidities such as negativism and conduct disorders; approximately 25% of them required treatment with atypical neuroleptic drugs.Atypical neuroleptic drugs are the preferred treatment for symptoms such as delusions, hallucinations, agitation and aggressive behaviour.Monotherapeutic treatment with new atypical neuroleptic drugs had a more positive effect on the mental health related quality of life (MCS) in comparison to treatment with polypharmacological treatment but not with oral conventional antipsychotics. \"}, {\"question\": \"Which are the characteristics of  Andersen syndrome?\", \"ideal_answer\": [\"the characteristics of Andersen syndrome are abnormal QT-U complex, ventricular arrhythmia, periodic paralysis, and facial and skeletal dysmorphisms\"], \"exact_answer\": [[\"abnormal QT-U complex\"], [\"ventricular arrhythmia\"], [\"periodic paralysis\"], [\"facial dysmorphisms\"], [\"skeletal dysmorphisms\"]], \"context\": \"Mutations of KCNJ2, the gene encoding the human inward rectifier potassium channel Kir2.1, cause Andersen-Tawil syndrome (ATS), a disease exhibiting ventricular arrhythmia, periodic paralysis, and dysmorphic features.Andersen cardiodysrhythmic periodic paralysis or Andersen-Tawil syndrome includes the distinct clinical features of periodic paralysis, cardiac arrhythmia, and facial and skeletal dysmorphisms and exhibits autosomal dominant inheritance. The patient was a 19-year-old woman with familial periodic paralysis, abnormal QT-U complex, and nonsustained ventricular tachycardia. Andersen-Tawil syndrome (ATS) is a rare inherited disorder, characterised by periodic paralysis, cardiac dysarrhythmias, and dysmorphic features, and is caused by mutations in the gene KCNJ2, which encodes the inward rectifier potassium channel, Kir2.1.Evaluation of candidate loci culminated in the identification of a heterozygous missense mutation (R67W) in KCNJ2, the gene encoding the inward-rectifying potassium current, Kir2.1, in 41 members of a kindred in which ventricular arrhythmias (13 of 16 female members [81%]) and periodic paralysis (10 of 25 male members [40%]) segregated as autosomal dominant traits with sex-specific variable expressivity. Some mutation carriers exhibited dysmorphic features, including hypertelorism, small mandible, syndactyly, clinodactyly, cleft palate, and scoliosis, which, together with cardiodysrhythmic periodic paralysis, have been termed \\\"Andersen syndrome.\\\" \"}, {\"question\": \"Mutations in which gene and which protein are associated with Netherton syndrome?\", \"ideal_answer\": [\"NS is due to loss-of-function mutations in the SPINK5 gene and to the consequent lack of expression of its encoded protein LEKTI in the skin and all stratified epithelial tissues.\"], \"exact_answer\": [[\"SPINK5 gene\"], [\"LEKTI protein\"]], \"context\": \"NS is due to loss-of-function mutations in the SPINK5 gene and to the consequent lack of expression of its encoded protein LEKTI in the skin and all stratified epithelial tissues.Recently, we identified SPINK5, which encodes the serine protease inhibitor Kazal-type 5 protein (LEKTI), as the defective gene in Netherton syndrome.Netherton syndrome (NS) is a rare, life-threatening ichthyosiform syndrome caused by recessive loss-of-function mutations in SPINK5 gene encoding lymphoepithelial Kazal-type-related inhibitor (LEKTI), a serine protease inhibitor expressed in the most differentiated epidermal layers and crucial for skin barrier function.Netherton syndrome (NS) is a debilitating congenital skin disorder caused by mutations in the SPINK5 gene encoding the lymphoepithelial Kazal-type-related inhibitor (LEKTI).Loss-of-function mutations in the Kazal-type serine protease inhibitor, LEKTI, encoded by the SPINK5 gene cause the rare autosomal recessive skin disease Netherton syndrome (NS).SPINK5 and Netherton syndrome: novel mutations, demonstration of missing LEKTI, and differential expression of transglutaminases.Netherton's syndrome is a rare autosomal recessive disorder caused by mutations of the SPINK5 gene, which encodes the lymphoepithelial Kazal-type-related inhibitor (LEKTI) protein.We observed microstructural changes and detected LEKTI activity and SPINK5 gene mutation in three Chinese patients with Netherton's syndrome.Several skin diseases and atopic disorders including Netherton syndrome and atopic dermatitis have been associated with mutations and deviations of expression of SPINK5, the gene encoding the human 15-domain serine proteinase inhibitor LEKTILoss-of-function mutations in the Kazal-type serine protease inhibitor, LEKTI, encoded by the SPINK5 gene cause the rare autosomal recessive skin disease Netherton syndrome (NS)Mutations in the serine protease inhibitor Kazal type 5 (SPINK5) gene leading to lymphoepithelial Kazal-type-related inhibitor (LEKTI) deficiency cause NS.  Netherton syndrome is a severe autosomal recessive skin disorder characterized by congenital erythroderma, a specific hair-shaft abnormality, and atopic manifestations with high IgE levels. Recently, we identified SPINK5, which encodes the serine protease inhibitor Kazal-type 5 protein (LEKTI), as the defective gene in Netherton syndrome.Recently, we identified SPINK5, which encodes the serine protease inhibitor Kazal-type 5 protein (LEKTI), as the defective gene in Netherton syndrome. Here we describe the intron-exon organization of the gene and characterize the SPINK5 mutations in patients from 21 families of different geographic origin, using denaturing high performance liquid chromatography and direct sequencing.Mutation of the SPINK5 gene has been identified as disease-causing in Netherton syndrome, but the pathophysiology still remains unclear. Almost all SPINK5 mutations result in the absence of the serine-protease inhibitor LEKTI protein in both keratinocytes and lymphocytes.Several skin diseases and atopic disorders including Netherton syndrome and atopic dermatitis have been associated with mutations and deviations of expression of SPINK5, the gene encoding the human 15-domain serine proteinase inhibitor LEKTI.Netherton&apos;s syndrome is a rare autosomal recessive disorder caused by mutations of the SPINK5 gene, which encodes the lymphoepithelial Kazal-type-related inhibitor (LEKTI) protein. We observed microstructural changes and detected LEKTI activity and SPINK5 gene mutation in three Chinese patients with Netherton&apos;s syndrome.NS is due to loss-of-function mutations in the SPINK5 gene and to the consequent lack of expression of its encoded protein LEKTI in the skin and all stratified epithelial tissues. Following the identification of the NS causative gene and protein, specific diagnostic tools have been developed, thus breaking up the challenge of distinguishing NS from other congenital ichthyoses with overlapping features, and from severe, early-onset forms of atopic dermatitis or psoriasis.Recently, we identified SPINK5, which encodes the serine protease inhibitor Kazal-type 5 protein (LEKTI), as the defective gene in Netherton syndrome.\"}, {\"question\": \"Which proteins participate in the formation of the Notch transcriptional activation complex?\", \"ideal_answer\": [\"The Notch intracellular domain (NICD) forms a transcriptional activation complex with the DNA-binding factor CSL and a transcriptional co-activator of the Mastermind family (MAML). ICN binds to a highly conserved DNA-binding transcription factor called CSL (also known as RBP-Jkappa, CBF1, Suppressor of Hairless, and Lag-1) and recruits Mastermind-like transcriptional co-activators to form a transcriptional activation complex.\", \"Although it is well understood that N(ICD) forms a transcriptional activation complex, little is known about how the complex is assembled. The Notch intracellular domain (NICD) forms a transcriptional activation complex with the DNA-binding factor CSL and a transcriptional co-activator of the Mastermind family (MAML).\"], \"exact_answer\": [[\"Notch intracellular domain (NICD)\", \"NICD\", \"Intracellular part of Notch\", \"ICN\", \"NIC\"], [\"Transcription factor CSL\", \"RBP-Jkappa\", \"CBF1\", \"Suppressor of Hairless\", \"Lag-1\"], [\"The coactivator protein Mastermind-like 1\", \"MAML-1\"]], \"context\": \"The Notch intracellular domain (NICD) forms a transcriptional activation complex with the DNA-binding factor CSL and a transcriptional co-activator of the Mastermind family (MAML). The \\\"RAM\\\" region of NICD recruits Notch to CSL, facilitating the binding of MAML at the interface between the ankyrin (ANK) repeat domain of NICD and CSL.Inappropriate release of the intracellular domain of Notch (N(ICD)) from the plasma membrane results in the accumulation of deregulated nuclear N(ICD) that has been linked to human cancers, notably T-cell acute lymphoblastic leukemia (T-ALL). Nuclear N(ICD) forms a transcriptional activation complex by interacting with the coactivator protein Mastermind-like 1 and the DNA binding protein CSL (for CBF-1/Suppressor of Hairless/Lag-1) to regulate target gene expression.Although it is well understood that N(ICD) forms a transcriptional activation complex, little is known about how the complex is assembled.Canonical Notch signaling is initiated when ligand binding induces proteolytic release of the intracellular part of Notch (ICN) from the cell membrane. ICN then travels into the nucleus where it drives the assembly of a transcriptional activation complex containing the DNA-binding transcription factor CSL, ICN, and a specialized co-activator of the Mastermind family. A consensus DNA binding site motif for the CSL protein was previously defined using selection-based methods, but whether subsequent association of Notch and Mastermind-like proteins affects the DNA binding preferences of CSL has not previously been examined.We report here the crystal structure of a Notch transcriptional activation complex containing the ankyrin domain of human Notch1 (ANK), the transcription factor CSL on cognate DNA, and a polypeptide from the coactivator Mastermind-like-1 (MAML-1). Ligand binding by Notch receptors triggers a series of proteolytic cleavages that liberate the intracellular portion of Notch (ICN) from the cell membrane, permitting it to translocate to the nucleus. Nuclear ICN binds to a highly conserved DNA-binding transcription factor called CSL (also known as RBP-Jkappa, CBF1, Suppressor of Hairless, and Lag-1) and recruits Mastermind-like transcriptional co-activators to form a transcriptional activation complex.On the basis of our results, we present a working structural model for the organization of the MAML1.ICN.CSL.DNA complex.The requirement for cooperative assembly of the MAML1.ICN.CSL.DNA complex suggests that a primary function of ICN is to render CSL competent for MAML loading.We report here that the astrogliogenic role of Notch is in part mediated by direct binding of the Notch intracellular domain to the CSL DNA binding protein, forming a transcriptional activation complex onto the astrocyte marker gene, glial fibrillary acidic protein (GFAP).Importantly, although the classical CSL-dependent Notch signaling pathway is intact and able to activate the Notch canonical target promoter during the neurogenic phase, it is unable to activate the GFAP promoter during neurogenesis.Mastermind (Mam) is a component of Notch pathway signaling. In combination with the intracellular domain of Notch and Suppressor of Hairless, Mam forms a transcriptional activation complex.High levels of Notch-mediated transcriptional activation require the formation of a ternary complex consisting of NotchICD, CSL (CBF-1, suppressor of hairless, LAG-1) and a Mastermind family member.This function relies on the formation of transcriptional activation complexes that include intracellular Notch, a Mastermind co-activator and the transcription factor CSL bound to cognate DNA.This function relies on the formation of transcriptional activation complexes that include intracellular Notch, a Mastermind co-activator and the transcription factor CSL bound to cognate DNAHigh levels of Notch-mediated transcriptional activation require the formation of a ternary complex consisting of NotchICD, CSL (CBF-1, suppressor of hairless, LAG-1) and a Mastermind family memberNotchIC, together with the transcriptional coactivator Mastermind, form a ternary complex with CSL that activates transcription from genes that are responsive to Notch signalingNuclear N(ICD) forms a transcriptional activation complex by interacting with the coactivator protein Mastermind-like 1 and the DNA binding protein CSL (for CBF-1/Suppressor of Hairless/Lag-1) to regulate target gene expressionNuclear ICN binds to a highly conserved DNA-binding transcription factor called CSL (also known as RBP-Jkappa, CBF1, Suppressor of Hairless, and Lag-1) and recruits Mastermind-like transcriptional co-activators to form a transcriptional activation complex\"}, {\"question\": \"Which are the main methods for pharmacophore modelling?\", \"ideal_answer\": [\"A pharmacophore describes the arrangement of molecular features a ligand must contain to efficaciously bind a receptor. Pharmacophore models are developed to improve molecular understanding of ligand\\u2013protein interactions, and can be used as a tool to identify novel compounds that fulfil the pharmacophore requirements and have a high probability of being biologically active. Protein structure-based pharmacophores (SBPs) derive these molecular features by conversion of protein properties to reciprocal ligand space. Unlike ligand-based pharmacophore models, which require templates of ligands in their bioactive conformation, SBPs do not depend on ligand information.\"], \"exact_answer\": [[\"Ligand-based, pharmacophore modeling\"], [\"Structure-base, pharmacophore modeling\"]], \"context\": \"protein interactions are becoming increasingly significant as potential drug targets; however, the rational identification of small molecule inhibitors of such interactions remains a challenge. Pharmacophore modelling is a popular tool for virtual screening of compound libraries, and has previously been successfully applied to the discovery of enzymatic inhibitors. However, the application of pharmacophore modelling in the field of protein:protein interaction inhibitors has historically been considered more of a challenge and remains limited. In this review, we explore the interaction mimicry by known inhibitors that originate from in vitro screening, demonstrating the validity of pharmacophore mapping in the generation of queries for virtual screening. We discuss the pharmacophore mapping methods that have been successfully employed in the discovery of first-in-class inhibitors.  a popular tool for virtual screening of libraries to identify novel active substances that can be potentially developed into drugs. While they have been applied for years on common drug targets, their application in the discovery of protein-protein interaction inhibitors remains limited. This review describes current pharmacophore modelling methods applied in the discovery of novel inhibitors targeting protein-protein interactions. We first address the mimicry of protein-protein interactions with their respective inhibitors as observed in crystal structure complexes. This mimicry can be exploited to derive a pharmacophore query from protein-protein complex structures. We then discuss several cases where pharmacophore queries were utilized for the discovery of first-in-class inhibitors of their respective protein-protein interaction targets.The best HypoGen pharmacophore model for ACC2 inhibitors (Hypo1_ACC2) consists of one hydrogen bond acceptor, one hydrophobic aliphatic and one hydrophobic aromatic feature, whereas the best pharmacophore (Hypo1_ACC1) for ACC1 consists of one additional hydrogen-bond donor (HBD) features. The best pharmacophore hypotheses were validated by various methods such as test set, decoy set and Cat-Scramble methodology. The validated pharmacophore models were used to screen several small-molecule databases, including Specs, NCI, ChemDiv and Natural product databases to identify the potential dual ACC inhibitors. The virtual hits were then subjected to several filters such as estimated [Formula: see text] value, quantitative estimation of drug-likeness and molecular docking analysis.pharmacophore model has been developed using diverse classes of epidermal growth factor receptor (EGFR) tyrosine kinase (TK) inhibitors useful in the treatment of human tumours. Among the top 10 generated hypotheses, the second hypothesis, with one hydrogen bond acceptor, one ring aromatic and three hydrophobic features, was found to be the best on the basis of Cat Scramble validation as well as test set prediction (r(training) = 0.89, r(test) = 0.82). The model also maps well to the external test set molecules as well as clinically active molecules and corroborates the docking studies. Finally, 10 hits were identified as potential leads after virtual screening of ZINC database for EGFR TK inhibition. In order to clarify the essential structure-activity relationship for the known Aurora-A inhibitors as well as identify new lead compounds against Aurora-A, 3D pharmacophore models were developed based on the known inhibitors. computational methods for molecular design are well established in medicinal chemistry research, their application in the field of natural products is still not exhaustively explored. This article gives a short introduction into both the potential for the application of computer-assisted approaches, such as pharmacophore modelling, virtual screening, docking, and neural networking to efficiently access the bioactive metabolites, and the requirements and limitations related to this specific field. The challenge is which selection criteria and/or multiple filtering tools to apply for a target-oriented isolation of potentially bioactive secondary metabolites. Application examples are provided where in silico tools and classical methods used by natural product scientists are used in an effort to maximize their efficacy in drug discovery. pharmacophore models derived from protein binding site atoms without the inclusion of any ligand information have become more popular in virtual screening studies. However, the accuracy of protein-based pharmacophore models for reproducing the critical protein-ligand interactions has never been explicitly assessed. Our results demonstrate that there are significant variations in the success of protein-based pharmacophore models to reproduce native contacts and consequently native ligand poses dependent on the details of the pharmacophore generation process. We show that the generation of optimized protein-based pharmacophore models is a promising approach for ligand pose prediction and pose rankingspharmacophore model does not describe a real molecule or a real association of functional groups but illustrates a molecular recognition of a biological target shared by a group of compounds. Pharmacophores also represent the spatial arrangement of essential interactions in a receptor-binding pocket. Structure based pharmacophores (SBPs) can work both with a free (apo) structure or a macromolecule-ligand complex (holo) structure. The SBP methods that derive pharmacophore from protein-ligand complexes use the potential interactions observed between ligand and protein, whereas, the SBP method that aims to derive pharmacophore from ligand free protein, uses only protein active site information3D pharmacophore methods are now commonly used as part of more complex workflows in drug discovery campaigns, and have been successfully and extensively applied in virtual screening (VS) approaches. \"}, {\"question\": \"Which are the inhibitors of histone methyltransferases?\", \"ideal_answer\": [\"In general, histone methyltransferases (HMTs) have no widely approved high-throughput screening assay format, and therefore reference inhibitors are not available for many of the HMTs. However, there are several selective HMT inhibitors: Trichostatin A (TSA), BIX-01294 and its derivative TM2-115, 2,4-pyridinedicarboxylic acid (2,4-PDCA), 3-deazaneplanocin A (DZNep), Psammaplin A (PsA) and Sulforaphane (SFN).\"], \"exact_answer\": [[\"BIX-01294\"], [\"TM2-115\"], [\"2,4-pyridinedicarboxylic acid\", \"2,4-PDCA\"], [\"3-deazaneplanocin A\", \"DZNep\"], [\"Trichostatin A\", \"TSA\"], [\"Psammaplin A\", \"PsA\"], [\"Sulforaphane\", \"SFN\"]], \"context\": \"However, in general, HMTs have no widely accepted high-throughput screening (HTS) assay format, and reference inhibitors are not available for many of the enzymes.Small-molecule histone methyltransferase inhibitors display rapid antimalarial activity against all blood stage forms in Plasmodium falciparum.We synthesized a compound library based upon a known specific inhibitor (BIX-01294) of the human G9a histone methyltransferase. Two compounds, BIX-01294 and its derivative TM2-115, inhibited P. falciparum 3D7 parasites in culture with IC(50) values of ~100 nM, values at least 22-fold more potent than their apparent IC(50) toward two human cell lines and one mouse cell line.Together, these results suggest that BIX-01294 and TM2-115 inhibit malaria parasite histone methyltransferases, resulting in rapid and irreversible parasite death.Synthesis and structure-activity relationship investigation of adenosine-containing inhibitors of histone methyltransferase DOT1L.A total of 55 adenosine-containing compounds were designed and synthesized, among which several potent DOT1L inhibitors were identified with K(i) values as low as 0.5 nM.Several new reagents and assays were developed to aid in the identification of EZH2 inhibitors, and these were used to execute two high-throughput screening campaigns.Of the five histone methyltransferases known to mediate methylation of the lysine 9 residue of histone H3 (H3K9), euchromatic histone-lysine N-methyltransferase 2 (EHMT2; also known as G9a) has been shown to be a primary mediator of H3K9 dimethylation; BIX-01294 has been shown to be a specific inhibitor of EHMT2We hypothesised that inhibition of EHMT2 by BIX-01294 would result in reduced levels of H3K9 dimethylation and compromised embryo development.We also demonstrate that peptides that mimic SET1 family Win motif sequences inhibit H3K4 dimethylation by the MLL1 core complex with varying degrees of efficiency.Studies of H3K4me3 demethylation by KDM5B/Jarid1B/PLU1 reveals strong substrate recognition in vitro and identifies 2,4-pyridine-dicarboxylic acid as an in vitro and in cell inhibitor.Inhibition studies of ccKDM5B showed both in vitro and in cell inhibition of ccKDM5B by 2,4-pyridinedicarboxylic acid (2,4-PDCA) with a potency similar to that reported for the HDM KDM4C.Treatment with 3-deazaneplanocin A (DZNep), an inhibitor of H3K27me3 and H4K20me3, significantly enhanced the BZLF1 transcription in Raji cells when in combination with an HDAC inhibitor, trichostatin A (TSA).All assays allowed profiling of known SET7/9 and LSD1 inhibitors. The results demonstrate that the optimized LANCE Ultra and AlphaLISA assay formats provide a relevant biochemical screening approach toward the identification of small-molecule inhibitors of HMTs and HDMs that could lead to novel epigenetic therapies.Selective inhibitors of histone methyltransferase DOT1L: design, synthesis, and crystallographic studies.We used structure- and mechanism-based design to discover several potent inhibitors of DOT1L with IC(50) values as low as 38 nM.Inhibition of histone lysine methylation enhances cancer-testis antigen expression in lung cancer cells: implications for adoptive immunotherapy of cancer.Short hairpin RNAs were used to inhibit several histone methyltransferases (KMT) and histone demethylases (KDM) that mediate histone methylation and repress gene expression.DZNep, a pharmacologic inhibitor of KMT6 expression, recapitulated the effects of KMT6 knockdown.Recent evidence shows that S-adenosylhomocysteine (AdoHcy) hydrolase inhibitors (AHI) such as 3-deazaneplanocin A (DZNep) modulate chromatin through indirect inhibition of histone methyltransferases including EZH2. We investigated the biological effects of AdoHcy hydrolase inhibition using DZNep and its structural analogues 3-deazaadenosine (DZA) and neplanocin A (Nep A) in breast cancer cells.A chemiluminescence-based method for identification of histone lysine methyltransferase inhibitors.The method is particularly well suited for detection of inhibitors acting by the desired histone peptide competitive mechanism and is applicable to testing other HMTs, demonstrated here with the G9a homolog EHMT1, also known as GLP.This study reports the pharmacokinetics and tissue distribution of a novel histone deacetylase and DNA methyltransferase inhibitor, psammaplin A (PsA), in mice.Human diet contains many histone deacetylase (HDAC) inhibitors, such as the bioactive component sulforaphane (SFN), whose epigenetic effects on MSTN gene in satellite cells are unknown.We found that DNA methyltransferase inhibitor (5-Aza-2'-deoxycytidine) and histone deacetylase inhibitor (trichostatin A) reduced leptin receptor expression.However, histone deacetylase (HDAC) inhibitors trichostatin A (TSA) and sodium butyrate (NaBt) significantly increased Wnt5a mRNA expression in SW620.\"}, {\"question\": \"Which genes are regulated by MEF-2 in the heart?\", \"ideal_answer\": [\"COX-2, ANF, estrogen receptor (ER)alpha gene, calsequestrin gene, casq2, cTnT, MCK, alpha-cardiac actin, sarco(endo)plasmic reticulum Ca2+-ATPase, SERCA, MLC-2, alpha-cardiac myosin heavy chain gene, phosphoglycerate mutase and PGAM-M are regulated by MEF-2 in the heart\"], \"exact_answer\": [[\"COX-2\"], [\"ANF\"], [\"estrogen receptor (ER)alpha gene\"], [\"calsequestrin gene\", \"casq2\"], [\"cTnT\"], [\"MCK\"], [\"alpha-cardiac actin\"], [\"sarco(endo)plasmic reticulum Ca2+-ATPase\", \"SERCA\"], [\"MLC-2\"], [\"alpha-cardiac myosin heavy chain gene\"], [\"phosphoglycerate mutase\", \"PGAM-M\"], [\"cardiac troponin C\", \"cTnC\"]], \"context\": \"Inhibition of MEF2A using siRNA attenuated HB-EGF-induced COX-2, ANF expression and cell size.This genetic reprogramming coincides with a pronounced increase in expression of the estrogen receptor (ER)alpha gene, which we show to be a direct MEF2 target genecardiac calsequestrin gene (casq2)Functional studies demonstrated that site-directed mutagenesis of the proximal MEF-2 and CArG box sites significantly decreased the transcription of the gene in cardiac and skeletal muscle cells, indicating that they are important to drive cardiac and skeletal muscle-specific transcription of the casq2 gene.DTEF-1 also interacts with MEF- 2 by coimmunoprecipitation and independently or cooperatively (with MEF-2) trans-activates the cTnT promoterAn 85-bp region within the enhancer is highly conserved between human and mouse and contains a central AT-rich site, which is essential for enhancer activity. This site binds myocyte enhancer factor (MEF)2 factors, principally MEF2D and MEF2A in cardiocyte nuclear extracts. These results are discussed in the context of MEF2 activity and of the regulation of the alpha-cardiac actin locus.The cis-acting elements, MEF-2, E boxes and A/T rich elements present in the enhancer region of the mouse MCK gene are known to regulate the expression of the geneThe sarco(endo)plasmic reticulum Ca2+-ATPases (SERCAs) belong to a family of active calcium transport enzymes encoded by the SERCA1, 2, and 3 genes. In this study, we describe the complete structure of the human SERCA2 gene and its 5 -regulatory region.Among the DNA cis-elements present in these two regulatory regions there are potential binding sites for: GATA-4, -5, -6, Nkx-2.5/Csx, OTF-1, USF, MEF-2, SRF, PPAR/RXR, AP-2, and TREs. Upstream from position -1.5 kb, there is no significant homology among the SERCA2 genes cloned.The cardiac calsequestrin gene consists of 11 exons and its 5' flanking region is characterized by the presence of a TATA-like box, muscle specific promoter elements such as 7 E-boxes, 1 MEF-2, 1 MCBF and 1 Repeat (musS) motifs, as well as several muscle non-specific transcriptional elements (AP-2A, NRE1, NRE2, p53, Spel and TFI-IIA).our laboratory identified a 28 bp HF-la/MEF-2 element in the MLC-2v promoter region, which confers cardiac ventricular chamber-specific gene expression during murine cardiogenesis,In this study, we investigated T3R alpha 1-vs. T3R beta 1-specific interactions with the myocyte enhancer-specific factor-2 (MEF-2) on the expression of the SERCA 2 gene in transient transfection assays in embryonal heart-derived H9c2 cells.point to T3R isoform-specific interactions with a cell type-specific transcription factor (MEF-2) in the regulation of SERCA 2 gene expression.In multiple independent transgenic mouse lines, we found that both a 250 base pair myosin light chain-2 ventricular promoter fragment, as well as a dimerized 28 bp sub-element (HF-1) containing binding sites for HF1a and HF1b/MEF2 factors, directed ventricular-specific reporter expression from as early as the endogenous gene, at day 7.5-8.0 post coitum.Myocyte-specific enhancer-binding factor (MEF-2) regulates alpha-cardiac myosin heavy chain gene expression in vitro and in vivoIn the MLC-2 gene, an AT-rich element (HF-1b) which contains a consensus MEF-2 site is required for cardiac tissue-specific expression.Role of myocyte-specific enhancer-binding factor (MEF-2) in transcriptional regulation of the alpha-cardiac myosin heavy chain gene.In order to analyze the transcriptional regulation of the muscle-specific subunit of the human phosphoglycerate mutase (PGAM-M) gene, chimeric genes composed of the upstream region of the PGAM-M geneThese observations define the PGAM-M enhancer as the only cardiac- and skeletal-muscle-specific enhancer characterized thus far that is mainly activated through MEF-2.Transcription of each gene is independently controlled but coordinately regulated. During each embryogenesis, the beta-MHC gene is expressed as part of the cardiac myogenic program under the control of NKX-2.5, MEF-2C, and GATA-4/5/6.We have characterized the specific DNA regulatory elements responsible for the function of the human cardiac troponin C gene (cTnC) muscle-specific enhancer in myogenic cells. We used functional transient transfection assays with deletional and site-specific mutagenesis to evaluate the role of the conserved sequence elements. Gel electrophoresis mobility shift assays (EMSA) demonstrated the ability of the functional sites to interact with nuclear proteins. We demonstrate that three distinct transcription activator binding sites commonly found in muscle-specific enhancers (a MEF-2 site, a MEF-3 site, and at least four redundant E-box sites) all contribute to full enhancer activity but a CArG box does not.\"}, {\"question\": \"Which are the newly identified DNA nucleases that can be used to treat thalassemia?\", \"ideal_answer\": [\"Thalassemia is genetic diseases of the blood caused by mutations in the globin gene.  Main goal for thalassemia treatment is to develop homologous recombination based gene therapy in order to cure these diseases. Zinc finger nucleases (ZFNs) and TAL effector nucleases (TALENs) are proper targets for the human globin gene. Genome editing using engineered nucleases such as ZFNs and TALENs has become a powerful technology for reverse genetics.\", \"The newly identified DNA nucleases that can be used to treat thalassemia are the transcription activator-like effector nucleases (TALEN). These are engineered proteins able to stimulate targeted integration of therapeutic wild-type beta-globin cDNAs to the endogenous beta-globin locus.\"], \"exact_answer\": [[\"TALEN\", \"Transcription activator-like effector nucleases\"], [\"Crispr/CAS\"], [\"zinc-finger nucleases\"]], \"context\": \"Transcription activator-like effector nuclease (TALEN) and zinc finger nuclease (ZFN) DNA editing technology enables site-directed engineering of the genomeWe used this opportunity to evaluate the effect of total donor homology on transcription activator-like effector nuclease (TALEN) mediated bi-allelic modification. Herein we describe our applications of site-specific nucleases, especially transcription activator-like effector nucleases, to engineer specific alterations in the genomes of pigs and cows.In this study, we used zinc finger nuclease-mediated knockout of the aryl hydrocarbon receptor (AHR) or AHR nuclear translocator (ARNT) in MCF7 and AHR knockout in MDA-MB-231 human breast cancer cells to investigate cross talk among AHR, ARNT, and estrogen receptor \\u03b1 (ER\\u03b1). For example, after binding DNA, TALEs fused to transcriptional activation domains can function as robust transcription factors (TALE-TFs), while fused to restriction endonucleases (TALENs) can cut DNA. Genome editing using engineered nucleases such as transcription activator-like effector nucleases (TALENs) has become a powerful technology for reverse geneticsHere, we describe a robust process combining efficient generation of integration-free \\u03b2-Thal iPSCs from the cells of patients and transcription activator-like effector nuclease (TALEN)-based universal correction of HBB mutations in situ.We report here the correction of \\u03b1-thalassemia major hydrops fetalis in transgene-free iPS cells using zinc finger-mediated insertion of a globin transgene in the AAVS1 site on human chromosome 19.  Here, we review the development of Cas9 as an important tool to not only edit the genomes of a number of different prokaryotic and eukaryotic species, but also as an efficient system for site-specific transcriptional repression or activation. Additionally, a specific Cas9 protein has been observed to target an RNA substrate, suggesting that Cas9 may have the ability to be programmed to target RNA as well.RNA-guided endonucleases (RGENs), derived from the prokaryotic adaptive immune system known as CRISPR/Cas, enable targeted genome engineering in cells and organisms. The CRISPR/Cas technology has been successfully used to stimulate the integration of small DNA sequences in a target locus to produce gene mutations.\"}, {\"question\": \"List inflammatory caspase proteins?\", \"ideal_answer\": [\"caspase-1\\ncaspase-4\\ncaspase-5\"], \"exact_answer\": [[\"caspase-1\"], [\"caspase-4\"], [\"caspase-5\"]], \"context\": \" In addition to identifying caspases-4 and -5 as potential targets for limiting intestinal inflammation, this study has identified epithelial-expressed caspases-4 and -5 as biomarkers with diagnostic and therapeutic potential in CRC.However, the involvement of two related inflammatory caspase members, caspases-4 and -5, during intestinal homeostasis and disease has not yet been established. This study demonstrates that caspases-4 and -5 are involved in IBD-associated intestinal inflammation.Altered activity of caspases, particularly caspase-1, has been implicated in the development of intestinal diseases, such as inflammatory bowel disease (IBD) and colorectal cancer (CRC)In contrast, engagement of various PRR in the recently identified inflammasome complexes lead to activation of a cysteine protease, caspase-1. ctivation of inflammatory caspase proteases. One such caspase, CASPASE-1 (CASP1), Inflammasomes are cytosolic multiprotein complexes assembled by intracellular nucleotide-binding oligomerization domain (NOD)-like receptors (NLRs) and they initiate innate immune responses to invading pathogens and danger signals by activating caspase-1 (ref. 1).e activation of inflammasome by different stimuli triggers the proteolytic cleavage of pro-caspase 1 into active caspase 1Formation of the inflammasome can lead to the activation of inflammatory caspases, such as Caspase-1, which then activate pro-inflammatory cytokines by proteolytic cleavage. inflammatory caspase-1 Human caspase-5 is classified as an inflammatory caspase, although its substrate has not been identified yet.e inflammatory Caspase-1 the inflammatory caspase-1 Although it is conventionally regarded as an inflammatory caspase, recent studies have shown that caspase-4 plays a role in induction of apoptosis by endoplasmic reticulum (ER) stress. \"}, {\"question\": \"List invertebrates where ultraconserved elements have been identified.\", \"ideal_answer\": [\"Ultraconserved elements have been identified in the following genomes of invertebrates: tunicates, diptera, worm and yeast.\"], \"exact_answer\": [[\"tunicates\"], [\"diptera\"], [\"worm\"], [\"yeast\"]], \"context\": \"Highly conserved elements discovered in vertebrates are present in non-syntenic loci of tunicatesTo trace the history of non-coding elements, which may represent candidate ancestral cis-regulatory modules affirmed during chordate evolution, we have searched for conserved elements in tunicate and vertebrate (Olfactores) genomes. We identified, for the first time, 183 non-coding sequences that are highly conserved between the two groupsIn a recent study that identified highly evolutionary conserved sequences in three genomes of Diptera species we described an ultraconserved element found at an internal exon-intron junction of the Drosophila melanogaster homothorax (hth) gene that appeared to be involved in the control of hth pre-mRNA splicingWe have explored the distributions of fully conserved ungapped blocks in genome-wide pair-wise alignments of recently completed species of Drosophila: D. melanogaster, D. yakuba, D. ananassae, D. pseudoobscura, D. virilis, and D. mojavensisParallel searches have been performed with multiple alignments of four insect species (three species of Drosophila and Anopheles gambiae), two species of Caenorhabditis, and seven species of Saccharomyces. Conserved elements were identified with a computer program called phastCons, which is based on a two-state phylogenetic hidden Markov model Ultraconserved elements in insect genomesHere, we used similar methods to identify ultraconserved genomic regions between the insect species Drosophila melanogaster and Drosophila pseudoobscura, as well as the more distantly related Anopheles gambiaeHere, we identified a wide range of ultraconserved elements common to distant species, from primitive aquatic organisms to terrestrial species with complicated body systems, including some novel elements conserved in fruit fly and human.RESULTS: Here, we identified a wide range of ultraconserved elements common to distant species, from primitive aquatic organisms to terrestrial species with complicated body systems, including some novel elements conserved in fruit fly and human.\"}, {\"question\": \"List two chemotherapeutic agents that are used for treatment of Subependymal Giant Cell Astrocytoma\", \"ideal_answer\": [\"Everolimus and rapamycin are chemotherapeutic agents that are used for treatment of Subependymal Giant Cell Astrocytoma.\"], \"exact_answer\": [[\"everolimus\"], [\"rapamycin\"]], \"context\": \"Everolimus has been approved by the FDA and the EMA for the treatment of advanced renal cell carcinoma (RCC), subependymal giant cell astrocytoma (SEGA) associated with tuberous sclerosis (TSC), pancreatic neuroendocrine tumors (PNET), in combination with exemestane in advanced hormone-receptor (HR)-positive, HER2-negative breast cancer.The effect of everolimus on renal angiomyolipoma in patients with tuberous sclerosis complex being treated for subependymal giant cell astrocytoma: subgroup results from the randomized, placebo-controlled, Phase 3 trial EXIST-1.Our objective was to evaluate everolimus, an mTOR inhibitor, in the treatment of angiomyolipoma in patients with subependymal giant cell astrocytoma (SEGA) associated with TSC. METHODS: EXamining everolimus In a Study of Tuberous Sclerosis Complex-1 (NCT00789828), a prospective, double-blind, randomized, placebo-controlled, Phase 3 study, examined everolimus in treating SEGA associated with TSC. Patients with serial SEGA growth from pre-baseline to baseline scans were randomly assigned (2:1) to receive 4.5 mg/m(2)/day everolimus (target blood trough: 5-15 ng/mL; n = 78) or placebo (n = 39). CONCLUSIONS: Everolimus showed efficacy in reducing angiomyolipoma lesion volume in patients with SEGA associated with TSC.Everolimus has demonstrated substantial clinical benefit in randomized, controlled, phase III studies leading to approval for the treatment of advanced renal cell carcinoma, advanced neuroendocrine tumors of pancreatic origin, renal angiomyolipoma and subependymal giant-cell astrocytoma associated with tuberous sclerosis complex, as well as advanced hormone-receptor-positive (HR(+)) and human epidermal growth factor receptor-2-negative advanced breast cancer. Everolimus for the treatment of subependymal giant cell astrocytoma probably causing seizure aggravation in a child with tuberous sclerosis complex: a case report.We are reporting on a 13.5-year-old girl with tuberous sclerosis complex (TSC) who was treated with everolimus because of giant cell astrocytoma and bilateral angiomyolipoma.Response of subependymal giant cell astrocytoma with spinal cord metastasis to everolimus.BACKGROUND: Brain subependymal giant cell astrocytomas (SEGAs) in patients with tuberous sclerosis have been reported to respond to everolimus. CONCLUSIONS: We describe a rare case of metastatic SEGA, which was successfully treated with everolimus.Additional oncology indications for everolimus include renal angiomyolipoma with tuberous sclerosis complex and subependymal giant-cell astrocytoma.Everolimus is a mammalian target of rapamycin (mTOR) inhibitor approved for the treatment of advanced renal cell carcinoma, pancreatic neuroendocrine tumors, subependymal giant cell astrocytoma associated with tuberous sclerosis complex, renal angiomyolipoma and tuberous sclerosis complex, and, in combination with exemestane, for hormone receptor-positive HER2-negative advanced breast cancer after failure of treatment with letrozole or anastrozole. Long-term effect of everolimus on epilepsy and growth in children under 3 years of age treated for subependymal giant cell astrocytoma associated with tuberous sclerosis complex.AIMS: To show the long-term safety data and the effect of everolimus treatment on epilepsy in children under the age of 3 who received everolimus for SEGAs associated with TSC. CONCLUSIONS: This study suggests that everolimus is effective and safe in infants and young children with epilepsy and SEGA associated with TSC and offers a valuable treatment option.Everolimus long-term safety and efficacy in subependymal giant cell astrocytoma.OBJECTIVE: To report long-term efficacy and safety data for everolimus for the treatment of subependymal giant cell astrocytoma (SEGA) in patients with tuberous sclerosis complex (TSC).CONCLUSION: Everolimus therapy is safe and effective for longer term (median exposure 34.2 months) treatment of patients with TSC with SEGA. CLASSIFICATION OF EVIDENCE: This study provides Class III evidence that everolimus, titrated to trough serum levels of 5-15 ng/mL, was effective in reducing tumor size in patients with SEGA secondary to TSC for a median of 34 months.Everolimus (RAD001): first systemic treatment for subependymal giant cell astrocytoma associated with tuberous sclerosis complex.In patients with subependymal giant cell astrocytomas (SEGAs) associated with tuberous sclerosis complex who are not candidates for surgery, single-agent everolimus has demonstrated the ability to significantly reduce SEGA volume with good tolerability. In the Phase III, randomized, placebo-controlled trial, everolimus was associated with a SEGA response rate of 35% compared with 0% in the placebo group. Efficacy and safety of everolimus for subependymal giant cell astrocytomas associated with tuberous sclerosis complex (EXIST-1): a multicentre, randomised, placebo-controlled phase 3 trial.In an open-label, phase 1/2 study, the mTOR inhibitor everolimus substantially and significantly reduced the volume of subependymal giant cell astrocytomas. We assessed the efficacy and safety of everolimus in patients with subependymal giant cell astrocytomas associated with tuberous sclerosis complex. INTERPRETATION: These results support the use of everolimus for subependymal giant cell astrocytomas associated with tuberous sclerosis.Everolimus for tumor recurrence after surgical resection for subependymal giant cell astrocytoma associated with tuberous sclerosis complex.A recent phase 1/2 study demonstrated that treatment with the mammalian target of rapamycin inhibitor everolimus reduced subependymal giant cell astrocytoma volume by 30% in 75% of the patients, all of whom were poor candidates for surgical resection. All 4 experienced over 50% initial reduction in the volume of their subependymal giant cell astrocytoma after 2 to 3 years of therapy with everolimus.METHODS: Recently, a phase I/II trial of everolimus demonstrated significant reductions in subependymal giant cell astrocytoma (SEGA) volume and decreased seizure frequency.TSC patients with SEGA received everolimus, titrated to tolerability to achieve target trough concentrations of 5-15 ng/mL.Everolimus: in patients with subependymal giant cell astrocytoma associated with tuberous sclerosis complex.Everolimus is an orally administered inhibitor of the mammalian target of rapamycin (mTOR). Everolimus (starting dosage 3.0\\u2009mg/m(2)) was associated with a significant reduction in the volume of the largest subependymal giant cell astrocytoma (SEGA) in 28 patients aged \\u22653 years with tuberous sclerosis complex (TSC) in a phase II trial (C2485). During the extension phase of this trial (median duration 34 months), the reduction in SEGA volume was maintained, with no everolimus recipient requiring surgery or other therapy for SEGA or hydrocephalus. Effective everolimus treatment of inoperable, life-threatening subependymal giant cell astrocytoma and intractable epilepsy in a patient with tuberous sclerosis complex.We present successful everolimus treatment of a huge subependymal giant cell astrocytoma in a 10-year old boy with tuberous sclerosis complex. Everolimus tablets for patients with subependymal giant cell astrocytoma.Recently, mTOR inhibitors such as everolimus have shown encouraging benefit for patients with SEGAs. The authors also examine the rationale for targeted agents against this pathway therapeutically and describe the clinical evidence underlying the FDA approval of everolimus for patients with inoperable SEGAs. EXPERT OPINION: Everolimus (Afinitor) selectively targets a molecular defect of SEGAs in TSC patients.Everolimus for subependymal giant-cell astrocytomas in tuberous sclerosis.An alternative may be the use of everolimus, which inhibits the mammalian target of rapamycin, a protein regulated by gene products involved in the tuberous sclerosis complex.Everolimus therapy was associated with a clinically meaningful reduction in volume of the primary subependymal giant-cell astrocytoma, as assessed on independent central review (P<0.001 for baseline vs. 6 months), with a reduction of at least 30% in 21 patients (75%) and at least 50% in 9 patients (32%). CONCLUSIONS: Everolimus therapy was associated with marked reduction in the volume of subependymal giant-cell astrocytomas and seizure frequency and may be a potential alternative to neurosurgical resection in some cases, though long-term studies are needed.The authors present a 21-year-old woman who has been receiving rapamycin for 5 months for bilateral subependymal giant cell astrocytomas.We present successful everolimus treatment of a huge subependymal giant cell astrocytoma in a 10-year old boy with tuberous sclerosis complex.A recent phase 1/2 study demonstrated that treatment with the mammalian target of rapamycin inhibitor everolimus reduced subependymal giant cell astrocytoma volume by 30% in 75% of the patients, all of whom were poor candidates for surgical resection.This review provides an overview of TSC, everolimus, and the clinical trials that led to its approval for the treatment of TSC-associated subependymal giant cell astrocytoma and renal angiomyolipoma.Everolimus in the treatment of subependymal giant cell astrocytomas, angiomyolipomas, and pulmonary and skin lesions associated with tuberous sclerosis complex.In patients with subependymal giant cell astrocytomas (SEGAs) associated with tuberous sclerosis complex who are not candidates for surgery, single-agent everolimus has demonstrated the ability to significantly reduce SEGA volume with good tolerabilityThe authors also examine the rationale for targeted agents against this pathway therapeutically and describe the clinical evidence underlying the FDA approval of everolimus for patients with inoperable SEGAs.\"}, {\"question\": \"List available databases containing information about conserved noncoding elements.\", \"ideal_answer\": [\"Ancora and TFCONES.\"], \"exact_answer\": [[\"Ancora\"], [\"TFCONES\"]], \"context\": \"Ancora: a web resource for exploring highly conserved noncoding elements and their association with developmental regulatory genes We describe Ancora http://ancora.genereg.net, a web resource that provides data and tools for exploring genomic organization of HCNEs for multiple genomes. Ancora includes a genome browser that shows HCNE locations and features novel HCNE density plots as a powerful tool to discover developmental regulatory genes and distinguish their regulatory elements and domainsTFCONES: a database of vertebrate transcription factor-encoding genes and their associated conserved noncoding elements We have created a database named TFCONES (Transcription Factor Genes & Associated COnserved Noncoding ElementS) (http://tfcones.fugu-sg.org) which contains all human, mouse and fugu TF-encoding genes and conserved noncoding elements (CNEs) associated with themThe conserved noncoding elements identified in TFCONES represent a catalog of highly prioritized putative cis-regulatory elements of TF-encoding genes and are candidates for functional assay.TFCONES: a database of vertebrate transcription factor-encoding genes and their associated conserved noncoding elements.We have created a database named TFCONES (Transcription Factor Genes & Associated COnserved Noncoding ElementS) (http://tfcones.We have created a database named TFCONES (Transcription Factor Genes & Associated COnserved Noncoding ElementS) (http://tfcones.fugu-sg.org) which contains all human, mouse and fugu TF-encoding genes and conserved noncoding elements (CNEs) associated with them\"}, {\"question\": \"What are the effects of ILK ablation?\", \"ideal_answer\": [\"Depending on the tissue or cell where ILK is ablated we see different effects:\\nAblation of ILK in heart results in dilated cardiomyopathy and spontaneous heart failure\\nAblation of ILK in fibroblasts leads to impaired healing due to a severe reduction in the number of myofibroblasts\\nAblation of ILK in osteoclasts inhibits bone resorption\\nAblation of ILK in liver results in enhanced and prolonged cell proliferation and hepatomegaly after phenobarbital administration and in abnormal histology\\nAblation of ILK in podocytes caused an aberrant distribution of nephrin and alpha-actinin-4\"], \"exact_answer\": [[\"Ablation of ILK in heart results in dilated cardiomyopathy and spontaneous heart failure\"], [\"Ablation of ILK in fibroblasts leads to impaired healing due to a severe reduction in the number of myofibroblasts\"], [\"Ablation of ILK in osteoclasts inhibits bone resorption\"], [\"Ablation of ILK in liver results in enhanced and prolonged cell proliferation and hepatomegaly after phenobarbital administration and in abnormal histology\"], [\"Ablation of ILK in podocytes caused an aberrant distribution of nephrin and alpha-actinin-4\"]], \"context\": \"ILK knockdown had no effect on the viability or survival pathway activity pattern of MM cells.We conclude that ILK does not play a prominent role in the promotion or sustenance of established MM.Defective granulation tissue formation in mice with specific ablation of integrin-linked kinase in fibroblastsHere, we show that fibroblast-restricted inactivation of ILK in mice leads to impaired healing due to a severe reduction in the number of myofibroblasts, whereas inflammatory infiltrate and vascularization of the granulation tissue are unaffected.in fibroblasts, ILK is crucial for limiting RhoA activity, thus promoting TGF\\u03b21 production, which is essential for dermal repair following injury.Our results show that ILK is important for the function, but not the differentiation, of osteoclasts.Osteoclast-specific inactivation of the integrin-linked kinase (ILK) inhibits bone resorption.ack of ILK in the hepatocytes imparts prolonged proliferative response not only to stimuli related to liver regeneration but also to xenobiotic chemical mitogens, such as PB.Liver-specific ablation of integrin-linked kinase in mice results in enhanced and prolonged cell proliferation and hepatomegaly after phenobarbital administration.Enhanced liver regeneration following changes induced by hepatocyte-specific genetic ablation of integrin-linked kinase.ILK deficiency caused an aberrant distribution of nephrin and alpha-actinin-4 in podocytes, whereas the localization of podocin and synaptopodin remained relatively intact.Targeted ablation of ILK from the murine heart results in dilated cardiomyopathy and spontaneous heart failure.Here we show that targeted ablation of the integrin-linked kinase (ILK) expression results in spontaneous cardiomyopathy and heart failure by 6 wk of age.Together, these results suggest that ILK plays a central role in protecting the mammalian heart against cardiomyopathy and failure.Our results show for the first time in vivo the significance of ILK and hepatic ECM-signaling for regulation of hepatocyte proliferation and differentiation.Liver-specific ablation of integrin-linked kinase in mice results in abnormal histology, enhanced cell proliferation, and hepatomegaly.\"}, {\"question\": \"Which receptors are bound by Tasimelteon?\", \"ideal_answer\": [\"Tasimelteon (HETLIOZ\\u2122) is an orally bioavailable agonist of the melatonin MT1 and MT2 receptors that has been approved in the US for the treatment of non-24-hour sleep-wake disorder.\"], \"exact_answer\": [[\"MT1\"], [\"MT2\"]], \"context\": \"We present here data on the in vitro binding affinity of tasimelteon for both human melatonin receptors MT1 and MT2, as well as the extended screen of other receptors and enzymes. Results indicate that tasimelteon is a potent Dual Melatonin Receptor Agonist (DMRA) with 2.1-4.4 times greater affinity for the MT2 receptor believed to mediate circadian rhythm phase-shifting (Ki = 0.0692 nM and Ki = 0.17 nM in NIH-3T3 and CHO-K1 cells, respectively), than for the MT1 receptor (Ki = 0.304 nM and Ki = 0.35 nM, respectively).The drugs covered target GABAA (zaleplon-CR, lorediplon, EVT-201), orexin (filorexant, MIN-202), histamine-H1 (LY2624803), serotonin 5-HT2A (ITI-007), melatonin/serotonin5-HT1A (piromelatine) and melatonin (indication expansions of prolonged-release melatonin and tasimelteon for pediatric sleep and circadian rhythm disorders) receptors.Several melatonin receptors agonists (ramelteon, prolonged-release melatonin, agomelatine and tasimelteon) have recently become available for the treatment of insomnia, depression and circadian rhythms sleep-wake disorders. Tasimelteon (HETLIOZ\\u2122) is an orally bioavailable agonist of the melatonin MT1 and MT2 receptors that has been approved in the US for the treatment of non-24-hour sleep-wake disorder. Numerous physiological functions of the pineal gland hormone melatonin are mediated via activation of two G-protein-coupled receptors, MT1 and MT2. The melatonergic drugs on the market, ramelteon and agomelatine, as well as the most advanced drug candidates under clinical evaluation, tasimelteon and TIK-301, are high-affinity nonselective MT1/MT2 agonists. A new melatonin 1 and melatonin 2 agonist tasimelteon improves sleep in these patients, resetting their circadian sleep-wake clocks.A general suitability in mitigating other symptoms of major depressive disorder cannot be deduced from the actions of tasimelteon via the melatonin receptors MT1 and MT2.Tasimelteon (HETLIOZ\\u2122) is an orally bioavailable agonist of the melatonin MT1 and MT2 receptors that has been approved in the US for the treatment of non-24-hour sleep-wake disorder.Ramelteon and tasimelteon are new chrono-hypnotic agents, selective for melatonin MT1 and MT2 receptors.Results indicate that tasimelteon is a potent Dual Melatonin Receptor Agonist (DMRA) with 2.1-4.4 times greater affinity for the MT2 receptor believed to mediate circadian rhythm phase-shifting (Ki = 0.0692 nM and Ki = 0.17 nM in NIH-3T3 and CHO-K1 cells, respectively), than for the MT1 receptor (Ki = 0.304 nM and Ki = 0.35 nM, respectively). A general suitability in mitigating other symptoms of major depressive disorder cannot be deduced from the actions of tasimelteon via the melatonin receptors MT1 and MT2. We present here data on the in vitro binding affinity of tasimelteon for both human melatonin receptors MT1 and MT2, as well as the extended screen of other receptors and enzymes.Results indicate that tasimelteon is a potent Dual Melatonin Receptor Agonist (DMRA) with 2.1-4.4 times greater affinity for the MT2 receptor believed to mediate circadian rhythm phase-shifting (Ki = 0.0692 nM and Ki = 0.17 nM in NIH-3T3 and CHO-K1 cells, respectively), than for the MT1 receptor (Ki = 0.304 nM and Ki = 0.35 nM, respectively).We present here data on the in vitro binding affinity of tasimelteon for both human melatonin receptors MT1 and MT2, as well as the extended screen of other receptors and enzymes. Results indicate that tasimelteon is a potent Dual Melatonin Receptor Agonist (DMRA) with 2.1-4.4 times greater affinity for the MT2 receptor believed to mediate circadian rhythm phase-shifting (Ki = 0.0692 nM and Ki = 0.17 nM in NIH-3T3 and CHO-K1 cells, respectively), than for the MT1 receptor (Ki = 0.304 nM and Ki = 0.35 nM, respectively).Results indicate that tasimelteon is a potent Dual Melatonin Receptor Agonist (DMRA) with 2.1-4.4 times greater affinity for the MT2 receptor believed to mediate circadian rhythm phase-shifting (Ki = 0.0692 nM and Ki = 0.17 nM in NIH-3T3 and CHO-K1 cells, respectively), than for the MT1 receptor (Ki = 0.304 nM and Ki = 0.35 nM, respectively). Tasimelteon was also shown to have no appreciable affinity for more than 160 other pharmacologically relevant receptors and several enzymes. Hetlioz(\\u00ae) (tasimelteon) is the first approved treatment in the United States for Non-24-Hour Sleep-Wake Disorder (Non-24). We present here data on the in vitro binding affinity of tasimelteon for both human melatonin receptors MT1 and MT2, as well as the extended screen of other receptors and enzymes.Results indicate that tasimelteon is a potent Dual Melatonin Receptor Agonist (DMRA) with 2.1-4.4 times greater affinity for the MT2 receptor believed to mediate circadian rhythm phase-shifting (Ki = 0.0692 nM and Ki = 0.17 nM in NIH-3T3 and CHO-K1 cells, respectively), than for the MT1 receptor (Ki = 0.304 nM and Ki = 0.35 nM, respectively).We present here data on the in vitro binding affinity of tasimelteon for both human melatonin receptors MT1 and MT2, as well as the extended screen of other receptors and enzymes.\"}, {\"question\": \"Which RNA polymerase is used for the replication of viroids?\", \"ideal_answer\": [\"DNA-dependent RNA polymerase II of plant origin transcribes viroid RNA into full-length copies\", \"DNA-dependent RNA polymerase II purified from healthy plant tissue is capable of synthesizing linear (-)-viroid RNA copies of full length from (+)-viroid RNA templates in vitro. The RNA genome of potato spindle tuber viroid (PSTV) is transcribed in vitro into complementary DNA and RNA by DNA-dependent DNA polymerase I and RNA polymerase, respectively, from Escherichia coli. Host DNA-dependent RNA polymerase II (RNAP II) was proposed to be critical for its replication, but no interaction site for RNAP II on the PSTVd RNA genome was identified. Whereas maximum total activity was observed in 1 mM Mn(2+) with a pronounced reduction (80%) in 5 mM Mn(2+), CEV synthesis was maintained in 1-15 mM Mn(2+).\", \"DNA-dependent RNA polymerase II purified from healthy plant tissue is capable of synthesizing linear (-)-viroid RNA copies of full length from (+)-viroid RNA templates in vitro. Research results support a role for RNA polymerase II in viroid replication and provide the first direct evidence of an association in vivo between host RNA polymerase II and CEV.\"], \"exact_answer\": [[\"DNA-dependent RNA polymerase II\"], [\"RNA polymerase II\"], [\"RNAP II\"]], \"context\": \"DNA-dependent RNA polymerase II of plant origin transcribes viroid RNA into full-length copiesDNA-dependent RNA polymerase II purified from healthy plant tissue is capable of synthesizing linear (-)-viroid RNA copies of full length from (+)-viroid RNA templates in vitro.Viroid RNA is accepted as a template for in vitro transcription by DNA-dependent DNA polymerase I and RNA polymerase from Escherichia coliThe RNA genome of potato spindle tuber viroid (PSTV) is transcribed in vitro into complementary DNA and RNA by DNA-dependent DNA polymerase I and RNA polymerase, respectively, from Escherichia coli.In vitro transcription of viroid RNA into full-length copies by RNA-dependent RNA polymerase from healthy tomato leaf tissueRNA-dependent RNA polymerase from healthy tomato plant tissue accepts potato spindle tuber viroid (PSTV) RNA as a template for the in vitro synthesis of full-length RNA copies of the PSTV genomeWhereas maximum total activity was observed in 1 mM Mn(2+) with a pronounced reduction (80%) in 5 mM Mn(2+), CEV synthesis was maintained in 1-15 mM Mn(2+). Inhibition of alpha-amanitin-sensitive CEV synthesis in 200 mM (NH(4))(2)SO(4) resembles the reaction of RNA polymerase II on a free nucleic acid templateCitrus exocortis viroid RNA is associated with the largest subunit of RNA polymerase II in tomato in vivo.The results support a role for RNA polymerase II in viroid replication and provide the first direct evidence of an association in vivo between host RNA polymerase II and CEV.Transcription of potato spindle tuber viroid by RNA polymerase II starts predominantly at two specific sitesPospiviroidae, with their main representative potato spindle tuber viroid (PSTVd), are replicated via a rolling circle mechanism by the host-encoded DNA-dependent RNA polymerase II (pol II)Transcription of potato spindle tuber viroid by RNA polymerase II starts in the left terminal loopViroids of the family Pospiviroidae, of which potato spindle tuber viroid (PSTVd) is the type strain, are replicated by the host's DNA-dependent RNA polymerase II in the nucleus.Tomato RNA polymerase II interacts with the rod-like conformation of the left terminal domain of the potato spindle tuber viroid positive RNA genomeHost DNA-dependent RNA polymerase II (RNAP II) was proposed to be critical for its replication, but no interaction site for RNAP II on the PSTVd RNA genome was identified. RNA polymerase II is implicated in the RNA-templated RNA synthesis during replication of viroids and Hepatitis Delta Virus (HDV); however, neither the RNA template nor protein factor requirements for this process are well defined.Viroids of the Pospiviroidae family, as represented by the Potato spindle tuber viroid (PSTVd), replicate in the nucleus by utilizing DNA-dependent RNA polymerase II.The results support a role for RNA polymerase II in viroid replication and provide the first direct evidence of an association in vivo between host RNA polymerase II and CEV.These results suggest that either RNA polymerase I or an unidentified RNA polymerase activity resistant to alpha-amanitin, acting on an RNA template, plays a role in the replication of ASBV, whereas for the rest of the viroids studied so far it appears that RNA polymerase II is involvedRNA polymerase II is implicated in the RNA-templated RNA synthesis during replication of viroids and Hepatitis Delta Virus (HDV); however, neither the RNA template nor protein factor requirements for this process are well definedViroids of the family Pospiviroidae, of which potato spindle tuber viroid (PSTVd) is the type strain, are replicated by the host&apos;s DNA-dependent RNA polymerase II in the nucleusViroids of the Pospiviroidae family, as represented by the Potato spindle tuber viroid (PSTVd), replicate in the nucleus by utilizing DNA-dependent RNA polymerase II. These results suggest that either RNA polymerase I or an unidentified RNA polymerase activity resistant to alpha-amanitin, acting on an RNA template, plays a role in the replication of ASBV, whereas for the rest of the viroids studied so far it appears that RNA polymerase II is involved. Inhibition studies with alpha-amanitin showed that the synthesis of ASBV-specific RNAs was not affected by concentrations of 1 and 200 micrograms/ml of the drug, which typically inhibit RNA polymerase II and III, respectively, from most animal and plant systems. These results suggest that either RNA polymerase I or an unidentified RNA polymerase activity resistant to alpha-amanitin, acting on an RNA template, plays a role in the replication of ASBV, whereas for the rest of the viroids studied so far it appears that RNA polymerase II is involved.Nucleoprotein complexes in the soluble fraction which bound to a monoclonal antibody to the carboxy terminal domain of the largest subunit of RNA polymerase II (8WG16) were affinity purified and contained plus- and minus-sense CEV RNA. The results support a role for RNA polymerase II in viroid replication and provide the first direct evidence of an association in vivo between host RNA polymerase II and CEV.\"}, {\"question\": \"List Parkin binding partners\", \"ideal_answer\": [\"HSP90\\nCDC37\\nGRP75\\nHSP60\\nLRPPRC\\nTUFM\\nPICK1\\nPSMA7\\nPael receptor\"], \"exact_answer\": [[\"HSP90\"], [\"CDC37\"], [\"GRP75\"], [\"HSP60\"], [\"LRPPRC\"], [\"TUFM\"], [\"PICK1\"], [\"PSMA7\", \"XAPC7\"], [\"Pael receptor\"]], \"context\": \"In addition to two known binding partners (HSP90, CDC37), 12 proteins were identified using the TAP assay; four of which are mitochondrially localized (GRP75, HSP60, LRPPRC, and TUFM).We find that parkin binds PICK1 via a PDZ-mediated interaction, which predominantly promotes PICK1 monoubiquitination rather than polyubiquitination. We have identified the 20S proteasomal subunit alpha4 (synonyms: PSMA7, XAPC7, subunit alpha type 7) as a new interacting partner of parkin.we cloned parkin-binding protein using a yeast two-hybrid system and identified a putative G protein-coupled receptor protein,which we named the Pael receptor (Pael-R), whose interactions with CASK are reviewed here, include the Parkinson's disease molecule parkin, \"}, {\"question\": \"What are the mobile applications fields of use for patients ?\", \"ideal_answer\": [\"Weight-loss mobile applications\\npediatric obesity prevention and treatment, healthy eating, and physical activity promotion\\nA total of 229 dermatology-related apps were identified in the following categories: general dermatology reference (61 [26.6%]), self-surveillance/diagnosis (41 [17.9%]), disease guide (39 [17.0%]), educational aid (20 [8.7%]), sunscreen/UV recommendation (19 [8.3%]), calculator (12 [5.2%]), teledermatology (8 [3.5%]), conference (6 [2.6%]), journal (6 [2.6%]), photograph storage/sharing (5 [2.2%]), dermoscopy (2 [0.9%]), pathology (2 [0.9%]), and other (8 [3.5%]). The most reviewed apps included Ultraviolet\\u2009~\\u2009UV Index (355 reviews), VisualDx (306), SPF (128), iSore (61), and SpotMole (50)\\nmobile health and fitness app\\nalcohol-use behavior change or recovery\\nMore than 17,000 mHealth apps now are available for smart phones and other devices, and they do everything from monitoring urine flow for patients with enlarged prostates to reminding people prone to kidney stones to drink more water.\"], \"exact_answer\": [[\"weight loss\"], [\"exercise\", \"physical activity\", \"fitness\"], [\"dermatology\"], [\"alcohol-use behavior change\"], [\"urine flow\"], [\"fluid intake\"]], \"context\": \"Evidence-based strategies in weight-loss mobile appsWeight-loss mobile applications (apps) have the potential to be a helpful toolMobile apps for pediatric obesity prevention and treatment, healthy eating, and physical activity promotionMobile applications (apps) offer a novel way to engage children in behavior change,Mobile applications in dermatologyA total of 229 dermatology-related apps were identified in the following categories: general dermatology reference (61 [26.6%]), self-surveillance/diagnosis (41 [17.9%]), disease guide (39 [17.0%]), educational aid (20 [8.7%]), sunscreen/UV recommendation (19 [8.3%]), calculator (12 [5.2%]), teledermatology (8 [3.5%]), conference (6 [2.6%]), journal (6 [2.6%]), photograph storage/sharing (5 [2.2%]), dermoscopy (2 [0.9%]), pathology (2 [0.9%]), and other (8 [3.5%]). The most reviewed apps included Ultraviolet\\u2009~\\u2009UV Index (355 reviews), VisualDx (306), SPF (128), iSore (61), and SpotMole (50). Twenty-seven mobile apps were identified and reviewed that involved general pharmacy practice, including apps that involved drug references, clinical references, medical calculators, laboratory references, news and continuing medical education, and productivity.Today, many high quality mobile apps are available for users and health professionals and cover the whole health care chain, i.e. information collection, prevention, diagnosis, treatment and monitoringOur team has developed a mobile health and fitness app called myFitnessCompanion\\u00ae to examine the current available mobile smartphone applications (e.g., apps) that utilize principles of ecological momentary assessment (EMA)-daily self-monitoring or near real-time self-assessment of alcohol-use behavior-to promote positive behavior change, alcohol harm reduction, psycho-education about alcohol use, or abstinence from alcohol.few apps addressed alcohol-use behavior change or recovery. Aside from tracking drinking consumption, a minority utilized empirically based components of alcohol treatment.More than 17,000 mHealth apps now are available for smart phones and other devices, and they do everything from monitoring urine flow for patients with enlarged prostates to reminding people prone to kidney stones to drink more water.\"}, {\"question\": \"Which are the genes responsible for Dyskeratosis Congenita?\", \"ideal_answer\": [\"To date, 8 genes have been associated with Dyskeratosis Congenita development. These are DKC1, TERC, TERT, NOP10, NHP2, TIN2, C16orf57, and TCAB1. Seven of these are important in telomere maintenance, because either they encode components of the telomerase enzyme complex (DKC1, TERC, TERT, NOP10, NHP2, and TCAB1) or the shelterin complex (TINF2).\", \"To date, CTC1, DKC1, TERC, TERT, TINF2, NHP2, NOP10, and WRAP53 are the genes in which mutations are known to cause DC and result in very short telomeres  Seven of these are important in telomere maintenance either because they encode components of the telomerase enzyme complex (DKC1, TERC, TERT, NOP10, NHP2, and TCAB1) or the shelterin complex (TINF2)  \"], \"exact_answer\": [[\"DKC1\"], [\"TERC\"], [\"TERT\"], [\"NOP10\"], [\"NHP2\"], [\"TIN2\"], [\"C16orf57\"], [\"TCAB1\"]], \"context\": \"Studies over the last 15 years have led to significant advances, with 8 DC genes (DKC1, TERC, TERT, NOP10, NHP2, TIN2, C16orf57, and TCAB1) having been characterizedSeven of these are important in telomere maintenance either because they encode components of the telomerase enzyme complex (DKC1, TERC, TERT, NOP10, NHP2, and TCAB1) or the shelterin complex (TINF2)Dyskeratosis congenita (DC) is a heterogeneous bone marrow failure syndrome with seven disease-causing genes identified to date, six of which are linked to telomere maintenance. Mutations in one of these genes (TINF2), which encodes a component of the shelterin complex, are associated with particularly short telomeresDyskeratosis congenita (DC) is an inheritable bone marrow failure syndrome characterized by reticulated hyperpigmentation, dystrophic nails and oral leukoplakiaFour genes, namely DKC1 (codes for dyskerin), TERC and TERT (code for telomerase) and NOP10, have been implicated in the pathogenesisTo date, CTC1, DKC1, TERC, TERT, TINF2, NHP2, NOP10, and WRAP53 are the genes in which mutations are known to cause DC and result in very short telomeresThe mode of inheritance of DC varies by gene: DKC1 (X-linked), TERC and TINF2 (autosomal dominant), TERT (autosomal dominant or autosomal recessive), CTC1, WRAP53, NHP2, and NOP10 (autosomal recessive)Six genes have been identified, defects in which cause different genetic subtypes (X-linked recessive, autosomal dominant, autosomal recessive) of DC. The products of these genes encode components that are critical for telomere maintenance; either because they are core constituents of telomerase (dyskerin, TERC, TERT, NOP10 and NHP2) or are part of the shelterin complex that protects the telomeric end (TIN2)BACKGROUND: Telomerase complex genes mutations (DKC1, TERC, TERT, and NOP10) lead to premature telomere shortening and are responsible for different forms of dyskeratosis congenita.Mutations in DKC1 gene encoding dyskerin are responsible for the X-linked dyskeratosis congenita.The DC1 gene responsible for the X linked form (MIM 305000) of dyskeratosis congenita has been mapped to Xq28.The DC1 gene responsible for the X linked form (MIM 305000) of dyskeratosis congenita has been mapped to Xq28Mutations in DKC1 gene encoding dyskerin are responsible for the X-linked dyskeratosis congenitaMutations in the DKC1 gene are responsible for causing X-linked recessive dyskeratosis congenita (DKC) and a more severe allelic variant of the disease, Hoyeraal-Hreidarsson syndromeTelomerase complex genes mutations (DKC1, TERC, TERT, and NOP10) lead to premature telomere shortening and are responsible for different forms of dyskeratosis congenitaLinkage analysis in multiplex families confirmed that the DKC1 gene, responsible for the X-linked form of DC, is located within Xq28 and facilitated its positional cloningDKC1 has been identified as the gene responsible for X-linked DC, and genetic analyses have been performed in a worldwide study\"}, {\"question\": \"Name five programs for transcript quantification from RNASeq experiments\", \"ideal_answer\": [\"Popular programs for transcript quantification from RNASeq experiments include: Cufflinks, RSEM, Flux Capacitor, Mitie, Miso, Tigar, Montebello, Drut, Traph, Pome, IsoformEx, Neuma,\"], \"exact_answer\": [[\"Cufflinks\"], [\"RSEM\"], [\"Flux Capacitor\"], [\"Mitie\"], [\"Miso\"], [\"Tigar\"], [\"Montebello\"], [\"Drut\"], [\"Traph\"], [\"Pome\"], [\"IsoformEx\"], [\"Neuma\"]], \"context\": \"Splicing Analysis Kit (SpankiORMAN ( O ptimal R esolution of M ultimapping A mbiguity of R N A-Seq Reads), which aims to compute the minimum number of potential transcript products for each gene and to assign each multimapping read to one of these transcripts based on the estimated distribution of the region covering the read.MITIE (Mixed Integer Transcript IdEntification) for simultaneous transcript reconstruction and quantification.Montebello, an integrated statistical approach which performs simultaneous isoform discovery and quantification by using a Monte Carlo simulation to find the most likely isoform compositionIGAR: transcript isoform abundance estimation method with gapped alignment of RNA-Seq dataa statistical method to estimate transcript isoform abundances from RNA-Seq data.RNA-Seq data with TopHat and CufflinksExperimental results on prediction accuracy show that our method is very competitive with popular tools such as Cufflinks and IsoLasso. Our tool, called TraphBy resolving the linear system with LASSO, our approach can infer an accurate set of dominantly expressed transcripts while existing methods tend to assign positive expression to every candidate isoformthe Minimum Unique Length Tool (MULTo), a framework for efficient and comprehensive representation of mappability information, through identification of the shortest possible length required for each genomic coordinate to become unique in the genome and transcriptome a pipeline for processing and analyzing RNA-Seq data, that we have named Grape (Grape RNA-Seq Analysis Pipeline Environment)Our method, referred as Discovery and Reconstruction of Unannotated Transcripts (DRUT), can be used to enhance existing transcriptome assemblers, such as Cufflinks, as well as to accurately estimate the transcript frequenciesDiffSpliceOmicsoft sequence aligner (OSA), a fast and accurate alignment tool for RNA-Seq data. two popular tools for isoform quantification, MISO and Cufflinks The method is implemented in the seqbias R/Bioconductor package, available freely under the LGPL licensePoisson mixed-effects (POME) model to characterize base-level read coverage within each transcriptSpliceTrap, a method to quantify exon inclusion levels using paired-end RNA-seq dataWe present RSEM, an user-friendly software package for quantifying gene and isoform abundances from single-end or paired-end RNA-Seq dataWe propose a novel algorithm (IsoformEx) that employs weighted non-negative least squares estimation method to estimate the expression levels of transcript isoforms.Rnnotator, an automated software pipeline that generates transcript models by de novo assembly of RNA-Seq data without the need for a reference genomeHere we introduce such algorithms in an open-source software program called Cufflinks.propose a novel, efficient and intuitive approach of estimating mRNA abundances from the whole transcriptome shotgun sequencing (RNA-Seq) data. Our method, NEUMA (Normalization by Expected Uniquely Mappable Area), is based on effective length normalization using uniquely mappable areas of gene and mRNA isoform models.\"}, {\"question\": \"List variants of the MC1R gene.\", \"ideal_answer\": [\"V60L\\nD84E\\nV92M\\nR151C\\nR160W\\nR163Q\\nD294H\"], \"exact_answer\": [[\"V60L\"], [\"D84E\"], [\"V92M\"], [\"R151C\"], [\"R160W\"], [\"R163Q\"], [\"D294H\"]], \"context\": \"All of the investigated variants showed positive associations with NMSC, with consistent significant results obtained for V60L, D84E, V92M, R151C, R160W, R163Q and D294H: SOR (95%CI) ranged from 1.42 (1.19-1.70) for V60L to 2.66 (1.06-6.65) for D84E variant.The melanocortin 1 receptor (MC1R) gene encodes for a seven-pass transmembrane receptor primarily expressed on melanocytes and melanoma cells.for V60L to 2.74 (1.53-4.89) for D84E.  We aim to examine the influence of the MC1R variants (RHC: D84E, R151C, R160W; NRHC: V60L, R163Q and the synonymous polymorphism T314T) on the MM risk in a population from the Canary Islands. \"}, {\"question\": \"Which phenotypes are associated with heterozygous mutations of the BSCL2 gene?\", \"ideal_answer\": [\"Heterozygous mutations in the Berardinelli-Seip congenital lipodystrophy (BSCL2) gene have been associated with different clinical phenotypes including Silver syndrome/spastic paraplegia 17, distal hereditary motor neuropathy type V, and Charcot-Marie-Tooth disease type 2 (CMT2) with predominant hand involvement.\", \"Heterozygous mutations in the Berardinelli-Seip congenital lipodystrophy (BSCL2) gene have been associated with different clinical phenotypes including Silver syndrome/spastic paraplegia 17, distal hereditary motor neuropathy type V, and Charcot-Marie-Tooth disease type 2 (CMT2) with predominant hand involvement\"], \"exact_answer\": [[\"Silver syndrome/spastic paraplegia 17\"], [\"Distal hereditary motor neuropathy type V\"], [\"Charcot-Marie-Tooth disease type 2 (CMT2) with predominant hand involvement\"]], \"context\": \"Heterozygous mutations in the Berardinelli-Seip congenital lipodystrophy (BSCL2) gene have been associated with different clinical phenotypes including Silver syndrome/spastic paraplegia 17, distal hereditary motor neuropathy type V, and Charcot-Marie-Tooth disease type 2 (CMT2) with predominant hand involvementDistal hereditary motor neuropathy type V (dHMN-V) and Charcot-Marie-Tooth syndrome (CMT) type 2 presenting with predominant hand involvement, also known as CMT2D and Silver syndrome (SS) are rare phenotypically overlapping diseases which can be caused by mutations in the Berardinelli-Seip Congenital Lipodystrophy 2 (BSCL2) and in the glycyl-tRNA synthetase encoding (GARS) genesHeterozygous mutations in the Berardinelli-Seip congenital lipodystrophy (BSCL2) gene have been associated with different clinical phenotypes including Silver syndrome/spastic paraplegia 17, distal hereditary motor neuropathy type V, and Charcot-Marie-Tooth disease type 2 (CMT2) with predominant hand involvement.In 2004, heterozygous mutations (N88S, S90L) in the Seipin/BSCL2 gene were identified in two autosomal dominant motor neuron diseases, distal hereditary motor neuropathy type V (OMIM #182960) and Silver syndrome (OMIM #270685).We recently found heterozygous mutations in the Berardinelli-Seip congenital lipodystrophy (BSCL2, seipin) gene causing SPG17 and distal hereditary motor neuropathy type V (distal HMN V).Distal hereditary motor neuropathy type V (dHMN-V) and Silver syndrome are rare phenotypically overlapping diseases which can be caused by mutations in the Berardinelli-Seip Congenital Lipodystrophy 2 (BSCL2) gene or Seipin.Heterozygous missense mutations in BSCL2 are associated with distal hereditary motor neuropathy and Silver syndrome.Heterozygous mutations in the Seipin/BSCL2 gene have recently been identified in two autosomal dominant motor neuron diseases, distal hereditary motor neuropathy type V and Silver's syndrome.We recently found heterozygous mutations in the Berardinelli-Seip congenital lipodystrophy (BSCL2, seipin) gene causing SPG17 and distal hereditary motor neuropathy type V (distal HMN V)Heterozygous missense mutations in BSCL2 are associated with distal hereditary motor neuropathy and Silver syndromeIn 2004, heterozygous mutations (N88S, S90L) in the Seipin/BSCL2 gene were identified in two autosomal dominant motor neuron diseases, distal hereditary motor neuropathy type V (OMIM #182960) and Silver syndrome (OMIM #270685)Heterozygous mutations in the Berardinelli-Seip congenital lipodystrophy (BSCL2) gene have been associated with different clinical phenotypes including Silver syndrome/spastic paraplegia 17, distal hereditary motor neuropathy type V, and Charcot-Marie-Tooth disease type 2 (CMT2) with predominant hand involvement. Heterozygous mutations in the Berardinelli-Seip congenital lipodystrophy (BSCL2) gene have been associated with different clinical phenotypes including Silver syndrome/spastic paraplegia 17,We recently found heterozygous mutations in the Berardinelli-Seip congenital lipodystrophy (BSCL2, seipin) gene causing SPG17 and distal hereditary motor neuropathy type V (distal HMN V). Here we report the clinical features of two families with heterozygous BSCL2 mutations.Heterozygous mutations in the Berardinelli-Seip congenital lipodystrophy (BSCL2) gene have been associated with different clinical phenotypes including Silver syndrome/spastic paraplegia 17, distal hereditary motor neuropathy type V, and Charcot-Marie-Tooth disease type 2 (CMT2) with predominant hand involvement.Heterozygous missense mutations in BSCL2 are associated with distal hereditary motor neuropathy and Silver syndrome.We recently found heterozygous mutations in the Berardinelli-Seip congenital lipodystrophy (BSCL2, seipin) gene causing SPG17 and distal hereditary motor neuropathy type V (distal HMN V).In 2004, heterozygous mutations (N88S, S90L) in the Seipin/BSCL2 gene were identified in two autosomal dominant motor neuron diseases, distal hereditary motor neuropathy type V (OMIM #182960) and Silver syndrome (OMIM #270685).Heterozygous mutations in the Seipin/BSCL2 gene have recently been identified in two autosomal dominant motor neuron diseases, distal hereditary motor neuropathy type V and Silver's syndrome.\"}, {\"question\": \"The CXCR2 receptor is targeted in cancer. Name five antagonists.\", \"ideal_answer\": [\"There are numerous CXCR2 receptor antagonists, such as SB225002, G31P, SCH-527123, AZ10397767, SCH-479833.\"], \"exact_answer\": [[\"SB225002\"], [\"G31P\"], [\"SCH-527123\"], [\"AZ10397767\"], [\"SCH-479833\"]], \"context\": \"CXCR2 small-molecule antagonist (SB225002)G31P, an antagonist against CXC chemokine receptors 1 and 2, inhibits growth of human prostate cancer cells in nude mice.The CXCR2 antagonist, SCH-527123, shows antitumor activityretreatment with the CXCR2 antagonist AZ10397767The aim of this study was to investigate whether the CXCR2 antagonist, SCH-527123, inhibits colorectal cancer proliferation and if it can sensitize colorectal cancer cells to oxaliplatin both in vitro and in vivo.we report that targeting CXCR2 and CXCR1 activity using orally active small molecule antagonist (SCH-527123, SCH-479833) inhibits human colon cancer liver metastasis mediated by decreased neovascularization and enhanced malignant cell apoptosis.CXCR2-specific small molecule inhibitor (AZ10397767) to investigate the recruitment and function of human neutrophils in tumors\"}, {\"question\": \"Which diseases are caused by mutations in Calsequestrin 2 (CASQ2) gene?\", \"ideal_answer\": [\"CASQ2 mutations are associated with autosomal recessive catecholaminergic polymorphic ventricular tachycardia (CPVT) and familial hypertrophic cardiomyopathy.\", \"Mutations in the gene encoding for cardiac calsequestrin, CASQ2, cause a rare but severe form of catecholaminergic polymorphic ventricular tachycardia (CPVT).\\nThere is also a publication that links mutations in CASQ2 gene to the disease of hypertrophic cardiomyopathy (HCM).\"], \"exact_answer\": [[\"catecholaminergic polymorphic ventricular tachycardia (CPVT)\", \"CPVT\", \"catecholaminergic polymorphic ventricular tachycardia\"], [\"familial hypertrophic cardiomyopathy\", \"HCM\", \"hypertrophic cardiomyopathy\"]], \"context\": \"Recessively inherited CPVT is caused by either missense or null-allele mutations in the cardiac calsequestrin (CASQ2) gene. It was suggested that defects in CASQ2 cause protein deficiency and impair Ca(2+) uptake to the sarcoplasmic reticulum and Ca(2+)-dependent inhibition of ryanodine channels, leading to diastolic Ca(2+) leak, after-depolarizations, and arrhythmia. The autosomal recessive form of CPVT is caused by mutations in the CASQ2 gene. In a consanguineous family, a novel homozygous CASQ2 mutation (p.L77P) was identified in a child with CPVT who required implantation of a cardioverter defibrillator due to episodes of syncope while on medical therapy. Catecholaminergic polymorphic ventricular tachycardia is a familial cardiac arrhythmia that is related to RYR2 or CASQ2 gene mutation.Thirteen mutations in the CASQ2 gene have been reported so far in association with CPVT. These data increased significantly the number of CASQ2 mutations described in association with CPVT, revealed the high prevalence of splicing and truncating mutations in this gene and brought new insight regarding the dominant inheritance of the disease. Moreover, our report of the first splicing abnormalities in CASQ2 caused by intronic mutation or synonymous change underlines the absolute necessity to perform extensive molecular analysis for genetic diagnosis and counseling of CPVT.Mutations in the human cardiac calsequestrin gene (CASQ2) are linked to catecholaminergic polymorphic ventricular tachycardia (CPVT-2).Catecholaminergic polymorphic ventricular tachycardia (CPVT) is a familial cardiac arrhythmia that is related to RYR2 or CASQ2 gene mutation. Catecholaminergic polymorphic ventricular tachycardia (CPVT) is an uncommon heritable disease presenting with syncope or sudden cardiac death. Two genes involved in calcium homeostasis, the ryanodine receptor gene and the calsequestrin 2 (CASQ2) gene, have been implicated in this disease. We describe a young man presenting with exercise-induced syncope, clinically diagnosed as CPVT. Genetic analysis revealed two mutations, p.Y55C (c.164A>G) and p.P308L (c.923C>T), in the CASQ2 gene. A novel heterozygous mutation, F189L, in CASQ2 gene was identified in one family with CPVT.Catecholamine-induced polymorphic ventricular tachycardia (CPVT) is a familial disorder caused by cardiac ryanodine receptor type 2 (RyR2) or calsequestrin 2 (CASQ2) gene mutations. We conclude that CPVT-causing CASQ2 missense mutations function as null alleles. The central role of RyR2 dysfunction in CASQ2 deficiency unifies the pathophysiologic mechanism underlying CPVT due to RyR2 or CASQ2 mutations and suggests a therapeutic approach for these inherited cardiac arrhythmias.Mutations of the RyR2 gene cause autosomal dominant CPVT, while mutations of the CASQ2 gene may cause an autosomal recessive or dominant form of CPVT. Mutations of two myocardial calcium signaling molecules, ryanodine receptor 2 (RYR2) and calsequestrin 2 (CASQ2), may cause catecholaminergic polymorphic ventricular tachycardia (CPVT), a severe inherited arrhythmic disease manifesting with salvoes of exercise-induced bidirectional and polymorphic tachycardias. These data, combined with our previous findings, show that RYR2 mutations are present in at least 6/16 (38%) of the catecholaminergic polymorphic ventricular tachycardia families, while CASQ2 mutations must be a rare cause of CPVT.This report reviews evidence that a missense mutation in the CASQ2 gene is associated with autosomal-recessive CPVT.An Australian cohort of 252 unrelated familial hypertrophic cardiomyopathy patients were screened for mutations in the Ca(2+) regulatory genes, sorcin (SRI), calstabin (FKBP1B), calsequestrin (CASQ2), phospholamban (PLN), sarcolipin (SLN), calreticulin (CALR3) and calmodulin (CALM). A total of 17 exonic DNA variants were identified in the 7 Ca(2+) regulatory genes studied, of which 4 were considered of pathogenic significance. Two novel mutations in the CALR3 gene were identified (Lys82Arg, Arg73Gln) and one truncation mutation in the PLN gene (Leu39Ter). A variant was also identified in the CASQ2 gene (Asp63Glu). These four variants were all novel, resulted in changes in conserved amino acids and were not identified in a normal population.\"}, {\"question\": \"In which diseases have electronic patient diaries been applied ?\", \"ideal_answer\": [\"Parkinson's disease\\nCOPD\\nFood hypersensitivity\\nNiacin induced flushing\\nHemophilia\\nHeartburn\\nHeadache\"], \"exact_answer\": [[\"Parkinson's disease\"], [\"COPD\", \"Chronic  obstructive pulmonary disease\", \"Chronic Obstructive Airways Disease\", \"Chronic Obstructive Lung Disease\"], [\"Food hypersensitivity\", \"Allergy\"], [\"Niacin induced flushing\"], [\"Hemophilia\"], [\"Heartburn\", \"Reflux\", \"GERD\", \"Esophageal reflux\"], [\"Headache\", \"Migraine\", \"migraine headache\", \"Migraine Disorder\", \"Migraine Headaches\"]], \"context\": \"WiiPD is an approach for the objective home based assessment of Parkinson's disease which utilizes the intuitive and sensor rich Nintendo Wii Remote. Combined with an electronic patient diary, a suite of mini-games, a metric analyzer, and a visualization engine, we propose that this system can complement existing clinical practice by providing objective metrics gathered frequently over extended periods of time.This research paper examines the challenges in the development and adoption of an electronic patient diary within the Pathways Home for Respiratory Illness ProjectThis project supported community-based patients suffering from chronic obstructive pulmonary disease (COPD) to achieve increased levels of self-management and self-efficacy using electronic-monitoring techniques and mentoring by community health nurses.Instead of measuring physiological parameters, a Smartphone based Personal Allergy Assistant (PAA) allows patients to keep an electronic patient diary by scanning the barcode of the consumed food products.Telemedicine assisted diet and diagnosis management in food hypersensitivityThe Flushing ASsessment Tool (FAST) was developed to assess flushing symptoms and their impact on patients receiving niacin therapy. This was a prospective, randomized, double-blind, placebo-controlled, parallel-group 8-week study conducted to evaluate the psychometric characteristics of the FAST. The instrument is administered daily using an electronic patient diary.Haemoassist--a hand-held electronic patient diary for haemophilia home care.A comparison of self-documentation in diabetics: electronic versus paper diaries.Subjects with self-perceived heartburn without known gastrointestinal disease or interfering treatments were selected with questionnaires. The study was performed unsupervised, whenever heartburn required medication. An electronic patient diary gave instructions when to take study medicationSelf-medication of a single headache episode with ketoprofen, ibuprofen or placebo, home-monitored with an electronic patient diary.\"}, {\"question\": \"Which are the Atg8 homologs in human?\", \"ideal_answer\": [\"Autophagy (Autophagy-related protein 8 or Atg8p or APG8 or AUT7 or CVT5) is a yeast protein involved in cytoplasm to vacuole transport (Cvt) vesicles and autophagosomes formation. In yeast it is represented by a single gene, the ATG8 family in humans contains 6 members (microtubule-associated protein-1 light chain 3A (MAP1LC3A), MAP1LC3B, MAP1LC3C, GABA(A) receptor-associated protein (GABARAP), GABARAPL1, and GABARAPL2/GATE-16).\"], \"exact_answer\": [[\"MAP1LC3A\", \"microtubule-associated protein-1 light chain 3A\", \"LC3A\"], [\"MAP1LC3B\", \"microtubule-associated protein-1 light chain 3B\", \"LC3B\"], [\"MAP1LC3C\", \"microtubule-associated protein-1 light chain 3C\", \"LC3C\"], [\"GABARAP\", \"GABA(A) receptor-associated protein\", \"Gamma-aminobutyric acid receptor-associated protein\", \"MM46\"], [\"GABARAPL1\", \"Gamma-aminobutyric acid receptor-associated protein-like 1\", \"Early estrogen-regulated protein\", \"GABA(A) receptor-associated protein-like 1\", \"Glandular epithelial cell protein 1\", \"GEC-1\"], [\"GABARAPL2\", \"Gamma-aminobutyric acid receptor-associated protein-like 2\", \"GABA(A) receptor-associated protein-like 2\", \"Ganglioside expression factor 2\", \"GEF-2\", \"General protein transport factor p16\", \"Golgi-associated ATPase enhancer of 16 kDa\", \"GATE-16\", \"MAP1 light chain 3-related protein\"]], \"context\": \"Strikingly, in addition to ULK1 and ULK2, ATG13 and FIP200 interacted with human ATG8 proteins, all with strong preference for the GABARAP subfamily.GABARAPL1 belongs to the small family of GABARAP proteins (including GABARAP, GABARAPL1 and GABARAPL2/GATE-16), one of the two subfamilies of the yeast Atg8 orthologue.Identification of the Atg8 family interacting motif (AIM) in Stbd1 required for interaction with GABARAPL1Stbd1 has been reported to interact with a known autophagy protein, GABARAPL1, a member of the Atg8 family.Atg8 is a yeast protein involved in the autophagic process and in particular in the elongation of autophagosomes. In mammals, several orthologs have been identified and are classed into two subfamilies: the LC3 subfamily and the GABARAP subfamily, referred to simply as the LC3 or GABARAP families. GABARAPL1 (GABARAP-like protein 1), one of the proteins belonging to the GABARAP (GABA(A) receptor-associated protein) familyThe proteins that make up the GABARAP family demonstrate conservation of their amino acid sequences and protein structures. In humans, GABARAPL1 shares 86% identity with GABARAP and 61% with GABARAPL2 (GATE-16).The selectivity is mediated by autophagy receptors, such as p62 and NBR1, which can bind to autophagic effector proteins (Atg8 in yeast, MAP1LC3 protein family in mammals) anchored in the membrane of autophagosomes.ATG3 is the E2-like enzyme necessary for ATG8/LC3 lipidation during autophagy.Recently, autophagy receptors, like p62/SQSTM1 and NBR1, which physically link autophagic cargo to ATG8/MAP1-LC3/GABARAP family members located on the forming autophagic membranes, have been identified.At least eight different Atg8 orthologs belonging to two subfamilies (LC3 and GATE-16/GABARAP) occur in mammalian cells, but their individual roles and modes of action are largely unknown.truncated DeltaN63 Atg4D displays increased activity against the Atg8 paralogue, gamma-aminobutyric acid receptor-associated protein-like 1 (GABARAP-L1)Structure of GATE-16, membrane transport modulator and mammalian ortholog of autophagocytosis factor Aut7p.The yeast ortholog of GATE-16 is the autophagocytosis factor Aut7p. GATE-16 is also closely related to the GABA receptor-associated protein (GABARAP),Three human Atg8 (hAtg8) homologs, LC3, GABARAP, and GATE-16, have been characterized as modifiers in reactions mediated by hAtg7 (an E1-like enzyme) and hAtg3 (an E2-like enzyme) as in yeast Atg8 lipidation, but their final targets have not been identified.\"}, {\"question\": \"List all clinical trials of the polypill.\", \"ideal_answer\": [\"'Use of a Multidrug Pill In Reducing cardiovascular Events' (UMPIRE) trial, European Clinical Trials database, as EudraCT: 2009-016278-34 and the Clinical Trials Registry, India as CTRI/2010/091/000250.\\n'IMProving Adherence using Combination Therapy (IMPACT)', Australian New Zealand Clinical Trial Registry (ACTRN12606000067572).\\n'Kanyini Guidelines Adherence with the Polypill (Kanyini-GAP)'\\nPhase II study of the Polycap, double-blind, randomised trial, registered with ClinicalTrials.gov, number NCT00443794\\nSecond Indian Polycap Study, TIPS-2\\nCluster Randomized Usual Care vs Caduet Investigation Assessing Long-term-risk (CRUCIAL trial)\\nGEMINI trial, 14-week, open-label trial conducted in 1220 patients from the USA\\nGEMINI-Australia, Asia, Latin America, Africa/Middle East (AALA) study \\nJEWEL study program, with JEWEL 1 conducted among 1138 patients from the UK and Canada and JEWEL 2 conducted in 1107 patients from Europe\\nCAPABLE54, the Clinical Utility of Caduet in Simultaneously Achieving Blood Pressure and Lipid End Points , in the USA\\nCUSP (The Caduet\\u00ae in an Untreated Subject Population trial)\\nTOGETHER trial\\nA randomised controlled trial in seven countries \\u2013 Australia, Brazil, India, Netherlands , New Zealand , United Kingdom  and United States.   Australian New Zealand Clinical Trials Registry (ACTRN 12607000099426)\", \"A number of clinical trials evaluating polypill, mainly in cardiovascular patients, have been performed and include the Use of a Multidrug Pill In Reducing cardiovascular Events (UMPIRE) trial, Cluster Randomized Usual Care vs Caduet Investigation Assessing Long-term-risk (CRUCIAL trial), Atorvastatin and Amlodipine in Patients with Elevated Lipids and Hypertension (AVALON) trial, Anglo-Scandinavian Cardiac Outcomes Trial (ASCOT), GEMINI trial, GEMINI-Australia, Asia, Latin America, Africa/Middle East (GEMINI-AALA) study, JEWEL 1 trial, JEWEL 2 trial, Clinical Utility of Caduet in Simultaneously Achieving Blood Pressure and Lipid End Points (CAPABLE54), The Caduet\\u00ae in an Untreated Subject Population trial (CUSP), TOGETHER trial, TIPS trial, TIPS-2 trial and IMProving Adherence using Combination Therapy (IMPACT).\"], \"exact_answer\": [[\"Use of a Multidrug Pill In Reducing cardiovascular Events' (UMPIRE) trial, European Clinical Trials database\\\\, as EudraCT: 2009-016278-34 and the Clinical Trials Registry\\\\, India as CTRI/2010/091/000250.\", \"UMPIRE\"], [\"IMProving Adherence using Combination Therapy (IMPACT)', Australian New Zealand Clinical Trial Registry (ACTRN12606000067572)\", \"IMPACT\"], [\"Kanyini Guidelines Adherence with the Polypill (Kanyini-GAP)\"], [\"Phase II study of the Polycap, double-blind\\\\, randomised trial\\\\, registered with ClinicalTrials.gov\\\\, number NCT00443794\", \"ndian Polycap Study, TIPS\", \"TIPS\"], [\"Cluster Randomized Usual Care vs Caduet Investigation Assessing Long-term-risk (CRUCIAL trial)\", \"CRUCIAL\"], [\"GEMINI trial, 14-week\\\\, open-label trial conducted in 1220 patients from the USA\", \"GEMINI\"], [\"GEMINI-Australia, Asia\\\\, Latin America\\\\, Africa/Middle East (AALA) study\", \"GEMINI-AALA\"], [\"JEWEL 1 conducted among 1138 patients from the UK and Canada\", \"JEWEL 1\"], [\"JEWEL 2 conducted in 1107 patients from Europe\", \"JEWEL 2\"], [\"CAPABLE54, the Clinical Utility of Caduet in Simultaneously Achieving Blood Pressure and Lipid End Points\\\\, in the USA\", \"CAPABLE54\"], [\"CUSP (The Caduet\\u00ae in an Untreated Subject Population trial)\", \"CUSP\"], [\"TOGETHER trial\", \"TOGETHER\"], [\"A randomised controlled trial in seven countries \\u2013 Australia, Brazil\\\\, India\\\\, Netherlands\\\\, New Zealand\\\\, United Kingdom and United States\\\\, Australian New Zealand Clinical Trials Registry (ACTRN 12607000099426)\"], [\"Second Indian Polycap Study, TIPS-2\", \"TIPS-2\"], [\" Atorvastatin and Amlodipine in Patients with Elevated Lipids and Hypertension (AVALON) trial\", \"AVALON\"], [\"Anglo-Scandinavian Cardiac Outcomes Trial (ASCOT)\", \"ASCOT\"]], \"context\": \"The 'Use of a Multidrug Pill In Reducing cardiovascular Events' (UMPIRE) trial assesses whether a polypill strategy (by combining aspirin, a statin and two blood pressure lowering agents) would improve adherence to guideline-indicated therapies and would lower both blood pressure and cholesterol, in people with established cardiovascular disease. UMPIRE, running in India and three European countries (England, Ireland and the Netherlands), is an open, randomised, controlled trial designed to include 1000 participants in India and 1000 in Europe, with a followup of 12-24 months.UMPIRE is registered with the European Clinical Trials database, as EudraCT: 2009-016278-34 and the Clinical Trials Registry, India as CTRI/2010/091/000250. The trial was part of the 'Single Pill Against Cardiovascular Events (SPACE)' collaboration, which encompasses the 'IMProving Adherence using Combination Therapy (IMPACT)' and 'Kanyini Guidelines Adherence with the Polypill (Kanyini-GAP)' trials.IMProving Adherence using Combination Therapy (IMPACT) is an open-label randomised controlled trial comparing a once-daily polypill containing four preventive medications with usual careThe trial is registered with the Australian New Zealand Clinical Trial Registry (ACTRN12606000067572).Effects of a polypill (Polycap) on risk factors in middle-aged individuals without cardiovascular disease (TIPS): a phase II, double-blind, randomised trial.This study is registered with ClinicalTrials.gov, number NCT00443794\"}, {\"question\": \"Which are the mammalian orthologs of Drosophila Yki?\", \"ideal_answer\": [\"There are two mammalian orthologs of Yki: YAP and TAZ\"], \"exact_answer\": [[\"YAP\"], [\"TAZ\"]], \"context\": \"Yorkie ortholog YAPYorkie ortholog, Yap1human ortholog of Yorkie, YAPYAP1, the ortholog of Drosophila YorkieYorkie ortholog YAP Yorkie homolog YAPYki (YAP/TAZ in vertebrates\"}, {\"question\": \"Which myosin isozymes are located within the pericuticular necklace of the hair cell?\", \"ideal_answer\": [\"The hair cell is located in the inner ear, a tissue that is particularly reliant on actin-rich structures and unconventional myosin isozymes. Within the pericuticular necklace, a domain of the hair cell, certain unconventional myosin isozymes are located, namely myosins-Ibeta, myosin-VI, and myosin-VIIa.\"], \"exact_answer\": [[\"myosins-Ibeta\"], [\"myosin-VI\"], [\"myosin-VIIa\"]], \"context\": \"Myosin 1 beta, VI, VIIa and probably XV are all expressed within a single cell in the inner ear, the hair cell.The myosin isozymes expressed in the hair cell all have unique domains of expression and in some areas, such as the pericuticular necklace, several domains overlap.Substantial amounts of myosins-Ibeta, -VI, and -VIIa are located in a pericuticular necklacefour unconventional myosin isozymes in the inner ear, a tissue that is particularly reliant on actin-rich structures and unconventional myosin isozymes.The myosin isozymes expressed in the hair cell all have unique domains of expression and in some areas, such as the pericuticular necklace, several domains overlap.Substantial amounts of myosins-Ibeta, -VI, and -VIIa are located in a pericuticular necklace that is largely free of F-actin, squeezed between (but not associated with) actin of the cuticular plate and the circumferential belt.To understand how cells differentially use the dozens of myosin isozymes present in each genome, we examined the distribution of four unconventional myosin isozymes in the inner ear, a tissue that is particularly reliant on actin-rich structures and unconventional myosin isozymes.This study examined the changes of myosin VI and myosin VIIa, two unconventional myosins that are critical for normal hair cell formation and function, during hair cell death and regeneration.Substantial amounts of myosins-Ibeta, -VI, and -VIIa are located in a pericuticular necklace that is largely free of F-actin, squeezed between (but not associated with) actin of the cuticular plate and the circumferential belt.Substantial amounts of myosins-Ibeta, -VI, and -VIIa are located in a pericuticular necklace that is largely free of F-actin, squeezed between (but not associated with) actin of the cuticular plate and the circumferential beltThe myosin isozymes expressed in the hair cell all have unique domains of expression and in some areas, such as the pericuticular necklace, several domains overlapTo understand how cells differentially use the dozens of myosin isozymes present in each genome, we examined the distribution of four unconventional myosin isozymes in the inner ear, a tissue that is particularly reliant on actin-rich structures and unconventional myosin isozymes\"}, {\"question\": \"Which proteins cause cytoplasmic sequestration of NF-kB?\", \"ideal_answer\": [\"In unstimulated cells, NF-kB transcription factors are retained in the cytoplasm with the inhibitory activity of I-kBs, Sef, NF-kB1 (p105) and NF-kB2 (p100).\"], \"exact_answer\": [[\"I-kBs\"], [\"Sef\"], [\"NF-kB1 (p105)\"], [\"NF-kB2 (p100)\"]], \"context\": \"Sef is an inhibitor of proinflammatory cytokine signaling, acting by cytoplasmic sequestration of NF-\\u03baBAccording to the classical model, NF-\\u03baB is retained in the cytoplasm of resting cells via binding to inhibitory, I\\u03baB proteins and translocates into the nucleus upon their ligand-induced degradation.Like I\\u03baBs, Sef sequesters NF-\\u03baB in the cytoplasm of resting cells.The activity of NF-\\u03baB is tightly controlled through its cytoplasmic sequestration by specific inhibitors, I\\u03baBs.The inhibitor of NF-kappaB (IkappaB) family of proteins is believed to regulate NF-kappaB activity by cytoplasmic sequestration.IkappaBalpha is an inhibitory molecule that sequesters NF-kappaB dimers in the cytoplasm of unstimulated cells.p105.Ikappa Bgamma and prototypical Ikappa Bs use a similar mechanism to bind but a different mechanism to regulate the subcellular localization of NF-kappa BWe show that the death domain of p105 (also of IkappaBgamma) is essential for the cytoplasmic sequestration of NF-kappaB by p105 and IkappaBgamma.In unstimulated cells, NF-kappaB transcription factors are retained in the cytoplasm by inhibitory IkappaB proteins.The ability of the IkappaB alpha protein to sequester dimeric NF-kappaB/Rel proteins in the cytoplasm provides an effective mechanism for regulating the potent transcriptional activation properties of NF-kappaB/Rel family members.The presence of a discrete nuclear import sequence in IkappaB alpha suggests that cytoplasmic sequestration of the NF-kappaB/Rel-IkappaB alpha complex is a consequence of the mutual masking of the NLS within NF-kappaB/Rel proteins and the import sequence within IkappaB alpha.The viral Tax protein, which is encoded by human T-cell leukaemia virus HTLV-I, activates nuclear translocation of the NF-kappa B/Rel transcription factors and relieves cytoplasmic sequestration of RelA and Rel by heterodimerization with NF-kappa B1/p1O5 (refs 1,2).The I-kappa B protein, which is necessary for the cytoplasmic sequestration of the NF-kappa B transcription factor complex, was identified specifically in regions of limbic, hypothalamic, and autonomic nuclei.Human T-cell leukemia virus type I Tax-protein-mediated activation of NF-kappa B from p100 (NF-kappa B2)-inhibited cytoplasmic reservoirsThe preexisting NF-kappa B proteins are retained in the cytoplasm of cells by association with inhibitory ankyrin-motif-containing I kappa B proteins, primarily I kappa B-alpha but also including the precursor proteins p105 (NF-kappa B1) and p100 (NF-kappa B2).\"}, {\"question\": \"Tumor-treating fields are effective for treatment of which cancers?\", \"ideal_answer\": [\"Clinical trials have shown that Tumor-treating fields are effective for treatment of non-small cell lung cancer and glioblastoma. Ongoing and future trials will evaluate TTFields in solid tumor brain metastases, and ovarian, pancreatic cancers and multidrug resistance cancer cells.\"], \"exact_answer\": [[\"non-small cell lung cancer\"], [\"glioblastoma\"]], \"context\": \"Alternating electric fields (tumor-treating fields therapy) can improve chemotherapy treatment efficacy in non-small cell lung cancer both in vitro and in vivo.e evaluated the effects of combining TTFields with standard chemotherapeutic agents on several NSCLC cell lines, both in vitro and in vivo. Frequency titration curves demonstrated that the inhibitory effects of TTFields were maximal at 150 kHz for all NSCLC cell lines tested, and that the addition of TTFields to chemotherapy resulted in enhanced treatment efficacy across all cell lines. Together, these findings suggest that combining TTFields therapy with chemotherapy may provide an additive efficacy benefit in the management of NSCLC.Response patterns of recurrent glioblastomas treated with tumor-treating fields.NovoTTF Therapy is a novel and US Food and Drug Administration (FDA)-approved antimitotic treatment for recurrent GBM with potential benefits compared with other options. Recurrent GBM patients from two prior trials with demonstrated radiologic tumor response to single-agent NovoTTF Therapy were analyzed to better characterize tumor response patterns and evaluate the associations between response, compliance, and OS. In addition, a compartmental tumor growth model was developed and evaluated for its ability to predict GBM response to tumor-treating fields (TTFields). The overall response rate across both trials was 15% (4% complete responses): 14% in the phase III trial (14/120) and 20% (2/10) in a pilot study. Tumor responses to NovoTTF Therapy developed slowly (median time to response, 5.2 months) but were durable (median duration, 12.9 months).NovoTTF Therapy is a novel antimitotic treatment for recurrent GBM associated with slowly developing but durable tumor responses in approximately 15% of patients.The effect of field strength on glioblastoma multiforme response in patients treated with the NovoTTF\\u2122-100A system. An ongoing trial is assessing its efficacy for newly diagnosed glioblastoma multiforme (GBM) and it has been FDA-approved for recurrent GBM.We present three patients with GBM in whom the fields were adjusted at recurrence and the effects of each adjustment.The first patient underwent subtotal resection, radiotherapy with temozolomide (TMZ), and then began NovoTTF Therapy with metronomic TMZ.  A second patient underwent two resections followed by radiotherapy/TMZ and NovoTTF Therapy/TMZ. Six months later, two new distal lesions were noted, and he underwent further resection with adjustment of his fields. He remained stable over the subsequent year on NovoTTF Therapy and bevacizumab. A third patient on NovoTTF Therapy/TMZ remained stable for two years but developed a small, slow growing enhancing lesion, which was resected, and his fields were adjusted accordingly. A phase I/II trial of Tumor Treating Fields (TTFields) therapy in combination with pemetrexed for advanced non-small cell lung cancer.Promising preclinical data have led to a single arm phase I/II trial in NSCLC patients.METHODS: Forty-two inoperable stage IIIB (with pleural effusion) and IV NSCLC patients who had had tumor progression received pemetrexed 500 mg/m(2) iv q3w together with daily TTFields therapy until disease progression.The median time to in-field progression was 28 weeks and the median time to systemic progression was 22 weeks.CONCLUSIONS: The combination of TTFields and pemetrexed as a second line therapy for NSCLC is safe and potentially more effective than pemetrexed alone. TTFields improved disease control within the treatment field and a phase III study is planned to further investigate its role as a novel treatment in NSCLC.Tumor treating fields (TTFields) is a noninvasive, regional antimitotic treatment modality that has been approved for the treatment of recurrent glioblastoma by the U.S. FDA and has a CE mark in Europe. The antimitotic effect of TTFields therapy has been demonstrated in multiple cell lines when the appropriate frequency was utilized. A phase III trial of TTFields monotherapy compared to active chemotherapy in recurrent glioblastoma patients established that TTFields therapy is associated with minimal toxicity, better quality of life, and comparable efficacy to chemotherapy. Ongoing and future trials will evaluate TTFields in newly diagnosed glioblastoma, solid tumor brain metastases, nonsmall cell lung cancer, and ovarian and pancreatic cancers.The U.S. Food and Drug Administration has approved the first device, the NovoTTF-100A\\u2122, that uses this technology and is indicated for use in progressive glioblastoma multiforme after standard therapies have failed. Promising clinical trial results will likely lead to expanded uses in primary brain tumors and other cancer types. Long-term survival of patients suffering from glioblastoma multiforme treated with tumor-treating fields.Tumor-treating fields (TTField) therapy is a novel treatment technique that has recently received CE and FDA approval for the treatment of RGBM, and is based on the principle that low intensity, intermediate frequency electric fields (100 to 300 kHz) may induce apoptosis in specific cell types. Our center was the first to apply TTField treatment to histologically proven GBM in a small pilot study of 20 individuals in 2004 and 2005, and four of those original 20 patients are still alive today. We report two cases of GBM and two cases of RGBM treated by TTField therapy, all in good health and no longer receiving any treatment more than seven years after initiating TTField therapy, with no clinical or radiological evidence of recurrence.Tumor treating fields (TTFields) are low intensity (1 ? 2 V/cm), intermediate frequency (100 ? 200 kHz) alternating electric fields administered using insulated electrodes placed on the skin surrounding the region of a malignant tumor. Furthermore, it summarizes the clinical experience with TTFields, mainly in two indications: one in recurrent glioblastoma multiforme: in a large prospective randomized Phase III trial TTFields was compared with best standard care (including chemotherapy): TTFields significantly improved median overall survival (OS) compared with standard therapy (7.8 vs 6.1 months) for the patients treated per protocol.The second indication was a Phase II study in second-line non-small cell lung cancer, where TTFields was administered concomitantly with pemetrexed. EXPERT OPINION: The proof of concept of TTFields has been well demonstrated in the preclinical setting, and the clinical data seem promising in various tumor types. CONCLUSIONS: The results indicate that TTFields alone and in combination with paclitaxel and doxorubicin effectively reduce the viability of both wild type and MDR cell sub-lines and thus can potentially be used as an effective treatment of drug resistant tumors.Alternating electric fields (TTFields) inhibit metastatic spread of solid tumors to the lungs.Tumor treating fields (TTFields) are low intensity, intermediate frequency, alternating electric fields used to treat cancerous tumors. This novel treatment modality effectively inhibits the growth of solid tumors in vivo and has shown promise in pilot clinical trials in patients with advanced stage solid tumors.BACKGROUND: The present study explores the efficacy and toxicity of combining a new, non-toxic, cancer treatment modality, termed Tumor Treating Fields (TTFields), with chemotherapeutic treatment in-vitro, in-vivo and in a pilot clinical trial.In addition, we studied the effects of combining chemotherapy with TTFields in an animal tumor model and in a pilot clinical trial in recurrent and newly diagnosed GBM patients.RESULTS: The efficacy of TTFields-chemotherapy combination in-vitro was found to be additive with a tendency towards synergism for all drugs and cell lines tested (combination indexA pilot study with very low-intensity, intermediate-frequency electric fields in patients with locally advanced and/or metastatic solid tumors.PATIENTS AND METHODS: This open, prospective pilot study was designed to evaluate the safety, tolerability, and efficacy profile of TTFields treatment in patients with locally advanced and/or metastatic solid tumors using the NovoTTF100A(TM) device.Outcomes showed 1 partial response of a treated skin metastasis from a primary breast cancer, 3 cases where tumor growth was arrested during treatment, and 1 case of disease progression. One mesothelioma patient experienced lesion regression near TTFields with simultaneous tumor stability or progression in distal areas.CONCLUSION: Although the number of patients in this study is small, the lack of therapy toxicity and the efficacy observed in data gathered to date indicate the potential of TTFields as a new treatment modality for solid tumors, definitely warranting further investigation.Mitotic disruption and reduced clonogenicity of pancreatic cancer cells in vitro and in vivo by tumor treating fields.OBJECTIVES: Tumor Treating Fields (TTFields) are a non-invasive cancer treatment modality approved for the treatment of patients with recurrent glioblastoma. The present study determined the efficacy and mechanism of action of TTFields in preclinical models of pancreatic cancer.RESULTS: Application of TTFields in vitro showed a significant decrease in cell count, an increase in cell volume and reduced clonogenicity. Further analysis demonstrated significant increase in the number of abnormal mitotic figures, as well as a decrease in G2-M cell population. In hamsters with orthotopic pancreatic tumors, TTFields significantly reduced tumor volume accompanied by an increase in the frequency of abnormal mitotic events. TTFields efficacy was enhanced both in vitro and in vivo when combined with chemotherapy.CONCLUSIONS: These results provide the first evidence that TTFields serve as an effective antimitotic treatment in preclinical pancreatic cancer models and have a long term negative effect on cancer cell survival. These results make TTFields an attractive candidate for testing in the treatment of patients with pancreatic cancer.Tumor Treating Fields (TTFields) are a non-invasive cancer treatment modality approved for the treatment of patients with recurrent glioblastoma.In addition, a compartmental tumor growth model was developed and evaluated for its ability to predict GBM response to tumor-treating fields (TTFields).Tumor-treating fields (TTField) therapy is a novel treatment technique that has recently received CE and FDA approval for the treatment of RGBM, and is based on the principle that low intensity, intermediate frequency electric fields (100 to 300 kHz) may induce apoptosis in specific cell types.Tumor treating fields (TTFields) is a noninvasive, regional antimitotic treatment modality that has been approved for the treatment of recurrent glioblastoma by the U.S. FDA and has a CE mark in Europe.Tumor-treating fields (TTField) therapy is a novel treatment technique that has recently received CE and FDA approval for the treatment of RGBM, and is based on the principle that low intensity, intermediate frequency electric fields (100 to 300 kHz) may induce apoptosis in specific cell types. In addition, a compartmental tumor growth model was developed and evaluated for its ability to predict GBM response to tumor-treating fields (TTFields). OBJECTIVES: Tumor Treating Fields (TTFields) are a non-invasive cancer treatment modality approved for the treatment of patients with recurrent glioblastoma. Tumor Treating Fields (TTFields) are a non-invasive cancer treatment modality approved for the treatment of patients with recurrent glioblastoma.\"}, {\"question\": \"List angiocrine factors\", \"ideal_answer\": [\"Angiocrine factors are: Ccl4, neurotensin, vascular endothelial growth factor, metalloproteinases-1, thrombospondin 3, Slit2, hepatocyte growth factor, Wnt2. \"], \"exact_answer\": [[\"Ccl4\"], [\"neurotensin\"], [\"vascular endothelial growth factor\"], [\"metalloproteinases-1\"], [\"thrombospondin 3\"], [\"Slit2\"], [\"hepatocyte growth factor\"], [\"Wnt2\"]], \"context\": \"Ccl4 and neurotensin (Nts) (angiocrine factors)vascular endothelial growth factor (VEGF)-A was identified as the most abundantly expressed factor, angiocrine factors tissue inhibitor of metalloproteinases-1 (Timp-1) and thrombospondin 3 (THBS3) We found that Slit2, which is negatively regulated by endothelial EphA2 receptor, is one such tumor suppressive angiocrine factor. angiocrine factors, including hepatocyte growth factor (HGF) and Wnt2.\"}, {\"question\": \"List the components of a Replisome Progression Complex (RPC).\", \"ideal_answer\": [\"RPC components include the essential initiation and elongation factor, Cdc45, the checkpoint mediator Mrc1, the Tof1-Csm3 complex that allows replication forks to pause at protein-DNA barriers, the histone chaperone FACT (facilitates chromatin transcription) and Ctf4, which helps to establish sister chromatid cohesion. RPCs also interact with Mcm10 and topoisomerase I.\", \"RPCs also interact with Mcm10 and topoisomerase I. Others have found recently that the Mrc1 subunit of RPCs binds DNA polymerase epsilon, which synthesises the leading strand at DNA replication forks. Here, we show that the RPC associates with DNA polymerase alpha that primes each Okazaki fragment during lagging strand synthesis. During initiation, GINS is essential for a specific subset of RPC proteins to interact with MCM. GINS is also important for the normal progression of DNA replication forks, and we show that it is required after initiation to maintain the association between MCM and Cdc45 within RPCs. This interaction requires the RPC components Mrc1 and Ctf4, both of which associate with a tetratricopeptide repeat (TPR) domain located at the amino terminus of Dia2. RPC components include the essential initiation and elongation factor, Cdc45, the checkpoint mediator Mrc1, the Tof1-Csm3 complex that allows replication forks to pause at protein-DNA barriers, the histone chaperone FACT (facilitates chromatin transcription) and Ctf4, which helps to establish sister chromatid cohesion. \"], \"exact_answer\": [[\"Cdc45\"], [\"Mrc1\"], [\"Tof1-Csm3 complex\"], [\"FACT\"], [\"Ctf4\"]], \"context\": \"This interaction requires the RPC components Mrc1 and Ctf4, both of which associate with a tetratricopeptide repeat (TPR) domain located at the amino terminus of Dia2Here we show that SCF(Dia2) associates with the replisome progression complex (RPC) that assembles around the MCM2-7 helicase at DNA replication forksWe found previously that many regulatory proteins assemble around the MCM2-7 helicase at yeast replication forks to form the replisome progression complex (RPC), which might link MCM2-7 to other replisome componentsHere, we show that the RPC associates with DNA polymerase alpha that primes each Okazaki fragment during lagging strand synthesis. Our data indicate that a complex of the GINS and Ctf4 components of the RPC is crucial to couple MCM2-7 to DNA polymerase alphaOthers have found recently that the Mrc1 subunit of RPCs binds DNA polymerase epsilon, which synthesises the leading strand at DNA replication forkswe show that the GINS (go ichi ni san) complex allows the MCM (minichromosome maintenance) helicase to interact with key regulatory proteins in large replisome progression complexes (RPCs) that are assembled during initiation and disassembled at the end of S phaseRPC components include the essential initiation and elongation factor, Cdc45, the checkpoint mediator Mrc1, the Tof1-Csm3 complex that allows replication forks to pause at protein-DNA barriers, the histone chaperone FACT (facilitates chromatin transcription) and Ctf4, which helps to establish sister chromatid cohesion. RPCs also interact with Mcm10 and topoisomerase IDuring initiation, GINS is essential for a specific subset of RPC proteins to interact with MCM. GINS is also important for the normal progression of DNA replication forks, and we show that it is required after initiation to maintain the association between MCM and Cdc45 within RPCsWe found previously that many regulatory proteins assemble around the MCM2-7 helicase at yeast replication forks to form the replisome progression complex (RPC), which might link MCM2-7 to other replisome components.Here we show that SCF(Dia2) associates with the replisome progression complex (RPC) that assembles around the MCM2-7 helicase at DNA replication forks [6].Compared with wild type, hydroxyurea-treated ctf18\\u0436\\u0434 cells exhibited increased chromatin association of replisome progression complex components including Cdc45, Ctf4, and GINS complex subunits, the polymerase processivity clamp PCNA and the single-stranded DNA-binding complex RPA.The amino-terminal TPR domain of Dia2 tethers SCF(Dia2) to the replisome progression complex.Ctf4 is a protein conserved in eukaryotes and a constituent of the replisome progression complex.Eukaryotic GINS also links with other key proteins at the fork to maintain an active replisome progression complex.Dia2 has previously been implicated in the control of replication and genome stability via its interaction with the replisome progression complex.The Cdc45-MCM-GINS complex could constitute the core of a larger macromolecular structure that has been termed the \\\"replisome progression complex\\\".FACT does not associate with the Mcm2-7 helicase at replication origins during G1 phase but is subsequently incorporated into the replisome progression complex independently of histone binding and uniquely among histone chaperones.Our data indicate that a complex of the GINS and Ctf4 components of the RPC is crucial to couple MCM2-7 to DNA polymerase alpha.Replisome progression complex links DNA replication to sister chromatid cohesion in Xenopus egg extracts.On the basis of the physical interactions between AND-1 and DNA polymerases, we discuss a model to describe how replisome progression complex establishes sister chromatid cohesion.The components of the replisome that preserve genomic stability by controlling the progression of eukaryotic DNA replication forks are poorly understood.RPC components include the essential initiation and elongation factor, Cdc45, the checkpoint mediator Mrc1, the Tof1-Csm3 complex that allows replication forks to pause at protein-DNA barriers, the histone chaperone FACT (facilitates chromatin transcription) and Ctf4, which helps to establish sister chromatid cohesion.\"}, {\"question\": \"Which CDK targets control cytokinesis?\", \"ideal_answer\": [\"Aip1, Ede1 and Inn1 are CDK targets whose dephosphorylation is required for cytokinesis.\"], \"exact_answer\": [[\"Aip1\"], [\"Ede1\"], [\"Inn1\"]], \"context\": \"We use phosphoproteome analysis of mitotic exit to identify Cdk targets that are dephosphorylated at the time of cytokinesis. We then apply a new and widely applicable tool to generate conditionally phosphorylated proteins to identify those whose dephosphorylation is required for cytokinesis. This approach identifies Aip1, Ede1 and Inn1 as cytokinetic regulators. Our results suggest that cytokinesis is coordinately controlled by the master cell cycle regulator Cdk together with its counteracting phosphatase and that it is executed by concerted dephosphorylation of Cdk targets involved in several cell biological processes.\"}, {\"question\": \"In what type(s) of plant organelles we can detect prolamellar bodies?\", \"ideal_answer\": [\"Prolamellar body (PLB) is a highly organized lipid structure, which is the main site of accumulation of the ternary light-harvesting POR complex LHPP (light-harvesting NADPH:protochlorophyllide oxidoreductase:protochlorophyllide). Prolamellar bodies have been discovered in etioplasts with the use of thin section electron microscopy. Etioplasts develop in the place of chloroplasts in the dark. During skotomorphogenesis in angiosperms, NADPH:protochlorophyllide oxidoreductase (POR) forms the photolabile NADPH-POR-protochlorophyllide (Pchlide) ternary complexes. Prolamellar bodies (PLBs) efficiently capture the light energy for photo conversion in etioplasts. Upon illumination, the etioplasts transformed into regular chloroplasts. PLBs are formed not only in etioplasts but also in chloroplasts in young developing leaves during the night.\"], \"exact_answer\": [[\"etioplasts (mainly)\"], [\"chloroplasts (in young developing leaves during the night)\"]], \"context\": \"CPP1 depletion also causes reduced POR accumulation in etioplasts of dark-grown plants and as a result impairs the formation of prolamellar bodies, which subsequently affects chloroplast biogenesis upon illumination.Prolamellar bodies (PLBs) isolated from etiolated wheat seedlings were studied with the use of atomic force microscopy (AFM), transmission electron microscopy (TEM) and fluorescence spectroscopyTEM analyses confirmed that PLBs of wheat leaf etioplasts also had an average diameter of appr. 1\\u03bcmetioplasts with prolamellar bodies (PLBs) and photoactive, 655nm emitting Pchlide form accumulated primarily in the basal leaf regions after 2 weeks of regeneration. When these latter leaves were illuminated with continuous light for 3 days, the etioplasts transformed into regular chloroplastsPOR complex assembly (including LHPP and POR dimers) did not affect the formation of prolamellar bodies (PLBs) that function for efficient capture of light energy for photo conversion in etioplasts.Etioplasts that develop in the place of chloroplasts in the dark contain a highly organized lipid structure termed prolamellar body (PLB), which is the main site of accumulation of the ternary Pchlide:POR:NADPH complexes.During skotomorphogenesis in angiosperms, NADPH:protochlorophyllide oxidoreductase (POR) forms an aggregate of photolabile NADPH-POR-protochlorophyllide (Pchlide) ternary complexes localized to the prolamellar bodies within etioplasts.The content of the supramolecular light-harvesting POR complex LHPP (light-harvesting NADPH:protochlorophyllide oxidoreductase:protochlorophyllide) and the density of prolamellar bodies in etioplasts are decreased in the mutant.The Arabidopsis porB-1 porC-1 mutant displays a severe xantha (highly chlorophyll-deficient) phenotype characterized by smaller prolamellar bodies in etioplastsThe most notable contributions of thin section electron microscopy include the elucidation of the 3-D organization of thylakoid membranes, the discovery of prolamellar bodies in etioplastsThis resulted in the recovery of thylakoid membrane stacking in chloroplasts in the light, and the formation of prolamellar bodies and plastoglobuli in etioplasts in the dark.The prolamellar body (PLB) proteome of dark-grown wheat leaves was characterized. PLBs are formed not only in etioplasts but also in chloroplasts in young developing leaves during the nightA fraction of highly purified prolamellar bodies was isolated from etioplasts of wheat (Triticum aestivum L.The highest specific activity of the enzyme in etiolated leaf tissue was confirmed to be in the fraction of prolamellar bodies.This work reveals a new view on the formation of prolamellar bodies and provides new clues about the function of POR in the etioplast-chloroplast transition.The Arabidopsis porB-1 porC-1 mutant displays a severe xantha (highly chlorophyll-deficient) phenotype characterized by smaller prolamellar bodies in etioplasts and decreased thylakoid stacking in chloroplasts.Transmission electron microscopy revealed partially developed, agranal plastids in the dark-grown mutant, unlike wild-type seedlings that contain etioplasts with prolamellar bodies.Organization of protochlorophyllide oxidoreductase in prolamellar bodies isolated from etiolated carotenoid-deficient wheat leaves as revealed by fluorescence probes.In fully etiolated leaves lamellae are absent and the prolamellar bodies become quite large, presumably because of inhibition of the fusion step which appears to require chlorophyll.POR is a peripheral membrane protein that accumulates to high levels in the prolamellar bodies of vascular plant etioplasts and is present at low levels in the thylakoid membranes of developing and mature plastids.This work reveals a new view on the formation of prolamellar bodies and provides new clues about the function of POR in the etioplast-chloroplast transition.The results presented in this paper are not consistent with the hypothesis that the molar ratio of MGDG/DGDG is responsible for the tubular structure of prolamellar bodies in etioplasts.\"}, {\"question\": \"What do mepolizumab and reslizumab have in common?\", \"ideal_answer\": [\"Mepolizumab and reslizumab are monoclonal antibodies that target and neutralize interleukin 5. They have been shown to reduce eosinophil counts and they are used for the treatment of refractory asthma (associated with eosiniphilia) and other eosinophilic diseases.\"], \"exact_answer\": [[\"monoclonal antibodies\"], [\"treatment for refractory asthma\"], [\"treatment for eosinophilic disease\"], [\"neutralize interleukin 5\"], [\"reduce eosinophil counts\"]], \"context\": \"The most promising agents are targeted against cytokines of Th2 pattern and related receptors, such as IL-2 (daclizumab) and IL-13 (lebrikizumab) or IL-5 in patients with hypereosinophilia (mepolizumab, reslizumab and benralizumab).Patients with severe asthma or COPD have often a suboptimal symptom control due to inadequate treatment. For example, patients with the hypereosinophilic syndrome have received mepolizumab, an anti-IL-5 monoclonal antibody, for as long as 6 years and have not developed any characteristic set of adverse events. Safety data for reslizumab, another anti-IL-5 monoclonal antibody, and benralizumab, a monoclonal antibody to the IL-5 receptor \\u03b1-chain, are comparatively limited, especially for benralizumab, although reports of administration of these antibodies to humans suggest that they are well tolerated.IL-5 antagonist therapies in current development include two monoclonal anti-IL-5 antibodies (mepolizumab, reslizumab), a monoclonal antibody directed at the IL-5 receptor (benralizumab), and anti-sense oligonucleotide therapy (TPI ASM8).In patients with severe, refractory asthma associated with eosinophilia, however, clinical trials have demonstrated significant reductions in asthma exacerbations. Clinical studies in other disorders, particularly eosinophilic esophagitis and hypereosinophilic syndrome, have also shown significant improvements in blood and/or tissue eosinophilia and variable alterations in clinical disease activity. Strategies aimed at the inhibition of IL-5 may hold great promise in the treatment of eosinophilic diseases.Two monoclonal antibodies have been designed to neutralize IL-5 (mepolizumab and reslizumab). Both antibodies have demonstrated the ability to reduce blood and tissue eosinophil counts. Two monoclonal antibodies have been designed to neutralize IL-5 (mepolizumab and reslizumab).To date, two humanized monoclonal antibodies, mepolizumab and reslizumab, have been developed that bind to human IL-5.Monoclonal antibodies directed towards interleukin-5, such as mepolizumab or reslizumab, were shown to be very effective at reducing blood and airways eosinophilia.IL-5 antagonist therapies in current development include two monoclonal anti-IL-5 antibodies (mepolizumab, reslizumab), a monoclonal antibody directed at the IL-5 receptor (benralizumab), and anti-sense oligonucleotide therapy (TPI ASM8).The most promising agents are targeted against cytokines of Th2 pattern and related receptors, such as IL-2 (daclizumab) and IL-13 (lebrikizumab) or IL-5 in patients with hypereosinophilia (mepolizumab, reslizumab and benralizumab).Asthma is a chronic inflammatory disease that often features eosinophilia, especially in its most severe forms. Monoclonal antibodies directed towards interleukin-5, such as mepolizumab or reslizumab, were shown to be very effective at reducing blood and airways eosinophilia. \"}, {\"question\": \"Which genes are associated with autosomal dominant Charcot-Marie-Tooth?\", \"ideal_answer\": [\"The genes associated with the X-linked and the autosomal dominant forms of Charcot-Marie-Tooth disease are GJB1, MPZ, INF2, DNM2, YARS, GNB4, NEFL, MFN2, LRSAM1, GDAP1, PMP22, LITAF, and EGR2. Identification of these genes has not only been important for patients and families, but also provided new information about disease pathogenesis.\"], \"exact_answer\": [[\"GJB1\"], [\"MPZ\"], [\"INF2\"], [\"DNM2\"], [\"YARS\"], [\"GNB4\"], [\"NEFL\"], [\"MFN2\"], [\"LRSAM1\"], [\"GDAP1\"], [\"PMP22\"], [\"LITAF\"], [\"EGR2\"]], \"context\": \" GJB1 is currently considered to be associated with X-linked DI-CMT, and MPZ, INF2, DNM2, YARS, GNB4, NEFL, and MFN2 are associated with autosomal DI-CMT. Moreover, GDAP1, KARS, and PLEKHG5 are associated with RI-CMT. Identification of these genes is not only important for patients and families but also provides new information about pathogenesisA novel mutation in LRSAM1 causes axonal Charcot-Marie-Tooth disease with dominant inheritanceCharcot-Marie-Tooth disease (CMT) refers to a heterogeneous group of genetic motor and sensory neuropathies. According to the primary site of damage, a distinction is made between demyelinating and axonal forms (CMT1 and 2, respectively, when inherited as an autosomal dominant trait). Leucine-rich repeat and sterile alpha motif-containing protein 1 (LRSAM1) is a ubiquitin-protein ligase with a role in sorting internalised cell-surface receptor proteins. So far, mutations in the LRSAM1 gene have been shown to cause axonal CMT in three different families and can confer either dominant or recessive transmission of the diseaseWe have identified a novel mutation in LRSAM1 in a small family with dominant axonal CMTPhenotypical features of the p.R120W mutation in the GDAP1 gene causing autosomal dominant Charcot-Marie-Tooth diseaseMutations in the ganglioside-induced-differentiation-associated protein 1 gene (GDAP1) can cause Charcot-Marie-Tooth (CMT) disease with demyelinating (CMT4A) or axonal forms (CMT2K and ARCMT2K). Most of these mutations present a recessive inheritance, but few autosomal dominant GDAP1 mutations have also been reported Our findings highlight the relevance of dominantly transmitted p.R120W GDAP1 gene mutations which can cause an axonal CMT with a wide clinical profileA novel GDAP1 Q218E mutation in autosomal dominant Charcot-Marie-Tooth diseaseThere have been very few reports of GDAP1 mutations in autosomal dominant (AD) CMTHere, we report an AD CMT family with a novel Q218E mutation in the GDAP1 geneCMT type 1 (CMT1; MIM 118200) is a group of autosomal dominant-inherited demyelinating neuropathies with a disease onset at or after childhood. Five different subtypes have been identified based on different causative genes. Among them, CMT1A (MIM #118220) is most common and is usually associated with a duplication of a 1.5-Mb region on chromosome 17p11.2, which includes peripheral myelin protein 22 gene (PMP22; MIM *601097)Mutations in the NF-L gene (NEFL) cause autosomal dominant neuropathies that are classified either as axonal Charcot-Marie-Tooth (CMT) type 2E (CMT2E) or demyelinating CMT type 1F (CMT1F)Recently, mutations affecting different domains of dynamin-2 (DNM2) were associated alternatively with autosomal dominant centronuclear myopathy or dominant intermediate (demyelinating and axonal) Charcot-Marie-Tooth disease (CMT) type BFour genes (PMP22, MPZ, LITAF, and EGR2) have been described in the last 15 yr associated with AD CMTI and a further gene (NEFL), originally described as causing AD CMT2 can also cause AD CMT1 (by neurophysiological criteria)Mutations in the pleckstrin homology domain of dynamin 2 cause dominant intermediate Charcot-Marie-Tooth diseaseWe refined the locus associated with DI-CMTB on chromosome 19p12-13.2 to 4.2 Mb in three unrelated families with CMT originating from Australia, Belgium and North America. After screening candidate genes, we identified unique mutations in dynamin 2 (DNM2) in all familiesHere, we report a large family showing characteristic phenotypes of Charcot-Marie-Tooth type 1A along with deafness in an autosomal dominant fashion. We detected a sequence variation (c.68C>G) co-segregating with the disease phenotype and leading to a T23R missense mutation in PMP22The authors recently mapped an autosomal dominant demyelinating form of CMT type 1 (CMT1C) to chromosome 16p13.1-p12.3The authors identified missense mutations (G112S, T115N, W116G) in the LITAFgene (lipopolysaccharide-induced tumor necrosis factor-alpha factor) in three CMT1C pedigreesWe identified a novel neurofilament-light missense mutation (C64T) that causes the disease in a large Slovenian CMT2 family. This novel mutation shows complete co-segregation with the dominantly inherited CMT2 phenotype in our familyPhenotypical features of the p.R120W mutation in the GDAP1 gene causing autosomal dominant Charcot-Marie-Tooth disease.Ganglioside-induced differentiation associated-protein 1 (GDAP1) mutations are commonly associated with autosomal recessive Charcot-Marie-Tooth (ARCMT) neuropathy; however, in rare instances, they also lead to autosomal dominant Charcot-Marie-Tooth (ADCMT).A novel GDAP1 Q218E mutation in autosomal dominant Charcot-Marie-Tooth disease.In this study, we aimed to elucidate the disease mechanisms in dominant intermediate Charcot-Marie-Tooth neuropathy type B and to find explanations for the tissue-specific defects that are associated with different DNM2 mutations in dominant intermediate Charcot-Marie-Tooth neuropathy type B versus autosomal dominant centronuclear myopathy.Mutations in dynamin 2 (DNM2) lead to dominant intermediate Charcot-Marie-Tooth neuropathy type B, while a different set of DNM2 mutations cause autosomal dominant centronuclear myopathy.A novel LRSAM1 mutation is associated with autosomal dominant axonal Charcot-Marie-Tooth diseaseRecently, mutations affecting different domains of dynamin-2 (DNM2) were associated alternatively with autosomal dominant centronuclear myopathy or dominant intermediate (demyelinating and axonal) Charcot-Marie-Tooth disease (CMT) type B.Mutations in the Dynamin 2 gene (DNM2) cause autosomal dominant centronuclear myopathy or autosomal dominant (AD) Charcot-Marie-Tooth (CMT) diseaseMutations in the ganglioside-induced differentiation-associated protein 1 gene cause either autosomal recessive demyelinating Charcot-Marie-Tooth disease type 4A or autosomal recessive axonal Charcot-Marie-Tooth disease with vocal cord paresisA novel mutation in the GDAP1 gene is associated with autosomal recessive Charcot-Marie-Tooth disease in an Amish familyPhenotypical features of a Moroccan family with autosomal recessive Charcot-Marie-Tooth disease associated with the S194X mutation in the GDAP1 geneAutosomal dominant Charcot-Marie-Tooth type-1A neuropathy (CMT1A) is a demyelinating peripheral nerve disorder that is commonly associated with a submicroscopic tandem DNA duplication of a 1.5-Mb region of 17p11.2p12 that contains the peripheral myelin gene PMP22. OBJECTIVE: Ganglioside-induced differentiation associated-protein 1 (GDAP1) mutations are commonly associated with autosomal recessive Charcot-Marie-Tooth (ARCMT) neuropathy; however, in rare instances, they also lead to autosomal dominant Charcot-Marie-Tooth (ADCMT). Mutations in the ganglioside-induced-differentiation-associated protein 1 gene (GDAP1) can cause Charcot-Marie-Tooth (CMT) disease with demyelinating (CMT4A) or axonal forms (CMT2K and ARCMT2K). Mutations in the NEFL gene were recently reported as a cause for autosomal dominant Charcot-Marie-Tooth type 2E (CMT2E) linked to chromosome 8p21. Mild functional differences of dynamin 2 mutations associated to centronuclear myopathy and Charcot-Marie Tooth peripheral neuropathy.Ganglioside-induced differentiation associated-protein 1 (GDAP1) mutations are commonly associated with autosomal recessive Charcot-Marie-Tooth (ARCMT) neuropathy; however, in rare instances, they also lead to autosomal dominant Charcot-Marie-Tooth (ADCMT).In this study, we aimed to elucidate the disease mechanisms in dominant intermediate Charcot-Marie-Tooth neuropathy type B and to find explanations for the tissue-specific defects that are associated with different DNM2 mutations in dominant intermediate Charcot-Marie-Tooth neuropathy type B versus autosomal dominant centronuclear myopathy. We used tissue derived from Dnm2-deficient mice to establish an appropriate peripheral nerve model and found that dominant intermediate Charcot-Marie-Tooth neuropathy type B-associated dynamin 2 mutants, but not autosomal dominant centronuclear myopathy mutants, impaired myelination. Mutations in dynamin 2 (DNM2) lead to dominant intermediate Charcot-Marie-Tooth neuropathy type B, while a different set of DNM2 mutations cause autosomal dominant centronuclear myopathy. In this study, we aimed to elucidate the disease mechanisms in dominant intermediate Charcot-Marie-Tooth neuropathy type B and to find explanations for the tissue-specific defects that are associated with different DNM2 mutations in dominant intermediate Charcot-Marie-Tooth neuropathy type B versus autosomal dominant centronuclear myopathy.We conclude that dynamin 2 mutations should be screened in the autosomal dominant Charcot-Marie-Tooth neuropathy families with intermediate or axonal NCV, and in patients with a classical mild to moderately severe Charcot-Marie-Tooth phenotype, especially when Charcot-Marie-Tooth is associated with neutropaenia or cataracts.  .A novel LRSAM1 mutation is associated with autosomal dominant axonal Charcot-Marie-Tooth disease.We conclude that dynamin 2 mutations should be screened in the autosomal dominant Charcot-Marie-Tooth neuropathy families with intermediate or axonal NCV, and in patients with a classical mild to moderately severe Charcot-Marie-Tooth phenotype, especially when Charcot-Marie-Tooth is associated with neutropaenia or cataracts.In this study, we aimed to elucidate the disease mechanisms in dominant intermediate Charcot-Marie-Tooth neuropathy type B and to find explanations for the tissue-specific defects that are associated with different DNM2 mutations in dominant intermediate Charcot-Marie-Tooth neuropathy type B versus autosomal dominant centronuclear myopathy.Phenotypical features of the p.R120W mutation in the GDAP1 gene causing autosomal dominant Charcot-Marie-Tooth disease.Mutations in the ganglioside-induced-differentiation-associated protein 1 gene (GDAP1) can cause Charcot-Marie-Tooth (CMT) disease with demyelinating (CMT4A) or axonal forms (CMT2K and ARCMT2K).A wide range of phenotypes have been reported in autosomal recessive (AR) Charcot-Marie-Tooth disease (CMT) patients carrying mutations in the ganglioside-induced differentiation-associated protein 1 (GDAP1) gene, such as axonal, demyelinating, and intermediate forms of AR CMT.A severe recessive and a mild dominant form of Charcot-Marie-Tooth disease associated with a newly identified Glu222Lys GDAP1 gene mutation.A novel GDAP1 Q218E mutation in autosomal dominant Charcot-Marie-Tooth disease.A novel LRSAM1 mutation is associated with autosomal dominant axonal Charcot-Marie-Tooth disease.\"}, {\"question\": \"Which transcription factors (TFs) participate in the formation of the interferon-beta (IFN-b) enhanceosome?\", \"ideal_answer\": [\"Transcriptional activation of the IFN beta gene in response to virus infection requires the assembly of an enhanceosome, consisting of the transcriptional activators NF-kappa B, IRF1, ATF2/c-Jun, and the architectural protein HMG I(Y).  Transcriptional activation of the human interferon-beta (IFN-beta) gene by virus infection requires the assembly of a higher order nucleoprotein complex, the enhanceosome, which consists of the transcriptional activators NF-kappa B (p50/p65), ATF-2/c-jun, IRF-3 and IRF-7, architectural protein HMGI(Y), and the coactivators p300 and CBP.   \", \"Transcriptional activation of the human interferon-beta (IFN-beta) gene by virus infection requires the assembly of a higher order nucleoprotein complex, the enhanceosome, which consists of the transcriptional activators NF-kappa B (p50/p65), ATF-2/c-jun, IRF-3 and IRF-7, architectural protein HMGI(Y), and the coactivators p300 and CBP. A functional interferon-beta gene enhanceosome was assembled in vitro using the purified recombinant transcriptional activator proteins ATF2/c-JUN, IRF1, and p50/p65 of NF-kappa B. However, HMG I(Y) plays an essential role in the assembly and function of the IFN beta gene enhanceosome.\"], \"exact_answer\": [[\"NF-kappa B (p50/p65)\"], [\"ATF-2\"], [\"c-jun\"], [\"IRF-3\"], [\"IRF-7\"], [\"IRF-1\"]], \"context\": \"The dimer formed by the ATF-2 and c-Jun transcription factors is one of the main components of the human interferon-beta enhanceosome.The induction of IFN transcription resulted from the activation of the components of the IFN-beta enhanceosome, i.e. IFN regulatory factor (IRF) 3, nuclear factor (NF)-kappaB, activating transcription factor (ATF)-2 and c-Jun.Transcriptional activation of the interferon-beta (IFN-beta) gene requires assembly of an enhanceosome containing the transcription factors ATF-2/c-Jun, IRF-3/IRF-7, NF-kappaB and HMGI(Y). These factors cooperatively bind a composite DNA site and activate expression of the IFN-beta gene.Here we report that within the IFN-beta enhanceosome the ATF-2-c-jun heterodimer binds in a specific orientation, which is required for assembly of a complex between ATF-2-c-jun and interferon regulatory factor 3 (IRF-3). Here, we identified a small molecule that induces the assembly of the interferon-beta (IFN-beta) enhanceosome by stimulating all the enhancer-binding activator proteins: ATF2/c-JUN, IRF3, and p50/p65 of NF-kappaB.Transcriptional activation of the virus inducible enhancer of the human interferon-beta (IFN-beta) gene in response to virus infection requires the assembly of an enhanceosome, consisting of the transcriptional activators NF-kappaB, ATF-2/c-Jun, IRFs and the architectural protein of the mammalian high mobility group I(Y) [HMG I(Y)].Here, we demonstrate that the first step in enhanceosome assembly, i.e. HMG I(Y)-dependent recruitment of NF-kappaB and ATF-2/c-Jun to the enhancer, is facilitated by discrete regions of HMG I and is mediated by allosteric changes induced in the DNA by HMG I(Y) and not by protein-protein interactions between HMG I(Y) and these proteins. However, we show that completion of the enhanceosome assembly process requires protein-protein interactions between HMG I(Y) and the activators.Transcriptional activation of the human interferon-beta (IFN-beta) gene by virus infection requires the assembly of a higher order nucleoprotein complex, the enhanceosome, which consists of the transcriptional activators NF-kappa B (p50/p65), ATF-2/c-jun, IRF-3 and IRF-7, architectural protein HMGI(Y), and the coactivators p300 and CBP. The transcriptional coactivators CBP and P/CAF are required for activation of transcription from the IFN beta enhanceosome. We show that CBP and P/CAF acetylate HMG I(Y), the essential architectural component required for enhanceosome assembly, at distinct lysine residues, causing distinct effects on transcription.The transcriptional activity of an in vitro assembled human interferon-beta gene enhanceosome is highly synergistic. This synergy requires five distinct transcriptional activator proteins (ATF2/c-JUN, interferon regulatory factor 1, and p50/p65 of NF-kappaB), the high mobility group protein HMG I(Y), and the correct alignment of protein-binding sites on the face of the DNA double helix.In addition, we provide evidence that recruitment of the holoenzyme by the enhanceosome is due, at least in part, to interactions between the enhanceosome and the transcriptional coactivator CREB, cAMP responsive element binding protein (CBP). Transcriptional activation of the IFN beta gene in response to virus infection requires the assembly of an enhanceosome, consisting of the transcriptional activators NF-kappa B, IRF1, ATF2/c-Jun, and the architectural protein HMG I(Y).Transcriptional synergy requires recruitment of the CBP/p300 coactivator to the enhanceosome, via a new activating surface assembled from the novel p65 domain and the activation domains of all of the activators.A functional interferon-beta gene enhanceosome was assembled in vitro using the purified recombinant transcriptional activator proteins ATF2/c-JUN, IRF1, and p50/p65 of NF-kappa B. Maximal levels of transcriptional synergy between these activators required the specific interactions with the architectural protein HMG I(Y) and the correct helical phasing of the binding sites of these proteins on the DNA helix.We present evidence that transcriptional activation of the human interferon-beta (IFN beta) gene requires the assembly of a higher order transcription enhancer complex (enhanceosome). This multicomponent complex includes at least three distinct transcription factors and the high mobility group protein HMG I(Y).Thus, HMG I(Y) plays an essential role in the assembly and function of the IFN beta gene enhanceosome.Transcriptional activation of the interferon-beta (IFN-beta) gene requires assembly of an enhanceosome containing the transcription factors ATF-2/c-Jun, IRF-3/IRF-7, NF-kappaB and HMGI(Y).A functional interferon-beta gene enhanceosome was assembled in vitro using the purified recombinant transcriptional activator proteins ATF2/c-JUN, IRF1, and p50/p65 of NF-kappa BTranscriptional activation of the human interferon-beta (IFN-beta) gene by virus infection requires the assembly of a higher order nucleoprotein complex, the enhanceosome, which consists of the transcriptional activators NF-kappa B (p50/p65), ATF-2/c-jun, IRF-3 and IRF-7, architectural protein HMGI(Y), and the coactivators p300 and CBPTranscriptional activation of the IFN beta gene in response to virus infection requires the assembly of an enhanceosome, consisting of the transcriptional activators NF-kappa B, IRF1, ATF2/c-Jun, and the architectural protein HMG I(Y)\"}, {\"question\": \"List packages for transcription factor binding sites' (TFBS) analysis available in R/Bioconductor\", \"ideal_answer\": [\"Neighbourhood Consistent PC (NCPC) algorithms, MMDiff and cosmo.\"], \"exact_answer\": [[\"Neighbourhood Consistent PC (NCPC) algorithms\"], [\"MMDiff\"], [\"cosmo\"], [\"dPattern\"], [\"TFBS\"]], \"context\": \"Clustered occurrence of multiple TFs at genomic sites may arise from chromatin accessibility and local cooperation between TFs, or binding sites may simply appear clustered if the profiles are generated from diverse cell populations. Overlaps in TF binding profiles may also result from measurements taken at closely related time intervals. It is thus of great interest to distinguish TFs that directly regulate gene expression from those that are indirectly associated with gene expression. Graphical models, in particular Bayesian networks, provide a powerful mathematical framework to infer different types of dependencies. However, existing methods do not perform well when the features (here: TF binding profiles) are highly correlated, when their association with the biological outcome is weak, and when the sample size is small. Here, we develop a novel computational method, the Neighbourhood Consistent PC (NCPC) algorithms, which deal with these scenarios much more effectively than existing methods do. We further present a novel graphical representation, the Direct Dependence Graph (DDGraph), to better display the complex interactions among variables. NCPC and DDGraph can also be applied to other problems involving highly correlated biological features. Both methods are implemented in the R package ddgraph, available as part of BioconductorHere, we present MMDiff, a robust, broadly applicable method for detecting differences between sequence count data sets. Based on quantifying shape changes in signal profiles, it overcomes challenges imposed by the highly structured nature of the data and the paucity of replicatesSupervised detection of conserved motifs in DNA sequences with cosmoWe here introduce an algorithm, called cosmo, that allows this search to be supervised by specifying a set of constraints that the position weight matrix of the unknown motif must satisfy. Such constraints may be formulated, for example, on the basis of prior knowledge about the structure of the transcription factor in question. The algorithm is based on the same two-component multinomial mixture model used by MEME, with stronger reliance, however, on the likelihood principle instead of more ad-hoc criteria like the E-valuedPattern: transcription factor binding site (TFBS) discovery in human genome using a discriminative pattern analysis.TFBS: Computational framework for transcription factor binding site analysis.\"}, {\"question\": \"List all reported treatment options for anxiety in autism spectrum disorder.\", \"ideal_answer\": [\"The predominant approach is to use versions of cognitive behavioural therapies, such as:\\nMindfulness Based Therapy (MBT)\\nMultimodal Anxiety and Social Skills Intervention (MASSI) program\\nmodified version of the Coping Cat program, (cognitive-behavioral therapy; CBT)\\nFamily cognitive-behavioral therapy has been found to be more effective than Individual cognitive-behavioral therapy \\nConflict management for couples, even when conflict and family distress is low\\n\\nDrugtherapy: \\nSertraline\"], \"exact_answer\": [[\"Mindfulness Based Therapy (MBT)\"], [\"Multimodal Anxiety and Social Skills Intervention (MASSI) program\"], [\"modified version of the Coping Cat program\", \"(cognitive-behavioral therapy; CBT\"], [\"Family cognitive-behavioral therapy\"], [\"Individual cognitive-behavioral therapy\"], [\"Conflict management for couples\", \"even when conflict and family distress is low\"], [\"Sertraline\"]], \"context\": \"A parent report of comorbid diagnosis of attention-deficit/hyperactivity disorder, bipolar disorder, obsessive-compulsive disorder, depression, or anxiety was associated with a high rate of use, with 80% receiving \\u2265 1 psychotropic medicationMindfulness-based therapy (MBT) has been found effective in reducing anxiety and depression symptoms, however research in autism is limited.Therefore, we examined the effects of a modified MBT protocol (MBT-AS) in high-functioning adults with ASD.the present study is the first controlled trial to demonstrate that adults with ASD can benefit from MBT-AS.This pilot, randomized controlled trial evaluated the feasibility and preliminary outcomes of the Multimodal Anxiety and Social Skills Intervention (MASSI) program in a sample of 30 adolescents with ASD and anxiety symptoms of moderate or greater severity.These findings suggest MASSI is a feasible treatment program and further evaluation is warranted.The purpose of this pilot study was to evaluate whether a modified version of the Coping Cat program could be effective in reducing anxiety in children with autism spectrum disorder (ASD).the Coping Cat program (cognitive-behavioral therapy; CBT)Results provide preliminary evidence that a modified version of the Coping Cat program may be a feasible and effective program for reducing clinically significant levels of anxiety in children with high-functioning ASD.The intent of this article is to explore the efficacy of both the literal and concrete externalization aspects within narrative therapy, and the implementation of interactive metaphors as a combined psychotherapeutic approach for decreasing anxiety with people who present with high-functioning autism.This paper reports a case series of children and adolescents with ASD and an anxiety disorder who were treated with a standard cognitive behaviour therapy (CBT) rationale adapted to take account of the neuropsychological features of ASD.children with moderate autistic symptomology (per SRS-P) were significantly more likely to improve from family CBT (FCBT) than individual CBT (ICBT; OR\\u00a0=\\u00a08.67)Though both treatments reduced anxiety, FCBT outperformed ICBT for children with moderate ASD symptoms, a benefit potentially linked to more at-home exposures and greater child involvement in FCBT.A structural model-building approach was used to test the extent to which family and peer variables directly or indirectly affected ASD via child anxiety/depression.The key findings were that anxiety/depression and ASD symptomatology were significantly related, and family conflict was more predictive of ASD symptomatology than positive family/peer influences. The results point to the utility of expanding interventions to include conflict management for couples, even when conflict and family distress is low.A family-based, cognitive behavioural treatment for anxiety in 47 children with comorbid anxiety disorders and High Functioning Autism Spectrum Disorder (HFA) was evaluated.Following treatment, 71.4% of the treated participants no longer fulfilled diagnostic criteria for an anxiety disorder. Comparisons between the two conditions indicated significant reductions in anxiety symptoms as measured by self-report, parent report and teacher report\"}, {\"question\": \"List adenosine A2A receptor antagonists that are used for Parkinson's disease treatment.\", \"ideal_answer\": [\"Istradefylline and preladenant are adenosine A2A receptor antagonists that are used for Parkinson's disease treatment.\"], \"exact_answer\": [[\"istradefylline\"], [\"preladenant\"]], \"context\": \"Adenosine A2A antagonists, such as istradefylline, improve motor function in PD, but their effect on cognitive impairment has not been determined.Both istradefylline and preladenant have demonstrated moderate efficacy in reducing off time in PD patients with motor fluctuations.  The available data also suggest that caffeine can improve the motor deficits of PD and that adenosine A2A receptor antagonists such as istradefylline reduces OFF time and dyskinesia associated with standard 'dopamine replacement' treatments. Istradefylline (KW-6002) is the first of several adenosine A2A receptor antagonists in development for PD to advance to phase III clinical trials. Initial studies indicate that in patients with motor fluctuations on levodopa, addition of istradefylline reduces 'off' time. These include alpha2 adrenergic receptor antagonists (eg, fipamezole), adenosine A2A receptor antagonists (eg, istradefylline), AMPA receptor antagonists (eg, talampanel), neuronal synchronization modulators (eg, levetiracetam) and agents that interact with serotonergic systems such as 5-hydroxytryptamine (5-HT)1A agonists (eg, sarizotan) and 5-HT2A antagonists (eg, quetiapine). BACKGROUND: We evaluated the efficacy and safety of istradefylline, a selective adenosine A2A receptor antagonist administered as adjunctive treatment to levodopa for 12 weeks in a double-blind manner in Parkinsons disease patients with motor complications in Japan.Istradefylline, a novel adenosine A2A receptor antagonist, for the treatment of Parkinsons disease.Istradefylline, a novel adenosine A2A receptor antagonist, for the treatment of Parkinson's disease.In this article, the author discusses the potential role of A2A adenosine receptor antagonists in the treatment of Parkinson's disease through the evaluation of istradefyllineSuitability of the adenosine antagonist istradefylline for the treatment of Parkinson's disease: pharmacokinetic and clinical considerations.\"}, {\"question\": \"Which are the clinical characteristics of TSC?\", \"ideal_answer\": [\"Tuberous sclerosis or tuberous sclerosis complex (TSC) is a rare multi-system genetic disease that causes benign tumors to grow in the brain and on other vital organs such as the kidneys, heart, eyes, lungs, and skin. A combination of symptoms may include seizures, intellectual disability, developmental delay, behavioral problems, skin abnormalities, lung and kidney disease. TSC is caused by a mutation of either of two genes, TSC1 and TSC2, which code for the proteins hamartin and tuberin respectively. These proteins act as tumor growth suppressors, agents that regulate cell proliferation and differentiation.\"], \"exact_answer\": [[\"Brain lesions\"], [\"Kidney lesions\"], [\"Lung lesions\"], [\"renal angiomyolipoma\"], [\"subependymal giant cell astrocytoma (SEGA)\"], [\"seizures\"], [\"mental detardation\"], [\"autism\"], [\"hepatic lesions\"], [\"Pulmonary lymphangioleiomyomatosis\"]], \"context\": \"Most cases of tuberous sclerosis complex are complicated with bilateral multiple renal angiomyolipoma.Periodic imaging surveillance for development of subependymal giant cell astrocytoma (SEGA), preferably by magnetic resonance imaging (MRI) every 1-3\\u00a0years, is now standard of care Although large tubers are less common than small to medium-sized ones, they are much more likely to be accompanied by severe clinical symptoms (seizures, mental retardation and autistic behaviour), even when the smaller tubers are quite numerousHepatic angiomyolipomas (AML), cysts, and other benign lesions were identified in 30% of the cohort, and some lesions grew significantly over time. Pulmonary lymphangioleiomyomatosis and bilateral renal angiomyolipomas are some presentations of tuberous sclerosis and the coexistence of both conditions may cause devastating morbidity and mortality.uberous sclerosis, a genetic, rare, variably expressed disease. Clinical symptoms were chest pain, and progressive dyspnea. Computed tomography scan of the chest showed bilateral, diffuse, small thin-walled cysts scattered throughout the lungs characteristic for pulmonary lymphangioleiomyomatosis.\"}, {\"question\": \"List fish anti-freeze proteins.\", \"ideal_answer\": [\"AFP-I\\nAFP-II\\nAFP-III\\nAnti-freeze glycoprotein\\nThermal hysteresis protein\"], \"exact_answer\": [[\"anti-freeze protein-3\", \"AFP-III\"], [\"anti-freeze glycoprotein\"], [\"Thermal hysteresis protein\"], [\"AFP-I\"], [\"AFP-II\"]], \"context\": \"the anti-freeze glycoprotein of Antarctic and Arctic notothenoids, pe III anti-freeze protein (AFP)AFP I or AFP IIItype III anti-freeze proteins wild type III thermal hysteresis proteinThermal hysteresis proteins (THPs) have been found in vertebrates, invertebrates, plants, bacteria and fungi and are able to depress the freezing point of water (in the presence of ice crystals) in a non-colligative manner by binding to the surface of nascent ice crystalsskin-type anti-freeze protein-3\"}, {\"question\": \"Name the major classes of small  non coding RNAs in mammalians?\", \"ideal_answer\": [\"microRNAs (miRNAs), small nuclear RNAs (snRNAs), small nucleolar RNAs (snoRNAs) are the major classes of small non coding RNAs. Recently, thanks mostly to massively parallel sequencing technologies, other classes of small RNAs have been discovered, such as piRNAs and scaRNAs.\"], \"exact_answer\": [[\"miRNA\"], [\"snRNA\"], [\"snoRNAs\"], [\"scaRNAs\"], [\"piRNAS\"]], \"context\": \"Some of these RNA classes, in particular microRNAs and snoRNAs, undergo maturation processes that lead to the production of shorter RNAs.\"}, {\"question\": \"List the components of mTOR Complex 2 (mTORC2).\", \"ideal_answer\": [\"Mammalian target of rapamycin complex 2 (mTORC2) is a kinase complex comprised of mTOR, Rictor, mSin1, mLST8/G\\u03b2L and PRR5.\"], \"exact_answer\": [[\"mTOR\"], [\"Rictor\"], [\"mSin1\"], [\"mLST8/G\\u03b2L\"], [\"PRR5\"]], \"context\": \"mTOR is the major component of two protein complexes: mTOR complex 1 (mTORC1) and mTORC2 Rictor, an mTORC2-specific componentf Rictor (rapamycin-insensitive companion of mTOR), an mTORC2 component, The mammalian target of rapamycin (mTOR) complex 2 (mTORC2), which contains the regulatory protein Rictor (rapamycin-insensitive companion of mTOR),  Rictor, a key component of mTORC2SIN1, a key component of mTOR complex 2 (mTORC2) the essential mTORC2 component rictor Sin1 is an essential component of mTOR complex 2 (mTORC2).  SIN1 or Rictor, two key components of mTOR complex 2 (mTORC2) Mammalian target of rapamycin complex 2 (mTORC2) is a kinase complex comprised of mTOR, Rictor, mSin1, mLST8/G\\u03b2L and PRR5 \"}, {\"question\": \"Which are the mains risk factors of metabolic syndrome?\", \"ideal_answer\": [\"Metabolic syndrome is a disorder of energy utilization and storage, diagnosed by a co-occurrence of three out of five of the following medical conditions: abdominal (central) obesity, elevated blood pressure, elevated fasting plasma glucose, high serum triglycerides, and low high-density cholesterol (HDL) levels. Metabolic syndrome increases the risk of developing cardiovascular disease, particularly heart failure, and diabetes.\"], \"exact_answer\": [[\"Obesity\"], [\"High blood pressure\", \"Hypertension\"], [\"High serum triglycerides\"], [\"Low high-density cholesterol (HDL) levels\"], [\"Diabetes type 2\"], [\"Dyslipidaemia\"]], \"context\": \"tigated the association between circulating levels of 60 and 70 kDa heat-shock proteins (HSP60 and 70) and cardiovascular risk factors in postmenopausal women with or without metabolic syndrome (MetS). This cross-sectional study included 311 Brazilian women (age \\u226545 years with amenorrhea \\u226512 months). Women showing three or more of the following diagnostic criteria were diagnosed with MetS: waist circumference (WC) \\u226588 cm, blood pressure \\u2265130/85 mmHg, triglycerides \\u2265150 mg/dl, high-density lipoprotein (HDL) <50 mg/dl, and glucose \\u2265100 mg/dl. Clinical, anthropometric, and biochemical parameters were collected. HSP60, HSP70, antibodies to HSP60 and HSP70, and C-reactive protein (CRP) levels were measured in serum. Student's t test, Kruskal-Wallis test, chi-square test, and Pearson correlation were used for statistical analysis. Of the 311 women, 30.9 % (96/311) were diagnosed with MetS. These women were, on average, obese with abdominal fat deposition and had lower HDL values as well as higher triglycerides and glucose levels. Homeostasis model assessment-insulin resistant (HOMA-IR) test values in these women were compatible with insulin resistance (P\\u2009<\\u20090.05). CRP and  is associated with an increased risk for cardiovascular disease (CVD) and type 2 diabetes mellitus (T2DM). Severalalence of the metabolic syndrome (MetS), a cluster of cardiovascular risk factors associated with obesity and insulin resistance, is dramatically increasing in Western and developing countries. This disorder is not only associated with a higher risk of appearance of type 2 diabetes and cardiovascular events, but impacts on the liver in different ways. Nonalcbolic syndrome (MS) components, such as dyslipidemia, prothrombotic status, and increased blood pressure, are risk factors for patients with renal disease. Visceral fat mass is closely related to the MS and atherosclerosis. We investigated the effects of body compositions and MS on anemia parameters and recombinant human erythropoietin (rHuEPO) requirements in maintenance hemodialysis patients. METHODS: Body composition (body mass index and bioimpedance analysis) and laboratory data were obtained from 110 dialysis patients. The MS was identified according to ATP-III criteria. Anemia parameters, hemoglobin (Hgb), albumin, C-reactive protein (CRP), calcium, phosphorus, parathormone levels, and rHuEPO requirements over the last 6 months were retrospectively analyzed. RESULTS: Patients with the MS seem to reach target Hgb levels more frequently (10-12 g/dL; 66.3% vs 84.8%; P = .03) without any difference in total intravenous iron therapy dosage. MS patients also required lower rHuEPO for reaching similar Hgb levels compared with patients without MS (2679.3 \\u00b1 1936.1 vs 3702.5 \\u00b1 2213.0 U/kg/6 mo; P = .02). There webolic syndrome (MetS) is typically diagnosed based on abnormalities in specific clustered clinical measures that are associated with increased risk for coronary heart disease (CHD) and Type 2 diabetes mellitus (T2DM). However, current MetS criteria result in racial/ethnic discrepancies. Our goals were to use confirmatory factor analysis (CFA) to delineate differential contributions to MetS by sub-group, and if contributions were discovered, develop sex and racial/ethnic-specific equations to calculate MetS severity. RESEARCH DESIGN AND METHODS: Using data on adults from the National Health and Nutrition Examination Survey 1999-2010, we performed a CFA of a single MetS factor that allowed differential loadings across groups, resulting in a sex and race/ethnicity-specific continuous MetS severity score. RESULTS: Loadings to the single MetS factor differed by sub-group for each MetS component (p<0.001), with lower factor loadings among non-Hispanic-blacks for triglycerides and among Hispanics for waist circumference. Systolic blood pressure exhibited low factor loadings among all groups. MetS severity scores were correlated with biomarkers of future disease (high-sensitivity C-reactive-protein, uric acid, insulin resistance). Non-Hispanic-black-males with diabetics had a low prevalence of MetS but high MetS severity scores that were not significantly different from other racial/ethnic groups. CONCLUSIONS: This analysis among adults uniquely demonstrated differences between sexes and racial/ethnic groups regarding contributions of traditional MetS components to an assumed single factor. The resulting equations provide a clinically-accessible and interpretable continuous measure of MetS for potential use in identifying adults at higher risk for MetS-related diseases and following changes within individuals over time. These eq study, we aim to examine the associations of obesity related loci with risk of metabolic syndrome (MetS) in a children population from China. A totaalence of obesity is on the increase, and consequently metabolic syndrome is also becoming a serious health problem in children and adolescents all over the world. This revciation between alopecia areata (AA), psoriasis, and other autoimmune diseases has been well reported in the literature, an association with metabolic syndrome has not been reported. We present two young women with the combination of severe psoriasis, androgen excess, metabolic syndrome, thyroiditis, and AA. Both w\"}, {\"question\": \"Which multiple kinase inhibitors are used in cancer therapy?\", \"ideal_answer\": [\"Multiple kinase inhibitors used in cancer therapy include ZD6474, SU11248, AEE 788, sorafenib, vatalanib, and AG-013736.\"], \"exact_answer\": [[\"ZD6474\"], [\"SU11248\"], [\"AEE 788\"], [\"sorafenib\"], [\"vatalanib\"], [\"AG-013736\"]], \"context\": \"These inhibitors generally hinder the phosphorylation of several protein kinases of membrane receptors, such as vascular endothelial growth factor receptors, platelet-derived growth factor receptors, the human epidermal growth factor receptor family, and cytoplasmic receptors such as c-Kit, Raf kinase, and FLT3.These inhibitors include ZD6474, SU11248, AEE 788, sorafenib, vatalanib, and AG-013736.\"}, {\"question\": \"List types of avoided words in bacterial genomes\", \"ideal_answer\": [\"Short palindromic sequences (4, 5 and 6 bp palindromes) are avoided at a statistically significant level in the genomes of several bacteria, including the completely sequenced Haemophilus influenzae and Synechocystis sp. genomes and in the complete genome of the archaeon Methanococcus jannaschii. Palindromes corresponding to sites for restriction enzymes from other species are also avoided, albeit less significantly, suggesting that in the course of evolution bacterial DNA has been exposed to a wide spectrum of restriction enzymes, probably as the result of lateral transfer mediated by mobile genetic elements, such as plasmids and prophages. Palindromic words appear to accumulate in DNA once it becomes isolated from restriction-modification systems, as demonstrated by the case of organellar genomes.\"], \"exact_answer\": [[\"Short palindromic sequences (4, 5 and 6 bp palindromes)\"], [\"Palindromes corresponding to sites for restriction enzymes from other species\"]], \"context\": \"Avoidance of palindromic words in bacterial and archaeal genomes: a close connection with restriction enzymes.Short palindromic sequences (4, 5 and 6 bp palindromes) are avoided at a statistically significant level in the genomes of several bacteria, including the completely sequenced Haemophilus influenzae and Synechocystis sp. genomes and in the complete genome of the archaeon Methanococcus jannaschii.Palindromes corresponding to sites for restriction enzymes from other species are also avoided, albeit less significantly, suggesting that in the course of evolution bacterial DNA has been exposed to a wide spectrum of restriction enzymes, probably as the result of lateral transfer mediated by mobile genetic elements, such as plasmids and prophagesPalindromic words appear to accumulate in DNA once it becomes isolated from restriction-modification systems, as demonstrated by the case of organellar genomesIn certain cases, a comparison of avoided palindromic words in taxonomically related bacteria shows a pattern of relatedness of their R-M systems.Recognition sites for type II restriction and modification enzymes in genomes of several bacteria are recognized as semi-palindromic motifs and are avoided at a significant degree.On the basis of the set of most avoided words, and in accordance to the IUPAC coding standards, suggestions are made regarding potential recognition sequences. In certain cases, a comparison of avoided palindromic words in taxonomically related bacteria shows a pattern of relatedness of their R-M systems.\"}, {\"question\": \"Which gene(s) should be genotyped in order to prescribe the drug Cetuximab (anti-EGFR)?\", \"ideal_answer\": [\"KRAS mutation has been unambiguously identified as a marker of resistance to cetuximab-based treatment in metastatic colorectal cancer (mCRC) patients.\\nOther genes are such as EGFR, BRAF and T53 have also been suggested to be genotyped in order to evaluate the drug responsivness.\"], \"exact_answer\": [[\"KRAS\"], [\"EGFR\"], [\"BRAF\"], [\"T53\"]], \"context\": \"Treatment of metastatic colorectal cancer with targeted anti-EGFR therapeutics such as cetuximab extends survival in only 25% of patients who test wild-type for KRAS, while the majority of patients prove resistant (J Clin Oncol 28(7):1254-1261, 2010).. Somatic mutation of the EGFR signalling pathway is a prevalent mechanism of resistance to cetuximab (Nature 486(7404):532-536, 2012). If the human genome harbours variants that influence susceptibility of the EGFR pathway to oncogenic mutation, such variants could also be prognostic for cetuximab responsiveness. KRAS mutations are strong predictors for clinical outcomes of EGFR-targeted treatments such as cetuximab and panitumumab in metastatic colorectal cancer (mCRC).KRAS mutation is widely accepted as a strong, negative predictive marker for anti-epidermal growth factor receptor antibodies, including cetuximab and panitumumab. KRAS status is now a mandatory prerequisite in order to treat metastatic colorectal patients with anti-Epidermal Growth Factor Receptor (EGFR) antibodies, such as cetuximab or panitumumab. KRAS mutations are unambiguously linked to a lack of response to these targeted therapies.The EGFR-R497K polymorphism is a potential predictor for overall survival in HNSCC patients treated with cetuximab based therapy in the palliative setting.Laboratories are increasingly required to perform molecular tests for the detection of mutations in the KRAS gene in metastatic colorectal cancers to allow better clinical management and more effective treatment for these patients. KRAS mutation status predicts a patient's likely response to the monoclonal antibody cetuximab.KRAS mutation has been unambiguously identified as a marker of resistance to cetuximab-based treatment in metastatic colorectal cancer (mCRC) patients.  This study suggested EGF A+61G polymorphism to be a predictive marker for pCR, independent of KRAS mutation status, to cetuximab-based neoadjuvant chemoradiation of patients with locally advanced rectal cancer.Intratumoral gene expression levels of EGFR, VEGFR2 and NRP as well as polymorphisms in FCGR3A, CyclinD1 and EGFR could predict clinical outcome in mCRC patients enrolled in BOND2, independent of KRAS mutation status.Numerous clinical studies have shown that anti-EGFR therapies are effective only in a subset of patients with colorectal cancer. Even though mutations in the KRAS gene have been confirmed as negative predictors of the response to EGFR-targeted therapies, not all KRAS wild-type (wt-KRAS) patients will respond to treatment. Recent studies have demonstrated that additionally wild-type BRAF (wt-BRAF) genotype is required for response to panitumumab or cetuximab, suggesting that BRAF genotype criteria should be used together with KRAS genotype for selecting the patients who are about to benefit from the anti-EGFR therapy.KRAS mutation is a dramatic example of single nucleotide polymorphism, which is able to identify a priori patients that could receive or not an anti-EGFR monoclonal antibody such as cetuximab or panitumumab. While confirming the negative effect of KRAS mutations on outcome after cetuximab, we show that BRAF, NRAS, and PIK3CA exon 20 mutations are significantly associated with a low response rate.Cetuximab and panitumumab, mAbs targeting EGFR, are registered for metastatic colorectal carcinoma (mCRC) patients whose tumors express EGFR as determined by immunohistochemistry. However, this method is not predictive of treatment efficacy.. This study suggests that TP53 mutations are predictive of cetuximab sensitivity, particularly in patients without KRAS mutation, and that TP53 genotyping could have a clinical interest to select patients who should benefit from cetuximab-based CT.A KRAS mutation confers resistance to cetuximab, which reduces treatment options, especially in first-line. KRAS genotyping was recently introduced as predictive biomarker, since only tumors carrying a wildtype were found to respond to treatment with panitumumab.\"}, {\"question\": \"Which micro-RNAs have been associated in the pathogenesis of Rheumatoid Arthritis?\", \"ideal_answer\": [\"Different expression patterns of mir-146a, miRNA-155, miRNA-124a, mir-203, mir-223, mir-346, mir-132, mir-363, mir-498, mir-15a, and mir-16 were documented in several tissue sample types of RA patients.\"], \"exact_answer\": [[\"miR-155\"], [\"miR-146a\"], [\"miR-124a\"], [\"miR-222\"], [\"miR-223\"], [\"miR-203\"], [\"miR-346\"], [\"miR-132\"], [\"miR-363\"], [\"miR-498\"], [\"miR-15a\"], [\"miR-16\"], [\"miR-18a\"]], \"context\": \"This study provides the first description of increased expression of miRNA miR-155 and miR-146a in RAIn this study we investigated the expression, regulation, and function of miR-155 and miR-146a in rheumatoid arthritis (RA) synovial fibroblasts (RASFs) and RA synovial tissueThe objective of this study was to identify the expression pattern of microRNA-146 (miR-146) in synovial tissue from patients with rheumatoid arthritis (RA)This study shows that miR-146 is expressed in RA synovial tissue and that its expression is induced by stimulation with TNFalpha and IL-1betapregulated miR-146a expression in peripheral blood mononuclear cells from rheumatoid arthritis patientsTotal RNA was isolated from peripheral blood mononuclear cells obtained from patients with rheumatoid arthritis, and healthy and disease control individuals, and the expression of miR-146a, miR-155, miR-132, miR-16, and microRNA let-7a was analyzed using quantitative real-time PCRRheumatoid arthritis peripheral blood mononuclear cells exhibited between 1.8-fold and 2.6-fold increases in miR-146a, miR-155, miR-132, and miR-16 expression, whereas let-7a expression was not significantly different compared with healthy control individualsOur data also suggest a possible mechanism contributing to rheumatoid arthritis pathogenesis, whereby miR-146a expression is increased but unable to properly function, leading to prolonged tumor necrosis factor-alpha production in patients with rheumatoid arthritisRepression of TRAF6 and/or IRAK-1 in THP-1 cells resulted in up to an 86% reduction in tumor necrosis factor-alpha production, implicating that normal miR-146a function is critical for the regulation of tumor necrosis factor-alpha productionicroRNA-124a is a key regulator of proliferation and monocyte chemoattractant protein 1 secretion in fibroblast-like synoviocytes from patients with rheumatoid arthritisWe found that miR-124a levels significantly decreased in RA synoviocytes as compared with OA synoviocytesInduction of miR-124a in RA synoviocytes significantly suppressed the production of the CDK-2 and MCP-1 proteinshe results of this study suggest that miR-124a is a key miRNA in the posttranscriptional regulatory mechanisms of RA synoviocytesiR-223 is overexpressed in T-lymphocytes of patients affected by rheumatoid arthritis.Although a multifactorial pathogenesis has been hypothesized, the precise mechanisms leading to the disease are still poorly understood at the molecular level. miRNA expression profile analysis highlighted that miR-223 is the only miRNA that is strikingly deregulated in peripheral T-lymphocytes from RA patients compared with healthy donorsIn summary, our data provide a first characterization of the miRNA expression profiles of peripheral T-lymphocytes of RA patients, identifying miR-223 as overexpressed in CD4(+) naive T-lymphocytes from these individualsAltered microRNA expression profile with miR-146a upregulation in CD4+ T cells from patients with rheumatoid arthritisThe role of miR-146a overexpression in regulating T cell apoptosis was evaluated by flow cytometryWe have detected increased miR-146a in CD4+ T cells of RA patients and its close correlation with TNF-alpha levels. Our findings that miR-146a overexpression suppresses T cell apoptosis indicate a role of miR-146a in RA pathogenesis and provide potential novel therapeutic targetsicroRNA-146a expresses in interleukin-17 producing T cells in rheumatoid arthritis patientsSix miRNAs, let-7a, miR-26, miR-146a/b, miR-150, and miR-155 were significantly up regulated in the IL-17 producing T cellsThese results indicated that miR-146a was associated with IL-17 expression in the PBMC and synovium in RA patientspolymorphism in the 3'-UTR of interleukin-1 receptor-associated kinase (IRAK1), a target gene of miR-146a, is associated with rheumatoid arthritis susceptibilityMicroRNA-146a was found to be increased in synovial fibroblasts, synovial tissue and PBMC from patients with RAThis is the first study that addresses association of a variant in a target of miR-146a, IRAK1 gene, with RA susceptibilityltered expression of microRNA-203 in rheumatoid arthritis synovial fibroblasts and its role in fibroblast activationPreviously, we described increased expression of miR-155 and miR-146a in rheumatoid arthritis (RA) and showed a repressive effect of miR-155 on matrix metalloproteinase (MMP) expression in RA synovial fibroblasts (RASFs)The current results demonstrate methylation-dependent regulation of miR-203 expression in RASFs Importantly, they also show that elevated levels of miR-203 lead to increased secretion of MMP-1 and IL-6 via the NF-\\u03baB pathway and thereby contribute to the activated phenotype of synovial fibroblasts in RAiR-124a as a key regulator of proliferation and MCP-1 secretion in synoviocytes from patients with rheumatoid arthritisTransfection of miR-124a into RA synoviocytes significantly suppressed their proliferation and arrested the cell cycle at the G1 phaseIt is proposed that miR-124a is a key miRNA in the post-transcriptional regulatory mechanisms of RA synoviocytes, and has a therapeutic potentialExpression of miR-146a and miR-16 in peripheral blood mononuclear cells of patients with rheumatoid arthritis and their correlation to the disease activityThe elevated expression levels of miR-146a and miR-16 are correlated to RA disease activity, suggesting their value in assessment of the clinical disease activity of RAiR-346 controls release of TNF-\\u03b1 protein and stability of its mRNA in rheumatoid arthritis via tristetraprolin stabilizationUsing a microarray, we found two miRNAs, miR-125b and miR-939 predicted to target the 3'-UTR of TNF-\\u03b1 mRNA, to be up-regulated in RA FLS in response to LPS, but their repression did not restore mature TNF-\\u03b1 expression in FLSWe showed previously that miR-346, which is upregulated in LPS-activated FLS, inhibited Btk expression that stabilized TNF-\\u03b1 mRNADifferent expression patterns of mir-146a, miRNA-155, miRNA-124a, mir-203, mir-223, mir-346, mir-132, mir-363, mir-498, mir-15a, and mir-16 were documented in several tissue sample types of RA patientsown-regulation of microRNA-34a* in rheumatoid arthritis synovial fibroblasts promotes apoptosis resistanceBasal expression levels of miR-34a* were found to be reduced in synovial fibroblasts from RA patients compared to osteoarthritis patients, whereas levels of miR-34a, miR-34b/b*, and miR-34c/c* did not differOur data provide evidence of a methylation-specific down-regulation of proapoptotic miR-34a* in RASFsdentification of microRNA-221/222 and microRNA-323-3p association with rheumatoid arthritis via predictions using the human tumour necrosis factor transgenic mouse modelmiR-seq demonstrated that TghuTNF-SF exhibit a distinct pathogenic profile with 22 significantly upregulated and 30 significantly downregulated miRValidation assays confirmed the dysregulation of miR-223, miR-146a and miR-155 previously associated with human rheumatoid arthritis (RA) pathology, as well as that of miR-221/222 and miR-323-3pthe authors identified miR-221/222 and miR-323-3p as novel dysregulated miR in RA SFFurther association studies may contribute to determining the role of miR-146a single-nucleotide polymorphisms in immune-mediated diseasesiR-146a polymorphism is associated with asthma but not with systemic lupus erythematosus and juvenile rheumatoid arthritis in Mexican patients.verexpression of microRNA-223 in rheumatoid arthritis synovium controls osteoclast differentiationMicroRNA-223 (miR-223) is reported to play critical roles in osteoclastogenesisMiR-223 is intensely expressed in RA synovium, and overexpression of miR-223 suppresses osteoclastogenesis in vitroThis study demonstrates the possibility of gene therapy with miR-223 to treat bone destruction in RA patientsMiR-223 was more highly expressed in RA synovium than in osteoarthritis (OA) synovium due to the increased number of miR-223-positive cells in RA synoviumssociation of pre-miRNA-146a rs2910164 and pre\\u2011miRNA-499 rs3746444 polymorphisms and susceptibility to rheumatoid arthritisOur findings demonstrated that the hsa-mir-499 rs3746444, but not mir-146a rs2910164, polymorphism is associated with an increased RA risk in a sample of the Iranian populationumor necrosis factor \\u03b1-induced microRNA-18a activates rheumatoid arthritis synovial fibroblasts through a feedback loop in NF-\\u03baB signalingTo elucidate whether the microRNA (miRNA) cluster miR-17-92 contributes to the activated phenotype of rheumatoid arthritis synovial fibroblasts (RASFsWe found that TNF\\u03b1 induces the expression of miR-17-92 in RASFs in an NF-\\u03baB-dependent mannerUsing reporter gene assays, we identified the NF-\\u03baB pathway inhibitor TNF\\u03b1-induced protein 3 as a new target of miR-18aOur data suggest that the miR-17-92-derived miR-18a contributes to cartilage destruction and chronic inflammation in the joint through a positive feedback loop in NF-\\u03baB signaling, with concomitant up-regulation of matrix-degrading enzymes and mediators of inflammation in RASFsAs inflammation and joint damage are the main hallmarks of RA, we focused on the three miRNAs, miR-146a, miR-155 and miR-223, whose functions have been studied in both the processes and the pathogenic role investigated in the experimental modelsFocusing on the role of miR-146a, miR-155 and miR-223 in RA pathogenesis emphasizes the intertwined relationships between bone homeostasis and immunity, and the prominent role of monocytes in RAIn a final perspective section we discuss the potential impact of therapeutic miR-155 modulation in RAcorrelation between whole blood and PBMC expression levels of miR-155 and miR-146a in healthy controls and rheumatoid arthritis patientsWe demonstrated a highly significant linear correlation between miR-146a and miR-155 expression in PBMC and whole blood, from both healthy individuals and RA patients.expression of specific microRNAs (miRNA) in peripheral blood-derived mononuclear cells (PBMC), particularly miR-146a and miR-155, is associated with rheumatoid arthritis (RA)icroRNA-323-3p with clinical potential in rheumatoid arthritis, Alzheimer's disease and ectopic pregnancyIt has been shown that miR-323-3p associates with the pathogenesis of several diseases, such as rheumatoid arthritis, Alzheimer's disease and ectopic pregnancy\"}, {\"question\": \"Which are the enzymes involved in the addition of 7-methylguanosine in mRNA?\", \"ideal_answer\": [\"The 7-methylguanosine cap added to the 5\\u2032 end of mRNA is essential for efficient gene expression and cell viability. Methylation of the guanosine cap is necessary for the translation of most cellular mRNAs in all eukaryotic organisms in which it has been investigated. In some experimental systems, cap methylation has also been demonstrated to promote transcription, splicing, polyadenylation and nuclear export of mRNA. In the addition of 7-methylguanosine in mRNA involved the RNA polymerase II, RNA guanylyltransferase and RNA guanine-7 methyltransferase enzymes.\", \"The enzymes involved in the addition of 7-methylguanosine in mRNA are RNA guanylyltransferase and 5'-phosphatase (RNGTT), RNA guanine-7 methyltransferase (RNMT or hMTr1), RNMT-activating mini-protein (RAM), RNA polymerase II, S-adenosylhomocysteine hydrolase (SAHH), and Myc.\"], \"exact_answer\": [[\"RNA guanylyltransferase and 5'-phosphatase (RNGTT)\", \"RNA guanylyltransferase and 5'-phosphatase\", \"RNGTT\"], [\"RNA guanine-7 methyltransferase\", \"RNA guanine-7 methyltransferase (RNMT or hMTr1)\", \"RNMT\", \"hMTr1\"], [\"RNMT-activating mini-protein\", \"RNMT-activating mini-protein (RAM)\", \"RAM\"], [\"RNA polymerase II\"], [\"S-adenosylhomocysteine hydrolase\", \"S-adenosylhomocysteine hydrolase (SAHH)\", \"SAHH\"], [\"Myc\"]], \"context\": \"In mammals, cap synthesis is catalysed by the sequential action of RNGTT (RNA guanylyltransferase and 5'-phosphatase) and RNMT (RNA guanine-7 methyltransferase), enzymes recruited to RNA pol II (polymerase II) during the early stages of transcription. We recently discovered that the mammalian cap methyltransferase is a heterodimer consisting of RNMT and the RNMT-activating subunit RAM (RNMT-activating mini-protein). RAM activates and stabilizes RNMT and thus is critical for cellular cap methylation and cell viability.Synthesis of the methyl cap initiates with the addition of 7-methylguanosine to the initiating nucleotide of RNA pol II (polymerase II) transcripts, which occurs predominantly during transcription and in mammals is catalysed by RNGTT (RNA guanylyltransferase and 5' phosphatase) and RNMT (RNA guanine-7 methyltransferase).The RNMT-activating subunit, RAM (RNMT-activating miniprotein), is also recruited to transcription initiation sites via an interaction with RNMT.Phosphorylation of the CTD recruits RNGTT and RNMT, the enzymes involved in mRNA capping, to the nascent transcript.Myc induces methyl cap formation by promoting RNA polymerase II phosphorylation which recruits the capping enzymes to RNA, and by up-regulating the enzyme SAHH (S-adenosylhomocysteine hydrolase), which neutralizes the inhibitory by-product of methylation reactions.In the present study, we identify the methyltransferase responsible for cap1 formation in human cells, which we call hMTr1 (also known as FTSJD2 and ISG95).The guanylyltransferase (GTP:mRNA guanylyltransferase, EC 2.7.7.50) reaction responsible for cap formation usually proceeds via a covalent enzyme-GMP intermediate.The methylation of the 5' terminal guanosine residue of the cap structure of Semliki Forest virus (SFV) mRNAs has been shown to occur in vitro concomitantly with their synthesis (R. K. Cross and P. J. Gomatos, Virology, 114, 542-554, 1981). The enzyme responsible for this methylation, a guanine-7-methyltransferase, is associated with the SFV replication complexAn RNA (guanine-7-)-methyltransferase that specifically methylates the 5'-terminal guanosine residue of RNAs ending in the dinucleoside triphosphate G(5')pppN- has been purified from the cytoplasm of HeLa cells.\"}, {\"question\": \"Which proteins are involved in actin bundling and filopodia formation and function?\", \"ideal_answer\": [\"A number of proteins have been found to regulate F-actin bundling and enhance filopodia formation and motility. Among these are Cysteine-rich protein 1 (CRP1), Fascin, Macrophage actin-associated tyrosine phosphorylated protein (MAYP/PSTPIP2), Insulin receptor tyrosine kinase substrate p53 (IRSp53), Missing in metastasis protein (MIM), Eps8, Diaphanous-related formin (dDia2) and Vasodilator-stimulated phosphoprotein (VASP).\"], \"exact_answer\": [[\"Cysteine-rich protein 1 (CRP1)\"], [\"Fascin\"], [\"Macrophage actin-associated tyrosine phosphorylated protein (MAYP/PSTPIP2)\"], [\"Insulin receptor tyrosine kinase substrate p53 (IRSp53)\"], [\"Missing in metastasis protein (MIM)\"], [\"Eps8\"], [\"Diaphanous-related formin (dDia2)\"], [\"Vasodilator-stimulated phosphoprotein (VASP)\"]], \"context\": \"CRP1, a protein localized in filopodia of growth cones, is involved in dendritic growthCRP1, which cross-links actin filaments to make actin bundles, is the only CRP family member expressed in the CNS with little known about its function in nerve cells.Here, we report that CRP1 colocalizes with actin in the filopodia of growth cones in cultured rat hippocampal neurons.Knockdown of CRP1 expression by short hairpin RNA interference results in inhibition of filopodia formation and dendritic growth in neurons.Overexpression of CRP1 increases filopodia formation and neurite branching, which require its actin-bundling activity.F-ascin is an actin-bundling protein involved in filopodia assembly and cancer invasion and metastasis of multiple epithelial cancer types.In this study, we investigated the role of fascin, a cytoskeleton actin-bundling protein involved in the formation of filopodia and cell migration, in prostate cancer progression.In cellular models, fascin gene silencing using small interfering RNA in the androgen-independent prostate cancer cell line DU145 decreased cell motility and invasiveness while increasing cell adhesive properties.Fascin is an actin-bundling protein that induces membrane protrusions and cell motility after the formation of lamellipodia or filopodia.The PCH family member MAYP/PSTPIP2 directly regulates F-actin bundling and enhances filopodia formation and motility in macrophagesOverexpression of MAYP decreased CSF-1-induced membrane ruffling and increased filopodia formation, motility and CSF-1-mediated chemotaxis.The opposite phenotype was observed with reduced expression of MAYP, indicating that MAYP is a negative regulator of CSF-1-induced membrane ruffling and positively regulates formation of filopodia and directional migration.Overexpression of MAYP led to a reduction in total macrophage F-actin content but was associated with increased actin bundling. Consistent with this, purified MAYP bundled F-actin and regulated its turnover in vitro.In addition, MAYP colocalized with cortical and filopodial F-actin in vivo.Because filopodia are postulated to increase directional motility by acting as environmental sensors, the MAYP-stimulated increase in directional movement may be at least partly explained by enhancement of filopodia formation.We report an unexpected direct association between fascin, an actin-bundling component of filopodia, microspikes and lamellipodial ribs, and protein kinase Calpha (PKCalpha), a regulator of focal adhesions.Fascin is an actin-bundling protein involved in filopodia assembly and cancer invasion and metastasis of multiple epithelial cancer types.Actin-crosslinking proteins organize actin into highly dynamic and architecturally diverse subcellular scaffolds that orchestrate a variety of mechanical processes, including lamellipodial and filopodial protrusions in motile cells.IRSp53, a multi-domain protein that can associate with the Rho-GTPases Rac and Cdc42, participates in these processes mainly through its amino-terminal IMD (IRSp53 and MIM domain). The isolated IMD has actin-bundling activity in vitro and is sufficient to induce filopodia in vivo.Eps8 activates and synergizes with IRSp53 in mediating actin bundling in vitro, enhancing IRSp53-dependent membrane extensions in vivo.Consistently, Cdc42-induced filopodia are inhibited following individual removal of either IRSp53 or Eps8.Collectively, these results support a model whereby the synergic bundling activity of the IRSp53-Eps8 complex, regulated by Cdc42, contributes to the generation of actin bundles, thus promoting filopodial protrusions.The bundling activity of vasodilator-stimulated phosphoprotein is required for filopodium formationFilopodia are highly dynamic finger-like cell protrusions filled with parallel bundles of actin filaments. Previously we have shown that Diaphanous-related formin dDia2 is involved in the formation of filopodia. Another key player for the formation of filopodia across many species is vasodilator-stimulated phosphoprotein (VASP).It has been proposed that the essential role of VASP for formation of filopodia is its competition with capping proteins for filament barbed-end interaction.Only WT DdVASP, but not a mutant lacking the F-actin bundling activity, could rescue the ability of these cells to form WT-like filopodia.Our data suggest that DdVASP is complexed with dDia2 in filopodial tips and support formin-mediated filament elongation by bundling nascent actin filaments.Insulin receptor tyrosine kinase substrate p53 (IRSp53) has been identified as an SH3 domain-containing adaptor that links Rac1 with a Wiskott-Aldrich syndrome family verprolin-homologous protein 2 (WAVE2) to induce lamellipodia or Cdc42 with Mena to induce filopodia.Here, we show that the N-terminal predicted helical stretch of 250 amino acids of IRSp53 is an evolutionarily conserved F-actin bundling domain involved in filopodium formation.The IMD alone, derived from either IRSp53 or MIM, induced filopodia in HeLa cells and the formation of tightly packed parallel F-actin bundles in vitro.These results suggest that IRSp53 and MIM belong to a novel actin bundling protein family.Role of the actin bundling protein fascin in growth cone morphogenesis: localization in filopodia and lamellipodiaFascin localized to radially oriented actin bundles in lamellipodia (ribs) and filopodia.Using a fascin antibody and a GFP fascin construct, we found that fascin incorporated into actin bundles from the beginning of growth cone formation at the cut end of axons.Later, during growth cone morphogenesis when actin ribs shortened, the proximal fascin-free zone of bundles increased, but fascin was retained in the distal, filopodial portion of bundles.Treatment with tumor promoter 12-O-tetradecanoylphorbol-13-acetate (TPA), which phosphorylates fascin and decreases its affinity for actin, resulted in loss of all actin bundles from growth cones.Our findings suggest that fascin may be particularly important for the linear structure and dynamics of filopodia and for lamellipodial rib dynamics by regulating filament organization in bundles.\"}, {\"question\": \"List clinical trials for prevention of sarcopenia\", \"ideal_answer\": [\"Several clinical trials with androgen replacement therapy. \\nStudy was to evaluate the effect of omega-3 fatty acid supplementation on the rate of muscle protein synthesis. This trial was registered at clinical trials.gov as NCT00794079\"], \"exact_answer\": [[\"androgen replacement trials\"], [\"omega-3 fatty acid supplementation\", \"NCT00794079\"]], \"context\": \"several clinical trials with androgen replacement therapy have failed to show clinical benefit.Clinical trials are needed to find better interventions for this syndrome.emergence of many promising interventions towards this age-related condition (e.g., physical exercise [in particular, resistance training], testosterone, antioxidant supplementations), the need for Phase II trial designs is high. The objective of this study was to evaluate the effect of omega-3 fatty acid supplementation on the rate of muscle protein synthesis in older adults.This trial was registered at clinical trials.gov as NCT00794079.The extreme paucity of clinical trials on sarcopenia in literature is mainly due to difficulties in designing studies able to isolate the aging process from its multiple and interconnected consequences.\"}, {\"question\": \"List symptoms of Hakim Triad.\", \"ideal_answer\": [\"Triad of Hakim--Adams is well known for normal pressure hydrocephalus (NPH): dementia, gait disturbances and urinary incontinence.\"], \"exact_answer\": [[\"dementia\"], [\"gait disturbances\"], [\"urinary incontinence\"]], \"context\": \"Triad of Hakim--Adams is well known for normal pressure hydrocephalus (NPH): dementia, gait disturbances and urinary incontinence.Normal pressure hydrocephalus (NPH) is a clinical triad of gait disturbance, dementia, and urinary incontinence combined with radiographic findings of ventriculomegaly and laboratory findings of normal cerebrospinal fluid pressures. Although it was first described by Hakim and Adams in 1965, there is no formal definition of NPH, causing discrepancy in its incidence in various studies. INTRODUCTION: Normal-pressure hydrocephalus (NPH) is a chronic neurological disorder characterized by enlarged ventricles and a triad of clinical symptoms affecting gait, cognition, and urinary continence. Salom\\u00f3n Hakim first identified the syndrome in 1957 at the Hospital San Juan de Dios in Bogot\\u00e1, Colombia. Although several reports of cases with the characteristic clinical manifestations of normal pressure hydrocephalus--progressive dementia, gait difficulty and urinary incontinence--have been published earlier, it was Adams and Hakim who emphasized the clinical triad and the effect of shunting the cerebrospinal fluid as a means of treatment.Although several reports of cases with the characteristic clinical manifestations of normal pressure hydrocephalus--progressive dementia, gait difficulty and urinary incontinence--have been published earlier, it was Adams and Hakim who emphasized the clinical triad and the effect of shunting the cerebrospinal fluid as a means of treatment.Chronic (normotensive or low pressure) hydrocephalus is characterized clinically by gait disturbance, cognitive and urinary impairment, known as Hakim's triad.Triad of Hakim--Adams is well known for normal pressure hydrocephalus (NPH): dementia, gait disturbances and urinary incontinenceAlthough several reports of cases with the characteristic clinical manifestations of normal pressure hydrocephalus--progressive dementia, gait difficulty and urinary incontinence--have been published earlier, it was Adams and Hakim who emphasized the clinical triad and the effect of shunting the cerebrospinal fluid as a means of treatmentBACKGROUND: Chronic (normotensive or low pressure) hydrocephalus is characterized clinically by gait disturbance, cognitive and urinary impairment, known as Hakim's triad. \"}, {\"question\": \"What kind of bonds are connecting keratin molecules?\", \"ideal_answer\": [\"cystine disulfide bonds\\namide bonds\\nhydrogen bonds\"], \"exact_answer\": [[\"cystine disulfide bonds\"], [\"amide bonds\"], [\"hydrogen bonds\"]], \"context\": \" Feathers and hair consist of cornified epidermal keratinocytes in which proteins are crosslinked via disulfide bonds between cysteine residues of structural proteins to establish mechanical resilience. The method involves the generation of thiols by controlled reduction of cystine disulfide bonds in keratinossible chemical bonds formed between alpha-keratins and KAbetaPs may derive from electrostatic interactions in addition to cross-linking through disulphide bonds.The method involves the generation of thiols by controlled reduction of cystine disulfide bonds in the keratine solubility test in the dithiothreitol/urea extraction buffer, the amino acid composition analysis and studies on keratin secondary structures suggest that the improved stability in water of thermally treated mats can be ascribed to the formation of amide bonds between acid and basic groups of some amino acid side chains.s the energy involved in the formation of disulfide bonds is much greater than that of hydrogen bonds or van der Waals interactions the structural transition is likely to be dominated by the requirement that the bonded cysteine residues occur at closely equivalent axial positions.The interface of the K5-K14 coiled-coil heterodimer has asymmetric salt bridges, hydrogen bonds and hydrophobic contacts, and its surface exhibits a notable charge polarization. A trans-dimer homotypic disulfide bond involving Cys367 in K14's stutter region occurs in the crystal and in skin keratinocytes, where it is concentrated in a keratin filament cage enveloping the nucleus.  the alpha helices which are stabilized by hydrogen bonds and the alpha-helical coiled coils which are stabilized by hydrophobic interactions, is more sensitive to radiation than the supramolecular architecture of the keratin filament and the filament packing within the keratin associated proteins matrix, which is stabilized by disulphide bonds.. A good solvent attacks the disulfide bonds between cystine molecules and hydrates the hair shaft.\"}, {\"question\": \"Which genes have been proposed as potential candidates for gene therapy of heart failure?\", \"ideal_answer\": [\"There are at least 6 genes which have been proposed as potential candidates of gene therapy in heart failure.\\n1. Cardiac Sarco-Endoplasmic Reticulum Calcium ATPase 2A (SERCA2A)\\n2. Inhibitor 1 (I-1) of Protein Phosphatase 1B\\n3. Protein Phosphatase 1B (PP1B)\\n4. Yes Associated Protein (YAP)\\n5. Survivin\\n6. S100A1\"], \"exact_answer\": [[\"Sarco-Endoplasmic Reticulum Calcium ATPase 2A\", \"(SERCA2A)\"], [\"Inhibitor 1 of Protein Phosphatase 1B\", \"(I-1)\"], [\"Protein Phosphatase 1B\", \"(PP1B)\"], [\"Yes Associated Protein\", \"(YAP)\"], [\"Survivin\"], [\"S100A1\"]], \"context\": \"The molecular abnormalities underlying HF are discussed along with potential targets for gene therapy, focusing on SERCA2a.The aim of this study was to improve cardiac function in chronic HF by overexpressing constitutively active inhibitor-1 (I-1c) In summary, I-1c overexpression using BNP116 improves cardiac function in a clinically relevant model of ischemic HF.therapeutic activation of YAP or its downstream targets, potentially through AAV-mediated gene therapy, may be a strategy to improve outcome after MIcalcium up-regulation by AAV1/SERCA2a gene therapy is safe and of potential benefit in advanced heart failure patients. Survivin gene therapy can attenuate the progression of LV systolic dysfunction in doxorubicin cardiomyopathy.Heart failure-inducible molecular targeting of PP1\\u03b2 has potential as a novel therapeutic strategy for heart failure.The recent successful and safe completion of a phase 2 trial targeting the sarcoplasmic reticulum calcium ATPase pump (SERCA2a), along with the start of more recent phase 1 trials, opens a new era for gene therapy for the treatment of heart failure.Our results present a strong rationale for a clinical trial of S100A1 gene therapy for human heart failure that could potentially complement current strategies to treat end-stage heart failure.S100A1, a positive inotropic regulator of myocardial contractility, becomes depleted in failing cardiomyocytes in humans and animals, and myocardial-targeted S100A1 gene transfer rescues cardiac contractile function by restoring sarcoplasmic reticulum calcium (Ca(2+)) handling in acutely and chronically failing hearts in small animal models.Use of gene therapy for heart failure is gaining momentum as a result of the recent successful completion of phase II of the Calcium Upregulation by Percutaneous Administration of Gene Therapy in Cardiac Disease (CUPID) trial, which showed clinical safety and efficacy of an adeno-associated viral vector expressing sarco-endoplasmic reticulum calcium ATPase (SERCA2a).The recent successful and safe completion of a phase 2 trial targeting the cardiac sarcoplasmic/endoplasmic reticulum Ca2+ ATPase pump (SERCA2a) has the potential to open a new era for gene therapy for heart failure.The recent successful and safe completion of a phase 2 trial targeting the sarcoplasmic reticulum calcium ATPase pump (SERCA2a) has the potential to open a new era for gene therapy in the treatment of heart failure.AIMS: Impaired myocardial sarcoplasmic reticulum calcium ATPase 2a (SERCA2a) activity is a hallmark of failing hearts, and SERCA2a gene therapy improves cardiac function in animals and patients with heart failure (HF).\"}, {\"question\": \"Which genes have been found to be associated with restless leg syndrome\", \"ideal_answer\": [\"Human L-Ferritin\\nThe genotypes of five specific single-nucleotide polymorphisms (SNPs) in three genes\\nHomozygosity for the T-allele of BTBD9 rs9296249\\nMEIS1\\nIntragenic guanosine triphosphate cyclohydrolase-1 duplication\\nLRRK2 gene mutation\"], \"exact_answer\": [[\"LRRK2 gene mutation\"], [\"Human L-Ferritin\"], [\"The genotypes of five specific single-nucleotide polymorphisms (SNPs) in three genes\"], [\"Homozygosity for the T-allele of BTBD9 rs9296249\"], [\"MEIS1\"], [\"Intragenic guanosine triphosphate cyclohydrolase-1 duplication\"]], \"context\": \"We describe a unique case of a 23-yr-old female patient affected by a homozygous loss of function mutation in the L-ferritin gene, idiopathic generalized seizures, and atypical restless leg syndrome (RLS). The genotypes of five specific single-nucleotide polymorphisms (SNPs) in three genes that have been previously associated with iron status and/or restless leg syndrome (RLS)Homozygosity for the T-allele of BTBD9 rs9296249 was associated with lower serum ferritin. A frequent polymorphism in BTBD9 was significantly associated with serum ferritin. This polymorphism has previously been associated with RLS, but not low iron stores in blood donors. In the German sample, variants in MEIS1 and BTBD9 were associated with RLS in ESRD (P(nom)\\u22640.004, ORs 1.52 and 1.55), whereas, in the Greek sample, there was a trend for association to MAP2K5/SKOR1 and BTBD9 This is the first demonstration of a genetic influence on RLS in ESRD patients with BTBD9 being significantly associated.Four family members developed dopa-responsive dystonia, with onset in their late teens, and subsequently developed restless leg syndrome and migraine. CONCLUSIONS: This is the first report of an intragenic guanosine triphosphate cyclohydrolase-1 duplication in a dopa-responsive dystonia family.To date, no gene mutation has been found, but five gene loci have been mapped in primary RLS to chromosomes 12q, 14q, 9p, 2q, and 20p (RLS1 through 5)We demonstrate linkage to a locus on chromosome 9p that is probably distinct from RLS3. Our family with a rather homogeneous phenotype and very early disease onset represents a unique opportunity to further elucidate the genetic causes of the frequent restless leg syndrome.A case of restless leg syndrome in a family with LRRK2 gene mutation.The association between RLS and LRRK2 gene mutation may be casual, but it can hypothesized that RLS is a possible phenotypic presentation in PARK8.\"}, {\"question\": \"What are the side effects of Nalmefene?\", \"ideal_answer\": [\"Side effects of nalmefene include nausea, dizziness / lightheadedness, insomina, fatigue, vomiting, reduced caloric intake / apetite, increased self-rated alertness and decreased tiredness. In horses some passage of semifluid fecal material, intermittent penile relaxation, and mild sedation has been described. In some studies nalmefene was well tolerated by all subjects, and no clinically significant adverse effects were observed.\"], \"exact_answer\": [[\"nausea\"], [\"dizziness / lightheadedness\"], [\"insomnia\"], [\"fatigue\"], [\"vomiting\"], [\"reduced caloric intake / apetite\"], [\"increased self-rated alertness\"], [\"decreased tiredness\"], [\"(horses) some passage of semifluid fecal material\"], [\"(horses) intermittent penile relaxation\"], [\"(horses) mild sedation\"]], \"context\": \"Adverse experiences included nausea, dizziness, and insomnia.Side effects to nalmefene were of greater duration and intensity in the subjects receiving 10 mg of nalmefene vs. those receiving 6 or 2 mg. These included most notably fatigue, lightheadedness, nausea and vomiting.The nalmefene treated group ate 22% less, both in terms of absolute weight and caloric intake, of a standardised buffet-meal than did the placebo group. Thus the apparent nutrient specificity of nalmefene appeared to be an indirect consequence of its effect on palatability. Nalmefene also caused slight increases in self-rated alertness, and decreases in ratings of tiredness and elation, although it was thought unlikely that these accounted for observed changes in eating behaviour.In the second study, six healthy men were initially administered a single 50-mg dose of drug, and plasma samples were obtained at selected time intervals for 48 hours. A dosing schedule of 20 mg q12h was then started and continued for seven days. Plasma samples were collected immediately before each dose and at selected times for up to 48 hours after the last dose. The drug was well tolerated by all subjects, and no clinically significant adverse effects were observed during the seven-day administration period. In the first study, single ascending doses of 50, 100, 200, and 300 mg of nalmefene HCl were administered in double-blind fashion to four groups of healthy men. There were six subjects in each group; four received nalmefene and two received placebo. The drug was well tolerated at all dose levels with only mild and transient side effects, such as lightheadedness, at the higher doses. Doses of nalmefene as large as 0.4 mg/kg, IV, produced only minor side effects. These side effects included some passage of semifluid fecal material, intermittent penile relaxation, and mild sedation. Treated horses responded normally to external stimuli, retained their appetites, and performed appropriately when ridden. Sedation wore off during the course of prolonged infusions. The drug was well tolerated at all dose levels with only mild and transient side effects, the most common of which was lightheadedness.\"}, {\"question\": \"Which are the most common methods for gene prioritization analysis?\", \"ideal_answer\": [\"Functional annotation-based approaches and literature-based approaches have been initially used. In recent years, network-based methods - which utilize a knowledge network derived from biological knowledge - have been utilized for gene prioritization. Currently network-based methods are the ones most widely used.\"], \"exact_answer\": [[\"network-based methods\"]], \"context\": \"We developed a new network inference algorithm called the Knowledge Network Gene Prioritization (KNGP) algorithm which can incorporate both link and node knowledge In recent years, network-based methods - which utilize a knowledge network derived from biological knowledge - have been utilized for gene prioritizationWe describe a bioinformatics approach, together with a freely accessible, interactive and flexible software termed Endeavour, to prioritize candidate genes underlying biological processes or diseases, based on their similarity to known genes involved in these phenomena We intend to use the ensemble boosting learning techniques to combine variant computational approaches for gene prioritization in order to improve the overall performance.The experimental results show that our ensemble learning approach outperforms the four gene-prioritization methods in ToppGene suite in the ranking results of the 13 known genes in terms of mean average precision, ROC and AUC measuresThe proposed enhanced RaJoLink rare-term model combines text mining and gene prioritization approaches. Computational gene prioritization is based on various pieces of correlative evidence that associate each gene with the given disease and suggest possible causal linksRecently, many network-based methods have been proposed that implicitly utilize the modularity principle, which states that genes causing the same or similar diseases tend to form physical or functional modules in gene/protein relationship networksthe random walk with restart (RWR) algorithm is considered to be a state-of-the-art approach, but the modularity principle has not been fully considered in traditional RWR approachesTherefore, we propose a novel method called ORIENT (neighbor-favoring weight reinforcement) to improve the performance of RWR through proper intensification of the weights of interactions close to the known disease genesNetwork-based methods have been successfully exploiting this concept by capturing the interaction of genes or proteins into a score Over the last years, computational approaches exploiting interaction network topology have been successfully applied to prioritize individual genes involved in diseases.Here, we propose a genome-wide network-based prioritization framework named GUILDThe traditional approach to reduce the number of candidate genes entails fine-mapping studies using markers and pedigreesFor this purpose, we evaluate our rule-based evolutionary machine learning systems, BioHEL and GAssist, on three public microarray cancer datasets, obtaining simple rule-based models for sample classification. hus, comparing the similarity between experimentally identified phenotypes and the phenotypes associated with human diseases can be used to suggest causal genes underlying a disease. In this manuscript, we present a method for disease gene prioritization based on comparing phenotypes of mouse models with those of human diseases.We present biomarker identification problem using gene prioritization method called gene prioritization from microarray data based on shortest paths, extended with structural and biological properties and edge flux using voting scheme (GP-MIDAS-VXEF).ensemble approach to microarray data-based gene prioritization after missing value imputationRecently, we have introduced the data mining tool ENDEAVOUR (Aerts et al., 2006), which performs this task automatically by relying on different genome-wide data sources, such as Gene Ontology, literature, microarray, sequence and moreSo far, biologists have relied on literature studies, extensive queries to multiple databases and hunches about expected properties of the disease gene to determine such an orderingWe study the effect of different data integration methods, and based on the validation studies, we show that our approach, ToppGene http://toppgene.cchmc.org, outperforms two of the existing candidate gene prioritization methods, SUSPECTS and ENDEAVOU Extending on an earlier hypothesis that the majority of genes that impact or cause disease share membership in any of several functional relationships we, for the first time, show the utility of mouse phenotype data in human disease gene prioritizationWe use co-expression data from yeast (S. cerevisiae), nematode worm (C. elegans), fruit fly (D. melanogaster), mouse and human and find that the use of evolutionary conservation can indeed improve the predictive value of co-expressionUsing a training set of genes known to be involved in a biological process of interest, our approach consists of (i) inferring several models (based on various genomic data sources), (ii) applying each model to the candidate genes to rank those candidates against the profile of the known genes and (iii) merging the several rankings into a global ranking of the candidate genesvocabularies, representations and ranking algorithms for gene prioritization by text miningRecently, text mining techniques have been applied to extract prior knowledge from text-based genomic information sources and this knowledge can be used to improve the prioritization processWe present an approach to prioritize single nucleotide polymorphisms for further follow-up in genome-wide association studies of type 2 diabetesThe proposed method combines both the use of open data access from two type 2 diabetes-genome-wide association studies (granted by the Diabetes Genetics Initiative and the Welcome Trust Case Control Consortium) and the comprehensive analysis of candidate regions generated by the freely accessible ENDEAVOUR software.ToppGene Suite (http://toppgene.cchmc.org; this web site is free and open to all users and does not require a login to access) is a one-stop portal for (i) gene list functional enrichment, (ii) candidate gene prioritization using either functional annotations or network analysis and (iii) identification and prioritization of novel disease candidate genes in the interactome.The protein-protein interaction network (PPIN)-based disease candidate gene prioritization uses social and Web networks analysis algorithms (extended versions of the PageRank and HITS algorithms, and the K-Step Markov method)We present gene prioritization system (GPSy), a cross-species gene prioritization system that facilitates the arduous but critical task of prioritizing genes for follow-up functional analysesGPEC: a Cytoscape plug-in for random walk-based gene prioritization and biomedical evidence collectionAmong these, network-based approaches are recently proposed and outperformed functional annotation-based onesIn the plug-in, gene prioritization is performed through a random walk with restart algorithm, a state-of-the art network-based method, along with a gene/protein relationship networHere we present a multi-dimensional evidence-based candidate gene prioritization approach for complex diseases and demonstrate it in schizophreniaWe present a multi-view approach to retrieve biomedical knowledge using different controlled vocabulariesGiven a query list, GeneMANIA extends the list with functionally similar genes that it identifies using available genomics and proteomics dataGenomic linkage and association studies are commonly performed for identifying disease-related geneHere, we introduce MedSim, a novel approach for ranking candidate genes for a particular disease based on functional comparisons involving the Gene OntologyMedSim uses functional annotations of known disease genes for assessing the similarity of diseases as well as the disease relevance of candidate genesHere, we report a large-scale analysis of spatial, i.e. 3D, gene-expression data from an entire organ (the mouse brain) for the purpose of evaluating and ranking positional candidate genes, showing that the spatial gene-expression patterns can be successfully exploited for the prediction of gene-phenotype associations not only for mouse phenotypes, but also for human central nervous system-related Mendelian disordersIn this paper, we propose an expandable framework for gene prioritization that can integrate multiple heterogeneous data sources by taking advantage of a unified graphic representationWe demonstrate that existing methods are likely to favor highly connected genes, making prioritization sensitive to the skewed degree distribution of PPI networks, as well as ascertainment bias in available interaction and disease association dataMotivated by this observation, we propose several statistical adjustment methods to account for the degree distribution of known disease and candidate genes, using a PPI network with associated confidence scores for interactionsHere we employ a previously described method of candidate gene prioritization based mainly on gene annotation, in accompaniment with a technique based on the evaluation of pertinent sequence motifs or signatures, in an attempt to refine the gene prioritization approach.We introduce the novel G\\u00e9nie algorithm that overcomes these problems by evaluating the literature attached to all genes in a genome and to their orthologs according to a selected topicPINTA (available at http://www.esat.kuleuven.be/pinta/; this web site is free and open to all users and there is no login requirement) is a web resource for the prioritization of candidate genes based on the differential expression of their neighborhood in a genome-wide protein-protein interaction networkWe have recently developed a computational method for constitutional genetic disorders that identifies the most promising candidate genes by replacing prior knowledge by experimental data of differential gene expression between affected and healthy individual\"}, {\"question\": \"Which are the major phycobiliproteins present in cyanobacteria?\", \"ideal_answer\": [\"Phycobiliproteins are derived from the photosynthetic apparatus of cyanobacteria and eukaryotic algae, and form their large extrinsic antenna complexes called phycobilisomes. Phycobilisomes have a core composed from allophycocyanin (APC) and rods, which are of variable phycobiliprotein composition. C-Phycocyanin (C-Pc) is one of the major light harvesting biliprotein pigments constitutively produced by many cyanobacteria, such as Spirulina platenesis (a blue-green alga). B-Phycoerythrin (B-PE) is an other major light-harvesting pigment found in red algae and cyanobacteria. R-phycoerythrin (R-PE) is the major light-harvesting pigment protein of most red algal phycobilisomes.\"], \"exact_answer\": [[\"allophycocyanin (APC)\"], [\"C-Phycocyanin (C-Pc)\"], [\"B-Phycoerythrin (B-PE)\"], [\"R-phycoerythrin (R-PE)\"]], \"context\": \"An extensive range of pigments including phycobiliproteins are present in algae. C-phycocyanin (C-PC), a phycobiliprotein, is one of the key pigments of SpirulinaSpirulina platensis produces nutraceutical product C-phycocyanin (C-PC)C-Phycocyanin (C-Pc) is one of the major biliprotein pigments of unicellular cyanbacterium of Spirulina platenesisMarine Synechococcus owe their specific vivid color (ranging from blue-green to orange) to their large extrinsic antenna complexes called phycobilisomes, comprising a central allophycocyanin core and rods of variable phycobiliprotein composition.Three major pigment types can be defined depending on the major phycobiliprotein found in the rods (phycocyanin, phycoerythrin I or phycoerythrin II)Phycocyanin--a major phycobiliprotein constitutively produced by many cyanobacteria--R-phycoerythrin was isolated and purified from a red alga, Polysiphonia urceolata GrevC-phycocyanin (C-PC), one of the major phycobiliproteins of Spirulina platensis (a blue-green alga)C-Phycocyanin (C-PC), the major light harvesting biliprotein from Spirulina platensis is of greater importance because of its various biological and pharmacological properties.B-Phycoerythrin (B-PE) is a major light-harvesting pigment of microalgae.Phycoerythrin is a major light-harvesting pigment of red algae and cyanobacteria that is widely used as a fluorescent probe and analytical reagent. In this paper, B-phycoerythrin and R-phycocyanin in native state, from the red alga Porphyridium cruentum were obtained by an inexpensive and simple process.Phycobiliproteins are derived from the photosynthetic apparatus of cyanobacteria and eukaryotic algae.the three major phycobiliprotein types, namely allophycocyanin, phycocyanin, and phycoerythrinPhycoerythrin is a major light-harvesting pigment of red algae and cyanobacteria widely used as a fluorescent probe.allophycocyanin (APC)R-phycoerythrin is the major light-harvesting pigment protein of most red algal phycobilisomes.The purified protein had three absorption peaks at 498, 535, and 565 nm and displayed a fluorescence maximum at 580 nm, which was consistent with the typical spectrum of R-phycoerythrin. The purified R-PE was also identified with electrophoresis.Phycobilisomes, the major light-harvesting complexes of cyanobacteria are multimolecular structures made up of chromophoric proteins called phycobiliproteins and non chromophoric linker polypeptides.We report here the isolation and nucleotide sequence of the genes, cpeA and cpeB, which in Calothrix PCC 7601 encode the alpha and beta subunits of phycoerythrin, one of the major phycobiliproteins.The major light-harvesting complex in eukaryotic red algae and prokaryotic cyanobacteria is the phycobilisome, a water-soluble complex located on the outer surface of the photosynthetic membranes and composed of both pigmented phycobiliproteins (85%) and non-pigmented linker (15%) polypeptides.Phycocyanin is a major protein produced by cyanobacteria, but very few phycocyanin-producing strains have been reported.In the present work, phycocyanin (PC) and phycoerythrin (PE) from a Nostoc species are proposed as protein markers for electrophoretic techniques.Rod structure of a phycoerythrin II-containing phycobilisome. I. Organization and sequence of the gene cluster encoding the major phycobiliprotein rod components in the genome of marine Synechococcus sp. WH8020.We now present data showing that the allophycocyanin genes and a second set of phycocyanin genes are transcribed into major mRNAs of 1400 and 1600 bases, respectivelyThe amino acid sequences deduced from both rpeA and rpeB present strong homologies with those previously reported for phycoerythrin subunits of cyanobacteria, rhodophyta, and cryptomonadsThese transcripts are present in RNA isolated from cultures grown in red and green light, although lower levels of the 1600-base phycocyanin transcript are present in cells grown in green lightPhycocyanin is a major protein produced by cyanobacteria, but very few phycocyanin-producing strains have been reportedThe major light-harvesting complex in eukaryotic red algae and prokaryotic cyanobacteria is the phycobilisome, a water-soluble complex located on the outer surface of the photosynthetic membranes and composed of both pigmented phycobiliproteins (85%) and non-pigmented linker (15%) polypeptidesPhycocyanin--a major phycobiliprotein constitutively produced by many cyanobacteria--holds several promising applications in diagnostics, biomedical research, and therapeuticsPhycobilisomes, the major light-harvesting complexes of cyanobacteria are multimolecular structures made up of chromophoric proteins called phycobiliproteins and non chromophoric linker polypeptides\"}, {\"question\": \"What is the inheritance pattern of Emery-Dreifuss muscular dystrophy? \", \"ideal_answer\": [\"The inheritance pattern of Emery-Dreifuss muscular dystrophy (EDMD) can be X-linked, autosomal dominant or autosomal recessive.\"], \"exact_answer\": [[\"X-linked\"], [\"autosomal dominant\"], [\"autosomal recessive\"]], \"context\": \". Loss-of-function truncation mutations in EMD, encoding the nuclear membrane protein emerin, cause X-linked Emery-Dreifuss muscular dystrophy (EDMD) characterized by localized contractures and skeletal myopathy in adolescence, sinus node dysfunction (SND) in early adulthood, and atrial fibrillation as a variably associated trait.Mutation of EMD can underlie X-linked familial AF. Lys37del is associated with epithelial cell emerin deficiency, as in EDMD, yet it causes electrical atriomyopathy in the absence of skeletal muscle disease. Targeted genetic testing of EMD should be considered in patients with SND-associated AF and/or family history suggesting X-linked inheritance.The Emery-Dreifuss muscular dystrophy is a form of muscular dystrophy that frequently presents early contractures and cardiac conduction defects, caused by emerin deficiency in the inner nuclear membrane of the muscular fibers.compatible with X-linked inheritance formThe first patient was a member of a family with molecularly proven X-linked EDMD.As these resemble the cardiac features of patients with the autosomal dominant variant of EDMD, we examined the lamin A/C gene, identifying a de-novo mutation in the propositus.contribute to disease severity in autosomal dominant EDMD.The STA gene encodes emerin and is one of the genes that is affected in Emery-Dreifuss muscular dystrophy (EDMD). Although it has been reported that EDMD caused by the STA gene mutation is associated with X-linked recessive inheritance, the genotype-phenotype correlations, with special reference to cardiac manifestations, are not well defined.Emery-Dreifuss muscular dystrophy (EDMD) is a muscular disorder characterized by 1) early contracture of the elbows. Achilles tendons and post-cervical muscles, 2) slowly progressive muscle wasting and weakness with a humeroperoneal distribution, and 3) life-threatening cardiomyopathy with conduction block. Most of families with EDMD show X-linked recessive inheritance with mutations in the STA gene on chromosome Xq28, which encodes a protein named emerin. A rare autosomal dominant form of EDMD (AD-EDMD) is caused by mutations in lamin A/C gene (LMNA) on chromosome 1q21. Both emerin and lamin A/C are located in the inner surface membrane of the nucleus.Emery-Dreifuss muscular dystrophy is an X-linked recessive myopathy. The patient had no familial background of the disease. This patient might have a sporadic inheritance pattern with severe cardiac involvement.autosomal dominant Emery-Dreifuss muscular dystrophy,Emery-Dreifuss muscular dystrophy is characterized by the clinical triad of early onset contractures of elbows, Achilles tendons and spine, wasting and weakness with a predominantly humero-peroneal distribution and life-threatening cardiac conduction defects and/or cardiomyopathy. Two main types of inheritance have been described: the X-linked form is caused by mutations in the STA gene on locus Xq28 and the gene for the autosomal dominant form (LMNA gene) has been localized on chromosome 1q11-q23. Recently, mutations in this LMNA gene have been also found to be responsible for the less frequent autosomal recessive form of the disease.Emery-Dreifuss muscular dystrophy (EDMD) is characterized by early contractures of elbows and Achilles tendons, slowly progressive muscle wasting and weakness, and a cardiomyopathy with conduction blocks which is life-threatening. Two modes of inheritance exist, X-linked (OMIM 310300) and autosomal dominant (EDMD-AD; OMIM 181350). EDMD-AD is clinically identical to the X-linked forms of the disease. Mutations in EMD, the gene encoding emerin, are responsible for the X-linked form.including X-linked Emery-Dreifuss MD,this form is not X-linked Emery-Dreifuss MD. We suggest that these patients represent a severe MD characterized by early onset distal wasting and severe rigidity of the spine, with probable autosomal recessive inheritance.Seventeen families with Emery-Dreifuss muscular dystrophy (EDMD) have been studied both by DNA sequencing and by emerin protein expression. Fourteen had mutations in the X-linked emerin gene, while three showed evidence of autosomal inheritance. Twelve of the 14 emerin mutations caused early termination of translation.One family presented a rare autosomal dominant variant of Emery-Dreifuss muscular dystrophy, another with X-linked recessive inheritance showed unusual intrafamilial variability.Emery-Dreifuss muscular dystrophy (EMD) is characterised by (1) early contractures of the Achilles tendons, elbows, and postcervical muscles, (2) slowly progressive muscle wasting and weakness with a predominantly humeroperoneal distribution in the early stages, and (3) cardiomyopathy with conduction defects and risk of sudden death. Inheritance is usually X linked recessive but can be autosomal dominant.Tendon contractures may be a partial expression of this myopathic disorder, suggesting an autosomal dominant inheritance with variable penetrance. A muscular dystrophy clinically similar to that of the Emery-Dreifuss (EDMD) type can thus occur in women.Since the disease was diagnosed in 3 brothers, the X-coupled recessive type of its inheritance is assumed. An opinion is advanced that the described form is a clinical variety of Emery-Dreyfus myodystrophy.Emery-Dreifuss syndrome (EDS)the term Emery-Dreifuss muscular dystrophy should be avoided. Instead, each case of EDS should be classified as myopathic or neurogenic with X chromosome recessive or autosomal dominant inheritance.Emery-Dreifuss muscular dystrophy is a syndrome with five salient features: early and unusual contractures; humeroperoneal muscle wasting; the slow progression of weakness, beginning in childhood; cardiac conduction defects; and X-linked inheritance. We present two cases and detail other reports with a similar constellation of findings with apparent autosomal dominant inheritance.Emery-Dreifuss muscular dystrophy with proximal weakness in both the upper and lower limbs and X-linked scapuloperoneal muscular dystrophy represent the same disorder.There is at least one other report of autosomal dominant transmission of this clinical picture, which had previously only been reported as Emery-Dreifuss muscular dystrophy with X-linked recessive inheritance.\"}, {\"question\": \"Which genes are regulated by TRalpha2  in the heart?\", \"ideal_answer\": [\"ARB1, ARB2, TAK1, p38, TRalpha1\"], \"exact_answer\": [[\"ARB1\"], [\"ARB2\"], [\"TAK1\"], [\"p38\"], [\"TRalpha1\"]], \"context\": \"n FM the reduced mRNA expression of ARB1 (p<0.05, -37%) and ARB2 (p<0.05, -42%) was associated with a reduction of the messenger for TRalpha1 (p<0.05, -85%) and TRalpha2 (p<0.05, -73%).hese data reveal that in human heart failure the reduction of beta-adrenergic receptors is associated with reduced expression of both TRalpha1 and TRalpha2 isoforms of thyroid hormone receptors.Mitigating TRalpha1 effects, both TRalpha2 and TRbeta1 attenuate TRalpha1-induced myocardial growth and gene expression by diminishing TAK1 and p38 activities, respectively.Selective ablation of TRalpha2 resulted in an inevitable, concomitant overexpression of TRalpha1.These data reveal that in human heart failure the reduction of beta-adrenergic receptors is associated with reduced expression of both TRalpha1 and TRalpha2 isoforms of thyroid hormone receptors.\"}, {\"question\": \"List available methods for transmembrane protein topology prediction.\", \"ideal_answer\": [\"HMMpTM, MetaTM, Philius, HMM_RA, HMMTOP, MEMSAT3, HMM-TM, TMHMM, Phobius and SignalP.\"], \"exact_answer\": [[\"HMMpTM\"], [\"MetaTM\"], [\"Philius\"], [\"HMM_RA\"], [\"HMMTOP\"], [\"MEMSAT3\"], [\"HMM-TM\"], [\"TMHMM\"], [\"Phobius\"], [\"SignalP\"]], \"context\": \"HMMpTM: improving transmembrane protein topology prediction using phosphorylation and glycosylation site predictionWe report the development of a Hidden Markov Model based method, capable of predicting the topology of transmembrane proteins and the existence of kinase specific phosphorylation and N/O-linked glycosylation sites along the protein sequence. Our method integrates a novel feature in transmembrane protein topology prediction, which results in improved performance for topology prediction and reliable prediction of phosphorylation and glycosylation sites. The method is freely available at http://bioinformatics.biol.uoa.gr/HMMpTMMetaTM - a consensus method for transmembrane protein topology prediction.A novel TM consensus method, named MetaTM, is proposed in this work. MetaTM is based on support vector machine models and combines the results of six TM topology predictors and two signal peptide predictors. On a large data set comprising 1460 sequences of TM proteins with known topologies and 2362 globular protein sequences it correctly predicts 86.7% of all topologies. CONCLUSION: Combining several TM predictors in a consensus prediction framework improves overall accuracy compared to any of the individual methods. Our proposed SVM-based system also has higher accuracy than a previous consensus predictor. MetaTM is made available both as downloadable source code and as DAS server at http://MetaTM.sbc.su.seWe present a support vector machine-based (SVM) TM protein topology predictor that integrates both signal peptide and re-entrant helix prediction, benchmarked with full cross-validation on a novel data set of 131 sequences with known crystal structures. The method achieves topology prediction accuracy of 89%, while signal peptides and re-entrant helices are predicted with 93% and 44% accuracy respectively. An additional SVM trained to discriminate between globular and TM proteins detected zero false positives, with a low false negative rate of 0.4%. We present the results of applying these tools to a number of complete genomes. Source code, data sets and a web server are freely available from http://bioinf.cs.ucl.ac.uk/psipred/Hidden Markov models (HMMs) have been successfully applied to the tasks of transmembrane protein topology prediction and signal peptide prediction. In this paper we expand upon this work by making use of the more powerful class of dynamic Bayesian networks (DBNs). Our model, Philius, is inspired by a previously published HMM, Phobius, and combines a signal peptide submodel with a transmembrane submodel. We introduce a two-stage DBN decoder that combines the power of posterior decoding with the grammar constraints of Viterbi-style decoding. Philius also provides protein type, segment, and topology confidence metrics to aid in the interpretation of the predictionsWe report a relative improvement of 13% over Phobius in full-topology prediction accuracy on transmembrane proteins, and a sensitivity and specificity of 0.96 in detecting signal peptidesHMM_RA: an improved method for alpha-helical transmembrane protein topology predictionThis paper presents a Hidden Markov Model (referred to as HMM_RA) that can predict the topology of alpha-helical transmembrane proteins with improved performance. HMM_RA adopts the same structure as the HMMTOP method, which has five modules: inside loop, inside helix tail, membrane helix, outside helix tail and outside loop. Each module consists of one or multiple statesA new method (MEMSAT3) for predicting transmembrane protein topology from sequence profiles is described and benchmarked with full cross-validation on a standard data set of 184 transmembrane proteinsThe algorithms presented here, are easily implemented in any kind of a Hidden Markov Model, whereas the prediction method (HMM-TM) is freely available for academic usersTMHMM, the current state-of-the-art method, has less than 52% accuracy in topology prediction on one set of transmembrane proteins of known topology. Based on the observation that there are functional domains that occur preferentially internal or external to the membrane, we have extended the model of TMHMM to incorporate functional domains, using a probabilistic approach originally developed for computational gene finding. Our extension is better than TMHMM in predicting the topology of transmembrane proteins. As prediction of functional domain improves, our system's prediction accuracy will likely improve as wellHere, we present Phobius, a combined transmembrane protein topology and signal peptide predictor. The predictor is based on a hidden Markov model (HMM) that models the different sequence regions of a signal peptide and the different regions of a transmembrane protein in a series of interconnected states. Training was done on a newly assembled and curated dataset. Compared to TMHMM and SignalP, errors coming from cross-prediction between transmembrane segments and signal peptides were reduced substantially by Phobius. False classifications of signal peptides were reduced from 26.1% to 3.9% and false classifications of transmembrane helices were reduced from 19.0% to 7.7%. Phobius was applied to the proteomes of Homo sapiens and Escherichia coli. Here we also noted a drastic reduction of false classifications compared to TMHMM/SignalP, suggesting that Phobius is well suited for whole-genome annotation of signal peptides and transmembrane regionsHMMpTM: improving transmembrane protein topology prediction using phosphorylation and glycosylation site prediction.The HMMTOP transmembrane topology prediction server.\"}, {\"question\": \"List available genetic multicolor cell labeling techiniques in Drosophila\", \"ideal_answer\": [\"Flybow and Drosophila Brainbow.\"], \"exact_answer\": [[\"Flybow\"], [\"Drosophila Brainbow\"]], \"context\": \"Flybow: genetic multicolor cell labeling for neural circuit analysis in Drosophila melanogaster.To facilitate studies of neural network architecture and formation, we generated three Drosophila melanogaster variants of the mouse Brainbow-2 system, called Flybow. Sequences encoding different membrane-tethered fluorescent proteins were arranged in pairs within cassettes flanked by recombination sites. Flybow combines the Gal4-upstream activating sequence binary system to regulate transgene expression and an inducible modified Flp-FRT system to drive inversions and excisions of cassettes. This provides spatial and temporal control over the stochastic expression of one of two or four reporters within one sample. Using the visual system, the embryonic nervous system and the wing imaginal disc, we show that Flybow in conjunction with specific Gal4 drivers can be used to visualize cell morphology with high resolution. Finally, we demonstrate that this labeling approach is compatible with available Flp-FRT-based techniques, such as mosaic analysis with a repressible cell marker; this could further support the genetic analysis of neural circuit assembly and function.Drosophila Brainbow: a recombinase-based fluorescence labeling technique to subdivide neural expression patterns.We developed a multicolor neuron labeling technique in Drosophila melanogaster that combines the power to specifically target different neural populations with the label diversity provided by stochastic color choice. This adaptation of vertebrate Brainbow uses recombination to select one of three epitope-tagged proteins detectable by immunofluorescence. Two copies of this construct yield six bright, separable colors. We used Drosophila Brainbow to study the innervation patterns of multiple antennal lobe projection neuron lineages in the same preparation and to observe the relative trajectories of individual aminergic neurons. Nerve bundles, and even individual neurites hundreds of micrometers long, can be followed with definitive color labeling. We traced motor neurons in the subesophageal ganglion and correlated them to neuromuscular junctions to identify their specific proboscis muscle targets. The ability to independently visualize multiple lineage or neuron projections in the same preparation greatly advances the goal of mapping how neurons connect into circuits.Brainbow: new resources and emerging biological applications for multicolor genetic labeling and analysis.\"}, {\"question\": \"Which growth factors are known to be involved in the induction of EMT?\", \"ideal_answer\": [\"EMT is characterized by acquisition of cell motility, modifications of cell morphology, and cell dissociation correlating with the loss of desmosomes from the cellular cortex. A number of growth factors have been shown to be involved in this process. These include fibroblast growth factors (FGFs), TGF-\\u03b21, TGF-\\u03b22, TNF-\\u03b1, CCN family, Sonic Hedgehog (SHh), Notch1, GF-\\u03b2, Wnt, EGF, bFGF, IGF-I and IGF-II.\"], \"exact_answer\": [[\"fibroblast growth factors (FGFs)\"], [\"TGF-\\u03b21\"], [\"TGF-\\u03b22\"], [\"TNF-\\u03b1\"], [\"CCN family\"], [\"Sonic Hedgehog (SHh)\"], [\"Notch1\"], [\"GF-\\u03b2\"], [\"Wnt\"], [\"EGF\"], [\"bFGF\"], [\"IGF-I\"], [\"IGF-II\"]], \"context\": \"Moreover, recent studies have shown that most EMT cases are regulated by soluble growth factors or cytokines. Among these factors, fibroblast growth factors (FGFs) execute diverse functions by binding to and activating members of the FGF receptor (FGFR) family, including FGFR1-4.Fibroblast growth factor receptor 1 is an oncoprotein that is involved in tumorigenesis, and PD173074 is known to be a selective inhibitor of FGFR1.Fibroblast growth factor receptor 1 was also overexpressed in EMT cell lines compared with non-EMT cell lines.Furthermore, treatment of HOC313 cells with PD173074 suppressed cellular proliferation and invasion and reduced ERK1/2 and p38 activation.In addition, the expression levels of certain matrix metalloproteinases (MMPs), whose genes contain activator protein-1 (AP-1) promoter sites, as well as Snail1 and Snail2 were reduced following PD173074 treatment.Taken together, these data suggest that PD173074 inhibits the MAPK pathway, which regulates the activity of AP-1 and induces MET.egulation of Na,K-ATPase \\u03b21-subunit in TGF-\\u03b22-mediated epithelial-to-mesenchymal transition in human retinal pigmented epithelial cellsThe EMT process is mediated via exposure to vitreous cytokines and growth factors such as TGF-\\u03b22.Previous studies have shown that Na,K-ATPase is required for maintaining a normal polarized epithelial phenotype and that decreased Na,K-ATPase function and subunit levels are associated with TGF-\\u03b21-mediated EMT in kidney cells.loss of Na,K-ATPase \\u03b21 is a potential contributor to TGF-\\u03b22-mediated EMT in RPE cells.MT (epithelial-mesenchymal transition) is crucial for cancer cells to acquire invasive phenotypes. In A549 lung adenocarcinoma cells, TGF-\\u03b2 elicited EMT in Smad-dependent manner and TNF-\\u03b1 accelerated this process, as confirmed by cell morphology, expression of EMT markers, capacity of gelatin lysis and cell invasion.Comprehensive expression analysis unraveled genes differentially regulated by TGF-\\u03b2 and TNF-\\u03b1, such as cytokines, chemokines, growth factors and ECM (extracellular matrices), suggesting the drastic change in autocrine/paracrine signals as well as cell-to-ECM interactions. Our recent studies found that CCN1 plays a critical role in pancreatic carcinogenesis through the induction of EMT and stemness.Here we show that CCN1 regulates the Sonic Hedgehog (SHh) signaling pathway, which is associated with the PDAC progression and poor prognosis.SHh regulation by CCN1 in pancreatic cancer cells is mediated through the active Notch-1.These extensive studies propose that targeting CCN1 can provide a new treatment option for patients with pancreatic cancer since blocking CCN1 simultaneously blocks two critical pathways (i.e. SHh and Notch1) associated with the development of the disease as well as drug resistance.GF-\\u03b2-induced epithelial-mesenchymal transitionThe cytokine TGF-\\u03b2, which is expressed by tumor-infiltrating immune cells, stands out as a master regulator of the pro-invasive tumor microenvironment. TGF-\\u03b2 cooperates with stem cell pathways, such as Wnt and Ras signaling, to induce EMT. In addition, TGF-\\u03b2 contributes to an EMT-permissive microenvironment by switching the phenotypes of tumor-infiltrating immune cells, which thereby mount pro-invasive and pro-metastatic immune responses.In this review, we discuss the role of TGF-\\u03b2-induced EMT as a link between cancer and inflammation in the context of questions, which from our point of view are key to answer in order to understand the functionality of EMT in tumors.ranscription factors c-Myc and CDX2 mediate E-selectin ligand expression in colon cancer cells undergoing EGF/bFGF-induced epithelial-mesenchymal transitionTreatment of HT29 and DLD-1 cells with EGF and/or basic FGF (bFGF) induced EMT and significantly increased sLe(x/a) expression resulting in enhanced E-selectin binding activity.Upon hypoxia, TGF\\u03b2 addition or EGFRvIII expression, MCF7, A549 and NMuMG epithelial cells acquired a spindle shape and lost cell-cell contacts. Expression of epithelial markers such as E-cadherin decreased, whereas mesenchymal markers such as vimentin and N-cadherin increased.The EMT process can be regulated by a diverse array of cytokines and growth factors, such as transforming growth factor (TGF)-beta, whose activities are dysregulated during malignant tumor progression.Transforming growth factor-beta-induced tubular cell EMT in C1.1 cells was inhibited by MMP-2/9 inhibitor. Our in vitro study provides evidence that MMPs, specifically MMP-9, secreted by effector macrophages can induce tubular cell EMT and thereby contribute to renal fibrosis.nflammatory cytokines augments TGF-beta1-induced epithelial-mesenchymal transition in A549 cells by up-regulating TbetaR-IIn this study, we report that cytomix (a mixture of IL-1beta, TNF-alpha and IFN-gamma) significantly enhances TGF-beta1-induced EMT in A549 cells as evidenced by acquisition of fibroblast-like cell shape, loss of E-cadherin, and reorganization of F-actin.IL-1beta or TNF-alpha alone can also augment TGF-beta1-induced EMT. However, a combination of IL-1beta and TNF-alpha or the cytomix is more potent to induce EMT.These results indicate that inflammatory cytokines together with TGF-beta1 may play an important role in the development of fibrosis and tumor progress via the mechanism of epithelial-mesenchymal transition.TGF-beta stimulated epithelial to mesenchyme transdifferentiation (EMT) in the presence of TGF-alpha, as characterized by increased expression of fibronectin and changes in TGF-beta receptor binding.EMT is typically induced by transforming growth factor-beta1 (TGF-beta1) and inhibited by hepatocyte growth factor (HGF).Exposure of MDCK cells to 10 ng/ml TGF-beta1 for 72 h induced EMT as evidenced by conversion to the spindle-like morphology, loss of E-cadherin, and activation of alpha-smooth muscle actin.In contrast, Akt1 down-regulation in IGF-IR-stimulated cells promoted dramatic neomorphic effects characteristic of an epithelial-mesenchymal transition (EMT) and enhanced cell migration induced by IGF-I or EGF stimulation. The phenotypic effects of Akt1 down-regulation were accompanied by enhanced extracellular signal-related kinase (ERK) activation, which contributed to the induction of migration and EMT.ransforming growth factors beta (TGF-betas) inhibit growth of epithelial cells and induce differentiation changes, such as epithelial-mesenchymal transition (EMT).Ha-Ras cooperates with transforming growth factor beta (TGFbeta) to cause epithelial mesenchymal transition (EMT) characterized by spindle-like cell morphology, loss of epithelial markers, and induction of mesenchymal markers.IGF-II induces rapid beta-catenin relocation to the nucleus during epithelium to mesenchyme transitionWe can show that (1) IGF-II induces a rapid epithelium to mesenchymal transition;Based on the given case of IGF-II and E-cadherin/beta-catenin complex, this study reveals the backbone of a cascade connecting growth factor signaling with cell-cell adhesion during EMT.The NBT-II rat carcinoma cell line exhibits two mutually exclusive responses to FGF-1 and EGF, entering mitosis at cell confluency while undergoing an epithelium-to-mesenchyme transition (EMT) when cultured at subconfluency.EMT requires continuous TGFbeta receptor (TGFbeta-R) and oncogenic Ras signaling and is stabilized by autocrine TGFbeta productionEMT seems to be a close in vitro correlate of metastasis, both requiring synergism between TGFbeta-R and Raf/MAPK signaling\"}, {\"question\": \"Which histone modification discriminates between active and poised enhancers?\", \"ideal_answer\": [\"Monomethylation of histone H3 on Lys 4 (H3K4me1) and acetylation of histone H3 on Lys 27 (H3K27ac) are histone modifications that are highly enriched over the body of actively transcribed genes and on enhancers. \"], \"exact_answer\": [[\"H3K4me1\"], [\"H3K27ac\"]], \"context\": \"Monomethylation of histone H3 on Lys 4 (H3K4me1) and acetylation of histone H3 on Lys 27 (H3K27ac) are histone modifications that are highly enriched over the body of actively transcribed genes and on enhancerSince Trr and mammalian Mll3/4 complexes are distinguished by bearing a unique subunit, the H3K27 demethylase UTX, we propose a model in which the H3K4 monomethyltransferases Trr/Mll3/Mll4 and the H3K27 demethylase UTX cooperate to regulate the transition from inactive/poised to active enhancers.We used ChIP-seq to measure changes in histone H3K27 acetylation, a mark of active enhancers, to identify enhancers in myelinating rat peripheral nerve and their dynamics after demyelinating nerve injury.However, the majority of Egr2-bound enhancers retain H3K27ac, indicating that other transcription factors maintain active enhancer status after nerve injury.Recent studies using mammalian cells showed that a chromatin state signature is associated with active developmental enhancers, defined by high levels of histone H3 lysine 27 acetylation (H3K27ac) and strong depletion of H3K27 trimethylation (H3K27me3Such permissive binding was largely restricted to open-chromatin regions showing histone modification marks characteristic of active enhancer and promoter regions, whereas open-chromatin regions lacking such marks did not show permissive binding. We find that histone H3K27ac distinguishes active enhancers from inactive/poised enhancer elements containing H3K4me1 aloneThe bivalent hypothesis posits that genes encoding developmental regulators required for early lineage decisions are poised in stem/progenitor cells by the balance between a repressor histone modification (H3K27me3), mediated by the Polycomb Repressor Complex 2 (PRC2), and an activator modification (H3K4me3)Histone H3K27ac separates active from poised enhancers and predicts developmental stateHistone H3K27ac separates active from poised enhancers and predicts developmental state.We find that histone H3K27ac distinguishes active enhancers from inactive/poised enhancer elements containing H3K4me1 alone. Furthermore, we compare the ability of eRNAs and H3K27ac to discriminate enhancer activity. We demonstrate that eRNA is more indicative of enhancer activity.\"}, {\"question\": \"What are the properties of super-enhancers?\", \"ideal_answer\": [\"Super-enhancers differ from typical enhancers in size, transcription factor density and content, ability to activate transcription, and sensitivity to perturbation. Defined by their magnitude of size, transcription factor density, and binding of transcriptional machinery, super-enhancers have been associated with genes driving cell differentiation. In this respect, the super-enhancer definition is useful in identifying regulatory elements likely to control genes important for cell type specification. Super-enhancers thus play key roles in the control of mammalian cell identity.\"], \"exact_answer\": [[\"their magnitude of size\"], [\"transcription factor density\"], [\"binding of transcriptional machinery\"], [\"associated with genes driving cell differentiation\"]], \"context\": \"Here we report that enhancer RNAs (eRNAs) identified by global nuclear run-on sequencing are extensively transcribed within super enhancers and are dynamically regulated in response to cellular signaling.Inflammation-sensitive super enhancers form domains of coordinately regulated enhancer RNAsThe expression of genes that specify cell type identity and function is associated with densely spaced clusters of active enhancers known as super-enhancers. The functions of enhancers and super-enhancers are influenced by, and affect, higher-order genomic organization. Defined by their magnitude of size, transcription factor density, and binding of transcriptional machinery, super-enhancers have been associated with genes driving cell differentiation.Here we review evidence for super-enhancer involvement in cancers, complex diseases, and developmental disorders and discuss interactions between super-enhancers and cofactors/chromatin regulators.The term 'super-enhancer' has been used to describe groups of putative enhancers in close genomic proximity with unusually high levels of Mediator binding, as measured by chromatin immunoprecipitation and sequencing (ChIP-seq).In this respect, the super-enhancer definition is useful in identifying regulatory elements likely to control genes important for cell type specification.Super-enhancers (SEs), also known as stretch-enhancers, are a subset of enhancers especially important for genes associated with cell identity and genetic risk of disease. CD4(+) T cells are critical for host defence and autoimmunity.Super-enhancers and stretch enhancers (SEs) drive expression of genes that play prominent roles in normal and disease cells, but the functional importance of these clustered enhancer elements is poorly understood, so it is not clear why genes key to cell identity have evolved regulation by such elementsSuper-enhancers thus provide a platform for signaling pathways to regulate genes that control cell identity during development and tumorigenesis.Here we show that super-enhancers underlie the identity, lineage commitment and plasticity of adult stem cells in vivo.We show that super-enhancers and their dense clusters ('epicentres') of transcription factor binding sites undergo remodelling upon lineage progression.New fate is acquired by decommissioning old and establishing new super-enhancers and/or epicentres, an auto-regulatory process that abates one master regulator subset while enhancing another.Finally, we identify SOX9 as a crucial chromatin rheostat of hair follicle stem cell super-enhancers, and provide functional evidence that super-enhancers are dynamic, dense transcription-factor-binding platforms which are acutely sensitive to pioneer master regulators whose levels define not only spatial and temporal features of lineage-status but also stemness, plasticity in transitional states and differentiation.We demonstrate that hotspots are highly enriched in large super-enhancer regions (several kilobases), which drive the early adipogenic reprogramming of gene expression.Our results indicate that cooperativity between transcription factors at the level of hotspots as well as super-enhancers is very important for enhancer activity and transcriptional reprogramming. Thus, hotspots and super-enhancers\\u00a0constitute important regulatory hubs that serve to integrate external stimuli on chromatin.We report here that the ESC master transcription factors form unusual enhancer domains at most genes that control the pluripotent state. These domains, which we call super-enhancers, consist of clusters of enhancers that are densely occupied by the master regulators and Mediator.Super-enhancers differ from typical enhancers in size, transcription factor density and content, ability to activate transcription, and sensitivity to perturbationSuper-enhancers thus play key roles in the control of mammalian cell identity.Super-enhancers are large clusters of transcriptional enhancers that drive expression of genes that define cell identity.We describe here the population of transcription factors, cofactors, chromatin regulators, and transcription apparatus occupying super-enhancers in embryonic stem cells and evidence that super-enhancers are highly transcribed. Finally, we identify SOX9 as a crucial chromatin rheostat of hair follicle stem cell super-enhancers, and provide functional evidence that super-enhancers are dynamic, dense transcription-factor-binding platforms which are acutely sensitive to pioneer master regulators whose levels define not only spatial and temporal features of lineage-status but also stemness, plasticity in transitional states and differentiation.Inflammation-sensitive super enhancers form domains of coordinately regulated enhancer RNAs.NF-\\u03baB directs dynamic super enhancer formation in inflammation and atherogenesis.Pioneer factors govern super-enhancer dynamics in stem cell plasticity and lineage choice.\"}, {\"question\": \"Which are the cardiac effects of thyronamines?\", \"ideal_answer\": [\"Thyronamines have negative chronotropy, negative inotropy; in particular thyronamines are considered negative inotropic agents\", \"In the heart, thyronamines cause negative chronotropy, negative inotropy,reduced cardiac output and resistance to ischemic injury.\"], \"exact_answer\": [[\"negative chronotropy\", \"lower heart rate\"], [\"negative inotropy\", \"negative inotropism\"], [\"resistance to ischemic injury\"], [\"reduced cardiac output\"]], \"context\": \"Most recently, thyroid hormone derivatives were identified, the thyronamines which are decarboxylated thyroid hormones initiating physiological actions like lowering body temperature and heart rate, thereby acting in opposite direction to the classical thyroid hormones.Intraperitoneal or central injection of 3-T(1)AM or T(0)AM into mice, rats, or Djungarian hamsters caused various prompt effects, such as metabolic depression, hypothermia, negative chronotropy, negative inotropy, hyperglycemia, reduction of the respiratory quotient, ketonuria, and reduction of fat mass. Functional effects have been observed after administration of exogenous T(1)AM: in the isolated heart, a negative inotropic and chronotropic action was produced, and the resistance to ischemic injury was increased, possibly as a consequence of an action on intracellular calcium homeostasis.Octopamine, beta-phenylethylamine, and tryptamine produced a dose-dependent negative inotropic effect as shown by reduced cardiac output (IC(50)=109 microM, 159 microM, and 242 microM, respectively). In the same preparation a similar effect was produced by thyronamine and 3-iodothyronamine, with IC(50)=94 microM and 27 microM, respectively. A class of thyroid hormone metabolites has dramatic physiological effects on metabolism and heart rate by still-unknown mechanisms of action. In the mouse, thyronamines act rapidly in a nongenomic fashion to initiate hypothermia and decrease cardiac output and heart rate.\"}, {\"question\": \"Matuzumab has been tested for treatment of which cancers?\", \"ideal_answer\": [\"Matuzumab has been tested for treatment of non-small cell lung, gastric, esophageal, colorectal, primary peritoneal, pancreatic, ovarian and cervical cancers.\"], \"exact_answer\": [[\"non-small cell lung\"], [\"gastric\"], [\"esophageal\"], [\"colorectal\"], [\"primary peritoneal\"], [\"pancreatic\"], [\"ovarian\"], [\"cervical\"]], \"context\": \" Matuzumab and panitumumab have also been studied in phase II trials.A phase I pharmacokinetic study of matuzumab in combination with paclitaxel in patients with EGFR-expressing advanced non-small cell lung cancer.This phase I study investigated the tolerability, safety and pharmacokinetics (PK) of matuzumab in combination with paclitaxel in patients with EGFR-expressing advanced non-small cell lung cancer (NSCLC). Matuzumab combined with paclitaxel was generally well tolerated in patients with advanced NSCLC. This article focuses on anti-Her-2 therapy, specifically trastuzumab, as well as other epidermal growth factor receptor antagonists such as cetuximab, panitumub, matuzumab, nimotzumab, gefitinib, and erlotinib.Phase I study of matuzumab in combination with 5-fluorouracil, leucovorin and cisplatin (PLF) in patients with advanced gastric and esophagogastric adenocarcinomas.BACKGROUND: To evaluate the safety and tolerability of two different weekly doses of the fully humanized epidermal growth factor receptor (EGFR)-targeting monoclonal antibody matuzumab combined with high-dose 5-fluorouracil, leucovorin and cisplatin (PLF) in the first-line treatment of patients with EGFR-positive advanced gastric and esophagogastric adenocarcinomas.Matuzumab and panitumumab have also been evaluated in phase II trials. Pemetrexed with or without matuzumab as second-line treatment for patients with stage IIIB/IV non-small cell lung cancer.INTRODUCTION: This randomized phase II study investigated pemetrexed in combination with the epidermal growth factor receptor (EGFR)-targeting monoclonal antibody matuzumab compared with pemetrexed alone as second-line therapy for patients with advanced non-small cell lung cancer. Matuzumab plus epirubicin, cisplatin and capecitabine (ECX) compared with epirubicin, cisplatin and capecitabine alone as first-line treatment in patients with advanced oesophago-gastric cancer: a randomised, multicentre open-label phase II study.BACKGROUND: Clinical data showed promising antitumour activity with feasible tolerability for matuzumab plus epirubicin, cisplatin and capecitabine (ECX) chemotherapy in untreated advanced oesophago-gastric (OG) cancer.  CONCLUSION: Matuzumab 800 mg weekly combined with ECX chemotherapy does not increase response or survival for patients with advanced OG cancer.In all studies, the patients had different types of advanced carcinoma - mainly colon, rectal and pancreatic cancer. They received matuzumab as multiple 1-hour intravenous infusions in a wide range of dosing regimens (development dataset: from 400 mg every 3 weeks to 2000 mg in the first week followed by 1600 mg weekly; evaluation dataset: from 100 mg weekly to 800 mg weekly).  Other anti-EGFR monoclonal antibodies, such as panitumumab, matuzumab, nimotuzumab, and ch806, are in different stages of development for the treatment of advanced NSCLC.Molecular determinants of response to matuzumab in combination with paclitaxel for patients with advanced non-small cell lung cancer.Antibodies targeting epidermal growth factor receptor (EGFR) have proven to be effective in patients with non-small cell lung cancer (NSCLC) that express EGFR. We recently published a phase I study of weekly matuzumab plus paclitaxel.Our data suggest that EGFR expression and KRAS mutation status is predictive for clinical response to matuzumab +/- paclitaxel in patients with advanced NSCLC.Phase I study of epirubicin, cisplatin and capecitabine plus matuzumab in previously untreated patients with advanced oesophagogastric cancer.To evaluate the safety, tolerability, efficacy, pharmacokinetics and pharmacodynamics of the humanised antiepidermal growth factor receptor monoclonal antibody matuzumab combined with epirubicin, cisplatin and capecitabine (ECX) in patients as first-line treatment for advanced oesophagogastric cancer that express epidermal growth factor receptor (EGFR). Therefore, these first successes led to the development of several drugs including monoclonal antibodies (trastuzumab, panitumumab, matuzumab), TK inhibitors targeting one receptor as well as TK pan-inhibitors (lapatinib, HKI 272, PKI 166, EKB-569, AEE-788), currently assessed through clinical trials worldwide.Antibodies to the epidermal growth factor receptor in non small cell lung cancer: current status of matuzumab and panitumumab.Matuzumab and panitumumab are antibodies against the epidermal growth factor receptor (EGFR) that are being evaluated in several malignancies including non-small cell lung cancer (NSCLC). In phase I trials of single-agent matuzumab in patients with EGFR-positive cancer, three tumor responses were documented in esophageal squamous cell carcinoma as well as colorectal carcinoma.A phase II trial of EMD72000 (matuzumab), a humanized anti-EGFR monoclonal antibody, in patients with platinum-resistant ovarian and primary peritoneal malignancies.CONCLUSIONS: Matuzumab at the dose and schedule selected is well tolerated. In this population of very heavily pretreated patients with epithelial ovarian and primary peritoneal malignancies, there was no evidence of significant clinical activity when matuzumab was administered as monotherapy.Phase I study of the humanised anti-EGFR monoclonal antibody matuzumab (EMD 72000) combined with gemcitabine in advanced pancreatic cancer. Three groups of chemotherapy-naive advanced pancreatic adenocarcinoma patients (n=17) received escalating doses of matuzumab (400 mg weekly, 800 mg biweekly, or 800 mg weekly) and gemcitabine (1000 mg m-2 weekly in weeks 1-3 of each 4-week cycle). A phase I study of the humanized monoclonal anti-epidermal growth factor receptor (EGFR) antibody EMD 72000 (matuzumab) in combination with paclitaxel in patients with EGFR-positive advanced non-small-cell lung cancer (NSCLC).This phase I study assessed the tolerability, pharmacokinetics and efficacy of the combination of matuzumab and paclitaxel in patients with advanced NSCLC.Other anti-EGFR monoclonal antibodies (panitunumab, matuzumab) are currently evaluated. It is the target for a class of agents at the forefront of development for the treatment of colorectal cancer, ie, the anti-EGFR monoclonal antibodies, which include cetuximab, panitumumab, and matuzumab. Two other monoclonal antibodies, matuzumab (EMD 72000) and panitumumab (ABG-EGF), also have shown activity against EGFR-expressing CRC but are still in the early stage of clinical development.Matuzumab is currently undergoing phase II clinical trials for gastric, cervical, pancreatic and ovarian cancers.\"}, {\"question\": \"Which are the synonyms of prostate-specific antigen?\", \"ideal_answer\": [\"Prostate-specific antigen (PSA) is a 33 kDa single chain glycoprotein belonging to the kallikrein family of serine proteases which is produced by epithelial cells of both normal and malignant prostate tissue. PSA is an important marker for the diagnosis of prostate cancer. PSA is also known as human kallikrein-related peptidase 3 (hK3).\", \"Human kallikrein-related peptidase 3 (hK3), also known as prostate-specific antigen (PSA), is a 33 kDa single chain glycoprotein belonging to the kallikrein family of serine proteases. (PMID: 19079621)\"], \"exact_answer\": [[\"human kallikrein-related peptidase 3\"], [\"hK3\"]], \"context\": \"Evidence for the novel expression of human kallikrein-related peptidase 3, prostate-specific antigen, in the brain.Human kallikrein-related peptidase 3 (hK3), also known as prostate-specific antigen (PSA), is a 33 kDa single chain glycoprotein belonging to the kallikrein family of serine proteases.\"}, {\"question\": \"List available biomedical question answering systems.\", \"ideal_answer\": [\"We live in an age of access to more information than ever before. The exponential growth in the volume of publications in the biomedical domain has made it impossible for an individual to keep pace with the advances. Thus, there is a need for intelligent information retrieval systems that can summarize relevant and reliable textual sources to satisfy a user's query. Question answering is a specialized type of information retrieval with the aim of returning precise short answers to queries posed as natural language questions. This accentuates the need for fast and accurate biomedical question answering systems. In this paper we introduce INDOC -- a biomedical question answering system based on novel ideas of indexing and extracting the answer to the questions posed. Increased access to information allows for more informed and empowered researchers, while information overload becomes an increasingly serious risk. INDOC displays the results in clusters to help the user arrive the most relevant set of documents quickly. Evaluation was done against the standard OHSUMED test collection. We present a review and comparison of three biomedical question answering systems: askHERMES, EAGLi ( http://eagl.unige.ch/EAGLi/ ), and HONQA ( http://services.hon.ch/cgi-bin/QA10/qa.pl ). \", \"askHERMES, EAGLi, HONQA and INDOC.\"], \"exact_answer\": [[\"askHERMES\"], [\"EAGLi\"], [\"HONQA\"], [\"INDOC\"]], \"context\": \"Question answering is a specialized type of information retrieval with the aim of returning precise short answers to queries posed as natural language questions. We present a review and comparison of three biomedical question answering systems: askHERMES (http://www.askhermes.org/), EAGLi (http://eagl.unige.ch/EAGLi/), and HONQA (http://services.hon.ch/cgi-bin/QA10/qa.pl)Question processing and clustering in INDOC: a biomedical question answering systemIn this paper we introduce INDOC--a biomedical question answering system based on novel ideas of indexing and extracting the answer to the questions posed. INDOC displays the results in clusters to help the user arrive the most relevant set of documents quickly. Evaluation was done against the standard OHSUMED test collection. Our system achieves high accuracy and minimizes user effortQuestion processing and clustering in INDOC: a biomedical question answering system.In this paper we introduce INDOC--a biomedical question answering system based on novel ideas of indexing and extracting the answer to the questions posed.We present a review and comparison of three biomedical question answering systems: askHERMES (http://www.In this paper we introduce INDOC--a biomedical question answering system based on novel ideas of indexing and extracting the answer to the questions posed.We present a review and comparison of three biomedical question answering systems: askHERMES (http://www.askhermes.org/), EAGLi (http://eagl.unige.ch/EAGLi/), and HONQA (http://services.hon.ch/cgi-bin/QA10/qa.pl).We present a review and comparison of three biomedical question answering systems: askHERMES (http://www.askhermes.org/), EAGLi (http://eagl.unige.ch/EAGLi/), and HONQA (http://services.hon.ch/cgi-bin/QA10/qa.pl).In this paper we introduce INDOC--a biomedical question answering system based on novel ideas of indexing and extracting the answer to the questions posedOnelook is a portal for online definitions, and MedQA is a question answering system that automatically generates short texts to answer specific biomedical questions\"}, {\"question\": \"Which dediodinases are present in kidney?\", \"ideal_answer\": [\"Type 1 and  Type 3 deiodinases are both present in liver\"], \"exact_answer\": [[\"Type 1 deiodinase\"], [\"Tipe 3 deiodinase\"]], \"context\": \"Iodothyronine deiodinase in vitro activity studies in the chicken showed the presence of type I and type III iodothyronine deiodinase activity in both liver and kidney.Co-expression of the deiodinases was also found in the kidney.high ID-I activities were found in liver, kidneythe kidney microsome 5'-deiodinase is type I.\"}, {\"question\": \"List common symptoms of patients with the DOORS syndrome.\", \"ideal_answer\": [\"DOORS syndrome is a constellation of deafness, onychodystrophy, osteodystrophy, mental retardation, and seizures. It is a rare autosomal recessive disorder of unknown cause.\"], \"exact_answer\": [[\"deafness\"], [\"onychodystrophy\"], [\"osteodystrophy\"], [\"mental retardation\"], [\"seizures\"]], \"context\": \"Deafness, onychodystrophy, osteodystrophy, mental retardation, and seizures (DOORS) syndrome is a rare autosomal recessive disorder of unknown cause. Through a search of available case studies and communication with collaborators, we identified families that included at least one individual with at least three of the five main features of the DOORS syndrome: deafness, onychodystrophy, osteodystrophy, intellectual disability, and seizures. \"}, {\"question\": \"Which genes does thyroid hormone receptor beta1 regulate in the liver?\", \"ideal_answer\": [\"LDL receptor\\\"//\\n\\\"ChREBP\\\"//\\n\\\"ME\\\", \\\"malic enzyme\\\"//\\n\\\"cytochrome P450 oxidoreductase\\\"//\"], \"exact_answer\": [[\"LDL receptor\"], [\"ChREBP\", \"Carbohydrate response element binding protein\"], [\"ME\", \"malic enzyme\"], [\"cytochrome P450 oxidoreductase\"]], \"context\": \"our data suggests that TRbeta1-mediated down regulation of hepatic LDLr gene may play a critical role in iodine excess-induced hypercholesterolemic effects.These data suggest that ChREBP mRNA expression is positively regulated by TR-beta1 and TH at the transcriptional level in mammals. This novel observation indicates that TH fine-tunes hepatic lipogenesis via regulating SREBP-1c and ChREBP gene expression reciprocally.In contrast treatment with L-T3 produced an increase in S14 and ME but no change in TR beta-/- mice. From these results, it can be concluded that regulation of HR and EE are independent of TR beta. With the exception of serum cholesterol concentration and liver ME mRNA accumulation, all other markers of TH action examined during TH deprivation exhibited the expected responses in the absence of TR beta.However, the T3-activated expression of the GH gene in GH3-PV and ME gene in SK-Hep-1-PV was repressed by approximately 30% and 90%, respectively, indicating the lack of correlation of PV/TRpbeta1 protein ratio with the dominant negative potency of mutant PV.Transient cotransfection of P450R promoter/chloramphenicol acetyl transferase (CAT) constructs and the thyroid hormone receptor beta1 (TR beta1) expression plasmid into rat hepatoma H4IIE cells resulted in a 2.4-fold induction of promoter activity that was both T3 and TR beta1 dependent.At the molecular level, we detected a dose-dependent attenuation of hepatic low density lipoprotein receptor (LDLr) and thyroid hormone receptor beta1 (TRbeta1) expression in parallel to the change of serum cholesterol.\"}, {\"question\": \"Which disease phenotypes are associated to PRPS1 mutations?\", \"ideal_answer\": [\"X-linked Charcot-Marie-Tooth disease type 5 (CMTX5), Arts syndrome, and non-syndromic sensorineural deafness (DFN2) are allelic syndromes, caused by reduced activity of phosphoribosylpyrophosphate synthetase 1 (PRS-I) due to loss-of-function mutations in PRPS1.\"], \"exact_answer\": [[\"X-linked Charcot-Marie-Tooth disease type 5 (CMTX5)\"], [\"Arts syndrome\"], [\"Non-syndromic sensorineural deafness (DFN2)\"]], \"context\": \"X-linked Charcot-Marie-Tooth disease, Arts syndrome, and prelingual non-syndromic deafness form a disease continuum: evidence from a family with a novel PRPS1 mutationX-linked Charcot-Marie-Tooth disease type 5 (CMTX5), Arts syndrome, and non-syndromic sensorineural deafness (DFN2) are allelic syndromes, caused by reduced activity of phosphoribosylpyrophosphate synthetase 1 (PRS-I) due to loss-of-function mutations in PRPS1Our findings demonstrate that CMTX5, Arts syndrome and DFN2 are phenotypic clusters on an intrafamilial continuum, including overlapping phenotypes even within individuals. The respective phenotypic presentation seems to be determined by the exact PRPS1 mutation and the residual enzyme activity, the latter being largely influenced by the degree of skewed X-inactivationMutations in PRPS1 are associated with a spectrum of non-syndromic to syndromic hearing lossGain of function mutations in PRPS1 cause a superactivity of the PRS-I protein whereas the loss-of-function mutations result in X-linked nonsyndromic sensorineural deafness type 2 (DFN2), or in syndromic deafness including Arts syndrome and X-linked Charcot-Marie-Tooth disease-5 (CMTX5)X-linked Charcot-Marie-Tooth disease, Arts syndrome, and prelingual non-syndromic deafness form a disease continuum: evidence from a family with a novel PRPS1 mutation.Mutations described thus far in PRPS1 are all missense mutations that result in PRS-I superactivity or in variable levels of decreased activity, resulting in X-linked Charcot-Marie-Tooth disease-5 (CMTX5), Arts syndrome, and X-linked nonsyndromic sensorineural deafness (DFN2).Mutations described thus far in PRPS1 are all missense mutations that result in PRS-I superactivity or in variable levels of decreased activity, resulting in X-linked Charcot-Marie-Tooth disease-5 (CMTX5), Arts syndrome, and X-linked nonsyndromic sensorineural deafness (DFN2)X-linked Charcot-Marie-Tooth disease type 5 (CMTX5), Arts syndrome, and non-syndromic sensorineural deafness (DFN2) are allelic syndromes, caused by reduced activity of phosphoribosylpyrophosphate synthetase 1 (PRS-I) due to loss-of-function mutations in PRPS1X-linked Charcot-Marie-Tooth disease type 5 (CMTX5) is caused by mutations in the gene encoding phosphoribosyl pyrophosphate synthetase I (PRPS1)Gain of function mutations in PRPS1 cause a superactivity of the PRS-I protein whereas the loss-of-function mutations result in X-linked nonsyndromic sensorineural deafness type 2 (DFN2), or in syndromic deafness including Arts syndrome and X-linked Charcot-Marie-Tooth disease-5 (CMTX5).The spectrum of PRPS1-related disorders associated with reduced activity includes Arts syndrome, Charcot-Marie-Tooth disease-5 (CMTX5) and X-linked non-syndromic sensorineural deafness (DFN2).X-linked Charcot-Marie-Tooth disease, Arts syndrome, and prelingual non-syndromic deafness form a disease continuum: evidence from a family with a novel PRPS1 mutation.\"}, {\"question\": \"Which kinases does baricitinib inhibit?\", \"ideal_answer\": [\"Baricitinib is an inhibitor of Janus kinase family of enzymes (JAKs) with selectivity for JAK1 and JAK2.\"], \"exact_answer\": [[\"JAK1\"], [\"JAK2\"]], \"context\": \"Reversal of Alopecia Areata Following Treatment With the JAK1/2 Inhibitor Baricitinib.A patient with AA was enrolled in a clinical trial to examine the efficacy of baricitinib, a JAK1/2 inhibitor, to treat concomitant CANDLE syndrome.Baricitinib (also known as LY3009104 or INCB028050), a novel and potent small molecule inhibitor of Janus kinase family of enzymes (JAKs) with selectivity for JAK1 and JAK2, is currently in clinical development for the treatment of rheumatoid arthritis (RA) and other inflammatory disorders.A patient with AA was enrolled in a clinical trial to examine the efficacy of baricitinib, a JAK1/2 inhibitor, to treat concomitant CANDLE syndrome. In vivo, preclinical studies were conducted using the C3H/HeJ AA mouse model to assess the mechanism of clinical improvement by baricitinib.Phase II data for four JAK inhibitors (baricitinib,Positive clinical trial results have also been reported for several other JAK inhibitors including baricitinib. Several other JAK inhibitors and other small molecular entities are also being developed in studies ranging from preclinical models to large clinical trials.\"}, {\"question\": \"Magnetic beads has been used in numerous applications. List some coatings used.\", \"ideal_answer\": [\"aptamers\\nenzymes\\nstreptavidin\\nconcanavalin A\\ncarboxylic-modified \\nTiO2\\nantibodies\\nSELEX library\\nsynthesized DNA\\nC18\\nC8\\noligo(dT)\"], \"exact_answer\": [[\"aptamers\"], [\"enzymes\"], [\"streptavidin\"], [\"concanavalin A\"], [\"carboxylic-modified\"], [\"TiO2\"], [\"antibodies\"], [\"SELEX library\"], [\"synthesized DNA\"], [\"C18\"], [\"C8\"], [\"oligo(dT)\"]], \"context\": \"aptamer-based magnetic separation system enzymes coated to magnetic beads Aptamers are short, single-stranded (ss) oligonucleotidesable to recognize target molecules with high affinity.  Recent developments involve immobilization of tagged enzymes onto magnetic nanoparticles. Using magnetic beads to immobilize DNAs containing various types of structures, we evaluated the in vitro binding activities of two well-characterized DNA repair proteins, Escherichia coli MutS and human p53.An affinity capture involved enzymatic assay for thrombin by using peptide aptamers as affinity ligands on magnetic beadsDuring phage selection the biotinylated antigens are bound to streptavidin coupled magnetic beads,  aptamer-functionalized magnetic beads native antibody immobilized to magnetic beadsPlasma membrane isolation using immobilized concanavalin A magnetic beads.streptavidin magnetic beads Digoxin was coated onto the surface of streptavidin magnetic beads. covalent immobilization of the antigen onto carboxylic-modified magnetic beads (HOOC-MBs) activated with N-(3-dimethylaminopropyl)-N'-ethylcarbodiimide (EDC) and N-hydroxysulfosuccinimide (sulfo-NHS), and further incubation in a mixture solution containing variable concentrations of the antigen and a fixed concentration of an HRP-labeled detection antibody. Aptamers against HNE were immobilized on magnetic beadsThe high specificity was obtained by using the magnetic beads and aptamers,Multiple targets (maltase, invertase, lipase) were immobilized on the magnetic beads by covalent linkage using 1-(3-dimethyl-aminopropyl)-3-ethyl-carbodiimide (EDC) and N-hydroxysuccinimide (NHS) as reaction reagents, respectively.TiO2 -coated magnetic beadsantibody-targeted magnetic beadsA special SELEX library was constructed with the aim to immobilize this library on magnetic beadsstreptavidin-coupled magnetic beadsaptamers loaded on the magnetic beads His-affinity magnetic beadstreptavidin-coated magnetic beads, synthesized DNA is bound to magnetic beadsstreptavidin-modified magnetic microbeads oligo(dT) magnetic beads\"}, {\"question\": \"Tumors of which three organs are classically associated with the multiple endocrine neoplasia type 1 syndrome?\", \"ideal_answer\": [\"Multiple endocrine neoplasia type 1 syndrome is an inherited cancer syndrome defined by occurrence of multiple neuro-endocrine tumors and is classically associated with the combined occurrence of two or more tumors involving parathyroid gland, pancreas and pituitary gland. Other tumors, including but not limited to adrenal cortical tumor, carcinoid tumors lipoma, leiomyoma, duodenal gastrinoma, hepatic focal nodular hyperplasia, and renal angiomyolipoma, angiofibroma, colagenoma, thyroid tumor and meningioma, may also be present.\"], \"exact_answer\": [[\"parathyroid gland\"], [\"pancreas\"], [\"pituitary gland\"]], \"context\": \"Multiple endocrine neoplasia type 1 (MEN1; formerly known as Wermer syndrome) is a rare disorder characterized by the combined occurrence of two or more tumors involving parathyroid, pancreatic islets and anterior pituitary glands; some other tumors have also been described. Hyperparathyroidism is the most common feature of MEN1 (95% of patients), pancreatic islet tumors or pancreatic NET (neuroendocrine tumor) occur in 40-70% and pituitary tumors in 30-40% of MEN 1 patients In addition, other tumors, such as adrenal cortical tumors, carcinoid tumors, lipomas, angiofibromas, colagenomas and meningiomas may be present. the most important causes malignant pancreatic neuroendocrine tumors (NET) and thymic carcinoids. Multiple endocrine neoplasia type 1 (MEN1) is inherited in an autosomal dominant fashion and predisposes to the development of hyperplastic or neoplastic changes in the parathyroid and pituitary glands and the endocrine pancreas, along with numerous other characteristic tumors and features. Multiple endocrine neoplasia type 1 (MEN1) is an autosomal dominant disorder characterized by the combined occurrence of parathyroid and adrenocortical tumors, and neuroendocrine tumors (NETs) of the pancreas and pituitary. The pancreatic NETs are predominantly gastrinomas and insulinomas, and the pituitary NETs are mostly prolactinomas and somatotrophinomas. We address the potential role of miRNAs in the endocrine pancreas, the pituitary gland, and the parathyroid glands-areas where MEN 1 shows high penetrance. Moreover, studies have provided evidence that dysregulation of miRNAs was responsible for endocrine carcinogenesis, including pancreatic, pituitary, and parathyroid tumors.MEN1 and MEN2 are rare inherited cancer syndromes which express a variety of endocrine and nonendocrine tumors.Multiple endocrine neoplasia Type 1 (MEN1) is a rare hereditary tumor syndrome predisposing to tumor development in several endocrine organs. Its major manifestations include hyperparathyroidism, tumors of endocrine pancreas and pituitary. eside these three, several other endocrine (adrenocortical, foregut carcinoid) and nonendocrine (lipoma, angiofibroma, collagenoma, ependymoma, meningioma) tumors have been described to be associated with this syndromeBoth familial and sporadic forms of the disease are known. The diagnosis of MEN1 can be established if two of the three major manifestations are found in the same patient, whereas the diagnosis of familial MEN1 requires one MEN1 patient and a first degree relative with at least one MEN1 manifestation. Both benign (parathyroid, anterior pituitary) and malignant (gastrinoma, glucagonoma) lesions may develop in MEN1 patientsMultiple endocrine neoplasia type 1 (MEN1) is a classic hereditary tumor syndrome characterized by a genetic predisposition to develop a variety of neuroendocrine neoplasias and hormone excess syndromes.Multiple endocrine neoplasia type 1 (MEN 1) is a familial syndrome characterized by parathyroid, enteropancreatic and pituitary tumors. Multiple endocrine neoplasia type 1 (MEN1) is an autosomal dominant inherited tumor syndrome characterized by the development of multiple endocrine tumorsMultiple facial angiofibromas were observed in 28 (88%) of the patients with MEN1, with 16 patients (50%) having 5 or more. Angiofibromas were clinically and histologically identical to those in individuals with tuberous sclerosis. Collagenomas were observed in 23 patients (72%). Also observed were cafe au lait macules in 12 patients (38%), lipomas in 11 patients (34%), confetti-like hypopigmented macules in 2 patients (6%), and multiple gingival papules in 2 patients (6%)Multiple angiofibromas, collagenomas, lipomas, confetti-like hypopigmented macules and multiple gingival papules are cutaneous manifestations of MEN1 and should be looked for in both family members of patients with MEN1 and individuals with hyperparathyroidism of other MEN1-associated tumors. Multiple endocrine neoplasia type 1 (MEN1) is an autosomal dominant inherited disorder characterized by nodular proliferation of the parathyroid glands and tumors of the anterior pituitary gland, the endocrine pancreas, and the neuroendocrine cell system of the gut.We report here a genetic study of a female MEN1 patient with the association of nodular hyperplasia of two parathyroid glands, an insulinoma, multiple duodenal gastrinomas, a prolactinoma, and a gastric carcinoid.Multiple endocrine neoplasia type 1 (MEN1) is defined clinically by the combined occurrence of multiple tumors, typically of the parathyroid glands, pancreatic islet cells, and anterior pituitary gland.In support of previous findings in islet tumors, we found down-regulation of the cell-cycle regulator, p18, in both the pancreatic islet and pituitary adenomas, suggesting that reduced p18 levels may be important for Men1-related tumorigenesis in multiple tissues. Surprisingly, we identified increased p16 transcript in pancreatic islet and pituitary tumors.  The patient was studied and diagnosed with a multiple endocrine neoplasia type I (MEN I), familiar (mother with MEN I). A scintigraphic study with 99mTc-MIBI was performed in order to localize hyperfunctioning parathyroid glands because of biochemical diagnosis of primary hyperparathyroidism. The tumor was removed and histologically confirmed as a carcinoid within a thymus in a MEN type I syndrome. MEN I patients can benefit from the examination with this agent which can potentially localize not only parathyroid endocrine pathology but also unknown associated tumors.Pancreatic endocrine tumors occur sporadically and as part of the multiple endocrine neoplasia type 1 (MEN 1) and von Hippel-Lindau (VHL) syndromes. We have analyzed 22 nonfamilial and 16 MEN 1-associated pancreatic endocrine tumors for loss of heterozygosity (LOH) at 3p, 11q13, and 18q. LOH at 3p was revealed in 45% and 36% of tumors from 31 patients with nonfamilial and MEN 1-associated disease, respectively.The data indicate involvement of tumor suppressor genes on 3p and 18q, in addition to the MEN1 gene at 11q13, in the tumorigenesis of both nonfamilial and MEN 1-associated pancreatic endocrine tumors.Multiple endocrine neoplasia type 1 (MEN1) is characterized by the development of endocrine tumors of the parathyroid and pituitary glands, pancreas, and duodenum. Less frequently occurring tumors associated with MEN1 include non-endocrine tumors such as lipomas and angiofibromas. An increased incidence of thyroid neoplasms, leiomyomas, adrenal cortical hyperplasia, hepatic focal nodular hyperplasia, and renal angiomyolipoma has been noted in the MEN1 population. A germline mutation of the MEN1 gene was detected, and deletions of the MEN1 gene were consistently detected in multiple neuroendocrine tumors involving the parathyroid glands and the pancreas and a hepatic neuroendocrine tumor metastasis, as predicted by Knudson's \\\"two hit\\\" hypothesis.Two hits of the MEN1 gene were also detected in esophageal leiomyoma tissue, suggesting that tumorigenesis was directly related to the patient's underlying MEN1. In contrast, follicular thyroid adenoma, papillary thyroid carcinoma, hepatic focal nodular hyperplasia, and adrenal cortical hyperplasia consistently showed retained heterozygosity of the MEN1 gene with flanking markers and an intragenic marker. Therefore, these tumors appear to develop along pathogenetic pathways that are different from classical MEN1-associated tumors.Twelve unrelated (German MEN1 families and their associated tumors (5 parathyroid tumors, 1 vipoma, 1 gastrinoma, 1 insulinoma) were characterized for MEN1 gene mutations by single-strand conformational variant (SSCV) analysis and DNA sequence analysis as well as for loss of heterozygosity on chromosome 11q13. Multiple endocrine neoplasia type 1 (MEN1) consists of benign, and sometimes malignant, tumors (often multiple in a tissue) of the parathyroids, enteropancreatic neuroendocrine system, anterior pituitary, and other tissues. Skin angiofibromas and skin collagenomas are common. \"}, {\"question\": \"Which are the different homologs or family members of the hedgehog proteins in mammals?\", \"ideal_answer\": [\"Hedgehog (Hh) signaling proteins stimulate cell proliferation, differentiation, and tissue patterning at multiple points in animal development. A single Hh homolog is present in Drosophila, but three Hh homologs, Sonic hedgehog (Shh), Indian hedgehog (Ihh) , and Desert hedgehog (Dhh), are present in mammals.\"], \"exact_answer\": [[\"Sonic hedgehog\", \"SHH\"], [\"Indian hedgehog\", \"IHH\"], [\"Desert hedgehog\", \"DHH\"]], \"context\": \"Hedgehog (Hh) signaling proteins stimulate cell proliferation, differentiation, and tissue patterning at multiple points in animal development. A single Hh homolog is present in Drosophila, but three Hh homologs, Sonic Hh, Indian Hh, and Desert Hh, are present in mammals. We report here biochemical and x-ray structural studies of Sonic, Indian, and Desert Hh proteins both alone and complexed with active domains of CDO and BOC.Previous work has shown that activin, a TGF(beta+) signaling molecule, permits pancreas development by repressing expression of Sonic hedgehog (Shh), a member of the hedgehog family of signaling molecules that antagonize pancreas development. Here we show that Indian hedgehog (Ihh), another hedgehog family member, and Patched 1 (Ptc1), a receptor and negative regulator of hedgehog activity, are expressed in pancreatic tissue.The original hedgehog (hh) gene was found in Drosophila and named for the appearance of a mutant phenotype which causes an embryo to be covered with pointy denticles, thus resembling a hedgehog. The hedgehog family consists of sonic hedgehog (Shh), desert hedgehog (Dhh), and Indian hedgehog (Ihh). Indian hedgehog (IHH) is a secreted signaling molecule of the hedgehog family known to play important roles in the regulation of chondrocyte differentiation, cortical bone formation, and the development of joints.Hedgehog (Hh) plays a pivotal role in various tissues during embryonic development, tissue homeostasis and tumorigenesis. In mammals, Hh exists in three homologs: Desert hedgehog (Dhh), Indian hedgehog (Ihh) and Sonic hedgehog (Shh).Sonic hedgehog (SHH), Desert hedgehog (DHH) and Indian hedgehog (IHH) bind to Patched family receptors (PTCH1 and PTCH2) to transduce signals to GLI1, GLI2 and GLI3. GLI family transcription factors then activate transcription of Hedgehog target genes, such as FOXE1 and FOXM1 encoding Forkhead-box transcription factors.Indian hedgehog (IHH) and its receptors patched (PTC) and smoothened (SMO) belong to the hedgehog family of signaling molecules, which are essential for a variety of patterning events during mammalian tIssue development.In mammals, Hh exists in three homologs: Desert hedgehog (Dhh), Indian hedgehog (Ihh) and Sonic hedgehog (Shh).In mammals, Hh exists in three homologs: Desert hedgehog (Dhh), Indian hedgehog (Ihh) and Sonic hedgehog (Shh).A single Hh homolog is present in Drosophila, but three Hh homologs, Sonic Hh, Indian Hh, and Desert Hh, are present in mammals.A single Hh homolog is present in Drosophila, but three Hh homologs, Sonic Hh, Indian Hh, and Desert Hh, are present in mammalsIn mammals, Hh exists in three homologs: Desert hedgehog (Dhh), Indian hedgehog (Ihh) and Sonic hedgehog (Shh)\"}, {\"question\": \"Which scales are recommended by the American Heart Association for depression screening in cardiovascular patients? \", \"ideal_answer\": [\"Patient Health Questionnaire-2 (PHQ-2) and Patient Health Questionnaire-9 (PHQ-9) are recommended by the American Heart Association for depression screening in cardiovascular patients.\"], \"exact_answer\": [[\"patient health questionnaire-2\"], [\"patient health questionnaire-9\"]], \"context\": \"In 2008, the American Heart Association (AHA) recommended a 2-step screening method, consisting of the 2-item Patient Health Questionnaire (PHQ-2) followed by the 9-item Patient Health Questionnaire (PHQ-9), for identifying depression in cardiovascular patients.The American Heart Association (AHA) statement has recommended routine screening for depression in coronary artery disease with a 2-stage implementation of the Patient Health Questionnaire (PHQ).A recent American Heart Association (AHA) Prevention Committee report recommended depression screening of all coronary heart disease patients using 2- and 9-item instruments from the Patient Health Questionnaire (PHQ-2 and PHQ-9) to identify patients who may need further assessment and treatment.Patients were screened for depression using a protocol identical to the one endorsed by the AHA in a cardiology community clinic in Elmhurst (Queens, New York). Depression was assessed using the Patient Health Questionnaire.Depression screening in cardiac patients has been recommended by the American Heart Association, but the best approach remains unclear. OBJECTIVE: To evaluate nurse-administered versions of the Patient Health Questionnaire for depression screening in patients hospitalized for acute coronary syndrome. METHODS: Staff nurses in an urban cardiac care unit administered versions 2, 9, and 10 of the questionnaire to 100 patients with acute coronary syndrome.\"}, {\"question\": \"Which proteins act as histone-like molecules in prokaryotes?\", \"ideal_answer\": [\"Prokaryotic histone-like proteins (Hlps) or nucleoid-associated proteins (NAPs) are abundant proteins found in bacterial and plastid nucleoids. HU protein is a small, basic, heat-stable DNA-binding protein that is well-conserved in prokaryotes and is associated with the bacterial nucleoid. HU is well conserved in all prokaryotes but surprisingly, it is also homologous to another E. coli DNA-binding protein, IHF. In prokaryotes, IHF and HU are key architectural proteins present at high concentrations. Histone-like Nucleoid Structuring (H-NS) protein can facilitate correct recognition of a promoter by RNA polymerase in AT-rich gene regulatory regions\", \"THe histone-like proteins HU, IHF, H-NS (Nucleoid Structuring) act as histones in prokaryotes\"], \"exact_answer\": [[\"HU\"], [\"IHF\"], [\"H-NS\"]], \"context\": \"We show that Histone-like Nucleoid Structuring (H-NS) protein can facilitate correct recognition of a promoter by RNA polymerase in AT-rich gene regulatory regionsOne feature is a histone-like protein that is associated with the DNA, condensing it into subunits similar to those in eukaryotic chromatinRecently the histone-like proteins were found in some primitive eucaryotes and procaryotesThe relationship between HMGs (1 + 2) and the \\\"primitive\\\" histone-like DNA-packaging proteins from prokaryotes and mitochondria is discussedThe amino acid composition of the protein resembled those of the other prokaryotic histone-like proteins and also to eukaryotic histones H2A and H2BIn prokaryotes and in the pool of vegetative phage DNA the most abundant histone-like protein HU is not associated with the bulk DNA, but localised in the border region with ribosomes where transcription and translation occurLike the histones, HU is able to condense DNA in vitro and to introduce negative super-coiling in covalently closed circular, relaxed DNA molecules in the presence of topoisomerase IHU is well conserved in all prokaryotes but surprisingly, it is also homologous to another E. coli DNA-binding protein, IHFHere we describe the identification of two developmental stage-specific genes, one of which is predicted to encode a 26-kDa lysine- and alanine-rich protein that appears to be homologous to several eukaryotic histone H1 proteinsNo sequence homology was observed between this protein and other bacterial \\\"histone-like\\\" chromosomal proteins, but homology does exist with two other recently described prokaryotic proteinsIn this study we show that these proteins can substitute for the prokaryotic DNA-bending protein HU in promoting the assembly of the Hin invertasome, an intermediate structure in Hin-mediated site-specific DNA inversionWe have characterized the role of HU in assembling the invertasome, an intermediate nucleoprotein complex involved in Hin-mediated site-specific recombinationUsing ligase-mediated circularization of short DNA fragments we also show that HU, the high mobility group (HMG) 1 and 2 proteins from mammals, and a protein from yeast can bend DNA extremely efficientlyThe repression of transcription of two overlapping promoters of the gal operon in Escherichia coli requires Gal repressor (GalR) and the histone-like protein HUWe report GalR mediated DNA looping in which HU plays an obligatory role by helping GalR tetramerizationThe protein has unique dual domains with homology to both bacterial histone-like proteins (HU) and eukaryotic histone H1 Four out of seven multi-copy suppressors were identified to encode HU, (3 for HUalpha and 1 for HUB) a histone-like DNA binding proteinoles of Escherichia coli histone-like protein HU in DNA replicationHU protein is a small, basic, heat-stable DNA-binding protein that is well-conserved in prokaryotes and is associated with the bacterial nucleoidrole of surface-exposed lysines in wrapping DNA about the bacterial histone-like protein HUbasic proteins, including the ubiquitous HU proteins, serve histone-like functions in prokaryotesmodification of E. coli histone-like protein H-NS and bovine histones by short-chain poly-(R)-3-hydroxybutyrate (cPHB)Here we examine Escherichia coli protein H-NS and calf thymus histones, H1, H2A, H2B, H3, and H4, for the presence of cPHBThe process requires negatively supercoiled DNA and the presence of the histone-like protein HUCharacterization of the major DNA-binding proteins of nucleoids revealed essential differences in the two lineages of photosynthetic eukaryotes, namely nucleoids of green plants contain sulfite reductase as a major DNA-binding protein that represses the genomic activity, whereas the prokaryotic DNA-binding protein HU is abundant in plastid nucleoids of the rhodophyte lineageIn prokaryotes, IHF and HU are key architectural proteins present at high concentrationsThe deficit of TFs in some genomes might be compensated by the presence of proteins organizing and compacting DNA, such as histone-like proteinsokaryotic histone-like proteins (Hlps) are abundant proteins found in bacterial and plastid nucleoidsn mesophilic prokaryotes, the DNA-binding protein HU participates in nucleoid organization as well as in regulation of DNA-dependent processesWe show here that HU from the hyperthermophilic eubacterium Thermotoga maritima HU bends DNA and constrains negative DNA supercoils in the presence of topoisomerase I We suggest that T. maritima HU serves an architectural function when associating with a single 35 bp site, but generates a very stable and compact aggregate at higher protein concentrations that organizes and protects the genomic DNAittle is known about the structure and function of most nucleoid-associated proteins (NAPs) in bacterian this study, we investigate the distribution of HU in Caulobacter crescentus using a combination of super-resolution fluorescence imaging and spatial point statistics\"}, {\"question\": \"Which trinucleotide repeat disorders are affecting the nervous system?\", \"ideal_answer\": [\"At least six neudegenerative disorders result from trinucleotide repeat expansion: X-linked spinal and bulbar muscular atrophy (SBMA), two fragile X syndromes of mental retardation (FRAXA and FRAXE), Huntington's disease (HD), spinocerebellar ataxia type 1 (SCA1), and dentatorubral-pallidoluysian atrophy (DRPLA).\"], \"exact_answer\": [[\"X-linked spinal and bulbar muscular atrophy (SBMA)\"], [\"Fragile X syndrome of mental retardation (FRAXA)\"], [\"Fragile X syndrome of mental retardation (FRAXE)\"], [\"Huntington's disease (HD)\"], [\"Spinocerebellar ataxia type 1 (SCA1)\"], [\"Dentatorubral-pallidoluysian atrophy (DRPLA)\"]], \"context\": \"The discovery that expansion of unstable repeats can cause a variety of neurological disorders has changed the landscape of disease-oriented research for several forms of mental retardation, Huntington disease, inherited ataxias, and muscular dystrophy.Expansion of trinucleotide repeats is now recognized as a major cause of neurological disease. At least seven disorders result from trinucleotide repeat expansion: X-linked spinal and bulbar muscular atrophy (SBMA), two fragile X syndromes of mental retardation (FRAXA and FRAXE), myotonic dystrophy, Huntington's disease, spinocerebellar ataxia type 1 (SCA1), and dentatorubral-pallidoluysian atrophy (DRPLA).Fragile X and myotonic dystrophy are multisystem disorders usually associated with large expansions of untranslated repeats, while the four neurodegenerative disorders, SBMA, Huntington's disease, SCA1, and DRPLA, are caused by smaller expansions of CAG repeats within the protein coding portion of the gene.\"}, {\"question\": \"Which are the cellular targets of imatinib mesylate?\", \"ideal_answer\": [\"The cellular targets of imatinib mesylate are BCR-ABL, platelet-derived growth factor receptor (PDGFR) and c-kit kinases.\"], \"exact_answer\": [[\"BCR-ABL\"], [\"Platelet-derived growth factor receptor (PDGFR)\"], [\"c-Kit\"]], \"context\": \"Imatinib mesylate is a tyrosine kinase inhibitor of the ABL, platelet-derived growth factor receptor (PDGFR), and c-kit kinases.Imatinib mesylate is a novel anti-tumor agent useful in the clinical management of chronic myelogenous leukemia and gastrointestinal stromal tumors with minimal toxicity relative to other forms of cancer therapy. Its clinical activity and minimal toxicity are related to specific inhibition of cellular targets including BCR-ABL, platelet-derived growth factor receptor and c-kit kinases, resulting in the collapse of downstream signaling cascades important for transformation.\"}, {\"question\": \"Which are the clinical characteristics of Tuberous Sclerosis?\", \"ideal_answer\": [\"The clinical characteristics of Tuberous Sclerosis include epilepsy, subependymal giant cell astrocytomas, lymphangioleiomyomatosis, rhabdomyoma, renal angiomyolipomas, cortical tubers, neurofibromas, angiofibromas, mental retardation, and behavioral disorders.\"], \"exact_answer\": [[\"epilepsy\"], [\"subependymal giant cell astrocytomas\"], [\"lymphangioleiomyomatosis\"], [\"rhabdomyoma\"], [\"renal angiomyolipomas\"], [\"cortical tubers\"], [\"neurofibromas\"], [\"angiofibromas\"], [\"mental retardation\"], [\"behavioral disorders\"]], \"context\": \"Prevalence and long-term outcome of epilepsy in tuberous sclerosis complex (TSC) is reported to be variableSubependymal giant cell astrocytomas (SEGAs) are benign tumors, most commonly associated with tuberous sclerosis complex (TSC).Lymphangioleiomyomatosis (LAM) is a rare, progressive, frequently lethal cystic lung disease that almost exclusively affects women.Rhabdomyoma is the most common type of cardiac tumor in fetuses and is often associated with tuberous sclerosis complex (TSC) with neurologic sequelae.In contrast to renal angiomyolipomas, which are often associated with tuberous sclerosis,clinical characteristics of subependymal nodule (SN) - subependymal giant cell astrocytoma (SGCA) complex in tuberous sclerosisNMI patients had a lower incidence of brain findings on imaging studies, neurological features, and renal findings than those with TSC2 mutations. In contrast, NMI patients had a lower incidence of seizures than TSC patients with TSC1 mutations, but had a higher incidence of both renal angiomyolipomas and pulmonary lymphangioleiomyomatosis.clinical characteristics of subjects with pulmonary lymphangioleiomyomatosis, both sporadic and tuberous sclerosis-related forms.Women with tuberous sclerosis-related lymphangioleiomyomatosis were younger and had less impaired lung function compared with those with the sporadic form.Balloon cells are histopathological hallmarks of various cortical malformations, i.e., focal cortical dysplasia (Taylor's type, FCD IIb), hemimegalencephaly (HME) or cortical tubers (tuberous sclerosis, TSC). Pulmonary lymphangioleiomyomatosis is a rare cause of recurrent pneumothorax and should be considered a differential diagnosis, especially in young women with diffuse bilateral bullous emphysema or tuberous sclerosis.The disease is characterized by the development of benign hamartomatous tumors: neurofibromas and angiofibromas, located in the skin, central nervous system, mucosas and other organs. Abnormal neural cell migration plays an important role in the neurological dysfunctions found in TS, the predominant features being mental retardation, seizures and behavioral disorders.The true prevalence of pulmonary lymphangioleiomyomatosis (LAM) in patients with tuberous sclerosis complex (TSC) is unknown.We retrospectively reviewed a cohort of patients with TSC and refractory epilepsy who started CLB over a 5-year period. Clinical characteristics and number of tubers on MRI were assessed.\"}, {\"question\": \"List drugs included in the DHAP-R chemotherapy regiment.\", \"ideal_answer\": [\"Dexamethasone (a steroid hormone), cytarabine (ara-C), cisplatin (platinum agent) and rituximab are included in the DHAP-R chemotherapy protocol.\"], \"exact_answer\": [[\"dexamethasone\"], [\"cytarabine\"], [\"cisplatin\"], [\"rituximab\"]], \"context\": \"Standard treatment of transplant-eligible patients with relapsed diffuse large B-cell lymphoma (DLBCL) consists of rituximab and platinum-based chemotherapy, either ifosfamide, carboplatin, and etoposide (ICE) or dexamethasone, cytarabine, and cisplatin (DHAP), with autologous transplant consolidation for those with chemosensitive disease.The intergroup Collaborative Trial in Relapsed Aggressive Lymphoma (CORAL) set the limits for this standard of treatment after first comparing 2 salvage regimens: rituximab, ifosfamide, etoposide, and carboplatin (R-ICE) and rituximab, dexamethasone, aracytine, and cisplatin (R-DHAP). The response rate (RR) to first salvage chemotherapy was 23% (RR by regimen: dexamethasone, cytosine arabinoside and cisplatin [DHAP] 15%, etoposide, Solu-Medrol, cytosine arabinoside and cisplatin [ESHAP] 36%, and gemcitabine, dexamethasone and cisplatin [GDP] 45%); 25% (n = 28) of patients underwent ASCT. PURPOSE: To evaluate the prognostic value of the cell of origin (COO) in patients with relapsed/refractory diffuse large B-cell lymphoma (DLBLC), prospectively treated by rituximab, dexamethasone, high-dose cytarabine, and cisplatin (R-DHAP) versus rituximab, ifosfamide, carboplatin, and etoposide and followed by intensive therapy plus autologous stem-cell transplantation on the Collaborative Trial in Relapsed Aggressive Lymphoma (CORAL) trial. The aim of this study was to evaluate the efficacy and toxicity of two chemotherapy regimens based on platinum and cytarabine in association with etoposide and methylprednisolone (ESHAP) or with dexamethasone (DHAP) with or without Rituximab (\\u00b1 R) in patients with refractory or a relapsed Primary Central Nervous System Lymphoma (PCNSL). R-DHAP combines rituximab with cisplatin, cytarabine, and dexamethasone. PATIENTS AND METHODS: Patients with CD20(+) DLBCL in first relapse or who were refractory after first-line therapy were randomly assigned to either rituximab, ifosfamide, etoposide, and carboplatin (R-ICE) or rituximab, dexamethasone, high-dose cytarabine, and cisplatin (R-DHAP).Because of the inability to safely administer unattenuated doses of anthracycline-based regimens, dexamethasone, high-dose cytarabine, and cisplatin (DHAP) was used at full doses (along with rituximab for the DLBCL patient) until hepatic function normalized (1-5 cycles). Randomized to DHAP (cisplatin-cytarabine-dexamethasone)-VIM (etoposide-ifosfamide-methotrexate)-DHAP (cisplatin-cytarabine-dexamethasone) chemotherapy with rituximab (R; R-DHAP arm) were 119 patients (113 evaluable) and to chemotherapy without rituximab (DHAP arm) 120 patients (112 evaluable). Patients are first randomised between ICE (ifosfamide, carboplatin, etoposide) and DHAP (dexamethasone, ara-C and cisplatin), both combined with rituximab (R-ICE or R-DHAP).BACKGROUND AND OBJECTIVE: The treatment of the patients with relapsed and refractory non-Hodgkin's lymphoma(NHL) remains difficult. It was reported that DHAP regimen(cisplatin + Ara-C + dexamthsone) was an effective salvage therapy, but there was no report about it in China. Randomized to DHAP (cisplatin-cytarabine-dexamethasone)-VIM (etoposide-ifosfamide-methotrexate)-DHAP (cisplatin-cytarabine-dexamethasone) chemotherapy with rituximab (R; R-DHAP arm) were 119 patients (113 evaluable) and to chemotherapy without rituximab (DHAP arm) 120 patients (112 evaluable).Randomized to DHAP (cisplatin-cytarabine-dexamethasone)-VIM (etoposide-ifosfamide-methotrexate)-DHAP (cisplatin-cytarabine-dexamethasone) chemotherapy with rituximab (R; R-DHAP arm) were 119 patients (113 evaluable) and to chemotherapy without rituximab (DHAP arm) 120 patients (112 evaluable)\"}, {\"question\": \"Which are the most common symptoms in Lambert-Eaton Myasthenic Syndrome?\", \"ideal_answer\": [\"Lambert-Eaton myasthenic syndrome is a neuromuscular junction disorder characterized by proximal limb muscle weakness, fatigability, decreased deep-tendon reflexes, and autonomic symptoms. There are 2 forms of Lambert-Eaton myasthenic syndrome: one most frequently associated with small-cell lung cancer (P-Lambert-Eaton myasthenic syndrome) and the other that is a pure autoimmune form (NP-Lambert-Eaton myasthenic syndrome)\", \"Lambert-Eaton myasthenic syndrome is a neuromuscular junction disorder characterized by proximal limb muscle weakness, fatigability, decreased deep-tendon reflexes, and autonomic symptoms, that may include dry mouth, constipation, blurred vision, impaired sweating, and orthostatic hypotension. There are 2 forms of Lambert-Eaton myasthenic syndrome: one most frequently associated with small-cell lung cancer, and the other that is a pure autoimmune form.\"], \"exact_answer\": [[\"proximal limb muscle weakness\"], [\"fatigability\"], [\"decreased deep-tendon reflexes\"], [\"autonomic symptoms\"], [\"small-cell lung cancer\"]], \"context\": \"Lambert-Eaton myasthenic syndrome is a neuromuscular junction disorder characterized by proximal limb muscle weakness, fatigability, decreased deep-tendon reflexes, and autonomic symptoms. There are 2 forms of Lambert-Eaton myasthenic syndrome: one most frequently associated with small-cell lung cancer (P-Lambert-Eaton myasthenic syndrome) and the other that is a pure autoimmune form (NP-Lambert-Eaton myasthenic syndrome)LEMS weakness typically involves lower and upper limbs and the proximal muscles are predominantly affected. A significant proportion of patients also have dysfunction of the autonomic nervous system that may include dry mouth, constipation, blurred vision, impaired sweating, and orthostatic hypotension. LEMS recognition is based on clinical, electrophysiological and immunological criteria. Nearly 50-60% of patients with LEMS have an underlying tumour that, in almost all cases, is a small-cell lung cancer;In Lambert-Eaton myasthenic syndrome (LEMS), antibodies against presynaptic voltage-gated calcium channels reduce the quantal release of acetylcholine, causing muscle weakness and autonomic dysfunction. More than half of the affected patients have associated small cell lung cancerThe Lambert-Eaton myasthenic syndrome (LEMS) is a disease of neuromuscular transmission in which autoantibodies against the P/Q-type voltage-gated calcium channel (VGCC) at the presynaptic nerve terminal play a major role in decreasing quantal release of acetylcholine (ACh), resulting in skeletal muscle weakness and autonomic symptoms.Primary respiratory failure as the presenting symptom in Lambert-Eaton myasthenic syndrome.Lambert-Eaton myasthenic syndrome is a neuromuscular junction disorder characterized by proximal limb muscle weakness, fatigability, decreased deep-tendon reflexes, and autonomic symptoms. Lambert-Eaton myasthenic syndrome is a neuromuscular junction disorder characterized by proximal limb muscle weakness, fatigability, decreased deep-tendon reflexes, and autonomic symptoms. There are 2 forms of Lambert-Eaton myasthenic syndrome: one most frequently associated with small-cell lung cancer (P-Lambert-Eaton myasthenic syndrome) and the other that is a pure autoimmune form (NP-Lambert-Eaton myasthenic syndrome).The Lambert-Eaton myasthenic syndrome (LEMS) is a neuromuscular disorder, often associated with small cell lung carcinoma (SCLC), which is characterized by reduced quantal release of acetylcholine from the motor nerve terminals. LAMBERT-EATON MYASTHENIC SYNDROME: The Lambert-Eaton Myasthenic Syndrome (LEMS) is characterized by proximal muscle weakness initially affecting gait, autonomic symptoms (dry mouth, constipation, erectile failure) and augmentation of strength during initial voluntary activation.LAMBERT-EATON MYASTHENIC SYNDROME: The Lambert-Eaton Myasthenic Syndrome (LEMS) is characterized by proximal muscle weakness initially affecting gait, autonomic symptoms (dry mouth, constipation, erectile failure) and augmentation of strength during initial voluntary activation.Primary respiratory failure as the presenting symptom in Lambert-Eaton myasthenic syndrome.\"}, {\"question\": \"Which types of bacterial microflora are associated with the progression of peri-implantitis?\", \"ideal_answer\": [\"Bacteria such as A. actinomycetemcomitans, P. gingivalis, T. forsythensis, T. denticola, P. intermedia and F. nucleatum are associated with the progression of peri-implantitis.\"], \"exact_answer\": [[\"A. actinomycetemcomitans\"], [\"P. gingivalis\"], [\"T. denticola\"], [\"T. forsythia\"], [\"P. intermedia\"], [\"F. nucleatum\"]], \"context\": \" Later, the specific plaque hypothesis established the role of some microorganisms such as A. actinomycetemcomitans, P. gingivalis, T. forsythensis, T. denticola, P. intermedia and F. nucleatum in different forms of periodontal diseasesThe microbiological analysis revealed that the total bacterial load increased during the period following ligature removal and established an anaerobic Gram-negative microflora.Most periodontal pathogens are common saprophytes of the oral cavity, expressing their virulence only in a susceptible host or when some changes come about in the oral environmentOverall, the types and relative proportions of putative periodontal pathogens in plaque associated with ligature-induced peri-implantitis and ligature-induced periodontitis were similar. Only levels of anaerobic Actinomyces and spirochetes were significantly different between both sites\"}, {\"question\": \"What are the known families of deadenylases?\", \"ideal_answer\": [\"All known deadenylases belong to either the DEDD or the exonuclease\\u2013endonuclease\\u2013phosphatase (EEP) superfamily. Members of DEDD include the POP2, CAF1Z, PARN and PAN2 families. Members of EEP include the CCR4, Nocturnin, ANGEL and 2'-PDE families.\"], \"exact_answer\": [[\"POP2\"], [\"CAF1Z\"], [\"PARN\"], [\"PAN2\"], [\"CCR4\"], [\"Nocturnin\"], [\"ANGEL\"], [\"2'-PDE\"]], \"context\": \"Among the deadenylase family, poly(A)-specific ribonuclease (PARN) is unique in its domain composition, which contains three potential RNA-binding domains: the catalytic nuclease domain, the R3H domain and the RRM domainOn the other hand, the interaction between CNOT7 and BTG2, a member of the antiproliferative BTG/Tob family involved in transcription and mRNA decay appears less important for proliferation of MCF7 cells, suggesting that CNOT7 does not function solely in conjunction with BTG2.The Tob/BTG family is a group of antiproliferative proteins containing two highly homologous regions, Box A and Box B. These proteins all associate with CCR4-associated factor 1 (Caf1), which belongs to the ribonuclease D (RNase D) family of deadenylases and is a component of the CCR4-Not deadenylase complex.The CCR4 family proteins are 3'-5'-deadenylases that function in the first step of the degradation of poly(A) mRNA.CCR4, an evolutionarily conserved member of the CCR4-NOT complex, is the main cytoplasmic deadenylase. It contains a C-terminal nuclease domain with homology to the endonuclease-exonuclease-phosphatase (EEP) family of enzymes.The yeast Pop2 protein, belonging to the eukaryotic Caf1 family, is required for mRNA deadenylation in vivo.Although CAF1 protein belongs to the DEDDh family of RNases, CCR4 appears to be the principle deadenylase of the CCR4-NOT complex.Like CCR4, increased length of RNA substrates converted mCAF1 into a processive enzyme. In contrast to two other DEDD family members, PAN2 and PARN, mCAF1 was not activated either by PAB1 or capped RNA substrates.The domain has the fold of the DnaQ family and represents the first structure of an RNase from the DEDD superfamily.The evolutionarily conserved CCR4 protein, which is part of the cytoplasmic deadenylase, contains a C-terminal domain that displays homology to an Mg2+-dependent DNase/phosphatase family of proteins.These results establish that CCR4 and most probably other members of a widely distributed CCR4-like family of proteins constitute a novel class of RNA-DNA exonucleases.Deadenylases specifically catalyze the degradation of eukaryotic mRNA poly(A) tail in the 3'- to 5'-end direction with the release of 5'-AMP as the product.Shortening and removal of the 3' poly(A) tail of mature mRNA by poly(A)-specific 3' exonucleases (deadenylases) is the initial and often rate-limiting step in mRNA degradation.PARN, Nocturnin and Angel are three of the multiple deadenylases that have been described in eukaryotic cells.\"}, {\"question\": \"Which packages are used for performing overlap analysis of genomic regions in R/bioconductor?\", \"ideal_answer\": [\"IRanges, GenomicRanges, and GenomicFeatures provide scalable data structures for representing annotated ranges on the genome, with special support for transcript structures, read alignments and coverage vectors. Computational facilities include efficient algorithms for overlap and nearest neighbor detection, coverage calculation and other range operations. This infrastructure directly supports more than 80 other Bioconductor packages, including those for sequence analysis, differential expression analysis and visualization.\", \"At the core of the infrastructure are three packages: IRanges, GenomicRanges, and GenomicFeatures. These packages provide scalable data structures for representing annotated ranges on the genome, with special support for transcript structures, read alignments and coverage vectors. Computational facilities include efficient algorithms for overlap and nearest neighbor detection, coverage calculation and other range operations. This infrastructure directly supports more than 80 other Bioconductor packages, including those for sequence analysis, differential expression analysis and visualization.\"], \"exact_answer\": [[\"IRanges\"], [\"GenomicRanges\"], [\"GenomicFeatures\"]], \"context\": \"At the core of the infrastructure are three packages: IRanges, GenomicRanges, and GenomicFeatures. These packages provide scalable data structures for representing annotated ranges on the genome, with special support for transcript structures, read alignments and coverage vectors. Computational facilities include efficient algorithms for overlap and nearest neighbor detection, coverage calculation and other range operations. This infrastructure directly supports more than 80 other Bioconductor packages, including those for sequence analysis, differential expression analysis and visualization.\"}, {\"question\": \"Global quantitative phosphoproteomic analyses are emerging. List the preferred technologies for the enrichment for phosphorylated peptides?\", \"ideal_answer\": [\"There are many different approaches to enrich for phosphorylated peptides: titanium dioxide, IMAC, simple derivatization through phosphoramidate chemistry and antibodies.\"], \"exact_answer\": [[\"enriched for phosphorylated peptides using titanium dioxide\"], [\"sequential elution from IMAC\"], [\"simple derivatization procedure based on phosphoramidate chemistry\"], [\"antiphosphotyrosine antibodies\"]], \"context\": \"TiSH--a robust and sensitive global phosphoproteomics strategy employing a combination of TiO2, SIMAC, and HILIC.An initial TiO(2) phosphopeptide pre-enrichment step is followed by post-fractionation using sequential elution from IMAC (SIMAC) to separate multi- and mono-phosphorylated peptides, and hydrophilic interaction liquid chromatography (HILIC) of the mono-phosphorylated peptides (collectively abbreviated \\\"TiSH\\\").After enrichment by anti-phosphotyrosine antibodies, we identified 29 potential novel c-Src substrate proteins.Combining immunoprecipitation with an antiphosphotyrosine antibody, titanium dioxide phosphopeptide enrichment, isobaric tag for the relative and absolute quantitation methodology, and strong cation exchange separation, we were able to identify 2814 phosphopeptides.Phosphopeptide fractionation by strong cation exchange chromatography combined with immobilized metal affinity chromatography (IMAC) enrichment enabled quantification of more than 8000 distinct phosphorylation sites in Ppt1 wild-type versus Ppt1-deficient yeast cellsBy employing titanium dioxide in addition to antiphosphotyrosine antibodies as enrichment methods, we identified 4164 phosphopeptides on 1670 phosphoproteins.Our method combines the use of strong cation exchange (SCX) and titanium dioxide (TiO(2)) for phosphopeptide enrichment, high-resolution MS for peptide and protein identification, and stable isotope labeling by amino acids in cell culture (SILAC) for quantification.Phosphopeptides were isolated with high specificity through a simple derivatization procedure based on phosphoramidate chemistry.\"}, {\"question\": \"Which histone modifications are correlated with transcription elongation?\", \"ideal_answer\": [\"This is accompanied by reductions in the level of H3K36 trimethylation, a posttranslational histone modification associated with efficient transcriptional elongation, and the number of full-length transcripts from these genes. The 3' ZNF exons contain H3K36me3, a mark of transcriptional elongation.\"], \"exact_answer\": [[\"H3K36me3\"]], \"context\": \" This is accompanied by reductions in the level of H3K36 trimethylation, a posttranslational histone modification associated with efficient transcriptional elongation, and the number of full-length transcripts from these genes.Also like BUR kinase and the PAF complex, the Spt5 CTR is important for histone H2B K123 monoubiquitination and histone H3 K4 and K36 trimethylation during transcription elongation.A positive feedback loop links opposing functions of P-TEFb/Cdk9 and histone H2B ubiquitylation to regulate transcript elongation in fission yeas Mono-ubiquitylation of histone H2B (H2Bub1) plays a key role in coordinating co-transcriptional histone modification by promoting site-specific methylation of histone H3We further characterized the histone modifications at the 3' ZNF exons and found that these regions also contain H3K36me3, a mark of transcriptional elongation.To test the hypothesis that the contradictory modifications are due to imprinting, we used a SNP analysis of RNA-seq data to demonstrate that both alleles of certain ZNF genes having H3K9me3 and H3K36me3 are transcribed. We next analyzed isolated ZNF 3' exons using stably integrated episomes.Set2-mediated H3 Lys(36) methylation is a histone modification that has been demonstrated to function in transcriptional elongation by recruiting the Rpd3S histone deacetylase complex to repress intragenic cryptic transcription.Overall, our results show and suggest that multiple H4, H2A, and H3 residues contribute to and form a Set2 docking/recognition site on the nucleosomal surface so that proper Set2-mediated H3 Lys(36) di- and trimethylation, histone acetylation, and transcriptional elongation can occur.Sp3 depletion correlated with increased H3K36me3 and H2Bub1, two histone modifications associated with transcription elongationUsing a SIR-regulated heat shock-inducible transgene, hsp82-2001, and a natural drug-inducible subtelomeric gene, YFR057w, as models we demonstrate that substantial transcriptional induction (&gt;200-fold) can occur in the context of restricted histone loss and negligible levels of H3K4 trimethylation, H3K36 trimethylation and H3K79 dimethylation, modifications commonly linked to transcription initiation and elongationSp3 depletion correlated with increased H3K36me3 and H2Bub1, two histone modifications associated with transcription elongation. Sp3 depletion correlated with increased H3K36me3 and H2Bub1, two histone modifications associated with transcription elongation.\"}, {\"question\": \"List disorders that have been associated to the polymorphism rs2535629.\", \"ideal_answer\": [\"schizophrenia and major depressive disorder. \"], \"exact_answer\": [[\"schizophrenia\"], [\"major depressive disorder\"]], \"context\": \"A recent genome-wide analysis indicated that a polymorphism (rs2535629) of ITIH3 showed the strongest association signal with susceptibility to psychiatric disorders in Caucasian populations. he aim of the study was to replicate the association of rs2535629 with schizophrenia and major depressive disorder (MDD) in Japanese subjects. In the MDD-bipolar cross-disorder analysis, 15 SNPs exceeded genome-wide significance (P<5 \\u00d7 10(-8)), and all were in a 248\\u2009kb interval of high LD on 3p21.1 (chr3:52\\u2009425\\u2009083-53\\u2009822\\u2009102, minimum P=5.9 \\u00d7 10(-9) at rs2535629).However, there was no preferential transmission at the ITIH3 rs52535629 polymorphism (\\u03c7\\u00b2=0.14, P=0.707).\"}, {\"question\": \"List diseases where protein citrullination plays an important role.\", \"ideal_answer\": [\"Rheumatoid arthritis, multiple sclerosis, prion diseases, cancer and Alzheimer's disease are examples of diseases where protein citrullination involvement has been demonstrated.\"], \"exact_answer\": [[\"Multiple Sclerosis\"], [\"Alzheimer disease\"], [\"Rheumatoid Arthritis\", \"RA\"], [\"Prion Diseases\"], [\"Cancer\"]], \"context\": \"his modification has been linked to the pathogenesis of autoimmune diseases including rheumatoid arthritis (RA). Over the past decade, PADs and protein citrullination have been commonly implicated as abnormal pathological features in neurodegeneration and inflammatory responses associated with diseases such as multiple sclerosis, Alzheimer disease and rheumatoid arthritis. These findings suggest that PAD2 and citrullinated proteins may play a key role in the brain pathology of prion diseases. We first discovered that abnormal myelin hypercitrullination, even in normal-appearing white matter, by peptidylarginine deiminases (PADs) correlates strongly with disease severity and might have an important role in MS progression. recent advances have shown that the once obscure modification known as citrullination is involved in the onset and progression of inflammatory diseases and cancer.These observations may reflect a role of P. gingivalis in the protein citrullination, which is related to the pathogenesis of RA.Protein citrullination has been detected in the CNS and associated with a number of neurological diseases.Protein and peptide citrullination are important in the development of rheumatoid arthritiscitrullination has been shown to be associated with several diseases, such as cancer, multiple sclerosis, rheumatoid arthritis, and Alzheimer's disease. Our data thus suggest that autoantibody formation in response to citrullinated vimentin directly induces bone loss, providing a link between the adaptive immune system and bone.citrullinated protein also has a negative side, because this protein's accumulation in the brain is a possible cause of Alzheimer's disease. The aim of the present work was to study the effect of tobacco smoking on disease progression in rheumatoid arthritis patients and its relation to anti-cyclical citrullinated peptide (anti-CCP) antibodies.It appears from our results that, tobacco smoking mostly play a role in progression of rheumatoid arthritis through tissue protein citrullinationwe have reported pathological characterization of PAD2 and citrullinated proteins in scrapie-infected mice, aberrant citrullinated proteins could play a role in pathogenesis and have value as a marker for the postmortem classification of neurodegenerative diseases.We previously reported that abnormal protein citrullination by PAD2 has been closely associated with the pathogenesis of neurodegenerative disorders such as Alzheimer's disease and prion diseaseThis study suggests that accumulated citrullinated proteins and abnormal activation of PAD2 may function in the pathogenesis of prion diseases and serve as potential therapeutic targets.PTMs such as citrullination due to alterations of peptidylarginine deiminase activity or generation of RA-specific epitopes, should be considered as a trigger in tolerance break.Protein citrullination is an important posttranslational modification recognized by rheumatoid arthritis (RA)-specific autoantibodies. Protein citrullination, a once-obscure post-translational modification (PTM) of peptidylarginine, has recently become an area of significant interest because of its suspected role in human disease states, including rheumatoid arthritis and multiple sclerosis, and also because of its newfound role in gene regulation.he increased levels of citrullinated CNS proteins associated with MS are also observed during the development of EAE. Rheumatoid arthritis, multiple sclerosis, and Alzheimer's disease are examples of human diseases where protein citrullination involvement has been demonstratedcitrullinated proteins may become a useful marker for human neurodegenerative diseases.\"}, {\"question\": \"List core circadian clock genes.\", \"ideal_answer\": [\"The core circadian clock genes are CLOCK, BMAL1, Per, and Cry.\"], \"exact_answer\": [[\"CLOCK\"], [\"BMAL1\"], [\"Per\"], [\"Cry\"], [\"CKl\\u03b5\"]], \"context\": \"PER1, CRY1, CRY2, CLOCK, BMAL1, and CKl\\u03b5  Expression levels of five clock genes (Rev-Erb\\u03b1, Per1, Per2, Bmal1 and Cry1)altered expression patterns of the circadian clock genes, Bmal1 and Per2. This review will focus on the core circadian clock genes CLOCK, BMAL1, Per, and Cry.CLOCK, RORA, and NPAS2Rev-erb\\u03b1 and Bmal1 he core circadian clock genes BMAL1, PER1/2 and CRY1/2.CLOCK/BMAL1, the core circadian clock components\"}, {\"question\": \"List clinical trials that have directly compared microsurgical clipping with endovascular coiling for treatment of ruptured brain aneurysms?\", \"ideal_answer\": [\"Barrow Ruptured Aneurysm Trial (BRAT) and international subarachnoid aneurysmal trial (ISAT) have directly compared microsurgical clipping with endovascular coiling for treatment of ruptured brain aneurysms. FIAT study, a clinical care trial aiming to compare angiographic and clinical outcomes following treatment with a Flow-Diverter or with the best conventional treatment option (including clipping) is underway.\"], \"exact_answer\": [[\"Barrow Ruptured Aneurysm Trial (BRAT)\"], [\"international subarachnoid aneurysmal trial (BRAT)\"], [\"FIAT study\"]], \"context\": \"The authors report the 3-year results of the Barrow Ruptured Aneurysm Trial (BRAT). The objective of this ongoing randomized trial is to compare the safety and efficacy of microsurgical clip occlusion and endovascular coil embolization for the treatment of acutely ruptured cerebral aneurysms and to compare functional outcomes based on clinical and angiographic data. We report the design of the FIAT study, a clinical care trial aiming to compare angiographic and clinical outcomes following treatment with a Flow-Diverter or with the best conventional treatment option. The FIAT study will include both a randomized and a registry portion. Patients will be proposed randomization to either FD stenting or best conventional treatment option (observation, coiling, stenting, or clipping) as determined by the treating physician. The effect of clipping and coiling in acute severe subarachnoid hemorrhage after international subarachnoid aneurysmal trial (ISAT) results.A critical appraisal of the ISAT of microsurgical clipping versus coiling is used for comparison of our results.International subarachnoid aneurysm trial (ISAT) of neurosurgical clipping versus endovascular coiling in 2143 patients with ruptured intracranial aneurysms: a randomised comparison of effects on survival, dependency, seizures, rebleeding, subgroups, and aneurysm occlusion.International Subarachnoid Aneurysm Trial (ISAT) of neurosurgical clipping versus endovascular coiling in 2143 patients with ruptured intracranial aneurysms: a randomised trial.\"}, {\"question\": \"Which are the biotracers used for detection of Alzheimer's disease using PET?\", \"ideal_answer\": [\"Pittsburgh compound B (PIB) was the first radiotracer capable of highlighting deposits of beta-amyloid\\u2014one pathological hallmark of Alzheimer's disease\\u2014in living individuals during a PET scan. The Alzheimer's Association helped fund early PIB development. The Association in 2006 also awarded a $2.1 million grant to the Alzheimer's Disease Neuroimaging Initiative (ADNI) to expand this long-term, nationwide study to include PIB-PET imaging.\\n\\n18F flutemetamol (flute), another radiotracer that highlights beta-amyloid in a PET scan, is structurally identical to PIB except for one fluorine atom in place of a carbon atom. That small chemical change enables flutemetamol to remain stable significantly longer than does PIB, potentially increasing its usefulness outside research settings. In phase II study results reported in the Annals of Neurology, flutemetamol performed similarly to PIB. Additional testing is under way.\\n\\nFlorbetapir F 18 (18F-AV-45) is also a radiotracer that highlights brain beta-amyloid during a PET scan. At the 2010 Alzheimer's Association International Conference on Alzheimer's Disease (AAICAD), florbetapir's developer first reported data, later published in the JAMA, showing nearly perfect correlation between brain amyloid levels detected by florbetapir PET scans in study volunteers and levels found in autopsies of the same individuals a few months later. The developer has sought Food and Drug Administration (FDA) approval to market florbetapir under the brand name Amyvid. The FDA has said it will withhold approval until the developer establishes a professional training program to ensure accuracy and consistency in reading and interpreting Amyvid scans.\\n\\nFlorbetaben (BAY 94-9172) is another radiotracer designed to detect beta-amyloid during a PET scan. Phase II study results and other florbetaben data were reported at the 2010 Alzheimer's Association International Conference on Alzheimer's Disease (AAICAD). Phase II data were also later published in Lancet Neurology. Further studies are now under way.\"], \"exact_answer\": [[\"Pittsburgh compound B\"], [\"Florbetaben (BAY 94-9172)\"], [\"Florbetapir F 18 (18F-AV-45)\"], [\"18F flutemetamol (flute)\"]], \"context\": \"(18)F-flutemetamol PET and structural MRI provided additive information in the diagnostic classification of aMCI subjects, suggesting an amyloid-independent neurodegenerative component among aMCI subjects in this sample.This pooled analysis of four studies determined the level of association between uptake of the fibrillar amyloid \\u03b2 positron emission tomography (PET) imaging agent [(18)F]flutemetamol (Pittsburgh Compound B analog with a 5.5 times longer half-life to enable it to be used in the clinical setting) is implicated in the development of late-onset Alzheimer's disease (AD). We sought to determine the associations between beta amyloid (A\\u03b2) plaque deposition in vivo using Pittsburgh compound B (PiB) and several indices of cholesterol homeostasis (i.e., total cholesterol, high-density lipoprotein cholesterol, low-density lipoprotein cholesterol, triglycerides, apolipoprotein E (ApoE), clusterin, oxysterol metabolites of cholesterol, and previously reported genes associated with late-onset AD) in 175 nondemented elderly subjects. High A\\u03b2 depg biomarkers of Alzheimer's disease include amyloid deposition [imaged with [(11)C]Pittsburgh compound B (PiB) PET], altered glucose metabolism (imaged with [(18)F]fluro-deoxyglucose PET), and structural atrophy (imaged by MRI). Recently we published the initial subset of imaging findings for specific regions in a cohort of individuals with autosomal dominant Alzheimer's disease. We now exteCriteria for preclinical Alzheimer disease (AD) propose \\u03b2-amyloid (A\\u03b2) plaques to initiate neurodegeneration within AD-affected regions. However, some cognitively normal older individuals harbor neural injury similar to patients with AD, without concurrent A\\u03b2 burden. Such findings challenge the proposed sequence and suggest that A\\u03b2-independent precursors underlie AD-typical neurodegenerative patterns. OBJECTIVE Tol is a positron emission tomography (PET) tracer for in vivo amyloid imaging. The ability to classify amyloid scans in a binary manner as 'normal' versus 'Alzheimer-like', is of high clinical relevance. We evaluated whether a supervised machine learning technique, support vector machines (SVM), can replicate the assignments made by visual readers blind to the clinical diagnosis, which image components have highest diagnostic value according to SVM and how (18)F-flutemetamol-based classification using SVM relates to structural MRI-based classification using SVM within the same subjects. By means of SVM with a linear kernel, we analyzed (18)F-flutemetamol scans and volumetric MRI scans from 72 cases from the (18)F-flutemetamol phase 2 study (27 clinically probable Alzheimer's disease (AD), 20 amnestic mild cognitive impairment (MCI), 25 controls). In a leave-one-out approach, we trained the (18)F-flutemetamol based classifier by means of the visual reads and tested whether the classifier was able to reproduce the assignment based on visual reads and which voxels had the highest feature weights. The (18)F-flutemetamol based classifier was able to replicate the assignments obtained by visual reads with 100% accuracy. The voxels with highest feature weights were in the striatum, precuneus, cingulate and middle frontal gyrus. Second, to determine concordance between the gray matter volume- and the (18)F-flutemetamol-based classification, we trained the classifier with the clinical diagnosis as gold standard. Overall sensitivity of the (18)F-flutemetamol- and the gray matter volume-based classifiers were identical (85.2%), albeit with discordant classification in three cases. Specificity of the (18)F-flutemetamol based classifier was 92% compared to 68% for MRI. In the MCI disease (AD) is defined histologically by the presence of extracellular \\u03b2-amyloid (A\\u03b2) plaques and intraneuronal neurofibrillary tangles in the cerebral cortex. The diagnosis of dementia, along with the prediction of who will develop dementia, has been assisted by magnetic resonance imaging and positron emission tomography (PET) by using [(18)F]fluorodeoxyglucose (FDG). These techniques, however, are not specific for AD. Based on the chemistry of histologic staining dyes, several A\\u03b2-specific positron-emitting radiotracers have been developed to image neuropathology of AD. Among these, [(11)C]PiB is the most studied A\\u03b2-binding PET radiopharmaceutical in the world. The histologic and biochemical specificity of PiB binding across different regions of the AD brain was demonstrated by showing a direct correlation between A\\u03b2-containing amyloid plaques and in vivo [(11)C]PiB retention measured by PET imaging. Because (11)C is not ideal for commercialization, several (18)F-labeled tracers have been developed. At this time, [(18)F]3'-F-PiB (Flutemetamol), (18)F-AV-45 (Florbetapir), and (18)F-AV-1 (Florbetaben) are undergoing extensive phase II and III clinical trials. This article provides a brief review of the amyloid biology and chemistry of A\\u03b2-specific (11)C and (18)F-PET radiopharmaceuticals. Clinical trials have clearly documented that PET radiopharmaceuticals capable of assessing A\\u03b2 content in vivo in the brains of AD subjects and subjects with mild cognitive impairment will be important as diagnostic agents to detect in vivo amyloid brain pathology. In addition,is well known that many clinical and genetic factors have been associated with beta-amyloid deposition, few studies have examined the interactions of such factors across different stages of Alzheimer's pathogenesis. METHODS: We used 18F-florbetapir F18 PET imaging to quantify neuritic beta-amyloid plaque density across four cortical regions in 602 elderly (55-94 years) subjects from the national ADNI biomarker study. The group comprised of 194 normal elderly, 212 early mild cognitive impairment [EMCI], 132 late mild cognitive impairment [LMCI], and 64 mild Alzheimer's (AD). FINDINGS: In a model incorporating multiple predictive factors, the effect of apolipoprotein E \\u03b54 and diagnosis was significant on all four cortical regions. The highest signals were seen in cingulate followed by frontal and parietal with lowest signals in temporal lobe (p<0.0001). The effect of apolipoprotein E \\u03b54 (Cohen's D 0.96) on beta-amyloid plaque density was approximately twice as large as the effect of a diagnosis of AD (Cohen's D 0.51) and thrice as large as the effect of a diagnosis of LMCI (Cohen's D 0.34) (p<0.0001). Surprisingly, ApoE \\u03b54+ normal controls had greater mean plaque density across all cortical regions than \\u03b54- EMCI and \\u03b54- LMCI (p<0.0001, p=0.0009) and showed higher, though non-significant, mean value than \\u03b54- AD patients (p<0.27). ApoE \\u03b54+ EMCI and LMCI subjects had significantly greater mean plaque density across all cortical regions than \\u03b54- AD patients (p<0.027, p<0.0001). INTERPRETATION: Neuritic amyloid plaque load across progressive clinical stages of AD varies strongly by ApoE4 genotype. These findings support the need for better pathology-based and supported diagnosis in routine practice. Our data alssitron emission tomography (PET) radiotracers, fluorodeoxyglucose F 18 ((18)F-FDG) is the compound most widely used in the diagnosis of neurodegenerative dementias. However, this compound shows a level of specificity and sensitivity for early Alzheimer's disease detection that is lower than that provided by high-affinity ligands for \\u03b2-amyloid (A\\u03b2). Among the new widely available fluorine 18 ((18)F)-labeled A\\u03b2 ligands, florbetapir F 18 ((18))F-AV-45; Amyvid\\u2122) showed clear qualitative and quantitative correlations between in vivo PET imaging and postmortem histopathologic analysis of A\\u03b2. Florbetapir F 18 stands out for its high A\\u03b2 affinity and its pharmacokinetic properties that allow 10-minute PET scan imaging within 90 minutes after administration (dose = 370 MBq). Importantly, no safety concerns for florbetapir F 18 were found in preclinical studies. In 2012, the U.S. Food and Drug Administration (FDA) approved Amyvid as a radiotracer helpful for excluding the presence of A\\u03b2 in the brain. It was the (Abeta) plaque formation is a hallmark of Alzheimer's disease (AD) and precedes the onset of dementia. Abeta imaging should allow earlier diagnosis, but clinical application is hindered by the short decay half-life of current Abeta-specific ligands. (18)F-BAY94-9172 is an Abeta ligand that, due to the half-life of (18)F, is suitable for clinical use. We thus studied the effectiveness of this ligand in identifying patients with AD. METHODS: 15 \"}, {\"question\": \"In which isochores are Alu elements enriched?\", \"ideal_answer\": [\"Alu elements are enriched in high GC% isochores due to reduced Alu loss by recombination in these regions. The frequency of Alu sequences increases with increasing GC, but attains a maximum in H2 isochores.\"], \"exact_answer\": [[\"H2\"], [\"H3\"]], \"context\": \" the vast majority of Alu sequences were shown to have the highest density in GC-rich isochoresAlu sequences being unstable in the GC-poor isochores but stable in the compositionally matching GC-rich isochores,Alu retrotransposons do not show a homogeneous distribution over the human genome but have a higher density in GC-rich (H) than in AT-rich (L) isochoresThis result, together with the known higher selective disadvantage of recombination products in H isochores, points to Alu-Alu recombination as the main agent provoking the density shift of Alus toward the GC-rich parts of the genomewhereas GC-rich Alus are mostly present in GC-rich isochoresthe present results on Alu and LINE stability/exclusion predict significant losses of Alu DNA from the GC-poor isochores during evolution, a phenomenon apparently due to negative selection against sequences that differ from the isochore compositionThe frequency of Alu sequences also increases with increasing GC, but attains a maximum in H2 isochores, in agreement with previous experimental dataThe CpG levels of both Alus and CpG islands increase with their GC levelscharacterized as a GC-rich isochore enriched for CpG islands, genes, and Alu-like repeatsMicrosatellites and SINES (Alu, B1, B2) are found at roughly equal frequencies in introns from all isochore classesThe results indicate that the short repeats of the B1 family of mouse and of the Alu I family of man are most frequent in the heavy components, whereas the long repeats of the BamHI family of mouse and of the Kpn I family of man are mainly present in the two light components.\"}, {\"question\": \"Which are the mutational hotspots of the human KRAS oncogene?\", \"ideal_answer\": [\"The KRAS oncogene has four main mutational hotspots located at codons 12, 13, 61 and 146.\", \"The mutational hotspots for the K-ras oncogene are codons 12 and 13\"], \"exact_answer\": [[\"Codon 12\"], [\"Codon 13\"], [\"Codon 61\"], [\"Codon 146\"], [\"Codon 18\"], [\"Codon 31\"], [\"Codon 15\"]], \"context\": \"In this study we analyzed 15 sMTC for mutations in the hotspots codon 12, 13 and 61 of the H- and K-ras oncogene.mutation in the codon 12, 13 and 61 of the H-ras and K-ras oncogenethe mutation hotspots of the K-ras proto-oncogene in human functional adrenocortical tumors are in codons 15, 16, 18, and 31mutations C742T, G746T, G747T in the TP53 gene and G35T in the KRAS gene have been repeatedly found in sectors of human tumorsexons 1 and 2 of KRASKRAS (codon 12/13 hotspot)three hotspots in KRAS (codons 12/13, 61 and 146)analysis of KRAS (codon12/13)mutations occurring in mutational hotspots (codons 12 and 13) of the KRAS proteinand N-, K-, and H-RAS (codons 12, 13, and 61)\"}, {\"question\": \"Which are the Chompret criteria for Li-Fraumeni syndrome?\", \"ideal_answer\": [\"1) According to the Chompret criteria for LFS, any patient with adrenocortical cancer (ACC), irrespective of age and family history, is at high risk for a TP53 germline mutation.\\n2) All families with a p53 mutation must have at least one family member with a sarcoma, breast, brain, or adrenocortical carcinoma (ACC).\"], \"exact_answer\": [[\"Any patient with adrenocortical cancer (ACC), irrespective of age and family history.\"], [\"All families with a p53 mutation must have at least one family member with a sarcoma, breast, brain, or adrenocortical carcinoma (ACC).\"]], \"context\": \"According to the Chompret criteria for LFS, any patient with adrenocortical cancer (ACC), irrespective of age and family history, is at high risk for a TP53 germline mutation. All families with a p53 mutation had at least one family member with a sarcoma, breast, brain, or adrenocortical carcinoma (ACC).\"}, {\"question\": \"When ceritinib used instead of crizotinib?\", \"ideal_answer\": [\"Ceritinib is approved for the treatment of ALK-positive metastatic NSCLC patients that are crizotinib-resistant and crizotinib-na\\u00efve.\"], \"exact_answer\": [[\"crizotinib-resistant ALK-positive metastatic NSCLC patients\"], [\"crizotinib-na\\u00efve ALK-positive metastatic NSCLC patients\"]], \"context\": \"Ceritinib is a second-generation ALK inhibitor that has demonstrated activity in crizotinib-resistant patients, becoming a promising treatment option in this population. Ceritinib is approved for the treatment of ALK-positive metastatic NSCLC in patients who are intolerant to or have progressed despite therapy with crizotinib. Ceritinib has activity in crizotinib-resistant and crizotinib-na\\u00efve patients and appears to be a viable alternative for ALK-positive NSCLC.In this review, we aim to present the current knowledge on acquired resistance of crizotinib known as a first-in-class ALK inhibitor and potential solutions to improve the cost-effectiveness, and to review the difference between ceritinib and crizotinib; preclinical data and results of the elegant early clinical trial of ceritinib which promoted its accelerated approval, pharmacokinetics, safety profile, and tolerability, the updated results (eg, efficacy on brain metastases), and robust design of ongoing phase II/III trials, and future directions of ceritinib to be a potent alternative to crizotinib for ALK-rearranged non-small-cell lung cancer are also presented.Altogether, our results demonstrate that ceritinib can overcome crizotinib resistance, consistent with clinical data showing marked efficacy of ceritinib in patients with crizotinib-resistant disease.Preclinical data showed impressive antitumor activity against crizotinib-resistant clones, and based on available data, ceritinib could represent a suitable option in crizotinib-resistant NSCLC.The ALK inhibitor ceritinib overcomes crizotinib resistance in non-small cell lung cancer.Ceritinib (LDK378): a potent alternative to crizotinib for ALK-rearranged non-small-cell lung cancer.However, we observed that ceritinib did not overcome two crizotinib-resistant ALK mutations, G1202R and F1174C, and one of these mutations was identified in 5 of 11 biopsies from patients with acquired resistance to ceritinib. Altogether, our results demonstrate that ceritinib can overcome crizotinib resistance, consistent with clinical data showing marked efficacy of ceritinib in patients with crizotinib-resistant disease.SIGNIFICANCE: The second-generation ALK inhibitor ceritinib can overcome several crizotinib-resistant mutations and is potent against several in vitro and in vivo laboratory models of acquired resistance to crizotinib. These findings provide the molecular basis for the marked clinical activity of ceritinib in patients with ALK-positive NSCLC with crizotinib-resistant disease. However, we observed that ceritinib did not overcome two crizotinib-resistant ALK mutations, G1202R and F1174C, and one of these mutations was identified in 5 of 11 biopsies from patients with acquired resistance to ceritinib. Altogether, our results demonstrate that ceritinib can overcome crizotinib resistance, consistent with clinical data showing marked efficacy of ceritinib in patients with crizotinib-resistant disease.Altogether, our results demonstrate that ceritinib can overcome crizotinib resistance, consistent with clinical data showing marked efficacy of ceritinib in patients with crizotinib-resistant disease.  The second-generation ALK inhibitor ceritinib can overcome several crizotinib-resistant mutations and is potent against several in vitro and in vivo laboratory models of acquired resistance to crizotinib.The accelerated approval of potent ALK inhibitors, such as crizotinib and more recently ceritinib (LDK378), based on the well designed phase I/II trials has been a landmark success in clinical cancer research and contributes a new era of oncogenic targeted therapy characterized by elegant clinical trial design. In this review, we aim to present the current knowledge on acquired resistance of crizotinib known as a first-in-class ALK inhibitor and potential solutions to improve the cost-effectiveness, and to review the difference between ceritinib and crizotinib; preclinical data and results of the elegant early clinical trial of ceritinib which promoted its accelerated approval, pharmacokinetics, safety profile, and tolerability, the updated results (eg, efficacy on brain metastases), and robust design of ongoing phase II/III trials, and future directions of ceritinib to be a potent alternative to crizotinib for ALK-rearranged non-small-cell lung cancer are also presented.The second-generation ALK inhibitor ceritinib can overcome several crizotinib-resistant mutations and is potent against several in vitro and in vivo laboratory models of acquired resistance to crizotinib. These findings provide the molecular basis for the marked clinical activity of ceritinib in patients with ALK-positive NSCLC with crizotinib-resistant disease.In this review, we aim to present the current knowledge on acquired resistance of crizotinib known as a first-in-class ALK inhibitor and potential solutions to improve the cost-effectiveness, and to review the difference between ceritinib and crizotinib; preclinical data and results of the elegant early clinical trial of ceritinib which promoted its accelerated approval, pharmacokinetics, safety profile, and tolerability, the updated results (eg, efficacy on brain metastases), and robust design of ongoing phase II/III trials, and future directions of ceritinib to be a potent alternative to crizotinib for ALK-rearranged non-small-cell lung cancer are also presented.  Copyright \\u00a9 2015 Elsevier Inc.\"}, {\"question\": \"Which are the thyroid hormone analogs utilized in human studies?\", \"ideal_answer\": [\"TRIAC and TETRAC are two different thyroid hormone analogs utilized in human studies\"], \"exact_answer\": [[\"TRIAC\", \"TETRAC\"]], \"context\": \"Here, we evaluate the therapeutic potential of the TH analog 3,5,3',5'-tetraiodothyroacetic acid (tetrac) as a replacement for T(4) in brain development. This treatment was sufficient to promote TH-dependent neuronal differentiation in the cerebellum, cerebral cortex, and striatum but was ineffective in suppressing hypothalamic TRH expression. In the current study, we investigated the effect of a natural thyroid hormone analogue - 3, 3?, 5-triiodo-thyroacetic acid (TRIAC) on regulating proliferation and differentiation and its possible molecular mechanism in normal human epidermal keratinocytes and C57BL/6 mice.TRIAC was effective and safe in ameliorating the effects of hyperthyroidism and ADHD symptoms in a child with known genetic RTH.\"}, {\"question\": \"List Kartagener Syndrome Triad.\", \"ideal_answer\": [\"The triad of situs inversus, bronchiectasis and sinusitis is known as Kartagener syndrome.\"], \"exact_answer\": [[\"situs inversus\"], [\"bronchiectasis\"], [\"sinusitis\"]], \"context\": \"The triad of situs inversus, bronchiectasis and sinusitis is known as Kartagener syndrome. BACKGROUND: KARTAGENER SYNDROME (KS) IS A RARE CONGENITAL DISEASE CHARACTERISED BY A CLINICAL TRIAD OF SYMPTOMS: situs inversus, chronic rhinosinusitis, and bronchiectasis. Kartagener's syndrome is a very rare congenital malformation comprising of a classic triad of sinusitis, situs inversus and bronchiectasis.We present a case of a patient with clinically definite ALS, who had earlier suffered from Kartagener syndrome, which is characterized by the triad comprising chronic sinusitis, bronchiectasis, and situs inversus.Kartagener syndrome (KS), an autosomal recessively inherited disease, is characterized by the triad of situs inversus, bronchiectasis and sinusitis.In 1933, Manes Kartagener, a Zurich pulmonary physician, reported four patients with the triad of sinusitis, bronchiectasis, and situs inversus.A case of a nine year and eight months old child with Kartagener's syndrome (triad) is described: chronic maxillary sinusitis, bronchiectasis and \\\"situs inversus totalis\\\".Kartagener's syndrome is an inherited disease characterized by a triad of symptoms--bronchiectasis, situs inversus and sinusitis--and is classified as an immotile cilia syndrome.Kartagener's syndrome is characterized by the clinical triad of bronchitis, sinusitis, and situs inversus.Kartagener's syndrome is a well known classical triad of presentations consisting of bronchiectasis, sinusitis and situs inversus.Kartagener's syndrome is a rare disorder characterized by the triad of situs inversus, including dextrocardia, bronchiectasis and paranasal sinusitis.Kartagener's syndrome is an inherited disease characterized by a triad of symptoms: bronchiectasis, situs inversus and sinusitis resulting from defective cilial motility.Ear, nose and throat symptoms and signs were studied in 15 patients with Kartagener's syndrome: a triad consisting of chronic rhinosinusitis, chronic bronchitis with bronchiectasis, and situs inversus.Kartagener syndrome (a clinical variant of primary ciliary dyskinesia) is a recessive autossomical disease characterized by the triad of chronic sinusitis, bronchiectasis and situs inversus with dextrocardiaKartagener Syndrome is a rare autosomal recessive disorder consisting of triad of sinusitis, bronchiectasis and situs inversus with dextrocardiaKartagener syndrome (KS), an autosomal recessively inherited disease, is characterized by the triad of situs inversus, bronchiectasis and sinusitisKartagener&apos;s syndrome is an inherited disease characterized by a triad of symptoms--bronchiectasis, situs inversus and sinusitis--and is classified as an immotile cilia syndromeKartagener&apos;s syndrome is a very rare congenital malformation comprising of a classic triad of sinusitis, situs inversus and bronchiectasisA comprehensive clinicomorphological examination of 24 children with Zivert-Kartagener syndrome ascertained the complete triad (bronchiectasis, maldevelopment of the sinuses and transposition of the viscera) in all of themThe triad situs viscerum inversus, bronchiectasis and sinusitis is known as Kartagener syndromeWe present a case of a patient with clinically definite ALS, who had earlier suffered from Kartagener syndrome, which is characterized by the triad comprising chronic sinusitis, bronchiectasis, and situs inversus. Recent linkage and mutational analyses identified several genes that are responsible for Kartagener syndrome.\"}, {\"question\": \"Which are the main functions of the APOBEC3 family of proteins?\", \"ideal_answer\": [\"The APOBEC3 family of cytidine deaminases play a critical role in host-mediated defense against exogenous viruses, most notably, human immunodeficiency virus type-1 (HIV-1), and endogenous transposable elements, such as LINE-1 and Alu retrotransposons.\"], \"exact_answer\": [[\"host-mediated defense against exogenous viruses\"], [\"defense against endogenous transposable elements\"]], \"context\": \"The APOBEC3 cytidine deaminases play a critical role in host-mediated defense against exogenous viruses, most notably, human immunodeficiency virus type-1 (HIV-1) and endogenous transposable elementsthe identification of APOBEC3G (A3G) as a potent restriction factor of HIV-1additional A3 family members have been identified as antiviral proteins, mechanistic details of the restrictive capacity of these proteins have been elucidatedHuman APOBEC3 (A3) proteins are cellular cytidine deaminases that potently restrict the replication of retroviruses by hypermutating viral cDNA and/or inhibiting reverse transcriptionA3F and A3G are the most potent inhibitors of HIV-1APOBEC3A is a specific inhibitor of the early phases of HIV-1 infection in myeloid cellswe demonstrate that the pool of APOBEC3A in target cells inhibits the early phases of HIV-1 infectionOur results indicate that APOBEC3A is a previously unrecognized antiviral factor that targets primate lentiviruses specifically in myeloid cells and that acts during the early phases of infection directly in target cellsFoamy virus Bet proteins function as novel inhibitors of the APOBEC3 family of innate antiretroviral defense factors.This inhibition correlated with the packaging of inhibitory APOBEC3 proteins into PFV virions, due to a specific PFV Gag/APOBEC3 interaction, and resulted in the G to A hypermutation of PFV reverse transcripts[Antiviral defense by APOBEC3 family proteins].All APOBEC3 family proteins differentially inhibit LINE-1 retrotransposition.These findings raise the possibility that the different APOBEC3 family members function to neutralize specific lentiviruses.\"}, {\"question\": \"Which histone modifications are associated with constitutive heterochromatin?\", \"ideal_answer\": [\"Strong methylation at H3 lysine 9 occurred preferentially in heterochromatic chromocenters of Arabidopsis nuclei. In general, heterochromatin has been linked to trimethylation of H3 at lysine 9 and parsimony analysis reveal that histone H3K9 methylation is, next to histone deacetylation, the evolutionary most stable heterochromatic mark. Classical histone modifications associated with heterochromatin also include H3K27me1 and H3K27me2. H3K36me3 function is not restricted to actively transcribed regions only and may contribute to the composition of heterochromatin. Other histone methylation marks usually found in constitutive heterochromatin are H4K20me3, H3K9me3, and H3K79me3. H3S10P is a good marker of pericentromeric heterochromatin.\", \"H3K9 methylation\\nH3S10 phosphorylation\\nH3K79 and H4K20 methylation\"], \"exact_answer\": [[\"H3K9Me1\", \"H3K9Me2\", \"H3K9me3\", \"H3K9 trimethylation\"], [\"H3K27me1\"], [\"H3K27me2\"], [\"H4K20me3\", \"H4K20 trimethylation\"], [\"H3K79me3\", \"H3K79 methylation\"], [\"H3S10\", \"H3S10 phosphorylation\"]], \"context\": \"Remarkably, in mammalian somatic cells, H3S10P initiates in the pericentromeric heterochromatin during the late G2 phase, and phosphorylation spreads throughout the chromosomes arms in prophase, being maintained until the onset of anaphase when it gets dephosphorylatedBased on these findings, we believe that H3S10P is a good marker of pericentromeric heterochromatin, especially in the late 1- and 2-cell stages as it labels both parental genomes and that it can be used to further investigate epigenetic regulation and heterochromatin mechanisms in early preimplantation embryos3S10 phosphorylation marks constitutive heterochromatin during interphase in early mouse embryos until the 4-cell stageH3 trimethylation at lysine 36 is associated with constitutive and facultative heterochromatinWe show here that the maternally inherited Snurf-Snrpn 3-Mb region, which is silenced by a potent transcription repressive mechanism, is uniformly enriched in histone methylation marks usually found in constitutive heterochromatin, such as H4K20me3, H3K9me3, and H3K79me3We show that H3K36me3 deposition within this large heterochromatin domain does not correlate with transcription events, suggesting the existence of an alternative pathway for the deposition of this histone modificationIn addition, we demonstrate that H3K36me3 is markedly enriched at the level of pericentromeric heterochromatin in mouse embryonic stem cells and fibroblastsThis result indicates that H3K36me3 is associated with both facultative and constitutive heterochromatinOur data suggest that H3K36me3 function is not restricted to actively transcribed regions only and may contribute to the composition of heterochromatin, in combination with other histone modifications.accumulation of histone H3.3 and euchromatic histone modifications in pericentromeric heterochromatin in response to a decrease in DNA methylation levelsIn this study, we show that inhibition of DNA methylation in mouse fibroblast cells affects histone modification and the subnuclear localization of histone H3.3 in a cell cycle-dependent mannerMoreover, we found that histone H3.3 was deposited on the pericentromeric heterochromatin prior to the accumulation of the euchromatic histone modificationsClassical histone modifications associated with heterochromatin, including H3K9me2, H3K27me1 and H3K27me2, were distributed throughout both A and B chromosomesHowever, H3K27me2 showed a reduced level on the B chromosome compared with the A chromosomes and was not associated with some classes of constitutive heterochromatinRecently analyzed H3K9 methyltransferases (HMTases) as SUV39H1, Clr4p, DIM-5, Su(var)3-9 or SUVH2 are responsible for the establishment of histone H3 lysine 9 methylation (H3K9me), which is intimately connected with heterochromatinizationheterochromatic histone H3K9 methyltransferases left its marks behindCompilation and parsimony analysis reveal that histone H3K9 methylation is, next to histone deacetylation, the evolutionary most stable heterochromatic mark, which is established by at least two subfamilies of specialized heterochromatic HMTases in almost all studied eukaryoteTrimethylated H3(K9) marks pericentromeric constitutive heterochromatin and the male Y chromosome, while H2A.Z is dramatically reduced at these chromosomal locationsOf eight tested antibodies, the one for histone H4 acetylated at lysine 4, 8, 12, or 16 was best for distinguishing constitutive heterochromatin from unexpressed euchromatin, but differences in the extent of immunoprecipitation of these two types of chromatin were only modest, although highly reproducibleIn addition, immunocytochemical analysis was done with an antibody to heterochromatin protein 1alpha (HP1alpha), whose preferential binding to heterochromatin has been linked to trimethylation of H3 at lysine 9Our combined ChIP and immunocytochemical results suggest that factors other than hypoacetylation of the N-terminal tails of H4 and hypermethylation of H3 at lysine 9 can play an important role in determining whether a chromatin sequence in mammalian cells is constitutively heterochromaticTo test for the putative heterochromatinization, we quantitated chromatin immunoprecipitation with an antibody for acetylated histone H4 that discriminates between constitutive heterochromatin and unexpressed euchromatinhigh methylation of H3 lysine 9 is dispensable for constitutive heterochromatinStrong methylation at H3 lysine 4 was restricted to euchromatin, while strong methylation at H3 lysine 9 occurred preferentially in heterochromatic chromocenters of Arabidopsis nuclei\"}, {\"question\": \"Which two catechol-O-methyl transferase (COMT) inhibitors can be used for treatment of Parkinson disease?\", \"ideal_answer\": [\"Tolcapone (central and peripheral) and entacapone (peripheral) are catechol-O-methyl transferase inhibitors that are used for treatment of Parkinson disease.\"], \"exact_answer\": [[\"tolcapone\"], [\"entacapone\"]], \"context\": \"Entacapone is a highly potent, reversible, peripherally acting catechol-O-methyl transferase (COMT) inhibitor that is used as an adjunct to L-dopa in the treatment of patients with Parkinson disease (PD).Stalevo is a combination tablet comprised of levodopa, carbidopa, and the COMT inhibitor entacapone. It is available in fixed-dose combinations of levodopa/carbidopa/entacapone, 50/12.5/200, 75/18.75/200, 100/25/200, 125/31.25/200, 150/37.5/200 and 200/50/200 mg. Stalevo is currently approved for use in Parkinson's disease patients with end-of-dose wearing off.Good news were marketing of ropinirole, a new non-ergot agonist, in December 2006 and entacapone, the first catechol-O-methyl transferase (COMT) inhibitor in Japan in April 2007. Having faced these new situations, Japanese Neurological Association has started revising \\\"the Guideline 2002 for the treatment of Parkinson's disease\\\".Stalevo (Orion) combines levodopa, the dopa-decarboxylase inhibitor carbidopa and the catechol-O-methyl transferase inhibitor entacapone in a single tablet.We investigated whether administration of the catechol-O-methyl transferase (COMT) inhibitor entacapone, at doses of 200 mg and 400 mg, alters the pharmacokinetics of apomorphine in Parkinson's disease patients experiencing severe motor fluctuations.The catechol-O-methyl transferase inhibitor entacapone is given in combination with levodopa/dopa decarboxylase inhibitor for Parkinson's disease (PD) patients experiencing end-of-dose wearing-off.Entacapone (Comtess/Comtan) is Orion Pharma's original proprietary catechol-O-methyl transferase (COMT) inhibitor. Entacapone is able to slow down degradation of levodopa and improve the availability and efficacy of each levodopa dose, hence its use as a complement to levodopa/carbidopa in patients with Parkinson's disease.Sleep attacks in Parkinson's disease induced by Entacapone, a COMT-inhibitor.We present a patient who suffered from sleep attacks after starting entacapone in addition to levodopa. Entacapone, a catechol-O-methyl transferase inhibitor, alters the pharmacokinetics of levodopa, leading to increase of levodopa concentration in plasma and brain.Two COMT inhibitors, tolcapone and entacapone, have recently been made available as adjunctive agents to levodopaTwo inhibitors of catechol-O-methyl transferase (COMT), tolcapone and entacapone, have recently been introduced as adjuncts to levodopa in the treatment of Parkinson's disease patients.Tolcapone, a peripheral and central COMT inhibitor, appears to be quite effective, producing a 47% reduction in 'off' time.. Entacapone, a purely peripheral COMT inhibitor with a lower potency than tolcapone, has also proved to be effective and has not been associated with liver damage, obviating the need for testing.Here we report that a treatment of the cerebral tissue with tolcapone, a central and peripheral inhibitor of COMT, does not change the membrane responses of midbrain dopamine neurons to dopamine and levodopa.Therefore, the therapeutic action of tolcapone in Parkinson's disease, might be dependent on the reduction of COMT activity in the extracerebral tissue.Pretreatment with entacapone (OR-611), a peripheral catechol O-methyl-transferase (COMT) inhibitor, greatly reduces the plasma 3OMFD fraction and provides an ideal situation to evaluate the contribution of the plasma 3OMFD fraction in several kinetic models of FDOPA uptake.The novel and reversible inhibitors are the catechol-O-methyl-transferase (COMT) inhibitors, Ro 40-7592 (3,4-dihydroxy-4'-methyl-5-nitrobenzophenone), and the monoamine oxidase type-B (MAO-B) inhibitor, Ro 19-6327 (N-(2-aminoethyl)-5-chloro-2-pyridine carboxamide HC1).\"}, {\"question\": \"Mutations of which genes have been associated with Catecholaminergic Polymorphic Ventricular Tachycardia (CPVT)?\", \"ideal_answer\": [\"Mutations in five genes \\u2013 ryanodine receptor 2 (RYR2), calsequestrin 2(CASQ2), triadic (TRDN), calmodulin 1 (CALM1) and potassium channel, inwardly rectifying subfamily J, member 2 (KCNJ2)  \\u2013 have been found to be associated with CPVT\"], \"exact_answer\": [[\"ryanodine receptor 2\", \"RYR2\"], [\"calsequestrin 2\", \"CASQ2\"], [\"triadin\", \"TRDN\"], [\"calmodulin 1\", \"CALM1\"], [\"potassium channel, inwardly rectifying subfamily J, member 2\", \"KCNJ2\"]], \"context\": \"Mutation in four genes \\u2013 RYR2, CASQ2, TRDN, and CALM1 \\u2013 is known to cause CPVT or related phenotypes of adrenergically induced life-threatening arrhythmias. The presence of other as-yet unidentified loci is postulated some KCNJ2 mutation carriers lack the ATS triad and sometimes share the phenotype of catecholaminergic polymorphic ventricular tachycardia (CPVT)In 50 Japanese probands from unrelated families who satisfied clinical criteria for CPVT, genetic testing was conducted in all exons on 3 CPVT-related genes: cardiac ryanodine receptor 2 (RYR2), calsequestrin 2 (CASQ2) and inward rectifier potassium channel 2 (KCNJ2)point mutations in RYR2, the gene encoding for the cardiac isoform of the RyR (RyR2), are associated with catecholaminergic polymorphic ventricular tachycardia (CPVT), an arrhythmogenic syndrome characterized by the development of adrenergically-mediated ventricular tachycardia in individuals with an apparently normal heart.Recessive CPVT variants are due to mutations in the CASQ2 gene. Recent studies have shown that CPVT is caused by mutations in the cardiac ryanodine receptor type 2 (RyR2) or calsequestrin 2 (CASQ2) genes.We identified three mutations in triadin which cosegregated with the disease on a recessive mode of transmission in two familiesCatecholaminergic polymorphic ventricular tachycardia (CPVT) is a familial arrhythmogenic disorder associated with mutations in the cardiac ryanodine receptor (RyR2) and cardiac calsequestrin (CASQ2) genes.BACKGROUND: Catecholaminergic polymorphic ventricular tachycardia (CPVT) is an inherited arrhythmia syndrome associated with mutations in the cardiac ryanodine receptor gene (Ryr2) in the majority of patients.BACKGROUND: Catecholaminergic polymorphic ventricular tachycardia (CPVT) is a familial arrhythmic syndrome caused by mutations in genes encoding the calcium-regulation proteins cardiac ryanodine receptor (RyR2) or calsequestrin-2 (CASQ2).Catecholaminergic polymorphic ventricular tachycardia (CPVT) is an inherited arrhythmogenic disease so far related to mutations in the cardiac ryanodine receptor (RYR2) or the cardiac calsequestrin (CASQ2) genes.Catecholaminergic polymorphic ventricular tachycardia (CPVT) is an inherited arrhythmia syndrome associated with mutations in the cardiac ryanodine receptor gene (Ryr2) in the majority of patients\"}, {\"question\": \"List programs suitable for pharmacophore modelling\", \"ideal_answer\": [\"A pharmacophore is an abstract description of molecular features which are necessary for molecular recognition of a ligand by a biological macromolecule. The IUPAC defines a pharmacophore to be \\\"an ensemble of steric and electronic features that is necessary to ensure the optimal supramolecular interactions with a specific biological target and to trigger (or block) its biological response\\\". A pharmacophore model explains how structurally diverse ligands can bind to a common receptor site. Furthermore pharmacophore models can be used to identify through denovo design or virtual screening novel ligands that will bind to the same receptor. Nowadays there are many programs suitable for pharmacophore modelling such as LigandScout, Discovery Studio, Catalyst, PharmaGist, Genetic Algorithm Similarity and Molecular Operating Environment.\"], \"exact_answer\": [[\"LigandScout program\"], [\"Discovery Studio program\", \"3D QSAR Pharmacophore Generation module\"], [\"Catalyst\", \"HipHop module\", \"HypoGen module\"], [\"PharmaGist\"], [\"Genetic Algorithm Similarity Program (GASP)\"], [\"Molecular Operating Environment (MOE)\", \"Pharmacophore Elucidation\"]], \"context\": \"Pharmacophore models based on thermolysin binding modes and activity profiles were generated using the LigandScout programIn this study, pharmacophore hypotheses based on known inhibitors were generated using common feature pharmacophore generation protocol available in Discovery Studio program A pharmacophore-based screening was then carried out using the program CatalystThe PharmaGist employed algorithm to identify the best pharmacophores by computing multiple flexible alignments between the input ligands. The multiple alignments are generated by combining alignments pair-wise between one of the gliptin input ligands, which acts as pivot and the other gliptin as ligand. The resulting multiple alignments reveal spatial arrangements of consensus features shared by different subsets of input ligands. The best pharmacophore model has been derived using both pair-wise and multiple alignment methods, which have been weighted in Pharmacophore Generation processPharmacophore models of c-Met kinase inhibitors have been developed based on 22 diverse compounds by using HypoGen algorithm implemented in Discovery studio program package.chemical feature based pharmacophore models of MMP-1, MMP-8 and MMP-13 inhibitors have been developed with the aid of HypoGen module within Catalyst program packageGenetic Algorithm Similarity Program (GASP) was used to derive a 3D pharmacophore model which was used in effective alignment of data set.Active site complimenting structure-based pharmacophore models were developed using Discovery Studio 2.5 program and validated using a dataset of known HDAC8 inhibitors.  pharmacophore model has been generated from the training set by means of the MOE (molecular operating environment) program.A training set of 20 compounds with known experimental activity was used to generate pharmacophore hypotheses using 3D QSAR Pharmacophore Generation module available in Discovery studio. Pharmacophore model was developed for the first time, on a training data set of 22 CCR3 antagonists, using CATALYST HypoRefine program.The CatalystHipHop approach was used to generate a pharmacophore model for cyclooxygenase-2 (COX-2) inhibitors based on a training set of 15 active inhibitors (1\\u00e215).  Molecular modeling study, including fitting to a 3D-pharmacophore model of the designed molecules (10a-g), with HCV NS3 protease hypothesis using catalyst program was fulfilled.pharmacophore model (Hypo1) with a well prediction capacity for CysLT(1) antagonists was developed using Catalyst/HypoGen program. Three pharmacophore models of opioid agonists were generated by the Catalyst/HypoGen programThe superposition of the ligands was performed with the tool Pharmacophore Elucidation (MOE-packageKnown PDE-5 inhibitors were used to construct a three dimensional quantitative structure-activity relationship (3D QSAR) model by HypoGen program. pharmacophore models were generated for AT1 and ET(A) receptors based on highly selective AT1 and ET(A) antagonists using the program Catalyst/HipHop In this study, chemical feature based pharmacophore models of inhibitors of 5-LOX have been developed with the aid of HipHop and HypoGen modules within Catalyst program packageA data set consisting of 24 inhibitors was selected on the basis of the information content of the structures and activity data as required by the Catalyst/HypoGen program.\"}, {\"question\": \"List disorders that are caused by mutations in the mitochondrial MTND6 gene.\", \"ideal_answer\": [\"Mitochondrial MTND6 gene mutations are the cause of Leigh syndrome and Leber's hereditary optic neuropathy and/or dystonia.\"], \"exact_answer\": [[\"Leigh syndrome\"], [\"Leber's hereditary optic neuropathy and/or dystonia\"]], \"context\": \"It was shown that m14487T>C can also cause the unusual combination of optic atrophy, ptosis, and encephalomyopathy leading to intractable seizures.  MTND6 genes were identified as 'hotspots' of mutations, with Leigh syndrome accounting for the large majority of associated phenotypes. he 14487T>C mutation in MTND6 were present in two probands with Leigh's-like and Leigh's syndrome, respectively. These results confirm the association of the MTND6*LDYT14459A mutation with Leber's hereditary optic neuropathy and/or dystonia. A novel point mutation in the ND6 subunit of complex I at position 14,459 of the mitochondrial DNA (MTND6*LDY T14459A) was identified as a candidate mutation for the highly tissue-specific disease. Leber's hereditary optic neuropathy plus dystonia.Representative of these mutations is a heteroplasmic mutation in MTND6 at np 14459 whose clinical presentations range from adult-onset blindness to pediatric dystonia and basal ganglial degeneration. \"}, {\"question\": \"Which are the most common methods for ctDNA (circulating tumour DNA) detection?\", \"ideal_answer\": [\"Recently, nanoplasmonics has emerged as a platform for one-step dual detection with high sensitivity and specificity. The practice of \\\"liquid biopsy\\\" as a diagnostic, prognostic and theranostic tool in non-small cell lung cancer (NSCLC) patients is an appealing approach, at least in theory, since it is noninvasive and easily repeated. Cancer personalized profiling by deep sequencing (CAPP-Seq), an economical and ultrasensitive method for quantifying ctDNA. A new DNA sensor using a nickel(II) phenanthroline complex ([Ni(phen)(2)PHPIP]\\u00b72ClO(4)) as the electrochemical probe was developed. The sensor is very sensitive and selective for calf thymus DNA (ctDNA) detection in aqueous medium.\", \"A new DNA sensor using a nickel(II) phenanthroline complex ([Ni(phen)(2)PHPIP]\\u00b72ClO(4)) as the electrochemical probe was developed. The calculated dynamics parameters of the electrode process indicate that there are obvious interactions between the probe and the ctDNA in aqueous solution. Under constant potential conditions, the redox current peak of the probe (Ni-complex) decreases obviously as the probe interacts/binds with ctDNAs. These results demonstrate that the sensor can simultaneously detect the hot-spot mutation and epigenetic changes on the ctDNA.\"], \"exact_answer\": [[\"nanoplasmonics\", \"biosensors\"], [\"liquid biopsy\"], [\"Cancer personalized profiling by deep sequencing\", \"CAPP-Seq\"], [\"electrochemical sensors\"]], \"context\": \"existing methods for ctDNA detection are restricted to genetic mutations.Recently, nanoplasmonics has emerged as a platform for one-step dual detection with high sensitivity and specificity.These results demonstrate that the sensor can simultaneously detect the hot-spot mutation and epigenetic changes on the ctDNA. Promisingly, other specific-tumor mutants and epigenetic changes can be detected at low concentration with this platform.The practice of \\\"liquid biopsy\\\" as a diagnostic, prognostic and theranostic tool in non-small cell lung cancer (NSCLC) patients is an appealing approach, at least in theory, since it is noninvasive and easily repeatedHere we introduce cancer personalized profiling by deep sequencing (CAPP-Seq), an economical and ultrasensitive method for quantifying ctDNA. Finally, we evaluated biopsy-free tumor screening and genotyping with CAPP-Seq. We envision that CAPP-Seq could be routinely applied clinically to detect and monitor diverse malignancies, thus facilitating personalized cancer therapy.Synthesis of a new Ni-phenanthroline complex and its application as an electrochemical probe for detection of nucleic acidA new DNA sensor using a nickel(II) phenanthroline complex ([Ni(phen)(2)PHPIP]\\u00b72ClO(4)) as the electrochemical probe was developed. The sensor is very sensitive and selective for calf thymus DNA (ctDNA) detection in aqueous medium. \"}, {\"question\": \"Which deiodinases are best known to be present in brain?\", \"ideal_answer\": [\"All the 3 deiodinases (Type 1, Type 2 and Type 3 deiodinase) are present in the \\\"brain\\\" but Type 1 deiodinase  is only found in neurohypophysis that cannot be actually considered true \\\"brain tissue\\\".\"], \"exact_answer\": [[\"Type 2 deiodinase\"], [\"Type 3 deiodinase\"]], \"context\": \"In brain, the presence of type III iodothyronine deiodinase (D3) seems to be important to maintain homeostasis of T(3) levels.We were able to detect a protein fragment corresponding to the expected molecular mass (30 kDa) for type III deiodinase by means of Western blot analysis. RT-PCR as well as Northern blot analysis confirmed the presence of D3 mRNA in the cerebellum.The presence of 5D (type-III) together with our previous report of 5'-deiodinase (type-I in euthyroidism and type-II in hypothyroidism) shows that the isozymes of deiodinases in the neurohypophysis are quite similar to those in the brain.\"}, {\"question\": \"What are the computational methods for the prediction of beta-barrel transmembrane proteins?\", \"ideal_answer\": [\"Computational tools have been developed for beta-barrel transmembrane protein discrimination, topology prediction and prediction of their structural features.\\nInitial methods developed for the prediction of the transmembrane beta strands were based on hydrophobicity analysis, using sliding windows along the sequence, in order to capture the alternating patterns of hydrophobic-hydrophilic residues of the transmembrane strands, or using generalized secondary structure prediction methods. Other approaches included the construction of special empirical rules using amino-acid propensities and prior knowledge of the structural nature of the proteins, and the development of Neural Network-based predictors to predict the location of alpha-carbon atoms with respect to the membrane. During the last few years, other more refined methods, appeared, including: Neural Networks, Hidden Markov Models, Support Vector Machines, k-Nearest Neighbors, Radial Basis Functions, Bayesian Networks, Genetic Algorithms, Mahalanobis Discriminant Functions, Cellular Automata, N-to-1 Extreme Learning Machines. Hidden Markov Model-based methods are among the most successful in topology prediction, being able to capture the unique architecture of beta-barrel transmembrane proteins. Consensus methods, as well as pipelines of several related tools (e.g. subcellular localization prediciton, alpha-helical transmembrane protein prediction, signal-peptide/lipoprotein prediction) have also used for discriminating beta-barrel transmembrane proteins. Recently, a number of methods for predicting more detailed structural features (e.g. surface accessibility, residue contacts, even detailed atomic 3D models) tailored to beta-barrel transmembrane proteins have been developed, based on knowledge-based potential functions, graph theoretic models, physical models and multi-tape S-attribute grammars. Methods/tools falling in the aforementioned classes are (listed in alphabetical order): BBF (beta-barrel finder), BETAWARE, BOCTOPUS, BOMP, BTMX (Beta barrel TransMembrane eXposure), HHomp, HMM-B2TMR, OMBBpred, PROFtmb, PRED-TMBB, TMB-Hunt, TBBPred, TMBETAPRED-RBF, TMBHMM, TransFold, TMBpro, TMBKNN, Wimley\"], \"exact_answer\": [[\"BBF\", \"beta-barrel finder\"], [\"BETAWARE\"], [\"BOCTOPUS\"], [\"BOMP\"], [\"BTMX\", \"Beta barrel TransMembrane eXposure\"], [\"HHomp\"], [\"HMM-B2TMR\"], [\"OMBBpred\"], [\"PROFtmb\"], [\"PRED-TMBB\"], [\"TMB-Hunt\"], [\"TBBPred\"], [\"TMBETAPRED-RBF\"], [\"TMBHMM\"], [\"TransFold\"], [\"TMBpro\"], [\"TMBKNN\"], [\"Wimley\"], [\"TMBETA-NET\"]], \"context\": \"BETAWARE: a machine-learning tool to detect and predict transmembrane beta-barrel proteins in prokaryotesRecently, we developed two top-performing methods based on machine-learning approaches to tackle both the detection of TMBBs in sets of proteins and the prediction of their topology. Here, we present our BETAWARE program that includes both approaches and can run as a standalone program on a linux-based computer to easily address in-home massive protein annotation or filtering.We introduce a graph-theoretic model for predicting the supersecondary structure of transmembrane \\u03b2-barrel proteins--a particular class of proteins that performs diverse important functions but it is difficult to determine their structure with experimental methods. This ab initio model resolves the protein folding problem based on pseudo-energy minimization with the aid of a simple probabilistic filter. It also allows for determining structures whose barrel follows a given permutation on the arrangement of \\u03b2-strands, and allows for rapidly discriminating the transmembrane \\u03b2-barrels from other kinds of proteins.BOCTOPUS: improved topology prediction of transmembrane \\u03b2 barrel proteinsImproving the detection of transmembrane \\u03b2-barrel chains with N-to-1 extreme learning machineswe introduce a new machine learning approach for TMBB detection based on N-to-1 Extreme Learning MachinesTMBHMM: a frequency profile based HMM for predicting the topology of transmembrane beta barrel proteins and the exposure status of transmembrane residuesWe present here TMBHMM, a computational method based on a hidden Markov model for predicting the structural topology of putative TMBs from sequence. In addition to predicting transmembrane strands, TMBHMM also predicts the exposure status (i.e., exposed to the membrane or hidden in the protein structure) of the residues in the transmembrane region, which is a novel feature of the TMBHMM method. Furthermore, TMBHMM can also predict the membrane residues that are not part of beta barrel forming strands.Prediction of the exposure status of transmembrane beta barrel residues from protein sequenceWe present BTMX (Beta barrel TransMembrane eXposure), a computational method to predict the exposure status (i.e. exposed to the bilayer or hidden in the protein structure) of transmembrane residues in transmembrane beta barrel proteins (TMBs).Combined prediction of transmembrane topology and signal peptide of beta-barrel proteins: using a hidden Markov model and genetic algorithmsWe present here, an HMM that combine a transmembrane barrel submodel and an SP submodel for both topology and SP predictions. A new genetic algorithm (GA) is presented here to training the model, at the same time the Posterior-Viterbi algorithm is adopted for decoding.In this work, we propose a method based on radial basis networks for predicting the number of beta-strands in OMPs and identifying their membrane spanning segments.We have developed a web server, TMBETAPRED-RBF for predicting the transmembrane beta-strands from amino acid sequence and it is available at http://rbf.bioinfo.tw/~sachen/tmrbf.htmlWe have developed a novel approach for dissecting transmembrane beta-barrel proteins (TMBs) in genomic sequences. The features include (i) the identification of TMBs using the preference of residue pairs in globular, transmembrane helical (TMH) and TMBs, (ii) elimination of globular/TMH proteins that show sequence identity of more than 70% for the coverage of 80% residues with known structures, (iii) elimination of globular/TMH proteins that have sequence identity of more than 60% with known sequences in SWISS-PROT, and (iv) exclusion of TMH proteins using SOSUI, a prediction system for TMH proteins.TMBpro: secondary structure, beta-contact and tertiary structure prediction of transmembrane beta-barrel proteinsWe develop a suite (TMBpro) of specialized predictors for predicting secondary structure (TMBpro-SS), beta-contacts (TMBpro-CON) and tertiary structure (TMBpro-3D) of transmembrane beta-barrel proteins.This paper presents a k-nearest neighbor (K-NN) method for discriminating TMB and non-TMB proteins.In this paper, based on the concept of pseudo amino acid composition (PseAA) that can incorporate sequence-order information of a protein sequence so as to remarkably enhance the power of discrete models (Chou, K. C., Proteins: Structure, Function, and Genetics, 2001, 43: 246-255), cellular automata and Lempel-Ziv complexity are introduced to predict the TM regions of integral membrane proteins including both alpha-helical and beta-barrel membrane proteins, validated by jackknife testwe present BOCTOPUS; an improved method for the topology prediction of TMBs by employing a combination of support vector machines (SVMs) and Hidden Markov Models (HMMs). The SVMs and HMMs account for local and global residue preferences, respectively.PRED-TMBB: a web server for predicting the topology of beta-barrel outer membrane proteinsWe present here a web server (PRED-TMBB, http://bioinformatics.biol.uoa.gr/PRED-TMBB) which is capable of predicting the transmembrane strands and the topology of beta-barrel outer membrane proteins of Gram-negative bacteria. The method is based on a Hidden Markov Model, trained according to the Conditional Maximum Likelihood criterion.Prediction of transmembrane regions of beta-barrel proteins using ANN- and SVM-based methodshis article describes a method developed for predicting transmembrane beta-barrel regions in membrane proteins using machine learning techniques: artificial neural network (ANN) and support vector machine (SVM). The ANN used in this study is a feed-forward neural network with a standard back-propagation training algorithm.We have also developed an SVM-based method using a primary sequence as input and achieved an accuracy of 77.4%. The SVM model was modified by adding 36 physicochemical parameters to the amino acid sequence information. Finally, ANN- and SVM-based methods were combined to utilize the full potential of both techniques.we have developed a Web server, TBBPred, for predicting transmembrane beta-barrel regions in proteinsPredicting transmembrane beta-barrels in proteomesHere we introduced the design, statistics and results of a novel profile-based hidden Markov model for the prediction and discrimination of TMBs.All proteome predictions and the PROFtmb prediction method are availableThe beta-barrel finder (BBF) program, allowing identification of outer membrane beta-barrel proteins encoded within prokaryotic genomesWe here report computational analyses of six outer membrane proteins of known 3D structures with respect to (1) secondary structure, (2) hydropathy, and (3) amphipathicity. Using these characteristics, as well as the presence of an N-terminal targeting sequence, a program was developed allowing prediction of integral membrane beta-barrel proteins encoded within any completely sequenced prokaryotic genome. This program, termed the beta-barrel finder (BBF) program, was used to analyze the proteins encoded within the Escherichia coli genome.A sequence-profile-based HMM for predicting and discriminating beta barrel membrane proteinsWe develop a HMM model, which can predict the topology of beta barrel membrane proteins using, as input, evolutionary information. The model is cyclic with 6 types of states: two for the beta strand transmembrane core, one for the beta strand cap on either side of the membrane, one for the inner loop, one for the outer loop and one for the globular domain state in the middle of each loop. The development of a specific input for HMM based on multiple sequence alignment is novel.Prediction of the transmembrane regions of beta-barrel membrane proteins with a neural network-based predictorA method based on neural networks is trained and tested on a nonredundant set of beta-barrel membrane proteins known at atomic resolution with a jackknife procedure. The method predicts the topography of transmembrane beta strands with residue accuracy as high as 78% when evolutionary information is used as input to the network.A web server based on the proposed method is available at http://yanbioinformatics.cs.usu.edu:8080/TMBKNNsubmit.Predicting three-dimensional structures of transmembrane domains of \\u03b2-barrel membrane proteinsWe have developed a computational method for predicting structures of the transmembrane (TM) domains of \\u03b2-barrel membrane proteins. Based on physical principles, our method can predict structures of the TM domain of \\u03b2-barrel membrane proteins of novel topology, including those from eukaryotic mitochondria. Our method is based on a model of physical interactions, a discrete conformational state space, an empirical potential function, as well as a model to account for interstrand loop entropy.exposure status classification of transmembrane beta barrel residuesSeveral computational methods exist for the identification of transmembrane beta barrel proteins (TMBs) from sequence. Some of these methods also provide the transmembrane (TM) boundaries of the putative TMBs. The aim of this study is to (1) derive the propensities of the TM residues to be exposed to the lipid bilayer and (2) to predict the exposure status (i.e. exposed to the bilayer or hidden in protein structure) of TMB residues.The modified Mahalanobis Discriminant for predicting outer membrane proteins by using Chou's pseudo amino acid composition.In this study, a method that combines increment of diversity with modified Mahalanobis Discriminant, called IDQD, is presented to predict 208 OMPs, 206 transmembrane helical proteins (TMHPs) and 673 globular proteins (GPs) by using Chou's pseudo amino acid compositions as parameters.Predicting transmembrane beta-barrels and interstrand residue interactions from sequenceA novel method using pairwise interstrand residue statistical potentials derived from globular (nonouter membrane) proteins is introduced to predict the supersecondary structure of transmembrane beta-barrel proteins. The algorithm transFold employs a generalized hidden Markov model (i.e., multitape S-attribute grammar) to describe potential beta-barrel supersecondary structures and then computes by dynamic programming the minimum free energy beta-barrel structure.Predicting the solvent accessibility of transmembrane residues from protein sequenceIn this study, we propose a novel method to predict the solvent accessible surface areas of transmembrane residues. For both transmembrane alpha-helix and beta-barrel residues, the correlation coefficients between the predicted and observed accessible surface areas are around 0.65.BOMP: a program to predict integral beta-barrel outer membrane proteins encoded within genomes of Gram-negative bacteriaThis work describes the development of a program that predicts whether or not a polypeptide sequence from a Gram-negative bacterium is an integral beta-barrel outer membrane protein. The program, called the beta-barrel Outer Membrane protein Predictor (BOMP), is based on two separate components to recognize integral beta-barrel proteins. The first component is a C-terminal pattern typical of many integral beta-barrel proteins. The second component calculates an integral beta-barrel score of the sequence based on the extent to which the sequence contains stretches of amino acids typical of transmembrane beta-strands.A HMM-based method to predict the transmembrane regions of beta-barrel membrane proteinsA novel method is developed to model and predict the transmembrane regions of beta-barrel membrane proteins. It is based on a Hidden Markov model (HMM) with architecture obeying those proteins' construction principles.First, an algorithm that calculates the mean hydrophobicity of one side of putative beta-strands predicted the positions of 16 transmembrane segments, a structure common to known porins. Second, outer loops typical of porins were assigned using an artificial neural network trained to predict the topology of bacterial outer-membrane proteins with a predominance of beta-strandsPrediction of the membrane-spanning beta-strands of the major outer membrane protein of ChlamydiaTopology prediction of Brucella abortus Omp2b and Omp2a porins after critical assessment of transmembrane beta strands prediction by several secondary structure prediction methodsFour porins of known structure were selected as test-cases and their secondary structure delineated. The specificity and sensitivity of 11 methods were separately evaluated. Our critical assessment shows that some secondary structure prediction methods (PHD, Dsc, Sopma) originally designed to predict globular protein structure are useful on porin topology prediction. The overall best prediction is obtained by combining these three \\\"generalist\\\" methods with a transmembrane beta strand prediction techniqueA knowledge-based potential highlights unique features of membrane \\u03b1-helical and \\u03b2-barrel protein insertion and foldingIn previous work, a depth-dependent insertion potential, E(z) , was derived for \\u03b1-helical inner membrane proteins. We have generated an equivalent potential for TM \\u03b2-barrel proteins. The similarities and differences between these two potentials provide insight into unique aspects of the folding and insertion of \\u03b2-barrel membrane proteins. This potential can predict orientation within the membrane and identify functional residues involved in intermolecular interactions.Improved identification of outer membrane beta barrel proteins using primary sequence, predicted secondary structure, and evolutionary informationWe present an accurate sequence-based predictor of OMBBs, called OMBBpred, which utilizes a Support Vector Machine classifier and a custom-designed set of 34 novel numerical descriptors derived from predicted secondary structures, hydrophobicity, and evolutionary information.Predicting weakly stable regions, oligomerization state, and protein-protein interfaces in transmembrane domains of outer membrane proteins.We have also discovered that out-clamps, in-plugs, and oligomerization are 3 general mechanisms for stabilizing weakly stable TM regions. In addition, we have found that extended and contiguous weakly stable regions often signal the existence of an oligomer and that strands located in the interfaces of protein-protein interactions are considerably less stable. Based on these observations, we can predict oligomerization states and can identify the interfaces of protein-protein interactions for beta-barrel membrane proteins by using either structure or sequence informationRecently, we developed two top-performing methods based on machine-learning approaches to tackle both the detection of TMBBs in sets of proteins and the prediction of their topology.We introduce a graph-theoretic model for predicting the supersecondary structure of transmembrane \\u03b2-barrel proteins--a particular class of proteins that performs diverse important functions but it is difficult to determine their structure with experimental methods.Several computational methods have been developed to discriminate TMBBs from other types of proteins.To conquer this limitation, we developed a new computational model that can be used for predicting the ASA of both TM \\u03b1-helix and \\u03b2-barrel residues.Transmembrane beta barrel (TMB) proteins are found in the outer membranes of bacteria, mitochondria and chloroplasts.RESULTS: We present here, an HMM that combine a transmembrane barrel submodel and an SP submodel for both topology and SP predictions.Prediction of membrane spanning segments in beta-barrel outer membrane proteins (OMP) and their topology is an important problem in structural and functional genomics.In the simplest of these, the target \\\"model\\\" is another protein sequence, while the more elaborate methods group together the entire set of t ansmembrane helical or transmembrane beta-barrel proteins.Based on this study, we have developed a Web server, TBBPred, for predicting transmembrane beta-barrel regions in proteins (available at http://www.imtech.res.in/raghava/tbbpred).\"}, {\"question\": \"How is active neurotoxin of Clostridium botulinum detected?\", \"ideal_answer\": [\"Active neurotoxin of Clostridium botulinum can be detected by:\\nmouse lethality assay\\nby mass spectrometry\\nbioassay\\ndifferentiated cell models\\npeptide cleavage assay\\nFDC (functional dual coating) microtitre plate immuno-biochemical assay\\nendopeptidase activity monitored via UV-Visible spectroscopy\"], \"exact_answer\": [[\"mouse lethality assay\"], [\"by mass spectrometry\"], [\"bioassay\"], [\"differentiated cell models\"], [\"peptide cleavage assay\"], [\"FDC (functional dual coating) microtitre plate immuno-biochemical assay\"], [\"endopeptidase activity monitored via UV-Visible spectroscopy\"]], \"context\": \"The neurotoxin detection is based on the mouse lethality assay. which could be shown by bioassay to produce biologically active type B toxin in culture.Human induced pluripotent stem cells (hiPSC) hold great promise for providing various differentiated cell models for in vitro toxigenicity testing. A direct comparison of BoNT detection using primary rat spinal cord cells and hiPSC-derived neurons showed equal or increased sensitivityBoNT/A intoxication studies demonstrate that the hiPSC-derived neurons reproducibly and quantitatively detect biologically active BoNT/A with high sensitivity (EC(50) \\u223c0.3 U). these data suggest that neurons derived from hiPSCs provide an ideal and highly sensitive platform for BoNT potency determination, neutralizing antibody detection and for mechanistic studies.Detection of botulinum neurotoxin-A activity in food by peptide cleavage assay.his study reports the successful use of an enzymatic assay employing an internally quenched fluorogenic peptide as a fast, simple and inexpensive alternative to the mouse bioassay. BoNT endopeptidase assaysin vivo mouse bioassay has for years been the 'gold standard' assay used for this purposeCell-based assays are currently the only in vitro alternative that detect fully functional BoNTs in a single assay and have been utilized for years for research purposes.mouse bioassay (MBA) is traditionally used to confirm the presence of toxin in serum or foodA novel FDC (functional dual coating) microtitre plate immuno-biochemical assay,This report demonstrates a novel assay that utilises the endopeptidase activity of the toxin to detect Botulinum neurotoxin in a pharmaceutical sample. The cleaving of SNAP-25 is monitored via UV-Visible spectroscopy with a limit of detection of 373 fg/mL and has been further developed into a high throughput method using a microplate reader detecting down to 600 fg/mL of active toxin. \"}, {\"question\": \"In which nuclear compartments is heterochromatin located?\", \"ideal_answer\": [\"This compartment localizes into three main regions: the peripheral heterochromatin, perinucleolar heterochromatin, and pericentromeric heterochromatin. Silencing appears to be associated with histone H3 lysine 9 trimethylation (H3K9me3), DNA methylation and the localization of the silenced gene to a specific nuclear compartment enriched in these modification\"], \"exact_answer\": [[\"peripheral heterochromatin\"], [\"perinucleolar heterochromatin\"], [\"pericentromeric heterochromatin\"]], \"context\": \"This compartment localizes into three main regions: the peripheral heterochromatin, perinucleolar heterochromatin, and pericentromeric heterochromatinBy separating cells according to transgene expression we show here that silencing appears to be associated with histone H3 lysine 9 trimethylation (H3K9me3), DNA methylation and the localization of the silenced gene to a specific nuclear compartment enriched in these modificationsThe cell nucleus is divided into chromosome territories and the extrachromosomal domain. The latter includes several structural and functional compartments involved in RNA processing and transport.We also describe a new apocentric nuclear compartment with a unique set of histone modifications that occurs as a zone of chromatin surrounding centromeric heterochromatin in differentiated lymphocytes. Little is known about the mechanisms of gene targeting within the nucleus and its effect on gene expression, but most studies have concluded that genes located near the nuclear periphery are silenced by heterochromatinAfter TSA and NaBt treatment, the HP1 proteins were repositioned more internally in the nucleus, being closely associated with interchromatin compartments, while centromeric heterochromatin was relocated closer to the nuclear peripheryTaking into account the different heterochromatic and euchromatic compartments of the genome, our data suggest that the relative abundance of Ty3/gypsy LTRrs along each chromosome arm is determined mainly by the different proportions of heterochromatin, particularly pericentric heterochromatin, relative to total arm length\"}, {\"question\": \"Which enzymes are involved in global genome nucleotide excision repair (GG-NER) in bacteria?\", \"ideal_answer\": [\"Nucleotide excision repair (NER) is universally used to recognize and remove many types of DNA damage. In eubacteria, the NER system typically consists of UvrA, UvrB, UvrC, the UvrD helicase, DNA polymerase I, and ligase.  Damage recognition during bacterial NER depends upon UvrA, which binds to the damage and loads UvrB onto the DNA. Subsequently, UvrA, UvrB and UvrC form the excinuclease protein UvrABC endonuclease, a multi-enzymatic complex which carries out repair of damaged DNA in sequential manner. In some cases, Cho may be the effective nuclease for NER, rather than UvrC. UvrC nuclease and the short oligonucleotide that contains the DNA lesion are removed from the post-incision complex by UvrD, a superfamily 1A helicase. In gram-positive organisms, PcrA helicase can also displace UvrC and the excised oligonucleotide from a post-incision NER complex.\"], \"exact_answer\": [[\"UvrA\"], [\"UvrB\"], [\"UvrC nuclease\"], [\"UvrD 1A helicase\"], [\"DNA polymerase I\"], [\"Ligase\"], [\"Cho nuclease (instead of UvrC)\"], [\"PcrA helicase (instead of Uvrd, in gram-positive bacteria)\"]], \"context\": \"we evaluated the in vivo role of NER in the repair of DNA adducts generated by psoralens (mono- or bi-functional) and UV-A light (PUVA) in E. coli. Cultures of wild-type E. coli K12 and mutants for uvrA, uvrB, uvrC or uvrAC genes were treated with PUVA and cell survival was determined.Nucleotide excision repair (NER) is universally used to recognize and remove many types of DNA damage. In eubacteria, the NER system typically consists of UvrA, UvrB, UvrC, the UvrD helicase, DNA polymerase I, and ligase.During nucleotide excision repair (NER) in bacteria the UvrC nuclease and the short oligonucleotide that contains the DNA lesion are removed from the post-incision complex by UvrD, a superfamily 1A helicase.UvrD interacts with UvrB, a component of the post-incision complex.PcrA helicase from Bacillus stearothermophilus can also displace UvrC and the excised oligonucleotide from a post-incision NER complex, which supports the idea that PcrA performs a UvrD-like function during NER in gram-positive organisms.Cho rather than UvrC seems to be an effective nuclease for the NER of DPCs.Nucleotide excision repair consists of excinuclease protein UvrABC endonuclease, multi-enzymatic complex which carries out repair of damaged DNA in sequential manner.Escherichia coli K12 mutant strains deficient in nucleotide excision repair (NER) were submitted to increasing concentrations of cisplatin, and the results revealed that uvrA and uvrB mutants are sensitive to this agent, while uvrC and cho mutants remain as the wild type strain.we used a genetic approach to investigate the roles of nucleotide excision repair (NER) pathway components in H. pylori mutation and recombination. RESULTS: Inactivation of any of the four uvr genes strongly increased the susceptibility of H. pylori to DNA damage by ultraviolet light. Inactivation of uvrA and uvrB significantly decreased mutation frequencies whereas only the uvrA deficient mutant exhibited a significant decrease of the recombination frequency after natural transformation. A uvrC mutant did not show significant changes in mutation or recombination rates; however, inactivation of uvrC promoted the incorporation of significantly longer fragments of donor DNA (2.2-fold increase) into the recipient chromosome. A deletion of uvrD induced a hyper-recombinational phenotype.Transcription-coupled DNA repair (TCR) is a subpathway of nucleotide excision repair (NER) that is triggered when RNA polymerase is stalled by DNA damage. Lesions targeted by TCR are repaired more quickly than lesions repaired by the transcription-independent \\\"global\\\" NER pathway, but the mechanism underlying this rate enhancement is not understood. Damage recognition during bacterial NER depends upon UvrA, which binds to the damage and loads UvrB onto the DNA. Bacterial TCR additionally requires the Mfd protein, a DNA translocase that removes the stalled transcription complexes.\"}, {\"question\": \"In which genomic regions are Alu enriched?\", \"ideal_answer\": [\"There are regions such as the four homeobox gene clusters, which are nearly devoid of these repeats that contrast with repeat dense regions in other transcriptionally active regions of the genome. Alu elements are more clustered in genes which are involved in metabolism, transport, and signaling processes. In contrast, they are significantly fewer in genes coding for information pathway components as well as structural proteins.\\nThis bias in Alu distribution is independent of the effect of Alu density of the flanking genomic region and is also not affected by the GC content of the gene and its upstream and downstream regions.\", \"There are regions such as the four homeobox gene clusters, which are nearly devoid of these repeats that contrast with repeat dense regions in other transcriptionally active regions of the genome.\"], \"exact_answer\": [[\"genes which are involved in metabolism, transport, and signaling processes\"], [\"gene conversion-prone regions\"]], \"context\": \"There are regions such as the four homeobox gene clusters, which are nearly devoid of these repeats that contrast with repeat dense regions in other transcriptionally active regions of the genome.These elements are more clustered in genes which are involved in metabolism, transport, and signaling processes. In contrast, they are significantly fewer in genes coding for information pathway components as well as structural proteins.This bias in Alu distribution is independent of the effect of Alu density of the flanking genomic region and is also not affected by the GC content of the gene and its upstream and downstream regions.The relative proportions of Alu subfamilies (Alu J, Alu S, and Alu Y) are not significantly different in genes with high Alu density belonging to the functional categories of transport, metabolism, and signaling.We suggest that Alu elements might be involved in regulatory mechanisms and are therefore differentially selected in primate genomes.Sequence analysis demonstrated these \\\"young\\\" Alu insertions represented gene conversion events of pre-existing ancient Alu elements or independent parallel insertions of older Alu elements in the same genomic region. This study suggests that the majority of Alu insertions in primate genomes are the products of unique evolutionary events.Our screening of the Alu insertion loci also resulted in the recovery of several \\\"young\\\" Alu elements that resided at orthologous positions in nonhuman primate genomes.\"}, {\"question\": \"Dracorhodin perchlorate was tested for treatment of which cancers?\", \"ideal_answer\": [\"Dracorhodin perchlorate induce apoptosis in prostate cancer, gastric tumor, melanoma and premyelocytic leukemia.\"], \"exact_answer\": [[\"prostate cancer\"], [\"gastric tumor\"], [\"melanoma\"], [\"premyelocytic leukemia\"]], \"context\": \"Dracorhodin perchlorate induced human breast cancer MCF-7 apoptosis through mitochondrial pathways.It was reported that DP could induce apoptosis in human prostate cancer, human gastric tumor cells and human melanoma, but the cytotoxic effect of DP on human breast cancer was not investigated.CONCLUSION: Therefore DP was a candidate for anti-breast cancer, DP induced apoptosis of MCF-7 through mitochondrial pathway.Dracorhodin perchlorate has been recently shown to induce apoptotic cell death in cancer cells. However, the molecular mechanisms underlying these effects are unknown in human gastric tumor cells.Dracorhodin perchlorate suppresses proliferation and induces apoptosis in human prostate cancer cell line PC-3.The growth inhibition and pro-apoptosis effects of dracorhodin perchlorate on human prostate cancer PC-3 cell line were examined. Dracorhodin perchlorate, an anthocyanin red pigment, induces human premyelocytic leukemia HL-60 cell death through apoptotic pathway. Dracorhodin perchlorate, an anthocyanin red pigment, induces human melanoma A375-S2 cell death through the apoptotic pathway. Dracorhodin perchlorate inhibited proliferation of several tumor cell lines. The growth inhibition and pro-apoptosis effects of dracorhodin perchlorate on human prostate cancer PC-3 cell line were examined.It was reported that DP could induce apoptosis in human prostate cancer, human gastric tumor cells and human melanoma, but the cytotoxic effect of DP on human breast cancer was not investigated.  Dracorhodin perchlorate (DP) was a synthetic analogue of the antimicrobial anthocyanin red pigment dracorhodin. It was reported that DP could induce apoptosis in human prostate cancer, human gastric tumor cells and human melanoma, but the cytotoxic effect of DP on human breast cancer was not investigated. The growth inhibition and pro-apoptosis effects of dracorhodin perchlorate on human prostate cancer PC-3 cell line were examined. After administration of 10-80 \\u03bcmol/L dracorhodin perchlorate for 12-48 h, cell viability of PC-3 cells was measured by MTT colorimetry.It was reported that DP could induce apoptosis in human prostate cancer, human gastric tumor cells and human melanoma, but the cytotoxic effect of DP on human breast cancer was not investigated. This study would investigate whether DP was a candidate chemical of anti-human breast cancer.Dracorhodin perchlorate (DP) was a synthetic analogue of the antimicrobial anthocyanin red pigment dracorhodin. It was reported that DP could induce apoptosis in human prostate cancer, human gastric tumor cells and human melanoma, but the cytotoxic effect of DP on human breast cancer was not investigated.The growth inhibition and pro-apoptosis effects of dracorhodin perchlorate on human prostate cancer PC-3 cell line were examined. After administration of 10-80 \\u03bcmol/L dracorhodin perchlorate for 12-48 h, cell viability of PC-3 cells was measured by MTT colorimetry.\"}, {\"question\": \"List mutations that are implicated in the Gray Platelet Syndrome.\", \"ideal_answer\": [\"GFI1B and NBEAL2 mutations are implicated in the Gray Platelet Syndrome.\"], \"exact_answer\": [[\"GFI1B\"], [\"NBEAL2\"]], \"context\": \"A dominant-negative GFI1B mutation in the gray platelet syndromeWe detected a nonsense mutation in the gene encoding the transcription factor GFI1B (growth factor independent 1B) that causes autosomal dominant gray platelet syndrome.The genetic defect responsible for gray platelet syndrome was recently identified in biallelic mutations in the NBEAL2 gene. Mutations in NBEAL2 underlie gray platelet syndrome (GPS), a rare inherited bleeding disorder characterized by a lack of \\u03b1-granules within blood platelets and progressive bone marrow fibrosis.NBEAL2 is mutated in gray platelet syndrome and is required for biogenesis of platelet \\u03b1-granules.Mutations in NBEAL2, encoding a BEACH protein, cause gray platelet syndrome.Mutations in NBEAL2 underlie gray platelet syndrome (GPS), a rare inherited bleeding disorder characterized by a lack of \\u03b1-granules within blood platelets and progressive bone marrow fibrosis.The genetic defect responsible for gray platelet syndrome was recently identified in biallelic mutations in the NBEAL2 gene.Our studies show that GFI1B, in addition to being causally related to the gray platelet syndrome, is key to megakaryocyte and platelet development.A dominant-negative GFI1B mutation in the gray platelet syndrome.We detected a nonsense mutation in the gene encoding the transcription factor GFI1B (growth factor independent 1B) that causes autosomal dominant gray platelet syndrome.GPS has been linked to loss of function mutations in NEABL2 (neurobeachin-like 2), and we describe here a murine GPS model, the Nbeal2(-/-) mouse. Here we show that mutations in NBEAL2 (neurobeachin-like 2), which encodes a BEACH/ARM/WD40 domain protein, cause GPS and that megakaryocytes and platelets from individuals with GPS express a unique combination of NBEAL2 transcripts.  The gray platelet syndrome is a rare inherited bleeding disorder characterized by macrothrombocytopenia and deficiency of alpha (\\u03b1)-granules in platelets. The genetic defect responsible for gray platelet syndrome was recently identified in biallelic mutations in the NBEAL2 gene.The gray platelet syndrome is a rare inherited bleeding disorder characterized by macrothrombocytopenia and deficiency of alpha (\\u03b1)-granules in platelets. The genetic defect responsible for gray platelet syndrome was recently identified in biallelic mutations in the NBEAL2 gene.The gray platelet syndrome is a hereditary, usually autosomal recessive bleeding disorder caused by a deficiency of alpha granules in platelets. We detected a nonsense mutation in the gene encoding the transcription factor GFI1B (growth factor independent 1B) that causes autosomal dominant gray platelet syndrome.Mutations in NBEAL2 underlie gray platelet syndrome (GPS), The gray platelet syndrome is a hereditary, usually autosomal recessive bleeding disorder caused by a deficiency of alpha granules in platelets. We detected a nonsense mutation in the gene encoding the transcription factor GFI1B (growth factor independent 1B) that causes autosomal dominant gray platelet syndrome.\"}, {\"question\": \"List interaction partners for the protein GATA1.\", \"ideal_answer\": [\"GATA-1 interact with factor Gfi-1b, the repressive MeCP1 complex, the chromatin remodeling ACF/WCRF complex, FOG-1, TAL-1, Ldb-1 and LMO2-C.\", \"GATA1 interacts with the essential hematopoietic factor Gfi-1b, the repressive MeCP1 complex, the chromatin remodeling ACF/WCRF complex, in addition to the known GATA-1/FOG-1 and GATA-1/TAL-1 complexes. Ldb1 is a known partners of GATA1. LMO2-C can bind endogenous GATA1. Novel transcription factor binding partners for PIAS3 include GATA1. Combinatorial interactions occurre between RUNX1 and its coregulator GATA1. Ski interacts with GATA1. GATA-1 interacts with multiple proteins including FOG-1, EKLF, SP1, CBP/p300 and PU.1.\"], \"exact_answer\": [[\"Gfi-1b\"], [\"MeCP1\", \"MeCP1 complex\"], [\"ACF/WCRF complex\"], [\"FOG-1\"], [\"TAL-1\"], [\"Ldb1\", \"Ldb-1\"], [\"LMO2-C\"], [\"PIAS3\"], [\"RUNX1\"], [\"Ski\"], [\"EKLF\"], [\"SP1\"], [\"CBP/p300\"], [\"PU.1\"]], \"context\": \"Our work describes, for the first time, distinct GATA-1 interactions with the essential hematopoietic factor Gfi-1b, the repressive MeCP1 complex, and the chromatin remodeling ACF/WCRF complex, in addition to the known GATA-1/FOG-1 and GATA-1/TAL-1 complexes. We also provide evidence that distinct GATA-1 complexes are associated with specific GATA-1 functions in erythroid differentiation, for example, GATA-1/Gfi-1b with the suppression of cell proliferation and GATA-1/FOG-1/MeCP1 with the repression of other hematopoietic transcription programs. We next applied the biotinylation tag to Ldb-1, a known partner of GATA-1, and characterized a number of novel interaction partners that are essential in erythroid development, in particular, Eto-2, Lmo4, and CdK9. Endogenous GATA1 and LDB1 proteins were confirmed to bind to LMO2-C by MBP pull down analysis. LMO2-C can bind endogenous GATA1 and LDB1 protein in K562 cells and down regulates the expression of GPA.We identify novel transcription factor binding partners for PIAS3 including ETS, EGR1, NR1I2, and GATA1.At other differentiation stages, additional combinatorial interactions occurred between RUNX1 and its coregulators, GATA1 and ETS. Ldb1, a ubiquitously expressed LIM domain binding protein, is essential in a number of tissues during development. It interacts with Gata1, Tal1, E2A and Lmo2 to form a transcription factor complex regulating late erythroid genes. Using a biotinylation tagging/proteomics approach in erythroid cells, we describe distinct GATA-1 interactions with the essential hematopoietic factor Gfi-1b, the repressive MeCP1 complex and the chromatin remodeling ACF/WCRF complex, in addition to the known GATA-1/FOG-1 and GATA-1/TAL-1 complexes. Importantly, we show that FOG-1 mediates GATA-1 interactions with the MeCP1 complex, thus providing an explanation for the overlapping functions of these two factors in erythropoiesis.Here we show that Ski interacts with GATA1, a transcription factor essential in erythropoiesis. Using a Ski mutant deficient in GATA1 binding, we show that this Ski-GATA1 interaction is critical for Ski's ability to repress GATA1-mediated transcription and block erythroid differentiation. The zinc finger transcription factor GATA-1, a central mediator of erythroid gene expression, interacts with multiple proteins including FOG-1, EKLF, SP1, CBP/p300 and PU.1. tudies involving point mutants of GATA-1 have shown that a direct physical interaction between GATA-1 and FOG-1 is essential for normal human erythroid and megakaryocyte maturation in vivo. In addition, evidence has emerged that physical interaction between GATA-1 and the myeloid/lymphoid specific factor PU.1, an oncogene implicated in murine erythroleukemia, acts to functionally cross-antagonize one another. \"}, {\"question\": \"What histone variants play a role in the DNA damage reponse?\", \"ideal_answer\": [\"Mostly H2A.X, but H2A.Z and H1R have also been associated to DNA damage\"], \"exact_answer\": [[\"H2A.X\"], [\"H2A.Z\"], [\"H1R\"]], \"context\": \"H2A.Z is an evolutionarily ancient and highly conserved H2A variant that regulates processes ranging from gene expression to the DNA damage responseactivation of DNA damage checkpoints (ataxia telangiectasia mutated (ATM) and H2A histone family, member X (H2AFX (H2AFX)))the specialization imparted by histone H2A (H2A.X and H2A.Z) variants to the nucleosome core particle constitutes the earliest response to DNA damage in the cellhistone variants (H2AX and H2A.Z), histone post-translational modifications (acetylation, phosphorylation, methylation and ubiquitination) and chromatin-remodeling complexes (INO80, SWR1, SWI/SNF, RSC and NuRD) are important and direct players in the DNA double-strand break (DSB)H2A.Z incorporation in the coding regions of genes encoding proteins involved in the regulation of meiosis and genotoxic stress responses phosphorylation of H2AX, a DNA damage signalDSB) are generally considered the most critical lesion induced by ionizing radiation (IRphosphorylated forms of histone H2AX H2AX has a complete SQ(E/D)varphi (where varphi denotes a hydrophobic residue) known to be phosphorylated in response to DNA damageH2A.X, plays a very important role in the cellular response to DNA damageH2A.X is an H2A variant present in multicellular organisms that is specifically phosphorylated on the serine in the C-terminal consensus sequence, canonically \\\"SQEY,\\\" in response to DNA damage.H2AFX) is important in maintaining chromatin structure and genetic stabilitySaccharomyces cerevisiae, the linker histone HHO1 is involved in DNA repairH1 variant, H1R is involved in DNA damage responsehigh levels of histone H2AX phosphorylation and an increased incorporation of the Htz1p histone variant into chromatin surrounding the DSBTRF2 upregulation was more dramatic in drug-resistant cells and occurred before the expression of ATM, gammaH2AX and p53DNA double-strand breaks induced by anticancer drugs or irradiation increase TRF2 expressionHistone variant H2AX and core histone H2A are phosphorylated in mammals and budding yeast, respectively. We demonstrate the DSB-induced phosphorylation of histone variant H2Av in Drosophila melanogasterformation of a double-strand break (DSB) in chromosomal DNA is highly conservedH2A variants, H2AX and H2AZ. H2AX is phosphorylated on a serine four residues from the carboxyl terminus in response to the introduction of DNA double-strand breaksIn response to IR, ZNF668 knockdown reduces Tip60-H2AX interaction and impairs IR-induced histone H2AX hyperacetylation, thus impairing chromatin relaxation.RPA foci formation were reduced in parallel with increasing levels of H2AX phosphorylationH2AX phosphorylation on a C-terminal serine residue to form \\\"\\u03b3-H2AX\\\" is a critical early event in the chromatin response to chromosomal DNA double strand breaks in eukaryotesThe phosphorylation of the histone variant H2AX (gammaH2AX) is one of the best examples of histone PTMs in response to DNA damage inductionOne of the most well-characterized modifications is caused by the formation of DNA double strand breaks (DSBs), resulting in phosphorylation of histone H2AX (the so-called gamma-H2AX) on the chromatin surrounding the DNA lesionH2AX appears to be mainly associated with maintaining the genome integrity by participating in the repair of the double-stranded DNA breaks exogenously introduced by environmental damage (ionizing radiation, chemicals) or in the process of homologous recombination during meiosisThe sustained localization of BMI1 to damage sites is dependent on intact ATM and ATR and requires H2AX phosphorylatiounique functional interactions among various chromatin modifiers, suggesting new regulatory pathways, such as a heterochromatin-specific modulation of DNA damage response involving H2A.X and WICH, both enriched in silent domains\"}, {\"question\": \"List tele monitoring applications of miniaturised sensors\", \"ideal_answer\": [\"Home-polysomnography (HPSG)\\nBody weight\\nBlood pressure control\\nHeart failure control\\nVital signs - disaster relief, dangerous outdoor sports and adventure monitoring, and antiterrorism activities.\\nTelemetric fetal home monitoring system for recording the trans-abdominal fetal heart signal and the uterine contractions\\nVital signs - electrocardiograms (ECGs), temperature (T), and oxygen saturation (SaO2) , breath rate\\nStep-counting for tele-rehabilitation\\nDetection of falls in elderly\"], \"exact_answer\": [[\"Sleep monitoring\"], [\"Body weight\"], [\"Blood pressure control\"], [\"Heart failure control\"], [\"vital signs\"], [\"fetal heart signal\"], [\"uterine contractions\"], [\"Electrocardiograms\", \"ECG\"], [\"Temperature\"], [\"Oxygen saturation\", \"SaO2\"], [\"breath rate\"], [\"Step-counting\"], [\"Falls detector\"]], \"context\": \"Home-polysomnography (HPSG) has been proposed as a cost-effective alternative for obstructive sleep apnea (OSA) diagnosis. We assessed, in a feasibility study, whether telematic transmission using the Dream\\u00ae and Sleepbox\\u00ae technologies was associated with low HPSG failure rate In this study, we monitored remotely self-measured body weight and blood pressure, in parallel with the data automatically transmitted by implantable cardioverter-defibrillators.Body weight, patient activity, and the difference between MHR and RHR are mutually correlated and may reasonably contribute to an algorithm for predicting heart failure deterioration. Blood pressure appears to offer no additional value. As both genesis and symptoms of heart failure exacerbation are non-uniform and complex, the telemonitoring concepts for heart failure patients should employ continuous monitoring of multiple diagnostic parameters, rather than rely on a single parameter.System and can collect users' vital signs with a set of wireless sensorsthis terminal is very helpful in special circumstances such as disaster relief, dangerous outdoor sports and adventure monitoring, and antiterrorism activities.A new, phonocardiography-based telemetric fetal home monitoring system.The input element of the system was the home monitor with two sensors for recording the trans-abdominal fetal heart signal and the uterine contractions. TTelemonitoring is currently being used in community-based healthcare, at the scene of medical emergencies, by ambulance services and in hospitals. Mobile telemonitoring for achieving tighter targets of blood pressure control in patients with complicated diabetes: Patients in the intervention arm transmitted weekly blood pressure readings wirelessly, using adapted sensors via mobile phones to a central server. In patients with diabetes, mobile telemonitoring has potential for delivering intensified care to improve blood pressure control, and its use may be associated with reduced exposure to hyperglycemia.A new codivilla-spring prosthesis with sensors for telemonitoring / telerehabilitation has been designed and constructed. It provides step countingThis article is a survey of systems, algorithms and sensors, for the automatic early detection of the fall of elderly persons. We describe a vital sign telemonitor (VST) that acquires, records, displays, and provides readings such as: electrocardiograms (ECGs), temperature (T), and oxygen saturation (SaO2) over the Internet to any site. The design of this system consisted of three parts: sensors, analog signal processing circuits, and a user-friendly graphical user interface (GUI). The paper describes the development of biomedical clothing for ambulatory telemonitoring of human vital parameters. VTAM (Vetement de Tele-Assistance Medicale) presents a T-shirt made from textile with woven wires and incorporating four smooth dry ECG electrodes, a breath rate sensor, a shock/fall detector and two temperature sensors. \"}, {\"question\": \"List the diseases for which there are point-of-care breath tests\", \"ideal_answer\": [\"Point of care breath tests are available for lung cancer, pulmonary embolism, respiratory distress syndrome, methanol intoxication, kidney diseases, liver diseases, Helicobacter pylori infection, asthma, sepsis, heart failure, diabetes and tuberculosis.\"], \"exact_answer\": [[\"lung cancer\"], [\"pulmonary embolism\"], [\"respiratory distress syndrome\"], [\"methanol intoxication\"], [\"kidney diseases\"], [\"liver diseases\"], [\"Helicobacter pylori infection\"], [\"asthma\"], [\"sepsis\"], [\"heart failure\"], [\"diabetes\"], [\"tuberculosis\"]], \"context\": \"Additionally, we try to clarify controversial issues concerning possible experimental malpractice in the field, and propose ways to translate the basic science results as well as the mechanistic understanding to tools (sensors) that could serve as point-of-care diagnostics of cancer.Point of care monitoring of hemodialysis patients with a breath ammonia measurement device based on printed polyaniline nanoparticle sensors. Excellent intraindividual correlations were demonstrated between breath ammonia and BUN (0.86 to 0.96), which demonstrates the possibility of using low cost point of care breath ammonia systems as a noninvasive means of monitoring kidney dysfunction and treatment.Recently, fractional exhaled nitric oxide (FeNO) has emerged as an important biomarker for the assessment and management of asthma.Here we discuss the potential of adapting a technology from the electronics industry, surface acoustic wave (SAW) sensors, for diagnosis of multiple markers of sepsis in real time, using non-invasive assays of exhaled breath condensate. Together these data indicate that FeNO testing has an important role in the assessment and management of adult asthma. Measurement of acetone in human breath samples has been previously shown to provide significant non-invasive diagnostic insight into the control of a patient's diabetic condition.  Urea breath testing may provide a useful diagnostic and biomarker assay for tuberculosis and treatment response.These animal model results suggest that exhaled breath can be used as a point-of-care tool for the diagnosis and monitoring of sepsis.Point-of-care breath test for biomarkers of active pulmonary tuberculosis.RATIONALE: Volatile organic compounds (VOCs) in breath provide biomarkers of tuberculosis (TB) because Mycobacterium tuberculosis manufactures VOC metabolites that are detectable in the breath of infected patients. CONCLUSIONS: A six-minute point-of-care breath test for volatile biomarkers accurately identified subjects with active pulmonary TB.Selected ion flow tube-mass spectrometry (SIFT-MS) can measure volatile compounds in breath on-line in real time and has the potential to provide accurate breath tests for a number of inflammatory, infectious and metabolic diseases, including diabetes.Breath analysis by SIFT-MS offers a rapid, reproducible and easily performed measurement of acetone concentration in ambulatory patients with type 2 diabetes. IGR is a new approach for non-invasive cardiac output measurement in mechanically ventilated individuals, but requires further investigation for clinical use.The liver metabolic breath test relies on measuring exhaled (13) C tagged methacetin, which is metabolized only by the liver. Measuring this liver-specific substrate by means of molecular correlation spectroscopy is a rapid, non-invasive method for assessing liver function at the point-of-care. Our recent findings regarding the ability of point-of-care (13) C MBT to assess the hepatic functional reserve in patients with acute and chronic liver disease are reviewed along with suggested treatment algorithms for common liver disorders.13[C]-urea breath test as a novel point-of-care biomarker for tuberculosis treatment and diagnosis.CONCLUSIONS/SIGNIFICANCE: Urea breath testing may provide a useful diagnostic and biomarker assay for tuberculosis and for treatment response.CONCLUSIONS: The non-invasive real-time BreathID GBT reliably assesses changes in liver glucose metabolism, and the degree of insulin resistance. DISCUSSION: The LiMAx test can validly determine liver function capacity and is feasible in every clinical situation.A P(ET,CO(2)) of >or=36 mmHg may reliably exclude PE. Accuracy is augmented by combination with Wells score. P( ET,CO(2)) should be prospectively compared to D-dimer in accuracy and simplicity to exclude PE.Point-of-care continuous (13)C-methacetin breath test improves decision making in acute liver disease: results of a pilot clinical trial.CONCLUSION: The (13)C-MBT provides a rapid, non-invasive assessment of liver function in acute severe liver disease of diverse etiologies. CONCLUSION: The determination of CO by IGR and CWD revealed a good agreement and reproducibility with a low rate of impossible measurements, suggesting that IGR and CWD can be used interchangeably in the clinical setting.BACKGROUND: Exhaled NO (FE(NO)) is a useful biomarker for the monitoring of asthma control and response to therapy. A high percentage of patients with different severities of asthma and regardless of their treatment with ICS and current smoking habit (current and/or ex-smokers) had highly elevated FE(NO) values, suggesting that their current therapy was possibly insufficient to control the underlying degree of airway inflammation and asthma symptoms.H. pylori-positive status was defined by positivity on at least 2 tests: a commercial H. pylori stool antigen enzyme immunoassay, an immunoglobulin G antibody enzyme immunoassay, and the C-urea breath testEnd-tidal carbon dioxide measurements in children with acute asthma.CONCLUSIONS: Noninvasive bedside measurement of EtCO2 values among children with acute asthma is feasible. Review article: the assessment of liver function using breath tests.The recent development of a real-time, point-of-care liver function breath test has made it straightforward to assess the metabolic function of the liver.RESULTS: The (13)C-methacetin breath test enables accurate follow-up of patients with acute or chronic liver damage, where overall hepatic function is significantly suppressed by known causes of liver disorders, including acute, sub-acute or chronic conditions. CONCLUSIONS: Breath testing that provides continuous quantification of methacetin metabolism may be a sensitive tool for the diagnosis and follow-up of patients with liver disorders. A multi-center study in a multi-ethnic population compared the RAPIRUN urine antibody test with the (13)C-urea breath test (C-UBT) and a traditional serologic test, the high-molecular-weight cell-associated protein enzyme immunoassay (HM-CAP EIA). CONCLUSION: EGA provided non-invasive, accurate, continuous, high-resolution COHb% measurements. Applying EGA to carboxyhaemoglobin dilution, we achieved RCV measurements with accuracy equivalent to that of CO-haemoximetry, with one-fifth of the COB infusion volume. These results suggest that the IR device, which is small, light-weight, and rugged may enable the foreign gas uptake method to be used in clinical, field, and point-of-care settings for Q (T) measurement.UNLABELLED: The 13C-urea breath test provides non-invasive testing for Helicobacter pylori infection with the possibility of analysis at the point of care. CONCLUSION: Fdlate, a variable of VCap, had a statistically better diagnostic performance in suspected PE than the PaCO(2)-EtCO(2) gradient.  In conclusion, the investigated rebreathing system allows for a non-invasive determination of PBF at the bedside. Non-invasive measurement of pulmonary blood flow during prone positioning in patients with early acute respiratory distress syndrome.The 13C-urea breath test provides accurate, noninvasive diagnosis of active Helicobacter pylori infection and can document posttherapy cure. A portable Fourier transform infrared (FT-IR) multicomponent point-of-care analyzer was tested for the diagnosis of methanol intoxications. Breath analysis with FT-IR was fast and easy, and no sample preparation was needed.CONCLUSIONS: The Fd(late) is a valuable tool for bedside screening of PE in surgical patients.\"}, {\"question\": \"What are the indications for alteplase?\", \"ideal_answer\": [\"Intravenous alteplase (recombinant tissue plasminogen activator) is the only approved thrombolytic agent at present indicated for acute ischaemic stoke.\\nFood and Drug Administration approval of alteplase for central venous catheter (CVC) occlusions. \\nAlteplase is now firmly established as a treatment of choice in the management of acute myocardial infarction. The efficacy of intravenous alteplase in the treatment of pulmonary thromboembolism has also been established and appears to be similar to that of streptokinase and urokinase in this indication and in arterial thrombotic occlusion. However, its use in this latter indication and in other vascular disorders has not been as extensively documented. \\nPreliminary data suggest efficacy of alteplase of deep vein thrombosis and arterial thrombotic occlusion.\"], \"exact_answer\": [[\"ischemic stroke\"], [\"central venous catheter (CVC) occlusions\"], [\"thrombolysis\"], [\"myocardial infarction\"], [\"acute pulmonary embolism\"], [\"deep vein thrombosis\"], [\"limb artery occlusions\"], [\"Ischaemic Stroke\"], [\"Acute Myocardial Infarction\", \"AMI\", \"MI\"], [\"Pulmonary Thromboembolism\"], [\"Vein Thrombosis\"], [\"Arterial Thrombotic Occlusion\"]], \"context\": \"In Japan, intravenous alteplase, a recombinant tissue-type plasminogen activator (rt-PA), was approved for an indication of ischemic stroke in 2005 on the basis of the results of a clinical trial with a unique dose of the drug (0.6 mg/kg). Alteplase appears to show efficacy for treatment of thrombus-related venous catheter occlusion in pediatric patients; however, data regarding its use in occluded dialysis catheters are limited.The trials used to support Food and Drug Administration approval of alteplase for central venous catheter (CVC) occlusions generally had low pediatric enrollment; however, additional small studies are available that support use of alteplase for this indication in children.To review the literature pertaining to the efficacy of alteplase for restoration of patency of occluded venous and dialysis catheters in pediatric patients. Intravenous thrombolysis with recombinant tissue Plasminogen Activator improves outcomes in patients treated early after stroke but at the risk of causing intracranial hemorrhage.Intravenous alteplase (recombinant tissue plasminogen activator) is the only approved thrombolytic agent at present indicated for acute ischaemic stoke.This paper presents a summary of the evidence review group report into the clinical effectiveness and cost-effectiveness of alteplase for the treatment of acute ischaemic stroke, in accordance with the licensed indication, based upon the evidence submission from the manufacturer to the National Institute for Health and Clinical Excellence (NICE) as part of the single technology appraisal (STA) process. The submitted clinical evidence included several randomised controlled trials indicating that, in highly selected patients, alteplase administered at a licensed dose within 3 hours of the onset of acute ischaemic stroke is associated with a statistically significant reduction in the risk of death or dependency at 3 months compared with placebo, despite a significantly increased risk of symptomatic intracranial haemorrhage within the first 7-10 days. Data from the National Institute of Neurological Disorders and Stroke (NINDS) trial suggest that the benefit of treatment is sustained at 6 and 12 months.The guidance issued by NICE in April 2007 as a result of the STA states that alteplase is recommended for the treatment of acute ischaemic stroke only when used by physicians trained and experienced in the management of acute stroke and in centres with the required facilities.Because of the risk of hemorrhage, especially in the brain, thrombolytic therapy with intravenous alteplase is restricted by guidelines, and only a small number of selected patients are being treated.Alteplase can be given to patients with cervical artery dissection, seizure at onset and evidence of acute ischemia on brain imaging, and after carefully weighing risk and benefit in pregnancy and during menstruation. Alteplase is the first thrombolytic drug to be approved in France for the treatment of ischaemic stroke within three hours of symptom onset.However, the Cochrane systematic review of thrombolysis for acute ischaemic stroke suggests that alteplase is the most promising treatment for acute ischaemic stroke.  To determine whether cryopreserved solutions of the thrombolytic agent alteplase could be used as a safe, effective, and economically reasonable alternative to urokinase in patients presenting with occluded central venous access devices (CVADs).Alteplase has been reported as an efficacious alternative to urokinase for treatment of occluded CVADs. Patients presenting with occluded CVADs received a sufficient volume of the thawed alteplase solution to fill the occluded catheter(s). Cryopreserved 1-mg/mL aliquots of alteplase are safe and effective in the clearance of occluded CVADs when stored at -20 degrees C for 30 days. The ability to cryopreserve alteplase aliquots makes it an economically reasonable alternative to urokinase in the setting of CVAD occlusion.Of the thrombolytic agents and therapeutic protocols in use, alteplase 100 mg/2 hours seems to be the best compromise between the risk of bleeding and efficacy in reducing pulmonary resistances by 30 to 40% and relatively early pulmonary revascularisation of 40-50%. To explore the best regimen of treatment for acute pulmonary embolism(APE), and to evaluate the efficacy and safety of 2-h infusion of recombinant tissue-type plasminogen activator, Alteplase (rt-PA) and urokinase(UK). A 2 h regimen of rt-PA or UK showed reliable efficacy and safety for treatment of APE. The indication of thrombolytic therapy can be extended.Tissue type plasminogen activator is available, through recombinant technology, for thrombolytic use as alteplase.Alteplase has been used successfully in evolving myocardial infarction (MI) to reopen occluded coronary arteries. Alteplase is most effective when given early in MI and is probably ineffective when given 12 h after the onset of symptoms. The effectiveness of alteplase in MI can be increased by front loading with a bolus of 15 mg, followed by an infusion of 50 mg over 30 min and 35 mg over 60 min.A reduced dose of alteplase to increase coronary artery patency prior to angioplasty may be useful in MI. An exciting new indication for the use of alteplase is in stroke, where it has become the first beneficial intervention. Alteplase is used to reopen occluded cerebral vessels but is associated with an increased risk of intracerebral haemorrhage. Alteplase is beneficial if given within 3 h of the onset of stroke but not after this time period.Clinical trials have not established a role for alteplase in the treatment of acute coronary syndromes or deep vein thrombosis. However, alteplase is useful in treating pulmonary thromboembolism and peripheral vascular disease.Our aim was to design and evaluate a new and easily administered recombinant tissue-type plasminogen activator (rt-PA) regimen for thrombolysis in acute myocardial infarction (AMI) based on established pharmacokinetic data that improve the reperfusion success rate. Rapid restoration of Thrombolysis in Myocardial Infarction (TIMI) grade 3 flow is a primary predictor of mortality after thrombolysis in AMI.The 60-min alteplase thrombolysis in AMI protocol achieved a TIMI grade 3 patency rate of 81.1% at 90 min with no indication of an increased bleeding hazard; it was associated with a 1.2% overall mortality rate.Alteplase is now firmly established as a treatment of choice in the management of acute myocardial infarction. The efficacy of intravenous alteplase in the treatment of pulmonary thromboembolism has also been established and appears to be similar to that of streptokinase and urokinase in this indication and in arterial thrombotic occlusion. However, its use in this latter indication and in other vascular disorders has not been as extensively documented. Although trials demonstrating the efficacy of intravenous alteplase in patients with deep vein thrombosis and intra-arterial alteplase in patients with arterial thrombotic occlusion exist, reliable data on the efficacy of the fibrinolytic in ischaemic stroke and intracranial haemorrhage are scarce.Further, preliminary data suggest efficacy in the therapy of deep vein thrombosis and arterial thrombotic occlusion, and alteplase has a proven place in the fibrinolytic treatment of pulmonary thromboembolism.The non-antigenic competitor for these two compounds for the indication of MI is alteplase (recombinant tissue plasminogen activator, rt-PA). In the framework of tailored thrombolytic therapy, alteplase or urokinase appear to be the drugs of choice in these patients.Patients with a recent (less than 10 days) proximal deep vein thrombosis of the leg or pelvis are candidates for thrombolysis as the major benefit over heparin seems to be the prevention of the postphlebitic limb, an aim which is still not proven in a satisfactory manner. Nonocclusive thrombi appear to lyse more readily than occlusive thrombi. For this indication the optimal dose regimens for the three thrombolytic drugs (streptokinase, urokinase, alteplase) are not established. To save 1 year of life, the costs of thrombolytic therapy using intravenous streptokinase, alteplase (recombinant tissue plasminogen activator; rt-PA) or anistreplase (anisoylated plasminogen streptokinase activator complex) under standard restricted indication criteria, vary from 1000 pounds British sterling to 1700 pounds British sterling in the UK, SEK3090 to 9660 in Scandinavia and $US35 000 to 800 000 in the US, depending on time delay in starting treatment after pain onset, size of infarct, thrombolytic agents used, study methodology, lists of clinical events considered in cost counting and the discount rate.Thus, the results of the ETTT trial show that the used low dosages of alteplase administered intravenously over 3-7 days in heparinized patients cannot be recommended as a treatment for patients with deep venous thrombosis of lower limbs and/or pelvis. Further studies are needed to define a more suitable dosage regimen of alteplase in this indication.Streptokinase, urokinase and alteplase have all been shown to accelerate the lysis of pulmonary emboli and to decrease pulmonary vascular obstruction and pulmonary hypertension. The local treatment of acute arterial occlusion in limb arteries results in rapid clearing of the artery in 67% of patients treated with streptokinase; the corresponding success rates for urokinase and alteplase are 81% and 88 to 94%, respectively. More recently, new drugs like alteplase, reteplase, lanoteplase and saruplase have been a breakthrough in the treatment of acute myocardial infarctionThe non-antigenic competitor for these two compounds for the indication of MI is alteplase (recombinant tissue plasminogen activator, rt-PA)The trials used to support Food and Drug Administration approval of alteplase for central venous catheter (CVC) occlusions generally had low pediatric enrollment; however, additional small studies are available that support use of alteplase for this indication in childrenSome acute ischemic strokes (before the 3rd hour) could be treated with alteplase if there is no absolute or relative contraindication for thrombolysis.Alteplase and reteplase are the most efficient thrombolytics despite a higher risk of cerebral bleeding. \"}, {\"question\": \"List scaffold proteins of the ERK signaling pathway.\", \"ideal_answer\": [\"Originally identified in yeast, scaffold proteins are now recognized to contribute to the specificity of MEK/ERK pathways in mammalian cells. These scaffolds include kinase suppressor of Ras (KSR), beta-arrestin, MEK partner-1 (MP-1), Sef and IQ motif-containing GTPase-activating protein 1(IQGAP1). Human disc-large homolog (hDlg) acts as a MEK2-specific scaffold protein for the ERK signaling pathway. Two scaffold proteins, caveolin-1 and IQGAP1, are required for phosphorylation of the actin associated pool of extracellular signal regulated kinase 1 and 2 (ERK1/2). Several 14-3-3 isotypes bind to protein kinase C (PKC)-zeta and facilitate coupling of PKC-zeta to Raf-1 an event that boosts the mitogen-activated protein kinase (ERK) pathway.\", \"kinase suppressor of Ras 1\\nMEK Partner 1\\nBeta-arrestin\\nIQ motif containing GTPase-activating protein 1\\nkinase suppressor of Ras 2\\nmitogen-activated protein kinase organizer 1\"], \"exact_answer\": [[\"Human disc-large homolog\", \"hDlg\"], [\"caveolin-1\"], [\"IQ motif-containing GTPase-activating protein 1\", \"IQGAP1\"], [\"kinase suppressor of Ras\", \"KSR\"], [\"MEK partner-1\", \"MP-1\"], [\"beta-arrestin\"], [\"Sef\"], [\"14-3-3\"], [\"mitogen-activated protein kinase organizer 1\"]], \"context\": \"Human disc-large homolog (hDlg), also known as synapse-associated protein 97, is a scaffold protein, a member of the membrane-associated guanylate kinase family, implicated in neuronal synapses and epithelial-epithelial cell junctions whose expression and function remains poorly characterized in most tissues, particularly in the vasculature. Using the yeast two-hybrid system to screen a human aorta cDNA library, we identified mitogen-activated protein/extracellular signal-responsive kinase (ERK) kinase (MEK)2, a member of the ERK cascade, as an hDlg binding partner.Taken together, these findings allow us to hypothesize that hDlg acts as a MEK2-specific scaffold protein for the ERK signaling pathway, and may improve our understanding of how scaffold proteins, such as hDlg, differentially tune MEK1/MEK2 signaling and cell responses.Here, we show that two scaffold proteins, caveolin-1 and IQGAP1, are required for phosphorylation of the actin associated pool of extracellular signal regulated kinase 1 and 2 (ERK1/2) in response to protein kinase C activation.ERK activation is enhanced by the scaffolding proteins KSR and MP1, localized near the cell membrane and late endosomes respectively, but little is known about their dynamic interplay. Originally identified in yeast, scaffold proteins are now recognized to contribute to the specificity of MEK/ERK pathways in mammalian cells. These scaffolds include KSR (kinase suppressor of Ras), beta-arrestin, MEK partner-1, Sef and IQGAP1.Here we analyze a potential scaffold of the Ras/mitogen-activated protein kinase (MAPK) pathway, kinase suppressor of Ras (KSR), by generating KSR-deficient mice. This demonstrates that KSR is a bona fide scaffold that is not required for but enhances signaling via the Ras/MAPK signaling pathway.We reported previously that several 14-3-3 isotypes bind to protein kinase C (PKC)-zeta and facilitate coupling of PKC-zeta to Raf-1 [van der Hoeven, van der Wal, Ruurs, van Dijk and van Blitterswijk (2000) Biochem. J. 345, 297-306], an event that boosts the mitogen-activated protein kinase (ERK) pathway in Rat-1 fibroblasts. In this study, IQGAP1 (IQ motif-containing GTPase-activating protein 1), a new Nrf2 interaction partner that we have published previously, was found to modulate MEK-ERK-mediated Nrf2 activation and induction of phase II detoxifying/antioxidant genes.In the aggregate, these results suggest that IQGAP1 may play an important role in the MEK-ERK-Nrf2 signaling pathway. Taken together, these findings allow us to hypothesize that hDlg acts as a MEK2-specific scaffold protein for the ERK signaling pathway, and may improve our understanding of how scaffold proteins, such as hDlg, differentially tune MEK1/MEK2 signaling and cell responses.The MAP kinase ERK and its scaffold protein MP1 interact with the chromatin regulator Corto during Drosophila wing tissue development.KSR-1 is a scaffold protein that is essential for Ras-induced activation of the highly conserved RAF-MEK-ERK kinase module. Previously, we identified a close homolog of KSR-1, called KSR-2, through structural homology-based data miningThe scaffold protein beta-arrestin2 interacted with both p-ERK and D1 DA receptor, triggering the cytosolic retention of p-ERK and inducing striatal neuronal apoptotic death. Involvement of the MP1 scaffold protein in ERK signaling regulation during Drosophila wing development.Beta-arrestin mediates desensitization and internalization of beta-adrenergic receptors (betaARs), but also acts as a scaffold protein in extracellular signal-regulated kinase (ERK) cascade.Recent studies indicate that kinase suppressor of Ras (KSR)is a scaffold protein for the Ras/Raf/MEK/ERK signaling cascade in mammals. The scaffold protein IQGAP1 regulates cell signaling through the RAF/MEK/ERK pathway.Kinase suppressor of Ras (KSR) is a molecular scaffold that interacts with the components of the Raf/MEK/ERK kinase cascade and positively regulates ERK signaling.Our results indicate that KSR-2 may act as a scaffold protein similar as KSR-1 to mediate the MAPK core (RAF-MEK-ERK) signaling but with a distinct RAF isoform specificity, namely KSR-2 may only mediate the A-RAF signaling while KSR-1 is responsible for transducing signals only from c-RAF.Kinase Suppressor of Ras (KSR) is a molecular scaffold that interacts with the core kinase components of the ERK cascade, Raf, MEK, ERK to provide spatial and temporal regulation of Ras-dependent ERK cascade signaling.These findings identify KSR2 as a Ca2+-regulated ERK scaffold and reveal a new mechanism whereby Ca2+ impacts Ras to ERK pathway signaling.MEK Partner-1 (MP1) was identified as a potential \\\"scaffold\\\" protein for the mammalian extracellular signal-regulated kinase (ERK) pathwayThe scaffold protein MP1 (MEK1 partner) assembles a scaffold complex in the ERK cascadeIQGAP1 scaffold-MAP kinase (ERK) interaction was noted in the human myeloma RPMI8226 cell lines. In conclusion, IQGAP1 plays an important role in the cell proliferation of MM via the MAP kinase (ERK) pathway.ERK-regulated, metastasis-associated scaffold protein Ezrinbsence of kinase suppressor of Ras 1 (KSR1), a scaffold protein of the ERK signaling pathwayGAB2 is a scaffold protein with diverse upstream and downstream effectors. MAPK and PI3K signaling pathways are known effectors of GAB2.In Drosophila, dMP1, that encodes an ERK scaffold protein, regulates ERK signaling during wing development and contributes to intervein and vein cell differentiation.Scaffold proteins regulate intracellular MAP kinase signaling by providing critical spatial and temporal specificities. We have shown previously that the scaffold protein MEK1 partner (MP1) is localized to late endosomes by the adaptor protein p14.The scaffold protein Shoc2/SUR-8 accelerates the interaction of Ras and Raf.We propose that Shoc2 regulates the spatio-temporal patterns of the Ras-ERK signaling pathway primarily by accelerating the Ras-Raf interaction.Proteomic characterization of the dynamic KSR-2 interactome, a signaling scaffold complex in MAPK pathway.Kinase modules are associated with scaffold proteins that regulate signaling by providing critical spatial and temporal specificities. Some of these scaffold proteins have been shown to be conserved, both in sequence and function. In mouse, the scaffold MP1 (MEK Partner 1) forms a signaling complex with MEK1 and ERK1.  extracellular signal-regulated kinase (ERK) pathway scaffold protein MP1. We propose that MORG1 is a component of a modular scaffold system that participates in the regulation of agonist-specific ERK signaling. IQGAP1, a scaffold protein linking cadherin-mediated cell adhesion to the cytoskeleton. This effect was accompanied by reduction of N-cadherin/IQGAP1/Erk-2 interactions.G protein-coupled receptor kinase 2 (GRK2) is a Rho-activated scaffold protein for the ERK MAP kinase cascade.GRK2 is a RhoA effector that serves as a RhoA-activated scaffold protein for the ERK MAP kinase cascade.\"}, {\"question\": \"Which are the families of mammalian DNA-(cytosine-5)-methyltransferases?\", \"ideal_answer\": [\"DNA (cytosine-5)-methyltransferases catalyze the specific transfer of a methyl group  to the C5 position of cytosine residues in DNA. Three families of DNA (cytosine-5)-methyltransferases have been identified in mammals: DNMT1, DNMT2 and DNMT3 (including DNMT3a, DNMT3b and DNMT3L isoforms). All of them share homologous catalytic domains. DNMT1 is the \\ufffd\\ufffd\\ufffdmaintenance\\u201d methyltransferase family. DNMT1 is specific for hemi-methylated DNA and ensures the faithful transmission of DNA methylation patterns in every replication cycle. DNMT3 is required for de novo methylation of DNA. DNMT3 targets unmethylated DNA and is responsible for the establishment of new methylation patterns. Dnmt2, in contrast to all other mammalian DNA (cytosine-5)-methyltransferases, does not possess a large N-terminal regulatory domain. The DNA methylation activity of DNMT2 is still controversial.\"], \"exact_answer\": [[\"DNMT1\"], [\"DNMT2\"], [\"DNMT3\"]], \"context\": \"Of the DNMTs, DNMT1 and DNMT3a are most highly expressed in postmitotic neurons.DNA methyltransferase 3B (Dnmt3b) belongs to a family of enzymes responsible for methylation of cytosine residues in mammals.Bovine DNA methylation imprints are established in an oocyte size-specific manner, which are coordinated with the expression of the DNMT3 family proteins.The DNMT3B de novo DNA methyltransferase (DNMT) plays a major role in establishing DNA methylation patterns in early mammalian development, but its catalytic mechanism remains poorly characterized.Dnmt2 proteins are the most conserved members of the DNA methyltransferase enzyme family, but their substrate specificity and biological functions have been a subject of controversy.This promoter hypermethylation may be mediated not only by DNMT-1 but also by an entire family of de novo DNA-methyltransferases, such as DNA-methyltransferase-3a (DNMT-3a) and -3b (DNMT-3b).Within the family of DNA methyltransferases (Dnmts), Dnmt3a and 3b establish methylation marks during early development, while Dnmt1 maintains methylation patterns after DNA replication. The maintenance function of Dnmt1 is regulated by its large regulatory N-terminal domain that interacts with other chromatin factors and is essential for the recognition of hemi-methylated DNA.In this study, we examined a role for the DNA methyltransferase DNMT3B, in particular, the truncated isoform DNMT3B7, which is generated frequently in cancer.DNMT1 is a widely expressed DNA methyltransferase maintaining methylation patterns in development, and mediating transcriptional repression by direct binding to HDAC2.DNA methyltransferases (DNMTs) are a family of related proteins that both catalyze the de novo formation of 5-methylcytosine and maintain these methylation marks in cell-specific patterns in virtually all mitotic cells of the body.This methylation is critical for imprinting; a reduction in the DNA methyltransferase DNMT1 causes a widespread loss of imprinting.DNA methyltransferase 3B (DNMT3B) is the key methyltransferase in DNA methylation regulations.DNA methyltransferases (DNMTs) are a family of enzymes that methylate DNA at the C5 position of cytosine residues, and their inhibition is a promising strategy for the treatment of various developmental and proliferative diseases, particularly cancers.DNA methylation is mediated by a family of DNA methyltransferases.These results suggest that DNMT1 and DNMT3B regulate BAG-1 expression via insulator protein DNA-binding and chromatin dynamics by regulating histone dimethylation.Here we show that human miR-148 represses DNA methyltransferase 3b (Dnmt3b) gene expression through a region in its coding sequence.DNA methylation is catalyzed by a family of DNA methyltransferases (DNMTs) including the maintenance enzyme DNMT 1 and de novo methyltransferases DNMT 3a and DNMT 3b.Among the reported down-regulated miRNAs in lung cancer, the miRNA (miR)-29 family (29a, 29b, and 29c) has intriguing complementarities to the 3'-UTRs of DNA methyltransferase (DNMT)3A and -3B (de novo methyltransferases), two key enzymes involved in DNA methylation, that are frequently up-regulated in lung cancer and associated with poor prognosis.Furthermore, we have examined the roles of the de novo methyltransferases (Dnmt3a and Dnmt3b) and related protein (Dnmt3L) in this process.DNA methyltransferase 1 (DNMT1) has been reported to interact with a wide variety of factors and to contain intrinsic transcriptional repressor activity.Inactivation of both Dnmt3a and Dnmt3b, DNA methyltransferases essential for the initiation of de novo DNA methylation, abolished the establishment of DNA methylation and the silencing of Rhox cluster genes in the embryo proper.Rather, in vitro analysis indicates that Dnmt3L stimulates DNA methylation by both Dnmt3a and Dnmt3b through direct binding to these proteins.OBJECTIVE: To investigate the association between single nucleotide polymorphism (SNP) in promoter of the DNA methyltransferase 3B (DNMT3B) gene and risk for development and lymphatic metastasis of gastric cardiac adenocarcinoma (GCA).Methylation at the 5-position of DNA cytosine on the vertebrate genomes is accomplished by the combined catalytic actions of three DNA methyltransferases (DNMTs), the de novo enzymes DNMT3A and DNMT3B and the maintenance enzyme DNMT1.There are two major categories of DNA methyltransferases: de novo and maintenance.Some, but not all, murine H1 subtypes interact with DNA methyltransferases DNMT1 and DNMT3B.\"}, {\"question\": \"Which classes of endogenous retroelements are known to date?\", \"ideal_answer\": [\"Endogenous retroelements fall into two distinct classes: retrotransposons containing LTRs (Long Terminal Repeats), and retrostransposons lacking LTRs.\"], \"exact_answer\": [[\"retrotransposons containing LTRs (Long Terminal Repeats)\"], [\"retrostransposons lacking LTRs (Long Terminal Repeats)\"]], \"context\": \"Transposons are divided into two general classes based on their transposition intermediate (DNA or RNA). Only one subclass, the non-LTR retrotransposons, which includes the Long INterspersed Element-1 (LINE-1 or L1), is currently active in humans as indicated by 96 disease-causing insertionsDiversity of LTR-retrotransposons and Enhancer/Suppressor Mutator-like transposons in cassavaMembers of two classes of LTR-retrotransposons, Ty1/copia-like and Ty3/gypsy-like, and of Enhancer/Suppressor Mutator (En/Spm)-like transposons were isolated and characterisedMammalian LINE-1 (L1) elements belong to the superfamily of autonomously replicating retrotransposable elements that lack the long terminal repeated (LTR) sequences typical of retroviruses and retroviral-like retrotransposons\"}, {\"question\": \"List GATA-1 interacting partners as discovered with the help of the biotinylation tagging approach.\", \"ideal_answer\": [\"Our work describes, for the first time, distinct GATA-1 interactions with the essential hematopoietic factor Gfi-1b, the repressive MeCP1 complex, and the chromatin remodeling ACF/WCRF complex, in addition to the known GATA-1/FOG-1 and GATA-1/TAL-1 complexes\", \"Using a biotinylation tagging/proteomics approach in erythroid cells, we describe distinct GATA-1 interactions with the essential hematopoietic factor Gfi-1b, the repressive MeCP1 complex and the chromatin remodeling ACF/WCRF complex, in addition to the known GATA-1/FOG-1 and GATA-1/TAL-1 complexes  \", \"The biotinylation tagging approach revealed, for the first time, distinct GATA-1 interactions with the essential hematopoietic factor Gfi-1b, the repressive MeCP1 complex, and the chromatin remodeling ACF/WCRF complex, in addition to the known GATA-1/FOG-1 and GATA-1/TAL-1 complexes.\"], \"exact_answer\": [[\"Gfi-1b\"], [\"MeCP1 complex\"], [\"ACF/WCRF complex\"], [\"fog-1\"], [\"tal-1\"]], \"context\": \"Our work describes, for the first time, distinct GATA-1 interactions with the essential hematopoietic factor Gfi-1b, the repressive MeCP1 complex, and the chromatin remodeling ACF/WCRF complex, in addition to the known GATA-1/FOG-1 and GATA-1/TAL-1 complexesWe also provide evidence that distinct GATA-1 complexes are associated with specific GATA-1 functions in erythroid differentiation, for example, GATA-1/Gfi-1b with the suppression of cell proliferation and GATA-1/FOG-1/MeCP1 with the repression of other hematopoietic transcription programsWe next applied the biotinylation tag to Ldb-1, a known partner of GATA-1, and characterized a number of novel interaction partners that are essential in erythroid development, in particular, Eto-2, Lmo4, and CdK9Using a biotinylation tagging/proteomics approach in erythroid cells, we describe distinct GATA-1 interactions with the essential hematopoietic factor Gfi-1b, the repressive MeCP1 complex and the chromatin remodeling ACF/WCRF complex, in addition to the known GATA-1/FOG-1 and GATA-1/TAL-1 complexes Importantly, we show that FOG-1 mediates GATA-1 interactions with the MeCP1 complex, thus providing an explanation for the overlapping functions of these two factors in erythropoiesis. We also show that subsets of GATA-1 gene targets are bound in vivo by distinct complexes, thus linking specific GATA-1 partners to distinct aspects of its functionsUsing a biotinylation tagging/proteomics approach in erythroid cells, we describe distinct GATA-1 interactions with the essential hematopoietic factor Gfi-1b, the repressive MeCP1 complex and the chromatin remodeling ACF/WCRF complex, in addition to the known GATA-1/FOG-1 and GATA-1/TAL-1 complexes.Using a biotinylation tagging/proteomics approach in erythroid cells, we describe distinct GATA-1 interactions with the essential hematopoietic factor Gfi-1b, the repressive MeCP1 complex and the chromatin remodeling ACF/WCRF complex, in addition to the known GATA-1/FOG-1 and GATA-1/TAL-1 complexes.We also provide evidence that distinct GATA-1 complexes are associated with specific GATA-1 functions in erythroid differentiation, for example, GATA-1/Gfi-1b with the suppression of cell proliferation and GATA-1/FOG-1/MeCP1 with the repression of other hematopoietic transcription programs.Our work describes, for the first time, distinct GATA-1 interactions with the essential hematopoietic factor Gfi-1b, the repressive MeCP1 complex, and the chromatin remodeling ACF/WCRF complex, in addition to the known GATA-1/FOG-1 and GATA-1/TAL-1 complexes.\"}, {\"question\": \"Which diseases are associated with Alu element insertion?\", \"ideal_answer\": [\"Diseases associated with Alu element insertion are the following: myotonic dystrophy type 2, Friedreich ataxia, spinocerebellar ataxia type 10, autosomal dominant optic atrophy, Menkes disease, hyper-IgM with immunodeficiency syndrome (HIGM), and anterior pituitary aplasia.\"], \"exact_answer\": [[\"myotonic dystrophy type 2\"], [\"Friedreich ataxia\"], [\"spinocerebellar ataxia type 10\"], [\"autosomal dominant optic atrophy\"], [\"Menkes disease\"], [\"hyper-IgM with immunodeficiency syndrome (HIGM)\"], [\"anterior pituitary aplasia\"], [\"HNPCC\"], [\"FAP\"]], \"context\": \"Alu elements are known to be the source of microsatellite repeats responsible for two other repeat expansion disorders: Friedreich ataxia and spinocerebellar ataxia type 10.We identified an Alu-element insertion located in intron 7 of OPA1 causing an in-frame deletion of exon 8 in 18 family members.Autosomal dominant optic atrophy (ADOA) is the most common form of hereditary optic neuropathy caused by mutations in the optic atrophy 1 (OPA1) gene.However, this is the first report of a pathogenic OPA1 gene mutation resulting from an Alu insertion.The first reported case of Menkes disease caused by an Alu insertion mutationWe present the first reported case of Menkes disease caused by an Alu element insertion mutation that interfered with splicing regulatory elements.HIGM syndrome caused by insertion of an AluYb8 element in exon 1 of the CD40LG geneA new mutation of the CD40LG gene that encodes the CD40 ligand molecule was characterized in a young patient harboring a hyper-IgM with immunodeficiency syndrome. Inactivation of CD40LG gene resulted from the insertion of an AluYb8 element in exon 1 responsible for a total deficiency of CD40 ligand expression by T lymphocytes.Alu-element insertion in the homeodomain of HESX1 and aplasia of the anterior pituitaryOne patient, who also has retinal coloboma, carries a HESX1 defect in the homozygous state: an Alu insertion in exon 3, a sequence that encodes the major part of the homeodomain.Anterior pituitary aplasia is a new example of a human disease caused by a germline retrotransposition event involving an Alu sequence.Myotonic dystrophy (DM) is associated with abnormal expansions of the CTG repeats in the 3' untranslated region of its gene. Previous studies in individuals of European origin demonstrated strong linkage disequilibrium between different CTG repeat length alleles and an Alu element insertion/deletion polymorphism in intron 8 of the DM gene\"}, {\"question\": \"Which proteins are the different members of the NF-kappaB family of transcription factors?\", \"ideal_answer\": [\"Nuclear factor kappa B (NF\\u03baB) is a dimeric transcription factor comprised of five family members RelA (p65), RelB, c-Rel, NF-kB1/p50 and NF-kB2/p52.\"], \"exact_answer\": [[\"RelA\", \"p65\"], [\"RelB\"], [\"c-Rel\"], [\"NF-kB1/p50\"], [\"NF-kB2/p52\"]], \"context\": \"The transcription factor NF-kappaB is composed of homodimeric and heterodimeric complexes of Rel/NF-kappaB-family polypeptides, which include Rel-A, c-Rel, Rel-B, NF-kappaB/p50 and NF-kappaB2/p52 .Nuclear factor kappa B (NF\\u03baB) is a dimeric transcription factor comprised of five family members RelA (p65), RelB, c-Rel, p50 and p52.In mammals, the genes rela, relb, crel, nfkappa\\u03921, and nfkappaB encode the five NF-kB protein family members RelA (p65), RelB, c-Rel, p50, and p52, respectively, which form homo- and heterodimeric DNA-binding complexes capable of regulating target gene transcription of specific biological responses differentially.These results suggest that the Rel-homology domain can act as an allosteric effector to promote transcription by p50/NFkappaB1 and that the configuration of p50 is important for its activity.\"}, {\"question\": \"What are the symptoms of Rotor syndrome?\", \"ideal_answer\": [\"Rotor syndrome is characterized by conjugated hyperbilirubinemia, coproporphyrinuria, and near-absent hepatic uptake of anionic diagnostics.\"], \"exact_answer\": [[\"conjugated hyperbilirubinemia\"], [\"coproporphyrinuria\"], [\"near-absent hepatic uptake of anionic diagnostics\"]], \"context\": \"The mechanistic basis of bilirubin excretion and hyperbilirubinemia syndromes is largely understood, but that of Rotor syndrome, an autosomal recessive disorder characterized by conjugated hyperbilirubinemia, coproporphyrinuria, and near-absent hepatic uptake of anionic diagnostics, has remained enigmatic. [A case with Rotor syndrome in hyperbilirubinemic family].Hyperbilirubinemia syndromes (Gilbert-Meulengracht, Crigler-Najjar, Dubin-Johnson, and Rotor syndrome)Rotor syndrome is a rare, benign familial disorder characterized by chronic fluctuating, nonhemolytic and predominantly conjugated hyperbilirubinemia with normal hepatic histologyJaundice is often the first and sometimes the only symptom of liver disease. Besides in parenchymal liver disease jaundice is also observed in chronic non-haemolytic hyperbilirubinemias (Crigler-Najjar type I and II, Gilbert's syndrome) and in hereditary conjugated hyperbilirubinaemias (Dubin-Johnson- and Rotor-syndrome). Rotor syndrome is a rare, benign familial disorder characterized by chronic fluctuating, nonhemolytic and predominantly conjugated hyperbilirubinemia with normal hepatic histology. In contrast to Dubin-Johnson syndrome, there is no liver pigmentation in Rotor syndrome.The mechanistic basis of bilirubin excretion and hyperbilirubinemia syndromes is largely understood, but that of Rotor syndrome, an autosomal recessive disorder characterized by conjugated hyperbilirubinemia, coproporphyrinuria, and near-absent hepatic uptake of anionic diagnostics, has remained enigmatic. Here, we analyzed 8 Rotor-syndrome families and found that Rotor syndrome was linked to mutations predicted to cause complete and simultaneous deficiencies of the organic anion transporting polypeptides OATP1B1 and OATP1B3.The mechanistic basis of bilirubin excretion and hyperbilirubinemia syndromes is largely understood, but that of Rotor syndrome, an autosomal recessive disorder characterized by conjugated hyperbilirubinemia, coproporphyrinuria, and near-absent hepatic uptake of anionic diagnostics, has remained enigmatic. Here, we analyzed 8 Rotor-syndrome families and found that Rotor syndrome was linked to mutations predicted to cause complete and simultaneous deficiencies of the organic anion transporting polypeptides OATP1B1 and OATP1B3.Hyperbilirubinemia syndromes (Gilbert-Meulengracht, Crigler-Najjar, Dubin-Johnson, and Rotor syndrome).Hyperbilirubinemia syndromes (Gilbert-Meulengracht, Crigler-Najjar, Dubin-Johnson, and Rotor syndrome).[A case with Rotor syndrome in hyperbilirubinemic family].\"}, {\"question\": \"List the existing methods for genetic manipulation of cells.\", \"ideal_answer\": [\"Genetic engineering, also called genetic modification, is the direct manipulation of an organism's genome using biotechnology. New DNA may be inserted in the host genome by first isolating and copying the genetic material of interest using molecular cloning methods to generate a DNA sequence, or by synthesizing the DNA, and then inserting this construct into the host organism. Genes may be removed, or \\\"knocked out\\\", using a nuclease. Gene targeting is a different technique that uses homologous recombination to change an endogenous gene, and can be used to delete a gene, remove exons, add a gene, or introduce point mutations. Based on results there are developed many methods for genetic manipulation of cells such as Microinjection, electroporation, liposomes and via viral vectors.\", \"The existing methods for genetic manipulation of cells include Agrobacterium-mediated or direct delivery methods, as well as methods using plant artificial chromosomes, site-directed plasmid mutagenesis, combined use of Red/ET recombination and unique restriction site elimination, Bacterial artificial chromosomes (BACs), direct in vitro transposition of viral genomes with a BAC cassette, and subsequent recovery in Escherichia coli, as well as transformation-competent artificial chromosome (TAC)-based acceptor vector, by exploiting the CreloxP recombination system and homing endonucleases.\"], \"exact_answer\": [[\"Agrobacterium-mediated delivery\"], [\"direct delivery methods\"], [\"methods using plant artificial chromosomes\"], [\"site-directed plasmid mutagenesis\"], [\"combined use of Red/ET recombination and unique restriction site elimination\"], [\"Bacterial artificial chromosomes (BACs)\"], [\"direct in vitro transposition of viral genomes with a BAC cassette, and subsequent recovery in Escherichia coli\"], [\"transformation-competent artificial chromosome (TAC)-based acceptor vector, by exploiting the CreloxP recombination system and homing endonucleases\"], [\"Microinjection\"], [\"electroporation\"], [\"viral vectors\"], [\"liposomes\"]], \"context\": \"For efficient genetic transformation, Agrobacterium-mediated as well as direct delivery methods have been used successfully. However, these methods suffer from many disadvantages especially in terms of transfer of large genes, gene complexes and gene silencing. To overcome these problems, recently, some efforts have been made to develop genetic transformation systems based on engineered plant chromosomes called minichromosomes or plant artificial chromosomes. Existing methods for site-directed plasmid mutagenesis are restrained by the small spectrum of modifications that can be introduced by mutagenic primers and the amplicon size limitations of in vitro DNA synthesis. As demonstrated here, the combined use of Red/ET recombination and unique restriction site elimination enables extensive manipulation regardless of plasmid size and DNA sequence elements.Bacterial artificial chromosomes (BACs) derived from genomes of large DNA viruses are powerful tools for functional delineation of viral genes.Here, we describe a novel method for cloning the genomes of large DNA viruses as BACs, which entails direct in vitro transposition of viral genomes with a BAC cassette, and subsequent recovery in Escherichia coli. Determination of insertion sites and adjacent viral sequences identify the BAC clones for genetic manipulation and functional characterization.Here, we describe a vector system for efficient multigene assembly and transformation. The system consists of a transformation-competent artificial chromosome (TAC)-based acceptor vector together with two donor vectors. By exploiting the CreloxP recombination system and homing endonucleases, multiple rounds of gene assembly cycling were carried out with alternate use of the donor vectors, and multiple genes were sequentially delivered into the TAC vector.\"}, {\"question\": \"List representatives of the major fungal hypoxanthine-adenine-guanine transporter families.\", \"ideal_answer\": [\"AzgA and Fcy21p are prototypes of the two major fungal hypoxanthine-adenine-guanine transporter families.\"], \"exact_answer\": [[\"AzgA\"], [\"Fcy21p\"]], \"context\": \"Comparative kinetic analysis of AzgA and Fcy21p, prototypes of the two major fungal hypoxanthine-adenine-guanine transporter familiesWe carried out a comparative kinetic analysis of two prototypes of these transporter families. The first was Fcy21p, a herein characterized protein of Candida albicans, and the second was AzgA, a transporter of Aspergillus nidulansThe azgA gene of Aspergillus nidulans encodes a hypoxanthine-adenine-guanine transporterIn Aspergillus nidulans, loss-of-function mutations in the uapA and azgA genes, encoding the major uric acid-xanthine and hypoxanthine-adenine-guanine permeases, respectively, result in impaired utilization of these purines as sole nitrogen sourcesComparative kinetic analysis of AzgA and Fcy21p, prototypes of the two major fungal hypoxanthine-adenine-guanine transporter families.The azgA gene of Aspergillus nidulans encodes a hypoxanthine-adenine-guanine transporter.In Aspergillus nidulans, loss-of-function mutations in the uapA and azgA genes, encoding the major uric acid-xanthine and hypoxanthine-adenine-guanine permeases, respectively, result in impaired utilization of these purines as sole nitrogen sources.Modelling, substrate docking and mutational analysis identify residues essential for function and specificity of the major fungal purine transporter AzgA.In Arabidopsis two proteins, AtAzg1 and AtAzg2, show substantial amino acid sequence similarity to the adenine-guanine-hypoxanthine transporter AzgA of Aspergillus nidulans\"}, {\"question\": \"Which proteins have been identified as RET ligands?\", \"ideal_answer\": [\"RET is activated by members of the glial cell line-derived neurotrophic factor (GDNF) family of ligands, which include GDNF, neurturin, artemin, and persephin.\", \"The RET proto-oncogene encodes a receptor tyrosine kinase, which is activated by members of the glial cell line-derived neurotrophic factor (GDNF) family of ligands, which include GDNF, neurturin (NRTN), artemin (ARTN), and persephin (PSPN).\"], \"exact_answer\": [[\"GDNF\", \"glial cell-line-derived neurotrophic factor\"], [\"ARTN\", \"artemin\"], [\"NTN\", \"neurturin\"], [\"PSPN\", \"persephin\"]], \"context\": \"Polymorphisms in the genes encoding the 4 RET ligands, GDNF, NTN, ARTN, PSPN, and susceptibility to Hirschsprung diseaseGiven the pivotal role of RET in HSCR, the genes encoding their ligands (GDNF, NRTN, ARTN, and PSPN) are also good candidates for the disease.Indeed, the RET ligands persephin and GDNF robustly stimulated calcitonin mRNA, which was blocked by pretreatment with NVP-AST487The ligands bind RET through GDNF family receptor alpha, termed GFRalpha1-4RET is activated by members of the glial cell line-derived neurotrophic factor (GDNF) family of ligands, which include GDNF, neurturin, artemin, and persephinGDNF availability determines enteric neuron number by controlling precursor proliferationFive germline mutations in the glial cell-line-derived neurotrophic factor (GDNF) gene, one of the RET ligands, have been detected in HSCR patients.The glial-cell-line-derived neurotrophic factor (GDNF) family receptors alpha (GFRalpha) are cell surface bound glycoproteins that mediate interactions of the GDNF ligand family with the RET receptorIn particular, the recent identification of neurotrophic factors acting as RET ligands (GDNF and Neurturin) provide additional candidate genes for HSCR. Mutations in genes that encode RET ligands (GDNF and NTN); components of another signaling pathway (EDNRB, EDN3, ECE-1); and the transcription factor, SOX10, have been identified in HSCR patients.Expression of RET and its ligand complexes, GDNF/GFRalpha-1 and NTN/GFRalpha-2, in medullary thyroid carcinomasMutations in the GDNF gene, and encoding one of the RET ligands, either alone or in combination with RET mutations, can also cause HSCR, as can mutations in four other genes (EDN3, EDNRB, ECE1, and SOX10)In particular, the recent identification of neurotrophic factors acting as RET ligands (GDNF and Neurturin) provide additional candidate genes for HSCR. Ret ligands have been recently identified as the neuron survival factor GDNF (Glial-Derived Neurotrophic Factor) and Neurturin.Mutation of the RET ligand, neurturin, supports multigenic inheritance in Hirschsprung diseaseConsidering that RET and glial cell line-derived neurotrophic factor (GDNF) mutations have been reported in the disease, we regarded the other RET ligand, neurturin (NTN), as an attractive candidate gene, especially as it shares large homologies with GDNFGiven the pivotal role of RET in HSCR, the genes encoding their ligands (GDNF, NRTN, ARTN, and PSPN) are also good candidates for the diseaseThe c-ret protooncogene, RET, encodes a receptor tyrosine kinase. RET is activated by members of the glial cell line-derived neurotrophic factor (GDNF) family of ligands, which include GDNF, neurturin, artemin, and persephinGlial cell line-derived neurotrophic factor (GDNF), a ligand of RET tyrosine kinase, and its family ligands promote the survival and differentiation of a variety of neurons.We demonstrate here that, after injury, the absence of c-Jun specifically in SCs caused impaired axonal regeneration and severely increased neuronal cell death. c-Jun deficiency resulted in decreased expression of several neurotrophic factors, and GDNF and Artemin, both of which encode ligands for the Ret receptor tyrosine kinase, were identified as novel direct c-Jun target genes.GDNF, now known to be the first identified member of a family of factors, signals through the previously known receptor tyrosine kinase, Ret. Unlike most ligands for receptor tyrosine kinases, GDNF does not bind and activate Ret directly, but requires the presence of GPI-linked coreceptors.BACKGROUND/PURPOSE: In 1996, the glial cell line-derived neurotrophic factor (GDNF) was identified as one of the ligands of the RET transmembrane receptor.The ret proto-oncogene encodes a receptor tyrosine kinase whose ligands belong to the glial cell line-derived neurotrophic factor (GDNF) protein family.RET is activated by members of the glial cell line-derived neurotrophic factor (GDNF) family of ligands, which include GDNF, neurturin, artemin, and persephin.Ret ligands have been recently identified as the neuron survival factor GDNF (Glial-Derived Neurotrophic Factor) and NeurturinGlial cell line-derived neurotrophic factor (GDNF) and its receptor molecule GDNFR-alpha, have recently been identified as members of the RET ligand binding complexRecently, the Glial cell line-derived neurotrophic factor (GDNF) has been identified to be a ligand for RETReplacing short segments of GDNF with the homologous segments from persephin (PSPN) (which cannot bind or activate GFRalpha1.RET or GFRalpha2.RET) identified sites along the second finger of GDNF critical for activating the GFRalpha1.RET and GFRalpha2.RET receptor complexesGDNF mediates its actions through a multicomponent receptor system composed of a ligand-binding glycosyl-phosphatidylinositol (GPI)-linked protein (designated GDNFR-alpha) and the transmembrane protein tyrosine kinase Ret\"}, {\"question\": \"Which substances are dangerous to g6PD deficient individuals?\", \"ideal_answer\": [\"Antimalarial drugs (primaquine, pamaquine, chloriquine), fava beans, sulfonamides, some antibiotics( nalidixic acid, nitrofurantoin, isoniazid, dapsone, and furazolidone) and henna\"], \"exact_answer\": [[\"primaquine\"], [\"pamaquine\"], [\"chloroquine\"], [\"fava beans\"], [\"sulfonamides\"], [\"antibiotics\"], [\"henna\"]], \"context\": \"The principal clinical manifestation encountered was favism.G6PD deficiency is the most common enzymopathy of red blood cells. The clinical symptoms of favism are jaundice, hematuria and haemolytic anaemia that seem to affect liver and kidney in long term. Thus we evaluate kidney and liver function of favism patients in an endemic area of the disease with a high rate of fava beans cultivation. \"}, {\"question\": \"Which mutations of SCN5A gene are implicated in Brugada syndrome?\", \"ideal_answer\": [\"The following mutations of SCN5A gene have been linked to Brugada syndrome:I137M, p.W1095X; c.3284G>A, R27H, E901K, G1743R, V728I, N1443S, E1152X, c.664C>T; p.Arg222X, Ala2>Thr, Ala735, Ala735>Thr, Val1340>Ile, IVS18-1G>A, E1784K (14x), F861WfsX90 (11x), D356N (8x), G1408R (7x), G400A, H558R, W822X, Q55X, V95I, A1649V, delF1617, c.4810+3_4810+6dupGGGT, K1527R, A1569P, R367H, A735V, R1192Q, D1795.\"], \"exact_answer\": [[\"I137M\"], [\"p.W1095X, c.3284G>A\"], [\"R27H\"], [\"E901K\"], [\"G1743R\"], [\"V728I\"], [\"N1443S\"], [\"E1152X\"], [\"c.664C>T; p.Arg222X\"], [\"Ala2>Thr, Ala735\"], [\"Ala735>Thr\"], [\"Val1340>Ile\"], [\"IVS18-1G>A\"], [\"E1784K (14x)\"], [\"F861WfsX90 (11x)\"], [\"D356N (8x)\"], [\"G1408R (7x)\"], [\"G400A\"], [\"H558R\"], [\"W822X\"], [\"Q55X\"], [\"V95I\"], [\"A1649V\"], [\"delF1617\"], [\"c.4810+3_4810+6dupGGGT\"], [\"K1527R\"], [\"A1569P\"], [\"R367H\"], [\"A735V\"], [\"R1192Q\"], [\"D1795\"], [\"p.I848fs\"], [\"p.R965C\"], [\"p.1876insM\"], [\"R1232W/T1620M\"]], \"context\": \"This was the first study to systematically investigate sodium channel variants in Chinese patients with ARVD; a new SCN5A mutation, I137M, was found. We describe a family showing the association between Brugada syndrome and epilepsy in which a known mutation in the SCN5A gene (p.W1095X, c.3284G>A) was identified. Direct sequencing and multiplex ligation-dependent probe amplification (MLPA) of the SCN5A gene for identification of mutations and larger rearrangements were performed, respectively. Eight patients (10.5%) had point mutations (R27H, E901K, G1743R (detected in three families), V728I, N1443S and E1152X).Brugada syndrome (BrS) is a life-threatening arrhythmia disorder associated with autosomal-dominant mutations in the SCN5A gene. We aimed to characterize the diagnostic challenges and clinical manifestations of a novel SCN5A mutation associated with BrS. RESULTS: From a novel SCN5A mutation (c.664C>T; p.Arg222X) identified in a proband with the characteristic electrocardiographic pattern and the history of sudden collapse, 122 family members were studied including 40 carriers of the mutation. We analyzed the sequence of this gene in 25 Spanish patients with Brugada syndrome. In 4 (16%), we found mutations that had not previously been described: three were amino acid changes (i.e. Ala2>Thr, Ala735>Thr and Val1340>Ile) and one was an intron mutation that affected messenger RNA processing (i.e. IVS18-1G>A).The 4 most frequent BrS1-associated mutations were E1784K (14x), F861WfsX90 (11x), D356N (8x), and G1408R (7x).One missense mutation (G400A) in SCN5A was detected in a conserved region among the cohort of 19 patients. A H558R polymorphism was detected on the same allele. Thus, we studied the functional restoration of nonsense-mutated SCN5A. RESULTS: HEK293 cells were transfected with SCN5A cDNA or its mutant carrying W822X, a nonsense mutation associated with Brugada syndrome and sudden cardiac death. Among the cohort of 19 patients, one missense mutation (G400A) in SCN5A was detected in a conserved region. An H558R polymorphism was detected on the same allele.Novel SCN5A mutation (Q55X) associated with age-dependent expression of Brugada syndrome presenting as neurally mediated syncope.A novel nonsense SCN5A mutation (Q55X) was identified in the proband, his mother, and his asymptomatic brother.Novel SCN5A gene mutations associated with Brugada syndrome: V95I, A1649V and delF1617An intronic mutation (c.4810+3_4810+6dupGGGT) in the SCN5A gene, located outside the consensus splice site, was detected in this study in a family with a highly variable clinical phenotype of Brugada syndrome and/or conduction disease and in a patient with Brugada syndrome.We genetically screened 30 asymptomatic probands (29 men and 1 woman; mean age 47.1 years) exhibiting a spontaneous Brugada-type ECG. Family members of patients with Brugada syndrome were excluded from the study. Genetic screening failed to identify SCN5A mutations in most cases but demonstrated a novel double missense mutation (K1527R and A1569P) located on the same allele in another asymptomatic subject (case 2).We screened patients with SUNDS for mutations in SCN5A, the gene known to cause Brugada syndrome, as well as genes encoding ion channels associated with the long-QT syndrome.One mutation, R367H, lies in the first P segment of the pore-lining region between the DIS5 and DIS6 transmembrane segments of SCN5A. A second mutation, A735V, lies in the first transmembrane segment of domain II (DIIS1) close to the first extracellular loop between DIIS1 and DIIS2, whereas the third mutation, R1192Q, lies in domain III.The substitution (D1790G) causes LQTS and the insertion (D1795) induces both LQTS and Brugada syndromes in carrier patients.\"}, {\"question\": \"List all articles on network meta-analysis for smoking cessation\", \"ideal_answer\": [\"Cardiovascular events associated with smoking cessation pharmacotherapies: a network meta-analysis.\\nPharmacological interventions for smoking cessation: an overview and network meta-analysis\\nEffectiveness and cost-effectiveness of computer and other electronic aids for smoking cessation: a systematic review and network meta-analysis.\\nSmoking cessation interventions in COPD: a network meta-analysis of randomised trials.\"], \"exact_answer\": [[\"Cardiovascular events associated with smoking cessation pharmacotherapies: a network meta-analysis.\"], [\"Pharmacological interventions for smoking cessation: an overview and network meta-analysis\"], [\"Effectiveness and cost-effectiveness of computer and other electronic aids for smoking cessation: a systematic review and network meta-analysis.\"], [\"Smoking cessation interventions in COPD: a network meta-analysis of randomised trials.\"]], \"context\": \"Cardiovascular events associated with smoking cessation pharmacotherapies: a network meta-analysis.Pharmacological interventions for smoking cessation: an overview and network meta-analysis.Effectiveness and cost-effectiveness of computer and other electronic aids for smoking cessation: a systematic review and network meta-analysis.network meta-analysis on interventions for smoking cessation including over 100 trialsSmoking cessation interventions in COPD: a network meta-analysis of randomised trials.\"}, {\"question\": \"Which are the most frequent syndromes associated with inherited bone marrow failure?\", \"ideal_answer\": [\"The inherited bone marrow failure syndromes (IBMFS) are a heterogeneous group of genetic disorders that share the inability of the bone marrow to produce an adequate number of blood cells. The 4 most frequent syndromes are Fanconi anemia (FA), dyskeratosis congenita (DC), Diamond-Blackfan anemia (DBA), and Shwachman-Diamond syndrome (SDS)  \", \"The inherited bone marrow failure syndromes (IBMFS) are a heterogeneous group of genetic disorders that share the inability of the bone marrow to produce an adequate number of blood cells. The 4 most frequent syndromes are Fanconi anemia (FA), dyskeratosis congenita (DC), Diamond-Blackfan anemia (DBA), and Shwachman-Diamond syndrome (SDS).\"], \"exact_answer\": [[\"Fanconi anemia\"], [\"Dyskeratosis congenita\"], [\"Diamond-Blackfan anemia\"], [\"Shwachman-Diamond syndrome\"]], \"context\": \"Ocular and orbital manifestations of the inherited bone marrow failure syndromes: Fanconi anemia and dyskeratosis congenitaThe inherited bone marrow failure syndromes (IBMFS) are a heterogeneous group of genetic disorders that share the inability of the bone marrow to produce an adequate number of blood cells. The 4 most frequent syndromes are Fanconi anemia (FA), dyskeratosis congenita (DC), Diamond-Blackfan anemia (DBA), and Shwachman-Diamond syndrome (SDS)The major inherited bone marrow failure syndromes associated with development of pancytopenia include Fanconi anemia, dyskeratosis congenita, Shwachman-Diamond syndrome, and amegakaryocytic thrombocytopenia.The major inherited bone marrow failure syndromes associated with development of pancytopenia include Fanconi anemia, dyskeratosis congenita, Shwachman-Diamond syndrome, and amegakaryocytic thrombocytopenia\"}, {\"question\": \"Which ones are the ESKAPE organisms?\", \"ideal_answer\": [\"The 6 ESKAPE pathogens are Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species.\"], \"exact_answer\": [[\"Enterococcus faecium\"], [\"Staphylococcus aureus\"], [\"Klebsiella pneumoniae\"], [\"Acinetobacter baumannii\"], [\"Pseudomonas aeruginosa\"], [\"Enterobacter species\"]], \"context\": \"The aim of this study was to test in the clinic whether antimicrobial diversity affects resistance of Enterococcus faecium, Staphylococcus aureus, Klebsiella species, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species (ESKAPE) pathogens in ventilator-associated pneumonia (VAP).Despite important geographical variations, Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa and Enterobacter species (ESKAPE) pathogens constitute more than 80% of ventilator-associated pneumonia (VAP) episodes. Multidrug resistance among the 'ESKAPE' organisms - encompassing Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa and Enterobacter spp. - is of particular concern because they are responsible for many serious infections in hospitals.Patients recovering from traumatic injuries or surgery often require weeks to months of hospitalization, increasing the risk for wound and surgical site infections caused by ESKAPE pathogens, which include A. baumannii (the ESKAPE pathogens are Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species).'ESKAPE' (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acintobacter baumannii, Pseudomonas aeruginosa and Enterobacter spp.) pathogens play a major role in the rapidly changing scenario of antimicrobial resistance in the 21st century.Although bacteremias caused by the 6 ESKAPE pathogens (Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa, and Enterobacter species) have recently been highlighted as a serious complication in solid organ transplant (SOT), more information is urgently needed. Multidrug resistance among the 'ESKAPE' organisms - encompassing Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa and Enterobacter spp.Multidrug resistance among the 'ESKAPE' organisms - encompassing Enterococcus faecium, Staphylococcus aureus, Klebsiella pneumoniae, Acinetobacter baumannii, Pseudomonas aeruginosa and Enterobacter spp. - is of particular concern because they are responsible for many serious infections in hospitals. \"}, {\"question\": \"Carbapenemase-producing gram-negative bacteria is a major health concern because their resistance to antibiotics.\\nList the most frequent carbapenemases found in Enterobacteriaceae.\", \"ideal_answer\": [\"The most frequent carbapenemases in Enterobacteriaceae  are OXA-48, KPC, VIM, NDM, IMP, SME, NMC, GES, IMI and MBL.\"], \"exact_answer\": [[\"OXA-48\"], [\"KPC\"], [\"VIM\"], [\"NDM\"], [\"IMP\"], [\"SME\"], [\"NMC\"], [\"GES\"], [\"IMI\"], [\"MBL\"]], \"context\": \"The most frequent carbapenemases in Enterobacteriaceae were OXA-48, KPC and VIM-1; in P. aeruginosa it was VIM-2 and in A. baumannii OXA-23.bla(IMP), bla(VIM), bla(SPM), bla(SIM) and bla(GIM), and of bla(KPC), bla(NDM) and bla(OXA-Detection of NDM-1, VIM-1, KPC, OXA-48, and OXA-162 carbapenemases by matrix-assisted laser desorption ionization-time of flight mass spectrometry.The Check-MDR CT102 DNA microarray enables detection of the most prevalent carbapenemases (NDM, VIM, KPC, OXA-48 and IMP)63 KPC producers, 47 MBL producers, and 31 KPC and MBL producersOXA-48 producers were poorly detected, but all systems reliably detected isolates with KPC and most with metallo-carbapenemases.Producers of IMP-, VIM-, and KPC-type carbapenemases with high levels of resistance to cephalosporins and to carbapenems were detected at 1x10(1) CFU/ml. The OXA-48 producers were not detected on ChromID ESBL medium unless coexpressing ESBLs, whereas carbapenemase-producing isolates with MICs of <4 microg/ml were not detected on CHROMagar KPC medium.The most frequent carbapenemase was KPC-2 or -3 (73 strains), followed by VIM-1 (14), IMP-1 (11), SME-2 (5), and NMC-A (1). All serine carbapenemases were detected in the United States, while MbetaL-producing strains were isolated in Europe. Carbapenemase-producing Enterobacteriaceae showed high rates of resistance to most antimicrobial agents tested.By using this methodology, isolates producing KPC, GES, Sme, IMI, and NMC-A carbapenemases were successfully distinguished from those producing other classes of beta-lactamasesKPC-2 and KPC-3 were the most frequently occurring carbapenemases (24 isolates, 20.2%) iK. pneumoniae strains with known KPC (n\\u2003=\\u200331) or VIM (n\\u2003=\\u200320) carbapenemasesCarbapenemases including Klebsiella pneumoniae carbapenemases (KPC) are widespread among clinical isolates in the family Enterobacteriaceae.Outbreak of vim-1-carbapenemase-producing Enterobacter cloacae in a pediatric intensive care unit.mergence of pediatric infections caused by carbapenemases-producing Enterobacteriaceae is a critical issue as they are resistant to most \\u03b2-lactam antibiotics.SME-2-producing S. marcescens were isolated in the New York City area, Texas, and Ohio, while NMC-A was found in one E. cloacae strain from New York. In contrast, metallo-beta-lactamases were prevalent in Europe. IMP-1-producing E. cloacae (11 isolates) were detected in Turkey, while VIM-1-producing strains were found in Italy (Enterobacter spp.) and Greece (Klebsiella pneumoniae).d several simple confirmatory methods that allow the recognition of bacteria producing class A carbapenemases, including KPC, Sme, IMI, NMC-A, and GES,\"}, {\"question\": \"Which are the clinical characteristics of isolated Non-compaction cardiomyopathy?\", \"ideal_answer\": [\"The clinical characteristics of isolated Non-compaction cardiomyopathy are excessively thickened endocardial layer with deep intertrabecular recesses (with ratio of non-compacted to compacted myocardium >2), heart failure, syncope, ventricular arrhythmias, stroke, pulmonary hypertension, complete left branch conductive block, sick sinus syndrome and paroxysmal supraventricular tachycardia\"], \"exact_answer\": [[\"excessively thickened endocardial layer with deep intertrabecular recesses\"], [\"ratio of non-compacted to compacted myocardium >2\"], [\"heart failure\"], [\"syncope\"], [\"ventricular arrhythmias\"], [\"stroke\"], [\"Pulmonary hypertension\"], [\"complete left branch conductive block\"], [\"sick sinus syndrome\"], [\"paroxysmal supraventricular tachycardia\"]], \"context\": \"Non compaction cardiomyopathy is a rare disorder caused by the arrest of myocardial compaction during embryogenesis, leading to a non compacted endocardial layer with marked hypertrabeculation and deep recesses.The form of presentation was heart failure in 53% of subjects, syncope in 20%o, ventricular arrhythmias in 13%o and stroke in 7%>. Left ventricular end-diastolic diameter was 66 \\u00b1 11 mm and estimated ejection fraction was 27 \\u00b1 10%>. Apical and/or mid-ventricular segments of the left ventricle were involved in all the cases. Pulmonary hypertension was present in 40%o.Prominent hypertrabecularisation (ratio of non-compacted to compacted myocardium >2) in the apical and mid left ventricular segments is typical for non-compaction cardiomyopathy. Isolated ventricular non-compaction or non-compaction cardiomyopathy (NCCM) is characterized by an excessively thickened endocardial layer with deep intertrabecular recesses, reminiscent of the myocardium during early embryogenesis.Sudden cardiac death (including 1 occurred while following-up) was reported in 7 family members (17.5%, 7/40). LVNC was diagnosed in 10 out of the 33 family members (30.3%) and heart enlargement was evidenced in 3, heart failure in 2, complete left branch conductive block in 3, serious sick sinus syndrome (SSS) treated with permanent pacemaker implantation in 1 and paroxysmal supraventricular tachycardia treated with radiofrequency ablation procedure in 1 out of these 10 LVNC patients. \"}, {\"question\": \"In which genomic positions is the histone variant macroH2A enriched?\", \"ideal_answer\": [\"macroH2A1 is enriched on the inactive X chromosome in female mammalian cells, where it functions to maintain gene silencing. The transcribed regions of most active genes are depleted of macroH2A, often in sharply localized domains that show depletion of 4-fold or more relative to bulk mouse liver chromatin. This repressor activity of marcroH2A is further supported by the substantial and relatively uniform macroH2A1 enrichment along the inactive X chromosome, which averages 4-fold. In addition to localizing to the MCB, macroH2A accumulates at a perinuclear structure centered at the centrosome\"], \"exact_answer\": [[\"Repressed regions\"], [\"Inactive X chromosome\"], [\"Centromeres\"]], \"context\": \"Although macroH2A1 nucleosomes are widely distributed across the genome, their local concentration varies over a range of 100-fold or moreThe transcribed regions of most active genes are depleted of macroH2A1, often in sharply localized domains that show depletion of 4-fold or more relative to bulk mouse liver chromatinThis repressor activity is further supported by the substantial and relatively uniform macroH2A1 enrichment along the inactive X chromosome, which averages 4-foldmH2A1 consistently colocalizes with a heterochromatin marker (H3K27me2; histone H3 trimethylated at lysine 27) and mH2A2 with a euchromatin marker (H3K4me3; histone H3 trimethylated at lysine 4)Similar results were found for the L41gene, with enrichment of mH2A in the promoter region.By using highly specific antibodies against mH2A1 and stable HEK 293 cell lines expressing either green fluorescent protein (GFP)-mH2A1 or GFP-H2A, we found that the Xi chromosome contains approximately 1.5-fold more mH2A1 than the autosomesThe data show that mH2A1 is uniformly distributed across the entire Xi chromosome.Interestingly, a stronger mH2A1 enrichment along the pseudoautosomal X chromosome region was observed in both sexes.macroH2A1 nucleosomes were enriched on endogenous MLVs, with the highest enrichment occurring on the 5' end of pro-polmacroH2A1 is enriched on the inactive X chromosome in female mammalian cells, where it functions to maintain gene silencing.We found that macroH2A1 was depleted on the transcribed regions of active genesIn contrast, macroH2A1 was concentrated on the inactive X chromosome, consistent with our previous immunofluorescence studies.These results support the hypothesis that macroH2As function as transcriptional repressorsTaking into account the properties of macroH2A toward chromatin structure and dynamics and its role in gene repression our data suggest that the increased expression of macroH2A and the hypermethylation of DNA which occurs upon winter-acclimatization plays a major role for the reorganization of chromatin structure and the regulation of gene expression during the physiological adaptation to a colder environmentInactivation is a multistep process that involves a large non-coding RNA termed XIST, a variety of epigenetic modifications of chromatin, and alterations in protein composition such as enrichment of the histone variant macroH2AOne of several features acquired by chromatin of the inactive X chromosome (Xi) is enrichment for the core histone H2A variant macroH2A within a distinct nuclear structure referred to as a macrochromatin body (MCB)In addition to localizing to the MCB, macroH2A accumulates at a perinuclear structure centered at the centrosomeThe centrosomal pool of macroH2A1 accumulates in the presence of an inhibitor of the 20S proteasome.Although the MCB dissipates during late S phase and G2 before reforming in late G1, macroH2A1 remains associated during mitosis with specific regions of the Xi, including at the X inactivation center\"}, {\"question\": \"Which agents are included in the FLAMSA chemotherapy regimen?\", \"ideal_answer\": [\"Fludarabine, cytarabine and amsacrine are included in the FLAMSA chemotherapy regimen.\"], \"exact_answer\": [[\"fludarabine\"], [\"cytarabine\"], [\"amsacrine\"]], \"context\": \"After 3 days of rest, RIC was carried out, consisting of 4 Gy total body irradiation, antithymocyte globulin (ATG-Fresenius), and cyclophosphamide (fludarabine, amsacrine, and cytarabine (FLAMSA)-RIC protocol).We report a retrospective single center analysis of aplasia-inducing chemotherapy using fludarabine, cytarabine, and amsacrine (FLAMSA) followed by reduced-intensity conditioning (RIC) for allogeneic hematopoietic cell transplantation (HCT) in 62 consecutive primary refractory or relapsed AML patients.A sequential regimen of cytoreductive chemotherapy (fludarabine, high-dose AraC, amsacrine) followed by reduced intensity conditioning (FLAMSA-RIC), was uniformly used for conditioning.Between November 2003 and June 2010, 30 patients (20 high-risk MDS, 10 sAML) received fludarabine (4 \\u00d7 30 mg/m(2)), amsacrine (4 \\u00d7 100 mg/m(2)), and Ara-C (4 \\u00d7 2 g/m(2), FLAMSA). The preparative regimen consisted of fludarabine, Ara-C and amsacrine (FLAMSA) chemotherapy, followed by reduced intensity conditioning (RIC) 3 days later. Fifteen patients received fludarabine 30 mg/m(2), cytarabine 2000 mg/m(2), amsacrine 100 mg/m(2) on days -10, -9, -8 and -7, anti-thymocyte globulin (ATG-Fresenius) 20 mg/kg body weight on days -6, -5 and -4 and fractionated total body irradiation 2 x 2 Gy on days -3, -2 and -1 (FLAMSA-ATG-TBI) before allogeneic hematopoietic stem cell transplantation. The goal of this analysis was to define the role of the moderate-intensity fludarabin Ara-C amsacrin (FLAMSA)-reduced intensity conditioning (RIC) regimen for patients with high-risk AML undergoing allogeneic SCT (alloSCT) in first CR1. The goal of this analysis was to define the role of the moderate-intensity fludarabin Ara-C amsacrin (FLAMSA)-reduced intensity conditioning (RIC) regimen for patients with high-risk AML undergoing allogeneic SCT (alloSCT) in first CR1The preparative regimen consisted of fludarabine, Ara-C and amsacrine (FLAMSA) chemotherapy, followed by reduced intensity conditioning (RIC) 3 days laterA sequential regimen of cytoreductive chemotherapy (fludarabine, high-dose AraC, amsacrine) followed by reduced intensity conditioning (FLAMSA-RIC), was uniformly used for conditioningA sequential regimen of cytoreductive chemotherapy (fludarabine, high-dose AraC, amsacrine) followed by reduced intensity conditioning (FLAMSA-RIC), was uniformly used for conditioningWe report a retrospective single center analysis of aplasia-inducing chemotherapy using fludarabine, cytarabine, and amsacrine (FLAMSA) followed by reduced-intensity conditioning (RIC) for allogeneic hematopoietic cell transplantation (HCT) in 62 consecutive primary refractory or relapsed AML patientsThe preparative regimen consisted of fludarabine, Ara-C and amsacrine (FLAMSA) chemotherapy, followed by reduced intensity conditioning (RIC) 3 days later\"}, {\"question\": \"Which are the subunits of the IkB protein kinase (IKK)?\", \"ideal_answer\": [\"Proinflammatory NF-kappaB activation requires the IkappaB (inhibitor of NF-kappaB) kinase (IKK) complex that contains two catalytic subunits named IKKalpha and IKKbeta and a regulatory subunit named NF-kappaB essential modulator (NEMO).Additional components may exist, transiently or permanently, but their characterization is still unsure.\", \"The IKK kinase complex is the core element of the NF-kappaB cascade. It is essentially made of two kinases (IKKalpha and IKKbeta) and a regulatory subunit, NEMO (NF-kB Essential Modulator)/IKKgamma. Additional components may exist, transiently or permanently, but their characterization is still unsure.\", \" The IKK protein complex is comprised of IKK\\u03b1, IKK\\u03b2 and NEMO subunits, with IKK\\u03b2 thought to play the dominant role in modulating NF-\\u03baB activity.   The IkappaB-Kinase (IKK) complex is a multisubunit protein complex crucial for signal-induced phosphorylation of the IkappaB proteins and thus controls the activity of the transcription factor NF-kappaB. Besides the two kinases IKKalpha and IKKbeta, the IKK complex contains NEMO/IKKgamma, an additional subunit with regulatory and adaptor functions.  \"], \"exact_answer\": [[\"IKKalpha\", \"IKK\\u03b1\", \"IKK1\"], [\"IKKbeta\", \"IKK\\u03b2\", \"IKK2\"], [\"IKKgamma\", \"IKK\\u03b3\", \"NEMO\"]], \"context\": \"NEMO (NF-\\u03baB essential modulator) associates with catalytic subunits IKK\\u03b1 and IKK\\u03b2 to form the I\\u03baB kinase (IKK) complex and is a key regulator of NF-\\u03baB pathway signaling.All known NF-\\u03baB activators converge on the IkappaB kinase (IKK) complex to activate the canonical and non-canonical NF-\\u03baB pathways. The IKK complex contains two catalytic subunits (IKK\\u03b1 and IKK\\u03b2) and a regulatory subunit NEMO/IKK\\u03b3 that regulates the canonical NF-\\u03baB pathway, whereas IKK\\u03b1 regulates the non-canonical pathway. The IKK protein complex is comprised of IKK\\u03b1, IKK\\u03b2 and NEMO subunits, with IKK\\u03b2 thought to play the dominant role in modulating NF-\\u03baB activity. The I\\u03baB kinase (IKK) complex plays a crucial role in the activation of the transcription factor NF-\\u03baB by phosphorylating an inhibitory molecule I\\u03baB\\u03b1. Recently, we showed that IKK2 (also called IKK\\u03b2), a catalytic subunit of the IKK complex, induces immunoglobulin E-mediated degranulation in mast cells by phosphorylating SNAP-23, the target-membrane soluble N-ethylmaleimide-sensitive fusion factor attachment protein receptor (SNARE).The depletion of any of the three subunits of the inhibitor of NF\\u03baB (I\\u03baB) kinase (IKK\\u03b1, IKK\\u03b2, IKK\\u03b3/NEMO), each of which is essential for the canonical NF\\u03baB activation pathway, limits autophagy induction by physiological or pharmacological triggers, while constitutive active IKK subunits suffice to stimulate autophagy. NF-\\u03baB activation is tightly regulated by the I\\u03baB kinase (IKK) complex, which is composed of two catalytic subunits IKK\\u03b1 and IKK\\u03b2, and a regulatory subunit IKK\\u03b3/NEMO.Activation of the classical NF-\\u03baB pathway is mediated through the activity of the IKK kinase complex, which consists of a heterotrimer of IKK\\u03b1, IKK\\u03b2, and IKK\\u03b3 subunits. NF-kB is a key regulator of inflammation and immunity in cancer development. The IkB kinase (IKK) is a multisubunit complex containing catalytic subunits termed IKK-\\u03b1, -\\u03b2 and -\\u03b3. Activation of NF-\\u03baB involves the I\\u03baB kinase (IKK)-complex composed of two catalytic subunits, IKK\\u03b1 and IKK\\u03b2, and a regulatory scaffold protein, IKK\\u03b3.The IKK kinase complex is the core element of the NF-kappaB cascade. It is essentially made of two kinases (IKKalpha and IKKbeta) and a regulatory subunit, NEMO/IKKgamma.Additional components may exist, transiently or permanently, but their characterization is still unsure.Activation of NF-kappaB subunits is regulated by the upstream IkappaB kinase (IKK) complex which contains IKKalpha, IKKbeta, and IKKgamma; the latter also known as NF-kappaB essential modulator (NEMO).Proinflammatory NF-kappaB activation requires the IkappaB (inhibitor of NF-kappaB) kinase (IKK) complex that contains two catalytic subunits named IKKalpha and IKKbeta and a regulatory subunit named NF-kappaB essential modulator (NEMO).Inhibition of NF-kappaB by suppressing the canonical NF-kappaB activation pathway, for instance by knockdown of the three subunits of the inhibitor of NF-kappaB (I kappaB) kinase (IKK) complex (IKK1, IKK2 and NEMO) triggers apoptosis in such cells. The IkappaB-Kinase (IKK) complex is a multisubunit protein complex crucial for signal-induced phosphorylation of the IkappaB proteins and thus controls the activity of the transcription factor NF-kappaB. Besides the two kinases IKKalpha and IKKbeta, the IKK complex contains NEMO/IKKgamma, an additional subunit with regulatory and adaptor functions.Activation of a large multisubunit protein kinase, called the inhibitor kappaB kinase (IKK) complex, is central to the induction of the family of transcription factors nuclear factor kappaB. IKK is comprised of two catalytic subunits, IKKalpha and IKKbeta, and a regulatory IKKgamma subunit.The IkB kinase (IKK) is a multisubunit complex containing catalytic subunits termed IKK-\\u03b1, -\\u03b2 and -\\u03b3\"}, {\"question\": \"Name five popular  computer  programs used to identify genes in genomic sequences\", \"ideal_answer\": [\"Any five from the following not exhaustive list:  AUGUSTUS, MGENE, CRAIG, Agene, EUGENE,  Fgenesh++C, Fgenesh++,  Fgenesh, GeneID, GeneMark.hmm,  GENOMIX, GESECA, GLEAN,  GlimmerHMM, Gramene,JIGSAW, MAKER, ,MGENE, N-SCAN,  SGP2, SNAP,  ExonHunter, Evigan, Genescan, HMMGene, MZEF, Genie, Twinscan, SLAM, GRAIL.\"], \"exact_answer\": [[\"Genscan\"], [\"Glimmer\"], [\"Fgenesh\"], [\"Augustus\"], [\"GeneID\"]], \"context\": \"This unit describes how to use the gene-finding programs GeneMark.hmm-E and GeneMark-ES for finding protein-coding genes in the genomic DNA of eukaryotic organisms.YACOP parses and combines the output of the three gene-predicting systems Criticia, Glimmer and ZCURVE.In this paper we have explored the benefits of combining predictions from already existing gene prediction programs. We have introduced three novel methods for combining predictions from programs Genscan and HMMgeneWe describe the data sets on which the experiments were performed, the approaches employed by the five algorithms: GenScan, HMMGene, GeneMark, Pombe and FFG, the methodology of our evaluation, and the results of the experiments.Phat is based on a generalized hidden Markov model (GHMM) similar to the models used in GENSCAN, Genie and HMMgene.We present an independent comparative analysis of seven recently developed gene-finding programs: FGENES, GeneMark.hmm, Genie, Genescan, HMMgene, Morgan, and MZEF.We have explored the effectiveness when the results of several gene-finding programs were re-analyzed and combined. We studied several methods with four programs (FEXH, GeneParser3, GEN-SCAN and GRAIL2)FGENES; Gene-Mark.hmm; Genie; Genescan; HMMgene; Morgan and MZEF,\"}, {\"question\": \"Which are the major transcription factors regulating glycolysis in mammals?\", \"ideal_answer\": [\"The main positive transcriptional regulator of Glycolysis in mammals is HIF1-alpha (Hypoxia Inducible Factor 1a). HIF1-alpha is upregulated by the oncogenes c-Myc and Src, which therefore also positively regulate glycolysis. Several reports have linked HIF-1\\u03b1 induction with STAT3 activation. SIRT6 appears to function as a corepressor of the transcription factor Hif1alpha. NF-\\u03baB also stimulates glycolysis through the inhibition of PDK4 expression. Loss of the widely expressed transcription factor Oct1 induces a coordinated metabolic shift: mitochondrial activity and amino acid oxidation are increased, while glucose metabolism is reduced. HNF4alpha is critical for regulating glucose transport and glycolysis and in doing so is crucial for maintaining glucose homeostasis.\"], \"exact_answer\": [[\"HIF1-alpha\"], [\"c-Myc\"], [\"Src\"], [\"STAT3\"], [\"SIRT6\"], [\"NF-\\u03baB\"], [\"Oct1\"], [\"HNF4-alpha\"], [\"PVHL\"]], \"context\": \"myc regulates hepatic glycolysisincrease in c-Myc protein was able to induce liver glucose utilization and accumulation, and suggested that c-Myc transcription factor is involved in the control in vivo of liver carbohydrate metabolismSTAT3 is a negative regulator of aerobic glycolysis.HIF1 alpha-mediated switches in the energy production of tumor cells from OXPHOS to glycolysisNF-\\u03baB organizes energy metabolism networks by controlling the balance between the utilization of glycolysis and mitochondrial respiration. NF-\\u03baB inhibition causes cellular reprogramming to aerobic glycolysis under basal conditions and induces necrosis on glucose starvationHypoxia-inducible factor-1\\u03b1 (HIF-1\\u03b1), which is a transcription factor that enhances glycolysis in cells in response to hypoxiaSIRT6 appears to function as a corepressor of the transcription factor Hif1alpha, a critical regulator of nutrient stress responsesSIRT6-deficient cells exhibit increased Hif1alpha activity and show increased glucose uptake with upregulation of glycolysis and diminished mitochondrial respirationc-Myc regulates genes involved in the biogenesis of ribosomes and mitochondria, and regulation of glucose and glutamine metabolismMyc regulates gene expression either directly, such as glycolytic genes including lactate dehydrogenase A (LDHA)loss of the widely expressed transcription factor Oct1 induces a coordinated metabolic shift: mitochondrial activity and amino acid oxidation are increased, while glucose metabolism is reduced.PVHL is a regulator of glucose metabolismCarbohydrate response element-binding protein (ChREBP) is a glucose-responsive transcription factor that plays a critical role in the glucose-mediated induction of gene products involved in hepatic glycolysis and lipogenesisActivation of NF-kappaB, by loss of p53, caused an increase in the rate of aerobic glycolysis and upregulation of Glut3p53 restricts activation of the IKK-NF-kappaB pathway through suppression of glycolysiscells express hypoxia-inducible factor (HIF)-1alpha, a transcription factor that responds to oxygen tension and regulates glycolysisThe HIF-1 dependent genic products are involved in tumoral angiogenesis, in the metabolic switch to anaerobic glycolysis and in prosurvival, proliferative and apoptotic mechanismshe hypoxia-inducible transcription factor (HIF) regulates the expression of genes involved in angiogenesis, erythropoiesis and glycolysishypoxia-inducible factor-1, which activates a gene program associated with angiogenesis, glycolysis, and adaptation to pHChREBP (carbohydrate response element-binding protein) was recently identified as a candidate transcription factor in the glucose-signaling pathwayHypoxia-inducible factor (HIF)-1 is a master transcription factor, which up-regulates glycolysis(Src) express HIF-1alpha protein under normoxia, which results in the expression of multiple HIF-1alpha target genes.HIF-1 alpha can then translocate to the nucleus and facilitate transcription of numerous target genes, the majority of which are involved in glycolysis and angiogenesis via heterodimerisation with the beta subunit (HIF-1 beta/ARNT)We previously observed that the c-Myc oncogenic transcription factor regulates lactate dehydrogenase A and induces lactate overproduction. We, therefore, sought to determine whether c-Myc controls other genes regulating glucose metabolismoverexpression of the c-Myc oncoprotein deregulates glycolysis through the activation of several components of the glucose metabolic pathwayHNF4alpha is critical for regulating glucose transport and glycolysis and in doing so is crucial for maintaining glucose homeostasis\"}, {\"question\": \"Which protein complexes contain mitofilin?\", \"ideal_answer\": [\"mitochondrial inter-membrane space bridging (MIB) complex \\nmitochondrial inner membrane organizing system (MINOS)\\nMitOS for mitochondrial organizing structure\"], \"exact_answer\": [[\"MIB\", \"mitochondrial inter-membrane space bridging complex\"], [\"MINOS\", \"mitochondrial inner membrane organizing system\"], [\"MitOS\"]], \"context\": \" and the mitochondrial structural proteins-components of mitochondrial inter-membrane space bridging (MIB) complex (Mitofilin, Sam50, and ChChd3), Mitofilin/MINOS protein complex  APOOL physically interacts with several subunits of the MINOS complex, namely Mitofilin, MINOS1, and SAMM50. Mitofilin/MINOS protein complex MINOS subunits (mitofilin, MINOS1, and CHCHD3)mitofilin, a core MINOS subunit,The mitochondrial inner membrane organizing system (MINOS) is a large protein complex required for maintaining inner membrane architecture. Mitochondria lacking mitofilin, the large core subunit of MINOS,The mitochondrial inner membrane contains a large protein complex crucial for membrane architecture, the mitochondrial inner membrane organizing system (MINOS). To study if outer membrane interactions and maintenance of cristae morphology are directly coupled, we generated mutant forms of mitofilin/Fcj1 (formation of crista junction protein 1), a core component of MINOS. The C-terminal domain of Fcj1 is required for formation of crista junctions and interacts with the TOB/SAM complex in mitochondria.Mitofilin/Fcj1, the central component of the MINOS complex,We provide evidence that the SAM components exist in a large protein complex together with the IMM proteins mitofilin and CHCHD3, which we term the mitochondrial intermembrane space bridging (MIB) complex.which we term MitOS for mitochondrial organizing structure, comprised of Fcj1/Mitofilin, a conserved inner membrane protein, and five additional components. Mitofilin is part of a large inner membrane complex, and we identify five partner proteins as constituents of the mitochondrial inner membrane organizing system (MINOS) that is required for keeping cristae membranes connected to the inner boundary membrane. Our findings indicate that mitofilin is a central component\\u00a0of MINOS\"}, {\"question\": \"Mutation of which gene and which chromosome cause Neurofibromatosis type I?\", \"ideal_answer\": [\"Neurofibromatosis Type I is an autosomal dominant condition associated with NF-1 gene mutation that is located in the long arm of chromosome 17 (17q11.2). In the majority (95%) of Neurofibromatosis Type I individuals, the mutation is found in the NF1 gene, while the remaining 5% of the patients have different types of genetic abnormalities.\\n\"], \"exact_answer\": [[\"NF1 gene\"], [\"chromosome 17\"]], \"context\": \"The von Recklinghausen neurofibromatosis (NF1) gene has been localized to the pericentromeric region of chromosome 17.Nine markers from the pericentromeric region of chromosome 17 were typed in 16 British and five South African families with neurofibromatosis type 1 (NF1).Inspection of recombinant events in families informative for several markers suggests that the NF1 gene is located between the markers EW301 (cen-p11.2) and EW206 (cen-q12) and possibly distal to pHHH202 (q11.2-q12).The gene for von Recklinghausen neurofibromatosis type 1 (NF1) has recently been mapped to the pericentromeric region of human chromosome 17.In addition to reporting, in accompanying papers, their individual analyses of mapping the neurofibromatosis type 1 (NF1) gene on chromosome 17, members of the International Consortium for NF1 Linkage contributed their data for our joint analysis to determine the exact sequence of flanking markers and to obtain precise estimates and confidence limits of the recombination fractions for the closest markers, in anticipation of clinical use.To better map the location of the von Recklinghausen neurofibromatosis (NF1) gene, we have characterized a somatic cell hybrid designated 7AE-11. This microcell-mediated, chromosome-transfer construct harbors a centromeric segment and a neo-marked segment from the distal long arm of human chromosome 17.The cosmids in the NF1 region will be an important resource for testing DNA blots of large-fragment restriction-enzyme digests from NF1 patient cell lines, to detect rearrangements in patients' DNA and to identify the 17;22 NF1 translocation breakpoint.The frequent LOH surrounding the NF1 locus and lack of neurofibromin expression in these tumors suggest that NF1 gene mutations may contribute to the development of adrenal gland neoplasms in patients with NF1.In the latter, the breakpoint in 17q occurred below the centromere and is at or in the region of the Neurofibromatosis Type 1 (NF1) gene.Neurofibromatosis type I (NF1) is an autosomal dominant condition associated with mutations in the long arm of chromosome 17, and characterised by neurofibromas, caf\\u00e9-au-lait spots and axillary freckling.In addition, the mother's and maternal grandmother's genetic analysis showed identical mutations in the neurofibromatosis I gene on the long arm of chromosome 17, confirming the diagnosis of NF1.Duplicon-mediated microdeletions around the NF1 gene are frequently associated with a severe form of neurofibromatosis type I in a subgroup of patients who show an earlier onset of cutaneous neurofibromas, dysmorphic facial features, and lower IQ values.Neurofibromatosis had been diagnosed in infancy, and genetic testing showed that the patient had a mutation in chromosome 17, consistent with neurofibromatosis type I.Neurofibromatosis Type 1 (NF1) is one of the most common inherited diseases in humans. It is caused by a mutation in the NF1 gene on chromosome 17, and is associated with numerous central and peripheral nervous system manifestations.In 95% of NF1 individuals, a mutation is found in the NF1 gene, and in 5% of the patients, the germline mutation consists of a microdeletion that includes the NF1 gene and several flanking genes.Herein, we describe a unique case of anaplastic de novo astroblastoma-sarcoma, in essence a variant of gliosarcoma, occurring in a 50-year-old female with documented NF1. Genetic study (fluorescence in situ hybridization) demonstrated no chromosomal losses or gains. Testing for abnormalities of chromosomes 7, 9, 10, 12, 17, 19 and 20, including the EGFR, p16, PTEN, MDM2 and NF1 gene regions, we found the tumor to exhibit a deletion of PTEN, monosomy 17 and gains of chromosomes 19 and 20q.Neurofibromatosis type I (NF1) is an autosomal dominant disorder affecting 1 in 3000 people. The NF1 gene is located on chromosome 17q11.2, spans 350 kb of genomic DNA, and contains 60 exons.A patient with recurrent-remittent multiple sclerosis associated with neurofibromatosis type I is described. The case is interesting for two reasons: 1) the difficulty of evaluating MRI findings, since both entities involve similar anomalies and 2) the relation between the two entities, according to evidence from recent genetic studies showing that the myelin protein gene associated to oligodendrocytes is part of an intron of the neurofibromatosis-1 gene of chromosome 17.Neurofibromatosis type I (NFI) is a common autosomal dominant disorder with an increased risk for developing benign and malignant tumors. The NFI gene has been cloned and maps to 17q11.2, and the gene product acts as a tumor suppressor gene.Direct sequencing of the NF1 gene in tumor DNA identified the presence of an inactivating NF1 somatic mutation in 41% (25/61) of analyzed sporadic tumors, associated with loss of the wild-type allele in 84% (21/25) of cases.A novel mutation c.4821delA was identified in NF1 gene, which predicted truncation of neurofibromin (p.Leu1607fs).She had a genotype of heterozygous c.6709C>T mutation of NF1 gene.RESULTS: The mutational analysis of the NF1 gene revealed a novel frameshift insertion mutation in exon 4c (c.654 ins A) in all affected family members.Using lymphoblastoid, fibroblast, and buccal cell samples collected from both twins and from other family members in three generations, we discovered a pathogenic nonsense mutation in exon 40 of the NF1 gene.We also found a novel non-synonymous change in exon 16 of the NF1 gene that was transmitted from the unaffected mother to both twins and co-segregated with the pathogenic mutation in the ensuing generation.The disease gene of the Chinese NF1 family was linked to NF1 locus, and a nonsense mutation, G1336X in the NF1 gene was identified.The present study demonstrated that G1336X mutation in the NF1 gene cause Neurofibromatosis type I in the family.\"}, {\"question\": \"Which genes does thyroid hormone receptor alpha1 regulate in the heart?\", \"ideal_answer\": [\"\\u03b2-myosin heavy chain, alpha-myosin heavy chain, SR(Ca)ATPase, phospholamban, nucleotide-gated potassium channel 2, KCNE1, HCN2, HCN4, KCND2, KCND3, KCNA4\"], \"exact_answer\": [[\"\\u03b2-myosin heavy chain\"], [\"alpha-myosin heavy chain\"], [\"SR(Ca)ATPase\"], [\"phospholamban\"], [\"nucleotide-gated potassium channel 2\"], [\"KCNE1\"], [\"HCN2\"], [\"HCN4\"], [\"KCND2\"], [\"KCND3\"], [\"KCNA4\"]], \"context\": \"A group of mice was treated with debutyl-dronedarone (DBD), a selective TR\\u03b11 inhibitor (AMI-DBD)Furthermore, AMI resulted in \\u03b2-myosin heavy chain overexpression and reduction in the ratio of SR(Ca)ATPase to phospholamban (PLB). The latter further declined in AMI-DBD mainly due to increased expression of PLBWe identified that mice expressing the mutant TRalpha1R384C (TRalpha1+m mice) displayed a mild bradycardia, which becomes more pronounced during night activity or on stress and is accompanied by a reduced expression of nucleotide-gated potassium channel 2 mRNA in the heartKCNE1 was four to 10-fold overexpressed in mice deficient in TRalpha1The bradycardia and prolonged QT(end) time in hypothyroid states can be explained by altered K(+) channel function due to decreased TRalpha1-dependent repression of KCNE1 expressionConsistently, TR alpha 1-infected myocytes exhibited larger I(f) current densities along with increased HCN2 and HCN4 mRNA and protein levels.T3 stimulation enhanced KCND2 and KCND3 expression and decreased KCNA4 transcription, while KCNIP2 remained unaffected.TRalpha1 increased I(to), while TRbeta1 significantly reduced I(to) in size, which was associated with TRalpha1-mediated increase and TRbeta1-mediated reduction of KCND2/3 transcription.While the TRalpha1 aporeceptor enhanced KCND3 transcription, the TRbeta1 aporeceptor suppressed KCND3 expression, with both effects exhibiting ligand-dependent amplification upon T3 stimulation.PE, in the absence of T3, resulted in 5.0 fold increase in TRalpha1 expression in nucleus and 2.0 fold decrease in TRalpha1 expression in cytosol, P<0.05. As a result, a fetal pattern of myosin isoform expression with marked expression of beta-MHC was observed in PE treated vs the untreated cells, P<0.05.PKC alpha overexpression in cardiomyocytes caused marked repression of triiodothyronine (T3)-responsive genes, alpha-myosin heavy chain, and the sarcoplasmic reticulum calcium-activated adenosinetriphosphatase SERCA2. Treatment with T3 for 4 h resulted in significant reductions of PKC alpha in nuclear and cytosolic compartments, and decreased TR alpha1 mRNA and protein, with normalization of phenotype.The mechanism of TH and TRalpha1-specific hypertrophy is novel for a nuclear hormone receptor and involves the transforming growth factor beta-activated kinase (TAK1) and p38Excised TRalpha(1)+/m hearts showed an increased expression of phospholamban (PLB)administration of dronedarone, an amiodarone-like compound that has been shown to preferentially antagonize thyroid hormone binding to thyroid hormone receptor alpha1 (TRalpha1),lpha-Myosin heavy chain (alpha-MHC) decreased in DRON while beta-myosin heavy chain (beta-MHC) and sarcoplasmic reticulum Ca2+ adenosine triphosphatase (ATPase) expression (SERCA) was similar to CONT.The activity of TR DNA binding to the transcriptional regulatory region in the alpha-MHC and SR Ca(2+)-ATPase genes and the mRNA and protein expression of alpha-MHC and SR Ca(2+)-ATPase in the heart and plasma 3,3'-triiodothyronine and thyroxine levels were altered in association with changes in the myocardial TR protein levels.addition, TR cotransfection and treatment with the TRbeta1-selective agonist GC-1 suggested different functional coupling of the TR isoforms, TRbeta1 to transcription of beta-MyHC, SERCA, and TRbeta1, and TRalpha1 to alpha-MyHC transcription and increased myocyte sizeOBJECTIVE: The reduced heart rate and prolonged QT(end) duration in mice deficient in thyroid hormone receptor (TR) alpha1 may involve aberrant expression of the K(+) channel alpha-subunit KCNQ1 and its regulatory beta-subunit KCNE1.\"}, {\"question\": \"Which computational frameworks are available for predicting enhancers?\", \"ideal_answer\": [\"DEEP integrates three components with diverse characteristics that streamline the analysis of enhancer's properties in a great variety of cellular conditions. DEEP uses features derived from histone modification marks or attributes coming from sequence characteristics. Experimental results indicate that DEEP performs better than four state-of-the-art methods on the ENCODE data. The PRISM (predicting regulatory information from single motifs) approach obtains 2543 TF function predictions in a large variety of contexts, at a false discovery rate of 16%. The predictions are highly enriched for validated TF roles, and 45 of 67 (67%) tested binding site regions in five different contexts act as enhancers in functionally matched cells.\"], \"exact_answer\": [[\"DEEP\"], [\"PRISM\"]], \"context\": \"In this study we developed DEEP, a novel ensemble prediction framework. DEEP integrates three components with diverse characteristics that streamline the analysis of enhancer's properties in a great variety of cellular conditions. In our method we train many individual classification models that we combine to classify DNA regions as enhancers or non-enhancers. DEEP uses features derived from histone modification marks or attributes coming from sequence characteristics. Experimental results indicate that DEEP performs better than four state-of-the-art methods on the ENCODE data. We report the first computational enhancer prediction results on FANTOM5 data where DEEP achieves 90.2% accuracy and 90% geometric mean (GM) of specificity and sensitivity across 36 different tissues. We further present results derived using in vivo-derived enhancer data from VISTA database. DEEP-VISTA, when tested on an independent test set, achieved GM of 80.1% and accuracy of 89.64%.Our novel PRISM (predicting regulatory information from single motifs) approach obtains 2543 TF function predictions in a large variety of contexts, at a false discovery rate of 16%. The predictions are highly enriched for validated TF roles, and 45 of 67 (67%) tested binding site regions in five different contexts act as enhancers in functionally matched cells.DEEP: a general computational framework for predicting enhancersWe report the first computational enhancer prediction results on FANTOM5 data where DEEP achieves 90.2% accuracy and 90% geometric mean (GM) of specificity and sensitivity across 36 different tissuesDEEP: a general computational framework for predicting enhancers.\"}, {\"question\": \"Which proteins in the cerebro-spinal fluid can be used for early diagnosis of Alzheimer's disease?\", \"ideal_answer\": [\"CSF is a clear fluid that bathes and cushions the brain and spinal cord. Adults have about 1 pint of CSF, which physicians can sample through a minimally invasive procedure called a lumbar puncture, or spinal tap. Research suggests that Alzheimer's disease in its earliest stages may cause changes in CSF levels of tau and beta-amyloid, two proteins that form abnormal brain deposits strongly linked to the disease.\"], \"exact_answer\": [[\"phosphorylated tau\"], [\"beta-amyloid\"]], \"context\": \"Alzheimer's disease (AD) patients have reduced amyloid (A\\u03b2 1-42) and elevated total tau (t-tau) and phosphorylated tau (p-tau 181p) in the cerebro-spinal fluid (CSF), suggesting that these same measures could be used to detect early AD pathology in healthy elderly (CN) and mild cognitive impairment (MCI)although high CSF tau and low CSF A\\u03b242 are predictive of Alzheimer's disease, the criterion \\\"positive CSF tau/A\\u03b242 ratio\\\" is not well defined. The qualification of biomarkers in the pre-dementia stage of Alzheimer's disease will allow better inclusion criteria of patients in pre-dementia trials in which the benefit/risk is higher for treatment with these novel compoundsinvestigate the dynamics of four of the most validated biomarkers for Alzheimer's disease (AD), cerebro-spinal fluid (CSF) Abeta 1-42, tau, hippocampal volume, and FDG-PET, in patients at different stage of ADAlzheimer's disease (AD) may be facilitated by cerebro-spinal fluid (CSF) biomarkers in combination with imaging and clinical assessments. By determining the concentration of beta amyloid fragments, total tau (t-tau) and phospho-tau (p-tau), it is possible to detect the conversion of mild cognitive impairment (MCI) to AD or distinguish AD vs. pseudo-dementia. However, these markers are poorly sensitive to the progressive disease stages\"}, {\"question\": \"Which genes are implicated in short QT syndrome?\", \"ideal_answer\": [\"The genes that are implicated in short QT syndrome are KCNJ2, KCNH2, CACNA2D1 and KCNQ1.\"], \"exact_answer\": [[\"KCNJ2\", \"inward rectifier potassium channel gene\"], [\"HERG\", \"KCNH2\"], [\"CACNA2D1\"], [\"KCNQ1\"], [\"CACNB2b\"]], \"context\": \"A novel gain-of-function KCNJ2 mutation associated with short-QT syndrome impairs inward rectification of Kir2.1 currents.In this study, we identified a novel KCNJ2 gain-of-function mutation, M301K, associated with SQTS. A mutation in genes related to SQTS was found in 23% of the probands; most of them had a gain of function mutation in HERG (SQTS1).We present, for the first time, a novel loss-of-function mutation coding for an L-type calcium channel subunit. however, a new variant at a heterozygous state was identified in the CACNA2D1 gene (nucleotide c.2264G > C; amino acid p.Ser755Thr), coding for the Ca(v)\\u03b1(2)\\u03b4-1 subunit of the L-type calcium channel.In the present study, we report the first pathogenic mutation in the CACNA2D1 gene in humans, which causes a new variant of SQTS.A novel mutation in the KCNH2 gene associated with short QT syndrome.The altered function of T618I-HERG channels suggests that this mutation in the KCNH2 gene is responsible for the SQTS phenotype in this family. Gain of function mutations in three genes encoding K(+) channels have been identified, explaining the abbreviated repolarization seen in this condition: KCNH2 for I(kr) (SQT1), KCNQ1 for I(ks) (SQT2) and KCNJ2 for I(k1) (SQT3). To evaluate the possible diagnosis of SQTS, DNA sequencing of genes known to be associated with SQTS was performed and identified a novel mutation in the KCNH2 gene.The mutation of genes (KCNH2, KCNQ1, and KCNJ2) encoding for cardiac potassium channels plays a central role in SQTS. To date, different mutations in genes encoding for cardiac ion channels (KCNH2, KCNQ1, and KCNJ2) have been identified to cause the short QT syndrome. Mutations in 3 different genes KCNQ1, KCNH2, and KCNJ2, all encoding cardiac ionic potassium channels have been identified in affected patients. Genetic analysis has, to date, identified three distinct forms of the condition, involving gain-of-function mutations to three different cardiac potassium channel genes: KCNH2 (SQT1), KCNQ1 (SQT2) and KCNJ2 (SQT3). Three different gain-of-function mutations in genes encoding for cardiac potassium channels (KCNH2, KCNQ1, and KCNJ2) have been identified up to now to cause short QT syndrome. To date, three different mutations in genes encoding cardiac ion channels (KCNH2, KCNQ1 and KCNJ2) have been identified as causing short QT syndrome.\"}, {\"question\": \"What is SCENAR therapy used for?\", \"ideal_answer\": [\"all patients experienced substantial relief of pain from the first treatment. an electronic biofeedback device (scenar) may be successfully utilized in the management of post-herpetic neuralgia. scenar) as effective in the treatment of neurogenic dysfunction of the bladder in children with nocturnal enuresis. post-herpetic neuralgia using a bioelectronical device (scenar). addition of scenar therapy to the complex conventional pharmacotherapy fastened ulcer healing, increased the effectiveness of helicobacter pylori eradication, and improved the condition of the gastroduodenal mucosa. scenar therapy to patients with localized suppurative peritonitis in the postoperative period. a new technique of low-frequency modulated electric current therapy, scenar therapy, was used in treatment of 103 patients with duodenal ulcer (du). \", \"localized suppurative peritonitis in the postoperative period\\nmanagement of post-herpetic neuralgia\\ntreatment of 103 patients with duodenal ulcer\\ntreatment of neurogenic dysfunction of the bladder in children with nocturnal enuresis\"], \"exact_answer\": [[\"peritonitis postoperative\"], [\"pain in postherpetic neuralgia\"], [\"duodenal ulcer\"], [\"nocturnal enuresis\"]], \"context\": \"SCENAR therapy to patients with localized suppurative peritonitis in the postoperative periodpost-herpetic neuralgia using a bioelectronical device (SCENAR)All patients experienced substantial relief of pain from the first treatment.An electronic biofeedback device (SCENAR) may be successfully utilized in the management of post-herpetic neuralgia.A new technique of low-frequency modulated electric current therapy, SCENAR therapy, was used in treatment of 103 patients with duodenal ulcer (DU).Addition of SCENAR therapy to the complex conventional pharmacotherapy fastened ulcer healing, increased the effectiveness of Helicobacter pylori eradication, and improved the condition of the gastroduodenal mucosa.SCENAR) as effective in the treatment of neurogenic dysfunction of the bladder in children with nocturnal enuresis. \"}, {\"question\": \"List symptoms of Gradenigo's syndrome.\", \"ideal_answer\": [\"Gradenigo's syndrome is a rare but life threatening complication of acute otitis media, which includes a classic triad of otitis media, deep facial pain and ipsilateral abducens nerve paralysis.\"], \"exact_answer\": [[\"otitis media\"], [\"deep facial pain\"], [\"abducens nerve paralysis\"]], \"context\": \"INTRODUCTION: Gradenigo's syndrome is nowadays a rare condition characterized by a triad of otorrhea, facial pain with trigeminal nerve involvement and abducens nerve palsy.Gradenigo's syndrome is a rare but life threatening complication of acute otitis media (AOM), which includes a classic triad of otitis media, deep facial pain and ipsilateral abducens nerve paralysis. We report a case of typical Gradenigo's syndrome, including left abducens nerve palsy, left facial pain and paresthesia in V1 and V2 distribution of trigeminal nerve caused by solitary osseous plasmacytoma of the left petrous apex.BACKGROUND: Gradenigo's syndrome is a rare disease, which is characterized by the triad of the following conditions: suppurative otitis media, pain in the distribution of the first and the second division of trigeminal nerve, and abducens nerve palsy.RESULTS: It was after Gradenigo that the well-known syndrome consisting of diplopia and facial pain due to a middle ear infection was named.Gradenigo's syndrome, the triad of suppurative otitis media, abducens nerve palsy and pain in the ophthalmic division of the trigeminal nerve, remains a rare complication of otitis media.In 1904, Giuseppe Gradenigo published his case series on the triad of ipsilateral abducens nerve palsy, facial pain in the trigeminal nerve distribution, and suppurative otitis media, which would subsequently be referred to as Gradenigo syndrome\"}, {\"question\": \"Which variables are included in the SPAN-100 score?\", \"ideal_answer\": [\"SPAN-100 score includes patient's age and NIH Stroke Scale score. SPAN-100 is used for prognostication of stroke patients.\"], \"exact_answer\": [[\"Age\"], [\"NIH Stroke Scale score\"]], \"context\": \"Furthermore, MR-proADM levels significantly improved reclassification of patients in the prediction of outcome by the Stroke Prognostication using Age and NIHSS-100 (SPAN-100; NRI\\u200a=\\u200a0.175; p\\u200a=\\u200a0.04). Stroke Prognostication using Age and NIH Stroke Scale: SPAN-100.METHODS: We created the Stroke Prognostication using Age and NIH Stroke Scale (SPAN) index by combining age in years plus NIH Stroke Scale (NIHSS) \\u2265100. The SPAN-100 index is considered positive if the sum of the age and the NIH Stroke Scale (a 15-item neurological examination scale with scores ranging from 0 to 42, with higher scores indicating more severe strokes) score is greater than or equal to 100The Stroke Prognostication using Age and the NIH Stroke Scale index, created by combining age in years plus a National Institutes of Health (NIH) Stroke Scale score of 100 or higher (and hereafter referred to as the SPAN-100 index), is a simple risk score for estimating clinical outcomes for patients with acute ischemic stroke (AIS)\"}, {\"question\": \"Which genes does thyroid hormone receptor alpha1 regulate in the liver?\", \"ideal_answer\": [\"phosphoenolpyruvate-carboxykinase\\\"//\\n\\\"pyruvate kinase\\\"//\\n\\\"D1\\\", \\\"deiodinase 1\\\"//\"], \"exact_answer\": [[\"phosphoenolpyruvate-carboxykinase\"], [\"pyruvate kinase\"], [\"D1\", \"deiodinase 1\"]], \"context\": \"Mice expressing the mutant thyroid hormone receptor TRalpha1R384C, which has a 10-fold reduced affinity to the ligand T(3), exhibit hypermetabolism due to an overactivation of the sympathetic nervous system. To define the consequences in the liver, we analyzed hepatic metabolism and the regulation of liver genes in the mutant mice. Our results showed that hepatic phosphoenolpyruvate-carboxykinase was up-regulated and pyruvate kinase mRNA down-regulated, contrary to what observed after T(3) treatmentRemarkably, there was an obligatory requirement for a TR, whether TRbeta or TRalpha1, for any detectable D1 expression in liver.Liver and kidney D1 mRNA and activity levels were reduced in TRbeta(-/-) but not TRalpha1(-/-) mice\"}, {\"question\": \"By which methods can we evaluate the reliability of a phylogenetic tree?\", \"ideal_answer\": [\"The methods for assessing the robustness/reliability of the topology of the inferred phylogenetic trees are: the widely used bootstrap method and the jackknife method.\", \"In contrast to other similar software, the program FreeTree (available at http://www.natur.cuni.cz/~flegr/programs/freetree or http://ijs.sgmjournals.org/content/vol51/issue3/) can also assess the robustness of the tree topology by bootstrap, jackknife or operational taxonomic unit-jackknife analysis. (PMID: 11411692)\"], \"exact_answer\": [[\"bootstrap\"], [\"jackknife\"]], \"context\": \"For robustness evaluation, we apply bootstrap and jackknife techniques to input data.An alignment confidence score capturing robustness to guide tree uncertainty.We build on the widely used bootstrap method for perturbing the phylogenetic tree.SuperTRI: A new approach based on branch support analyses of multiple independent data sets for assessing reliability of phylogenetic inferences.To overcome these limitations, we propose a new approach, called SuperTRI, which is based on the branch support analyses of the independent data sets, and where the reliability of the nodes is assessed using three measures: the supertree Bootstrap percentage and two other values calculated from the separate analyses: the mean branch support (mean Bootstrap percentage or mean posterior probability) and the reproducibility index.In contrast to other similar software, the program FreeTree (available at http://www.natur.cuni.cz/~flegr/programs/freetree or http://ijs.sgmjournals.org/content/vol51/issue3/) can also assess the robustness of the tree topology by bootstrap, jackknife or operational taxonomic unit-jackknife analysis.Unlike any previous tool for inferring phylogenies from rearrangement data, TIBA uses novel methods of robustness estimation to provide support values for the edges in the inferred tree.Fast and accurate phylogenetic reconstruction from high-resolution whole-genome data and a novel robustness estimator.Moreover, whereas phylogenetic analyses from sequence data are deemed incomplete unless bootstrapping scores (a measure of confidence) are given for each tree edge, no equivalent to bootstrapping exists for rearrangement-based phylogenetic analysis.We also describe a novel approach to estimate the robustness of results-an equivalent to the bootstrapping analysis used in sequence-based phylogenetic reconstruction.The standard deviation also suggests the reliability level of the branch order.Assessment of the reliability of a given phylogenetic hypothesis is an important step in phylogenetic analysis.The reliability of the phylogenetic trees was probed with the bootstrapping of 100 replicates of the original sequence alignments.\"}, {\"question\": \"List FDA approved treatments for androgenetic allopecia\", \"ideal_answer\": [\"Recently, in addition to the two currently approved U.S. Food and Drug Administration (FDA) medications (minoxidil and finasteride), a novel device was FDA-approved for the treatment of hair loss, the laser hair comb.\"], \"exact_answer\": [[\"Minoxidil\"], [\"Finasteride\"], [\"Laser hair comb\"]], \"context\": \"Topical minoxidil 2%-5% 1 mL twice daily or finasteride 1 mg daily are recommended as first line treatments, followed by the use of Food and Drug Administration-cleared HairMax LaserComb\\u00ae in patients who do not respond to first line modalities.Recently, in addition to the two currently approved U.S. Food and Drug Administration (FDA) medications (minoxidil and finasteride), a novel device was FDA-approved for the treatment of hair loss, the laser hair comb.In addition, low-level light therapy (LLLT) has recently been approved by the United States Food and Drug Administration (FDA) for the treatment of hair loss. Currently there are two U.S. Food and Drug Administration (FDA)-approved agents that promote hair regrowth: over-the-counter topical minoxidil solution for men and women and prescription oral finasteride tablets for men. Topical monoxidil and oral finasteride are commonly in use and have FDA approval for the treatment of male androgenetic alopecia; dutasteride, a type I and II 5-alpha-reductase inhibitor, is on hold in Phase III trials.The only FDA-approved dermatological indication of finasteride is androgenetic alopecia.Finasteride (Propecia) was approved by the FDA in 1998 for treating men with androgenetic alopecia.\"}, {\"question\": \"Which receptors are targeted by suvorexant?\", \"ideal_answer\": [\"Suvorexant is a potent, selective, and orally bioavailable antagonist of orexin 1 receptor and orexin 2 receptor currently under clinical investigation as a novel therapy for insomnia.\"], \"exact_answer\": [[\"orexin 1\"], [\"orexin 2\"]], \"context\": \"Dual orexin receptor (OXR) antagonists (DORAs) such as almorexant, 1 (SB-649868), or suvorexant have shown promise for the treatment of insomnias and sleep disorders in several recent clinical trials in volunteers and primary insomnia patients. Several such compounds have entered into clinical development, including the dual orexin receptor antagonists, suvorexant and almorexant. Suvorexant (MK-4305) is an orexin receptor antagonist being developed for the treatment of insomnia. Indeed, phase III clinical trials were completed last year of suvorexant, a non-selective (dual) antagonist for orexin receptors, for the treatment of primary insomnia, and demonstrate promising results.METHODS: We evaluated suvorexant, an orexin receptor antagonist, for treating patients with primary insomnia in a randomized, double-blind, placebo-controlled, 2-period (4 weeks per period) crossover polysomnography study.Suvorexant is a dual orexin antagonist currently in Phase III clinical trials for the modulation of sleep and is being developed by Merck.Merck's suvorexant (MK-4305) is the first compound of the so-called dual orexin receptor antagonist (DORA) class expected to be submitted for FDA approval, with a new drug application anticipated in 2012. A concise, enantioselective synthesis of the potent dual orexin inhibitor suvorexant (1) is reported. Reaction development and mechanistic study of a ruthenium catalyzed intramolecular asymmetric reductive amination en route to the dual Orexin inhibitor Suvorexant (MK-4305).The first example of an intramolecular asymmetric reductive amination of a dialkyl ketone with an aliphatic amine has been developed for the synthesis of Suvorexant (MK-4305), a potent dual Orexin antagonist under development for the treatment of sleep disorders. Suvorexant (MK-4305) is a potent, selective, and orally bioavailable antagonist of OX(1)R and OX(2)R currently under clinical investigation as a novel therapy for insomnia.Suvorexant is a pharmacologically novel dual antagonist of orexin receptors OX1R and OX2R, which has an effect that promotes sleep by reducing arousal and wakefulness. Indeed, phase III clinical trials were completed last year of suvorexant, a non-selective (dual) antagonist for orexin receptors, for the treatment of primary insomnia, and demonstrate promising results. In order to understand the differential contribution of both receptors in regulating sleep/wakefulness states we compared the pharmacological effects of a newly developed OX2R antagonist (2-SORA), Compound 1 m (C1 m), with those of a dual orexin receptor antagonist (DORA), suvorexant, in C57BL/6J mice.Promotion of sleep by suvorexant-a novel dual orexin receptor antagonist.\"}, {\"question\": \"A common problem in proteomics is the contamination of samples with exogenous proteins (often from other species). These proteins can be found in specific databases. List some contaminants.\", \"ideal_answer\": [\"Some common contaminants in proteomics are proteases (used for the digestion of the proteins), keratins (usually from the skin),  proteins originated from the serum of the culture media and antibodies if used in the experiment.\"], \"exact_answer\": [[\"Proteases\"], [\"Keratins\"], [\"Bovine serum proteins\"], [\"Antibodies\"]], \"context\": \"protease autolysis or keratin contaminants.Most of the interfering masses originate from trypsin autolysis and human keratins.5-15% FBS. Contamination by bovine proteins is difficult to avoid because of adherence to the plastic vessel and the cultured cells.The main disadvantage of traditional IP and co-IP is that the conditions used to elute the precipitated antigen also release the antibody thus contaminating the antigenMaConDa: a publicly accessible mass spectrometry contaminants database.\"}, {\"question\": \"List features of the DEND syndrome.\", \"ideal_answer\": [\"Clinical features of the DEND syndrome include developmental delay, epilepsy and neonatal diabetes.\"], \"exact_answer\": [[\"developmental delay\"], [\"epilepsy\"], [\"neonatal diabetes\"]], \"context\": \"Some mutations in this gene, including p.Q52R, are associated with the developmental delay, epilepsy, neonatal diabetes (DEND) syndrome.Developmental delay was noted on follow-up which raised the possibility of intermediate DEND syndrome.We have verified a lack of clinical response for both glycemic control and neurological features in an infant with permanent neonatal diabetes mellitus and DEND syndrome due to a V59A mutation in the KCNJ11 gene. They also may cause neurologic symptoms such as mental retardation and motor problems (iDEND syndrome) and epilepsy (DEND syndrome).  Wide phenotype variability is associated with single ABCC8 mutations, ranging from transient or permanent neonatal diabetes (ND) with or without developmental delay (DEND syndrome) to very mild phenotypes. Specific uncommon mutations KCNJ11give rise to a syndrome defined as developmental delay, epilepsy, and neonatal diabetes (DEND), or - more frequently - to a milder sub-type lacking epilepsy, denoted as intermediate-DEND (iDEND).Approximately 20% have associated neurologic features: the most severe form, which includes epilepsy and developmental delay, is called developmental delay, epilepsy, and neonatal diabetes (DEND) syndrome and the milder form, with less severe developmental delay and without epilepsy, is designated intermediate DEND syndrome.  The patient with the V59M mutation successfully switched from insulin injections to oral glibenclamide; 2 years of follow-up revealed that the patient had intermediate developmental delay, epilepsy and neonatal diabetes (DEND) syndrome. Approximately 20% have associated neurologic features: the most severe form, which includes epilepsy and developmental delay, is called developmental delay, epilepsy, and neonatal diabetes (DEND) syndrome and the milder form, with less severe developmental delay and without epilepsy, is designated intermediate DEND syndrome.We have verified a lack of clinical response for both glycemic control and neurological features in an infant with permanent neonatal diabetes mellitus and DEND syndrome due to a V59A mutation in the KCNJ11 gene.There is a striking genotype-phenotype relationship with specific Kir6.2 mutations being associated with transient neonatal diabetes, permanent neonatal diabetes alone, and a novel syndrome characterized by developmental delay, epilepsy, and neonatal diabetes (DEND) syndrome.In a few patients, KCNJ11 mutations cause a triad of developmental delay, epilepsy, and neonatal diabetes (DEND syndrome).Glibenclamide unresponsiveness in a Brazilian child with permanent neonatal diabetes mellitus and DEND syndrome due to a C166Y mutation in KCNJ11 (Kir6.2) gene.It is proposed that some patients with KCNJ11 mutations have neurological features that are part of a discrete neurological syndrome termed developmental Delay, Epilepsy and Neonatal Diabetes (DEND), but there are also neurological consequences of chronic or acute diabetes. Medical and developmental impact of transition from subcutaneous insulin to oral glyburide in a 15-yr-old boy with neonatal diabetes mellitus and intermediate DEND syndrome: extending the age of KCNJ11 mutation testing in neonatal DM.In a few patients, KCNJ11 mutations cause a triad of developmental delay, epilepsy, and neonatal diabetes (DEND syndrome).We have verified a lack of clinical response for both glycemic control and neurological features in an infant with permanent neonatal diabetes mellitus and DEND syndrome due to a V59A mutation in the KCNJ11 gene.It is proposed that some patients with KCNJ11 mutations have neurological features that are part of a discrete neurological syndrome termed developmental Delay, Epilepsy and Neonatal Diabetes (DEND), but there are also neurological consequences of chronic or acute diabetes.We report a case of developmental delay, epilepsy, and neonatal diabetes (DEND) syndrome that shows neurologic improvement with sulfonylurea therapy.Some patients have DEND syndrome (developmental delay, epilepsy and neonatal diabetes) with the neurological features arising from mutated KATP channels in muscle, nerve and brain.Approximately 20% have associated neurologic features: the most severe form, which includes epilepsy and developmental delay, is called developmental delay, epilepsy, and neonatal diabetes (DEND) syndrome and the milder form, with less severe developmental delay and without epilepsy, is designated intermediate DEND syndrome.Intermediate developmental delay, epilepsy and neonatal diabetes mellitus (DEND) syndrome as a result of a 59V>M Kir6.2 mutation.Five patients with developmental delay, epilepsy and neonatal diabetes (DEND) syndrome had unique mutations not associated with other phenotypes.Approximately 20% have associated neurologic features: the most severe form, which includes epilepsy and developmental delay, is called developmental delay, epilepsy, and neonatal diabetes (DEND) syndrome and the milder form, with less severe developmental delay and without epilepsy, is designated intermediate DEND syndrome.We have verified a lack of clinical response for both glycemic control and neurological features in an infant with permanent neonatal diabetes mellitus and DEND syndrome due to a V59A mutation in the KCNJ11 gene. Thus, our case reinforces that most cases with DEND syndrome are insensitive to SU.Some patients have DEND syndrome (developmental delay, epilepsy and neonatal diabetes) with the neurological features arising from mutated KATP channels in muscle, nerve and brain.We have verified a lack of clinical response for both glycemic control and neurological features in an infant with permanent neonatal diabetes mellitus and DEND syndrome due to a V59A mutation in the KCNJ11 gene.It is proposed that some patients with KCNJ11 mutations have neurological features that are part of a discrete neurological syndrome termed developmental Delay, Epilepsy and Neonatal Diabetes (DEND), but there are also neurological consequences of chronic or acute diabetes.\"}, {\"question\": \"Which genes are known to be involved in Diamond-Blackfan anemia?\", \"ideal_answer\": [\"Diamond-Blackfan anemia (DBA) is an inherited red blood cell aplasia that usually presents during the first year of life. The disease has been associated with point mutations and large deletions in ten ribosomal protein (RP) genes RPS19, RPS24, RPS17, RPL35A, RPL5, RPL11, RPS7, RPS10, RPS26, and RPL26, as well as in GATA1, in about 60-65 % of patients.\"], \"exact_answer\": [[\"RPS19\"], [\"RPS24\"], [\"RPS17\"], [\"RPL35A\"], [\"RPL5\"], [\"RPL11\"], [\"RPS7\"], [\"RPS10\"], [\"RPS26\"], [\"RPL26\"], [\"GATA1\"]], \"context\": \"Diamond-Blackfan anemia (DBA) is an inherited red blood cell aplasia that usually presents during the first year of life. The main features of the disease are normochromic and macrocytic anemia, reticulocytopenia, and nearly absent erythroid progenitors in the bone marrow. The patients also present with growth retardation and craniofacial, upper limb, heart and urinary system congenital malformations in ~30-50 % of cases. The disease has been associated with point mutations and large deletions in ten ribosomal protein (RP) genes RPS19, RPS24, RPS17, RPL35A, RPL5, RPL11, RPS7, RPS10, RPS26, and RPL26 and GATA1 in about 60-65 % of patientsThis study was aimed to explore the mutations of ribosomal protein (RP) genes in patients with Diamond Blackfan anemia (DBA). Twenty-one cases of DBA admitted in our hospital from Dec 2008 to Aug 2012 were screened by PCR for mutations in the nine known genes associated with DBA: RPS19, RPS24, RPS17, RPL5, RPL11, RPS7, RPL35a, RPS10 and RPS26The results found that 8 patients (38.1%) with DBA had mutations in the genes coding for ribosomal protein, in which RPS19 mutation was identified in 3 patients, RPS24, RPS7, RPL5, RPL11 and RPL35A mutations were identified respectively in 1 of the patientDiamond-Blackfan anemia (DBA) is a hypoplastic anemia characterized by impaired production of red blood cells, with approximately half of all cases attributed to ribosomal protein gene mutations. We performed exome sequencing on two siblings who had no known pathogenic mutations for DBA and identified a mutation in the gene encoding the hematopoietic transcription factor GATA1As a proof of concept, we designed a multiplex ligation-dependent probe amplification assay targeted to screen the six genes that are most frequently mutated in Diamond-Blackfan anemia patients: RPS17, RPS19, RPS26, RPL5, RPL11, and RPL35A. Using this assay we showed that deletions represent approximately 20% of all mutationsDESIGN AND METHODS: We screened 49 Japanese patients with Diamond-Blackfan anemia (45 probands) for mutations in the six known genes associated with Diamond-Blackfan anemia: RPS19, RPS24, RPS17, RPL5, RPL11, and RPL35A.Defects in the RPS19 gene, encoding the ribosomal protein S19, are the main known cause of Diamond-Blackfan anemia and account for more than 25% of cases.OBJECTIVE: Diamond-Blackfan anemia (DBA) is a rare congenital hypoplastic anemia caused by mutations in ribosomal protein (RP) genes.Fifty percent of Diamond-Blackfan anemia (DBA) patients possess mutations in genes coding for ribosomal proteins (RPs).Mutations in the gene coding for the ribosomal protein RPS19 have been identified in 25% of patients with DBA, with resulting impairment of 18S rRNA processing and 40S ribosomal subunit formationWe screened 49 Japanese patients with Diamond-Blackfan anemia (45 probands) for mutations in the six known genes associated with Diamond-Blackfan anemia: RPS19, RPS24, RPS17, RPL5, RPL11, and RPL35ADefects in the RPS19 gene, encoding the ribosomal protein S19, are the main known cause of Diamond-Blackfan anemia and account for more than 25% of casesThe first ribosomal gene involved in DBA, ribosomal protein (RP) gene S19 (RPS19 gene), was identified in 1999These data identify RPL15 as a new gene involved in DBA and further support the presence of large deletions in RP genes in DBA patients.\"}, {\"question\": \"What are the main clinical characteristics of Pendred syndrome?\", \"ideal_answer\": [\"Pendred syndrome is an autosomal recessive disorder characterized by congenital sensorineural deafness, goiter, and impaired iodide organification.\"], \"exact_answer\": [[\"congenital sensorineural deafness\"], [\"goiter\"], [\"impaired iodide organification\"]], \"context\": \"Obligatory features are profound deafness in childhood and defective organic binding of iodine in the thyroid gland. Therefore, goiter is a common symptom. Hypoplasia of the cochlea is another feature.If a child has progressive sensorineural deafness and a widened vestibular aqueduct, it is important to consider a diagnosis of Pendred syndrome. Inherited as an autosomal recessive trait, Pendred syndrome is a disease that shows congenital sensorineural hearing loss and goiter, with a positive finding in the perchlorate discharge test. Pendred syndrome results from various mutations in the PDS/SLC26A4 gene that cause production of an abnormal pendrin proteinPendred syndrome is an autosomal recessive disorder characterized by congenital sensorineural deafness, goiter, and impaired iodide organification. It is caused by mutations in the PDS gene. Malformations of the inner ear, specifically enlargement of the vestibular aqueduct, are common in Pendred syndrome and mutations in the PDS (Pendred Syndrome) gene have been recorded in patients presenting with deafness and vestibular aqueduct dilatation only, without other features of Pendred syndrome.Aside from the branchio-otorenal syndrome, Pendred syndrome is the only other known genetic disorder with a widened vestibular aqueduct.The clinical diagnosis of Pendred syndrome was based on the laboratory and sonographic signs of thyroid dyshormonogenesis in association with sensorineural hearing loss.Although the textbook view of Pendred syndrome is that of an autosomal recessive condition characterized by deafness and goitre, it is increasingly clear that not all such patients present this classical clinical picture.Progressive sensorineural hearing loss and widened vestibular aqueducts are characteristic features of Pendred's syndrome, which provides the opportunity to diagnose Pendred's syndrome clinically in the first few years of life, as has recently been suggested in a case report (Cremers et al., Progressive sensorineural hearing loss and a widend vestibular aqueduct in Pendred syndrome, Arch.Pendred syndrome is the most common syndromal form of deafness, yet the primary defect remains unknown.Pendred syndrome is one of the most common hereditary determined diseases in patients with syndromic sensorineural hearing impairment.The clinical diagnosis of Pendred syndrome was based on the laboratory and ultrasonographic signs of thyroid dyshormonogenesis (elevated TSH, low T4/fT4, goitre or normal thyroid volume) in association with sensorineural hearing loss. The clinical diagnosis of Pendred syndrome was based on the laboratory and ultrasonographic signs of thyroid dyshormonogenesis (elevated TSH, low T4/fT4, goitre or normal thyroid volume) in association with sensorineural hearing loss. Individuals were assigned affected status based on the characteristic clinical features of Pendred&apos;s syndrome, namely the presence of congenital sensorineural hearing loss and the appearance in early life of a goitre. Additionally, at least one affected member from each sibship had a characteristic positive perchlorate discharge test (Morgans &amp; Trotter, 1958).Individuals were assigned affected status based on the characteristic clinical features of Pendred&apos;s syndrome, namely the presence of congenital sensorineural hearing loss and the appearance in early life of a goitre. Additionally, at least one affected member from each sibship had a characteristic positive perchlorate discharge test (Morgans &amp; Trotter, 1958).\"}, {\"question\": \"List the three most prevalent pathogenic species of Borrelia in Europe.\", \"ideal_answer\": [\"The most prevalent pathogenic species of Borrelia in Europe are: B. afzelii, B. garinii and B. burgdorferi ss.\"], \"exact_answer\": [[\"B. afzelii\"], [\"B. garinii\"], [\"B. burgdorferi ss.\"]], \"context\": \" The Borrelia species causing Lyme borreliosis in Europe express different OspA serotypes on their surface, B. burgdorferi (serotype 1), B. afzelii (serotype 2), B. garinii (serotypes, 3, 5 and 6) and B. bavariensis (serotype 4), while only B. burgdorferi is present in the US. Borrelia afzelii is the predominant Borrelia species causing Lyme borreliosis in Europe.Moreover we detected Borrelia spielmanii, Borrelia miyamotoi, Babesia divergens, and Babesia venatorum for the first time in Danish ticks. We successfully determined the prevalence of expected (Borrelia burgdorferi sensu latoBorrelia miyamotoi is a newly described emerging pathogen transmitted to people by Ixodes species ticks and found in temperate regions of North America, Europe, and Asia. There is limited understanding of large scale entomological risk patterns of B. miyamotoi and of Borreila burgdorferi sensu stricto (ss), the agent of Lyme disease, in western North America. In France as elsewhere in Europe the most prevalent TBD in humans is Lyme borreliosis, caused by different bacterial species belonging to Borrelia burgdorferi sensu lato complexB. miyamotoi belonging to the relapsing fever group and transmitted by the same Ixodes species has been involved in human disease in Russia, the USA and the Netherlands. Laboratory diagnosis of Lyme disease is based on the serological detection of antibodies against the etiologic agent Borrelia burgdorferi.B. burgdorferi sl complex, currently comprising at least 19 genospecies, includes the main pathogenic species responsible for human disease in Europe: B. burgdorferi sensu stricto (ss), B. afzelii, and B. garinii. The European red squirrel (Sciurus vulgaris) has long been suspected to be a reservoir host of the agents of Lyme borreliosis, in particular B. burgdorferi sensu stricto (s.s.).n adults (n = 227), the prevalence of infection with B. burgdorferi s.l. was 27.3%, with 18.9% B. burgdorferi s.s., 11.9% B. afzelii, and 3.5% B. garinii. : The overall estimated prevalence of B. burgdorferi s.l was 9.7% . The most prevalent genospecies was B. afzelii which was detected in 53.5% of Borrelia-positive ticks, followed by B. garinii and B. valaisiana with 26.2% and 5.5%, respectively. Also, B. bavariensis and B. burgdorferi s.s. DNA in single I. ricinus ti Among all the analysed antigens those of B. burgdorferi s.s. were the most frequent cause of immunological reaction, followed by antigens of B. afzelii and B. garinii. Reaction to antigens of B. spielmanii was rarely detected.Overall, 15.8% ticks were infected with B. burgdorferi s. I. Borrelia afzelii (43.1%) was the predominant genospecies, followed by Borrelia valaisiana (14.7%), Borrelia garinii (13.7%) and Borrelia burgdorferi sensu stricto (6.3%). In cases where the genospecies could be identified, B. garinii was most frequently found (8x), followed with B. burgdorferi s.s. (4\\u00d7) and B. afzelii (3\\u00d7).In Europe, Ixodes ricinus is the vector of many pathogens of medical and veterinary relevance, among them Borrelia burgdorferi sensu lato and tick-borne encephalitis virus, which have been the subject of numerous investigationsThe minimum prevalence of Borrelia burgdorferi sensu lato was 2.5%. Borr. afzelii, Borr. burgdorferi sensu stricto, Borr. garinii, Borr. lusitaniae, and Borr. valaisiana were identified by hybridization.\"}, {\"question\": \"Which are the subunits of the transcription factor NF-kappaB in the canonical pathway activation?\", \"ideal_answer\": [\"The NF-\\u03baB canonical pathway is mediated by the p65/relA and p50 subunits.\"], \"exact_answer\": [[\"p65\", \"RelA\"], [\"p50\", \"NF-\\u03baB1\"]], \"context\": \"In this study, we demonstrated that the HSV-1 UL42 protein, a DNA polymerase processivity factor, was a novel antagonism of the canonical NF-\\u03baB signaling pathway. UL42 was shown to significantly suppress TNF-\\u03b1 mediated NF-\\u03baB activation. Co-immunoprecipitation experiment revealed that UL42 bound to the NF-\\u03baB subunits p65 and p50. Thus, it was first time we demonstrated that HSV-1 UL42 appeared to prevent NF-\\u03baB-dependent gene expression by retaining p65 and p50 in the cytoplasm, and UL42-dependent transcriptional activation were inherently coupled to promote HSV-1 lytic replication, which also may contribute to immune evasion and pathogenesis of HSV-1.The review describes NF-\\u03baB and its two pathways: the canonical, mediated by the p65 and p50 subunits, and the non-canonical, mediated by the subunits RelB, p52 and p50. The function of the TTV open reading frame (ORF) in the nuclear factor kappaB (NF-kappaB) pathway has not yet been established. In this study, we found for the first time that the TTV ORF2 protein could suppress NF-kappaB activity in a dose-dependent manner in the canonical NF-kappaB pathway. By Western blot analysis, we proved that the TTV ORF2 protein did not alter the level of NF-kappaB expression but prevented the p50 and p65 subunits from entering the nucleus due to the inhibition of IkappaBalpha protein degradation.Herpes simplex virus 1 DNA polymerase processivity factor UL42 inhibits TNF-\\u03b1-induced NF-\\u03baB activation by interacting with p65/RelA and p50/NF-\\u03baB1.\"}, {\"question\": \"Which receptors are targeted by a drug Macitentan?\", \"ideal_answer\": [\"Endothelin receptor A and endothelin receptor B are targeted by a drug Macitentan. Macitentan is a potent, orally active, non-peptide dual antagonist of endothelin receptors A and B that is approved for the treatment of pulmonary arterial hypertension.\"], \"exact_answer\": [[\"endothelin receptor A\"], [\"endothelin receptor B\"]], \"context\": \"Macitentan is a novel dual endothelin receptor antagonist with sustained receptor binding in clinical development for pulmonary arterial hypertension. This prospective, randomised, double-blind, multicentre, parallel-group, placebo-controlled phase II trial (NCT00903331) investigated the efficacy and safety of the endothelin receptor antagonist macitentan in idiopathic pulmonary fibrosis. Investigation of the effects of ketoconazole on the pharmacokinetics of macitentan, a novel dual endothelin receptor antagonist, in healthy subjects.Macitentan is a potent, orally active, non-peptide antagonist of endothelin receptors with tissue-targeting properties, currently undergoing clinical development for the treatment of pulmonary arterial hypertension. Interaction profile of macitentan, a new non-selective endothelin-1 receptor antagonist, in vitro.Macitentan is a new non-selective endothelin-1 receptor antagonist under development for the treatment of pulmonary arterial hypertension. This article highlights how the introduction of new compounds such as macitentan, riociguat and selexipag, which act on the endothelin, nitric oxide and prostacyclin pathways, respectively, have the potential to further improve the prognosis for patients with PAH.Dual ETRA/ETRB blockade with macitentan (or the combination of the ETRA and ETRB antagonists BQ123 and BQ788) did not enhance antitumor immune cell recruitment. The novel ERA macitentan has recently concluded testing in a Phase III morbidity/mortality clinical trial in PAH patients. Given that the newest PAH drugs include a receptor tyrosine kinase antagonist (imatinib), a soluble guanylate cyclase stimulator (riociguat), an oral analog of prostacyclin (selexipag), and a tissue targeting ERA (macitentan) determination of appropriate combinations for various etiologies and clinical stages is urgently required. The discovery of N-[5-(4-bromophenyl)-6-[2-[(5-bromo-2-pyrimidinyl)oxy]ethoxy]-4-pyrimidinyl]-N'-propylsulfamide (Macitentan), an orally active, potent dual endothelin receptor antagonist.Among these, compound 17 (macitentan, ACT-064992) emerged as particularly interesting as it is a potent inhibitor of ET(A) with significant affinity for the ET(B) receptor and shows excellent pharmacokinetic properties and high in vivo efficacy in hypertensive Dahl salt-sensitive rats. Renal, retinal and cardiac changes in type 2 diabetes are attenuated by macitentan, a dual endothelin receptor antagonist.Here we investigated the effects of macitentan, an orally-active, tissue-targeting dual ET receptor antagonist on chronic complications in type 2 diabetes. Several specific therapeutic agents were developed for the medical management of PAH including prostanoids (epoprostenol, trepoprostenil, iloprost), endothelin receptor antagonists (bosentan, ambrisentan, macitentan soon available) and phosphodiesterase type 5 inhibitors (sildenafil, tadalafil). The aim of the present study is to evaluate the in vitro effect of macitentan, an orally active tissue-targeting dual endothelin receptor antagonist, and its major metabolite (ACT-132577) on alpha smooth muscle actin (alphaSMA) expression, evaluated on dermal fibroblasts from healthy subjects and on dermal fibroblasts from lesional and non-lesional skin from sclerodermic patients. Safety, tolerability, pharmacokinetics, and pharmacodynamics of macitentan, an endothelin receptor antagonist, in an ascending multiple-dose study in healthy subjects.This multiple-ascending-dose study investigated safety, tolerability, pharmacokinetics, and pharmacodynamics, of macitentan, a new endothelin receptor antagonist (ERA) with sustained receptor binding and enhanced tissue penetration properties compared to other ERAs.We assessed the efficacy of macitentan, a new dual endothelin-receptor antagonist, using a primary end point of morbidity and mortality in a long-term trial.Many clinical studies targeting the vasoconstrictor ET-1 pathway with receptor antagonists like bosentan and ambrisentan have shown strong results, even more optimism coming from macitentan, the newest drug.Promising candidates include tyrosine kinase antagonists (e.g. imatinib); soluble guanylate cyclase stimulators (riociguat); an oral analog of prostacyclin (selexipag); and a tissue targeting endothelin receptor antagonist (macitentan).Pharmacokinetics of the novel dual endothelin receptor antagonist macitentan in subjects with hepatic or renal impairment.Efficacy, safety and clinical pharmacology of macitentan in comparison to other endothelin receptor antagonists in the treatment of pulmonary arterial hypertension.Macitentan is a novel dual endothelin receptor antagonist (ERA) showing sustained receptor occupancy. Macitentan (Opsumit\\u00ae) is a novel dual endothelin receptor antagonist (ERA) with sustained receptor binding properties developed by Actelion Pharmaceuticals Ltd. Endothelin (ET)-1 influences pathological changes via two ET receptor subtypes (ETA and ETB), to which it binds with high affinity. Macitentan was developed by modifying the structure of bosentan in the search for an optimal dual ERA with improved efficacy and tolerability compared with other ERAs. Absorption, distribution, metabolism, and excretion of macitentan, a dual endothelin receptor antagonist, in humans.Macitentan is a tissue-targeting, dual endothelin receptor antagonist, currently under phase 3 investigation in pulmonary arterial hypertension. Antivascular therapy for multidrug-resistant ovarian tumors by macitentan, a dual endothelin receptor antagonist.The nonselective endothelin type A and B receptor antagonists bosentan (100 nmol/L) and macitentan (30 nmol/L) alone relaxed endothelin-contracted rings by 30\\u00b15% and 24\\u00b13%, respectively. Macitentan is a dual endothelin receptor antagonist under phase 3 investigation in pulmonary arterial hypertension.This rise could largely be prevented with the endothelin receptor antagonist macitentan (\\u0394BP: 12.3\\u00b11.5 mm Hg) and only mildly with Tempol, a superoxide dismutase mimetic (\\u0394BP: 25.9\\u00b12.3 mm Hg). Promising study results for the treatment of IPF were published for the triple tyrosine kinase inhibitor intedanib, other drugs such as the endothelin receptor antagonist (ERA) macitentan are currently investigated in clinical trials.To study the pharmacokinetics, pharmacodynamics, and tolerability of rising single doses of macitentan, an endothelin receptor antagonist, in healthy male subjects. Macitentan (ACT-064992), a tissue-targeting endothelin receptor antagonist, enhances therapeutic efficacy of paclitaxel by modulating survival pathways in orthotopic models of metastatic human ovarian cancer.We investigated the therapeutic efficacy of the dual ETR antagonist, macitentan, on human ovarian cancer cells, SKOV3ip1 and IGROV1, growing orthotopically in nude mice. Macitentan, a tissue-targeting endothelin receptor antagonist for the potential oral treatment of pulmonary arterial hypertension and idiopathic pulmonary fibrosis.Macitentan (ACT-064992), under development by Actelion Ltd in collaboration with Japanese licensee Nippon Shinyaku Co Ltd, is an orally active, non-peptide dual endothelin (ET)(A) and ET(B) receptor antagonist for the potential treatment of idiopathic pulmonary fibrosis (IPF) and pulmonary arterial hypertension (PAH). Macitentan, because of its ability to target the tissues and to block both ET(A) and ET(B) receptors, is emerging as a new agent to treat cardiovascular disorders associated with chronic tissue ET system activation. Comparison of the dissolution and pharmacokinetic profiles of two galenical formulations of the endothelin receptor antagonist macitentan.Macitentan (ACT-064992) is an orally active endothelin receptor antagonist. Pharmacology of macitentan, an orally active tissue-targeting dual endothelin receptor antagonist.Macitentan, also called Actelion-1 or ACT-064992 [N-[5-(4-bromophenyl)-6-(2-(5-bromopyrimidin-2-yloxy)ethoxy)-pyrimidin-4-yl]-N'-propylaminosulfonamide], is a new dual ET(A)/ET(B) endothelin (ET) receptor antagonist designed for tissue targeting.Macitentan and its metabolite antagonized the specific binding of ET-1 on membranes of cells overexpressing ET(A) and ET(B) receptors and blunted ET-1-induced calcium mobilization in various natural cell lines, with inhibitory constants within the nanomolar range. In functional assays, macitentan and ACT-132577 inhibited ET-1-induced contractions in isolated endothelium-denuded rat aorta (ET(A) receptors) and sarafotoxin S6c-induced contractions in isolated rat trachea (ET(B) receptors). In conclusion, macitentan, by its tissue-targeting properties and dual antagonism of ET receptors, protects against end-organ damage in diabetes and improves survival in pulmonary hypertensive rats.\"}, {\"question\": \"Which drugs are utilized to treat eosinophilic esophagitis?\", \"ideal_answer\": [\"Therapeutic options of eosinophilic esophagitis include use of proton-pump inhibitors, immunosuppressive drugs, elimination diets, and esophageal dilatation.\\nOral viscous budesonide (OVB) is an effective treatment of pan-esophageal disease in children with EoE. OVB improves symptoms and endoscopic and histologic features.\"], \"exact_answer\": [[\"proton pump inhibitors\"], [\"immunosoppressive drugs\"], [\"budesonide\"]], \"context\": \"Current therapeutic options include use of proton-pump inhibitors, immunosuppressive drugs, elimination diets, and esophageal dilatation.Aerosolized, swallowed fluticasone leads to a histologic but not a symptomatic response in adults with EoEOVB is an effective treatment of pan-esophageal disease in children with EoE. OVB improves symptoms and endoscopic and histologic features. Proton pump inhibitor single therapy did not significantly improve esophageal eosinophilia or symptoms of EoE.\"}, {\"question\": \"Where is X-ray free electron laser used?\", \"ideal_answer\": [\"X-ray free electron laser (XFEL) technologies provide coherent and extremely intense photon pulses of short duration. XFELs are particularly useful in structural biology and imaging, in structural studies of single biological macromolecules (e.g. high resolution protein structure determination) and assemblies (e.g. viruses) or nanocrystals, which are not amenable to investigation with traditional crystallographic methods. Moreover, XFELs have the potential to be used for studying enzyme kinetics.\"], \"exact_answer\": [[\"high resolution protein structure determination\"], [\"molecular imaging\", \"single-particle imaging\"], [\"study of enzyme kinetics\", \"time resolved protein crystallography\"]], \"context\": \"In conjunction with the XFEL temporal profile and high-flux, it is a powerful tool for studying the dynamics of time-dependent systems. Photo-induced processes and fast catalytic reaction kinetics, ranging from femtoseconds to milliseconds, will be resolvable in a wide array of systems circumventing radiation damage.We discuss the pertinent physics of the intense X-ray-matter interactions, and illustrate the importance of electron-ion collisions. Detailed simulations of the interaction process conducted with a radiative-collisional code show good qualitative agreement with the experimental results. We obtain insights into the evolution of the charge state distribution of the system, the electron density and temperature, and the timescales of collisional processes. Our results should inform future high-intensity X-ray experiments involving dense samples, such as X-ray diffractive imaging of biological systems, material science investigations, and the study of matter in extreme conditions.employment of \\\"exotic\\\" systems, such as the Free Electron LASER (FEL), that are expected to focus on the fundamental processes of life, following chemical reactions and biological processes as they happen, on unprecedented time and size scales.New data analysis approaches are outlined for the correlated fluctuations in fast WAXS, for protein nanocrystals just a few molecules on a side, and for the continuous x-ray scattering from a single virusResearch opportunities and techniques are reviewed for the application of hard x-ray pulsed free-electron lasers (XFEL) to structural biology. These include the imaging of protein nanocrystals, single particles such as viruses, pump--probe experiments for time-resolved nanocrystallography, and snapshot wide-angle x-ray scattering (WAXS) from molecules in solution. The use of femtosecond exposure times, rather than freezing of samples, as a means of minimizing radiation damage is shown to open up new opportunities for the molecular imaging of biochemical reactions at room temperature in solutionNew opportunities for solving the phase problem for XFEL data are outlined. A summary of the latest results is given, which now extend to atomic resolution for nanocrystals. Possibilities for time-resolved chemistry using fast WAXS (solution scattering) from mixtures is reviewed, toward the general goal of making molecular movies of biochemical processes.Molecular imaging using X-ray free-electron lasers.The recent development of X-ray free-electron laser sources has created new opportunities for the structural analysis of protein nanocrystals.X-ray free electron lasers hold the promise of enabling atomic-resolution diffractive imaging of single biological molecules.The emergence of femtosecond diffractive imaging with X-ray lasers has enabled pioneering structural studies of isolated particles, such as viruses, at nanometer length scales.In this paper we report room temperature X-ray diffraction data of PS\\u00a0II microcrystals obtained using ultrashort (<\\u00a050\\u00a0fs) 9\\u00a0keV X-ray pulses from a hard X-ray free electron laser, namely the Linac Coherent Light Source.High-resolution protein structure determination by serial femtosecond crystallographyWe applied serial femtosecond crystallography (SFX) using an x-ray free-electron laser (XFEL) to obtain high-resolution structural information from microcrystals (less than 1 micrometer by 1 micrometer by 3 micrometers) of the well-characterized model protein lysozyme.X-ray free electron laser (X-FEL)-based serial femtosecond crystallography is an emerging method with potential to rapidly advance the challenging field of membrane protein structural biology. Here we recorded interpretable diffraction data from micrometer-sized lipidic sponge phase crystals of the Blastochloris viridis photosynthetic reaction center delivered into an X-FEL beamHere we show that high-quality diffraction data can be obtained with a single X-ray pulse from a non-crystalline biological sample, a single mimivirus particle, which was injected into the pulsed beam of a hard-X-ray free-electron laser, the Linac Coherent Light Source.Single mimivirus particles intercepted and imaged with an X-ray laserFemtosecond X-ray protein nanocrystallographyHere we present a method for structure determination where single-crystal X-ray diffraction 'snapshots' are collected from a fully hydrated stream of nanocrystals using femtosecond pulses from a hard-X-ray free-electron laser, the Linac Coherent Light Source. We prove this concept with nanocrystals of photosystem I, one of the largest membrane protein complexes.These methods can be used to image biological and materials science samples at high resolution with x-ray undulator radiation and establishes the techniques to be used in atomic-resolution ultrafast imaging at x-ray free-electron laser sources.With the prospects of the x-ray free electron lasers, this approach could provide a major new opportunity for the high-resolution three-dimensional structure determination of single biomolecules.X-ray free-electron lasers (XFELs) generate sequences of ultra-short spatially coherent pulses of X-ray radiation.Diffractive imaging using the intense and coherent beam of X-ray free-electron lasers opens new perspectives for structural studies of single nanoparticles and biomolecules.The opening of hard X-ray free-electron laser facilities, such as the Linac Coherent Light Source (LCLS) at SLAC National Accelerator Laboratory in the United States, has ushered in a new era in structural determination.Where possible we also highlight areas that we think could push this field forward with emerging technologies, such as X-ray free electron lasers (X-feL), which could have a big impact on the membrane protein structural biology community.X-ray free electron lasers (XFELs) deliver short (<100 fs) and intense (\\u223c10(12) photons) pulses of hard X-rays, making them excellent sources for time-resolved studies.Modern imaging techniques at the molecular scale rely on utilizing novel coherent light sources like X-ray free electron lasers for the ultimate goal of visualizing such objects as individual biomolecules rather than crystals.In single-particle coherent x-ray diffraction imaging experiments, performed at x-ray free-electron lasers (XFELs), samples are exposed to intense x-ray pulses to obtain single-shot diffraction patterns.The structure reveals the mechanism of native TbCatB inhibition and demonstrates that new biomolecular information can be obtained by the \\\"diffraction-before-destruction\\\" approach of x-ray free-electron lasers from hundreds of thousands of individual microcrystals.Coherent diffraction imaging (CDI) of single molecules at atomic resolution is a major goal for the x-ray free-electron lasers (XFELs).Ultrashort X-ray pulses from free-electron laser X-ray sources make it feasible to conduct small- and wide-angle scattering experiments on biomolecular samples in solution at sub-picosecond timescales.The short and intense pulses of the new X-ray free electron lasers, now operational or under construction, may make possible diffraction experiments on single molecule-sized objects with high resolution, before radiation damage destroys the sample.The recent development of x-ray free electron lasers providing coherent, femtosecond-long pulses of high brilliance and variable energy opens new areas of scientific research in a variety of disciplines such as physics, chemistry, and biology.X-ray free electron laser (X-FEL)-based serial femtosecond crystallography is an emerging method with potential to rapidly advance the challenging field of membrane protein structural biology.X-ray free-electron lasers (FELs) show promise for revealing the structure of single molecules or nanocrystals, but the phase problem remains largely unsolved.Coherent diffractive imaging using x-ray free-electron lasers (XFELs) may provide a unique opportunity for high-resolution structural analysis of single particles sprayed from an aqueous solution into the laser beam.Here we present a method for structure determination where single-crystal X-ray diffraction 'snapshots' are collected from a fully hydrated stream of nanocrystals using femtosecond pulses from a hard-X-ray free-electron laser, the Linac Coherent Light Source.This work is directly relevant to the time-resolved imaging of magnetic dynamics using coherent and ultrafast radiation from x-ray free-electron lasers and also to broader classes of diffractive imaging experiments which suffer noisy data, missing data, or both.By combining serial nanocrystallography with x-ray free-electron laser sources in the future, it may be possible to produce molecular-resolution electron-density maps using membrane protein crystals that contain only a few hundred or thousand unit cells.Very strong soft X-ray free electron laser (FEL) pulses must be short to allow for investigations of ultra-fast wet chemistry, according to the principle of collect and destroy.\"}, {\"question\": \"Which protein kinases have been found to phosphorylate Phospholamban and affect its biological activity?\", \"ideal_answer\": [\"Phosphorylation of phospholamban at Ser16 is mainly mediated by PKA and at Thr17 by Ca(2+) /calmodulin-dependent protein kinase (CaMKII). Phospholamban is also a reporter for PKG activity and akt kinase interacts with and phosphorylates PLN at Thr(17).\"], \"exact_answer\": [[\"protein kinase A\", \"PKA\"], [\"protein kinase G\", \"PKG\"], [\"Ca(2+) /calmodulin-dependent protein kinase II\", \"CaMKII\"], [\"akt kinase\", \"Akt\"]], \"context\": \"Dephosphorylated PLN is an inhibitor of SERCA2a and phosphorylation by protein kinase A (PKA) or calcium-calmodulin-dependent protein kinases (CAMKII) relieves these inhibitory effects. We have found that phosphorylation of PLN by protein kinase A (PKA) leads to an increase in the rate of Ca(2+) leak from Ca(2+)-loaded SR vesicles.PKG activity of cardiac myocytes was assessed by phospholamban (PLB) phosphorylation determined by western blot. PLB is not a unique reporter for PKG activity since it is also phosphorylated by protein kinase A (PKA), Ca(2+)-calmodulin-dependent protein kinase II (CaMKII) plays an important role mediating apoptosis/necrosis during ischemia-reperfusion (IR).The onset of reperfusion increased the phosphorylation of Thr(17) site of phospholamban, without changes in total protein, consistent with an increase in CaMKII activity. We found that when activated, Akt interacts with and phosphorylates PLN at Thr(17), the Ca(2+)-calmodulin-dependent kinase IIdelta site, whereas silencing Akt signaling, through the knock-out of phosphatidylinositol-dependent kinase-1, resulted in reduced phosphorylation of PLN at Thr(17). Deletion of Arg(14) disrupts the protein kinase A recognition motif, which abrogates phospholamban phosphorylation and results in constitutive SERCA inhibition.Dephosphorylated PLN tonically inhibits the SR Ca-ATPase (SERCA2a), while phosphorylation at Ser16 by PKA and Thr17 by Ca(2+) /calmodulin-dependent protein kinase (CaMKII) relieves the inhibition, and this increases SR Ca(2+) uptake. Both CaMKIIdelta(C) and the novel CaMKIIbeta(4) isoforms were found to exist in complex with alphaKAP and SERCA2a at the SR and were able to phosphorylate Thr-17 on phospholamban (PLN), an accessory subunit and known regulator of SERCA2a activityPhospholamban is the major membrane protein of the heart phosphorylated in response to beta-adrenergic stimulation. In cell-free systems, cAMP-dependent protein kinase catalyzes exclusive phosphorylation of serine 16 of phospholamban, whereas Ca2+/calmodulin-dependent protein kinase gives exclusive phosphorylation of threonine 17 (Simmerman, H. K. B., Collins, J. H., Theibert, J. L., Wegener, A. D., and Jones, L. R. (1986) J. Biol. Chem. 261, 13333-13341).\"}, {\"question\": \"List autoimmune disorders associated with GAD65 autoantibodies.\", \"ideal_answer\": [\"Autoantibodies to the smaller isoform of glutamate decarboxylase (GAD) can be found in patients with type 1 diabetes and a number of neurological disorders, including stiff-person syndrome, cerebellar ataxia and limbic encephalitis.\"], \"exact_answer\": [[\"type 1 diabetes\"], [\"stiff-person syndrome\"], [\"cerebellar ataxia\"], [\"limbic encephalitis\"]], \"context\": \"Evidence for somatic mutation and affinity maturation of diabetes associated human autoantibodies to glutamate decarboxylase.The circulating autoantibodies to multiple islet autoantigens including GAD, insulin, and IA-2 are the important immunological features of type 1 diabetes. 100% (39/39) selected type 1 diabetes mellitus (DM) patients,Autoantibodies to the GM2-1 islet ganglioside and to GAD-65 at type 1 diabetes onset.Autoantibodies to the smaller isoform of glutamate decarboxylase (GAD) can be found in patients with type 1 diabetes and a number of neurological disorders, including stiff-person syndrome, cerebellar ataxia and limbic encephalitis. Anti-glutamic acid decarboxylase antibody (GAD-ab)-associated cerebellar ataxia is a rare neurological disorder characterized by cerebellar symptoms concomitant with high GAD-ab levels in serum and cerebrospinal fluid (CSF).The unique association of autoantibody-mediated cerebellar ataxia and late-onset T1DM in 2 siblings with similar clinical and paraclinical phenotypes strengthens the concept that hereditary factors might play a relevant role also in autoimmune diseases so far considered to be sporadic. \"}, {\"question\": \"List the different subtypes of thyroid cancer.\", \"ideal_answer\": [\"The different histologic subtypes of thyroid cancer include papillary, follicular, anaplastic, medullary, and H\\u00fcrthle cell carcinomas.\"], \"exact_answer\": [[\"Papillary thyroid carcinoma\"], [\"Follicular thyroid carcinoma\"], [\"Anaplastic thyroid carcinoma\"], [\"Medullary thyroid carcinoma\"], [\"H\\u00fcrthle cell carcinoma\"]], \"context\": \"Thyroid cancer is the most common cancer of the endocrine system and is responsible for the majority of deaths from endocrine malignancies. Although a large proportion of thyroid cancers belong to well differentiated histologic subtypes, which in general show a good prognosis after surgery and radioiodine ablation, the treatment of radio-resistant papillary-type, of undifferentiated anaplastic, and of medullary-type thyroid cancers remains unsatisfactorySupervised hierarchical cluster analysis demonstrated grouping of 2 histologies (papillary and follicular thyroid carcinoma)FDG-PET/CT is also useful in the initial (post thyroidectomy) staging of high-risk patients with less differentiated (and thus less iodine-avid and clinically more aggressive) subtypes, such as tall cell variant and H\\u00fcrthle cell carcinoma, but in particular poorly differentiated and anaplastic carcinoma. FDG-PET/CT may help in defining the extent of disease in some patients with medullary thyroid carcinoma and rising postoperative calcitonin levelsHistologic subtypes included papillary (41%), follicular (22%), anaplastic (19%), medullary (15%), and H\\u00fcrthle cell carcinomas (4%)Follicular thyroid cancer is the second most common thyroid malignancy after PTCThyroid cancer subtypes were 90 patients with papillary thyroid cancer PTC, 24 with follicular FTC, eight with medullary MTC and two with anaplastic cancer.Papillary thyroid carcinoma is the most common type of thyroid malignancyThe main objectives of this review article are to (1) summarize the gross and histopathologic features of papillary thyroid carcinoma; (2) provide an overview of the subtypes of papillary thyroid carcinoma and their prognosis; (3) discuss established and emerging data on the immunohistochemical findings that are helpful in differential diagnosis; and (4) summarize molecular findings and pathogenesis of these lesionsAlthough the vast majority of papillary thyroid carcinomas have an excellent prognosis, some variants of papillary thyroid carcinoma can have a more aggressive course\"}, {\"question\": \"List Pentalogy of Fallot.\", \"ideal_answer\": [\"Pentalogy of Fallot consists of a pulmonic stenosis, a ventricular septal defect, an overriding aorta, a right ventricular hypertrophy and a patent foramen ovale.\"], \"exact_answer\": [[\"pulmonic stenosis\"], [\"ventricular septal defect\"], [\"overriding aorta\"], [\"right ventricular hypertrophy\"], [\"patent foramen ovale\"]], \"context\": \"Pentalogy of Fallot (POF) is a rare form of congenital heart disease characterized by the association of Tetralogy of Fallot (TOF) with an atrial septal defect (ASD).Additionally, an atrial septal defect was found on necropsy, resulting in the final diagnosis of Tetralogy of Fallot with an atrial septal defect (a subclass of Pentalogy of Fallot).Pentalogy of Fallot is a rare cyanotic congenital heart disease characterized by biventricular origin of the aorta above a large ventricular septal defect, obstruction of the pulmonary outflow, right ventricular hypertrophy (tetralogy of Fallot), and an atrial septal defect. Postmortem examination of the ram's heart showed a pentalogy of Fallot, consisting of a pulmonic stenosis, a ventricular septal defect, an overriding aorta, a right ventricular hypertrophy and a patent foramen ovale.Additionally, an atrial septal defect was found on necropsy, resulting in the final diagnosis of Tetralogy of Fallot with an atrial septal defect (a subclass of Pentalogy of Fallot).Pentalogy of Fallot is a rare cyanotic congenital heart disease characterized by biventricular origin of the aorta above a large ventricular septal defect, obstruction of the pulmonary outflow, right ventricular hypertrophy (tetralogy of Fallot), and an atrial septal defect.Based on these findings, the dog was diagnosed as a case of tetralogy of Fallot with atrial septal defect (pentalogy of Fallot).the dog was diagnosed as a case of tetralogy of Fallot with atrial septal defect (pentalogy of Fallot).In conclusion, whenever the diagnosis pentalogy of fallot is suspected, a multidisciplinary approach is essential. Pentalogy of fallot; Overriding Aorta; Ventricular Septal Defect; Atrial Septal Defect; Pulmonary Atresia; Doppler Echocardiography.resulting in the final diagnosis of Tetralogy of Fallot with an atrial septal defect (a subclass of Pentalogy of Fallot).Pentalogy of Fallot (POF) is a rare form of congenital heart disease characterized by the association of Tetralogy of Fallot (TOF) with an atrial septal defect (ASD).In conclusion, whenever the diagnosis pentalogy of fallot is suspected, a multidisciplinary approach is essential.Pentalogy of fallot; Overriding Aorta; Ventricular Septal Defect; Atrial Septal Defect; Pulmonary Atresia; Doppler Echocardiography.\"}, {\"question\": \"List algorithms suitable for predicting protein complexes\", \"ideal_answer\": [\"Protein-Protein interactions (PPI) play a key role in determining the outcome of most cellular processes. The correct identification and characterization of protein interactions and the networks, which they comprise, is critical for understanding the molecular mechanisms within the cell. Large-scale techniques such as pull down assays and tandem affinity purification are used in order to detect protein interactions in an organism. Today, relatively new high-throughput methods like yeast two hybrid, mass spectrometry, microarrays, and phage display are also used to reveal protein interaction networks. Some suitable algorithms for predicting protein complexes are Naive Bayes Classifier, Negatome, Support Vector Machine, PEWCC, iPTMClust, NDComplex, PROCOMOSS, PPI network, metaPIS, EPOF, EAGLE, NFC, MCODE, DPClus, IPCA, CPM, MCL, CMC, SPICi, Core-Attachment, ProRank, ClusterONE, CFinder, Spectral, RNSC, Affinity Propagation, HKC, NWE, CP-DR, Struct2net and PIPE.\"], \"exact_answer\": [[\"Na\\u00efve Bayes Classifier\"], [\"Negatome\"], [\"Support Vector Machine\"], [\"PEWCC\"], [\"iPTMClust\"], [\"NDComplex\"], [\"PROCOMOSS\"], [\"PPI network\"], [\"metaPIS\"], [\"EPOF\"], [\"EAGLE\"], [\"NFC\"], [\"MCODE\"], [\"DPClus\"], [\"IPCA\"], [\"CPM\"], [\"MCL\"], [\"CMC\"], [\"SPICi\"], [\"Core-Attachment\"], [\"ProRank\"], [\"ClusterONE\"], [\"CFinder\"], [\"Spectral\"], [\"RNSC\"], [\"Affinity Propagation\"], [\"HKC\"], [\"NWE\"], [\"CP-DR\"], [\"Struct2net\"], [\"PIPE\"]], \"context\": \"Protein complexes are basic cellular entities that carry out the functions of their componentswe have designed several features characterizing heterodimeric protein complexes based on genomic data sets, and proposed a supervised-learning method for the prediction of heterodimeric protein complexes. This method learns the parameters of the features, which are embedded in the na\\u00efve Bayes classifier non-interacting proteins (NIPs) is important for training the algorithms to predict protein-protein interactions (PPIs) and for assessing the false positive rates of PPI detection efforts. We present the second version of Negatome,Negatome is derived by manual curation of literature and by analyzing three-dimensional structures of protein complexes.Identification of protein-ligand binding site is an important task in structure-based drug design and docking algorithmsSupport Vector Machine (SVM) is used to cluster the pockets that are most likely to bind ligands with the attributes of geometric characteristics, interaction potential, offset from protein, conservation score and properties surrounding the pocketsPredicting protein complexes from protein-protein interaction data is becoming a fundamental problem in computational biology. The identification and characterization of protein complexes implicated are crucial to the understanding of the molecular events under normal and abnormal physiological conditionsa novel graph mining algorithm (PEWCC) to identify such protein complexes. Firstly, the algorithm assesses the reliability of the interaction data, then predicts protein complexes based on the concept of weighted clustering coefficient. To demonstrate the effectiveness of the proposed method, the performance of PEWCC was compared to several methodsWe developed a novel PTM refinement algorithm, iPTMClust, which extends a recently introduced PTM refinement algorithm PTMClust and uses a non-parametric Bayesian model to better account for uncertainties in the quantity and identity of PTMs in the input data. The use of this new modeling approach enables iPTMClust to provide a confidence score per modification site that allows fine-tuning and interpreting resulting PTM predictionsWe develop an algorithm for identifying protein complexes by considering these two types of complexes separately. We test our algorithm on a few yeast protein interaction networks, and show that our algorithm is able to identify complexes more accurately than existing algorithms. A software program NDComplexStudying protein complexes is very important in biological processes since it helps reveal the structure-functionality relationships in biological networks and much attention has been paid to accurately predict protein complexes from the increasing amount of protein-protein interaction (PPI) data To provide an investigation of the roles of edges in PPI networks, we show that the edges connecting less similar vertices in topology are more significant in maintaining the global connectivity, indicating the weak ties phenomenon in PPI networksIdentification of protein complexes in protein-protein interaction (PPI) networks is the first step in understanding the organization and dynamics of cell function.We have developed an algorithm PROCOMOSS (Protein Complex Detection using Multi-objective Evolutionary Approach based on Semantic Similarity) for partitioning the whole PPI network into clusters, which serve as predicted protein complexes.With the exponentially growing amount of protein sequence data, an exploration of new methods that predict protein interaction sites based solely on sequence information is becoming increasingly urgentThis paper describes a new method that predicts interaction sites based on protein sequences by integrating five different algorithms employing meta-method, Majority Vote and SVMhmm Regression techniquesThe 'metaPIS' web-server was implemented for meta-prediction. In this paper, a novel protein complex identifying method, named EPOF, is proposed, using essential proteins and the local metric of vertex fitnessWe compare the performances of EPOF to other ten previous algorithms, including EAGLE, NFC, MCODE, DPClus, IPCA, CPM, MCL, CMC, SPICi, and Core-Attachment. We compare the performance of ProRank to some of the well known protein complex prediction methods (ClusterONE, CMC, CFinder, MCL, MCode and Core) in terms of precision and recallIn this paper we evaluated four different clustering algorithms using six different interaction datasets. We parameterized the MCL, Spectral, RNSC and Affinity Propagation algorithms and applied them to six PPI datasets produced experimentally by Yeast 2 Hybrid (Y2H) and Tandem Affinity Purification (TAP) methodsthis paper, we present a new topology-based algorithm, HKC, to detect protein complexes in genome-scale PPI networks. HKC mainly uses the concepts of highest k-core and cohesion to predict protein complexes by identifying overlapping clusters.  Protein complexes are important entities to organize various biological processes in the cell, like signal transduction, gene expression, and molecular transmission. In most cases, proteins perform their intrinsic tasks in association with their specific interacting partners, forming protein complexeMacropol et al. proposed a protein complex prediction algorithm, called RRW, which repeatedly expands a current cluster of proteins according to the stationary vector of a random walk with restarts with the cluster whose proteins are equally weighted. In the cluster expansion, all the proteins within the cluster have equal influences on determination of newly added protein to the cluster. In this paper, we extend the RRW algorithm by introducing a random walk with restarts with a cluster of proteins, each of which is weighted by the sum of the strengths of supporting evidence for the direct physical interactions involving the protein. The resulting algorithm is called NWE (Node-Weighted Expansion of clusters of proteins). In this paper, we propose a new topological model by extending the definition of k-clique community of algorithm CPM and introduced distance restriction, and develop a novel algorithm called CP-DR based on the new topological model for identifying protein complexesThis paper also describes a random forest classifier which can effectively combine the structure-based prediction results and other functional annotations together to predict protein interactionsHere we explain the development and implementation of a novel Protein-Protein Interaction Prediction Engine termed PIPE. This tool is capable of predicting protein-protein interactions for any target pair of the yeast Saccharomyces cerevisiae proteins from their primary structure and without the need for any additional information or predictions about the proteins\"}, {\"question\": \"What are the computational tools for the prediction of beta-barrel transmembrane proteins?\", \"ideal_answer\": [\"The computational tools for the prediction of beta-barrel transmembrane proteins (TMBs) are based mainly on the following methodologies: Hidden Markov Models (HMMs), hydrophobicity, structural data, k-nearest neighbor algorithm, Neural Networks and Support Vector Machines. The state-of-the-art computational tools for the prediction of TMBs are: BETAWARE, BOCTOPUS, BOMP, BTMX, HMM-B2TMR, OMPdb,PRED-TMBB, PROB, ProfTMB, PV, TMBETA-NET, TMB finding pipeline, TMBETADISC-RBF, TMBETAPRED-RBF, TMBHMM, TMB-Hunt, TMB-Hunt2, TMBKNN, TMBpro, transFold.\"], \"exact_answer\": [[\"BETAWARE\"], [\"BOCTOPUS\"], [\"BOMP\"], [\"BTMX\"], [\"HMM-B2TMR\"], [\"OMPdb\"], [\"PRED-TMBB\"], [\"PROB\"], [\"ProfTMB\"], [\"PV\"], [\"TMB finding pipeline\"], [\"TMBETADISC-RBF\"], [\"TMBETAPRED-RBF\"], [\"TMBHMM\"], [\"TMB-Hunt\", \"TMB-Hunt2\"], [\"TMBKNN\"], [\"TMBpro\"], [\"transFold\"], [\"TMBETA-NET\"]], \"context\": \"In this work, we propose a method based on radial basis networks for predicting the number of beta-strands in OMPs and identifying their membrane spanning segments.We have developed a web server, TMBETAPRED-RBF for predicting the transmembrane beta-strands from amino acid sequence and it is available at http://rbf.bioinfo.tw/~sachen/tmrbf.html.We have developed a prediction server, TMBETADISC-RBF, which is available at http://rbf.bioinfo.tw/~sachen/OMP.html.TMB finding pipeline: novel approach for detecting beta-barrel membrane proteins in genomic sequences.Interestingly, the present approach identified TMBs from all 15 families in TCDB.A new method is presented for identification of beta-barrel membrane proteins. It is based on a hidden Markov model (HMM) with an architecture obeying these proteins' construction principles. Once the HMM is trained, log-odds score relative to a null model is used to discriminate beta-barrel membrane proteins from other proteins.PRED-TMBB: a web server for predicting the topology of beta-barrel outer membrane proteins.We present here a web server (PRED-TMBB, http://bioinformatics.biol.uoa.gr/PRED-TMBB) which is capable of predicting the transmembrane strands and the topology of beta-barrel outer membrane proteins of Gram-negative bacteria. The method is based on a Hidden Markov Model, trained according to the Conditional Maximum Likelihood criterion.Predicting transmembrane beta-barrels in proteomes.Here we introduced the design, statistics and results of a novel profile-based hidden Markov model for the prediction and discrimination of TMBs.A sequence-profile-based HMM for predicting and discriminating beta barrel membrane proteins.RESULTS: We develop a HMM model, which can predict the topology of beta barrel membrane proteins using, as input, evolutionary information.BOMP: a program to predict integral beta-barrel outer membrane proteins encoded within genomes of Gram-negative bacteria.This work describes the development of a program that predicts whether or not a polypeptide sequence from a Gram-negative bacterium is an integral beta-barrel outer membrane protein. The program, called the beta-barrel Outer Membrane protein Predictor (BOMP), is based on two separate components to recognize integral beta-barrel proteins.On the basis of existing knowledge of beta-barrel outer-membrane proteins, several state of the art prediction methods, as well as a new in-house program (PROB) were employed for the systematic exploration of Mycobacterium tuberculosis predicted proteomes for potential beta-barrel structures.Computational identification of beta-barrel outer-membrane proteins in Mycobacterium tuberculosis predicted proteomes as putative vaccine candidates.BETAWARE: a machine-learning tool to detect and predict transmembrane beta-barrel proteins in prokaryotes.Recently, we developed two top-performing methods based on machine-learning approaches to tackle both the detection of TMBBs in sets of proteins and the prediction of their topology.Supersecondary structure prediction of transmembrane beta-barrel proteins.We introduce a graph-theoretic model for predicting the supersecondary structure of transmembrane \\u03b2-barrel proteins--a particular class of proteins that performs diverse important functions but it is difficult to determine their structure with experimental methods.BOCTOPUS: improved topology prediction of transmembrane \\u03b2 barrel proteins.Here, we present BOCTOPUS; an improved method for the topology prediction of TMBs by employing a combination of support vector machines (SVMs) and Hidden Markov Models (HMMs).TMBHMM: a frequency profile based HMM for predicting the topology of transmembrane beta barrel proteins and the exposure status of transmembrane residues.We present here TMBHMM, a computational method based on a hidden Markov model for predicting the structural topology of putative TMBs from sequence. In addition to predicting transmembrane strands, TMBHMM also predicts the exposure status (i.e., exposed to the membrane or hidden in the protein structure) of the residues in the transmembrane region, which is a novel feature of the TMBHMM method. Furthermore, TMBHMM can also predict the membrane residues that are not part of beta barrel forming strands.We present BTMX (Beta barrel TransMembrane eXposure), a computational method to predict the exposure status (i.e. exposed to the bilayer or hidden in the protein structure) of transmembrane residues in transmembrane beta barrel proteins (TMBs). BTMX predicts the exposure status of known TM residues with an accuracy of 84.2% over 2,225 residues and provides a confidence score for all predictions.A method for discovering transmembrane beta-barrel proteins in Gram-negative bacterial proteomes.A web server based on the proposed method is available at http://yanbioinformatics.cs.usu.edu:8080/TMBKNNsubmit.This paper presents a k-nearest neighbor (K-NN) method for discriminating TMB and non-TMB proteins.TMBpro: secondary structure, beta-contact and tertiary structure prediction of transmembrane beta-barrel proteins.We develop a suite (TMBpro) of specialized predictors for predicting secondary structure (TMBpro-SS), beta-contacts (TMBpro-CON) and tertiary structure (TMBpro-3D) of transmembrane beta-barrel proteins. We compare our results to the recent state-of-the-art predictors transFold and PRED-TMBB using their respective benchmark datasets, and leave-one-out cross-validation.A consensus algorithm to screen genomes for novel families of transmembrane beta barrel proteins.Here we describe a new algorithm combining composition and hidden Markov model topology based classifiers (called TMB-Hunt2), which achieves a crossvalidation accuracy of >95%, with 96.7% precision and 94.2% recall.Tools and datasets are made available through a website called TMB-Web (http://www.bioinformatics.leeds.ac.uk/TMB-Web/TMB-Hunt2).\"}, {\"question\": \"Where can we find the protein lacritin?\", \"ideal_answer\": [\"The protein lacritin can be found in lacrimal and salivary glands as well as in tear fluid and in the thyroid.\"], \"exact_answer\": [[\"lacrimal gland\"], [\"salivary glands\"], [\"thyroid\"], [\"tear fluid\"]], \"context\": \"Lacritin, a glycoprotein secreted from lacrimal acinar cells,Essential for the viability of the surface epithelium of the eye and for normal vision is the thin, but protein-rich, tear film in which the small tear glycoprotein lacritin appears to play a prominent prosecretory, cytoprotective, and mitogenic roleLacritin is a prosecretory mitogen in tears and, although a tear protein, it promotes basal tearing and lacrimal gland secretion.cDNA and genomic cloning of lacritin, a novel secretion enhancing factor from the human lacrimal gland.Lacritin mRNA and protein are highly expressed in human lacrimal gland, moderately in major and minor salivary glands and slightly in thyroid. No lacritin message or protein is detected elsewhere among more than 50 human tissues examined.and tear-specific proteins such as lacritinThis study assessed lacritin levels in tears from healthy individualsThe lipocalin 1, lactotransferrin, lacritin, lysozyme C, lipophilin A and immunoglobulin lambda chain were identified as possible biomarker candidates with significantly higher relative levels in the tear of patients with diabetic retinopathy.Lacritin is a novel human tear glycoproteinIn primates, lacritin was produced in the lacrimal gland and secreted into tear fluid.Here we show by quantitative real-time PCR that lacritin is expressed in human breast tumors, breast cancer cell lines, and normal breast. The previously reported restricted expression pattern of lacritin is therefore incorrect. Lacritin transcripts were not detected in peripheral blood which makes lacritin a potential candidate as a breast cancer marker gene\"}, {\"question\": \"What is the triple screening test performed during pregnancy measuring?\", \"ideal_answer\": [\"Alpha-fetoprotein (AFP), human chorionic gonadotropin (hCG) and unconjugated estriol (uE3)\"], \"exact_answer\": [[\"AFP\"], [\"Estriol\"], [\"hCG\"]], \"context\": \"AFP--1.18 MoM and estriol--1.29 MoM and significantly higher mean median value for the free beta-hCGalpha-fetoprotein (AFP), 1.02 for human chorionic gonadotropin (hCG) and 1.01 for unconjugated estriol (uE3alpha fetoprotein (AFP), beta human chorionic gonadotropin (beta-hCG), and unconjugated estriol (uE3) alpha-fetoprotein (AFP), intact HCG and unconjugated estriol (uE3)lpha-fetoprotein (AFP), total beta human chorionic gonadotrophin (hCG) and estriol (uE(3)alpha feto-protein, HCG and unconjugated estriolmaternal serum alpha-fetoprotein, human chorionic gonadotropin and unconjugated oestriol maternal serum alpha fetoprotein, unconjugated oestriol and human chorionic gonadotrophin\"}, {\"question\": \"What are the generic versions of Viagra?\", \"ideal_answer\": [\"Generic versions of sildenafil are Elonza, Caverta, Zenegra-100, Vega Asia, Suhagra-100, Vega, Revatio.\"], \"exact_answer\": [[\"Elonza\"], [\"Caverta\"], [\"Zenegra-100\"], [\"Vega Asia\"], [\"Suhagra-100\"], [\"Vega\"]], \"context\": \"The authors conducted a 12-week case series to study the efficacy and safety of Elonza (generic sildenafil) in PAH patients.\"}, {\"question\": \"Against which organisms has reverse vaccinology been used?\", \"ideal_answer\": [\"Reverse Vaccinology (RV) was first applied to serogroup B Neisseria meningitidis. This work induced further research of other pathogens in the same way: Porphyromonas gingivalis, Streptococcus pneumoniae, Chlamydia pneumoniae, Bacillus anthracis, group B streptococci, Helicobacter pylori and Mycobacterium tuberculosis. Concerning animal-affecting organisms, RV has been applied for vaccine design against Theileria parva, Brachyspira hyodysenteriae,  Echinococcus granulosus, Ehrlichia ruminantium, Leishmania spp, Rhipicephalus microplus and Brucella melitensis.\"], \"exact_answer\": [[\"Neisseria meningitidis (serogroup B)\"], [\"Porphyromonas gingivalis\"], [\"Streptococcus pneumoniae\"], [\"Chlamydia pneumoniae\"], [\"Bacillus anthracis\"], [\"group B streptococci\"], [\"Helicobacter pylori\"], [\"Mycobacterium tuberculosis\"], [\"Theileria parva\"], [\"Brachyspira hyodysenteriae\"], [\"Echinococcus granulosus\"], [\"Ehrlichia ruminantium\"], [\"Leishmania spp\"], [\"Rhipicephalus microplus\"], [\"Brucella melitensis\"]], \"context\": \"The potential of this new approach is illustrated by the use of reverse vaccinology for the development of a vaccine against serogroup B meningococcusThe Neisseria meningitidis serogroup B project, the first example of Reverse VaccinologyReverse vaccinology and vaccines for serogroup B Neisseria meningitidisThis process, first applied to serogroup B Neisseria meningitidis, has been termed as reverse vaccinologyA reverse vaccinology approach to swine dysentery vaccine developmentSwine dysentery (SD) is a mucohaemorrhagic colitis of pigs resulting from infection of the large intestine with the anaerobic intestinal spirochaete Brachyspira hyodysenteriaeReverse vaccinology approach identify an Echinococcus granulosus tegumental membrane protein enolase as vaccine candidateReverse vaccinology process identified E. granulosus tegumental membrane protein enolase as vaccine candidateThe application of RV to Neisseria meningitidis serogroup B represents the first success of this novel approachIdentification of Ehrlichia ruminantium proteins that activate cellular immune responses using a reverse vaccinology strategyEhrlichia ruminantium is an obligate intracellular bacterial pathogen which causes heartwater, a serious tick-borne disease of ruminants throughout sub-Saharan AfricaA reverse vaccinology approach for the identification of potential vaccine candidates from Leishmania sppA systematic, functional genomics, and reverse vaccinology approach to the identification of vaccine candidates in the cattle tick, Rhipicephalus microplus\"}, {\"question\": \"Which compounds exist that are thyroid hormone analogs?\", \"ideal_answer\": [\"Compoounds such as 3,5-diiodo-L-thyronine, T2, GC-24, CO23, DITPA, 3,5-diiodothyropropionic acid, GC-1, Tetrac, 3,3',5,5'-tetraiodo-thyroacetic acid, KB- 2115, KB - 141, thyronamines, T4-agarose, CGS 23425, D-T3, 3,3',5-triiodo-D-thyronine, 3,5-T2, 3,5-diiodo-L-thyronine, DIT, 3,5-diiodo-L-tyrosine, MIT, 3-monoiodo-L-tyrosine, triac, 3, 3',5-triiodo-thyroacetic acid, 3,5-Diiodo-4-hydroxyphenylpropionic acid, DIHPA, 3,5-Dimethyl-3'-isopropyl-L-thyronine, DIMIT, 3,5-diiodo-3'-isopropylthyroacetic acid and IpTA2 are compounds that are thyroid hormone analogs.\"], \"exact_answer\": [[\"3,5-diiodo-L-thyronine\", \"T2\"], [\"GC-24\"], [\"CO23\"], [\"DITPA\", \"3,5-diiodothyropropionic acid\"], [\"GC-1\"], [\"Tetrac\", \"3,3',5,5'-tetraiodo-thyroacetic acid\"], [\"KB-2115\"], [\"KB-141\"], [\"thyronamines\"], [\"T4-agarose\"], [\"CGS 23425\"], [\"D-T3\", \"3,3',5-triiodo-D-thyronine\"], [\"3,5-T2\", \"3,5-diiodo-L-thyronine\"], [\"DIT\", \"3,5-diiodo-L-tyrosine\"], [\"MIT\", \"3-monoiodo-L-tyrosine\"], [\"triac\", \"3,3',5-triiodo-thyroacetic acid\"], [\"3,5-Diiodo-4-hydroxyphenylpropionic acid\", \"DIHPA\"], [\"3,5-Dimethyl-3'-isopropyl-L-thyronine\", \"DIMIT\"], [\"3,5-diiodo-3'-isopropylthyroacetic acid\", \"IpTA2\"]], \"context\": \"naturally occurring iodothyronine, 3,5-diiodo-L-thyronine (T2)thyroid hormone receptor beta- and \\u03b1-selective agonists GC-24 and CO23two thyroid hormone agonists, the TR\\u03b2-selective GC-24 and the TR\\u03b1-selective CO23Diiodothyropropionic acid (DITPA) and GC-1, a noniodinated thyroid hormone analog,analogs DITPA and GC-1Tetrac inhibits binding of thyroid hormone analogs to the receptor on alphavbeta3 and lacks thyromimetic activity at this sitethyroid hormone agonist, called GC1Other selective thyromimetic, KB- 2115 and KB - 141Another class of thyroid hormone analogs, the thyronaminesT4-agarose, a formulation of thyroid hormonehe thyroid hormone analog 3,5-diiodothyropropionic acid (DITPA)The thyroid hormone analogs DITPA, T(4), and T(4)-agarose,,5-diiodothyropropionic acid (DITPA), a thyroid hormone analogthyroid hormone analogs 3,5-diiodothyropropionic acid and CGS 23425wo thyroid hormone analogs, 3,5-diidodothyropropionic acid (DITPA) and CGS 23425several thyroid hormone analogs, including T4(3,3',5,5'-tetraiodo-L-thyronine), rT3(3,3',5'-triiodo-L-thyronine), D-T3(3,3',5-triiodo-D-thyronine), 3,5-T2(3,5-diiodo-L-thyronine), DIT (3,5-diiodo-L-tyrosine), MIT (3-monoiodo-L-tyrosine), tetrac (3,3',5,5'-tetraiodo-thyroacetic acid), triac (3, 3',5-triiodo-thyroacetic acid),,5-Diiodo-4-hydroxyphenylpropionic acid (DIHPA) was found to bind specifically to bacterially expressed alpha-1 and beta-1 thyroid hormone receptorsDIHPA is a novel thyromimetic compoundIdentification of simple substituted phenols with thyromimetic activity: cardiac effects of 3,5-diiodo-4-hydroxyphenylpropionic acidThe relative affinity of the receptor for thyroid hormone analogs was triiodothyroacetic acid = T3 greater than T4 greater than tetraiodothyroacetic acid in oligodendrocyte and astrocyte nucleiThe relative binding affinity of the receptor for thyroid hormone analogs was in the order TRIAC greater than L-T3 greater than D-T3 greater than L-T4 in both cell lines3,5-Dimethyl-3'-isopropyl-L-thyronine (DIMIT) and 3,5-diiodo-3'-isopropylthyroacetic acid (IpTA2), two thyroid hormone analogs,synthetic thyroid analogs (3,5,3'-L-triiodothyronine, 3,5,3'-D-triiodothyronine, 3,3',5'-L- triiodothyronine , 3,5,3'-L-triiodothyroacetic acid and 3,5-L-diiodothyronine)In the present study, we characterized the interactions of the synthetic triiodo L-thyronine analogs and thyroid hormone nuclear receptor TR\\u03b2-selective agonists GC-1 and GC-24 with the wild type and V30M variant of human transthyretin (TTR).3-Iodothyronamine (T(1)AM) is a naturally occurring thyroid hormone metabolite with distinct biological effects that are opposite those of thyroid hormone.\"}, {\"question\": \"What genes are related to breast cancer?\", \"ideal_answer\": [\"Breast cancer is a disease in which certain cells in the breast become abnormal and multiply without control or order to form a tumor. The most common form of breast cancer begins in cells lining the ducts that carry milk to the nipple (ductal cancer). Other forms of breast cancer begin in the glands that produce milk (lobular cancer) or in other parts of the breast.\\nEarly breast cancer usually does not cause pain and may exhibit no noticeable symptoms. As the cancer progresses, signs and symptoms can include a lump or thickening in or near the breast; a change in the size or shape of the breast; nipple discharge, tenderness, or retraction (turning inward); and skin irritation, dimpling, or scaliness. These changes can occur as part of many different conditions, however. Having one or more of these symptoms does not mean that a person definitely has breast cancer.\\nIn some cases, cancerous tumors can invade surrounding tissue and spread to other parts of the body. If breast cancer spreads, cancerous cells most often appear in the bones, liver, lungs, or brain. Tumors that begin at one site and then spread to other areas of the body are called metastatic cancers.\\nA small percentage of all breast cancers cluster in families. Hereditary cancers are those associated with inherited gene mutations. Hereditary breast cancers tend to occur earlier in life than noninherited (sporadic) cases and are more likely to involve both breasts.\\n\\nVariations of the BRCA1, BRCA2, BRCT, and P53 genes increase the risk of developing breast cancer.\\nThe and NFR2, HER2 and TOP2A genes are associated with breast cancer.\"], \"exact_answer\": [[\"p53\", \"TP53\"], [\"BRCA2\"], [\"NRF2\"], [\"BRCA1\"], [\"BRCT\"], [\"HER2\", \"ERBB2\", \"RPS2\"], [\"TOP2A\"], [\"TRIM22\"]], \"context\": \"Network analysis showed increased expression of a majority of components in p53 and BRCA1 subnetworks in AA breast tumor samples, and members of the aurora B and polo-like kinase signaling pathways were also highly expressedThese results indicate that nuclear NRF2 protein plays important roles in the proliferation and/or progression of breast carcinoma, and nuclear NRF2 immunoreactivity is therefore considered a potent prognostic factor in breast cancer patientsOur previous studies in breast cancer cells indicate that 2CdA and F-ara-A are involved in epigenetic regulation of gene transcription It seems to be a new important element of ClF anticancer activity and may indicate its potential efficacy in epigenetic therapy of solid tumours, especially at early stages of carcinogenesismany molecular components that regulate HR are tumour suppressors p53, a negative regulator and breast cancer early-onset (BRCA)2, a positive regulator. Both the players not only interact with each other but also directly interact with human RAD51 (hRAD51), the key recombinase in HR Our findings suggest a rigorous p53-mediated regulation on hRAD51 functions in HR even in the presence of BRCA2coordinated replication and transcription of pericentromeric repeats enable RNA interference (RNAi)-mediated transmission of pericentromeric heterochromatin in fission yeast, which is essential for the proper function of centromeres. Rad3/ATR kinase phosphorylates histone H2A on serine-128/-129 to create \\u03b3H2A in pericentromeric heterochromatin during S phase, which recruits Brc1 through its breast cancer gene 1 protein (BRCA1) C-terminal (BRCT) domainsprotein BRCA2 affect its interactions with the recombinase RAD51 and are associated with an increased risk of cancer serial deletion mutation experiments, binding strengths were increased when the C-terminal BRC repeat was removed from BRC1-8, BRC1-5 and BRC1-3. These results may provide an insight into the effects of missense or truncation mutations in BRCA2 in canine tumours.Triple-negative breast cancers (TNBC) do not represent a single disease subgroup and are often aggressive breast cancers with poor prognoses. Unlike estrogen/progesterone receptor and HER2 (human epidermal growth factor receptor 2) breast cancers, which are responsive to targeted treatments, there is no effective targeted therapy for TNBC, although approximately 50% of patients respond to conventional chemotherapies, including taxanes, anthracyclines, cyclophosphamide, and platinum saltsGenetic BRCA2 insufficiency is associated with breast cancer development; however, in sporadic breast cancer cases, high BRCA2 expression is paradoxically correlated with poor prognosisHER-2 and TOP2A gene amplification showed a tendency to be associated with larger tumor size, positive lymph node status, high level of apoptotic and proliferative indexes, and low level of p53 and Bcl-2 expression, which all together indicate group of patients with similar outcome during the progression of the disease\"}, {\"question\": \"List fluorescent reporter proteins.\", \"ideal_answer\": [\"Fluorescent and luminescent reporter genes have become popular tools for the real-time monitoring of gene expression in living cells:\\ngreen fluorescent protein\\nTimer\\nred fluorescent protein\\nyellow fluorescent protein\\nbeta-phycoerythrin\\ncoral fluorescent reporter protein\\nenhanced green fluorescent reporter protein\\nmCherry\"], \"exact_answer\": [[\"green fluorescent protein\", \"Gfp\"], [\"Timer\", \"DsRed1-E5\"], [\"red fluorescent protein\", \"dsRed\"], [\"yellow fluorescent protein\"], [\"beta-phycoerythrin\"], [\"coral fluorescent reporter protein\"], [\"enhanced green fluorescent reporter protein\"], [\"mCherry\"]], \"context\": \"Fluorescent Timer, or DsRed1-E5, is a mutant of the red fluorescent protein, dsRed, in which fluorescence shifts over time from green to red as the protein matures.green fluorescent protein (Gfp)Characterization of flavin-based fluorescent proteins: an emerging class of fluorescent reporters.Flavin-based fluorescent proteins (FbFPs)mCherry was employed as a reporter protein yellow fluorescent proteinred cytoplasmic fluorescent reporter protein. red fluorescent reporter protein enhanced green fluorescent reporter proteinFluorescent and luminescent reporter genes have become popular tools for the real-time monitoring of gene expression in living cells.  fluorescent reporter protein [green fluorescent protein (GFP) and DsRed2]coral fluorescent reporter protein driving a cassette for the enhanced green \\\"live\\\" fluorescent reporter protein (eGFP) GFP as a fluorescent reporter protein proteinred fluorescent reporter protein (RFP) gene. The red fluorescent protein eqFP611the green fluorescent reporter protein gene  red fluorescent reporter proteinbeta-phycoerythrin, a fluorescent reporter protein derived from algae live-cell fluorescent reporter protein, Timerred fluorescent reporter protein\"}, {\"question\": \"Which transcription factors are known as the four (4) \\\"Yamanaka factors\\\" that have been used to create induced pluripotent stem cells  (iPSCs)?\", \"ideal_answer\": [\"Fibroblasts can be reprogrammed into induced pluripotent stem cells (iPSCs) by the application of Yamanaka factors (OSKM). In particular, the mechanisms how the Yamanaka factors (Oct4, Sox2, Klf4, and c-Myc) directly drive reprogramming and which additional components are involved are still not yet understood.\", \"Through the ectopic expression of four transcription factors, Oct4, Klf4, Sox2 and cMyc, human somatic cells can be converted to a pluripotent state, generating so-called induced pluripotent stem cells (iPSCs)(1-4).  Delivery of the transcription factors Oct4, Klf4, Sox2 and c-Myc via integrating viral vectors has been widely employed to generate induced pluripotent stem cell (iPSC) lines from both normal and disease-specific somatic tissues, providing an invaluable resource for medical research and drug development.  \", \"Through the ectopic expression of four transcription factors, Oct4, Klf4, Sox2 and cMyc, human somatic cells can be converted to a pluripotent state, generating so-called induced pluripotent stem cells (iPSCs)(1-4).Fibroblasts can be reprogrammed into induced pluripotent stem cells (iPSCs) by the application of Yamanaka factors (OSKM), but the mechanisms underlying this reprogramming remain poorly understood.\"], \"exact_answer\": [[\"Oct4\", \"Oct 3/4\"], [\"Sox2\"], [\"Klf4\"], [\"c-Myc\"]], \"context\": \"The recently established reprogramming of somatic cells into induced pluripotent stem cells (iPSCs) by Takahashi and Yamanaka represents a valuable tool for future therapeutic applications.In particular, the mechanisms how the Yamanaka factors (Oct4, Sox2, Klf4, and c-Myc) directly drive reprogramming and which additional components are involved are still not yet understood.Here we show that Yamanaka factors (OCT4, SOX2, MYC, and KLF4)-expressing EV can also reprogram adult peripheral blood mononuclear cells (PBMNCs) into pluripotency, yet at a very low efficiency.The CytoTune\\u2122-iPS Reprogramming Kit contains four SeV-based reprogramming vectors, each capable of expressing one of the four Yamanaka factors (i.e., Oct4, Sox2, Klf4, and c-Myc) and are optimized for generating iPSCs from human somatic cells.Delivery of the transcription factors Oct4, Klf4, Sox2 and c-Myc via integrating viral vectors has been widely employed to generate induced pluripotent stem cell (iPSC) lines from both normal and disease-specific somatic tissues, providing an invaluable resource for medical research and drug development.We demonstrate that residual expression of the Yamanaka factors prevents iPSCs from acquiring the transcriptional program exhibited by embryonic stem cells (ESCs) and that the expression profiles of iPSCs generated with and without c-Myc are indistinguishable.Fibroblasts can be reprogrammed into induced pluripotent stem cells (iPSCs) by the application of Yamanaka factors (OSKM), but the mechanisms underlying this reprogramming remain poorly understood.We also discovered that HMGA1 enhances cellular reprogramming of somatic cells to iPSCs together with the Yamanaka factors (OCT4, SOX2, KLF4, cMYC - OSKM). Through the ectopic expression of four transcription factors, Oct4, Klf4, Sox2 and cMyc, human somatic cells can be converted to a pluripotent state, generating so-called induced pluripotent stem cells (iPSCs)(1-4).We have shown the generation of transgene-free human iPSCs from patients with different lung diseases using a single excisable polycistronic lentiviral Stem Cell Cassette (STEMCCA) encoding the Yamanaka factors(6).These iPSCs were established in a feeder-free system by lentiviral transduction of the Yamanaka factors, Oct3/4, Sox2, Klf4, and c-Myc.Induced pluripotent stem cells (iPSCs) are created by the reprogramming of somatic cells via overexpression of certain transcription factors, such as the originally described Yamanaka factors: Oct4, Sox2, Klf4, and c-Myc (OSKM)iPSC production was first achieved by transducing, with the use of retroviral vectors, four specific transcription factors: Oct4, Klf4, Sox2 and c-Myc (OKSM), into primary cells in culture Takahashi and Yamanaka, (Cell 126(4):663-676, 2006)\"}, {\"question\": \"Which are the most abundant human lincRNA?\", \"ideal_answer\": [\"MALAT1 (metastasis-associated lung adenocarcinoma transcript 1) locus is misregulated in many human cancers and produces an abundant long nuclear-retained noncoding RNA. MALAT/NEAT2 highly abundant, its expression is strongly regulated in many tumor entities including lung adenocarcinoma and hepatocellular carcinoma as well as physiological processes, and it is associated with many RNA binding proteins and highly conserved throughout evolution. H19 large intergenic non-coding RNA (lincRNA) is one of the most highly abundant and conserved transcripts in mammalian development, being expressed in both embryonic and extra-embryonic cell lineages, yet its physiological function is unknown. Our genome-wide screens in two mammalian species reveal no more than three abundant large non-coding polyadenylated RNAs in the nucleus; the canonical large noncoding RNA XIST and NEAT1 and NEAT2.\"], \"exact_answer\": [[\"MALAT1\", \"MALAT-1\", \"NEAT2\"], [\"NEAT1\"], [\"H19\"], [\"XIST\"]], \"context\": \"One abundant such RNA, ci-ankrd52, largely accumulates to its sites of transcription, associates with elongation Pol II machinery, and acts as a positive regulator of Pol II transcriptionlong noncoding RNA MALAT1 (metastasis-associated lung adenocarcinoma transcript 1), also known as MALAT-1 or NEAT2 (nuclear-enriched abundant transcript 2), is a highly conserved nuclear noncoding RNA (ncRNA) and a predictive marker for metastasis development in lung cancer.Because viruses borrow molecular mechanisms from their hosts, we searched highly abundant human long-noncoding RNAs and identified putative ENE-like structures in metastasis-associated lung adenocarcinoma transcript 1 (MALAT1) and multiple endocrine neoplasia-\\u03b2 (MEN\\u03b2) RNAs. MALAT1 (metastasis-associated lung adenocarcinoma transcript 1) locus is misregulated in many human cancers and produces an abundant long nuclear-retained noncoding RNA.Many long noncoding RNAs (lncRNAs) appear to have epigenetic regulatory function in humans, including HOTAIR and XIST.metastasis-associated lung adenocarcinoma transcript 1, MALAT1, is a long non-coding RNA (lncRNA) that has been discovered as a marker for lung cancer metastasis.It is highly abundant, its expression is strongly regulated in many tumor entities including lung adenocarcinoma and hepatocellular carcinoma as well as physiological processes, and it is associated with many RNA binding proteins and highly conserved throughout evolution.Thus, loss of the abundant nuclear long ncRNA MALAT1 is compatible with cell viability and normal development.bundant expression of the long noncoding (lnc) PAN (polyadenylated nuclear) RNA by the human oncogenic gammaherpesvirus Kaposi's sarcoma-associated herpesvirus (KSHV) depends on a cis-element called the expression and nuclear retention element (ENE)alat1 is an abundant long, noncoding RNA that localizes to nuclear bodies known as nuclear speckles, which contain a distinct set of pre-mRNA processing factors.Malat1 (metastasis associated lung adenocarcinoma transcript 1) is among the most abundant and highly conserved lncRNAs, and it exhibits an uncommon 3'-end processing mechanismH19 large intergenic non-coding RNA (lincRNA) is one of the most highly abundant and conserved transcripts in mammalian development, being expressed in both embryonic and extra-embryonic cell lineages, yet its physiological function is unknown. Our genome-wide screens in two mammalian species reveal no more than three abundant large non-coding polyadenylated RNAs in the nucleus; the canonical large noncoding RNA XIST and NEAT1 and NEAT219, which is one of the most abundantly expressed imprinted genes during mammalian embryonic and foetal development, has been cloned from a ruminantH19 mRNA is highly abundant in most ovine embryonic and foetal tissues of mesodermal and endodermal origins but was not detected in tissues of ectodermal origin such as the trophectoderm and the foetal brainAbundant expression of H19 was evident in fetal bladder but was absent in normal adult bladderurthermore, we discuss results showing an abundant expression of H19 gene in some adenocarcinomas of bad prognosis, in the context of the otherwise established tumor-suppressor role of this gene, or the strictly controlled gene dosage, which could be overridden in these particular cases.One of the first lncRNA genes discovered was MALAT1, the metastasis-associated lung adenocarcinoma transcript 1,21 later also referred to as NEAT2 for nuclear-enriched abundant transcript 2. MALAT1 is highly abundant and is expressed in many healthy organs, most strongly in pancreas and lung\"}, {\"question\": \"Which drugs have been found effective for the treatment of chordoma?\", \"ideal_answer\": [\"Established chordoma cell lines, and patient-derived primary cell cultures, as well as chordoma tumors in vivo were found to be sensitive to treatment with bortezomib, vincristine, doxorubicin, etoposide, cisplatin, fludarabine and SD-1029 Stat3 inhibitor. Moreover, percutaneous intratumoral injection with pingyangmycin lipiodol emulsion was shown to be effective against chordoma. It should be stressed that combination treatment with the use of the above drugs was always able to increase the therapeutic potency.\"], \"exact_answer\": [[\"bortezomib\"], [\"vincristine\"], [\"doxorubicin\"], [\"etoposide\"], [\"cisplatin\"], [\"fludarabine\"], [\"SD-1029 Stat3 inhibitor\"], [\"pingyangmycin lipiodol emulsion\"]], \"context\": \"The combination treatment of bortezomib with topoisomerase I and II inhibitors increased the therapeutic potency in U-CH2 and patient-derived primary cultures.Vincristine, doxorubicin, etoposide, cisplatin, and fludarabine, each at a concentration of 10 \\u03bcM, decreased the number of chordoma cells when given alone down to 11%, 0%, 30%, 67%, and 3%, respectively. Etoposide and cisplatin, each at a concentration of 10 \\u03bcM, reduced the percentage of viable chordoma cells in a more effective way when given with 1 \\u03bcM ATRA simultaneously, reducing the number of viable cells to 14% and 9%, respectively.Percutaneous intratumoral injection with pingyangmycin lipiodol emulsion for the treatment of recurrent sacrococcygeal chordomasPreliminary results showed that PIIT with pingyangmycin lipiodol emulsion under fluoroscopic guidance is effective and safe and may be considered as a treatment option.Percutaneous intratumoral injection with pingyangmycin lipiodol emulsion for treatment of recurrent sacrococcygeal chordomasDuring the follow-up (median time of 21.7 months, range 10-26 months), all the patients showed obviously reduced tumor size and VAS, and partial remission was achieved in 6 patients and stable disease (SD) in 1 patient.Fluoroscopy-guided percutaneous intratumoral injection of PLE can be effective and safe and may serve as a alternative for treatment of recurrent sacrococcygeal chordomas.The expression of Stat3 signaling cascade was inhibited in all chordoma cell lines after treatment with SD-1029. The cytotoxicity of the combination of SD-1029 and chemotherapeutic drugs is significantly better than either agent alone. Phosphorylation of Stat3 in chordoma cells in vitro and cellular proliferation in three-dimensional culture were inhibited by SD-1029.In conclusion, the Stat3 pathway is constitutively activated in chordomas and the level of expression may serve as a predictor for prognosis. Blockade of the Stat3 pathway represents a potential strategy for future treatment.\"}, {\"question\": \"Which components of the stress granules are known to be related to motor neuron degeneration in Amyotrophic Lateral Sclerosis?\", \"ideal_answer\": [\"Of note, both ALS and FTD are characterized by pathological inclusions, where some well-known SG markers localize with the ALS related proteins TDP-43 and FUS.\", \"TDP-43 and FUS have been identified as key proteins in the pathogenesis of some cases of ALS. Although their role in motor neuron degeneration is not yet known, TDP-43 and FUS have been shown to accumulate in RNA stress granules (SGs) in cell models and in spinal cord tissue from Amyotrophic Lateral Sclerosis (ALS) patients.\"], \"exact_answer\": [[\"TDP-43\"], [\"FUS\"]], \"context\": \"Here, we demonstrate that C-terminal ALS mutations disrupt the nuclear localizing signal (NLS) of FUS resulting in cytoplasmic accumulation in transfected cells and patient fibroblasts. FUS mislocalization is rescued by the addition of the wild-type FUS NLS to mutant proteins. We also show that oxidative stress recruits mutant FUS to cytoplasmic stress granules where it is able to bind and sequester wild-type FUS.Recently, TDP-43 has been identified as a key protein in the pathogenesis of some cases of ALS. Although the role of TDP-43 in motor neuron degeneration is not yet known, TDP-43 has been shown to accumulate in RNA stress granules (SGs) in cell models and in spinal cord tissue from ALS patients. in response to oxidative stress or heat shock conditions in cultures and in vivo, the ALS-linked FUS mutants, but not wild-type FUS, assembled into perinuclear stress granules in proportion to their cytoplasmic expression levels. These findings demonstrate a potential link between FUS mutations and cellular pathways involved in stress responses that may be relevant to altered motor neuron homeostasis in ALS.both stress granules (TIA-1 immunoreactive) and processing bodies (P-bodies; XRN-1 immunoreactive) were more prevalent in ALS motor neurons than in controls and demonstrated strong co-localization with TDP-43.These data suggest that NFL mRNA processing is fundamentally altered in ALS spinal motor neurons to favour compartmentalization within both stress granules and P-bodies, and that TDP-43 plays a fundamental role in this process.Transactive response DNA-binding protein 43 (TDP-43) forms abnormal ubiquitinated and phosphorylated inclusions in brain tissues from patients with amyotrophic lateral sclerosis (ALS) and frontotemporal lobar degeneration. TDP-43 is a DNA/RNA-binding protein involved in RNA processing, such as transcription, pre-mRNA splicing, mRNA stabilization and transport to dendrites. We found that in response to oxidative stress and to environmental insults of different types TDP-43 is capable to assemble into stress granules (SGs), ribonucleoprotein complexes where protein synthesis is temporarily arrested. Of note, both ALS and FTD are characterized by pathological inclusions, where some well-known SG markers localize with the ALS related proteins TDP-43 and FUS.Mutations in fused in sarcoma (FUS), a DNA/RNA binding protein, have been associated with familial amyotrophic lateral sclerosis (fALS), which is a fatal neurodegenerative disease that causes progressive muscular weakness and has overlapping clinical and pathologic characteristics with frontotemporal lobar degeneration.Our results indicate that mutant-FUS alters the dynamic properties of stress granules, which is consistent with a gain-of-toxic mechanism for mutant-FUS in stress granule assembly and cellular stress response.Expression of ALS-linked FUS mutations resulted in their assembly into cytoplasmic stress granules (SGs), cellular structures that package mRNA and RNA-binding proteins during cell stress.TDP-43 is an RNA-binding protein linked to amyotrophic lateral sclerosis (ALS) that is known to regulate the splicing, transport, and storage of specific mRNAs into stress granulesAmyotrophic lateral sclerosis (ALS) is a neurodegenerative disorder consisting of progressive loss of motor neurons. TDP-43 has been identified as a component of ubiquitin-immunoreactive inclusions of motor neurons in ALS.SGs have been linked to several pathologies including inflammatory diseases, cancer, viral infection, and neurodegenerative diseases such as amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD)PTE significantly reduced the incorporation of R521C FUS/TLS into stress granules under stress conditions.\"}, {\"question\": \"Which could be some of the possible causes of hypersomnia?\", \"ideal_answer\": [\"Sleep-wake disturbances (SWD) with hypersomnia are common after traumatic brain injury (TBI), with decreased CSF levels of hypocretin-1, a wake-promoting neurotransmitter, in cases of sleep apnea, as well as in up to half the patients with dementia, particularly in vascular dementia, Korsakow syndrome, Parkinson's disease, and depression, where the alteration of sleep architecture may be pronounced, whereas in Alzheimer's disease prominent hypersomnolence or insomnia is typically only found in later stages of the diseases.\"], \"exact_answer\": [[\"traumatic brain injury (TBI)\"], [\"sleep apnea\"], [\"dementia\"]], \"context\": \"Sleep-wake disturbances (SWD) are common after traumatic brain injury (TBI). In acute TBI, we recently found decreased CSF levels of hypocretin-1, a wake-promoting neurotransmitterHypocretin levels 6 months after TBI were significantly lower in patients with post-traumatic EDSThese results suggest that sleep-wake disturbances, particularly EDS, fatigue and hypersomnia are common after TBI, and significantly impair quality of lifeWhen sleepiness is excessive, undesirable, inappropriate or unexplained, it often indicates a clinical disorder that is generically termed hypersomnia. One of the leading causes of hypersomnia is sleep apneaA clinically relevant sleep-wake disturbance is found in up to half the patients with dementia, and the sundowning agitation is a common cause of institutionalisation of demented geriatric patients. The circadian rhythm of demented patients is levelled off with increased daytime sleep and disrupted night sleep. Particularly in vascular dementia, Korsakow syndrome, Parkinson's disease, and depression the alteration of sleep architecture may be pronounced, whereas in Alzheimer's disease prominent hypersomnolence or insomnia is typically only found in later stages of the diseasesWe report on an 83 yr old man with hypersomnia and central sleep apnoea (CSA)One of the leading causes of hypersomnia is sleep apnea.In this article, the causes of hypersomnia are detailed following the conventional classification of hypersomnic syndromes: narcolepsy, idiopathic hypersomnia, recurrent hypersomnia, insufficient sleep syndrome, medication- and toxin-dependent sleepiness, hypersomnia associated with psychiatric disorders, hypersomnia associated with neurological disorders, posttraumatic hypersomnia, infection (with a special emphasis on the differences between bacterial and viral diseases compared with parasitic diseases, such as sleeping sickness) and hypersomnia, hypersomnia associated with metabolic or endocrine diseases, breathing-related sleep disorders and sleep apnea syndromes, and periodic limb movements in sleep.In the absence of sleep deprivation (either because of behavioral or medical causes) or pharmacologically induced sleepiness, hypersomnia is a manifestation of one of the central disorders of hypersomnolence, such as narcolepsy types 1 and 2, idiopathic hypersomnia, and recurrent hypersomnias such as Kleine-Levin syndrome.Sleep apnea syndrome (SAS), narcolepsy, chronic sleep deprivation (insufficiency), and restless legs/periodic limb movements in sleep syndrome (RLS/PLMS) represent the most common causes of hypersomnia.Hypocretin (orexin) deficiency in narcolepsy and primary hypersomnia.One of the leading causes of hypersomnia is sleep apnea.\"}, {\"question\": \"List BRAF inhibitors that have been tested in clinical trials for treatment of melanoma patients\", \"ideal_answer\": [\"Vemurafenib and dabrafenib are BRAF inhibitors that have been tested in clinical trials for treatment of melanoma patients.\"], \"exact_answer\": [[\"Vemurafenib\"], [\"Dabrafenib\"]], \"context\": \"Combined BRAF (Dabrafenib) and MEK inhibition (Trametinib) in patients with BRAFV600-mutant melanoma experiencing progression with single-agent BRAF inhibitor. Here, we report patients treated with combination therapy after disease progression with BRAF inhibitor treatment administered before study enrollment (part B; n = 26) or after cross-over at progression with dabrafenib monotherapy (part C; n = 45). This randomized phase 3 study evaluated the combination of the BRAF inhibitor vemurafenib and the MEK inhibitor cobimetinib. METHODS: All patients treated with the BRAF inhibitors vemurafenib or dabrafenib or combination BRAF inhibitor and mitogen-activated protein kinase kinase (MEK) inhibitor therapy at Westmead Hospital, Sydney, Australia underwent regular dermatological assessments for the duration of therapy.We assessed the safety and efficacy of combined BRAF inhibition with vemurafenib and MEK inhibition with cobimetinib in patients with advanced BRAF-mutated melanoma. We included individuals who had either recently progressed on vemurafenib or never received a BRAF inhibitor. However, it is becoming evident that the effects of paradoxical mitogen-activated protein kinase pathway activation by BRAF inhibitors in non-BRAF-mutant cells needs to be taken into account, which may be implicated in the problems encountered in the first clinical trial testing a combination of the BRAF inhibitor vemurafenib with ipilimumab (anti-CTLA4), with significant liver toxicities.The BRAF inhibitor dabrafenib recently was approved for use in patients with BRAF V600-mutated metastatic melanoma.His treatment course to date has included surgery, adjuvant radiotherapy, and interferon, metastasectomies, granulocyte-macrophage colony-stimulating factors, a clinical trial with the BRAF inhibitor vemurafenib (PLX-4032), clinical trial with combination BRAF plus MEK inhibition with vemurafenib plus GDC-0973, and combination targeted and immune therapy with vemurafenib plus the anti-CTLA4 antibody ipilimumab.Thirty-one patients commenced dabrafenib therapy including six individuals who had progressed on a prior BRAF-inhibitor treatment.Population pharmacokinetics of dabrafenib, a BRAF inhibitor: effect of dose, time, covariates, and relationship with its metabolites.Dabrafenib is a BRAF kinase inhibitor indicated for the treatment of BRAF V600E mutation-positive melanoma. This was a prospective study of melanoma patients harboring the BRAF V600 mutation and treated with iBRAF within a clinical trial (dabrafenib) or as part of an expanded access program (vemurafenib).We investigated the BRAF kinase inhibitor vemurafenib in patients with advanced melanoma with symptomatic brain metastases.This discovery led to the development of BRAF kinase inhibitors like vemurafenib and dabrafenib. CONCLUSIONS: Dabrafenib joins vemurafenib to confirm the superior clinical outcome of the BRAF inhibitors when compared with dacarbazine in patients with BRAF(V600E)-positive advanced melanoma. Tumor genetic analyses of patients with metastatic melanoma treated with the BRAF inhibitor dabrafenib (GSK2118436).PURPOSE: Dabrafenib is a selective inhibitor of V600-mutant BRAF kinase, which recently showed improved progression-free survival (PFS) as compared with dacarbazine, in metastatic melanoma patients. Phase II trial (BREAK-2) of the BRAF inhibitor dabrafenib (GSK2118436) in patients with metastatic melanoma.PURPOSE: Dabrafenib (GSK2118436) is a potent inhibitor of mutated BRAF kinase.Cutaneous squamous cell carcinoma (cSCC) is a concerning toxicity with BRAF inhibitors in the treatment for melanoma. While the two drugs shown to improve survival, vemurafenib, and dabrafenib, have similar efficacy, the reported rates of cSCC are quite different.PURPOSE To assess pharmacodynamic effects and intrinsic and acquired resistance mechanisms of the BRAF inhibitor vemurafenib in BRAF(V600)-mutant melanoma, leading to an understanding of the mechanism of action of vemurafenib and ultimately to optimization of metastatic melanoma therapy. BACKGROUND: The cobas 4800 BRAF V600 Mutation Test is a CE-marked and FDA-approved in vitro diagnostic assay used to select patients with metastatic melanoma for treatment with the selective BRAF inhibitor vemurafenibThe purpose of this study is to review the development of BRAF inhibitors, with emphasis on the trials conducted with dabrafenib (GSK2118436) and the evolving role of dabrafenib in treatment for melanoma patients.Dabrafenib inhibits the mutant BRAF (BRAF(mut)) protein in melanomas with BRAF(V600E) and BRAF(V600K) genotypes.To address this problem, we conducted a phase 1 and 2 trial of combined treatment with dabrafenib, a selective BRAF inhibitor, and trametinib, a selective MAPK kinase (MEK) inhibitor.Cutaneous manifestations of dabrafenib (GSK2118436): a selective inhibitor of mutant BRAF in patients with metastatic melanoma.We conducted a systematic prospective dermatological review of all patients enrolled at a single institution in the phase I/II clinical trial of the mutant BRAF inhibitor dabrafenib (GSK2118436).CONCLUSIONS: Administration of the mutant BRAF inhibitor dabrafenib is associated with induction of keratinocytic proliferation, which in some cases develops features of low-grade malignancy.Neoplastic leptomeningitis presenting in a melanoma patient treated with dabrafenib (a V600EBRAF inhibitor): a case report.For this reason, she was enrolled into another clinical trial with the GSK2118436 BRAF inhibitor, dabrafenib, as a second line of therapy.BACKGROUND: The development of keratoacanthomas (KAs) and well-differentiated squamous cell carcinomas (SCCs) is a known adverse effect of novel BRAF inhibitors such as vemurafenib.OBSERVATIONS: We describe a patient with stage IV melanoma who received the BRAF inhibitor vemurafenib (recently approved by the US Food and Drug Administration) as part of a clinical trial and developed numerous diffuse, pathology-proven KAs and SCCs. CONCLUSIONS: Given vemurafenib's recent approval by the US Food and Drug Administration, we provide a timely case report on the effective use of PDT in the treatment of BRAF inhibitor-associated KAs and SCCs. The BRAF inhibitor, vemurafenib, has shown up to 78% objective response rates in melanoma patients harboring the BRAF-V600E mutation but not in patients lacking the mutation. The oral BRAF inhibitor vemurafenib (PLX4032) frequently produced tumor regressions in patients with BRAF V600-mutant metastatic melanoma in a phase 1 trial and improved overall survival in a phase 3 trial. CONTEXT: A polymerase chain reaction-based companion diagnostic (cobas 4800 BRAF V600 Mutation Test) was recently approved by the US Food and Drug Administration to select patients with BRAF-mutant metastatic melanoma for treatment with the BRAF inhibitor vemurafenib. However, in the recent clinical trial of the BRAF inhibitor, vemurafenib (PLX4032), a significant percentage of BRAF V600E mutant melanoma patients did not meet the RECIST criteria for a response. Vemurafenib (RG7204/PLX4032), a small-molecule inhibitor of mutant BRAF, has shown striking clinical efficacy in BRAFV600 mutant melanoma, creating the need for a well-validated companion diagnostic to select patients for treatment.BACKGROUND: Phase 1 and 2 clinical trials of the BRAF kinase inhibitor vemurafenib (PLX4032) have shown response rates of more than 50% in patients with metastatic melanoma with the BRAF V600E mutation.Vemurafenib is the first molecularly targeted therapy to be licensed in the US and Europe for treatment of advanced melanomaClinical trials to test the efficacy of vemurafenib in combination with immunomodulatory agents, such as ipilimumab, and MAPK kinase (MEK) inhibitors, such as GDC-0973, in the treatment of advanced melanoma are currently underway\"}, {\"question\": \"which are the risk factors for sudden cardiac death in patients with hypertrophic cardiomyopathy?\", \"ideal_answer\": [\"The following risk factors for sudden cardiac death in patients with hypertrophic cardiomyopathy have been identified: 1) previous cardiac arrest; 2) sustained ventricular tachycardia; 3) family history of sudden cardiac death; 4) high-risk genetic mutations; 5) unexplained syncope; 6) non-sustained ventricular tachycardia; 7) hypotensive response to exercise; 8) marked left ventricular hypertrophy; 9) J-wave on ECG and 10) myocardial fibrosis using late gadolinium enhancement\"], \"exact_answer\": [[\"myocardial fibrosis using late gadolinium enhancement\"], [\"J-wave on ECG\"], [\"Extreme left ventricular hypertrophy (>30mm)\"], [\"Nonsustained ventricular tachycardia\"], [\"Sustained ventricular tachycardia\"], [\"previous cardiac arrest\"], [\"family history of sudden cardiac death\"], [\"unexplained syncope\"], [\"hypotensive response to exercise\"], [\"high-risk genetic mutations\"]], \"context\": \"Cardiac magnetic resonance (CMR) is used in the diagnosis and risk stratification of hypertrophic cardiomyopathy (HCM) and can detect myocardial replacement fibrosis (anindependent predictor of adverse cardiac outcomes) using late gadolinium enhancement (LGE). J wave is associated with increased risk of sudden cardiac arrest in patients with hypertrophic cardiomyopathy.The J wave may be a risk factor for SCA in patients with HCM.Extreme left ventricular hypertrophy was the most common risk factor present (alone or in combination with other markers) in patients experiencing primary prevention interventions (17 of 26 [65%]).Extreme left ventricular hypertrophy was most frequently associated with appropriate interventions.In conclusion, it is demonstrated that LGE has incremental value in addition to clinical risk factors for risk stratification and management of patients with HC.Fibrosis as detected by CMR should be evaluated as an additional risk factor to further delineate risk of SCD in carriers of an HCM causing mutation.Nonsustained VT was the only predictive risk factor (RF) for an appropriate ICD intervention in the PP (positive predictive value 22%, negative predictive value 96%).Nonsustained ventricular tachycardia seems to be the most predictive RF for appropriate device discharges. However, LGE was strongly associated with surrogates of arrhythmia and remained a significant associate of subsequent SCD and/or ICD discharge after controlling for other variables. If replicated, LGE may be considered an important risk factor for sudden death in patients with HCM.In conclusion, our cohort study results do not support LVOTO as an independent risk factor for SD in patients with hypertrophic cardiomyopathy.The appearance of a \\\"hump\\\" at the ST segment during exercise testing appears to be a risk factor for SCD in patients with HCM. However, further studies are necessary to validate this finding in larger populations and to elucidate the mechanism of the appearance of the \\\"hump.\\\"The ICD is definitely indicated for secondary prevention of sudden death in patients with HCM who have survived a cardiac arrest with documented ventricular fibrillation, or experienced one or more episodes of sustained ventricular tachycardia.A number of risk markers are used to assess the magnitude of risk, including family history of premature sudden death; extreme left ventricular (LV) hypertrophy (> 30 mm) in young patients; nonsustained ventricular tachycardia on Holter electrocardiographic recording; unexplained (not neurally mediated) syncope, particularly in young patients; and blood pressure decrease or inadequate increase during upright exercise.Measured risk factors for sudden death included family history of sudden death, massive left ventricular hypertrophy, nonsustained ventricular tachycardia on Holter monitoring, and unexplained prior syncope.The following risk factors were analyzed: 1) previous cardiac arrest or sustained ventricular tachycardia; 2) family history of sudden cardiac death; 3) high-risk genetic mutations; 4) syncope; 5) non-sustained ventricular tachycardia; 6) hypotensive response to exercise; and 7) marked left ventricular hypertrophy.Family history of sudden death was associated with a positive predictive value of 25% for appropriate therapies, 40% for syncope and 50% for non-sustained ventricular tachycardia. In patients with an implantable cardioverter-defibrillator for primary prevention, non-sustained ventricular tachycardia was the risk factor with the highest predictive value. It is concluded that there is an association of the mutation Arg719Trp in the beta-myosin heavy chain with sudden cardiac death in a young child. \"}, {\"question\": \"Which proteins are related to the loss of cell-cell adhesion during EMT (epithelial-mesenchymal transition)?\", \"ideal_answer\": [\"Transcriptional and post-transcriptional regulatory mechanisms mediated by several inducers of EMT, in particular the ZEB and Snail factors, downregulate the expression and/or functional organization of core polarity complexes. Functional loss of the cell-cell adhesion molecule E-cadherin is an essential event for epithelial-mesenchymal transition (EMT), a process that allows cell migration during embryonic development and tumour invasion. Recently, we found that aPKC can also phosphorylate Par6 to drive EMT and increase the migratory potential of non-small cell lung cancer cells. We propose that the regulation of EMT by SIRT1 involves modulation of, and cooperation with, the EMT inducing transcription factor ZEB1. Knockdown of Numb by shRNA in MDCK cells led to a lateral to apical translocation of E-cadherin and beta-catenin, active F-actin polymerization, mis-localization of Par3 and aPKC, a decrease in cell-cell adhesion and an increase in cell migration and proliferation. Growth factors such as TGFb and EGF have also been shown to be related to EMT.\"], \"exact_answer\": [[\"ZEB\"], [\"SNAIL\"], [\"SIRT1\"], [\"EGF\"], [\"TGFb\"], [\"aPKC\"], [\"Par6\"]], \"context\": \"conserved polarity proteins Par6 and aPKC regulate cell polarization processes In addition to signaling pathways that alter genetic programes that trigger the loss of cell-cell adhesion, alternative pathways can alter cell plasticity to regulate cell-cell cohesion and increase invasive potentialOne such pathway involves TGF\\u03b2-induced phosphorylation of Par6Recently, we found that aPKC can also phosphorylate Par6 to drive EMT and increase the migratory potential of non-small cell lung cancer cellsHere, we propose a novel mechanism through which the nicotinamide adenine dinucleotide-dependent histone deacetylase SIRT1 regulates EMT in prostate cancer cells through cooperation with the EMT inducing transcription factor ZEB1In contrast, silencing SIRT1 in metastatic prostate tumor cells restores cell-cell adhesion and induces a shift toward an epithelial morphology concomitant with increased expression of E-cadherin and decreased expression of mesenchymal markersWe also found that SIRT1 has a physiologically relevant role in endogenous EMT induced by EGF signaling in prostate cancer cellsWe propose that the regulation of EMT by SIRT1 involves modulation of, and cooperation with, the EMT inducing transcription factor ZEB1We thus identify a necessary role for ZEB1 in SIRT1-mediated EMTNumb has been shown to play an important role in the proper functions of Par protein complex and in cell-cell junctions, both of which are associated with EMTRecently, we showed that Numb is capable of binding to both Par3 and E-cadherinKnockdown of Numb by shRNA in MDCK cells led to a lateral to apical translocation of E-cadherin and beta-catenin, active F-actin polymerization, mis-localization of Par3 and aPKC, a decrease in cell-cell adhesion and an increase in cell migration and proliferationThis is coupled with the transcriptional repression of E-cadherin, often mediated by Snail1, Snail2 and Zeb1/deltaEF1.EGF induces EMT in the breast cancer cell line PMC42-LA and the kinase inhibitor staurosporine (ST) induces EMT in embryonic neural epithelial cells, with F-actin de-bundling and disruption of cell-cell adhesion, via inhibition of aPKCST in combination with EGF directed a greater EMT via actin depolymerisation and focal contact size reduction, resulting in a loosening of cell-ECM attachment along with Snail1-Zeb1/deltaEF1 inductionThis appeared fundamentally different to the EGF-induced EMT, highlighting the multiple pathways which can regulate EMTunctional loss of the cell-cell adhesion molecule E-cadherin is an essential event for epithelial-mesenchymal transition (EMT), a process that allows cell migration during embryonic development and tumour invasionTranscriptional and post-transcriptional regulatory mechanisms mediated by several inducers of EMT, in particular the ZEB and Snail factors, downregulate the expression and/or functional organization of core polarity complexesTGFbeta has pleiotropic effects that range from regulation of proliferation and apoptosis to morphological changes and epithelial-mesenchymal transition (EMT)Morphological changes were accompanied by decrease in E-cadherin and increased Snail expression but the mesenchymal markers (N-cadherin, SMAalpha and Vimentin) studied remained unchangedNeutralising E-cadherin function promoted the invasion and inhibited the aggregation of endocrine-sensitive MCF7 cells, whilst having little effect on the behaviour of TamR cells\"}, {\"question\": \"List common features of Shapiro syndrome\", \"ideal_answer\": [\"Shapiro syndrome is a rare entity, comprising a triad of recurrent hypothermia, hyperhidrosis and congenital agenesis of the corpus callosum. Hypermelatoninemia has also been described in a patient with Shapiro syndrome.\"], \"exact_answer\": [[\"recurrent hypothermia\"], [\"hyperhidrosis\"], [\"congenital agenesis of the corpus callosum\"]], \"context\": \"Shapiro syndrome is defined as the constellation of periodic hypothermia and hyperhidrosis along with agenesis of the corpus callosum by Shapiro et al. in 1969.Shapiro syndrome is a rare entity, comprising a triad of recurrent hypothermia, hyperhidrosis and congenital agenesis of the corpus callosum. We present a case of an 80 year old woman presenting with recurrent bouts of shivering, sweating and profound malaise, who sought medical attention because the frequency and severity of attacks worsened in her later years. MRI Brain demonstrated agenesis of the corpus callosum; a rigorous work-up excluded other causes for her symptomatology.These findings imply that aberrant thermoregulation in Shapiro syndrome involves a number of structures remote from the callosal region.A 6-year-old girl previously diagnosed with Shapiro's syndrome was admitted to our hospital on several occasions during a 1-year period with complaints of altered consciousness, syncope, hypothermia and episodes of sweating.Hypermelatoninemia should be considered in patients with spontaneous periodic hypothermia and hyperhidrosis, and also in patients with Shapiro's syndrome.We present a patient diagnosed with Shapiro syndrome without corpus callosum agenesis. A 4-year-old-girl was admitted to the hospital with complaints of sweating, cooling, and drowsiness that continued during the last week of her admission.Postmortem data regarding the hypothalamic and surrounding areas from future cases of Shapiro syndrome and spontaneous periodic hypothermia would be of great interest.\"}, {\"question\": \"List mouse models for autism spectrum disorder (ASD).\", \"ideal_answer\": [\"Numerous mouse models exists for autism spectrum disorder, such as: BTBR T+tf/J (BTBR), maternal immune, activation (MIA) mouse model of gestational poly(IC) exposure, C58/J and ProSAP1/Shank2.\"], \"exact_answer\": [[\"maternal immune activation (MIA) mouse model of gestational poly(IC) exposure\", \"C58/J\", \"Grin1\", \"Shank3-deficient mice\", \"C57BL/6J\", \"SYNGAP1\", \"Tsc2f/-\", \"MALTT\", \"(Dp(11)17/+)\", \"NR1(neo-/-)\", \"TS2-neo mouse\", \"Tbx1 heterozygous (HT) mice\"], [\"BTBR T+tf/J (BTBR)\", \"VPA600\", \"ProSAP1/Shank2\"]], \"context\": \"maternal immune activation (MIA) mouse model that is known to display features of ASD.BTBR T+tf/J (BTBR), a model of ASD with cognitive deficits,C58/J mouse strain, a model of ASD core symptoms. the C58/J mouse model of autismBoth C58/J and Grin1 knockdown mice, another model of ASD-like behavior, Mice injected subcutaneously with 600\\u2009mg/kg valproic acid (VPA600) at gestational day 12.5 show reduced social interaction in adulthood (at 8 weeks of age), and they have been proposed as a mouse model of autism.BTBR T+tf/J (BTBR), a commonly-used model presenting all core autism-related phenotypes The Autism ProSAP1/Shank2 mouse model Shank3-deficient micea panel of Shank mutant mouse modelsmaternal immune activation (MIA) mouse model of gestational poly(IC) exposure poly(IC) mouse model of autism spectrum disordersBTBR T+ tf/J strain, a mouse model of autism.The inbred BTBR T+ tf/J (BTBR) strain, a putative mouse model of autism,C57BL/6J strainSYNGAP1 mouse model of ID/ASDTsc2f/-;Cre mice demonstrated increased repetitive behavior as assessed with marble burying activityBTBR T+ tf/J mouse model for autism spectrum disordersFurthermore, maternal infection (maternal immune activation, MIA) is a risk factor for ASD.NR1(neo-/-) micemice heterozygous for the Gabrb3 geneGestational immune activation and Tsc2 haploinsufficiencynovel transgenic mouse, MALTT,sing a mouse model, CD38, a transmembrane protein expressed in immune cells but also in brain, was found to be critical for social behavior via regulation of oxytocin secretion. fmr1 knockout mouse modelTS2-neo mouseCongenic Tbx1 heterozygous (HT) mice the valproic acid (VPA) mouse model of autism indicates that the MeCP2+/- model may be useful for preclinical development targeting specific cortical processing abnormalities in RTT with potential relevance to ASD.(Dp(11)17/+) BTBR T+tf/J (BTBR) inbred mice are frequently used as a model of autism spectrum disorders (ASD)Intact and impaired executive abilities in the BTBR mouse model of autism.\"}, {\"question\": \"Which are the main functions of G3BP1 and G3BP2 proteins?\", \"ideal_answer\": [\"The main functions of G3BP1 and/or G3BP2 include translation of interferon stimulated mRNAs during dengue virus infection, initiation of assembly of stress granules, regulation of PMP22 mRNA which was found to affect cell proliferation in breast cancer cells, participation in several signaling pathways involved in growth, differentiation and apoptosis in human tumor cells after overexpression, limit viral replication events during Sindbis virus (SINV) infection, and modulation of p53 and MDM2 activity.\"], \"exact_answer\": [[\"Translation of interferon stimulated mRNAs during dengue virus infection\"], [\"Initiation of assembly of stress granules\"], [\"Regulation of PMP22 mRNA which was found to affect cell proliferation in breast cancer cells\"], [\"Participation in several signaling pathways involved in growth, differentiation and apoptosis in human tumor cells after overexpression\"], [\"Limit viral replication events during Sindbis virus (SINV) infection\"], [\"Modulation of p53 and MDM2 activity\"]], \"context\": \"Both G3BP1 and G3BP2 contribute to stress granule formationG3BP1, G3BP2 and CAPRIN1 are required for translation of interferon stimulated mRNAs and are targeted by a dengue virus non-coding RNAWe examined three conserved host RNA-binding proteins (RBPs) G3BP1, G3BP2 and CAPRIN1 in dengue virus (DENV-2) infection and found them to be novel regulators of the interferon (IFN) response against DENV-2Ras-GTPase-activating protein SH3 domain-binding protein 1 (G3BP1) is a component of SGs that initiates the assembly of SGs by forming a multimerAlthough single knockdown of either G3BP1 or G3BP2 in 293T cells partially reduced the number of SG-positive cells induced by arsenite, the knockdowns of both genes significantly reduced the numberlike G3BP1, the overexpression of G3BP2 induced SGs even without stress stimuliCollectively, these results suggest that both G3BP1 and G3BP2 play a role in the formation of SGs in various human cells and thereby recovery from these cellular stressesRegulation of PMP22 mRNA by G3BP1 affects cell proliferation in breast cancer cellsG3BP has been reported to both stabilize and induce degradation of specific mRNAsGlobal gene expression analyses of G3BP1- and G3BP2-depleted cells indicate that primarily G3BP1, and much less G3BP2, influences mRNA expression levels. Peripheral myelin protein 22 (PMP22) was one gene that was significantly influenced by G3BP1 depletion which led to a 2-3 fold increased expressionDepletion of PMP22 resulted in increased proliferation and the G3BP1-mediated effect on proliferation was not seen upon PMP22-depletionRas-GTPase-activating protein SH3 domain-binding proteins (G3BP) are overexpressed in various human tumors and participate in several signaling pathways involved in growth, differentiation and apoptosisTwo related proteins found in association with nsP4 at both times of infection, GTPase-activating protein (SH3 domain) binding protein 1 (G3BP1) and G3BP2 were also previously identified as associated with SINV nsP2 and nsP3. We demonstrate a likely overlapping role for these host factors in limiting SINV replication eventsModulation of p53 and MDM2 activity by novel interaction with Ras-GAP binding proteins (G3BP)we have found that members of the Ras network of proteins, Ras-GTPase activating protein-SH3-domain-binding proteins 1 and 2 (G3BP1 and 2), bind to p53 in vitro and in vivo. Our data show that expression of G3BPs leads to the redistribution of p53 from the nucleus to the cytoplasm. The G3BP2 isoform additionally associated with murine double minute 2 (MDM2), a negative regulator of p53. G3BP2 expression resulted in significant reduction in MDM2-mediated p53 ubiquitylation and degradation. Interestingly, MDM2 was also stabilized in G3BP2-expressing cells and its ability to ubiquitylate itself was compromisedWe identified several proteins, including DDX6, G3BP1, G3BP2, Caprin1, and USP10, implicated in P body (PB) and stress granule (SG) function, and not previously known to bind DENV RNAs.Additional experiments demonstrate that, in contrast to DDX6, the SG proteins G3BP1, G3BP2, Caprin1 and USP10 bind to the variable region (VR) in the 3' UTR.G3BP1, G3BP2 and CAPRIN1 are required for translation of interferon stimulated mRNAs and are targeted by a dengue virus non-coding RNA.Both G3BP1 and G3BP2 contribute to stress granule formation.Although single knockdown of either G3BP1 or G3BP2 in 293T cells partially reduced the number of SG-positive cells induced by arsenite, the knockdowns of both genes significantly reduced the number. Collectively, these results suggest that both G3BP1 and G3BP2 play a role in the formation of SGs in various human cells and thereby recovery from these cellular stresses.Furthermore, expression of shRNA targeting either G3BP1 or G3BP2 in human cancer cell lines resulted in marked upregulation of p53 levels and activity. Ras-GTPase-activating protein SH3 domain-binding protein 1 (G3BP1) is a component of SGs that initiates the assembly of SGs by forming a multimer.Although single knockdown of either G3BP1 or G3BP2 in 293T cells partially reduced the number of SG-positive cells induced by arsenite, the knockdowns of both genes significantly reduced the number.Moreover, like G3BP1, the overexpression of G3BP2 induced SGs even without stress stimuli.We found that HDAC6 interacts with another SG protein, G3BP (Ras-GTPase-activating protein SH3 domain-binding protein 1), and localizes to SGs under all stress conditions tested.Dengue virus (DENV) is a rapidly re-emerging flavivirus that causes dengue fever (DF), dengue hemorrhagic fever (DHF) and dengue shock syndrome (DSS), diseases for which there are no available therapies or vaccines.\\u00a0 The DENV-2 positive-strand RNA genome contains 5' and 3' untranslated regions (UTRs) that have been shown to form secondary structures required for virus replication and interaction with host cell proteins.\\u00a0 In order to comprehensively identify host cell factors that bind the DENV-2 UTRs, we performed RNA chromatography, using the DENV-2 5' and 3' UTRs as \\\"bait\\\", combined with quantitative mass spectrometry.\\u00a0 We identified several proteins, including DDX6, G3BP1, G3BP2, Caprin1, and USP10, implicated in P body (PB) and stress granule (SG) function, and not previously known to bind DENV RNAs.\\u00a0Additional experiments demonstrate that, in contrast to DDX6, the SG proteins G3BP1, G3BP2, Caprin1 and USP10 bind to the variable region (VR) in the 3' UTR.\"}, {\"question\": \"Which post-translational histone modifications are characteristic of facultative heterochromatin?\", \"ideal_answer\": [\"Nuclear VapB methyltransferase diminishes the establishment of facultative heterochromatin by decreasing histone 3 lysine 9 trimethylation (H3K9me3). Dramatic changes in exposure of a repressive chromatin mark, H3K9me2, indicate that during development linker histone plays a role in establishing the facultative heterochromatin territory and architecture in the nucleus. Histone H3 trimethylation at lysine 36 is associated with constitutive and facultative heterochromatin.\"], \"exact_answer\": [[\"H3K9me3\"], [\"H3K9me2\"], [\"H3K36me3\"]], \"context\": \"We used chromatin immunoprecipitation combined with high-throughput sequencing (\\\"ChIP-seq\\\") of DNA associated with the centromere-specific histone H3, CENP-A (CenH3), to identify centromeric DNA, and ChIP-seq with antibodies against dimethylated H3K4, trimethylated H3K9 and trimethylated H3K27 to determine the relative distribution and proportion of euchromatin, obligate and facultative heterochromatin, respectivelyDramatic changes in exposure of a repressive chromatin mark, H3K9me2, indicate that during development linker histone plays a role in establishing the facultative heterochromatin territory and architecture in the nucleusWe show here that the maternally inherited Snurf-Snrpn 3-Mb region, which is silenced by a potent transcription repressive mechanism, is uniformly enriched in histone methylation marks usually found in constitutive heterochromatin, such as H4K20me3, H3K9me3, and H3K79me3. Strikingly, we found that trimethylated histone H3 at lysine 36 (H3K36me3), which was previously identified as a hallmark of actively transcribed regions, is deposited onto the silenced, maternally contributed 3-Mb imprinted regionHistone H3 trimethylation at lysine 36 is associated with constitutive and facultative heterochromatin.In addition, we demonstrate that H3K36me3 is markedly enriched at the level of pericentromeric heterochromatin in mouse embryonic stem cells and fibroblasts.Remarkably, H3K9me3 is predominant in coding regions of active genes, a phenomenon that is not restricted to the X chromosome.We detected the H3K9me2, H3K9me3, and H3K27me3 modifications, with H3K27me3, which is indicative of facultative heterochromatin, exhibiting the highest enrichment on all viral promoters tested. Our study identifies a unique heterochromatin state marked by the presence of both H3.3 and H3K9me3, and establishes an important role for H3.3 in control of ERV retrotransposition in embryonic stem cells.\"}, {\"question\": \"Mutations in which genes have been associated with Aicardi-Goutieres syndrome?\", \"ideal_answer\": [\"Aicardi-Gouti\\u00e8res syndrome (AGS) is an inflammatory disorder caused by mutations in any of six genes (TREX1, RNASEH2A, RNASEH2B, RNASEH2C, SAMHD1, and ADAR).\"], \"exact_answer\": [[\"TREX1\"], [\"RNASEH2A\"], [\"RNASEH2B\"], [\"RNASEH2C\"], [\"SAMHD1\"], [\"ADAR\"]], \"context\": \"Aicardi-Gouti\\u00e8res syndrome (AGS) is an inflammatory disorder caused by mutations in any of six genes (TREX1, RNASEH2A, RNASEH2B, RNASEH2C, SAMHD1, and ADAR). Aicardi-Gouti\\u00e8res syndrome (AGS) is an inflammatory disorder caused by mutations in any of six genes (TREX1, RNASEH2A, RNASEH2B, RNASEH2C, SAMHD1, and ADAR)\"}, {\"question\": \"List bacteria that may be useful in uranium bioremediation.\", \"ideal_answer\": [\"The main bacteria studied in uranium bioremediation are Geobacteraceae. Other bacteria are: \\nFirmicutes, \\nShewanella oneidensis\\nPseudomonas aeruginosa\\nAnaeromyxobacter dehalogenans  \\nstrain Rf4T\"], \"exact_answer\": [[\"Betaproteobacteria\"], [\"Firmicutes\"], [\"Geobacter\", \"Geobacteraceae\", \"Geobacter uraniireducens\"], [\"Shewanella oneidensis\"], [\"Pseudomonas aeruginosa\"], [\"Anaeromyxobacter dehalogenans\"], [\"strain Rf4T\"]], \"context\": \"Members of the Betaproteobacteria (i.e. Dechloromonas, Ralstonia, Rhodoferax, Polaromonas, Delftia, Chromobacterium) and the Firmicutes dominated the biostimulated aquifer community.Shewanella oneidensis is an important model organism for bioremediation studies because of its diverse respiratory capabilities, conferred in part by multicomponent, branched electron transport systems.metal-reducing Geobacter species Identifications based on bacterial 16S rRNA sequence analysis showed that the dominant cultivable bacteria belonged to the genus Bacillus. Members of the genera Paenibacillus, Lysinibacillus, Klebsiella, Microbacterium and Chryseobacterium were also isolated from the LAAs soil samples. GeobacterGeobacteraceaeGeobacteraceae will be responsible for much of the Fe(III) and U(VI) reduction during uranium bioremediation in these sediments. Our previous studies have demonstrated that the microbial communities involved in uranium bioremediation and energy harvesting are both dominated by microorganisms in the family Geobacteraceae and that the organisms in this family are responsible for uranium bioremediation and electron transfer to electrodes. the organisms in the family Geobacteraceae that have been found to be associated with metal reduction in previous studies to stimulate the activity of Geobacter species during in situ uranium bioremediation.Geobacteraceae Geobacter species are known to be important members of the microbial community: (1) a uranium-contaminated aquifer located in Rifle, CO, USA undergoing in situ bioremediation;Geobacteraceae feoB transcripts in groundwater samples from a site undergoing in situ uranium bioremediation  Geobacteraceae activity during metal reduction in carbon-amended microcosms, with the highest expression observed in the glucose treatment.Geobacter uraniireducens sp. nov., isolated from subsurface sediment undergoing uranium bioremediation.A Gram-negative, rod-shaped, motile bacterium, strain Rf4T, which conserves energy from dissimilatory Fe(III) reduction concomitant with acetate oxidation, was isolated from subsurface sediment undergoing uranium bioremediation. . Constant levels of Geobacter ompB transcripts were detected in groundwater during a field experiment in which acetate was added to the subsurface to promote in situ uranium bioremediation. Geobacter species are the predominant organisms, Firmicutes were the predominant organisms whereas no Firmicutes sequences were detected in background sediments which did not have the capacity to sorb U(VI),Geobacter species that predominates during in situ uranium bioremediation at this site.As part of an effort to diagnose the physiological status of Geobacter species during in situ bioremediation of uranium-contaminated groundwater, Geobacter uraniireducens Geobacter species are often the dominant members of the groundwater community during active bioremediation and the primary organisms catalysing U(VI) reduction.Anaeromyxobacter dehalogenans strains implicated in hexavalent uranium reduction and immobilization are present in the fractured saprolite subsurface environment at the U.S. Department of Energy Integrated Field-Scale Subsurface Research Challenge (IFC) site near Oak Ridge, TN. Both enrichments were capable of reducing U(VI) rapidly. 16S rRNA gene clone libraries of the two enrichments revealed that Desulfovibrio spp. are dominant in the sulfate-reducing enrichment, and Clostridium spp. are dominant in the iron-reducing enrichment.. The results demonstrate that Geobacter species can effectively reduce U(VI)Uranium biomineralization by a metal-resistant Pseudomonas aeruginosa strain isolated from uranium mine waste was characterized for its potential in bioremediation. Uranium biomineralization by a metal resistant Pseudomonas aeruginosa strain isolated from contaminated mine waste.Geobacter-mediated bioremediation of uraniumThe cytochrome genes detected were primarily from Geobacter sp.Acetate amendment at uranium contaminated sites in Rifle, CO. leads to an initial bloom of Geobacter accompanied by the removal of U(VI) from the groundwater, followed by an increase of sulfate-reducing bacteria (SRBs) which are poor reducers of U(VI).Whole-genome microarray analyses of a subsurface isolate of Geobacter uraniireducens, At the US Department of Energy's Integrated Field Research Challenge (IFRC) site in Rifle, CO, the stimulation of Geobacter growth and activity via subsurface acetate addition leads to precipitation of U(VI) from groundwater as U(IV)Here, we report a proteomics-based approach for simultaneously documenting the strain membership and microbial physiology of the dominant Geobacter community members during in situ acetate amendment of the U-contaminated Rifle,\"}, {\"question\": \"The protein neprilysin  has an positive effect on Alzheimer disease, how can it be delivered to the brain?\", \"ideal_answer\": [\"The protein neprilysin can be deliverered to the brain (crossing the blood brain barrier) through: gene tranfer, transgenesis, gene induction, ex-vivo gene therapy, intracardiac (peripheral) administration of viral neprilysin construct, syringe-focused ultrasound device, convection-enhanced delivery and the use of human adipose tissue-derived mesenchymal stem cells that secrete functional neprilysin-bound exosomes\"], \"exact_answer\": [[\"Gene transfer with viral vectors\"], [\"Gene transfer with non-viral vectors\"], [\"Transgenesis\"], [\"Gene Induction\"], [\"Fusion of the Apolipoprotein B (ApoB) low-density lipoprotein (LDL) receptor-binding domain to a targeted protein\"], [\"ex vivo gene therapy\"], [\"Intracardiac (peripheral) administration of viral neprilysin construct\"], [\"Syringe-focused ultrasound device\"], [\"Convection-enhanced delivery\"], [\"Use of human adipose tissue-derived mesenchymal stem cells that secrete functional neprilysin-bound exosomes\"]], \"context\": \"he novel, syringe-focused US device was used to introduce purified plasmid DNA encoding human neprilysin (hNEP) into mouse hindlimb skeletal muscle concurrent with US protocolThis technique provides a safe and efficient non-viral method for in vivo gene delivery with potential applications in both basic research and in gene therapy of neuronal disease.A novel system for in vivo neprilysin gene delivery using a syringe electrode.Previous studies have demonstrated the therapeutic potential of viral vector-mediated neprilysin (NEP) gene therapy in mouse models of Alzheimer's disease (AD).Convection-enhanced delivery of neprilysin: a novel amyloid-\\u03b2-degrading therapeutic strategy.\"}, {\"question\": \"Which major signaling pathways are regulated by RIP1?\", \"ideal_answer\": [\"necroptosis\\napoptosis  \\npro-survival/inflammation NF-\\u03baB activation\"], \"exact_answer\": [[\"necroptosis\"], [\"apoptosis\"], [\"pro-survival/inflammation NF-\\u03baB activation\"]], \"context\": \"Major signaling pathways regulated by RIP1 include necroptosis, apoptosis, and pro-survival/inflammation NF-\\u03baB activation. receptor-interacting protein-1 (RIP1)-mediated necroptosisTNF-induced apoptotic signaling pathways were assessed by monitoring levels of the anti-apoptotic protein, A20, and cleavage products of the caspase-8 substrate, RIP1.Receptor-interacting protein (RIP) kinases promote the induction of necrotic cell death pathways. Together our results reveal a specific role for the RIP1-RIP3-DRP1 pathway in RNA virus-induced activation of the NLRP3 inflammasome and establish a direct link between inflammation and cell-death signaling pathways.Necroptosis exhibits a unique signaling pathway that requires the involvement of receptor interaction protein kinases 1 and 3 (RIP1 and RIP3),\"}, {\"question\": \"What type of cancers and inherited diseases have been associated to mutations in the Notch pathway?\", \"ideal_answer\": [\"So far, mutations in Notch and other components of its signaling pathway have been implicated in an array of human diseases (T-cell leukemia and other cancers,  Multiple Sclerosis,  CADASIL,  Alagille Syndrome,  Spondylocostal Dysostosis), but more pathologies are likely to be associated with Notch in the future due to its network complexity.\"], \"exact_answer\": [[\"Alagille syndrome\"], [\"spondylocostal dysostosis\"], [\"CADASIL (cerebral autosomal dominant arteriopathy with subcortical infarcts and leukoencephalopathy)\"], [\"T-cell acute lymphoblastic leukemia\"]], \"context\": \"Comparative functional genomic analysis identified a signature of Notch activation in 30% of HCC samples from patients.Here, we performed whole transcriptome sequencing on a discovery cohort of 18 primary tissue MCL samples and 2 cell lines. We found recurrent mutations in NOTCH1, a finding that we confirmed in an extension cohort of 108 clinical samples and 8 cell lines.We found activating mutations in Notch in more than 30% of ATL patients. These activating mutations are phenotypically different from those previously reported in T-ALL leukemias and may represent polymorphisms for activated Notch in human cancers.We detected Notch1 missense mutations in 8.3% of the tumors (only in the posterior fossa location and in case of 9q33-34 gain).Importantly, mutations of the Notch protein and components of its signaling pathway have been implicated in an array of human diseases (T-cell leukemia and other cancers, Multiple Sclerosis, CADASIL, Alagille Syndrome, Spondylocostal Dysostosis).Here, we demonstrate that endothelial-specific deletion of Jag1 leads to cardiovascular defects in both embryonic and adult mice that are reminiscent of those in Alagille syndrome.In this study, analysis of 21 Vietnamese ALGS individuals led to the identification of 19 different mutations (18 JAG1 and 1 NOTCH2), 17 of which are novel, including the third reported NOTCH2 mutation in Alagille Syndrome.Genetics of Holoprosencephaly (HPE), a congenital malformation of the developing human forebrain, is due to multiple genetic defects. Most genes that have been implicated in HPE belong to the sonic hedgehog signaling pathway. Here we describe a new candidate gene isolated from array comparative genomic hybridization redundant 6qter deletions, DELTA Like 1 (DLL1), which is a ligand of NOTCH.Four genes causing a subset of autosomal recessive forms of this disease have been identified: DLL3 (SCDO1: MIM 277300), MESP2 (SCDO2: MIM 608681), LFNG (SCDO3: MIM609813) and HES7 (SCDO4). These genes are all essential components of the Notch signalling pathway, which has multiple roles in development and disease. Previously, only a single SCD-causative missense mutation was described in HES7. In this study, we have identified two new missense mutations in the HES7 gene in a single family,Here we have used autozygosity mapping to identify a mutation in a fourth Notch pathway gene, Hairy-and-Enhancer-of-Split-7 (HES7), in an autosomal recessive SCD family.Alagille syndrome (AGS) is caused by mutations in the gene for the Notch signaling pathway ligand Jagged1 (JAG1), which are found in 94% of patients. To identify the cause of disease in patients without JAG1 mutations, we screened 11 JAG1 mutation-negative probands with AGS for alterations in the gene for the Notch2 receptor (NOTCH2). We found NOTCH2 mutations segregating in two families and identified five affected individuals.Mutations in the JAG1 (Jagged 1) gene, coding a ligand in the evolutionarily conserved Notch signaling pathway, are responsible for AGS. Here we present sixteen different JAG1 gene mutations, among them twelve novel, not described previously.Altogether 226 different JAG1 mutations have been described in association with AGS, including our novel 36 mutations.Previous studies have demonstrated that a wide spectrum of JAG1 mutations result in AGS. These include total gene deletions, protein truncating, splicing and missense mutations which are distributed across the coding region of the gene. Here we present results of JAG1 mutation screening by SSCP and FISH in 105 patients with AGS. For these studies, new primers were designed for 12 exons. Mutations were identified in 63/105 patients (60%)We have screened 54 AGS probands and family members to determine the frequency of mutations in JAG1. Three patients (6%) had deletions of the entire gene. Of the remaining 51 patients, 35 (69%) had mutations within JAG1, identified by SSCP analysis. Of the 35 identified intragenic mutations, all were unique, with the exceptions of a 5-bp deletion in exon 16, seen in two unrelated patients, and a C insertion at base 1618 in exon 9, also seen in two unrelated patients.Analyses of many patients with cytogenetic deletions or rearrangements have mapped the gene to chromosome 20p12, although deletions are found in a relatively small proportion of patients (< 7%). We have mapped the human Jagged1 gene (JAG1), encoding a ligand for the developmentally important Notch transmembrane receptor, to the Alagille syndrome critical region within 20p12.\"}, {\"question\": \"What is the treatment of triiodothyronine toxicosis?\", \"ideal_answer\": [\"Treatment of T3 toxicosis is a complex medical problem because not well responsive to the various options. Usual treatment includes antithyroid drugs such as propyltiouracil, radioactive iodine or beta blockers like propanol; surgery may be also necessary in some cases.\"], \"exact_answer\": [[\"Propyltiouracil\"], [\"Radioiodine\"], [\"beta blockers\"], [\"thyroidectomy\"]], \"context\": \"Three patients met the criteria for free T3 toxicosis and three had subclinical hyperthyroidism. All six patients had either multinodular glands or a single nodule on thyroid exam. Four patients were treated with radioactive iodine or surgery, resulting in reversal of the TSH suppression in three cases.6 months treatment with propranolol (160 mg/day) in eight patients with T3 (triiodothyronine) toxicosis. Serum total T3 concentrations showed a significant (p less than 0.01) and sustained fall to approximately 80% of pre-treatment values.Both patients responded to therapy with propylthiouracil.\"}, {\"question\": \"Which are the known human transmembrane nucleoporins?\", \"ideal_answer\": [\"Transmembrane nucleoporins (NUPs) are integral membrane components of the eukaryotic nuclear pore, playing an important role in the Nuclear Pore Complex (NPC) assembly. Even though the NPC is a conserved feature of all eukaryotes, different lineages possess some distinct transmembrane NUP subunits. Currently, four human transmembrane NUPs have been characterized, namely: NDC1 (also known as TMEM48 or NET3 or hNDC1), POM121 (also known as Nup121), GP210 (also known as Nuclear pore membrane glycoprotein 210 or Nuclear envelope pore membrane protein POM 210, POM210 or Nup210) and TMEM33 (or DB83).\"], \"exact_answer\": [[\"NDC1\", \"TMEM48\", \"NET3\", \"hNDC1\"], [\"POM121\", \"Nup121\"], [\"GP210\", \"Nuclear pore membrane glycoprotein 210\", \"Nuclear envelope pore membrane protein POM 210\", \"POM210\", \"Nup210\"], [\"TMEM33\", \"DB83\"]], \"context\": \"We investigated the interplay between import receptors and the transmembrane nucleoporin Pom121the transmembrane nucleoporin POM121 is critical for the incorporation of the Nup107/160 complex into new assembly sites specifically during interphase.The transmembrane nucleoporin NDC1NDC1 is a transmembrane nucleoporin that is required for NPC assembly and nucleocytoplasmic transport.The only protein known to localize to and be important in the assembly of both of these yeast structures is the integral membrane protein, Ndc1p. However, no homologues of Ndc1p had been characterized in metazoa. Here, we identify and analyze NDC1 homologues that are conserved throughout evolution. We show that the overall topology of these homologues is conserved. Each contains six transmembrane segments in its N-terminal half and has a large soluble C-terminal half of approximately 300 amino acids.Although it is not known whether vertebrate NDC1 protein localizes to nuclear pores like its yeast counterpart, the human homologue contains three FG repeats in the C-terminus, a feature of many nuclear pore proteins.we bring together data from another study to demonstrate that the human homologue of NDC1 is the known inner nuclear membrane protein, NET3.The conserved transmembrane nucleoporin NDC1we characterize vertebrate NDC1--a transmembrane nucleoporin conserved between yeast and metazoansgp210 is a major constituent of the nuclear pore complex (NPC) with possible structural and regulatory roles. It interacts with components of the NPC via its C-terminal domain (CTD), which follows a transmembrane domain and a massive ( approximately 200 kDa) N-terminal region that resides in the lumen of the perinuclear space.The membrane-spanning glycoprotein gp210 is a major component of the nuclear pore complex. This nucleoporin contains a large cisternal N-terminal domain, a short C-terminal cytoplasmic tail, and a single transmembrane segment.We propose that gp210 is organized into the pore membrane as a large array of gp210 dimers that may constitute a luminal submembranous protein skeleton.Patients with primary biliary cirrhosis (PBC) frequently produce autoantibodies against gp210, an integral glycoprotein of the nuclear pores. this protein consists of three main domains: a large glycosylated lumenal domain, a single hydrophobic transmembrane segment and a short cytoplasmic tail.Patients with primary biliary cirrhosis frequently develop autoantibodies directed to gp210, a major glycoprotein of the nuclear pore complex. This protein contains a large glycosylated cisternal domain, a single transmembrane segment, and a short cytoplasmic tail.Autoantibodies from patients with primary biliary cirrhosis recognize a restricted region within the cytoplasmic tail of nuclear pore membrane glycoprotein Gp210Patients with primary biliary cirrhosis (PBC) frequently have autoantibodies against a 210-kD integral glycoprotein of the nuclear envelope pore membrane. This protein, termed gp210, has a 1,783-amino acid amino-terminal domain located in the perinuclear space, a 20-amino acid transmembrane segment, and a 58-amino acid cytoplasmic carboxy-terminal tail.We investigated the interplay between import receptors and the transmembrane nucleoporin Pom121.Conversely, the transmembrane nucleoporin POM121 is critical for the incorporation of the Nup107/160 complex into new assembly sites specifically during interphase.Here, we characterize vertebrate NDC1--a transmembrane nucleoporin conserved between yeast and metazoans.Here, we report the solution structure of the human gp210 CTD as determined by various spectroscopic techniques.Using human gp210 isolated from HeLa cells we found the lumenal domain as the major target.This protein, termed gp210, has a 1,783-amino acid amino-terminal domain located in the perinuclear space, a 20-amino acid transmembrane segment, and a 58-amino acid cytoplasmic carboxy-terminal tail. In vertebrate cells, GLYCOPROTEIN OF 210 kD (gp210), PORE MEMBRANE PROTEIN OF 121 kD (Pom121), and NUCLEAR DIVISION CYCLE1 (NDC1) have been identified as integral pore membrane proteins (Cronshaw et al., 2002). \"}, {\"question\": \"Which genes were found to be methylated in bladder cancer cells?\", \"ideal_answer\": [\"In bladder cancer, hepaCAM (hepatocyte cell adhesion molecule), RAR\\u03b2(2), APC, TPEF (transmembrane protein containing epidermal growth factor and follistatin domain), RASSF1A, p14(ARF) and p16 genes were found to be methylated. These methylation events were demostrated to associate with downregulation of gene expression.\"], \"exact_answer\": [[\"hepaCAM (hepatocyte cell adhesion molecule)\"], [\"RAR\\u03b2(2)\"], [\"APC\"], [\"TPEF (transmembrane protein containing epidermal growth factor and follistatin domain)\"], [\"RASSF1A\"], [\"p14(ARF)\"], [\"p16\"]], \"context\": \"CpG island in promoter of hepaCAM gene was hyper-methylated both in bladder carcinoma tissues and cell lines (T24 and BIU-87). Otherwise, aberrant methylation of its promoter was associated with its decreased expression.Abnormal hypermethylation in CpG island of hepaCAM promoter is involved in absence of hepaCAM gene expression when bladder cancer occurs.Methylated RAR\\u03b2(2) and APC were significantly higher in bladder cancer patients (62.8%, 59.5%) than benign (16.4%, 5%) but not detected in healthy volunteers (0%) at (P < 0.0001).Among the 128 patients with bilharzial bladder cancer, 94 (73.4%) showed methylated RAR\\u03b2(2) and 86 (67.2%) showed methylated APC.Thus, methylated RAR\\u03b2(2) and APC genes might be valuable urinary molecular markers for early detection of bilharzial and nonbilharzial bladder cancer.Exon 2 methylation inhibits hepaCAM expression in transitional cell carcinoma of the bladderthe expression of hepaCAM was absent in T24 and BIU-87 cells, and we found that exon 2 of hepaCAM was methylated in the 2 cells.hepaCAM mRNA was re-expressed and the methylation status of hepaCAM exon 2 was reversed after treatment with 5-Aza-CdR.The methylation rate of hepaCAM exon 2 was significantly higher in bladder cancer tissues than in adjacent tissues.Recently, methylation of the TPEF (transmembrane protein containing epidermal growth factor and follistatin domain) gene was reported in human colon, gastric, and bladder cancer cells.Hypermethylation of at least one of three suppressor genes (APC, RASSF1A, and p14(ARF)) was found in all 45 tumor DNAs (100% diagnostic coverage).Methylation of the CpG island in the promoter of the p16 gene in human bladder cancer cells did not stop the formation of a transcript initiated 20 kb upstream by the p19 promoter but did prevent the expression of a p16 transcript.We also present the first functional evidence that methylation of only a small number of CpG sites can significantly down-regulate p16 promoter activity, thus providing support for the model of progressive inactivation of this tumor suppressor gene by DNA methylation.Hypermethylation of hepaCAM gene was reversed and expression of its mRNA and protein were re-activated in two cell lines by DNA methyltransferases inhibitor 5-aza-CdR\"}, {\"question\": \"Which are the bioinformatics tools for gene structure prediction?\\n\", \"ideal_answer\": [\"The in silico prediction of the complete structure of genes is one of the main challenges of bioinformatics. A critical part in the gene structure prediction is to identify the boundaries between exons and introns (i.e. splice sites) in the coding region. Several advanced bioinformatics tools have been developed for the precise delineation of a given gene structure: WPSS, SCGPred, TICO, GLIMMER, MetWAMer, WebScipio, GeneSeqer,  SplicePredictor, DGSplicer, Transcript Assembly Program (TAP), GeneBuilder, SeqHelp, HSPL, RNASPL, HEXON, CDSB, HBR, FGENE and FGENEH for human genes.\", \"SCGPred: a score-based method for gene structure prediction by combining multiple sources of evidence. MetWAMer.gthXML is a special-purpose variant of the software, specifically tailored to refine gene structure predictions generated by the GenomeThreader [30] and GeneSeqer [31] programs for spliced alignment-based gene structure annotation. WebScipio: an online tool for the determination of gene structures using protein sequences. \"], \"exact_answer\": [[\"WPSS\"], [\"SCGPred\"], [\"TICO\"], [\"GLIMMER\"], [\"MetWAMer\"], [\"WebScipio\"], [\"GeneSeqer\"], [\"SplicePredictor\"], [\"DGSplicer\"], [\"TAP\"], [\"GeneBuilder\"], [\"SeqHelp\"], [\"FGENE\", \"FGENEH\"], [\"HSPL\"], [\"RNASPL\"], [\"HEXON\"], [\"CDSB\"], [\"HBR\"]], \"context\": \"The prediction of the complete structure of genes is one of the very important tasks of bioinformatics, especially in eukaryotes. A crucial part in the gene structure prediction is to determine the splice sites in the coding region. Identification of splice sites depends on the precise recognition of the boundaries between exons and introns of a given DNA sequence. This problem can be formulated as a classification of sequence elements into 'exon-intron' (EI), 'intron-exon' (IE) or 'None' (N) boundary classes.The proposed WPSS method poses efficient results compared with the performance of many methods proposed in the literature.SCGPred: a score-based method for gene structure prediction by combining multiple sources of evidence.Moreover, computational gene finding in newly sequenced genomes is especially a difficult task due to the absence of a training set of abundant validated genes. Here we present a new gene-finding program, SCGPred, to improve the accuracy of prediction by combining multiple sources of evidence.Therefore, SCG-Pred can serve as an alternative gene-finding tool for newly sequenced eukaryotic genomes. The program is freely available at http://bio.scu.edu.cn/SCGPred/.Incorporation of splice site probability models for non-canonical introns improves gene structure prediction in plants.We pursued one such approach and describe the training and implementation of GC-donor splice site models for Arabidopsis and rice, with the goal of exploring whether specific modeling of non-canonical introns can enhance gene structure prediction accuracy.Source code for the updated versions of GeneSeqer and SplicePredictor (distributed with the GeneSeqer code) isavailable at http://bioinformatics.iastate.edu/bioinformatics2go/gs/download.html. Web servers for Arabidopsis, rice and other plant species are accessible at http://www.plantgdb.org/PlantGDB-cgi/GeneSeqer/AtGDBgs.cgi, http://www.plantgdb.org/PlantGDB-cgi/GeneSeqer/OsGDBgs.cgi and http://www.plantgdb.org/PlantGDB-cgi/GeneSeqer/PlantGDBgs.cgi, respectively. A SplicePredictor web server is available at http://bioinformatics.iastate.edu/cgi-bin/sp.cgi. Software to generate training data and parameterizations for Bayesian splice site models is available at http://gremlin1.gdcb.iastate.edu/~volker/SB05B/BSSM4GSQ/Prediction of splice sites with dependency graphs and their expanded bayesian networks.A crucial part in the gene structure prediction is to determine the precise exon-intron boundaries, i.e. the splice sites, in the coding region.Software (a program called DGSplicer) and datasets used are available at http://csrl.ee.nthu.edu.tw/bioinf/ BACKGROUND: cclu@ee.nthu.edu.tw.Gene structure prediction from consensus spliced alignment of multiple ESTs matching the same genomic locus.Accurate gene structure annotation is a challenging computational problem in genomics.The splice site prediction tool (SplicePredictor) is distributed with the GeneSeqer code. A SplicePredictor web server is available at http://bioinformatics.iastate.edu/cgi-bin/sp.cgiWe have developed a software tool, Transcript Assembly Program (TAP), to delineate gene structures using genomically aligned EST sequences.GeneBuilder: interactive in silico prediction of gene structure.In the case of low homology, GeneBuilder is still able to predict the gene structure. The GeneBuilder system has been tested by using the standard set (Burset and Guigo, Genomics, 34, 353-367, 1996) and the performances are: 0.89 sensitivity and 0.91 specificity at the nucleotide level.The computer program SeqHelp organizes information from database searches, gene structure prediction, and other information to generate multiply aligned, hypertext-linked reports to allow for fast analysis of molecular sequences.Identification of human gene structure using linear discriminant functions and dynamic programming.Development of advanced technique to identify gene structure is one of the main challenges of the Human Genome Project.A gene structure prediction system FGENE has been developed based on the exon recognition functions.Analysis of uncharacterized human sequences based on our methods for splice site (HSPL, RNASPL), internal exons (HEXON), all type of exons (FEXH) and human (FGENEH) and bacterial (CDSB) gene structure prediction and recognition of human and bacterial sequences (HBR) (to test a library for E. coli contamination) is available through the University of Houston, Weizmann Institute of Science network server and a WWW page of the Human Genome Center at Baylor College of Medicine.In the bioinformatics field, many computer algorithmic and data mining technologies have been developed for gene prediction, protein-protein interaction analysis, sequence analysis, and protein folding predictions, to name a few.The accurate prediction of intron boundaries largely facilitates the correct prediction of gene structure in nuclear genomes.\"}, {\"question\": \"Name the factors required for selenoprotein synthesis in eukaryotes\", \"ideal_answer\": [\"eFSec, SBP2, SECp43, PSTK, Sec synthase (Sec S, SLA/LP), SPS2 (SelD), tRNASec, SECIS element, (L30), SPS1\"], \"exact_answer\": [[\"eFSec\"], [\"SBP2\"], [\"SECp43\"], [\"PSTK\"], [\"Sec synthase\", \"Sec S\", \"SLA/LP\"], [\"SPS2\", \"SelD\"], [\"tRNASec\"], [\"SECIS element\"], [\"(L30)\"], [\"SPS1\"]], \"context\": \"The process requires the Sec insertion sequence (SECIS) element, tRNASec, and protein factors including the SECIS binding protein 2 (SBP2)Taken together, these data establish the role of SECp43 and SLA in selenoprotein biosynthesis through interaction with tRNA([Ser]Sec) in a multiprotein complex.Selenophosphate synthetase (SelD) generates the selenium donor for selenocysteine biosynthesis in eubacteria. One homologue of SelD in eukaryotes is SPS1 (selenophosphate synthetase 1) and a second one, SPS2, was identified as a selenoprotein in mammals.These in vivo studies indicate that SPS2 is essential for generating the selenium donor for selenocysteine biosynthesis in mammals, whereas SPS1 probably has a more specialized, non-essential role in selenoprotein metabolism.\"}, {\"question\": \"Which genes/proteins have been found to inhibit cyclin dependent kinase 4 (CDK4)?\", \"ideal_answer\": [\"The p15(ink4b) and p16(ink4a) CDK4 inhibitor genes map within the chromosome band 9p21 region deleted frequently in various cancers.The Cdk4 inhibitor p18(Ink4c) is a tumor suppressor. Recent studies of Cyclin D1/Cdk4 have proposed that p21(Waf1/Cip1/Sdi1) plays a key role as a potent Cdk4 inhibitor. p27KIP1 is also a cdk4 ihibitor.\"], \"exact_answer\": [[\"p16\", \"p16INK4\", \"p16INK4a\", \"MTS1\", \"CDKN2\"], [\"p15\", \"p15INK4B\"], [\"p18(Ink4c)\"], [\"p21(Waf1/Cip1/Sdi1)\"], [\"p27KIP1\"]], \"context\": \"Interestingly, the Cdk inhibitor p18(Ink4c) was induced in the transgenic pineal glands independently of p53, and transgenic mice that lacked Ink4c developed invasive PNET, although at an older age than those lacking p53.Our finding that the Cdk4 inhibitor p18(Ink4c) is a tumor suppressor in cyclin D1-driven PNET suggests that pharmacologic interventions to inhibit Cdk4 activity may be a useful chemoprevention or therapeutic strategy in cancer driven by primary RB pathway disruption.Our previous studies showed that while both specific CDK4 inhibitor p16INK4A (P16) and gankyrin bind to cyclin-dependent kinase 4 (CDK4) in similar fashion, only P16 inhibits the kinase activity of CDK4.Recent biological studies of Cyclin D1/Cdk4 have proposed that p21 C-terminal domain (p21(CT)) plays a key role as a potent Cdk4 inhibitor. Especially, our data suggests that the D(149)FYHSKRR(156) region of p21 is critical for Cdk4 binding, indicating that the major driving force for complex originates from hydrophobic interaction between p21 and Cdk4.We show that c-Myc prevents induction of the cdk4 inhibitor p15(Ink4b) and the subsequent inhibition of G(1) cdks by TGF-beta.CDK activity is modulated by inhibitors such as p15INK4b and p16INK4a. Loss of function of p15INK4b and p16INK4a (multiple tumor suppressor-I and CDK4 inhibitor) determines impairment in the control of the cell cycle and contributes to the transformation of several cell types. D-type cyclins, in association with the cyclin-dependent kinases CDK4 and CDK6, promote progression through the G1 phase of the cell cycle.In the present study, we analyzed human ovarian carcinoma cell lines for abnormalities in the tumor suppressor gene Rb (retinoblastoma) and in cyclin-dependent kinase 4 (CDK4) inhibitor genes (p16INK4 and p15INK4B) using molecular biology techniques.These data suggest that abnormalities of Rb and CDK4 inhibitor genes (p16INK4, p15INK4B) may be involved in human ovarian carcinogenesis.The mutation, an arginine-to-cysteine exchange at residue 24, was part of the CDK4 peptide recognized by CTLs and prevented binding of the CDK4 inhibitor p16INK4a, but not of p21 or of p27KIP1. The p15 and p16 CDK4 inhibitor genes map within the chromosome band 9p21 region deleted frequently in malignant mesothelioma and other cancers.The cyclin-dependent kinase 4 (cdk4) inhibitor (p16INK4/MTS1/CDKN2) gene has been recently identified as a putative tumor suppressor gene because of the high frequency of homozygous deletion observed in numerous human tumor cell lines, including leukemias. The identification of a cdk4 inhibitor, p16INK4, as a target for mutations in cultured tumor lines and primary tumors suggested that RB activity may be affected in these cells.Roles of cyclin A-Cdk2 as a p27 target and cyclin D2-Cdk4 as a p27 reservoir may result from the differential ability of bound p27 to inhibit the kinase subunit in these complexes.Role of the cyclin-dependent kinase 4 and 6 inhibitor gene family p15, p16, p18 and p19 in leukemia and lymphoma.Identification of functional elements of p18INK4C essential for binding and inhibition of cyclin-dependent kinase (CDK) 4 and CDK6.One of the INK4 molecules, p16, is also known as multiple tumor suppressor and has been found to be mutated or deleted in various tumors and cell lines. We have previously identified p18 as a member of the INK4 family.Members of the INK4 family of cyclin-dependent kinase (CDK) inhibitors specifically bind and inhibit the G1-specific CDK molecules CDK4 and CDK6.The subcellular locations of p15(Ink4b) and p27(Kip1) coordinate their inhibitory interactions with cdk4 and cdk2.\"}, {\"question\": \"Which disorders are associated to mutated Hepcidin (HAMP)?\", \"ideal_answer\": [\"Juvenile hemochromatosis (JH) is the most severe form of heriditary hemochromatosis, usually caused by mutations in hemojuvelin (HJV) or hepcidin (HAMP).\"], \"exact_answer\": [[\"juvenile hemochromatosis\"]], \"context\": \"juvenile hemochromatosis due to hemojuvelin and hepcidin mutation Mutations in all of the currently known genes implicated in non-HFE HH (hemojuvelin, hepcidin, transferrin receptor 2, and ferroportin) have been reported in patients from the Asia-Pacific region. In conclusion, HAMP and hemojuvelin mutations are rare among Spanish HH patients, and their impact in this population is not significant.Juvenile hemochromatosis (JH) is the most severe form, usually caused by mutations in hemojuvelin (HJV) or hepcidin (HAMP). We have identified novel mutations in HJV, HAMP, and SLC40A1 in countries not normally associated with hereditary hemochromatosis (Pakistan, Bangladesh, Sri Lanka, and Thailand).In humans, loss of TfR2, HJV, and hepcidin itself or FPN mutations result in full-blown hemochromatosis. Targeting hepcidin with replacement therapy to decrease iron may be a treatment of not only HCV, HH, and alcoholic cirrhosis, but also PCT.Hepcidin deficiency underlies iron overload in HFE-hemochromatosis as well as in several other genetic iron excess disorders, such as hemojuvelin or hepcidin-related hemochromatosis and transferrin receptor 2-related hemochromatosis. In addition to HFE gene, mutations in the genes that encode hemojuvelin (HJV), hepcidin (HAMP), transferrin receptor 2 (TFR2) and ferroportin (SLC40A1) have been associated with regulation of iron homeostasis and development of HH\"}, {\"question\": \"Which are the most under-represented oligonucleotides in higher eukaryote genomes?\", \"ideal_answer\": [\"The oligonucleotides containing the CG and TA dinucleotide are generally under-represented in higher eukaryote genomes\", \"TpA is the most underepresented dinucleotide followed closely by CpG. For trinucleotides, GCA/TGC tends to be under-represented in phage, human viral, and eukaryotic sequences, and CTA/TAG is strongly under-represented in many prokaryotic, eukaryotic, and viral sequences. For higher lengts alternating Purine/Pyrimidine tracts are underepresented up to 60%.\", \"Oligonucleotides containing CG and TA dinucleoides\"], \"exact_answer\": [[\"TA\"], [\"CG\"], [\"(PuPy)n\"], [\"GCA/TGC\"], [\"CTA/TAG\"]], \"context\": \"Unusual frequencies of certain alternating purine-pyrimidine runs in natural DNA sequencesOctanucleotides are the most deficient, occurring at only 60% of the frequency expected in random sequences. An unexpectedly high proportion of these octamers consists of alternating tetramers with the repeat structure (PuPyPuPy)2 or (PyPuPyPu)2For dinucleotides, TA is almost universally under-represented, with the exception of vertebrate mitochondrial genomes, and CG is strongly under-represented in vertebrates and in mitochondrial genomesFor trinucleotides, GCA.TGC tends to be under-represented in phage, human viral, and eukaryotic sequences, and CTA.TAG is strongly under-represented in many prokaryotic, eukaryotic, and viral sequencesThe dinucleotide CpG is significantly under-represented in vertebrate DNA and is usually methylatedOne such pattern recognition system is based on unmethylated CpG dinucleotides in particular sequence contexts (CpG motifs); these motifs are common in bacterial DNA but are under-represented (\\\"CpG suppression\\\") and methylated in vertebrate DNAWe argue that the significant under-represented motif pattern of CpG in an AU context--which is found in both the ssRNA viruses and innate genes, and has decreased throughout the history of H1N1 influenza replication in humans--is immunostimulatory and has been selected against during the co-evolution of viruses and host innate immune genes\"}, {\"question\": \"List co-morbidities that may occur together with \\\"Stiff man Syndrome\\\"\", \"ideal_answer\": [\"SMS (Stiff man Syndrome) is is a rare disorder of the central nervous system of probable autoimmune origin and as such is associated with other autoimmune diseases, such as Insulin Dependent Diabetes Mellitus . GAD-65 is a dominant auto-antigen that is found both in in stiff-man syndrome and insulin-dependent diabetes mellitus. TRAB -positive Graves' disease has been reported to occur together with SMS. \\nIn a subgroup of patients with the stiff-man syndrome, the condition is likely to have an autoimmune paraneoplastic origin. The detection of autoantibodies against the 128-kd antigen in patients with this syndrome should be considered an indication to search for an occult breast cancer.\\n\\nHCV may be the etiologic virus of progressive encephalomyelitis with rigidity; a rare disorder similar to stiff-man syndrome although different because it is progressive and fatal.\\nIt is possible that the reported case of association of progressive dementia with concomitant development of stiff-man syndrome in an elderly man represents an exaggerated form of such motor disturbances in dementia, and that clinical and electromyographic features of stiff-man syndrome may be present with increased incidence in patients with dementia.\"], \"exact_answer\": [[\"Insulin Dependent Diabetes Mellitus\", \"type I (insulin-dependent) diabetes mellitus (IDDM)\"], [\"Grave's disease\"], [\"Breast Cancer\"], [\"Dementia\"]], \"context\": \"To report an association between two autoimmune conditions, Graves' disease and stiff-person (stiff-man) syndrome, and discuss the relevant literature.This case illustrates the association between TRAB-positive Graves' disease and stiff-person syndrome and the improvement of Graves' disease with immunosuppressive therapy.HCV may be the etiologic virus of progressive encephalomyelitis with rigidity; a rare disorder similar to stiff-man syndrome although different because it is progressive and fatal.The triad of stiff-man syndrome, breast cancer, and autoantibodies against amphiphysin identifies a new autoimmune paraneoplastic syndrome of the CNS.GAD has been suggested as an autoantigen in insulin-dependent diabetes mellitus and stiff-man syndromeStiff-man syndrome (SMS) is a rare disorder of the central nervous system of probable autoimmune origin.Patients with SMS often have other autoimmune diseases, in particular type I (insulin-dependent) diabetes mellitus (IDDM).GAD-65 is a dominant autoantigen in stiff-man syndrome and insulin-dependent diabetes mellitus.In this group, there is a striking association of the stiff-man syndrome with organ-specific autoimmune diseases, primarily insulin-dependent diabetes mellitus.We studied three women with the stiff-man syndrome and breast cancer,RESULTS: Autoantibodies directed against a 128-kd brain protein were found in two of the women with the stiff-man syndrome and breast cancer. These results led to a search for breast cancer in the third patient with the stiff-man syndrome, who also had autoantibodies. A small invasive ductal carcinoma was detected by ultrasonography and removed.CONCLUSIONS: In a subgroup of patients with the stiff-man syndrome, the condition is likely to have an autoimmune paraneoplastic origin. The detection of autoantibodies against the 128-kd antigen in patients with this syndrome should be considered an indication to search for an occult breast cancer.We have previously reported the presence of autoantibodies against glutamic acid decarboxylase (GAD) in a patient with stiff-man syndrome, epilepsy, and insulin-dependent diabetes mellitus.These findings support the hypothesis that stiff-man syndrome is an autoimmune disease and suggest that GAD is the primary autoantigen involved in stiff-man syndrome and the associated insulin-dependent diabetes mellitus.An elderly man with progressive dementia and concomitant development of stiff-man syndrome is described. He had not had stiff-man syndrome one year earlier, when he had only mild dementia. An association between stiff-man syndrome and dementia has not been previously described.It is possible that this patient represents an exaggerated form of such motor disturbances in dementia, and that clinical and electromyographic features of stiff-man syndrome may be present with increased incidence in patients with dementia.\"}, {\"question\": \"What personality traits can be evaluated with the Ten Item Personality Inventory.\", \"ideal_answer\": [\"The Ten Item Personality Inventory measures each of the five major facets of personality: openness, extroversion, conscientiousness, agreeableness and neuroticism.\"], \"exact_answer\": [[\"openness\"], [\"extroversion\"], [\"conscientiousness\"], [\"agreeableness\"], [\"neuroticism\"]], \"context\": \"Analyses included correlations and a regression analysis between depressive symptoms and unmet expectations with the Five-Factor Model personality traits (extraversion, agreeableness, conscientiousness, emotional stability, openness to experience) as measured by the Ten-Item Personality Inventory.The questionnaire measures each of the five major facets of personality: openness, extroversion, conscientiousness, agreeableness and neuroticism.DESIGN: Adoptive mothers (N = 136) were surveyed for depressive symptoms using the Center for Epidemiologic Studies-Depression Scale (CES-D) and the Edinburgh Postnatal Depression Scale (EPDS). Analyses included correlations and a regression analysis between depressive symptoms and unmet expectations with the Five-Factor Model personality traits (extraversion, agreeableness, conscientiousness, emotional stability, openness to experience) as measured by the Ten-Item Personality Inventory. Analyses included correlations and a regression analysis between depressive symptoms and unmet expectations with the Five-Factor Model personality traits (extraversion, agreeableness, conscientiousness, emotional stability, openness to experience) as measured by the Ten-Item Personality Inventory. \"}, {\"question\": \"Which are the enzymes involved in the control of tubulin acetylation?\", \"ideal_answer\": [\"Acetyltransferase MEC-17, and deacetylases SIRT2 (Sirtuin 2), HDAC6 (histone deacetylase 6) and dTip60 are known to control the levels of tubulin acetylation.\"], \"exact_answer\": [[\"MEC-17\"], [\"SIRT2 (Sirtuin 2)\"], [\"HDAC6 (histone deacetylase 6)\"], [\"dTip60\"]], \"context\": \"Inhibition of HDAC6 deacetylase activity increases its binding with microtubules and suppresses microtubule dynamic instability in MCF-7 cellsHDAC6, the tubulin deacetylase, plays a key role in maintaining typical distribution of acetylated microtubules in cells.We found that whereas both pharmacological inhibition of HDAC6 as well as its depletion enhance microtubule acetylation, only pharmacological inhibition of HDAC6 activity leads to an increase in microtubule stability against cold and nocodazole-induced depolymerizing conditions.The evidence presented in this study indicated that the increased binding of HDAC6, rather than the acetylation per se, causes microtubule stability.The results are in support of a hypothesis that in addition to its deacetylase function, HDAC6 might function as a MAP that regulates microtubule dynamics under certain conditions.Moreover, acetylation of \\u03b1-tubulin is under the control of the acetyltransferase MEC-17 and deacetylases SIRT2 (Sirtuin 2) and HDAC6 (histone deacetylase 6).MEC-17 deficiency leads to reduced \\u03b1-tubulin acetylation and impaired migration of cortical neuronsMEC-17 is a newly discovered \\u03b1-tubulin acetyltransferase that has been found to play a major role in the acetylation of \\u03b1-tubulin in different species in vivo.Thus, MEC-17, which regulates the acetylation of \\u03b1-tubulin, appears to control the migration and morphological transition of cortical neurons. This finding reveals the importance of MEC-17 and \\u03b1-tubulin acetylation in cortical development.NT secretion is prevented by overexpression of HDAC6, an \\u03b1-tubulin deacetylase.Moreover, \\u03b1-tubulin acetylation levels of microtubules specifically extending into the terminal synaptic boutons are reduced in response to dTip60 HAT reduction.In this study, angiotensin II induced disassembly and deacetylation of \\u03b1-tubulin, which were blocked by pretreatment with an angiotensin II type 1 receptor blocker losartan and a sirtuin class deacetylase inhibitor sirtinol, and by depletion of a deacetylase SIRT2 using RNA interference.These data show that angiotensin II and mechanical stretch stimulate microtubule redistribution and deacetylation via SIRT2 in endothelial cells, suggesting the emerging role of SIRT2 in hypertension-induced vascular remodeling.Tubulin is a major substrate of the cytoplasmic class II histone deacetylase HDAC6. Inhibition of HDAC6 results in higher levels of acetylated tubulin and enhanced binding of the motor protein kinesin-1 to tubulin, which promotes transport of cargoes along microtubules.Alcohol-induced alterations in hepatic microtubule dynamics can be explained by impaired histone deacetylase 6 functionAddition of trichostatin A (TSA), an HDAC6 inhibitor, induced microtubule acetylation to the same extent as in ethanol-treated cells (approximately threefold).Interestingly, HDAC6 from ethanol-treated cells was able to bind and deacetylate exogenous tubulin to the same extent as control, suggesting that ethanol-induced tubulin modifications prevented HDAC6 binding to endogenous microtubules.We propose that lower HDAC6 levels combined with decreased microtubule binding lead to increased tubulin acetylation in ethanol-treated cells.The results also showed that both pan-HDAC and class-I-specific inhibitor treatment resulted in increased acetylation of histones, but only pan-HDAC inhibitor treatment resulted in increased tubulin acetylation, which is in agreement with their activity towards the HDAC6 isoform.Nicotinamide but not 3-aminobenzamide, an inhibitor for poly(ADP)ribose polymerase, enhanced tubulin acetylation and resistance to axonal degeneration in cultured cerebellar granule cells from wild-type (WT) mice, suggesting that mammalian Sir2-related protein (SIRT) 2, a nicotinamide adenine dinucleotide (NAD)--dependent tubulin deacetylase, could modulate resistance to axonal degeneration.Moreover, SIRT2 overexpression abrogated microtubule hyperacetylation and resistance to axonal degeneration in these cells.Conversely, SIRT2 knockdown by using a lentiviral vector expressing small interfering RNA, enhanced microtubule acetylation and resistance to axonal degeneration in WT granule cells.Taken together, these results suggest that SIRT2-mediated tubulin deacetylation is involved in both microtubule hyperacetylation and resistance to axonal degeneration in Wld(S) granule cells.Compared to control cultures, higher levels of acetylated tubulin were found in neurons treated with tubacin\"}, {\"question\": \"List all approved indications for Glivec\", \"ideal_answer\": [\"CML - blast crisis, in accelerated phase, and in chronic phase after interferon failure or intolerance.  Glivec received orphan drug status from the U.S. Food and Drug Administration (FDA) Office of Orphan Products Development on January 31, 2001, and accelerated approval from the FDA for the above three indications on May 10, 2001.\\n\\nGastrointestinal stromal tumor (GIST\\nTreatment with adjuvant imatinib following surgical resection of localized Kit-positive GIST\\n In locally advanced inoperable patients and metastatic patients, Imatinib is the standard treatment.\", \"Dermatofibrosarcoma protuberans (DFSP) is an uncommon cutaneous neoplasm.\"], \"exact_answer\": [[\"Chronic myelogenous leukemia (CML)\"], [\"Gastrointestinal stromal tumor (GIST)\"], [\"Dermatofibrosarcoma protuberans (DFSP)\"]], \"context\": \"The limited available experience suggests that imatinib could be considered as an individualized treatment approach in severe SSc and underscores the need to identify markers for selecting particular patients, who will safely respond to therapeutic inhibition of tyrosine kinases.Dermatofibrosarcoma protuberans (DFSP) is an uncommon cutaneous neoplasmMoreover, even when surgery is utilized, there are cases where positive margins remain, in which case imatinib would be an option, often in combination with surgery. Imatinib can decrease tumor size preoperatively and help to improve postsurgical aesthetic appearance and minimize functional impairment.for CML we analysed imatinib, dasatinib and nilotinib. RESULTS: Most drugs included in this study had received market authorization in all countries, but the range of indications for which they had been authorized differed by countryImatinib mesylate, an orally administered kinase inhibitor that targets the Kit (CD117) protein, currently has 10 approved indications including treatment of chronic myelogenous leukemia and metastatic gastrointestinal stromal tumors (GIST). Treatment with adjuvant imatinib following surgical resection of localized Kit-positive GIST, the most recent FDA-approved indication (December 2008), has been shown to significantly improve recurrence-free survival (RFS) compared with surgical resection aloneCurrently, there are no clinical recommendations regarding how to incorporate imatinib drug plasma monitoring in patients with either chronic myeloid leukemia or gastrointestinal stromal tumors, indications for which imatinib is approved.Imatinib was effective against GIST that were positive for KIT proteinThe introduction of alternative therapies, such as imatinib for CML, has challenged well-established indications.More micro-GIST are discovered with the development of investigations, rising the question of wait and see policy for some of them. In locally advanced inoperable patients and metastatic patients, Imatinib is the standard treatment.The introduction of alternative therapies has challenged well-established indications such as imatinib for chronic myeloid leukaemia.Imatinib mesylate (Gleevec, Novartis Pharmaceuticals East Manruer, NJ) received accelerated approval on May 10, 2001 for the treatment of patients with chronic myeloid leukemia (CML) in (a) chronic phase after failure of IFN-alpha therapy, (b) accelerated phase, and (c) blast crisis.Gleevec (imatinib mesylate), a highly promising new drug for the treatment of chronic myelogenous leukemia in blast crisis, in accelerated phase, and in chronic phase after interferon failure or intolerance, received orphan drug status from the U.S. Food and Drug Administration (FDA) Office of Orphan Products Development on January 31, 2001, and accelerated approval from the FDA for the above three indications on May 10, 2001.Imatinib is the first effective systemic therapy for advanced GIST.\"}, {\"question\": \"Which proteins induce inhibition of LINE-1 and Alu retrotransposition?\", \"ideal_answer\": [\"It was demonstrated that antiretroviral restriction factors, human APOBEC3 proteins A to H, differentially inhibit LINE-1 and Alu retrotransposition. The same effect was shown to be induced by the Aicardi-Gouti\\u00e8res syndrome gene product SAMHD1.\"], \"exact_answer\": [[\"human APOBEC3 proteins A to H\"], [\"Aicardi-Gouti\\u00e8res syndrome gene product SAMHD1\"]], \"context\": \"APOBEC3G oligomerization is associated with the inhibition of both Alu and LINE-1 retrotranspositionWe have previously demonstrated that antiretroviral restriction factors, human APOBEC3 (hA3) proteins (A-H), differentially inhibit L1 retrotranspositionIn this present study, we found that hA3 members also restrict Alu retrotransposition at differential levels that correlate with those observed previously for L1 inhibitionModulation of LINE-1 and Alu/SVA retrotransposition by Aicardi-Gouti\\u00e8res syndrome-related SAMHD1Here, we demonstrate that the Aicardi-Gouti\\u00e8res syndrome gene product SAMHD1, recently revealed to be an inhibitor of HIV/simian immunodeficiency virus (SIV) infectivity and neutralized by the viral Vpx protein, is also a potent regulator of LINE-1 and LINE-1-mediated Alu/SVA retrotranspositionWe also found that mutant SAMHD1s of Aicardi-Gouti\\u00e8res syndrome patients are defective in LINE-1 inhibitionSAMHD1 inhibits LINE-1 retrotransposition in dividing cellswe found that the predominantly nuclear A3B only weakly restricted infection by HIV-1, HIV-1\\u0394vif, and human T-cell leukemia virus type 1 (HTLV-1), while significantly inhibiting LINE-1 retrotranspositionThe chimeric construct A3G/B, in which the first 60 amino acids of A3B were replaced with those of A3G, restricted HIV-1, HIV-1\\u0394vif, and HTLV-1 infection, as well as LINE-1 retrotranspositionExpression of cellular A3 proteins is sufficient for inhibition of L1 and Alu retrotransposition, but the effect of A3 proteins transferred in exosomes on retroelement mobilization is unknown.Inhibition of LINE-1 and Alu retrotransposition by exosomes encapsidating APOBEC3G and APOBEC3F.APOBEC3G oligomerization is associated with the inhibition of both Alu and LINE-1 retrotransposition.Previous studies revealed that the overexpression of some A3 proteins could restrict engineered human Long INterspersed Element-1 (LINE-1 or L1) retrotransposition in HeLa cells.APOBEC3 proteins inhibit LINE-1 retrotransposition in the absence of ORF1p binding.We concluded that APOBEC3 proteins interfere indirectly with the LINE-1 retrotransposition pathway, probably through interference with RNA targeting.Expression of cellular A3 proteins is sufficient for inhibition of L1 and Alu retrotransposition, but the effect of A3 proteins transferred in exosomes on retroelement mobilization is unknownHere, we demonstrate that the Aicardi-Gouti\\u00e8res syndrome gene product SAMHD1, recently revealed to be an inhibitor of HIV/simian immunodeficiency virus (SIV) infectivity and neutralized by the viral Vpx protein, is also a potent regulator of LINE-1 and LINE-1-mediated Alu/SVA retrotranspositionHuman cytidine deaminases, including APOBEC3G (A3G) and A3F, are part of a cellular defense system against retroviruses and retroelements including non-LTR retrotransposons LINE-1 (L1) and AluSAMHD1 inhibits LINE-1 retrotransposition in dividing cellsExpression of cellular A3 proteins is sufficient for inhibition of L1 and Alu retrotransposition, but the effect of A3 proteins transferred in exosomes on retroelement mobilization is unknownSelective inhibition of Alu retrotransposition by APOBEC3G.Our data further indicate that APOBEC3G recognizes 7SL RNA and Alu RNA by its common structure, the Alu domain, suggesting a mechanism for APOBEC3G- mediated inhibition of Alu retrotranspositionFunctional analysis and structural modeling of human APOBEC3G reveal the role of evolutionarily conserved elements in the inhibition of human immunodeficiency virus type 1 infection and Alu transposition.\"}, {\"question\": \"Which are the available biomedical text mining tools for the detection of protein-protein interactions?\", \"ideal_answer\": [\"Protein-protein interactions (PPI) can be extracted from biomedical literature using text mining approaches. These approaches have been classified into two categories, the statistical calculation of the co-occurrence of proteins and the computational linguistic method. Moreover, bioinformatics methods based on sequence, structural, or evolutionary information have been developed to predict protein-protein interactions. The available state-of-the-art biomedical text mining tools are: eFIP (Extracting Functional Impact of Phosphorylation), a system for text mining of protein interaction networks of phosphorylated proteins; GeneView, a suite of state-of-the-art text-mining tools designed for the automated identification of protein\\u2013protein interactions; PPI finder, a text mining tool for human protein-protein interactions based on computational linguistic methods. PPI Finder system consists of the Information Retrieval module and Information Extraction module; PreBIND and Textomy are two components of a biomedical literature-mining system optimized to discover protein-protein interactions using a support vector machine; BioMap system is optimized for the identification of protein-protein interactions from large biomedical literature datasets; Protopia searches for and integrates protein-protein interactions and the information about them contained in five different Protein Interaction Web Databases; STRING uses Natural Language Processing to extract a subset of semantically specified known and predicted protein-protein interactions.\", \"Several tools have been developed to detect protein-protein interactions (PPIs) by text mining the biomedical literature. Two main computational approaches used for this task are co-occurrence-based methods and Natural Language Processing methods. Biomedical text mining tools for PPI identification are the following (in alphabetical order): BioCreative Meta Server, BioMap, eFIP, Extracting Functional Impact of Phosphorylation, GeneView, HAPPI, Hidden Vector State model, iHop, LAITOR, OntoGene, OpenDMAP, PIE (Protein interaction information extraction system), PPI finder (Paired-PPI Finder), PPInterFinder, PPIs, PolySearch, PreBind and Textomy, Protopia, STRING, TafTalent.\"], \"exact_answer\": [[\"BioCreative Meta Server\"], [\"Hidden Vector State model\"], [\"iHop\"], [\"LAITOR\"], [\"OntoGene\"], [\"PIE\", \"Protein interaction information extraction system\"], [\"PPInterFinder\"], [\"PPIs\"], [\"TafTalent\"], [\"PolySearch\"], [\"OpenDMAP\"], [\"HAPPI\"], [\"GeneView\"], [\"PPI finder\", \"Paired-PPI Finder\"], [\"PreBind\", \"PreBIND\", \"Textomy\"], [\"BioMap\"], [\"eFIP\", \"Extracting Functional Impact of Phosphorylation\"], [\"Protopia\"], [\"STRING\"]], \"context\": \"In this study, we propose a novel algorithm for extracting PPIs from literature which consists of two phases. First, we automatically categorize the data into subsets based on its semantic properties and extract candidate PPI pairs from these subsets. Second, we apply support vector machines (SVMs) to classify candidate PPI pairs using features specific for each subset.The source code and scripts used in this article are available for academic use at http://staff.science.uva.nl/~bui/PPIs.zipWe describe a system for the detection of mentions of protein-protein interactions in the biomedical scientific literature. The original system was developed as a part of the OntoGene projectExtracting Protein-Protein Interactions from MEDLINE using the Hidden Vector State modelWe have constructed an information extraction system based on the Hidden Vector State (HVS) model for protein-protein interactions.When applied in extracting protein-protein interactions, we found that it performed better than other established statistical methodsThe developed BioMap system allows discovering implicit P-P interactions from large quantity of biomedical literature data.In this paper we examine if P-P interactions in regenerating tissues and cells of the anuran Xenopus laevis can be discovered from biomedical literature using computational and literature mining techniques. Using literature mining techniques, we have identified a set of implicitly interacting proteins in regenerating tissues and cells of Xenopus laevis that may interact with Cdc2 to control cell division.P-P interactions that are implicitly appearing in literature can be effectively discovered using literature mining techniques.\"}, {\"question\": \"Which are the most commonly reported pathological states associated with the formation of DNA G-quadruplexes?\", \"ideal_answer\": [\"There is a growing recognition for the profound role of G-quadruplexes in a wide spectrum of diseases, such as cancer, diabetes and cardiovascular disease. Amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD) were recently shown to be caused by expansion of a (GGGGCC)n/(GGCCCC)n repeat in the C9ORF72 gene.  Treatment with a G-quadruplex interactive ligand was shown to achieve antifibrotic action. G-quadruplex forming sequences have also been linked with ADAM10 a primary candidate for anti-amyloidogenic activity in Alzheimer's. A G-quadruplex-interactive agent, telomestatin (SOT-095), induces telomere shortening with apoptosis and enhances chemosensitivity in acute myeloid leukemia.\"], \"exact_answer\": [[\"Amyotrophic lateral sclerosis (ALS)\"], [\"frontotemporal dementia (FTD)\"], [\"Alzheimer's disease\"], [\"fibrosis\"], [\"acute myeloid leukemia\"]], \"context\": \"Amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD) were recently shown to be caused by expansion of a (GGGGCC)n/(GGCCCC)n repeat in the C9ORF72 geneMyocardial fibrosis is a key pathological change in a variety of heart diseases contributing to the development of heart failure, arrhythmias, and sudden death. Recent studies have shown that relaxin prevents and reverses cardiac fibrosis. Endogenous expression of relaxin was elevated in the setting of heart disease; the extent of such up-regulation, however, is insufficient to exert compensatory actions, and the mechanism regulating relaxin expression is poorly defined. In the rat relaxin-1 (RLN1, Chr1) gene promoter region we found presence of repeated guanine (G)-rich sequences, which allowed formation and stabilization of G-quadruplexes with the addition of a G-quadruplex interactive ligand berberine.Up-regulating relaxin expression by G-quadruplex interactive ligand to achieve antifibrotic actionOur findings document a novel therapeutic strategy for fibrosis through up-regulating expression of endogenous relaxin.We identified a G-rich sequence within exon 3 of BACE1 involved in controlling splice site selection\\u03b2-Site amyloid precursor protein (APP) cleaving enzyme 1 (BACE1) is the transmembrane aspartyl protease that catalyzes the first cleavage step in the proteolysis of the APP to the amyloid \\u03b2-protein (A\\u03b2), a process involved in the pathogenesis of Alzheimer diseaseAnti-amyloidogenic processing of the amyloid precursor protein APP by \\u03b1-secretase prevents formation of the amyloid-\\u03b2 peptide, which accumulates in senile plaques of Alzheimer disease patients. \\u03b1-Secretase belongs to the family of a disintegrin and metalloproteases (ADAMs), and ADAM10 is the primary candidate for this anti-amyloidogenic activityUsing circular dichroism spectroscopy, we demonstrate that a G-rich region between nucleotides 66 and 94 of the ADAM10 5'-UTR forms a highly stable, intramolecular, parallel G-quadruplex secondary structure under physiological conditionsFANCJ helicase defective in Fanconia anemia and breast cancer unwinds G-quadruplex DNA to defend genomic stability.Telomerase inhibition with a novel G-quadruplex-interactive agent, telomestatin: in vitro and in vivo studies in acute leukemiaRecently, we have demonstrated that treatment with a G-quadruplex-interactive agent, telomestatin reproducibly inhibited telomerase activity in the BCR-ABL-positive leukemic cell linesA G-quadruplex-interactive agent, telomestatin (SOT-095), induces telomere shortening with apoptosis and enhances chemosensitivity in acute myeloid leukemiaWe examined G-quadruplex interactive agent, telomestatin (SOT-095), for its ability to inhibit the proliferation of human leukemia cells, including freshly obtained leukemia cells.\"}, {\"question\": \"Which drugs are included in TAS-102?\", \"ideal_answer\": [\"TAS-102 is a novel oral nucleoside antitumor agent consisting of trifluridine and tipiracil hydrochloride at a molar ratio of 1:0.5.\"], \"exact_answer\": [[\"trifluridine\"], [\"tipiracil\"]], \"context\": \"PURPOSE: Trifluridine (TFT) is an antitumor component of a novel nucleoside antitumor agent, TAS-102, which consists of TFT and tipiracil hydrochloride (thymidine phosphorylase inhibitor).TAS-102, a combination of trifluorothymidine and the thymidine phosphorylase inhibitor TPI in a 1:0.5 ratio, is a novel oral formulation, which is active in 5FU-resistant models, both in vitro and in xenograft models.TAS-102 is a novel oral nucleoside antitumor agent consisting of trifluridine and tipiracil hydrochloride at a molar ratio of 1:0.5. PURPOSE: TAS-102 is an orally administered anticancer agent composed of \\u03b1,\\u03b1,\\u03b1-trifluorothymidine (FTD) and thymidine phosphorylase inhibitor (TPI). Novel DNA synthesis inhibitors for the treatment of gastrointestinal malignancies include a combination of the antimetabolite TAS-102, which consists of trifluorothymidine with a thymidine phosphorylase inhibitor, and a novel micellar formulation of cisplatin NC-6004 that uses a nanotechnology-based drug delivery system. Repeated oral dosing of TAS-102 confers high trifluridine incorporation into DNA and sustained antitumor activity in mouse models.TAS-102 is a novel oral nucleoside antitumor agent containing trifluridine (FTD) and tipiracil hydrochloride (TPI).\"}, {\"question\": \"What are the current treatments for generalised anxiety disorder in teenagers?\", \"ideal_answer\": [\"Cognitive-behavioral treatment (CBT) - both in individual and in group treatment\\nRandomised, placebo controlled trials have found Sertraline efficacious for GAD in adults, children and adolescents.\\nWhile both CBT and SSRIs are beneficial, some evidence suggests that the effects of CBT may be more long lasting.\"], \"exact_answer\": [[\"Cognitive-behavioral treatment (CBT)\", \"CBT\", \"group treatment\", \"manualised cognitive-behavioural therapy, FRIENDS\"], [\"Sertraline\", \"SSRI\"]], \"context\": \"49.3% of the youths with depression had comorbid conditions: anxiety disorders in 23.37% of casesTo evaluate the feasibility and effectiveness of a school-based group cognitive-behavioral treatment (CBT) for anxiety disorders with African-American adolescents. METHODS: Twelve adolescents (mean age = 15.6 years) with anxiety disorders were randomly assigned to CBT (n = 6) or a group attention-support control conditionAt posttreatment and among those who attended more than one treatment session, 3/4 adolescents in CBT no longer met diagnostic criteria for their primary anxiety disorder, compared with 1/5 in AS-Control. Clinician ratings of impairment and self-report levels of overall anxiety were significantly lower at posttreatment in CBT compared with AS-ControlThe present study compares an individual versus a group format in the delivery of manualised cognitive-behavioural therapy (FRIENDS) for children with anxiety disorders. Clinically referred children (aged 8 to 12) diagnosed with Separation Anxiety Disorder (n = 52), Generalised Anxiety Disorder (n = 37), Social Phobia (n = 22) or Specific Phobia (n = 16) were randomly assigned to individual (n = 65) or group (n = 62) treatment.Forty-eight percent of the children in the individual versus 41% in the group treatment were free of any anxiety disorder at post-treatment; 62% versus 54% were free of their primary anxiety disorder. Regression analyses showed no significant difference in outcome between individual and group treatmentChildren improved in both conditions. Choice between treatments could be based on pragmatic considerations such as therapeutic resources, referral rates, and the preference of the parents and the child.All child anxiety disorders were associated with several forms of anxiety disorder in the mother. Some specificity in the form of anxiety disorder in the child and the mother was apparent for social phobia and separation anxiety disorder. The findings have implications for the management of child anxiety.\"}, {\"question\": \"Which drugs acting via bradykinin system are effective for treatment of ACE-inhibitor-induced angioedema?\", \"ideal_answer\": [\"Icatibant and ecallantide are medication acting via bradykinin system that are used for treatment of ACE-inhibitor-induced angioedema.\"], \"exact_answer\": [[\"icatibant\"], [\"ecallantide\"]], \"context\": \"Medications recently developed, primarily icatibant and ecallantide, to control hereditary angioedema, a disorder also associated with kallikrein-kinin activation, have been used to treat ACEI angioedema with some success. A patient with acute ACE inhibitor-induced angioedema was treated with icatibant, a specific bradykinin B2 receptor antagonist approved for the treatment of hereditary angioedema. A patient with acute ACE inhibitor-induced angioedema was treated with icatibant, a specific bradykinin B2 receptor antagonist approved for the treatment of hereditary angioedema.\"}, {\"question\": \"Which forms of cancer is the Tpl2 gene associated with?\", \"ideal_answer\": [\"Tpl2/Map3K8, also known as tumor progression locus 2 has been identified as an oncogene, its mutation or overexpression is reported in a variety of human cancers. Types of cancer associated with Tpl2 include skin and epithelial cancers, ADI prostate cancer, gastric and colon adenocarcinomas, colitis-associated cancer (CAC), breast cancer, Hodgkin lymphomas, nasopharyngeal carcinomas and several types of T-cell neoplasias.\"], \"exact_answer\": [[\"Skin cancer\"], [\"ADI prostate cancer\"], [\"gastric adenocarcinoma\"], [\"colon adenocarcinoma\"], [\"colitis-associated cancer (CAC)\"], [\"breast cancer\"], [\"Hodgkin lymphoma\"], [\"nasopharyngeal carcinoma\"], [\"T-cell neoplasia\"]], \"context\": \"These findings establish a mesenchyme-specific role for Tpl2 in the regulation of HGF production and suppression of epithelial tumorigenesis.Tpl2 regulates intestinal myofibroblast HGF release to suppress colitis-associated tumorigenesisTpl2 ablation promotes intestinal inflammation and tumorigenesisLoss of tumor progression locus 2 (tpl2) enhances tumorigenesis and inflammation in two-stage skin carcinogenesis Previously identified as an oncogene, its mutation or overexpression is reported in a variety of human cancersThese studies indicate that Tpl2 may serve more as a tumor suppressor than as an oncogene in chemically induced skin carcinogenesis, with its absence contributing to both tumorigenesis and inflammation\"}, {\"question\": \"What are the indications for hydrochlorothiazide?\", \"ideal_answer\": [\"Hydrochlorothiazide is a diuretic, often used in combination with others. Hydrochlorothiazide are used to treat hypertension. Hydrochlorothiazide has been shown to decrease diastolic blood pressure.\"], \"exact_answer\": [[\"diuretic\"], [\"hypertension\"]], \"context\": \"The most frequently used diuretic in combination is again hydrochlorothiazide, which is combined with reninangiotensin system blockersAlso a combination of two diuretics is popular -\\u200a mainly hydrochlorothiazide + amiloridearticle gives an overview of the risk factors for hypertension and the appropriate indication for using a fixed combination of telmisartan and hydrochlorothiazideHydrochlorothiazide 25-200 mg daily, chlorothiazide 500 mg twice daily, and indapamide 2.5 mg daily provided long-term blood pressure reduction in patients with severe renal disease who were not on dialysisThis review provides an overview of the efficacy and safety of antihypertensive therapy based on olmesartan medoxomil\\u2009\\u00b1\\u2009hydrochlorothiazide and amlodipine/olmesartan medoxomil in high-risk patient populations enrolled in studies that reported ambulatory BP endpointsIn this regard, ARB-based SPCs are available in combination with the diuretic, hydrochlorothiazide (HCTZ) or the calcium CCB, amlodipineTelmisartan has a favourable safety and tolerability profile, and has demonstrated efficacious and long-lasting 24-hour BP reductions, whether as monotherapy or in combination with hydrochlorothiazide or amlodipineThe combination of irbesartan and hydrochlorothiazide resulted in additive antihypertensive effectsPatients with stable hypertension, not taking antihypertensive or NSAID medications, were treated with HCTZ 50 mg/dayis study determined the effect of nonsteroidal anti-inflammatory drug (NSAID) administration on blood pressure in hypertensive patients taking hydrochlorothiazide (HCTZ). Hydrochlorothiazide treatment decreased diastolic blood pressure to 83.1 +/- 5.6 mm Hg, and MAP to 101.1 +/- 6.5 mm HgTreating systolic hypertension in the very elderly with valsartan-hydrochlorothiazide vs. either monotherapy: ValVET primary results.This 16-week trial investigated the efficacy and safety of single-pill valsartan/hydrochlorothiazide (HCTZ) vs. the individual components in patients 70years and older with systolic hypertension.The article gives an overview of the risk factors for hypertension and the appropriate indication for using a fixed combination of telmisartan and hydrochlorothiazide\"}, {\"question\": \"What hand deformities do patients with Apert syndrome present with?\", \"ideal_answer\": [\"In patients with Apert syndrome, the hands demonstrate many disturbances of soft tissue and bony structures. These include a short thumb with radial clinodactyly, complex syndactyly with a bony fusion involving the index, long and ring fingers, symphalangism and simple syndactyly of the fourth web space. The soft tissue anomalies involve the intrinsic muscles, the extrinsic tendon insertions and the neurovascular bundles.\"], \"exact_answer\": [[\"complex syndactyly with bony fusion involving the index\"], [\"polydactyly\"], [\"short thumb with radial clinodactyly\"], [\"long and ring fingers\"], [\"symphalangism\"], [\"simple syndactyly of the fourth web space\"], [\"intrinsic muscle anomalies\"], [\"extrinsic tendon insertions\"], [\"neurovascular bundles\"]], \"context\": \"Five patients, ranging from Upton type 1 to type 3 Apert hand deformities, have had preoperative computed tomography angiography that delineated the vascular anatomy. This allowed planning and execution of a single-stage syndactyly release in all patients.The protocol presented allows preoperative planning and single-stage operation for complete release of syndactyly in patients with Apert syndrome.In patients with Apert syndrome, the hands demonstrate many disturbances of soft tissue and bony structures. These include a short thumb with radial clinodactyly, complex syndactyly with a bony fusion involving the index, long and ring fingers, symphalangism and simple syndactyly of the fourth web space. The soft tissue anomalies involve the intrinsic muscles, the extrinsic tendon insertions and the neurovascular bundles. Two of the patients had Apert syndrome (syndromic craniosynostosis with symmetrical syndactyly) and two had polydactyly. The overall strategy involved early bilateral separation of syndactylous border digits at 1 year of age, followed by sequential unilateral middle syndactyly mass separation with thumb osteotomy and bone grafting as needed. In these 10 patients, a total of 53 web spaces were released, 49 of which involved osteotomies for complex syndactyly. Apert syndrome, characterised by craniosynostosis, craniofacial anomalies, and symmetrical syndactyly of the digits (cutaneous and bony fusion), has been associated with two canonical mutations in the FGFR2 gene (S252W, P253R) in the great majority of cases. The Apert syndrome hand demonstrates many typical clinical features including syndactyly, symbrachyphalangism, and growth disturbances.In Apert syndrome, characterised by syndactyly of the hands and feet, recurrent mutations of a serine-proline dipeptide (either Ser252Trp or Pro253Arg) in the linker between the IgII and IgIII extracellular immunoglobulin-like domains, have been documented in more than 160 unrelated individuals. urgical correction of syndactyly of the Apert hand should begin by 6 months and be completed by 3 years of age. As much surgery as possible is carried out at each sitting. The syndactyly cases were more complicated than the average, among them 4 cases of Apert syndrome were noted. \"}, {\"question\": \"Which genes are thought to be involved in medulloblastoma development?\", \"ideal_answer\": [\"Medulloblastomas are the most frequent malignant brain tumors affecting children. Disease development has been suggested to be associated with a significant number of genes, such as PTCH1, SUFU, PTEN, CREBBP, PTEN, MYT1L, NFIA, NFIB, TEAD1, TGIF2, IGF2, PCDH10, BMI1, MYC, OTX2, RASSF1A, HIC1, and CASP8.\"], \"exact_answer\": [[\"PTCH1\"], [\"SUFU\"], [\"PTEN\"], [\"CREBBP\"], [\"PTEN\"], [\"MYT1L\"], [\"NFIA\"], [\"NFIB\"], [\"TEAD1\"], [\"TGIF2\"], [\"IGF2\"], [\"PCDH10\"], [\"BMI1\"], [\"MYC\"], [\"OTX2\"], [\"RASSF1A\"], [\"HIC1\"], [\"CASP8\"]], \"context\": \"Medulloblastomas, the most frequent malignant brain tumours affecting children, comprise at least 4 distinct clinicogenetic subgroups. Aberrant sonic hedgehog (SHH) signalling is observed in approximately 25% of tumours and defines one subgroup. Although alterations in SHH pathway genes (e.g. PTCH1, SUFU) are observed in many of these tumours, high throughput genomic analyses have identified few other recurring mutationsMutagenesis significantly increased medulloblastoma frequency and identified 17 candidate cancer genes, including orthologs of genes somatically mutated (PTEN, CREBBP) or associated with poor outcome (PTEN, MYT1L) in the human disease. Strikingly, these candidate genes were enriched for transcription factors (p=2x10-5), the majority of which (6/7; Crebbp, Myt1L, Nfia, Nfib, Tead1 and Tgif2) were linked within a single regulatory network enriched for genes associated with a differentiated neuronal phenotypeIgf2, previously implicated in medulloblastoma, was the most differentially expressed gene in murine tumours with network perturbationPCDH10 is a candidate tumour suppressor gene in medulloblastomaThe aim of this study was to investigate the genetic and epigenetic mechanisms contributing to PCDH10 down-regulation in medulloblastomaWe report a very focal homozygous deletion on chromosome 4q28.3 harbouring the PCDH10 gene. We demonstrate that PCDH10 transcription is down-regulated in 19/26 (73%) of medulloblastomas suggesting that other mechanisms also could be involved in gene repressionOur findings suggest that genetic and epigenetic deregulation of PCDH10 occurs in a significant portion of medulloblastoma patients. Failure to express PCDH10 may result in loss of inhibition of cell migration, thereby contributing to medulloblastoma progressionWe found that the higher expression levels of BMI1 and PCGF2 genes were associated with significantly decreased patient survivalOur analysis showed correlation between BMI1 and PCGF2 gene's expression and survival in children with medulloblastomaBmi1 is required for Hedgehog pathway-driven medulloblastoma expansionThese data provide the first demonstration that Bmi1 is required for spontaneous de novo development of a solid tumor arising in the brain, suggest a crucial role for Bmi1-dependent, nestin-expressing progenitor cells in medulloblastoma expansion, and implicate Bmi1 as a key factor required for Hh pathway-driven tumorigenesisGenomic amplification of orthodenticle homologue 2 in medulloblastomasGenomic alterations normally associated with medulloblastomas, including MYC amplification and isochromosome 17q, were easily detected. Surprisingly, analysis of only five genomes revealed novel amplicons on chromosome 14q, one of which contained the orthodenticle homologue 2 (OTX2) homeobox gene. DNA copy number analysis showed that OTX2 had undergone genomic amplification in 2 of 11 medulloblastoma cell lines and 8 of 42 primary tumorsSerial analysis of gene expression of 240 different human tumors or normal tissues revealed that 96% of all 783 OTX2 transcripts sequenced were in medulloblastomas or embryonic stem cellsOTX2 functions to specify the fate of neuroectoderm in various regions of the developing brain. This developmental role is consistent with the evidence suggesting that OTX2 is a medulloblastoma oncogeneExtensive hypermethylation of RASSF1A was detected frequently in medulloblastomas but not in the normal cerebellum (41/44 primary tumours versus 0/5 normal cerebella)In contrast, complete methylation of HIC1 and CASP8 in a subset of primary tumours (17/44 and 14/39) occurred against a consistent background of partial methylation in the normal cerebellumThese data therefore indicate that extensive methylation of RASSF1A, HIC1 and CASP8 are tumour-specific events in medulloblastomaWe conclude that epigenetic TSG inactivation is a significant feature of medulloblastoma, and identify RASSF1A, HIC1 and CASP8 as potentially critical genes in its pathogenesisBiallelic epigenetic inactivation of the RASSF1A tumor suppressor gene in medulloblastoma developmentSeventy-nine % (27 of 34) of primary tumors and 100% (8 of 8) of medulloblastoma cell lines displayed extensive tumor-specific DNA hypermethylation across the RASSF1A promoter-associated CpG islandHypermethylation was associated with epigenetic silencing of RASSF1A transcription in medulloblastoma cell lines, and RASSF1A expression in these lines was restored after treatment with the DNA-methyltransferase inhibitor 5-aza-2'-deoxycytidineEpigenetic inactivation by biallelic hypermethylation therefore represents the primary mechanism of RASSF1A gene inactivation in medulloblastomaRASSF1A hypermethylation is a frequent event in medulloblastoma tumorigenesis detectable in adult (5 of 7) and pediatric patients (22 of 27) and in all histological variants and age and sex groupingsMedulloblastoma, a brain tumor, develops in about 3% of NBCCS patients, and mutations in PTCH have also been described in a subset of sporadic medulloblastomasRecently, a transgenic mouse hemizygous for the Ptch gene was generated by homologous recombination. Medulloblastomas were found in about 19% of these mice within the first 25 weeks after birthWe also examined the mRNA expression levels for the remaining Ptch allele, as well as for Gli1, a gene known to be transcriptionally activated by Ptch inactivation. Blot analysis of RNA from the 13 tumors shows that Ptch mRNA of appropriate size is expressed in all tumors at varying levels. Expression of Gli1 was increased in tumors compared to normal cerebellum. These results suggest that deletion of one copy of Ptch may be sufficient to promote medulloblastoma development in mice\"}, {\"question\": \"Which sports have a risk for commotio cordis?\", \"ideal_answer\": [\"Participation in sports such as baseball, football, soccer, cricket, hockey and lacrosse has a risk for commotio cordis.\"], \"exact_answer\": [[\"baseball\"], [\"football\"], [\"soccer\"], [\"cricket\"], [\"hockey\"], [\"lacrosse\"]], \"context\": \"This study sought to characterize the demographics of commotio cordis globally in comparison to the U.S. experience.Not unexpectedly, the groups differed with baseball/softball and football predominant in the United States (55% of events) and soccer, cricket, and hockey most common internationally (47% of events). The authors reviewed 41 incidents of baseball injuries reported to the National Center for Catastrophic Sports Injury Research from 1982 until 2002.Catastrophic injuries included 23 severe head injuries, 8 cervical injuries, 3 cases of commotio cordis, and 2 cases each of a collapsed trachea and facial fractures.Commotio cordis events occurred most commonly during organized sporting events (79 [62%]), such as baseball, but 49 (38%) occurred as part of daily routine and recreational activities.Twenty-two (28%) participants were wearing commercially available chest barriers, including 7 in whom the projectile made direct contact with protective padding (baseball catchers and lacrosse/hockey goalies), and 2 in whom the projectile was a baseball specifically designed to reduce risk.In a recent study of fatal chest impacts by baseballs, 28% of the children were wearing a chest protector. This study evaluates the effectiveness of chest protectors in reducing the risk of commotio cordis. It has been described after blows to the chest from baseballs, softballs, hockey pucks, and other objects.Commotio cordis due to blunt trauma to the precordium is a rare cause of death in young athletes, occurring less frequently than all of the other athletics-related deaths. Several measures, such as the use of safety baseballs and the use of chest protectors, can help protect young athletes from commotio cordis. \"}, {\"question\": \"What are the breath test biomarkers of pulmonary tuberculosis\", \"ideal_answer\": [\"Nitric oxide, urea, volatile organic compounds, hydrogen peroxide and end products of lipid peroxidation are the breath test biomarkers of pulmonary tuberculosis.\"], \"exact_answer\": [[\"nitric oxide\"], [\"urea\"], [\"volatile organic compounds\"], [\"hydrogen peroxide\"], [\"end products of lipid peroxidation\"]], \"context\": \"BACKGROUND: Nitric oxide (NO), a key macrophage antimycobacterial mediator that ameliorates immunopathology, is measurable in exhaled breath in individuals with pulmonary tuberculosis. CONCLUSIONS: Among patients with pulmonary tuberculosis, impaired pulmonary NO bioavailability is associated with more-severe disease and delayed mycobacterial clearance.Four KI windows corresponded with KI values of VOCs previously identified as biomarkers of pulmonary TB and metabolic products of M. tuberculosis, principally derivatives of naphthalene, benzene and alkanes. Measurement of exhaled nitric oxide as a potential screening tool for pulmonary tuberculosis.CONCLUSION: While eNO measurement has limited value in the direct diagnosis of pulmonary TB, it may be worth developing and evaluating as a cost-effective replacement of chest X-ray in screening algorithms of pulmonary TB where X-ray is not available.13[C]-urea breath test as a novel point-of-care biomarker for tuberculosis treatment and diagnosis.CONCLUSIONS/SIGNIFICANCE: Urea breath testing may provide a useful diagnostic and biomarker assay for tuberculosis and for treatment response. RESULTS: Breath VOCs contained apparent biomarkers of active pulmonary tuberculosis comprising oxidative stress products (alkanes and alkane derivatives) and volatile metabolites of M. tuberculosis (cyclohexane and benzene derivatives).Pulmonary tuberculosis may alter volatile organic compounds (VOCs) in breath because Mycobacteria and oxidative stress resulting from Mycobacterial infection both generate distinctive VOCs. We conclude that volatile biomarkers in breath were sensitive and specific for pulmonary tuberculosis: the breath test distinguished between \\\"sick versus well\\\" i.e. between normal controls and patients hospitalized for suspicion of pulmonary tuberculosis, and between infected versus non-infected patients i.e. between those whose sputum cultures were positive or negative for Mycobacteria.RESULTS: The concentrations of H202 and TBARs (1022.96+/-186.02 nM and 4.22+/-0. 80 microM, respectively) were significantly higher in patients with tuberculosis as compared with the controls (398.15+/-37.10 nM and 0.48+/-0.17 microM, respectively). We conclude that volatile biomarkers in breath were sensitive and specific for pulmonary tuberculosis: the breath test distinguished between \\\"sick versus well\\\" i.Volatile organic compounds (VOCs) in breath may contain biomarkers of active pulmonary tuberculosis derived from the infectious organism (metabolites of Mycobacterium tuberculosis) and from the infected host (products of oxidative stress).Volatile biomarkers of pulmonary tuberculosis in the breath.Breath VOCs contained apparent biomarkers of active pulmonary tuberculosis comprising oxidative stress products (alkanes and alkane derivatives) and volatile metabolites of M.CONCLUSIONS: A six-minute point-of-care breath test for volatile biomarkers accurately identified subjects with active pulmonary TB.BACKGROUND: Volatile organic compounds (VOCs) in breath may contain biomarkers of active pulmonary tuberculosis derived from the infectious organism (metabolites of Mycobacterium tuberculosis) and from the infected host (products of oxidative stress).We conclude that volatile biomarkers in breath were sensitive and specific for pulmonary tuberculosis: the breath test distinguished between \\\"sick versus well\\\" iWe conclude that volatile biomarkers in breath were sensitive and specific for pulmonary tuberculosis: the breath test distinguished between \\\"sick versus well\\\" i.eWe evaluated breath VOC biomarkers in subjects with active pulmonary TB, using an internet-linked rapid point-of-care breath test.Urea breath testing may provide a useful diagnostic and biomarker assay for tuberculosis and for treatment responseA six-minute point-of-care breath test for volatile biomarkers accurately identified subjects with active pulmonary TB.\"}, {\"question\": \"What types of DNA mutations are induced by 2-hydroxy-dATP (2-OH-dATP)?\", \"ideal_answer\": [\"2-hydroxy-dATP mainly elicits G:C --> A:T transitions, and, to a lesser extent, G:C --> T:A transversions The induction of G:C --> T:A transversions by 2-OH-dATP indicates the formation of G*2-OH-dATP pairs. 2-OH-dATP also induces tandem (CC --> TT) mutations. Altogether, 2-OH-dATP induces both transition and transvertion mutations, such as A:T --> G:C, A:T --> C:G and G:C --> T:A mutations.\"], \"exact_answer\": [[\"G:C --> A:T transitions\"], [\"tandem (CC --> TT) mutations\"], [\"G:C --> T:A transversions\"], [\"A:T --> C:G mutations\"]], \"context\": \"2-OH-dATP induced both transition and transversion mutations, including A:T-->G:C, A:T-->C:G and G:C-->T:A mutations.a G:C-->T:A transversion, the same type of mutation elicited by 2-OH-dATP.2-OH-dATP mainly elicited a G x C to A x T transition, and a G x C to T x A transversion to a lesser extent.2-OH-dATP induced mutations in a dose-dependent manner and elicited substitution and deletion mutations. Of the substitutions, a G.C-->A.T transition including a tandem (CC-->TT) mutation was mainly observed.2-OH-dATP elicits G.C-->T.A transversionsthe alpha subunit incorporated 2-OH-dATP 10 times more frequently opposite T than opposite G.2-OH-dATP elicited G-->T transversions, indicating the formation of G*2-OH-dATP pairs.Interestingly, 2-OH-dATP induced both transition and transversion mutations, including A:T-->G:C, A:T-->C:G and G:C-->T:A mutations.The polymerase almost exclusively incorporated 8-hydroxy-dGTP (8-OH-dGTP) opposite template adenine (A) at 60% efficiency of normal dTTP incorporation, and incorporated 2-hydroxy-dATP (2-OH-dATP) opposite template thymine (T), guanine (G), or cytosine (C) at substantial rates.2-OH-dATP induced mutations in a dose-dependent manner and elicited substitution and deletion mutations.Steady-state kinetic studies indicated that the alpha subunit incorporated 2-OH-dATP 10 times more frequently opposite T than opposite G. On the other hand, the incorporation of 2-OH-dATP opposite T by the exonuclease-deficient Klenow fragment was 2 orders of magnitude more efficient than that opposite G. These results indicate that the misinsertion specificity of 2-OH-dATP differs between replicative and repair-type DNA polymerases, and provide a biochemical basis for the mutations induced by 2-OH-dATP in E. coli.Interestingly, 2-OH-dATP induced both transition and transversion mutations, including A:T-->G:C, A:T-->C:G and G:C-->T:A mutations.2-OH-dATP induced mutations in a dose-dependent manner and elicited substitution and deletion mutations.On the other hand, in an experiment using the Klenow fragment, incorporation of 2-OH-dATP was observed only opposite T. Steady-state kinetic studies indicated that incorporation of 2-OH-dATP by DNA polymerase alpha opposite T was favored over that opposite C by a factor of only 4.5.2-OH-dATP induced mutations in a dose-dependent manner and elicited substitution and deletion mutations.These mutations include various mutations involving a G:C-->T:A transversion, the same type of mutation elicited by 2-OH-dATP2-OH-dATP induced mutations in a dose-dependent manner and elicited substitution and deletion mutations\"}, {\"question\": \"Which diseases can be treated with Afamelanotide?\", \"ideal_answer\": [\"Afamelanotide was ivestigated for treatment of erythropoietic protoporphyria, vitiligo, Hailey-Hailey disease,  acne vulgaris, polymorphic light eruption, prevention of actinic keratoses in organ transplant recipients and nonmelanoma skin cancer.\"], \"exact_answer\": [[\"erythropoietic protoporphyria\"], [\"vitiligo\"], [\"Hailey-Hailey disease\"], [\"acne vulgaris\"], [\"polymorphic light eruption\"], [\"ctinic keratoses\"], [\"nonmelanoma skin cancer\"]], \"context\": \"Afamelanotide for Erythropoietic Protoporphyria.CONCLUSIONS: Afamelanotide had an acceptable side-effect and adverse-event profile and was associated with an increased duration of sun exposure without pain and improved quality of life in patients with erythropoietic protoporphyria.Afamelanotide and narrowband UV-B phototherapy for the treatment of vitiligo: a randomized multicenter trial.CONCLUSIONS AND RELEVANCE: A combination of afamelanotide implant and NB-UV-B phototherapy resulted in clinically apparent, statistically significant superior and faster repigmentation compared with NB-UV-B monotherapy.UNLABELLED: Afamelanotide is an \\u03b1-melanocyte-stimulating hormone (\\u03b1-MSH) agonist with proven efficacy in photodermatoses such as erythropoietic protoporphyria (EPP). Afamelanotide (CUV1647) in dermal phototoxicity of erythropoietic protoporphyria.CONCLUSIONS: Afamelanotide is effective for the treatment of skin lesions in HHD.CONCLUSIONS: Afamelanotide appears to have anti-inflammatory effects in patients with mild-to-moderate acne vulgaris.In a recent trial, afamelanotide administered as controlled release implants protected erythropoietic protoporphyria (EPP) patients from sunlight induced phototoxic skin reactions.CONCLUSIONS: We propose that afamelanotide represents a novel and potentially effective treatment for vitiligo.Afamelanotide is currently undergoing phase II and III trials in Europe and the US for skin diseases including vitiligo, erythropoietic protoporphyria, polymorphic light eruption and prevention of actinic keratoses in organ transplant recipients. Currently, afamelanotide is already on the market in Italy and Switzerland for patients with erythropoietic protoporphyria. Agents that are showing promising results in early phases of clinical trials include betulinic acid; hedgehog signaling pathway inhibitors, such as cyclopamine and GDC-0449; alpha-melanocyte-stimulating hormone analogs, such as afamelanotide; epidermal growth factor receptor inhibitors, such as gefitinib and erlotinib; anti-epidermal growth factor receptor monoclonal antibodies, such as cetuximab and panitumumab; and the 5-fluorouracil prodrug capecitabine.Meanwhile, the regulated \\u03b1-MSH analogue afamelanotide (Clinuvel Pharmaceuticals Ltd, Melbourne, Australia) is showing promise for its photoprotective potential, and is undergoing phase II and III clinical trials in people with photosensitivity disorders and those prone to nonmelanoma skin cancer.CONCLUSIONS: Among the six models proposed to assess the effectiveness of therapeutic interventions in PP the ETFP model demonstrates the highest sensitivity using the existing data from a clinical trial of afamelanotide in PP.Afamelanotide, an agonistic analog of \\u03b1-melanocyte-stimulating hormone, in dermal phototoxicity of erythropoietic protoporphyria.IMPORTANCE OF THE FIELD: Afamelanotide, an \\u03b1-melanocyte stimulating hormone (MSH) agonistic analog is a first-in-class therapeutic. Its application to protoporphyria (PP), a disease associated with absolute sunlight-intolerance is discussed.TAKE HOME MESSAGE: Although early, results of the first trials of afamelanotide for PP are promising and the risk-safety profile appears favorable today.This study examines the efficacy of afamelanotide in preventing symptoms in patients with EPP.The findings demonstrate beneficial effects of afamelanotide in patients with EPP.Afamelanotide is currently undergoing phase II and III trials in Europe and the US for skin diseases including vitiligo, erythropoietic protoporphyria, polymorphic light eruption and prevention of actinic keratoses in organ transplant recipients.The application of afamelanotide, an \\u03b1-melanocyte stimulating hormone agonistic analogue to protoporphyria, a disease with absolute sunlight-intolerance is discussed.UNLABELLED: Afamelanotide is an \\ufffd-melanocyte-stimulating hormone (\\ufffd-MSH) agonist with proven efficacy in photodermatoses such as erythropoietic protoporphyria (EPP). This peptide drug, repeatedly administered over prolonged time, may induce anti-drug antibodies (ADA). Adverse events were mostly mild; serious adverse events were not thought to be related to the study drug.CONCLUSIONS: Afamelanotide had an acceptable side-effect and adverse-event profile and was associated with an increased duration of sun exposure without pain and improved quality of life in patients with erythropoietic protoporphyria. (Funded by Clinuvel Pharmaceuticals and others; ClinicalTrials.gov numbers, NCT01605136 and NCT00979745.).\"}, {\"question\": \"List features of the Perry syndrome.\", \"ideal_answer\": [\"Perry syndrome is a familial parkinsonism associated with central hypoventilation, mental depression, and weight loss.\"], \"exact_answer\": [[\"parkinsonism\"], [\"central hypoventilation\"], [\"depression\"], [\"weight loss\"]], \"context\": \"BACKGROUND: Depression, parkinsonism, and hypoventilation (Perry syndrome) or familial motor neuron disease have been linked to mutations in dynactin P150(Glued) (DCTN1).BACKGROUND: Perry syndrome (PS) caused by DCTN1 gene mutation is clinically characterized by autosomal dominant parkinsonism, depression, severe weight loss, and hypoventilation.CONCLUSIONS AND RELEVANCE: Mutations of the DCTN1 gene have been previously associated with amyotrophic lateral sclerosis and with Perry syndrome, a rare autosomal dominant disorder characterized by weight loss, parkinsonism, central hypoventilation, and psychiatric disturbances. OBJECTIVES: Perry syndrome consists of autosomal dominant Parkinsonism, depression, weight loss, and central hypoventilation.A patient with a mood disorder and a Parkinsonian syndrome with frontal cognitive impairment thought to resemble progressive supranuclear palsy defied precise diagnosis until the development of respiratory compromise, prompting consideration of the diagnosis of Perry syndrome.Perry syndrome is a familial parkinsonism associated with central hypoventilation, mental depression, and weight loss. Perry syndrome (PS) caused by DCTN1 gene mutation is clinically characterized by autosomal dominant parkinsonism, depression, severe weight loss, and hypoventilationPerry syndrome is a familial parkinsonism associated with central hypoventilation, mental depression, and weight lossPerry syndrome due to the DCTN1 G71R mutation: a distinctive levodopa responsive disorder with behavioral syndrome, vertical gaze palsy, and respiratory failurePerry syndrome consists of early-onset parkinsonism, depression, severe weight loss and hypoventilation, with brain pathology characterized by TDP-43 immunostaining. We carried out genome-wide linkage analysis and identified five disease-segregating mutations affecting the CAP-Gly domain of dynactin (encoded by DCTN1) in eight families with Perry syndrome; these mutations diminish microtubule binding and lead to intracytoplasmic inclusions. \"}, {\"question\": \"For which diseases members of the 2-aminobenzamide class of histone deacetylase (HDAC) inhibitors show promise as therapeutics?\", \"ideal_answer\": [\"Members of the 2-aminobenzamide class of histone deacetylase (HDAC) inhibitors show promise as therapeutics for the neurodegenerative diseases Friedreich's ataxia (FRDA) and Huntington's disease (HD).\", \"Members of the 2-aminobenzamide class of histone deacetylase (HDAC) inhibitors show promise as therapeutics for the neurodegenerative diseases Friedreich's ataxia (FRDA) and Huntington's disease (HD)\"], \"exact_answer\": [[\"Friedreich's ataxia (FRDA)\"], [\"Huntington's disease (HD)\"]], \"context\": \"Members of the 2-aminobenzamide class of histone deacetylase (HDAC) inhibitors show promise as therapeutics for the neurodegenerative diseases Friedreich's ataxia (FRDA) and Huntington's disease (HD)Histone deacetylase (HDAC) inhibitors have received considerable attention as potential therapeutics for a variety of cancers and neurological disorders. Recent publications on a class of pimelic diphenylamide HDAC inhibitors have highlighted their promise in the treatment of the neurodegenerative diseases Friedreich's ataxia and Huntington's disease, based on efficacy in cell and mouse modelsWe recently identified a novel class of pimelic o-aminobenzamide histone deacetylase (HDAC) inhibitors that show promise as therapeutics in the neurodegenerative diseases Friedreich's ataxia (FRDA) and Huntington's disease (HD)We recently identified a class of pimelic diphenylamide histone deacetylase (HDAC) inhibitors that show promise as therapeutics in the neurodegenerative diseases Friedreich's ataxia (FRDA) and Huntington's diseaseMembers of the 2-aminobenzamide class of histone deacetylase (HDAC) inhibitors show promise as therapeutics for the neurodegenerative diseases Friedreich's ataxia (FRDA) and Huntington's disease (HD).We recently identified a novel class of pimelic o-aminobenzamide histone deacetylase (HDAC) inhibitors that show promise as therapeutics in the neurodegenerative diseases Friedreich's ataxia (FRDA) and Huntington's disease (HD).Rationale for the development of 2-aminobenzamide histone deacetylase inhibitors as therapeutics for Friedreich ataxia.While it is clear that HDAC3 is one of the important targets of the 2-aminobenzamide HDAC inhibitors, inhibition of other class I HDACs (HDACs 1 and 2) may also be involved in the beneficial effects of these compounds in FRDA and HD, and other HDAC interacting proteins may be impacted by the compound.The authors' laboratories have identified a novel class of histone deacetylase inhibitors (2-aminobenzamides) that reverses heterochromatin-mediated silencing of the frataxin (FXN) gene in Friedreich ataxia.Members of the 2-aminobenzamide class of histone deacetylase (HDAC) inhibitors show promise as therapeutics for the neurodegenerative diseases Friedreich&apos;s ataxia (FRDA) and Huntington&apos;s disease (HD)While it is clear that HDAC3 is one of the important targets of the 2-aminobenzamide HDAC inhibitors, inhibition of other class I HDACs (HDACs 1 and 2) may also be involved in the beneficial effects of these compounds in FRDA and HD, and other HDAC interacting proteins may be impacted by the compoundMembers of the 2-aminobenzamide class of histone deacetylase (HDAC) inhibitors show promise as therapeutics for the neurodegenerative diseases Friedreich's ataxia (FRDA) and Huntington's disease (HD). We recently identified a novel class of pimelic o-aminobenzamide histone deacetylase (HDAC) inhibitors that show promise as therapeutics in the neurodegenerative diseases Friedreich's ataxia (FRDA) and Huntington's disease (HD). We recently identified a novel class of pimelic o-aminobenzamide histone deacetylase (HDAC) inhibitors that show promise as therapeutics in the neurodegenerative diseases Friedreich's ataxia (FRDA) and Huntington's disease (HD).The authors' laboratories have identified a novel class of histone deacetylase inhibitors (2-aminobenzamides) that reverses heterochromatin-mediated silencing of the frataxin (FXN) gene in Friedreich ataxia.Members of the 2-aminobenzamide class of histone deacetylase (HDAC) inhibitors show promise as therapeutics for the neurodegenerative diseases Friedreich's ataxia (FRDA) and Huntington's disease (HD). While it is clear that HDAC3 is one of the important targets of the 2-aminobenzamide HDAC inhibitors, inhibition of other class I HDACs (HDACs 1 and 2) may also be involved in the beneficial effects of these compounds in FRDA and HD, and other HDAC interacting proteins may be impacted by the compound.We recently identified a novel class of pimelic o-aminobenzamide histone deacetylase (HDAC) inhibitors that show promise as therapeutics in the neurodegenerative diseases Friedreich's ataxia (FRDA) and Huntington's disease (HD).However, histone deacetylase inhibitors tested to date either are highly cytotoxic or have very low specificities for different histone deacetylase enzymes. The authors' laboratories have identified a novel class of histone deacetylase inhibitors (2-aminobenzamides) that reverses heterochromatin-mediated silencing of the frataxin (FXN) gene in Friedreich ataxia.Members of the 2-aminobenzamide class of histone deacetylase (HDAC) inhibitors show promise as therapeutics for the neurodegenerative diseases Friedreich's ataxia (FRDA) and Huntington's disease (HD).Rationale for the development of 2-aminobenzamide histone deacetylase inhibitors as therapeutics for Friedreich ataxia.The authors' laboratories have identified a novel class of histone deacetylase inhibitors (2-aminobenzamides) that reverses heterochromatin-mediated silencing of the frataxin (FXN) gene in Friedreich ataxia.\"}, {\"question\": \"List anti-amyloid-beta monoclonal antibodies that have been investigated in clinical trials for treatment of Alzheimer disease.\", \"ideal_answer\": [\"Ponezumab, solanezumab and bapineuzumab are humanized antiamyloid beta (A\\u03b2) monoclonal antibodies that have been designed for treatment of Alzheimer disease.\", \"Bapineuzumab\\nSolanezumab\\nPonezumab\\nGantenerumab\"], \"exact_answer\": [[\"Bapineuzumab\", \"bapineuzumab\"], [\"Solanezumab\", \"solanezumab\"], [\"Ponezumab\", \"ponezumab\"], [\"Gantenerumab\"]], \"context\": \"The first active immunization clinical trial with AN1792 in AD patients was halted when a subset of patients developed meningoencephalitis.The first passive immunotherapy trial with bapineuzumab,Preliminary results of a prematurely terminated clinical trial where AD patients were actively vaccinated with aggregated Abeta strategies based on A\\u03b21-42 peptide induced encephalomyelitis and possible microhemorrhages.Phase III trials showed that bapineuzumab failed to improve cognitive and functional performances in AD patients, and was associated with a high incidence of amyloid-related imaging abnormalities (ARIA).Solanezumab's two Phase III trials in AD patients failed to meet endpoints when analyzed independently. Unfortunately, the first active vaccine (AN1792, consisting of preaggregate A\\u03b2 and an immune adjuvant, QS-21) was abandoned because it caused meningoencephalitis in approximately 6% of treated patientsAnti-A\\u03b2 monoclonal antibodies (bapineuzumab and solanezumab) are now being developed.several monoclonal and polyclonal antibodies are in clinical trials. These are bapineuzumab, solanezumab, ponezumab, gantenerumab, BAN2401, gammaguard and octagam. Since each antibody has a different antigen epitope of Abeta, anti-amyloid activities are different.Several types of A\\u03b2 peptide immunotherapy for Alzheimer's disease are under investigation, direct immunization with synthetic intact A\\u03b2(42) , active immunization involving the administration of synthetic fragments of A\\u03b2 peptide conjugated to a carrier protein and passive administration with monoclonal antibodies directed against A\\u03b2 peptideAn active anti-A\\u03b2 vaccine preparation, AN1792, has been used in AD patients with some clues of clinical efficacy but causing meningoencephalitis in about 6% of patients and it has been abandonedThe most advanced of these immunological approaches is bapineuzumab, composed of humanized anti-A\\u03b2 monoclonal antibodies, that has been tested in two Phase II trials, demonstrating to reduce A\\u03b2 burden in the brain of AD patients. The results of four ongoing large Phase III trials on bapineuzumab will tell us if passive anti-A\\u03b2 immunization is able to alter the course if this devastating disease.Solanezumab is a monoclonal antibody that binds to \\u03b2-amyloid (A\\u03b2), a protein that plays a key role in the pathogenesis of AD. The drug is currently being investigated in Phase III trials as a disease-modifying treatment for AD. The most advanced of these immunological approaches is bapineuzumab, which is composed of humanized anti-A\\u03b2 monoclonal antibodies that has been tested in two Phase II trials. The results of four ongoing large Phase III trials on bapineuzumab will provide answers regarding whether passive anti-A\\u03b2 immunization is able to alter the course of this devastating disease.\"}, {\"question\": \"Which anticancer drugs target human topoisomerase II?\", \"ideal_answer\": [\"Etoposide (VP-16) and Teniposide (VM-26) are effective as an anti-tumour drug by inhibiting eukaryotic DNA topoisomerase II via establishing a covalent complex with DNA. Doxorubicin, Daunorubicin and Aclarubicin are anthracyclins that act as DNA topoisomerase II inhibitors and may be used in combination. Benzoxazoles, benzimidazoles and related fused heterocyclic compounds, which exhibited significant eukaryotic DNA topoisomerase II inhibitory activity. F14512 is a polyamine-containing epipodophyllotoxin derivative that acts as an inhibitor of DNA topoisomerase II. Bisdioxopiperazine drugs such as ICRF-187 are catalytic inhibitors of DNA topoisomerase II. \\nAmong topoisomerase II inhibitors, the cytostatic potency was by decreasing order: mitoxantrone; doxorubicin, which was slightly greater than DuP 941, azatoxin; DuP 937; and amsacrine, which was much greater than VP-16\"], \"exact_answer\": [[\"Etoposide (VP-16)\"], [\"Teniposide (VM-26)\"], [\"Doxorubicin\"], [\"Daunorubicin\"], [\"Aclarubicin\"], [\"Mitoxantrone\"], [\"Amsacrine (m-AMSA)\"]], \"context\": \"Etoposide is effective as an anti-tumour drug by inhibiting eukaryotic DNA topoisomerase II via establishing a covalent complex with DNAbenzoxazoles, benzimidazoles and related fused heterocyclic compounds, which exhibited significant eukaryotic DNA topoisomerase II inhibitory activityF14512, a polyamine-containing inhibitor of DNA topoisomerase IIF14512, an epipodophyllotoxin derivative equipped with a spermine moiety, is selectively taken up by the polyamine transport system over-active in tumor cellsF14512 combines an epipodophyllotoxin core-targeting topoisomerase II with a spermine moiety introduced as a cell delivery vector. The polyamine tail supports three complementary functions: (a) facilitate formulation of a water-soluble compound, (b) increase DNA binding to reinforce topoisomerase II inhibition, and (c) facilitate selective uptake by tumor cells via the PTSTwenty previously synthesized fused heterocyclic DNA-topoisomerase II (Topo II)-inhibiting compounds were investigated for their potential efficacy in various human cancer cell lines that were derived from different tumor entitiesIn conclusion, compounds BD 13, BD 14, BD 16, D 23 and D 24 may be useful for the treatment of different multidrug-resistant cancer cells with cross resistance against \\\"classical\\\" Topo II-targeting drugs.ATPase domain of eukaryotic DNA topoisomerase II. Inhibition of ATPase activity by the anti-cancer drug bisdioxopiperazine and ATP/ADP-induced dimerizationWe tested the effects of two DNA topoisomerase II poisons, etoposide and doxorubicinthe bisdioxopiperazine topoisomerase II catalytic inhibitor ICRF-187Bisdioxopiperazine drugs such as ICRF-187 are catalytic inhibitors of DNA topoisomerase II, with at least two effects on the enzyme: namely, locking it in a closed-clamp form and inhibiting its ATPase activityThese findings suggest that the difference in drug sensitivities to doxorubicin and etoposide in human lung cancer cell lines might not be explainable by the topoisomerase II alpha levels and topoisomerase II catalytic activityReduced expression of DNA topoisomerase II confers resistance to etoposide (VP-16) in small cell lung cancer cell linestopoisomerase II-targeting anticancer agents, etoposide (VP-16) and teniposide (VM-26)the anti-cancer DNA topoisomerase II poison etoposide (VP-16New DNA topoisomerase II inhibitors less prone to redox reactions, such as mitoxantrone and more recently the anthrapyrazoles, were developed to circumvent this toxicityTwo anthrapyrazoles currently in clinical evaluation, DuP 941 (Losoxantrone) and DuP 937, were compared to other topoisomerase II inhibitorsAmong topoisomerase II inhibitors, the cytostatic potency was by decreasing order: mitoxantrone; doxorubicin, which was slightly greater than DuP 941, azatoxin; DuP 937; and amsacrine, which was much greater than VP-16n an attempt to clarify the role of drug-induced protein-associated DNA breaks (i.e., DNA topoisomerase II-mediated DNA cleavage) in the cytotoxic activity of doxorubicin and etoposideThe effect of combinations of the anthracycline aclarubicin and the topoisomerase II targeting drugs 4'-demethylepipodophyllotoxin-9-(4,6-O-ethylidene-beta-D-glucopyra noside) (VP-16) and 4'-(9-acridinylamino)methanesulfon-m-anisidide (m-AMSA) was investigated in a clonogenic assayAntagonistic effect of aclarubicinThe effect of combinations of the anthracyclines aclarubicin and daunorubicin was investigated in a clonogenic assay using the human small cell lung cancer cell line OC-NYH and a multidrug-resistant (MDR) murine subline of Ehrlich ascites tumor (EHR2/DNR+)the DNA topoisomerase II inhibitors doxorubicin and etoposide in both types of cancer cell linesthe DNA topoisomerase II inhibitors doxorubicin and etoposide in human lung cancer cell linescells are cross-resistant to some drugs that interact with topoisomerase II but not mitoxantronesuramin decreases the phosphorylation of Topo II mediated by PKC. This effect of suramin might cause the inhibition of Topo II activity resulting in the growth inhibition of tumor cells.\"}, {\"question\": \"Which are the known inhibitors of the TPL2/MAP3K8 protein?\", \"ideal_answer\": [\"[1,7]naphthyridine-3-carbonitriles and quinoline-3-carbonitriles were the first Tumor Progression Loci-2 (Tpl2) kinase inhibitors. 4-alkylamino-[1,7]naphthyridine-3-carbonitriles are also known to inhibit Tpl2 function as well as quinoline-3-carbonitrile derivatives, thieno[3,2-d]pyrimidines and 2,4-disubstituted thieno[2,3-c]pyridines, indazoles, 4-Alkylamino-[1,7]naphthyridine-3-carbonitriles and generally molecules belonging to the wide categories of quinoline-3-carbonitriles, indazoles and thieno-pyrimidines.\", \"Honokiol\\nThieno[3,2-d]pyrimidines and thieno[2,3-c]pyridine\\nQuinoline-3-carbonitrile derivatives (8-halo-4-(3-chloro-4-fluoro-phenylamino)-6-[(1H-[1,2,3]triazol-4-ylmethyl)-amino]-quinoline-3-carbonitriles; 8-bromo-4-(3-chloro-4-fluorophenylamino)-6-[(1-methyl-1H-imidazol-4-yl)methylamino]quinoline-3-carbonitrile; 4-alkylamino-[1,7]naphthyridine-3-carbonitrile; 1,7-naphthyridine-3-carbonitriles)\\nIndazoles\\nLuteolin\\n1,7-naphtyridine-3-carbonitrile\"], \"exact_answer\": [[\"Honokiol\"], [\"Thieno[3,2-d]pyrimidines and thieno[2,3-c]pyridine\", \"thieno-pyrimidines\"], [\"Quinoline-3-carbonitrile derivatives (8-halo-4-(3-chloro-4-fluoro-phenylamino)-6-[(1H-[1,2,3]triazol-4-ylmethyl)-amino]-quinoline-3-carbonitriles\", \"quinoline-3-carbonitriles\"], [\"8-bromo-4-(3-chloro-4-fluorophenylamino)-6-[(1-methyl-1H-imidazol-4-yl)methylamino]quinoline-3-carbonitrile; 4-alkylamino-[1,7]naphthyridine-3-carbonitrile; 1,7-naphthyridine-3-carbonitriles)\"], [\"Indazoles\", \"indazoles\"]], \"context\": \"Taken together, our results suggest that the therapeutic inhibition of Tpl2 by Honokiol thwarts both gastric tumor growth and peritoneal dissemination by inducing ER stress and inhibiting EMT.The article describes the development of a robust pharmacophore model and the investigation of structure-activity relationship analysis of quinoline-3-carbonitrile derivatives reported for Tpl2 kinase inhibition. In the present study, we found that luteolin inhibited TNF-\\u03b1-induced COX-2 expression by down-regulating the transactivation of nuclear factor-\\u03baB and activator protein-1.we studied the molecular mechanisms of Tpl2-mediated TNFalpha production using a potent Tpl2 kinase inhibitor, 1,7-naphtyridine-3-carbonitrile, and LPS-stimulated RAW264.7 cellsa selective and potent inhibitor of Tpl2, 1,7-naphtyridine-3-carbonitrile, was usedWe have discovered that 8-halo-4-(3-chloro-4-fluoro-phenylamino)-6-[(1H-[1,2,3]triazol-4-ylmethyl)-amino]-quinoline-3-carbonitriles (4) are potent inhibitors of this enzymeurther structure-activity based modifications led to the identification of 8-bromo-4-(3-chloro-4-fluorophenylamino)-6-[(1-methyl-1H-imidazol-4-yl)methylamino]quinoline-3-carbonitrile, which demonstrated in vitro as well as in vivo efficacy in inhibition of LPS-induced TNF-alpha production.nhibitors of tumor progression loci-2 (Tpl2) kinase and tumor necrosis factor alpha (TNF-alpha) production: selectivity and in vivo antiinflammatory activity of novel 8-substituted-4-anilino-6-aminoquinoline-3-carbonitrilesthieno[3,2-d]pyrimidines as Tpl2 kinase inhibitors selective thieno[2,3-c]pyridine inhibitor of COT kinasewe report new SAR efforts which have led to the identification of 4-alkylamino-[1,7]naphthyridine-3-carbonitrilenhibition of Tpl2 kinase and TNF-alpha production with 1,7-naphthyridine-3-carbonitriles\"}, {\"question\": \"Which proteins compose the error prevention GO (8-oxo-G) system in Pseudomonas putida?\", \"ideal_answer\": [\"In P. putida (Pseudomonas putida) the error prevention GO (8-oxo-G) system is composed of MutY, MutM, and MutT enzymes.\"], \"exact_answer\": [[\"MutY\"], [\"MutM\"], [\"MutT\"]], \"context\": \"the GO repair enzymes MutY, MutM, and MutT are involved in the prevention of base substitution mutations in carbon-starved P. putida.Our results demonstrated that the GO repair enzymes MutY, MutM, and MutT are involved in the prevention of base substitution mutations in carbon-starved P. putida.\"}, {\"question\": \"List five applications of machine learning algorithms in medical diagnosis.\", \"ideal_answer\": [\"Machine learning technology is well suited for the induction of diagnostic and prognostic rules and solving of small and specialized diagnostic and prognostic problems. The medical diagnostic knowledge can be automatically derived from the description of cases solved in the past. In several medical domains we actually applied machine learning algorithms. Typically, the automatically generated diagnostic rules achieved the same or slightly better diagnostic accuracy than physicians specialists. There many several  application of machine learning algorithms in medical diagnosis such as Brain glioma progression, Microarray classification, Mass spectral proteomics, Lymph disease classification and Parkinson's disease.\"], \"exact_answer\": [[\"Brain glioma progression\"], [\"Microarray classification\"], [\"Mass spectral proteomics\"], [\"Lymph disease classification\"], [\"Parkinson's disease\"]], \"context\": \"The ability to differentiate between brain tumor progression and radiation therapy induced necrosis is critical for appropriate patient management. IExperimental results on public leukemia, prostate, and colon cancer datasets show that fuzzy support vector machine applied in combination with filter or wrapper feature selection methods develops a robust model with higher accuracy than the conventional microarray classification models such as support vector machine, artificial neural network, decision trees, k nearest neighbors, and diagonal linear discriminant analysisMass spectrometry based proteomics technologies have allowed for a great progress in identifying disease biomarkers for clinical diagnosis and prognosis. However, they face acute challenges from a data reproducibility standpoint, in that no two independent studies have been found to produce the same proteomic patterns. Such reproducibility issues cause the identified biomarker patterns to lose repeatability and prevent real clinical usage. In this work, we propose a profile biomarker approach to overcome this problem from a machine-learning viewpoint by developing a novel derivative component analysis (DCA)In this paper, a random forest classifier (RFC) approach is proposed to diagnose lymph diseasesThe algorithm provides excellent discrimination of PD patients from PSP patients at an individual level, thus encouraging the application of computer-based diagnosis in clinical practice.The objective of this project was the development and validation of a multiparameter machine learning algorithm and system capable of predicting the need for life-saving interventions (LSIs) in trauma patients.\"}, {\"question\": \"List functional roles of the FtsZ protein.\", \"ideal_answer\": [\"Four major roles of FtsZ have been characterized: cell elongation, GTPase, cell division, and bacterial cytoskeleton.\"], \"exact_answer\": [[\"cell elongation\"], [\"GTPase\"], [\"cell division\"], [\"bacterial cytoskeleton\"]], \"context\": \"bacterial homologue FtsZ establishes the cytokinetic ring that constricts during cell division. In Escherichia coli, initial assembly of the Z ring for cell division requires FtsZ plus the essential Z ring-associated proteins FtsA and ZipA.FtsZ is an essential cell division protein in Escherichia coli, and its localization, filamentation, and bundling at the mid-cell are required for Z-ring stability.the bacterial division FtsZ proteinThe GTPase-dependent polymerization/depolymerization dynamics of FtsZ regulate bacterial cell division in vivo. Bacterial cytokinesis depends upon the tubulin-like GTPase FtsZ, which polymerizes into an annular structure at midcell (the Z-ring) that defines the division site.FtsZ is the first of the divisome proteins to accumulate at the division site and is widely thought to function as a force generator that constricts the cell envelope. the cell division protein FtsZFtsZ, the key protein of bacterial cell division was selected as a potent anti bacterial target. the cell-division protein FtsZ FtsZ and PG together function in orchestrating cell division and maintaining cell shape in almost all other bacteria. the tubulin-like protein of the division septum FtsZFtsZ is a widely distributed major cytoskeletal protein involved in the archaea and bacteria cell division. Four major roles of FtsZ have been characterized: cell elongation, GTPase, cell division, and bacterial cytoskeleton. Bacterial cell division involves a complex and dynamic sequence of events whereby polymers of the protein FtsZ assemble at the division plane and rearrange to achieve the goal of contracting the cell membrane at the site of cell division, thus dividing the parent cell into two daughter cells. the cell division protein FtsZFtsZ, the homolog of eukaryotic tubulin, is a GTPase that assembles into cytokinetic Z rings essential for cell division in prokaryotic cells\"}, {\"question\": \"Intetumumab has been tested in clinical trials for treatment of which cancers?\", \"ideal_answer\": [\"Intetumumab has been tested in clinical trials for treatment of prostate cancer, melanoma and angiosarcoma.\"], \"exact_answer\": [[\"prostate cancer\"], [\"melanoma\"], [\"angiosarcoma\"]], \"context\": \"A randomized, double-blind, multicenter, phase 2 study of a human monoclonal antibody to human \\u03b1\\u03bd integrins (intetumumab) in combination with docetaxel and prednisone for the first-line treatment of patients with metastatic castration-resistant prostate cancer.BACKGROUND: Intetumumab is a fully human mAb with antiangiogenic, antitumor properties which has shown potential therapeutic effect in castration-resistant prostate cancer (CRPC) patients. PATIENTS AND METHODS: In a phase 2, randomized, double-blind, multicenter study, men with metastatic CRPC without prior systemic nonhormonal therapy were randomly assigned to 75-mg/m(2) docetaxel (Taxotere) and 5-mg prednisone plus placebo (N = 65) or 10-mg/kg intetumumab (N = 66) q3w. CONCLUSION: The addition of intetumumab to docetaxel resulted in shorter PFS without additional toxicity among CRPC patients.Data from the Melanoma Subscale (MS) of the Functional Assessment of Cancer Therapy-Melanoma and the worst pain question from the Brief Pain Inventory (BPI) were taken from a clinical trial evaluating intetumumab alone or with dacarbazine in Stage IV metastatic melanoma.A randomised, phase II study of intetumumab, an anti-\\u03b1v-integrin mAb, alone and with dacarbazine in stage IV melanoma.CONCLUSION: With its favourable safety profile and a nonsignificant trend towards improved OS, intetumumab merits further investigation in advanced melanoma.METHODS: In a multicentre, randomised, phase II study, stage IV melanoma patients were randomised 1:1:1:1 to 1000 mg m(-2) dacarbazine+placebo (n=32), 1000 mg m(-2) dacarbazine+10 mg kg(-1) intetumumab (n=32), 10 mg kg(-1) intetumumab (n=33), or 5 mg kg(-1) intetumumab (n=32) q3w. Clinical and pharmacologic evaluation of two dose levels of intetumumab (CNTO 95) in patients with melanoma or angiosarcoma.PURPOSE: In this Phase 1, multicenter, open-label study, intetumumab (CNTO 95), a fully human anti-\\u03b1v integrin monoclonal antibody was evaluated for safety, pharmacokinetics, and pharmacodynamic activity in patients with melanoma or angiosarcoma. PATIENTS AND METHODS: Patients with histologically-confirmed inoperable melanoma or angiosarcoma refractory to standard treatment were allocated to treatment with 10 mg/kg or 20 mg/kg intetumumab, administered once every 3 weeks for up to four cycles unless unacceptable toxicity or disease progression occurred.CONCLUSIONS: In patients with metastatic malignant melanoma and angiosarcoma in this study, intetumumab demonstrated manageable toxicity, was well tolerated, and presented approximately dose-proportional pharmacokinetics for the 10 mg/kg and 20 mg/kg doses.A phase 1, multicenter, open-label study of the safety of two dose levels of a human monoclonal antibody to human \\u03b1(v) integrins, intetumumab, in combination with docetaxel and prednisone in patients with castrate-resistant metastatic prostate cancer.PURPOSE: We evaluated the safety and efficacy of intetumumab in combination with docetaxel in patients with castrate-resistant metastatic prostate cancer. Patients and methods In this phase 1, open-label, multicenter, nonrandomized study, 75 mg/m\\u00b2 docetaxel was administered on Day 1 of each of nine 21-day treatment cycles and intetumumab 5 or 10 mg/kg was administered on Days 1, 8, and 15 of Cycles 2 and 3 and on Day 1 of all subsequent cycles. In a multicentre, randomised, phase II study, stage IV melanoma patients were randomised 1:1:1:1 to 1000 mg m(-2) dacarbazine+placebo (n=32), 1000 mg m(-2) dacarbazine+10 mg kg(-1) intetumumab (n=32), 10 mg kg(-1) intetumumab (n=33), or 5 mg kg(-1) intetumumab (n=32) q3w.Patients with histologically-confirmed inoperable melanoma or angiosarcoma refractory to standard treatment were allocated to treatment with 10 mg/kg or 20 mg/kg intetumumab, administered once every 3 weeks for up to four cycles unless unacceptable toxicity or disease progression occurred.Data from the Melanoma Subscale (MS) of the Functional Assessment of Cancer Therapy-Melanoma and the worst pain question from the Brief Pain Inventory (BPI) were taken from a clinical trial evaluating intetumumab alone or with dacarbazine in Stage IV metastatic melanoma.Data from the Melanoma Subscale (MS) of the Functional Assessment of Cancer Therapy-Melanoma and the worst pain question from the Brief Pain Inventory (BPI) were taken from a clinical trial evaluating intetumumab alone or with dacarbazine in Stage IV metastatic melanomaData from the Melanoma Subscale (MS) of the Functional Assessment of Cancer Therapy-Melanoma and the worst pain question from the Brief Pain Inventory (BPI) were taken from a clinical trial evaluating intetumumab alone or with dacarbazine in Stage IV metastatic melanoma\"}, {\"question\": \"List available tools for genomic visualisation in comparative genomics\", \"ideal_answer\": [\"Insyght, Genomicus and Sockeye.\"], \"exact_answer\": [[\"Insyght\"], [\"Genomicus\"], [\"Sockeye\"]], \"context\": \"Insyght: navigating amongst abundant homologues, syntenies and gene functional annotations in bacteria, it's that symbol!Insyght is a comparative genomic visualization tool that combines three complementary displays: (i) a table for thoroughly browsing amongst homologues, (ii) a comparator of orthologue functional annotations and (iii) a genomic organization view designed to improve the legibility of rearrangements and distinctive lociGenomicus: five genome browsers for comparative genomics in eukaryotaGenomicus (http://www.dyogen.ens.fr/genomicus/) is a database and an online tool that allows easy comparative genomic visualization in>150 eukaryote genomesSockeye: a 3D environment for comparative genomics.We have developed a Java-based application called Sockeye that uses three-dimensional (3D) graphics technology to facilitate the visualization of annotation and conservation across multiple sequences.\"}, {\"question\": \"Which species may be used for the biotechnological production of itaconic acid?\", \"ideal_answer\": [\"In 1955, the production of itaconic acid was firstly described for Ustilago maydis. Some Aspergillus species, like A. itaconicus and A. terreus, show the ability to synthesize this organic acid and A. terreus can secrete significant amounts to the media. Itaconic acid is mainly supplied by biotechnological processes with the fungus Aspergillus terreus. Cloning of the cadA gene into the citric acid producing fungus A. niger showed that it is possible to produce itaconic acid also in a different host organism.\"], \"exact_answer\": [[\"Aspergillus terreus\"], [\"Aspergillus niger\"], [\"Ustilago maydis\"]], \"context\": \"continuous itaconic acid production from sucrose with Aspergillus terreusA potent itaconic acid producing strain, Aspergillus terreus SKR10, was isolated from horticulture wasteTwo high itaconic acid yielding mutants, N45 and UNCS1 were obtained by gradient platingItaconic acid production using sago starch hydrolysate by Aspergillus terreus TN484-M1.production of itaconic acid and lovastatin by Aspergillus terreus ATCC 20542Fermentation products of Aspergillus terreus ATCC 20542 (a parent strain for lovastatin production) were collected, and the coexistence of itaconic acid (IA) with lovastatin was confirmed in this studyEnhancing itaconic acid production by Aspergillus terreusAspergillus terreus is successfully used for industrial production of itaconic acidFrom a number of different Aspergillus terreus controlled batch fermentations, those cultures with the largest difference in itaconic acid titer and productivity were selected for mRNA isolationa systematic process optimization was performed with an own isolated strain of Aspergillus terreusCells of Aspergillus terreus, free and immobilized in polyurethane foam, were employed in itaconic acid fermentation processes on glycerol-based media.current biotechnological production processes with the fungus Aspergillus terreus has to be improvedReduced by-product formation and modified oxygen availability improve itaconic acid production in Aspergillus nigerPreviously, it was shown that expression of the cis-aconitate decarboxylase gene (cadA) from Aspergillus terreus converted A. niger into an itaconic acid producer\"}, {\"question\": \"Which types of cancer can be recognized and treated by the use of immunotherapy?\", \"ideal_answer\": [\"When normal cells turn into cancer cells, some of the antigens on their surface change. These cells, like many body cells, constantly shed bits of protein from their surface into the circulatory system. Often, tumor antigens are among the shed proteins.\\nThese shed antigens prompt action from immune defenders, including cytotoxic T cells, natural killer cells, and macrophages. According to one theory, patrolling cells of the immune system provide continuous bodywide surveillance, catching and eliminating cells that undergo malignant transformation. Tumors develop when this immune surveillance breaks down or is overwhelmed.\\nA new approach to cancer therapy uses antibodies that have been specially made to recognize specific cancers such as Melanoma, Leukaemia, Lung Cancer, Colorectal Cancer, Breast Cancer, Head Cancer and Pancreatic Cancer.\"], \"exact_answer\": [[\"Melanoma\"], [\"Leukaemia\"], [\"Lung Cancer\"], [\"Colorectal Cancer\"], [\"Breast Cancer\"], [\"Head Cancer\"], [\"Pancreatic Cancer\"], [\"Prostate tumor\"]], \"context\": \"omising expansion of antigen-specific T cells, the clinical responses following vaccination have been limited, indicating that further improvements of DC vaccine potency are necessary. Pre-clinical studies suggest that vaccination with combination of primary DC subsets, such as myeloid and plasmacytoid blood DCs (mDCs and pDCs, respectively), may result in stronger clinical responses. However, it is a challenge to obtain high enough numbers of primary DCs for immunotherapy, since their frequency in blood is very low. We therefore explored the possibility to generate them from hematopoietic progenitor cells (HPCs). Here, we show that by inhibiting the aryl hydrocarbon receptor with its antagonist StemRegenin 1 (SR1), clinical-scale numbers of functional BDCA2(+)BDCA4(+) pDCs, BDCA1(+) mDCs, and BDCA3(+)DNGR1(+) mDCs can be efficiently generated from human CD34(+) HPCs. The ex vivo-generated DCs were phenotypically and functionally comparable to peripheral blood DCs. They secreted high levels of pro-inflammatory cytokines such as interferon (IFN)-\\u03b1, interleukin (IL)-12, and tumor necrosis factor (TNF)-\\u03b1 and upregulated co-stimulatory molecules and maturation markers following stimulation with Toll-like receptor (TLR) ligands. Further, theodent models of prostate cancer have served as valuable preclinical models to evaluate novel treatments and understand malignant disease progression. In particular, a transgenic rat autochthonous model of prostate cancer using the SV40 large T antigen expressed under a prostate-specific probasin promoter was previously developed as a model of androgen-dependent prostate cancer (TRAP). In the current report, we backcrossed this strain to the Lewis strain, an inbred rat strain better characterized for immunological analyses. We demonstrate that Lewis transgenic rats (Lew-TRAP) developed prostate adenocarcinomas with 100% penetrance by 25 weeks of age. Tumors were predominantly androgen-dependent, as castration prevented tumor growth in the majority of animals. Finally, we demonstrate that Lew-TRAP rats could be immunized with a DNA vaccine encoding a human prostate tumor antigen (prostatic acid phosphatase) with the development of Lewis strain-specific T-cell responses. We propose we analyzed NY-ESO-1 expression in 222 melanoma specimens, including 16 primary and 206 metastatic tumors. Our results support previous findings showing higher expression of NY-ESO-1 in metastatic (58/206; 28.2%) versus primary (0/16) tumors. In addition, our results show that the epithelioid subtype of melanoma has the highest incidence of NY-ESO-1 expression. These findin Immunotherapy is a highly promising form of cancer treatment, but its efficacy can be severely compromised by an immunosuppressive tumor microenvironment.\"}, {\"question\": \"Which proteins constitute the methyl-directed mismatch repair system (MMR) in bacteria?\", \"ideal_answer\": [\"The mismatch repair system (MMR) recognizes and corrects mismatched or unpaired bases caused mainly by DNA polymerase, and contributes to the fidelity of DNA replication in living cells. In bacteria, the methyldirected mismatch repair (MMR) is comprised of MutS and MutL proteins, encoded by the mutS/L operon.\"], \"exact_answer\": [[\"mutS\"], [\"mutL\"], [\"mutH\"]], \"context\": \"MutY and MutSL (mismatch repair system [MMR])Escherichia coli DNA Mismatch Repair (MMR) protein MutS We have characterized the mismatch repair system (MMR) of the highly radiation-resistant type strain of Deinococcus radiodurans, ATCC 13939. We show that the MMR system is functional in this organism, where it participates in ensuring the fidelity of DNA replication and recombination. The system relies on the activity of two key proteins, MutS1 and MutL, which constitute a conserved core involved in mismatch recognition.the mutS/L operon essential for methyldirected mismatch repair (MMR) activityThe mismatch repair system (mutS, mutL and uvrD genes) in Pseudomonas aeruginosaThe mismatch repair system (MMR) recognizes and corrects mismatched or unpaired bases caused mainly by DNA polymerase, and contributes to the fidelity of DNA replication in living cells. In Escherichia coli, the MutHLS system is known to function in MMR, and homologues of MutS and MutL are widely conserved in almost all organisms.A simple genetic system has been developed to test the effect of over-expression of wild-type or mutated human MutL homologue 1 (hMLH1) proteins on methyl-directed mismatch repair (MMR) in Escherichia coli.Among these six mutated genes was mutS, which encodes a component of the methyl-directed mismatch repair (MMR) system.The majority of naturally occurring, strong mutators contain defects in the methyl-directed mismatch repair (MMR) system, with mutations in mutS predominating.The picture of MutH-independent MMR in other bacteria is less clear, as MMR components other than MutS and MutL have not been identified in the majority of bacteria.The methyl-directed mismatch repair (MMR) mechanism has been extensively studied in vitro and in vivo, but one of the difficulties in determining the biological relationships between the MMR-related proteins is the tendency of MutL to self-aggregate.Among these six mutated genes was mutS, which encodes a component of the methyl-directed mismatch repair (MMR) system.MutL is required to assist the mismatch repair protein MutS during initiation of the methyl-directed mismatch repair (MMR) response in various organisms ranging from prokaryotes to eukaryotes.Moreover, methyl-directed mismatch repair (MMR) is known to trigger sensitivity to methylating agents via a mechanism that involves recognition by MutS of the O(6)-mG:T replication intermediates.The system relies on the activity of two key proteins, MutS1 and MutL, which constitute a conserved core involved in mismatch recognition.In Escherichia coli, the best studied mismatch repair (MMR) pathway is the methyl-directed long patch repair system which is mediated by three gene products; MutS, MutL and MutH.Cells devoid of MutS1 or MutL proteins were as resistant to gamma-rays, mitomycin C and UV-irradiation as wild-type bacteria, suggesting that the mismatch repair system is not essential for the reconstitution of a functional genome after DNA damage.The system relies on the activity of two key proteins, MutS1 and MutL, which constitute a conserved core involved in mismatch recognition\"}, {\"question\": \"Which are the main components of mTORC1?\", \"ideal_answer\": [\"The mTOR (mammalian target of rapamycin) protein kinase is an important regulator of cell growth and is a key target for therapeutic intervention in cancer. Two complexes of mTOR have been identified: complex 1 (mTORC1), consisting of mTOR, Raptor (regulatory associated protein of mTOR) and mLST8 (mammalian lethal with SEC13 protein 8) and complex 2 (mTORC2) consisting of mTOR, Rictor (rapamycin-insensitive companion of mTOR), Sin1 (stress-activated protein kinase-interacting protein 1), and mLST8.\"], \"exact_answer\": [[\"mTOR\"], [\"Raptor\"], [\"mLST8/GbetaL\"]], \"context\": \"Raptor-mTOR and Rictor-mTOR complexes compete for association with LST8, and this mechanism may contribute to the reciprocal negative regulations of mTORC1 and mTORC2 activities, in terms of their LST8 components.The mTOR kinase controls cell growth, proliferation, and survival through two distinct multiprotein complexes, mTORC1 and mTORC2. mTOR and mLST8 are in both complexes, while raptor and rictor are part of only mTORC1 and mTORC2, respectively.The mTOR (mammalian target of rapamycin) protein kinase is an important regulator of cell growth. Two complexes of mTOR have been identified: complex 1, consisting of mTOR-Raptor (regulatory associated protein of mTOR)-mLST8 (termed mTORC1), and complex 2, comprising mTOR-Rictor (rapamycininsensitive companion of mTOR)-mLST8-Sin1 (termed mTORC2).composition of mTORC1 (raptor, mTOR, and GbetaL)The mammalian target of rapamycin (mTOR) is part of two distinct complexes, mTORC1, containing raptor and mLST8, and mTORC2, containing rictor, mLST8 and sin1.The mTORC1 protein kinase complex consists of mTOR, raptor, mLST8/GbetaL and PRAS40.The mTOR (mammalian target of rapamycin) protein kinase is an important regulator of cell growth and is a key target for therapeutic intervention in cancer. Two complexes of mTOR have been identified: complex 1 (mTORC1), consisting of mTOR, Raptor (regulatory associated protein of mTOR) and mLST8 (mammalian lethal with SEC13 protein 8) and complex 2 (mTORC2) consisting of mTOR, Rictor (rapamycin-insensitive companion of mTOR), Sin1 (stress-activated protein kinase-interacting protein 1), mLST8 and Protor-1 or Protor-2.\"}, {\"question\": \"Which receptor(s) recognize lysosomal hydrolases in trans-Golgi network (TGN)?\", \"ideal_answer\": [\"The majority of lysosomal hydrolases in trans-Golgi network (TGN) are specifically recognized by mannose 6-phosphate (M6P) receptors (MPRs), which ensure their transport to the endosomal/lysosomal system. Other receptors can also carry lysosomal hydrolases from the trans-Golgi network (TGN) to the endosomal system. These M6P alternative receptors are the lysosomal integral membrane protein (LIMP-2) and the multi-ligand receptor sortilin. These reseptors enable a mannose-6-phosphate-independent pathway from TGN to lysosomes.\"], \"exact_answer\": [[\"mannose 6-phosphate receptors (MPRs)\"], [\"sortilin\"], [\"lysosomal integral membrane protein (LIMP-2)\"], [\"cation-independent mannose 6-phosphate receptor (CI-MPR)\"]], \"context\": \"The majority of the enzymes leave the TGN after modification with mannose-6-phosphate (M6P) residues, which are specifically recognized by M6P receptors (MPRs), ensuring their transport to the endosomal/lysosomal system.M6P receptors play a major role in the intracellular transport of newly synthesized lysosomal enzymes in mammalian cellsMannose 6-phosphate receptors (MPRs) are known to be shuttled between the trans-Golgi network (TGN) and endosomes, thereby several lysosomal hydrolases are delivered through the endocytic pathway into lysosomes.Most soluble lysosomal hydrolases are sorted in the trans-Golgi network (TGN) and delivered to the lysosomes by the mannose 6-phosphate receptor (M6PR).certain soluble lysosomal hydrolases, is sorted and trafficked to the lysosomes by sortilin.Mannose-6-phosphate receptors (MPRs) transport lysosomal hydrolases from the trans Golgi network (TGN) to endosomes. Recently, the multi-ligand receptor sortilin has also been implicated in this transportA shortcut to the lysosome: the mannose-6-phosphate-independent pathwayseveral lines of evidence suggest the existence of alternative processes of lysosomal targeting. Among them, the two that are mediated by the M6P alternative receptors, lysosomal integral membrane protein (LIMP-2) and sortilin, have gained unequivocal support.delivery of mannose 6-phosphate receptors carrying lysosomal hydrolases from the trans-Golgi network (TGN) to the endosomal systemThe cation-independent mannose 6-phosphate receptor (CI-MPR) mediates sorting of lysosomal hydrolase precursors from the TGN to endosomes.mannose 6-phosphate (Man-6-P) recognition marker on lysosomal hydrolases.The delivery of mannose 6-phosphate receptors carrying lysosomal hydrolases from the trans-Golgi network (TGN) to the endosomal system is mediated by selective incorporation of the receptor-hydrolase complexes into vesicular transport carriers (TCs) that are coated with clathrin and the adaptor proteins, GGA and AP-1.Mannose-6-phosphate receptors (MPRs) transport lysosomal hydrolases from the trans Golgi network (TGN) to endosomes.Mannose 6-phosphate receptors carry newly synthesized lysosomal hydrolases from the trans-Golgi network to endosomes, then return to the trans-Golgi network for another round of enzyme delivery.In this way, the M6P receptors help package the hydrolases into vesicles that bud from the trans-Golgi network to deliver their contents to endosomes that ultimately will develop into mature lysosomes, where recently-delivered hydrolases may start digesting the endocyted material.However, LS fibroblasts displayed reduced mannose 6-phosphate receptor (MPR)-mediated re-uptake of the lysosomal enzyme arylsulfatase B. In addition, endosome-to-trans Golgi network (TGN) transport of MPRs was decreased significantly, leading to higher levels of cell surface MPRs and their enrichment in enlarged, retromer-positive endosomes in OCRL-depleted HeLa cells.The M6P groups are then recognized by two independent transmembrane M6P receptors, present in the trans-Golgi network: the cation-independent M6P receptor and/or the cation-dependent M6P receptor.Two related mannose 6-phosphate receptors (MPRs) recognize this feature in the trans Golgi network (TGN) and deliver the hydrolases to the late endosome.The cation-independent mannose 6-phosphate receptor (CIMPR) cycles between the trans-Golgi network (TGN) and endosomes to mediate sorting of lysosomal hydrolases.The M6P groups are then recognized by two independent transmembrane M6P receptors, present in the trans-Golgi network: the cation-independent M6P receptor and/or the cation-dependent M6P receptorMannose-6-phosphate receptors (MPRs) transport lysosomal hydrolases from the trans Golgi network (TGN) to endosomesThe delivery of mannose 6-phosphate receptors carrying lysosomal hydrolases from the trans-Golgi network (TGN) to the endosomal system is mediated by selective incorporation of the receptor-hydrolase complexes into vesicular transport carriers (TCs) that are coated with clathrin and the adaptor proteins, GGA and AP-1Mannose 6-phosphate receptors (MPRs) are known to be shuttled between the trans-Golgi network (TGN) and endosomes, thereby several lysosomal hydrolases are delivered through the endocytic pathway into lysosomesThe cation-independent mannose 6-phosphate receptor (CIMPR) cycles between the trans-Golgi network (TGN) and endosomes to mediate sorting of lysosomal hydrolasesTwo related mannose 6-phosphate receptors (MPRs) recognize this feature in the trans Golgi network (TGN) and deliver the hydrolases to the late endosome\"}, {\"question\": \"List the diseases that can be treated using Vedolizumab.\", \"ideal_answer\": [\"Ulcerative colitis and Crohn's disease are inflammatory bowel diseases that have been successfully treated with Vedolizumab, a gut-selective, anti-inflammatory monoclonal antibody.\"], \"exact_answer\": [[\"Ulcerative colitis\"], [\"Crohn's disease\"]], \"context\": \"A new gut-specific monoclonal antibody, vedolizumab, has been shown to be effective in inflammatory bowel disease, and in continued trials no patients have developed PML. Finally, several studies were presented on new drugs with new therapeutic targets, such as vedolizumab, in the treatment of inflammatory bowel disease. We review the role of vedolizumab, a humanized antibody against the \\u03b14\\u03b27 - integrin, in both ulcerative colitis (UC) and Crohn's disease (CD). Results from clinical trials show that vedolizumab is effective in the induction and maintenance of remission in active CD and UC and has a very good safety profile.Vedolizumab as induction and maintenance therapy for Crohn's disease.NCLUSIONS: Vedolizumab-treated patients with active Crohn's disease were more likely than patients receiving placebo to have a remission, but not a CDAI-100 response, at week 6; patients with a response to induction therapy who continued to receive vedolizumab (rather than switching to placebo) were more likely to be in remission at week 52. Vedolizumab as induction and maintenance therapy for ulcerative colitis.CONCLUSIONS: Vedolizumab was more effective than placebo as induction and maintenance therapy for ulcerative colitis.New and emerging treatments for ulcerative colitis: a focus on vedolizumab. This article reviews the emerging data on the use of vedolizumab for UC.  Newer treatments such as vedolizumab and tofacitinib may represent valuable future therapies.SUMMARY: New therapeutic targets in IBD patients who failed anti-TNF-\\u03b1 therapy are urgently required, and tofacitinib, vedolizumab and ustekinumab appear to be the most promising drugs. BACKGROUND: Vedolizumab, a gut-selective, anti-inflammatory monoclonal antibody, has shown preliminary efficacy in ulcerative colitis (UC) and Crohn's disease (CD). CONCLUSION: Vedolizumab every 8 weeks for up to 78 weeks had an adverse event profile similar to that previously observed. Mean disease activity indices (partial Mayo score and Crohn's Disease Activity Index score) improved with all 3 doses investigated.The new drugs discussed here are those that disturb lymphocyte homing to the gut (natalizumab, vedolizumab and anti-mucosal addressin cellular adhesion molecule); one that blocks interleukin (IL)-12 as well as the IL-23/T helper 17 (Th17) axis (ustekinumab) and one that blocks the signaling of multiple cytokines (tofacitinib).Vedolizumab  for Crohn's disease.AREAS COVERED: This review discusses the potential role of vedolizumab, a humanised monoclonal antibody that selectively blocks lymphocyte trafficking to the gut, for the treatment of CD. All randomised placebo-controlled trials that evaluated vedolizumab for the treatment of CD were reviewed and safety and efficacy data evaluated. EXPERT OPINION: Vedolizumab is an effective and well-tolerated drug that is an important advance for the treatment of CD.Treatment strategies that appear particularly appealing include selective anti-integrin therapy with vedolizumab (anti-\\u03b14\\u03b27), etrolizumab (anti-\\u03b27 antibody) and PF-00547,659 (anti-MAdCAM-1 antibody), anti-interleukin 12/23p40 therapy with ustekinumab and Janus kinase 1, 2 and 3 inhibition with toafacitinib.We discuss a number of them with specific focus on vedolizumab, a monoclonal antibody directed against the alpha4beta7 integrin on lymphocytes, ustekinumab, a monoclonal antibody against the p40 subunit of interleukin-12 and interleukin-23, and tofacitinib, a small molecule targeting Janus-activated kinase. Most likely, these three agents will find their way to the market and offer significant therapeutic alternatives for the management of Crohn's disease and/or ulcerative colitis.Natalizumab, vedolizumab, alicaforsen AJM300, rhuMAb \\u03b27, CCX282-B, and PF-00547,659 are few of monoclonal antibodies that have shown high promise in trials with the potential for more attractive benefit:risk ratio than currently available therapies.Vedolizumab for the treatment of ulcerative colitis and Crohn's disease.Here, we discuss the current treatment options available for patients with Crohn's disease or ulcerative colitis, the history of CAM inhibitors, the current state of development of vedolizumab and its future role in inflammatory bowel disease, if approved by regulatory agencies.Biological agents will probably be used earlier and more widely; new information on levels of biological agents, mucosal healing and new comparative studies will also allow these agents to be used in a more precise and personalized way. In addition to infliximab, adalimumab, natalizumab and certolizumab, other biological agents will be employed; among the first of these to be used will be ustekinumab, golimumab and vedolizumab.Several antibodies exist which constitute selection adhesion molecule inhibitors, including Natalizumab, MLN-0002 (Vedolizumab) and ISIS 2302 (Alicaforsen) will be discussed in this review.Vedolizumab for the treatment of active ulcerative colitis: a randomized controlled phase 2 dose-ranging study.BACKGROUND: Vedolizumab is a gut-selective biologic that has shown efficacy in ulcerative colitis (UC) and Crohn's disease (CD).They are anti CD4+ T cell cytokine including interleukin (IL)-12/23 and IL-17 blockers, selective anti-adhesion molecule known as natalizumab, vedolizumab and alicaforsen, T-cell proliferation inhibitor, anti-inflammatory cytokine, immune stimulator, growth factor, and mitogen-activated protein kinase inhibitor. Vedolizumab, a humanized mAb against the \\u03b14\\u03b27 integrin for the potential treatment of ulcerative colitis and Crohn's disease.Data from phase II clinical trials of vedolizumab demonstrated efficacy with an attractive safety profile, especially in ulcerative colitis. Large phase III, multicenter trials in both ulcerative colitis and Crohn's disease will provide valuable data for the ongoing development of vedolizumab, which might evolve as a new anti-inflammatory treatment option for the management of therapy-refractory patients.The development of a new specific molecule, vedolizumab, is currently under investigation in a large clinical trial. This novel specific anti-integrin drug seems to hold promise in the treatment of gut inflammation.The binding specificity and selective antagonism of vedolizumab, an anti-alpha4beta7 integrin therapeutic antibody in development for inflammatory bowel diseases.These pharmacologic properties of vedolizumab, in conjunction with the gastrointestinal tropism of alpha(4)beta(7) integrin function, may ultimately confer an improved risk-to-benefit profile for patients with inflammatory bowel diseases.In the Crohn's disease study that included maintenance treatment, vedolizumab was significantly more effective at 52\\u00a0weeks than placebo in both endpoints (clinical remission was the only primary endpoint in the maintenance study)Vedolizumab is a useful addition to the treatment options available for patients with moderately to severely active ulcerative colitis and Crohn's diseaseVedolizumab-treated patients with active Crohn's disease were more likely than patients receiving placebo to have a remission, but not a CDAI-100 response, at week 6; patients with a response to induction therapy who continued to receive vedolizumab (rather than switching to placebo) were more likely to be in remission at week 52\"}, {\"question\": \"Which genes code for the alpha subunit of the DNA polymerase III in most Firmicutes?\", \"ideal_answer\": [\"Bacterial DNA polymerase III is the primary complex of DNA replication. In most Firmicutes, which are low-GC, gram-positive bacteria, the alpha subunit of their DNA polymerase III is encoded by polC and dnaE. DnaE is widely conserved in most bacteria, while PolC is present mainly in Firmicutes clade.\"], \"exact_answer\": [[\"dnaE\"], [\"polC\"]], \"context\": \"PolC co-evolves with RNA degradation enzymes that are present only in the A+T-rich Firmicutes cladeBacterial genomes displaying a strong bias between the leading and the lagging strand of DNA replication encode two DNA polymerases III, DnaE and PolC, rather than a single one.SGD, PAS, and polC are all features associated with a group of low-GC, gram-positive bacteria (Firmicutes)PAS is a characteristic of organisms with a heterodimeric DNA polymerase III alpha-subunit constituted by polC and dnaEBacterial DNA polymerase III, the primary complex of DNA replication, consists of PolC and DnaE. PolC is conserved in Gram-positive bacteria, especially in the Firmicutes with low GC content, whereas DnaE is widely conserved in most Gram-negative and Gram-positive bacteria.PolC-PolE DNA polymerasesWe conclude that PAS is a characteristic of organisms with a heterodimeric DNA polymerase III alpha-subunit constituted by polC and dnaE, which may play a direct role in the maintenance of SGD.Furthermore, SGD, PAS, and polC are all features associated with a group of low-GC, gram-positive bacteria (Firmicutes)In contrast, PolC co-evolves with RNA degradation enzymes that are present only in the A+T-rich Firmicutes clade, suggesting at least two origins for the degradosome.A method for the construction of in frame substitutions in operons: deletion of the essential Escherichia coli holB gene coding for a subunit of the DNA polymerase III holoenzyme.\"}, {\"question\": \"List the neurotransmitters that are metabolized by MAOA.\", \"ideal_answer\": [\"The monoamine oxidase-A (MAOA) gene plays a vital role in the metabolism of neurotransmitters, e.g, serotonin, norepinephrine, and dopamine.\"], \"exact_answer\": [[\"serotonin\"], [\"norepinephrine\"], [\"dopamine\"]], \"context\": \" MAOA metabolizes monoamine neurotransmitters. Monoamine oxidase A (MAOA) modulates metabolism of serotonin and dopamine metabolism, neurotransmitters involved in regulation of appetite and food intake. Tumors from patients with high-grade aggressive prostate cancer (PCa) exhibit increased expression of monoamine oxidase A (MAOA), a mitochondrial enzyme that degrades monoamine neurotransmitters and dietary amines.Monoamine oxidase A (MAOA) catalyzes monoamine neurotransmitters including dopamine, 5-hydroxytryptamine (5-HT, serotonin), and norepinephrine. These traits are controlled by neurotransmitters like dopamine, serotonin and norepinephrine. Monoamine oxidase A (MAOA), a mitochondrial enzyme involved in the degradation of amines, has been reported to be associated with aggression, impulsivity, depression, and mood changes.Monoamine Oxidase A (MAOA) is a critical enzyme in the catabolism of monoaminergic neurotransmitters. Monoamine oxidases (MAO-A and MAO-B) have a key role in the degradation of amine neurotransmitters, such as dopamine, norepinephrine and serotonin. Inhibitors of monoamine oxidase A (MAOA), a mitochondrial enzyme that degrades neurotransmitters including serotonin and norepinephrine, are commonly used to treat neurological conditions including depression. Monoamine oxidase A (MAOA) is one of the major enzymes responsible for the degradation of neurotransmitters.Monoamine oxidase membrane enzymes are responsible for the catalytic breakdown of extra- and intracellular neurotransmitters Monoamine oxidase-A (MAO-A) is a key mitochondrial enzyme that metabolizes biogenic amine neurotransmitters such as dopamine and serotoninThe monoamine oxidase-A (MAOA) gene plays a vital role in the metabolism of neurotransmitters, e.g, serotonin, norepinephrine, and dopamine. \"}, {\"question\": \"Which are the subtypes of Pfeiffer syndrome?\", \"ideal_answer\": [\"Pfeiffer syndrome is divided into three clinical subtypes.\", \"Pfeiffer syndrome is divided into three clinical subtypes. Type 1 \\\"classic\\\" Pfeiffer syndrome involves individuals with mild manifestations including brachycephaly, midface hypoplasia and finger and toe abnormalities; it is associated with normal intelligence and generally good outcome. Type 2 consists of cloverleaf skull, extreme proptosis, finger and toe abnormalities, elbow ankylosis or synostosis, developmental delay and neurological complications. Type 3 is similar to type 2 but without a cloverleaf skull.\"], \"exact_answer\": [[\"Type I\", \"Type I (1)\"], [\"Type II\", \"Type II (2)\"], [\"Type III\", \"Type III (3)\"]], \"context\": \"Of 802 patients treated for craniosynostosis, 28 were identified with Pfeiffer syndrome: 17 were classified as type I (61 percent), seven were classified as type II (25 percent), and four were classified as type III (14 percent). Pfeiffer syndrome is divided into three clinical subtypes. Type 1 \\\"classic\\\" Pfeiffer syndrome involves individuals with mild manifestations including brachycephaly, midface hypoplasia and finger and toe abnormalities; it is associated with normal intelligence and generally good outcome. Type 2 consists of cloverleaf skull, extreme proptosis, finger and toe abnormalities, elbow ankylosis or synostosis, developmental delay and neurological complications. Type 3 is similar to type 2 but without a cloverleaf skull. Recently, based on clinical findings, the disorder has been divided into three subtypes: type 1, characterized by mild expression; type 2, in which clover leaf skull deformity and multiple congenital anomalies are present at birth; and type 3, which is similar to type 2, but lacks the presence of the clover leaf skull at birth. It is a clinically variable disorder and has been divided into three subtypes [Cohen, 1993: Am J Med Genet 45:300-307]. Type 1 represents the less severe cases, while types 2 and 3 are the more severe cases. These latter types tend to have a higher risk for neurodevelopmental problems and a reduced life expectancy.Classic Pfeiffer syndrome is designated type I. Type 2 consists of cloverleaf skull with Pfeiffer hands and feet together with ankylosis of the elbows. Such patients do poorly with an early death. All reported instances to date have been sporadic. Type 3 is similar to type 2 but without cloverleaf skull. Ocular proptosis is severe in degree and the anterior cranial base is markedly short. These patients also do poorly and tend to have an early death. To date all cases have occurred sporadically. Although these 3 clinical subtypes do not have status as separate entities, their diagnostic and prognostic implications are important. Type 1 is commonly associated with normal intelligence, generally good outcome, and can be found dominantly inherited in some families. Types 2 and 3 generally have severe neurological compromise, poor prognosis, early death, and sporadic occurrence. \"}, {\"question\": \"Which genes does thyroid hormone receptor beta1 regulate in the heart?\", \"ideal_answer\": [\"\\u03b2-MHC, HCN4, KCND2/3, SERCA, TRbeta1, alpha-MHC\"], \"exact_answer\": [[\"\\u03b2-MHC\"], [\"HCN4\"], [\"KCND2/3\"], [\"SERCA\"], [\"TRbeta1\"], [\"alpha-MHC\"]], \"context\": \"we identified thyroid hormone receptor \\u03b21 (TR\\u03b21), which negatively regulates \\u03b2-MHC transcription, as a target of miR-27aHCN2 gene expression was not significantly affected by TR beta 1. TR beta 1 exclusively suppressed HCN4 transcription.TRalpha1 increased I(to), while TRbeta1 significantly reduced I(to) in size, which was associated with TRalpha1-mediated increase and TRbeta1-mediated reduction of KCND2/3 transcription.he TRbeta1 aporeceptor suppressed KCND3 expressionincreasing TR expression in the hypertrophied heart is associated with an improvement in contractile function and increased SERCA expression.TRbeta1 to transcription of beta-MyHC, SERCA, and TRbeta1,Our study investigates both the regulation of cardiac-specific genes and contractile behavior of the heart in the presence of a mutant thyroid hormone receptor beta1 (T3Rbeta1-delta337T) derived from the S kindredThe messenger RNA levels for MHC alpha and SERCA2 were markedly down-regulated, MHC beta messenger RNA was up-regulated. Although T3 levels were normal in these animals, this pattern of cardiac gene expression mimics a hypothyroid phenotype.\"}, {\"question\": \"Which hormone abnormalities are common in Williams syndrome\", \"ideal_answer\": [\"Elevated Thyrotropin - TSH\\nLow FT4\\nGrowth Hormone deficiency\\nCalcitonin deficiency\\nElevated Prolactin\\nElevated Cortisol\\nElevated Oxytocin\\nElevated Vasopressin\"], \"exact_answer\": [[\"Elevated Thyrotropin -\", \"Elevated TSH\"], [\"Low FT4\", \"Low Thyroxine\"], [\"Growth Hormone deficiency\", \"GHD\"], [\"Calcitonin deficiency\"], [\"Elevated Prolactin\"], [\"Elevated Cortisol\"], [\"Elevated Oxytocin\", \"Elevated OT\"], [\"Elevated Vasopressin\", \"Elevated AVP\"]], \"context\": \"A girl with Williams syndrome (WS) presented with elevated thyrotropin (TSH) levels (7.0 microU/ml), normal free thyroid hormone concentrations, and absent antithyroid autoantibodies.TSH response to thyrotropin-releasing hormone (TRH) injection (200 microg/mq, i.v.) was exaggerated and prolonged, suggesting subclinical hypothyroidism. The biological activity of circulating TSH was slightly below the normal range [TSH bioactivity (B) to immunoreactivity (I) ratio (TSH B/I) = 0.4, normal: 0.6-2.2].We report a boy with confirmed Williams-Beuren syndrome, who was found to have classical growth hormone deficiency and responded well to growth hormone therapy.A 31-year-old man who had been under regular hemodialysis for 6 months was diagnosed as Williams syndrome (WS) by fluorescence in situ hybridization (FISH) chromosomal analysis. The association of WS and chronic renal failure (CRF) is only rarely encountered. Endocrinological examinations revealed hypergonadotropic hypogonadism. Prolonged and exaggerated responses of adrenocorticotropin (ACTH) to insulin-induced hypoglycemia and corticotropin releasing hormone (CRH) were also noted.Involvement of the calcitonin/CGRP gene in Williams syndrome is postulated on the basis that Williams syndrome children often have infantile hypercalcemia and deficient expression of calcitonin, a hormone that lowers serum calcium levels. To test the hypothesis that mutations in the calcitonin/CGRP gene might be responsible for the reduced calcitonin levelsIt has been suggested that a defect in calcitonin function may play a role in Williams syndromeImparied calcitonin secretion in patients with Williams syndrome.in patients with Williams syndrome, we studied five such children, with intravenous calcium and parathyroid hormone infusions as provocative stimuli. These patients were found to have significantly higher mean baseline calcium concentrations, delayed clearance of calcium after intravenous calcium loading, and blunted calcitonin responses after calcium infusionOur studies demonstrate that patients with Williams syndrome have a defect in the synthesis or release of immunoreactive calcitonin. A deficiency of calcitonin may explain the abnormalities of calcium metabolism seen in these patients and can serve as an important endocrine marker for Williams syndrome.A 16-year-old male adolescent diagnosed to have the Williams-Beuren syndrome was referred to our obesity outpatient clinic, due to his morbid obesity (body mass index 39.2 kilograms per square metre) and gluttony. After several unsuccessful dietary treatments, we started therapy with sibutramine. As growth hormone (GH) deficiency was diagnosed by an additional GH-stimulation test, we commenced with a GH-treatment.  Subclinical hypothyroidism is a frequent but stable finding in young children with WS.Thyroid involvement in Williams syndrome (WS) was recently reported in two small groups of patients, both showing an increased prevalence of elevation of TSH serum concentration; in a population of 95 WS patients, half of them followed for more than 5 years. Our study confirms the increased incidence of both elevated TSH serum values (37.9% in our sample)This study confirms the presence of alterations of thyroid function in WS We report a female infant with confirmed WBS who, through provocative testing, was found to have GHDthe elevated levels of prolactin and the response of hGH to growth hormone releasing hormone (GHRH) administration are indicative of a hypothalamic rather than pituitary defect. In conclusion, GH deficiency might contribute to the growth failure in a number of patients with WBS a female presenting with congenital heart defects, liver hemangiomas, and facial dysmorphisms admitted to hospital at 3 months of age because of feeding difficulties and poor growth. She had hypotonia and large tongue, \\\"coarse\\\" face, and umbilical hernia in presence of complex congenital cardiovascular malformations. In spite of normal neonatal screening we performed serum levels of thyroid hormones. Thyrotropin level was very high WS and TD participants had similar profiles in a familiar setting, while participants with WS had elevated cortisol late in the day in the novel setting when social demands were higher. Results suggest that adults with WS have a typical diurnal cortisol profile Results revealed significantly higher median levels of OT in WS versus controls at baseline, with a less marked increase in AVPhigh prevalence of impaired glucose regulation, adults with WS Thyroid function tests showed a raised TSH (42 mIU/l; normal range 0.5-4 mIU/l) with a low FT(4) concentrationWBS and abnormalities of thyroid function are common in patients with this feature. T\"}, {\"question\": \"Which assays can be used for detecting DNA fragmentation resulting from programmed cell death (apoptosis)?\", \"ideal_answer\": [\"The biochemical hallmark of apoptosis is internucleosomal DNA cleavage into oligonucleosome-length fragments. Measurement of apoptosis-associated internucleosomal DNA fragmentation through determination of the percentage of fragmented DNA by electrophoresis or centrifugation of whole cell lysates is by far the most common means of quantifying apoptosis. DNA fragmentation due to apoptosis can also be identified using terminal deoxynucleotidyl transferase-mediated dUTP-biotin nick end labeling of DNA fragments (TUNEL), in situ end labeling (ISEL) of the genomic DNA in fragmented nuclei, and measurement of cytosolic histone-bound DNA fragments (cell death ELISA assays).\"], \"exact_answer\": [[\"electrophoresis of whole cell lysates\"], [\"centrifugation of whole cell lysates\"], [\"terminal deoxynucleotidyl transferase-mediated dUTP-biotin nick end labeling of DNA fragments (TUNEL)\"], [\"in situ end labeling (ISEL) of the genomic DNA in fragmented nuclei\"], [\"measurement of cytosolic histone-bound DNA fragments (cell death ELISA assays)\"], [\"comet assay\"], [\"DAPI staining assay\"], [\"anti-DNA immunoelectron microscopy\"], [\"fluorescence correlation spectroscopy (FCS)\"]], \"context\": \"the biochemical hallmark of apoptosis is internucleosomal DNA cleavage into oligonucleosome-length fragments. terminal deoxynucleotidyl transferase-mediated dUTP-biotin nick end labeling of DNA fragments (TUNEL) and in situ end labeling (ISEL) techniquesTUNEL labeling of DNA fragmentsApoptotic cells in the specimens were quantitated after terminal deoxynucleotidyl transferase labeling of DNA fragments with digoxigenin-deoxyuridine triphosphateThe outcome of chronic ethanol consumption recorded in liver by in situ staining of the genomic DNA in fragmented nuclei indicates the course of cellular events that has been coined as apoptosis or programmed cell death. levels of cytosolic histone-bound DNA fragments (cell death ELISA assays)Measurement of apoptosis-associated internucleosomal DNA fragmentation through determination of the percentage of fragmented DNA by electrophoresis or centrifugation of whole cell lysates is by far the most common means of quantifying apoptosis.Treatment with silica resulted in morphological changes typical of apoptotic cells, enhanced DNA fragmentation (a characteristic feature of programmed cell death), and significant alveolar macrophage apoptosis as observed by cell death ELISA assays.Programmed cell death and DNA fragmentation were detected by Nile blue staining and cytochemical labeling of DNA fragmentation, respectively, in the interdigital mesoderm and in the regions of presumptive joints of the digit.Treatment with silica resulted in morphological changes typical of apoptotic cells, enhanced DNA fragmentation (a characteristic feature of programmed cell death), and significant alveolar macrophage apoptosis as observed by cell death ELISA assays.Cell were treated with these particulates in vitro for 6 and 24 hr and examined for apoptosis by morphological analysis, DNA fragmentation, and levels of cytosolic histone-bound DNA fragments (cell death ELISA assays).Featured in this unit is the TUNEL method of detecting cells that exhibit DNA fragmentation, which can also be performed on tissue sections to locate apoptotic cells in situ.Single-cell gel electrophoresis, or the comet assay, a technique widely used for DNA damage analysis, has been used recently for detecting DNA fragmentation in cells undergoing apoptosisProgrammed cell death and DNA fragmentation were detected by Nile blue staining and cytochemical labeling of DNA fragmentation, respectively, in the interdigital mesoderm and in the regions of presumptive joints of the digitSingle-cell gel electrophoresis assay monitors precise kinetics of DNA fragmentation induced during programmed cell death.In addition, flow cytometry and DNA fragmentation assays confirmed DAPI staining assay results and indicated the occurrence of a programmed cell death (apoptosis) in the treated cellsFour frequently used flow cytometric techniques were evaluated: (i) Annexin V/Propidium Iodide assay, detecting the translocation of phosphatidylserine to the outer leaflet of the plasma membrane, simultaneously with preservation of the membrane integrity; (ii) Terminal deoxynucleotidyl Transferase (TdT) Uridine triphosphate (UTP) nick end labelling (TUNEL), revealing the presence of DNA strand breaks; (iii) DNA-flow cytometry, measuring DNA-stainability (DNA-fragmentation assay) and (iv) Phycoerythrin-labelled (PE) Apo2.7-assay, a monoclonal antibody against 7A6 antigen, a protein, which becomes exposed upon the mitochondrial membrane during apoptosisThe present single-cell gel electrophoresis assay offers a significant improvement in monitoring the kinetics of DNA fragmentation induced during programmed cell death\"}, {\"question\": \"Which are the APOBEC3 protein family members able to inhibit Vif-deficient HIV-1 replication?\", \"ideal_answer\": [\"APOBEC3G, APOBEC3F, APOBEC3DE, APOBEC3A, and APOBEC3H haplotypes II, V, and VII, provide protection against Vif-deficient HIV-1, through hypermutation of the viral genome, inhibition of reverse transcription, and inhibition of viral DNA integration into the host genome.\"], \"exact_answer\": [[\"APOBEC3G\"], [\"APOBEC3F\"], [\"APOBEC3DE\"], [\"APOBEC3A\"], [\"APOBEC3H haplotypes II, V, and VII\"]], \"context\": \"APOBEC3DE, APOBEC3F, APOBEC3G, and APOBEC3H haplotypes II, V, and VII provide protection against HIV-1\\u0394vif through hypermutation of the viral genome, inhibition of reverse transcription, and inhibition of viral DNA integration into the host genomelow quantities of IFN-\\u03b1 failed to upregulate costimulatory molecules, did not induce IL-12p40 or migration, but significantly induced A3G, A3A, and A3F mRNA expression and restricted viral replication in MDDCsWithout Vif, A3 proteins, particularly APOBEC3G (A3G) and APOBEC3F (A3F), inhibit HIV-1 replication by blocking reverse transcription and/or integration and hypermutating nascent viral cDNAIn addition to APOBEC3G, we find that three other human APOBEC3 proteins, APOBEC3D, APOBEC3F, and APOBEC3H, are all potent HIV-1 restriction factors.These data strongly implicate a combination of four APOBEC3 proteins--APOBEC3D, APOBEC3F, APOBEC3G, and APOBEC3H--in HIV-1 restriction.Furthermore, APOBEC3D, APOBEC3F, APOBEC3G, and APOBEC3H of the rhesus macaque also are packaged into and restrict Vif-deficient HIV-1 when stably expressed in T cells, and they are all neutralized by the simian immunodeficiency virus Vif protein.APOBEC3G is a retroviral restriction factor that can inhibit the replication of human immunodeficiency virus, type 1 (HIV-1) in the absence of the viral infectivity factor (Vif) protein.Human and rhesus APOBEC3D, APOBEC3F, APOBEC3G, and APOBEC3H demonstrate a conserved capacity to restrict Vif-deficient HIV-1.The human APOBEC3 (A3) cytidine deaminases, such as APOBEC3G (A3G) and APOBEC3F (A3F), are potent inhibitors of Vif-deficient human immunodeficiency virus type 1 (HIV-1).Human APOBEC3G (A3G) and APOBEC3F (A3F) inhibit the replication of Vif-deficient human immunodeficiency virus type 1 (HIV-1)Human cytidine deaminases APOBEC3G (A3G) and APOBEC3F (A3F) inhibit replication of Vif-deficient human immunodeficiency virus type 1 (HIV-1)Human APOBEC3G and other APOBEC3 cytidine deaminases inhibit a variety of retroviruses, including Vif-deficient HIV-1Three of these, rat APOBEC1, mouse APOBEC3, and human APOBEC3B, are able to inhibit HIV infectivity even in the presence of VifTwo members of this family, APOBEC3G and APOBEC3F, have been found to have potent activity against virion infectivity factor deficient (Deltavif) human immunodeficiency virus 1 (HIV-1)\"}, {\"question\": \"Which genetic defects are observed in Prader-Willi syndrome?\", \"ideal_answer\": [\"The predominant genetic defects in Prader-Willi syndrome are 15q11-13 deletions of paternal origin and maternal chromosome 15 uniparental disomy, or rare imprinting mutations, combined with monoallelic expression of the paternal alleles.\"], \"exact_answer\": [[\"15q11-13 deletions of paternal origin\"], [\"maternal chromosome 15 uniparental disomy\"], [\"rare imprinting mutations\"], [\"monoallelic expression of the paternal alleles\"]], \"context\": \"Prader-Willi syndrome is a complex genetic disease caused by lack of expression of paternally inherited genes on chromosome 15q11-q13.Prader-Willi syndrome (PWS) is caused by the disturbed expression of genes from the imprinted region of 15q11-q13, but the specific contributions of individual genes remain unknown.Prader-Willi syndrome (PWS) is a rare disorder caused by genetic defects in certain regions of chromosome 15q11-13.Prader-Willi syndrome (PWS) is an example of a human genetic disorder that involves imprinting genes on the proximal long arm of chromosome 15 and SNRPN gene as a candidate gene for this syndrome.Genomic imprinting plays an important role in the molecular pathogenesis of PWS that caused by paternal microdeletions of 15q11-q13 or maternal UPD of chromosome 15. The basic defect seemed to be an absence of function of PWS genes that are normally expressed only from the paternal chromosome 15.Abnormalities in imprinted inheritance occur in several genetic diseases and cancer, and are exemplified by the diverse genetic defects involving chromosome 15q11-q13 in Prader-Willi (PWS) and Angelman (AS) syndromes.The predominant genetic defects in PW are 15q11-13 deletions of paternal origin and maternal chromosome 15 uniparental disomy.In both disorders, these mutations are associated with parent-of-origin specific methylation at several 15q11-13 loci.Prader-Willi syndrome (PWS) is caused by absence of a paternal contribution of the chromosome region 15q11-q13, resulting from paternal deletions, maternal uniparental disomy, or rare imprinting mutations.Because methylation analysis can detect all three major classes of genetic defects associated with PWS (deletion, UPD, or imprinting mutations), methylation analysis with either PW71 or SNRPN is an efficient primary screening test to rule out a diagnosis of PWS.The predominant genetic defects in Prader-Willi syndrome (PWS) are 15q11-q13 deletions of paternal origin and maternal chromosome 15 uniparental disomy (UPD).In both disorders, these mutations are associated with parent-of-origin specific methylation at several 15q11-q13 loci. The critical PWS region has been narrowed to a approximately 320-kb region between D15S63 and D15S174, encoding several imprinted transcripts, including PAR5, IPW, PAR1 (refs 7,8) and SNRPN, which has so far been considered a strong candidate for the PWS gene.The genetic defects in Prader-Willi syndrome (PWS) and Angelman syndrome (AS) map to 15q11-13.\"}, {\"question\": \"Which are the state-of-the-art computational tools for the prediction of gene fusion events?\", \"ideal_answer\": [\"Gene fusion detection - also known as the 'Rosetta Stone' method - involves the identification of fused composite genes in a set of reference genomes, which indicates potential interactions between its un-fused counterpart genes in query genomes. A few methods/tools and computational pipelines for the detection of gene fusion events have been introduced. The basic steps followed in these approaches consist of (i) all-against-all sequence comparison, (ii) detection of non-overlapping similarities of two genes/proteins (components) to a single gene/protein (composite), and optionally (iii) elimination of putative spurious hits (e.g. due to promiscuous domains) achieves via clustering based on sequence similarity and examining dense regions of the resulting graph or by querying the PFAM database. An advantage of gene fusion analysis is that functional associations can be predicted even in cases of genes of unknown function. Due to the computationally intense nature of these approaches, precompiled data of this type are often organized in specialized databases. Tools and databases developed for this purpose include (in alphabetical order): fdfBLAST, FusionDB, InPrePPI, (Integrated method for Prediction of Protein-Protein Interactions), MosaicFinder, Phydbac2, PLEX, Predictome, Rosetta Stone method, STRING.\"], \"exact_answer\": [[\"fdfBLAST\"], [\"FusionDB\"], [\"InPrePPI\", \"Integrated method for Prediction of Protein-Protein Interactions\"], [\"MosaicFinder\"], [\"Phydbac2\"], [\"PLEX\"], [\"Predictome\"], [\"Rosetta Stone method\"], [\"STRING\"]], \"context\": \"MosaicFinder: Identification of fused gene families in sequence similarity networksThis leads to an efficient formulation of previous methods of fused gene identification, which we implemented in the Python program FusedTriplets.We implemented this method in the C++ program MosaicFinder, which additionally uses local alignments to discard false positive candidates and indicates potential fusion points.Inference of gene function based on gene fusion events: the rosetta-stone method.The basic idea is based on the principle of \\\"guilt by association.\\\" It is assumed that two proteins, which are found to be transcribed by a single transcript in one (or several) genomes are likely to be functionally linked, for example by acting in a same metabolic pathway or by forming a multiprotein complex. This method is of particular interest for studying genes that exhibit no, or only remote, homologies with already well-characterized proteinsThis chapter uses the FusionDB database (http://www.igs.cnrs-mrs.fr/FusionDB/) as source of information. FusionDB provides a characterization of a large number of gene fusion events at hand of multiple sequence alignments.PLEX can be searched iteratively and also enables searches for chromosomal gene neighbors and Rosetta Stone linkages.While phylogenomic profiles remain the central focus of Phydbac2, it now integrates chromosomal proximity and gene fusion analyses as two additional non-similarity-based indicators for inferring pairwise gene functional relationships.Functional links between proteins can often be inferred from genomic associations between the genes that encode them: groups of genes that are required for the same function tend to show similar species coverage, are often located in close proximity on the genome (in prokaryotes), and tend to be involved in gene-fusion events. The database STRING is a precomputed global resource for the exploration and analysis of these associations.Protein interaction maps for complete genomes based on gene fusion eventsHere we present a method that identifies gene-fusion events in complete genomes, solely based on sequence comparison.Gene fusions have been suggested to be useful characters for identifying evolutionary relationships because they constitute synapomorphies or cladistic characters. To investigate the fidelity of gene-fusion characters, we developed an approach for identifying differentially distributed gene fusions among whole-genome datasets: fdfBLAST.Genome-scale comparative analysis of gene fusions, gene fissionsHere we present Predictome, a database of predicted links between the proteins of 44 genomes based on the implementation of three computational methods--chromosomal proximity, phylogenetic profiling and domain fusionPairs of genes that function together in a pathway or cellular system can sometimes be found fused together in another organism as a Rosetta Stone protein--a fusion protein whose separate domains are homologous to the two functionally-related proteins.Using the Rosetta Stone method and this scoring scheme, we find all significant functional linkages for proteins of E. coli, P. horikshii and S. cerevisiae, and measure the extent of the resulting protein networks.\"}, {\"question\": \"List the results of mutated casein kinase 1 epsilon. \", \"ideal_answer\": [\"Mutation in casein kinase 1 epsilon results in a short circadian period, abnormal entrainment to light cycles, and potentiated resetting responses to light.\\nMutations of CK1epsilon found in breast cancer can suppress Wnt/beta-catenin as well as promote the Wnt/Rac-1/JNK and Wnt/NFAT pathways, thus contributing to breast cancer development via effects on cell adhesion and migration.\\nCsnk1e is regulating not only the timing of sleep, but also the REM sleep amount and NREM sleep architecture.\"], \"exact_answer\": [[\"short circadian period\"], [\"abnormal entrainment to light cycles\"], [\"potentiated resetting responses to light\"], [\"REM sleep amount\"], [\"NREM sleep architecture\"], [\"effects on cell adhesion\"], [\"effects on cell migration\"], [\"timing of sleep\"]], \"context\": \"These results demonstrate a role for Csnk1e in regulating not only the timing of sleep, but also the REM sleep amount and NREM sleep architecture, and support Csnk1e as a causal gene in the sleep QTL on chromosome 15.mutations of CK1epsilon found in breast cancer can suppress Wnt/beta-catenin as well as promote the Wnt/Rac-1/JNK and Wnt/NFAT pathways, thus contributing to breast cancer development via effects on cell adhesion and migrationresults in a short circadian period, abnormal entrainment to light cycles, and potentiated resetting responses to light.We now demonstrate that the mutation results in a heightened sensitivity to light, suggesting that CK1epsilon also regulates the photic entrainment pathway.\"}, {\"question\": \"Which diseases can Oncotype DX be used for?\", \"ideal_answer\": [\"Oncotype can be used for predicting breast cancer and colon cancer recurrence.\"], \"exact_answer\": [[\"breast cancer\"], [\"colon cancer\"]], \"context\": \"Prognostic and predictive markersOncotype DX\\u00ae currently allow avoidance of an over therapy or under therapya high score on the Oncotype DX gene expression profile rpoor-prognosis ER-positive breast cancers in need of more aggressive therapyluminal B classThe 21-gene recurrence score (Oncotype DX: RS) appears to augment clinico-pathologic prognostication and is predictive of adjuvant chemotherapy benefit in node-negative (N-) and node-positive (N+), endocrine-sensitive breast cancer.Half of all breast cancers are early stage, lymph node negative, and hormone receptor positive. A 21-gene (Oncotype DX\\u00ae; Genomic Health, Inc., Redwood City, CA) recurrence score (RS) is prognostic for recurrence and predictive of chemotherapy benefit.We hypothesized that a distinct network of immune function genes at the tumor site can predict a low risk versus high risk of distant relapse in breast cancer patients regardless of the status of ER, PR, or HER-2/neu in their tumors.Real-time RT-PCR confirmed the 5-gene prognostic signature that was distinct from an FDA-cleared 70-gene signature of MammaPrint panel and from the Oncotype DX recurrence score assay panel. These data suggest that neoadjuvant immunotherapy in patients with high risk of relapse may reduce tumor recurrence by inducing the immune function genes.Molecular tests such as the Oncotype DX(\\u00ae) breast cancer test are proving to be more effective tools for individualized patient stratification and treatment planning than traditional methods such as patient demographic variables and histopathology indicators.Node-negative breast cancer does not automatically suggest a good prognosis, or the preclusion of chemotherapy benefits, and additional biomarkers are needed to help identify patients who do benefit from chemotherapy.The oncotype DX\\u00ae genomic assay has also been used to help prognosis estimation and treatment decisions.The Oncotype DX Recurrence Score (RS) is a validated genomic predictor of outcome and response to adjuvant chemotherapy in ER-positive breast cancer.The 21-gene signature has been intensively studied and incorporated into major guidelines for treatment decision in early breast cancer.Among early-stage breast cancer patients who received Oncotype DX, we found low knowledge about many aspects of genomic recurrence risk testing.the gene expression signatures that define specific prognostic subtypes in other breast cancer datasets, such as luminal A and B, basal, normal-like, and ERBB2+, and prognostic signatures including MammaPrint and Oncotype DX, predicted genomic instability in our samples. This remarkable congruence suggests a biological interdependence of poor-prognosis gene signatures, breast cancer subtypes, genomic instability, and clinical outcome.The Oncotype DX assay is one of the molecular tests that provide predictive and prognostic information to breast cancer patients with estrogen receptor (ER)-positive and node-negative disease. This study evaluates the association of Forkhead-box protein A1 (FOXA1) and GATA-binding protein 3 (GATA3) expressions with Oncotype DX recurrences scores in 77 cases of patients with ER-positive node-negative breast carcinomas diagnosed at Indiana University.the use of the Oncotype DX assay in estrogen receptor-positive node-negative breast cancer patients were incorporated into guidelines from both the American Society of Clinical Oncology and the National Comprehensive Cancer Network. The Oncotype DX assay is a diagnostic test which measures changes in a set of 21 genes in order to predict the likelihood of disease recurrence and also to predict which patients are most likely to respond to chemotherapy.The EWG found adequate evidence regarding the association of the Oncotype DX Recurrence Score with disease recurrence and adequate evidence for response to chemotherapy.EWG) found insufficient evidence to make a recommendation for or against the use of tumor gene expression profiles to improve outcomes in defined populations of women with breast cancer.confirmed the clinical validity of the Oncotype DX breast cancer assay, not only as a way to predict recurrence but also as a tool for determining therapeutic benefit from adjuvant chemotherapy.Over several decades, investigators working through National Cancer Institute-sponsored Cooperative Groups have contributed to major advances in the endocrine treatment of breast cancer.Cooperative Group studies have contributed to the development of a molecular profiling test, Oncotype Dx, which identifies women who have an excellent prognosis with hormonal therapy alone.In this report we compare the prognostic and predictive utility of these two tools in node-negative, ER-positive breast cancerThe Oncotype-DX Breast Cancer Assay (Genomic Health, Redwood City, CA) quantifies gene expression for 21 genes in breast cancer tissue by performing reverse transcription polymerase chain reaction (RT-PCR) on formalin-fixed paraffin-embedded (FFPE) tumour blocks that are obtained during initial surgery (lumpectomy, mastectomy, or core biopsy) of women with early breast cancer that is newly diagnosed.78 women, treated for early-stage, estrogen receptor-positive breast cancer with 0-3 positive lymph nodes, whose medical records indicated they received Oncotype DX testing earlier.A retrospective chart review of 58 T1/T2, node-negative, estrogen-receptor positive breast cancer patients that underwent Oncotype DX gene assay testing between January and December 2006 was performed.\"}, {\"question\": \"Which are the DNA (cytosine-5-)-methyltransferases inhibitors?\", \"ideal_answer\": [\"DNA (Cytosine-5-)-methyltransferases are a family of enzymes that methylate DNA at the C5 position of cytosine residues. Given that methylation of tumour suppressor gene promoters leads to carcinogenesis, inhibition of DNA (Cytosine-5-)-methyltransferases is a promising strategy for the treatment of cancer. There are several inhibitors of DNA (Cytosine-5-)-methyltransferases that uses different modes of action: 5-azacytidine (5-aza-CR, Vidaza\\u00ae), 5-azadeoxycytidine (5-aza-CdR, decitabine, Dacogen\\u00ae), 5-azacytosine (ZCyt), 5-fluorodeoxycytidine (FdC), 5,6-dihydro-5-azacytosine (DZCyt), 4'-thio-2'-deoxycytidine, hydralazine, 2-(1H)-pyrimidinone riboside (zebularine), 2-(1H)-pyrimidinone (zebularine aglycon), procaine, procainamide, psammaplin A, and RSC133, a new synthetic derivative of indoleacrylic acid/indolepropionic acid.\"], \"exact_answer\": [[\"5-azacytidine\", \"5-aza-CR\", \"Vidaza\\u00ae\"], [\"5-azadeoxycytidine\", \"5-aza-CdR\", \"Dacogen\\u00ae\", \"5-aza-2'-deoxycytidine\", \"Decitabine\"], [\"5-azacytosine\", \"ZCyt\"], [\"5-fluorodeoxycytidine\", \"FdC\"], [\"5,6-dihydro-5-azacytosine\", \"DZCyt\"], [\"4'-thio-2'-deoxycytidine\"], [\"hydralazine\"], [\"2-(1H)-pyrimidinone riboside\", \"zebularine\"], [\"2-(1H)-pyrimidinone\", \"zebularine aglycon\"], [\"procaine\"], [\"procainamide\"], [\"RSC133\"], [\"Psammaplin A\"]], \"context\": \"5-Aza-2'-deoxycytidine (5-Aza-CdR), a nucleoside analog that can inhibit DNA cytosine methylation, possesses potent antitumorigenic activities for myeloid disorders. Although 5-Aza-CdR is known to be incorporated into DNA and inhibit DNA (cytosine-5)-methyltransferases, the precise mechanisms underlying the drug's antineoplastic activity remain unclear.The mechanism of inhibition of DNA (cytosine-5-)-methyltransferases by 5-azacytosine is likely to involve methyl transfer to the inhibitor.The mechanism of inhibition of DNA (cytosine-5-)-methyltransferases by the mechanism-based inhibitor 5-azacytosine has remained unclear, mainly because of the unavailability of a substrate in which the inhibitor, but not normal cytosine, is present at the target site.These results suggest that, when the enzyme binds to 5-azacytosine in the presence of the cofactor, a methyl group is transferred to the N-5 position of the base, resulting in the inactivation of the enzyme.Rapid synthesis of new DNMT inhibitors derivatives of procainamide.Six compounds resulted in potent inhibitors of the murine catalytic Dnmt3A/3L complex and of human DNMT1, at least 50 times greater than that of the parent compounds. The inhibitors showed selectivity for C5 DNA methyltransferases.DNA (Cytosine-C5) methyltransferase inhibition by oligodeoxyribonucleotides containing 2-(1H)-pyrimidinone (zebularine aglycon) at the enzymatic target site.A number of DNA methyltransferase inhibitors are known to reactivate silenced genes; including 5-azacytidine and 2-(1H)-pyrimidinone riboside (zebularine). Zebularine is a more stable, less cytotoxic inhibitor compared to 5-azacytidine.ODNs containing 2-(1H)-pyrimidinone at the enzymatic target site are competitive inhibitors of both prokaryotic and mammalian DNA C5 methyltransferases. We determined that the ternary complexes between the enzymes, 2-(1H)-pyrimidinone inhibitor, and the cofactor S-adenosyl methionine are maintained through the formation of a reversible covalent interaction. The differing stability and reversibility of the covalent bonds may partially account for the observed differences in cytotoxicity between zebularine and 5-azacytidine inhibitors.DNA methyltransferases (DNMTs) are a family of enzymes that methylate DNA at the C5 position of cytosine residues, and their inhibition is a promising strategy for the treatment of various developmental and proliferative diseases, particularly cancers.The inhibitory activity of hydralazine toward DNMT may be rationalized at the molecular level by similar interactions within the binding pocket (e.g., by a similar pharmacophore) as established by substrate-like deoxycytidine analogues.Despite the different scaffolds of other non-nucleoside DNMT inhibitors such as procaine and procainamide, the current modeling work reveals that these drugs exhibit similar interactions within the DNMT1 binding site. These findings are valuable in guiding the rational design and virtual screening of novel DNMT inhibitors.Inhibition of HhaI DNA (Cytosine-C5) methyltransferase by oligodeoxyribonucleotides containing 5-aza-2'-deoxycytidine: examination of the intertwined roles of co-factor, target, transition state structure and enzyme conformation.The presence of 5-azacytosine (ZCyt) residues in DNA leads to potent inhibition of DNA (cytosine-C5) methyltranferases (C5-MTases) in vivo and in vitro.Thus, it is important to understand the critical mechanistic determinants of ZCyt's inhibitory action.Since methyl transfer can occur only in the presence of AdoMet, these results suggest (1) that the inhibitory capacity of ZCyt in DNA is based on its ability to induce a stable, tightly closed conformation of M.HhaI that prevents DNA and co-factor release and (2) that methylation of ZCyt in DNA is not required for inhibition of M.HhaI.Zebularine: a novel DNA methylation inhibitor that forms a covalent complex with DNA methyltransferases.The inhibition of cytosine-[C5]-specific DNA methyltransferases (C5 MTases) by oligodeoxynucleotides containing 5-azadeoxycytidine (AzadC) and 5-fluorodeoxycytidine (FdC) provides a well-documented example of mechanism-based inhibition of enzymes central to nucleic acid metabolism.Synthesis of oligonucleotide inhibitors of DNA (Cytosine-C5) methyltransferase containing 5-azacytosine residues at specific sites.The incorporation of 5-azacytosine residues into DNA causes potent inhibition of DNA (Cytosine-C5) methyltransferases. TMechanism of inhibition of DNA (cytosine C5)-methyltransferases by oligodeoxyribonucleotides containing 5,6-dihydro-5-azacytosine.Substitution of DZCyt for target cytosines in C-G dinucleotides of single-stranded or double-stranded oligodeoxyribonucleotide substrates led to complete inhibition of methylation by murine DNA C5-MTase.Oligodeoxyribonucleotides containing DZCyt formed a tight but reversible complex with M.HhaI, and were consistently more potent as inhibitors of DNA methylation than oligodeoxyribonucleotides identical in sequence containing 5-fluorocytosine.These results indicate that DZCyt can occupy the active site of M.HhaI as a transition state mimic and, because of the high degree of affinity of its interaction with the enzyme, it can act as a potent inhibitor of methylation.Analysis of DNA methylation processes related to the inhibition of DNA synthesis by 5-azacytidine in Streptomyces antibioticus ETH 7451.Our results suggest that some of the 5-azacytidine effects on DNA and RNA synthesis might indeed be related to the complex formation and inhibition of a cytosine-specific DNA methyltransferase.DNA containing 4'-thio-2'-deoxycytidine inhibits methylation by HhaI methyltransferase.The inhibitory effect of the 4'sulfur atom on enzymatic activity may be traced to perturbation of a step in the methylation reaction after DNA binding but prior to methyl transfer.Treatment of Schizosaccharomyces pombe with the C5 DNA methyltransferase (C5Mtase) inhibitor 5-azacytidine (5-azaC) has previously been shown to induce G2 checkpoint-dependent cell cycle arrest.Inhibitors of DNA methyltransferase (Dnmt) enzymes have been advocated as a means to promote and stabilize Foxp3 expression in Tregs undergoing expansion in vitro before their injection in vivo.Booster of pluripotency: RSC133, a new synthetic derivative of indoleacrylic acid/indolepropionic acid, exhibits dual activity by inhibiting histone deacetylase and DNA methyltransferase.DNA methyltransferase inhibitors (DNMTIs), including decitabine (DAC) and azacitidine (AZA), have recently been highlighted for the treatment of high-risk myelodysplastic syndrome (MDS); however, their action mechanisms have not been clearly defined.Structurally, however, gemcitabine has similarities to             5-aza-2-deoxycytidine (decitabine/Dacogen\\u00ae), a DNA methyltransferase inhibitor             (DNMTi).Although 5-Aza-CdR is known to be incorporated into DNA and inhibit DNA (cytosine-5)-methyltransferases, the precise mechanisms underlying the drug's antineoplastic activity remain unclear.Human colon cancer cell lines (HCT116, HT29, SW48, SW480) were treated with 5-aza-2'-deoxycytidine (DAC), as a DNA methyltransferase inhibitor, followed by trichostatin A (TSA), as a histone deacetylase inhibitor.With increasing 5-Aza-CdR concentrations, the expression of DNA methyltransferases, DNMT3A and DNMT3B, significantly decreased in a dose-dependent manner.Psammaplin A (11c) is a marine metabolite previously reported to be a potent inhibitor of two classes of epigenetic enzymes: histone deacetylases and DNA methyltransferases.Defining the parameters of zebularine-mediated tumor inhibition may advance the future development of DNA methyltransferase inhibitors as an effective cancer treatment.The DNA methyltransferase inhibitor (5-aza-2-deoxycytidine) and the histone deacetylase inhibitor [trichostatin A (TSA)] both significantly up-regulated miR-146a transcriptional activation by altering the DNA-binding activity of NF-\\u03baB in macrophages isolated from aged mice, which suggests that DNA methylation and histone acetylation are involved in the suppression of age-dependent miR-146a expression.Diverse DNA methyltransferase inhibitors are being studied as potential anticancer drugs, and there is interest in developing novel and more effective DNMTIs.Currently, there are only a few DNMT1 inhibitors with potential application as therapeutic agents or research tools.The effects of the deoxycytidine analogue decitabine (5-aza-2'-deoxycytidine), which depletes DNA methyl transferase 1 (DNMT1), on melanoma differentiation were examined.Cooperative action of HDACi and DNA methylation inhibitors (DNMTi) has been reported, making combined treatment an attractive choice for cancer therapy.\"}, {\"question\": \"Which are the main histone modifications associated with enhancers?\", \"ideal_answer\": [\"Histone 3 lysine 4 mono- (H3K4me1) and di-methylation (H3K4me2) are the main post-transcriptional histone modifications related to enhancer activity.\", \"Using H3K4me2 as a mark for active enhancers (PMID: 22270183) Hyperacetylation of histones H3 and H4, a mark of active chromatin, is established broadly across target loci by enhancers that function over long distances (PMID: 19021773) The enhancer region itself was marked by mono-methylation at K4 and K9, distinguishing it from the methyl marks in the gene coding region (PMID: 19021773) H3K4 methylation to monovalent and bivalent domains (PMID: 20621055)\", \"H3K4 methylation to monovalent and bivalent domains. The enhancer region itself was marked by mono-methylation at K4 and K9, distinguishing it from the methyl marks in the gene coding region. Hyperacetylation of histones H3 and H4, a mark of active chromatin, is established broadly across target loci by enhancers that function over long distances. Using H3K4me2 as a mark for active enhancers. \"], \"exact_answer\": [[\"Histone 3 lysine 4 mono-methylation (H3K4me1)\"], [\"Histone 3 lysine 4 di-methylation (H3K4me2)\"]], \"context\": \"Using H3K4me2 as a mark for active enhancersHyperacetylation of histones H3 and H4, a mark of active chromatin, is established broadly across target loci by enhancers that function over long distancesThe enhancer region itself was marked by mono-methylation at K4 and K9, distinguishing it from the methyl marks in the gene coding regionH3K4 methylation to monovalent and bivalent domains\"}, {\"question\": \"With which complexes is the protein SUS1 associated?\", \"ideal_answer\": [\"Sus1/ENY2 is a component of the SAGA and TREX-2 complexes\"], \"exact_answer\": [[\"SAGA complex\"], [\"TREX-2 complex\"]], \"context\": \"The yeast DUBm comprises a catalytic subunit, Ubp8, and three additional subunits, Sgf11, Sus1 and Sgf73, all of which are required for DUBm activity.the deubiquitination module (DUBm) of SAGA, which is composed of Ubp8, Sus1, Sgf11, and Sgf73.The conserved Sac3:Thp1:Sem1:Sus1:Cdc31 (TREX2) complex binds to nuclear pore complexes (NPCs)Sus1 (hENY2) participates in this coordination as part of two protein complexes: SAGA, a transcriptional co-activator; TREX-2, which functions in mRNA biogenesis and export.The deubiquitinating module (DUBm) of the SAGA coactivator contains the Ubp8 isopeptidase, Sgf11, Sus1, and Sgf73, which form a highly interconnected complex. Sus1/ENY2, a component of the SAGA and TREX-2 complexes, is involved in both transcription and mRNA exportSus1 is a component of both the SAGA transcriptional co-activator complex and the TREX-2 complex that binds to nuclear pore complexes. distinct subcomplex called the deubiquitinating module (DUBm), which contains the ubiquitin-specific protease Ubp8, bound to Sgf11, Sus1, and Sgf73.The yeast deubiquitinase Ubp8 protein is recruited and activated by the SAGA complex and, together with Sgf11, Sus1, and Sgf73, forms a DUB module responsible for deubiquitinating histone H2B during gene expression. In the nucleus, Sus1 is associated to the transcriptional co-activator SAGA and to the NPC associated complex termed TREX2/THSC\"}, {\"question\": \"Which are the main NMD factors in Saccharomyces cerevisiae?\", \"ideal_answer\": [\"Nonsense-mediated mRNA decay (NMD) is a surveillance mechanism that accelerates the degradation of mRNAs containing premature translation termination codons. This quality control pathway depends on the NMD-specific factors, Upf1p, Upf2p/Nmd2p, and Upf3p, as well as the two release factors, eRF1 and eRF3 (respectively designated Sup45p and Sup35p in yeast). NMD activation is also enabled by the absence of the poly(A)-binding protein, Pab1p, downstream of a termination event\", \"In Saccharomyces cerevisiae, rapid degradation of nonsense-containing mRNAs requires the three nonsense-mediated mRNA decay (NMD) factors, Upf1p, Nmd2p, and Upf3p.\", \"In addition to their well-documented roles in the promotion of nonsense-mediated mRNA decay (NMD), yeast Upf proteins (Upf1, Upf2/Nmd2, and Upf3) also manifest translational regulatory functions, at least in vitro, including roles in premature translation termination and subsequent reinitiation\"], \"exact_answer\": [[\"Upf1p\"], [\"Nmd2p\"], [\"Upf3p\"]], \"context\": \"In addition to their well-documented roles in the promotion of nonsense-mediated mRNA decay (NMD), yeast Upf proteins (Upf1, Upf2/Nmd2, and Upf3) also manifest translational regulatory functions, at least in vitro, including roles in premature translation termination and subsequent reinitiationIn Saccharomyces cerevisiae, nonsense-mediated mRNA decay (NMD) requires Upf1p, Upf2p, and Upf3p to accelerate the decay rate of two unique classes of transcripts: (1) nonsense mRNAs that arise through errors in gene expression, and (2) naturally occurring transcripts that lack coding errors but have built-in features that target them for accelerated decay (error-free mRNAs)In Saccharomyces cerevisiae, rapid degradation of nonsense-containing mRNAs requires the decapping enzyme Dcp1p, the 5'-to-3' exoribonuclease Xrn1p, and the three nonsense-mediated mRNA decay (NMD) factors, Upf1p, Nmd2p, and Upf3pOur results indicate that Upf1p, Nmd2p, and Upf3p regulate decapping and exonucleolytic degradation of nonsense-containing mRNAs. In addition, we show that these factors also regulate the same processes in the degradation of wild-type mRNAsThis quality control pathway depends on the NMD-specific factors, Upf1p, Upf2p/Nmd2p, and Upf3p, as well as the two release factors, eRF1 and eRF3 (respectively designated Sup45p and Sup35p in yeast)In addition to their well-documented roles in the promotion of nonsense-mediated mRNA decay (NMD), yeast Upf proteins (Upf1, Upf2/Nmd2, and Upf3) also manifest translational regulatory functions, at least in vitro, including roles in premature translation termination and subsequent reinitiation. Association of yeast Upf1p with direct substrates of the NMD pathway Translation termination factors eRF1 (Sup45) and eRF3 (Sup35) participate not only in the last step of protein synthesis but also in mRNA degradation and translation initiation via interaction with such proteins as Pab1, Upf1, Upf2 and Upf3.From these results, we conclude that sup111, sup112 and sup113 are mutant alleles of UPF2, UPF3 and UPF1, respectively, and thus attribute suppressor activity of these mutations to defects in the NMD (nonsense-mediated mRNA decay) machinery.In Saccharomyces cerevisiae, nonsense-mediated mRNA decay (NMD) requires Upf1p, Upf2p, and Upf3p to accelerate the decay rate of two unique classes of transcripts: (1) nonsense mRNAs that arise through errors in gene expression, and (2) naturally occurring transcripts that lack coding errors but have built-in features that target them for accelerated decay (error-free mRNAs).In Saccharomyces cerevisiae, rapid degradation of nonsense-containing mRNAs requires the decapping enzyme Dcp1p, the 5'-to-3' exoribonuclease Xrn1p, and the three nonsense-mediated mRNA decay (NMD) factors, Upf1p, Nmd2p, and Upf3p.To date, human (h) Upf1 protein (p) (hUpf1p), a group 1 RNA helicase named after its Saccharomyces cerevisiae orthologue that functions in both translation termination and NMD, has been the only factor shown to be required for NMD in mammalian cells.In Saccharomyces cerevisiae, rapid degradation of nonsense-containing mRNAs requires the decapping enzyme Dcp1p, the 5-to-3 exoribonuclease Xrn1p, and the three nonsense-mediated mRNA decay (NMD) factors, Upf1p, Nmd2p, and Upf3p.In Saccharomyces cerevisiae, three trans-acting factors (Upf1p to Upf3p) are required for NMD.Interactions between Upf1 and the decapping factors Edc3 and Pat1 in Saccharomyces cerevisiae.In yeast Saccharomyces cerevisiae, the activity of this pathway depends on the recognition of the PTC by the translational machinery and interaction of translation termination factors eRF1 and eRF3 with Upf1, Upf2 and Upf3 proteins.In yeast Saccharomyces cerevisiae, the activity of this pathway depends on the recognition of the PTC by the translational machinery and interaction of translation termination factors eRF1 and eRF3 with Upf1, Upf2 and Upf3 proteinsNMD activation is also enabled by the absence of the poly(A)-binding protein, Pab1p, downstream of a termination eventOf those, a group of weak and recessive suppressors, sup111, sup112 and sup113, is of particular interest because of their dependency on [PSI+], a yeast prion.\"}, {\"question\": \"Which histone marks are deposited by Set7?\", \"ideal_answer\": [\"Set7 is H4K20 monomethyltransferase. Upregulation of PR-Set7 expression upon loss of HCF-1 leads to improper mitotic H4-K20 methylation. Set7 (or some variant) has also been reported to perform mono-methylation on lysine-4 of H3.\"], \"exact_answer\": [[\"H4K20 monomethylation\"], [\"H3K4 monomethylation\"]], \"context\": \"the loss of H4 lysine 20 methylationTranscriptionally competent regions lack H4 lysine 20 methylationa human histone H4 lysine 20 methyltransferase and cloned the encoding gene, PR/SET07Localization of PR-Set7 to mitotic chromosomes and subsequent increase in H4 Lys 20 methylationthe H4-K20 methyltransferase PR-Set7. Indeed, upregulation of PR-Set7 expression upon loss of HCF-1 leads to improper mitotic H4-K20 methylation\"}, {\"question\": \"List non-surgical treatment modalities that are included in the Stupp protocol.\", \"ideal_answer\": [\"Radiotherapy and chemotherapy are non-surgical treatment modalities that are included in the Stupp protocol. This protocol is widely used for treatment of glioblastoma.\"], \"exact_answer\": [[\"radiotherapy\"], [\"chemotherapy\"]], \"context\": \"OBJECTIVE: It is now accepted that the concomitant administration of temozolomide with radiotherapy (Stupp regime), in the treatment of patients with newly diagnosed glioblastoma multiforme (GBM), significantly improves survival and this practice has been adopted locally since 2004.BACKGROUND: The introduction of ALA-Fluorescence-guided surgery (FGS) followed by concomitant radiochemotherapy according to the Stupp-protocol is representative of the major changes in glioblastoma therapy in the past years.CONCLUSIONS: FGS and radiochemotherapy according to the Stupp protocol have induced an impressive improvement in overall survival in glioblastoma patients. Current treatment strategies in patients with newly-diagnosed glioblastoma include surgical resection with post-operative radiotherapy and concomitant/adjuvant temozolomide (the \\\"Stupp protocol\\\") or resection with implantation of 1,3-bis(2-chloroethyl)-1-nitrosourea (BCNU) wafers in the surgical cavity followed by radiotherapy. The standard treatment today is maximal surgical resection followed by concomitant chemo-radiation therapy followed by adjuvant TMZ according to Stupp protocol. PATIENTS AND METHODS: A total of 191 patients with primary GBM were postoperatively treated with either radiation and concomitant TMZ, followed by adjuvant TMZ (Stupp protocol) (n = 154), or radiation followed by adjuvant TMZ (n = 37).Afterward, patients received concomitant radiochemotherapy according to the Stupp protocol. The treatment of glioblastoma includes surgery followed by chemoradiation with the protocol of Stupp et al. Since the publication of a study by Stupp et al in 2005, which used a protocol of conventional fractionated irradiation with concomitant TMZ followed by standard TMZ for six cycles, many clinical studies in the People's Republic of China have demonstrated that such a treatment strategy has significantly improved efficacy with limited side effects for newly diagnosed glioblastoma after surgery as compared with strategies that do not contain TMZ.Standard chemoradiotherapy as per the Stupp protocol, together with multimodal rehabilitation, resulted in substantial functional improvement within 6 weeks of initiation.BACKGROUND: Patients with glioblastoma treated with BCNU wafer implantation for recurrence frequently receive frontline chemoradiotherapy with temozolomide as part of the Stupp protocol.This is followed by concurrent radio-chemotherapy with temozolomide (TMZ) according to the Stupp protocol. CONCLUSIONS: MGMT promoter methylation status and low MGMT expression both were identified as positive prognosticators in patients with newly diagnosed glioblastoma who underwent surgical resection and received Gliadel wafer implants followed by adjuvant radiotherapy and concomitant oral TMZ chemotherapy (the Stupp protocol).INTRODUCTION: To evaluate the incidence and impact of early post-chemoradiation (cRT) 'pseudoprogression' (PsPD) amongst glioblastoma multiforme (GBM) patients treated with the current standard of care - 60 Gy conformal radiotherapy with concurrent low-dose temozolomide, followed by six cycles of high-dose temozolomide (the 'Stupp protocol'). In patients with glioblastoma multiforme (GBM), there is no consensus on the sequential use of two existing regimens: post-operative Gliadel implantation into the surgical cavity and concomitant temozolomide with radiotherapy followed by adjuvant temozolomide ('Stupp protocol'). We also screened for associations between the level of methylation of CpG sites and overall survival in a cohort of 50 patients uniformly treated by surgery, radiotherapy and chemotherapy with concomitant and adjuvant temozolomide (STUPP protocol). Surprisingly, less than half the patients having surgery received concomitant radiochemotherapy according to the Stupp protocol. We analyzed MGMT methylation by methylation-specific PCR in 90 GBM patients from four Portuguese hospitals, uniformly treated with radiotherapy combined with concomitant and adjuvant temozolomide (Stupp protocol).The current care standards for newly diagnosed glioblastoma consist, when feasible, in surgical resection, radiotherapy, and chemotherapy, as described in the Stupp protocol. Moreover, there are no data concerning the effect of concomitant radiochemotherapy according to the STUPP protocol on the course ofMS in patients with coexisting glioblastoma. Current treatment strategies in patients with newly-diagnosed glioblastoma include surgical resection with post-operative radiotherapy and concomitant/adjuvant temozolomide (the \\\"Stupp protocol\\\") or resection with implantation of 1,3-bis(2-chloroethyl)-1-nitrosourea (BCNU) wafers in the surgical cavity followed by radiotherapyCurrent treatment strategies in patients with newly-diagnosed glioblastoma include surgical resection with post-operative radiotherapy and concomitant/adjuvant temozolomide (the \\\"Stupp protocol\\\") or resection with implantation of 1,3-bis(2-chloroethyl)-1-nitrosourea (BCNU) wafers in the surgical cavity followed by radiotherapyThe standard treatment today is maximal surgical resection followed by concomitant chemo-radiation therapy followed by adjuvant TMZ according to Stupp protocol\"}, {\"question\": \"Which genes are associated with Ehlers-Danlos syndrome type I/II?\", \"ideal_answer\": [\"It is currently estimated that approximately 50% of patients with a clinical diagnosis of classic Ehlers-Danlos syndrome harbor mutations in the COL5A1 and the COL5A2 gene, encoding the \\u03b11 and the \\u03b12-chain of type V collagen, respectively\", \"Ehlers-Danlos syndrome (EDS) type I (the classical variety) is a dominantly inherited, genetically heterogeneous connective-tissue disorder. Mutations in the COL5A1 and COL5A2 genes, which encode type V collagen, have been identified in several individuals. Most mutations affect either the triple-helical domain of the protein or the expression of one COL5A1 allele  It is currently estimated that approximately 50% of patients with a clinical diagnosis of classic Ehlers-Danlos syndrome harbor mutations in the COL5A1 and the COL5A2 gene, encoding the \\u03b11 and the \\u03b12-chain of type V collagen, respectively  \", \"It is currently estimated that approximately 50% of patients with a clinical diagnosis of classic Ehlers-Danlos syndrome harbor mutations in the COL5A1 and the COL5A2 gene, encoding the \\u03b11 and the \\u03b12-chain of type V collagen, respectively.\", \"Ehlers-Danlos syndrome (EDS) type I (the classical variety) is a dominantly inherited, genetically heterogeneous connective-tissue disorder. Mutations in the COL5A1 and COL5A2 genes, which encode type V collagen, have been identified in several individuals. Most mutations affect either the triple-helical domain of the protein or the expression of one COL5A1 allele  \"], \"exact_answer\": [[\"COL5A1\"], [\"COL5A2\"]], \"context\": \"It is currently estimated that approximately 50% of patients with a clinical diagnosis of classic Ehlers-Danlos syndrome harbor mutations in the COL5A1 and the COL5A2 gene, encoding the \\u03b11 and the \\u03b12-chain of type V collagen, respectivelyIn the majority of patients with molecularly characterized classic Ehlers-Danlos syndrome, the disease is caused by a mutation leading to a nonfunctional COL5A1 allele and resulting in haploinsufficiency of type V collagenA smaller proportion of patients harbor a structural mutation in COL5A1 or COL5A2, causing the production of a functionally defective type V collagen proteinOrder of intron removal influences multiple splice outcomes, including a two-exon skip, in a COL5A1 acceptor-site mutation that results in abnormal pro-alpha1(V) N-propeptides and Ehlers-Danlos syndrome type IEhlers-Danlos syndrome (EDS) type I (the classical variety) is a dominantly inherited, genetically heterogeneous connective-tissue disorder. Mutations in the COL5A1 and COL5A2 genes, which encode type V collagen, have been identified in several individuals. Most mutations affect either the triple-helical domain of the protein or the expression of one COL5A1 alleleWe identified a novel splice-acceptor mutation (IVS4-2A-->G) in the N-propeptide-encoding region of COL5A1, in one patient with EDS type IMutations of the alpha2(V) chain of type V collagen impair matrix assembly and produce ehlers-danlos syndrome type IWe have identified mutations of the COL5A2 gene, which encodes the alpha2(V) chain of type V collagen, in two unrelated patients with the severe type I form of EDSOur findings show that heterozygous mutations of the COL5A2 gene can produce the EDS type I phenotypeA splice-junction mutation in the region of COL5A1 that codes for the carboxyl propeptide of pro alpha 1(V) chains results in the gravis form of the Ehlers-Danlos syndrome (type I)We report that genomic polymorphisms at the pro alpha 1(V) gene (COL5A1) locus cosegregated with the gravis form of Ehlers-Danlos syndrome (EDS) (type I) in a three generation familySince exon 65 encodes 78 residues of the carboxyl propeptide, the expected result of this mutation is reduced efficiency in incorporating mutant pro alpha 1(V) chains into type V collagen molecules and reduced type V collagen synthesisThese studies indicate that heterozygous mutations in COL5A1 can result in EDS type IThe gene encoding collagen alpha1(V)(COL5A1) is linked to mixed Ehlers-Danlos syndrome type I/IIRecently, genetic linkage has been demonstrated between the COL5A1 gene, which encodes the alphal chain of type V collagen, and EDS type II in a large British kindredUsing a polymorphic intragenic simple sequence repeat at the COL5A1 locus, we now demonstrate tight linkage to EDS type I/II in a three-generation familyThe variation in expression in this family suggests that EDS types I and II are allelic, and the linkage data support the hypothesis that mutation in COL5A1 can cause both phenotypes.Null alleles of the COL5A1 gene of type V collagen are a cause of the classical forms of Ehlers-Danlos syndrome (types I and II).Recently mutations have been found in the genes for type V collagen in a small number of people with the most common forms of EDS, types I and IIThe gene encoding collagen alpha1(V)(COL5A1) is linked to mixed Ehlers-Danlos syndrome type I/II.Linkage of the gene that encodes the alpha 1 chain of type V collagen (COL5A1) to type II Ehlers-Danlos syndrome (EDS II).Genetic linkage to the collagen alpha 1 (V) gene (COL5A1) in two British Ehlers-Danlos syndrome families with variable type I and II phenotypes.\"}, {\"question\": \"Which factors are considered in the ABCD2 score?\", \"ideal_answer\": [\"Age, Blood pressure, Clinical features, Duration of symptoms and Diabetes are included in the ABCD2 score, which is used to identify patients having a transient ischemic attack who are at high risk for imminent stroke.\"], \"exact_answer\": [[\"Age\"], [\"Blood pressure\"], [\"Clinical features\"], [\"Duration of symptoms\"], [\"Diabetes\"]], \"context\": \"The 'accuracy' of age, blood pressure, clinical features, duration and diabetes (ABCD(2)) scoring by non-stroke specialists referring patients to a daily Rapid Access Stroke Prevention (RASP) service is unclear, as is the accuracy of ABCD(2) scoring by trainee residents.  Classification of low risk was based on an age, blood pressure, clinical features, duration of symptoms and diabetes (ABCD2) score <4 and the absence of high risk features, including known carotid disease, crescendo TIA, or atrial fibrillation. Our goal was to compare the very early predictive accuracy of the most relevant clinical scores [age, blood pressure, clinical features and duration of symptoms (ABCD) score, ABCD and diabetes (ABCD2) score, ABCD and brain infarction on imaging score, ABCD2 and brain infarction on imaging score, ABCD and prior TIA within 1 week of the index event (ABCD3) score, California Risk Score, Essen Stroke Risk Score and Stroke Prognosis Instrument II] in consecutive transient ischemic attack (TIA) patients. We examined the association of the age, blood pressure, clinical features, duration of symptoms and diabetes (ABCD2) score, a validated risk prediction model for stroke after TIA, and the presence of ICS or ECS. BACKGROUND: The ABCD2 score (Age, Blood pressure, Clinical features, Duration of symptoms and Diabetes) is used to identify patients having a transient ischemic attack who are at high risk for imminent stroke. BACKGROUND AND PURPOSE: The risk of stroke after a transient ischaemic attack (TIA) can be predicted by scores incorporating age, blood pressure, clinical features, duration (ABCD-score), and diabetes (ABCD2-score). BACKGROUND: Modifications to the age, blood pressure, clinical symptoms, duration of symptoms, and diabetes (ABCD2) score, which incorporate history of hypertension and acute hyperglycemia in addition to acute blood pressure (BP) elevation and history of diabetes, have been proposed to increase the predictive value of the score. ABCD2 (Age, Blood pressure, Clinical features, Duration, Diabetes) score and magnetic resonance imaging abnormalities help to identify patients at high risk of stroke.METHODS: We prospectively calculated the ABCD(2) score (age [> or = 60 years: 1 point]; blood pressure [systolic >140 mm Hg or diastolic >90 mm Hg: 1[; clinical features [unilateral weakness: 2, speech disturbance without weakness: 1, other symptom: 0]; duration of symptoms [ <10 minutes: 0, 10-59 minutes: 1, > or = 60 minutes: 2]; diabetes mellitus [yes: 1]) in consecutive TIA patients hospitalized in 3 tertiary care neurology departments across 2 different racial populations (white and Asian). Early stroke risk is especially high in TIA patients with a high ABCD2 score of 4 or more (A age over 60 years [1 point]: B blood pressure > 140/90 mmHg [1 point]: C Clinical features, including unilateral weakness [2 points] and speech disturbance without weakness [1 point] D2: Diabetes [1 point] and Duration of symptoms [1 point for < 60 min and 2 points for > 60 min]), acute ischemic lesions on diffusion weighted image, > 50% carotid stenosis, severe intracranial artery stenosis, microembolic signals on transcranial Doppler, atrial fibrillation, or hypercoagulable states.STUDY OBJECTIVE: We evaluate, in admitted patients with transient ischemic attack, the accuracy of the ABCD(2) (age [A], blood pressure [B], clinical features [weakness/speech disturbance] [C], transient ischemic attack duration [D], and diabetes history [D]) score in predicting ischemic stroke within 7 days. Our goal was to compare the very early predictive accuracy of the most relevant clinical scores [age, blood pressure, clinical features and duration of symptoms (ABCD) score, ABCD and diabetes (ABCD2) score, ABCD and brain infarction on imaging score, ABCD2 and brain infarction on imaging score, ABCD and prior TIA within 1 week of the index event (ABCD3) score, California Risk Score, Essen Stroke Risk Score and Stroke Prognosis Instrument II] in consecutive transient ischemic attack (TIA) patients.The risk of stroke after a transient ischaemic attack (TIA) can be predicted by scores incorporating age, blood pressure, clinical features, duration (ABCD-score), and diabetes (ABCD2-score).BACKGROUND AND PURPOSE: The risk of stroke after a transient ischaemic attack (TIA) can be predicted by scores incorporating age, blood pressure, clinical features, duration (ABCD-score), and diabetes (ABCD2-score).\"}, {\"question\": \"Which are the musculoskeletal manifestations of Marfan syndrome?\", \"ideal_answer\": [\"Musculoskeletal manifestations of Marfan syndrome include scoliosis, dural ectasia, pectus excavatum and carinatum, arachnodactyly, otto pelvis (protrusio acetabuli), dolichostenomelia and ligamentous laxity.\"], \"exact_answer\": [[\"scoliosis\"], [\"dural ectasia\"], [\"pectus excavatum and carinatum\"], [\"arachnodactyly\"], [\"otto pelvis\", \"protrusio acetabuli\"], [\"dolichostenomelia\"], [\"ligamentous laxity\"]], \"context\": \"Musculoskeletal manifestations include scoliosis, dural ectasia, protrusio acetabuli, and ligamentous laxity.Scoliosis, pectus excavatum and carinatum, arachnodactyly, and acetabular protrusion are common musculoskeletal manifestations.Two cases are described in which protrusio acetabuli was a major problem. Otto pelvis should be considered as one of the musculoskeletal manifestations of Marfan's syndrome.The following musculoskeletal abnormalities were found: pectus in 3 patients (11%), pectus and scoliosis in 19 (73%), dolichostenomelia in 11 (42%) and arachnodactyly in 21 (80%).Musculoskeletal clinicians should be aware of the diagnostic features of Marfan syndrome. Patients with three to four physically evident features, or two highly specific features (e.g., thumb and wrist signs, craniofacial features, dural ectasia, or protrusio), should be carefully reexamined and possibly referred for an echocardiogram or a genetics consultation.\"}, {\"question\": \"Which proteins are the different isoforms of the p38 MAP kinase?\", \"ideal_answer\": [\"The p38 Mitogen-Activated Protein (MAP) kinase, a serine/threonine kinase, is one of the best characterized kinases in the inflammatory process. There are four isoforms of the enzyme (p38alpha, p38beta, p38gamma and p38delta), which differ in tissue distribution, regulation of kinase activation and subsequent phosphorylation of downstream substrates. Among the four identified p38 isoforms (p38\\u03b1, p38\\u03b2, p38\\u03b3, and p38\\u03b4), the \\u03b1-form is the most fully studied.\", \"The mammalian p38 mitogen-activated protein kinases (MAPKs) family is composed of four members (p38\\u03b1, p38\\u03b2, p38\\u03b3, and p38\\u03b4), which are very similar in amino acid sequence but differ in their expression patterns.\", \"The mitogen-activated protein kinase (MAPK) p38 is a Ser/Thr kinase, originally isolated from lipopolysaccharide-stimulated monocytes. There are four isoforms of the enzyme (p38alpha, p38beta, p38gamma and p38delta), which differ in tissue distribution, regulation of kinase activation and subsequent phosphorylation of downstream substrates.  The p38 Mitogen-Activated Protein (MAP) kinase, a serine/threonine kinase, is one of the best characterized kinases in the inflammatory process. Among the four identified p38 isoforms (p38\\u03b1, p38\\u03b2, p38\\u03b3, and p38\\u03b4), the \\u03b1-form is the most fully studied and plays a central role in the biosynthesis of the proinflammatory cytokines i.e. IL-1\\u03b2 and TNF-\\u03b1 at the translational and transcriptional levels.  \"], \"exact_answer\": [[\"p38alpha\", \"p38\\u03b1\"], [\"p38beta\", \"p38\\u03b2\"], [\"p38gamma\", \"p38\\u03b3\"], [\"p38delta\", \"p38\\u03b4\"]], \"context\": \"The p38 Mitogen-Activated Protein (MAP) kinase, a serine/threonine kinase, is one of the best characterized kinases in the inflammatory process. Among the four identified p38 isoforms (p38\\u03b1, p38\\u03b2, p38\\u03b3, and p38\\u03b4), the \\u03b1-form is the most fully studied and plays a central role in the biosynthesis of the proinflammatory cytokines i.e. IL-1\\u03b2 and TNF-\\u03b1 at the translational and transcriptional levels.The p38 mitogen-activated protein kinase (MAPK) is one of the kinase pathways that regulate the production of IL-1\\u03b2 and TNF\\u03b1. Importantly, small molecule inhibitors of the p38 MAPK family have been developed and show efficacy in blocking the production of IL-1\\u03b2 and TNF\\u03b1. The p38 family consists of at least four isoforms (p38\\u03b1, \\u03b2, \\u03b3, \\u03b4) encoded by separate genes.Defining the roles of the various p38 family members, specifically p38alpha and p38beta, in these processes has been difficult. The anomalous p38 inhibitor effect on IL-1alpha induction of MMP-3 and phosphorylation of p38 delta/gamma suggests complex interactions between p38 MAP kinase isoforms and their differential uses by TNFalpha and IL-1alpha in TM.Human and porcine TM cells expressed p38 alpha, beta, delta, and gamma isoforms, which migrate coincident with bands of specific phosphorylation.The mitogen-activated protein kinase (MAPK) p38 is a Ser/Thr kinase, originally isolated from lipopolysaccharide-stimulated monocytes. There are four isoforms of the enzyme (p38alpha, p38beta, p38gamma and p38delta), which differ in tissue distribution, regulation of kinase activation and subsequent phosphorylation of downstream substrates.We report the molecular cloning of a novel isoform of p38 MAP kinase, p38 beta 2.The p38 MAP kinase kinase MKK6 is identified as a common activator of p38 alpha, p38 beta 2, and p38 gamma MAP kinase isoforms, while MKK3 activates only p38 alpha and p38 gamma MAP kinase isoforms. four identified p38 isoforms (p38\\u03b1, p38\\u03b2, p38\\u03b3, and p38\\u03b4), The mammalian p38 mitogen-activated protein kinases (MAPKs) family is composed of four members (p38\\u03b1, p38\\u03b2, p38\\u03b3, and p38\\u03b4), which are very similar in amino acid sequence but differ in their expression patterns. Four p38 isoforms (p38alpha, p38beta, p38gamma and p38delta) exist in mammalian cells, being p38alpha and p38gamma the most abundantly expressed isoforms in adult skeletal muscle. Among the four identified p38 isoforms (p38\\u03b1, p38\\u03b2, p38\\u03b3, and p38\\u03b4), the \\u03b1-form is the most fully studied and plays a central role in the biosynthesis of the proinflammatory cytokines iThe p38 mitogen-activated protein kinases are activated in response to various extracellular signals in eukaryotic cells and play a critical role in the cellular responses to these signals. The four mammalian isoforms (p38alpha, p38beta, p38gamma, and p38delta) are coexpressed and coactivated in the same cells.four p38 isoforms p38 MAPK isoforms are becoming increasingly important for a variety of cellular functions. The physiological functions of p38\\u03b1 and -\\u03b2 are now well documented in contrast to -\\u03b3 and -\\u03b4 which are comparatively under-studied and ill-defined.Of the four p38 isoforms known to date, two (p38alpha and p38beta) have been identified as targets for cytokine-suppressive anti-inflammatory drugs. the contribution of each of the four p38 isoforms (encoded by different genes) to senescence induction is unclear. In the current study, we demonstrated that p38alpha and p38gamma, but not p38beta, play an essential role in oncogenic ras-induced senescence. Involvement of the p38 mitogen-activated protein kinase alpha, beta, and gamma isoforms in myogenic differentiation.we showed here that p38alpha, beta, and gamma are expressed with distinct expression patterns during differentiation.The p38 family of kinases is a subgroup of the mitogen-activated protein kinase family. It is composed of four isoforms and is involved in critical biological processes as well as in inflammatory diseasesp38alpha and -beta2 are ubiquitously expressed, whereas p38gamma and -delta appear to have more restricted expression.\"}, {\"question\": \"What are the reported adverse effects of gabapentin used in children?\", \"ideal_answer\": [\"Limited literature data, suggest that gabapentin may cause rash that is severe enough to necessitate discontinuation in a small percentage of children.\\nIn a large survey of all age groups: The commonest adverse effects seen were somnolence, fainting, ataxia, nystagmus, tremor and headache, fatigue. However, their incidence was low and intensity mild. In a pediatric group only somnolence and dizziness were reported in 2 out of 33 patients.\\nBehavioural adverse effects are more common  in children with intellectual disability and attention deficit, worse in <10yo :  hyperactivity, defiance, irritability, agitation, aggression, explosive outbursts, oppositional behavior, often  warranting discontinuation of the medication.\"], \"exact_answer\": [[\"Skin reactions\"], [\"Somnolence\", \"drowsiness\", \"sedation\", \"lassitude\"], [\"Dizziness\"], [\"hyperactivity\"], [\"defiance\"], [\"irritability\"], [\"agitation\"], [\"aggression\"], [\"explosive outbursts\"], [\"oppositional behavior\"], [\"malaise\"]], \"context\": \"he purpose of our study was to review our experience with using gabapentin to treat insomnia in children. We identified 23 children,Adverse effects were noted in 6 children.To determine the effectiveness of gabapentin as an add-on therapy in children presenting with overactive bladder (OAB)Data were analyzed in 30 patients as treatment was terminated in 1 due to adverse effects.Gabapentin gives moderate results in children with OAB refractory to conventional anticholinergics. In general, the drug is well tolerated with fewer adverse effects.To describe a child who developed a skin reaction during gabapentin therapy and discuss how we evaluated the probability of an adverse drug reactionIn contrast to other antiepileptic drugs, skin reactions to gabapentin are considered uncommon. In adults, reported prevalence of rash possibly related to gabapentin range from 1% to 10%. A postmarketing surveillance study reported gabapentin treatment failure as a consequence of rash in 0.4% of 3000 patients. The product monograph does not mention rash in children.This case, and limited literature data, suggest that gabapentin may cause rash that is severe enough to necessitate discontinuation in a small percentage of children. Further research is needed to determine the actual incidence and severity of gabapentin-related rash in this population.Five patients treated in the Children's Hospital Pain Control Service for intractable neuropathic pain were included in gabapentin treatmentWe noticed a rapid improvement, in 1 week, of our patients' VAS scores (from 9 or 10 to 4 or 3) with minimal adverse effectsThe charts of all children given gabapentin for seizures were reviewed.Only 8 children (7%) reported adverse effects, and the drug was discontinued in two.A postmarketing surveillance study of gabapentin as add-on therapy for 3,100 patients in England.The cohort comprised 3,100 patients, of whom 136 (4%) were children.The most frequently reported adverse events reported during the first month of treatment, drowsiness/sedation, dizziness, and malaise/lassitude, also were the commonest reasons for discontinuing GBP and reported as suspected adverse drug reactions (ADRs)Neurologic-related events were the most frequently reported adverse events. They also were the commonest reasons for discontinuing treatment and reported as suspected ADRs.We reviewed the reports in the literature on the safety, adverse effects and tolerance of GBP as treatment, used as monotherapy or associated with other drugs, in epilepsies in patients of all age groups, including children, adults and the elderlyGBP is a well tolerated drug when used for monotherapy or associated with other antiepileptic drugs. The commonest adverse effects seen were somnolence, fainting, ataxia, nystagmus, tremor and headache. However, their incidence was low and intensity mild. There was no clear relationship between the dose given and appearance of side effects, except for fainting and somnolence.GBP is a drug which is free of unwanted side effects. Its safety and tolerability profile is excellent for treatment of simple and complex partial crises, with or without generalization in epileptic patients of all ages.Two studies were conducted to determine the single-dose pharmacokinetics of gabapentin in healthy subjects age 1 month to 12 years and to guide dose selection in safety and efficacy trials in pediatric patientsSingle doses of gabapentin were well tolerated by healthy pediatric subjects.The efficacy and safety of gabapentin as add-on therapy for refractory partial seizures in 237 children, aged 3 to 12 years were evaluated over a 6-month period.Thirteen patients (5%) withdrew during the 6-month period because of adverse events.Male or female patients, at least 12 years old, with a recent history of partial seizures with or without secondary generalization, were randomized to receive gabapentinOf the four most common adverse events (somnolence, dizziness, ataxia, fatigue), only one, dizziness, occurred more often in the nontitrated (Rapid initiation) group than in the titrated (Slow initiation) group.Twenty-six children with intellectual disability and six normal children, all suffering from refractory partial seizures, received open-label gabapentin (range = 10-50 mg kg(-1) day(-1); mean = 26.7 mg kg(-1) day(-1) as an add-on medication to their antiepileptic drug regimenIn the present patient population, patients younger than 10 years of age, all of whom had intellectual disability, were more likely to have side-effects than those older than 10 years of age. Observed adverse effects, which were generally mild, occurred in patients with baseline intellectual disability, attention deficit disorder and behavioural problems. Behavioural adverse effects warranted discontinuation of the medication in only three patients. The severity of intellectual disability (mild versus moderate or severe) did not affect the extent of the response or the occurrence of side-effects.However, children with intellectual disability who also are less than 10 years of age with baseline attention deficit appear to be at a higher risk of behavioural side-effects.Efficacy and safety of gabapentin monotherapy were evaluated in 33 children with newly diagnosed absence epilepsySomnolence and dizziness were the only adverse events reported by at least two patients during gabapentin treatment. No clinically important changes in laboratory assessments or other safety parameters were observed.Thirty-two children with refractory partial epilepsy received open-label gabapentin as an additional medication to their antiepileptic drug regimenThe major reported side effects were behavioral. These consisted of hyperactivity, irritability, and agitation that occurred in patients with baseline mental retardation with attention deficit.We report 2 children who received GBP for intractable seizures and who developed intolerable aggressive behavior requiring dose reduction or drug discontinuation. Behavioral changes should be recognized as a possible side effect of GBP, especially in mentally retarded children.We report 7 children who received gabapentin (GBP) as adjunctive medic ation and subsequently developed behavioral side effects. These behavioral changes consisted of intensification of baseline behaviors as well as new behavioral problems. Behaviors that parents considered most troublesome were tantrums, aggression directed toward others, hyperactivity, and defiance. All behavioral changes were reversible and were managed by dose reduction or discontinuation of GBP. All children had baseline attention deficit hyperactivity disorder and developmental delays.We described 3 learning disabled children, 1 aged 7 and 2 aged 10 years, with intractable partial seizures who developed severe behavioral problems while receiving modest doses of GBP. The children became hyperactive and had explosive outburst consisting of aggressive and oppositional behavior. The behavioral problems were sufficiently severe to require discontinuation of GBP despite moderately improved seizure control.\"}, {\"question\": \"Which are the 3 basic transcription factors that have been used for the direct reprogramming of fibroblasts into cardiomyocytes or cardiomyocyte like-cells?\", \"ideal_answer\": [\"Direct reprogramming of human cardiac fibroblasts (HCFs) into cardiomyocytes may hold great potential for this purpose. We found that functional cardiomyocytes can be directly induced from fibroblasts by a combination of three cardiac transcription factors, Gata4, Mef2c and Tbx5, in vitro and in vivo.\", \"Direct reprogramming of human cardiac fibroblasts (HCFs) into cardiomyocytes may hold great potential for this purpose. We reported previously that induced cardiomyocyte-like cells (iCMs) can be directly generated from mouse cardiac fibroblasts in vitro and vivo by transduction of three transcription factors: Gata4, Mef2c, and Tbx5, collectively termed GMT.  \", \"Cardiac fibroblasts, which represent 50% of the cells in the mammalian heart, can be directly reprogrammed to adult cardiomyocyte-like cells in vitro by the addition of Gata4, Mef2c and Tbx5 (GMT). Induced cardiomyocytes expressed cardiac-specific markers, had a global gene expression profile similar to cardiomyocytes, and contracted spontaneously.\"], \"exact_answer\": [[\"Gata4\"], [\"Mef2c\"], [\"Tbx5\"]], \"context\": \"Direct reprogramming of human cardiac fibroblasts (HCFs) into cardiomyocytes may hold great potential for this purpose. We reported previously that induced cardiomyocyte-like cells (iCMs) can be directly generated from mouse cardiac fibroblasts in vitro and vivo by transduction of three transcription factors: Gata4, Mef2c, and Tbx5, collectively termed GMT.Cardiac fibroblasts can be reprogrammed to cardiomyocyte-like cells by the introduction of three transcription factors: Gata4, Mef2c and Tbx5 (collectively referred to here as GMT).Here we describe a detailed step-by-step protocol for in vitro cardiac reprogramming using retroviruses encoding GMT.Overexpression of transcription factors MYOCD and SRF alone or in conjunction with Mesp1 and SMARCD3 enhanced the basal but necessary cardio-inducing effect of the previously reported GATA4, TBX5, and MEF2C. Using calcium activity as our primary outcome measure, we compared several published combinations of transcription factors along with novel combinations in mouse embryonic fibroblasts. The most effective combination consisted of Hand2, Nkx2.5, Gata4, Mef2c, and Tbx5 (HNGMT). This combination is >50-fold more efficient than GMT alone and produces iCMs with cardiomyocyte marker expression, robust calcium oscillation, and spontaneous beating that persist for weeks following inactivation of reprogramming factors.Here we show that four human cardiac transcription factors, including GATA binding protein 4, Hand2, T-box5, and myocardin, and two microRNAs, miR-1 and miR-133, activated cardiac marker expression in neonatal and adult human fibroblasts. We found that functional cardiomyocytes can be directly induced from fibroblasts by a combination of three cardiac transcription factors, Gata4, Mef2c and Tbx5, in vitro and in vivo.To assess the efficiency of direct fibroblast reprogramming via viral overexpression of GATA4, Mef2c, and Tbx5 (GMT).  In addition, transplantation of GMT infected CFs into injured mouse hearts resulted in decreased cell survival with minimal induction of cardiomyocyte genes.Here, to identify a cardiac transcription factor combination facilitating mouse fibroblast reprogramming into cardiomyocytes, we directly screened all triplet combinations of 10 candidate factors combined with a Q-PCR assay reporting induction of multiple cardiac-specific genes. Through this screening method the combination of Tbx5, Mef2c, and Myocd was identified to upregulate a broader spectrum of cardiac genes compared to the combination of Tbx5, Mef2c, and Gata4 that was recently shown to induce reprogramming of fibroblasts into cardiomyocytes.We reported previously that cardiac fibroblasts,which represent 50%of the cells in the mammalian heart, can be directly reprogrammed to adult cardiomyocyte-like cells in vitro by the addition of Gata4, Mef2c and Tbx5 (GMT).Here, we report that a combination of three developmental transcription factors (i.e., Gata4, Mef2c, and Tbx5) rapidly and efficiently reprogrammed postnatal cardiac or dermal fibroblasts directly into differentiated cardiomyocyte-like cells. Induced cardiomyocytes expressed cardiac-specific markers, had a global gene expression profile similar to cardiomyocytes, and contracted spontaneously.More recently, we showed that reprogramming of cardiac fibroblasts to cardiomyocyte-like cells in vivo after coronary artery ligation using three cardiac transcription factors (Gata4/Mef2c/Tbx5) offers an alternative approach to regenerate heart muscle.Here, we report that a combination of three developmental transcription factors (i.e., Gata4, Mef2c, and Tbx5) rapidly and efficiently reprogrammed postnatal cardiac or dermal fibroblasts directly into differentiated cardiomyocyte-like cells.Srivastava's group reported the first direct reprogramming of mouse fibroblast cells into mesoderm lineage cells (cardiomyocytes) with the enforced expression of three cardiac transcriptional factors: Gata4, Mef2c, and Tbx5.Srivastava's group reported the first direct reprogramming of mouse fibroblast cells into mesoderm lineage cells (cardiomyocytes) with the enforced expression of three cardiac transcriptional factors: Gata4, Mef2c, and Tbx5Cardiac fibroblasts can be reprogrammed to cardiomyocyte-like cells by the introduction of three transcription factors: Gata4, Mef2c and Tbx5 (collectively referred to here as GMT)\"}, {\"question\": \"Which are the major intramolecular phosphorylation sites of human Chk2 involved in cell cycle control?\", \"ideal_answer\": [\"The major phosphorylation sites of human Chk2 involved in cell cycle control are T68, S19, and S33/35.\"], \"exact_answer\": [[\"T68\"], [\"S19\"], [\"S33/35\"]], \"context\": \"Transfection of HEK293 cells with Chk2 wildtype and Chk2 mutants in the absence or presence of DNA damage showed significant T68 phosphorylation already in the absence of DNA damaging reagentsUpon DNA damage, phosphorylation of additional Chk2 sites was observed (S19, S33/35)Transfection of HEK293 cells with Chk2 wildtype and Chk2 mutants in the absence or presence of DNA damage showed significant T68 phosphorylation already in the absence of DNA damaging reagentsUpon DNA damage, phosphorylation of additional Chk2 sites was observed (S19, S33/35)Despite distinct time-dependent autophosphorylation kinetics by monitoring the phosphorylation of amino acid residues T68, S19, S33/35, T432, in Chk2 wildtype and Chk2 mutants (T68A, T68D and Q69E) they gave identical specific activities\"}, {\"question\": \"Which are the methods for in silico prediction of the origin of replication (ori) among bacteria?\", \"ideal_answer\": [\"Several in silico methods have been applied for prediction of the origin of replication (ori). DNA base composition asymmetry, such as GC skew, is the basis of numerous in silico methods used to detect the ori in prokaryotes. The Z curve analysis is also used for ori identification. Comparative genomics, by BLAST analyses of the intergenic sequences compared to related species have been applied in ori prediction. The finding of the dnaA gene and its binding sites, DnaA boxes, as well as the finding of the binding sites of other proteins, such as CtrA and IHF, are fundamental characteristics used for in silico prediction of the ori. Also, the localization of boundary genes, such as cell division cycle (cdc6) gene, and consensus origin recognition box (ORB) sequences have been employed for ori detection. The study of the gene order around the origin sequence and the distribution of the genes encoded in the leading versus the lagging strand are also used for in silico detection of the ori.\"], \"exact_answer\": [[\"GC skew analysis\"], [\"Z curve analysis\"], [\"comparative genomics\"], [\"dnaA gene location\"], [\"localization of dnaA binding sites (DnaA boxes)\"], [\"localization of CtrA binding sites\"], [\"localization of IHF binding sites\"], [\"localization of cell division cycle (cdc6) gene\"], [\"localization of other boundary genes\"], [\"localization of other consensus origin recognition box (ORB) sequences\"], [\"the gene order around the ori\"], [\"gene distribution in leading versus lagging strand\"]], \"context\": \"Based on the Z-curve theory, we have developed a web-based system Ori-Finder to predict oriCs in bacterial genomes with high accuracy and reliability by taking advantage of comparative genomicswe have applied a combination of multiple in silico approaches - Z curve, the cell division cycle (cdc6) gene location and location of consensus origin recognition box (ORB) sequences for location of origin of replicationUsing features that have been previously experimentally verified in the alpha-Proteobacterium Caulobacter crescentus, the origin of DNA replication (ori) regions were identified in silico for Wolbachia strains and eleven other related bacteria belonging to Ehrlichia, Anaplasma, and Rickettsia genera. These features include DnaA-, CtrA- and IHF-binding sites as well as the flanking genes in C. crescentus.The sequences of the ori regions described here are only similar among closely related bacteria while fundamental characteristics like presence of DnaA and IHF binding sites as well as the boundary genes are more widely conserved.We propose a correlation measure and show that it is correctly able to predict the origin of replication in most of the bacterial genomes. When applied to Methanocaldococcus jannaschii, Plasmodium falciparum apicoplast and Nicotiana tabacum plastid, this correlation based method is able to correctly predict the origin of replication whereas the generally used GC skew measure fails.DNA base composition asymmetry is the basis of numerous in silico methods used to detect the origin and terminus of replication in prokaryotes.These analyses include finding the dnaA gene and its binding sites; making BLAST analyses of the intergenic sequences compared to related species; studying the gene order around the origin sequence; and studying the distribution of the genes encoded in the leading versus the lagging strand.DnaA binds a single DnaA box as a monomerIn silico comparative analysis of sequenced mollicute genomes indicated the lack of conservation of gene order in the region containing the predicted origin of replication (oriC) and the existence, in most of the mollicute genomes examined, of putative DnaA boxes lying upstream and downstream from the dnaA gene.In this paper we have applied a combination of multiple in silico approaches - Z curve, the cell division cycle (cdc6) gene location and location of consensus origin recognition box (ORB) sequences for location of origin of replication in Thermococcus onnurineus, Thermococcus gammatolerans and other Themococcales and compared the results to that of the well-documented case of Pyrococcus abyssi.DNA base composition asymmetry is the basis of numerous in silico methods used to detect the origin and terminus of replication in prokaryotes.It increases the prediction accuracy of the replication termini compared with previously documented methods based on genomic base composition.Using compositional skews such as the GC skew, replication origin and terminus can be predicted in silico by observing the shift points.Here, we summarize the recent advances of in silico prediction of oriCs in bacterial and archaeal genomes using the Z-curve based methodDNA base composition asymmetry is the basis of numerous in silico methods used to detect the origin and terminus of replication in prokaryotesRecent Advances in the Identification of Replication Origins Based on the Z-curve Method.We present a web-based tool Ori-Finder 2 to predict oriCs in the archaeal genomes automatically, based on the integrated method comprising the analysis of base composition asymmetry using the Z-curve method, the distribution of origin recognition boxes identified by FIMO tool, and the occurrence of genes frequently close to oriCs\"}, {\"question\": \"Which diseases have been associated with the PTPN22 620W allele?\", \"ideal_answer\": [\"The functional polymorphism 620W in the intracellular tyrosine phosphatase PTPN22 gene has been shown to confer susceptibility to the development of type 1 diabetes, seropositive rheumatoid arthritis, systemic lupus erythematosus, Hashimoto thyroiditis, and Wegener's granulomatosis (granulomatosis with polyangiitis).\"], \"exact_answer\": [[\"type 1 diabetes\"], [\"seropositive rheumatoid arthritis\"], [\"systemic lupus erythematosus\"], [\"Hashimoto thyroiditis\"], [\"Wegener's granulomatosis (granulomatosis with polyangiitis)\"]], \"context\": \"The frequency of the minor allele (620W) was significantly higher in GPA patients than in controls [P\\u2009=\\u20090.005, \\u03c7(2\\u2009)=\\u20097.858, odds ratio (OR)\\u2009=\\u20091.91], while no statistically significant association was found with MPA or CSS. Among GPA patients, the 620W allele was particularly enriched in ANCA-positive patients as compared with controls (P\\u2009=\\u20090.00012, \\u03c7(2\\u2009)=\\ufffd\\ufffd\\ufffd14.73, OR\\u2009=\\u20092.31)The PTPN22 620W allele confers susceptibility to the development of GPA (but not of MPA or CSS), and particularly of its ANCA-positive subsetThe PTPN22 620W allele is a risk factor for Wegener's granulomatosisAnalyses of families with multiple autoimmune disorders have revealed a functional polymorphism, 620W, in the intracellular tyrosine phosphatase gene PTPN22 as a predisposing factor for type 1 diabetes, seropositive rheumatoid arthritis, systemic lupus erythematosus, and Hashimoto thyroiditis, and the presence of the PTPN22 protein appears to herald the development of autoantibodies in these disordersThe PTPN22 620W allele frequency was significantly increased in antineutrophil cytoplasmic antibody (ANCA)-positive WG patients compared with healthy controlsThe PTPN22 620W allele appears to be involved in the pathogenesis of WG, and ANCA positivity seems to be the hallmarkRheumatoid arthritis seropositive for the rheumatoid factor is linked to the protein tyrosine phosphatase nonreceptor 22-620W alleleThe protein tyrosine phosphatase nonreceptor type 22 (PTPN22) gene encodes for lymphoid tyrosine phosphatase LYP, involved in the negative regulation of early T-cell activation. An association has recently been reported between the PTPN22-620W functional allele and rheumatoid factor-positive (RF+) rheumatoid arthritis (RA), among other autoimmune diseasesA functional variant (R620W) of the intracellular protein tyrosine phosphatase N22 (PTPN22) has now been conclusively shown to confer approximately two-fold risk for seropositive RA as well as several other autoimmune disordersthe PTPN22 620W allele associates with multiple autoimmune phenotypesWe report that a recently described functional single-nucleotide polymorphism (rs2476601, encoding R620W) in the intracellular tyrosine phosphatase (PTPN22) confers risk of four separate autoimmune phenotypes in these families: T1D, RA, SLE, and Hashimoto thyroiditisThe 620W allele of PTPN22 has been associated with susceptibility to several different forms of chronic inflammatory disease, including Type 1 diabetes (T1D), rheumatoid arthritis (RA), systemic lupus erythematosus (SLE), and autoimmune thyroiditis (AIT).The risk of developing RA later in life in patients with type 1 DM may be attributed, in part, to the presence of the 620W PTPN22 allele, suggesting that this risk factor may represent a common pathway for the pathogenesis of these 2 diseases.The minor allele of the R620W missense single-nucleotide polymorphism (SNP; rs2476601) in the PTPN22 (protein tyrosine phosphatase non-receptor 22) gene has been reported to be associated with multiple autoimmune diseases, including type 1 diabetes, systemic lupus erythematosus, rheumatoid arthritis, juvenile idiopathic arthritis, autoimmune thyroiditis and vitiligo.An association has recently been reported between the PTPN22-620W functional allele and rheumatoid factor-positive (RF+) rheumatoid arthritis (RA), among other autoimmune diseasesThe minor allele of the R620W missense single-nucleotide polymorphism (SNP; rs2476601) in the PTPN22 (protein tyrosine phosphatase non-receptor 22) gene has been reported to be associated with multiple autoimmune diseases, including type 1 diabetes, systemic lupus erythematosus, rheumatoid arthritis, juvenile idiopathic arthritis, autoimmune thyroiditis and vitiligoThe 620W allele of PTPN22 has been associated with susceptibility to several different forms of chronic inflammatory disease, including Type 1 diabetes (T1D), rheumatoid arthritis (RA), systemic lupus erythematosus (SLE), and autoimmune thyroiditis (AIT)The 620W allele of PTPN22 has been associated with susceptibility to several different forms of chronic inflammatory disease, including Type 1 diabetes (T1D), rheumatoid arthritis (RA), systemic lupus erythematosus (SLE), and autoimmune thyroiditis (AIT)PTPN22 620W risk allele was associated with ACPA production [odds ratio (OR)\\u2009=\\u20092.21, 95% CI 1.4, 3.4, P\\u2009<\\u20090.0001]An association has recently been reported between the PTPN22-620W functional allele and rheumatoid factor-positive (RF+) rheumatoid arthritis (RA), among other autoimmune diseases\"}, {\"question\": \"Which extra thyroid tissues have thyrotropin (TSH) receptors?\", \"ideal_answer\": [\"TSH receptors are expressed also in extrathyroid tissues. TSH receptors seem to be functional. Extrathyroid tissues include fibrobasts of the orbit and adipose tissue\\nThe principal tissues with TSH receptors are:\\nadippose tissue\\n orbital fibrotic tissue\"], \"exact_answer\": [[\"adipose tissue\"], [\"fibrotic tissue\"]], \"context\": \"GD orbital fibroblasts, which comprise a mixture of CD34(+) and CD34(-) cells, express much lower levels of Tg and TSHRPreviously, we found that CD34(+) progenitor cells, known as fibrocytes, express functional TSHR, infiltrate the orbit, and comprise a large subset of orbital fibroblasts in TAO. TSH induced lipolysis in adipose tissues.TSH worked as a lipolytic factor in white adipose tissues,These fibrocytes infiltrate orbital connective tissues in thyroid-associated ophthalmopathy and express functional TSH receptor (TSHR). TSHR levels are higher than those in orbital fibroblasts.\"}, {\"question\": \"Which histone modifications distinguish between promoters and enhancers?\", \"ideal_answer\": [\"H3K27ac is a marker of active enhancers. An enhancer chromatin state signature associated with active developmental enhancers may be defined by high levels of H3K27ac marking, nucleosome displacement, hypersensitivity to sonication, and strong depletion of H3K27me3.\"], \"exact_answer\": [[\"H3K27ac enrichment\"], [\"H3K27me3 depletion\"]], \"context\": \"Here, we used chromatin immunoprecipitation followed by high-throughput sequencing (ChIP-seq) to measure genome-wide changes in histone H3 acetylation at lysine 27 (H3K27ac), a marker of active enhancers, in unstimulated HUVECs and HUVECs stimulated with VEGFA for 1, 4, and 12 h.Here we use whole-transcriptome analysis coupled with genome-wide profiling of H3K27ac and H3K27me3 to map chromatin states and enhancers in mouse embryonic forelimb and hindlimb.Using H3K27ac profiles, we identified 28,377 putative enhancers, many of which are likely to be limb specific based on strong enrichment near genes highly expressed in the limb and comparisons with tissue-specific EP300 sites and known enhancers. a chromatin state signature associated with active developmental enhancers, defined by high levels of H3K27ac marking, nucleosome displacement, hypersensitivity to sonication, and strong depletion of H3K27me3We also find that some developmental enhancers exhibit components of this signature, including hypersensitivity, H3K27ac enrichment, and H3K27me3 depletion, at lower levels in tissues in which they are not active.At some locations, H3K27 monomethylation was also found to be associated with chromatin signatures of gene enhancers.Individual chromatin marks, such as H3K27ac and H3K27me3, have been identified to distinguish active from inactive enhancers. We find that histone H3K27ac distinguishes active enhancers from inactive/poised enhancer elements containing H3K4me1 alone. Although certain chromatin features, such as the histone acetyltransferase P300 and the histone modification H3K4me1, indicate the presence of enhancers, only a fraction of enhancers are functionally active. Individual chromatin marks, such as H3K27ac and H3K27me3, have been identified to distinguish active from inactive enhancers.We find that histone H3K27ac distinguishes active enhancers from inactive/poised enhancer elements containing H3K4me1 alone.\"}, {\"question\": \"What are the major classes of retrotransposons active in the human genome?\", \"ideal_answer\": [\"LINE-1 (L1), Alu, SVA\"], \"exact_answer\": [[\"LINE-1\", \"L1\"], [\"Alu\"], [\"SVA\"]], \"context\": \"Half of the human genome is composed of repeated DNA, and some types are mobile within our genome (transposons and retrotransposons). Despite their abundance, only a small fraction of them are currently active in our genome (Long Interspersed Element-1 (LINE-1), Alu, and SVA elements).Since certain types of retrotransposons, particularly members of the Alu, L1, and SVA families, are still active, their recent and ongoing propagation generates a unique and important class of human genomic diversity/polymorphismAlu elements are the most successful SINEs (Short INterspersed Elements) in primate genomes and have reached more than 1,000,000 copies in the human genome1 and Alu represent the most prolific human LINE and SINE families, respectively.Only a few Alu elements are able to retropose, and the factors determining their retroposition capacity are poorly understood.\"}, {\"question\": \"Which are the clinical characteristics of Diamond-Blackfan anemia?\", \"ideal_answer\": [\"Diamond-Blackfan anemia (DBA) is a rare congenital erythroid hypoplastic anemia that usually presents early in infancy and is characterized by red cell aplasia, congenital anomalies, and a predisposition to cancer.\"], \"exact_answer\": [[\"Red cell aplasia\"], [\"Congenital anomalies\"], [\"Predisposition to cancer\"]], \"context\": \"The results indicated that out of 45 children diagnosed as DBA, 14 cases (31.1%) had short stature and physical malformation. All patients had anemia with reticulocytopenia. Thirty-four patients (75.6%) had mean corpuscular volume. Eleven patients (24.4%) had macrocytic anemia. Bone marrow examination showed a marked erythroid hypoplasia in all patientsDiamond-Blackfan anemia (DBA) is a rare congenital erythroid hypoplastic anemia that usually presents early in infancy and is inherited in up to 45% of cases. It is characterized by red cell aplasia, congenital anomalies, and a predisposition to cancerDiamond-Blackfan Anemia (DBA) is characterized by a defect of erythroid progenitors and, clinically, by anemia and malformations.Diamond Blackfan Anemia (DBA) is a rare hypoplastic anemia that presents in infancy with macrocytic anemia and reticulocytopenia.Diamond Blackfan anemia is characterized by a severe hypoplastic anemia and a heterogeneous collection of other clinical features.Diamond-Blackfan anaemia (congenital hypoplastic anaemia) is a rare hereditary disease with isolated congenital hypoplasia of red blood cells precursors in bone marrow, and its important characteristic is successful treatmentDiamond-Blackfan anemia is a rare, inherited disease that characteristically presents as a chronic, normochromic macrocytosis due to red cell lineage bone marrow failureDiamond Blackfan anemia is characterized by a severe hypoplastic anemia and a heterogeneous collection of other clinical features\"}, {\"question\": \"Which factors play a role in promoter proximal pausing of RNA polymerase II?\", \"ideal_answer\": [\"NELF (negative elongator factor) and DSIF (DRB Sensitivity Inducing Factor)\"], \"exact_answer\": [[\"NELF\"], [\"DSIF\"]], \"context\": \"Among the factors we describe are the pausing factors--NELF (negative elongation factor) and DSIF (DRB sensitivity-inducing factor)BMP target gene expression requires the pause-inducing negative elongation factor (NELF) complexNA polymerase II (Pol II) and the pausing complex, NELF and DSIF, are detected near the transcription start site (TSS) of many active and silent genespausing complex NELF/DSIFknockdown of the pause-inducing factor NELF leads to broadly attenuated immune gene activationpausing factors NELF and DSIFpausing factors NELF and DSIFpausing factor DSIF (DRB sensitivity-inducing factor)(NELF) and 5,6-dichloro-1-beta-D-ribofuranosylbenzimidazole sensitivity-inducing factor (DSIF) are involved in pausing RNA Polymerase II (Pol II) in the promoter-proximal regionPausing and resumption of the elongation of transcripts is under the control of transcription elongation factors. Three of them, P-TEFb, DSIF, and NELF have been well characterized NELF-mediated stalling of RNAPII At several eukaryotic promoters, DSIF, together with NELF (negative elongation factor), leads to promoter-proximal pausing of RNA polymerase IInegative transcription elongation factor NELF(DSIF), is involved in regulating the processivity of RNA polymerase II. DSIF plays also a role in transcriptional activation, and in concert with the negative elongation factor NELF causes promoter proximal pausing of RNA polymerase IINELF causes Pol II to pause in the promoter-proximal region(DSIF) regulates RNA polymerase II (RNAPII) processivity by promoting, in concert with negative elongation factor (NELF), promoter-proximal pausing of RNAPI(DSIF) and negative elongation factor (NELF) negatively regulate transcription elongation by RNA polymerase II (RNAPII) in vitro.pausing caused by NELF ELF and DSIF collaborate to inhibit elongation by RNA polymerase IIaThe presence of DSIF reduced pausing,\"}, {\"question\": \"Which are the main clinical features of Fanconi anemia?\", \"ideal_answer\": [\"Fanconi anaemia (FA) is an autosomal recessive disease characterised by congenital abnormalities, defective haemopoiesis, and increased risk of malignancies.\", \"Fanconi anemia (FA) is a genetically and phenotypically heterogeneous recessive disorder characterized by diverse congenital malformations, progressive pancytopenia and predisposition to both hematologic malignancies and solid tumors\"], \"exact_answer\": [[\"congenital abnormalities\"], [\"defective haemopoiesis\"], [\"increased risk of malignancies\"], [\"cellular hypersensitivity to DNA crosslinking agents\"]], \"context\": \"The clinical features of cytopenia, developmental defects, and tumor predisposition are similar in each group, suggesting that the gene products participate in a common pathway.Fanconi anemia (FA) is a genetically and phenotypically heterogeneous recessive disorder characterized by diverse congenital malformations, progressive pancytopenia and predisposition to both hematologic malignancies and solid tumorsPatients with FA exhibit a heterogeneous spectrum of clinical features. The most significant and consistent phenotypic characteristics are stem cell loss, causing progressive bone marrow failure and sterility, diverse developmental abnormalities and a profound predisposition to neoplasiaFanconi anemia (FA) is an autosomal recessive, cancer susceptibility disorder characterized by diverse clinical features, such as short stature, skeletal or skin abnormalities, progressive bone marrow (BM) failure, and increased risk of malignanciesFanconi anaemia (FA) is an autosomal recessive disease characterised by congenital abnormalities, defective haemopoiesis, and a high risk of developing acute myeloid leukaemia and certain solid tumoursFanconi anemia (FA), a recessive syndrome with both autosomal and X-linked inheritance, features diverse clinical symptoms, such as progressive bone marrow failure, hypersensitivity to DNA cross-linking agents, chromosomal instability and susceptibility to cancer.Fanconi anemia (FA) consists of a group of at least five autosomal recessive disorders that share both clinical (e.g., birth defects and hematopoietic failure) and cellular (e.g., sensitivity to cross-linking agents and predisposition to apoptosis) features with each other.Features of chromosomal aberrations, hypersensitivity to DNA crosslinking agents, and predisposition to malignancy have suggested a fundamental anomaly of DNA repair in Fanconi anemia.Progressive bone marrow failure starting in the first decade of life is one of the main characteristics of Fanconi anemiaFA is an autosomal recessive disease with three main features: chromosome instability, hypersensitivity to DNA cross-linking agents such as mitomycin C (MMC), cisplatin and so on, and susceptible to a number of cancer types, mainly leukemia and squamous cell carcinomas of the head and neck or gynecologic systemBone marrow failure resulting in pancytopenia is the main cause of death of FA patients\"}, {\"question\": \"Which drugs may interfere thyroxine absorption?\", \"ideal_answer\": [\"bile acid sequestrants, ferrous sulphate, sucralfate, calcium carbonate, aluminium-containing antacids, phosphate binders, raloxifene and proton-pump inhibitors, have also been shown to interfere with the absorption of levothyroxine\\nsevelamer hydrochloride or chromium picolinate should be advised to separate the time of ingestion of these drugs from their thyroid hormone preparation by several hours\"], \"exact_answer\": [[\"bile acide sequestrant\"], [\"ferrous sulphate\"], [\"sucralfate\"], [\"Calcium carbonate\"], [\"aluminium-containing antacids\"], [\"raloxifene\"], [\"proton pump inhibithors\"], [\"sevelamer\"], [\"chromium picolinate\"]], \"context\": \"ommonly used drugs, such as bile acid sequestrants, ferrous sulphate, sucralfate, calcium carbonate, aluminium-containing antacids, phosphate binders, raloxifene and proton-pump inhibitors, have also been shown to interfere with the absorption of levothyroxineHypothyroid patients taking sevelamer hydrochloride or chromium picolinate should be advised to separate the time of ingestion of these drugs from their thyroid hormone preparation by several hours.\"}, {\"question\": \"In which types of DNA repair is the UvrAB complex involved?\", \"ideal_answer\": [\"UvrB and the lesion-recognition factor UvrA form the UvrAB complex, which plays a key role in bacterial nucleotide excision repair (NER). In transcription-coupled repair (TCR), the transcription repair coupling factor Mfd recruits uvrA, and the assembled UvrAB complex initiates repair. UvrAB complex also suppresses illegitimate recombination.\"], \"exact_answer\": [[\"nucleotide excision repair (NER)\"], [\"transcription-coupled repair (TCR)\"], [\"suppression of illegitimate recombination\"]], \"context\": \"illegitimate recombination is mostly suppressed by UvrA and UvrB.UvrAB complex suppresses illegitimate recombinationTranscription-coupled repair (TCR) is a subpathway of nucleotide excision repair (NER) that acts specifically on lesions in the transcribed strand of expressed genes.the transcription repair coupling factor, Mfd. This protein recruits the NER lesion-recognition factor UvrA, and then dissociates from the DNA. UvrA binds UvrB, and the assembled UvrAB* complex initiates repair.UvrB plays a key role in bacterial nucleotide excision repair. It is the ultimate damage-binding protein that interacts with both UvrA and UvrC.UvrB subunits bind to UvrA, most likely as part of a UvrA2B2 complexEscherichia coli nucleotide excision repair (NER) is responsible for removing bulky DNA adducts by dual incisions of the UvrABC endonuclease.the activity of the UvrAB complex which can induce DNA conformational change is employed in NERThe recognition by Escherichia coli Uvr nucleotide excision repair proteins of a variety of lesions with diverse chemical structures and the presence of helicase activity in the UvrAB complex which can displace short oligonucleotides annealed to single-stranded DNA led to a model in which this activity moves UvrAB along undamaged DNA to damaged sites where the lesion blocks further translocation and the protein-DNA pre-incision complex is formed.Transcription when coupled to nucleotide excision repair specifies the location in active genes where preferential DNA repair is to take place.Transcription when coupled to nucleotide excision repair specifies the location in active genes where preferential DNA repair is to take placeThe recognition by Escherichia coli Uvr nucleotide excision repair proteins of a variety of lesions with diverse chemical structures and the presence of helicase activity in the UvrAB complex which can displace short oligonucleotides annealed to single-stranded DNA led to a model in which this activity moves UvrAB along undamaged DNA to damaged sites where the lesion blocks further translocation and the protein-DNA pre-incision complex is formedTranscription-coupled DNA repair in prokaryotes.\"}, {\"question\": \"List programs suitable for protein docking\", \"ideal_answer\": [\"Macromolecular docking is the computational modelling of the quaternary structure of complexes formed by two or more interacting biological macromolecules. Protein\\u2013protein complexes are the most commonly attempted targets of such modelling, followed by protein\\u2013nucleic acid complexes.\\nThe ultimate goal of docking is the prediction of the three-dimensional structure of the macromolecular complex of interest as it would occur in a living organism. Docking itself only produces plausible candidate structures. These candidates must be ranked using methods such as scoring functions to identify structures that are most likely to occur in nature.\\nNowadays there a lot of programs suitable for proteins docking  such as CSBB-ConeExclusion, HADDOCK, ZDOCK, GalaxyDock, PHASE, DockRank, HotLig, SOL, AutodockVina, DockoMatic, DockoMatic, DockTrina, CAVITY, LiGenDock and DOCK.\"], \"exact_answer\": [[\"CSBB-ConeExclusion\"], [\"HADDOCK\"], [\"ZDOCK\"], [\"GalaxyDock\"], [\"PHASE\"], [\"DockRank\"], [\"HotLig\"], [\"SOL\"], [\"AutodockVina\"], [\"DockoMatic\"], [\"DockTrina\"], [\"CAVITY\"], [\"LiGenDock\"], [\"DOCK\"], [\"DISCORE\"]], \"context\": \"We present CSBB-ConeExclusion, a methodology and computer program which provides a measure of the applicability of solution dockings to solid supportThe structure of the \\u03b11I-peptide complex was investigated using data from NMR, small angle x-ray scattering, and size exclusion chromatography that were used to generate and validate a model of the complex using the data-driven docking program, HADDOCK (High Ambiguity Driven Biomolecular Docking). We report the performance of our approaches for protein-protein docking and interface analysis in CAPRI rounds 20-26. At the core of our pipeline was the ZDOCK program for rigid-body protein-protein dockingGalaxyDock protein-ligand docking program is introduced. GalaxyDock performs conformational space annealing (CSA) global optimization to find the optimal binding pose of a ligand both in the rigid-receptor mode and the flexible-receptor modeUtilizing NMR titration data, we generated the structural models of S100B-FGF2 complex from the computational docking program, HADDOCK which were further proved stable during 15ns unrestrained molecular dynamics (MD) simulationsThereafter, all molecules were docked into the newly generated active site environment of the selected protein using glide docking program, and the 3D-QSAR analysis was performed in PHASE program utilizing the docking based alignment of the moleculesDockRank uses interface residues predicted by partner-specific sequence homology-based protein-protein interface predictor (PS-HomPPI), which predicts the interface residues of a query protein with a specific interaction partner. We compared the performance of DockRank with several state-of-the-art docking scoring functions using Success Rate (the percentage of cases that have at least one near-native conformation among the top m conformations) and Hit Rate (the percentage of near-native conformations that are included among the top m conformations). In this study, we developed a novel scoring program, HotLig, which applies the Connolly surface of a protein to calculate hydrophobic interaction and paired pharmacophore interactions with ligands. In addition to molecular surface distance, ligand-contacting areas and hydrogen-bond angles were also introduced to the scoring functions in HotLiga method called residue contact frequency (RCF), which uses the complex structures generated by the protein-protein docking algorithm ZDOCK to predict interface residuesdevoted to results obtained by the docking program SOL and the post-processing program DISCORE at the CSAR benchmark. SOL and DISCORE programs are described. SOL is the original docking program developed on the basis of the genetic algorithm, MMFF94 force field, rigid protein, precalculated energy grid including desolvation in the frame of simplified GB model, vdW, and electrostatic interactions and taking into account the ligand internal strain energyimproves the binding energy scoring by the local energy optimization of the ligand docked pose and a simple linear regression on the base of available experimental dataThe template-based methods showed similar performance to a docking method (ZDOCK) when the latter was allowed one prediction for each complex, but when the same number of predictions was allowed for each method, the docking approach outperformed template-based approachesVinaMPI is a massively parallel Message Passing Interface (MPI) program based on the multithreaded virtual docking program AutodockVina, and is used to distribute tasks while multithreading is used to speed-up individual docking tasksPrime and the binding energy function in YASARA suggested it could be possible to evaluate the quality of the orthosteric binding site based on the prediction of relative binding energies is a free and open source application that unifies a suite of software programs within a user-friendly graphical user interface (GUI) to facilitate molecular docking experimentsThis article presents DockTrina, a novel protein docking method for modeling the 3D structures of nonsymmetrical triangular trimersTo account for the important docking interactions between the UBSAs ligand and hCA II enzyme, a molecular docking program AutoDock Vina is used. The molecular docking results obtained by AutoDock Vina revealed that the docked conformer has root mean square deviation value less than 1.50 \\u00c5 compared to X-ray crystal structures.Four possible binding pockets (Pocket A, B, C, and D) at the stalk region of hemagglutinin were detected and defined using the CAVITY programWithin this context, low-resolution shape data obtained from either ion-mobility mass spectrometry (IM-MS) or SAXS experiments have been integrated into the conventional scoring function of the information-driven docking program HADDOCKa novel de novo design program, called LiGen, we developed a docking program, LiGenDock, based on pharmacophore models of binding sites, including a non-enumerative docking algorithm.we present the functionalities of LiGenDock and its accompanying module LiGenPocket, aimed at the binding site analysis and structure-based pharmacophore definitionHere we describe the application of the program AutoDock to the design of a focused library that was used in the \\\"click chemistry in-situ\\\" generation of the most potent noncovalent inhibitor of the native enzyme acetylcholinesterase (AChE) yet developed (K(d) = ~100 fM).  the method yields improved success over the standard DOCK energy function for pose identification across a large test set of experimental co-crystal structures, for crossdocking, and for database enrichmentThe pose prediction success rate of each docking program alone was found in this trial to be 55% for Autodock, 58% for DOCK, and 64% for Vina.\"}, {\"question\": \"Which genes have been found mutated in Gray platelet syndrome patients?\", \"ideal_answer\": [\"The genetic defects responsible for gray platelet syndrome are mutations in the genes NBEAL2, GATA1 and GFI1B.\", \"The genetic defect responsible for gray platelet syndrome was recently identified in biallelic mutations in the NBEAL2 gene.\\nA nonsense mutation in the gene encoding the transcription factor GFI1B (growth factor independent 1B) that causes autosomal dominant gray platelet syndrome.\\nX-linked gray platelet syndrome due to a GATA1 Arg216Gln mutation.\"], \"exact_answer\": [[\"neurobeachin-like 2\", \"NBEAL2\"], [\"GATA1\"], [\"GFI1B\"]], \"context\": \"We detected a nonsense mutation in the gene encoding the transcription factor GFI1B (growth factor independent 1B) that causes autosomal dominant gray platelet syndrome. The genetic defect responsible for gray platelet syndrome was recently identified in biallelic mutations in the NBEAL2 gene.Here we show that mutations in NBEAL2 (neurobeachin-like 2), which encodes a BEACH/ARM/WD40 domain protein, cause GPS and that megakaryocytes and platelets from individuals with GPS express a unique combination of NBEAL2 transcripts. Linkage analysis revealed a 63 cM region on the X chromosome between markers G10578 and DXS6797, which segregated with the platelet phenotype and included the GATA1 gene. Sequencing of GATA1 revealed a G-to-A mutation at position 759 corresponding to amino acid change Arg216Gln. We identified a family with gray platelet syndrome (GPS) segregating as a sex-linked trait. X-linked gray platelet syndrome due to a GATA1 Arg216Gln mutation\"}, {\"question\": \"Which mutations in the cardiac isoform of the ryanodine receptor (RyR2) have been found to be related to CPVT?\", \"ideal_answer\": [\"Recently, a novel CPVT RyR2 mutation, G230C, was found to increase the cytosolic, but not the luminal, Ca2+ sensitivity of single RyR2 channels in lipid bilayers. The novel RYR2-S4153R mutation has been implicated as a cause of CPVT and atrial fibrillation. A novel RyR2-V2475F mutation is associated with CPVT in humans. 3 CPVT mouse models are: RyR2-R2474S+/-, RyR2-N2386I+/-, and RyR2-L433P+/-. The E189D RyR2 mutation is causative for CPVT. A knock-in mouse model carrier of the R4496C mutation is the mouse equivalent to the R4497C mutations identified in CPVT families. Scanning of 12 Finnish CPVT probands identified three novel RYR2 mutations (V2306I, P4902L, R4959Q), which were absent in unaffected and control individuals. Three CPVT-linked human RyR2 (hRyR2) mutations are: S2246L, N4104K, and R4497C.\"], \"exact_answer\": [[\"RyR2-G230C\"], [\"RyR2-S4153R\"], [\"RyR2-V2475F\"], [\"RyR2-R2474S\"], [\"RyR2-N2386I\"], [\"RyR2-L433P\"], [\"RyR2-E189D\"], [\"RyR2-R4497C (RyR2-R4496C in the mouse)\"], [\"RyR2-V2306I,\"], [\"RyR2-P4902L\"], [\"RyR2-R4959Q\"], [\"RyR2-S2246L\"], [\"RyR2-N4104K\"]], \"context\": \"Recently, a novel CPVT RyR2 mutation, G230C, was found to increase the cytosolic, but not the luminal, Ca2+ sensitivity of single RyR2 channels in lipid bilayers.These data suggest that the G230C mutation enhances the propensity for SOICR by sensitizing the channel to luminal and cytosolic Ca2+ activation, and that G230C has an intrinsic structural impact on the N-terminal domains of RyR2.The novel RYR2-S4153R mutation has been implicated as a cause of CPVT and atrial fibrillation. The mutation has been functionally characterized via store-overload-induced Ca(2+) release (SOICR) and tritium-labelled ryanodine ([(3)H]ryanodine) binding assays. The S4153R mutation enhanced propensity for spontaneous Ca(2+) release and reduced SOICR threshold but did not alter Ca(2+) activation of [(3)H]ryanodine binding, a common feature of other CPVT gain-of-function RYR2 mutations. We conclude that the S4153R mutation is a gain-of-function RYR2 mutation associated with a clinical phenotype characterized by both CPVT and atrial fibrillation.To determine the molecular and cellular mechanisms by which a novel RyR2-V2475F mutation associated with CPVT in humans triggers Ca(2+)-dependent arrhythmias in whole hearts and intact mice.The mutation RyR2-V2475F is phenotypically strong among other CPVT mutations and produces heterogeneous mechanisms of RyR2 dysfunction. We generated 3 knock-in mice with mutations introduced into RyR2 that result in leaky channels and cause exercise induced polymorphic ventricular tachycardia in humans [catecholaminergic polymorphic ventricular tachycardia (CPVT)]. We examined AF susceptibility in these three CPVT mouse models harboring RyR2 mutations to explore the role of diastolic SR Ca2+ leak in AF. AF was stimulated with an intra-esophageal burst pacing protocol in the 3 CPVT mouse models (RyR2-R2474S+/-, 70%; RyR2-N2386I+/-, 60%; RyR2-L433P+/-, 35.71%) but not in wild-type (WT) mice (P<0.05).The RyR2-G230C mutant exhibits similar biophysical defects compared with previously characterized CPVT mutations: decreased binding of the stabilizing subunit calstabin2 and a leftward shift in the Ca(2+) dependence for activation under conditions that simulate exercise, consistent with a \\\"leaky\\\" channel. Both RyR2-G230C and RyR2-P2328S channels exhibit normal luminal Ca(2+) activation. Thus, diastolic sarcoplasmic reticulum Ca(2+) leak caused by reduced calstabin2 binding and a leftward shift in the Ca(2+) dependence for activation by diastolic levels of cytosolic Ca(2+) is a common mechanism underlying CPVT.A novel CPVT mutation, E189D, was identified. The E189D mutation does not alter the affinity of the channel for FKBP12.6, but it increases the propensity for store-overload-induced Ca(2+) release (SOICR). Furthermore, the E189D mutation enhances the basal channel activity of RyR2 and its sensitivity to activation by caffeine. The E189D RyR2 mutation is causative for CPVT and functionally increases the propensity for SOICR without altering the affinity for FKBP12.6.We found that mice heterozygous for the R2474S mutation in Ryr2 (Ryr2-R2474S mice) exhibited spontaneous generalized tonic-clonic seizures (which occurred in the absence of cardiac arrhythmias), exercise-induced ventricular arrhythmias, and sudden cardiac death.  Thus, CPVT-associated mutant leaky Ryr2-R2474S channels in the brain can cause seizures in mice, independent of cardiac arrhythmias.We screened 12 Finnish CPVT probands for mutations in these genes and identified three novel RYR2 mutations (V2306I, P4902L, R4959Q), which were absent in unaffected and control individuals.RyR2 mutations suggested to cause defective Ca2+ channel function have recently been identified in catecholaminergic polymorphic ventricular tachycardia (CPVT) and arrhythmogenic right ventricular dysplasia (ARVD) affected individuals. We report expression of three CPVT-linked human RyR2 (hRyR2) mutations (S2246L, N4104K, and R4497C) in HL-1 cardiomyocytes displaying correct targeting to the endoplasmic reticulum. We developed a conditional knock-in mouse model carrier of the R4496C mutation, the mouse equivalent to the R4497C mutations identified in CPVT families, to evaluate if the animals would develop a CPVT phenotype and if beta blockers would prevent arrhythmias.These data provide the first experimental demonstration that the R4496C RyR2 mutation predisposes the murine heart to VT and VF in response caffeine and/or adrenergic stimulation.We developed a conditional knock-in mouse model carrier of the R4496C mutation, the mouse equivalent to the R4497C mutations identified in CPVT families, to evaluate if the animals would develop a CPVT phenotype and if beta blockers would prevent arrhythmias\"}, {\"question\": \"Which are the most widely reported side-effects in the treatment of Crohn's disease?\", \"ideal_answer\": [\"Leukopenia, paresthesia, psoriasis, alopecia and hemolysis are the most commonly reported side effects depending on the treatment. Severe adverse effects include myelosuppression, liver toxicity and hyperplasia, pancreatitis and pericarditis. The most severe but rare side-effects reported are progressive multifocal leukoencephalopathy (PML), serious infections, and lymphoma.\"], \"exact_answer\": [[\"Leukopenia\"], [\"paresthesia\"], [\"psoriasis\"], [\"alopecia\"], [\"hemolysis\"], [\"pancreatitis\"], [\"liver toxicity\"], [\"pericarditis\"]], \"context\": \"Different cutaneous side effects have been described for anti-TNF-\\u03b1 therapy such as psoriasisAnti-TNF drug-induced alopecia is a less well-known side effect of this class of drugs.3 patients who developed scalp alopeciaPsoriasis and psoriatic arthritis induced in a patient treated with infliximab for Crohn's diseaseside effects appearing in about a fifth of the patients, including myelosuppression and liver toxicityPersistent corneal endothelial deposits associated with rifabutin therapy for Crohn's diseasecomplained of alopecia 3 days after starting 6-mercaptopurine (6-MP) and then developed severe myelosuppressionLeucopoenia was the more frequent side effect observed, occurring in 36 (34%)Nausea, vomit, although slight, occurred in 29 (27.4%)Nodular regenerative hyperplasia as a side effect of azathioprine in a patient with Crohn's diseaseThe whole blood count revealed a pancytopenia, hyperbilirubinemia and slightly elevated transaminasesAnalysis of the liver histology was highly suggestive of an azathioprine-related, nodular regenerative hyperplasia3 serious adverse events (SAEs) known to be associated with CD treatment (progressive multifocal leukoencephalopathy (PML), serious infections, and lymphoma)Severe pulmonary toxicity after azathioprine/6-mercaptopurine initiation for the treatment of inflammatory bowel diseaseA severe side effect is acute pancreatitis, which is specific for Crohn's diseaseRecurrent myopericarditis in association with Crohn's diseaseInterstitial nephritis in patients with inflammatory bowel disease treated with mesalaminePancreatitis in a patient with Crohn disease treated with mesalazine and azathioprineLeukopenia and thrombocytopenia are observed mostly as a side effect of therapy, particularly with use of immunosuppressive drugsImmune thrombocytopenic purpura is rarely reported in association with inflammatory bowel disease.resulted to watery stools, vomiting, and feverEosionophilia was present and the lymphocyte stimulation test with mesalazine was positivePericarditis as a side effect induced by sulfasalazine or 5-aminosalicylic acidThe commonest side-effect was hyperaesthesia, but one patient developed nephrotoxicity and one developed hepatotoxicityThe only troublesome side effect is paresthesia, which apparently is dose-related and not progressiveThe only major side effect observed was paresthesias. These occurred in 50% of the patients and developed in the patients at a mean of 6.5 mo after the onset of treatment.Hemolysis is not a rare side-effect of sulfasalazine therapy17 of 40 (43%) patients with inflammatory bowel disease receiving sulfasalazine had evidence of hemolysis as detected by starch gel electrophoresis\"}, {\"question\": \"Which microRNAs are involved in exercise adaptation?\", \"ideal_answer\": [\"miR-1, miR-133, miR-208a, miR-206, miR-494, miR-146a, miR-222, miR-21, miR-221, miR-20a, miR-133a, miR-133b, miR-23, miR-107 and miR-181 are involved in exercise adaptation\"], \"exact_answer\": [[\"miR-1\"], [\"miR-133\"], [\"miR-208a\"], [\"miR-206\"], [\"miR-494\"], [\"miR-146a\"], [\"miR-222\"], [\"miR-21\"], [\"miR-221\"], [\"miR-20a\"], [\"miR-133a\"], [\"miR-133b\"], [\"miR-23\"], [\"miR-107\"], [\"miR-181\"]], \"context\": \"Some miRNAs as miR-1, miR-133 and miR-208a are highly expressed in the heart and strongly associated with the development of cardiac hypertrophy. Recent data indicate that these miRNAs as well as miR-206 change their expression quickly in response to physical activity.the miR-494 content significantly decreased after endurance exercise in C57BL/6J mice, accompanied by an increase in expression of mtTFA and Foxj3 proteins. These results suggest that miR-494 regulates mitochondrial biogenesis by downregulating mtTFA and Foxj3 during myocyte differentiation and skeletal muscle adaptation to physical exercise.1) c-miRNA up-regulated by acute exercise before and after sustained training (miR-146a and miR-222), (2) c-miRNA responsive to acute exercise before but not after sustained training (miR-21 and miR-221), (3) c-miRNA responsive only to sustained training (miR-20a),MicroRNA 1 expression was decreased independent of the training modality, and was paralleled by an increased expression of IGF-1 representing a potential target.we investigated the expression of these myomiRs, including miR-1, miR-133a, miR-133b and miR-206 in muscle biopsies from vastus lateralis of healthy young males (n = 10) in relation to a hyperinsulinaemic\\u2013euglycaemic clamp as well as acute endurance exercise before and after 12 weeks of endurance training.In resting biopsies, endurance training for 12 weeks decreased basal expression of all four myomiRs (P < 0.05)These results suggest that miR-494 regulates mitochondrial biogenesis by downregulating mtTFA and Foxj3 during myocyte differentiation and skeletal muscle adaptation to physical exercise.\"}, {\"question\": \"List inhibtors targeting the mitochondrial permeability transition pore.\", \"ideal_answer\": [\"Cyclosporine A\\nAtractyloside\\nN-metyl-4-isoleucine-cyclosporine\\nSanglifehrin A \\nTRO-19622\", \"The opening of the mitochondrial permeability transition pore appears to be inhibited by KB-R 7943, diacylglycerol analog 1-oleoyl-2-acetyl-sn-glycerol, cyclosporin A (CsA) and BRL37344.\"], \"exact_answer\": [[\"Cyclosporine A\"], [\"Atractyloside\"], [\"N-metyl-4-isoleucine-cyclosporine\", \"NIM811\"], [\"Sanglifehrin A\"], [\"TRO-19622\"]], \"context\": \"Postconditioning with cyclosporine a reduces early renal dysfunction by inhibiting mitochondrial permeability transition.mitochondrial permeability transition pore inhibitor (Cyclosporine A). treatment with the MPTP inhibitor atractyloside cyclosporine A, an MPTP inhibitor,NIM-811 (N-metyl-4-isoleucine-cyclosporine) is a mitochondria specific drug, which can prevent ischemic-reperfusion (IR) injury, by inhibiting mitochondrial permeability transition pores (mPTP). The mPTP inhibitor sanglifehrin A (SfA) mPTP inhibitor, cyclosporin A (CsA),cyclosporine A (CsA), a MPTP inhibitor, The mPTP inhibitor NIM811,A mPTP inhibitor (TRO-19622)Cyclosporine A (CsA), a mitochondria permeability transition pore (MPTP) inhibitor,  and N-methyl-4-isoleucine cyclosporine (NIM811), an mPTP inhibitor, mPTP inhibitor cyclosporin A (CsA) ADP is not only a key substrate for ATP generation, but also a potent inhibitor of mitochondrial permeability transition pore (mPTP). MPTP inhibitor cyclosporin A mPTP inhibitor cyclosporin A MPTP inhibitor cyclosporine A (CsA) the mPTP inhibitor cyclosporine AThe mPTP inhibitor cyclosporine A (CsA)Rather, we find that KB-R7943 inhibits opening of the mitochondrial permeability transition pore in permeabilized cells and isolated liver mitochondria. Our data reveal another mechanism through which KB-R7943 may protect against calcium-induced injury, as well as a novel means to inhibit the mitochondrial permeability transition pore.KB-R7943, a plasma membrane Na(+)/Ca(2+) exchanger inhibitor, blocks opening of the mitochondrial permeability transition pore.Inhibitors of calcium/calmodulin-dependent protein kinase II, a mitochondrial Ca(2+) uniporter (MCU) regulator, also prevented MPTP formation and arachidonic acid release induced by A23187 and H2O2. Unlike pyrrophenone, the diacylglycerol analog 1-oleoyl-2-acetyl-sn-glycerol and CsA blocked cell death and arachidonic acid release not by preventing mitochondrial calcium uptake but by inhibiting MPTP formation.Incubation with \\u03b23AR agonist (BRL37344, 7\\u00a0\\u03bcmol/L) significantly reduced cell death in isolated adult mouse cardiomyocytes during hypoxia/reoxygenation and decreased susceptibility to deleterious opening of the mitochondrial permeability transition pore (mPTP), via a mechanism dependent on the Akt-NO signaling pathway. Here we show that antamanide inhibits the mitochondrial permeability transition pore, a central effector of cell death induction, by targeting the pore regulator cyclophilin D. Indeed, (i) permeability transition pore inhibition by antamanide is not additive with the cyclophilin D-binding drug cyclosporin A, (ii) the inhibitory action of antamanide on the pore requires phosphate, as previously shown for cyclosporin A; (iii) antamanide is ineffective in mitochondria or cells derived from cyclophilin D null animals, and (iv) abolishes CyP-D peptidyl-prolyl cis-trans isomerase activity.Furthermore, in wild-type (non-transfected) SHSY5Y cells, the effects of lactacystin on mitochondrial function and cell viability are also prevented by cyclosporin A (1\\u00a0\\u03bcM) which blocks the activity of the mitochondrial permeability transition poreExogenous zinc protects cardiac cells from reperfusion injury by targeting mitochondrial permeability transition pore through inactivation of glycogen synthase kinase-3beta.Bradykinin prevents reperfusion injury by targeting mitochondrial permeability transition pore through glycogen synthase kinase 3beta.HIF-1 reduces ischaemia-reperfusion injury in the heart by targeting the mitochondrial permeability transition pore.Pre\\u2014treatment of gallic acid and mitochondrially targeted gallic acid to sodium nitroprusside treated mitochondria not only significantly reduced the oxidative stress but also prevented mitochondrial permeability pore transition to a significant difference. Mitochondrially targeted gallic acid was found more effective in reducing oxidative stress and mitochondrial permeability pore transition than gallic acid.Mitochondrially targeted gallic acid was found more effective in reducing oxidative stress and mitochondrial permeability pore transition than gallic acid. We conclude that mitochondrially targeted gallic acid can be used for preventing mitochondrial impairment caused by oxidative stress.The involvement of mitochondrial permeability transition pore was determined with a mitochondrial permeability transition pore opener atractyloside and a specific mitochondrial permeability transition pore inhibitor cyclosporin A.Here we show that antamanide inhibits the mitochondrial permeability transition pore, a central effector of cell death induction, by targeting the pore regulator cyclophilin D.Mitochondrially targeted gallic acid was found more effective in reducing oxidative stress and mitochondrial permeability pore transition than gallic acid.Pre\\u2014treatment of gallic acid and mitochondrially targeted gallic acid to sodium nitroprusside treated mitochondria not only significantly reduced the oxidative stress but also prevented mitochondrial permeability pore transition to a significant difference.\"}, {\"question\": \"List bacterial species identified in the iceman tissues.\", \"ideal_answer\": [\"Spirochete Treponema denticola\\nClostridium perfringens\\nClostridium ghonii\\nClostridium sordellii\\nEubacterium tenue\\nBacteroides sp\\nVibrio\\nSphingomonas\\nAfipia\\nCurtobacterium\\nMicrobacterium\\nAgromyces\"], \"exact_answer\": [[\"Spirochete Treponema denticola\"], [\"Clostridium perfringens\"], [\"Clostridium ghonii\"], [\"Clostridium sordellii\"], [\"Eubacterium tenue\"], [\"Bacteroides sp\"], [\"Vibrio\"], [\"Sphingomonas\"], [\"Afipia\"], [\"Curtobacterium\"], [\"Microbacterium\"], [\"Agromyces\"]], \"context\": \"Here, we report the molecular detection of the oral spirochete Treponema denticola in ancient human tissue biopsies of the Iceman, a 5,300-year-old Copper Age natural ice mummy. The colon, on the other hand, contains several members of the fecal flora of humans, such as Clostridium perfringens, C. ghonii, C. sordellii, Eubacterium tenue, and Bacteroides sp. The Iceman's colon, however, was found to contain, rather unexpectedly, also some members of the genus Vibrio.  while the untreated skin still bears the remains of large numbers of bacteria belonging to the genera Sphingomonas, Afipia, Curtobacterium, Microbacterium, Agromyces, and others.\"}, {\"question\": \"List some ways to reverse Tau hyperphosphorylation in Tauopathies?\", \"ideal_answer\": [\"Different ways have been used to try to reverse Tau hyperphosphorylation through administration of inhibitors such as: 7-nitroindazole,  memantine, glycogen synthase kinase-3 inhibitors. \\nOther approaches are transplantation of Human umbilical cord blood-derived mesenchymal stem cells and administration of M1 muscarinic agonists.\"], \"exact_answer\": [[\"Human umbilical cord blood-derived mesenchymal stem cells (hUCB-MSC) transplantation\"], [\"Administration of 7-nitroindazole  (neuronal nitric oxide synthase inhibitor)\"], [\"Administration of Memantine  (un-competitive inhibitor of N-methyl-D-aspartate receptors)\"], [\"Administration glycogen synthase kinase-3  (GSK-3) inhibitors\"], [\"Administration of M1 muscarinic agonists such as AF150(S) and AF267B\"]], \"context\": \"tau hyperphosphorylation were dramatically reduced in hUCB-MSC transplanted APP/PS1 miceAdministration of 7-nitroindazole to CMS-exposed old rats significantly (p=0.002) increased GAD activity, decreased glutamate levels (7.19+/-3.19 vs. 763.9+/-91 micromol/g tissue protein; p=0.0005), and decreased phosphorylation of tau proteins compared to CMS exposed rats.Changes in glutamate decarboxylase enzyme activity and tau-protein phosphorylation in the hippocampus of old rats exposed to chronic mild stress: reversal with the neuronal nitric oxide synthase inhibitor 7-nitroindazole.Involvement of I2PP2A in the abnormal hyperphosphorylation of tau and its reversal by Memantine.Full reversal of Alzheimer's disease-like phenotype in a mouse model with conditional overexpression of glycogen synthase kinase-3.Here, we show that transgene shutdown in symptomatic mice leads to normal GSK-3 activity, normal phospho-tau levels, diminished neuronal death, and suppression of the cognitive deficit, thus further supporting the potential of GSK-3 inhibitors for AD therapeutics.These effects can be combined with the beneficial effects of these compounds on some other major hallmarks of Alzheimer's disease (AD) (e.g. tau hyperphosphorylation and paired helical filaments [PHF]; and loss of cholinergic function conducive to cognitive impairments.)The M1 muscarinic agonists AF102B (Cevimeline, EVOXACTM: prescribed in USA and Japan for Sjogren's Syndrome), AF150(S) and AF267B--1) are neurotrophic and synergistic with neurotrophins such as nerve growth factor and epidermal growth factor; 2) elevate the non-amyloidogenic amyloid precursor protein (alpha-APPs) in vitro and decrease beta-amyloid (A beta) levels in vitro and in vivo; and 3) inhibit A beta- and oxidative-stress-induced cell death and apoptosis in PC12 cells transfected with the M1 muscarinic receptor\"}, {\"question\": \"List components of the CRSP/Med complex.\", \"ideal_answer\": [\"Mediator of RNA polymerase II transcription subunit 7\\nMediator of RNA polymerase II transcription subunit 14\\nMediator of RNA polymerase II transcription subunit 17\\nMediator of RNA polymerase II transcription subunit 23\\nMediator of RNA polymerase II transcription subunit 24\\nMediator of RNA polymerase II transcription subunit 26\\nMediator of RNA polymerase II transcription subunit 27\"], \"exact_answer\": [[\"Mediator of RNA polymerase II transcription subunit 7\"], [\"Mediator of RNA polymerase II transcription subunit 14\"], [\"Mediator of RNA polymerase II transcription subunit 17\"], [\"Mediator of RNA polymerase II transcription subunit 23\"], [\"Mediator of RNA polymerase II transcription subunit 26\"], [\"Mediator of RNA polymerase II transcription subunit 27\"], [\"Mediator of RNA polymerase II transcription subunit 24\"]], \"context\": \" Here we describe a new human factor, CRSP, that is required together with the TAF(II)s for transcriptional activation by Sp1. Purification of CRSP identifies a complex of approximate relative molecular mass 700,000 (M(r) approximately 700K) that contains nine subunits with M(r) values ranging from 33K to 200K. Cloning of genes encoding CRSP subunits reveals that CRSP33 is a homologue of the yeast mediator subunit Med7, whereas CRSP150 contains a domain conserved in yeast mediator subunit Rgr1. CRSP p200 is identical to the nuclear hormone-receptor co-activator subunit TRIP2/PBP. CRSPs 34, 77 and 130 are new proteins, but the amino terminus of CRSP70 is homologous to elongation factor TFIIS.CRSP contains unique subunits as well as polypeptides that are shared with other cofactor complexes.The multi-subunit, human CRSP coactivator-also known as Mediator (Med)-regulates transcription by mediating signals between enhancer-bound factors (activators) and the core transcriptional machinery. Interestingly, different activators are known to bind distinct subunits within the CRSP/Med complex. We have isolated a stable, endogenous CRSP/Med complex (CRSP/Med2) that specifically lacks both the Med220 and the Med70 subunits. \"}, {\"question\": \"Which mitochondrial genes are regulated by thyroid hormone?\", \"ideal_answer\": [\"subunit 6 of ATP synthase, ATPase-6, mitochondrial II and III subunits of cytochrome-c oxidase, NADH dehydrogenase subunit 3\"], \"exact_answer\": [[\"subunit 6 of ATP synthase\", \"ATPase-6\"], [\"mitochondrial II and III subunits of cytochrome-c oxidase\"], [\"NADH dehydrogenase subunit 3\", \"ND3\"]], \"context\": \"We used a polymerase chain reaction (PCR)-based mRNA differential display (DD) analysis to obtain a profile of thyroid hormone-responsive genes in osteoblast-like cellsAt the 2-hour time point, 1 true-positive novel clone was identified and shown to be the mitochondrial gene, subunit 6 of ATP synthase (ATPase-6).We studied the coordination of the two genomes by measuring transcript levels for nuclear (IV, Va, and VIc) and mitochondrial (II and III) subunits of cytochrome-c oxidase after altering the mitochondrial content of rat muscle and liver by altering the thyroid state of the animals. Tissue levels of these mRNAs were generally decreased in hypothyroid animals and were up-regulated again after thyroid hormone (T3) treatment.Following this procedure, we now report the identification of the mitochondrial NADH dehydrogenase subunit 3 (ND3) gene as target of thyroid hormone.Sequencing and electrophoretic mobility shift assays confirmed the presence of a thyroid hormone receptor (TR)/c-erbA specific binding site in the mitochondrial ND3 gene.\"}, {\"question\": \"Which are the best treatment options to treat Helicobacter pylori?\", \"ideal_answer\": [\"The best treatment options  for eradication of Helicobacter pylori involve triple or quadruple drugs therapy with different types of antibiotics.\\nBismuth may be also an additional option. Proton pump inhibitors are also included in treatment.\\nThe more effective drug list includes: amoxicillin, claritromycin, metronidazole rifabutin.\\nAlso chitosan microspheres with Eudragit L100 have been tested.\"], \"exact_answer\": [[\"amoxicillin\"], [\"metronidazole\"], [\"claritromycin\"], [\"Proton pump inhibithors\"], [\"rifabutin\"], [\"Eudragit L100\"]], \"context\": \"Triple therapy, which has been the mainstay of treatment in many countries over the last decade, now has suboptimal results in many parts of the world.equential therapy and quadruple therapy (either bismuth-based or non-bismuth-based) are the best current options to replace initial treatment with triple therapy.olecular tests can be used to detect H. pylori and clarithromycin and/or fluoroquinolone resistance in gastric biopsies without necessitating culture. In regions of high clarithromycin resistance, such as France, sequential treatment or bismuth-containing quadruple therapies are replacing standard triple therapies for the first-line empirical treatment.The evidence in favour of bismuth compounds for treating infected children is still not clear.Chitosan microspheres with multiple Eudragit L100 cores were easily prepared by a new emulsification/coagulation encapsulating method.the multi-core chitosan microspheres could serve as a satisfactory vehicle for stomach-specific delivery of hydrophilic antibiotics.Treatment consisted of a one-week regimen containing a PPI twice daily, amoxicillin (A) 1 g twice daily and rifabutin (R) 300 mg once daily (PPI-AR).Pretreatment antibiotic susceptibility to metronidazole, clarithromycin and A was evaluated using a validated epsilometer test.four different strategies for prevention of rebleeding in patients with peptic ulcer hemorrhage: 1) test for H. pylori and treatment, if positive; 2) proton pump inhibitor maintenance; 3) no preventive treatment; 4) empirical H. pylori eradication immediately after bleeding.Empirical H. pylori eradication was the dominant strategy: its estimated rate of recurrent bleeding was lower (6.1%) than those of strategies 1 (7.4%), 2 (11.1%), and 3 (18.4%) and it was the least expensive strategy. The results remained stable when variables were changed inside a wide range of plausible values.In patients with bleeding peptic ulcer, empirical treatment of H. pylori infection immediately after feeding is restarted is the most cost-effective strategy for preventing recurrent hemorrhage.\"}, {\"question\": \"List sclerostin interaction partners.\", \"ideal_answer\": [\"alkaline phosphatase\\ncarbonic anhydrase\\ngremlin-1\\nfetuin A\\nmidkine\\nannexin A1 \\nannexin A2\\ncollagen \\u03b11\\ncasein kinase II \\nsecreted frizzled related protein 4\\nPhex\\nasporin\\nfollistatin\\nerbB-3\\nLRP5 \\nnoggin\"], \"exact_answer\": [[\"alkaline phosphatase\"], [\"carbonic anhydrase\"], [\"gremlin-1\"], [\"fetuin A\"], [\"midkine\"], [\"annexin A1\"], [\"annexin A2\"], [\"collagen \\u03b11\"], [\"casein kinase II\"], [\"secreted frizzled related protein 4\"], [\"Phex\"], [\"asporin\"], [\"follistatin\"], [\"erbB-3\"], [\"LRP5\"], [\"noggin\"]], \"context\": \" Several previously unidentified full-length sclerostin-interacting proteins such as alkaline phosphatase, carbonic anhydrase, gremlin-1, fetuin A, midkine, annexin A1 and A2, and collagen \\u03b11, which have established roles in bone formation or resorption processes, were bound to the sclerostin-MBP amylose resin but not to the MBP amylose resin. Other full-length sclerostin-interacting proteins such as casein kinase II and secreted frizzled related protein 4 that modulate Wnt signaling were identified. Several peptides derived from proteins such as Phex, asporin and follistatin that regulate bone metabolism also bound sclerostin. We identified an interaction between sclerostin and the carboxyl-terminal portion of the receptor tyrosine-protein kinase erbB-3.Sclerostin, SOST, the gene affected in this disease, has been postulated to exert its activity by functioning as a BMP antagonist. However, recent evidence indicates that SOST is highly related to Wise, which can also modulate the Wnt pathway by binding to LRP5 and LRP6.Bone density ligand, Sclerostin, directly interacts with LRP5 but not LRP5G171V to modulate Wnt activity.We show that recombinant sclerostin and noggin bound to each other with high affinity (K(D) = 2.92 nm).\"}, {\"question\": \"What imaging modalities have been listed as method of choice  to diagnose CSF leak?\", \"ideal_answer\": [\"CT cisternography in the investigation of cerebrospinal fluid rhinorrhoea. CTC is an accurate, well-tolerated procedure and should be regarded as the method of choice for investigation of this condition.\\n...unenhanced (three-dimensional constructive interference in steady state (3D-CISS)...In conclusion, 3D-CISS is a non-invasive and reliable technique, and should be the first-choice method to localise CSF leak.\"], \"exact_answer\": [[\"CT cisternography\", \"CTC\"], [\"three-dimensional constructive interference in steady state\", \"3D-CISS\"]], \"context\": \"CT cisternography in the investigation of cerebrospinal fluid rhinorrhoea.CTC is an accurate, well-tolerated procedure and should be regarded as the method of choice for investigation of this condition.Currently the method of choice for cerebrospinal fluid detection is qualitative determination of beta-2-transferrin.surgery the use of intrathecal sodium-fluorescein improves visualisation of the site of leakageThe contribution of 3D-CISS and contrast-enhanced MR cisternography in detecting cerebrospinal fluid leak in patients with rhinorrhoea. In conclusion, 3D-CISS is a non-invasive and reliable technique, and should be the first-choice method to localise CSF leak.unenhanced (three-dimensional constructive interference in steady state (3D-CISS)) and contrast-enhanced MR cisternography (CE-MRC) in detecting the localisation of cerebrospinal fluid (CSF) leak in patients with rhinorrhoeaThe diagnostic work-up included lumbar tapping and measurement of CSF opening pressure, radioisotope cisternography, brain and spinal magnetic resonance imaging (MRI), and computed tomography (CT) myelography.A Tc-99m diethyltriaminepentacetic acid radionuclide cisternography (RNC) showed the accumulation of radioactivity in the area of the subarachnoid space, the poor migration of the isotope over the convexities, and the early appearance of kidney and bladder activity\"}, {\"question\": \"Which are currently available software tools for detecting rare codon clusters in coding sequences?\", \"ideal_answer\": [\"Rare codon clusters (RCCs) correspond to regions along mRNA sequences where among the possible choices of synonymous codons those  with lower usage are observed. Due to the fact that relative codon frequencies have been shown to correlate with their cognate tRNA frequencies, RCCs indicate possible translational attenuation sites. A few tools specific for this task have been described in the literature, namely: LaTcOm, %MinMax, PAUSE, Sherlocc, Sliding Window (RiboTempo)\"], \"exact_answer\": [[\"LaTcOm\"], [\"%MinMax\"], [\"PAUSE\"], [\"Sherlocc\"], [\"Sliding Window\", \"RiboTempo\"]], \"context\": \"Our program Sherlocc, detects statistically relevant conserved rare codon clusters and produces a user-friendly HTML output.LaTcOm: a web server for visualizing rare codon clusters in coding sequencesWe present LaTcOm, a new web tool, which offers several alternative methods for 'rare codon cluster' (RCC) identificationthree RCC detection schemes are implemented: the recently described %MinMax algorithm and a simplified sliding window approach, along with a novel modification of a linear-time algorithm for the detection of maximally scoring subsequences tailored to the RCC detection problemThe PAUSE software has been developed as a new tool to study translational control over protein targeting. This makes it possible to correlate the position of clusters of rare codons in a gene, predicted to cause a translational pause, with the position of hydrophobic stretches in the encoded protein, predicted to span a membrane or to act as a cleavable signal for targeting to the secretory pathway.\"}, {\"question\": \"Name triad of Wernicke encephalopathy.\", \"ideal_answer\": [\"Wernicke's encephalopathy is a triad of ophthalmoplegia, ataxia and confusion seen in alcoholics with dietary vitamin B1 (thiamine) deficiency.\"], \"exact_answer\": [[\"ophthalmoplegia\"], [\"ataxia\"], [\"confusion\"]], \"context\": \"In the classical form it is characterized by a typical triad (confusion, oculomotor disturbance and ataxia), however, in the majority of the cases only confusion is present. Wernicke's encephalopathy is an acute neurological disorder characterized by mental confusion, oculomotor dysfunction, and ataxia. The classic triad consists of encephalopathy, oculomotor dysfunction and gait ataxia but is not seen in a majority of patients.After frequent vomiting, he presented with mental status changes, ocular abnormalities, and truncal ataxia (the classic triad).Wernicke's encephalopathy is a triad of ophthalmoplegia, ataxia and confusion seen in alcoholics with dietary vitamin B1 (thiamine) deficiency.Wernicke encephalopathy is caused by thiamine deficiency in the central nervous system, and is defined by the triad of confusional symptoms, ocular alterations and ataxia.The classical triad of clinical symptoms described by Wernicke (gait ataxia, ophthalmoplegia, and confusion) are found in only a third of patients upon initial examination.It was found that only 0.4% of the population studied had the classical triad of Wernicke's encephalopathy, namely confusion, ophthalmoplegia, and ataxia.The triad of Wernicke's encephalopathy--global confusional state, ophthalmoplegia and nystagmus, and ataxia--is occasionally seen in chronic alcoholics and is often attenuated by immediate thiamine treatment.Only six presented with the Wernicke's encephalopathy clinical triad (mental status changes, ocular signs, and ataxia) at neurologic onset; nine eventually demonstrated this triad.Wernicke's encephalopathy is a well-described syndrome characterized by the classic triad of confusion, ataxia, and ophthalmoplegia.Wernicke&apos;s encephalopathy is a triad of ophthalmoplegia, ataxia and confusion seen in alcoholics with dietary vitamin B1 (thiamine) deficiencyWernicke encephalopathy (WE) is an acute neurologic disorder characterized by a triad of ophthalmoplegia, ataxia, and mental confusionWernicke&apos;s encephalopathy is a metabolic disorder caused by deficiency of thiamine (vitamin B1) seen in alcoholics and even in nonalcoholic patients, classically presenting with a triad of ataxia, ophthalmoplegia, and altered mental statusWernicke&apos;s encephalopathy is an acute neurological syndrome due to thiamine deficiency, which is characterized by a typical triad of mental status changes, oculomotor dysfunction and ataxiaThe triad of Wernicke&apos;s encephalopathy--global confusional state, ophthalmoplegia and nystagmus, and ataxia--is occasionally seen in chronic alcoholics and is often attenuated by immediate thiamine treatmentIn 25/73 (35%) of the patients the classic triad of Wernicke&apos;s encephalopathy with ocular symptoms, ataxia and confusion, was foundIt was found that only 0.4% of the population studied had the classical triad of Wernicke&apos;s encephalopathy, namely confusion, ophthalmoplegia, and ataxiaWernicke&apos;s encephalopathy (WE) is a serious, potentially fatal, neurologic disorder caused by thiamine deficiency (vitamin B(1)), classically described as presenting with a triad of ocular abnormalities, ataxia, and confusionOnly six presented with the Wernicke&apos;s encephalopathy clinical triad (mental status changes, ocular signs, and ataxia) at neurologic onset; nine eventually demonstrated this triad. The high rate of patients diagnosed only at postmortem examination (41.9%) confirms that Wernicke&apos;s encephalopathy is underdiagnosed in children.\"}, {\"question\": \"How many and which are the different isoforms for the ryanodine receptor?\", \"ideal_answer\": [\"Generally, three ryanodine receptor isoforms (RyR1-RyR3) are known. RyR1, expressed in skeletal muscle; RyR2, expressed in cardiac muscle; and RyR3, expressed in various cells. RyR3 is preferentially expressed in the brain especially in the hippocampus and striatum.\"], \"exact_answer\": [[\"Ryanodine Receptor type 1\", \"RyR1\"], [\"Ryanodine Receptor type 2\", \"RyR2\"], [\"Ryanodine Receptor type 3\", \"RyR3\"]], \"context\": \"Skeletal (RyR1) and cardiac muscle (RyR2) isoforms of ryanodine receptor calcium channelsRyanodine receptors (RyRs) are located in the sarcoplasmic/endoplasmic reticulum membrane and are responsible for the release of Ca(2+) from intracellular stores during excitation-contraction coupling in both cardiac and skeletal muscle. RyRs are the largest known ion channels (> 2MDa) and exist as three mammalian isoforms (RyR 1-3),Ryanodine receptors (RyRs) are a family of intracellular calcium release channels that mediate calcium-induced calcium release from the endoplasmic reticulum. Among the three RyR isoforms, RyR3 is preferentially expressed in the brain especially in the hippocampus and striatum.In excitable cells such as skeletal and cardiac myocytes excitation-contraction coupling is an important intermediate step between initiation of the action potential and induction of contraction. This process is predominantly controlled by Ca(2+) release from the sarcoplasmic reticulum via the ryanodine receptor. This very large protein (MW 560 kDa) exists as a homotetramer (~2.2 MDa) and is expressed in three isoforms: RyR1, expressed in skeletal muscle; RyR2, expressed in cardiac muscle; and RyR3, expressed in various cells at lower levels than the other isoforms. Generally, three ryanodine receptor isoforms (RyR1-RyR3) are known; Ryanodine receptor (RyR) is a Ca(2+) channel that mediates Ca(2+) release from intracellular stores.There are three RyR isoforms,Ryanodine receptors (RyRs) are intracellular Ca(2+) channels that mediate the release of calcium from internal stores and therefore play an important role in Ca(2+) signaling and homeostasis. Three RyR isoforms have been described thus far,RyR1 and RyR2RyR3Three different isoforms of the SR Ca2+ release channels, or ryanodine receptors (RyRs), have been isolated (RyR1, RyR2, and RyR3).\"}, {\"question\": \"Which protein phosphatases have been found to dephosphorylate phospholamban?\", \"ideal_answer\": [\"The protein phosphatases which dephosphorylate native, sarcoplasmic reticulum (SR)-associated phospholamban were studied in cardiac muscle extracts and in a Triton fraction prepared by detergent extraction of myofibrils, the latter fraction containing 70-80% of the SR-associated proteins present in the tissue. At physiological concentrations of free Mg2+ (1 mM), protein phosphatase 1 (PP1) accounted for approximately 70% of the total phospholamban phosphatase activity in these fractions towards either Ser-16 (the residue labelled by cAMP-dependent protein kinase, PK-A) or Thr-17 (the residue phosphorylated by an SR-associated Ca2+/calmodulin-dependent protein kinase). Protein phosphatase 2A (PP2A) and protein phosphatase 2C (PP2C) accounted for the remainder of the activity.  \", \"The protein phosphatases which dephosphorylate native, sarcoplasmic reticulum (SR)-associated phospholamban were studied in cardiac muscle extracts and in a Triton fraction prepared by detergent extraction of myofibrils, the latter fraction containing 70-80% of the SR-associated proteins present in the tissue. At physiological concentrations of free Mg2+ (1 mM), protein phosphatase 1 (PP1) accounted for approximately 70% of the total phospholamban phosphatase activity in these fractions towards either Ser-16 (the residue labelled by cAMP-dependent protein kinase, PK-A) or Thr-17 (the residue phosphorylated by an SR-associated Ca2+/calmodulin-dependent protein kinase). Protein phosphatase 2A (PP2A) and protein phosphatase 2C (PP2C) accounted for the remainder of the activity.\"], \"exact_answer\": [[\"Protein Phosphatase 1\", \"PP1\"], [\"Protein phosphatase 2A\", \"PP2A\"], [\"Protein Phosphatase 2C\", \"PP2C\"], [\"Protein Phosphatase 2B\", \"PP2B\"]], \"context\": \"The protein phosphatases which dephosphorylate native, sarcoplasmic reticulum (SR)-associated phospholamban were studied in cardiac muscle extracts and in a Triton fraction prepared by detergent extraction of myofibrils, the latter fraction containing 70-80% of the SR-associated proteins present in the tissue. At physiological concentrations of free Mg2+ (1 mM), protein phosphatase 1 (PP1) accounted for approximately 70% of the total phospholamban phosphatase activity in these fractions towards either Ser-16 (the residue labelled by cAMP-dependent protein kinase, PK-A) or Thr-17 (the residue phosphorylated by an SR-associated Ca2+/calmodulin-dependent protein kinase). Protein phosphatase 2A (PP2A) and protein phosphatase 2C (PP2C) accounted for the remainder of the activity.These results are consistent with the views that protein phosphatase 1 is capable of dephosphorylating membrane-associated phospholamban when it is phosphorylated by protein kinase A, but not by calcium/calmodulin kinase, and that under certain conditions, calcium/calmodulin-stimulated protein phosphatase (protein phosphatase 2B) is also able to dephosphorylate PLN phosphorylated by protein kinase A. Additionally, the observations show that protein phosphatase 1 is extremely active against the three protein kinase A substrates (M(r) 23, 19 and 17 kDa) that were present in the isolated microsomes and whose state of phosphorylation was particularly affected in the presence of dimethylsulfoxide.These results are consistent with the views that protein phosphatase 1 is capable of dephosphorylating membrane-associated phospholamban when it is phosphorylated by protein kinase A, but not by calcium/calmodulin kinase, and that under certain conditions, calcium/calmodulin-stimulated protein phosphatase (protein phosphatase 2B) is also able to dephosphorylate PLN phosphorylated by protein kinase AThese results are consistent with the views that protein phosphatase 1 is capable of dephosphorylating membrane-associated phospholamban when it is phosphorylated by protein kinase A, but not by calcium/calmodulin kinase, and that under certain conditions, calcium/calmodulin-stimulated protein phosphatase (protein phosphatase 2B) is also able to dephosphorylate PLN phosphorylated by protein kinase A\"}, {\"question\": \"What is the most prominent sequence consensus for the polyadenylation site?\", \"ideal_answer\": [\"Functional polyadenylation [poly(A)] sites consist of two sequence elements, the AAUAAA and G/U box signals, that closely flank the site of mRNA 3'-end formation. The canonical polyadenylation signal sequence AATAAA\"], \"exact_answer\": [[\"AATAAA\"], [\"AAUAAA\"]], \"context\": \"the cannonical polyadenylation signal sequence AATAAATwo AATAAA motifs are coded in the last exon of this geneThe results demonstrate that the intact AAUAAA is not required for RNA polyadenylation but is required for the cleavage step preceding polyadenylation to occur efficientlythe early poly(A) consensus signal was mutated from AAUAAA to UGUAAAFunctional polyadenylation [poly(A)] sites consist of two sequence elements, the AAUAAA and G/U box signals, that closely flank the site of mRNA 3'-end formationthe appropriate spacing of the AAUAAA and G/U box signals is critical for poly(A) site function Two AATAAA hexanucleotide sequences are present in the 2092 nucleotide interval. The first one functions as the major polyA signal\"}, {\"question\": \"What are the main benefits of pharmacophore models?\", \"ideal_answer\": [\"As researchers continue to search for new targets of therapeutic interest, transmembrane and G-protein coupled receptors are of ever-increasing importance. However, crystal structures for these targets may be impossible to resolve, posing great challenges in rational drug design. Structure-based virtual screening is not an option when the active site geometry is unknown, but assaying an entire library for hits is an inefficient and expensive proposition.\\nPharmacophore modeling solves this problem by determining the spatial arrangement of chemical features that confer drug activity toward a target receptor. Having established the chemical space occupied by active ligands, pharmacophore modeling software allows researchers to create 3-D structure-activity relationships, screen databases, and generate hits without the benefit of a receptor structure.\"], \"exact_answer\": [[\"They represent chemical functions valid not only for the existing bounds but also for unknown molecules.\"], [\"Due their simplicity they are proper for large scale virtual screening\"], [\"They are comprehensive and editable, so by changing chemical feature contains information can be easily trace back.\"]], \"context\": \"To discover novel RNAP inhibitors, a pharmacophore based on the alignment of described inhibitors was used for virtual screening. In an optimization process of hit compounds, novel derivatives with improved in vitro potency were discovered. Investigations concerning the molecular mechanism of RNAP inhibition reveal that they prevent the protein-protein interaction (PPI) between \\u03c3(70) and the RNAP core enzyme.the primary structural features required for estrogen receptor binding, the phenols, are not required for inhibiting parasitic growth. Significantly, the most active antileishmanial benzothiophenes lack the pharmacophore for estrogen receptor activity and therefore address potential concerns about the undesirable effects of using selective estrogen receptor modulators in women and children with leishmaniasis. series of novel diastereoisomeric \\u03c3 ligands 3 was designed, synthesized and pharmacologically evaluated. The highly rigid [4.3.3]propellane scaffold was used to fix the three dimensional orientation of the pharmacophoric moieties required for \\u03c3 affinityThis study was performed to identify natural iNOS inhibitors from traditional Chinese herbs through a combination of pharmacophore modeling, molecular docking and virtual screening. First, the pharmacophore models were generated though six known iNOS inhibitors and validated by a test database. The pharmacophore model_017 showed good performance in external validation and was employed to screen Traditional Chinese Medicine Database (Version 2009), which resulting in a hit list of 498 compounds with matching score (QFIT) above 40. Then, the hits were subjected to molecular docking for further refinement. An empirical scoring function was used to evaluate the affinity of the compounds and the target protein. Parts of compounds with high docking scores have been reported to have the related pharmacological activity from the literatures.Hit compound 8 was identified as an inhibitor of tankyrases through a combination of substructure searching of the Amgen compound collection based on a minimal binding pharmacophore hypothesis and high-throughput screening. Herein we report the structure- and property-based optimization of compound 8 leading to the identification of more potent and selective tankyrase inhibitors 22 and 49 with improved pharmacokinetic properties in rodents, which are well suited as tool compounds for further in vivo validation studies\"}, {\"question\": \"Which drugs affect insulin resistance in obesity?\", \"ideal_answer\": [\"Enistein treatment could help reduce insulin resistance\\nACE inhibitor drugs may improve insulin resistance\"], \"exact_answer\": [[\"Enistein\"], [\"ACE inhibitors\"]], \"context\": \"enistein treatment could help reduce insulin resistance through the amelioration of OVX-induced metabolic dysfunction, and the GSEA approach may be useful in proposing putative targets related to insulin resistance.he positive effects of ACE inhibitor drugs, particularly on hypertriglyceridemia and insulin resistance, might bring them forth as first-line drugs in the treatment of obese and hypertensive children. \"}, {\"question\": \"List two most common symptoms of Aagenaes syndrome.\", \"ideal_answer\": [\"Aagenaes syndrome, also called lymphoedema cholestasis syndrome 1, is characterized by neonatal intrahepatic cholestasis, often lessening and becoming intermittent with age and severe chronic lymphoedema, mainly affecting the lower extremities.\"], \"exact_answer\": [[\"intrahepatic cholestasis\"], [\"lymphoedema\"]], \"context\": \"Aagenaes syndrome, also called lymphoedema cholestasis syndrome 1 (LSC1), is characterized by neonatal intrahepatic cholestasis, often lessening and becoming intermittent with age and severe chronic lymphoedema, mainly affecting the lower extremities. In this study, 17 out of 20 Norweigian adult patients with lymphedema cholestasis syndrome 1 (LCS1)/Aagenaes syndrome were examined. The patients exhibited lymphedema and sporadic cholestasis.BACKGROUND: Lymphedema-cholestasis syndrome (LCS; Aagenaes syndrome) is a rare autosomal recessive disorder, characterized by 1) neonatal intrahepatic cholestasis, often lessening and becoming intermittent with age, and 2) severe chronic lymphedema, mainly lower limb.The disease was thought to be caused by a congenital cholestatic syndrome associated with intermittent oedema in childhood, resembling the rare Aagenaes syndrome.Do patients with lymphoedema cholestasis syndrome 1/Aagenaes syndrome need dietary counselling outside cholestatic episodes?BACKGROUND & AIMS: Patients with lymphoedema cholestasis syndrome 1/Aagenaes Syndrome need a fat reduced diet when cholestatic.Aagenaes syndrome, also called Lymphedema Cholestasis Syndrome (LSC 1), is a form of idiopathic familial intrahepatic cholestasis associated with lymphedema of the lower extremities. Aagenaes syndrome, also called Lymphedema Cholestasis Syndrome (LSC 1), is a form of idiopathic familial intrahepatic cholestasis associated with lymphedema of the lower extremities.Parent-child transmission of infantile cholestasis with lymphoedema (Aagenaes syndrome).Mapping of the locus for cholestasis-lymphedema syndrome (Aagenaes syndrome) to a 6.6-cM interval on chromosome 15q.Lymphedema-cholestasis syndrome (LCS, Aagenaes syndrome) is the only known form of hereditary lymphedema associated with cholestasis. OBJECTIVE: To investigate the prognosis of liver disease in Aagenaes syndrome (lymphoedema cholestasis syndrome 1 (LCS1)), which is an autosomal recessive inherited syndrome consisting of neonatal cholestasis with intermittent cholestatic episodes in childhood into adulthood and development of lymphoedema. BACKGROUND: Lymphedema-cholestasis syndrome (LCS; Aagenaes syndrome) is a rare autosomal recessive disorder, characterized by 1) neonatal intrahepatic cholestasis, often lessening and becoming intermittent with age, and 2) severe chronic lymphedema, mainly lower limb. [Aagenaes syndrome--lymphedema and intrahepatic cholestasis].Parent-child transmission of infantile cholestasis with lymphoedema (Aagenaes syndrome).Hereditary lymphedema, characteristics, and variations in 17 adult patients with lymphedema cholestasis syndrome 1/Aagenaes syndrome.Lymphedema-cholestasis syndrome (LCS, Aagenaes syndrome) is the only known form of hereditary lymphedema associated with cholestasis.\"}, {\"question\": \"Which are the side effects during tacrine administration in patients with Alzheimer's Disease?\", \"ideal_answer\": [\"The side effects during tacrine administration in patients with Alzheimer's Disease are:\\r\\n1) Hepatotoxicity\\r\\n2) Gastrointestinal (diarrhea, anorexia, dyspepsia, abdominal pain, nausea, vomiting)\\r\\n3) Mitochondrial impairement\"], \"exact_answer\": [[\"Hepatotoxicity\"], [\"Gastrointestinal\"], [\"diarrhea\"], [\"anorexia\"], [\"dyspepsia\"], [\"abdominal pain\"], [\"nausea\"], [\"vomiting\"], [\"anorexia\"], [\"Mitochondrial impairement\"]], \"context\": \"Tacrine, the first of the cholinesterase inhibitors to undergo extensive trials for this purpose, was associated with significant adverse effects including hepatotoxicity.Tacrine and its analogues impair mitochondrial function and bioenergetics: a lipidomic analysis in rat brain.However, its low therapeutic efficiency and a high incidence of side effects have limited its clinical use. Tacrine induced significant perturbations in the mitochondrial PL profile, which were detected by means of changes in the relative abundance of phosphatidylcholine (PC), PE, phosphatidylinositol (PI) and CL and by the presence of oxidized phosphatidylserines.These results indicate that tacrine and its analogues impair mitochondrial function and bioenergetics, thus compromising the activity of brain cells.Tacrine, the first of the cholinesterase inhibitors to undergo extensive trials for this purpose, was associated with significant adverse effects including hepatotoxicity. The antidementives are well tolerated and undesired effects are rare; except hepatotoxicity of tacrine and gastrointestinal side effects of donepezil, rivastigmine, galantamin and tacrine that result from acetylcholinesterase inhibition. Nausea, diarrhea, vomiting, and weight loss are the most common side effects of the acetylcholinesterase inhibitors. Raised serum liver enzymes was the major reason for withdrawal. Gastrointestinal side effects (diarrhoea, anorexia, dyspepsia and abdominal pain) were the other major cause of adverse events and for withdrawal, and the odds ratio for withdrawal was also significantly different from one in favour of the control group (OR 3.8; 95%CI 2.8-5.1).To determine the clinical efficacy of tacrine for the symptoms of Alzheimer's disease.In Alzheimer's disease, cognition now responds to several drugs. Anticholinesterases target the acetylcholine deficit. In mild-to-moderate Alzheimer's disease, they all provide significant benefit versus placebo on the Alzheimer's Disease Assessment ScheduleCognitive Section (ADAS-Cog), Side effects, in 5% to 15% of cases, include nausea, vomiting, diarrhea, anorexia, and dizziness. Tacrine, the leading anticholinesterase, caused frequent hepatic enzyme elevation and was withdrawnThis 12-week randomised, double-blind, placebo-controlled study was undertaken to evaluate the ability of silymarin to antagonise or prevent the hepatotoxic effects of tacrine and to analyse its action on tacrine efficacy and tolerability.METHODS: Outpatients suffering from mild-to-moderate dementia of the Alzheimer type were randomly assigned to two treatment groups: tacrine + silymarin and tacrine + placebo.Hepatotoxic effects of tacrine administration in patients with Alzheimers diseaseHepatotoxic effects of tacrine administration in patients with Alzheimer's disease.Tremulous jaw movements produced by acute tacrine administration: possible relation to parkinsonian side effects.Administration of tacrine (THA) for the treatment of Alzheimer's disease results in a reversible hepatotoxicity in 30-50% of patients, as indicated by an increase in transaminase levels.Tacrine, the first drug marketed for Alzheimer's disease (AD), induces an elevation of serum liver transaminase prohibiting an effective dosage in many patients. This 12-week randomised, double-blind, placebo-controlled study was undertaken to evaluate the ability of silymarin to antagonise or prevent the hepatotoxic effects of tacrine and to analyse its action on tacrine efficacy and tolerability.METHODS: Outpatients suffering from mild-to-moderate dementia of the Alzheimer type were randomly assigned to two treatment groups: tacrine + silymarin and tacrine + placebo.BACKGROUND: Silymarin is a well-known hepatoprotective agent. Tacrine, the first drug marketed for Alzheimer's disease (AD), induces an elevation of serum liver transaminase prohibiting an effective dosage in many patients. This 12-week randomised, double-blind, placebo-controlled study was undertaken to evaluate the ability of silymarin to antagonise or prevent the hepatotoxic effects of tacrine and to analyse its action on tacrine efficacy and tolerability.METHODS: Outpatients suffering from mild-to-moderate dementia of the Alzheimer type were randomly assigned to two treatment groups: tacrine + silymarin and tacrine + placebo.Tacrine, the first drug marketed for Alzheimer's disease (AD), induces an elevation of serum liver transaminase prohibiting an effective dosage in many patients. This 12-week randomised, double-blind, placebo-controlled study was undertaken to evaluate the ability of silymarin to antagonise or prevent the hepatotoxic effects of tacrine and to analyse its action on tacrine efficacy and tolerability.METHODS: Outpatients suffering from mild-to-moderate dementia of the Alzheimer type were randomly assigned to two treatment groups: tacrine + silymarin and tacrine + placebo. The study was double-blind for silymarin and open for tacrine and was conducted in 22 French neurology and geriatric centres.Tacrine, the first drug marketed for Alzheimer's disease (AD), induces an elevation of serum liver transaminase prohibiting an effective dosage in many patients.Hepatotoxic effects of tacrine administration in patients with Alzheimer's disease.\"}, {\"question\": \"Which enzymes synthesize catecholamines in adrenal glands?\", \"ideal_answer\": [\"The enzymes that synthesize catecholamines in adrenal glands are:\\n1) Tyrosine Hydroxylase (TH)\\n2) Aromatic L-amino acid decarboxylase (AAAD)\\n3) Dopamine \\u03b2-hydroxylase (DBH)\\n4) Phenylethanolamine N-methyltransferase (PNMT)\"], \"exact_answer\": [[\"Tyrosine Hydroxylase\", \"TH\"], [\"Aromatic L-amino acid decarboxylase\", \"AAAD\", \"AADC\", \"DDC\"], [\"Dopamine \\u03b2-hydroxylase\", \"DBH\"], [\"Phenylethanolamine N-methyltransferase\", \"PNMT\"]], \"context\": \"We analyzed the mRNA levels of catecholamine-synthesizing enzymes: tyrosine hydroxylase (TH), aromatic L-amino acid decarboxylase (AAAD), dopamine-\\u03b2-hydroxylase (DBH) and phenylethanolamine N-methyltransferase (PNMT) in adrenal glands of 18 pigs with chronic systolic non-ischaemic HF (tachycardia-induced cardiomyopathy due to right ventricle pacing) and 6 sham-operated controls.This study aimed to investigate physical exercise-related changes in gene expression of catecholamine biosynthetic enzymes (tyrosine hydroxylase, dopamine-\\u00df-hydroxylase and phenylethanolamine N-methyltransferase) and cyclic adenosine monophosphate response element-binding (CREB) in the adrenal medulla, concentrations of catecholamines and corticosterone (CORT) in the plasma and the weight of adrenal glands of chronically psychosocially stressed adult rats exposed daily to 20 min treadmill running for 12 weeks. In the adrenal medulla, despite the absence of morphological changes, immunohistochemistry for tyrosine hydroxylase, dopamine \\u03b2-hydroxylase and phenyl-ethanolamine-N-methyltransferase demonstrated an increased immunopositivity for these cathecolamine-synthesizing enzymes after intense exercise. Using isolated adrenal medulla we observed no difference in basal catecholamine secretion percentile between obese and lean animals. However, the percentile of catecholamine secretion stimulated by high K+ concentration was lower in the obese group. There was a decrease in the tyrosine hydroxylase enzyme expression (57.3%, P<0.004) in adrenal glands of obese mice. Interestingly, the expression of dopamine beta-hydroxylase was also reduced (47.0%, P<0.005). Phenylethanolamine N-methyltransferase expression was not affected. Differential housing (single vs. group) and social defeat of rats is known to alter the activity of catecholamine-synthesizing enzymes in the medulla. The present studies examined the effect of 70 days of triad (3 rats per large cage) and individual housing of male rats on adrenal mRNA levels of tyrosine hydroxylase (TH), dopamine-beta-hydroxylase (DBH) and phenylethanolamine-N-methyltransferase (PNMT) and on TH protein levels.As a first step toward understanding the molecular mechanisms by which catecholamine synthesis is controlled in the tumor, we measured the levels of mRNA coding for the catecholamine synthesizing enzyme, tyrosine hydroxylase (TH) and catecholamines in 6 pheochromocytomas and 2 normal adrenal glands. We also examined the gene expression of the messengers of other catecholamine synthesizing enzymes, dopamine beta-hydroxylase (DBH) and aromatic 1-amino acid decarboxylase (AADC) in pheochromocytomas. These findings indicate that catecholamine overproduction in pheochromocytomas is mediated by the overexpression of genes coding for catecholamines synthesizing enzymes, TH, DBH, and AADC.he present study investigated the cellular localization of 3 catecholamine biosynthetic enzymes, tyrosine hydroxylase (TH), dopamine beta-hydroxylase (DBH), and phenylethanolamine N-methyltransferase (PNMT) tyrosine hydroxylase (TH) the rate limiting catecholamine biosynthetic enzyme and also of dopamine beta-hydroxylase (DBH)tyrosine hydroxylase (TH), dopamine-beta-hydroxylase (DBH) and phenylethanolamine-N-methyltransferase (PNMT) in sympathetic ganglia and adrenalsThis study aimed to investigate physical exercise-related changes in gene expression of catecholamine biosynthetic enzymes (tyrosine hydroxylase, dopamine-\\u00df-hydroxylase and phenylethanolamine N-methyltransferase) and cyclic adenosine monophosphate response element-binding (CREB) in the adrenal medulla, concentrations of catecholamines and corticosterone (CORT) in the plasma and the weight of adrenal glands of chronically psychosocially stressed adult rats exposed daily to 20 min treadmill running for 12 weeksAs a first step toward understanding the molecular mechanisms by which catecholamine synthesis is controlled in the tumor, we measured the levels of mRNA coding for the catecholamine synthesizing enzyme, tyrosine hydroxylase (TH) and catecholamines in 6 pheochromocytomas and 2 normal adrenal glandsWe analyzed the mRNA levels of catecholamine-synthesizing enzymes: tyrosine hydroxylase (TH), aromatic L-amino acid decarboxylase (AAAD), dopamine-\\u03b2-hydroxylase (DBH) and phenylethanolamine N-methyltransferase (PNMT) in adrenal glands of 18 pigs with chronic systolic non-ischaemic HF (tachycardia-induced cardiomyopathy due to right ventricle pacing) and 6 sham-operated controlsThis study aimed to investigate physical exercise-related changes in gene expression of catecholamine biosynthetic enzymes (tyrosine hydroxylase, dopamine-\\u00df-hydroxylase and phenylethanolamine N-methyltransferase) and cyclic adenosine monophosphate response element-binding (CREB) in the adrenal medulla, concentrations of catecholamines and corticosterone (CORT) in the plasma and the weight of adrenal glands of chronically psychosocially stressed adult rats exposed daily to 20 min treadmill running for 12 weeks.We used immunocytochemistry to study the ontogeny of leu-enkephalin and the catecholamine-synthesizing enzymes dopamine beta-hydroxylase and phenylethanolamine N-methyltransferase in adjacent sections of 14 fetal rhesus and 31 fetal human adrenal glands.As a first step toward understanding the molecular mechanisms by which catecholamine synthesis is controlled in the tumor, we measured the levels of mRNA coding for the catecholamine synthesizing enzyme, tyrosine hydroxylase (TH) and catecholamines in 6 pheochromocytomas and 2 normal adrenal glands.\"}, {\"question\": \"Which two drugs are included in the Harvoni pill?\", \"ideal_answer\": [\"Harvoni contains 400 mg sofosbuvir and 90 mg ledipasvir. It used for treatment of hepatitis C virus infection.\"], \"exact_answer\": [[\"sofosbuvir\"], [\"ledipasvir\"]], \"context\": \"Will Sofosbuvir/Ledipasvir (Harvoni) Be Cost-Effective and Affordable for Chinese Patients Infected with Hepatitis C Virus?The developed method was applied to the analysis of the two drugs after a single oral administration of Harvoni 400/90 mg film-coated tablets containing 400 mg sofosbuvir and 90 mg ledipasvir to four healthy volunteers.After the introductions of sofosbuvir (Sovaldi) and ledipasvir plus sofosbuvir (Harvoni) for the treatment of hepatitis C, employers have become very sensitive to new, and especially unforeseen, factors that significantly raise healthcare costs.The single-tablet regimen of the hepatitis C virus (HCV) NS5A inhibitor ledipasvir and the HCV NS5B polymerase inhibitor sofosbuvir (ledipasvir/sofosbuvir; Harvoni(\\u00ae)) was recently approved in the US and the EU. Ledipasvir/Sofosbuvir (harvoni): improving options for hepatitis C virus infection.Ledipasvir/sofosbuvir (Harvoni): improving options for hepatitis C virus infection.Also reviewed were recent practice guidelines on the management of HCV infections, prescribing information on all HCV drugs approved by the US Food and Drug Administration, and health technology assessments of Sovaldi\\u00ae and Harvoni(TM) (sofosbuvir/ledipasvir).Ledipasvir/Sofosbuvir (harvoni): improving options for hepatitis C virus infection.This review will summarize the data regarding these interferon-free regimens, including Gilead's Harvoni (sofosbuvir/ledipasvir), AbbVie's Viekira Pak (paritaprevir/ritonavir/ombitasvir with dasabuvir), and Janssen's Olysio (simeprevir) with sofosbuvir.\"}, {\"question\": \"Which mutated genes are associated with the Tourette's syndrome?\", \"ideal_answer\": [\"A mutation in histidine decarboxylase (Hdc) gene as well as mutations in the SLITRK1 (Slit and Trk-like 1) gene have been implicated as rare genetic causes of Tourette's syndrome.\"], \"exact_answer\": [[\"histidine decarboxylase\", \"Hdc\"], [\"Slit and Trk-like 1\", \"SLITRK1\"]], \"context\": \" A rare genetic form of Tourette syndrome due to L-histidine-decarboxylase mutation, with similar features in human and rodent, has inspired new research on functional anatomy of Tourette syndrome.A mutation in histidine decarboxylase (Hdc), the key enzyme for the biosynthesis of histamine (HA), has been implicated as a rare genetic cause.Histidine decarboxylase deficiency causes tourette syndrome: parallel findings in humans and mice.Tourette syndrome (TS), the most severe end of the continuum of tic disorders, is substantially genetic, but causative mutations have been elusive.The SLITRK1 gene encodes a developmentally regulated stimulator of neurite outgrowth and previous studies have implicated rare variants in this gene in disorders in the OC spectrum, specifically Tourette syndrome (TS)The recent implication of the histamine decarboxylase (HDC) gene, the key enzyme in histamine production, raises the intriguing hypothesis of a possible role of histaminergic dysfunction leading to TS onset.we describe an analysis of linkage in a two-generation pedigree leading to the identification of a rare functional mutation in the HDC gene encoding L-histidine decarboxylase, the rate-limiting enzyme in histamine biosynthesis.Mutations in the gene SLITRK1 (Slit and Trk-like 1) have been reported in patients with Tourette's disorder (TD)Recently, two variants, including a single-base deletion resulting in a truncated protein and a 3'-untranslated-region variant altering a binding site for micro-RNA in the Slit and Trk-like 1 gene, were found to be a genetic cause of Tourette syndrome.Collectively, these findings support the association of rare SLITRK1 sequence variants with TS.\"}, {\"question\": \"Data from which major epigenome projects are contained in the DeepBlue epigenomic data server?\", \"ideal_answer\": [\"The DeepBlue Epigenomic Data Server contains data from four major epigenome projects, namely ENCODE, ROADMAP, BLUEPRINT and DEEP.\"], \"exact_answer\": [[\"ENCODE\"], [\"ROADMAP\"], [\"BLUEPRINT\"], [\"DEEP\"]], \"context\": \"Here we present the DeepBlue Epigenomic Data Server, which streamlines epigenomic data analysis as well as software development. DeepBlue provides a comprehensive programmatic interface for finding, selecting, filtering, summarizing and downloading region sets. It contains data from four major epigenome projects, namely ENCODE, ROADMAP, BLUEPRINT and DEEP. \"}, {\"question\": \"Which diseases that can be treated using the focused ultrasound thalamotomy.\", \"ideal_answer\": [\"Focused ultrasound thalamotomy is used for treatment of Parkinson disease, essential tremor, obsessive-compulsive disorder and chronic neuropathic pain.\"], \"exact_answer\": [[\"Parkinson disease\"], [\"essential tremor\"], [\"obsessive-compulsive disorder\"], [\"chronic neuropathic pain\"]], \"context\": \"Besides DBS and standard thalamotomy, novel surgical approaches for ET are on the horizon. The development of MRI-guided focused ultrasound technique has been the new frontier of deep brain lesional therapies.132\\u2003A Randomized, Sham-Controlled Trial of Transcranial Magnetic Resonance-Guided Focused Ultrasound Thalamotomy Trial for the Treatment of Tremor-Dominant, Idiopathic Parkinson Disease.Recently, transcranial magnetic resonance-guided focused ultrasound (MRgFUS) has been used to successfully perform thalamotomy for essential tremor. We designed a double-blinded, randomized controlled trial to investigate the effectiveness of MRgFUS thalamotomy in tremor-dominant PD.Thalamotomy at the ventralis intermedius nucleus has been an effective treatment method for essential tremor, but how the brain network changes immediately responding to this deliberate lesion and then reorganizes afterwards are not clear. Taking advantage of a non-cranium-opening MRI-guided focused ultrasound ablation technique, we investigated functional network changes due to a focal lesion. Unilateral magnetic resonance guided focused ultrasound thalamotomy for essential tremor: practices and clinicoradiological outcomes.CONCLUSIONS: Our results demonstrate that MRgFUS thalamotomy is a safe, effective and less-invasive surgical method for treating medication-refractory ET. OBJECT: The authors report different MRI patterns in patients with essential tremor (ET) or obsessive-compulsive disorder (OCD) after transcranial MR-guided focused ultrasound (MRgFUS) and discuss possible causes of occasional MRgFUS failure.METHODS: Between March 2012 and August 2013, MRgFUS was used to perform unilateral thalamotomy in 11 ET patients and bilateral anterior limb capsulotomy in 6 OCD patients; in all patients symptoms were refractory to drug therapy. To discern the neurophysiologic correlates of its therapeutic effects, we applied MRgHIFU to an intractable neurologic disorder, essential tremor, while measuring magnetoencephalogram mu rhythms from the motor cortex. MRI Guided Focused Ultrasound Thalamotomy for Moderate-to-Severe Tremor in Parkinson's Disease.Methods. Seven PD patients, mean age 59.4 \\u00b1 9.8 years (range, 46-74) with a mean disease duration of 5.4 \\u00b1 2.8 years (range, 2-10) suffering from severe refractory tremor, underwent ventral intermediate nucleus thalamotomy using MRI guided focused ultrasound (MRgFUS), an innovative technology that enables noninvasive surgery. Thalamotomy using MRgFUS is safe and effective in PD patients.Evolution of Movement Disorders Surgery Leading to Contemporary Focused Ultrasound Therapy for Tremor.A noninvasive approach using transcranial high-energy focused ultrasound has emerged for the treatment of intractable tremor.This article describes the evolution of magnetic resonance-guided focused ultrasound for ventrolateral thalamotomy for tremor.Thalamic connectivity in patients with essential tremor treated with MR imaging-guided focused ultrasound: in vivo fiber tracking by using diffusion-tensor MR imaging.PURPOSE: To use diffusion-tensor (DT) magnetic resonance (MR) imaging in patients with essential tremor who were treated with transcranial MR imaging-guided focused ultrasound lesion inducement to identify the structural connectivity of the ventralis intermedius nucleus of the thalamus and determine how DT imaging changes correlated with tremor changes after lesion inducement.MATERIALS AND METHODS: With institutional review board approval, and with prospective informed consent, 15 patients with medication-refractory essential tremor were enrolled in a HIPAA-compliant pilot study and were treated with transcranial MR imaging-guided focused ultrasound surgery targeting the ventralis intermedius nucleus of the thalamus contralateral to their dominant hand.A pilot study of focused ultrasound thalamotomy for essential tremor.This preliminary study investigates the use of transcranial MRI-guided focused ultrasound thalamotomy for the treatment of essential tremor.METHODS: From February 2011 through December 2011, in an open-label, uncontrolled study, we used transcranial MRI-guided focused ultrasound to target the unilateral ventral intermediate nucleus of the thalamus in 15 patients with severe, medication-refractory essential tremor.CONCLUSIONS: In this pilot study, essential tremor improved in 15 patients treated with MRI-guided focused ultrasound thalamotomy.MR-guided focused ultrasound thalamotomy for essential tremor: a proof-of-concept study.MR-guided focused ultrasound has been developed as a non-invasive means of generating precisely placed focal lesions. We examined its application to the management of essential tremorFour patients with chronic and medication-resistant essential tremor were treated with MR-guided focused ultrasound to ablate tremor-mediating areas of the thalamus.INTERPRETATION: MR-guided focused ultrasound might be a safe and effective approach to generation of focal intracranial lesions for the management of disabling, medication-resistant essential tremor.Transcranial magnetic resonance imaging-guided focused ultrasound: noninvasive central lateral thalamotomy for chronic neuropathic pain.The goal of this study was to apply the new transcranial magnetic resonance imaging-guided focused ultrasound (tcMRgFUS) technology to perform noninvasive central lateral thalamotomies (CLTs) as a treatment for chronic neuropathic pain.METHODS: In 12 patients suffering from chronic therapy-resistant neuropathic pain, tcMRgFUS CLT was proposed. CONCLUSIONS: The authors assert that tcMRgFUS represents a noninvasive, precise, and radiation-free neurosurgical technique for the treatment of neuropathic pain.A new less invasive treatment of tremor using MR guided focused ultrasound has started and is promising.Quality-of-life scores improved from 37% to 11% (P=0.001).CONCLUSIONS: In this pilot study, essential tremor improved in 15 patients treated with MRI-guided focused ultrasound thalamotomy.Seven PD patients, mean age 59.4 \\u00b1 9.8 years (range, 46-74) with a mean disease duration of 5.4 \\u00b1 2.8 years (range, 2-10) suffering from severe refractory tremor, underwent ventral intermediate nucleus thalamotomy using MRI guided focused ultrasound (MRgFUS), an innovative technology that enables noninvasive surgeryWe hypothesized that all three would effectively treat the dominant hand and positively impact functional outcomes and quality of life as measured with the Clinical Rating Scale for Tremor and the Quality of Life in Essential Tremor Questionnaire.This is a retrospective study of medication-refractory essential tremor patients treated at the University of Virginia with bilateral Vim DBS (n\\u2009=\\u200957), unilateral Vim DBS (n\\u2009=\\u200913), or unilateral focused ultrasound Vim thalamotomy (n\\u2009=\\u200915)Quality-of-life scores improved from 37% to 11% (P=0.001).In this pilot study, essential tremor improved in 15 patients treated with MRI-guided focused ultrasound thalamotomyMRI Guided Focused Ultrasound Thalamotomy for Moderate-to-Severe Tremor in Parkinsons DiseaseSeven PD patients, mean age 59.4 \\u00b1 9.8 years (range, 46-74) with a mean disease duration of 5.4 \\u00b1 2.8 years (range, 2-10) suffering from severe refractory tremor, underwent ventral intermediate nucleus thalamotomy using MRI guided focused ultrasound (MRgFUS), an innovative technology that enables noninvasive surgery.Transcranial magnetic resonance imaging-guided high-intensity focused ultrasound (MRgHIFU) is gaining attention as a potent substitute for surgical intervention in the treatment of neurologic disorders. To discern the neurophysiologic correlates of its therapeutic effects, we applied MRgHIFU to an intractable neurologic disorder, essential tremor, while measuring magnetoencephalogram mu rhythms from the motor cortex. Focused ultrasound sonication destroyed tissues by focusing a high-energy beam on the ventralis intermedius nucleus of the thalamus.BACKGROUND: Recent advances have enabled delivery of high-intensity focused ultrasound through the intact human cranium with magnetic resonance imaging (MRI) guidance. This preliminary study investigates the use of transcranial MRI-guided focused ultrasound thalamotomy for the treatment of essential tremor.METHODS: From February 2011 through December 2011, in an open-label, uncontrolled study, we used transcranial MRI-guided focused ultrasound to target the unilateral ventral intermediate nucleus of the thalamus in 15 patients with severe, medication-refractory essential tremor.MRI Guided Focused Ultrasound Thalamotomy for Moderate-to-Severe Tremor in Parkinson's Disease.132\\u2003A Randomized, Sham-Controlled Trial of Transcranial Magnetic Resonance-Guided Focused Ultrasound Thalamotomy Trial for the Treatment of Tremor-Dominant, Idiopathic Parkinson Disease.In this pilot study, essential tremor improved in 15 patients treated with MRI-guided focused ultrasound thalamotomy.A Randomized Trial of Focused Ultrasound Thalamotomy for Essential Tremor.MR-guided focused ultrasound thalamotomy for essential tremor: a proof-of-concept study.Unilateral magnetic resonance guided focused ultrasound thalamotomy for essential tremor: practices and clinicoradiological outcomes.Functional assessment and quality of life in essential tremor with bilateral or unilateral DBS and focused ultrasound thalamotomy.Transcranial magnetic resonance imaging-guided focused ultrasound: noninvasive central lateral thalamotomy for chronic neuropathic pain.This is a retrospective study of medication-refractory essential tremor patients treated at the University of Virginia with bilateral Vim DBS (n\\u2009=\\u200957), unilateral Vim DBS (n\\u2009=\\u200913), or unilateral focused ultrasound Vim thalamotomy (n\\u2009=\\u200915).Twenty-one consecutive patients suffering from chronic (mean disease duration 29.9\\u00a0years), therapy-resistant ET were treated with MRgFUS CTT.\"}, {\"question\": \"Which genes of the marmoset genome exhibit rapid sequence evolution?\", \"ideal_answer\": [\"Both protein-coding and microRNA genes related to reproduction exhibit evidence of rapid sequence evolution in the marmoset genome.\"], \"exact_answer\": [[\"protein-coding genes related to reproduction\"], [\"microRNA genes related to reproduction\"]], \"context\": \"We observed positive selection in growth hormone/insulin-like growth factor genes (growth pathways), respiratory complex I genes (metabolic pathways), and genes encoding immunobiological factors and proteases (reproductive and immunity pathways). In addition, both protein-coding and microRNA genes related to reproduction exhibited evidence of rapid sequence evolution.In addition, both protein-coding and microRNA genes related to reproduction exhibited evidence of rapid sequence evolution.In addition, both protein-coding and microRNA genes related to reproduction exhibited evidence of rapid sequence evolution. \"}, {\"question\": \"List selective estrogen receptor degraders.\", \"ideal_answer\": [\"Selective estrogen receptor degraders (SERD) are fulvestrant, RAD1901 and ARN-810. Fulvestrant is the only SERD approved for the treatment of breast cancer.\"], \"exact_answer\": [[\"fulvestrant\"], [\"RAD1901\"], [\"ARN-810\"]], \"context\": \"For women with hormone receptor-positive advanced breast cancer, endocrine therapies, including the selective estrogen receptor modulator tamoxifen, the aromatase inhibitors anastrozole, letrozole, and exemestane, and the selective estrogen receptor degrader fulvestrant, are recommended in clinical guidelines.Here we describe and characterize a novel, orally bioavailable small-molecule selective estrogen receptor degrader, RAD1901, and evaluate its therapeutic potential for the treatment of breast cancer.Identification of GDC-0810 (ARN-810), an Orally Bioavailable Selective Estrogen Receptor Degrader (SERD) that Demonstrates Robust Activity in Tamoxifen-Resistant Breast Cancer Xenografts.Fulvestrant is a steroid-based, selective estrogen receptor degrader (SERD) that both antagonizes and degrades ER-\\u03b1 and is active in patients who have progressed on antihormonal agents. Evaluation of the pharmacological activities of RAD1901, a selective estrogen receptor degrader.However, being the only SERD currently known to access the brain, RAD1901 merits evaluation as a targeted therapy for the treatment of breast cancer brain metastases.Tamoxifen, toremifene, and the selective estrogen receptor degrader fulvestrant are used to treat breast cancer, and tamoxifen and raloxifene protect against breast cancer in high-risk women.The turnover of estrogen receptor \\u03b1 by the selective estrogen receptor degrader (SERD) fulvestrant is a saturable process that is not required for antagonist efficacy.Currently, ICI 182,780 (ICI, fulvestrant) is the only SERD approved for the treatment of breast cancer. Fulvestrant is a steroid-based, selective estrogen receptor degrader (SERD) that both antagonizes and degrades ER-\\u00e1 and is active in patients who have progressed on antihormonal agents.Fulvestrant is a steroid-based, selective estrogen receptor degrader (SERD) that both antagonizes and degrades ER-\\u03b1 and is active in patients who have progressed on antihormonal agentsRAD1901: a novel, orally bioavailable selective estrogen receptor degrader that demonstrates antitumor activity in breast cancer xenograft models.RAD1901 selectively binds to and degrades the ER and is a potent antagonist of ER-positive breast cancer cell proliferation. The selective estrogen receptor degrader, fulvestrant, significantly reduced the anchorage-independent growth of ESR1 mutant-expressing cells, while combination treatments with the mTOR inhibitor everolimus, or an inhibitor blocking IGF1R, and the insulin receptor significantly enhanced anti-proliferative responses.SUM-44\\u00a0cells do not require exogenous estrogen for growth in\\u00a0vitro; however, they are dependent on ER\\u03b1 expression, as ESR1 knock-down or exposure to the selective estrogen receptor degrader fulvestrant resulted in growth inhibition.Evaluation of the pharmacological activities of RAD1901, a selective estrogen receptor degrader.Fulvestrant is a steroid-based, selective estrogen receptor degrader (SERD) that both antagonizes and degrades ER-\\u03b1 and is active in patients who have progressed on antihormonal agents.\"}, {\"question\": \"List all the available databases of super enhancers\", \"ideal_answer\": [\"dbSUPER and SEA are the available databases of super enhancers.\", \"dbSUPER and SEA are the most well-known and widely used Super-Enhancer Databases.\"], \"exact_answer\": [[\"dbSUPER\"], [\"SEA\"]], \"context\": \"dbSUPER: a database of super-enhancers in mouse and human genomeWe developed dbSUPER (http://bioinfo.au.tsinghua.edu.cn/dbsuper/), the first integrated and interactive database of super-enhancers, with the primary goal of providing a resource for assistance in further studies related to transcriptional control of cell identity and disease. dbSUPER provides a responsive and user-friendly web interface to facilitate efficient and comprehensive search and browsing. The data can be easily sent to Galaxy instances, GREAT and Cistrome web-servers for downstream analysis, and can also be visualized in the UCSC genome browser where custom tracks can be added automatically. The data can be downloaded and exported in variety of formats. Furthermore, dbSUPER lists genes associated with super-enhancers and also links to external databases such as GeneCards, UniProt and Entrez. dbSUPER also provides an overlap analysis tool to annotate user-defined regions. We believe dbSUPER is a valuable resource for the biology and genetic research communitiesSEA: a super-enhancer archiveHere, we present a specifically designed web-accessible database, Super-Enhancer Archive (SEA, http://sea.edbc.org). SEA focuses on integrating super-enhancers in multiple species and annotating their potential roles in the regulation of cell identity gene expression. The current release of SEA incorporates 83 996 super-enhancers computationally or experimentally identified in 134 cell types/tissues/diseases, including human (75 439, three of which were experimentally identified), mouse (5879, five of which were experimentally identified), Drosophila melanogaster (1774) and Caenorhabditis elegans (904). To facilitate data extraction, SEA supports multiple search options, including species, genome location, gene name, cell type/tissue and super-enhancer name. The response provides detailed (epi)genetic information, incorporating cell type specificity, nearby genes, transcriptional factor binding sites, CRISPR/Cas9 target sites, evolutionary conservation, SNPs, H3K27ac, DNA methylation, gene expression and TF ChIP-seq data. Moreover, analytical tools and a genome browser were developed for users to explore super-enhancers and their roles in defining cell identity and disease processes in depthSEA focuses on integrating super-enhancers in multiple species and annotating their potential roles in the regulation of cell identity gene expression. SEA incorporates 83 996 super-enhancers computationally or experimentally identified in 134 cell types/tissues/diseases, including human (75 439, three of which were experimentally identified), mouse (5879, five of which were experimentally identified), Drosophila melanogaster (1774) and Caenorhabditis elegans (904).We developed dbSUPER (http://bioinfo.au.tsinghua.edu.cn/dbsuper/), the first integrated and interactive database of super-enhancers, with the primary goal of providing a resource for assistance in further studies related to transcriptional control of cell identity and diseaseFurthermore, dbSUPER lists genes associated with super-enhancers and also links to external databases such as GeneCards, UniProt and Entrez. dbSUPER also provides an overlap analysis tool to annotate user-defined regionsWe developed dbSUPER (http://bioinfo.au.tsinghua.edu.cn/dbsuper/), the first integrated and interactive database of super-enhancers, with the primary goal of providing a resource for assistance in further studies related to transcriptional control of cell identity and disease. dbSUPER provides a responsive and user-friendly web interface to facilitate efficient and comprehensive search and browsingdbSUPER: a database of super-enhancers in mouse and human genome.\"}, {\"question\": \"Which are the consequences of the hyperphosphorylated tau in Alzheimers' Disease?\", \"ideal_answer\": [\"The consequences of the hyperphosphorylated tau in Alzheimers' Disease is:\\n1) The formation of neurofibrillary tangles (NFTs)\\n2) Impaired glutamate metabolism\\n3) Decreased tau affinity for microtubules binding\\n4) Dendritic and axonal instability\\n5) Synaptic degeneration\\n6) Neuronal loss.\"], \"exact_answer\": [[\"Neurofibrillary tangles formatio\", \"NFTs\", \"NFT formation\"], [\"Impaired glutamate metabolism\"], [\"Decreased affinity for microtubules binding\"], [\"Dendritic instability\"], [\"Axonal instability\"], [\"Synaptic degeneration\"], [\"Neuronal loss\"]], \"context\": \"The formation of neurofibrillary tangles from the assembly of hyperphosphorylated tau leads to dendritic and axonal instability, synaptic degeneration, and neuronal loss. Glutamate metabolism is impaired in transgenic mice with tau hyperphosphorylation.In neurodegenerative diseases including Alzheimer's disease and frontotemporal dementia, the protein tau is hyperphosphorylated and eventually aggregates to develop neurofibrillary tangles. Neurofibrillary tangles (NFTs), a marker of neuronal alterations in Alzheimer's disease (AD) and other tauopathies, are comprised of aggregates of hyperphosphorylated tau protein.At 24 months, NFTs progress, tau inclusions propagate to the dentate gyrus, and neuronal loss is evident. At autopsy, brains of AD and a subset of TBI victims display some similarities including accumulation of A\\u03b2 peptides and neurofibrillary tangles of hyperphosphorylated tau proteins. The pathological hallmarks of Alzheimer's disease (AD)--widespread synaptic and neuronal loss and the pathological accumulation of amyloid-beta peptide (A\\u03b2) in senile plaques, as well as hyperphosphorylated tau in neurofibrillary tangles--have been known for many decades, but the links between AD pathology and dementia and effective therapeutic strategies remain elusive. The microtubule-associated protein tau, in a hyperphosphorylated form, aggregates into insoluble paired-helical filaments (PHFs) in Alzheimer's disease (AD) and other tauopathies.Pseudohyperphosphorylation causing AD-like changes in tau has significant effects on its polymerization.In AD, kinases such as glycogen synthase kinase-3beta (GSK-3beta) are believed to be involved in the generation of hyperphosphorylated tau. However, the functional consequences of hyperphosphorylation on the microtubule binding and polymerization of tau are not well understood. Pseudophosphorylation of tau decreases its affinity for microtubules, and pseudohyperphosphorylated forms of tau do not have significantly decreased levels of microtubule binding as compared to single and double sites. On the basis of the observations that pseudohyperphosphorylated tau has decreased affinity for microtubules and reduced inducer-initiated rates of nucleation and polymerization, we propose that this combination could be the cause of the increased cytotoxicity of hyperphosphorylated tau in Alzheimer's disease and also explain the potentially beneficial role of tau polymerization and NFT formation.Microtubule-associated protein tau is abnormally hyperphosphorylated and aggregated in affected neurons of Alzheimer disease brain.In several neurodegenerative diseases, including Alzheimer disease, the neuronal microtubule-associated protein tau becomes hyperphosphorylated, accumulates in the somatodendritic compartment, and aggregates into insoluble filaments.Tau is the major component of the neurofibrillar tangles that are a pathological hallmark of Alzheimers' disease.Aggregates of hyperphosphorylated tau protein are found in a group of diseases called tauopathies, which includes Alzheimers diseaseIn several neurodegenerative diseases, including Alzheimer disease, the neuronal microtubule-associated protein tau becomes hyperphosphorylated, accumulates in the somatodendritic compartment, and aggregates into insoluble filamentsOn the basis of the observations that pseudohyperphosphorylated tau has decreased affinity for microtubules and reduced inducer-initiated rates of nucleation and polymerization, we propose that this combination could be the cause of the increased cytotoxicity of hyperphosphorylated tau in Alzheimers disease and also explain the potentially beneficial role of tau polymerization and NFT formationIn the brains of individuals with Alzheimers disease, tau is hyperphosphorylated and aggregated into intraneuronal deposits called neurofibrillary tangles (NFTs)Neurofibrillary tangles and neuropil threads, both made of hyperphosphorylated tau proteins, point to an alteration of microtubules in Alzheimers diseaseHyperphosphorylation, which diminishes the microtubule-binding capacity of tau, destabilizes microtubules and may enhance the formation of paired helical filaments that constitute neurofibrillary tangles in Alzheimer's disease. In several neurodegenerative diseases, including Alzheimer disease, the neuronal microtubule-associated protein tau becomes hyperphosphorylated, accumulates in the somatodendritic compartment, and aggregates into insoluble filaments. The consequences of the accumulation of hyperphosphorylated tau in the somatodendritic compartment remain poorly characterized at the early stage of disease before the formation of tau insoluble filaments.Microtubule-associated protein tau is abnormally hyperphosphorylated and aggregated in affected neurons of Alzheimer disease brain. This hyperphosphorylated tau can be dephosphorylated at some of the abnormal phosphorylated sites by purified protein phosphatase-1, 2A, and 2B in vitro.Adult human nerve cells contain tau protein, a phosphorylated microtubule-associated protein, that is hyperphosphorylated in the fetus and in patients with Alzheimer's disease. Hyperphosphorylation, which diminishes the microtubule-binding capacity of tau, destabilizes microtubules and may enhance the formation of paired helical filaments that constitute neurofibrillary tangles in Alzheimer's disease.Importantly, hyperphosphorylated tau comprises the paired helical filament, whose pathological deposition as neurofibrillary tangles is implicated in Alzheimer's disease; hyperphosphorylated tau is also implicated in the pathogenesis of other neurodegenerative disorders.In Alzheimer disease (AD) brain tau is ~three to four-fold more hyperphosphorylated than the normal adult brain tau and in this hyperphosphorylated state it is polymerized into paired helical filaments ([PHF) admixed with straight filaments (SF) forming neurofibrillary tangles.\"}, {\"question\": \"Which diseases are involved in the severe cutaneous reactions (SCAR) spectrum?\", \"ideal_answer\": [\"The diseases that are involved in the severe cutaneous reactions (SCAR) spectrum are:\\n1) Stevens-Johnson syndrome (SJS)\\n2) Toxic epidermal necrolysis (TEN)\\n3) Acute generalized exanthematous pustulosis (AGEP).\"], \"exact_answer\": [[\"Stevens-Johnson syndrome\", \"SJS\"], [\"Toxic epidermal necrolysis\", \"TEN\"], [\"Acute generalized exanthematous pustulosis\", \"AGEP\"]], \"context\": \"Stevens-Johnson Syndrome (SJS) and toxic epidermal necrolysis (TEN) are diseases within the spectrum of severe cutaneous adverse reactions affecting skin and mucous membranes. Toxic epidermal necrolysis (TEN) and Stevens Johnson Syndrome (SJS) are severe adverse cutaneous drug reactions that predominantly involve the skin and mucous membranes.Currently, TEN and SJS are considered to be two ends of a spectrum of severe epidermolytic adverse cutaneous drug reactions, differing only by their extent of skin detachment. Toxic epidermal necrolysis (TEN) and Stevens-Johnson syndrome (SJS) are considered part of a spectrum of adverse cutaneous drug reactions showing severe and extensive skin detachment.Our aim was to study patch testing in severe cutaneous adverse drug reactions (ADRs) (Stevens-Johnson syndrome/toxic epidermal necrolysis (SJS/TEN), acute generalized exanthematous pustulosis (AGEP), and other cutaneous ADRs)However, allopurinol is also one of the most common causes of severe cutaneous adverse reactions (SCARs), which include drug hypersensitivity syndrome, Stevens\\u2013Johnson syndrome, and toxic epidermal necrolysis.Drug eruptions range from transient erythema to the life threatening severe cutaneous adverse reactions (SCAR) that encompass Stevens-Johnson syndrome (SJS), toxic epidermal necrolysis (TEN), acute generalized exanthematous pustulosis (AGEP) and drug reaction with eosinophilia and systemic symptoms complex (DRESS).To study the clinical and epidemiological aspects of cutaneous adverse drug reactions (CADR).Ethical clearance was obtained from the institutional ethics committeeStevens-Johnson syndrome (SJS), toxic epidermal necrolysis (TEN), and drug reaction with eosinophilia and systemic symptoms (DRESS) are severe cutaneous adverse reactions (SCAR) which are majorly caused by drugsThis study investigated the association between the HLA class I genotype and carbamazepine-induced severe cutaneous adverse reaction (SCAR) in Koreans.Twenty-four patients who had developed carbamazepine-induced SCAR (7 Stevens-Johnson syndrome (SJS), 17 drug hypersensitivity syndrome (HSS)), 50 carbamazepine-tolerant controls from the Korean Pharmacogenetic Adverse Drug Reaction Research Network and data of 485 Korean general population from a previously published study were recruitedSevere cutaneous adverse reactions (SCARs) include Stevens-Johnson syndrome (SJS), toxic epidermal necrolysis (TEN), and drug reaction with eosinophilia and systemic symptoms (DRESS)Severe Cutaneous Adverse Reaction (SCAR) represents the spectrum of adverse drug reactions from erythema multiforme, Stevens - Johnson syndrome (SJS) to Toxic Epidermal Necrolysis (TEN). Toxic epidermal necrolysis (TEN) and Stevens-Johnson syndrome (SJS) are uncommon but extremely serious and often life-threatening mucocutaneous reactions characterized by extensive epithelial sloughing and systemic symptoms. There is no effective evidence-based treatment for severe cutaneous adverse reactions (SCAR) to drugs and no consensus on how to treat these patients.However, allopurinol is also one of the most common causes of severe cutaneous adverse reactions (SCARs), which include drug hypersensitivity syndrome, Stevens\\u2013Johnson syndrome, and toxic epidermal necrolysis. A variant allele of the human leukocyte antigen (HLA)-B, HLA-B*58:01, associates strongly with allopurinolinduced SCAR.The skin is often involved in ADRs and although most cutaneous ADRs have a favorable course, they may present as severe adverse cutaneous drug reactions (SCARs), such as Stevens-Johnson syndrome, toxic epidermal necrolysis, drug reaction with eosinophilia and systemic symptoms (also referred to as drug-induced hypersensitivity syndrome), and acute generalized exanthematous pustulosis.By comparison, Stevens-Johnson syndrome (SJS) is a life-threatening severe cutaneous adverse reaction (SCAR), mainly caused by drugs.The study sought to identify the magnitude and characteristic of severe cutaneous adverse reactions (SCAR's) like Steven-Johnson syndrome (SJS) and Toxic Epidermal Necrolysis (TEN).Allopurinol, an antihyperuricaemic agent, is one of the common causes of life-threatening severe cutaneous adverse reactions (SCAR), including drug rash with eosinophilia and systemic symptoms (DRESS), Stevens-Johnson syndrome (SJS) and toxic epidermal necrosis (TEN).\"}, {\"question\": \"For what indications is thalidomide currently marketed?\", \"ideal_answer\": [\"Drug repositioning, exemplified by sildenafil and thalidomide, is a promising way to explore alternative indications for existing drugs. THE USE OF A DRUG WITH A TEMPORARY MARKETING AUTHORISATION: Thalidomide is currently available in France for nominative or cohort use with a temporary marketing authorisation (TMA). Currently, it is used for a few indications; in Brazil, where leprosy is endemic, thalidomide is used for the treatment of erythema nodosum leprosum, and recent cases of thalidomide embryopathy have been reported.We analyzed the frequency of births with phenotypes consistent with thalidomide embryopathy (TEP) and correlated this with the distribution of thalidomide and the prevalence of leprosy between 2005 and 2010 in Brazil.A total of 5,889,210 thalidomide tablets were distributed; the prevalence of limb reduction defects was 1.60 (CI95%: 1.54-1.66) and TEP was 0.11 (CI95%: 0.10-0.13) Currently it includes a group of new drugs (immunosuppressives tacrolimus mycophenolate, thalidomide, biologic therapy, probiotics, neuroinflammation blockers), new treatment techniques (cytaphereses, sequential immunosuppression, immunosuppression with high doses), and finally new indications (chemoprophylaxis). A review of the therapeutic indications for thalidomide in dermatology as well as the mechanisms of action and side-effects of this drug are presented.\", \"Thalidomide can be used to treat multiple myeloma, polyneuropathy, organomegaly, endocrinopathy, M-protein and skin changes (POEMS) syndrome and possibly Irritable Bowel Syndrome.\"], \"exact_answer\": [[\"Multiple myeloma\"], [\"polyneuropathy, organomegaly, endocrinopathy, M-protein and skin changes\", \"POEMS\"], [\"Irritable Bowels Syndrome \", \"IBS\"]], \"context\": \"In this retrospective study, pharmacy claims were analyzed for those patients with a diagnosis of MM who received thalidomide, lenalidomide, or pomalidomide from a large specialty pharmacy in the US between January 1, 2011, and December 31, 2013.Polyneuropathy, organomegaly, endocrinopathy, M-protein and skin changes (POEMS) syndrome is a fatal systemic disorder associated with plasma cell dyscrasia and the overproduction of the vascular endothelial growth factor (VEGF). Recently, the prognosis of POEMS was substantially improved by introduction of therapeutic intervention for myeloma. However, no randomised clinical trial has been performed because of the rarity and severity of the disease.METHODS AND ANALYSIS: The Japanese POEMS syndrome with Thalidomide (J-POST) Trial is a phase II/III multicentre, double-blinded, randomised, controlled trial that aims to evaluate the efficacy and safety of a 24-week treatment with thalidomide in POEMS syndromeThalidomide could relieve clinical symptoms and intestinal mucosal lesions effectively in children with refractory inflammatory bowel disease (IBD) from the pre-clinical study.In reactional states of leprosy the use of thalidomide is established.The use of thalidomide was never discontinued in Brazil where it is prescribed for leprosy type 2 reaction.New uses of thalidomide.Currently, it is used for a few indications; in Brazil, where leprosy is endemic, thalidomide is used for the treatment of erythema nodosum leprosum, and recent cases of thalidomide embryopathy have been reported.We analyzed the frequency of births with phenotypes consistent with thalidomide embryopathy (TEP) and correlated this with the distribution of thalidomide and the prevalence of leprosy between 2005 and 2010 in Brazil.A total of 5,889,210 thalidomide tablets were distributed; the prevalence of limb reduction defects was 1.60 (CI95%: 1.54-1.66) and TEP was 0.11 (CI95%: 0.10-0.13) per 10,000 births.The comeback of thalidomide to the legitimate status of a marketed drug came in 1998 when it received FDA approval for the treatment of erythema nodosum leprosum (ENL)To explore the inhibitory effect of thalidomide combined with interferon (IFN) on the human acute myeloid leukemia cell line Kasumi- 1 and its mechanism.The inhibitiory effect of Kasumi- 1 cells by thalidomide, interferon or combination was detected by CCK- 8 method, the apoptosis by flow cytometry, the expression of apoptosis related proteins by Western blot, vascular endothelial growth factor (VEGF) concentration in culture supernatant by ELISA.Thalidomide inhibited the proliferation of Kasumi- 1 in a dose- dependent manner from 50 \\u03bcg/ml to 500 \\u03bcg/ml with an IC\\u2085\\u2080 of (451.Interferon (INF)-\\u03b1 was the maintenance treatment of choice after autologous stem cell transplantation in multiple myeloma in the past, but currently Thalidomide is commonly usedHowever, no randomised clinical trial has been performed because of the rarity and severity of the disease.The Japanese POEMS syndrome with Thalidomide (J-POST) Trial is a phase II/III multicentre, double-blinded, randomised, controlled trial that aims to evaluate the efficacy and safety of a 24-week treatment with thalidomide in POEMS syndrome, with an additional 48-week open-label safety studyThalidomide, mainly used for the treatment of leprosy, is a current teratogen in South America, and it is reasonable to assume that at present this situation is affecting many births in underdeveloped countries. Currently, it is used for a few indications; in Brazil, where leprosy is endemic, thalidomide is used for the treatment of erythema nodosum leprosum, and recent cases of thalidomide embryopathy have been reported.We analyzed the frequency of births with phenotypes consistent with thalidomide embryopathy (TEP) and correlated this with the distribution of thalidomide and the prevalence of leprosy between 2005 and 2010 in Brazil.A total of 5,889,210 thalidomide tablets were distributed; the prevalence of limb reduction defects was 1.60 (CI95%: 1.54-1.66) and TEP was 0.11 (CI95%: 0.10-0.13)A review of the therapeutic indications for thalidomide in dermatology as well as the mechanisms of action and side-effects of this drug are presented.Thalidomide first was marketed as a sedative in the 1950s and withdrawn from the market in 1961 following reports of teratogenicity. Later, it was used as an investigational agent for the treatment of Hansen's disease, Kaposi's sarcoma, myelofibrosis, RAUs, and wasting associated with HIV.The drug has since been found effective for several different indications.Based on present publications we review indications of the therapy of dermatoses with thalidomide as well as possible mechanisms of action and side effects of this drug.Use of thalidomide in dermatological indications.Current data demonstrates that thalidomide is clinically promising in multiple myeloma, glioblastoma multiforme and renal cell cancer.Erythema nodosum leprosum is the only registered indication for the use of thalidomide in the United States of America.Thalidomide is currently under investigation for the treatment of a wide variety of diseases, including conditions thought to have an inflammatory or immune basis, malignancies and complications of infection with HIV.Gradually, thalidomide was reintroduced for the treatment of a few skin diseases including leprous erythema nodosum, severe mucosal ulcers (e.g., associated with HIV infection or Beh\\u00e7et's disease), lymphocytic skin infiltrations, cutaneous lupus erythematosus, and chronic graft-versus-host diseaseThalidomide has been used in several cutaneous inflammatory disorders (such as erythema nodosum leprosum in lepromatous leprosy, cutaneous lupus erythematosus, severe aphtosis), cancers (relapsed/refractory multiple myeloma) and inflammatory conditions.This drug was more recently rediscovered as a powerful immunomodulatory and antiinflammatory agent and was approved by the FDA in 1998 for treatment of erythema nodosum leprosum. Since, however, it has been found to be an effective drug in erythema nodosum leprosum\"}, {\"question\": \"Which human syndromes have been detected with Fluorescence in situ hybridization (FISH)?\", \"ideal_answer\": [\"To explore the clonal evolution of monosomy 7 in patients with aplastic anemia (AA).Monosomy 7 (-7) in 81 AA patients with normal karyotype at diagnosis and 46 AA treated with immunosuppressive therapy (IST) and more than 6 months of recombinant human granulocyte colony-stimulating factor (rhuG-CSF) were detected by interphase- fluorescence in situ hybridization (FISH) retrospectively.There were 5.4% - 7.6% of -7 cells in 11 (13.6%) of 81 patients at diagnosis, the survival and response rate to IST in -7 positive patients did not differ s Former molecular cytogenetic studies showed that such aberrations as an equivalent of intrinsic radiosensitivity can be detected by fluorescence in-situ hybridization (FISH) techniques using whole chromosome painting (wcp) probes. Five-color FISH was performed using human DNA segments specific for peri-CEN or subTEL, which were labeled with five different fluorescent dyes [7-diethylaminocoumarin (DEAC): blue, fluorescein isothiocyanate (FITC): green, rhodamine 6G (R6G): yellow, TexRed: red, and cyanine5 (Cy5): purple]. Using this technique, followed by cell-by-cell multicolor fluorescence in situ hybridization (FISH) analysis of purified FNRBCs, we were able to detect some of the most common human aneuploidies (including Down syndrome, Klinefelter syndrome, and trisomy 13) in 33 pregnant women referred for amniocentesis.\", \"The human syndromes which can be detected with FISH are:\\n1) Myelodysplastic Syndrome (MDS)\\n2) Genetic syndromes caused by somatic chromosomal mosaicism\\n3) Prader Willi syndrome (PWS)\\n4) Angelman syndrome (AS)\\n5) Williams syndrome\\n6) Smith Magenis syndrome \\n7) Velocardiofacial syndrome\\n8) Jacobsen syndrome\\n9) Shwachman-Diamond syndrome\\n10) Saethre-Chotzen syndrome \\n11) DiGeorge syndrome (DGS) \\n12) Velo-cardio-facial syndrome (VCFS)\\n13) Miller-Dieker syndrome\\n14) Deletion 22q11.2 syndrome (D22S)\"], \"exact_answer\": [[\"Myelodysplastic Syndrome\", \"MDS\"], [\"Genetic syndromes caused by somatic chromosomal mosaicism\"], [\"Prader Willi syndrome\", \"PWS\"], [\"Angelman syndrome\", \"AS\"], [\"Williams syndrome\"], [\"Smith Magenis syndrome\"], [\"Velocardiofacial syndrome\"], [\"Jacobsen syndrome\"], [\"Shwachman-Diamond syndrome\"], [\"Saethre-Chotzen syndrome\"], [\"DiGeorge syndrome\", \"DGS\"], [\"Velo-cardio-facial syndrome\", \"VCFS\"], [\"Miller-Dieker syndrome\"], [\"Deletion 22q11.2 syndrome\", \"D22S\"], [\"Down syndrome\", \"Trisomy 21\", \"DS\", \"DNS\"], [\"Klinefelter Syndrome\", \"47\", \"XXY\", \"KS\"], [\"Trisomy 13\", \"Patau Syndrome\"]], \"context\": \"We demonstrate that, using a sensitive FISH technique, 12p deletion occurs significantly more frequently in MDS than previously described (7.6% by CD34+ PB-FISH vs. 1.6% by CBA, P<0.001) and is often associated with other aberrations (93% by CD34+ PB-FISH vs. 60% by CBA)Somatic chromosomal mosaicism arising from post-zygotic errors is known to cause several well-defined genetic syndromes as well as contribute to phenotypic variation in diseases.The mosaicism was characterized and confirmed by fluorescence in situ hybridization (FISH) and/or chromosome analysis. Different categories of abnormal cell lines were detected: (1) aneuploidy, including sex chromosome abnormalities and isochromosomes (22 cases), (2) ring or marker chromosomes (12 cases), (3) single deletion/duplication copy number variations (CNVs) (11 cases), (4) multiple deletion/duplication CNVs (5 cases), (5) exonic CNVs (4 cases), and (6) unbalanced translocations (3 cases). Positive cases with MLPA technique were confirmed using either fluorescence in situ hybridization (FISH) or follow up confirmatory MLPA probe sets.The deletions/duplications detected in non telomeric regions include regions for Prader Willi/Angelman regions, Williams syndrome, Smith Magenis syndrome and Velocardiofacial syndrome.The bone marrow specimen from a patient with MDS was comprehensively analyzed by a combination of genomic approaches, including chromosomal karyotyping, fluorescence in situ hybridization (FISH), genome scanning using Affymetrix high density SNP microarray platform, and next-generation sequencing (NGS) analysis using IonTorrent Cancer Gene Panel.Detection of TET2 abnormalities by fluorescence in situ hybridization in 41 patients with myelodysplastic syndrome.Fluorescence in situ hybridization of TP53 for the detection of chromosome 17 abnormalities in myelodysplastic syndromes.Fluorescence in situ hybridization (FISH) can effectively demonstrate TET2 deletions and is often used to validate molecular results. These findings demonstrate that 1) FISH is an effective and economical method to reveal cryptic abnormalities of band 4q22-q24 resulting in TET2 deletions;FISH and SNP-A karyotyping in myelodysplastic syndromes: improving cytogenetic detection of del(5q), monosomy 7, del(7q), trisomy 8 and del(20q). Using a standardized approach, we focused our investigation on detection of -5/del(5q), -7/del(7q), trisomy 8 and del(20q) in patients with MDS (N=52), MDS/myeloproliferative overlap syndromes (N=7) and acute myeloid leukemia (N=15) using MC, FISH and SNP-A karyotyping. We conclude that MC, FISH and SNP-A are complementary techniques that, when applied and interpreted together, can improve the diagnostic yield for identifying genetic lesions in MDS and contribute to the better description of abnormal karyotypes.Here, we report the detection of a reciprocal translocation t(7;21)(p22;q22) in the marrow of two adults with MDS and AML, using conventional cytogenetic analysis and fluorescence-in situ-hybridization (FISH).Detection of cytogenetic abnormalities involving chromosomes 5,7 and 8 in myelodysplastic syndromes with fluorescence in situ hybridization and its clinical significanceTo identify the abnormal karyotypes by fluorescence in situ hybridization (FISH) and explore prognostic implications in patients with myelodysplastic syndromes (MDS).FISH was used to detect the frequently occurring chromosome abnormalities (-5/5q, +8, -7/7q-) in 37 MDS cases. FISH detected monosomy 7 in 6 samples at the time MDS was diagnosed and in 2 samples at the time AA was diagnosed.As chromosomal instability may contribute to leukemogenesis in patients with congenital bone marrow failure (CBMF) disorders, it was the aim of this study to characterize chromosomally aberrant clones that arise during the clinical course of disease by means of R-banding and fluorescence in situ hybridization (FISH) analyses.In one child with congenital thrombocytopenia, Jacobsen syndrome [del(11)(q24)c] was diagnosed, and thus a CBMF could be excluded. In a girl with Shwachman-Diamond syndrome, two independent clones, one with an isochromosome i(7)(q10), another with a complex aberrant karyotype, were identified. Simultaneously, transition into a myelodysplastic syndrome (MDS) occurred. The brother, who was also afflicted with Shwachman-Diamond syndrome, showed an isochromosome i(7q) as a single aberration. In summary, we suggest that follow-up of patients with CBMF using chromosome and FISH analyses will be helpful for the early detection of transition into MDS or AML and thus should be an integral part of the clinical management of these patients.Here we assess this capability by applying array CGH to the analysis of copy number alterations in 44 patients with a phenotype of the 22q11.2 deletion syndrome. Twenty-five patients had the deletion on chromosome 22 characteristic of this syndrome as determined by fluorescence in situ hybridization (FISH). Subtelomeric aberrations previously detected by fluorescence in situ hybridization (FISH) analysis were confirmed in two patients, and accurate diagnosis was provided in two previously undiagnosed complex cases. Microdeletions at 15q11.2-q13 in a newborn with hypotonia, cryptorchidism, and hypopigmentation were detected with few discrepancies between the array results and FISH analysis. Contiguous microdeletion of GSCL, HIRA and TBX1 genes at 22q11.2 was identified in a previously undiagnosed boy with an unusual presentation of the VCF/DiGeorge spectrum. Detection of 20q(-) chromosome abnormality in myelodysplastic syndrome by interphase fluorescence in situ hybridizationThe results showed that among 52 cases of MDS, 7 (13.5%) cases were positive by FISH, however, of which, 4 cases were positive and the other 3 cases were negative by CC assay. It is concluded that YAC912C3 and interphase FISH providing a powerful technique in the detection of 20q(-) in MDS is an important complement to CC assay.A FISH assay was performed to analyze 70 AML/MDS patients who had received conventional cytogenetic analysis (CCA). FISH is a powerful tool to identify or refine chromosomal structural aberrations involving 7q, and it provides accurate evaluation of -7/7q- in all the patients. -7 and 7q- clone frequently coexist in the same specimen, and the significantly increasing percentage of 7q- cells implies that -7 clone secondary to 7q- clone is a result from loss of 7q-.Saethre-Chotzen syndrome is a common craniosynostosis syndrome characterized by craniofacial and limb anomalies.We propose that initial screening for the FGFR3 P250R mutation, followed by sequencing of TWIST and then fluorescence in situ hybridization (FISH) for deletion detection of TWIST, is sufficient to detect mutations in>80% of patients with the Saethre-Chotzen phenotype.We clearly present that the use of SKY combined with conventional G-banding analysis and FISH has assisted in the identification of important chromosomal events that may play a key role in the development of t-MDS.n this study, we used spectral karyotyping (SKY) and fluorescence in situ hybridization (FISH) as complementary techniques for the analysis of two therapy-related secondary myelodysplastic syndrome (t-MDS) cases with complex karyotypes, previously analyzed by G-banding. Comparison of detection of trisomy 8 with fluorescence in situ hybridization and conventional karyotype analysis in myelodysplastic syndromeThe trisomy 8 clones were simultaneously detected in 48 MDS cases with FISH and conventional cytogenetic analysis (CCA).Trisomy 8 was detected in 1 of 15 specimens with normal or abnormal karyotype without trisomy 8 by FISH.In conclusion, our results showed that FISH is a sensitive and accurate technique to detect trisomy 8 in MDS patients. It can provide contribute to diagnosis, assessment of curative effect and predicting progress of disease in MDS. Detection of common chromosome abnormalities in myelodysplastic syndrome with a panel fluorescence in situ hybridizationPanel FISH is a useful tool of molecular cytogenetics in the detection of common chromosome abnormalities in MDS.Among 20 cases, 13 cases were found to carry common chromosome abnormalities by panel FISH (trisomy 8, five cases; -5/5q-, one case; 20q-, five cases; 5q- accompanying 20q-, one case; complex abnormalities, one case)Detection of -5/5q- chromosome abnormality in myelodysplastic syndromes by interphase fluorescence in situ hybridizationIn 48 MDS patients, 13 were positive for interphase FISH, of whom, 7 were positive and 6 were negative for conventional cytogenetics (CC).Interphase FISH is more sensitive than CC for the detection of -5/5q- in MDS.Three cases displayed -7/7q- by conventional cytogenetics(CC) and were confirmed by interphase FISH. Six cases in 43 cases who did not show -7/7q- by CC displayed -7/7q- by interphase FISH.CONCLUSION: Interphase FISH is very useful for the detection of -7 or 7q- in MDS and it is more sensitive than CC.Diagnosis of microdeletion syndromes by fluorescence in situ hybridization (FISH).Detection of trisomy 8 with interphase fluorescence in situ hybridization in myelodysplastic syndromesInterphase FISH was a useful method for the detection of trisomy 8 in MDS, especially in patients with normal karyotype or marker chromosome. Congenital cardiovascular (c-v) malformations are the leading signs of two syndromes of highly variable phenotypes, the DiGeorge syndrome (DGS) and the velo-cardio-facial syndrome (VCFS), both of which in the majority of cases are caused by microdeletion in the chromosome region 22q11.2. In patients with congenital c-v and associated malformations of dysmorphism microdeletion diagnosis of 22q11.2 by FISH is indicated in addition to conventional cytogenetic testing. Discordant detection of monosomy 7 by GTG-banding and FISH in a patient with Shwachman-Diamond syndrome without evidence of myelodysplastic syndrome or acute myelogenous leukemia.Fluorescence in situ hybridization of progenitor cells obtained by fluorescence-activated cell sorting for the detection of cells affected by chromosome abnormality trisomy 8 in patients with myelodysplastic syndromes.We conclude that the combination of FACS/FISH/BUdR, which determines the number, phenotype and proliferation rate of very rare leukaemic cells in patients with AML or MDS in clinical remission, provides information that is useful in the identification of patients with high and low likelihood of relapse.FISH detection of chromosome 15 deletions in Prader-Willi and Angelman syndromes.We conclude that FISH analysis is a rapid and reliable method for detection of deletions within 15q11-q13 and whenever a deletion is found, FISH analysis of parental chromosomes should also be considered.We have evaluated fluorescence in situ hybridization (FISH) analysis for the clinical laboratory detection of the 15q11-q13 deletion seen in Prader-Willi syndrome (PWS) and Angelman syndrome (AS) using probes for loci D15S11, SNRPN, D15S10, and GABRB3. Study of clonality in myelodysplastic syndromes: detection of trisomy 8 in bone marrow cell smears by fluorescence in situ hybridization.Investigations with fluorescence in situ hybridization (FISH) demonstrate loss of the telomeres on the reciprocal chromosome in three unbalanced translocations involving chromosome 15 in the Prader-Willi and Angelman syndromes.Fluorescence in situ hybridization (FISH) was used for the detection of chromosome 15(q11-13) deletions (with probes from the PWS/AS region) and to define the involvement of the telomere in the derivative chromosomes (with library probes and telomere-specific probes).Fluorescence in situ hybridization improves the detection of monosomy 7 in myelodysplastic syndromes.Detection of monosomy 7 by fluorescence in situ hybridization in acute nonlymphocytic leukemia and myelodysplastic syndrome.Fluorescence in situ hybridization (FISH) with a chromosome 7 specific alpha satellite DNA probe was used to detect monosomy 7 in interphase and metaphase cells obtained from patients with myelodysplastic syndrome (MDS) and acute nonlymphocytic leukemia (ANLL).A 15-year-old male with myelodysplastic syndrome (MDS) characterized by monosomy 7 was cytogenetically evaluated by metaphase karyotyping and fluorescence in situ hybridization (FISH) of interphase cells at six different points during the course of his disease.Fluorescence in situ hybridization (FISH) using two cosmid probes (41A and P13) from the Miller-Dieker syndrome (MDS) critical region in 17p13.3 was performed in a blinded comparison of three MDS patients with submicroscopic deletions and in four normal relatives used as controls.These studies demonstrate that in situ hybridization is an efficient method for deletion detection in Miller-Dieker syndrome. This study identified cytogenetic abnormalities in a population screened for deletion 22q11.2 syndrome (D22S) by fluorescence in situ hybridization (FISH) and G-banding and correlated these abnormalities to referring specialty and submitted indications. Positive test results for D22S FISH and other abnormalities found by other FISH assays and G-banding were correlated to submitting specialist and indicationUse of fluorescence in situ hybridization (FISH) in the diagnosis of DiGeorge sequence and related diseases.Deletions of 22q11.2 have been detected in the majority of patients with DiGeorge, velocardiofacial, and conotruncal anomaly face syndromes by either cytogenetic analysis, fluorescence in situ hybridization (FISH), or Southern blot hybridization.Fluorescence in situ hybridization improves the detection of 5q31 deletion in myelodysplastic syndromes without cytogenetic evidence of 5q-.To explore the value of multiplex fluorescence in situ hybridization (M-FISH) technique in the detection of the complex chromosomal aberrations (CCAs) in myelodysplastic syndromes (MDS).M-FISH was used in ten MDS patients with R-banding CCAs to refine the complex chromosomal rearrangements, the constitute of marker chromosomes, and to identify the cryptic translocations.Thirty-seven kinds of structural rearrangements were detected by M-FISH including insertion, deletion, translocation and derivative chromosomes, among which 34 kinds were unbalanced rearrangements, and 3 were balanced rearrangements including t(6;22) (q21; q12), t(9; 19) (q13; p13) and t(3;5) (?; ?).To explore the clonal evolution of monosomy 7 in patients with aplastic anemia (AA).Monosomy 7 (-7) in 81 AA patients with normal karyotype at diagnosis and 46 AA treated with immunosuppressive therapy (IST) and more than 6 months of recombinant human granulocyte colony-stimulating factor (rhuG-CSF) were detected by interphase- fluorescence in situ hybridization (FISH) retrospectively.There were 5.4% - 7.6% of -7 cells in 11 (13.6%) of 81 patients at diagnosis, the survival and response rate to IST in -7 positive patients did not differ significantly from that in -7 negative patients (P = 0.To compare the results of fluorescence in situ hybridization (FISH) versus conventional cytogenetics (CC) in the detection of common chromosomal abnormalities and evaluate the significance of FISH in myelodysplastic syndrome (MDS) .A total of 344 patients with de novo MDS from June 2008 to October 2012 were detected by 6 pairs of probes, including CSF1R/D5S23-D5S721 (5q33) , EGR1/ D5S23-D5S721 (5q31) , D7S486 (7q31) /CSP7, D7S522 (7q31) /CSP7, D20S108/CSP8 (20q12/CSP8) and CSPX/CSPYUnbalanced translocation, t(18;21), detected by fluorescence in situ hybridization (FISH) in a child with 18q- syndrome and a ring chromosome 21To evaluate the conventional cytogenetic methods in genetic diagnosis and prenatal diagnosis in the family with a proband of Angelman syndrome (AS).High-resolution G-banding karyotyping and fluorescence in situ hybridization (FISH) on metaphase chromosomes were performed.Two AS patients and 1 normal fetus in the family were successfully detected by FISH.Our result demonstrated that patient with type I AS could be detected by combining the techniques of high-resolution G-banding and FISH with clinical observation, which would offer accurate genetic counseling information to the geneticists and provide the prenatal diagnosis for the AS familyTo investigate the prenatal diagnosis of trisomy 21 syndrome using chromosome 13/21 alpha satellite probe fluorescence in situ hybridization (FISH) on uncultured interphase cells from amniotic fluid.The interphase amniocytes of 10 fetuses who were detected normal and 3 fetus who were detected trisomy by prenatal cytogenetic diagnosis were selectedTo optimize the prenatal diagnosis platform by using domestically made fluorescence in situ hybridization(FISH) kit and to explore the clinical application of FISH to rapid prenatal diagnosis of a wide range of chromosomal abnormalities.Amniotic fluid samples from 110 pregnant women were studied with the rapid prenatal diagnosis method of FISH and the conventional cell culture method of karyotyping, the results from both methods were compared.Four cases of trisomy 21, 1 case of trisomy 18, 58 cases of 46, XX, and 47 cases of 46, XY were detected by FISH in the 110 amniotic fluid samples.Implementation of Fluorescent in situ hybridization (FISH) as a method for detecting microdeletion syndromes - our first experiences.Prenatal detection of cri du chat syndrome on uncultured amniocytes using fluorescence in situ hybridization (FISH).Fluorescence in situ hybridization (FISH) using non-commercial probes in the diagnosis of clinically suspected microdeletion syndromes.Unbalanced translocation, t(18;21), detected by fluorescence in situ hybridization (FISH) in a child with 18q- syndrome and a ring chromosome 21.Using this technique, followed by cell-by-cell multicolor fluorescence in situ hybridization (FISH) analysis of purified FNRBCs, we were able to detect some of the most common human aneuploidies (including Down syndrome, Klinefelter syndrome, and trisomy 13) in 33 pregnant women referred for amniocentesis.FISH with 5p subtelomeric probes and 18p subtelomeric probe further confirmed that the derivative chromosome 5 has derived from a translocation between 5p and 18p, which has given rise to a 46,XY,der(5)t(5;18)(p15.1;p11.31)dn karyotype.A de novo 5p partial deletion in conjunction with a cryptic 18p duplication has been detected in a boy featuring Cri-du-Chat syndrome.We conclude that FISH is a useful, easily applied technique for the diagnosis of 22q11.2 microdeletion syndromes, particularly DGS.In group B (n=79), the 5q deletion had been observed by cytogenetics and was confirmed to be present in all cases by fluorescence in situ hybridization of 5q31.CONCLUSIONS: Fluorescence in situ hybridization of 5q31 detected the 5q deletion in 6% of cases without clear evidence of del(5q) by conventional banding cytogenetics. We suggest that fluorescence in situ hybridization of 5q31 should be performed in cases of a suspected '5q- syndrome' and/or if the cytogenetic study shows no metaphases or an aberrant karyotype with chromosome 5 involved (no 5q- chromosome).Although conventional cytogenetics is considered the gold standard to detect chromosomal abnormalities in myelodysplastic syndromes (MDS), fluorescence in situ hybridization (FISH) is being increasingly used additionally.Cytogenetic aberrations in myelodysplastic syndrome detected by comparative genomic hybridization and fluorescence in situ hybridization.To detect chromosomal abnormalities in myelodysplastic syndrome (MDS) patients by fluorescence in situ hybridization (FISH) and conventional cytogenetic analysis (CCA).Comparison of high resolution chromosome banding and fluorescence in situ hybridization (FISH) for the laboratory evaluation of Prader-Willi syndrome and Angelman syndrome.Deletion of small nuclear ribonucleoprotein polypeptide N (SNRPN) in Prader-Willi syndrome detected by fluorescence in situ hybridization: two sibs with the typical phenotype without a cytogenetic deletion in chromosome 15q.We describe two sibs with typical PWS presenting deletion of SNRPN detected by fluorescence in situ hybridization (FISH).Use of fluorescence in situ hybridization (FISH) in the diagnosis of DiGeorge sequence and related diseases.\"}, {\"question\": \"List symptoms of EAST syndrome.\", \"ideal_answer\": [\"Epilepsy, ataxia, sensorineural deafness, and tubulopathy comprise EAST syndrome that is associated with recessive mutations in the KCNJ10 gene.\"], \"exact_answer\": [[\"epilepsy\"], [\"ataxia\"], [\"sensorineural deafness\"], [\"tubulopathy\"]], \"context\": \"This condition is characterized by 4 cardinal features; Epilepsy, Ataxia, Sensorineural deafness, and (a renal salt-wasting) Tubulopathy, hence the acronym EAST syndrome. Generation and validation of a zebrafish model of EAST (epilepsy, ataxia, sensorineural deafness and tubulopathy) syndrome.Recessive mutations in KCNJ10, which encodes an inwardly rectifying potassium channel, were recently identified as the cause of EAST syndrome, a severe and disabling multi-organ disorder consisting of epilepsy, ataxia, sensorineural deafness and tubulopathy that becomes clinically apparent with seizures in infancy.We next injected splice- and translation-blocking kcnj10 antisense morpholino oligonucleotides and reproduced the cardinal symptoms of EAST syndrome - ataxia, epilepsy and renal tubular defects. AIM: Recently, we reported a previously unrecognized symptom constellation comprising epilepsy, ataxia, sensorineural deafness, and tubulopathy (EAST syndrome) associated with recessive mutations in the KCNJ10 gene.EAST syndrome should be considered in any patient with a renal Gitelman-like phenotype with additional neurological signs and symptoms like ataxia, epilepsy or sensorineural deafness.<CopyrightInformation>Copyright \\u00a9 2011 S. Karger AG, Basel.</CRecently, mutations in KCNJ10 were recognised as pathogenic in man, causing a constellation of symptoms, including epilepsy, ataxia, sensorineural deafness and a renal tubulopathy designated as EAST syndrome.Recently, we reported a previously unrecognized symptom constellation comprising epilepsy, ataxia, sensorineural deafness, and tubulopathy (EAST syndrome) associated with recessive mutations in the KCNJ10 gene.EAST syndrome should be considered in any patient with a renal Gitelman-like phenotype with additional neurological signs and symptoms like ataxia, epilepsy or sensorineural deafness..\"}, {\"question\": \"Which are the lipid lowering drugs administered in patients with Coronary Artery Disease (CAD)?\", \"ideal_answer\": [\"The lipid lowering drugs administered in patients with Coronary Artery Disease (CAD) are:\\n1) Statins\\n2) Fibrates\\n3) Resins\\n4) Niacin\\n5) Cholesterol absorption-inhibiting drugs.\"], \"exact_answer\": [[\"Statins\"], [\"Fibrates\"], [\"Resins\"], [\"Niacin\"], [\"Cholesterol absorption-inhibiting drugs\"]], \"context\": \"Impact of adding ezetimibe to statin to achieve low-density lipoprotein cholesterol goal (from the Clinical Outcomes Utilizing Revascularization and Aggressive Drug Evaluation [COURAGE] trial).After maximizing statin dose, ezetimibe was added to reach the LDL cholesterol goal in 34% of patients (n\\u00a0= 734). Using Cox regression analysis, the most significant factors associated with achieving LDL cholesterol goals were lower baseline LDL cholesterol, average statin dose, and ezetimibe use. In conclusion, after maximizing statin dose, the addition of ezetimibe results in a substantial increase in the percentage of patients who reach LDL cholesterol goal, a key component of optimal medical therapy.Comparative study of bezafibrate and pravastatin in patients with coronary artery disease and high levels of remnant lipoprotein.Remnant lipoproteinemia is a strong risk factor for cardiovascular (CV) diseases. This study examined which of 2 common lipid-lowering drugs (fibrates and statins) is more effective in patients with remnant lipoproteinemia and if lowering remnant lipoprotein levels can reduce CV risk.RLP-C levels at 1 year of treatment were reduced more by bezafibrate than pravastatin (37% and 25% from baseline, respectively). Bezafibrate therapy decreased RLP-C levels to a greater extent than pravastatin and a decrease in RLP-C level may be associated with a reduction in CV events in patients with high RLP-C levels.Statins are powerful lipid-lowering drugs, widely used in patients with hyperlipidemia and coronary artery disease. Patients are frequently treated with lipid-lowering drugs such as resins, fibrates, niacin, statins and cholesterol absorption-inhibiting drugs (ezetimibe). This study sought to investigate the effect of daily glucose fluctuation on coronary plaque properties in patients with coronary artery disease (CAD) pre-treated with lipid-lowering therapy.There is growing evidence that glucose fluctuation, as a residual risk apart from dyslipidemia, is an important factor contributing to the development of CAD.This prospective study enrolled 70 consecutive CAD patients who were referred for percutaneous coronary intervention and whose low-density lipoprotein cholesterol level was &lt;120 mg/dl under statin treatment or &lt;100 mg/dl without statinsTo investigate the impact of lipid lowering therapy by different means on skin microvascular function in patients with dysglycaemia and coronary artery disease (CAD).Thirty-six patients were randomized to simvastatin 80 mg daily (S80, n = 19) or ezetimibe 10 mg and simvastatin 10 mg daily (E10/S10, n = 17) for 6 weeksLarge clinical trials have elucidated that lipid-lowering therapy using statins reduces cardiovascular events in patients with coronary artery disease (CAD)The aim of the present study was to investigate whether LLT (lipid-lowering therapy) with statins could attenuate EPC telomere erosion in patients with CAD (coronary artery disease)Beneficial effects of pravastatin (+/-colestyramine/niacin) initiated immediately after a coronary event (the randomized Lipid-Coronary Artery Disease [L-CAD] Study).\"}, {\"question\": \"List drugs withdrawn from the market for cardiovascular adverse events.\", \"ideal_answer\": [\"Over the years, a number of different drugs have been withdrawn for Cardiotoxicity. Drugs like Clobutinol, that  induce cardiac arrhythmias by a blockade of the potassium channel coded by the hERG channel, sibutramine for weight loss and Cox2 inhibitors, Rofecoxib and Valdecoxib have been withdrawn from the market.\"], \"exact_answer\": [[\"Rofecoxib\"], [\"Valdecoxib\"], [\"Sibutramine\"], [\"Clobutinol\"], [\"Sertindole\"], [\"aprotinin\"]], \"context\": \" rofecoxib was voluntarily removed from the market for increased cardiovascular risk and in April 2005, valdecoxib was also withdrawn, at least in part, due to excess cardiovascular risk.A common over-the-counter (OTC) non-opioid antitussive drug, clobutinol, was recently withdrawn from the market due to its potential to induce cardiac arrhythmias by a blockade of the potassium channel coded by the human ether-\\u00e0-go-go-related gene (hERG).n. However, despite this, at the time of writing orlistat (Europe; USA) remains the only drug currently marketed for the treatment of obesity, with sibutramine having recently been withdrawn from sale globally due to the increased incidence of serious, non-fatal cardiovascular eventsDiclofenac has a risk very similar to rofecoxib, which was withdrawn from worldwide markets owing to cardiovascular toxicity.Sibutramine was associated with an increase in major adverse cardiovascular events in the Sibutramine Cardiovascular Outcomes (SCOUT) trial and it was withdrawn from the market in 2010The safety of the most popular agent, aprotinin, has been challenged, and it was withdrawn from world markets in May 2008 because of concerns that it increased the risk of cardiovascular complications and death.To assess the comparative effects of the anti-fibrinolytic drugs aprotinin, tranexamic acid (TXA), and epsilon aminocaproic acid (EACA) on blood loss during surgery, the need for red blood cell (RBC) transfusion, and adverse events, particularly vascular occlusion, renal dysfunction, and death.We searched: the Cochrane Injuries Groups Specialised Register (July 2010), Cochrane Central Register of Controlled Trials (The Cochrane Library 2010, Issue 3), MEDLINE (Ovid SP) 1950 to July 2010, EMBASE (Ovid SP) 1980 to July 2010.The safety of the most popular agent, aprotinin, has been challenged, and it was withdrawn from world markets in May 2008 because of concerns that it increased the risk of cardiovascular complications and death.To assess the comparative effects of the anti-fibrinolytic drugs aprotinin, tranexamic acid (TXA), and epsilon aminocaproic acid (EACA) on blood loss during surgery, the need for red blood cell (RBC) transfusion, and adverse events, particularly vascular occlusion, renal dysfunction, and death.We searched: the Cochrane Injuries Groups Specialised Register (July 2010), Cochrane Central Register of Controlled Trials (The Cochrane Library 2010, Issue 3), MEDLINE (Ovid SP) 1950 to July 2010, EMBASE (Ovid SP) 1980 to July 2010.Rofecoxib and valdecoxib have been withdrawn from the market worldwide due to safety concerns (most importantly for cardiovascular adverse events) and lumiracoxib has been withdrawn in many countries due to liver toxicityIn the SCOUT (Sibutramine Cardiovascular OUTcomes) study, sibutramine increased serious cardiovascular events, such as stroke or myocardial infarction, compared with placebo, and was consequently withdrawn from the marketDiclofenac has a risk very similar to rofecoxib, which was withdrawn from worldwide markets owing to cardiovascular toxicity. Rofecoxib and valdecoxib have been withdrawn from the market worldwide due to safety concerns (most importantly for cardiovascular adverse events) and lumiracoxib has been withdrawn in many countries due to liver toxicity.Sertindole , first withdrawn from the market for cardiovascular safety concerns, is currently available in many countries.In the SCOUT (Sibutramine Cardiovascular OUTcomes) study, sibutramine increased serious cardiovascular events, such as stroke or myocardial infarction, compared with placebo, and was consequently withdrawn from the market. The lesson learnt from this is the importance of patient selection, limiting the duration of treatment and stopping treatment in non-responders. Currently, phentermine and amfepramone (diethylpropion) are approved for short-term treatment of obesity (up to 3 months) and orlistat is approved for longer-term treatment; however, the gastrointestinal adverse effects of orlistat may be intolerable for some patients.BACKGROUND: Certain non-steroidal anti-inflammatory drugs (NSAIDs) (e.g., rofecoxib [Vioxx]) increase the risk of heart attack and stroke and should be avoided in patients at high risk of cardiovascular events.In the SCOUT (Sibutramine Cardiovascular OUTcomes) study, sibutramine increased serious cardiovascular events, such as stroke or myocardial infarction, compared with placebo, and was consequently withdrawn from the market.Diclofenac has a risk very similar to rofecoxib, which was withdrawn from worldwide markets owing to cardiovascular toxicity. Rofecoxib, a selective cyclo-oxygenase (COX)-2 inhibitor, was a widely marketed drug that was used for relief of pain and inflammation in arthritic conditions. It was withdrawn from the market worldwide in September 2004 because of an increased risk of cardiovascular events. \"}, {\"question\": \"Which are the types of viral meningitis?\", \"ideal_answer\": [\"Aseptic meningitis is the most common type of meningitis and is characterized by meningeal inflammation that is not linked to identifiable bacterial pathogens in cerebrospinal fluid (CSF). It can be originated from infection from:\\n1) Varicella-zoster virus (VZV)\\n2) Herpes simplex types I and II (HSV-1, HSV-2)\\n3) Epstein-Barr virus (EBV) \\n4) Cytomegalovirus (CMV) \\n5) Enteroviruses (EV) \\n6) Parechoviruses (HPeV) \\n7) Human rhinoviruses (HRVs) \\n8) Echovirus types 6, 9, 11\\n9) West Nile Virus (WNV)\"], \"exact_answer\": [[\"Varicella-zoster virus\", \"VZV\"], [\"Herpes simplex types I and II\", \"HSV-1\", \"HSV-2\"], [\"Epstein-Barr virus\", \"EBV\"], [\"Cytomegalovirus\", \"CMV\"], [\"Enteroviruses\", \"EV\"], [\"Parechoviruses\", \"HPeV\"], [\"Human rhinoviruses\", \"HRVs\"], [\"Echovirus types 6, 9, 11\"], [\"West Nile Virus\", \"WNV\"]], \"context\": \"Aseptic meningitis is the most common type of meningitis and is characterized by meningeal inflammation that is not linked to identifiable bacterial pathogens in cerebrospinal fluid (CSF).The results indicate a considerable rate of herpesvirus infection in patients with aseptic meningitis, and that VZV is the most common herpesvirus to cause infection followed by HSV-1. Detection of herpes viruses in the cerebrospinal fluid of adults with suspected viral meningitis in Malawi.Of these, 59 (32 %) had proven meningitis (bacterial, tuberculous or cryptococcal), 39 (21 %) had normal CSF and 14 (8 %) had aseptic meningitis. For the latter group, a herpes virus was detected in 9 (64 %): 7 (50 %) had EBV and 2 (14 %) had CMV, all were human immunodeficiency virus (HIV)-positive. Results confirm that EV-specific RT-PCR can detect HRVs, and at a practical level, identify potential problems of interpretation if fecal samples are used for surrogate screening in cases of suspected viral meningitis. Human rhinoviruses (HRVs) can be divided into three species; HRV-A to HRV-C.Enteroviruses (EV) and parechoviruses (HPeV) are the most common causes of aseptic meningitis, encephalitis and sepsis-like syndrome in neonates.Serous meningitis associated with primary genital herpes infectionAseptic meningitis is not an uncommon complication to primary genital herpes infection caused by herpes simplex virus type 2 (HSV-2). Compared with other types of viral meningitis, HSV-2-meningitis is associated with a significant rate of neurological complications in the acute stage. In addition, some patients will suffer from recurrent aseptic meningitis (Mollaret's meningitis) later. We describe six patients, five women and one man, age 26-35 years, with aseptic meningitis caused by HSV-2.Viral meningitis due to echovirus types 6 and 9: epidemiological data from Western Australia.During the autumn of 1992, Western Australia experienced a large viral meningitis outbreak of dual aetiology. Echovirus 9 caused 41% of cases and occurred mainly in the metropolitan areas of Western Australia whereas echovirus 6, which caused 37% of cases, was more widespread. Sensitive virologic evaluation in suspected meningitis: study of a radioimmunotest for the detection of IgM antibodies against ECHO 9 and ECHO 11 virusesThe majority of viral meningitis cases is known to be due to ECHO virus infections on one hand, and mumps on the other. The West Nile Virus (WNV) is a Flavivirus which is transmitted to man by means of different species of mosquitoes and causes outbreaks and sporadic cases of illness in different regions of the Old World, including the Mediterranean Basin.Bearing in mind the high percentage of neurological complications found to exist in the most recent outbreaks of WNV infections recorded in the Mediterranean Basin (Bucharest, Algeria), it can be thought that the WNV plays some role in the factors contributing to viral meningitis and encephalitis which occur within the population of the areas at risk within Spain.This study aimed to evaluate the frequency of aseptic meningitis caused by herpesviruses, namely herpes simplex types I and II (HSV-1, HSV-2), Epstein-Barr virus (EBV), cytomegalovirus (CMV) and varicella-zoster virus (VZV).We compared data obtained from patients with mumps meningitis and patients with aseptic meningitis caused by other viruses in order to identify mumps meningitis-specific cytokine/chemokine alterations in cerebrospinal fluid (CSF).METHODS: \\u2002 We elucidated the cytokine/chemokine network based on the cytokine/chemokine profiles in CSF from children with mumps meningitis and meningitis due to other viral infections using multiplex cytokine measurement.SPORADIC OCCURRENCE OF ECHO VIRUS TYPES 27 AND 31 ASSOCIATED WITH ASEPTIC MENINGITIS IN ONTARIO.Seventeen cytokines/chemokines, namely interleukin (IL)-1\\u00e2, IL-2, IL-4, IL-5, IL-6, IL-7, IL-8, IL-10, IL-12 (p70), IL-13, IL-17, interferon (IFN)-\\u00e3, tumor necrosis factor (TNF)-\\u00e1, granulocyte colony-stimulating factor (G-CSF), granulocyte monocyte colony-stimulating factor (GM-CSF), monocyte chemoattractant protein-1 (MCP-1) and macrophage inflammatory protein-1\\u00e2 (MIP-1\\u00e2), were measured simultaneously in CSF supernatants from eight children with mumps meningitis, 11 children with other types of viral meningitis and eight children with fever without neurological complications such as convulsion.RESULTS: \\u2002 We found that IL-8, IL-10, IL-12, IL-13 and IFN-\\u00e3 showed a statistically significant increase in CSF from mumps meningitis when compared to other types of viral meningitis and fever without neurological complications.CONCLUSION: \\u2002 Mumps meningitis may induce a distinct immunological response when compared with other types of viral meningitis.We found that IL-8, IL-10, IL-12, IL-13 and IFN-\\u03b3 showed a statistically significant increase in CSF from mumps meningitis when compared to other types of viral meningitis and fever without neurological complicationsMumps meningitis may induce a distinct immunological response when compared with other types of viral meningitis.<CopyrightInformation>\\u00a9 2011 The Authors.Pediatrics International \\u00a9 2011 Japan Pediatric Society.</[Adult cases of viral meningitis caused by echovirus type 13].[Epidemic outbreak of viral meningitis caused by type 30 ECHO virus].[Outbreak of viral meningitis caused by echovirus type 4 in Misiones province].Compared with other types of viral meningitis, HSV-2-meningitis is associated with a significant rate of neurological complications in the acute stage.Seventeen cytokines/chemokines, namely interleukin (IL)-1\\u03b2, IL-2, IL-4, IL-5, IL-6, IL-7, IL-8, IL-10, IL-12 (p70), IL-13, IL-17, interferon (IFN)-\\u03b3, tumor necrosis factor (TNF)-\\u03b1, granulocyte colony-stimulating factor (G-CSF), granulocyte monocyte colony-stimulating factor (GM-CSF), monocyte chemoattractant protein-1 (MCP-1) and macrophage inflammatory protein-1\\u03b2 (MIP-1\\u03b2), were measured simultaneously in CSF supernatants from eight children with mumps meningitis, 11 children with other types of viral meningitis aWe found that IL-8, IL-10, IL-12, IL-13 and IFN-\\u03b3 showed a statistically significant increase in CSF from mumps meningitis when compared to other types of viral meningitis and fever without neurological complications.Mumps meningitis may induce a distinct immunological response when compared with other types of viral meningitis.<CopyrightInformation>\\u00a9 2011 The Authors.Pediatrics International \\u00a9 2011 Japan Pediatric Society.</CBACKGROUND: \\u2002 The mumps virus is frequently the causative agent in aseptic meningitis and mumps has still prevailed in Japan. We compared data obtained from patients with mumps meningitis and patients with aseptic meningitis caused by other viruses in order to identify mumps meningitis-specific cytokine/chemokine alterations in cerebrospinal fluid (CSF).METHODS: \\u2002 We elucidated the cytokine/chemokine network based on the cytokine/chemokine profiles in CSF from children with mumps meningitis and meningitis due to other viral infections using multiplex cytokine measurement.BACKGROUND: \\u2002 The mumps virus is frequently the causative agent in aseptic meningitis and mumps has still prevailed in Japan. We compared data obtained from patients with mumps meningitis and patients with aseptic meningitis caused by other viruses in order to identify mumps meningitis-specific cytokine/chemokine alterations in cerebrospinal fluid (CSF).METHODS: \\u2002 We elucidated the cytokine/chemokine network based on the cytokine/chemokine profiles in CSF from children with mumps meningitis and meningitis due to other viral infections using multiplex cytokine measurement. Seventeen cytokines/chemokines, namely interleukin (IL)-1\\u00e2, IL-2, IL-4, IL-5, IL-6, IL-7, IL-8, IL-10, IL-12 (p70), IL-13, IL-17, interferon (IFN)-\\u00e3, tumor necrosis factor (TNF)-\\u00e1, granulocyte colony-stimulating factor (G-CSF), granulocyte monocyte colony-stimulating factor (GM-CSF), monocyte chemoattractant protein-1 (MCP-1) and macrophage inflammatory protein-1\\u00e2 (MIP-1\\u00e2), were measured simultaneously in CSF supernatants from eight children with mumps meningitis, 11 children with other types of viral meningitis and eight children with fever without neurological complications such as convulsion.RESULTS: \\u2002 We found that IL-8, IL-10, IL-12, IL-13 and IFN-\\u00e3 showed a statistically significant increase in CSF from mumps meningitis when compared to other types of viral meningitis and fever without neurological complications.CONCLUSION: \\u2002 Mumps meningitis may induce a distinct immunological response when compared with other types of viral meningitis.[Adult cases of viral meningitis caused by echovirus type 13].\\u2002 Mumps meningitis may induce a distinct immunological response when compared with other types of viral meningitis..A hospitalization with viral meningitis was defined as any discharge with diagnoses of mumps meningitis (B26.1), enteroviral meningitis (A85.0), tick-born encephalitis (A84.1) and viral undifferentiated meningitis (A87.9) due to ICD-10.Molecular typing of enteroviruses associated with viral meningitis in Cyprus, 2000-2002.\"}, {\"question\": \"What are the different classes of orally administered drugs used to treat diabetes\", \"ideal_answer\": [\"There are a number of classes of medications tha are used to treat Type 2 diabetes. These include biguanides like metformin, which decreased hepatic glucose release; sulfonyureas like Glimepride, metglitinides like repaglin and d-phenylalanine derivatives, all of which stimulate pancreatic insulin release; Glitizones or thiazolidinediones which make the body more sensitive to insulin; DPP-4 inhibitors which lower the amount of glucose made in the body; alpha-glucosidase inhibitors which slow the absorbtion of carbohydrate in the blood; and bile acid sequestrants that lower blood glucose\"], \"exact_answer\": [[\"Biguanides\"], [\"Sulfonylureas\"], [\"Meglitinides\"], [\"D-Phenylalanine Derivatives\"], [\"glitazones\"], [\"DPP-4 Inhibitors\"], [\"Alpha-glucosidase inhibitors\"], [\"bile acid sequestrants\"]], \"context\": \" achieved by selectively inhibiting sodium-glucose co-transporter 2 (SGLT2).After a decade-long effort to improve the pharmacokinetics of GLP1, a number of GLP1 analogues are currently available on the market short-acting glucagon-like peptide-1 receptor agonists exenatide and lixisenatide.  Sulphonylurea (SU) drugs have a history stretching back over 60 years, and have traditionally been the mainstay choice as second-line agents to be added to metformin once glycaemic control with metformin monotherapy deteriorates;In 2013, sulfonylureas were the most common second-line treatment after metformin ithis position was taken by DPP-4 inhibitors[Studies on resorption of orally administered antibiotics and chemotherapeutic agents in children and its modification.The drugs used to treat diabetes mellitus are diverse and include several classes.Many of the therapeutic agents used to treat diabetes mellitus have the ability to lower blood glucose to dangerous concentrations; these include the sulfonylurea, meglitinide, and thiazolidinedione drug classes.The most common therapeutic classes of drugs used in all participating regions, in the previous week, were cardiovascular, gastrointestinal, metabolic (including drugs to treat diabetes) and nervous system.Use of oral antidiabetic agents, by drug class, did not differ significantly by race/ethnicity (p = 0.33 for TZDs, p = 0.43 for metformin, p = 0.38 for sulfonylureas). The use of these drugs was also common among insulin treated diabetics but did not differ significantly from among non-diabetics. Drug use among those treated with insulin and those treated orally was substantially higher than among non-diabetics while the difference between diabetics on dietary regimen and non-diabetics was much smaller. UNLABELLED: The aims of the study were to evaluate differences in dietary, oral hygiene habits and social class in children with Type I diabetes mellitus (DM), compared to non-diabetics, and to investigate relationship between selected caries-risk factors and caries experience in diabetics. Vildagliptin is a potent and selective inhibitor of dipeptidyl peptidase-IV (DPP-4), orally active, that improves glycemic control in patients with type 2 diabetes (T2DM) primarily by enhancing pancreatic (alpha and beta) islet function.The literature covered includes original studies and meta-analyses identified in PubMed, recent abstracts presented at major diabetes scientific conferences, and clinical trials registered at ClinicalTrials.gov.Although there are some differences in the pharmacokinetic and pharmacodynamic profiles of the different DPP-4 inhibitors, all are small orally active compounds with broadly similar HbA1c-lowering efficacy.The authors conclude that rosiglitazone can be safely administered with metformin and, due to the different mechanisms of action of these agents, may offer a therapeutic advantage in patients with type 2 diabetes mellitus.There are different classes of anti-diabetic drugs reported to treat diabetes.There are now four different classes of oral medications which are available to treat diabetes-sulfonylureas, biguanides, thiazolidinediones, and alpha-glucosidase inhibitors.During followup, incident treated diabetes was defined as a self-report of a new physician diagnosis of diabetes treated with insulin or oral drugs.This systematic review will compare adherence and persistence rates across the different classes of medication available to people with type 2 diabetes.Five classes of oral hypoglycaemic drugs and two trace minerals used to treat diabetes mellitus in humans are reviewed and current knowledge on the use of these drugs in diabetic dogs and cats is presented.Sulfonylureas, biguanides, alpha-glucosidase, meglitinides, DPP-4 inhibitors and thiazolidinediones are among the classes of oral hypoglycemic drugs available to treat Type II diabetes, but concerns exist regarding safety and efficacy of these drugs.Currently, four classes of orally administered antidiabetic agents are available for use in patients with type 2 diabetes: insulin secretagogues, biguanides, a-glucosidase inhibitors, and thiazolidinediones.\"}, {\"question\": \"Which pipelines are used for analyzing data from ChIP-nexus experiments?\", \"ideal_answer\": [\"PeakXus and Q-nexus enable comprehensive transcription factor binding site discovery from ChIP-nexus experiments.\"], \"exact_answer\": [[\"PeakXus\"], [\"Q-nexus\"]], \"context\": \"PeakXus: comprehensive transcription factor binding site discovery from ChIP-Nexus and ChIP-Exo experiments.We describe a peak caller PeakXus that is specifically designed to leverage the increased resolution of ChIP-exo/Nexus and developed with the aim of making as few assumptions of the data as possible to allow discoveries of novel binding patterns. We apply PeakXus to ChIP-Nexus and ChIP-exo experiments performed both in Homo sapiens and in Drosophila melanogaster cell lines.Q-nexus: a comprehensive and efficient analysis pipeline designed for ChIP-nexus.Here, we present a comprehensive software package for ChIP-nexus data that exploits the random barcodes for selective removal of PCR duplicates and for quality control. Furthermore, we developed bespoke methods to estimate the width of the protected region resulting from protein-DNA binding and to infer binding positions from ChIP-nexus data. Finally, we applied our peak calling method as well as the two other methods MACE and MACS2 to the available ChIP-nexus data.Q-nexus: a comprehensive and efficient analysis pipeline designed for ChIP-nexus.PeakXus: comprehensive transcription factor binding site discovery from ChIP-Nexus and ChIP-Exo experiments.\"}, {\"question\": \"SPAG5 was implicated in which cancers?\", \"ideal_answer\": [\"SPAG5 was implicated in prostate cancer, lung cancer and cervical cancer.\"], \"exact_answer\": [[\"prostate cancer\"], [\"lung cancer\"], [\"cervical cancer\"], [\"breast cancer\"]], \"context\": \"BACKGROUND: We conducted multiple microarray datasets analyses from clinical and xenograft tumor tissues to search for disease progression-driving oncogenes in prostate cancer (PCa). Sperm-associated antigen 5 (SPAG5) attracted our attention. SPAG5 was recently identified as an oncogene participating in lung cancer and cervical cancer progression.CONCLUSION: Our results collectively showed a progression-driving role of SPAG5 in PCa which can be regulated by miR-539, suggesting that miR-539/SPAG5 can serve as a potential therapeutic target for PCa.RESULTS: We identified that SPAG5 expression was gradually increased in PCa progression and its level was significantly associated with lymph node metastasis, clinical stage, Gleason score, and biochemical recurrence. Our results indicated that SPAG5 knockdown can drastically inhibit PCa cell proliferation, migration, and invasion in vitro and supress tumor growth and metastasis in vivo. miR-539 inhibits prostate cancer progression by directly targeting SPAG5.SPAG5 as a prognostic biomarker and chemotherapy sensitivity predictor in breast cancer: a retrospective, integrated genomic, transcriptomic, and protein analysis.Copy number aberration leading to SPAG5 gain or amplification and high SPAG5 transcript and SPAG5 protein concentrations were associated with shorter overall breast cancer-specific survival (METABRIC cohort [copy number aberration]: hazard ratio [HR]1\\u00b750, 95% CI 1\\u00b718-1\\u00b792, p=0\\u00b700010; METABRIC cohort [transcript]: 1\\u00b768, 1\\u00b740-2\\u00b701, p<0\\u00b70001; and Nottingham-HES-breast cancer cohort [protein]: 1\\u00b768, 1\\u00b732-2\\u00b712, p<0\\u00b70001). In multivariable analysis, high SPAG5 transcript and SPAG5 protein expression were associated with reduced breast cancer-specific survival at 10 years compared with lower concentrations (Uppsala: HR 1\\u00b762, 95% CI 1\\u00b703-2\\u00b753, p=0\\u00b7036; METABRIC: 1\\u00b727, 1\\u00b702-1\\u00b758, p=0\\u00b7034; untreated lymph node-negative cohort: 2\\u00b734, 1\\u00b724-4\\u00b742, p=0\\u00b70090; and Nottingham-HES: 1\\u00b773, 1\\u00b723-2\\u00b746, p=0\\u00b70020). In patients with oestrogen receptor-negative breast cancer with high SPAG5 protein expression, anthracycline-based adjuvant chemotherapy increased breast cancer-specific survival overall compared with that for patients who did not receive chemotherapy (Nottingham-oestrogen receptor-negative-ACT cohort: HR 0\\u00b737, 95% CI 0\\u00b720-0\\u00b760, p=0\\u00b70010). Multivariable analysis showed high SPAG5 transcript concentrations to be independently associated with longer distant relapse-free survival after receiving taxane plus anthracycline neoadjuvant chemotherapy (MD Anderson-NeoACT: HR 0\\u00b768, 95% CI 0\\u00b748-0\\u00b797, p=0\\u00b7031). INTERPRETATION: SPAG5 is a novel amplified gene on Ch17q11.2 in breast cancer. The transcript and protein products of SPAG5 are independent prognostic and predictive biomarkers that might have clinical utility as biomarkers for combination cytotoxic chemotherapy sensitivity, especially in oestrogen receptor-negative breast cancer.SPAG5 upregulation predicts poor prognosis in cervical cancer patients and alters sensitivity to taxol treatment via the mTOR signaling pathway.Previously, we found that sperm-associated antigen 5 (SPAG5) was upregulated in pelvic lymph node metastasis-positive cervical cancer.In general, SPAG5 upregulation relates to poor prognosis in cervical cancer patients, and SPAG5 is a regulator of mTOR activity during taxol treatment in cervical cancer.RESULTS: We identified novel genes whose expression was upregulated in NSCLC, including SPAG5, POLH, KIF23, and RAD54L, which are associated with mitotic spindle formation, DNA repair, chromosome segregation, and dsDNA break repair, respectively.Another module identified among male breast cancers, regulated by SPAG5, was closely associated with proliferation.In our study, SPAG5 expression in cervical cancer patients was detected using quantitative real-time polymerase chain reaction, western blotting, and immunohistochemistry; cervical cancer cell function with downregulated SPAG5 in vitro was explored using tetrazolium assay, flow cytometry, and colony formation and Transwell assays. In patients with oestrogen receptor-negative breast cancer with high SPAG5 protein expression, anthracycline-based adjuvant chemotherapy increased breast cancer-specific survival overall compared with that for patients who did not receive chemotherapy (Nottingham-oestrogen receptor-negative-ACT cohort: HR 0\\u00b737, 95% CI 0\\u00b720-0\\u00b760, p=0\\u00b70010).We investigated the clinicopathological relevance of SPAG5 gene copy number aberrations, mRNA transcript expression, and protein expression and analysed the associations of SPAG5 copy number aberrations, transcript expression, and protein expression with breast cancer-specific survival, disease-free survival, distant relapse-free survival, pathological complete response, and residual cancer burden in the Nottingham discovery cohort, Uppsala cohort, METABRIC cohort, a pooled untreated lymph node-negative cohort (n=684), a multicentre combined cohort (n=5439), the Nottingham historical early stage breast cancer cohort (Nottingham-HES; n=1650), NIn multivariable analysis, high SPAG5 transcript and SPAG5 protein expression were associated with reduced breast cancer-specific survival at 10 years compared with lower concentrations (Uppsala: HR 1\\u00b762, 95% CI 1\\u00b703-2\\u00b753, p=0\\u00b7036; METABRIC: 1\\u00b727, 1\\u00b702-1\\u00b758, p=0\\u00b7034; untreated lymph node-negative cohort: 2\\u00b734, 1\\u00b724-4\\u00b742, p=0\\u00b70090; and Nottingham-HES: 1\\u00b773, 1\\u00b723-2\\u00b746, p=0\\u00b70020).THY1 may thus be a promising novel prognostic marker for male breast cancer. Another module identified among male breast cancers, regulated by SPAG5, was closely associated with proliferation.Furthermore, men with THY1 positive breast cancers had significantly inferior survival. THY1 may thus be a promising novel prognostic marker for male breast cancer. Another module identified among male breast cancers, regulated by SPAG5, was closely associated with proliferation.SPAG5 was recently identified as an oncogene participating in lung cancer and cervical cancer progression.SPAG5 is a novel amplified gene on Ch17q11.2 in breast cancer.Copy number aberration leading to SPAG5 gain or amplification and high SPAG5 transcript and SPAG5 protein concentrations were associated with shorter overall breast cancer-specific survival (METABRIC cohort [copy number aberration]: hazard ratio [HR] 1\\u00b750, 95% CI 1\\u00b718-1\\u00b792, p=0\\u00b700010; METABRIC cohort [transcript]: 1\\u00b768, 1\\u00b740-2\\u00b701, p<0\\u00b70001; and Nottingham-HES-breast cancer cohort [protein]: 1\\u00b768, 1\\u00b732-2\\u00b712, p<0\\u00b70001).\"}, {\"question\": \"List kinases that phosphorylates the protein Bora.\", \"ideal_answer\": [\"During cell division Bora becomes multiply phosphorylated by a variety of cell cycle kinases, including Aurora A and Plk1, and GSK3\\u03b2 and Cdk1 albeit at distinctive sites.\"], \"exact_answer\": [[\"Cdk1\"], [\"Plk1\"], [\"Aurora A\"], [\"GSK3\\u03b2\"]], \"context\": \"During cell division Bora becomes multiply phosphorylated by a variety of cell cycle kinases, including Aurora A and Plk1, albeit at distinctive sites.Likewise, we find that phosphorylation of Bora by Cdk1 promotes phosphorylation of human Plk1 by Aurora A suggesting that this mechanism is conserved in humans. we show that the master mitotic kinase Cdk1 contributes to Plk1 activation through SPAT-1/Bora phosphorylationthe potential role of Bora phosphorylation by Cdk1 in this processphosphorylation of Bora on the Cdk consensus site T52 blocks Bora degradation.Here, we used the LC-MS/MS phosphopeptide mapping assay to identify 13 in vivo hBora phosphorylation sites and characterized that GSK3\\u03b2 can interact with hBora and phosphorylate hBora at Ser274 and Ser278.\"}, {\"question\": \"Which fimA genotypes are associated with disease?\", \"ideal_answer\": [\"FimA has been characterized as an important virulence factor for P. gingivalis, and many studies, both animal experiments and clinical investigations, have characterized fimA genotypes II, Ib, and IV to be associated with disease (periodontitis and cardiovascular disease)\"], \"exact_answer\": [[\"genotypes II\"], [\"genotypes Ib\"], [\"genotypes IV\"]], \"context\": \" FimA has been characterized as an important virulence factor for P. gingivalis, and many studies, both animal experiments and clinical investigations, have characterized fimA genotypes II, Ib, and IV to be associated with disease (periodontitis and cardiovascular disease) Long fimbriae (FimA) are important virulence factors of Porphyromonas gingivalis. Based on the diversity of the fimA gene, this species is classified into 6 genotypes.  In cases of chronic apical periodontitis, P. gingivalis variant type IV was the most prevalent (24%), followed by types I (20%), II (16%), and III (8%). In acute abscess samples, variant type II was the most prevalent (12%), followed by types III and IV (8% of each) and type I (4%).We show that P. gingivalis strains with genotype I and II of FimA are efficient in interaction with saliva or S. gordonii. A strong association between Porphyromonas gingivalis fimA genotypes II and Ib and chronic periodontitis exists in the Spanish population. The most prevalent genotype in periodontal patients is II.Our results suggest that the presence of P. gingivalis is associated with periodontal diseases, and that the type II, IV and Ib/II combination are the most common among fimA genotypes.Statistical analysis, however, revealed that a more significant correlation was found between periodontitis and the occurrence of type Ib fimA. Type II of fimA was the most prevalent genotype of P. gingivalis in patients with AgP. The population of Tregs further decreased in patients with type II FimA compared with the other types. P.gingivlias FimA genotype II was the dominant type associated with decreased Treg population. A trend toward a greater frequency of FimA II genotype in patients with moderate and severe periodontitis was determined.The fimA type Ib genotype of P. gingivalis was found to play a critical role in the destruction of peri-implant tissue, suggesting that it may be a distinct risk factor for peri-implantitis.\"}, {\"question\": \"Please list 3 diseases associated with the PIEZO2 gene.\", \"ideal_answer\": [\"Recently, Gordon syndrome has been associated to heterozygous mutations in the piezo-type mechanosensitive ion channel component 2 gene (PIEZO2). Different mutations of this gene also cause distal arthrogryposis type 5 and Marden-Walker syndrome.\"], \"exact_answer\": [[\"Gordon syndrome\"], [\"distal arthrogryposis type 5\"], [\"Marden-Walker syndrome\"]], \"context\": \"Recently, Gordon syndrome has been associated to heterozygous mutations in the piezo-type mechanosensitive ion channel component 2 gene (PIEZO2). Different mutations of this gene also cause distal arthrogryposis type 5 and Marden-Walker syndrome.Mutations in PIEZO2 cause Gordon syndrome, Marden-Walker syndrome, and distal arthrogryposis type 5.Both types of changes in kinetics result in increased channel activity in response to a given mechanical stimulus, suggesting that Distal Arthrogryposis Type 5 can be caused by gain-of-function mutations in PIEZO2.Here, we describe two distinct PIEZO2 mutations in patients with a subtype of Distal Arthrogryposis Type 5 characterized by generalized autosomal dominant contractures with limited eye movements, restrictive lung disease, and variable absence of cruciate knee ligaments.Recently, Gordon syndrome has been associated to heterozygous mutations in the piezo-type mechanosensitive ion channel component 2 gene (PIEZO2).In contrast, heterozygous gain-of-function missense mutations, mainly localized at the C terminus, cause dominant distal arthrogryposis 3 (DA3), distal arthrogryposis 5 (DA5), or Marden-Walker syndrome (MWKS), which encompass contractures of hands and feet, scoliosis, ophthalmoplegia, and ptosis.Dominant mutations in PIEZO2, which codes for the principal mechanotransduction channel for proprioception and touch sensation, have been found to cause different forms of distal arthrogryposi\"}, {\"question\": \"List diseases that could be targeted by disaggregases?\", \"ideal_answer\": [\"UBQLN2 acts with the HSP70-HSP110 disaggregase machinery to clear protein aggregates via the 26S proteasome. UBQLN2 recognizes client-bound HSP70 and links it to the proteasome to allow for the degradation of aggregated and misfolded proteins. We further show that this process is active in the cell nucleus, where another system for aggregate clearance, autophagy, does not act.\"], \"exact_answer\": [[\"ALS\", \"amyotrophic lateral sclerosis\"], [\"PD\", \"Parkinson's disease\"], [\"AD\", \"Alzheimer's disease\"], [\"Prions\"]], \"context\": \"Therapeutic agents are urgently required to cure several common and fatal neurodegenerative disorders caused by protein misfolding and aggregation, including amyotrophic lateral sclerosis (ALS), Parkinson's disease (PD), and Alzheimer's disease (AD).Human Hsp70 Disaggregase Reverses Parkinson's-Linked \\u03b1-Synuclein Amyloid Fibrils.Disaggregases, molecular chaperones that resolubilize protein aggregates.The structural basis by which Hsp104 dissolves disordered aggregates and prions is unknown.\"}, {\"question\": \"List RNA modifications databases\", \"ideal_answer\": [\"RMBase and MODOMICS\"], \"exact_answer\": [[\"RMBase\", \"RNA Modification Base\"], [\"MODOMICS\"]], \"context\": \"RMBase: a resource for decoding the landscape of RNA modifications from high-throughput sequencing data.we developed a novel resource, RMBase (RNA Modification Base, http://mirlab.sysu.edu.cn/rmbase/), to decode the genome-wide landscape of RNA modifications identified from high-throughput modification data generated by 18 independent studies.  known pathways of RNA modification from the MODOMICS databaseMODOMICS is a database of RNA modifications that provides comprehensive information concerning the chemical structures of modified ribonucleosides, their biosynthetic pathways, RNA-modifying enzymes and location of modified residues in RNA sequences. \"}, {\"question\": \"Please list the 3 findings in HELLP syndrome.\", \"ideal_answer\": [\"hemolysis, elevated liver enzymes and low platelets constitute the hellp syndrome.\", \"HELLP syndrome (hemolysis, elevated liver enzymes, low platelets) is an obstetric complication and is characterized by microangiopathic hemolytic anemia, elevated liver enzymes by intravascular breakdown of fibrin in hepatic sinusoids and reduction of platelet circulation by its increased consumption\"], \"exact_answer\": [[\"hemolytic anemia\"], [\"elevated liver enzymes\"], [\"low platlets\"]], \"context\": \"HELLP syndrome (hemolysis, elevated liver enzymes, low platelets) is an obstetric complication with heterogonous presentation and multisystemic involvement. It is characterized by microangiopathic hemolytic anemia, elevated liver enzymes by intravascular breakdown of fibrin in hepatic sinusoids and reduction of platelet circulation by its increased consumptionHemolysis, elevated liver enzymes and low platelets constitute the HELLP syndrome.Our objective was to describe the hepatic imaging findings in selected patients with HELLP syndrome (hemolysis, elevated liver enzymes, and low platelet count) and to correlate these findings with the severity of concurrent clinical and laboratory abnormalities.Our objective was to categorize the histologic findings in the liver in patients with HELLP syndrome (hemolysis, elevated liver enzymes, and low platelet count) and to correlate these findings with the severity of clinical laboratory abnormalities.OBJECTIVE: Our purpose was to test the hypothesis that the HELLP (hemolysis, elevated liver enzymes, and low platelets) syndrome is the result of excessive vasoconstriction of the hepatic arterial circulation.STUDY DESIGN: Doppler ultrasonography was used to measure the pulsatility index of the common hepatic artery in 14 women with preeclampsia, 15 with preeclampsia complicated by HELLP syndrome, and 8 with HELLP syndrome but without proteinuria.Extensive hepatic infarction in severe preeclampsia as part of the HELLP syndrome (hemolysis, elevated liver enzymes, and low platelets): evolution of CT findings and successful treatment with plasma exchange therapy.OBJECTIVE: Our objective was to categorize the histologic findings in the liver in patients with HELLP syndrome (hemolysis, elevated liver enzymes, and low platelet count) and to correlate these findings with the severity of clinical laboratory abnormalities. The syndrome of haemolysis, elevated liver enzymes and low platelets (HELLP syndrome) is a life threatening, severe complication of pre-eclampsia with typical laboratory findings.Hemolysis, elevated liver enzymes, and low platelet count (HELLP) syndrome hemolysis, elevated liver enzymes, and low platelet count (HELLP) syndromeHELLP syndrome, a syndrome of hemolysis, elevated liver enzymes and low plateletsHELLP syndrome is a collection of symptoms described as hemolysis, elevated liver enzymes and low plateletsThe HELLP syndrome is characterized by the presence of hypertension disorder more a triad: microangiopathic hemolysis, elevated liver enzymes and low platelet count.HELLP (hemolysis, elevated liver enzymes, and low platelets) HELLP (hemolysis, elevated liver enzymes, and low platelets) syndrome, Hepatic hemorrhage occurs in less than 5% of patients with hemolysis, elevated liver enzymes, low platelets (HELLP) syndrome HELLP (haemolysis, elevated liver enzymes and low platelets)Hemolysis, elevated liver enzymes, and low platelet count syndrome (HELLP syndrome)  hemolysis, elevated liver enzymes, low platelets (HELLP) syndrome hemolysis, elevated liver enzymes and low platelet (HELLP) syndrome, iHemolysis, elevated liver enzymes, and low platelets (HELLP) syndromeHELLP (hemolysis, elevated liver enzymes, and low platelet count)HELLP (hemolysis, elevated liver enzymes, low platelets) syndromehemolysis, elevated liver enzymes, low platelets (HELLP) syndromehemolysis, elevated liver enzymes, and low platelets (HELLP),\"}, {\"question\": \"List the three most abundant bacterial phyla present in mouse feces.\", \"ideal_answer\": [\"Firmicutes\\nProteobacteria \\nBacteroidetes\"], \"exact_answer\": [[\"Firmicutes\"], [\"Proteobacteria\"], [\"Bacteroidetes\"]], \"context\": \"Amplicons representing the major phyla encountered in the rumen (Firmicutes, 43.7%; Proteobacteria, 28.7%; Bacteroidetes, 25.3%; Spirochea, 1.1%, and Synergistes, 1.1%) were recovered, alterations in the gut microbiota (increased Firmicutes and decreased Bacteroidetes)The taxonomic analysis of the metagenomic reads indicated that pygmy loris fecal microbiomes were dominated by Bacteroidetes and Proteobacteria phyla. Specifically, intermittent hypoxia-exposed mice showed a higher abundance of Firmicutes and a smaller abundance of Bacteroidetes and Proteobacteria phyla than controls.\"}, {\"question\": \"List the classical triad of symptoms of the Melkersson\\u2013Rosenthal syndrome.\", \"ideal_answer\": [\"The Melkersson-Rosenthal syndrome consists of the classical triad of symptoms:\\n1) orofacial oedema \\n2) fissured tongue (lingua plicata) and\\n3) facial paralysis.\"], \"exact_answer\": [[\"orofacial oedema\", \"lip and face swelling\"], [\"fissured tongue\", \"lingua plicata\"], [\"facial paralysis\", \"facial palsy\"]], \"context\": \"The Melkersson-Rosenthal syndrome consists of a triad of recurrent lip and/or face swelling, fissured tongue, and intermittent facial palsy. The patient has a typical for Melkersson-Rosenthal syndrome triad of symptoms: bilateral facial nerve palsy, lingua plicata and facial oedema. Melkersson-Rosenthal syndrome (MRS) is a rare disorder of unknown cause. The classical triad of MRS is orofacial edema, recurrent facial paralysis, and a fissured tongue.Melkersson-Rosenthal syndrome (MRS) is a rare granulomatous inflammatory disease characterised by the triad of orofacial oedema, facial nerve palsy and furrowed tongue.Melkersson-Rosenthal syndrome (MRS) is a rare neuro-mucocutaneous granulomatous disorder of unknown etiology, characterized by the triad of facial palsy, lingua plicata (fissured tongue), and orofacial edema. Melkerrson-Rosenthal syndrome is a rare disorder of unknown etiology. The classical triad of recurrent facial paralysis, swelling of the face, lips and deep furrowed tongue (Lingua Plicata) is seen in very few cases, majority of the patients often present with one or two symptoms only, which often leads to misdiagnosis and mismanagement. Melkersson-Rosenthal syndrome (MRS) is a rare neuromucocutaneous syndrome marked by the triad of recurrent nonpitting orofacial edema, fissured dorsal tongue (lingua plicata), and lower motoneuron facial paralysis.Recurrent facial nerve palsy, facial swelling, and fissured tongue are the symptoms and signs of this condition. However, this triad is not typical in all patients as patients may present with one or more of the symptoms, which makes management of this condition difficult. Melkersson-Rosenthal syndrome may manifest as the classical triad (orofacial edema, facial nerve palsy and stable lingua plicata) but monosymptomatic manifestations or combinations of typical symptoms are not infrequent.Melkersson-Rosenthal Syndrome (MRS) is a systemic neuro-mucocutaneous granulomatous disease, characterized in its classical form by a triad of recurrent facial nerve paralysis, swelling of the lips and lingua plicata.Melkersson-Rosenthal syndrome is classically described as a triad of orofacial swelling, facial palsy, and fissured tongue.Melkersson-Rosenthal syndrome is a rare disorder consisting of the triad of persistent or recurrent orofacial edema, relapsing facial paralysis and fissured tongue.The Melkersson-Rosenthal syndrome consists of a triad of recurrent lip and/or face swelling, fissured tongue, and intermittent facial palsy.The Melkersson-Rosenthal syndrome consists of a triad of symptoms: peripheral facial nerve paralysis, congenital \\\"lingua plicata\\\" and noninflammatory facial oedema.The Melkersson-Rosenthal syndrome consists of triad of symptoms: recurrent oedema of lips, recurrent facial nerve paralysis and lingua plicata.Melkersson-Rosenthal syndrome is characterised by a triad of recurrent orofacial swelling, relapsing facial paralysis, and fissured tongue.The Melkersson-Rosenthal syndrome consists of triad of symptoms: recurrent oedema of lips, recurrent facial nerve paralysis and lingua plicataMelkersson-Rosenthal syndrome may manifest as the classical triad (orofacial edema, facial nerve palsy and stable lingua plicata) but monosymptomatic manifestations or combinations of typical symptoms are not infrequentMelkersson-Rosenthal syndrome (MRS) is a rare disorder of unknown etiology characterized by a triad of symptoms: recurrent orofacial swelling, relapsing facial palsy. and a fissured tongueThe Melkersson-Rosenthal syndrome consists of a triad of symptoms: peripheral facial nerve paralysis, congenital \\\"lingua plicata\\\" and noninflammatory facial oedemaMelkersson-Rosenthal syndrome (MRS) is characterized by the triad of facial paralysis, facial oedema, and lingua plicata in association with other symptoms, such as headache and mental changesPeripheral facial nerve palsy, recurrent or persistent oral or facial swelling, and fissured tongue constitute a triad of symptoms known as Melkersson-Rosenthal syndromeMelkersson-Rosenthal (MRS) syndrome is characterized by a classical triad of recurrent or persistent orofacial swelling, peripheral facial nerve paralysis and lingua plicata. BACKGROUND: Melkersson-Rosenthal syndrome may manifest as the classical triad (orofacial edema, facial nerve palsy and stable lingua plicata) but monosymptomatic manifestations or combinations of typical symptoms are not infrequent. The syndrome is classically characterized by a triad of signs consisting of facial edema, recurrent peripheral facial nerve paralysis, and congenital fissured tongue, although it may also present in a mono- or oligosymptomatic form. The classic triad of signs includes recurrent orofacial edema, recurrent facial nerve palsy, and lingua plicata. Melkersson-Rosenthal syndrome (MRS) is characterized by the triad of facial paralysis, facial oedema, and lingua plicata in association with other symptoms, such as headache and mental changes.The patient has a typical for Melkersson-Rosenthal syndrome triad of symptoms: bilateral facial nerve palsy, lingua plicata and facial oedema.Peripheral facial nerve palsy, recurrent or persistent oral or facial swelling, and fissured tongue constitute a triad of symptoms known as Melkersson-Rosenthal syndrome.Classical signs include recurrent facial palsy, lingua plicata and orofacial edema.The classical triad includes recurrent orofacial oedema involving predominantly the lips (macrocheilitis), intermittent peripheral facial palsy and scrotal tongue.The patient has a typical for Melkersson-Rosenthal syndrome triad of symptoms: bilateral facial nerve palsy, lingua plicata and facial oedema.\"}, {\"question\": \"Which factors are considered in the FUNC score for intracerebral hemorrhage?\", \"ideal_answer\": [\"FUNC score includes Age, Glasgow Coma Scale, ICH location, volume and pre-ICH cognitive impairment.\"], \"exact_answer\": [[\"Age\"], [\"Glasgow Coma Scale\"], [\"ICH location\"], [\"ICH volume\"], [\"pre-ICH cognitive impairment\"]], \"context\": \"Age, Glasgow Coma Scale, ICH location, volume (all P<0.0001), and pre-ICH cognitive impairment (P=0.005) were independently associated with Glasgow Outcome Score>or = 4. The FUNC score was developed as a sum of individual points (0-11) based on strength of association with outcome. Age, Glasgow Coma Scale, ICH location, volume (all P<0.0001), and pre-ICH cognitive impairment (P=0.005) were independently associated with Glasgow Outcome Score>or = 4.\"}, {\"question\": \"What genes are drug targets for Fibrodysplasia Ossificans Progressiva (FOP)?\", \"ideal_answer\": [\" Recently, FOP has been associated with a specific mutation of ACVR1, the gene coding for a bone morphogenetic protein type I receptor. \", \"here, it is noted that if b cells are found to be the lymphocytes responsible for excess bmp-4 production in fop, use of rituximab, a monoclonal anti-cd20 antibody which effectively targets b cells, could be a less permanent and less risky treatment alternative for fop.\", \"Inhibitors of ALK2(ACVR1) are used for the treatment of FOP. Current therapies for FOP are Dorsomorphin analogues which function as BMP inhibitor.  if B cells control excess BMP-4 production in FOP, use of Rituximab, a monoclonal which targets B cells, could be a treatment alternative for FOP.\"], \"exact_answer\": [[\"ALK2\"], [\"BMP\"]], \"context\": \" Recently, FOP has been associated with a specific mutation of ACVR1, the gene coding for a bone morphogenetic protein type I receptor. Small molecule inhibitors of the bone morphogenetic protein (BMP) receptor kinase ALK2 (ACVR1) are needed urgently to treat the progressively debilitating musculoskeletal disease fibrodysplasia ossificans progressiva (FOP).Dorsomorphin analogues, first identified in zebrafish, remain the only BMP inhibitor chemotype reported to date.By screening an assay panel of 250 recombinant human kinases we identified a highly selective 2-aminopyridine-based inhibitor K02288 with in vitro activity against ALK2 at low nanomolar concentrations similar to the current lead compound LDN-193189Here, it is noted that if B cells are found to be the lymphocytes responsible for excess BMP-4 production in FOP, use of Rituximab, a monoclonal anti-CD20 antibody which effectively targets B cells, could be a less permanent and less risky treatment alternative for FOP.A new mutation of the Noggin gene in a French Fybrodysplasia ossificans progressiva (FOP) family: Fibrodysplasia ossificans progressiva (FOP) is a very rare disease characterized by congenital malformation of the great toes and progressive heterotopic ossification of the muscles.Localization of the gene for fibrodysplasia ossificans progressiva (FOP) to chromosome 17q21-22.A new mutation of the noggin gene in a French Fibrodysplasia ossificans progressiva (FOP) family.Rarely occurring mutation of ACVR1 gene in Moroccan patient with fibrodysplasia ossificans progressiva.A new mutation of the Noggin gene in a French Fybrodysplasia ossificans progressiva (FOP) family: Fibrodysplasia ossificans progressiva (FOP) is a very rare disease characterized by congenital malformation of the great toes and progressive heterotopic ossification of the musclesMutations in the ACVR1 gene are associated with Fibrodysplasia Ossificans Progressiva (FOP), a rare and extremely disabling disorder characterized by congenital malformation of the great toes and progressive heterotopic endochondral ossification in muscles and other non-skeletal tissuesMutations of the noggin (NOG) and of the activin A type I receptor (ACVR1) genes in a series of twenty-seven French fibrodysplasia ossificans progressiva (FOP) patientsACVR1, a therapeutic target of fibrodysplasia ossificans progressiva, is negatively regulated by miR-148a.These mutations represent new targets for therapeutic intervention in this otherwise incurable disease.In this study, we examined downstream signaling targets to study the BMP-Smad and BMP-p38 mitogen-activated protein kinase (MAPK) pathways in FOP. Mutations in the ACVR1 gene are associated with Fibrodysplasia Ossificans Progressiva (FOP), a rare and extremely disabling disorder characterized by congenital malformation of the great toes and progressive heterotopic endochondral ossification in muscles and other non-skeletal tissues. Linkage exclusion and mutational analysis of the noggin gene in patients with fibrodysplasia ossificans progressiva (FOP).Fibrodysplasia ossificans progressiva (FOP), a rare disorder characterized by progressive ossification of connective tissue, is caused by an activating mutation in Acvr1 (the gene that encodes ACVR1/ALK2).This gene encodes the type I bone morphogenic protein receptor ALK2, with the residues affected identical to those that, when mutated in the germline, give rise to the congenital malformation syndrome fibrodysplasia ossificans progressiva (FOP), resulting in the transformation of soft tissue into bone.ACVR1, a therapeutic target of fibrodysplasia ossificans progressiva, is negatively regulated by miR-148a.A new mutation of the noggin gene in a French Fibrodysplasia ossificans progressiva (FOP) family.Neofunction of ACVR1 in fibrodysplasia ossificans progressiva.The discovery of the FOP gene establishes a crucial milestone in understanding FOP, reveals a highly conserved target in the BMP signaling pathway for drug development and specifically stimulates therapeutic approaches for the development of inhibitors for ACVR1/ALK2 signaling.The discovery of the FOP gene establishes a critical milestone in understanding FOP, reveals a highly conserved druggable target in the TGF-beta/bone morphogenetic protein signaling pathway and compels therapeutic approaches for the development of small molecule signal transduction inhibitors for activin-like kinase-2.\"}, {\"question\": \"Which syndromes are associated with heterochromia iridum?\", \"ideal_answer\": [\"The syndromes that are associated with heterochromia iridum are:\\n1) Ascher's syndrome\\n2) Waardenburg Syndrome type II (WS2)\\n3) Horner's syndrome.\"], \"exact_answer\": [[\"Ascher's syndrome\"], [\"Waardenburg Syndrome type II\", \"WS2\", \"WS Type II\"], [\"Horner's syndrome\"]], \"context\": \"To our knowledge, this is the first reported case of Ascher's syndrome associated with iris coloboma, heterochromia iridum, and narrowing of horizontal palpebral fissure and decreased outer intercanthal distance secondary to lengthening of lateral canthal ligament. Sensorineural hearing loss (17/20, 85.0%) and heterochromia iridum (20/20, 100.0%) were the most commonly observed clinical features in Chinese WS2 patients. Waardenburg Syndrome (WS) is an autosomal-dominant disorder characterized by sensorineural hearing loss and pigmentary abnormalities of the eyes, hair, and skin. Microphthalmia-associated transcription factor (MITF) gene mutations account for about 15% of WS type II (WS2) cases. The association of congenital Horner's syndrome and hypochromia iridum without anhidrosis is highly suggestive of sympathetic pathway injury early in life. The Waardenburg syndrome (WS) consists of at least two distinct autosomal dominant hereditary disorders. WS Type I has been mapped to the distal part of chromosome 2q and the gene identified as PAX3. Other gene(s) are responsible for WS Type II. Sensorineural hearing loss (77%) and heterochromia iridum (47%) were the two most important diagnostic indicators for WS Type II. WAARDENBURG'S SYNDROME AND HETEROCHROMIA IRIDUM IN A DEAF SCHOOL POPULATION.Waardenburg's syndrome consists of lateral displacement of the inner canthi of the eyes (dystopia canthorum), a broad nasal root and confluent eyebrows, heterochromia iridum, a white forelock and congenital deafness.To our knowledge, this is the first reported case of Ascher's syndrome associated with iris coloboma, heterochromia iridum, and narrowing of horizontal palpebral fissure and decreased outer intercanthal distance secondary to lengthening of lateral canthal ligament.Sensorineural hearing loss (77%) and heterochromia iridum (47%) were the two most important diagnostic indicators for WS Type II.Sensorineural hearing loss (17/20, 85.0%) and heterochromia iridum (20/20, 100.0%) were the most commonly observed clinical features in Chinese WS2 patients.We present the genetic analyses conducted on a three-generation family (14 individuals) with three members affected with isolated-Hirschsprung disease (HSCR) and one with HSCR and heterochromia iridum (syndromic-HSCR), a phenotype reminiscent of Waardenburg-Shah syndrome (WS4).Heterochromia iridis and Horner's syndrome due to paravertebral neurilemmoma.[Iris heterochromia in acquired Horner's syndrome].BACKGROUND: Heterochromia iridis, asymmetry of iris pigmentation, has been well described with congenital Horner syndrome.A case of heterochromia iridis and Horner's syndrome is reported in a 7-year old girl with paravertebral neurilemmoma.We present the genetic analyses conducted on a three-generation family (14 individuals) with three members affected with isolated-Hirschsprung disease (HSCR) and one with HSCR and heterochromia iridum (syndromic-HSCR), a phenotype reminiscent of Waardenburg-Shah syndrome (WS4)To our knowledge, this is the first reported case of Aschers syndrome associated with iris coloboma, heterochromia iridum, and narrowing of horizontal palpebral fissure and decreased outer intercanthal distance secondary to lengthening of lateral canthal ligamentWAARDENBURGS SYNDROME AND HETEROCHROMIA IRIDUM IN A DEAF SCHOOL POPULATIONWaardenburgs syndrome consists of lateral displacement of the inner canthi of the eyes (dystopia canthorum), a broad nasal root and confluent eyebrows, heterochromia iridum, a white forelock and congenital deafnessSensorineural hearing loss (17/20, 85.0%) and heterochromia iridum (20/20, 100.0%) were the most commonly observed clinical features in Chinese WS2 patientsSensorineural hearing loss (17/20, 85.0%) and heterochromia iridum (20/20, 100.0%) were the most commonly observed clinical features in Chinese WS2 patients. A dominantly inherited syndrome associated with hypopigmentation, heterochromia irides, colobomatous eyes and bilateral hearing loss has been ascertained in Fleckvieh cattle (German White Fleckvieh syndrome). Sensorineural hearing loss (77%) and heterochromia iridum (47%) were the two most important diagnostic indicators for WS Type II.Sensorineural hearing loss (17/20, 85.0%) and heterochromia iridum (20/20, 100.0%) were the most commonly observed clinical features in Chinese WS2 patients.WAARDENBURG'S SYNDROME AND HETEROCHROMIA IRIDUM IN A DEAF SCHOOL POPULATION.\"}, {\"question\": \"List peptide fragmentations methods in mass spectrometry\", \"ideal_answer\": [\"CID, HCD, ECD, ETD and PSD are different peptide fragmentation technologies used in mass spectrometry.\"], \"exact_answer\": [[\"CID\", \"collision-induced dissociation\"], [\"HCD\", \"Higher-energy collisional dissociation\"], [\"ECD\", \"electron capture dissociation\"], [\"ETD\", \"electron transfer dissociation\"], [\"PSD\", \"post-source decay\"]], \"context\": \" using the CID and HCD fragmentation techniquesBy integrating HCD-MS/MS, CID-MS/MSECD and ETD induced different fragmentation processesMALDI post-source decay (PSD), MALDI 157 nm photodissociation, tris(2,4,6-trimethoxyphenyl)phosphonium bromide (TMPP) charge tagging in PSD and photodissociation, ESI collision-induced dissociation (CID), electron transfer dissociation (ETD), and free-radical initiated peptide sequencing (FRIPS) with CID were applied to peptides containing either aspartic or isoaspartic acid.automated analysis of HCD and ETD fragmentation spectraThe combination of DIA analysis and ETD fragmentation with supplemental CID energy \"}, {\"question\": \"List available circular RNA prediction tools.\", \"ideal_answer\": [\"circRNA_finder, find_circ, CIRCexplorer, CIRI, and MapSplice.\"], \"exact_answer\": [[\"circRNA_finder\"], [\"find_circ\"], [\"CIRCexplorer\"], [\"CIRI\"], [\"MapSplice\"]], \"context\": \"CircRNAs are novel members of the non-coding RNA family. For several decades circRNAs have been known to exist, however only recently the widespread abundance has become appreciated. Annotation of circRNAs depends on sequencing reads spanning the backsplice junction and therefore map as non-linear reads in the genome. Several pipelines have been developed to specifically identify these non-linear reads and consequently predict the landscape of circRNAs based on deep sequencing datasets. Here, we use common RNAseq datasets to scrutinize and compare the output from five different algorithms; circRNA_finder, find_circ, CIRCexplorer, CIRI, and MapSplice and evaluate the levels of bona fide and false positive circRNAs based on RNase R resistance. By this approach, we observe surprisingly dramatic differences between the algorithms specifically regarding the highly expressed circRNAs and the circRNAs derived from proximal splice sites. Collectively, this study emphasizes that circRNA annotation should be handled with care and that several algorithms should ideally be combined to achieve reliable predictions.\"}, {\"question\": \"List scales that are used for scoring of patients with spinal metastasis?\", \"ideal_answer\": [\"Tokuhashi, Tomita, Bauer, and Oswestry scores are used for survival prediction of patients with spinal metastases.\"], \"exact_answer\": [[\"Tokuhashi\"], [\"Tomita\"], [\"Bauer\"], [\"Oswestry\"]], \"context\": \"METHODS: Outcomes of surgery, prognostic factors for survival, and relevance of Tomita and Tokuhashi scores were investigated.PURPOSE: To assess variability in the use of Tomita and modified Bauer scores in spine metastases.Only the Tomita scoring system was shown to be an independent prognostic factor for overall survival. .CONCLUSION: The Tomita scoring system represents a practicable and highly predictive prognostic tool. Analysis of the predictive role and new proposal for surgical strategies based on the modified Tomita and Tokuhashi scoring systems for spinal metastasis.We also aimed to evaluate the validity of the prognostic scores in the Tomita and Tokuhashi systems and discuss several aspects to improve the predictive accuracy of these systems. CONCLUSIONS: Tomita scores more accurately predicted survival than Tokuhashi scores. A comparison of the modified Tokuhashi and Tomita scores in determining prognosis for patients afflicted with spinal metastasis. It is for this reason that scoring systems, such as the modified Tokuhashi and Tomita scores, have been createdThe modified Tokuhashi score had better accuracy in determining actual survival.Tokuhashi, Tomita, Bauer, and Oswestry scores have been devised for survival prediction; however, none of these systems have been evaluated in nasopharyngeal carcinoma (NPC).All four scoring systems could be used to prognosticate these patients. The modified Tokuhashi score is the best in doing so.Primary tumor type and Tokuhashi score independently predicted survival in patients with spinal metastases. Primary tumor type and Tokuhashi scoring independently predicted survival in patients with spinal metastases after surgery.A predictive score was then developed and compared against that of the modified Bauer score alone in terms of prognosticating 1-year survival after surgery.Our predictive score performed better than the modified Bauer alone and may be used to predict survival after surgical intervention for metastatic disease.Tokuhashi Scoring System has limited applicability in the majority of patients with spinal cord compression secondary to vertebral metastasis.Tokuhashi Scoring System has limited applicability in the majority of patients with spinal cord compression secondary to vertebral metastasisTokuhashi, Tomita, Bauer, and Oswestry scores have been devised for survival prediction; however, none of these systems have been evaluated in nasopharyngeal carcinoma (NPC).To investigate the accuracy of these scoring systems in predicting survival and to identify prognostic factors for survival of the patients with spinal metastases from NPC.Retrospective analysis of the patients with spinal metastases from NPC who were treated in our institution.The study included 87 patients with spinal metastases from NPC.The primary outcome measure was the survival time of these patientsPredictive value of survival by modified Tokuhashi score was the highest among all four scoring systems.Patients with spinal metastases from NPC have relatively good survival prognosisThe data of the present study emphasize that the original Bauer score and a modified Bauer score without scoring for pathologic fracture seem to be practicable and highly predictive preoperative scoring systems for patients with spinal metastasesA retrospective study of 180 patients with lung cancer spinal metastases, wherein prognostic score-predicted survival was compared with actual survival.To evaluate and compare the accuracy of prognostic scoring systems in lung cancer spinal metastases.The modified Tokuhashi, Tomita, modified Bauer, and Oswestry scores are currently used to guide decisions regarding operative treatment of patients with spinal metastases.To evaluate the predictive values of Tokuhashi score, revised Tokuhashi score and Tomita score systems for life expectancy and treatment options in patients with spinal metastasis.A comparison of the modified Tokuhashi and Tomita scores in determining prognosis for patients afflicted with spinal metastasis.The combination of Tokuhashi score and Tomita score may be applied to better predict postoperative survival prognosis and guide the surgical options for patients with spinal metastasis..reported a composite model taking into account a modified Bauer score, preoperative albumin, and ambulatory status of patients with spinal metastasis.\"}, {\"question\": \"What are the side effects during statins administration in patients with atherosclerosis?\", \"ideal_answer\": [\"The side effects during statins administration in patients with atherosclerosis are:\\n1) Myopathy\\n2) Transaminase elevations\\n3) Diabetes mellitus \\n4) Renal and neurologic adverse effects.\"], \"exact_answer\": [[\"Myopathy\"], [\"Transaminase elevations\"], [\"Diabetes mellitus\"], [\"Renal adverse effects\"], [\"Neurologic adverse effects\"]], \"context\": \"A history of side effects was reported by 40 (19.5%) of the patients, and mainly consisted of muscle complaints and gastrointestinal symptoms. Treatment Options for Statin-Associated Muscle Symptoms.Muscle symptoms are a clinically relevant side effect of statin treatment.At least 5% of patients taking statins have statin-associated muscle symptoms (SAMS). Muscle problems and other adverse symptoms associated with statin use are frequent reasons for non-adherence and discontinuation of statin therapy, which results in inadequate control of hyperlipidemia and increased cardiovascular risk. It was estimated that rosuvastatin-associated absolute hazards of muscle-, liver- and renal-related adverse effects are lower than the corresponding vascular benefits in moderate vascular risk individuals. Rosuvastatin-induced proteinuria appears to be of tubular origin, not relating to kidney injury. Rosuvastatin increases the risk of new-onset diabetes by dose-dependently impairing insulin sensitivity. Statins are associated with myopathy, transaminase elevations, and an increased risk of incident diabetes mellitus among some patients; connections between statins and other processes, such as renal and neurologic function, have also been studied with mixed results. Muscle symptoms are a clinically relevant side effect of statin treatment.This review is based on pertinent publications retrieved by a selective literature search, and on the current recommendations of the European Atherosclerosis Society.At least 5% of patients taking statins have statin-associated muscle symptoms (SAMS).Nevertheless, patients may develop side effects such as muscle pain and weakness years after starting statin therapy; however, the absolute number of patients affected by statin myopathy increases with treatment duration.Better understanding of the molecular mechanisms involved in statin-induced myopathy may help identify patient groups susceptible to statins' side effects, thereby increasing their safety..\"}, {\"question\": \"Entresto is composed of which two drugs?\", \"ideal_answer\": [\"Entresto is composed of sacubitril and valsartan. It is newly FDA-approved medication that dually inhibits angiotensin and neprilysin, in the treatment of heart failure.\"], \"exact_answer\": [[\"sacubitril\"], [\"valsartan\"]], \"context\": \"The PARADIGM-HF study, a large outcome trial in heart failure and reduced ejection fraction (HFrEF), has recently shown improved cardiovascular outcomes with sacubitril/valsartan (Entresto\\u00ae, Novartis), still commonly referred to as LCZ696, compared to ACE-inhibitor therapy, possibly leading us to a new era for heart failure (HF) treatment. This article reviews the background, current knowledge and data supporting the use of sacubitril/valsartan (Entresto(\\u00ae) ), the newly FDA-approved medication that dually inhibits angiotensin and neprilysin, in the treatment of heart failure.\\u25bc Sacubitril valsartan (Entresto-Novartis) is a new oral drug licensed for the treatment of symptomatic chronic heart failure in adults with reduced ejection fraction.Rhabdomyolysis After Coadministration of Atorvastatin and Sacubitril/Valsartan (Entresto\\u2122) in a 63-Year-Old Woman.A 63-year-old woman previously stable on a regimen of atorvastatin 40\\u00a0mg daily, carvedilol 25\\u00a0mg twice daily, digoxin 0.125\\u00a0mg daily, furosemide 40\\u00a0mg daily, spironolactone 25\\u00a0mg daily, rivaroxaban 15\\u00a0mg daily, and enalapril 20\\u00a0mg twice daily for heart failure developed rhabdomyolysis 26\\u00a0days after enalapril was stopped and sacubitril/valsartan (Entresto\\u2122) started.Sacubitril/valsartan [LCZ696 (Entresto), Novartis Pharmaceuticals Corp.] is the first in a new class of drugs that combines neprilysin inhibition with angiotensin II receptor antagonism, the combination of which acts to increase endogenous natriuretic peptides while inhibiting the renin-angiotensin-aldosterone system.In July 2015, the US Food and Drug Administration approved the first of a new class of drugs for the treatment of heart failure: Valsartan/sacubitril (formerly known as LCZ696 and currently marketed by Novartis as Entresto) combines the angiotensin receptor blocker valsartan and the neprilysin inhibitor prodrug sacubitril in a 1:1 ratio in a sodium supramolecular complex.Sacubitril/valsartan (Entresto) for chronic heart failure; brexpiprazole (Rexulti) for major depressive disorder and schizophrenia; and lumacaftor/ivacaftor (Orkambi) for cystic fibrosis involving specific CFTR mutations.Sacubritil\\u2217valsartan (Entresto, Novartis, still commonly referred to as LCZ696) is a combination drug described as a new class of dual-acting angiotensin receptor-neprilysin inhibitor (ARNi).The molecular combination of sacubitril and valsartan (Entresto) is a new drug for reducing the risk of cardiovascular death and hospitalization for heart failure in patients with chronic heart failure (NYHA Class II-IV) and reduced ejection fraction.\\u25bc Sacubitril valsartan (Entresto-Novartis) is a new oral drug licensed for the treatment of symptomatic chronic heart failure in adults with reduced ejection fraction.(1) It is described as an angiotensin receptor neprilysin inhibitor and contains the neprilysin inhibitor, sacubitril and the angiotensin II receptor antagonist, valsartan.(1-3) Here, we review the evidence for sacubitril valsartan and consider its place in the management of heart failure.\"}, {\"question\": \"What are clinical features of the de Morsier syndrome?\", \"ideal_answer\": [\"Classic triad of the De Morsier syndrome (septooptic dysplasia) includes optic nerve hypoplasia, the absence of septum pellucidum, and pituitary hypoplasia.\"], \"exact_answer\": [[\"optic nerve hypoplasia\"], [\"absence of septum pellucidum\"], [\"pituitary hypoplasia\"]], \"context\": \" SOD was formerly known as de Morsier syndrome, which associated a midline brain defect such as an absent septum pellucidum with optic nerve hypoplasia.The triad consists of optic nerve hypoplasia, pituitary hormone abnormalities, and midline brain defects, although it can vary in the severity of clinical presentation and phenotype. INTRODUCTION: Previous studies have described septooptic dysplasia (SOD) to describe patients who have optic nerve hypoplasia, the absence of septum pellucidum, and pituitary hypoplasia.BACKGROUND: Septo-optic dysplasia, also referred to as de Morsier syndrome, is a congenital condition characterized by classic triad features: midline brain abnormalities, optic nerve hypoplasia and pituitary endocrine dysfunction. Septo-optic dysplasia (SOD), otherwise called De Morsier syndrome, is a developmental anomaly of mid-line brain structures and includes optic nerve hypoplasia, absence of the septum pellucidum and hypothalamo-pituitary abnormalities). The frequently associated features of hypopituitarism and absent septum pellucidum were felt to have embryonic linkage as \\\"septo-optic dysplasia\\\" or \\\"de Morsier's syndrome.\\\"An MRI of the brain demonstrated the absence of the septum pellucidum, which confirmed a diagnosis of septo-optic dysplasia or de Morsier syndrome.Septo-optic dysplasia (De Morsier syndrome) is a developmental anomaly of mid-line brain structures and includes optic nerve hypoplasia, absence of the septum pellucidum and hypothalamo-pituitary abnormalities.Septo-optic dysplasia, also known as de Morsier syndrome, is a rare congenital entity almost always characterized by hypoplasia/dysplasia of the optical nerve, chiasma or optic radiations and the complete or partial absence of the septum pellucidum.The de Morsier syndrome, or septo-optic dysplasia, is a developmental anomaly characterized by involvement of the optic system, hypothalamic-pituitary axis and septum pellucidum.The term septooptic dysplasia was coined in 1956 by de Morsier, who pointed out the association of optic nerve hypoplasia and absence of the septum pellucidum.Septo-optic dysplasia, also referred to as de Morsier syndrome, is a congenital condition characterized by classic triad features: midline brain abnormalities, optic nerve hypoplasia and pituitary endocrine dysfunctionSepto-optic dysplasia (SOD), also referred to as de Morsier syndrome, is a rare congenital condition, characterized by two of the classic triad features: midline brain abnormalities, optic nerve hypoplasia (ONH) and pituitary endocrine dysfunctionSepto-optic dysplasia (SOD), also referred to as de Morsier syndrome, is a rare congenital condition, characterized by two of the classic triad features: midline brain abnormalities, optic nerve hypoplasia (ONH) and pituitary endocrine dysfunction. Septo-optic dysplasia, also referred to as de Morsier syndrome, is a congenital condition characterized by classic triad features: midline brain abnormalities, optic nerve hypoplasia and pituitary endocrine dysfunction.Septo-optic dysplasia (SOD), also referred to as de Morsier syndrome, is a rare congenital condition, characterized by two of the classic triad features: midline brain abnormalities, optic nerve hypoplasia (ONH) and pituitary endocrine dysfunction.[Optic nerve hypoplasia and growth hormone deficiency: de Morsier's syndrome].\"}, {\"question\": \"Which protein complexes recognize centromeric (CEN) DNA in yeast?\", \"ideal_answer\": [\"The Schizosaccharomyces pombe centromere-linked genes, LYS1 and CYH1 on chromosome I and TPS13 and RAN1 on chromosome II, have been isolated. In budding yeast, as well as in other eukaryotes, the Cse4 histone variant (known in vertebrates as CENP-A) is believed to substitute for histone H3 at the centromeric nucleosome.\", \"The Schizosaccharomyces pombe centromere-linked genes, LYS1 and CYH1 on chromosome I and TPS13 and RAN1 on chromosome II, have been isolated Histone H3 localizes to the centromeric DNA in budding yeast  In budding yeast, as well as in other eukaryotes, the Cse4 histone variant (known in vertebrates as CENP-A) is believed to substitute for histone H3 at the centromeric nucleosome\", \"The Schizosaccharomyces pombe centromere-linked genes, LYS1 and CYH1 on chromosome I and TPS13 and RAN1 on chromosome II, have been isolated\"], \"exact_answer\": [[\"Lys1\"], [\"Cyh1\"], [\"Tps13\"], [\"Ran1\"], [\"H3\"], [\"Cse4\"]], \"context\": \"The Schizosaccharomyces pombe centromere-linked genes, LYS1 and CYH1 on chromosome I and TPS13 and RAN1 on chromosome II, have been isolatedHistone H3 localizes to the centromeric DNA in budding yeast In budding yeast, as well as in other eukaryotes, the Cse4 histone variant (known in vertebrates as CENP-A) is believed to substitute for histone H3 at the centromeric nucleosomeSurprisingly, we observed a strong interaction of H3, as well as Cse4, H4, H2A, and H2B, but not histone chaperone Scm3 (HJURP in human) with the centromeric DNA. H3 localizes to centromeric DNA at all stages of the cell cycle.Our results favor a H3-Cse4 heterotypic octamer at the budding yeast centromereIn budding yeast, a single right-handed cenH3/H4/H2A/H2B tetramer wraps the \\u223c80-bp Centromere DNA Element II (CDE II) sequence of each centromere into a 'hemisome'. Here, we show that Cse4 octamers remain intact under conditions of low salt and urea that dissociate H3 octamers.\"}, {\"question\": \"Which proteins form part of the NRD complex in S. cerevisiae?\", \"ideal_answer\": [\"The purification of an ATR complex allowed identification of chromodomain-helicase-DNA-binding protein 4 (CHD4) as an ATR-associated protein by tandem mass spectrometric sequencing. CHD4 (also called Mi-2beta) is a component of a histone-deacetylase-2 (HDAC2)-containing complex, the nucleosome remodeling and deacetylating (NRD) complex. Endogenous ATR, CHD4, and HDAC2 are shown to coimmunoprecipitate, and ATR and HDAC2 coelute through two biochemical purification steps. Other members of the NRD complex, HDAC1, MTA1, and MTA2, are also detectable in ATR immunoprecipitates. Sen1 of S. cerevisiae is a known component of the NRD complex implicated in transcription termination of nonpolyadenylated as well as some polyadenylated RNA polymerase II transcripts.  We now show that Pcf11, a component of the cleavage and polyadenylation complex (CPAC), is also generally required for NRD-dependent transcription termination through the action of its C-terminal domain (CTD)-interacting domain (CID).\"], \"exact_answer\": [[\"Chd2\"], [\"Chd4\"], [\"Hdac2\"], [\"Mta1\"], [\"Mta2\"], [\"Sen1\"], [\"Pcf11\"]], \"context\": \"This activity is derived from the CHD3 and CHD4 proteins which contain helicase/ATPase domains found in SWI2-related chromatin remodelling factors, and facilitates the deacetylation of oligonucleosomal histones in vitro. We refer to this complex as the nucleosome remodelling and deacetylating (NRD) complexMolecular association between ATR and two components of the nucleosome remodeling and deacetylating complex, HDAC2 and CHD4The purification of an ATR complex allowed identification of chromodomain-helicase-DNA-binding protein 4 (CHD4) as an ATR-associated protein by tandem mass spectrometric sequencing. CHD4 (also called Mi-2beta) is a component of a histone-deacetylase-2 (HDAC2)-containing complex, the nucleosome remodeling and deacetylating (NRD) complexEndogenous ATR, CHD4, and HDAC2 are shown to coimmunoprecipitate, and ATR and HDAC2 coelute through two biochemical purification steps. Other members of the NRD complex, HDAC1, MTA1, and MTA2, are also detectable in ATR immunoprecipitates.Sen1 of S. cerevisiae is a known component of the NRD complex implicated in transcription termination of nonpolyadenylated as well as some polyadenylated RNA polymerase II transcripts. We now show that Pcf11, a component of the cleavage and polyadenylation complex (CPAC), is also generally required for NRD-dependent transcription termination through the action of its C-terminal domain (CTD)-interacting domain (CID).In Saccharomyces cerevisiae, short noncoding RNA (ncRNA) generated by RNA polymerase II (Pol II) are terminated by the NRD complex consisting of Nrd1, Nab3, and Sen1.Sen1 of S. cerevisiae is a known component of the NRD complex implicated in transcription termination of nonpolyadenylated as well as some polyadenylated RNA polymerase II transcripts.CHD4 (also called Mi-2beta) is a component of a histone-deacetylase-2 (HDAC2)-containing complex, the nucleosome remodeling and deacetylating (NRD) complex.Other members of the NRD complex, HDAC1, MTA1, and MTA2, are also detectable in ATR immunoprecipitates.In Saccharomyces cerevisiae, short noncoding RNA (ncRNA) generated by RNA polymerase II (Pol II) are terminated by the NRD complex consisting of Nrd1, Nab3, and Sen1. \"}, {\"question\": \"Which are the causes of the Koebner phenomenon?\", \"ideal_answer\": [\"The causes of the Koebner phenomenon are:\\n1) Lichen sclerosus\\n2) Vitiligo\\n3) Psoriasis and \\n4) Physical stress.\"], \"exact_answer\": [[\"Lichen sclerosus\"], [\"Vitiligo\"], [\"Psoriasis\"], [\"Physical stress\"], [\"Lichen planus\"]], \"context\": \"Lichen sclerosus has been reported at sites of injury as a Koebner phenomenon. Therefore, lower expression of keratinocyte-derived factors, including SCF, in vitiliginous keratinocytes, which could result from keratinocyte apoptosis, might be responsible for passive melanocyte death and may explain the Koebner phenomenon.Deprivation of survival factors causes the apoptosis of melanocytes. Vitiligo often develops following physical trauma, even if this is minor. The exact mechanism of the Koebner phenomenon in vitiligo is unclear.First described in 1877 as the appearance of psoriatic lesions in the uninvolved skin of psoriatic patients as a consequence of trauma, the Koebner phenomenon has since been described in numerous diseases. The Koebner phenomenon is known to occur (lichen sclerosus occurs in skin already scarred or damaged), so trauma, injury, and sexual abuse have been suggested as possible triggers of symptoms in genetically predisposed people. The triggering role of physical stress is seen in the \\\"deep Koebner\\\" phenomenon, which causes inflammation in the synovial membrane and in enthesis, resulting in peripheral arthritis. The triggering role of physical stress is seen in the \\\"deep Koebner\\\" phenomenon, which causes inflammation in the synovial membrane and in enthesis, resulting in peripheral arthritis.Two patients with unusual skin stimuli causing koebner phenomenon in psoriasis are reported.To document the role of striae distensae and striae gravidarum in causing Koebner phenomenon in cases of vitiligo, psoriasis and lichen planus.In addition, psoriasis can be caused by a\\u00a0local cutaneus trauma, known as Koebner phenomenon.Striae are documented to cause Koebner phenomenon in patients with preexisting vitiligo, psoriasis and lichen planus, the three conditions where 'true kobenerisation' has been suggested according to Boyd and Nelder classification.[Unusual Koebner phenomenon in psoriasis caused by varicella and UVB].A 48 year-old woman developed an exacerbation of a latent psoriasis as a Koebner phenomenon in a migrating erythema caused by Borrelia afzelii infection.[Psoriasis as Koebner phenomenon in erythma migrans].[Characteristics of the Koebner phenomenon in patients with psoriasis vulgaris].Vitiligo appearing in striae distensae as a Koebner phenomenon.The appearance of vitiligo in striae distensae as a form of Koebner phenomenon has been reported rarely.The Koebner phenomenon originally described the appearance of psoriatic lesions in the uninvolved skin of patients with psoriasis as a consequence of trauma.The triggering role of physical stress is seen in the \\\"deep Koebner\\\" phenomenon, which causes inflammation in the synovial membrane and in enthesis, resulting in peripheral arthritisTo document the role of striae distensae and striae gravidarum in causing Koebner phenomenon in cases of vitiligo, psoriasis and lichen planus.Striae are documented to cause Koebner phenomenon in patients with preexisting vitiligo, psoriasis and lichen planus, the three conditions where true kobenerisation has been suggested according to Boyd and Nelder classification.Striae distensae and striae gravidarum are examples of blunt trauma[Unusual Koebner phenomenon in psoriasis caused by varicella and UVB]A 48 year-old woman developed an exacerbation of a latent psoriasis as a Koebner phenomenon in a migrating erythema caused by Borrelia afzelii infectionIntradermal antigen tests and the Koebner phenomenon in psoriasisIn addition, intensity of delayed hypersensitivity reaction and resolution times demonstrate no significant difference between psoriatic and nonpsoriatic subjects statistically (P > 0.05).These findings may indicate that intradermal antigens used in this study were more effective in inducing the Koebner phenomenon than injury aloneRESULTS: Striae are documented to cause Koebner phenomenon in patients with preexisting vitiligo, psoriasis and lichen planus, the three conditions where 'true kobenerisation' has been suggested according to Boyd and Nelder classification. According to the presence of Koebner phenomenon they were divided in two groups, one with positive and the other with negative Koebner phenomenon which presented the control group at the same time.RESULTS AND DISCUSSION: The Koebner reaction is often thought to be more frequent in actively spreading, severe psoriasis.We describe a case of concurrent lichen planus and sarcoidosis in the auditory canal, which represents an unusual manifestation of the Koebner phenomenon.According to the presence of Koebner phenomenon they were divided into two groups, 20 patients with positive and 40 patients with negative Koebner reaction, who were the control group at the same time.RESULTS AND DISCUSSION: 95% of patients treated with PUVA, were cleared of psoriatic changes in the Koebner positive, as well as in the Koebner negative group.Striae are documented to cause Koebner phenomenon in patients with preexisting vitiligo, psoriasis and lichen planus, the three conditions where 'true kobenerisation' has been suggested according to Boyd and Nelder classification.\"}, {\"question\": \"Which are the typical symptoms of M\\u00e9ni\\u00e8re's disease?\", \"ideal_answer\": [\"The typical symptoms of M\\u00e9ni\\u00e8re's disease are:\\n1) sensorineural hearing loss, \\n2) vertigo and \\n3) tinnitus.\"], \"exact_answer\": [[\"sensorineural hearing loss\", \"hypoacusia\"], [\"vertigo\", \"dizziness\"], [\"tinnitus\"]], \"context\": \"To evaluate the onset of vertigo, hearing loss and tinnitus in M\\u00e9ni\\u00e8re's disease and the associated endolymphatic hydrops (EH) of the inner ear.We suggest that a 3T MRI measurement should be carried out in patients with sensory-neural hearing loss, vertigo and tinnitus, 4 h after the intravenous injection of a gadolinium-contrast agent to verify the inner ear pathology. Eleven of the 17425 veterans appeared to have typical M\\u00e9ni\\u00e8re's Disease. Their symptoms included attacks of vertigo, lasting between half an hour and a few hours and no more than 24 hours; the sensation of aural fullness; and tinnitus accompanied by a fluctuating or permanent low-tone hearing loss. Diagnosis of Meni\\u00e8re's Disease is based upon the wellknown labyrinthic syndrome (hypoacusia, tinnitus and dizziness) which manifests with the typical abscessual, recurrent and unforeseeable course. Typical clinical manifestations of M\\u00e9ni\\u00e8re's disease (vertigo, sensorineural hearing loss and tinnitus) were found in 6/11 patients (54.5%) in the Lyme disease group.Among 93 patients presenting the typical symptoms of a M\\u00e9ni\\u00e8re's disease associating an unilateral fluctuating hearing loss of sensorineural type, tinnitus and vertiginous attacks lasting minutes to hours, 40 patients (43%) presented in their personal history a particular otologic insult in the ear which later on developed into the full M\\u00e9ni\\u00e8re's symptomatology, or a particular systemic disease with otologic manifestations. Fourteen children (aged 14 years or younger) with typical M\\u00e9ni\\u00e8re's triad with cochlear sensorineural hearing loss, tinnitus, and intermittent vertigo attacks lasting from minutes to hours were investigated in four different neuro-otologic centers. Among 93 patients presenting the typical symptoms of a M\\u00e9ni\\u00e8re's disease associating an unilateral fluctuating hearing loss of sensorineural type, tinnitus and vertiginous attacks lasting minutes to hours, 40 patients (43%) presented in their personal history a particular otologic insult in the ear which later on developed into the full M\\u00e9ni\\u00e8re's symptomatology, or a particular systemic disease with otologic manifestations.To evaluate the onset of vertigo, hearing loss and tinnitus in M\\u00e9ni\\u00e8res disease and the associated endolymphatic hydrops (EH) of the inner ear.Multicentre evaluation of three patient groups.Disease-specific symptoms were reviewed among referred patients in a tertiary referral hospital in Finland and in members of a Finnish M\\u00e9ni\\u00e8re Association in FinlandIn monosymptomatic patients with vertigo, tinnitus or hearing loss; EH was demonstrated in 55-90% of the patients either in the cochlea and/or the vestibulum of the symptomatic ear.M\\u00e9ni\\u00e8res disease often shows bilateral EH and comprises a continuum from a monosymptomatic disease to the typical symptom complex of the diseaseTypical clinical manifestations of M\\u00e9ni\\u00e8re's disease (vertigo, sensorineural hearing loss and tinnitus) were found in 6/11 patients (54.5%) in the Lyme disease group. Nine children, labeled as having \\\"idiopathic M\\u00e9ni\\u00e8re's disease,\\\" developed the auditory and vestibular symptoms without any detectable causative factor.To evaluate the onset of vertigo, hearing loss and tinnitus in M\\u00e9ni\\u00e8re's disease and the associated endolymphatic hydrops (EH) of the inner ear.Multicentre evaluation of three patient groups.Disease-specific symptoms were reviewed among referred patients in a tertiary referral hospital in Finland and in members of a Finnish M\\u00e9ni\\u00e8re Association in Finland.In each twin pair, one twin had migraine and M\\u00e9ni\\u00e8re's disease, whereas the other experienced migraine and episodic vertigo without auditory symptoms.The frequent association of episodic vertigo, migraine, and M\\u00e9ni\\u00e8re's disease in closely related individuals, including identical twins supports the heritability of a migraine-M\\u00e9ni\\u00e8re's syndrome, with variable expression of the individual features of hearing loss, episodic vertigo, and migraine headaches.There has been literature, beginning with M\\u00e9ni\\u00e8re himself, suggesting a relationship between M\\u00e9ni\\u00e8re's disease and migraine-associated dizziness.In the characterizations of the migraine-associated dizziness, the signs and symptoms show overlap with those characterizing Meniere's disease.\"}, {\"question\": \"List metalloenzyme inhibitors.\", \"ideal_answer\": [\"Foscarnet\\nVT-1129\\nVT-1161 \\nBB-3497\\nhydroxamate molecules\\nsiderophores\"], \"exact_answer\": [[\"VT-1129\"], [\"VT-1161\"], [\"BB-3497\"], [\"hydroxamate molecules\"], [\"siderophores\"], [\"Foscarnet\"]], \"context\": \" Clinically approved inhibitors were selected as well as several other reported metalloprotein inhibitors in order to represent a broad range of metal binding groups (MBGs), including hydroxamic acid, carboxylate, hydroxypyridinonate, thiol, and N-hydroxyurea functional groups.A total of 21 different raltegravir-chelator derivative (RCD) compounds were prepared that differed only in the nature of the MBG. At least two compounds (RCD-4, RCD-5) containing a hydroxypyrone MBG were found to display superior strand-transfer inhibition when compared to an abbreviated analogue of raltegravir (RCD-1). By screening a library of metalloenzyme inhibitors, the N-formyl-hydroxylamine derivative BB-3497 was identified as a potent inhibitor of Escherichia coli peptide deformylase with antibacterial activity both in vitro and in vivo. The 8-hydroxyquinoline represents a promising new chelator scaffold for the development of MMP inhibitors that was discovered by use of a metalloprotein-focused chelator fragment library. the metalloenzyme inhibitors VT-1129 and VT-1161 (both Viamet Pharmaceuticals)Based on their ability to chelate metals, hydroxamate molecules and siderophores have been successfully used as metalloenzyme inhibitors. Using this model, we identified two nitrogen donor compounds--2,2'-dipyridylamine (DPA) and triazacyclononane (TACN)--as the most selective ZBGs for zinc metalloenzyme inhibitor development. Foscarnet (phosphonoformate trisodium salt), an antiviral used for the treatment of HIV and herpes virus infections, also acts as an activator or inhibitor of the metalloenzyme carbonic anhydrase (CA, EC 4.2.1.1)\"}, {\"question\": \"Which proteins remove H2A.Z in the yeast Saccharomyces cerevisiae?\", \"ideal_answer\": [\"Removal of H2A.Z by INO80 promotes homologous recombination Budding yeast INO80 can remove H2A.Z/H2B dimers from chromatin and replace them with H2A/H2B dimers. Here, we show that H2A.Z in human cells is indeed rapidly removed from chromatin flanking DNA damage by INO80.\", \"Here, we show that H2A.Z in human cells is indeed rapidly removed from chromatin flanking DNA damage by INO80. We also report that the histone chaperone ANP32E, which is implicated in removing H2AZ from chromatin, similarly promotes HR and appears to work on the same pathway as INO80 in these assays. Importantly, we demonstrate that the HR defect in cells depleted of INO80 or ANP32E can be rescued by H2A.Z co-depletion, suggesting that H2A.Z removal from chromatin is the primary function of INO80 and ANP32E in promoting homologous recombination.\", \"Budding yeast INO80 can remove H2A.Z/H2B dimers from chromatin and replace them with H2A/H2B dimers. Here, we show that H2A.Z in human cells is indeed rapidly removed from chromatin flanking DNA damage by INO80. We also report that the histone chaperone ANP32E, which is implicated in removing H2AZ from chromatin, similarly promotes HR and appears to work on the same pathway as INO80 in these assays.\", \"we also report that the histone chaperone anp32e, which is implicated in removing h2az from chromatin, similarly promotes hr and appears to work on the same pathway as ino80 in these assays.\", \"Budding yeast INO80 can remove H2A.Z/H2B dimers from chromatin and replace them with H2A/H2B dimers. Removal of H2A.Z by INO80 promotes homologous recombination H2A.Z removal from chromatin is the primary function of INO80 and ANP32E in promoting homologous recombination.\", \"Budding yeast INO80 can remove H2A.Z/H2B dimers from chromatin and replace them with H2A/H2B dimers. Removal of H2A.Z by INO80 promotes homologous recombination\", \"Budding yeast INO80 can remove H2A.Z/H2B dimers from chromatin and replace them with H2A/H2B dimers. H2A.Z removal from chromatin is the primary function of INO80 and ANP32E in promoting homologous recombination.\"], \"exact_answer\": [[\"INO80\"], [\"ANP32E\"]], \"context\": \"Removal of H2A.Z by INO80 promotes homologous recombinationBudding yeast INO80 can remove H2A.Z/H2B dimers from chromatin and replace them with H2A/H2B dimers.Here, we show that H2A.Z in human cells is indeed rapidly removed from chromatin flanking DNA damage by INO80.We also report that the histone chaperone ANP32E, which is implicated in removing H2AZ from chromatin, similarly promotes HR and appears to work on the same pathway as INO80 in these assays.H2A.Z removal from chromatin is the primary function of INO80 and ANP32E in promoting homologous recombination.Instead, the INO80 complex, which removes H2A.Z(Htz1) from nucleosomes, promotes the ectopic deposition of CENP-A(Cse4).\"}, {\"question\": \"Which diseases are associated with Primary intestinal lymphangiectasia (PIL)?\", \"ideal_answer\": [\"Primary intestinal lymphangiectasia (PIL) is associated with:\\n1) Waldmann's disease and\\n2) Hennekam syndrome (HS).\"], \"exact_answer\": [[\"Waldmann's disease\"], [\"Hennekam syndrome\", \"HS\"]], \"context\": \"Primary intestinal lymphangiectasia (PIL), also known as Waldmann's disease, is an exudative enteropathy resulting from morphologic abnormalities in the intestinal lymphatics. Secondary hypogammaglobulinemia in Waldmann's disease treated with subcutaneous immunoglobulins.Octreotide in Hennekam syndrome-associated intestinal lymphangiectasia.Primary intestinal lymphangiectasia (PIL) is one mechanism leading to PLE. Few syndromes are associated with PIL; Hennekam syndrome (HS) is one of them. There are no publications on the treatment of PIL with octreotide in patients with HS. To the best of our knowledge, this is the first case report on the use of octreotide in HS-associated PIL.Primary intestinal lymphangiectasia (PIL), also known as Waldmann's disease, is a rare disorder characterized by dilated intestinal lacteals resulting in lymph leakage into the small bowel lumen and responsible for protein-losing enteropathy leading to lymphopenia, hypoalbuminemia and hypogammaglobulinemia.Primary intestinal lymphangiectasia (Waldmann's disease).Limb lymphedema as a first manifestation of primary intestinal lymphangiectasia (Waldmann's disease)Primary intestinal lymphangiectasia (Waldmann's disease) is characterized by protein-losing enteropathy occurring more frequently in childhood. Primary intestinal lymphangiectasia (PIL), also known as Waldmann's disease, is an exudative enteropathy resulting from morphologic abnormalities in the intestinal lymphatics.Primary intestinal lymphangiectasia (PIL), so-called Waldmann's disease, is an uncommon condition, characterized by dilated intestinal submucosal and subserosal lymphatics of the gastrointestinal tract.Primary intestinal lymphangiectasia (PIL) or Waldmann's disease is a rare protein-losing gastroenteropathy of unknown etiology.Primary intestinal lymphangiectasia (PIL), also known as Waldmanns disease, is a rare disorder characterized by dilated intestinal lacteals resulting in lymph leakage into the small bowel lumen and responsible for protein-losing enteropathy leading to lymphopenia, hypoalbuminemia and hypogammaglobulinemiaPrimary intestinal lymphangiectasia (PIL) or Waldmanns disease is a rare protein-losing gastroenteropathy of unknown etiologyPrimary intestinal lymphangiectasia (PIL), so-called Waldmanns disease, is an uncommon condition, characterized by dilated intestinal submucosal and subserosal lymphatics of the gastrointestinal tractPrimary intestinal lymphangiectasia (Waldmanns disease) is a rare disease characterized by dilated lymphatics in the small bowel leading to an exudative enteropathy with lymphopenia, hypoalbuminemia and hypogammaglobulinemia.We report the case of a 23 year-old male who presented with chronic anemia and in whom primary intestinal lymphangiectasia was diagnosedPrimary intestinal lymphangiectasis (PIL), also known as Waldmanns disease, is a rare protein-losing enteropathy characterized by abnormal enlargement of the lymphatic ducts in the bowel wallFew syndromes are associated with PIL; Hennekam syndrome (HS) is one of them. Primary intestinal lymphangiectasia (PIL), also known as Waldmann's disease, is a rare disorder characterized by dilated intestinal lacteals resulting in lymph leakage into the small bowel lumen and responsible for protein-losing enteropathy leading to lymphopenia, hypoalbuminemia and hypogammaglobulinemia. Primary intestinal lymphangiectasia (Waldmann's disease) is characterized by protein-losing enteropathy occurring more frequently in childhood.Primary intestinal lymphangiectasia (Waldmann's disease).Octreotide in Hennekam syndrome-associated intestinal lymphangiectasia.\"}, {\"question\": \"Please list 2 treatments for a torn rotator cuff\", \"ideal_answer\": [\"to compare clinical outcomes between conventional en masse repair and separate double-layer double-row repair for the treatment of delaminated rotator cuff tears.\", \"Surgical repair earlier or later than 3 months after injury may result in similar outcomes and patient satisfaction.\"], \"exact_answer\": [[\"early surgery\"], [\"later surgery(3 months after injury)\"], [\"en masse repair\"], [\"double-layer double-row repair\"], [\"arthroscopic\"], [\"open\"], [\"knotless transosseous-equivalent\"]], \"context\": \"To compare clinical outcomes between conventional en masse repair and separate double-layer double-row repair for the treatment of delaminated rotator cuff tears To systematically review and evaluate the effectiveness of grafts in the augmentation of large-to-massive rotator cuff repairsTo evaluate the effectiveness of arthroscopic debridement (DB), partial (PR), and complete repair (CR) for massive rotator cuff tears (mRCT) The purpose was to investigate whether surgical repair earlier or later than 3\\u00a0months after injury may result in similar outcomes and patient satisfactiondouble-row (DR) and knotless transosseous-equivalent (KL-TOE) arthroscopic subacromial decompression and rotator-cuff debridement; the other comparable group of 23 patients had open repair and acromioplasty\"}, {\"question\": \"Which are the key players on radial glial specification to ependymal cells?\", \"ideal_answer\": [\"Mcidas and GemC1/Lynkeas specify embryonic radial glial cells. Both proteins were initially described as cell cycle regulators revealing sequence similarity to Geminin. They are expressed in radial glial cells committed to the ependymal cell lineage during embryogenesis, while overexpression and knock down experiments showed that are sufficient and necessary for ependymal cell generation.\"], \"exact_answer\": [[\"Mcidas\"], [\"GemC1\", \"Lynkeas\"]], \"context\": \"Mcidas and GemC1/Lynkeas specify embryonic radial glial cells.Here we discuss recent findings suggesting that 2 novel molecules, Mcidas and GemC1/Lynkeas are key players on radial glial specification to ependymal cells. Both proteins were initially described as cell cycle regulators revealing sequence similarity to Geminin. They are expressed in radial glial cells committed to the ependymal cell lineage during embryogenesis, while overexpression and knock down experiments showed that are sufficient and necessary for ependymal cell generation. We propose that Mcidas and GemC1/Lynkeas are key components of the molecular cascade that promotes radial glial cells fate commitment toward multiciliated ependymal cell lineage operating upstream of c-Myb and FoxJ1.Here we discuss recent findings suggesting that 2 novel molecules, Mcidas and GemC1/Lynkeas are key players on radial glial specification to ependymal cells.Overexpression and knockdown experiments show that Mcidas and GemC1 are sufficient and necessary for cell fate commitment and differentiation of radial glial cells to multiciliated ependymal cellsHere we discuss recent findings suggesting that 2 novel molecules, Mcidas and GemC1/Lynkeas are key players on radial glial specification to ependymal cells.Our results suggest that Mcidas and GemC1 are key players in the generation of multiciliated ependymal cells of the adult neurogenic niche.We propose that Mcidas and GemC1/Lynkeas are key components of the molecular cascade that promotes radial glial cells fate commitment toward multiciliated ependymal cell lineage operating upstream of c-Myb and FoxJ1.Mcidas and GemC1 are key regulators for the generation of multiciliated ependymal cells in the adult neurogenic niche.\"}, {\"question\": \"List diseases associated with the  Dopamine Receptor D4 (DRD4).\", \"ideal_answer\": [\"The 5-repeat allele of a common length polymorphism in the gene that encodes the dopamine D4 receptor (DRD4) is robustly associated with the risk of attention deficit hyperactivity disorder (ADHD).\"], \"exact_answer\": [[\"attention deficit hyperactivity disorder\", \"ADHD\"], [\"susceptibility to neuropsychiatric diseases\"]], \"context\": \"The 5-repeat allele of a common length polymorphism in the gene that encodes the dopamine D4 receptor (DRD4) is robustly associated with the risk of attention deficit hyperactivity disorder (ADHD) and substantially exists in Asian populations, which have a lower ADHD prevalence.The variable number of tandem repeats (VNTR) of the dopamine receptor D4 (DRD4) gene among humans may elucidate individual differences in susceptibility to neuropsychiatric diseases. Dopamine dysfunction may be involved with Attention Deficit Hyperactivity Disorder (ADHD) symptoms.This review summarized the ongoing research of DA receptor genes in ADHD pathogenesis and gathered the past published data with meta-analysis and revealed the high risk of DRD5, DRD2, and DRD4 polymorphisms in ADHD.\"}, {\"question\": \"Which bacteria cause diphtheria?\", \"ideal_answer\": [\"Diphtheria is caused by the bacteria:\\n1) Corynebacterium ulcerans and \\n2) Corynebacterium diphtheriae.\"], \"exact_answer\": [[\"Corynebacterium ulcerans\", \"C. ulcerans\"], [\"Corynebacterium diphtheriae\", \"C. diphtheriae\"]], \"context\": \"Corynebacterium ulcerans is a zoonotic pathogen that can produce diphtheria toxin and causes an illness categorized as diphtheria in the European Union because its clinical appearance is similar to that of diphtheria caused by Corynebacterium diphtheriae. The zoonotic bacterium Corynebacterium ulcerans may be pathogenic both in humans and animals: toxigenic strains can cause diphtheria or diphtheria-like disease in humans via diphtheria toxin, while strains producing the dermonecrotic exotoxin phospholipase D may lead to caseous lymphadenitis primarily in wild animals. Diphtheria toxin-positive Corynebacterium ulcerans strains have been isolated mainly from cattle, dogs and cats.Corynebacterium diphtheriae, a pathogenic bacterium that causes diphtheria, has been performed. Corynebacterium ulcerans 0102 carries the gene encoding diphtheria toxin on a prophage different from the C. diphtheriae NCTC 13129 prophage.Corynebacterium ulcerans can cause a diphtheria-like illness, especially when the bacterium is lysogenized with a tox gene-carrying bacteriophage that produces diphtheria toxin. Acquisition of toxigenicity upon phage lysogenization is a common feature of C. ulcerans and C. diphtheriae. Corynebacterium ulcerans (toxigenic C. ulcerans) produces the diphtheria toxin, which causes pharyngeal and cutaneous diphtheria-like disease in people, and this bacterium is commonly detected in dogs and cats that are reared at home. This case demonstrates that strains of C. ulcerans that produce diphtheria toxin can cause infections of the skin that completely mimic typical cutaneous diphtheria, thereby potentially providing a source of bacteria capable of causing life-threatening diseases in the patient's environment.Diphtheria is a serious upper respiratory tract disease caused by the diphtheria toxin (DT) secreted from the bacteria Corynebacterium diphtheriae.ulcerans produce diphtheria toxin, which can cause life-threatening cardiopathies and neuropathies in humans.A high-density growth approach was utilized to produce mutated diphtheria toxin from two strains of Corynebacterium diphtheria: C7 (beta)(tox-201,tox-9) and C7 (beta)(tox-107).For the primary prevention of disease caused by diphtheria toxin-producing corynebacteria, vaccination with diphtheria toxoid is recommended.ulcerans that produce diphtheria toxin can cause infections of the skin that completely mimic typical cutaneous diphtheria, thereby potentially providing a source of bacteria capable of causing life-threatening diseases in the patient's environment.[A case of diphtheria in the Netherlands due to an infection with Corynebacterium ulcerans].Diphtheria is an acute bacterial illness caused by toxigenic strains of Corynebacterium diphtheriae (C. diphtheriae)Diphtheria is caused by diphtheria toxin-producing Corynebacterium speciesDiphtheria, an acute infectious condition caused by Corynebacterium diphtheriae, was once a major killer of childrenDiphtheria, caused by toxigenic strains of Corynebacterium diphtheriae, is an ancient disease with high incidence and mortality that has always been characterized by epidemic waves of occurrenceInfections caused by Corynebacterium diphtheriae frequently induce situations in which very small doses of antigens injected intradermally can cause strong inflammatory reactionsCorynebacterium diphtheriae is the etiological agent of diphtheria, a potential fatal disease caused by a corynephage toxin[Diphtheria in Denmark 1956-1989. Occurrence of Corynebacterium diphtheriae and other diphtheria toxigenic bacteria].Matrix-assisted laser desorption/ionisation time-of-flight (MALDI-TOF) mass spectrometry as a tool for rapid diagnosis of potentially toxigenic Corynebacterium species in the laboratory management of diphtheria-associated bacteria.This case demonstrates that strains of C. ulcerans that produce diphtheria toxin can cause infections of the skin that completely mimic typical cutaneous diphtheria, thereby potentially providing a source of bacteria capable of causing life-threatening diseases in the patient's environment. Detection of differences in the nucleotide and amino acid sequences of diphtheria toxin from Corynebacterium diphtheriae and Corynebacterium ulcerans causing extrapharyngeal infections.Heterogeneity of diphtheria toxin gene, tox, and its regulatory element, dtxR, in Corynebacterium diphtheriae strains causing epidemic diphtheria in Russia and Ukraine.Diphtheria is a serious upper respiratory tract disease caused by the diphtheria toxin (DT) secreted from the bacteria Corynebacterium diphtheriae. Laboratory guidelines for the diagnosis of infections caused by Corynebacterium diphtheriae and C. ulcerans. A possible biological mechanism for a diphtheria effect on hearing ability exists: The toxin produced by the Corynebacterium diphtheriae bacteria can cause damage to cranial nerves and therefore may affect the auditory neural pathway. A possible biological mechanism for a diphtheria effect on hearing ability exists: The toxin produced by the Corynebacterium diphtheriae bacteria can cause damage to cranial nerves and therefore may affect the auditory neural pathway.In cases of human infection with potentially toxigenic corynebacteria, it is important to determine the species and examine the isolate for diphtheria toxin production.AIM: Study of the apoptogenic effect of Corynebacterium diphtheriae toxigenic strains on mice peritoneal macrophages in vitro.MATERIALS AND METHODS: Evaluation ofapoptosis induced by Corynebacterium diphtheriae, Corynebacterium pseudodiphtheriticum, Staphylococcus aureus, Streptococcus pyogenes strains was performed by characteristic morphological changes in macrophages in smears stained by azure eosin by Romanovsky-Giemsa.RESULTS: Apoptogenic activity of diphtheria infectious agent was established to be determined by diphtheria exotoxin at early (after 1 hour) and surface structures and pathogenicity enzymes at later (3 hours) stages of effect.CONCLUSION: The ability of diphtheria infectious agent to cause macrophage apoptosis is one of the mechanisms of realization of its pathogenic properties determined by the effect of diphtheria exotoxin, as well as its surface structures and pathogenicity enzymes.Extrapharyngeal infections caused by Corynebacterium ulcerans have rarely been reported previously, and diphtheria toxin production has usually not been addressed.Corynebacterium ulcerans may cause diphtheria in humans and caseous lymphadenitis in animals.ulcerans for the first time, we show that related sequence types (STs) might be associated with the presence of the diphtheria toxin gene, which encodes diphtheria toxin (DT), the most important diphtheria-causing virulence factor.Human-to-human-transmitted Corynebacterium diphtheriae was historically the main pathogen causing diphtheria and has therefore been studied extensively in the past.Toxigenic Corynebacterium diphtheriae, the organism causing diphtheria, was thought to have become rare or even have disappeared from previously endemic areas such as South Dakota.Infection of the skin caused by Corynebacterium ulcerans and mimicking classical cutaneous diphtheria.\"}, {\"question\": \"List drug that were evaluated in the CHAMP trial for migraine.\", \"ideal_answer\": [\"Childhood and Adolescent Migraine Prevention (CHAMP) trial evaluated effectiveness of amitriptyline and topiramate in the prevention of migraine in children and adolescents.\"], \"exact_answer\": [[\"amitriptyline\"], [\"topiramate\"]], \"context\": \"OBJECTIVE: To describe baseline headache characteristics of children and adolescents participating in a multicenter, randomized, double-blinded, placebo-controlled, comparative effectiveness study of amitriptyline, topiramate, and placebo for the prevention of migraine (CHAMP Study).Childhood and Adolescent Migraine Prevention (CHAMP) study: a double-blinded, placebo-controlled, comparative effectiveness study of amitriptyline, topiramate, and placebo in the prevention of childhood and adolescent migraine.METHODS: CHAMP is a double-blinded, placebo-controlled, multicenter, comparative effectiveness study of amitriptyline and topiramate for the prevention of episodic and chronic migraine, designed to mirror real-world practice, sponsored by the US National Institute of Neurological Disorders and Stroke/National Institutes of Health (U01NS076788). The subjects will be randomized in a 2:2:1 (amitriptyline: topiramate: placebo) ratio. Doses are weight based and will be slowly titrated over an 8-week period to a target dose of 1\\u2009mg/kg of amitriptyline and 2\\u2009mg/kg of topiramate. CONCLUSIONS: The goal of the CHAMP study is to obtain level 1 evidence for the effectiveness of amitriptyline and topiramate in the prevention of migraine in children and adolescents.To describe baseline headache characteristics of children and adolescents participating in a multicenter, randomized, double-blinded, placebo-controlled, comparative effectiveness study of amitriptyline, topiramate, and placebo for the prevention of migraine (CHAMP Study).The aim of the Childhood and Adolescent Migraine Prevention (CHAMP) study is to determine the most effective preventive agent to use in children and adolescents.METHODS: CHAMP is a double-blinded, placebo-controlled, multicenter, comparative effectiveness study of amitriptyline and topiramate for the prevention of episodic and chronic migraine, designed to mirror real-world practice, sponsored by the US National Institute of Neurological Disorders and Stroke/National Institutes of Health (U01NS076788).Childhood and Adolescent Migraine Prevention (CHAMP) study: a double-blinded, placebo-controlled, comparative effectiveness study of amitriptyline, topiramate, and placebo in the prevention of childhood and adolescent migraineThe aim of the Childhood and Adolescent Migraine Prevention (CHAMP) study is to determine the most effective preventive agent to use in children and adolescents.CHAMP is a double-blinded, placebo-controlled, multicenter, comparative effectiveness study of amitriptyline and topiramate for the prevention of episodic and chronic migraine, designed to mirror real-world practice, sponsored by the US National Institute of Neurological Disorders and Stroke/National Institutes of Health (U01NS076788)CONCLUSIONS: The goal of the CHAMP study is to obtain level 1 evidence for the effectiveness of amitriptyline and topiramate in the prevention of migraine in children and adolescents. Childhood and Adolescent Migraine Prevention (CHAMP) study: a double-blinded, placebo-controlled, comparative effectiveness study of amitriptyline, topiramate, and placebo in the prevention of childhood and adolescent migraine.\"}, {\"question\": \"Which two drugs were compared in the ARISTOTLE Trial?\", \"ideal_answer\": [\"Apixaban for Reduction In Stroke and Other Thromboembolic Events in Atrial Fibrillation (ARISTOTLE) trial compared apixaban and warfarin.\"], \"exact_answer\": [[\"apixaban\"], [\"warfarin\"]], \"context\": \"METHODS: We analyzed patients from ARISTOTLE, a randomized trial of 18,201 patients with atrial fibrillation comparing the effects of apixaban versus warfarin on the risk of stroke or systemic embolism.  In this study we evaluated the relationship between inflammatory biomarkers at baseline and future risk of cardiovascular events in the Apixaban for Reduction In Stroke and Other Thromboembolic Events in Atrial Fibrillation (ARISTOTLE) trial.METHODS: The ARISTOTLE trial randomised 18,201 patients with AF to apixaban or warfarin.Moreover, apixaban was associated with decreased all-cause mortality (RR 0.89, 95% CI 0.81 to 0.99) although this finding was driven by the results of the ARISTOTLE trial.Compared with warfarin, apixaban (in the ARISTOTLE trial) and edoxaban (in the ENGAGE-AF trial) were shown to be superior or non-inferior, respectively, for reduction in stroke or SE risk in patients with AF. METHODS: A decision-analytic Markov model was constructed to assess the cost-effectiveness of apixaban versus warfarin, based on data from the Apixaban for Reduction in Stroke and Other Thromboembolic Events in AF (ARISTOTLE) trial. The following phase 3 randomized trials with novel oral anticoagulants versus warfarin for stroke prevention in AF have been completed: the Randomized Evaluation of Long-term Anticoagulant therapy (RE-LY) trial with dabigatran, the Rivaroxaban Once daily oral direct Factor Xa inhibition Compared with vitamin K antagonism for prevention of stroke and Embolism Trial in Atrial Fibrillation (ROCKET-AF) trial with rivaroxaban, and the Apixaban for Reduction of Stroke and Other Thromboembolism Events in Atrial Fibrillation (ARISTOTLE) trial with apixaban.The following phase 3 randomized trials with novel oral anticoagulants versus warfarin for stroke prevention in AF have been completed: the Randomized Evaluation of Long-term Anticoagulant therapy (RE-LY) trial with dabigatran, the Rivaroxaban Once daily oral direct Factor Xa inhibition Compared with vitamin K antagonism for prevention of stroke and Embolism Trial in Atrial Fibrillation (ROCKET-AF) trial with rivaroxaban, and the Apixaban for Reduction of Stroke and Other Thromboembolism Events in Atrial Fibrillation (ARISTOTLE) trial with apixabanWe sought to compare the efficacy and safety of the 3 new agents based on data from their published warfarin-controlled randomized trials, using the method of adjusted indirect comparisons.We included findings from 44 535 patients enrolled in 3 trials of the efficacy of dabigatran (Randomized Evaluation of Long-Term Anticoagulation Therapy [RELY]), apixaban (Apixaban for Reduction in Stroke and Other Thromboembolic Events in Atrial Fibrillation [ARISTOTLE]), and rivaroxaban (Rivaroxaban Once Daily Oral Direct Factor Xa Inhibition Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial in Atrial Fibrillation [ROCKET-AF]), each compared with warfarin.A comparative analysis of three major clinical trials with factor Xa inhibitor oral anticoagulant (XOAC) drugs versus warfarin in atrial fibrillation-Rocket-AF (rivaroxaban), Aristotle (apixaban) and Engage AF Timi 48 (edoxaban; two different doses and sets of data)-was carried out.Data were extracted from the original reports (study level) and a meta-analysis was carried out.When compared with warfarin, XOAC therapy was associated with a decrease in haemorrhagic stroke, with a similar pattern for all regimens and meta-analysis showing a risk ratio of 0.488 (95% CI 0.Apixaban compared with warfarin in patients with atrial fibrillation and previous stroke or transient ischaemic attack: a subgroup analysis of the ARISTOTLE trial.Efficacy and safety of apixaban compared with warfarin for stroke prevention in patients with atrial fibrillation from East Asia: a subanalysis of the Apixaban for Reduction in Stroke and Other Thromboembolic Events in Atrial Fibrillation (ARISTOTLE) Trial.Efficacy of apixaban when compared with warfarin in relation to renal function in patients with atrial fibrillation: insights from the ARISTOTLE trial.A comparative analysis of three major clinical trials with factor Xa inhibitor oral anticoagulant (XOAC) drugs versus warfarin in atrial fibrillation-Rocket-AF (rivaroxaban), Aristotle (apixaban) and Engage AF Timi 48 (edoxaban; two different doses and sets of data)-was carried out.Data were extracted from the original reports (study level) and a meta-analysis was carried out.When compared with warfarin, XOAC therapy was associated with a decrease in haemorrhagic stroke, with a similar pattern for all regimens and meta-analysis showing a risk ratio of 0.488 (95% CThis study sought to characterize major bleeding on the basis of the components of the major bleeding definition, to explore major bleeding by location, to define 30-day mortality after a major bleeding event, and to identify factors associated with major bleeding.Apixaban was shown to reduce the risk of major hemorrhage among patients with atrial fibrillation in the ARISTOTLE (Apixaban for Reduction in Stroke and Other Thromboembolic Events in Atrial Fibrillation) trial.All patients who received at least 1 dose of a study drug were included.Little is known about patients with 1 dose-reduction criterion who received the 5 mg twice daily dose of apixaban.To determine the frequency of 1 dose-reduction criterion and whether the effects of the 5 mg twice daily dose of apixaban on stroke or systemic embolism and bleeding varied among patients with 1 or no dose-reduction criteria.Among 18 201 patients in the ARISTOTLE trial, 17\\u202f322 were included in this analysis.We sought to compare the efficacy and safety of the 3 new agents based on data from their published warfarin-controlled randomized trials, using the method of adjusted indirect comparisons.We included findings from 44 535 patients enrolled in 3 trials of the efficacy of dabigatran (Randomized Evaluation of Long-Term Anticoagulation Therapy [RELY]), apixaban (Apixaban for Reduction in Stroke and Other Thromboembolic Events in Atrial Fibrillation [ARISTOTLE]), and rivaroxaban (Rivaroxaban Once Daily Oral Direct Factor Xa Inhibition Compared With Vitamin K Antagonism for Prevention of Stroke and Embolism Trial iThis study aimed to evaluate the medical cost reductions associated with the use of individual NOACs instead of warfarin from the US payer perspective.Rates for efficacy and safety clinical events for warfarin were estimated as the weighted averages from the RE-LY, ROCKET-AF and ARISTOTLE trials, and event rates for NOACs were determined by applying trial hazard ratios or relative risk ratios to such weighted averages.The ARISTOTLE trial showed apixaban was superior to warfarin for stroke prevention, significantly reduced all major bleeding, and resulted in a significant reduction in all-cause mortality.The following phase 3 randomized trials with novel oral anticoagulants versus warfarin for stroke prevention in AF have been completed: the Randomized Evaluation of Long-term Anticoagulant therapy (RE-LY) trial with dabigatran, the Rivaroxaban Once daily oral direct Factor Xa inhibition Compared with vitamin K antagonism for prevention of stroke and Embolism Trial in Atrial Fibrillation (ROCKET-AF) trial with rivaroxaban, and the Apixaban for Reduction of Stroke and Other Thromboembolism Events in Atrial Fibrillation (ARISTOTLE) trial with apixaban.In the ARISTOTLE trial, apixaban was superior to warfarin in the reduction of stroke or systemic embolism, major bleeding, intracranial hemorrhage, and all-cause mortality, with a similar reduction in the rate of ischemic stroke and better tolerability.The results of the ARISTOTLE trial, which is evaluating apixaban against warfarin in \\u223c18\\u2009000 patients with AF, are expected to be available later this year.The Medco data did not contain information for patients receiving apixaban as it was not on the market at the time of analysis.RESULTS: Stroke and MB rates among RW NVAF patients during warfarin exposure were higher compared with event rates in patients treated with warfarin in ARISTOTLE (stroke: 5.29 vs.In the ARISTOTLE trial, apixaban when compared with warfarin reduced the rate of stroke, death, and bleeding.The ARISTOTLE trial compared apixaban with warfarin in 18 201 patients with AF and \\u2265 1 additional risk factor for stroke.Compared with warfarin, apixaban (in the ARISTOTLE trial) and edoxaban (in the ENGAGE-AF trial) were shown to be superior or non-inferior, respectively, for reduction in stroke or SE risk in patients with AF.Efficacy and safety of apixaban compared with warfarin according to age for stroke prevention in atrial fibrillation: observations from the ARISTOTLE trial.Apixaban compared with warfarin in patients with atrial fibrillation and previous stroke or transient ischaemic attack: a subgroup analysis of the ARISTOTLE trial.In the ARISTOTLE trial, the rate of stroke or systemic embolism was reduced by apixaban compared with warfarin in patients with atrial fibrillation (AF).\"}, {\"question\": \"List 3 indications for Bupropion\", \"ideal_answer\": [\"Bupropion is used to treat Obesity, for smoking cessation and for depression\"], \"exact_answer\": [[\"weight loss in obesity\"], [\"smoking cessation\"], [\"depression\"]], \"context\": \" For genotype, genes associated with nicotinic acetylcholine receptors (nAChRs) and related proteins have been found to predict response to first-line medications (e.g. nicotine replacement therapy [NRT], bupropion, or varenicline) or quitting over time without a controlled treatment trial. The approval of drug combinations, such as phentermine/topiramate and bupropion/naltrexone are also noteworthy, the components of which have been previously approved, but not necessarily for obesity as main indication.Two drugs, phentermine plus topiramate and lorcaserin, have recently been approved by the US Food and Drug Administration for the indication of obesity; a third, bupropion plus naltrexone, is under consideration for approval. Bupropion SR is preferred to subjects with depression or smokers who have failed with the previous two agents, due to the many contra-indications and side effects of bupropion SR. With one of the 3 agents combined with follow-up visits with counselling, one can expect a 1-year quit rate around 20-25%.In vitro intrinsic clearances were likewise different for bupropion enantiomers.CONCLUSIONS: Stereoselective bupropion hydroxylation may have implications for the therapeutic efficacy of bupropion as an antidepressant or smoking cessation therapy, and for the use of bupropion as an in vivo phenotypic probe for CYP2B6 activity.PURPOSE: Hydroxylation of the antidepressant and smoking deterrent drug bupropion is a clinically important bioactivation and elimination pathway.Although not U.S. Food and Drug Administration approved for these indications, bupropion has also been used as an adjunctive treatment to reverse antidepressant-induced sexual dysfunction and to augment anti-depressant efficacy in partial responders and non-responders to other agents.Bupropion has played and will continue to play an important role as a treatment for major depressive disorder in adults, as well as for other related disordersClinically, bupropion is used as a treatment for two indications, as an antidepressant, the indication for which it was developed, and as a tobacco use cessation agentSustained release bupropion (amfebutamone) is a non-nicotine agent that is indicated as an aid to smoking cessation. Bupropion was introduced for smoking cessation following a pivotal trial showing that it gave improved efficacy over the nicotine patch and also suggesting combination treatment was beneficial. Compared with a placebo control, bupropion approximately doubles smoking quit rates. Over the past decade, bupropion has become a major pharmacotherapy for smoking cessation in the Western world. Unlike other smoking cessation pharmacotherapies, bupropion is a non-nicotine treatmentBupropion has been tested in over 40 controlled clinical trials and has been associated with higher rates of treatment discontinuation due to adverse events than NRTs. Bupropion SR is a useful oral and non-nicotine form of pharmacotherapy for smoking cessation.everal new drugs for the treatment of obesity are either licensed in some parts of the world, submitted for registration, or completing phase III trials. These include combinations (at low dose) of existing drugs, e.g., bupropion + naltrexone (Contrave),\"}, {\"question\": \"Which are the additions of the JASPAR 2016 open-access database of transcription factor binding profiles?\", \"ideal_answer\": [\"Compared to the JASPAR CORE collection, JASPAR 2016 has been expanded with 494 new TF binding profiles (315 in vertebrates, 11 in nematodes, 3 in insects, 1 in fungi and 164 in plants) and 59 profiles (58 in vertebrates and 1 in fungi) have been updated. The introduced profiles represent an 83% expansion and 10% update when compared to the previous release. The structural annotation of the TF DNA binding domains (DBDs) has been updated following a published hierarchical structural classification. In addition, 130 transcription factor flexible models trained on ChIP-seq data for vertebrates, which capture dinucleotide dependencies within TF binding sites were introduced . The new JASPAR release is accompanied by a new web tool to infer JASPAR TF binding profiles recognized by a given TF protein sequence. Moreover, users are provided with a Ruby module complementing the JASPAR API to ease programmatic access and use of the JASPAR collection of profiles. JASPAR2016 R/Bioconductor data package is also provided with the data of this release.\"], \"exact_answer\": [[\"494 new TF binding profiles (315 in vertebrates, 11 in nematodes, 3 in insects, 1 in fungi and 164 in plants) and updated 59 profiles (58 in vertebrates and 1 in fungi)\"], [\"Updated structural annotation of the TF DNA binding domains (DBDs) following a published hierarchical structural classification\"], [\"130 transcription factor flexible models trained on ChIP-seq data for vertebrates, which capture dinucleotide dependencies within TF binding sites\"], [\"A new web tool to infer JASPAR TF binding profiles recognized by a given TF protein sequence\"], [\"A Ruby module complementing the JASPAR API to ease programmatic access and use of the JASPAR collection of profiles\"], [\"JASPAR2016 R/Bioconductor data package\"]], \"context\": \"For this 2016 release, we expanded the JASPAR CORE collection with 494 new TF binding profiles (315 in vertebrates, 11 in nematodes, 3 in insects, 1 in fungi and 164 in plants) and updated 59 profiles (58 in vertebrates and 1 in fungi). The introduced profiles represent an 83% expansion and 10% update when compared to the previous release. We updated the structural annotation of the TF DNA binding domains (DBDs) following a published hierarchical structural classification. In addition, we introduced 130 transcription factor flexible models trained on ChIP-seq data for vertebrates, which capture dinucleotide dependencies within TF binding sites. This new JASPAR release is accompanied by a new web tool to infer JASPAR TF binding profiles recognized by a given TF protein sequence. Moreover, we provide the users with a Ruby module complementing the JASPAR API to ease programmatic access and use of the JASPAR collection of profiles. Finally, we provide the JASPAR2016 R/Bioconductor data package with the data of this release\"}, {\"question\": \"Which driver mutations have been identified for Diffuse Intrinsic Pontine Glioma (DIPG)?\", \"ideal_answer\": [\"We found conservation of heterozygous K27M mutations in H3F3A (n\\u2009=\\u20094) or HIST1H3B (n\\u2009=\\u20093) across all primary, contiguous, and metastatic tumor sites in all DIPGs\", \"found conservation of heterozygous k27m mutations in h3f3a (n\\u2009=\\u20094) or hist1h3b (n\\u2009=\\u20093) across all primary , contiguous , and metastatic tumor sites in all dipgs . h3k27m ubiquitously-associated mutations involve alterations in tp53 cell-cycle (tp53/ppm1d) or specific growth factor pathways (acvr1/pik3r1) . reconstruction indicates histone 3 (h3) k27m--including h3.2k27m--mutations potentially arise first and are invariably associated with specific , high-fidelity obligate partners throughout the tumour and its spread , from diagnosis to end-stage disease , suggesting mutual need for tumorigenesis. . aberrations (n\\u2009=\\u20093 patients) varied by type and location between primary and metastatic tumors sites but were intra-tumorally conserved. . \", \"We found conservation of heterozygous K27M mutations in H3F3A (n\\u00e2\\u0080\\u0089=\\u00e2\\u0080\\u00894) or HIST1H3B (n\\u00e2\\u0080\\u0089=\\u00e2\\u0080\\u00893) across all primary, contiguous, and metastatic tumor sites in all DIPGs. Evolutionary reconstruction indicates histone 3 (H3) K27M--including H3.2K27M--mutations potentially arise first and are invariably associated with specific, high-fidelity obligate partners throughout the tumour and its spread, from diagnosis to end-stage disease, suggesting mutual need for tumorigenesis. These H3K27M ubiquitously-associated mutations involve alterations in TP53 cell-cycle (TP53/PPM1D) or specific growth factor pathways (ACVR1/PIK3R1).\", \"We found conservation of heterozygous K27M mutations in H3F3A (n=4) or HIST1H3B (n=3) across all primary, contiguous, and metastatic tumor sites in all DIPGs. ACVR1 (n=2), PIK3CA (n=2), FGFR1 (n=2), and MET (n=1) were also intra-tumorally conserved. TP53 aberrations (n=3 patients) varied by type and location between primary and metastatic tumors sites but were intra-tumorally conserved. Evolutionary reconstruction indicates histone 3 (H3) K27M--including H3.2K27M--mutations potentially arise first and are invariably associated with specific, high-fidelity obligate partners throughout the tumour and its spread, from diagnosis to end-stage disease, suggesting mutual need for tumorigenesis. These H3K27M ubiquitously-associated mutations involve alterations in TP53 cell-cycle (TP53/PPM1D) or specific growth factor pathways (ACVR1/PIK3R1). \", \"Recurrent mutations within the histone H3 genes H3F3A and HIST1H3B that convert K27 to methionine (H3K27M) and disrupt the global H3K27 methylation landscape and PRC2-dependent silencing, have recently been identified in pediatric high-grade gliomas including Diffuse Intrinsic Pontine Glioma (DIPG) and Glioblastoma multiforme (GBM; Type IV glioma). The spatial and temporal homogeneity of main driver mutations in DIPG implies they will be captured by limited biopsies and emphasizes the need to develop therapies specifically targeting obligate oncohistone partnerships. TP53 aberrations (n\\u2009=\\u20093 patients) varied by type and location between primary and metastatic tumors sites but were intra-tumorally conserved.Spatial conservation of prognostically-relevant and therapeutically-targetable somatic mutations in DIPG and mHGG contrasts the significant heterogeneity of driver mutations seen in adult HGG and supports uniform implementation of diagnostic biopsy in DIPG and mHGG to classify molecular risk groups and guide therapeutic strategy. Recently, a subset of these same mutations of ACVR1 have been identified in diffuse intrinsic pontine glioma (DIPG) tumors. Sequencing analysis showed c.83A>T mutations in the H3F3A or HIST1H3B gene in 77\\u00a0% of our DIPG cohort. Protein profiling identified 2,305 unique proteins indicating distinct DIPG protein expression patterns compared to other pediatric brain tumors. Two distinct subgroups of DIPG were identified.\", \"We found conservation of heterozygous K27M mutations in H3F3A (n\\u2009=\\u20094) or HIST1H3B (n\\u2009=\\u20093) across all primary, contiguous, and metastatic tumor sites in all DIPGs ACVR1 (n\\u2009=\\u20092), PIK3CA (n\\u2009=\\u20092), FGFR1 (n\\u2009=\\u20092), and MET (n\\u2009=\\u20091) were also intra-tumorally conserved  TP53 aberrations (n\\u2009=\\u20093 patients) varied by type and location between primary and metastatic tumors sites but were intra-tumorally conserved.\", \"Evolutionary reconstruction indicates histone 3 (H3) K27M--including H3.2K27M--mutations potentially arise first and are invariably associated with specific, high-fidelity obligate partners throughout the tumour and its spread, from diagnosis to end-stage disease, suggesting mutual need for tumorigenesis. Conservation of heterozygous K27M mutations in H3F3A (n\\u2009=\\u20094) or HIST1H3B (n\\u2009=\\u20093) was observed across all primary, contiguous, and metastatic tumor sites in all DIPGs. ACVR1 (n\\u2009=\\u20092), PIK3CA (n\\u2009=\\u20092), FGFR1 (n\\u2009=\\u20092), and MET (n\\u2009=\\u20091) were also intra-tumorally conserved.  TP53 aberrations (n\\u2009=\\u20093 patients) varied by type and location between primary and metastatic tumors sites but were intra-tumorally conserved.\", \"We found conservation of heterozygous K27M mutations in H3F3A (n\\u2009=\\u20094) or HIST1H3B (n\\u2009=\\u20093) across all primary, contiguous, and metastatic tumor sites in all DIPGs  TP53 aberrations (n\\u2009=\\u20093 patients) varied by type and location between primary and metastatic tumors sites but were intra-tumorally conserved. These H3K27M ubiquitously-associated mutations involve alterations in TP53 cell-cycle (TP53/PPM1D) or specific growth factor pathways (ACVR1/PIK3R1).\", \"We found conservation of heterozygous K27M mutations in H3F3A (n=4) or HIST1H3B (n=3) across all primary, contiguous, and metastatic tumor sites in all DIPGs. Evolutionary reconstruction indicates histone 3 (H3) K27M--including H3.2K27M--mutations potentially arise first and are invariably associated with specific, high-fidelity obligate partners throughout the tumour and its spread, from diagnosis to end-stage disease, suggesting mutual need for tumorigenesis. These H3K27M ubiquitously-associated mutations involve alterations in TP53 cell-cycle (TP53/PPM1D) or specific growth factor pathways (ACVR1/PIK3R1). TP53 aberrations (n=3 patients) varied by type and location between primary and metastatic tumors sites but were intra-tumorally conserved. ACVR1 (n=2), PIK3CA (n=2), FGFR1 (n=2), and MET (n=1) were also intra-tumorally conserved. \", \"We found conservation of heterozygous K27M mutations in H3F3A (n\\u2009=\\u20094) or HIST1H3B (n\\u2009=\\u20093) across all primary, contiguous, and metastatic tumor sites in all DIPGs These H3K27M ubiquitously-associated mutations involve alterations in TP53 cell-cycle (TP53/PPM1D) or specific growth factor pathways (ACVR1/PIK3R1). \"], \"exact_answer\": [[\"K27M in H3F3A\"], [\"HIST1H3B\"]], \"context\": \"We found conservation of heterozygous K27M mutations in H3F3A (n\\u2009=\\u20094) or HIST1H3B (n\\u2009=\\u20093) across all primary, contiguous, and metastatic tumor sites in all DIPGsACVR1 (n\\u2009=\\u20092), PIK3CA (n\\u2009=\\u20092), FGFR1 (n\\u2009=\\u20092), and MET (n\\u2009=\\u20091) were also intra-tumorally conserved TP53 aberrations (n\\u2009=\\u20093 patients) varied by type and location between primary and metastatic tumors sites but were intra-tumorally conserved.Evolutionary reconstruction indicates histone 3 (H3) K27M--including H3.2K27M--mutations potentially arise first and are invariably associated with specific, high-fidelity obligate partners throughout the tumour and its spread, from diagnosis to end-stage disease, suggesting mutual need for tumorigenesis.These H3K27M ubiquitously-associated mutations involve alterations in TP53 cell-cycle (TP53/PPM1D) or specific growth factor pathways (ACVR1/PIK3R1). Recently, sequencing of tumor cells revealed that histone H3 is frequently mutated in pediatric HGG, with up to 78\\u00a0% of diffuse intrinsic pontine gliomas (DIPGs) carrying K27M and 36\\u00a0% of non-brainstem gliomas carrying either K27M or G34R/V mutations.Moreover, in diffuse intrinsic pontine gliomas (DIPG), recurrent somatic mutations of ACVR1 were found in association with HIST1H3B mutationsRecently, sequencing of tumor cells revealed that histone H3 is frequently mutated in pediatric HGG, with up to 78\\u00a0% of diffuse intrinsic pontine gliomas (DIPGs) carrying K27M and 36\\u00a0% of non-brainstem gliomas carrying either K27M or G34R/V mutationsK27M mutation in histone H3.3 defines clinically and biologically distinct subgroups of pediatric diffuse intrinsic pontine gliomas.Recurrent mutations within the histone H3 genes H3F3A and HIST1H3B that convert K27 to methionine (H3K27M) and disrupt the global H3K27 methylation landscape and PRC2-dependent silencing, have recently been identified in pediatric high-grade gliomas including Diffuse Intrinsic Pontine Glioma (DIPG) and Glioblastoma multiforme (GBM; Type IV glioma).Sequencing analysis showed c.83A>T mutations in the H3F3A or HIST1H3B gene in 77\\u00a0% of our DIPG cohort.Recently, sequencing of tumor cells revealed that histone H3 is frequently mutated in pediatric HGG, with up to 78\\u00a0% of diffuse intrinsic pontine gliomas (DIPGs) carrying K27M and 36\\u00a0% of non-brainstem gliomas carrying either K27M or G34R/V mutations.K27M mutation in histone H3.3 defines clinically and biologically distinct subgroups of pediatric diffuse intrinsic pontine gliomas.\"}, {\"question\": \"List genes associated with hypolipidemia.\", \"ideal_answer\": [\"PCSK9\\nAPOB \\nANGPTL3\\nANGPTL4\\nMTP\"], \"exact_answer\": [[\"PCSK9\"], [\"APOB\"], [\"ANGPTL3\"], [\"ANGPTL4\"], [\"MTP\"]], \"context\": \" Conversely familial hypobetalipoproteinemia is caused by inactivation of the PCSK9 gene which increases the number of LDL receptors and decreases plasma cholesterol.Mutations in the genes APOB, and ANGPTL3 and ANGPTL4 (that encode angiopoietin-like proteins which inhibit lipoprotein lipase activity) can further cause low levels of apoB containing lipoproteinsSeveral mutations in the apoB, proprotein convertase subtilisin/kexin type 9 (PCSK9), and MTP genes result in low or absent levels of apoB and LDL-cholesterol in plasma, which cause familial hypobetalipoproteinemia and abetalipoproteinemia. Mutations in the ANGPTL3 gene cause familial combined hypolipidemia. Mutations of the ANGPTL3 gene have been associated with a novel form of primary hypobetalipoproteinemia, the combined hypolipidemia (cHLP), characterized by low total cholesterol and low HDL-cholesterol levels. These results demonstrated that in a cohort of subjects with severe primary hypobetalipoproteinemia the prevalence of ANGPTL3 gene mutations responsible for a combined hypolipidemia phenotype is about 10%, whereas mutations of APOB gene are absent.Angptl3 knockout mice have marked hypolipidemia, and heterozygous carriers of ANGPLT3, loss-of-function mutations were found among individuals in the lowest quartile of plasma triglycerides in population studies.Complete ANGPTL3 deficiency caused by loss-of-function mutations of ANGPTL3 is associated with a recessive hypolipidemia characterized by a reduction of apolipoprotein B and apolipoprotein A-I-containing lipoproteins, changes in subclasses of high-density lipoprotein, and reduced cholesterol efflux potential of serum.\"}, {\"question\": \"Which are the symptoms of glucose-6-phosphate dehydrogenase (G6PD) deficiency?\", \"ideal_answer\": [\"Glucose-6-phosphate dehydrogenase deficiency (G6PD deficiency) is the most common red blood cell (RBC) enzyme disorder. The decrease as well as the absence of the enzyme increase RBC vulnerability to oxidative stress caused by exposure to certain medications or intake of fava beans. Among the most common symptoms of this condition are:\\n1) acute hemolysis, \\n2) chronic hemolysis, \\n3) neonatal hyperbilirubinemia.\"], \"exact_answer\": [[\"acute hemolysis\"], [\"chronic hemolysis\"], [\"neonatal hyperbilirubinemia\"]], \"context\": \"Glucose-6-phosphate dehydrogenase deficiency (G6PD deficiency) is the most common red blood cell (RBC) enzyme disorder. The decrease as well as the absence of the enzyme increase RBC vulnerability to oxidative stress caused by exposure to certain medications or intake of fava beans. Among the most common clinical manifestations of this condition, acute hemolysis, chronic hemolysis, neonatal hyperbilirubinemia, and an asymptomatic form are observed.Red blood cells carry oxygen in the body and Glucose-6-Phosphate Dehydrogenase protects these cells from oxidative chemicals. If there is a lack of Glucose-6-Phosphate Dehydrogenase, red blood cells can go acute hemolysis. Convulsion is a rare presentation for acute hemolysis due to Glucose-6-Phosphate Dehydrogenase deficiency. By far the most common form worldwide is the Glucose-6-phosphate deficiency. In the most frequent variants of this disease hemolysis occurs only during stress, imposed for example by infection, \\\"oxidative\\\" drugs or after ingestion of fava beans. The most serious clinical complication of the Glucose-6-phosphate deficiency is the rarely observed neonatal icterus. Some enzyme variants can cause chronic hemolysis which is described as chronic nonsperocytic hemolytic anemia. One limitation of primaquine is that, at certain doses, primaquine can cause hemolytic anemia in individuals with glucose-6-phosphate dehydrogenase (G6PD) deficiency (G6PDd)Glucose-6-phosphate dehydrogenase (G6PD) deficiency is a common human enzymopathy that affects cellular redox status and may lower flux into nonoxidative pathways of glucose metabolismThe 6PGD deficiency was associated with a variable reticulocyte count and recurrent increased unconjugated bilirubinemia without anemia in the propositus, while no clinical or hematological symptoms were evident in her mother.The authors of this article make 5 particular recommendations: (1) Anyone suspected of G6PD deficiency should be screened; (2) exposure to oxidative stressors in these individuals should be avoided; (3) these patients should be informed of risks along with signs and symptoms of an acute hemolytic crisis; (4) the clinician should be able to identify both laboratory and clinical signs of hemolysis; and finally, (5) if an acute hemolytic crisis is identified, the patient should be admitted for close observation and care.None of the patients with or without G6PD deficiency showed symptoms, signs, or laboratory findings indicating hemolysis before administration of the drug and 4 days thereafter.Kinetic and electrophoretic properties of 230--300 fold purified preparations of glucose-6-phosphate dehydrogenase (G6PD) from red cells of donors and patients with acute drug hemolytic anemia due to G6PD deficiency were studied.Glucose-6-phosphate dehydrogenase (G6PD) deficiency is an X-linked recessive disorder in which haemolytic anaemia is the major symptom.Glucose-6-phosphate dehydrogenase (G6PD) deficiency is an X-linked recessive genetic defect that can cause hemolytic crisis.However, its use is contraindicated in patients with glucose-6-phosphate dehydrogenase (G6PD) deficiency (risk for haemolytic anaemia).The role of glucose-6-phosphate dehydrogenase (G6PD) deficiency in the development of stroke in children with SCA is controversial.Glucose-6-phosphate dehydrogenase (G6PD) deficiency is an X-linked recessive hemolytic anemia caused by a mutation in the G6PD gene on Xq28.Antiplatelet and invasive treatment in patients with glucose-6-phosphate dehydrogenase (G6PD) deficiency and acute coronary syndrome. The safety of aspirin.\"}, {\"question\": \"Which are the clinical symptoms of left ventricular noncompaction?\", \"ideal_answer\": [\"The clinical symptoms of left ventricular noncompaction are:\\n1) heart failure, \\n2) systemic thromboembolic events, \\n3) ventricular arrhythmias and\\n4) sudden cardiac death.\"], \"exact_answer\": [[\"heart failure\"], [\"systemic thromboembolic events\", \"systemic thromboembolism\"], [\"ventricular arrhythmias\"], [\"sudden cardiac death\"]], \"context\": \"Perioperative management of the patient with LVNC might be challenging due to the clinical symptoms of heart failure, systemic thromboembolic events, and fatal left ventricular arrhythmias.Left ventricular noncompaction cardiomyopathy is a rare type of congenital cardiomyopathy characterized by prematurely arrested compaction of the endocardial and myocardial fibers and the progressive deterioration of left ventricular contractility. Clinical features associated with LVNC vary in asymptomatic and symptomatic patients, and include the potential for heart failure, conduction defects (eg, left bundle branch block), supraventricular and ventricular arrhythmias, thromboembolic events, and sudden cardiac death. The clinical presentations are variable ranging from asymptomatic patients to patients who develop ventricular arrhythmias, thromboembolism, heart failure, and sudden cardiac death. The clinical presentation and the natural history of LVNC are highly variable, ranging from no symptoms to congestive heart failure, arrhythmias, and systemic thromboemboli. The clinical manifestations of the disease are variable, ranging from no symptoms to signs of heart failure, systemic emboli, and ventricular arrhythmias. We describe the case of a 62-year-old woman who presented with symptoms of heart failure secondary to left ventricular noncompaction.Multiple left ventricular thrombi in a patient with left ventricular noncompaction.We present three children with myocardial noncompaction: one male with isolated left ventricular noncompaction, another with right ventricular noncompaction and dysplastic tricuspid valve, and the last with left ventricular noncompaction, ventricular septal defect and coarctation of aorta, to stress especially the different clinical forms of the disorder and the importance of early diagnosis, as it may result in a fatal outcome.Left ventricular noncompaction has a heterogeneous clinical presentation that includes arrhythmia and sudden cardiac death.We retrospectively reviewed all children diagnosed with left ventricular noncompaction at Texas Childrens Hospital from January 1990 to January 2009Left ventricular noncompaction is a cardiomyopathy characterized by excessive trabeculation of the left ventricle, progressive myocardial dysfunction, and early mortalityWe describe the case of a 62-year-old woman who presented with symptoms of heart failure secondary to left ventricular noncompaction. Left ventricular noncompaction (LVNC) and dilated CMP are the most common cardiac phenotypes reported and can lead to symptoms of heart failure as well as ventricular arrhythmias.Multiple left ventricular thrombi in a patient with left ventricular noncompaction.Left ventricular noncompaction has a heterogeneous clinical presentation that includes arrhythmia and sudden cardiac death.Clinical symptoms include signs of left ventricular systolic dysfunction even to the point of heart failure, ventricular arrhythmias, and embolic events.\"}, {\"question\": \"Which mutated genes are associated with isolated ectopia lentis?\", \"ideal_answer\": [\"Isolated ectopia lentis (EL) is caused by mutation in genes:\\n1) ADAMTSL4 and \\n2) Fibrillin-1 (FBN1).\"], \"exact_answer\": [[\"ADAMTSL4\"], [\"Fibrillin-1\", \"FBN1\"]], \"context\": \"A founder mutation in ADAMTSL4 causes early-onset bilateral ectopia lentis among Jews of Bukharian origin.The term isolated ectopia lentis (EL; subluxation or dislocation of the human crystalline lens) is applied to patients with EL, without skeletal features and in the absence of aortic root dilatation. To date, the only gene shown to cause autosomal-recessive isolated EL is ADAMTSL4. Here we report a novel founder mutation in ADAMTSL4 gene in children of Bukharian Jewish origin presenting with early-onset bilateral EL. Early onset ectopia lentis due to a FBN1 mutation with non-penetrance.Isolated ectopia lentis is usually autosomal dominant and commonly due to the mutations of FBN1 gene. In conclusion, we report on a case of early-onset autosomal dominant isolated ectopia lentis caused by FBN1 mutation that has previously been reported only in Marfan syndrome.ADAMTSL4-associated isolated ectopia lentis: Further patients, novel mutations and a detailed phenotype description.ADAMTSL4 mutations seem to be the most common cause of isolated ectoplia lentis (EL) and thus are important concerning the differential diagnosis of connective tissue syndromes with EL as main feature. A genotype-phenotype comparison of ADAMTSL4 and FBN1 in isolated ectopia lentis.To describe the genotype-phenotype relationship of a cohort of consecutive patients with isolated ectopia lentis (EL) secondary to ADAMTSL4 and FBN1 mutations.ADAMTSL4 is the most important known causative gene in isolated EL. Mutations in ADAMTSL4 appear to cause earlier manifestation of EL and are associated with increased axial length as compared to FBN1. We suggest that ADAMTSL4 be screened in all patients with isolated EL and that physicians be vigilant for the more severe ocular phenotype associated with mutations in this gene.A novel FBN1 mutation in a Chinese family with isolated ectopia lentis.To identify the genetic defect in an autosomal dominant isolated ectopia lentis (EL) family.A novel mutation of FBN1 results in an arginine to cysteine residue (p.R974C) substitution, which is responsible for the patients with isolated EL in this Chinese family.A homozygous mutation in ADAMTSL4 causes autosomal-recessive isolated ectopia lentis.Isolated ectopia lentis is usually autosomal dominant and commonly due to the mutations of FBN1 gene.To examine the fibrillin-1 (FBN1) gene for mutations in members of a Chinese family with isolated ectopia lentis.Ophthalmic findings in a family with early-onset isolated ectopia lentis and the p.Arg62Cys mutation of the fibrillin-1 gene (FBN1).Craniosynostosis with ectopia lentis and a homozygous 20-base deletion in ADAMTSL4.ADAMTSL4-associated isolated ectopia lentis: Further patients, novel mutations and a detailed phenotype descriptionOur study presented detailed clinical manifestations, including some novel ophthalmic findings, such as pupillary abnormality, different types of glaucoma, and progressive hyperopia.Ophthalmic findings and the p.Arg62Cys mutation of FBN1 gene were reported in a family with early-onset isolated ectopia lentisTo examine the fibrillin-1 (FBN1) gene for mutations in members of a Chinese family with isolated ectopia lentis.Clinically relevant laboratory investigation.Family members underwent clinical examinationsOphthalmic findings in a family with early-onset isolated ectopia lentis and the p.Arg62Cys mutation of the fibrillin-1 gene (FBN1)The 65 exons of FBN1 were amplified by polymerase chain reaction and screened for mutations by a combination of denaturing high-performance liquid chromatography analysis and direct DNA sequencing.A mutation, c.184C-->T in exon 2 of FBN1, which results in substitution of arginine by cysteine at position 62 of the fibrillin-1 protein (p.R62C) in all affected family members but in none of the unaffected individuals.A recurrent mutation of FBN1 gene resulted in an arginine-to-cysteine residue (p.R62C), is responsible for the patients with isolated ectopia lentis in a Chinese familyIsolated ectopia lentis is usually autosomal dominant and commonly due to the mutations of FBN1 geneCONCLUSIONS: The results emphasize the association of ADAMTSL4 null mutations with isolated ectopia lentis and the presence of a founder mutation in the European population. Herein we show a consanguineous family that carries a novel homozygous splice mutation IVS4-1G>A/IVS4-1G>A in ADAMTSL4 responsible for isolated autosomal recessive EL, thus confirming the involvement of this gene in this condition and underlining the major role of ADAMTS proteases in zonular fibers homeostasis.Two novel FBN1 mutations associated with ectopia lentis and marfanoid habitus in two Chinese families.Novel FBN1 mutations associated with predominant ectopia lentis and marfanoid habitus in Chinese patients.FBN1 mutations are associated with multiple clinical phenotypes, including Marfan syndrome (MFS), MASS phenotype, and familial ectopia lentis, but rarely with isolated aortic aneurysm and dissection.This was correlated with Sanger sequencing of ADAMTSL4 and FBN1 genes.Seventeen patients were examined, including one with ectopia lentis et pupillae.Recently, ADAMTSL4 has been shown to be responsible for isolated autosomal recessive ectopia lentis in an inbred family.Confirmation of ADAMTSL4 mutations for autosomal recessive isolated bilateral ectopia lentis.A recurrent FBN1 mutation in an autosomal dominant ectopia lentis family of Indian origin.We suggest that ADAMTSL4 be screened in all patients with isolated EL and that physicians be vigilant for the more severe ocular phenotype associated with mutations in this gene..A homozygous mutation in ADAMTSL4 causes autosomal-recessive isolated ectopia lentis.Recurrent FBN1 mutation (R62C) in a Chinese family with isolated ectopia lentis.A genotype-phenotype comparison of ADAMTSL4 and FBN1 in isolated ectopia lentis.Ophthalmic findings and the p.Arg62Cys mutation of FBN1 gene were reported in a family with early-onset isolated ectopia lentis..Ophthalmic findings in a family with early-onset isolated ectopia lentis and the p.Arg62Cys mutation of the fibrillin-1 gene (FBN1).A recurrent mutation of FBN1 gene resulted in an arginine-to-cysteine residue (p.R62C), is responsible for the patients with isolated ectopia lentis in a Chinese family..\"}, {\"question\": \"Signaling of which pathways is inhibited by Dupilumab?\", \"ideal_answer\": [\"Dupilumab, a fully human anti-interleukin-4 receptor \\u03b1 monoclonal antibody, inhibits interleukin-4 and interleukin-13 signalling. It is used for treatment of atopic or allergic diseases.\"], \"exact_answer\": [[\"interleukin-4\"], [\"interleukin-13\"]], \"context\": \"This study reviewed all studies about any roles of IL-4 that can directly and indirectly be played in the development of pemphigus and IL-4 inhibition with interferons and dupilumab therapy were introduced as a novel pemphigus treatment for patients who are in relapse phase of the disease. Dupilumab was also introduced as a possible treatment for patients with severe pemphigus. It can directly inhibit IL-4 by targeting IL-4 \\u03b1-chain receptor. OBJECTIVE: To assess inhibition of interleukins 4 and 13 with dupilumab in patients with chronic sinusitis and nasal polyposis.BACKGROUND: Dupilumab, a fully human anti-interleukin-4 receptor \\u03b1 monoclonal antibody, inhibits interleukin-4 and interleukin-13 signalling, key drivers of type-2-mediated inflammation. Anti-IL-4 and IL-13 agents (dupilumab, lebrikizumab, and tralokinumab) which block different Th-2 inflammatory pathways and agents targeting the Th-17 inflammatory pathway in severe refractory asthma are under development.Best evidence of the clinical efficacy of novel immunologic approaches using biological agents in patients with AD is available for the anti-IL-4 receptor \\u03b1-chain antibody dupilumab, but a number of studies are currently ongoing with other specific antagonists to immune system players.Background Dupilumab, a human monoclonal antibody against interleukin-4 receptor alpha, inhibits signaling of interleukin-4 and interleukin-13, type 2 cytokines that may be important drivers of atopic or allergic diseases such as atopic dermatitis. The world's first prospective controlled studies with the biologic human anti-IL4R antibody dupilumab for the indication \\\"atopic dermatitis\\\" were published in 2014.An IL-4R antagonist, dupilumab, is the first drug that shows great promise in phase II trials. Dupilumab, a humanized monoclonal antibody to the interteukin-4R is the first antibody (i.e. 'biological') with published efficacy shown in controlled prospective studies in atopic dermatitis. Dupilumab is a fully human monoclonal antibody directed against the IL-4 receptor \\u03b1 subunit that blocks the signaling of IL-4 and IL-13, both key cytokines in Th2-mediated pathways.Dupilumab inhibits interleukin-4 (IL-4) and interleukin-13 (IL-13) signaling and was previously found to be effective in asthma.Data from early-stage studies suggested that interleukin (IL)-4 and IL-13 are requisite drivers of atopic dermatitis, evidenced by marked improvement after treatment with dupilumab, a fully-human monoclonal antibody that blocks both pathways.Background Dupilumab, a human monoclonal antibody against interleukin-4 receptor alpha, inhibits signaling of interleukin-4 and interleukin-13, type 2 cytokines that may be important drivers of atopic or allergic diseases such as atopic dermatitis.The robust effects of dupilumab on skin inflammation and pruritus confirm the pathogenic role of IL-4 and IL-13 signaling in adult AD, and further support the application of Th2 cytokine antagonists in the treatment of this disease.Dupilumab, a fully human anti-interleukin-4 receptor \\u03b1 monoclonal antibody, inhibits interleukin-4 and interleukin-13 signalling, key drivers of type-2-mediated inflammation.Dupilumab is a fully human monoclonal antibody directed against the IL-4 receptor \\u03b1 subunit that blocks the signaling of IL-4 and IL-13, both key cytokines in Th2-mediated pathwaysIn early-phase trials, dupilumab, a fully human mAb targeting IL-4 receptor \\ufffd, markedly improved disease activity, but the effect of IL-4/IL-13 blockade on AD at the molecular level has not been characterized.OBJECTIVES: We sought to evaluate dupilumab modulation of the AD molecular signature.METHODS: We performed transcriptomic analyses of pretreatment and posttreatment skin biopsy specimens from patients with moderate-to-severe AD treated weekly with 150 or 300 mg of dupilumab or placebo.RESULTS: Exacerbation of the AD transcriptome was observed in placebo-treated patients. Dupilumab is a fully human monoclonal antibody directed against the IL-4 receptor \\u03b1 subunit that blocks the signaling of IL-4 and IL-13, both key cytokines in Th2-mediated pathways.Dupilumab inhibits interleukin-4 (IL-4) and interleukin-13 (IL-13) signaling and was previously found to be effective in asthma.The robust effects of dupilumab on skin inflammation and pruritus confirm the pathogenic role of IL-4 and IL-13 signaling in adult AD, and further support the application of Th2 cytokine antagonists in the treatment of this disease.In early-phase trials, dupilumab, a fully human mAb targeting IL-4 receptor \\u00e1, markedly improved disease activity, but the effect of IL-4/IL-13 blockade on AD at the molecular level has not been characterized.OBJECTIVES: We sought to evaluate dupilumab modulation of the AD molecular signature.METHODS: We performed transcriptomic analyses of pretreatment and posttreatment skin biopsy specimens from patients with moderate-to-severe AD treated weekly with 150 or 300 mg of dupilumab or placebo.RESULTS: Exacerbation of the AD transcriptome was observed in placebo-treated patients.\"}, {\"question\": \"List the three main structures of the cytoskeleton.\", \"ideal_answer\": [\"Fibrillar polymers-actin filaments, microtubules, and intermediate filaments-are major constituents of the cytoskeleton.\"], \"exact_answer\": [[\"actin filaments\"], [\"microtubules\"], [\"intermediate filaments\"]], \"context\": \"Fibrillar polymers-actin filaments, microtubules, and intermediate filaments-are major constituents of the cytoskeleton,\"}, {\"question\": \"Which treatment methods were compared in the EXCEL Trial?\", \"ideal_answer\": [\"EXCEL trial compared Everolimus Eluting Stent vs. Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization.\"], \"exact_answer\": [[\"Everolimus Eluting Stent\"], [\"Coronary Artery Bypass Surgery\"]], \"context\": \"Long-term forecasting and comparison of mortality in the Evaluation of the Xience Everolimus Eluting Stent vs. Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization (EXCEL) trial: prospective validation of the SYNTAX Score II.AIMS: To prospectively validate the SYNTAX Score II and forecast the outcomes of the randomized Evaluation of the Xience Everolimus-Eluting Stent Versus Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization (EXCEL) Trial.Based on 4-year mortality predictions in EXCEL, clinical characteristics shifted long-term mortality predictions either in favour of PCI (older age, male gender and COPD) or CABG (younger age, lower creatinine clearance, female gender, reduced left ventricular ejection fraction).Given that the completed randomized trials did not include contemporary DESs, the upcoming results of the ongoing trials evaluating the performance of new-generation DES compared with CABG (such as the EXCEL trial), may further help to clarify the current role and future recommendations of PCI for left main coronary artery disease. The ongoing EXCEL trial will help elucidate the role of ULMCA PCI in the treatment of left main disease compared with coronary artery bypass graft surgery.The ongoing Evaluation of XIENCE V Everolimus Eluting Stent System Versus Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization (EXCEL) trial is expected to provide a better answer on the optimal treatment strategy for LMCAD patients. The Evaluation of Xience Prime or Xience V versus Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization (EXCEL) trial is a multicenter, ongoing trial conducted in patients with left main disease and SYNTAX score \\u2264 32 to establish the presumptive advantage of percutaneous coronary intervention (PCI) versus bypass surgery in patients with less complex coronary artery disease than those enrolled in the Synergy between PCI with Taxus and Cardiac Surgery (SYNTAX) trial. The EXCEL trial will address the value of PCI in relation to CABG for the treatment of ULMCA stenosis in more than 2000 patients. OBJECTIVES: The aim of this study is to verify the study hypothesis of the EXCEL trial by comparing percutaneous coronary intervention (PCI) and coronary artery bypass graft (CABG) in an EXCEL-like population of patients.BACKGROUND: The upcoming EXCEL trial will test the hypothesis that left main patients with SYNTAX score \\u2264 32 experience similar rates of 3-year death, myocardial infarction (MI), or cerebrovascular accidents (CVA) following revascularization by PCI or CABG.CONCLUSIONS: In an EXCEL-like cohort of patients with left main disease, there seems to be a clinical equipoise between PCI and CABG in terms of death/MI/CVA. The aim of this study is to verify the study hypothesis of the EXCEL trial by comparing percutaneous coronary intervention (PCI) and coronary artery bypass graft (CABG) in an EXCEL-like population of patients.To prospectively validate the SYNTAX Score II and forecast the outcomes of the randomized Evaluation of the Xience Everolimus-Eluting Stent Versus Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization (EXCEL) Trial.Evaluation of the Xience Everolimus Eluting Stent vs. Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization is a prospective, randomized multicenter trial designed to establish the efficacy and safety of percutaneous coronary intervention (PCI) with the everolimus-eluting stent compared with coronary artery bypass graft (CABG) surgery in subjects with unprotected left-main coronary artery (ULMCA) disease and low-intermediate anatomical SYNTAX scores (&lt;33).OBJECTIVES: The aim of this study is to verify the study hypothesis of the EXCEL trial by comparing percutaneous coronary intervention (PCI) and coronary artery bypass graft (CABG) in an EXCEL-like population of patients.BACKGROUND: The upcoming EXCEL trial will test the hypothesis that left main patients with SYNTAX score ?The results of the ongoing EXCEL trial, which compares left main percutaneous coronary intervention with drug-eluting stents and CABG, will provide insight regarding the ideal revascularization strategy for these patients..Given that the completed randomized trials did not include contemporary DESs, the upcoming results of the ongoing trials evaluating the performance of new-generation DES compared with CABG (such as the EXCEL trial), may further help to clarify the current role and future recommendations of PCI for left main coronary artery disease.The introduction of the newer-generation drug-eluting stents (DES) -with documented improvements in both safety and efficacy- has prompted the interventional community to design two new dedicated randomised trials comparing CABG and PCI: the NOBLE (Coronary Artery Bypass Grafting Vs Drug Eluting Stent Percutaneous Coronary Angioplasty in the Treatment of Unprotected Left Main Stenosis) and EXCEL (Evaluation of XIENCE Everolimus Eluting Stent Versus Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization) trials.The EXCEL trial will address the value of PCI in relation to CABG for the treatment of ULMCA stenosis in more than 2000 patients.The ongoing Evaluation of XIENCE V Everolimus Eluting Stent System Versus Coronary Artery Bypass Surgery for Effectiveness of Left Main Revascularization (EXCEL) trial is expected to provide a better answer on the optimal treatment strategy for LMCAD patients.\"}, {\"question\": \"Which proteins does the yeast Cleavage and Polyadenylation Complex contain?\", \"ideal_answer\": [\"The proteins Nrd1, Rap1, Trf4, Rrp6, Ssu72, Cstf64, Pcf11 and PAP are the major components of the 3' cleavage and polyadenylation complex.\"], \"exact_answer\": [[\"Nrd1\"], [\"Rap1\"], [\"Trf4\"], [\"Rrp6\"], [\"Ssu72\"], [\"PAP\"], [\"Cstf64\"], [\"Pcf11\"]], \"context\": \"Gene loops have been described in different organisms from yeast to human and form through interaction between components of the transcription pre-initiation complex and Ssu72, a member of the 3' end cleavage and polyadenylation complexA large cleavage and polyadenylation complex containing the major poly(A) polymerase Pap1 produces mRNA 3' ends, whereas those of nonpolyadenylated snoRNAs in yeast are formed either by endonucleolytic cleavage or by termination, followed by trimming by the nuclear exosome.Poly(A) tails are added by Pap1 to both forms, whereas the alternative poly(A) polymerase Tfr4 adenylates major precursors and processing intermediates to facilitate further polyadenylation by Pap1 and maturation by the exosome/Rrp6A more important role of Trf4/TRAMP, however, is to enhance Nrd1 association with snoRNA genes.Screening crude natural product extracts with this technology has resulted in the identification of a novel family of antifungal natural products, named the parnafungins, which inhibit the enzyme polyadenosine polymerase (PAP), a key component of the mRNA cleavage and polyadenylation complex.We show that phosphorylation of TFIIB at serine 65 regulates the interaction between TFIIB and the CstF-64 component of the CstF 3' cleavage and polyadenylation compleWe now show that Pcf11, a component of the cleavage and polyadenylation complex (CPAC), is also generally required for NRD-dependent transcription termination through the action of its C-terminal domain (CTD)-interacting domain (CID)Like the yeast counterpart, mammalian Ssu72 associates with TFIIB and the yeast cleavage/polyadenylation factor Pta1, and exhibits intrinsic phosphatase activity.We also provide evidence that Pti1p probably acts by uncoupling cleavage and polyadenylation, and functions in coordination with the Nrd1p-dependent pathway for 3' end formation of non-polyadenylated transcripts.Like the yeast counterpart, mammalian Ssu72 associates with TFIIB and the yeast cleavage/polyadenylation factor Pta1, and exhibits intrinsic phosphatase activity.\"}, {\"question\": \"Which deep learning-based algorithms are used for enhancer prediction?\", \"ideal_answer\": [\"EP-DNN and DEEP.\"], \"exact_answer\": [[\"EP-DNN\"], [\"DEEP\"]], \"context\": \"EP-DNN: A Deep Neural Network-Based Global Enhancer Prediction AlgorithmWe present EP-DNN, a protocol for predicting enhancers based on chromatin features, in different cell typesDEEP: a general computational framework for predicting enhancers In this study we developed DEEP, a novel ensemble prediction framework. DEEP integrates three components with diverse characteristics that streamline the analysis of enhancer's properties in a great variety of cellular conditions. In our method we train many individual classification models that we combine to classify DNA regions as enhancers or non-enhancers. DEEP uses features derived from histone modification marks or attributes coming from sequence characteristics. in this paper, we use recent state-of-the-art Deep Learning methods and develop a deep neural network (DNN)-based architecture, called EP-DNN, to predict the presence and types of enhancers in the human genome.In this paper, we developed EP-DNN, which has high accuracy of prediction, with validation rates above 90\\u00a0% for the operational region of enhancer prediction for all four cell lines that we studied, outperforming DEEP-ENCODE and RFECSWe present EP-DNN, a protocol for predicting enhancers based on chromatin features, in different cell types. Specifically, we use a deep neural network (DNN)-based architecture to extract enhancer signatures in a representative human embryonic stem cell type (H1) and a differentiated lung cell type (IMR90)We find that EP-DNN has superior accuracy with a validation rate of 91.6%, relative to 85.3% for DEEP-ENCODE and 85.5% for RFECS, for a given number of enhancer predictions and also scales better for a larger number of enhancer predictions.PEDLA: predicting enhancers with a deep learning-based algorithmic framework.DEEP: a general computational framework for predicting enhancers.PEDLA: predicting enhancers with a deep learning-based algorithmic framework.EP-DNN: A Deep Neural Network-Based Global Enhancer Prediction Algorithm.We developed a deep learning-based algorithmic framework named PEDLA (https://github.com/wenjiegroup/PEDLA), which can directly learn an enhancer predictor from massively heterogeneous data and generalize in ways that are mostly consistent across various cell types/tissues.\"}, {\"question\": \"What are the roles of Smyd3 in zebrafish?\", \"ideal_answer\": [\"Smyd3 is required for the development of cardiac and skeletal muscle in zebrafish. Transcripts of smyd3 are expressed in zebrafish embryos at all developmental stages and knockdown of smyd3 in embryos resulted in pericardial edema and defects in the trunk structure. In addition, these phenotypes are associated with abnormal expression of three heart-chamber markers including cmlc2, amhc and vmhc, and abnormal expression of myogenic regulatory factors including myod and myog.\"], \"exact_answer\": [[\"The development of cardiac muscle\"], [\"The development of skeletal muscle\"]], \"context\": \"Smyd3 is required for the development of cardiac and skeletal muscle in zebrafishWe here show that transcripts of smyd3 were expressed in zebrafish embryos at all developmental stages examined and that knockdown of smyd3 in embryos resulted in pericardial edema and defects in the trunk structure. In addition, these phenotypes were associated with abnormal expression of three heart-chamber markers including cmlc2, amhc and vmhc, and abnormal expression of myogenic regulatory factors including myod and myog. These data suggest that Smyd3 plays an important role in the development of heart and skeletal muscle.We here show that transcripts of smyd3 were expressed in zebrafish embryos at all developmental stages examined and that knockdown of smyd3 in embryos resulted in pericardial edema and defects in the trunk structure.We find both smyd3 and setd7 are highly expressed within developing zebrafish heart and knock-down of these genes led to severe defects in cardiac morphogenesis without altering the expressions pattern of heart markers, including cmlc2, vmhc, and amhc.Smyd3 is required for the development of cardiac and skeletal muscle in zebrafish.SMYD1, SMYD2, SMYD3, and SMYD4 have been found to play critical roles in carcinogenesis and/or the development of heart and skeletal muscle.SMYD1, SMYD2, SMYD3, and SMYD4 have been found to play critical roles in carcinogenesis and/or the development of heart and skeletal muscle.Smyd3 is required for the development of cardiac and skeletal muscle in zebrafish.These data suggest that Smyd3 plays an important role in the development of heart and skeletal muscle..\"}, {\"question\": \"Which histone mutations have been associated with pediatric gliomas?\", \"ideal_answer\": [\"About 80% of Diffuse intrinsic pontine glioma (DIPG) cases and 70% of midline glioblastomas contain a mutation at one allele of the H3F3A gene (encoding histone H3 variant H3.3), replacing the lysine 27 with methionine (K27M). Moreover, approximately 30% of pediatric high grade gliomas (pedHGG) including GBM and DIPG harbor a lysine 27 mutation (K27M) in histone 3.3 (H3.3) which is correlated with poor outcome. Recent studies on high-grade pediatric GBM have identified two recurrent mutations (K27M and G34R/V) in genes encoding histone H3 (H3F3A for H3.3 and HIST1H3B for H3.1)\", \"Moreover, approximately 30% of pediatric high grade gliomas (pedHGG) including GBM and DIPG harbor a lysine 27 mutation (K27M) in histone 3.3 (H3.3) which is correlated with poor outcome and was shown to influence EZH2 function Recent studies on high-grade pediatric GBM have identified two recurrent mutations (K27M and G34R/V) in genes encoding histone H3 (H3F3A for H3.3 and HIST1H3B for H3.1) Recent studies have identified a Lys 27-to-methionine (K27M) mutation at one allele of H3F3A, one of the two genes encoding histone H3 variant H3.3, in 60% of high-grade pediatric glioma cases.\", \"Recent studies on high-grade pediatric GBM have identified two recurrent mutations (K27M and G34R/V) in genes encoding histone H3 (H3F3A for H3.3 and HIST1H3B for H3.1). Two new studies show that the known histone H3 alteration p.Lys27Met in pediatric glioma leads to globally diminished trimethylation at histone H3 lysine 27. These results indicate that H3.3K27M mutation reprograms epigenetic landscape and gene expression, which may drive tumorigenesis.\", \", approximately 30% of pediatric high grade gliomas (pedhgg) including gbm and dipg harbor a lysine 27 mutation (k27m) in histone 3.3 (h3.3) which is correlated with poor outcome and was shown to influence ezh2 function . the h3f3a mutant allele found in high-grade pediatric glioma by real-time pcr . studies on high-grade pediatric gbm have identified two recurrent mutations (k27m and g34r/v) in genes encoding histone h3 (h3f3a for h3.3 and hist1h3b for h3.1) . has been reported recently that about 80% of dipg cases and 70% of midline glioblastomas contain a mutation at one allele of the h3f3a gene (encoding histone h3 variant h3.3) , replacing the lysine 27 with methionine (k27m). . , discuss vaccine treatment for children diagnosed with malignant glioma , through targeting epha2 , il-13r\\u03b12 and/or histone h3 k27m , while in adults , treatments with rintega , prophage series g-100 and dendritic cells are explored. . studies have identified a lys 27-to-methionine (k27m) mutation at one allele of h3f3a , one of the two genes encoding histone h3 variant h3.3 , in 60% of high-grade pediatric glioma cases. . new studies show that the known histone h3 alteration p.lys27met in pediatric glioma leads to globally diminished trimethylation at histone h3 lysine 27 . were able to discern the h3f3a k27m mutation in a newly obtained pediatric brainstem glioblastoma sample whose h3.3 status was not known previously , and in three other dipg samples as well as paraffin embedded samples these results demonstrate that have developed a new reliable procedure for detecting the h3f3a k27m mutation in pediatric glioblastoma patient samples. . \"], \"exact_answer\": [[\"K27M in H3F3A\"], [\"G34R/V in HIST1H3B\"]], \"context\": \"Detecting the H3F3A mutant allele found in high-grade pediatric glioma by real-time PCR It has been reported recently that about 80% of DIPG cases and 70% of midline glioblastomas contain a mutation at one allele of the H3F3A gene (encoding histone H3 variant H3.3), replacing the lysine 27 with methionine (K27M).In order to facilitate diagnosis of DIPG patients, a quick and reliable method to identify the H3F3A K27M mutation is needed.Using this optimized real-time PCR assay, we analyzed eleven samples, two of which containing H3F3A K27M mutation, and found that these two samples were differentially amplified from the nine others. we were able to discern the H3F3A K27M mutation in a newly obtained pediatric brainstem glioblastoma sample whose H3.3 status was not known previously, and in three other DIPG samples as well as paraffin embedded samples. These results demonstrate that we have developed a new reliable procedure for detecting the H3F3A K27M mutation in pediatric glioblastoma patient samples.Moreover, approximately 30% of pediatric high grade gliomas (pedHGG) including GBM and DIPG harbor a lysine 27 mutation (K27M) in histone 3.3 (H3.3) which is correlated with poor outcome and was shown to influence EZH2 functionA lesson learned from the H3.3K27M mutation found in pediatric glioma: a new approach to the study of the function of histone modifications in vivo?Recent studies on high-grade pediatric GBM have identified two recurrent mutations (K27M and G34R/V) in genes encoding histone H3 (H3F3A for H3.3 and HIST1H3B for H3.1)Genome-wide studies using ChIP-seq on H3.3K27M patient samples indicate a global reduction of H3K27me3 on chromatin.Remarkably, we also found a dramatic enrichment of H3K27me3 and EZH2 (the catalytic subunit H3K27 methyltransferase) at hundreds of gene loci in H3.3K27M patient cellsTwo new studies show that the known histone H3 alteration p.Lys27Met in pediatric glioma leads to globally diminished trimethylation at histone H3 lysine 27. The histone H3.3K27M mutation in pediatric glioma reprograms H3K27 methylation and gene expression.Recent studies have identified a Lys 27-to-methionine (K27M) mutation at one allele of H3F3A, one of the two genes encoding histone H3 variant H3.3, in 60% of high-grade pediatric glioma cases.These results indicate that H3.3K27M mutation reprograms epigenetic landscape and gene expression, which may drive tumorigenesis.Here, we discuss vaccine treatment for children diagnosed with malignant glioma, through targeting EphA2, IL-13R\\u03b12 and/or histone H3 K27M, while in adults, treatments with RINTEGA, Prophage Series G-100 and dendritic cells are explored.Histone H3 lysine(27)-to-methionine (H3K27M) gain-of-function mutations occur in highly aggressive pediatric gliomas.Sequencing of pediatric gliomas has identified missense mutations Lys27Met (K27M) and Gly34Arg/Val (G34R/V) in genes encoding histone H3.3 (H3F3A) and H3.1 (HIST3H1B).Over 70% of diffuse intrinsic pediatric gliomas, an aggressive brainstem tumor, harbor heterozygous mutations that create a K27M amino acid substitution (methionine replaces lysine 27) in the tail of histone H3.3.Recently, sequencing of tumor cells revealed that histone H3 is frequently mutated in pediatric HGG, with up to 78\\u00a0% of diffuse intrinsic pontine gliomas (DIPGs) carrying K27M and 36\\u00a0% of non-brainstem gliomas carrying either K27M or G34R/V mutations.Use of human embryonic stem cells to model pediatric gliomas with H3.3K27M histone mutation.Exon sequencing has identified a mutation in K27M of the histone H3.3 gene (H3F3A K27M and G34R/V) in about 20% of pediatric glioblastomas, but it remains to be seen whether these mutations can be considered specific for pediatric diffuse high-grade astrocytomas or also occur in other pediatric brain tumors.K27M mutation in histone H3.3 defines clinically and biologically distinct subgroups of pediatric diffuse intrinsic pontine gliomas.Recently, sequencing of tumor cells revealed that histone H3 is frequently mutated in pediatric HGG, with up to 78\\u00a0% of diffuse intrinsic pontine gliomas (DIPGs) carrying K27M and 36\\u00a0% of non-brainstem gliomas carrying either K27M or G34R/V mutationsRecent sequencing studies have shown that ~30\\u00a0% of pediatric GBM and ~80\\u00a0% of diffuse intrinsic pontine gliomas show K27M mutations in the H3F3A gene, a variant encoding histone H3.3Histone H3 lysine(27)-to-methionine (H3K27M) gain-of-function mutations occur in highly aggressive pediatric gliomasSequencing of pediatric gliomas has identified missense mutations Lys27Met (K27M) and Gly34Arg/Val (G34R/V) in genes encoding histone H3.3 (H3F3A) and H3.1 (HIST3H1B)Over 70% of diffuse intrinsic pediatric gliomas, an aggressive brainstem tumor, harbor heterozygous mutations that create a K27M amino acid substitution (methionine replaces lysine 27) in the tail of histone H3.3Exon sequencing has identified a mutation in K27M of the histone H3.3 gene (H3F3A K27M and G34R/V) in about 20% of pediatric glioblastomas, but it remains to be seen whether these mutations can be considered specific for pediatric diffuse high-grade astrocytomas or also occur in other pediatric brain tumorsPediatric brainstem gliomas often harbor oncogenic K27M mutation of histone H3.3K27M mutation in histone H3.3 defines clinically and biologically distinct subgroups of pediatric diffuse intrinsic pontine gliomasOver 70% of diffuse intrinsic pediatric gliomas, an aggressive brainstem tumor, harbor heterozygous mutations that create a K27M amino acid substitution (methionine replaces lysine 27) in the tail of histone H3.3. Recent studies have identified a Lys 27-to-methionine (K27M) mutation at one allele of H3F3A, one of the two genes encoding histone H3 variant H3.3, in 60% of high-grade pediatric glioma cases. We established a Drosophila animal model for the pathogenic histone H3K27M mutation and show that its overexpression resembles polycomb repressive complex 2 (PRC2) loss-of-function phenotypes, causing derepression of PRC2 target genes and developmental perturbations. Recent sequencing studies have shown that ~30\\u00a0% of pediatric GBM and ~80\\u00a0% of diffuse intrinsic pontine gliomas show K27M mutations in the H3F3A gene, a variant encoding histone H3.3.Evolutionary reconstruction indicates histone 3 (H3) K27M--including H3.2K27M--mutations potentially arise first and are invariably associated with specific, high-fidelity obligate partners throughout the tumour and its spread, from diagnosis to end-stage disease, suggesting mutual need for tumorigenesis.Moreover, approximately 30% of pediatric high grade gliomas (pedHGG) including GBM and DIPG harbor a lysine 27 mutation (K27M) in histone 3.3 (H3.3) which is correlated with poor outcome and was shown to influence EZH2 function.The present study investigated the correlation of expression of EZH2 and other PRC2 genes (EZH1, SUZ12, EED) with overall survival of pediatric GBM patients and the cytotoxic impact of EZH2 inhibition by the novel agent Tazemetostat in pediatric GBM/DIPG cells harboring either a H3.3 mutation or a H3 wildtype.EZH2 gene expression does not cRecently, sequencing of tumor cells revealed that histone H3 is frequently mutated in pediatric HGG, with up to 78\\u00a0% of diffuse intrinsic pontine gliomas (DIPGs) carrying K27M and 36\\u00a0% of non-brainstem gliomas carrying either K27M or G34R/V mutations.Important insights have also been uncovered in the field of pediatric glioma, including the identification of recurrent mutation, fusion, and/or duplication events of the BRAF, FGFR1, MYB, and MYBL1 genes in pediatric low-grade gliomas, mutations affecting histone components (H3F3A p.K27M or p.G34) in pediatric high-grade gliomas, and aggressive subsets developing in midline central nervous system structures.K27M-H3.3 is universally associated with short survival in DIPG, while patients wild-type for H3.3 show improved survival.Taken together, our results suggest that mutations disrupting the histone code at H3K36, including H3.3 G34R/V, IDH1 and/or SETD2 mutations, are central to the genesis of hemispheric HGGs in older children and young adults.Gliomas with H3F3A K27M from pediatric or young adult patients had similar, characteristic DNA methylation profiles.K27M-H3.3 is universally associated with short survival in DIPG, while patients wild-type for H3.3 show improved survival.Moreover, the gain of H3K27me3 and Ezh2 at gene promoters alters the expression of genes that are associated with various cancer pathways.Use of human embryonic stem cells to model pediatric gliomas with H3.3K27M histone mutation.The histone H3.3K27M mutation in pediatric glioma reprograms H3K27 methylation and gene expression.K27M mutation in histone H3.3 defines clinically and biologically distinct subgroups of pediatric diffuse intrinsic pontine gliomas.A lesson learned from the H3.3K27M mutation found in pediatric glioma: a new approach to the study of the function of histone modifications in vivo?\"}, {\"question\": \"Which are the triad symptoms of pheochromocytoma?\", \"ideal_answer\": [\"The classic triad of symptoms are episodic headache, excessive sweating (diaphoresis) and palpitation.\"], \"exact_answer\": [[\"headache\"], [\"excessive sweating\", \"diaphoresis\"], [\"palpitation\"]], \"context\": \"Diaphoresis (LR+ 2.2, LR- 0.45), Palpitation (LR+ 1.9, LR- 0.52) and headache (LR+ 1.6, LR- 0.24) were significant symptoms in clinical diagnosis of pheochromocytoma. In the 52 cases analyzed, 40 of the patients had symptoms: 31 patients had hypertension; 10 had the triad of palpitations, diaphoresis, and headaches; and all had elevated urinary metanephrine concentrations. The symptoms and signs of pheochromocytoma include the classic triad of episodic headache, increased sweating, and palpitations. Among the presenting symptoms, episodes of palpitations, headaches, and profuse sweating are typical and constitute a classic triad.The classic triad of symptoms are episodic headache, excessive sweating and palpitation. The symptoms and signs of pheochromocytoma include the classic triad of episodic headache, increased sweating, and palpitations.Combinations of symptoms can be of value: 2 or more symptoms from the triad of headache, palpitations, and diaphoresis were present in the majority of pheochromocytoma patients, but in a smaller number of non-pheochromocytoma patients.The symptoms and signs of pheochromocytoma include the classic triad of episodic headache, increased sweating, and palpitationsPheochromocytoma in pregnancy is rare, and if unrecognized, can cause serious perinatal morbidity and mortality.A patient with severe hypertension, postpartum pulmonary edema, and a recognized pheochromocytoma is described.Abdominal palpation after vaginal childbirth reproduced the diagnostic triad of hypertension, headaches, and palpitationsCombinations of symptoms can be of value: 2 or more symptoms from the triad of headache, palpitations, and diaphoresis were present in the majority of pheochromocytoma patients, but in a smaller number of non-pheochromocytoma patientsAlthough hypertension, headache, and diaphoresis are common symptoms in a dialyzed patient, pheochromocytoma has to be eliminated in the presence of this clinical triadPheochromocytomas are catecholamine producing tumors that classically present with the triad of sweating, palpitations and headache.9-year-old boy whose only presenting complaints were polyuria and polydipsia for 2 years.Routine measurement of blood pressure detected mild hypertension, and subsequent investigations revealed bilateral pheochromocytoma.Surgical removal of the tumors resulted in complete resolution of polyuria and polydipsia.The case highlights the importance of measuring BP for children as part of physical examinationThe symptoms and signs of pheochromocytoma include the classic triad of episodic headache, increased sweating, and palpitations.Although hypertension, headache, and diaphoresis are common symptoms in a dialyzed patient, pheochromocytoma has to be eliminated in the presence of this clinical triad.Combinations of symptoms can be of value: 2 or more symptoms from the triad of headache, palpitations, and diaphoresis were present in the majority of pheochromocytoma patients, but in a smaller number of non-pheochromocytoma patients.In the 52 cases analyzed, 40 of the patients had symptoms: 31 patients had hypertension; 10 had the triad of palpitations, diaphoresis, and headaches; and all had elevated urinary metanephrine concentrations.The classic triad of symptoms are episodic headache, excessive sweating and palpitation.The clinical symptoms of these tumors vary from isolated hypertension or hypertension accompanied by paroxysmal episodes -including the classical triad of headache, palpitations and diaphoresis-to potentially serious manifestations such as acute pulmonary edema, arrhythmias and sudden death.A very few symptoms were specific, like the triad \\\"headaches, sweating, palpitations\\\" whose onlyone third of patients was concerned by.Classically, a triad of symptoms includes sweating, palpitations, and headache.The triad of diaphoresis, tachycardia, and headache in hypertensive patients is highly suggestive of pheochromocytoma.A very few symptoms were specific, like the triad \\\"headaches, sweating, palpitations\\\" whose onlyone third of patients was concerned by.We noticed, like others, when the triad of headache, sweating and palpitations is accompanied by hypertension, the diagnosis of pheochromocytoma can be made with specify and sensitivity over 93%.\"}, {\"question\": \"List the types of the Cardiorenal syndrome (CRS) according to the five-part classification system.\", \"ideal_answer\": [\"Cardiorenal syndromes (CRS) have been recently classified into five distinct entities, each with different major pathophysiologic mechanisms. \\nCRS type 1: acute worsening of heart function (AHF-ACS) leading to kidney injury and/or dysfunction. \\nCRS type 2: chronic abnormalities in heart function (CHF-CHD) leading to kidney injury or dysfunction. \\nCRS type 3: acute worsening of kidney function (AKI) leading to heart injury and/or dysfunction. \\nCRS type 4: chronic kidney disease (CKD) leading to heart injury, disease and/or dysfunction. \\nCRS type 5: systemic conditions leading to simultaneous injury and/or dysfunction of heart and kidney.\"], \"exact_answer\": [[\"CRS type 1\"], [\"CRS type 2\"], [\"CRS type 3\"], [\"CRS type 4\"], [\"CRS type 5\"]], \"context\": \"The general definition has been expanded into five subtypes: CRS type 1 = acute worsening of heart function (acute heart failure-acute coronary syndrome) leading to kidney injury and/or dysfunction; CRS type 2 = chronic abnormalities in heart function (chronic heart failure-chronic heart disease) leading to kidney injury or dysfunction; CRS type 3 = acute worsening of kidney function (acute kidney injury) leading to heart injury and/or dysfunction; CRS type 4 = chronic kidney disease (chronic kidney disease) leading to heart injury, disease and/or dysfunction; and CRS type 5 = systemic conditions leading to simultaneous injury and/or dysfunction of heart and kidney. Cardiorenal syndromes (CRS) have been recently classified into five distinct entities, each with different major pathophysiologic mechanisms. CRS type 1 most commonly occurs in the setting of acutely decompensated heart failure where approximately 25% of patients develop a rise in serum creatinine and a reduction of urine output after the first several doses of intravenous diuretics. CRS type 2 is the hastened progression of chronic kidney disease (CKD) in the setting of chronic heart failure. CRS type 3 is acutely decompensated heart failure after acute kidney injury from inflammatory, toxic, or ischemic insults. CRS type 4 is manifested by the acceleration of the progression of chronic heart failure in the setting of CKD. Cardiac myocyte dysfunction and fibrosis, so-called 'CKD cardiomyopathy', is believed to be the predominant pathophysiologic mechanism. Type 5 CRS is simultaneous acute cardiac and renal injury in the setting of an overwhelming systemic insult such as sepsis. CRS type I: acute worsening of heart function (AHF-ACS) leading to kidney injury and/or dysfunction. CRS type II: chronic abnormalities in heart function (CHF-CHD) leading to kidney injury or dysfunction. CRS type III: acute worsening of kidney function (AKI) leading to heart injury and/or dysfunction. CRS type IV: chronic kidney disease (CKD) leading to heart injury, disease and/or dysfunction. CRS type V: systemic conditions leading to simultaneous injury and/or dysfunction of heart and kidney. Acute CRS (type 1): acute worsening of heart function (AHF-ACS) leading to kidney injury and/or dysfunction. Chronic cardio-renal syndrome (type 2): chronic abnormalities in heart function (CHF-CHD) leading to kidney injury and/or dysfunction. Acute reno-cardiac syndrome (type 3): acute worsening of kidney function (AKI) leading to heart injury and/or dysfunction. Chronic reno-cardiac syndrome (type 4): chronic kidney disease leading to heart injury, disease, and/or dysfunction. Secondary CRS (type 5): systemic conditions leading to simultaneous injury and/or dysfunction of heart and kidney. For mentioning the interaction of cardiovascular and renal diseases the cardiorenal syndrome (CRS) term was introduced, with its classification on 5 types, according to the presence of acute/chronic heart failure and primary/secondary origination of heart and kidney injury.For mentioning the interaction of cardiovascular and renal diseases the cardiorenal syndrome (CRS) term was introduced, with its classification on 5 types, according to the presence of acute/chronic heart failure and primary/secondary origination of heart and kidney injuryThe most recent classification recognizes five types of CRS: types I and II originate from heart failure (acute and chronic, respectively), type III and IV from kidney failure (again acute and chronic), while type V originates from a range of systemic diseasesThe most recent classification recognizes five types of CRS: types I and II originate from heart failure (acute and chronic, respectively), type III and IV from kidney failure (again acute and chronic), while type V originates from a range of systemic diseases.For mentioning the interaction of cardiovascular and renal diseases the cardiorenal syndrome (CRS) term was introduced, with its classification on 5 types, according to the presence of acute/chronic heart failure and primary/secondary origination of heart and kidney injury.Based on the pathophysiological primum movens, the actual classification recognizes five CRS types: in type I and II CRS, the initiating event is heart failure (acute or chronic), while it is kidney failure in type III and IV CRS; type V is linked to systemic diseases.An effective classification of CRS in 2008 essentially divides CRS in two main groups, cardiorenal and renocardiac CRS, based on primum movens of disease (cardiac or renal); both cardiorenal and renocardiac CRS are then divided into acute and chronic, according to onset of disease.Of particular interest to the critical care specialist are CRS type 1 (acute cardiorenal syndrome) and type 3 (acute renocardiac syndrome).At present, the classification of the syndrome includes five types of CRS: types I and II which are strictly related to initial heart failure (both acute and chronic), types III and IV which include initial kidney failure, and type V which includes several systemic diseases.The general definition has been expanded into five subtypes: CRS type 1 = acute worsening of heart function (acute heart failure-acute coronary syndrome) leading to kidney injury and/or dysfunction; CRS type 2 = chronic abnormalities in heart function (chronic heart failure-chronic heart disease) leading to kidney injury or dysfunction; CRS type 3 = acute worsening of kidney function (acute kidney injury) leading to heart injury and/or dysfunction; CRS type 4 = chronic kidney disease (chronic kidney disease) leading to heart injury, disease and/or dysfunction; and CRS type 5 = systemic conditions leading to simultaneous injury and/or dysfunction of heart and kidney.The general definition has been expanded into five subtypes reflecting the primacy of organ dysfunction and the time-frame of the syndrome: CRS type 1 = acute worsening of heart function leading to kidney injury and/or dysfunction; CRS type 2 = chronic abnormalities in heart function leading to kidney injury or dysfunction; CRS type 3 = acute worsening of kidney function leading to heart injury and/or dysfunction; CRS type 4 = chronic kidney disease leading to heart injury, disease and/or dysfunction, and CRS type 5 = systemic conditions leading to simultaneous injury and/or dysfunction of heart and kidney.\"}, {\"question\": \"Which are the components of the pre-replication complex (pre-RC) in eukaryotes?\", \"ideal_answer\": [\"The components of the pre-replication complex (pre-RC) in eukaryotes are:\\n1) Cdc6/Cdc18, \\n2) MCM, \\n3) ORC1-6,\\n4) Cdt1 and\\n5) Sap1/Gi.\"], \"exact_answer\": [[\"Cdc6/Cdc18\"], [\"MCM\"], [\"ORC1-6\"], [\"Cdt1\"], [\"Sap1/Gi\"]], \"context\": \" The first step of replication initiation is the assembly of pre-replication complex (pre-RC). Since 1973, four proteins, Cdc6/Cdc18, MCM, ORC and Cdt1, have been extensively studied and proved to be pre-RC components. Recently, a novel pre-RC component called Sap1/Girdin was identified. Sap1/Girdin is required for loading Cdc18/Cdc6 to origins for pre-RC assembly in the fission yeast and human cells, respectively. In eukaryotes, the pre-replication complex is composed of the Origin Recognition Complex (ORC), Cdc6 and the MCM replicative helicase in conjunction with Cdt1. Eukaryotic ORC is considered to be composed of six subunits, named Orc1-6, and monomeric Cdc6 is closely related in sequence to Orc1.The sequencing of the Plasmodium genome has also revealed the apparent absence of many key components (e.g. Cdt1, DDK and Cdc45) of the eukaryotic cell cycle machinery that are responsible for the formation of the pre-replication complex (pre-RC). We have characterized the Plasmodium falciparum minichromosome maintenance complex (MCM) that plays a key role in the transition of pre-RC to the RC. Similar to other eukaryotes, the Plasmodium genome encodes six MCM subunits. During pre-replication complex (pre-RC) formation, origin recognition complex (ORC), Cdc6, and Cdt1 cooperatively load the 6-subunit mini chromosome maintenance (MCM2-7) complex onto DNA.ORC1 is an essential component of the pre-replicative complex (pre-RC) that licenses eukaryote DNA replication origins.In higher eukaryotes, the pre-replication complex (pre-RC) component Cdt1 is the major regulator in licensing control for DNA replication.The sequencing of the Plasmodium genome has also revealed the apparent absence of many key components (e.g. Cdt1, DDK and Cdc45) of the eukaryotic cell cycle machinery that are responsible for the formation of the pre-replication complex (pre-RC)We examined the mRNA expression of the components of pre-replication complex (Pre-RC), CDC6, CDT1, and MCM2-7Replication initiation is a multi-step process involving many factors including ORC, Cdt1p, Mcm2-7p and other proteins that bind to replication origins to form a pre-replicative complex (pre-RC)The sequencing of the Plasmodium genome has also revealed the apparent absence of many key components (e.g. Cdt1, DDK and Cdc45) of the eukaryotic cell cycle machinery that are responsible for the formation of the pre-replication complex (pre-RC).Different components of the ORC complex and Cdc6 stimulated prereplicative complex (pre-RC) formation and replication initiation when fused to the GAL4 DNA-binding domain and recruited to plasmid DNA containing a tandem array of GAL4-binding sites.Cdt1, DDK and Cdc45) of the eukaryotic cell cycle machinery that are responsible for the formation of the pre-replication complex (pre-RC).Recently, a novel pre-RC component called Sap1/Girdin was identified.MCM2-7 is a key component of the prereplicative complex (pre-RC), which is loaded onto chromatin by the concerted action of origin recognition complex, Cdc6, and Cdt1.The pre-replicative complex (pre-RC) is formed at all potential origins of replication through the action of the origin recognition complex (ORC), Cdc6, Cdt1, and the Mcm2-7 complex.\"}, {\"question\": \"What is the inheritance of hypophosphatemic rickets?\", \"ideal_answer\": [\"Hypophosphatemic rickets are transmitted with:\\n1) autosomal recessive\\n2) autosomal dominant\\n3) X-linked recessive and\\n4) X-linked dominant inheritance.\"], \"exact_answer\": [[\"autosomal recessive\"], [\"autosomal dominant\"], [\"X-linked recessive\"], [\"X-linked dominant\"]], \"context\": \"Among 35 candidate genes in this region, the dentin matrix protein 1 gene (DMP1) was sequenced to reveal a nonsense mutation 250C/T on exon 6. Previous research has shown that mutations in the DMP1 gene are responsible for autosomal recessive hypophosphatemic rickets in humans.A novel nonsense mutation in the DMP1 gene in a Japanese family with autosomal recessive hypophosphatemic rickets.Autosomal recessive hypophosphatemic rickets (ARHR) is an extremely rare disorder of autosomal recessive inheritance, characterized by hypophosphatemia resulting from renal phosphate wasting. Prenatal diagnosis for a novel splice mutation of PHEX gene in a large Han Chinese family affected with X-linked hypophosphatemic rickets.X-linked hypophosphatemia (XLH) is the most common form of heritable rickets characterized by X-linked dominant inheritance, renal phosphate wasting, hypophosphatemia, and defective bone mineralization.Molecular analysis of DMP1 mutants causing autosomal recessive hypophosphatemic rickets.We previously demonstrated that the mutations Met1Val (M1V) and the deletion of nucleotides 1484-1490 (1484-1490del) in Dentin matrix protein-1 (DMP1) cause the novel disorder autosomal recessive hypophosphatemic rickets (ARHR), which is associated with elevated fibroblast growth factor-23 (FGF23).First, we examined a kindred with a severe hypophosphatemic rickets phenotype and recessive inheritance. Discordance for X-linked hypophosphataemic rickets in identical twin girls.We report monozygotic twin girls with a family history consistent with X-linked hypophosphataemic rickets (XLH). Microsatellite analysis was performed to confirm monozygosity and bi-parental inheritance of the X chromosome. Hypophosphatemia due to isolated renal phosphate wasting is a genetically heterogeneous disease. Two new genes linked to two different forms of hereditary hypophosphatemias have recently been described. Autosomal recessive form of hypophosphatemic rickets was mapped to chromosome 4q21 and identified homozygous mutations in dentin matrix protein 1 (DMP1) gene, which encodes a non-collagenous bone matrix protein.Somatic and germline mosaicism for a mutation of the PHEX gene can lead to genetic transmission of X-linked hypophosphatemic rickets that mimics an autosomal dominant trait.Familial hypophosphatemic rickets is usually transmitted as an X-linked dominant disorder (XLH), although autosomal dominant forms have also been observed. Genetic studies of these disorders have identified mutations in PHEX and FGF23 as the causes of X-linked dominant disorder and autosomal dominant forms, respectively.Direct nucleotide sequencing of FGF23 and PHEX revealed that the elder daughter was heterozygous for an R567X mutation in PHEX, rather than FGF23, suggesting that the genetic transmission occurred as an X-linked dominant trait. Somatic and germline mosaicism for an X-linked dominant mutation in PHEX may mimic autosomal dominant inheritance.We report two cases of x-linked dominant hypophosphatemic rickets involving a man and his daughter. X-linked hypophosphatemic rickets--a report of 2 cases and review of literature.The study of this family tree strongly suggests an x-linked dominant inheritance.Autosomal dominant hypophosphatemic rickets/osteomalacia: clinical characterization of a novel renal phosphate-wasting disorder.We performed clinical and biochemical evaluations of individuals from a large kindred with autosomal dominant hypophosphatemic rickets/osteomalacia. In conclusion, autosomal dominant hypophosphatemic rickets/osteomalacia is an inherited disorder of isolated renal phosphate wasting. [Scriver type autosomal hypophosphatemic rachitis: a family case].A familial observation of hypophosphatemic rickets with unusual inheritance and evolution, different from that of X linked hypophosphatemia, is reported. This observation is quite similar to the 'autosomal hypophosphatemic bone disease' described by Scriver et al. The HHRH is transmitted by autosomal recessive inheritance and is an extremely rare form of hypophosphatemic rickets.First, we examined a kindred with a severe hypophosphatemic rickets phenotype and recessive inheritance.A familial observation of hypophosphatemic rickets with unusual inheritance and evolution, different from that of X linked hypophosphatemia, is reported.Autosomal recessive hypophosphatemic rickets (ARHR) is an extremely rare disorder of autosomal recessive inheritance, characterized by hypophosphatemia resulting from renal phosphate wasting.Hereditary hypophosphatemic rickets with hypercalciuria (HHRH) is a rare disorder of autosomal recessive inheritance that was first described in a large consanguineous Bedouin kindred.Autosomal recessive hypophosphatemic rickets (ARHR) is an extremely rare disorder of autosomal recessive inheritance, characterized by hypophosphatemia resulting from renal phosphate wastingA familial observation of hypophosphatemic rickets with unusual inheritance and evolution, different from that of X linked hypophosphatemia, is reportedThe HHRH is transmitted by autosomal recessive inheritance and is an extremely rare form of hypophosphatemic ricketsFirst, we examined a kindred with a severe hypophosphatemic rickets phenotype and recessive inheritanceHereditary hypophosphatemic rickets with hypercalciuria (HHRH) is a rare disorder of autosomal recessive inheritance that was first described in a large consanguineous Bedouin kindredGenetic studies of these disorders have identified mutations in PHEX and FGF23 as the causes of X-linked dominant disorder and autosomal dominant forms, respectively.The objective of the study was to describe the molecular genetic findings in a family affected by hypophosphatemic rickets with presumed autosomal dominant inheritance.We studied a family in which the father and the elder of his two daughters, but not the second daughter, were affected by hypophosphatemic ricketsHereditary hypophosphatemic rickets with hypercalciuria is another rare disorder of autosomal recessive inheritanceFirst, we examined a kindred with a severe hypophosphatemic rickets phenotype and recessive inheritance.Autosomal recessive form of hypophosphatemic rickets was mapped to chromosome 4q21 and identified homozygous mutations in dentin matrix protein 1 (DMP1) gene, which encodes a non-collagenous bone matrix protein.In addition, such CLCN5 mutations that would result in a functional loss have also been demonstrated in Japanese children with idiopathic low molecular weight proteinuria, hypercalciuria and nephrocalcinosis, and an Italian kindred with X-linked recessive hypophosphatemic rickets (XLRH) and hypercalciuria.Inactivating mutations of phosphate-regulating gene with homologies to endopeptidases on the X chromosome, dentin matrix acidic phosphoprotein 1, and ectonucleotide pyrophosphatase/phosphodiesterase 1 are associated with X-linked hypophosphatemic rickets, autosomal recessive hypophosphatemic rickets 1, and autosomal recessive hypophosphatemic rickets 2, respectively.The objective of the study was to describe the molecular genetic findings in a family affected by hypophosphatemic rickets with presumed autosomal dominant inheritance.The HHRH is transmitted by autosomal recessive inheritance and is an extremely rare form of hypophosphatemic rickets.X-linked familial hypophosphatemic rickets (X.L.F.H.R.) is one of the D resistant rickets.Iron deficiency may affect autosomal dominant hypophosphatemic rickets phenotype by regulating FGF23 production.Current treatment with activated vitamin D metabolites and oral inorganic phosphate salts may partially correct skeletal lesions and linear growth in patients with hypophosphatemic rickets.[X-linked familial hypophosphatemic rickets report of six cases (author's transl)].\"}, {\"question\": \"Which NGS alignment software implement the Burrows-Wheeler Transform?\", \"ideal_answer\": [\"The most widely used software belong to the family of the Burrows-Wheeler Aligner (BWA) and its variants for local alignment BWASW, map reduce BWASW-PMR and multi-threaded implementation BWA-MT. Other approaches include Bowtie, SOAP2, BWBBLE, SOAP2 and FANSe2.\"], \"exact_answer\": [[\"Bowtie\"], [\"Burrows-Wheeler Alignment Tool\", \"BWA\"], [\"SOAP2\"], [\"Burrows-Wheeler Aligner's Smith-Waterman Alignment\", \"BWA-SW\"], [\"TotalReCaller\"], [\"BWBBLE\"], [\"FANSe2\"], [\"Burrows-Wheeler Aligner's Smith-Waterman Alignment Map Reduce\", \"BWASW-PMR\"], [\"Burrows-Wheeler Aligner Multi-Thread\", \"BWA-MT\"]], \"context\": \"For the human genome, Burrows-Wheeler indexing allows Bowtie to align more than 25 million reads per CPU hour with a memory footprint of approximately 1.3 gigabytes. Bowtie extends previous Burrows-Wheeler techniques with a novel quality-aware backtracking algorithm that permits mismatches.We implemented Burrows-Wheeler Alignment tool (BWA), a new read alignment package that is based on backward search with Burrows-Wheeler Transform (BWT), to efficiently align short sequencing reads against a large reference sequence such as the human genome, allowing mismatches and gaps.SOAP2 is a significantly improved version of the short oligonucleotide alignment program that both reduces computer memory usage and increases alignment speed at an unprecedented rate. We used a Burrows Wheeler Transformation (BWT) compression index to substitute the seed strategy for indexing the reference sequence in the main memory.We designed and implemented a new algorithm, Burrows-Wheeler Aligner's Smith-Waterman Alignment (BWA-SW), to align long sequences up to 1 Mb against a large sequence database (e.g. the human genome) with a few gigabytes of memory.Here, we propose a new base-calling algorithm, TotalReCaller, to achieve improved performance. A linear error model for the raw intensity data and Burrows-Wheeler transform (BWT) based alignment are combined utilizing a Bayesian score function, which is then globally optimized over all possible genomic locations using an efficient branch-and-bound approach.To tackle this problem, a new alignment tool BWBBLE is introduced in this article. We (i) introduce a new compressed representation of a collection of genomes, which explicitly tackles the genomic variation observed at every position, and (ii) design a new alignment algorithm based on the Burrows-Wheeler transform that maps short reads from a newly sequenced genome to an arbitrary collection of two or more (up to millions of) genomes with high accuracy and no inherent bias to one specific genome.Seed-based algorithms are generally slow but robust, while Burrows-Wheeler Transform (BWT) based algorithms are fast but less robust. To have both advantages, we developed an algorithm FANSe2 with iterative mapping strategy based on the statistics of real-world sequencing error distribution to substantially accelerate the mapping without compromising the accuracy.Therefore, in this paper, we introduce Burrows-Wheeler Aligner's Smith-Waterman Alignment on Parallel MapReduce (BWASW-PMR) cloud platform for long sequence alignment.In this paper, we propose a BWA-MT tool, evolved from BWA but supporting multi-thread computation, designed to fully utilize the underlying multi-core architecture of computing resources. By using multi-thread computation, BWA-MT can significantly shorten the time needed to generate an alignment for single-end read sequences.The speed of MAQ is also a concern when the alignment is scaled up to the resequencing of hundreds of individuals.We implemented Burrows-Wheeler Alignment tool (BWA), a new read alignment package that is based on backward search with Burrows-Wheeler Transform (BWT), to efficiently align short sequencing reads against a large reference sequence such as the human genome, allowing mismatches and gapsAlthough the Burrows-Wheeler Aligner (BWA) tool is one of the most widely used open-source software tools to align read sequences, it is still limited in that it does not fully support multi-thread mechanisms during the alignment stepsWe implemented Burrows-Wheeler Alignment tool (BWA), a new read alignment package that is based on backward search with Burrows-Wheeler Transform (BWT), to efficiently align short sequencing reads against a large reference sequence such as the human genome, allowing mismatches and gaps.We designed and implemented a new algorithm, Burrows-Wheeler Aligner's Smith-Waterman Alignment (BWA-SW), to align long sequences up to 1 Mb against a large sequence database (e.g.Although the Burrows-Wheeler Aligner (BWA) tool is one of the most widely used open-source software tools to align read sequences, it is still limited in that it does not fully support multi-thread mechanisms during the alignment steps.\"}, {\"question\": \"List the human acrocentric chromosomes that are involved in Robertsonian translocation.\", \"ideal_answer\": [\"Robertsonian translocations (ROBs) are the most common chromosomal rearrangements in humans. ROBs are whole-arm rearrangements between the acrocentric chromosomes 13, 14, 15, 21, and 22.\"], \"exact_answer\": [[\"13\"], [\"14\"], [\"15\"], [\"21\"], [\"22\"]], \"context\": \"Sperm segregation analysis of a (13;22) Robertsonian translocation carrier by FISH: a comparison of locus-specific probe and whole chromosome painting.The t(13;22) Robertsonian translocation constitutes a rare form of rearrangement between acrocentric human chromosomes. Most of the meiotic segregation studies of human Robertsonian translocations have been performed on common t(13;14) and t(14;21) translocations. Robertsonian translocations (ROBs) are the most common chromosomal rearrangements in humans. ROBs are whole-arm rearrangements between the acrocentric chromosomes 13-15, 21, and 22. ROBs can be classified into two groups depending on their frequency of occurrence, common (rob(13q14q) and rob(14q21q)), and rare (all remaining possible nonhomologous combinations). Robertsonian translocations (ROBs) are the most common rearrangements in humans, contributing significantly to genetic imbalance, fetal wastage, mental retardation and birth defects. Rob(14q21q) and rob(13q14q), which are formed predominantly during female meiosis, comprise the majority (approximately 85%) of all ROBs. Previous studies have shown that the breakpoints are consistently located within specific regions of the proximal short arms of chromosomes 13, 14, and 21. To localize the chAB4 sequences more precisely on the acrocentrics, chAB4-specific probes together with rDNA and a number of satellite sequences were hybridized to metaphase chromosomes of normal probands and of carriers of Robertsonian translocations of the frequent types rob(13q14q) and rob(14q21q). Chromosome 22, most probably, is the only chromosome where chAB4 is found in the direct neighbourhood of the centromere. Fluorescence in situ hybridization analyses of metaphase chromosomes of carriers of rob(21q22q) revealed breakpoint diversity for this rare type of Robertsonian translocation chromosome. This process is suggested for the evolution of an alpha satellite subfamily recently found on chromosomes 13, 14 and 21. The presence of this alpha subfamily may allow pairing between these chromosomes leading to the observed non-random participation of these chromosomes in t(13q14q) and t(14q21q) Robertsonian translocations.The pattern of association of acrocentric chromosomes was examined in ten and five carriers of a 15/21 and a 13/14 Robertsonian translocation, respectively, and was compared with that of the same numbers of relatives with normal karyotypes.Homologous alpha satellite sequences on human acrocentric chromosomes with selectivity for chromosomes 13, 14 and 21: implications for recombination between nonhomologues and Robertsonian translocations.Out of 21 structural chromosomal anomalies and polymorphic variants, six deletions and microdeletions, four situations with abnormal long \\\"p\\\" arm of acrocentric chromosomes, two duplications, two reciprocal translocations, two inversions, two additions, one Robertsonian translocation associating trisomy 13, one 9q heteromorphism and one complex chromosome rearrangement were noticedThe purpose of this study was to search for cytologic evidence of robertsonian translocation formation that involves chromosomes 14q and 21q in human oogenesis with the use of dual color fluorescent in situ hybridization with whole chromosome paints.The oocytes from a chromosomally normal fetus at 23.5 weeks of gestation underwent cohybridization with chromosome specific DNA libraries from chromosomes 14 and 21The t(13;22) Robertsonian translocation constitutes a rare form of rearrangement between acrocentric human chromosomesHomologous alpha satellite sequences on human acrocentric chromosomes with selectivity for chromosomes 13, 14 and 21: implications for recombination between nonhomologues and Robertsonian translocations.The t(13;22) Robertsonian translocation constitutes a rare form of rearrangement between acrocentric human chromosomes.\"}, {\"question\": \"List viral vectors used in gene therapy.\", \"ideal_answer\": [\"adeno-associated viruses\\nlentiviruses\\nherpes simplex viral vector\"], \"exact_answer\": [[\"adeno-associated viruses\"], [\"lentiviruses\"], [\"herpes simplex viral vector\"]], \"context\": \"Not only can some genetically engineered adenoviral vectors achieve remarkably efficient and specific gene delivery to target cells, but they also may act as anticancer agents by selectively replicating within cancer cells. Using these criteria, we then evaluate approaches made to model PD using viral vectors to date, including both adeno-associated viruses and lentivirusesRecombinant AAV (rAAV) vectors are a suitable vector for gene therapy studies because of desired characteristics such as low immunogenicity, transfection of non-dividing and dividing cells, and long-term expression of the transgene.Over the last five years, the number of clinical trials involving AAV (adeno-associated virus) and lentiviral vectors continue to increase by about 150 trials each year. For continued success, AAV and lentiviral expression cassettes need to be designed to meet each disease's specific needs. describes commonly used gene therapeutics (herpes simplex viral vector (HSV) and adeno-associated viral vector (AAV))This study examined the efficacy of gene therapy of lung adenocarcinoma using specifically controlled type I herpes simplex virus recombinant vector expressing Gibbon ape leukemia virus membrane fusion glycoprotein gene (GALV.fus)\"}, {\"question\": \"List the classical symptoms of the Moschcowitz syndrome (Thrombotic thrombocytopenic purpura).\", \"ideal_answer\": [\"The typical manifestations of Moschocowitz syndrome (Thrombotic-thrombocytopenic purpura) are:\\n1) thrombocytopenia, \\n2) haemolysis, \\n3) fever, \\n4) coma and \\n5) renal failure.\"], \"exact_answer\": [[\"thrombocytopenia\", \"thrombotic microangiopathy\"], [\"haemolysis\", \"hemolytic anemia\"], [\"fever\"], [\"coma\", \"neurological symptoms\"], [\"renal failure\"]], \"context\": \"The combination of neurological symptoms, thrombocytopenia, fever, renal failure and hemolytic anemia in a patient taking ticlopidine points to a diagnosis of TTP.In addition to the typical manifestations of thrombotic-thrombocytopenic purpura like thrombocytopenia, haemolysis, fever, coma and renal failure, signs of a beginning DIC could be seen in a patient after abdominal surgery. Moschcowitz syndrome or thrombotic thrombocytopenic purpura is a rare disorder with a poor prognosis. This syndrome is characterized by a microangiopathic hemolytic anemia with thrombocytopenia, neurologic symptoms and renal disease. Moschcowitz's syndrome is a rare condition with poor prognosis. It is characterized by a microangiopathic haemolytic anaemia associated with thrombocytopenia, neurological symptoms and renal involvement.Thrombotic thrombocytopenic purpura (TTP), in 1924 first described by Moschcowitz, is a clinically heterogeneous syndrome associated with thrombocytopenia, Coombs-negative hemolytic anemia, neurologic changes, renal impairment, and fever.Thrombotic thrombocytopenic purpura (TTP), in 1924 first described by Moschcowitz, is a clinically heterogeneous syndrome associated with thrombocytopenia, Coombs-negative hemolytic anemia, neurologic changes, renal impairment, and feverThrombotic thrombocytopenic purpura (TTP, Moschcowitz disease) is characterized by thrombotic microangiopathy leading to microvascular occlusion and ischemic dysfunction of various organs including the brainSevere deficiency of von Willebrand factor-cleaving protease (ADAMTS-13) activity (<5% of normal) is specific for classical thrombotic thrombocytopenic purpura (TTP), a disorder presenting with thrombocytopenia, microangiopathic haemolytic anaemia and often with organ dysfunction such as neurological symptoms, renal failure, and fever.In a 57-year-old female patient, who was admitted with fluctuating central neurological abnormalities and generalized purpura, was made the diagnosis of a thrombotic thrombocytopenic purpura (TTP, Moschcowitz' syndrome).Thrombotic thrombocytopenic purpura (TTP) is a syndrome that occurs mainly in adults with multiorgan microvascular thrombosis consisting of thrombocytopenia, microangiopathic hemolytic anemia, neurologic symptoms, renal involvement, and fever.\"}, {\"question\": \"List features of the Kaufman Oculocerebrofacial Syndrome.\", \"ideal_answer\": [\"Clinical features of the Kaufman Oculocerebrofacial Syndrome include hypotonia, developmental delay, intellectual disability, low cholesterol levels, microcephaly, long narrow face, ocular anomalies, and long thin hands and feet.\"], \"exact_answer\": [[\"hypotonia\"], [\"developmental delay\"], [\"intellectual disability\"], [\"low cholesterol levels\"], [\"microcephaly\"], [\"long narrow face\"], [\"ocular anomalies\"], [\"long thin hands and feet\"]], \"context\": \"A pair of sisters was ascertained for multiple congenital defects, including marked craniofacial dysmorphisms with blepharophimosis, and severe psychomotor delay. Genetic syndromes involving both brain and eye abnormalities are numerous and include syndromes such as Warburg micro syndrome, Kaufman oculocerebrofacial syndrome, Cerebro-oculo-facio-skeletal syndrome, Kahrizi syndrome and others. Biallelic mutations of UBE3B have recently been shown to cause Kaufman oculocerebrofacial syndrome (also reported as blepharophimosis-ptosis-intellectual disability syndrome), an autosomal recessive condition characterized by hypotonia, developmental delay, intellectual disability, congenital anomalies, characteristic facial dysmorphic features, and low cholesterol levels.BACKGROUND: Kaufman oculocerebrofacial syndrome (KOS) is a developmental disorder characterised by reduced growth, microcephaly, ocular anomalies (microcornea, strabismus, myopia, and pale optic disk), distinctive facial features (narrow palpebral fissures, telecanthus, sparse and laterally broad eyebrows, preauricular tags, and micrognathia), mental retardation, and generalised hypotonia. Kaufman oculocerebrofacial syndrome (KOS) is a rare autosomal recessive disorder characterized by severe mental retardation, microcephaly, long narrow face, ocular anomalies, and long thin hands and feet. Both showed psychomotor retardation, microcephaly, blepharophimosis and delayed growth as the main features; the infant also presented preauricular tags and large clitoris. Kaufman oculocerebrofacial syndrome (KOS) is a developmental disorder characterised by reduced growth, microcephaly, ocular anomalies (microcornea, strabismus, myopia, and pale optic disk), distinctive facial features (narrow palpebral fissures, telecanthus, sparse and laterally broad eyebrows, preauricular tags, and micrognathia), mental retardation, and generalised hypotonia.Biallelic mutations of UBE3B have recently been shown to cause Kaufman oculocerebrofacial syndrome (also reported as blepharophimosis-ptosis-intellectual disability syndrome), an autosomal recessive condition characterized by hypotonia, developmental delay, intellectual disability, congenital anomalies, characteristic facial dysmorphic features, and low cholesterol levelsKaufman oculocerebrofacial syndrome (KOS) is a developmental disorder characterised by reduced growth, microcephaly, ocular anomalies (microcornea, strabismus, myopia, and pale optic disk), distinctive facial features (narrow palpebral fissures, telecanthus, sparse and laterally broad eyebrows, preauricular tags, and micrognathia), mental retardation, and generalised hypotoniaKaufman oculocerebrofacial syndrome (KOS) is a rare autosomal recessive disorder characterized by severe mental retardation, microcephaly, long narrow face, ocular anomalies, and long thin hands and feetKaufman oculocerebrofacial syndrome (KOS) is a rare autosomal recessive disorder characterized by severe mental retardation, microcephaly, long narrow face, ocular anomalies, and long thin hands and feet.\"}, {\"question\": \"The MMR vaccine protects against what 3 viruses?\", \"ideal_answer\": [\"The MMR vaccine provides immunity to measles, mumps and rubella.\", \" measles, mumps and rubella (mmr) vaccine .\"], \"exact_answer\": [[\"Measles\"], [\"Mumps\"], [\"Rubella\"]], \"context\": \" measles, mumps and rubella (MMR) vaccine Measles, mumps, rubella (MMR) vaccine is a live vaccine preparation containing attenuated strains of all 3 viruses.\"}, {\"question\": \"List cardinal features of the Triple A syndrome.\", \"ideal_answer\": [\"Triple A syndrome, also known as Allgrove syndrome, is a rare disease, and presents mainly in children. Its cardinal symptoms are achalasia, alacrima, and adrenal insufficiency.\"], \"exact_answer\": [[\"achalasia\"], [\"alacrima\"], [\"adrenal insufficiency\"]], \"context\": \"UNLABELLED: Triple A syndrome (alacrima, achalasia, adrenal failure, progressive neurodegenerative disease) is caused by mutations in the AAAS gene which encodes the protein alacrima achalasia adrenal insufficiency neurologic disorder (ALADIN).Triple A syndrome, also known as Allgrove syndrome, is a rare disease, and presents mainly in children. Its cardinal symptoms are achalasia, alacrima, and adrenocorticotropic hormone (ACTH) insensitivity.BACKGROUND: Oesophageal achalasia is well-recognized but relatively rare in children, occasionally appearing as the \\\"triple A\\\" syndrome (with adrenal insufficiency and alacrima).Patients usually display the triad of achalasia, alacrima, and adrenocorticotropin (ACTH) insensitive adrenal insufficiency, though the presentation is inconsistent. Allgrove (Triple A) syndrome is a rare autosomal recessive disorder characterized by cardinal features of adrenal insufficiency due to adrenocorticotropic hormone (ACTH) resistance, achalasia, and alacrimia.Triple A syndrome, also known as Allgrove syndrome, is a rare autosomal recessive disorder characterized by three cardinal symptoms: adrenal insufficiency due to ACTH insensitivity, achalasia and alacrima.Allgrove (Triple A) syndrome is a rare autosomal recessive disorder characterized by cardinal features of adrenal insufficiency due to adrenocorticotropic hormone (ACTH) resistance, achalasia, and alacrimia\"}, {\"question\": \"List 3 features of IRVAN syndrome.\", \"ideal_answer\": [\"Idiopathic retinal vasculitis, aneurysms, and neuroretinitis is coined as IRVAN syndrome.\"], \"exact_answer\": [[\"Idiopathic retinal vasculitis\"], [\"Aneurysms\"], [\"Neuroretinitis\"]], \"context\": \"Fluorescein photodiagnosis of idiopathic retinal vasculitis, aneurysms, and neuroretinitis (IRVAN) syndrome: A case report and long-term outcome of photocoagulation therapy.Idiopathic retinal vasculitis, aneurysms, and neuroretinitis (IRVAN) syndrome is a disease characterized by multiple retinal macroaneurysms, neuroretinitis and peripheral capillary non-perfusion, leading to irreversible visual loss.[Early treatment of idiopathic vasculitis, aneurysms and neuroretinitis (IRVAN). A case report].CASE REPORT: A 55 year old woman presented with retinal vasculitis, multiple aneurysms, macular exudation and widespread retinal nonperfusion and was diagnosed with IRVAN.A 7-year-old girl with IRVAN (idiopathic retinal vasculitis, aneurysms, and neuroretinitis) syndrome was monitored for 9 years.Vision loss associated with the idiopathic retinal vasculitis, aneurysms, and neuroretinitis (IRVAN) syndrome most commonly occurs from macular edema or complications related to neovascularization. PURPOSE: To report a case of idiopathic retinal vasculitis, aneurysms and neuroretinitis (IRVAN) syndrome associated with positive perinuclear antineutrophil cytoplasmic antibody (P-ANCA).PURPOSE: We report our experience in treating 2 patients of idiopathic retinal vasculitis, aneurysm, and neuroretinitis (IRVAN) syndrome with antitumor necrosis factor agent, infliximab, who showed a very favorable response to treatmentFluorescein angiography delineated saccular aneurysms of the retinal arteriolar vasculature, and IRVAN syndrome was diagnosed.The authors describe the clinical feature of ten patients with a new syndrome characterized by the presence of retinal vasculitis, multiple macroaneurysms, neuro-retinitis, and peripheral capillary nonperfusion.The authors evaluated ten patients identified to have clinical features compatible with the syndrome of idiopathic retinal vasculitis, aneurysms and neuroretinits (IRVAN)The authors evaluated ten patients identified to have clinical features compatible with the syndrome of idiopathic retinal vasculitis, aneurysms and neuroretinits (IRVAN).\"}, {\"question\": \"Which are the Proprotein Convertase Subtilisin Kexin 9 (PCSK9) inhibitors that are FDA approved?\", \"ideal_answer\": [\"The PCSK9 inhibitors that are FDA approved are:\\n1) Alirocumab and \\n2) Evolocumab.\"], \"exact_answer\": [[\"Alirocumab\", \"Praluent\"], [\"Evolocumab\", \"Repatha\"]], \"context\": \"Inhibition of the PCSK9 protein by monoclonal antibodies results in a dramatic 40%-60% lowering of serum low-density lipoprotein cholesterol (LDL-C). Alirocumab and evolocumab have been approved by the FDA for the treatment of patients with heterozygous familial hypercholesterolemia and patients with clinical atherosclerotic cardiovascular disease) who do not achieve their LDL-C target on maximal tolerated statin treatment and dietary modification. In addition, evolocumab has been approved by the FDA for homozygous familial hypercholesterolemia.LDL cholesterol (LDLC) lowering has been revolutionized by PCSK9 inhibitors, Alirocumab (Praluent) and Evolocumab (Repatha), approved as adjuncts to maximally tolerated cholesterol lowering therapy in heterozygous (HeFH) or homozygous (HoFH) familial hypercholesterolemia, and/or clinical atherosclerotic cardiovascular disease (CVD) where LDLC lowering is insufficient.A new class of lipid-lowering drugs, inhibitors of PCSK9 has been generating impressive clinical trial data over the last several years, and alirocumab (Praluent) has become the first to be approved by the US FDA.The second FDA-approved PCSK9 inhibitor evolocumab (Repatha) appears to be similar in efficacy and safety to alirocumab (Praluent), but no comparative studies are available. Two proprotein convertase subtilisin/kexin type 9 (PCSK9) inhibitors, evolocumab and alirocumab, have recently been approved by both the Food and Drug Administration (FDA) and the European Medicines Agency (EMA) for the treatment of hypercholesterolemia.Newer agents for LDL cholesterol reduction include the cholesterol ester transfer protein inhibitors, the microsomal triglyceride transfer protein inhibitor lomitapide, the apolipoprotein B antisense oligonucleotide mipomersen and several molecules that inhibit or interfere with proprotein convertase subtilisin/kexin 9 (PCSK9).Among the various PCSK9 inhibitors, human data are available for monoclonal antibodies against PCSK9 of which the two most advanced are alirocumab (SAR236553/REGN727) and AMG 145The 2 or 4\\u2011week subcutaneous therapy with the recently approved antibodies alirocumab and evolocumab for inhibition of proprotein convertase subtilisin-kexin type 9 (PCSK9) reduces low-density lipoprotein cholesterol (LDL-C) in addition to statins and ezetimibe by 50-60\\u2009%.In 2015 the U.S. Food and Drug Administration approved the first two proprotein convertase subtilisin/kexin type 9 (PCSK9) inhibitors, alirocumab (Praluent\\u00ae; Sanofi/ Regeneron) and evolocumab (Repatha\\u00ae; Amgen), for use in patients with heterozygous and homozygous familial hypercholesterolemia and for patients intolerant of statins or those with a major risk of cardiovascular disease (CVD) but unable to lower their LDL cholesterol (LDL-C) to optimal levels with statins and ezetimibe.Very recent clinical trials have proven overwhelmingly the effectiveness and safety of PCSK9 inhibitors for lowering LDL-C. Both alirocumab and evolocumab have now been approved by the US FDA and there are some initial favorable outcomes data.Food and Drug Administration approved the first two proprotein convertase subtilisin/kexin type 9 (PCSK9) inhibitors, alirocumab (Praluent\\u00ae; Sanofi/ Regeneron) and evolocumab (Repatha\\u00ae; Amgen), for use in patients with heterozygous and homozygous familial hypercholesterolemia and for patients intolerant of statins or those with a major risk of cardiovascular disease (CVD) but unable to lower their LDL cholesterol (LDL-C) to optimal levels with statins and ezetimibe.\"}, {\"question\": \"List active ingredients of the Stribild polypill.\", \"ideal_answer\": [\"Active ingredients of Stribild are elvitegravir, cobicistat, emtricitabine and tenofovir. It is used for treatment of HIV infection.\"], \"exact_answer\": [[\"elvitegravir\"], [\"cobicistat\"], [\"emtricitabine\"], [\"tenofovir\"]], \"context\": \"PURPOSE: The purpose of this study is to assess postmarketing safety and tolerability of Stribild (elvitegravir [EVG]/cobicistat [COBI]/tenofovir disoproxil fumarate [TDF]/emtricitabine [FTC]).INTRODUCTION: Co-formulated elvitegravir, cobicistat, emtricitabine and tenofovir (EVG/COBI/FTC/TDF or Stribild\\u2122) is the latest antiretroviral tablet approved in the EU. BACKGROUND: Co-formulated elvitegravir, cobicistat, emtricitabine, and tenofovir disoproxil fumarate (EVG/COBI/FTC/TDF; Stribild(\\u00ae)) is a guideline-recommended regimen for HIV treatment-na\\u00efve patients and a switch option for virologically suppressed patients.BACKGROUND: Coformulated elvitegravir, cobicistat, emtricitabine, and tenofovir disoproxil fumarate (EVG/COBI/FTC/TDF; Stribild(\\u00ae)) is a recommended integrase inhibitor-based regimen in treatment guidelines from the US Department of Health and Human Services and the British HIV Association. Elvitegravir/cobicistat/emtricitabine/tenofovir disoproxil fumarate single-tablet regimen (Stribild\\u00ae): a review of its use in the management of HIV-1 infection in adults.A new single-tablet, fixed-dose formulation consisting of elvitegravir, an HIV-1 integrase strand transfer inhibitor (INSTI); cobicistat, a pharmacokinetic enhancer; emtricitabine, a nucleoside reverse transcriptase inhibitor; and tenofovir disoproxil fumarate (tenofovir DF), a nucleotide reverse transcriptase inhibitor (elvitegravir/cobicistat/emtricitabine/tenofovir DF 150 mg/150 mg/200 mg/300 mg; Stribild\\u00ae) is available in some countries for the once-daily treatment of HIV-1 infection in antiretroviral therapy-na\\u00efve adults. To review the clinical trials, pharmacologic characteristics, safety, and efficacy of the elvitegravir/cobicistat/emtricitabine/tenofovir single tablet formulation (Stribild).Literature searches were performed in MEDLINE (1948-September 2012) and PubMed (1966-September 2012) using the search terms GS-9137, elvitegravir, GS 9350, cobicistat, quad pill, Stribild, and integrase inhibitors.To evaluate dolutegravir and elvitegravir/cobicistat pharmacokinetics in HIV-negative volunteers up to 10 days after drug cessation.Healthy volunteers received 50 mg of dolutegravir once-daily for 10 days, then underwent a 9 day wash-out period, and then received elvitegravir/cobicistat as part of Stribild(\\u00ae) (245 mg of tenofovir, 200 mg of emtricitabine, 150 mg of elvitegravir and 150 mg of cobicistat) for 10 days.\"}, {\"question\": \"List clinical features of the IMAGe syndrome.\", \"ideal_answer\": [\"Clinical features of IMAGe syndrome include intra-uterine growth restriction, metaphyseal dysplasia, adrenal hypoplasia congenita and genital abnormalities. It is s caused by gain-of-function mutations of maternally expressed gene CDKN1C on chromosome 11p15.5.\"], \"exact_answer\": [[\"intra-uterine growth restriction\"], [\"metaphyseal dysplasia\"], [\"adrenal hypoplasia congenita\"], [\"genital abnormalities\"]], \"context\": \"Recently, our group described gain-of-function mutations in the PCNA-binding site of CDKN1C that result in an undergrowth syndrome called IMAGe Syndrome (Intrauterine Growth Restriction, Metaphyseal dysplasia, Adrenal hypoplasia, and Genital anomalies), with life-threatening consequences. IMAGe syndrome is an exceedingly rare condition first described in 1999. Components of the syndrome are intrauterine growth retardation (IUGR), metaphyseal dysplasia, congenital adrenal hypoplasia and genital anomalies. OBJECTIVE: Arboleda et al. have recently shown that IMAGe (intra-uterine growth restriction, metaphyseal dysplasia, adrenal hypoplasia congenita and genital abnormalities) syndrome is caused by gain-of-function mutations of maternally expressed gene CDKN1C on chromosome 11p15.5. Intriguing is that CDKN1C gain-of-function variations were recently found in patients with IMAGe syndrome (intrauterine growth restriction, metaphyseal dysplasia, congenital adrenal hypoplasia, and genital anomalies). \"}, {\"question\": \"Which gene mutations are predictive of response to anti-TNF therapy in Rheumatoid Arthritis patients?\", \"ideal_answer\": [\"\\u039cutations in TLR5 and TLR1 genes contribute to differential response to anti-TNF treatment in RA. Variation at FCGR2A and functionally related genes such as DHX32 and RGS12 is also associated with the response to anti-TNF therapy in rheumatoid arthritis.\"], \"exact_answer\": [[\"TLR5\"], [\"TLR1\"], [\"FCGR2A\"], [\"DHX32\"], [\"RGS12\"]], \"context\": \"Genetic Variations in Pattern Recognition Receptor Loci Are Associated with Anti-TNF Response in Patients with Rheumatoid ArthritisSix of twenty successfully genotyped polymorphisms were nominally associated with EULAR treatment response. Three of these were in weak to moderate linkage disequilibrium with polymorphisms previously reported associated with anti-TNF treatment response. TLR5(rs5744174) variant allele carriers (odds ratio(OR) = 1.7(1.1-2.5),p = 0.010,q = 0.46) and TLR1(rs4833095) homozygous variant carriers (OR = 2.8(1.1-7.4),p = 0.037,q = 0.46) had higher odds for a positive treatment response. NLRP3(rs10754558) variant allele carriers (odds ratio(OR) = 0.6(0.4-1.0),p = 0.045,q = 0.46) were more likely to have a negative treatment response. The association in TLR5(rs5744174) remained significant after correction for multiple comparisons among patients negative for RF (OR = 6.2(2.4-16.3),p = 0.0002,q = 0.024). Variation at FCGR2A and functionally related genes is associated with the response to anti-TNF therapy in rheumatoid arthritisWe aimed to validate this genetic association in a patient cohort from the Spanish population, and also to identify new genes functionally related to FCGR2A that are also associated with anti-TNF response.A total of 348 RA patients treated with an anti-TNF therapy were included and genotyped for FCGR2A polymorphism rs1081274. Using gene expression profiles from macrophages obtained from synovial fluid of RA patients, we searched for genes highly correlated with FCGR2A expression.We found a significant association between FCGR2A and the response to adalimumab (P=0.022). Analyzing the subset of anti-CCP positive RA patients (78%), we also found a significant association between FCGR2A and the response to infliximab (P=0.035). DHX32 and RGS12 were the most consistently correlated genes with FCGR2A expression in RA synovial fluid macrophages (P<0.001).We found a significant association between the genetic variation at DHX32 (rs12356233, corrected P=0.019) and a nominally significant association between RGS12 and the response to adalimumab (rs4690093, uncorrected P=0.040).In the anti-CCP positive group of patients, we also found a nominally significant association between RGS12 and the response to infliximab (rs2857859, uncorrected P=0.042).\"}, {\"question\": \"List clinical features of EEM syndrome.\", \"ideal_answer\": [\"EEM syndrome is characterized by ectodermal dysplasia, ectrodactyly and macular dystrophy.\"], \"exact_answer\": [[\"ectodermal dysplasia\"], [\"ectrodactyly\"], [\"macular dystrophy\"]], \"context\": \"Hypotrichosis with juvenile macular dystrophy (HJMD) and ectodermal dysplasia, ectrodactyly and macular dystrophy (EEM) are both caused by mutations in the CDH3 gene. Loss-of-function mutations in CDH3, which encodes P-cadherin, result in two allelic autosomal recessive disorders: hypotrichosis with juvenile macular dystrophy (HJMD) and ectodermal dysplasia, ectrodactyly, and macular dystrophy (EEM) syndromes. Recently, mutations in the P-cadherin gene (CDH3) have been shown to cause two inherited diseases in humans: hypotrichosis with juvenile macular dystrophy (HJMD) and ectodermal dysplasia, ectrodactyly, macular dystrophy (EEM syndrome).EEM syndrome is a rare condition characterised by ectodermal dysplasia, ectrodactyly and macular dystrophy. BACKGROUND: EEM syndrome is the rare association of ectodermal dysplasia, ectrodactyly, and macular dystrophy.Distinct CDH3 mutations cause ectodermal dysplasia, ectrodactyly, macular dystrophy (EEM syndrome).Ectodermal dysplasia, ectrodactyly and macular dystrophy (EEM syndrome) in siblings.Association of ectodermal dysplasia, ectrodactyly, and macular dystrophy: the EEM syndrome.Association of ectodermal dysplasia, ectrodactyly and macular dystrophy: EEM syndrome (case report).EEM syndrome is the rare association of ectodermal dysplasia, ectrodactyly, and macular dystrophy.We report a brother and sister with ectodermal dysplasia, ectrodactyly, and macular dystrophy (the EEM syndrome).Ectodermal dysplasia, ectrodactyly and macular dystrophy (EEM syndrome) in siblings.Distinct CDH3 mutations cause ectodermal dysplasia, ectrodactyly, macular dystrophy (EEM syndrome).\"}, {\"question\": \"Which are the most common methods for circular RNA detection from RNASeq?\", \"ideal_answer\": [\"The main algorithms are circRNA_finder, find_circ, CIRCexplorer, CIRI, and MapSplice.\", \"Here, we use common RNAseq datasets to scrutinize and compare the output from five different algorithms; circRNA_finder, find_circ, CIRCexplorer, CIRI, and MapSplice and evaluate the levels of bona fide and false positive circRNAs based on RNase R resistance. \", \"Here, we use common RNAseq datasets to scrutinize and compare the output from five different algorithms; circRNA_finder, find_circ, CIRCexplorer, CIRI, and MapSplice and evaluate the levels of bona fide and false positive circRNAs based on RNase R resistance.\", \"CircRNAs are novel members of the non-coding RNA family. Several pipelines have been developed to specifically identify these non-linear reads and consequently predict the landscape of circRNAs based on deep sequencing datasets. Here, we use common RNAseq datasets to scrutinize and compare the output from five different algorithms; circRNA_finder, find_circ, CIRCexplorer, CIRI, and MapSplice and evaluate the levels of bona fide and false positive circRNAs based on RNase R resistance.\", \", use common rnaseq datasets to scrutinize and compare the output from five different algorithms; circrna_finder , find_circ , circexplorer , ciri , and mapsplice and evaluate the levels of bona fide and false positive circrnas based on rnase r resistance. . \", \"here, we use common rnaseq datasets to scrutinize and compare the output from five different algorithms; circrna.finder, find.circ, circexplorer, ciri, and mapsplice and evaluate the levels of bona fide and false positive circrnas based on rnase r resistance.\"], \"exact_answer\": [[\"circRNA_finder\"], [\"find_circ\"], [\"CIRCexplorer\"], [\"CIRI\"], [\"MapSplice\"]], \"context\": \"Here, we use common RNAseq datasets to scrutinize and compare the output from five different algorithms; circRNA_finder, find_circ, CIRCexplorer, CIRI, and MapSplice and evaluate the levels of bona fide and false positive circRNAs based on RNase R resistance.\"}, {\"question\": \"Which proteins control the degradation of cryptic unstable transcripts (CUTs) in yeast?\", \"ideal_answer\": [\"Termination of cryptic unstable transcripts is directed by yeast RNA-binding proteins Nrd1 and Nab3. These cryptic unstable transcripts (CUTs) are rapidly degraded by the nuclear exosome. Key substrates for exosomal degradation include aberrant functional RNAs and cryptic unstable transcripts (CUTs). Yeast RNA binding proteins Nrd1 and Nab3 direct termination of sn/snoRNAs and recently have also been implicated in premature transcription termination of the NRD1 gene. These results suggest that transcription termination of CUTs directed by Nrd1 and Nab3 is a prerequisite for rapid degradation by the nuclear exosome. \", \"Termination of cryptic unstable transcripts is directed by yeast RNA-binding proteins Nrd1 and Nab3. Key substrates for exosomal degradation include aberrant functional RNAs and cryptic unstable transcripts (CUTs). These cryptic unstable transcripts (CUTs) are rapidly degraded by the nuclear exosome. These transcripts are targeted for degradation immediately after synthesis by the action of the Nrd1-exosome-TRAMP complexes. Cryptic unstable transcripts (CUTs) were recently described as a principal class of RNA polymerase II transcripts in Saccharomyces cerevisiae. \", \"The exosome and its nuclear specific subunit Rrp6 form a 3'-5' exonuclease complex that regulates diverse aspects of RNA biology including 3' end processing and degradation of a variety of noncoding RNAs (ncRNAs) and unstable transcripts. Known targets of the nuclear exosome include short (<1000 bp) RNAPII transcripts such as small noncoding RNAs (snRNAs), cryptic unstable transcripts (CUTs), and some stable unannotated transcripts (SUTs) that are terminated by an Nrd1, Nab3, and Sen1 (NNS) dependent mechanism. The MTREC complex physically interacts with the nuclear exosome and with various RNA-binding and RNA-processing complexes, coupling RNA processing to the RNA degradation machinery.\", \"Termination of cryptic unstable transcripts is directed by yeast RNA-binding proteins Nrd1 and Nab3 These cryptic unstable transcripts (CUTs) are rapidly degraded by the nuclear exosome. Yeast RNA binding proteins Nrd1 and Nab3 direct termination of sn/snoRNAs and recently have also been implicated in premature transcription termination of the NRD1 gene.\", \"Termination of cryptic unstable transcripts is directed by yeast RNA-binding proteins Nrd1 and Nab3. Yeast RNA binding proteins Nrd1 and Nab3 direct termination of sn/snoRNAs and recently have also been implicated in premature transcription termination of the NRD1 gene. These cryptic unstable transcripts (CUTs) are rapidly degraded by the nuclear exosome. These results suggest that transcription termination of CUTs directed by Nrd1 and Nab3 is a prerequisite for rapid degradation by the nuclear exosome. Key substrates for exosomal degradation include aberrant functional RNAs and cryptic unstable transcripts (CUTs). \", \"Termination of cryptic unstable transcripts is directed by yeast RNA-binding proteins Nrd1 and Nab3\", \"Cryptic unstable transcripts (CUTs) were recently described as a principal class of RNA polymerase II transcripts in Saccharomyces cerevisiae. These transcripts are targeted for degradation immediately after synthesis by the action of the Nrd1-exosome and Trf4/5-Air1/2-Mtr4 polyadenylation (TRAMP) complexes. The termination of cryptic unstable transcripts is directed by yeast RNA-binding proteins Nrd1 and Nab3. Known targets of the nuclear exosome include short (<1000 bp) RNAPII transcripts such as small noncoding RNAs (snRNAs), cryptic unstable transcripts (CUTs), and some stable unannotated transcripts (SUTs) that are terminated by an Nrd1, Nab3, and Sen1 (NNS) dependent mechanism. Recent work suggests Nrd1 is necessary for transcriptome surveillance, regulating promoter directionality and suppressing antisense transcription independently of, or prior to, Rrp6 activity.\"], \"exact_answer\": [[\"Rrp6\"], [\"Nrd1\"], [\"Nab3\"], [\"TRAMP complex\", \"Trf4/5-Air1/2-Mtr4 polyadenylation\"], [\"Sen1\"]], \"context\": \"Termination of cryptic unstable transcripts is directed by yeast RNA-binding proteins Nrd1 and Nab3These cryptic unstable transcripts (CUTs) are rapidly degraded by the nuclear exosome. Yeast RNA binding proteins Nrd1 and Nab3 direct termination of sn/snoRNAs and recently have also been implicated in premature transcription termination of the NRD1 gene. These results suggest that transcription termination of CUTs directed by Nrd1 and Nab3 is a prerequisite for rapid degradation by the nuclear exosome.Key substrates for exosomal degradation include aberrant functional RNAs and cryptic unstable transcripts (CUTs).Lack of TRAMP activity stabilises \\u223c 1600 CUTs in meiotic cells, which occupy 40% of the binding capacity of the nuclear cap binding complex (CBC). CBC mutants display defects in the formation of meiotic double strand breaks (DSBs), and we see similar defects in TRAMP mutants, suggesting that a key function of the nuclear exosome is to prevent saturation of the CBC complex by CUTsWe find that TRAMP mutants produce high levels of CUTs during meiosis that are undetectable in wild-type cells, showing that the nuclear exosome remains functional for CUT degradation, and we further report that the meiotic exosome complex contains Rrp6.The exosome component Rrp6 is required for RNA polymerase II termination at specific targets of the Nrd1-Nab3 pathway.Known targets of the nuclear exosome include short (<1000 bp) RNAPII transcripts such as small noncoding RNAs (snRNAs), cryptic unstable transcripts (CUTs), and some stable unannotated transcripts (SUTs) that are terminated by an Nrd1, Nab3, and Sen1 (NNS) dependent mechanismRecent work suggests Nrd1 is necessary for transcriptome surveillance, regulating promoter directionality and suppressing antisense transcription independently of, or prior to, Rrp6 activity.Cryptic unstable transcripts (CUTs) were recently described as a principal class of RNA polymerase II transcripts in Saccharomyces cerevisiae. These transcripts are targeted for degradation immediately after synthesis by the action of the Nrd1-exosome-TRAMP complexes.In Saccharomyces cerevisiae, non-coding RNAs, including cryptic unstable transcripts (CUTs), are subject to degradation by the exosome. The Trf4/5-Air1/2-Mtr4 polyadenylation (TRAMP) complex in S. cerevisiae is a nuclear exosome cofactor that recruits the exosome to degrade RNAs The MTREC complex specifically binds to CUTs, meiotic mRNAs and unspliced pre-mRNA transcripts and targets these RNAs for degradation by the nuclear exosome, while the TRAMP complex has only a minor role in this processThe eukaryotic exosome exoribonuclease Rrp6 forms a complex with Rrp47 that functions in nuclear RNA quality control mechanisms, the degradation of cryptic unstable transcripts (CUTs), and in the 3 end maturation of stable RNAsNrd1 and Nab3 colocalize to regions of the genome expressing antisense CUTs, and these transcripts require yeast nuclear exosome and TRAMP components for degradationGenome-wide studies have identified abundant small, noncoding RNAs, including small nuclear RNAs, small nucleolar RNAs (snoRNAs), cryptic unstable transcripts (CUTs), and upstream regulatory RNAs (uRNAs), that are transcribed by RNA polymerase II (pol II) and terminated by an Nrd1-dependent pathwayNrd1 and Nab3 colocalize to regions of the genome expressing antisense CUTs, and these transcripts require yeast nuclear exosome and TRAMP components for degradation.Here, by examining the overlap of stable (SUTs, stable unannotated transcripts) and unstable (CUTs, cryptic unstable transcripts) transcripts with protein-coding genes, we show that the predicted Nrd1 and Nab3-binding site sequences occur at differing frequencies.Examples are the consensus binding site sequences of the RNA-binding proteins Nrd1 and Nab3 that target non-coding transcripts for degradation.Termination of cryptic unstable transcripts is directed by yeast RNA-binding proteins Nrd1 and Nab3.Some are designated Cryptic Unstable Transcripts (CUTs) because they are terminated by the Nrd1-Nab3-Sen1 pathway and then rapidly degraded by the nuclear exosome.\"}, {\"question\": \"In which fields of DNA sequencing are Bloom filters applied?\", \"ideal_answer\": [\"A novel algorithm, fast and accurate classification of sequences (FACSs), is introduced that can accurately and rapidly classify sequences as belonging or not belonging to a reference sequence.  Classification of DNA sequences using Bloom filters Lighter is a fast, memory-efficient tool for correcting sequencing errors.\", \"A novel algorithm, fast and accurate classification of sequences (FACSs), is introduced that can accurately and rapidly classify sequences as belonging or not belonging to a reference sequence. Classification of DNA sequences using Bloom filters. Further, we note that Bloom filters would be suitable to implicitly store spaced seeds and be tolerant to sequencing errors. Lighter avoids counting k-mers. Fast lossless compression via cascading Bloom filters. \", \"A novel algorithm, fast and accurate classification of sequences (FACSs), is introduced that can accurately and rapidly classify sequences as belonging or not belonging to a reference sequence. Classification of DNA sequences using Bloom filters. Instead, it uses a pair of Bloom filters, one holding a sample of the input k-mers and the other holding k-mers likely to be correct. Fast lossless compression via cascading Bloom filters. Lighter is a fast, memory-efficient tool for correcting sequencing errors. \", \"A novel algorithm, fast and accurate classification of sequences (FACSs), is introduced that can accurately and rapidly classify sequences as belonging or not belonging to a reference sequence. Classification of DNA sequences using Bloom filters. Instead, it uses a pair of Bloom filters, one holding a sample of the input k-mers and the other holding k-mers likely to be correct. Lighter is a fast, memory-efficient tool for correcting sequencing errors. Lighter avoids counting k-mers. \", \"Bloom Filters (BFs) reduce the memory footprint required to store millions of k-mers while allowing for fast set containment queries, at the cost of a low false positive rate (FPR). Cascading Bloom filters have been used to improve the memory usage and speed of DNA sequence compression.\", \"Further, we note that Bloom filters would be suitable to implicitly store spaced seeds and be tolerant to sequencing errors.  It uses a pair of cache oblivious Bloom filters, one holding a uniform sample of [Formula: see text]-spaced sequenced [Formula: see text]-mers and the other holding [Formula: see text]-mers classified as likely correct, using a simple statistical test.\"], \"exact_answer\": [[\"storage\"], [\"compression\"], [\"alignment-free comparisons\"]], \"context\": \"A novel algorithm, fast and accurate classification of sequences (FACSs), is introduced that can accurately and rapidly classify sequences as belonging or not belonging to a reference sequence. Classification of DNA sequences using Bloom filtersLighter is a fast, memory-efficient tool for correcting sequencing errors. Lighter avoids counting k-mers. Instead, it uses a pair of Bloom filters, one holding a sample of the input k-mers and the other holding k-mers likely to be correct.Fast lossless compression via cascading Bloom filtersIn contrast to reference-based methods that first align reads to the genome, we hash all reads into Bloom filters to encode, and decode by querying the same Bloom filters using read-length subsequences of the reference genome. Further compression is achieved by using a cascade of such filtersUsing cascading Bloom filters to improve the memory usage for de Brujin graphsIn this work, we show how to reduce the memory required by the data structure of Chikhi and Rizk (WABI'12) that represents de Brujin graphs using Bloom filters.We introduce a novel hash-based method for constructing the string graph. We use incremental hashing, and specifically a modification of the Karp-Rabin fingerprint, and Bloom filtersFurther, we note that Bloom filters would be suitable to implicitly store spaced seeds and be tolerant to sequencing errors. It uses a pair of cache oblivious Bloom filters, one holding a uniform sample of [Formula: see text]-spaced sequenced [Formula: see text]-mers and the other holding [Formula: see text]-mers classified as likely correct, using a simple statistical test.Improving Bloom Filter Performance on Sequence Data Using k-mer Bloom FiltersBFs reduce the memory footprint required to store millions of k-mers while allowing for fast set containment queries, at the cost of a low false positive rate (FPR).We consider several variants of such k-mer Bloom filters (kBFs), derive theoretical upper bounds for their FPR, and discuss their range of applications and limitations.Fast lossless compression via cascading Bloom filters.Fast lossless compression via cascading Bloom filters.\"}, {\"question\": \"Which 2 medications are included in the Qsymia pill?\", \"ideal_answer\": [\"Qsymia pill includes phentermine and topiramate. It is used for treatment of obesity.\"], \"exact_answer\": [[\"phentermine\"], [\"topiramate\"]], \"context\": \"OBJECTIVE: To quantify the incremental cost-effectiveness of Qsymia (phentermine and topiramate extended-release) for health-related quality of life improvements.The intervention was diet and lifestyle advice plus the recommended dose of Qsymia (phentermine 7.5\\u00a0mg plus topiramate 46.0\\u00a0mg) vs. control, which included diet and lifestyle advice plus placebo.These include combinations (at low dose) of existing drugs, e.g., bupropion + naltrexone (Contrave), phentermine + topiramate (Qsymia), higher doses of existing drugs licensed for other indications (liraglutide, 3 mg), and new entities (lorcaserin).RECENT FINDINGS: Two new antiobesity drugs - naltrexone/bupropion (Contrave) and liraglutide (Saxenda) - were approved by the US Food and Drug Administration in 2014 and join four other approved obesity medications, including phentermine/topiramate XR (Qsymia) and lorcaserin (Belviq), to form the largest number of medications available for the treatment of obesity.Adding to the current available pharmacotherapies for obesity, the Food and Drug Administration has recently approved 2 new combination medications known as lorcaserin (Belviq) and phentermine-topiramate (Qsymia).After a long period of failure in development, two new medications (phentermine/topiramate ER - Qsymia\\u2122 and lorcaserin - Belviq\\u00ae) have been approved by the US Food and Drug Administration for long-term weight management in persons with obesity (BMI \\u2265 30 kg/m(2)) or in overweight persons (BMI \\u2265 27 kg/m(2)) with comorbidities.The intervention was diet and lifestyle advice plus the recommended dose of Qsymia (phentermine 7.5\\u00a0mg plus topiramate 46.0\\u00a0mg) vs. control, which included diet and lifestyle advice plus placeboCurrent antiobesity medications and pharmacological strategies will be reviewed.Two new antiobesity drugs - naltrexone/bupropion (Contrave) and liraglutide (Saxenda) - were approved by the US Food and Drug Administration in 2014 and join four other approved obesity medications, including phentermine/topiramate XR (Qsymia) and lorcaserin (Belviq), to form the largest number of medications available for the treatment of obesity.Almost a decade after the Food and Drug Administration approved the first weight loss medication, it recently approved two novel anti-obesity drugs Belviq (lorcaserin) and Qsymia (topiramate and phentermine), thus signalling the beginning of a new era in the pharmacotherapy of obesity.Obesity is a public health crisis affecting approximately more than 33% of Americans and costing the healthcare system more than $190 billion annually.To review the 2 new drugs that were recently approved by the US Food and Drug Administration (FDA) for the treatment of obesity, lorcaserin HCl (Belviq) and phentermine/topiramate (Qsymia) and their potential impact on the treatment of obese patients.Lifestyle modification is the first and mainstay treatment for obesity.Adding to the current available pharmacotherapies for obesity, the Food and Drug Administration has recently approved 2 new combination medications known as lorcaserin (Belviq) and phentermine-topiramate (Qsymia).Two new antiobesity drugs - naltrexone/bupropion (Contrave) and liraglutide (Saxenda) - were approved by the US Food and Drug Administration in 2014 and join four other approved obesity medications, including phentermine/topiramate XR (Qsymia) and lorcaserin (Belviq), to form the largest number of medications available for the treatment of obesity.To review the 2 new drugs that were recently approved by the US Food and Drug Administration (FDA) for the treatment of obesity, lorcaserin HCl (Belviq) and phentermine/topiramate (Qsymia) and their potential impact on the treatment of obese patients.\"}, {\"question\": \"Which markers are screened with the triple test for the detection of syndromes in fetus?\", \"ideal_answer\": [\"The markers that are screened with the triple test for the detection of syndromes in fetus are:\\n1) alpha-fetoprotein (AFP), \\n2) beta-chorionic gonadotrophin (beta-CG) and \\n3) unconjugated oestriol (uE3).\"], \"exact_answer\": [[\"alpha-fetoprotein\", \"AFP\", \"\\u03b1-fetoprotein\"], [\"beta-chorionic gonadotrophin\", \"beta-CG\"], [\"unconjugated oestriol\", \"uE3\"]], \"context\": \"Our study aimed at calculation and validation of the triple marker medians used in screening Egyptian females as well as to recommend programme conventions to unify screening in this country.Chorionic gonadotropin (CG), \\u03b1-fetoprotein (AFP) and unconjugated oestriol (uE3) were measured on Siemens Immulite analyzer.Triple test screening for Down syndrome: an Egyptian-tailored study.The purpose of this article was to evaluate the reliability of maternal serum triple marker screening of alpha-fetoprotein, human chorionic gonadotropin, and unconjugated estriol for the prenatal diagnosis of fetal chromosomal abnormalities in Turkish pregnant women.Second trimester triple test is an effective screening tool for detecting fetal Down syndrome in Turkish women.The aim of this prospective study was to compare triple test screening (alpha-fetoprotein, beta-chorionic gonadotrophin and unconjugated oestriol) with amniocentesis in the detection of fetuses with Down's syndrome in women of 35 years of age or more. The aim of this prospective study was to compare triple test screening (alpha-fetoprotein, beta-chorionic gonadotrophin and unconjugated oestriol) with amniocentesis in the detection of fetuses with Down's syndrome in women of 35 years of age or more.None of the other five markers added more than 2% detection for the same false-positive rate.The performance of screening using maternal age and serum-free beta-hCG and PAPP-A at 10 weeks of pregnancy was better than the double test (alpha-fetoprotein and hCG with maternal age) and similar to the triple test (alpha-fetoprotein, unconjugated oestriol and hCG with maternal age) at 15-22 weeks.To make recommendations to physicians providing prenatal care on (1) whether prenatal screening for and diagnosis of Down syndrome (DS) is advisable and (2) alternative screening and diagnosis manoeuvres.\\\"Triple-marker\\\" screening of maternal serum levels of alpha-fetoprotein, human chorionic gonadotropin and unconjugated estriol; fetal ultrasonographic examination; amniocentesis; and chorionic villus sampling (CVS).Accuracy of detection of DS in fetuses, and risks to the mother, including psychologic distress, and to the fetus from the screening and diagnostic interventions.A MEDLINE search for relevant aA decision analytic model was designed for women at increased risk for a DS fetus due to either advanced maternal age (AMA) or a positive expanded maternal serum alpha fetoprotein (MSAFP) screening test, also known as a triple screen (+triple) test.These tests have a limited detection rate for Down syndrome: approximately 40% for hCG or free beta-subunit alone, approximately 60% for the triple screen test, and approximately 70% for the quadruple marker test, all at 5%, or a relatively high, false-positive rate.Serum human chorionic gonadotropin (hCG) and hCG free beta-subunit tests are used in combination with unconjugated estriol and alpha-fetoprotein in the triple screen test, and with the addition of inhibin-A in the quadruple marker test for detecting Down syndrome in the second trimester of pregnancy.Initial and revised screen-positive rates and detection rates were reviewed for women undergoing triple-marker testing (maternal serum alpha-fetoprotein, human chorionic gonadotropin, and unconjugated estriol).\"}, {\"question\": \"Which factors drive replisome disassembly during DNA replication termination and mitosis?\", \"ideal_answer\": [\"CUL-2LRR-1 and UBXN-3.\"], \"exact_answer\": [[\"CUL-2LRR-1\"], [\"UBXN-3\"]], \"context\": \"Using Caenorhabditis elegans early embryos and Xenopus laevis egg extracts, we show that the E3 ligase CUL-2LRR-1associates with the replisome and drives ubiquitylation and disassembly of CMG, together with the CDC-48 cofactors UFD-1 and NPL-4. Removal of CMG from chromatin in frog egg extracts requires CUL2 neddylation, and our data identify chromatin recruitment of CUL2LRR1as a key regulated step during DNA replication termination. Interestingly, however, CMG persists on chromatin until prophase in worms that lack CUL-2LRR-1, but is then removed by a mitotic pathway that requires the CDC-48 cofactor UBXN-3, orthologous to the human tumour suppressor FAF1.CUL-2 and UBXN-3 drive replisome disassembly during DNA replication termination and\\u00a0mitosis.\"}, {\"question\": \"Which two cotransporters are inhibited by sotagliflozin?\", \"ideal_answer\": [\"Sotagliflozin works by inhibiting sodium-glucose cotransporter 1 (SGLT1) and sodium-glucose cotransporter 2 (SGLT2). It is used for treatment of diabetes.\"], \"exact_answer\": [[\"sodium-glucose cotransporter 1\"], [\"sodium-glucose cotransporter 2\"]], \"context\": \"The present study used molecular docking by Autodock4.2 using our \\\"Click-By-Click\\\"-protocol, Ligplot1.4.3 and \\\"change in accessible surface area (\\u0394ASA)-calculations\\\" to investigate the binding of two investigational anti-diabetic drugs, Ertugliflozin and Sotagliflozin to an established target (SGLT2) and a research target (human brain AChE).SGLT2 inhibitors such as canagliflozin and sotagliflozin (a SGLT1/SGLT2 dual inhibitor) also have a mild or moderate intestinal and renal SGLT1 inhibitory effect because of their relatively weak selectivity for SGLT2 over SGLT1.Hydrogen-bonded structures and interaction energies in two forms of the SGLT-2 inhibitor sotagliflozin.We evaluated the safety and efficacy of sotagliflozin, an oral inhibitor of sodium-glucose cotransporters 1 and 2, in combination with insulin treatment in patients with type 1 diabetes. Phase 2 studies with the SGLT2 inhibitor empagliflozin and the dual SGLT1 and SGLT2 inhibitor sotagliflozin, which acts in the gut and the kidney, have demonstrated reductions in A1C, weight, and glucose variability without an increased incidence of hypoglycemia.Sotagliflozin may meet this need, because this compound lowers blood glucose through the insulin-independent mechanisms of inhibiting kidney SGLT2 and intestinal SGLT1. AREAS COVERED: Herein, the authors focus specifically on sotagliflozin (LX4211), the first-in-class dual SGLT1/SGLT2 inhibitor.Despite a weaker effect on UGE than selective SGLT2 inhibitors, sotagliflozin is as effective as SGLT2 inhibitors on HbA1C reduction, with a similar safety profile in short-term studies. Therefore, sotagliflozin (LX4211) has been developed as a dual inhibitor of SGLT1 and SGLT2. Sotagliflozin, a Dual SGLT1 and SGLT2 Inhibitor, as Adjunct Therapy to Insulin in Type 1 Diabetes.OBJECTIVE: To assess the safety and efficacy of dual sodium-glucose cotransporter (SGLT) 1 and SGLT2 inhibition with sotagliflozin as adjunct therapy to insulin in type 1 diabetes.RESEARCH DESIGN AND METHODS: We treated 33 patients with sotagliflozin, an oral dual SGLT1 and SGLT2 inhibitor, or placebo in a randomized, double-blind trial assessing safety, insulin dose, glycemic control, and other metabolic parameters over 29 days of treatment.CONCLUSIONS: As adjunct to insulin, dual SGLT1 and SGLT2 inhibition with sotagliflozin improved glycemic control and the CGM profile with bolus insulin dose reduction, weight loss, and no increased hypoglycemia in type 1 diabetes.We evaluated the safety and efficacy of sotagliflozin, an oral inhibitor of sodium-glucose cotransporters 1 and 2, in combination with insulin treatment in patients with type 1 diabetes.Despite a weaker effect on UGE than selective SGLT2 inhibitors, sotagliflozin is as effective as SGLT2 inhibitors on HbA1C reduction, with a similar safety profile in short-term studies.OBJECTIVE To assess the safety and efficacy of dual sodium-glucose cotransporter (SGLT) 1 and SGLT2 inhibition with sotagliflozin as adjunct therapy to insulin in type 1 diabetes.Therefore, sotagliflozin (LX4211) has been developed as a dual inhibitor of SGLT1 and SGLT2.CONCLUSIONS As adjunct to insulin, dual SGLT1 and SGLT2 inhibition with sotagliflozin improved glycemic control and the CGM profile with bolus insulin dose reduction, weight loss, and no increased hypoglycemia in type 1 diabetes.AREAS COVERED Herein, the authors focus specifically on sotagliflozin (LX4211), the first-in-class dual SGLT1/SGLT2 inhibitor.To assess the safety and efficacy of dual sodium-glucose cotransporter (SGLT) 1 and SGLT2 inhibition with sotagliflozin as adjunct therapy to insulin in type 1 diabetes.\"}, {\"question\": \"List the two most important hematological features of the Evans syndrome\", \"ideal_answer\": [\"Evans syndrome is a rare autoimmune disorder, which is characterized by immune thrombocytopenia and autoimmune hemolytic anemia.\"], \"exact_answer\": [[\"immune thrombocytopenia\"], [\"autoimmune hemolytic anemia\"]], \"context\": \"Evans syndrome (ES) was defined by the combination of immune thrombocytopenia (platelet count<100,000/mm3) and AIHA.Thrombocytopenia was only found in this cluster (36% of cases, manifested as Immune Thrombocytopenia (ITP) or Evans syndrome).BACKGROUND: Autoimmune cytopenias are characterized by immune-mediated destruction of hematopoietic cell lines with immune thrombocytopenia (ITP) affecting platelets and Evans syndrome (ES) affecting platelets and red blood cells. Primary Evans syndrome (ES) is defined by the concurrent or sequential occurrence of immune thrombocytopenia and autoimmune hemolytic anemia in the absence of an underlying etiology. Evans syndrome (ES) is a rare autoimmune disorder, which is characterized by immune thrombocytopenia and autoimmune hemolytic anemia.BACKGROUND: Evans syndrome (ES) is a rare immune disorder in children, manifested by simultaneous or sequential autoimmune cytopenias (ACs) of unknown cause and having a chronic course with periods of exacerbation and remission. Evans syndrome (ES) is a rare autoimmune disorder whose exact pathophysiology is unknown. It is characterized by the simultaneous or subsequent development of autoimmune hemolytic anemia (AIHA) and immune thrombocytopenia (ITP). Evans' syndrome is an unusual illness of autoimmune etiology, characterized by thrombocytopenia and hemolytic anemia.Evans syndrome is an uncommon condition characterised by simultaneous or sequential development of immune thrombocytopenia (ITP) and autoimmune haemolytic anaemia (AIHA) with a positive direct antiglobulin test (DAT) in the absence of a known underlying aetiology.Evans' syndrome is an uncommon condition defined by the combination (either simultaneously or sequentially) of immune thrombocytopenia purpura and autoimmune hemolytic anemia with a positive direct antiglobulin test in the absence of known underlying etiology.Evans syndrome is a very rare hematologic autoimmune disease, characterized by a direct Coombs' positive hemolytic anemia and immune thrombocytopenic purpura without a known underlying etiology.Evans syndrome usually is complicated by hemolytic or thrombocytopenic symptoms.The selection criteria were Evans' syndrome in pregnancy; autoimmune haemolytic anaemia; immune thrombocytopenia.Evans' syndrome is an unusual illness of autoimmune etiology, characterized by thrombocytopenia and hemolytic anemia.Evans syndrome refers to a hematological autoimmune disorder with autoimmune hemolytic anemia accompanied by immune thrombocytopenia.Evans syndrome is a very rare hematologic autoimmune disease, characterized by a direct Coombs' positive hemolytic anemia and immune thrombocytopenic purpura without a known underlying etiology.\"}, {\"question\": \"List the four most important interferonopathies\", \"ideal_answer\": [\"Aicardi-Gouti\\u00e8res syndrome\\nchilblain lupus\\nubiquitin specific peptidase 18 (USP18)-deficiency\\nSingleton-Merten syndrome\", \"Inappropriate upregulation of type I IFN signaling and interferon-stimulated gene expression have been linked to several CNS diseases termed \\\"interferonopathies\\\" including Aicardi-Goutieres syndrome, ubiquitin specific peptidase 18 (USP18)-deficiency, haploinsufficiency in A20, otulipenia, familial chiblain lupus.\"], \"exact_answer\": [[\"Aicardi-Gouti\\u00e8res syndrome\"], [\"chilblain lupus\"], [\"ubiquitin specific peptidase 18 (USP18)-deficiency\"], [\"Singleton-Merten syndrome\"]], \"context\": \"As defined, the type I interferonopathies represent discrete examples of a disturbance of the homeostatic control of this system caused by Mendelian mutations.Familial chilblain lupus belongs to the group of type I interferonopathies and is characterized by typical skin manifestations and acral ischaemia.interferonopathies such as Aicardi-Gouti\\u00e8res syndrome.Inappropriate upregulation of type I IFN signaling and interferon-stimulated gene expression have been linked to several CNS diseases termed \\\"interferonopathies\\\" including Aicardi-Goutieres syndrome and ubiquitin specific peptidase 18 (USP18)-deficiency. interferonopathies (Singleton-Merten syndrome)Aicardi-Gouti\\u00e8res syndrome (AGS) is an inflammatory disorder belonging to the recently characterized group of type I interferonopathies.Inappropriate upregulation of type I IFN signaling and interferon-stimulated gene expression have been linked to several CNS diseases termed \\\"interferonopathies\\\" including Aicardi-Goutieres syndrome and ubiquitin specific peptidase 18 (USP18)-deficiency. Then, major players driving autoinflammation, such as type-1 interferonopathies (including the recently described haploinsuffiency in A20 and otulipenia), TNF-associated periodic syndromes, defects in ubiquitination, and SAID with overlapping features of autoimmunity or immunodeficiency. Familial chilblain lupus belongs to the group of type I interferonopathies and is characterized by typical skin manifestations and acral ischaemia. \"}, {\"question\": \"List the partners of budding yeast Cdc48 that are important for disassembly of ubiquitylated CMG helicase at the end of chromosome replication\", \"ideal_answer\": [\"The ubiquitin-binding Ufd1-Npl4 complex recruits Cdc48 to ubiquitylated CMG helicase at the end of chromosome replication.\"], \"exact_answer\": [[\"Ufd1\"], [\"Npl4\"]], \"context\": \"Ufd1-Npl4 Recruit Cdc48 for Disassembly of Ubiquitylated CMG Helicase at the End of Chromosome Replication.We demonstrate that the ubiquitin-binding Ufd1-Npl4 complex recruits Cdc48 to ubiquitylated CMG.Mutation of K29 abrogates in\\u00a0vitro recruitment of Ufd1-Npl4-Cdc48 to the CMG helicase, supporting a model whereby Ufd1-Npl4 recruits Cdc48 to ubiquitylated CMG at the end of chromosome replication, thereby driving the disassembly reaction.Mutation of K29 abrogates in\\u00a0vitro recruitment of Ufd1-Npl4-Cdc48 to the CMG helicase, supporting a model whereby Ufd1-Npl4 recruits Cdc48 to ubiquitylated CMG at the end of chromosome replication, thereby driving the disassembly reaction.\"}, {\"question\": \"Which are the main manifestations of Ohdo syndrome?\", \"ideal_answer\": [\"Severe ID, absent or deficient language, skeletal manifestations including bilateral patella dislocations.\"], \"exact_answer\": [[\"intellectual disability\", \"ID\"], [\"deficient language\"], [\"bilateral patella dislocations\"], [\"skeletal manifestations\"]], \"context\": \" This finding together with the phenotypic analogies shared with the carriers of c.5898dupC mutation suggests the existence of a fourth MED12-related disorder, characterized by severe ID, absent or deficient language and, milder, clinical manifestation in heterozygotes.Skeletal manifestations in Ohdo syndrome: a case with bilateral patella dislocations.We report a new case of Ohdo syndrome with bilateral patella dislocations where surgical intervention has been indicated. A review of the skeletal manifestations reported in the literature on Ohdo syndrome reveals that joint laxity and skeletal deformities are important aspects of the phenotype.We confirm the existence of this condition that, although similar to Ohdo syndrome, seems to be an independent clinical entity. We report a new case of Ohdo syndrome with bilateral patella dislocations where surgical intervention has been indicated.Skeletal manifestations in Ohdo syndrome: a case with bilateral patella dislocations.This finding together with the phenotypic analogies shared with the carriers of c.5898dupC mutation suggests the existence of a fourth MED12-related disorder, characterized by severe ID, absent or deficient language and, milder, clinical manifestation in heterozygotes.\"}, {\"question\": \"Which are the best methods for the prediction of circular RNA (circRNA)?\", \"ideal_answer\": [\"A circRNA prediction software for plants . Circular RNA profile in gliomas revealed by identification tool UROBORUS. Numerous algorithms that are used to detect genome-wide circRNA expression from RNA sequencing  data have been developed in the past few years, but there is little overlap in their predictions and no clear gold-standard method to assess the accuracy of these algorithms. MiARma-Seq: a comprehensive tool for miRNA, mRNA and circRNA analysis. Here, we use common RNAseq datasets to scrutinize and compare the output from five different algorithms; circRNA_finder, find_circ, CIRCexplorer, CIRI, and MapSplice and evaluate the levels of bona fide and false positive circRNAs based on RNase R resistance. \", \"Numerous algorithms that are used to detect genome-wide circRNA expression from RNA sequencing (RNA-seq) data have been developed in the past few years, but there is little overlap in their predictions and no clear gold-standard method to assess the accuracy of these algorithms. We developed the software tandem, DCC and CircTest DCC uses output from the STAR read mapper to systematically detect back-splice junctions in next-generation sequencing data. A circRNA prediction software for plants (termed PcircRNA_finder) was developed that is more sensitive in detecting circRNAs than other frequently used programs (such as find_circ and CIRCexplorer)\", \"Numerous algorithms that are used to detect genome-wide circRNA expression from RNA sequencing (RNA-seq) data have been developed in the past few years, but there is little overlap in their predictions and no clear gold-standard method to assess the accuracy of these algorithms. We developed the software tandem, DCC and CircTest DCC uses output from the STAR read mapper to systematically detect back-splice junctions in next-generation sequencing data. Circular RNA profile in gliomas revealed by identification tool UROBORUS.  Here, we use common RNAseq datasets to scrutinize and compare the output from five different algorithms; circRNA_finder, find_circ, CIRCexplorer, CIRI, and MapSplice and evaluate the levels of bona fide and false positive circRNAs based on RNase R resistance.\", \"PcircRNAfinder : a software for circRNA prediction in plants. A circRNA expression from RNA sequencing  here, we use common RNAseq datasets to scrutinize and compare the output from five different algorithms; circRNAfinder, findcirc, CIRCexplorer, CIRI, and MapSplice and evaluate the levels of bona fide and false positive circRNAs based on RNase R resistance. Numerous algorithms that are used to detect genome-wide circRNA prediction software for plants  was developed in the past few years, but there is little overlap in their predictions and no clear gold-standard method to assess the accuracy of these algorithms. We developed the software tandem, DCC and CircTest DCC uses output from the STAR read mapper to systematically detect circRNAs in total RNA-seq data. MiARma-Seq : a comprehensive tool for miRNA, mRNA and circRNA analysis. Circular RNA profile in gliomas revealed by identification tool UROBORUS. \", \"The best methods for the prediction of circular RNA are: circRNA_finder, find_circ, CIRCexplorer, CIRI, MapSplice, UROBORUS, Circ TEST DCC and miARma-Seq. \", \"Numerous algorithms that are used to detect genome-wide circRNA expression from RNA sequencing (RNA-seq) data have been developed in the past few years, but there is little overlap in their predictions and no clear gold-standard method to assess the accuracy of these algorithms. We developed the software tandem, DCC and CircTest DCC uses output from the STAR read mapper to systematically detect back-splice junctions in next-generation sequencing data. Circular RNA profile in gliomas revealed by identification tool UROBORUS.\"], \"exact_answer\": [[\"circRNA_finder\"], [\"find_circ\"], [\"CIRCexplorer\"], [\"CIRI\"], [\"MapSplice\"], [\"UROBORUS\"], [\"Circ TEST DCC\"], [\"miARma-Seq\"]], \"context\": \"Numerous algorithms that are used to detect genome-wide circRNA expression from RNA sequencing (RNA-seq) data have been developed in the past few years, but there is little overlap in their predictions and no clear gold-standard method to assess the accuracy of these algorithms.We developed the software tandem, DCC and CircTest DCC uses output from the STAR read mapper to systematically detect back-splice junctions in next-generation sequencing data.Circular RNA profile in gliomas revealed by identification tool UROBORUS.In this work, we have developed a computational pipeline named UROBORUS to detect circRNAs in total RNA-seq data.  Here, we use common RNAseq datasets to scrutinize and compare the output from five different algorithms; circRNA_finder, find_circ, CIRCexplorer, CIRI, and MapSplice and evaluate the levels of bona fide and false positive circRNAs based on RNase R resistance.miARma-Seq: a comprehensive tool for miRNA, mRNA and circRNA analysis.PcircRNA_finder: a software for circRNA prediction in plants.A circRNA prediction software for plants (termed PcircRNA_finder) was developed that is more sensitive in detecting circRNAs than other frequently used programs (such as find_circ and CIRCexplorer)Recently, circular RNA (circRNA), a class of endogenous non-protein coding RNAs that contain a circular loop, was found to exhibit multiple biological effects.RESULTS A circRNA prediction software for plants (termed PcircRNA_finder) was developed that is more sensitive in detecting circRNAs than other frequently used programs (such as find_circ and CIRCexplorer), Based on analysis of simulated and real rRNA-/RNAase R RNA-Seq data from Arabidopsis thaliana and rice PcircRNA_finder provides a more comprehensive sensitive, precise prediction method for plants circRNAs.\"}, {\"question\": \"Which tendons are affected in the Dequervain's tenosynovitis?\", \"ideal_answer\": [\"DeQuervain's tenosynovitis is a common cause of radial-sided wrist pain. Symptoms result from a narrow first dorsal compartment and associated tendinosis of the enclosed extensor pollicis brevis and/or abductor pollicis longus.\"], \"exact_answer\": [[\"extensor pollicis brevis\"], [\"abductor pollicis longus\"]], \"context\": \"DeQuervain's tenosynovitis is a common cause of radial-sided wrist pain. Symptoms result from a narrow first dorsal compartment and associated tendinosis of the enclosed extensor pollicis brevis and/or abductor pollicis longus (APL). DeQuervain's disease of the first dorsal compartment of the wrist, is a common wrist pathology, pain results from resisted gliding of the abductor pollicis longus and the extensor pollicis brevis tendon in the fibroosseous canal. DeQuervain tenosynovitis, which involves the abductor pollicis longus and extensor pollicis brevis tendons, is much more common in women than men and is due to repetitive movements of the hand such as grasping and twisting.DeQuervain tenosynovitis, which involves the abductor pollicis longus and extensor pollicis brevis tendons, is much more common in women than men and is due to repetitive movements of the hand such as grasping and twisting.\"}, {\"question\": \"Which proteins are controlling sterol metabolism in S. cerevisiae?\", \"ideal_answer\": [\"The yeast genome encodes seven oxysterol binding protein homologs, Osh1p-Osh7p, which have been implicated in regulating intracellular lipid and vesicular transport The yeast genome encodes seven oxysterol binding protein homologs, Osh1p-Osh7p, which have been implicated in regulating intracellular lipid and vesicular transport Upc2p, a transcription factor of the zinc cluster family, is an important regulator of sterol biosynthesis and azole drug resistance in Candida albicans\", \"UCP2 and its major targets ERG11, ERG2, NCP1 as well as Osh1p-Osh7p\", \"Sterol metabolism and ERG2 gene regulation in the yeast Saccharomyces cerevisiae. Upc2p, a transcription factor of the zinc cluster family, is an important regulator of sterol biosynthesis and azole drug resistance in Candida albicans. The yeast genome encodes seven oxysterol binding protein homologs, Osh1p-Osh7p, which have been implicated in regulating intracellular lipid and vesicular transport. Northern blot analysis showed that increased binding correlates with increased expression for the analyzed Upc2p targets . Taken together, our results indicate that Upc2p is a key regulator of ergosterol metabolism. \", \"The yeast genome encodes seven oxysterol binding protein homologs, Osh1p-Osh7p, which have been implicated in regulating intracellular lipid and vesicular transport northern blot analysis showed that increased binding correlates with increased expression for the analyzed Upc2p included 12 genes involved in ergosterol biosynthesis  to better understand Upc2p function in C. albicans, we used genomewide location profiling to identify the transcriptional targets of Upc2p in vivo. Upc2p, a transcription factor of the zinc cluster family, is an important regulator of sterol biosynthesis and azole drug resistance in Candida albicans sterol metabolism and ERG2 gene regulation in the yeast Saccharomyces cerevisiae. They also suggest that Upc2p may contribute to azole resistance by regulating the expression of drug efflux pump-encoding genes in addition to ergosterol metabolism. Overrepresented functional groups of genes whose promoters were bound by Upc2p is a key regulator of ergosterol metabolism. We have taken advantage of this property to study the regulation of the Delta8-Delta7-sterol isomerase-encoding ERG2 gene in an ergosterol auxotrophic mutant devoid of squalene-synthase activity. Ergosterol starvation leads to an 8-16-fold increase in ERG2 gene expression. \", \"Sterol metabolism and ERG2 gene regulation in the yeast Saccharomyces cerevisiae. Upc2p, a transcription factor of the zinc cluster family, is an important regulator of sterol biosynthesis and azole drug resistance in Candida albicans. The yeast genome encodes seven oxysterol binding protein homologs, Osh1p-Osh7p, which have been implicated in regulating intracellular lipid and vesicular transport. Northern blot analysis showed that increased binding correlates with increased expression for the analyzed Upc2p targets . Taken together, our results indicate that Upc2p is a key regulator of ergosterol metabolism. To better understand Upc2p function in C. albicans, we used genomewide location profiling to identify the transcriptional targets of Upc2p in vivo. Overrepresented functional groups of genes whose promoters were bound by Upc2p included 12 genes involved in ergosterol biosynthesis . They also suggest that Upc2p may contribute to azole resistance by regulating the expression of drug efflux pump-encoding genes in addition to ergosterol biosynthesis genes. We have taken advantage of this property to study the regulation of the Delta8-Delta7-sterol isomerase-encoding ERG2 gene in an ergosterol auxotrophic mutant devoid of squalene-synthase activity. \", \"The yeast genome encodes seven oxysterol binding protein homologs, Osh1p-Osh7p, which have been implicated in regulating intracellular lipid and vesicular transport northern blot analysis showed that increased binding correlates with increased expression for the analyzed Upc2p targets ( ERG11, MDR1, CDR1, YOR1, SUT1, SMF12, and CBP1 ).Taken together, our results indicate that upc2p is a key regulator of ergosterol metabolism.Overrepresented functional groups of genes whose promoters were bound by upc2p included 12 genes involved in ergosterol biosynthesis ( ncp1, erg11, erg2, and others ) to better understand upc2p function in c. albicans, we used genomewide location profiling to identify the transcriptional targets of upc2p in vivo.Upc2p, a transcription factor of the zinc cluster family, is an important regulator of sterol biosynthesis and azole drug resistance in candida albicans sterol metabolism and erg2 gene regulation in the yeast saccharomyces cerevisiae.We have taken advantage of this property to study the regulation of the delta8-delta7-sterol isomerase-encoding erg2 gene expression.Ergosterol starvation leads to an 8-16-fold increase in ERG2 gene expression.\"], \"exact_answer\": [[\"ERG2\"], [\"UCP2\"], [\"Osh1p\"], [\"Osh7p\"], [\"ERG11\"], [\"NCP1\"]], \"context\": \"We have taken advantage of this property to study the regulation of the Delta8-Delta7-sterol isomerase-encoding ERG2 gene in an ergosterol auxotrophic mutant devoid of squalene-synthase activity. Ergosterol starvation leads to an 8-16-fold increase in ERG2 gene expression.The yeast genome encodes seven oxysterol binding protein homologs, Osh1p-Osh7p, which have been implicated in regulating intracellular lipid and vesicular transportSterol metabolism and ERG2 gene regulation in the yeast Saccharomyces cerevisiae.Genomewide location analysis of Candida albicans Upc2p, a regulator of sterol metabolism and azole drug resistance.To better understand Upc2p function in C. albicans, we used genomewide location profiling to identify the transcriptional targets of Upc2p in vivo.Upc2p, a transcription factor of the zinc cluster family, is an important regulator of sterol biosynthesis and azole drug resistance in Candida albicans Taken together, our results indicate that Upc2p is a key regulator of ergosterol metabolism. They also suggest that Upc2p may contribute to azole resistance by regulating the expression of drug efflux pump-encoding genes in addition to ergosterol biosynthesis genes.Northern blot analysis showed that increased binding correlates with increased expression for the analyzed Upc2p targets (ERG11, MDR1, CDR1, YOR1, SUT1, SMF12, and CBP1). Overrepresented functional groups of genes whose promoters were bound by Upc2p included 12 genes involved in ergosterol biosynthesis (NCP1, ERG11, ERG2, and others)Cell wall maintenance genes TIR4 and CCW12, sterol metabolism gene UPC2, small molecule transport genes AUS1 and YHK8, and stress response gene CUP1-1 were expressed at a level at least 2.5-fold higher than the expression level found in 9763S.\"}, {\"question\": \"Which enzymes are responsible for base J creation in Trypanosoma brucei?\", \"ideal_answer\": [\"The base is synthesized in a two-step pathway. Initially, a thymidine residue in DNA is hydroxylated by a thymidine hydroxylase (TH). This intermediate (HOMedU) is then glucosylated to form base J. Two proteins involved in J synthesis, JBP1 (J binding protein 1) and JBP2, contain a putative TH domain related to the family of Fe(2+)/2-oxoglutarate-dependent hydroxylases. JBP2 is a chromatin re-modeling protein that induces de novo J-synthesis, allowing JBP1, a J-DNA binding protein, to stimulate additional J-synthesis. A recent computational screen identified a possible candidate for the base J-associated glucosyltransferase (JGT) in trypanosomatid genomes.\", \"JBP2 and JBP1\", \"The deletion of both alleles of JGT from the genome of Trypanosoma brucei generates a cell line that completely lacks base J. We conclude that JBP2 and JBP1 are the TH enzymes involved in J biosynthesis. -d-glucopyranosyloxymethyluracil . The base is synthesized in a two-step pathway. Synthesis of the modified thymine base, beta-d-glucosyl-hydroxymethyluracil or J, within telomeric DNA of Trypanosoma brucei correlates with the bloodstream form specific epigenetic silencing of telomeric variant surface glycoprotein genes involved in antigenic variation. Two thymidine hydroxylases differentially regulate the formation of glucosylated DNA at regions flanking polymerase II polycistronic transcription units throughout the genome of Trypanosoma brucei. A recent computational screen identified a possible candidate for the base J-associated glucosyltransferase  in trypanosomatid genomes. This intermediate  is then glucosylated to form base J. Here we discuss the regulation of hmU and base J formation in the trypanosome genome by JGT and base J-binding protein. Chromosome-internal J deposition is primarily mediated by JBP1, whereas JBP2-stimulated J deposition at the telomeric regions. Two proteins involved in J synthesis, JBP1 /2-oxoglutarate-dependent hydroxylases. JBP2 and JBP1 are capable of stimulating de novo J-synthesis. \", \"We have previously characterized two thymidine-hydroxylases (TH), JBP1 and JBP2, which regulate J-biosynthesis.JBP2 is a chromatin re-modeling protein that induces de novo J-synthesis, allowing JBP1, a J-DNA binding protein, to stimulate additional J-synthesis.Chromosome-internal J deposition is primarily mediated by JBP1, whereas JBP2-stimulated J deposition at the telomeric regions.Here we show that mutation of key residues in the TH domain of JBP2 ablate its ability to induce de novo J synthesis.JBP1 and JBP2 are two distinct thymidine hydroxylases involved in J biosynthesis in genomic DNA of African trypanosomes.Here we discuss the regulation of hmU and base J formation in the trypanosome genome by JGT and base J-binding protein.The deletion of both alleles of JGT from the genome of Trypanosoma brucei generates a cell line that completely lacks base JWe have recently proposed a model in which chromatin remodeling by a SWI2/SNF2-like protein (JBP2) regulates the developmental and de novo site-specific localization of J synthesis within bloodstream form trypanosome DNA.\"], \"exact_answer\": [[\"JBP1\"], [\"JBP2\"], [\"JGT\"]], \"context\": \"Synthesis of the modified thymine base, beta-d-glucosyl-hydroxymethyluracil or J, within telomeric DNA of Trypanosoma brucei correlates with the bloodstream form specific epigenetic silencing of telomeric variant surface glycoprotein genes involved in antigenic variation. We have recently proposed a model in which chromatin remodeling by a SWI2/SNF2-like protein (JBP2) regulates the developmental and de novo site-specific localization of J synthesis within bloodstream form trypanosome DNA.JBP1 and JBP2 are two distinct thymidine hydroxylases involved in J biosynthesis in genomic DNA of African trypanosomes.The base is synthesized in a two-step pathway. Initially, a thymidine residue in DNA is hydroxylated by a thymidine hydroxylase (TH). This intermediate (HOMedU) is then glucosylated to form base J. Two proteins involved in J synthesis, JBP1 (J binding protein 1) and JBP2, contain a putative TH domain related to the family of Fe(2+)/2-oxoglutarate-dependent hydroxylases.Here we show that mutation of key residues in the TH domain of JBP2 ablate its ability to induce de novo J synthesis.  We conclude that JBP2 and JBP1 are the TH enzymes involved in J biosynthesis.Two thymidine hydroxylases differentially regulate the formation of glucosylated DNA at regions flanking polymerase II polycistronic transcription units throughout the genome of Trypanosoma brucei.We have previously characterized two thymidine-hydroxylases (TH), JBP1 and JBP2, which regulate J-biosynthesis.JBP2 is a chromatin re-modeling protein that induces de novo J-synthesis, allowing JBP1, a J-DNA binding protein, to stimulate additional J-synthesis.JBP2 and JBP1 are capable of stimulating de novo J-synthesis.Chromosome-internal J deposition is primarily mediated by JBP1, whereas JBP2-stimulated J deposition at the telomeric regions. \\u03b2-d-glucopyranosyloxymethyluracil (base J) synthesis is initiated by the JBP1/2 enzymes that hydroxylate thymine, forming 5-hydroxymethyluracil (hmU)A recent computational screen identified a possible candidate for the base J-associated glucosyltransferase (JGT) in trypanosomatid genomes.Here we discuss the regulation of hmU and base J formation in the trypanosome genome by JGT and base J-binding protein.The thymidine hydroxylases (JPB1 and JBP2) that catalyze the first step have been characterized, but the identity of the glucosyltransferase catalyzing the second step has proven elusive.  The deletion of both alleles of JGT from the genome of Trypanosoma brucei generates a cell line that completely lacks base JThe deletion of both alleles of JGT from the genome of Trypanosoma brucei generates a cell line that completely lacks base J.\\u03b2-d-glucopyranosyloxymethyluracil (base J) synthesis is initiated by the JBP1/2 enzymes that hydroxylate thymine, forming 5-hydroxymethyluracil (hmU).\"}, {\"question\": \"Which drugs were tested in the KEYNOTE-006 study?\", \"ideal_answer\": [\"KEYNOTE-006 study compared pembrolizumab versus ipilimumab for advanced melanoma.\"], \"exact_answer\": [[\"pembrolizumab\"], [\"ipilimumab\"]], \"context\": \"The model used Kaplan-Meier estimates of progression-free survival and overall survival from a recent randomized phase 3 study (KEYNOTE-006) that compared pembrolizumab and ipilimumab.Pembrolizumab versus ipilimumab for advanced melanoma: final overall survival results of a multicentre, randomised, open-label phase 3 study (KEYNOTE-006).BACKGROUND: Interim analyses of the phase 3 KEYNOTE-006 study showed superior overall and progression-free survival of pembrolizumab versus ipilimumab in patients with advanced melanoma.INTERPRETATION: Substantiating the results of the interim analyses of KEYNOTE-006, pembrolizumab continued to provide superior overall survival versus ipilimumab, with no difference between pembrolizumab dosing schedules. OBJECTIVE: Report results of patient-reported health-related quality of life (HRQoL) and symptoms from phase III KEYNOTE-006 study of pembrolizumab versus ipilimumab in patients with ipilimumab-naive advanced melanoma.Patient-reported outcomes in KEYNOTE-006, a randomised study of pembrolizumab versus ipilimumab in patients with advanced melanomaPembrolizumab (Keytruda(\\u00ae)) is a humanized monoclonal antibody against programmed death receptor-1 (PD-1), a key immunoinhibitory checkpoint protein implicated in down-regulating anti-tumour immune responses. This intravenous drug is indicated for the treatment of advanced (unresectable or metastatic) melanoma, on the basis of its clinical benefit in this setting in the phase I KEYNOTE 001 trial (expansion cohorts) and the phase II and III trials, KEYNOTE 002 and 006.CONCLUSIONS: The anti-PD-1 antibody pembrolizumab prolonged progression-free survival and overall survival and had less high-grade toxicity than did ipilimumab in patients with advanced melanoma. (Funded by Merck Sharp&Dohme; KEYNOTE-006 ClinicalTrials.gov number, NCT01866319.).Interim analyses of the phase 3 KEYNOTE-006 study showed superior overall and progression-free survival of pembrolizumab versus ipilimumab in patients with advanced melanoma.Report results of patient-reported health-related quality of life (HRQoL) and symptoms from phase III KEYNOTE-006 study of pembrolizumab versus ipilimumab in patients with ipilimumab-naive advanced melanoma.The model used Kaplan-Meier estimates of progression-free survival and overall survival from a recent randomized phase 3 study (KEYNOTE-006) that compared pembrolizumab and ipilimumab.A cost-effectiveness model was developed to analyze the costs and consequences of treatment with pembrolizumab compared to treatment with ipilimumab in patients with advanced melanoma not previously treated with ipilimumab. The model was parameterized by using data from a head-to-head phase III randomized clinical trial, KEYNOTE-006.BACKGROUND Interim analyses of the phase 3 KEYNOTE-006 study showed superior overall and progression-free survival of pembrolizumab versus ipilimumab in patients with advanced melanoma.OBJECTIVE Report results of patient-reported health-related quality of life (HRQoL) and symptoms from phase III KEYNOTE-006 study of pembrolizumab versus ipilimumab in patients with ipilimumab-naive advanced melanoma.INTERPRETATION Substantiating the results of the interim analyses of KEYNOTE-006, pembrolizumab continued to provide superior overall survival versus ipilimumab, with no difference between pembrolizumab dosing schedules.The model used Kaplan-Meier estimates of progression-free survival and overall survival from a recent randomized phase 3 study (KEYNOTE-006) that compared pembrolizumab and ipilimumab.In the trials with active comparator arms, pembrolizumab regimens significantly improved progression-free survival (PFS), overall survival (OS) and overall response rates (ORR) relative to ipilimumab in ipilimumab-na\\u00efve patients (KEYNOTE 006), and significantly improved PFS and ORR, but not OS (although OS data are immature), relative to chemotherapy in ipilimumab-refractory patients, who had also received BRAF/MEK inhibitor therapy if BRAF-mutation positive (KEYNOTE 002).\"}, {\"question\": \"List main clinical features of the POEMS syndrome.\", \"ideal_answer\": [\"POEMS is an acronym for the main clinical features of the syndrome, namely, Polyneuropathy, Organomegaly, Endocrinopathy, M protein, and Skin abnormalities. Other features include papilledema, extravascular volume overload, sclerotic bone lesions, thrombocytosis, and Castleman disease. It is a multisystemc disorder with a good long-term prognosis.\"], \"exact_answer\": [[\"Polyneuropathy\"], [\"Organomegaly\"], [\"Endocrinopathy\"], [\"M protein\"], [\"Skin abnormalities\"]], \"context\": \"POEMS syndrome is a rare paraneoplastic syndrome secondary to a plasma cell dyscrasia. Recognition of a combination of peripheral neuropathy, organomegaly, endocrinopathy, monoclonal plasmaproliferative disorder, skin changes, papilledema, extravascular volume overload, sclerotic bone lesions, thrombocytosis, and Castleman disease is the first step in managing the disease.Treatment of polyneuropathy, organomegaly, endocrinopathy, monoclonal gammopathy, and skin changes (POEMS) syndrome should be directed at the underlying plasma cell clone with risk-adapted therapy based on the extent of the plasma cell disorder.Polyneuropathy, organomegaly, endocrinopathy, monoclonal plasma cell disorder, skin changes (POEMS) syndrome is a rare paraneoplastic disorder.POEMS is an acronym for the main clinical features of the syndrome, namely, Polyneuropathy, Organomegaly, Endocrinopathy, M protein, and Skin abnormalities.POEMS syndrome (polyneuropathy, organomegaly, endocrinopathy, monoclonal gammopathy and skin changes) is a multisystem disorder with a good long-term prognosis.The association of polyneuropathy, organomegaly, endocrinopathy, monoclonal gammopathy, and skin changes forms a characteristic multisystem syndrome, Crow-Fukase syndrome, or acronym POEMS syndrome.Polyneuropathy, organomegaly, endocrinopathy, monoclonal gammopathy and skin changes (POEMS) syndrome is a multisystem disorder arising from underlying plasma cell dyscrasia.Polyneuropathy, organomegaly, endocrinopathy, monoclonal gammopathy, and skin changes (POEMS) syndrome is an uncommon condition related to a paraneoplastic syndrome secondary to an underlying plasma cell disorder.Pulmonary hypertension is one of the well-known clinical manifestations of polyneuropathy, organomegaly, endocrinopathy, monoclonal gammopathy, and skin changes (POEMS) syndrome, occurring in approximately 25-30% of the affected individuals.Polyneuropathy, organomegaly, endocrinopathy, monoclonal gammopathy, and skin changes (POEMS) syndrome is a rare disorder.Polyneuropathy, organomegaly, endocrinopathy, monoclonal gammopathy and skin changes (POEMS) syndrome, a plasma cell dyscrasia associated with pulmonary hypertension, has been treated in the past with anticytokine strategies with a poor outcome.A 48-year-old man with polyneuropathy, organomegaly, endocrinopathy, monoclonal gammopathy, and skin changes (POEMS) syndrome had bilateral optic disc edema (ODE), bilateral cystoid macular edema (CME), anasarca, and elevated serum vascular endothelial growth factor (VEGF).A 39-year-old woman presented with polyneuropathy, hepatomegaly, splenomegaly, endocrinopathy, monoclonal protein and skin changes, several of the many clinical features of the recently described POEMS syndrome.The acronym POEMS is derived from main features of the syndrome namely 'polyneuropathy, organomegaly, endocrinopathy, monoclonal gammopathy and skin lesions'.POEMS syndrome, also known as Crow-Fukase syndrome, represents a rare multisystem syndrome characterized by polyneuropathy, organomegaly, endocrinopathy, M protein, and skin changes.Polyneuropathy is often an initial manifestation of polyneuropathy, organomegaly, endocrinopathy, M protein and skin changes (POEMS) syndrome and therefore this disorder is frequently misdiagnosed as chronic inflammatory demyelinating polyneuropathy (CIDP).POEMS syndrome is a rare paraneoplastic condition, commonly associated with Castleman disease, that manifests with progressive distal polyneuropathy and a monoclonal plasma cell disorder, often accompanied by endocrinopathy, organomegaly, skin changes, sclerotic bone lesions, ascites, erythrocytosis, and thrombocytosis.POEMS syndrome, a rare multisystem disease, is a variant of osteosclerotic myeloma and is characterized by polyneuropathy, organomegaly, endocrinopathy, monoclonal proteins, and skin changes.We report a case of a 64-year-old man with POEMS (polyneuropathy, organomegaly, endocrinopathy, monoclonal gammopathy and skin changes) syndrome that had been previously misdiagnosed as systemic sclerosis.POEMS syndrome is a rare plasma cell dyscrasia characterized by polyneuropathy, organomegaly, endocrinopathy, M protein, and skin changes.The POEMS (polyneuropathy, organomegaly, endocrinopathy, monoclonal gammapathy, and skin changes) syndrome is a rare variant of plasma cell dyscrasia with multisystemic manifestations.a 39 year old woman presented with polyneuropathy hepatomegaly splenomegaly endocrinopathy monoclonal protein and skin changes several of the many clinical features of the recently described poems syndrome in addition she had a castleman s disease angiofollicular lymph node hyperplasia in this case ascites was a main presenting feature thus the poems syndrome must be added to the list of rare causes of ascites electron microscopy of the liver showed perisinusoidal fibrosis.the poems syndrome is a rare multisystemic disorder with polyneuropathy organomegaly endocrinopathy of various forms production of monoclonal m component and skin changes we describe a 46 year old man who developed ascites one year after the onset of peripheral neuropathy with accompanying muscle atrophies and increasing weakness extensive evaluation revealed that the patient had no underlying liver disease malignancy infection or cardiac or renal disease the ascites initially responded to high dose corticosteroid therapy the patient had many clinical features of the described poems syndrome including sclerotic bone lesions a persistent lambda paraprotein and refractory ascites in this case ascites was a main presenting feature thus the poems syndrome must be added to the list of rare causes of refractory ascites.poems syndrome is a paraneoplastic manifestation associated with hematopoietic disorders such as multiple myeloma and castleman disease poems is an acronym for the main clinical features of the syndrome namely polyneuropathy organomegaly endocrinopathy m protein and skin abnormalities glomeruloid hemangiomas are considered to be a specific clinical marker of poems syndrome however while they are not pathognomonic their presence should raise suspicion of this syndrome or alert clinicians to its possible future development as these lesions can appear years before the onset of the syndrome we report the cases of 2 women with plasma cell dyscrasias and sudden onset of lesions with a vascular appearance and histologic findings consistent with glomeruloid hemangioma recognition of this vascular tumor is important for the early diagnosis of poems syndrome.poems syndrome is a rare conglomeration of disorders associated with plasma cell dyscrasia the acronym poems is derived from main features of the syndrome namely polyneuropathy organomegaly endocrinopathy monoclonal gammopathy and skin lesions other clinical features include presence of sclerotic bone lesions castleman s disease papilledema pleural effusion edema ascites erythrocytosis and thrombocytosis myeloma is the most common plasma cell dyscrasia associated with poems syndrome renal involvement is rare and renal biopsy is characterized by glomerular involvement with membranoproliferative glomerulonephritis and endothelial injury we report a case of a 67 year old male who presented with clinical features satisfying the diagnostic criteria of poems syndrome and had rapidly progressive renal failure renal biopsy showed extensive interstitial infiltration by plasma cells and concomitant presence of classic polyarteritis nodosa although association with small vessel vasculitis has been reported in patients with poems syndrome to the best of our knowledge this is the first report of poems syndrome associated with medium sized vessel vasculitis.poems syndrome also known as crow fukase syndrome represents a rare multisystem syndrome characterized by polyneuropathy organomegaly endocrinopathy m protein and skin changes hypothyroidism is one of the common endocrine abnormalities which are central features of poems syndrome the clinical data associated with the measurement of thyroid function and its clinical significance in poems syndrome is still rare herein we report 24 cases with poems syndrome which were studied thyroid function and clinical manifestations and performed an associated analysis between hypothyroidism and edema effusions of the 24 patients with poems syndrome 17 70 8 had a recognized hypothyroidism including 11 clinical hypothyroidism and 6 subclinical hypothyroidism fourteen patients 58 3 had some form of extravascular volume overload in 14 patients with edema effusions 12 were diagnosed as having hypothyroidism hypothyroidism may be one of causes of edema effusions after thyroid hormone treatment and chemotherapy symptoms of hypothyroidism and edema effusions were improved greatly.we present a case of a woman with unique multisystem disorder poems syndrome and endocrine abnormalities coexisting with it the poems acronym comprises the dominant features polyneuropathy organomegaly endocrinopathy monoclonal protein m protein skin changes association between plasma cell dyscrasia and polyneuropathy was described in 1956 year by crow the main features were coined in the acronym poems by bardwick in 1980 year the polysymptomatic clinical picture progressive course and no concurrent manifestations of main features impede the diagnosis in this case the first symptoms were the sensomotor polyneuropathy peripheral oedema osteosclerotic bone lesions skin changes organomegaly they preceded diagnosis by 3 years the first endocrinopathy was hypothyroidism definite diagnosis was delayed because we couldn t detect the presence of m protein immunoelectrophoresis didn t detect it but analysis by immunofixation detected m protein in serum and urine within 3 years of the first symptoms she developed hypogonadism hypergonadotropic at first the monotherapy with corticosteroids was used then melfalan with prednisone due to the progression of the disease a thalidomide was used in therapy it is anti vegf agent one of the side effects of the treatment of thalidomide is the progression of polyneuropathy which was observed in this patient after finishing this therapy she received chemotherapy this case report imposes the necessity of constants observation of patients with poems syndrome because there is a possibility of their developing other disorders in the event of coexistence polyneuropathy and plasma cell dyscrasia this disease should be taken into consideration.\"}, {\"question\": \"What are the 4 cardinal signs of inflammation according to Celsus?\", \"ideal_answer\": [\"redness or rubor , heat or calor, swelling or tumor, and pain or dolor\", \"Tumor, calor, rubor, and dolor describe four cardinal signs of inflammation. \"], \"exact_answer\": [[\"redness or rubor\"], [\"heat or calor\"], [\"swelling or tumor\"], [\"pain or dolor\"]], \"context\": \"Tumor, calor, rubor, and dolor describe four cardinal signs of inflammation. Reference was paid to the cardinal signs of inflammation (pain, heat, redness, and swelling)Inflammation is the body's response to injury or infection. As early as 2000 years ago, the Roman encyclopaedist Aulus Cornelius Celsus recognised four cardinal signs of this response-redness, heat, swelling and pain;  inflammation gives rise to clinical cardinal signs: rubor, calor, dolor, tumorThe cardinal signs of inflammation dolor, calor, tumor and ruborThe concept of the four cardinal signs of acute inflammation comes from antiquity as rubor et tumor cum calore et dolore, (redness and swelling with heat and pain) Inflammation is considered a nonspecific response to injury, characterized by exudation of serum into damaged tissue, and identified by the cardinal signs of rubor, calor, dolor, and tumor.four well-known cardinal signs of Celsus (rubor, calor, tumor, dolor)four cardinal signs of Celsus (redness, heat, swelling and pain). Inflammation (the famous four cardinal signs of Celsus: pain, redness, swelling, heat) As early as 2000 years ago, the Roman encyclopaedist Aulus Cornelius Celsus recognised four cardinal signs of this response-redness, heat, swelling and pain; a fifth sign is loss of function.[...It was Galen who added the disturbance of function (functio laesa) as the fifth cardinal sign of inflammation to the four well-known cardinal signs of Celsus (rubor, calor, tumor, dolor).The concept of the four cardinal signs of acute inflammation comes from antiquity as rubor et tumor cum calore et dolore, (redness and swelling with heat and pain) extended later by functio laesa (loss of function).The cardinal signs of inflammation dolor, calor, tumor and rubor are intrinsically associated with events including vasodilatation, edema and leukocyte trafficking into the site of inflammation.Tumor, calor, rubor, and dolor describe four cardinal signs of inflammation.The four cardinal signs of inflammation are redness, swelling, heat, and pain, phenomena explained at the cellular level.inflammation is the body s response to injury or infection as early as 2000 years ago the roman encyclopaedist aulus cornelius celsus recognised four cardinal signs of this response redness heat swelling and pain a fifth sign is loss of function.\"}, {\"question\": \"Which two genes are implicated in Juvenile polyposis syndrome?\", \"ideal_answer\": [\"Juvenile polyposis syndrome (JPS) is a rare autosomal dominant disorder predisposing to gastrointestinal hamartomatous polyps and cancer with a pathogenic SMAD4 or BMPR1A germline mutation being identified in about 40-50% of patients.\"], \"exact_answer\": [[\"SMAD4\"], [\"BMPR1A\"]], \"context\": \" JPS is most frequently caused by mutations in the SMAD4 or BMPR1A genes. Herein, we report a child with juvenile polyposis syndrome (JPS) with a novel mutation in the SMAD4 gene. Subsequent genetic screening revealed a novel mutation in SMAD4, exon 5 (p.Ser144Stop). A SMAD4 mutation indicative of juvenile polyposis syndrome in a family previously diagnosed with Menetrier's disease.CONCLUSION: The 1244_1247delACAG mutation of SMAD4 is the cause of JPS and the likely cause of MD in a large family initially diagnosed with MD.A dominant 1244_1247delACAG mutation of SMAD4 was identified in each of the subjects with JPS as well as in each of the subjects with MD. ince his initial diagnosis of JPS further studies have confirmed an association between JPS and mutations in BMPR1A in chromosome band 10q23.2, which is in close proximity to PTEN.Prevalence of thoracic aortopathy in patients with juvenile Polyposis Syndrome-Hereditary Hemorrhagic Telangiectasia due to SMAD4.Exome sequencing identified a germline heterozygous mutation in SMAD9 (SMAD9(V90M)). Causative mutations for JPS have been identified in two genes to date, SMAD4 and BMPR1A. Somatic alterations in juvenile polyps from BMPR1A and SMAD4 mutation carriers.Juvenile polyposis syndrome (JPS) is a rare autosomal dominant disorder predisposing to gastrointestinal hamartomatous polyps and cancer with a pathogenic SMAD4 or BMPR1A germline mutation (1st-hit) being identified in about 40-50% of patients.The JPS is caused by mutations in SMAD4 and BMPR1A. Germline mutations in SMAD4 and BMPR1A disrupt the transforming growth factor \\u03b2 signal transduction pathway, and are associated with juvenile polyposis syndrome.This study evaluated the differential impact of SMAD4 and BMPR1A gene mutations on cancer risk and oncological phenotype in patients with juvenile polyposis syndrome.Juvenile polyposis syndrome (JPS) is caused by heterozygous mutations in either SMAD4 or BMPR1A. Individuals with JPS due to mutations in SMAD4 are at greater risk to manifest signs of hereditary hemorrhagic telangiectasia (HHT).BMPR1A and SMAD4 germline mutations have been found in patients with juvenile polyposis syndrome.In about 50%-60% of patients diagnosed with juvenile polyposis syndrome a germline mutation in the SMAD4 or BMPR1A gene is found.Juvenile polyposis syndrome (JPS) is caused by heterozygous mutations in either SMAD4 or BMPR1A.BACKGROUND In patients with juvenile polyposis syndrome (JPS) the frequency of large genomic deletions in the SMAD4 and BMPR1A genes was unknown.in patients with juvenile polyposis syndrome jps the frequency of large genomic deletions in the smad4 and bmpr1a genes was unknown mutation and phenotype analysis was used in 80 unrelated patients of whom 65 met the clinical criteria for jps typical jps and 15 were suspected to have jps by direct sequencing of the two genes point mutations were identified in 30 patients 46 of typical jps using mlpa large genomic deletions were found in 14 of all patients with typical jps six deletions in smad4 and three deletions in bmpr1a mutation analysis of the pten gene in the remaining 41 mutation negative cases uncovered a point mutation in two patients 5 smad4 mutation carriers had a significantly higher frequency of gastric polyposis 73 than did patients with bmpr1a mutations 8 p 0 001 all seven cases of gastric cancer occurred in families with smad4 mutations smad4 mutation carriers with gastric polyps were significantly older at gastroscopy than those without p 0 001 in 22 of the 23 unrelated smad4 mutation carriers hereditary hemorrhagic telangiectasia hht was also diagnosed clinically the documented histologic findings encompassed a wide distribution of different polyp types comparable with that described in hereditary mixed polyposis syndromes hmps screening for large deletions raised the mutation detection rate to 60 in the 65 patients with typical jps a strong genotype phenotype correlation for gastric polyposis gastric cancer and hht was identified which should have implications for counselling and surveillance histopathological results in hamartomatous polyposis syndromes must be critically interpreted.juvenile polyposis syndrome is a rare autosomal dominant syndrome characterized by multiple distinct juvenile polyps in the gastrointestinal tract and an increased risk of colorectal cancer the cumulative life time risk of colorectal cancer is 39 and the relative risk is 34 juvenile polyps have a distinctive histology characterized by an abundance of edematous lamina propria with inflammatory cells and cystically dilated glands lined by cuboidal to columnar epithelium with reactive changes clinically juvenile polyposis syndrome is defined by the presence of 5 or more juvenile polyps in the colorectum juvenile polyps throughout the gastrointestinal tract or any number of juvenile polyps and a positive family history of juvenile polyposis in about 50 60 of patients diagnosed with juvenile polyposis syndrome a germline mutation in the smad4 or bmpr1a gene is found both genes play a role in the bmp tgf beta signalling pathway it has been suggested that cancer in juvenile polyposis may develop through the so called landscaper mechanism where an abnormal stromal environment leads to neoplastic transformation of the adjacent epithelium and in the end invasive carcinoma recognition of this rare disorder is important for patients and their families with regard to treatment follow up and screening of at risk individuals each clinician confronted with the diagnosis of a juvenile polyp should therefore consider the possibility of juvenile polyposis syndrome in addition juvenile polyposis syndrome provides a unique model to study colorectal cancer pathogenesis in general and gives insight in the molecular genetic basis of cancer this review discusses clinical manifestations genetics pathogenesis and management of juvenile polyposis syndrome.juvenile polyposis syndrome is an autosomal dominant inherited disorder characterized by multiple juvenile polyps arising in the gastrointestinal tract and an increased risk of gastrointestinal cancers specifically colon cancer bmpr1a and smad4 germline mutations have been found in patients with juvenile polyposis syndrome we identified a bmpr1a mutation which involves a duplication of coding exon 3 c 230 452 333 441dup1995 on multiple ligation dependent probe amplification in a patient with juvenile polyposis syndrome the mutation causes a frameshift producing a truncated protein p d112nfsx2 therefore the mutation is believed to be pathogenic we also identified a duplication breakpoint in which alu sequences are located these results suggest that the duplication event resulted from recombination between alu sequences to our knowledge partial duplication in the bmpr1a gene has not been reported previously this is the first case report to document coding exon 3 duplication in the bmpr1a gene in a patient with juvenile polyposis syndrome.juvenile polyposis syndrome is a dominant gi polyposis syndrome defined by 5 gi juvenile polyps or 1 juvenile polyps with a family history of juvenile polyposis mutations in bmpr1a or smad4 are found in 50 of individuals hereditary hemorrhagic telangiectasia is a dominant disorder characterized by epistaxis visceral arteriovenous malformations and telangiectasias hereditary hemorrhagic telangiectasia is diagnosed when 3 criteria including clinical manifestations or a family history are present a juvenile polyposis hereditary hemorrhagic telangiectasia overlap syndrome has previously been reported in 22 of patients with juvenile polyposis due to a smad4 mutation our objective was to determine the prevalence and clinical manifestations of hereditary hemorrhagic telangiectasia by curacao criteria in our juvenile polyposis smad4 patients this was a cohort study of juvenile polyposis patients in our inherited colon cancer registries hereditary hemorrhagic telangiectasia manifestations were obtained from medical records patient contact and or prospective hereditary hemorrhagic telangiectasia screening the curacao criteria was used for diagnosis of hereditary hemorrhagic telangiectasia 3 criteria diagnostic 2 criteria suspect of prevalence and clinical manifestations of hereditary hemorrhagic telangiectasia in juvenile polyposis smad4 patients forty one juvenile polyposis families were identified genetic testing was available for individuals within 18 families smad4 mutations were found in 21 relatives in 9 families eighty one percent of smad4 patients had hereditary hemorrhagic telangiectasia and 14 were suspected of having hereditary hemorrhagic telangiectasia epistaxis and asthma are the most common symptoms in our overlap patients symptomatic and subclinical arteriovenous malformations were noted near universally there was a single tertiary referral center nearly all juvenile polyposis smad4 patients have the overlap syndrome the clinical implications and need for hereditary hemorrhagic telangiectasia screening are important factors for genetic testing in juvenile polyposis health care providers must be cognizant of the juvenile polyposis hereditary hemorrhagic telangiectasia overlap syndrome and the implications for management of these patients.\"}, {\"question\": \"List the 6 genes associated with the autosomal recessive form of Osteogenesis imperfecta\", \"ideal_answer\": [\"There are at least 6 genes associated with osteogenesis imperfecta, Sp7/Osx, FK506-binding protein, Hsp47/SERPINH1, WNT1, CRTAP, P3H1, and PPIB, LEPRE1,PLOD2, TMEM38B\"], \"exact_answer\": [[\"Sp7/Osx\"], [\"FK506-binding protein\"], [\"Hsp47/SERPINH1\"], [\"CRTAP\"], [\"P3H1\"], [\"PPIB\"], [\"WNT1\"], [\"LEPRE1\"], [\"PLOD2\"], [\"TMEM38B\"], [\"BMP1\"], [\"SEC24D\"], [\"cartilage associated protein\"]], \"context\": \"Patients offspring of nonconsanguineous parents were mostly identified withCOL1A1orCOL1A2heterozygous changes, although there were also a few cases withIFITM5andWNT1heterozygous mutations. Only one sporadic patient was a compound heterozygote for two recessive mutations. Patients offspring of consanguineous parents showed homozygous changes in a variety of genes includingCRTAP,FKBP10,LEPRE1,PLOD2,PPIB,SERPINF1,TMEM38B, andWNT1.here is minimal literature on the mechanism of action for variants in SERPINH1 resulting in osteogenesis imperfecta The majority of patients (about 90%) with a clinical diagnosis of OI have a mutation in the COL1A1 or COL1A2 genes, which shows an autosomal dominant pattern of inheritance. Six other genes, CRTAP, LEPRE1, FKBP10, PP1B, SP7/Osterix (OSX), and SERPINH1, are associated with autosomal recessive forms of OI.hese results expand the range of CRTAP/LEPRE1 mutations that result in recessive OI Autosomal recessive osteogenesis imperfecta (OI) accounts for 10% of all OI cases, and, currently, mutations in 10 genes (CRTAP, LEPRE1, PPIB, SERPINH1, FKBP10, SERPINF1, SP7, BMP1, TMEM38B, and WNT1) are known to be responsible for this form of the disease.We have recently shown that FKBP10 previously reported as a novel autosomal recessive OI gene also defines a novel Bruck syndrome locus (BKS3).Autosomal recessive mutations in a number of genes have also been described, including the BMP1 gene that encodes the mammalian Tolloid (mTLD) and its shorter isoform bone morphogenic protein-1 (BMP1).n this article, we performed clinical assessment and sought mutations in patients from 10 unrelated families with AR-OI, one of whom was presented with the additional features of Bruck syndrome (BS). Pathogenic changes were identified in five different genes: three families had mutations in FKBP10, three in SERPINF1, two in LEPRE1, one in CRTAP, and one in PPIB. Two of the genes encoding proteins involved in that enzyme complex, LEPRE1 and cartilage-associated protein, when mutated have been shown to cause autosomal recessive osteogenesis imperfectaThe discovery of mutations involving CRTAP and LEPRE1 genes in severe/lethal and recessively inherited osteogenesis imperfecta has provided partial answers to questions about 'other' osteogenesis imperfecta genes in patients with an osteogenesis imperfecta phenotype but no COL1A1 and COL1A2 mutations\"}, {\"question\": \"Which chromosomes are implicated in the Emanuel syndrome?\", \"ideal_answer\": [\"Emanuel syndrome is associated with supernumerary chromosome t(11;22)(q23;q11), which consists of the extra genetic material from chromosomes 11 and 22.\"], \"exact_answer\": [[\"11\"], [\"22\"]], \"context\": \"Single\\u2011nucleotide polymorphism\\u2011array analysis of the proband indicated a partial duplication of chromosomes 22 and 11 at 22q11.1\\u2011q11.21 and 11q23.3\\u2011q25, respectively, which confirmed the diagnosis of ES.NGS and FISH analysis verified that the supernumerary marker chromosome carried by the fetus was der(22) t(11; 22) (q23;q11). Emanuel syndrome is associated with supernumerary chromosome, which consists of the extra genetic material from chromosome 11 and 22. Multiple congenital anomalies and craniofacial dysmorphism are characterizing the so-called Emanuel or supernumerary der(22)t(11;22) syndrome (OMIM609029). Complex sSMCs consist of chromosomal material derived from more than one chromosome, for example, the derivative der(22)t(11;22)(q23;q11.2) in Emanuel syndrome.A case with Emanuel syndrome: extra derivative 22 chromosome inherited from the mother. Affected children are usually identified in the newborn period as the offspring of balanced (11;22) translocation carriers.We report a 3-year-old boy with the t(11;22)(q23;q11) chromosome, transmitted in an unbalanced fashion from his mother. Balanced carriers of t(11;22) usually manifest no clinical symptoms, and are often identified after the birth of offspring with an unbalanced form of this translocation, known as Emanuel syndrome. Derivative 11;22 (emanuel) syndrome: a case report and a review.Complex sSMC consist of chromosomal material derived from more than one chromosome; the best known representative of this group is the derivative chromosome 22 {der(22)t(11;22)} or Emanuel syndrome. The syndrome is caused by chromosomal imbalance due to a supernumerary derivative chromosome 22. We have generated iPSCs from monosomy X [Turner syndrome (TS)], trisomy 8 (Warkany syndrome 2), trisomy 13 (Patau syndrome) and partial trisomy 11;22 (Emanuel syndrome), using either skin fibroblasts from affected individuals or amniocytes from antenatal diagnostic tests. Emanuel syndrome is an inherited chromosomal abnormality resulting from 3:1 meiotic segregation from parental balanced translocation carrier t(11;22)(q23;q11), mostly of maternal origin. Here in, we describe a female patient with supernumerary der(22) syndrome (Emanuel syndrome) due to balanced translocation carrier father t(11;22) (q23;q11). The t(11;22)(q23;q11.2) is the main recurrent germline translocation in humans. Unbalanced translocation can be transmitted to the progeny and can cause Emanuel syndrome.Complex small supernumerary marker chromosomes (sSMCs) constitute one of the smallest subsets within the patients with an sSMC. Complex sSMCs consist of chromosomal material derived from more than one chromosome, for example, the derivative der(22)t(11;22)(q23;q11.2) in Emanuel syndrome.Emanuel syndrome is associated with supernumerary chromosome, which consists of the extra genetic material from chromosome 11 and 22.The observed number of Emanuel syndrome cases was 36 and that of t(11;22) balanced translocation carriers, 40.Emanuel syndrome is an inherited chromosomal abnormality resulting from 3:1 meiotic segregation from parental balanced translocation carrier t(11;22)(q23;q11), mostly of maternal origin.Emanuel syndrome results from +der(22)t(11q23;22q11).Here in, we describe a female patient with supernumerary der(22) syndrome (Emanuel syndrome) due to balanced translocation carrier father t(11;22) (q23;q11).Complex sSMC consist of chromosomal material derived from more than one chromosome; the best known representative of this group is the derivative chromosome 22 {der(22)t(11;22)} or Emanuel syndrome.Complex sSMCs consist of chromosomal material derived from more than one chromosome, for example, the derivative der(22)t(11;22)(q23;q11.2) in Emanuel syndrome.Balanced carriers of t(11;22) usually manifest no clinical symptoms, and are often identified after the birth of offspring with an unbalanced form of this translocation, known as Emanuel syndrome.supernumerary chromosome der 22 t 11 22 emanuel syndrome associates with novel featuresa case with emanuel syndrome extra derivative 22 chromosome inherited from the motheremanuel syndrome is a rare chromosomal disorder characterized by severe mental retardation and multiple anomalies the syndrome is caused by chromosomal imbalance due to a supernumerary derivative chromosome 22 little is known regarding the characteristics of prenatal biochemical screening or ultrasonographic markers in this syndrome we aimed to identify a prenatal screening pattern characteristic of emanuel syndrome we report the prenatal characteristics of five fetuses with emanuel syndrome four of which were diagnosed prenatally we found no consistent pattern of prenatal biochemical markers or other prenatal characteristics nevertheless increased nt low papp a and ultrasound features such as intra uterine growth restriction posterior fossa cardiac and bowel abnormalities may be helpful in raising the suspicion for this rare genetic syndrome review of the biochemical screening results ultrasound findings and demographic characteristics of this emanuel syndrome case series as well as of the relevant literature fail to suggest a characteristic prenatal pattern.emanuel syndrome is associated with supernumerary chromosome which consists of the extra genetic material from chromosome 11 and 22 the frequency of this syndrome has been reported as 1 in 110 000 it is a rare anomaly associated with multiple systemic malformations such as micrognathia and congenital heart disease in addition patients with emanuel syndrome may have seizure disorders we experienced anesthetic management of a patient with emanuel syndrome who underwent palatoplasty this patient had received tracheotomy due to micrognathia in addition he had atrial septal defect mild pulmonary artery stenosis and cleft palate palatoplasty was performed without any complication during anesthesia close attention was directed to cardiac function seizure and airway management.complex small supernumerary marker chromosomes ssmc constitute one of the smallest subgroups of ssmc in general complex ssmc consist of chromosomal material derived from more than one chromosome the best known representative of this group is the derivative chromosome 22 der 22 t 11 22 or emanuel syndrome in 2008 we speculated that complex ssmc could be part of an underestimated entity here the overall yet reported 412 complex ssmc are summarized they constitute 8 4 of all yet in detail characterized ssmc cases the majority of the complex ssmc is contributed by patients suffering from emanuel syndrome 82 besides there are a der 22 t 8 22 q24 1 q11 1 and a der 13 t 13 18 q11 p11 21 or der 21 t 18 21 p11 21 q11 1 der 13 or 21 t 13 or 21 18 syndrome the latter two represent another 2 6 and 2 2 of the complex ssmc cases respectively the large majority of complex ssmc has a centric minute shape and derives from an acrocentric chromosome nonetheless complex ssmc can involve material from each chromosomal origin most complex ssmc are inherited form a balanced translocation in one parent and are non mosaic interestingly there are hot spots for the chromosomal breakpoints involved complex ssmc need to be considered in diagnostics especially in non mosaic centric minute shaped ssmc as yet three complex ssmc associated syndromes are identified as recurrent breakpoints in the complex ssmc were characterized it is to be expected that more syndromes are identified in this subgroup of ssmc overall complex ssmc emphasize once more the importance of detailed cytogenetic analyses especially in patients with idiopathic mental retardation.emanuel syndrome is characterized by multiple congenital anomalies and developmental disability it is caused by the presence of a supernumerary derivative chromosome that contains material from chromosomes 11 and 22 the origin of this imbalance is 3 1 malsegregation of a parental balanced translocation between chromosomes 11 and 22 which is the most common recurrent reciprocal translocation in humans little has been published on the clinical features of this syndrome since the 1980s and information on natural history is limited we designed a questionnaire to collect information from families recruited through an international online support group chromosome 22 central data gathered include information on congenital anomalies medical and surgical history developmental and behavioral issues and current abilities we received information on 63 individuals with emanuel syndrome ranging in age from newborn to adulthood as previously recognized congenital anomalies were common the most frequent being ear pits 76 micrognathia 60 heart malformations 57 and cleft palate 54 our data suggest that vision and hearing impairment seizures failure to thrive and recurrent infections particularly otitis media are common in this syndrome psychomotor development is uniformly delayed however the majority of individuals over 70 eventually learn to walk with support language development and ability for self care are also very impaired this study provides new information on the clinical spectrum and natural history of emanuel syndrome for families and physicians caring for these individuals.emanuel syndrome is an inherited chromosomal abnormality resulting from 3 1 meiotic segregation from parental balanced translocation carrier t 11 22 q23 q11 mostly of maternal origin it is characterized by mental retardation microcephaly preauricular tag or sinus ear anomalies cleft or high arched palate micrognathia congenital heart diseases kidney abnormalities structural brain anomalies and genital anomalies in male here in we describe a female patient with supernumerary der 22 syndrome emanuel syndrome due to balanced translocation carrier father t 11 22 q23 q11 she was mentally and physically disabled and had most of the craniofacial dysmorphism of this syndrome our patient had cleft palate maldeveloped corpus callosum and hind brain with normal internal organs additionally arachnodactyly hyperextensibility of hand joints abnormal deep palmar and finger creases extra finger creases and bilateral talipus were evident and not previously described with this syndrome cytogenetic analysis and fish documented that the patient had both translocation chromosomes plus an additional copy of der 22 with karyotyping 47 xx t 11 22 q23 q11 der 22 t 11 22 q23 q11 we postulated that this rare chromosomal complement can arise from 2 2 segregation in the first meiotic division of the balanced translocation father followed by non disjunction at meiosis ii in the balanced spermatocyte.emanuel syndrome results from der 22 t 11q23 22q11 cleft palate ear anomalies heart defects genital anomalies hypotonia and mental retardation are the main features of the syndrome we report a nine year old boy with the t 11 22 q23 q11 chromosome transmitted in an unbalanced fashion from his mother and originated in the maternal grandmother s meiosis in addition to mental retardation hypotonia craniofacial anomalies and cryptorchidism he has novel findings such as joint hyperextensibility left liver lobe agenesis left sided malposition of the gallbladder and pancreas hypoplasia this is the first report associating these features with emanuel syndrome.\"}, {\"question\": \"List polyubiquitin binding proteins involved in NF-kappaB signaling.\", \"ideal_answer\": [\"NEMO\\nA20\\nABIN-1\\nABIN-2\\noptineurin\\np62\"], \"exact_answer\": [[\"NEMO\"], [\"A20\"], [\"ABIN-1\"], [\"ABIN-2\"], [\"optineurin\"], [\"p62\"]], \"context\": \"he discovery that NEMO is a polyubiquitin-binding protein and that the IKK complex is modulated by other protein kinases that are themselves controlled by polyubiquitin chains has provided a deeper molecular understanding of the non-degradative roles of ubiquitylationEssential roles of K63-linked polyubiquitin-binding proteins TAB2 and TAB3 in B cell activation via MAPKs.Novel polyubiquitin binders AWP1, CALCOCO2, N4BP1, RIO3, TEX27, TTC3, UBFD1 and ZNF313 were identified using this approach, while known NF-kappaB regulators including NEMO, A20, ABIN-1, ABIN-2, optineurin and p62 were also identified\"}, {\"question\": \"What are the prednisone side effects in DMD patients?\", \"ideal_answer\": [\"Side effects of prednisone in DMD patients include reduced growth rate and increase in body weight.\"], \"exact_answer\": [[\"Reduced growth rate\"], [\"Increase in weight\", \"weight gain\"]], \"context\": \" Side effects included a decline in growth rate in the prednisone-treated patients and excessive weight gain in one control and three treated patients.\"}, {\"question\": \"What induces Arabidopsis ROF1 expression?\", \"ideal_answer\": [\"The abundance of ROF1 increased several-fold under stress conditions such as  wounding, heat stress or exposure to elevated NaCl levels.\"], \"exact_answer\": [[\"Wounding\"], [\"Elevated NaCl levels\"], [\"Heat\"]], \"context\": \"Endogenous ROF1 was shown to accumulate in response to high salt treatment in Arabidopsis thaliana young leaves as well as in seedlings germinated on high salt media (0.15 and 0.2 M NaCl) at both an mRNA and protein level. Arabidopsis ROF1 (AtFKBP62) is a peptidyl prolyl cis/trans isomerase and a member of the FKBP (FK506 binding protein) family. ROF1 expression is induced by heat stress and developmentally regulated.Exposure of plants to low levels of MDA using a recently developed protocol powerfully upregulated many genes on a cDNA microarray with a bias towards those implicated in abiotic/environmental stress (e.g. ROF1 and XERO2).Arabidopsis immunophilins ROF1 (AtFKBP62) and ROF2 (AtFKBP65) exhibit tissue specificity, are heat-stress induced, and bind HSP90.The abundance of ROF1 mRNA increased several-fold under stress conditions such as wounding or exposure to elevated NaCl levels.\"}, {\"question\": \"What are the side effects of deflazacort in DMD patients?\", \"ideal_answer\": [\"The side effects observed in DMD patients following deflazacort treatment include growth failure and weight gain, facial fullness, blood pressure, bone health, cataracts, gastrointestinal symptoms, behavior, hypertrichosis, and need for medication interventions.\"], \"exact_answer\": [[\"growth failure\"], [\"weight gain\"], [\"facial fullness\"], [\"blood pressure\"], [\"bone health\"], [\"cataracts\"], [\"gastrointestinal symptoms\"], [\"behavioral\"], [\"hypertrichosis\"], [\"need for medical intervention\"]], \"context\": \"Side effects were growth failure and weight gain, facial fullness, blood pressure, bone health, cataracts, gastrointestinal symptoms, behavior, hypertrichosis, and need for medication interventions.\"}, {\"question\": \"Name the phase 3 clinical trials for tofacitinib in colitis.\", \"ideal_answer\": [\"There are three phase 3, randomized, double-blind, placebo-controlled trials of tofacitinib therapy in adults with ulcerative colitis: OCTAVE Induction 1, OCTAVE Induction 2, OCTAVE Sustain.\", \"OCTAVE Induction 1, OCTAVE Induction 2, and OCTAVE Sustain ClinicalTrials.gov numbers, NCT01465763 , NCT01458951 , and NCT01458574 .\"], \"exact_answer\": [[\"OCTAVE Induction 1\"], [\"OCTAVE Induction 2\"], [\"OCTAVE Sustain\"]], \"context\": \"We conducted three phase 3, randomized, double-blind, placebo-controlled trials of tofacitinib therapy in adults with ulcerative colitis. In the OCTAVE Induction 1 and 2 trials, 598 and 541 patients, respectively, who had moderately to severely active ulcerative colitis despite previous conventional therapy or therapy with a tumor necrosis factor antagonist were randomly assigned to receive induction therapy with tofacitinib (10 mg twice daily) or placebo for 8 weeks. In the OCTAVE Sustain trial, 593 patients who had a clinical response to induction therapy were randomly assigned to receive maintenance therapy with tofacitinib (either 5 mg or 10 mg twice daily) or placebo for 52 weeksIn the OCTAVE Induction 1 trial, remission at 8 weeks occurred in 18.5% of the patients in the tofacitinib group versus 8.2% in the placebo group (P=0.007); in the OCTAVE Induction 2 trial, remission occurred in 16.6% versus 3.6% (P<0.001). In the OCTAVE Sustain trial, remission at 52 weeks occurred in 34.3% of the patients in the 5-mg tofacitinib group and 40.6% in the 10-mg tofacitinib group versus 11.1% in the placebo group (P<0.001 for both comparisons with placebo). In the OCTAVE Induction 1 and 2 trials, the rates of overall infection and serious infection were higher with tofacitinib than with placebo. In the OCTAVE Sustain trial, the rate of serious infection was similar across the three treatment groups, and the rates of overall infection and herpes zoster infection were higher with tofacitinib than with placebo.Funded by Pfizer; OCTAVE Induction 1, OCTAVE Induction 2, and OCTAVE Sustain ClinicalTrials.gov numbers, NCT01465763 , NCT01458951 , and NCT01458574 , respectively.In patients with moderately to severely active ulcerative colitis, tofacitinib was more effective as induction and maintenance therapy than placebo. (Funded by Pfizer; OCTAVE Induction 1, OCTAVE Induction 2, and OCTAVE Sustain ClinicalTrials.gov numbers, NCT01465763 , NCT01458951 , and NCT01458574 , respectively.).\"}, {\"question\": \"Which phase III clinical trials for rheumatoid arthritis involve baricitinib? (November 2017)\", \"ideal_answer\": [\"The phase 3 clinical trials of baricitinib in rheumatoid arthritis patients are: RA-BEACON, RA-BUILD, RA-BEAM.\"], \"exact_answer\": [[\"RA-BEACON\"], [\"RA-BUILD\"], [\"RA-BEAM\"]], \"context\": \"Patient-reported outcomes from a randomised phase III study of baricitinib in patients with rheumatoid arthritis and an inadequate response to biological agents (RA-BEACON)Baricitinib in patients with inadequate response or intolerance to conventional synthetic DMARDs: results from the RA-BUILD study.Patient-reported outcomes from a phase 3 study of baricitinib versus placebo or adalimumab in rheumatoid arthritis: secondary analyses from the RA-BEAM study.\"}, {\"question\": \"List indications for palivizumab for treatment of RSV-induced bronchiolitis. \", \"ideal_answer\": [\"According to guidelines palivizumab is available for treatment of RSV-induced bronchiolitis in high-risk infants: born before 29 weeks' gestation, infants with chronic lung disease of prematurity, and infants and children with hemodynamically significant heart disease. Palivizumab reduces the burden of bronchiolitis but does not prevent infection.\"], \"exact_answer\": [[\"infants born before 29 weeks' gestation\"], [\"infants with chronic lung disease of prematurity\"], [\"infants and children with hemodynamically significant heart disease\"]], \"context\": \" Palivizumab reduces the burden of bronchiolitis but does not prevent infection.Children at risk of severe lower respiratory tract infection should receive immunoprophylaxis with palivizumab, a humanized monoclonal antibody, in up to five monthly doses. Prophylaxis guidelines are restricted to infants born before 29 weeks' gestation, infants with chronic lung disease of prematurity, and infants and children with hemodynamically significant heart disease.Nowadays, only a humanized monoclonal antibody, known as palivizumab, is available to protect against hRSV infection in high-risk infants. Palivizumab, a humanized murine monoclonal antibody that recognizes antigenic site II on both the prefusion (pre-F) and postfusion (post-F) conformations of the respiratory syncytial virus (RSV) F glycoprotein, is the only prophylactic agent approved for use for the treatment of RSV infection. Palivizumab prophylaxis is recommended for infants with qualifying high risk conditions. Recent evidence-based clinical practice guidelines have been published by the American Academy of Pediatrics to guide diagnosis, treatment, and prevention of bronchiolitis.Palivizumab is effective in reducing RSV hospitalization in high risk patient groups. .CONCLUSION: Palivizumab was effective in reducing RSV-related hospitalization infection in CF patients.CONCLUSIONS: Among infants 29 to 32 weeks' gestation without chronic illness, palivizumab use was associated with reduced RSV hospitalizations but increased hospitalizations for bronchiolitis without RSV diagnosis.There is currently no licensed vaccine to prevent RSV infection but passive immunoprophylaxis using a monoclonal antibody, palivizumab, reduces the risk of hospitalization due to RSV infection by 39-78% in various high-risk infants predisposed to developing severe RSV disease.Eligibility for palivizumab prophylaxis in a cohort of children with severe bronchiolitis.n 2014, the American Academy of Pediatrics (AAP) updated their recommendations for palivizumab prophylaxis for children who are at high risk for severe respiratory syncytial virus (RSV) infection. To investigate the potential impact of the more restrictive 2014 criteria on the eligibility for palivizumab prophylaxis, we applied the 2012 and 2014 AAP recommendations for palivizumab prophylaxis to a multicenter cohort of 2207 US children hospitalized for bronchiolitis.The decrease was largely driven by the restriction of eligibility to preterm infants with gestational age<29 weeks.To assess the cost effectiveness of palivizumab, a humanised monoclonal antibody, used as prevention against severe respiratory syncytial virus (RSV) infection requiring hospitalisation, in the indication of preterm infants and infants with preterm/bronchopulmonary dysplasia and in the second indication of children with congenital heart disease in the Dutch healthcare setting.Preventing RSV bronchiolitis in vulnerable infants: the role of palivizumab.Palivizumab a humanized monoclonal antibody directed against the F protein of RSV is the only agent licensed to prevent severe RSV disease in high-risk children.OBJECTIVES Palivizumab, a humanized monoclonal antibody directed against respiratory syncytial virus (RSV), is the only existent immunoprophylaxis therapy for prevention of serious lower respiratory tract disease caused by RSV in infants (up to 2\\u00a0years of age), particularly in those who meet high-risk criteria (preterm infants and/or those with bronchopulmonary or congenital heart disease).CONCLUSION Palivizumab provides cost-effective prophylaxis against RSV in high-risk infants.OBJECTIVE To assess the cost effectiveness of palivizumab, a humanised monoclonal antibody, used as prevention against severe respiratory syncytial virus (RSV) infection requiring hospitalisation, in the indication of preterm infants and infants with preterm/bronchopulmonary dysplasia and in the second indication of children with congenital heart disease in the Dutch healthcare setting.\"}, {\"question\": \"List factors that promote lymphangiogenesis.\", \"ideal_answer\": [\"VEGF-C\\nVEGF-D\\nVEGF-R3\"], \"exact_answer\": [[\"VEGF-C\"], [\"VEGF-D\"], [\"VEGF-R3\"]], \"context\": \"Lymphangiogenesis plays an important role in cancer progression and is regulated by a complex mechanism that includes vascular endothelial growth factor (VEGF) signaling.ur results demonstrate that NHT stimulates lymphangiogenesis via upregulation of VEGF-C and -DVegfc is essential for secondary angiogenesis, giving rise to veins and lymphaticsvegfc and vegfd cooperatively control lymphangiogenesis throughout the embryo, including during the formation of the trunk lymphatic vasculature. The signals for lymphangiogenesis, VEGF-C and its receptor VEGF-R3Vascular endothelial growth factor (VEGF) members VEGF-C and VEGF-D are both potent candidates for stimulating lymphangiogenesis,\"}, {\"question\": \"Which are the main transcriptional activators of circadian oscillations?\", \"ideal_answer\": [\"Mammalian CLOCK and BMAL1 are two members of bHLH-PAS-containing family of transcription factors that represent the positive elements of circadian autoregulatory feedback loop.\", \"BMAL1 and CLOCK.\", \"Mammalian CLOCK and BMAL1 are two members of bHLH-PAS-containing family of transcription factors that represent the positive elements of circadian autoregulatory feedback loop.  We have examined abundance, posttranslational modifications, cellular localization of endogenous and ectopically expressed CLOCK and BMAL1 proteins. Nuclear/cytoplasm distribution of CLOCK was found to be under circadian regulation.\", \"Mammalian CLOCK and BMAL1 are two members of bHLH-PAS-containing family of transcription factors that represent the positive elements of circadian autoregulatory feedback loop. Nuclear/cytoplasm distribution of CLOCK was found to be under circadian regulation.  Two basic helix-loop-helix (bHLH) PAS (for Period-Arnt-Sim) domain-containing transcriptional activators, CLOCK and BMAL1, are known to regulate gene expression by interacting with a promoter element termed the E-box (CACGTG).\", \"Mammalian CLOCK and BMAL1 are two members of bHLH-PAS-containing family of transcription factors that represent the positive elements of circadian autoregulatory feedback loop.  We have examined abundance, posttranslational modifications, cellular localization of endogenous and ectopically expressed CLOCK and BMAL1 proteins. Formation of CLOCK/BMAL1 complex following ectopic coexpression of both proteins is followed by their codependent phosphorylation, which is tightly coupled to CLOCK nuclear translocation and degradation\"], \"exact_answer\": [[\"BMAL1\"], [\"CLOCK\"]], \"context\": \"Mammalian CLOCK and BMAL1 are two members of bHLH-PAS-containing family of transcription factors that represent the positive elements of circadian autoregulatory feedback loop. We have examined abundance, posttranslational modifications, cellular localization of endogenous and ectopically expressed CLOCK and BMAL1 proteins. Nuclear/cytoplasm distribution of CLOCK was found to be under circadian regulation.Formation of CLOCK/BMAL1 complex following ectopic coexpression of both proteins is followed by their codependent phosphorylation, which is tightly coupled to CLOCK nuclear translocation and degradationAltogether, these results provide evidence for an additional level of circadian system control, which is based on regulation of transcriptional activity or/and availability of CLOCK/BMAL1 complex. Two basic helix-loop-helix (bHLH) PAS (for Period-Arnt-Sim) domain-containing transcriptional activators, CLOCK and BMAL1, are known to regulate gene expression by interacting with a promoter element termed the E-box (CACGTG).Circadian rhythms in mammals are generated by a transcriptional negative feedback loop that is driven primarily by oscillations of PER and CRY, which inhibit their own transcriptional activators, CLOCK and BMAL1.Our biochemical evidence supports an elegant mechanism for the disparity: PER2 directly and rhythmically binds to CLOCK:BMAL1, while CRY only interacts indirectly; PER2 bridges CRY and CLOCK:BMAL1 to drive the circadian negative feedback loop.<br>Two basic helix-loop-helix (bHLH) PAS (for Period-Arnt-Sim) domain-containing transcriptional activators, CLOCK and BMAL1, are known to regulate gene expression by interacting with a promoter element termed the E-box (CACGTG).The CRY proteins in turn inhibit CLOCK:BMAL-mediated transcription closing the negative feedback loop.In the vertebrate circadian feedback loop, CLOCK:BMAL heterodimers induce the expression of Cry genes.Two basic helix-loop-helix (bHLH) PAS (for Period-Arnt-Sim) domain-containing transcriptional activators, CLOCK and BMAL1, are known to regulate gene expression by interacting with a promoter element termed the E-box (CACGTG).In the present study, we demonstrate that Id2 is involved in stabilization of the amplitudes of the circadian oscillations by suppressing transcriptional activation of clock genes Clock and Bmal1.bmal1 is an essential transcriptional activator within the mammalian circadian clock we report here that the suprachiasmatic nucleus scn of bmal1 null mutant mice unexpectedly generates stochastic oscillations with periods that overlap the circadian range dissociated scn neurons expressed fluctuating levels of per2 detected by bioluminescence imaging but could not generate circadian oscillations intrinsically inhibition of intercellular communication or cyclic amp signaling in scn slices which provide a positive feed forward signal to drive the intracellular negative feedback loop abolished the stochastic oscillations propagation of this feed forward signal between scn neurons then promotes quasi circadian oscillations that arise as an emergent property of the scn network experimental analysis and mathematical modeling argue that both intercellular coupling and molecular noise are required for the stochastic rhythms providing a novel biological example of noise induced oscillations the emergence of stochastic circadian oscillations from the scn network in the absence of cell autonomous circadian oscillatory function highlights a previously unrecognized level of circadian organization.transcriptional regulation appears to be fundamental to circadian oscillations of clock gene expression these oscillations are believed to control output rhythms the transcriptional feedback loop and a model of interlocked loops have been proposed as the basis for these oscillations we characterized the genomic structure of the mouse bmal1 gene mbmal1 and defined the mbmal1 promoter region transcription of mbmal1 was activated by cry1 cry2 and per2 and was repressed by bmal1 clock dimers therefore cry per2 and bmal1 clock play bidirectional roles in transcription when they are at high levels by late day and midnight respectively this underlies the opposite phase of bmal1 compared to cry and per we propose that a bmal1 negative feedback loop interlocks with the cry and per2 negative feedback loop by inter activation forming a third positive forward loop this transcriptional model suggests a molecular basis for the maintenance of stability persistence and period of circadian rhythms the transcriptional potency of cry is predominant within the mammalian clock suggesting a clearance mechanism for cry in period maintenance c 2002 elsevier science usa.the circadian expression of the mammalian clock genes is based on transcriptional feedback loops two basic helix loop helix bhlh pas for period arnt sim domain containing transcriptional activators clock and bmal1 are known to regulate gene expression by interacting with a promoter element termed the e box cacgtg the non canonical e boxes or e box like sequences have also been reported to be necessary for circadian oscillation we report a new cis element required for cell autonomous circadian transcription of clock genes this new element consists of a canonical e box or a non canonical e box and an e box like sequence in tandem with the latter with a short interval 6 base pairs between them we demonstrate that both e box or e box like sequences are needed to generate cell autonomous oscillation we also verify that the spacing nucleotides with constant length between these 2 e elements are crucial for robust oscillation furthermore by in silico analysis we conclude that several clock and clock controlled genes possess a direct repeat of the e box like elements in their promoter region we propose a novel possible mechanism regulated by double e box like elements not to a single e box for circadian transcriptional oscillation the direct repeat of the e box like elements identified in this study is the minimal required element for the generation of cell autonomous transcriptional oscillation of clock and clock controlled genes.clock bmal independent circadian oscillation of zebrafish cryptochrome1a gene\"}, {\"question\": \"Which clinical trials for psoriasis involved tofacitinib? (November 2017)\", \"ideal_answer\": [\"Four phase 3 clinical trials have been performed to assess tofacitinib in psoriasis patients: OPT Retreatment, OPT Pivotal 1, OPT Pivotal 2, OPT Compare\"], \"exact_answer\": [[\"OPT Retreatment\"], [\"OPT Pivotal 1\"], [\"OPT Pivotal 2\"], [\"OPT Compare\"]], \"context\": \"A Phase 3 withdrawal/re-treatment study (NCT01186744; OPT Retreatment) showed tofacitinib re-treatment was effective in patients with chronic plaque psoriasis.In 2 identical phase 3 studies (OPT Pivotal 1 and 2), patients were randomized 2:2:1 to receive tofacitinib 5\\u00a0mg, tofacitinib 10\\u00a0mg, or placebo, twice daily. To evaluate HRQoL with tofacitinib, vs. placebo or etanercept, in the Phase 3, randomized, placebo-controlled, non-inferiority, Oral-treatment Psoriasis Trial (OPT) Compare Study (NCT01241591).\"}, {\"question\": \"Which two interleukins are inhibited by Ustekinumab?\", \"ideal_answer\": [\"Ustekinumab, a monoclonal antibody that binds to the shared p40 subunit of interleukin-12 and interleukin-23, is approved in the USA and Europe for moderate to severe plaque psoriasis.\"], \"exact_answer\": [[\"interleukin-12\"], [\"interleukin-23\"]], \"context\": \"The safety and efficacy of ustekinumab, a human monoclonal anti-IL-12/23 p40 antibody, and guselkumab, a human monoclonal anti-IL-23 antibody, were evaluated in adults with active RA despite methotrexate (MTX) therapy.The purpose of this review is to update the reader on treatment options for PsA using conventional synthetic disease modifying agents (csDMARDs) and novel therapies including tumour necrosis factor alpha inhibitors, interleukin 12/23 inhibitor (ustekinumab), the interleukin 17 antagonists including secukinumab, brodalumab, ixekizumab, and the phosphodiesterase-4 inhibitor, apremilast. Expert opinion: Based on the rationale of the involvement of the IL-23/Th17 axis in AS, novel biological agents have been developed and include secukinumab, an anti-IL-17A agent and ustekinumab, an anti-IL-23 antibody.Besides systemic corticosteroids and cyclosporine A (CsA), options also include intravenous immunoglobulins (IVIG) and biologics such as the TNF\\u03b1 inhibitors infliximab, adalimumab, and etanercept; the interleukin (IL) 12/23 antibody ustekinumab; the IL-1 receptor antagonist anakinra; and the IL-1\\u03b2 antibody canakinumab.Objective: To determine whether blockade of the interleukin 23-helper T cell 17 (IL-23-TH17) pathway with ustekinumab represents an efficacious and, based on its proinflammatory cytokine profile, targeted treatment option in PRP.The use of monoclonal antibodies against interleukin (IL)-12 and -23, such as ustekinumab, has considerably reduced the disease burden in many patients with moderate to severe psoriasis.Ustekinumab, a monoclonal antibody that binds to the shared p40 subunit of interleukin (IL)-12 and IL-23, is approved in the USA and Europe for moderate to severe plaque psoriasis. The commercialization of tumor necrosis factor inhibitors (adalimumab, certolizumab pegol, etanercept, golimumab, and infliximab) as well as interleukin (IL)-12/23 (ustekinumab) and IL-17 (secukinumab) inhibitors is representative of a revolution in the treatment of PsA. This study is a retrospective analysis of routine parameter dynamics and laboratory adverse events (LAE) in psoriasis patients on long-term treatment (n\\u2009=\\u2009199) with tumour necrosis factor (TNF)-\\u03b1-antagonists (adalimumab, etanercept), and the interleukin (IL)12/23-antagonist ustekinumab. Ustekinumab is a monoclonal antibody directed against the p40 subunit, which is part of interleukins IL-12 and IL-23.The biologic, Ustekinumab (Stelara\\u00ae, Centocor, Inc., Malvern, PA, USA), is a fully human monoclonal antibody with a high affinity for the shared p40 subunit of interleukins 12 and 23 (IL-12 and IL-23).Ustekinumab is a human monoclonal antibody directed against the shared p40 subunit of interleukins 12 and 23.A recent example is ustekinumab, a first-in-class therapeutic human immunoglobulin G1 kappa mAb that binds to the interleukins (IL)-12 and IL-23, cytokines that modulate lymphocyte function, including T-helper (Th) 1 and Th17 cell subsets.Ustekinumab (CNTO 1275, Centocor Inc, Malvern, PA, USA) is a human monoclonal antibody that binds to the shared p40 protein subunit of human interleukins 12 and 23 with high affinity and specificity, thereby preventing interaction with their surface IL-12R\\u03b21 receptor.Discovery and mechanism of ustekinumab: a human monoclonal antibody targeting interleukin-12 and interleukin-23 for treatment of immune-mediated disorders.); (2) Ustekinumab is an inhibitor of interleukins 12 and 23, which are believed to be implicated in the onset of psoriasis.Ustekinumab (Stelara) is a therapeutic monoclonal antibody (mAb) targeted against the p40 shared subunit of IL-12 and IL-23.In the past years, the IL-23/Th17 axis has emerged as an important mechanism in the pathogenesis of PsA. Ustekinumab, a fully human IgG1\\u03ba monoclonal antibody that targets the common subunit p40 of IL-12 and IL-23, has been shown in clinical trials, to be well-tolerated and effective in patients with active PsA.We compared two biologic agents, ustekinumab (an interleukin-12 and interleukin-23 blocker) and etanercept (an inhibitor of tumor necrosis factor alpha), for the treatment of psoriasis.Ustekinumab, a human immunoglobulin G1 kappa (IgG1k) monoclonal antibody that binds with high affinity to human interleukin-12 and interleukin-23, has demonstrated efficacy in patients with psoriasis.Anti-p40 antibodies ustekinumab and briakinumab: blockade of interleukin-12 and interleukin-23 in the treatment of psoriasis.A randomized trial of Ustekinumab, a human interleukin-12/23 monoclonal antibody, in patients with moderate-to-severe Crohn's disease.Ustekinumab is a monoclonal antibody against the p40 subunit of interleukin-12/23.ABT-874 (Abbott Laboratories, Saint-Laurent, QC,) and ustekinumab (CNTO 1275, Ortho Biotech, Toronto, ON) are two monoclonal antibodies against interleukins 12 and 23 (IL-12/23), key mediators of T-cell differentiation in the pathogenesis of psoriasis.Ustekinumab is a fully human immunoglobulin G1\\u03ba monoclonal antibody that targets the common p40 subunit that is shared by both interleukin (IL)-12 and IL-23, consequently inhibiting T-cell differentiation along both Th1 and Th17 pathways.Ustekinumab is a fully human monoclonal antibody directed against the p40 subunit shared by interleukin 12 and interleukin 23, two naturally occurring protein regulators that play an important role in immune-mediated inflammatory diseases, including psoriatic arthritis (PsA).BACKGROUND Ustekinumab, a fully human immunoglobulin (Ig) G1K monoclonal antibody directed against the p40 subunit of interleukin (IL)-12/23, has demonstrated efficacy in patients with moderate-to-severe psoriasis.Ustekinumab (CNTO 1275, Centocor Inc, Malvern, PA, USA) is a human monoclonal antibody that binds to the shared p40 protein subunit of human interleukins 12 and 23 with high affinity and specificity, thereby preventing interaction with their surface IL-12R\\u03b21 receptor.Ustekinumab is a monoclonal antibody belonging to a newly developed class of biological, anti-cytokine medications that notably targets the p40 subunit of both IL-12 and -23, both naturally occurring proteins that are important in regulating the immune system and are understood to play a role in immune-mediated inflammatory disorders.Ustekinumab binds to the p40 subunit common to IL-12 and IL-23 and prevents their interaction with the IL-12 receptor \\u03b21 subunit of the IL-12 and IL-23 receptor complexes.INTRODUCTION Ustekinumab is a human monoclonal antibody directed against the shared p40 subunit of interleukins 12 and 23.BACKGROUND Ustekinumab is a human monoclonal antibody that binds to the p40 subunit of interleukin (IL) 12 and IL-23 and inhibits their pharmacological activity.Ustekinumab is a fully human monoclonal antibody that binds with high specificity and affinity to the cytokines interleukin (IL)-12 and IL-23, thereby suppressing IL-12- and IL-23-mediated inflammation associated with psoriasis.Two human anti-p40 monoclonal antibodies targeting both IL-12 and IL-23 via their shared p40 subunit have been developed: briakinumab and ustekinumab.evaluate ustekinumab an anti interleukin il 12 and il 23 antibody effects on radiographic progression in psoriatic arthritis psa we conducted preplanned integrated analyses of combined radiographic data from psummit 1 and psummit 2 phase 3 randomised controlled trials patients had active psa despite prior conventional and or biologic disease modifying antirheumatic drugs 5 66 swollen 5 68 tender joints c reactive protein 3 0 mg l documented plaque psoriasis patients psummit 1 n 615 psummit 2 n 312 were randomised to ustekinumab 45 mg 90 mg or placebo at weeks wk 0 4 and every q 12 wks at wk 16 patients with 5 improvement in tender swollen joint counts entered blinded early escape all other placebo patients received ustekinumab 45 mg at wk 24 and wk 28 then q 12 wks radiographs of hands feet at wks 0 24 52 were assessed using psa modified van der heijde sharp vdh s scores combined psummit 1 and psummit 2 changes in total vdh s scores from wk 0 to wk 24 comprised the prespecified primary radiographic analysis treatment effects were assessed using analysis of variance on van der waerden normal scores factors treatment baseline methotrexate usage and study integrated data analysis results indicated that ustekinumab treated patients regardless of dose demonstrated significantly less radiographic progression at wk 24 than did placebo recipients wk 0 24 total vdh s score mean changes 0 4 combined individual ustekinumab dose groups 1 0 placebo all p 0 02 from wk 24 to wk 52 inhibition of radiographic progression was maintained for ustekinumab treated patients and progression was substantially reduced among initial placebo recipients who started ustekinumab at wk 16 or wk 24 wk 24 wk 52 total vdh s score mean change 0 08 ustekinumab 45 and 90 mg treatments significantly inhibited radiographic progression of joint damage in patients with active psa.discovery and mechanism of ustekinumab a human monoclonal antibody targeting interleukin 12 and interleukin 23 for treatment of immune mediated disordersa randomized trial of ustekinumab a human interleukin 12 23 monoclonal antibody in patients with moderate to severe crohn s diseasehuman leucocyte antigen cw6 as a predictor for clinical response to ustekinumab an interleukin 12 23 blocker in chinese patients with psoriasis a retrospective analysisustekinumab a human immunoglobulin g1 kappa igg1k monoclonal antibody that binds with high affinity to human interleukin 12 and interleukin 23 has demonstrated efficacy in patients with psoriasis the objective of this study was to perform exposure response modeling to increase the understanding of reduction in disease severity following treatment with ustekinumab in patients with moderate to severe psoriasis who participate in two phase iii studies phoenix 1 and phoenix 2 patients were randomly assigned to receive ustekinumab 45 mg or 90 mg n 1312 11 624 psoriasis area and severity index pasi scores or placebo n 665 3278 pasi scores disease severity was assessed using pasi scores a population mechanism based exposure response model of ustekinumab using nonmem was developed using serum ustekinumab concentrations and pasi scores the pharmacodynamic response effect was the reduction in pasi score the placebo effect although minor was also integrated into the model none of the covariate factors evaluated eg demographics baseline disease characteristics comorbidities significantly contributed to the between subject variability in the pharmacodynamic parameters the developed exposure response model can serve as a basis to support future alternative dosing regimens for ustekinumab in patients with moderate to severe plaque psoriasis a robust exposure response relationship has been confirmed for ustekinumab in psoriasis.interleukin 23 is thought to be critical to the pathogenesis of psoriasis we compared risankizumab bi 655066 a humanized igg1 monoclonal antibody that inhibits interleukin 23 by specifically targeting the p19 subunit and thus prevents interleukin 23 signaling and ustekinumab an interleukin 12 and interleukin 23 inhibitor in patients with moderate to severe plaque psoriasis we randomly assigned a total of 166 patients to receive subcutaneous injections of risankizumab a single 18 mg dose at week 0 or 90 mg or 180 mg doses at weeks 0 4 and 16 or ustekinumab 45 or 90 mg according to body weight at weeks 0 4 and 16 the primary end point was a 90 or greater reduction from baseline in the psoriasis area and severity index pasi score at week 12 at week 12 the percentage of patients with a 90 or greater reduction in the pasi score was 77 64 of 83 patients for risankizumab 90 mg and 180 mg groups pooled as compared with 40 16 of 40 patients for ustekinumab p 0 001 the percentage of patients with a 100 reduction in the pasi score was 45 in the pooled 90 mg and 180 mg risankizumab groups as compared with 18 in the ustekinumab group efficacy was generally maintained up to 20 weeks after the final dose of 90 or 180 mg of risankizumab in the 18 mg and 90 mg risankizumab groups and the ustekinumab group 5 patients 12 6 patients 15 and 3 patients 8 respectively had serious adverse events including two basal cell carcinomas and one major cardiovascular adverse event there were no serious adverse events in the 180 mg risankizumab group in this phase 2 trial selective blockade of interleukin 23 with risankizumab was associated with clinical responses superior to those associated with ustekinumab this trial was not large enough or of long enough duration to draw conclusions about safety funded by boehringer ingelheim clinicaltrials gov number nct02054481.1 for adults with plaque psoriasis after failure of topical symptomatic treatments and puva therapy several systemic immunosuppressive agents are acceptable for severe disease methotrexate then ciclosporin and possible a tnf alpha antagonist etanercept etc 2 ustekinumab is an inhibitor of interleukins 12 and 23 which are believed to be implicated in the onset of psoriasis it is authorized in the european union for patients who fail to respond to conventional systemic treatments 3 in one trial with a low level of evidence single blind 2 subcutaneous injections of ustekinumab at an interval of 4 weeks appeared to be statistically more effective than twice weekly subcutaneous injections of etanercept for 12 weeks more patients achieved a 75 reduction in the score most widely used to evaluate the extent and intensity of plaque psoriasis lesions pasi score about 71 versus 57 the results beyond this period have not been reported 4 two randomised double blind placebo controlled trials in a total of 1996 patients showed that at least two thirds of patients treated with ustekinumab achieved at least a 75 reduction in their pasi score versus fewer than 4 with placebo 5 in animal studies interleukin 12 and 23 inhibitors cause cancer there is therefore a high risk of cancer developing during prolonged treatment with ustekinumab 6 the main adverse effects identified in clinical trials include infections injection site reactions psychological disorders and development of anti ustekinumab antibodies 7 there is insufficient follow up to evaluate the cardiac risks associated with ustekinumab 8 as maintenance therapy ustekinumab is administered as one subcutaneous injection every 12 weeks this practical advantage compared to tnf alpha antagonists must be weighed against the risks inherent in prolonged immunosuppression 9 in summary for symptomatic relief of patients whose psoriasis poses major problems despite treatment with methotrexate or ciclosporin in the absence of a better alternative it is better to use a tnf alpha antagonist and to avoid exposing patients to the risks associated with ustekinumab particularly its carcinogenic risk.biologic agents offer a range of new therapeutic options for patients with psoriasis however the relative benefit risk profiles of such therapies are not well known we compared two biologic agents ustekinumab an interleukin 12 and interleukin 23 blocker and etanercept an inhibitor of tumor necrosis factor alpha for the treatment of psoriasis we randomly assigned 903 patients with moderate to severe psoriasis to receive subcutaneous injections of either 45 or 90 mg of ustekinumab at weeks 0 and 4 or high dose etanercept 50 mg twice weekly for 12 weeks the primary end point was the proportion of patients with at least 75 improvement in the psoriasis area and severity index pasi at week 12 a secondary end point was the proportion with cleared or minimal disease on the basis of the physician s global assessment assessors were unaware of the treatment assignments the efficacy and safety of a crossover from etanercept to ustekinumab were evaluated after week 12 there was at least 75 improvement in the pasi at week 12 in 67 5 of patients who received 45 mg of ustekinumab and 73 8 of patients who received 90 mg as compared with 56 8 of those who received etanercept p 0 01 and p 0 001 respectively similarly 65 1 of patients who received 45 mg of ustekinumab and 70 6 of patients who received 90 mg of ustekinumab had cleared or minimal disease according to the physician s global assessment as compared with 49 0 of those who received etanercept p 0 001 for both comparisons among patients who did not have a response to etanercept 48 9 had at least 75 improvement in the pasi within 12 weeks after crossover to ustekinumab one or more adverse events occurred through week 12 in 66 0 of patients who received 45 mg of ustekinumab and 69 2 of patients who received 90 mg of ustekinumab and in 70 0 who received etanercept 1 9 1 2 and 1 2 respectively had serious adverse events safety patterns were similar before and after crossover from etanercept to ustekinumab the efficacy of ustekinumab at a dose of 45 or 90 mg was superior to that of high dose etanercept over a 12 week period in patients with psoriasis clinicaltrials gov number nct00454584.repeated subcutaneous injections of a monoclonal antibody against the p40 subunit of interleukins 12 and 23 ustekinumab were used to treat patients with relapsing remitting multiple sclerosis rrms to assess the drug s safety efficacy and pharmacokinetics in this phase ii multicentre randomised double blind placebo controlled study 249 patients with rrms aged 18 65 years were eligible to be assigned equally by a central randomisation procedure based on study site and presence or absence of gadolinium enhancing t1 weighted lesions at baseline to one of five groups that received placebo or four different ustekinumab dosages at weeks 0 1 2 3 7 11 15 and 19 ustekinumab doses were 27 mg 90 mg q8w 90 mg or 180 mg the 90 mg q8w dosage group received placebo substitute at weeks 7 and 15 the primary endpoint was the cumulative number of new gadolinium enhancing t1 weighted lesions on serial cranial mri through week 23 patients were followed up through week 37 analysis was by intention to treat this trial is registered with clinicaltrials gov number nct00207727 from august 2004 to december 2006 249 patients underwent randomisation 49 for placebo 50 for each ustekinumab group ustekinumab treatment did not show a significant reduction in the primary endpoint for any dosage groups versus placebo at week 37 adverse events occurred in 38 78 placebo treated patients and 170 85 ustekinumab treated patients with infections most commonly reported serious adverse events occurred in one 2 placebo treated patient and six 3 ustekinumab treated patients malignant diseases were reported in two patients shortly after the initiation of ustekinumab treatment both patients were withdrawn from the trial and given appropriate treatment which resulted in complete remission no serious infections cardiovascular events or exacerbation of demyelinating events occurred a dose dependent increase in serum concentrations of ustekinumab was recorded ustekinumab is generally well tolerated but does not show efficacy in reducing the cumulative number of gadolinium enhancing t1 weighted lesions in multiple sclerosis.interleukin 12 and interleukin 23 are inflammatory cytokines implicated in crohn s disease pathophysiology ustekinumab is a monoclonal antibody against the p40 subunit of interleukin 12 23 we performed a double blind cross over trial of the clinical effects of ustekinumab in 104 patients with moderate to severe crohn s disease population 1 patients were given subcutaneous placebo at weeks 0 3 then ustekinumab at weeks 8 11 subcutaneous ustekinumab at weeks 0 3 then placebo at weeks 8 11 intravenous placebo at week 0 then ustekinumab at week 8 or intravenous ustekinumab at week 0 then placebo at week 8 furthermore an open label trial evaluated the effects of 4 weekly subcutaneous injections or 1 intravenous infusion of ustekinumab in 27 patients who were primary or secondary nonresponders to infliximab population 2 in population 1 clinical response rates for the combined groups given ustekinumab and placebo were 53 and 30 p 02 respectively at weeks 4 and 6 and 49 and 40 p 34 respectively at week 8 in a subgroup of 49 patients who were previously given infliximab neither primary nor secondary nonresponders clinical response to ustekinumab was significantly greater than the group given placebo p 05 through week 8 in population 2 the clinical responses at week 8 to subcutaneous and intravenous ustekinumab were 43 and 54 respectively there was no increase in the number of adverse or serious adverse events in patients given ustekinumab through week 8 compared with placebo ustekinumab induced a clinical response in patients with moderate to severe crohn s disease especially in patients previously given infliximab.anti p40 antibodies ustekinumab and briakinumab blockade of interleukin 12 and interleukin 23 in the treatment of psoriasisefficacy and safety of ustekinumab a human interleukin 12 23 monoclonal antibody in patients with psoriasis 76 week results from a randomised double blind placebo controlled trial phoenix 1ustekinumab is a fully human monoclonal antibody directed against the p40 subunit shared by interleukin 12 and interleukin 23 two naturally occurring protein regulators that play an important role in immune mediated inflammatory diseases including psoriatic arthritis psa in september of 2009 the us fda approved ustekinumab for the treatment of adult patients with moderate to severe plaque psoriasis beginning in november of 2009 janssen biotech formerly centocor biotech the developer of ustekinumab initiated clinical trials to investigate the efficacy of ustekinumab in the treatment of other inflammatory disorders including psa phase ii and phase iii studies showed both a good safety profile and significant efficacy for ustekinumab in the treatment of psa leading to the drug s approval in both europe and the usa in an immunotherapy market currently dominated by anti tnf \\u03b1 drugs for the treatment of psa ustekinumab offers an alternative option for patients with psa including those unresponsive to methotrexate and the tnf \\u03b1 inhibitory agents currently approved for this potentially debilitating disease.ustekinumab is a fully human monoclonal antibody that binds with high specificity and affinity to the cytokines interleukin il 12 and il 23 thereby suppressing il 12 and il 23 mediated inflammation associated with psoriasis in two large phase iii trials in patients with moderate to severe plaque psoriasis significantly more subcutaneous ustekinumab 45 or 90 mg recipients administered as two injections 4 weeks apart than placebo recipients achieved a 75 improvement on the psoriasis area and severity index pasi 75 score at 12 weeks other efficacy measures including the physician s global assessment of clinical response at week 12 also favored ustekinumab over placebo psoriatic symptom control was maintained during ustekinumab maintenance therapy administered once every 12 weeks for up to 76 weeks in a phase ii trial in patients with active plaque psoriasis and psoriatic arthritis signs and symptoms of arthritis and psoriatic symptom control were improved to a greater extent with ustekinumab than with placebo at 12 weeks based on the proportion of patients achieving a 20 improvement in american college of rheumatology response criteria arthritis or pasi 75 skin symptoms health related quality of life assessed using the dermatology life quality index and the health assessment questionnaire disability index was improved to a significantly greater extent with ustekinumab than with placebo at week 12 subcutaneous ustekinumab was generally well tolerated in clinical trials with most treatment emergent adverse events being of mild severity.the choice of therapeutic agents for patients with moderate to severe psoriasis has expanded significantly in the past decade with new understanding of the immunologic basis of psoriasis multiple new potential targets for therapy have been identified it is likely that a series of new medications to focus on the newly identified pathways is on the horizon the first pathway targeted by new medications focuses on the p40 subunit that is shared by interleukin il 12 and il 23 two human anti p40 antibodies have been used therapeutically in psoriasis to date ustekinumab cnto 1275 stelara centocor horsham pa and briakinumab abt 874 abbott abbott park il ustekinumab was recently approved by the united states food and drug administration making it the first medication approved in the united states to work by this pathway while briakinumab is currently in phase iii clinical trials.\"}, {\"question\": \"List proteins that are targeted by \\\"immune checkpoints inhibitors\\\".\", \"ideal_answer\": [\"The treatment landscape of advanced melanoma has changed significantly following the discovery and marketing authorisation of immune checkpoints inhibitors. Ipilimumab (anti-CTLA-4) was the first one to be approved, and it. demonstrated long-term survival in about 20% of patients. Subsequently, anti-programmed cell death-1 (a-PD-1) antibodies (pembrolizuamb, nivolumab), inhibitors of PD-1/programmed cell death-1 ligand (PD1-L) synapse, showed higher clinical efficacy with lower toxicity comparing to ipilimumab.\"], \"exact_answer\": [[\"CTLA-4\"], [\"PD-1\"], [\"PD1-L\"]], \"context\": \"The recent discovery of immune checkpoints inhibitors, especially anti-programmed cell death protein 1 (PD-1) and anti-programmed cell death protein ligand 1 (PD-L1) monoclonal antibodies, has opened new scenarios in the management of non-small cell lung cancer (NSCLC) and this new class of drugs has achieved a rapid development in the treatment of this disease.  S100A9-targeting agents with immune checkpoints inhibitors,Programmed cell death 1 checkpoint inhibitors in the treatment of patients with advanced melanoma.The treatment landscape of advanced melanoma has changed significantly following the discovery and marketing authorisation of immune checkpoints inhibitors. Ipilimumab (anti-CTLA-4) was the first one to be approved, and it. demonstrated long-term survival in about 20% of patients. Subsequently, anti-programmed cell death-1 (a-PD-1) antibodies (pembrolizuamb, nivolumab), inhibitors of PD-1/programmed cell death-1 ligand (PD1-L) synapse, showed higher clinical efficacy with lower toxicity comparing to ipilimumab.\"}, {\"question\": \"Which genes belong to the AUX/IAA family of transcription repressors in plants?\", \"ideal_answer\": [\"The Aux/IAA proteins are auxin-sensitive repressors that mediate diverse physiological and developmental processes in plants [1, 2]. There are 29 Aux/IAA genes in Arabidopsis that exhibit unique but partially overlapping patterns of expression Plant Stress Tolerance Requires Auxin-Sensitive Aux/IAA Transcriptional Repressors\", \"The Aux/IAA proteins are auxin-sensitive repressors that mediate diverse physiological and developmental processes in plants [1, 2]. There are 29 Aux/IAA genes in Arabidopsis that exhibit unique but partially overlapping patterns of expression\", \"The Aux/IAA proteins are auxin-sensitive repressors that mediate diverse physiological and developmental processes in plants [1, 2]. Several DREB/CBF TFs directly promote transcription of the IAA5 and IAA19 genes in response to abiotic stress.  Auxin enhances the binding of Aux/IAA proteins to the receptor TIR1, which is an F-box protein that is part of the E3 ubiquitin ligase complex SCF(TIR1).\", \"Auxin is sensed by SCFTIR1-IAA6 and SCFTIR1-IAA19 co-receptor complexes, which leads to IAA6/IAA19 ubiquitylation in vitro and IAA6/IAA19 degradation in vivo. EgrIAA4 protein is localized in the nucleus and functions as an auxin-responsive repressor. Evidence also exists for SCF(TIR1)-mediated ubiquitination of the Aux/IAA proteins SHY2/IAA3 and BDL/IAA12\", \"The Aux/IAA proteins are auxin-sensitive repressors that mediate diverse physiological and developmental processes in plants [1, 2]. Auxin dose-response assays revealed that IAA9 downregulated lines were hypersensitive to auxin, although the only early auxin-responsive gene that was found to be upregulated in the antisense lines was IAA3. We report here that the downregulation of IAA9, a tomato (Solanum lycopersicum) gene from a distinct subfamily of Aux/IAA genes, results in a pleiotropic phenotype, consistent with its ubiquitous expression pattern. While no mutation in any member of subfamily IV has been reported to date, the phenotypes associated with the downregulation of IAA9 reveal distinct and novel roles for members of the Aux/IAA gene family. Aux/IAA proteins are short-lived nuclear proteins that repress expression of primary/early auxin response genes in protoplast transfection assays. Here, we systematically dissect auxin sensing by SCFTIR1-IAA6 and SCFTIR1-IAA19 co-receptor complexes, and assess IAA6/IAA19 ubiquitylation in vitro and IAA6/IAA19 degradation in vivo. Binding of Aux/IAA proteins leads to degradation via the 26S proteasome, but evidence for SCF(TIR1)-mediated poly-ubiquitination of Aux/IAA proteins is lacking. \", \"The Aux/IAA proteins are auxin-sensitive repressors that mediate diverse physiological and developmental processes in plants [1, 2]. The tomato Aux/IAA transcription factor IAA9 is involved in fruit development and leaf morphogenesis. There are 29 Aux/IAA genes in Arabidopsis that exhibit unique but partially overlapping patterns of expression. Several DREB/CBF TFs directly promote transcription of the IAA5 and IAA19 genes in response to abiotic stress. Plant Stress Tolerance Requires Auxin-Sensitive Aux/IAA Transcriptional Repressors. While no mutation in any member of subfamily IV has been reported to date, the phenotypes associated with the downregulation of IAA9 reveal distinct and novel roles for members of the Aux/IAA gene family. Nce for SCF-mediated ubiquitination of the Aux/IAA proteins SHY2/IAA3 and BDL/IAA12. Aux/IAA proteins are short-lived nuclear proteins that repress expression of primary/early auxin response genes in protoplast transfection assays. We showed that EgrIAA4 protein is localized in the nucleus and functions as an auxin-responsive repressor. Here, we systematically dissect auxin sensing by SCFTIR1-IAA6 and SCFTIR1-IAA19 co-receptor complexes, and assess IAA6/IAA19 ubiquitylation in vitro and IAA6/IAA19 degradation in vivo. \", \"Plant Stress Tolerance Requires Auxin-Sensitive Aux/IAA Transcriptional Repressors. The Aux/IAA proteins are auxin-sensitive repressors that mediate diverse physiological and developmental processes in plants [1, 2]. There are 29 Aux/IAA genes in Arabidopsis that exhibit unique but partially overlapping patterns of expression. The tomato Aux/IAA transcription factor IAA9 is involved in fruit development and leaf morphogenesis. Several DREB/CBF TFs directly promote transcription of the IAA5 and IAA19 genes in response to abiotic stress. Here, we systematically dissect auxin sensing by SCFTIR1-IAA6 and SCFTIR1-IAA19 co-receptor complexes, and assess IAA6/IAA19 ubiquitylation in vitro and IAA6/IAA19 degradation in vivo. We showed that EgrIAA4 protein is localized in the nucleus and functions as an auxin-responsive repressor. Auxin dose-response assays revealed that IAA9 downregulated lines were hypersensitive to auxin, although the only early auxin-responsive gene that was found to be upregulated in the antisense lines was IAA3. While no mutation in any member of subfamily IV has been reported to date, the phenotypes associated with the downregulation of IAA9 reveal distinct and novel roles for members of the Aux/IAA gene family. \"], \"exact_answer\": [[\"IAA6\", \"IAA6\"], [\"IAA9\", \"IAA9\"], [\"IAA4\"], [\"SCIFTR1\"], [\"SHY2/IAA3\", \"IAA3\"], [\"BDL/IAA12\", \"IAA12\"]], \"context\": \"The Aux/IAA proteins are auxin-sensitive repressors that mediate diverse physiological and developmental processes in plants [1, 2]. There are 29 Aux/IAA genes in Arabidopsis that exhibit unique but partially overlapping patterns of expressionPlant Stress Tolerance Requires Auxin-Sensitive Aux/IAA Transcriptional RepressorsSeveral DREB/CBF TFs directly promote transcription of the IAA5 and IAA19 genes in response to abiotic stress. Here, we systematically dissect auxin sensing by SCFTIR1-IAA6 and SCFTIR1-IAA19 co-receptor complexes, and assess IAA6/IAA19 ubiquitylation in vitro and IAA6/IAA19 degradation in vivo.We showed that EgrIAA4 protein is localized in the nucleus and functions as an auxin-responsive repressor.Auxin-induced, SCF(TIR1)-mediated poly-ubiquitination marks AUX/IAA proteins for degradation.evidence for SCF(TIR1)-mediated ubiquitination of the Aux/IAA proteins SHY2/IAA3 and BDL/IAA12Auxin enhances the binding of Aux/IAA proteins to the receptor TIR1, which is an F-box protein that is part of the E3 ubiquitin ligase complex SCF(TIR1). Binding of Aux/IAA proteins leads to degradation via the 26S proteasome, but evidence for SCF(TIR1)-mediated poly-ubiquitination of Aux/IAA proteins is lackingThe tomato Aux/IAA transcription factor IAA9 is involved in fruit development and leaf morphogenesisWe report here that the downregulation of IAA9, a tomato (Solanum lycopersicum) gene from a distinct subfamily of Aux/IAA genes, results in a pleiotropic phenotype, consistent with its ubiquitous expression patternAuxin dose-response assays revealed that IAA9 downregulated lines were hypersensitive to auxin, although the only early auxin-responsive gene that was found to be upregulated in the antisense lines was IAA3The activity of the IAA3 promoter was stimulated in the IAA9 antisense genetic background, indicating that IAA9 acts in planta as a transcriptional repressor of auxin signaling. While no mutation in any member of subfamily IV has been reported to date, the phenotypes associated with the downregulation of IAA9 reveal distinct and novel roles for members of the Aux/IAA gene family.Aux/IAA proteins are short-lived nuclear proteins that repress expression of primary/early auxin response genes in protoplast transfection assays.AUX/IAA proteins are active repressors, and their stability and activity are modulated by auxin.The Aux/IAA proteins are auxin-sensitive repressors that mediate diverse physiological and developmental processes in plants [1, 2].There are 29 Aux/IAA genes in Arabidopsis that exhibit unique but partially overlapping patterns of expression [3].Functional characterization of SlIAA15, a member of the tomato (Solanum lycopersicum)Functional characterization of SlIAA15, a member of the tomato (Solanum lycopersicum) Aux/IAA family, shows that the encoded protein acts as a strong repressor of auxin-dependent transcription.The Aux/IAA genes are best characterized among the early auxin-responsive genes, which encode short-lived transcriptional repressors.plant stress tolerance requires auxin sensitive aux iaa transcriptional repressorsgenome wide analysis and characterization of aux iaa family genes in brassica rapathe aux iaa proteins are auxin sensitive repressors that mediate diverse physiological and developmental processes in plants 1 2 there are 29 aux iaa genes in arabidopsis that exhibit unique but partially overlapping patterns of expression 3 although some studies have suggested that individual aux iaa genes have specialized function genetic analyses of the family have been limited by the scarcity of loss of function phenotypes 4 furthermore with a few exceptions our knowledge of the factors that regulate aux iaa expression is limited 1 5 we hypothesize that transcriptional control of aux iaa genes plays a central role in the establishment of the auxin signaling pathways that regulate organogenesis growth and environmental response here we describe a screen for transcription factors tfs that regulate the aux iaa genes we identify tfs from 38 families including 26 members of the dreb cbf family several dreb cbf tfs directly promote transcription of the iaa5 and iaa19 genes in response to abiotic stress recessive mutations in these iaa genes result in decreased tolerance to stress conditions demonstrating a role for auxin in abiotic stress our results demonstrate that stress pathways interact with the auxin gene regulatory network grn through transcription of the aux iaa genes we propose that the aux iaa genes function as hubs that integrate genetic and environmental information to achieve the appropriate developmental or physiological outcome.auxin is a central hormone that exerts pleiotropic effects on plant growth including the development of roots shoots flowers and fruit the perception and signaling of the plant hormone auxin rely on the cooperative action of several components among which auxin indole 3 acetic acid aux iaa proteins play a pivotal role in this study we identified and comprehensively analyzed the entire aux iaa gene family in tomato solanum lycopersicum a reference species for solanaceae plants and the model plant for fleshy fruit development functional characterization using a dedicated single cell system revealed that tomato aux iaa proteins function as active repressors of auxin dependent gene transcription with however different aux iaa members displaying varying levels of repression phylogenetic analysis indicated that the aux iaa gene family is slightly contracted in tomato compared with arabidopsis with a lower representation of non canonical proteins sl iaa genes display distinctive expression pattern in different tomato organs and tissues and some of them display differential responses to auxin and ethylene suggesting that aux iaas may play a role in linking both hormone signaling pathways the data presented here shed more light on sl iaa genes and provides new leads towards the elucidation of their function during plant development and in mediating hormone cross talk.the aux iaa genes encode a large family of short lived proteins known to regulate auxin signalling in plants functional characterization of sliaa15 a member of the tomato solanum lycopersicum aux iaa family shows that the encoded protein acts as a strong repressor of auxin dependent transcription the physiological significance of sliaa15 was addressed by a reverse genetics approach revealing that sliaa15 plays multiple roles in plant developmental processes the sliaa15 down regulated lines display lower trichome number reduced apical dominance with associated modified pattern of axillary shoot development increased lateral root formation and decreased fruit set moreover the leaves of sliaa15 inhibited plants are dark green and thick with larger pavement cells longer palisade cells and larger intercellular space of spongy mesophyll cells the sliaa15 suppressed plants exhibit a strong reduction in type i v and vi trichome formation suggesting that auxin dependent transcriptional regulation is required for trichome initiation concomitant with reduced trichome formation the expression of some r2r3 myb genes putatively involved in the control of trichome differentiation is altered these phenotypes uncover novel and specialized roles for aux iaas in plant developmental processes clearly indicating that members of the aux iaa gene family in tomato perform both overlapping and specific functions.auxins are the key players in plant growth development involving leaf formation phototropism root fruit and embryo development auxin indole 3 acetic acid aux iaa are early auxin response genes noted as transcriptional repressors in plant auxin signaling however many studies focus on aux arf gene families and much less is known about the aux iaa gene family in brassica rapa b rapa here we performed a comprehensive genome wide analysis and identified 55 aux iaa genes in b rapa using four conserved motifs of aux iaa family pf02309 chromosomal mapping of the b rapa aux iaa briaa genes facilitated understanding cluster rearrangement of the crucifer building blocks in the genome phylogenetic analysis of briaa with arabidopsis thaliana oryza sativa and zea mays identified 51 sister pairs including 15 same species briaa briaa and 36 cross species briaa atiaa iaa genes among the 55 briaa genes expression of 43 and 45 genes were verified using genebank b rapa ests and in home developed microarray data from mature leaves of chiifu and rcbr lines despite their huge morphological difference tissue specific expression analysis of briaa genes between the parental lines chiifu and rcbr showed that the genes followed a similar pattern of expression during leaf development and a different pattern during bud flower and siliqua development stages the response of the briaa genes to abiotic and auxin stress at different time intervals revealed their involvement in stress response single nucleotide polymorphisms between iaa genes of reference genome chiifu and rcbr were focused and identified our study examines the scope of conservation and divergence of aux iaa genes and their structures in b rapa analyzing the expression and structural variation between two parental lines will significantly contribute to functional genomics of brassica crops and we belive our study would provide a foundation in understanding the aux iaa genes in b rapa.the plant hormone auxin plays a central role in regulating many aspects of plant growth and development this largely occurs as a consequence of changes in gene expression the aux iaa genes are best characterized among the early auxin responsive genes which encode short lived transcriptional repressors in most plants examined including arabidopsis soybean and rice the aux iaa genes constitute a large gene family by screening the available databases at least 15 expressed sequence tags ests have been identified from wheat triticum aestivum which exhibit high sequence identity with aux iaa homologues in other species one of these aux iaa genes taiaa1 harbouring all the four conserved domains characteristic of the aux iaa proteins has been characterized in detail the expression of taiaa1 is light sensitive tissue specific and is induced within 15 30 min of exogenous auxin application also the taiaa1 transcript levels increase in the presence of a divalent cation ca 2 and this effect is reversed by the calcium chelating agent egta the taiaa1 gene qualifies as the primary response gene because an increase in its transcript levels by auxin is unaffected by cycloheximide in addition to auxin the taiaa1 gene is also induced by brassinosteroid providing evidence that interplay between hormones is crucial for the regulation of plant growth and development.auxin regulates various aspects of plant growth and development the auxin indole 3 acetic acid aux iaa genes encode short lived transcriptional repressors that are targeted by the transport inhibitor response1 auxin receptor f box proteins the aux iaa proteins regulate auxin mediated gene expression by interacting with members of the auxin response factor protein family aux iaa function is poorly understood herein we report the identification and characterization of insertion mutants in 12 of the 29 aux iaa family members the mutants show no visible developmental defects compared with the wild type double or triple mutants of closely related aux iaa genes such as iaa8 1 iaa9 1 or iaa5 1 iaa6 1 iaa19 1 also exhibit wild type phenotypes global gene expression analysis reveals that the molecular phenotypes of auxin treated and untreated light grown seedlings are unaffected in the iaa17 6 and iaa5 1 iaa6 1 iaa19 1 mutants by contrast similar analysis with the gain of function axr3 1 iaa17 1 mutant seedlings reveals dramatic changes in basal and auxin induced gene expression compared with the wild type expression of several type a arabidopsis response regulator genes and a number of genes involved in cell wall biosynthesis and degradation is repressed in axr3 1 iaa17 1 the data suggest extensive functional redundancy among aux iaa gene family members and that enhanced stability of the axr3 iaa17 protein severely alters the molecular phenotype resulting in developmental defects.the regulatory interactions between auxin response factors arfs and aux iaa repressors play a central role in auxin signal transduction yet the systems properties of this regulatory network are not well established we generated a steroid inducible arf5 monopteros mp transgenic background to survey the involvement of this factor in the transcriptional regulation of the entire aux iaa family in arabidopsis thaliana target genes of arf5 mp identified by this approach were confirmed by chromatin immunoprecipitation in vitro gel retardation assays and gene expression analyses our study shows that arf5 mp is indispensable for the correct regulation of nearly one half of all aux iaa genes and that these targets coincide with distinct subclades further genetic analyses demonstrate that the protein products of multiple aux iaa targets negatively feed back onto arf5 mp activity this work indicates that arf5 mp broadly influences the expression of the aux iaa gene family and suggests that such regulation involves the activation of specific subsets of redundantly functioning factors these groups of factors may then act together to control various processes within the plant through negative feedback on arf5 similar detailed analyses of other aux iaa arf regulatory modules will be required to fully understand how auxin signal transduction influences virtually every aspect of plant growth and development.a well known physiological adaptation process of plants encountering drying soil is to achieve water balance by reducing shoot growth and maintaining or promoting root elongation but little is known about the molecular basis of this process this study investigated the role of a drought up regulated triticum aestivum nac69 1 tanac69 1 in the modulation of root growth in wheat tanac69 1 was predominantly expressed in wheat roots at the early vegetative stage overexpression of tanac69 1 in wheat roots using osrsp3 essentially root specific and ospip2 3 root predominant promoters resulted in enhanced primary seminal root length and a marked increase in maturity root biomass competitive growth analysis under water limited conditions showed that osrsp3 promoter driven tanac69 1 transgenic lines produced 32 and 35 more above ground biomass and grains than wild type plants respectively tanac69 1 overexpression in the roots down regulated the expression of tashy2 and taiaa7 which are from the auxin iaa aux iaa transcriptional repressor gene family and are the homologs of negative root growth regulators shy2 iaa3 and iaa7 in arabidopsis the expression of tashy2 and taiaa7 in roots was down regulated by drought stress and up regulated by cytokinin treatment which inhibited root growth dna binding and transient expression analyses revealed that tanac69 1 bound to the promoters of tashy2 and taiaa7 acted as a transcriptional repressor and repressed the expression of reporter genes driven by the tashy2 or taiaa7 promoter these data suggest that tanac69 1 is a transcriptional repressor of tashy2 and taiaa7 homologous to arabidopsis negative root growth regulators and is likely to be involved in promoting root elongation in drying soil.auxin mediates numerous plant responses some of which have been shown to require transcriptional regulation one auxin response pathway which depends on the relief of transcriptional repression is mediated by tir1 transport inhibitor response protein 1 tir1 is an auxin receptor and also a subunit of an scf type ubiquitin ligase in the presence of a low concentration of auxin in the nucleus members of the aux iaa family of transcriptional repressors bind to arf proteins and inhibit the transcription of specific auxin response genes increased nuclear concentrations of auxin promote auxin binding to tir1 causing the aux iaa proteins to associate with tir1 and leading to their degradation by a proteasome mediated pathway this decreases the concentration of aux iaa proteins in the nucleus and thereby enables the expression of certain auxin response genes.recent studies of auxin response have focused on the functions of three sets of proteins the auxin aux response factors arfs the aux iaas and the f box protein tir1 the arf proteins bind dna and directly activate or repress transcription of target genes while the aux iaa proteins repress arf function tir1 is part of a ubiquitin protein ligase required for degradation of aux iaa proteins here we report the isolation and characterization of a novel mutant of arabidopsis called axr5 1 mutant plants are resistant to auxin and display a variety of auxin related growth defects including defects in root and shoot tropisms further the axr5 1 mutation results in a decrease in auxin regulated transcription the molecular cloning of axr5 revealed that the gene encodes the iaa1 protein a member of the aux iaa family of proteins axr5 is expressed throughout plant development consistent with the pleiotropic mutant phenotype the axr5 1 mutation results in an amino acid substitution in conserved domain ii of the protein similar to gain of function mutations recovered in other members of this gene family biochemical studies show that iaa1 axr5 interacts with tir1 in an auxin dependent manner the mutation prevents this interaction suggesting that the mutant phenotype is caused by the accumulation of iaa1 axr5 our results provide further support for a model in which most members of the aux iaa family are targeted for degradation by scftir1 in response to auxin.\"}, {\"question\": \"List the ten types of conjoined twins.\", \"ideal_answer\": [\"Thoraco-omphalopagus: Two bodies fused from the upper chest to the lower chest. These twins usually share a heart, and may also share the liver or part of the digestive system.\\nThoracopagus:Two bodies fused from the upper thorax to lower belly. The heart is always involved in these cases.\\nOmphalopagus: Two bodies fused at the lower abdomen. Unlike thoracopagus, the heart is never involved in these cases.\\nPyopagus: joined at the buttocks with sacrum and coccyx anomalies\\nRachipagus: Joined at the spine with vertebral and neural tube defects\\nIschiopagus: joined at the hip from umbilicus to conjoined pelvis\\nParasitic twins: Twins that are asymmetrically conjoined, resulting in one twin that is small, less formed, and dependent on the larger twin for survival.\\nCraniopagus: Fused skulls, but separate bodies\\nCephalopagus: head but not face or foramen magnum, brains are usually separate\\nDicephalus dipus dibrachius: 2 heads and one body\"], \"exact_answer\": [[\"Thoraco-omphalopagus\"], [\"Thoracopagus\"], [\"Omphalopagus\"], [\"Pyopagus\"], [\"Craniopagus\"], [\"Parasitic\"], [\"Rachipagus\"], [\"Ischiopagus\"], [\"diprosopus\"], [\"Dicephalus dipus dibrachius\"]], \"context\": \"Separation of Thoracoomphalopagus Conjoined Twins. Eight different types of conjoined twins have been described in the literature. MPygopagus are one of the rare types of conjoined twins with only a handful of cases reported in the literature. Ischiopagus and diprosopus in India: two pairs of conjoined twinsDicephalus dipus dibrachius: conjoined twins conjoined tripus twins with features of rachipagus, parapagus dicephalus, and cephalopagus.craniopagus conjoined twindiagnosis of conjoined twins, thoracopagus,Eight different types of conjoined twins can be distinguished; one is cephalopagus, which is fourth in rarity of occurrence. They are classified into eight different subtypes, with 18% representing pyopagus conjoints.Craniopagus twins (CPT) are an uncommon, highly fascinating accident of natureIschiopagus and pygopagus conjoined twins:The most common varieties encountered were thoraco-omphalopagus (28%), thoracopagus (18.5%), omphalopagus (10%), parasitic twins (10%) and craniopagus (6%)Dicephalus dipus tetrabrachius conjoined twins of Zaria: case report and literature review.These anterolaterally united parapagus twins must result from two nearly parallel notochords in close proximity; craniopagi and pygopagi from fusion at the cranial and caudal neuropores, respectively; cephalopagi and ischiopagi from union at the pharyngeal and cloacal membranes, respectively; thoracopagi from merging of the cardiac anlage; and omphalopagi from fusion of the umbilicus or of the edges of two embryonic discs in any area not including the above sites.The most common varieties encountered were thoraco-omphalopagus (28%), thoracopagus (18.5%), omphalopagus (10%), parasitic twins (10%) and craniopagus (6%).The most common types of conjoined twins were thoracoomphalopagus (28%), thoracopagus (18%), omphalopagus (10%), parasitic twins (10%), and craniopagus (6%).RESULTS Three sets of female conjoined twins underwent successful separation 2 pygopagus, one ischiopagus tripus) with 5 surviving infants.Craniopagus-type conjoined twins (joined at the head) are exceedingly rare.Cephalopagus is a rare variety of conjoined twins. Magnetic resonance imaging (MRI) was used for the first time in the preoperative planning for separation of conjoined twins. In these omphalopagus infantsCephalothoracopagus janiceps is a prototype of facing anomaly in which the two bodies demonstrated a cross symmetry to the midline, Dicephalus dipus dibrachius is a case of side-by-side union, in which the bodies facing nearly the same direction were symmetrical to the middle sagittal plane.Frequency of thoracoomphalopagus conjoined twins in Thailand.Forty cases of conjoined twins were included in the study. There were 72.5% cases of thoracophagus, 12.5% of paraphagus, 7.5% of omphalo-ischiophagus, 5.0% of omphalophagus, and 2.5% of cephalophagus.Dicephalus is one of the rarest types of conjoined twins. In such cases, the twins are usually stillborn or die shortly after birthThere were five omphalopagus, two pygopagus, two heterpagus and one ishiopagus twins.\"}, {\"question\": \"Which are the constitutive parts of a Genomic Regulatory Block (GRB)?\", \"ideal_answer\": [\"GRBs are spanned by highly conserved noncoding elements (HCNEs), their developmental regulatory target genes, and phylogenetically and functionally unrelated \\\"bystander\\\" genes.\"], \"exact_answer\": [[\"highly conserved noncoding elements (HCNEs)\"], [\"developmental regulatory target genes\"], [\"phylogenetically and functionally unrelated \\\"bystander\\\" genes\"]], \"context\": \"We found the largest mammal-teleost conserved chromosomal segments to be spanned by highly conserved noncoding elements (HCNEs), their developmental regulatory target genes, and phylogenetically and functionally unrelated \\\"bystander\\\" genes. Bystander genes are not specifically under the control of the regulatory elements that drive the target genes and are expressed in patterns that are different from those of the target genes. Reporter insertions distal to zebrafish developmental regulatory genes pax6.1/2, rx3, id1, and fgf8 and miRNA genes mirn9-1 and mirn9-5 recapitulate the expression patterns of these genes even if located inside or beyond bystander genes, suggesting that the regulatory domain of a developmental regulatory gene can extend into and beyond adjacent transcriptional units. We termed these chromosomal segments genomic regulatory blocks (GRBs). We found the largest mammal-teleost conserved chromosomal segments to be spanned by highly conserved noncoding elements (HCNEs), their developmental regulatory target genes, and phylogenetically and functionally unrelated \\\"bystander\\\" genes.\"}, {\"question\": \"Which R packages have been developed for the discovery of mutational signatures in cancer?\", \"ideal_answer\": [\"signeR: an empirical Bayesian approach to mutational signature discovery. Mutational signatures can be used to understand cancer origins and provide a unique opportunity to group tumor types that share the same origins and result from similar processes. These signatures have been identified from high throughput sequencing data generated from cancer genomes by using non-negative matrix factorisation (NMF) techniques. The extraction of mutational signatures from high-throughput data still remains a daunting task.RESULTS: Here we present a new method for the statistical estimation of mutational signatures based on an empirical Bayesian treatment of the NMF model.\", \"SomaticSignatures: inferring mutational signatures from single-nucleotide variants.Mutational signatures are patterns in the occurrence of somatic single-nucleotide variants that can reflect underlying mutational processes.The SomaticSignatures package provides flexible, interoperable and easy-to-use tools that identify such signatures in cancer sequencing data.It facilitates large-scale, cross-dataset estimation of mutational signatures, implements existing methods for pattern decomposition, supports extension through user-defined approaches and integrates with existing Bioconductor workflows.AVAILABILITY AND IMPLEMENTATION: The R package SomaticSignatures is available as part of the Bioconductor project.Its documentation provides additional details on the methods and demonstrates applications to biological datasets.These mutagenic processes often produce characteristic mutational patterns called mutational signatures.The decomposition of a cancer genome's mutation catalog into mutations consistent with such signatures can provide valuable information about cancer etiology.Hence, one needs to be able to not only decompose a patient's mutational profile into signatures but also establish the accuracy of such decomposition.Results: We proposed two complementary ways of measuring confidence and stability of decomposition results and applied them to analyze mutational signatures in breast cancer genomes.\", \"signeR is an R package for a new method for the statistical estimation of mutational signatures based on an empirical Bayesian treatment of the NMF model.\", \"signeR: an empirical Bayesian approach to mutational signature discovery. Mutational signatures can be used to understand cancer origins and provide a unique opportunity to group tumor types that share the same origins and result from similar processes. These signatures have been identified from high throughput sequencing data generated from cancer genomes by using non-negative matrix factorisation (NMF) techniques.\", \"signeR: an empirical Bayesian approach to mutational signature discovery. The extraction of mutational signatures from high-throughput data still remains a daunting task.RESULTS: Here we present a new method for the statistical estimation of mutational signatures based on an empirical Bayesian treatment of the NMF model. Results on real data agree well with current knowledge.AVAILABILITY AND IMPLEMENTATION: signeR is implemented in R and C\\u2009++, and is available as a R package at http://bioconductor.org/packages/signeR\", \"signeR: an empirical Bayesian approach to mutational signature discovery.\", \"It facilitates large-scale, cross-dataset estimation of mutational signatures, implements existing methods for pattern decomposition, supports extension through user-defined approaches and integrates with existing Bioconductor workflows.AVAILABILITY AND IMPLEMENTATION: The R package SomaticSignatures is available as part of the Bioconductor project. \", \"Cancers arise as the result of somatically acquired changes in the DNA of cancer cells. However, in addition to the mutations that confer a growth advantage, cancer genomes accumulate a large number of somatic mutations resulting from normal DNA damage and repair processes as well as carcinogenic exposures or cancer related aberrations of DNA maintenance machinery. These mutagenic processes often produce characteristic mutational patterns called mutational signatures. The decomposition of a cancer genome's mutation catalog into mutations consistent with such signatures can provide valuable information about cancer etiology. SigneR, SignatureEstimation, pmsignature and SomaticSignatures are R packages that have been developed for the discovery of mutational signatures in cancer.\"], \"exact_answer\": [[\"signeR\"], [\"SignatureEstimation\"], [\"pmsignature\"], [\"SomaticSignatures\"]], \"context\": \"signeR: an empirical Bayesian approach to mutational signature discovery.Mutational signatures can be used to understand cancer origins and provide a unique opportunity to group tumor types that share the same origins and result from similar processes. These signatures have been identified from high throughput sequencing data generated from cancer genomes by using non-negative matrix factorisation (NMF) techniques. Current methods based on optimization techniques are strongly sensitive to initial conditions due to high dimensionality and nonconvexity of the NMF paradigm. In this context, an important question consists in the determination of the actual number of signatures that best represent the data. The extraction of mutational signatures from high-throughput data still remains a daunting task.RESULTS: Here we present a new method for the statistical estimation of mutational signatures based on an empirical Bayesian treatment of the NMF model. While requiring minimal intervention from the user, our method addresses the determination of the number of signatures directly as a model selection problem. In addition, we introduce two new concepts of significant clinical relevance for evaluating the mutational profile. The advantages brought by our approach are shown by the analysis of real and synthetic data. The later is used to compare our approach against two alternative methods mostly used in the literature and with the same NMF parametrization as the one considered here. Our approach is robust to initial conditions and more accurate than competing alternatives. It also estimates the correct number of signatures even when other methods fail. Results on real data agree well with current knowledge.AVAILABILITY AND IMPLEMENTATION: signeR is implemented in R and C\\u2009++, and is available as a R package at http://bioconductor.org/packages/signeRCancers arise as the result of somatically acquired changes in the DNA of cancer cells. However, in addition to the mutations that confer a growth advantage, cancer genomes accumulate a large number of somatic mutations resulting from normal DNA damage and repair processes as well as carcinogenic exposures or cancer related aberrations of DNA maintenance machinery. These mutagenic processes often produce characteristic mutational patterns called mutational signatures. The decomposition of a cancer genome's mutation catalog into mutations consistent with such signatures can provide valuable information about cancer etiology. However, the results from different decomposition methods are not always consistent. Hence, one needs to be able to not only decompose a patient's mutational profile into signatures but also establish the accuracy of such decomposition.Results: We proposed two complementary ways of measuring confidence and stability of decomposition results and applied them to analyze mutational signatures in breast cancer genomes. We identified both very stable and highly unstable signatures, as well as signatures that previously have not been associated with breast cancer. We also provided additional support for the novel signatures. Our results emphasize the importance of assessing the confidence and stability of inferred signature contributions.Availability: All tools developed in this paper have been implemented in an R package, called SignatureEstimation, which is available from https://www.ncbi.nlm.nih.gov/CBBresearch/Przytycka/index.cgi#signatureestimation.Recent advances in sequencing technologies have enabled the production of massive amounts of data on somatic mutations from cancer genomes. These data have led to the detection of characteristic patterns of somatic mutations or \\\"mutation signatures\\\" at an unprecedented resolution, with the potential for new insights into the causes and mechanisms of tumorigenesis. Here we present new methods for modelling, identifying and visualizing such mutation signatures. Our methods greatly simplify mutation signature models compared with existing approaches, reducing the number of parameters by orders of magnitude even while increasing the contextual factors (e.g. the number of flanking bases) that are accounted for. This improves both sensitivity and robustness of inferred signatures. We also provide a new intuitive way to visualize the signatures, analogous to the use of sequence logos to visualize transcription factor binding sites. We illustrate our new method on somatic mutation data from urothelial carcinoma of the upper urinary tract, and a larger dataset from 30 diverse cancer types. The results illustrate several important features of our methods, including the ability of our new visualization tool to clearly highlight the key features of each signature, the improved robustness of signature inferences from small sample sizes, and more detailed inference of signature characteristics such as strand biases and sequence context effects at the base two positions 5' to the mutated site. The overall framework of our work is based on probabilistic models that are closely connected with \\\"mixed-membership models\\\" which are widely used in population genetic admixture analysis, and in machine learning for document clustering. We argue that recognizing these relationships should help improve understanding of mutation signature extraction problems, and suggests ways to further improve the statistical methods. Our methods are implemented in an R package pmsignature (https://github.com/friend1ws/pmsignature) and a web application available at https://friend1ws.shinyapps.io/pmsignature_shiny/.SomaticSignatures: inferring mutational signatures from single-nucleotide variants.Mutational signatures are patterns in the occurrence of somatic single-nucleotide variants that can reflect underlying mutational processes. The SomaticSignatures package provides flexible, interoperable and easy-to-use tools that identify such signatures in cancer sequencing data. It facilitates large-scale, cross-dataset estimation of mutational signatures, implements existing methods for pattern decomposition, supports extension through user-defined approaches and integrates with existing Bioconductor workflows.AVAILABILITY AND IMPLEMENTATION: The R package SomaticSignatures is available as part of the Bioconductor project. Its documentation provides additional details on the methods and demonstrates applications to biological datasets.The SomaticSignatures package provides flexible, interoperable and easy-to-use tools that identify such signatures in cancer sequencing data.signeR: an empirical Bayesian approach to mutational signature discovery.Here we present a new method for the statistical estimation of mutational signatures based on an empirical Bayesian treatment of the NMF model.Results on real data agree well with current knowledge.<br><b>AVAILABILITY AND IMPLEMENTATION</b>: signeR is implemented in R and C\\u2009++, and is available as a R package at http://bioconductor.org/packages/signeR CONTACT: itojal@cipe.accamargo.org.brSupplementary information: Supplementary data are available at Bioinformatics online.<br>signeR is implemented in R and C\\u2009++, and is available as a R package at http://bioconductor.org/packages/signeR CONTACT: itojal@cipe.accamargo.org.brSupplementary information: Supplementary data are available at Bioinformatics online..\"}, {\"question\": \"According to guidelines, insulin resistance is one risk factor in the diagnosis of metabolic syndrome, name 3 more risk factors.\", \"ideal_answer\": [\"Metabolic syndrome (MetS) is generally defined as a cluster of metabolically related cardiovascular risk factors which are often associated with the condition of insulin resistance, elevated blood pressure, and abdominal obesity.\", \"Metabolic syndrome (MetS) is generally defined as a cluster of metabolically related cardiovascular risk factors which are often associated with the condition of insulin resistance, elevated blood pressure, and abdominal obesity. \"], \"exact_answer\": [[\"obesity\"], [\"hypertension\"], [\"dyslipdemia\"]], \"context\": \" Metabolic syndrome (MetS) is a collection of clinical conditions, including central obesity, hypertension, glucose intolerance and dyslipidemia.Metabolic syndrome (MetS) is generally defined as a cluster of metabolically related cardiovascular risk factors which are often associated with the condition of insulin resistance, elevated blood pressure, and abdominal obesityMetabolic syndrome, defined as a cluster of obesity, hypertension, dyslipidemia, and insulin resistance/glucose intolerance, has been identified as a major risk factor for coronary heart disease in women.Metabolic syndrome is a cluster of conditions that synergistically increase the risk of cardiovascular disease, type 2 diabetes, and premature mortality. The components are abdominal obesity, impaired glucose metabolism, dyslipidemia, and hypertension.MetS is recognized clinically by numerous constitutive traits, including abdominal obesity, hypertension, dyslipidemia (elevated triglycerides, low high-density lipoprotein cholesterol), and hyperglycemia. Abdominal obesity is frequently associated with a collection of metabolic disorders that include elevated blood pressure, characteristic lipid abnormalities (low high-density lipoprotein cholesterol and high triglycerides) and increased fasting glucose, with an underlying situation of insulin resistance, which has been defined as metabolic syndromeThe presence of 3 or more of the following 5 components (abdominal obesity, hypertriglyceridemia, low level of high density lipoprotein cholesterol, hypertension and high fasting blood glucose) allows to diagnose metabolic syndrome.The metabolic syndrome has 5 criteria: central obesity, hypertriglyceridemia, low high-density lipoprotein (HDL) levels, fasting hyperglycemia, and hypertension.Metabolic syndrome is a constellation of interrelated risk factors of metabolic origin including abdominal obesity, atherogenic dyslipidemia, elevated blood pressure, elevated plasma glucose level, and prothrombotic and proinflammatory states that promote atherosclerotic CVD and increase the risk of type 2 diabetes mellitus.The waist circumference cut points for diagnosing metabolic syndrome (as defined using the new harmonised guidelines), insulin resistance, dysglycaemia, hypertension and dyslipidaemia were obtained using receiver operator characteristic curve analysis.The metabolic syndrome is a complex association of several risk factors including insulin resistance, dyslipidemia, and essential hypertension.  insulin resistance syndrome (often referred to as syndrome X or the metabolic syndrome) ie, a clustering of metabolic, atheromatous risk factors, including hypertriglyceridaemia, hyperinsulinaemia, hyper-tension, hypercholesterinaemia and obesity. The metabolic syndrome is a clustering of risk factors which predispose an individual to cardiovascular morbidity and mortality. There is general consensus regarding the main components of the syndrome (glucose intolerance, obesity, raised blood pressure and dyslipidaemia [elevated triglycerides, low levels of high-density lipoprotein cholesterol]) Metabolic syndrome (MS) refers to the clustering of cardiometabolic risk factors - including abdominal obesity, hyperglycaemia, dyslipidaemia and elevated blood pressureThe metabolic syndrome is a constellation of risk factors including glucose dysregulation, central obesity, dyslipidemia, and hypertension.Insulin resistance syndrome (IRS), also termed syndrome X, is a distinctive constellation of risk factors for the development of type 2 diabetes mellitus and cardiovascular disease. The syndrome's hallmarks are glucose intolerance, hyperinsulinemia, a characteristic dyslipidemia (high triglycerides; low high-density lipoprotein cholesterol, and small, dense low-density lipoprotein cholesterol), obesity, upper-body fat distribution, hypertension, and increased prothrombotic and antifibrinolytic factors\\\"The metabolic syndrome\\\" is the name for a clustering of risk factors for cardiovascular disease and type 2 diabetes that are of metabolic origin. These risk factors consist of atherogenic dyslipidemia, elevated blood pressure, elevated plasma glucose, a prothrombotic state, and a proinflammatory state. Metabolic Syndrome X is a clinical entity which comprises the following factors: diabetes mellitus, arterial hypertension, high levels of triglyceride and/or low levels of HDL cholesterol, central obesity and microalbuminuriametabolic syndrome, produced by genetic, hormonal and lifestyle factors such as obesity, physical inactivity and certain nutrient excesses. This disease is characterized by the clustering of insulin resistance and hyperinsulinemia, and is often associated with dyslipidemia (atherogenic plasma lipid profile), essential hypertension, abdominal (visceral) obesity, glucose intolerance or noninsulin-dependent diabetes mellitus and an increased risk of cardiovascular events\"}, {\"question\": \"List the two most important synaptic markers.\", \"ideal_answer\": [\"postsynaptic density 95\\nsynaptophysin\", \"Synaptic markers include acetylcholine, gamma-aminobutyric acid (GABA), glutamine, glycine and synaptophysin.\"], \"exact_answer\": [[\"postsynaptic density 95\", \"PSD-95\"], [\"synaptophysin\", \"SVP\"]], \"context\": \"hese improvements in neuron viability and function were correlated with significant increases in the levels of post-synaptic marker PSD95 and the pre-synaptic marker synaptophysin. post-synaptic density (PSD)-95 protein, a post-synaptic markerHippocampal levels of synaptic marker postsynaptic density 95 (PSD-95) and synaptophysin (SVP) were measured using western blot analysisThese animals showed a significant loss of presynaptic markers (synaptophysin; p<0.001),increases in the levels of the pre-synaptic marker, synaptophysin, and the post-synaptic marker, PSD95, were also presentThe most widely used neuronal markers comprise phosphorylated and nonphosphorylated neurofilaments, microtubule-associated protein-2, NeuN, or synaptic markers such as synaptophysin.Synaptic markers were acetylcholine, gamma-aminobutyric acid (GABA), glutamine, and glycine. \"}, {\"question\": \"What is included in the fourth generation HIV test?\", \"ideal_answer\": [\"Fourth generation assays detect simultaneously antibodies for HIV and the p24 antigen. It identifies HIV infection earlier than previous generation tests.\"], \"exact_answer\": [[\"antibodies for HIV\"], [\"p24 antigen\"]], \"context\": \"Diagnosing acute HIV infection at point of care: a retrospective analysis of the sensitivity and specificity of a fourth-generation point-of-care test for detection of HIV core protein p24.Previous evaluation of the fourth-generation Alere Determine HIV-1/2 Ag/Ab Combo POCT showed only 50% sensitivity for HIV core protein p24 (p24 antigen) detection, which is suboptimal for diagnosis of acute HIV infection with limited advantage over third-generation POCT.CONCLUSIONS: This new POCT shows improved sensitivity for detection of p24 antigen and may be of value for clinical use in detecting acute HIV infection.While fourth generation antigen-antibody combination assays have been successful in high-resource settings, rapid point of care (POC) versions of these assays have yet to demonstrate high sensitivity to detect AHI. STUDY DESIGN: A total of 30,201 sera were tested for HIV diagnosis using Abbott Architect\\u00aeHIV-Ag/Ab-Combo 4th-gen-EIA at a hospital in Spain during 17 months. OBJECTIVE: To determine if fourth-generation antigen/antibody (Ag/Ab) rapid diagnostic tests (RDT) would have detected HIV infection earlier than the third-generation RDT used in MTN-003 (VOICE).BACKGROUND: Fourth generation assays detect simultaneously antibodies for HIV and the p24 antigen, identifying HIV infection earlier than previous generation tests.The status of clients on ART was additionally confirmed by fourth-generation HIV Ag/Ab combo tests, Architect and Genscreen Ultra.INTRODUCTION: We describe the overall accuracy and performance of a serial rapid HIV testing algorithm used in community-based HIV testing in the context of a population-based household survey conducted in two sub-districts of uMgungundlovu district, KwaZulu-Natal, South Africa, against reference fourth-generation HIV-1/2 antibody and p24 antigen combination immunoassays.The sample was later routinely tested with a fourth generation antigen/antibody assay as per local protocol and was strongly positive.The fourth-generation HIV EIA was positive presumably because it detects p24 HIV antigen as well as antibodies, unlike rapid HIV tests and third-generation HIV EIAs.This case highlights not only the importance of frontline providers to understand the different testing methodologies for HIV screening and their limitations but the importance of clinical suspicion as well.This second diagnostic window was caused by the absence of HIV specific antibodies and the decline of HIV p24 antigen concentrations below the detection limits of the fourth generation assays.Fourth- and fifth-generation HIV assays added p24 antigen detection to the screening assay, reducing the test-negative window to 11 to 14 days.\"}, {\"question\": \"List four principles of medical ethics.\", \"ideal_answer\": [\"The four principles of medical ethics proposed by Beauchamp and Childress are autonomy, non-maleficence, beneficence and justice. They have been extremely influential in the field of medical ethics, and are fundamental for understanding the current approach to ethical assessment in health care.\"], \"exact_answer\": [[\"autonomy\"], [\"non-maleficence\"], [\"beneficence\"], [\"justice\"]], \"context\": \"Respect for Autonomy (RFA) has been a mainstay of medical ethics since its enshrinement as one of the four principles of biomedical ethics by Beauchamp and Childress' in the late 1970sThere are four principles of medical ethics; Beneficence, Respect for autonomy, Non-maleficence, and Justice.This paper argues that the four prima facie principles-beneficence, non-maleficence, respect for autonomy and justice-afford a good and widely acceptable basis for 'doing good medical ethics'. Contemporary clinical ethics was founded on principlism, and the four principles: respect for autonomy, nonmaleficence, beneficence and justice, remain dominant in medical ethics discourse and practice.Each of the four principles (beneficence, nonmaleficence,justice and autonomy) is investigated in turn, looking in particular at the extent to which each is rooted in the Islamic paradigm.Guided by the medical ethics principles of \\\"four principles plus scope,\\\" Chinese aesthetic medical practitioners have proposed some extremely valuable ethical principles combined with the construction of aesthetic medicine and the requirements of clinical practice such as the principle of general nonmaleficence, the principle of local minimal invasiveness, the principle of informed consent, and the principle of respect and confidentiality.This commentary briefly argues that the four prima facie principles of beneficence, non-maleficence, respect for autonomy and justice enable a clinician (and anybody else) to make ethical sense of the author's proposed reliance on professional guidance and rules, on law, on professional integrity and on best interests, and to subject them all to ethical analysis and criticism based on widely acceptable basic prima facie moral obligations; and also to confront new situations in the light of those acceptable principles.BACKGROUND: The four principles of Beauchamp and Childress--autonomy, non-maleficence, beneficence and justice--have been extremely influential in the field of medical ethics, and are fundamental for understanding the current approach to ethical assessment in health care. Difficulties with the application of the four principles (autonomy, beneficence, nonmaleficence and justice) to judge medical practitioner behaviour are highlighted. Tom Beauchamp and James Childress have always maintained that their four principles approach (otherwise known as principlism) is a globally applicable framework for biomedical ethics. This claim is grounded in their belief that the principles of respect for autonomy, non-maleficence, beneficence and justice form part of a 'common morality', or collection of very general norms to which everyone who is committed to morality subscribes.I will argue that there are difficulties with the application of the four principles approach to incompetent children. The most important principle - respect for autonomy - is not directly applicable to incompetent children and the most appropriate modification of the principle for them is not clear. The principle of beneficence - that one should act in the child's interests - is complicated by difficulties in assessing what a child's interests are and to which standard of interests those choosing for children should be held.These principles are: respect for autonomy, beneficence, non-maleficence and justice, along with concern for their scope of application.BACKGROUND: The four principles of Beauchamp and Childress--autonomy, non-maleficence, beneficence and justice--have been extremely influential in the field of medical ethics, and are fundamental for understanding the current approach to ethical assessment in health care.Difficulties with the application of the four principles (autonomy, beneficence, nonmaleficence and justice) to judge medical practitioner behaviour are highlighted.Four main principles underlie medical ethics: autonomy, nonmaleficence, beneficence, and justice.This approach to treatment is discussed in the context of the four principles of medical ethics, namely respect for autonomy, justice, beneficence and non-maleficence.\"}, {\"question\": \"List symptoms of Heerfordt syndrome.\", \"ideal_answer\": [\"Heerfordt syndrome (also known as Heerfordt-Waldenstr\\u00f6m or uveoparotid fever)  is a rare presentation of sarcoidosis characterized by the presence of parotid gland enlargement, facial palsy, anterior uveitis, and fever.\"], \"exact_answer\": [[\"parotid gland enlargement\"], [\"facial palsy\"], [\"anterior uveitis\"], [\"fever\"]], \"context\": \"Case 3 showed Heerfordt syndrome with facial nerve paralysis.Heerfordt's syndrome is defined as a combination of facial palsy, parotid swelling, uveitis and fever in sarcoidosis cases.Heerfordt-Waldenstr\\u00f6m syndrome, a rare presentation of sarcoidosis, is characterized by the presence of parotid gland enlargement, facial palsy, anterior uveitis, and fever. The patient was given a diagnosis of Heerfordt-Waldenstr\\u00f6m syndrome, or uveoparotid fever.Physical examination revealed right facial paralysis, parotid gland swelling, high fever and poor general condition. Ophthalmoscopy revealed anterior and posterior uveitis including macular edema and chorioretinal infiltrates. Angiography revealed a dense pattern of hyperfluorescent lesions and these observations resulted in the diagnosis of Heerfordt syndrome. BACKGROUND: Heerfordt syndrome is rare and is characterized by fever, uveitis, parotid gland enlargement, and facial nerve palsy.Heerfordt's syndrome with uveitis, enlargement of the parotid glands and optional paralysis of the Nn. facialis is a form of sarcoidosis.The Heerfordt syndrome is characterized by fever, uveitis, swelling of parotid gland and facial nerve palsy, and 53 cases have been reported in Japan until 2000. The Heerfordt syndrome is characterized by fever, uveitis, swelling of parotid gland and facial nerve palsy, and 53 cases have been reported in Japan until 2000.Heerfordt syndrome is an unusual manifestation of systemic sarcoidosis and is characterized by parotitis, uveitis, and facial nerve paralysis.Heerfordt syndrome is a rare manifestation of neurosarcoidosis and has to be included in the differential diagnosis of facial nerve palsy.Heerfordt syndrome is rare and is characterized by fever, uveitis, parotid gland enlargement, and facial nerve palsy.Heerfordt syndrome with unilateral facial nerve palsy: a rare presentation of sarcoidosis.<b>BACKGROUND</b>: Heerfordt syndrome is rare and is characterized by fever, uveitis, parotid gland enlargement, and facial nerve palsy.We hereby present a case of Heerfordt syndrome with unilateral facial nerve palsy as a presentation of sarcoidosis.<br><b>HISTORY AND SIGNS</b>: A 29-year-old male patient from Sri Lanka presented with eye redness OU, blurred vision OD, fever, headache, night sweat, fatigue, and weight loss (5 kg over 1 month).One year after onset of treatment, no recurrence was noted.<br><b>CONCLUSIONS</b>: Heerfordt syndrome is a rare manifestation of neurosarcoidosis and has to be included in the differential diagnosis of facial nerve palsy.<br>Heerfordt's syndrome (HS) consists in its complete form of uveitis, parotid or salivary gland enlargement and cranial nerve palsy.However, the otolaryngologist may encounter Heerfordt's syndrome as this syndrome presents with facial nerve palsy and swelling of the parotid gland.Heerfordt's syndrome is a rare manifestation of sarcoidosis characterized by the presence of facial nerve palsy, parotid gland enlargement, anterior uveitis, and low grade fever.As a subtype of the clinical presentations associated with sarcoidosis, the combination of uveitis, parotid gland swelling, and facial nerve palsy is known as Heerfordt's syndrome.Heerfordt's syndrome as a cause of facial palsy is very rare.Heerfordt's syndrome is a rare manifestation of sarcoidosis and is defined as a combination of facial palsy, parotid swelling, and uveitis, associated with a low-grade fever.Heerfordt's Syndrome Presenting with Recurrent Facial Nerve Palsy:BACKGROUND Heerfordt syndrome is rare and is characterized by fever, uveitis, parotid gland enlargement, and facial nerve palsy.We hereby present a case of Heerfordt syndrome with unilateral facial nerve palsy as a presentation of sarcoidosis.\"}, {\"question\": \"In quadruped mammals, what bones make up the stifle?\", \"ideal_answer\": [\"In quadruped mammals, the stifle is composed of 3 bones, the femur, the tibia and the patella.\"], \"exact_answer\": [[\"femur\"], [\"tibia\"], [\"patella\"]], \"context\": \"MR images showed the bone, articular cartilage, menisci and ligaments of the normal tiger stifle. SE T1-weighted sequence provided excellent resolution of the subchondral bones of the femur, tibia and patella compared with the GE STIR T2-weighted MR images.\"}, {\"question\": \"List 2 approved drug treatments for Inflammatory Bowel Disease (IBD).\", \"ideal_answer\": [\"Patients with IBD, inflammtory Bowel Disease can be treated with steroids, or 2 approved biosimilar drugs infliximab (IFX) or adalimumab (ADA)\"], \"exact_answer\": [[\"infliximab (IFX)\"], [\"adalimumab (ADA)\"]], \"context\": \"Nonetheless, infliximab and adalimumab are the only biological agents that have been approved for this group of patients. In September 2013, the European Medicines Agency approved two infliximab biosimilars for treatment of adult and paediatric inflammatory bowel disease patients,The introduction of infliximab into clinical practice is one of the most significant advances in the care of patients who have IBD. Infliximab has become an important part of the medical armamentarium to treat extraintestinal manifestations that often are refractory to other medications and are a significant cause of morbidity in these patients. Two other TNF inhibitors recently have demonstrated efficacy in CD: certolizumab pegol and adalimumab.Infliximab, a chimeric monoclonal antibody to tumor necrosis factor-alpha, is effective for induction and maintenance of remission of CD and UC. The role of infliximab for EIMs related to IBD has been less studied, but it is likely as effective.The Italian Group for the Study of Inflammatory Bowel Disease (IG-IBD) has prepared clinical practice guidelines to help physicians prescribe corticosteroids and immunosuppressive drugs for these patients. lthough the TNF inhibitor infliximab is known to improve IBD outcomes in many different ways, several questions remain regarding the optimal way to employ this drug in the clinic, Infliximab biosimilars for the treatment of IBD have been available in Europe and Asia for a few years and are expected to become available in the United States within the next 1 to 2 years\"}, {\"question\": \"Name curated data resources for ChIP-seq data\", \"ideal_answer\": [\"The MGA repository, Cistrome Data Browser and CR Cistrome are curated data resources for ChIP-seq data.\"], \"exact_answer\": [[\"MGA repository\"], [\"Cistrome Data Browser\"], [\"CR Cistrome\"], [\"CistromeMap\"]], \"context\": \"MGA repository: a curated data resource for ChIP-seq and other genome annotated data.Cistrome Data Browser: a data portal for ChIP-Seq and chromatin accessibility data in human and mouse.Although rapidly accumulating publicly available ChIP-seq, DNase-seq and ATAC-seq data are a valuable resource for the systematic investigation of gene regulation processes, a lack of standardized curation, quality control and analysis procedures have hindered extensive reuse of these data. To overcome this challenge, we built the Cistrome database, a collection of ChIP-seq and chromatin accessibility data (DNase-seq and ATAC-seq) published before January 1, 2016, including 13 366 human and 9953 mouse samples. All the data have been carefully curated and processed with a streamlined analysis pipeline and evaluated with comprehensive quality control metrics. We have also created a user-friendly web server for data query, exploration and visualization. The resulting Cistrome DB (Cistrome Data Browser), available online at http://cistrome.org/db, is expected to become a valuable resource for transcriptional and epigenetic regulation studies.CR Cistrome: a ChIP-Seq database for chromatin regulators and histone modification linkages in human and mouse.We have created CistromeMap as a web server to provide a comprehensive knowledgebase of all of the publicly available ChIP-Seq and DNase-Seq data in mouse and human.To overcome this challenge, we built the Cistrome database, a collection of ChIP-seq and chromatin accessibility data (DNase-seq and ATAC-seq) published before January 1, 2016, including 13 366 human and 9953 mouse samples.We have created CistromeMap as a web server to provide a comprehensive knowledgebase of all of the publicly available ChIP-Seq and DNase-Seq data in mouse and human.The resulting Cistrome DB (Cistrome Data Browser), available online at http://cistrome.org/db, is expected to become a valuable resource for transcriptional and epigenetic regulation studies.<br>Cistrome Data Browser: a data portal for ChIP-Seq and chromatin accessibility data in human and mouse.\"}, {\"question\": \"Which comparisons demonstrate the applicability of StereoGene in regulatory genomics?\", \"ideal_answer\": [\"StereoGene rapidly estimates genome-wide correlation among pairs of genomic features. These features may represent high-throughput data mapped to reference genome or sets of genomic annotations in that reference genome. StereoGene enables correlation of continuous data directly, avoiding the data binarization and subsequent data loss. Correlations are computed among neighboring genomic positions using kernel correlation. Representing the correlation as a function of the genome position, StereoGene outputs the local correlation track as part of the analysis. StereoGene also accounts for confounders such as input DNA by partial correlation. Numerous comparisons of ChIP-Seq datasets from the Human Epigenome Atlas and FANTOM CAGE demonstrate the wide applicability of StereoGene in regulatory genomics.\"], \"exact_answer\": [[\"Comparisons of ChIP-Seq datasets from the Human Epigenome Atlas\"], [\"Comparisons of ChIP-Seq datasets from FANTOM CAGE\"]], \"context\": \"StereoGene: rapid estimation of genome-wide correlation of continuous or interval feature data.Genomics features with similar genome-wide distributions are generally hypothesized to be functionally related, for example, colocalization of histones and transcription start sites indicate chromatin regulation of transcription factor activity. Therefore, statistical algorithms to perform spatial, genome-wide correlation among genomic features are required.Results: Here, we propose a method, StereoGene, that rapidly estimates genome-wide correlation among pairs of genomic features. These features may represent high-throughput data mapped to reference genome or sets of genomic annotations in that reference genome. StereoGene enables correlation of continuous data directly, avoiding the data binarization and subsequent data loss. Correlations are computed among neighboring genomic positions using kernel correlation. Representing the correlation as a function of the genome position, StereoGene outputs the local correlation track as part of the analysis. StereoGene also accounts for confounders such as input DNA by partial correlation. We apply our method to numerous comparisons of ChIP-Seq datasets from the Human Epigenome Atlas and FANTOM CAGE to demonstrate its wide applicability. We observe the changes in the correlation between epigenomic features across developmental trajectories of several tissue types consistent with known biology and find a novel spatial correlation of CAGE clusters with donor splice sites and with poly(A) sites. These analyses provide examples for the broad applicability of StereoGene for regulatory genomics.\"}, {\"question\": \"List symptoms of the Zieve's syndrome.\", \"ideal_answer\": [\"Zieve's syndrome, characterized by jaundice, hyperlipidaemia and haemolytic anaemia. It usually develops in young, chronically alcoholic subjects with enlarged fatty liver. It may rarely occur with intracranial haemorrhage.\"], \"exact_answer\": [[\"jaundice\"], [\"hyperlipidaemia\"], [\"haemolytic anaemia\"]], \"context\": \"Hemolysis in Acute Alcoholic Hepatitis: Zieve's Syndrome.Lab studies revealed hemolysis as the cause of anemia. The patient was diagnosed with Zieve's syndrome and managed with supportive measures.After exclusion of other possibilities, Zieve's syndrome was diagnosed. This is a condition of hyperbilirubinaemia, Coombs' negative haemolytic anaemia and hyperlipidaemia associated with alcoholism.We report a case of recurrent Zieve's syndrome, consisting of jaundice, hemolytic anemia and hyperlipoproteinemia, initiated by alcohol abuse. The crucial feature of this syndrome, which allows differentiation with common acute alcoholic hepatitis, is the presence of hemolysisZieve's syndrome consists of transient haemolytic anaemic, jaundice, hyperlipidaemia and alcohol-induced liver disease.Zieve's syndrome (hyperlipidaemia, anaemia and fatty liver degeneration) may rarely occur with intracranial haemorrhage. Alcohol-associated haemolysis in Zieve's syndrome: a clinical and laboratory study of five cases.In 1958 Zieve described a syndrome of jaundice, hyperlipidaemia, and transient haemolytic anaemia associated with alcohol abuse. Zieve's syndrome, characterized by jaundice, hyperlipaemia and haemolytic anaemia, usually develops in young, chronically alcoholic subjects with enlarged fatty liver. Zieve's syndrome (ZS), which consists of transient haemolytic anaemia, jaundice, hyperlipoproteinaemia, and alcohol-induced liver disease, was studied in male patients during the acute (n = 20) and the remittent (n = 10) phase. Red cell metabolic and membrane features in haemolytic anaemia of alcoholic liver disease (Zieve's syndrome).[Changes of erythrocyte membrane lipids in ethanol induced hyperlipidemia (Zieve's syndrome) (author's transl)].In 11 patients with alcohol-induced hyperlipemia, of whom 6 showed a Zieve Syndrome increased phospholipids, triglycerides and total cholesterol were found in the red cells stromal.The macroscopic and microscopic findings of a case of Zieve's syndrome are described (fatty liver, icterus, hyperlipemia and hemolytic anemia in chronic alcoholism). Reversible haemolytic anaemia associated with decreased red cell half-life and reticulocytosis was studied in 10 patients with Zieve's syndrome. The Zieve's syndrome consists of the transient association of cholestatic jaundice, hemolytic anemia, hyperlipemia, in a chronic alcoholic. Laboratory findings revealed Zieve-syndrome (alcoholic hyperlipemia, hemolytic anemia, and alcoholic fatty liver) and negative Hounsfield Unit measurement of the hypodense rim finally identified it as a layer of fat around the clot.The Zieve syndrome is characterized by hemolytic anemia in conjunction with secondary hyperlipidemia in patients suffering from alcohol-related toxic liver damage.Zieve's syndrome, characterized by jaundice, hyperlipaemia and haemolytic anaemia, usually develops in young, chronically alcoholic subjects with enlarged fatty liver.The Zieve's syndrome consists of the transient association of cholestatic jaundice, hemolytic anemia, hyperlipemia, in a chronic alcoholic.We report a case of recurrent Zieve's syndrome, consisting of jaundice, hemolytic anemia and hyperlipoproteinemia, initiated by alcohol abuse.The macroscopic and microscopic findings of a case of Zieve's syndrome are described (fatty liver, icterus, hyperlipemia and hemolytic anemia in chronic alcoholism).\"}, {\"question\": \"Which cancers compose Carney's triad?\", \"ideal_answer\": [\"Carney's triad is a rare pathogenic entity which consists of 3 rare soft tissue tumors: gastric leiomyosarcoma, pulmonary chondroma and extraadrenal paraganglioma. It is usually diagnosed in young women. The presence of three tumors at the same time is not required for its diagnosis (incomplete Carney's Triad).\"], \"exact_answer\": [[\"gastric leiomyosarcoma\"], [\"pulmonary chondroma\"], [\"extraadrenal paraganglioma\"]], \"context\": \"New adrenal tumors have also been associated with nonhereditary Carney's triad.Carney designated the combination of 3 rare soft tissue tumors (gastric leiomyosarcoma, pulmonary chondroma, and extraadrenal paraganglioma) as a syndrome. In sporadic Carney's triad additional pulmonary chondromas are observed and there are no SDH mutations.A case of bilateral multiple pulmonary chondroma: necessity of follow-up for Carney's triad.We report a case of pulmonary bilateral multiple chondromas that were possibly an initial clinical presentation of Carney's triad. These chondromas were possibly components of Carney's triad, because each nodule had a thin fibrous pseudocapsule and did not have an entrapped epithelium and fat.A curious association of three rare tumours was described by Carney in 1977. 'Carney's triad' characteristically includes multifocal pulmonary chondroma, gastric stromal sarcoma and extra-adrenal paraganglioma. We report a female teenager who presented with a gastrointestinal stromal tumor of the stomach and a paraganglioneuroma. She later developed a pulmonary chondroma, fulfilling the requirements of Carney's triad.A 34-year-old man, who had undergone the gastrectomy for gastrointestinal stromal tumor (GIST) and para-aortic paraganglioma 3 years before, was found to have a left lung tumor on a computed tomography. The tumor was revealed to be a pulmonary hamartoma, and diagnosed as Carney's triad.INTRODUCTION: Carney's triad is a rare pathogenic entity which consists of the association in young women of multiple condromatosis in the lung, gastric leiomyosarcoma, and extradrenal paraganglioma; although the presence of three at the same time is not required for its diagnosis.Interstitial cell of Cajal (ICC) hyperplasia has been documented in conditions associated with multiple gastrointestinal stromal tumours (GISTs) (familial GIST syndromes, Carney's triad and von Recklinghausen's disease) and rarely in the vicinity of sporadic GISTs.Multiple gastric tumors and pararenal non functioning paraganglioma were found. No chondromas were detected. An incomplete Carney's Triad was diagnosed. Gastrointestinal stromal tumours (GIST) occur mostly as sporadic solitary lesions involving the tubular GI tract and are only rarely associated with other benign or malignant neoplasms or occur as part of a multi-neoplastic disease as in the setting of Carney's triad and von Recklinghausen's disease.Carney's triad is a very unusual syndrome, associating three different tumours on the same patient, a young woman generally: a gastric leiomyoblastoma, a pulmonary chondroma and an extra-adrenal paraganglioma. This report encourages clinicians to consider a diagnosis of Carney's triad in patients with multifocal gastric stromal sarcoma, extraadrenal paraganglioma (predominantly mediastinal), or pulmonary chondroma.Carney's triad represents the association of gastric gastrointestinal stromal tumor, pulmonary chondroma, and extraadrenal paraganglioma.Thus, the Triad may be a disorder of the autonomic nervous system rather than a multiple endocrine neoplasia syndrome or multiple hamartoma syndrome.Carney's triad is a syndrome of unknown etiology, representing a combination of gastrointestinal stromal tumors, bronchial chondromas and vagal, adrenal or paraadrenal paragangliomas.Two of the Carney's triad components-the paragangliomas and the gastrointestinal stromal tumors-are potentially lethal.Carney's triad, a rare disorder affecting young females, is characterized by the presence of at least two of the three following neoplasms: gastric epithelioid leiomyosarcoma, extra-adrenal paraganglioma, and pulmonary chondroma.Carney's triad--gastric leiomyosarcoma, pulmonary chondroma, and extra-adrenal paraganglioma--is a syndrome that occurs primarily in young women.We communicate a case with the Carney triad (gastric leiomyosarcoma, pulmonary chondromatosis and extra-adrenal paraganglioma).The etiology of the Carney's triad (gastrointestinal stromal tumors, pulmonary chondromas, and paragangliomas) is unknown, and only 57 cases have been reported since its identification in 1977.Carney's triad is defined by the coexistence of at least two of three rare disorders, including gastric epithelioid leiomyosarcoma (malignant leiomyoblastoma), pulmonary chondroma, and paraganglioma, most often extra-adrenal and functioning.\"}, {\"question\": \"List 3 symptoms of Wernicke encephalopathy.\", \"ideal_answer\": [\"Wernicke's encephalopathy (WE) is a neurological syndrome caused by thiamine deficiency, and clinically characterized by ophthalmoplegia, ataxia and acute confusion.\", \"Wernicke's encephalopathy (WE) is a neurological syndrome caused by thiamine deficiency, and clinically characterized by ophthalmoplegia, ataxia and acute confusion. \"], \"exact_answer\": [[\"ophthalmoplegia\"], [\"ataxia\"], [\"acute confusion\"]], \"context\": \"Wernicke's encephalopathy (WE) is a neurological syndrome caused by thiamine deficiency, and clinically characterized by ophthalmoplegia, ataxia and acute confusion. entire triad of classic features of Wernicke-Korsakoff syndrome: confusion, ataxia, and ophthalmoplegia or nystagmus. Wernicke encephalopathy (WE) is a medical emergency caused by thiamine deficiency, characterized by cerebellar ataxia, ophthalmoplegia, and cognitive disturbancesWernicke encephalopathy is an acute neuropsychiatric disease with heterogeneous symptoms, including changes in mental status, ataxia and ocular abnormalities; if left untreated, these symptoms can lead to morbidity and even to mortality.Wernicke's encephalopathy is an acute neurological disorder characterized by mental confusion, oculomotor dysfunction, and ataxia.Wernicke's encephalopathy is an acute neurolopsychiatric syndrome caused by thiamine deficiency, and classically presents with the triad of opthalmopathy, ataxia and altered mentality.Wernicke's encephalopathy is a serious neurologic disorder caused by vitamin-B1 or thiamine deficiency. The classical triad of clinical symptoms described by Wernicke (gait ataxia, ophthalmoplegia, and confusion) are found in only a third of patients upon initial examination. Wernicke's encephalopathy is a well-described syndrome characterized by the classic triad of confusion, ataxia, and ophthalmoplegia. Wernicke encephalopathy is a severe neurologic disorder that results from a dietary vitamin B1 deficiency. It is characterized by changes in consciousness, ocular abnormalities, and ataxiaWernicke encephalopathy (WE) is a neurologic disorder characterized by clinical symptoms, such as nystagmus, ataxia, and mental confusion. Wernicke encephalopathy is a neurologic disorder that results from thiamine deficiency. It is associated with a classic triad of symptoms consisting of ataxia, ocular motor cranial neuropathies, and changes in consciousness.\"}, {\"question\": \"Which molecules are inhibited by anticancer drug Dovitinib?\", \"ideal_answer\": [\"Dovitinib (TKI-258/CHIR-258) is a pan receptor tyrosine kinase (RTK) inhibitor that targets VEGFR, FGFR, PDGFR, and KIT. It is being widely tested for treatment of various cancers.\"], \"exact_answer\": [[\"VEGFR\"], [\"FGFR\"], [\"PDGFR\"], [\"KIT\"]], \"context\": \"Dovitinib, a potent FGFR inhibitor, has demonstrated antitumor activity in heavily pretreated patients with FGFR pathway-amplified breast cancer.Dovitinib inhibits multiple tyrosine receptor kinases, predominantly the vascular endothelial growth factor receptors (VEGFR), but also FGFRs, and could be active in MPM.Dovitinib (TKI-258/CHIR-258) is a pan receptor tyrosine kinase (RTK) inhibitor that targets VEGFR, FGFR, PDGFR, and KIT.Dovitinib, a suppressor of FGFR activity, may be active in ACC.We tested dovitinib, an inhibitor of both FGF and VEGF receptors, in patients progressing on anti-VEGF treatment.Five HNSCC cell lines were characterized based on VEGFR1-3 and FGFR1-4 expression by sqRT-PCR and treated with three different tyrosine kinase inhibitors (TKIs) (nintedanib, dovitinib and pazopanib), all of which are effective against VEGFR and FGFR family members.PURPOSE: Dovitinib is an orally available multi tyrosine kinase inhibitor which inhibits VEGFR 1-3, FGFR 1-3, and PDGFR. .RESULTS: Four trials involving the selective VEGFR inhibitors axitinib, tivozanib and dovitinib were analysed, all using sorafenib as the comparator. PURPOSE: Dovitinib (TKI258) is an oral multi-tyrosine kinase inhibitor of FGFR, VEGFR, PDGFR \\u03b2, and c-Kit. Using this PDX model, we confirmed the ability of the FGFR inhibitors, ponatinib, dovitinib and BGJ398, to modulate FGFR signaling, inhibit cell proliferation and induce cell apoptosis in cholangiocarcinoma tumors harboring FGFR2 fusions.In addition to VEGFR and PDGFR, dovitinib inhibits FGFR. AZD4547, BGJ398 (infigratinib), Debio-1347 and dovitinib are FGFR1/2/3 inhibitors; BLU9931 is a selective FGFR4 inhibitor; FIIN-2, JNJ-42756493, LY2874455 and ponatinib are pan-FGFR inhibitors. AZD4547, dovitinib and ponatinib are multi-kinase inhibitors targeting FGFRs, colony stimulating factor\\u00a01 receptor\\u00a0(CSF1R), vascular endothelial growth factor\\u00a0(VEGF)R2, and others. In vitro, N87-TR cell lines demonstrated a higher sensitivity than did trastuzumab-sensitive parental cells to the FGFR3 inhibitor dovitinib, which reduced expression of pAKT, ZEB1, and cell migration.The current phase 2 study evaluated the efficacy and tolerability of dovitinib, an FGFR inhibitor, in patients with advanced SCC of the lung.Dovitinib (TKI258) is a small molecule multi-kinase inhibitor currently in clinical phase I/II/III development for the treatment of various types of cancers. We tested various T-ALL cell lines against a small-molecule inhibitor screen of 66 compounds targeting two-thirds of the tyrosine kinome and found that combined treatment of T-ALL cells with dovitinib, an orally active multi-targeted small-molecule receptor tyrosine kinase inhibitor, and OP449 synergistically reduced the viability of all tested T-ALL cell lines. We sought to identify fibroblast growth factor receptor 2 (FGFR2) kinase domain mutations that confer resistance to the pan-FGFR inhibitor, dovitinib, and explore the mechanism of action of the drug-resistant mutations.Dovitinib is a novel multi-target tyrosine kinase inhibitor that inhibits vascular endothelial growth factor receptor-1-3, platelet-derived growth factor receptor-\\u03b2, Fms-like tyrosine kinase 3, c-Kit and fibroblast growth factor receptor-1-3.Dovitinib (TKI258/CHIR258) is a multi-kinase inhibitor in phase III development for the treatment of several cancers.INTRODUCTION Erlotinib is a FDA approved small molecule inhibitor of epidermal growth factor receptor and dovitinib is a novel small molecule inhibitor of fibroblast growth factor and vascular endothelial growth factor receptor.Here, we explore the therapeutic potential of 2 FGFR inhibitors, the multikinase inhibitor dovitinib (TKI258) and the more selective FGFR inhibitor NVP-BGJ398 for the treatment of endometrial cancer.PURPOSE Dovitinib is an orally available multi tyrosine kinase inhibitor which inhibits VEGFR 1-3, FGFR 1-3, and PDGFR.The multiple kinase inhibitor dovitinib is currently under clinical investigation for hepatocellular carcinoma (HCC).\"}, {\"question\": \"List two human monoclonal antibodies against Clostridium difficile toxins.\", \"ideal_answer\": [\"Actoxumab and bezlotoxumab are human monoclonal antibodies against C. difficile toxins A and B, respectively. They were shown to decrease Clostridium difficile recurrence. Bezlotoxumab was approved by the Food and Drug Administration and the European Medicines Agency for Clostridium difficile recurrence.\"], \"exact_answer\": [[\"actoxumab\"], [\"bezlotoxumab\"]], \"context\": \"Actoxumab and bezlotoxumab are human monoclonal antibodies against C. difficile toxins A and B, respectively.Notably, the authors lay emphasis on the phase III clinical trial (MODIFY II), which allowed bezlotoxumab to be approved by the Food and Drug Administration and the European Medicines Agency. Bezlotoxumab, a human monoclonal antibody that binds and neutralizes C. difficile toxin B, is the first therapeutic agent to receive United States Food and Drug Administration approval for the prevention of CDI recurrence.The Monoclonal Antitoxin Antibodies (Actoxumab-Bezlotoxumab) Treatment Facilitates Normalization of the Gut Microbiota of Mice withHuman monoclonal antibodies MK-3415A (actoxumab-bezlotoxumab) toC. difficiletoxin A and toxin B, as an emerging non-antibiotic approach, significantly reduced the recurrence of CDI in animal models and human clinical trials. Bezlotoxumab (Zinplava\\u2122) is a human monoclonal antibody against Clostridium difficile toxin B developed by Merck&Co. Mechanisms of protection against Clostridium difficile infection by the monoclonal antitoxin antibodies actoxumab and bezlotoxumab.A combination of human monoclonal antibodies, actoxumab and bezlotoxumab, directed against TcdA and TcdB, respectively, has been shown to decrease the rate of recurrence in patients treated with standard-of-care antibiotics.  Actoxumab and bezlotoxumab are human monoclonal antibodies that neutralize the cytotoxic/cytopathic effects of TcdA and TcdB, respectively.Bezlotoxumab (Zinplava\\u2122) is a human monoclonal antibody against Clostridium difficile toxin B developed by Merck & Co.\"}, {\"question\": \"List ribosomal biogenesis proteins.\", \"ideal_answer\": [\"FGF13\\np53\\nTGF\\u03b2/Activin\\nPTEN\\nNucleostemin\\nHEATR1\"], \"exact_answer\": [[\"FGF13\"], [\"p53\"], [\"TGF\\u03b2/Activin\"], [\"PTEN\"], [\"Nucleostemin\"], [\"HEATR1\"]], \"context\": \"Regulatory module involving FGF13, miR-504, and p53 regulates ribosomal biogenesis and supports cancer cell survival.TGF\\u03b2/Activin signalling is required for ribosome biogenesis and cell growth in Drosophila salivary glands. our results show that TGF\\u03b2/Activin signalling is required for ribosomal biogenesis, a key aspect of cellular growth control. PTEN\\u03b2 is an alternatively translated isoform of PTEN that regulates rDNA transcription.Nucleostemin (NS) is a nucleolar protein that controls ribosomal biogenesisThese findings reveal an important role for HEATR1 in ribosome biogenesis and further support the concept that perturbation of ribosome biosynthesis results in p53-dependent cell cycle checkpoint activation, with implications for human pathologies including cancer.\"}, {\"question\": \"List BET proteins.\", \"ideal_answer\": [\"BRD2\\nBRD3\\nBRD4\\nBdf1\"], \"exact_answer\": [[\"Brd2\"], [\"Brd4\"], [\"Bdf1\"], [\"Brd3\"]], \"context\": \"Bromodomain-containing protein 2 (Brd2), which belongs to the bromodomain and extraterminal domain family of proteins, suppresses adipocyte differentiation. BRD4 is a member of the BET (bromodomain and extraterminal domain) family proteins the role of the BET protein Bdf1 has been explored in Saccharomyces cerevisiaeBET proteins BRD2, BRD3 and BRD4 inhibition of Brd4, a BET family member\"}, {\"question\": \"What illness is transmitted by the Lone Star Tick, Amblyomma americanum?\", \"ideal_answer\": [\"Amblyomma americanum (Lone star tick) is an important disease vector in the United States. It transmits several human pathogens, including the agents of human monocytic ehrlichiosis, tularemia, and southern tick-associated rash illness [STARI] or Masters disease\", \"Today,A americanumremains an important vector for tick-borne illness. In addition to others, species ofRickettsia,Ehrlichia, andBorreliaare all transmitted by the lone star tick.  It transmits several human pathogens, including the agents of human monocytic ehrlichiosis, tularemia, and southern tick-associated rash illness.\"], \"exact_answer\": [[\"human monocytic ehrlichiosis\"], [\"tuleremia\"], [\"southern tick-associated rash illness [STARI] or Masters disease\"], [\"tide water spotted fever\"]], \"context\": \"Today,A americanumremains an important vector for tick-borne illness. In addition to others, species ofRickettsia,Ehrlichia, andBorreliaare all transmitted by the lone star tick. Southern tick-associated rash illness and anaphylaxis to red meat following tick bites have been attributed to the lone star tick.Lone Star tick (Amblyomma americanum) and the rash is called STARI (southern tick-associated rash illness).Amblyomma americanum (Lone star tick) is an important disease vector in the United States. It transmits several human pathogens, including the agents of human monocytic ehrlichiosis, tularemia, and southern tick-associated rash illness.E. chaffeensis is maintained in a complex cycle involving white-tailed deer (WTD; Odocoileus virginianus) as a primary reservoir and the lone star tick (LST; Amblyomma americanum) as a primary vector.Amblyomma americanum, the lone star tick, is the most common and most aggressive human biting tick in the Southeastern United States. It is known to transmit the agents of human ehrlichioses, Ehrlichia chaffeensis and Ehrlichia ewingii. Rickettsia parkeri is a member of the spotted fever group rickettsiae and causes a febrile illness in humans commonly referred to as \\\"Tidewater spotted fever\\\" or \\\"R. parkeri rickettsiosis.\\\" Although the Gulf Coast tick, Amblyomma maculatum Koch, is the primary vector of R. parkeri, a small proportion of A. americanum have also been shown to harbor R. parkeri. Lyme-like illness (also known as southern tick-associated rash illness [STARI] or Masters disease) is vectored by the Lone Star tick (Amblyomma americanum).Transmission of Ehrlichia chaffeensis from lone star ticks (Amblyomma americanum) to white-tailed deer (Odocoileus virginianus).Monocytic ehrlichiosis in people caused by the intracellular bacterium, Ehrlichia chaffeensis, is an emerging infectious disease transmitted by the lone star tick, Amblyomma americanumSouthern tick-associated rash illness is a Lyme-like syndrome that occurs in the southern states. Borrelia lonestari, which has been suggested as a possible causative agent of southern tick-associated rash illness, naturally infects white-tailed deer (WTD; Odocoileus virginianus) and is transmitted by the lone star tick (Amblyomma americanum)\"}, {\"question\": \"List the continent of origin for the brown marmorated stinkbug(Halyomorpha halys)\", \"ideal_answer\": [\"The brown marmorated stinkbug (Halyomorpha halys) is native to Asia\"], \"exact_answer\": [[\"Asia\"]], \"context\": \"The brown marmorated stink bug, Halyomorpha halys (St\\u00e5l) (BMSB) is an exotic invasive insect originating in East Asia, currently causing significant damage to fruits, vegetables and other crops throughout most of the Mid-Atlantic states of the U.S.The brown marmorated stink bug, Halyomorpha halys (St\\u00e5l), is a polyphagous pest indigenous to northeastern Asia where it damages various trees, vegetables, and leguminous crops.The brown marmorated stink bug (BMSB, Halyomorpha halys) is an East Asian species now established across North America and Europe, that in the Eastern United States of America (US) and Italy is causing significant economic losses to agriculture.The brown marmorated stink bug, Halyomorpha halys (St\\u00e5l), is an invasive species native to regions of China, Japan, Korea, and Taiwan.The brown marmorated stink bug, Halyomorpha halys, a native of Asia, has become a serious invasive pest in the USA.The brown marmorated stink bug, Halyomorpha halys (St\\u00e5l), native to China, Japan, and Korea, has emerged as a harmful invasive pest of a variety of crops in North America and Europe.The brown marmorated stink bug, Halyomorpha halys, is a highly polyphagous Asian herbivore and an economically important invasive pest in North America and Europe.Brown marmorated stink bug, Halyomorpha halys (St\\u00e5l), (Hemiptera: Pentatomidae) is an invasive polyphagous agricultural and urban nuisance pest of Asian origin that is becoming widespread in North America and Europe.The brown marmorated stink bug, Halyomorpha halys, is native to Asia (China, Taiwan, Japan, and the Korean peninsula).The brown marmorated stink bug, Halyomorpha halys (St\\u00e5l), is an invasive species from Asia capable of causing severe agricultural damage.The brown marmorated stink bug, Halyomorpha halys (St\\u00e5l) (BMSB) is an exotic invasive insect originating in East Asia, currently causing significant damage to fruits, vegetables and other crops throughout most of the Mid-Atlantic states of the U.S.The brown marmorated stink bug (BMSB), Halyomorpha halys (St\\u00e5l), is an invasive pentatomid introduced from Asia into the United States, Canada, multiple European countries, and Chile. The invasive brown marmorated stink bug, Halyomorpha halys (St\\u00e5l; Hemiptera: Pentatomidae), has recently emerged as a harmful pest of horticultural crops in North America and Europe. Native to East Asia, this highly polyphagous insect is spreading rapidly worldwide. The brown marmorated stink bug (BMSB), Halyomorpha halys (St\\u00e5l) (Hemiptera: Pentatomidae), an invasive and polyphagous insect pest from Asia,The brown marmorated stink bug, Halyomorpha halys (St\\u00e5l), is native to eastern Asia and is presently invading North America.\"}, {\"question\": \"Please list 6 symptoms of Scarlet fever.\", \"ideal_answer\": [\"Symptoms of scarlet fever include fever, rash, strawberry tongue and sore throat. In some cases other symptoms, angular stomatitis, tonsular exudate, and swollen lymph nodes are seen.\"], \"exact_answer\": [[\"fever\"], [\"rash\"], [\"sore throat\"], [\"strawberry tongue\"], [\"angular stomatitis\"], [\"swollen lymph nodes\"]], \"context\": \"A fever is likely to occur with roseola, erythema infectiosum (fifth disease), and scarlet fever.The rash associated with scarlet fever usually develops on the upper trunk, then spreads throughout the body, sparing the palms and solesScarlet fever consists in a diffuse exanthem associated with mucous changes.Cases of above-38 degrees C temperature were seen in about 81.4%, and from 2 to 5 days-duration of temperature were seen in 86.6% of the patients in the year 1976. Cases of above-moderate rash were observed in 68.2%, sever redness of throat in 29.9%, strawberry tongue in 86.3% and angular stomatitis in 37.7% of the patientsulture supernatants of strains without a detectable amount of the known ETs were highly mitogenic, indicating the production of other streptococcal mitogens. A correlation with clinical symptoms was determined with regard to exanthema and fever. Strains producing two or three toxins caused a more intense exanthema. Patient temperature was higher (greater than or equal to 38 degrees C) when the infecting strain produced ETB.) SI with a pronounced generalization including a pharyngeal one (31 cases), 4 of them with a rash (scarlet fever); extrapharyngeal (7 cases), 6 of them with a rash (scarlet fever); 2) SI without pronounced generalization (localized) including 33 cases with the involvement of the lungs and tonsilla and having an ordinary course and 11 cases of a sudden death. A total of 171 out of 252 scarlet fever diagnoses were microbiologically verified in 158 patients. The median age was 3.8 years (interquartile range: 2.91-4.78), with the majority (57%) under the age of 4 years. There was fever in 89% of the processes (95% CI: 84-94%), with a temperature of>38\\u00b0C in 73% (95% CI: 65-80%), enlarged lymph nodes in 70% (95% CI: 58-82%), absence of cough in 73% (95% CI: 65-80%), and tonsillar exudate in only 24% (95% CI: 17-31%).\"}, {\"question\": \"What is included in the Mentzer index?\", \"ideal_answer\": [\"Mentzer index (MCV/RBC) is mean corpuscular volume (MCV) and red blood cell count (RBC) ratio. It is used for differentiation of thalassemia and iron deficiency anemia.\"], \"exact_answer\": [[\"mean corpuscular volume\"], [\"red blood cell count\"]], \"context\": \"In this study, macrocytic anemia due to FAD was defined as having an MCV \\u2265100 fL and folic acid \\u22646 ng/mL; pernicious anemia as having MCV \\u2265100 fL, vitamin B12<200 pg/mL, and serum gastric parietal cell antibody positivity; iron deficiency anemia as having MCV<80 fL and iron<60 \\u03bcg/dL; and thalassemia trait as having MCV<74 fL, red blood cell (RBC) count>5.0 \\u00d7 10(12)/L, and Mentzer index (MCV/RBC)<13.Complete blood count (CBC) parameters of 2196 certainly diagnosed (1272 beta-thalassaemia minor and 924 iron deficiency) samples were used to evaluate the following indices and formulas: Bessman index (RDW), Mentzer formula (MCV/RBC), England and Fraser formula (MCV - RBC - 5 x Hb- 3.4), Shine and Lal formula (MCV2 x MCH/100), Ehsani formula (MCV-10 x RBC), Srivastava formula (MCH/RBC), Green and King formula (MCV2 x RDW/Hb x 100), red distribution width index RDWI (RDW x MCV/RBC), RDW/RBC, as well as our formula (MCV-RBC -3 x Hb). The ratio of the mean corpuscular volume (MCV) and red blood cell count (RBC) can be automatically calculated with any of the newer hematology analyzers.METHODS: The results of 398 patient screens were collected. Data from the set were divided into training and validation subsets. The Mentzer ratio was determined through a receiver operating characteristic (ROC) curve on the first subset, and screened for thalassemia using the second subset. Mentzer ratio MCV/RBC detected all cases of beta thalassemia but was a poor index for iron deficiency detection. \"}, {\"question\": \"List drugs that are included in the Vosevi polypill.\", \"ideal_answer\": [\"Vosevi pill includes sofosbuvir, velpatasvir and voxilaprevir. It is approved by the US Food and Drug Administration (FDA) for adult patients with chronic hepatitis C virus (HCV) infection without cirrhosis or with compensated cirrhosis (Child-Pugh A) who have: genotype 1, 2, 3, 4, 5, or 6 infection and have previously been treated with an HCV regimen containing an NS5A inhibitor; and genotype 1a or 3 infection and have previously been treated with an HCV regimen containing sofosbuvir without an NS5A inhibitor.\"], \"exact_answer\": [[\"sofosbuvir\"], [\"velpatasvir\"], [\"voxilaprevir\"]], \"context\": \"On July 18, 2017, the US Food and Drug Administration (FDA) approved sofosbuvir/velpatasvir/voxilaprevir (SOF/VEL/VOX) (Vosevi) fixed-dose combination (FDC), an interferon-free, complete regimen for adult patients with chronic hepatitis C virus (HCV) infection without cirrhosis or with compensated cirrhosis (Child-Pugh A) who have: genotype 1, 2, 3, 4, 5, or 6 infection and have previously been treated with an HCV regimen containing an NS5A inhibitor; and genotype 1a or 3 infection and have previously been treated with an HCV regimen containing sofosbuvir without an NS5A inhibitor.\"}, {\"question\": \"Which algorithms are used for compression of SAM files?\", \"ideal_answer\": [\"The most popular format for genomic data is the SAM (Sequence Alignment/Map) format, which contains information such as alignment, quality values, etc. These files are large (on the order of terabytes), which necessitates compression. GeneComp, NGC, SAMZIP and QVZ are algorithms which perform compression of data stored in SAM files.\"], \"exact_answer\": [[\"GeneComp\"], [\"NGC\"], [\"SAMZIP\"], [\"QVZ\"], [\"CSAM\"]], \"context\": \"GeneComp, a new reference-based compressor for SAM filesThe most popular format for genomic data is the SAM format, which contains information such as alignment, quality values, etc. These files are large (on the order of terabytes), which necessitates compression. In this work we propose a new reference-based compressor for SAM files, which can accommodate different levels of compression, based on the specific needs of the user. In particular, the proposed compressor GeneComp allows the user to perform lossy compression of the quality scores, which have been proven to occupy more than half of the compressed file (when losslessly compressed). We show that the proposed compressor GeneComp overall achieves better compression ratios than previously proposed algorithms when working on lossless mode.The enormous size of these data motivates the development of data compression algorithms usable for the implementation of the various storage policies that are applied to the produced intermediate and final result files. In this article, we present NGC, a tool for the compression of mapped short read data stored in the wide-spread SAM format. NGC enables lossless and lossy compression and introduces the following two novel ideas: first, we present a way to reduce the number of required code words by exploiting common features of reads mapped to the same genomic positions; second, we present a highly configurable way for the quantization of per-base quality values, which takes their influence on downstream analyses into account. NGC, evaluated with several real-world data sets, saves 33-66% of disc space using lossless and up to 98% disc space using lossy compression. By applying two popular variant and genotype prediction tools to the decompressed data, we could show that the lossy compression modes preserve>99% of all called variants while outperforming comparable methods in some configurations.Research in bioinformatics primarily involves collection and analysis of a large volume of genomic data. Naturally, it demands efficient storage and transfer of this huge amount of data. In recent years, some research has been done to find efficient compression algorithms to reduce the size of various sequencing data. One way to improve the transmission time of large files is to apply a maximum lossless compression on them. In this paper, we present SAMZIP, a specialized encoding scheme, for sequence alignment data in SAM (Sequence Alignment/Map) format, which improves the compression ratio of existing compression tools available.Recent advancements in sequencing technology have led to a drastic reduction in the cost of sequencing a genome. This has generated an unprecedented amount of genomic data that must be stored, processed and transmitted. To facilitate this effort, we propose a new lossy compressor for the quality values presented in genomic data files (e.g. FASTQ and SAM files), which comprise roughly half of the storage space (in the uncompressed domain). Lossy compression allows for compression of data beyond its lossless limit.RESULTS: The proposed algorithm QVZ exhibits better rate-distortion performance than the previously proposed algorithms, for several distortion metrics and for the lossless case. Moreover, it allows the user to define any quasi-convex distortion function to be minimized, a feature not supported by the previous algorithms. Finally, we show that QVZ-compressed data exhibit better performance in the genotyping than data compressed with previously proposed algorithms, in the sense that for a similar rate, a genotyping closer to that achieved with the original quality values is obtained.In this paper, we present SAMZIP, a specialized encoding scheme, for sequence alignment data in SAM (Sequence Alignment/Map) format, which improves the compression ratio of existing compression tools available.We describe CSAM (Compressed SAM format), a compression approach offering lossless and lossy compression for SAM files.GeneComp, a new reference-based compressor for SAM files.\"}, {\"question\": \"List 4 drugs used to treat opioid addiction or overdose\", \"ideal_answer\": [\"Suboxone (buprenorphine/naloxone) and  methadone are used to assist in opioid withdrawal and Naloxone is used to treat overdoses\"], \"exact_answer\": [[\"methadone\"], [\"naloxone\"], [\"suboxone\"], [\"buprenorphine\"]], \"context\": \"Currently, the American College of Obstetricians and Gynecologists recommends methadone, buprenorphine, or buprenorphine/naloxone in the treatment of prenatal opioid abuse.PubMed was searched and literature was reviewed on the effectiveness, safety, and side effect profiles of abstinence-based treatment, buprenorphine-naloxone treatment, and methadone treatment.Methadone is an opioid prescribed for pain management and is also provided through opioid treatment programs to treat opioid use disorders.Fortunately, there are effective medications (ie, methadone, buprenorphine, and oral and injectable naltrexone) available for the treatment of opioid addiction. We describe the major classes of drug treatments available, including opioid agonist (e.g., methadone, buprenorphine, LAAM), antagonist (e.g., naltrexone)he development of buprenorphine, a partial opioid agonist, as an effective treatment for opioid addiction reopened the possibility for having a less burdensome oversight process,Although the synthetic opioid buprenorphine has been available clinically for almost 30 years, its use has only recently become much more widespread for the treatment of opioid addiction\"}, {\"question\": \"List major risk factors for Alzheimer's disease. \", \"ideal_answer\": [\"Apolipoprotein E4\\ntype 2 diabetes\\nClusterin\\nHypertension\\nadvancing age\\nobesity\"], \"exact_answer\": [[\"Apolipoprotein E4\"], [\"type 2 diabetes\"], [\"Clusterin\"], [\"Hypertension\"], [\"advancing age\"], [\"obesity\"]], \"context\": \"Apolipoprotein E4 (E4) and type 2 diabetes are major risk factors for cognitive decline and late onset Alzheimer's disease (AD)Cardiovascular risk factors, especially hypertension, are also major risk factors for Alzheimer's disease (AD).Clusterin (Apo J) is one of the major risk factors for sporadic form of AD. Hypertension is one of the major risk factors for central nervous system (CNS) disorders like stroke and Alzheimer's disease (AD).Major risk factors for AD are advancing age and diabetes. Lately, obesity has been associated with an increased risk of dementia.\"}, {\"question\": \"What do the trispecific HIV antibodies target?\", \"ideal_answer\": [\"Trispecific HIV antibodies (Abs) allow a single molecule to interact with three independent HIV-1 envelope determinants: the CD4 binding site, the membrane-proximal external region (MPER), and the V1V2 glycan site.\"], \"exact_answer\": [[\"CD4 binding site\"], [\"membrane-proximal external region (MPER)\"], [\"V1V2 glycan site\"]], \"context\": \"We engineered trispecific antibodies (Abs) that allow a single molecule to interact with three independent HIV-1 envelope determinants: the CD4 binding site, the membrane-proximal external region (MPER), and the V1V2 glycan site.\"}, {\"question\": \"Which cellular functions are affected by lncRNA H19 in the heart?\", \"ideal_answer\": [\"H19 could inhibit autophagy in cardiomyocytes by epigenetically silencing of DIRAS3. Elevated H19 promotes apoptosis through PA2G4. Downregulation of H19 promotes proliferation and inhibits apoptosis. H19 induces mineralization of valve interstitial cells. H19 contributes to cardiac fibroblast proliferation and fibrosis, which act in part through repression of DUSP5/ERK1/2.\"], \"exact_answer\": [[\"inhibits autophagy\"], [\"promotes apoptosis\"], [\"inhibits proliferation\"], [\"induced mineralization of valve interstitial cells\"], [\"promotes cardiac fibroblast proliferation\"], [\"promotes fibrosis\"]], \"context\": \"In addition, overexpression of H19 was found to downregulate DIRAS3 expression, promote mTOR phosphorylation and inhibit autophagy activation in cardiomyocytes exposed to high glucose. In conclusion, our study suggested that H19 could inhibit autophagy in cardiomyocytes by epigenetically silencing of DIRAS3, which might provide novel insights into understanding the molecular mechanisms of diabetic cardiomyopathy.LncRNA H19 inhibits autophagy by epigenetically silencing of DIRAS3 in diabetic cardiomyopathy.The long non-coding RNA H19 promotes cardiomyocyte apoptosis in dilated cardiomyopathy.H19 knockdown in the myocardium of DCM rats attenuated cardiomyocyte apoptosis and improved left ventricular structure and function. Adriamycin treatment was associated with elevated H19 and miR-675 expression and increased apoptosis in neonatal cardiomyocytes.Moreover, H19 knockdown was found to increase PA2G4 expression and suppress apoptosis in cardiomyocytes exposed to adriamycin. In conclusion, our study suggests that H19/miR-675 axis is involved in the promotion of cardiomyocyte apoptosis by targeting PA2G4, which may provide a new therapeutic strategy for the treatment of adriamycin-induced DCM.Downregulation of long non-coding RNA H19 promotes P19CL6 cells proliferation and inhibits apoptosis during late-stage cardiac differentiation via miR-19b-modulated Sox6.Biological function analysis showed that knockdown of H19 promoted cell proliferation and inhibits cell apoptosis. H19 suppressed miR-19b expression and miR-19b targeted Sox6, which inhibited cell proliferation and promoted apoptosis in P19CL6 cells during late-stage cardiac differentiation.Downregulation of H19 promoted cell proliferation and inhibited cell apoptosis during late-stage cardiac differentiation by regulating the negative role of miR-19b in Sox6 expression, which suggested that the manipulation of H19 expression could serve as a potential strategy for heart disease.Knockdown and overexpression experiments showed that H19 induces a strong osteogenic phenotype by altering the NOTCH1 pathway.A knockdown of H19 in valve interstitial cells (VICs) increased the expression of NOTCH1 and decreased the level of RUNX2 and BMP2, 2 downstream targets repressed by NOTCH1. In rescue experiments, the transfection of a vector encoding for the active Notch intracellular domain prevented H19-induced mineralization of valve interstitial cellsLong noncoding RNA H19 controls DUSP5/ERK1/2 axis in cardiac fibroblast proliferation and fibrosis.H19 ectopic overexpression reduces DUSP5 abundance and increases the proliferation of cardiac fibroblast, whereas H19 silencing causes the opposite effects. In a broader perspective, these results demonstrated that LncRNA H19 contributes to cardiac fibroblast proliferation and fibrosis, which act in part through repression of DUSP5/ERK1/2.\"}, {\"question\": \"Name 4 circular RNA molecules associated with carcinogenesis.\", \"ideal_answer\": [\"circ-ABCB10 knockdown suppressed the proliferation and increased apoptosis of breast cancer cells.\\nHsa_circ_0058246 was elevated in tumor specimens of patients with poor clinical outcomes.\\nCirc-FBXW7 expression positively associated with glioblastoma patient overall survival.\\nciRS-7 promotes the development of cancer by acting as sponge of miR-7.\"], \"exact_answer\": [[\"ciRS-7\"], [\"FBXW7\"], [\"circ-ABCB10\"], [\"hsa_circ_0058246\"]], \"context\": \" In particular, we found that the expression of hsa_circ_0058246 was elevated in tumor specimens of patients with poor clinical outcomes.Circular RNA circ-ABCB10 promotes breast cancer proliferation and progression through sponging miR-1271.Results found circ-ABCB10, was significantly up-regulated in breast cancer tissue.In vitro, loss-of-function experiments showed circ-ABCB10 knockdown suppressed the proliferation and increased apoptosis of breast cancer cells. Overall, results identified a new functional circ-ABCB10 in breast cancer tumorigenesis, and reveal the important regulatory role of circ-ABCB10 through sponging miR-1271, providing a novel insight for breast cancer pathogenesis.Novel Role of FBXW7 Circular RNA in Repressing Glioma Tumorigenesis.Upregulation of FBXW7-185aa in cancer cells inhibited proliferation and cell cycle acceleration, while knockdown of FBXW7-185aa promoted malignant phenotypes invitro and invivo. Moreover, circ-FBXW7 and FBXW7-185aa levels were reduced in glioblastoma clinical samples compared with their paired tumor-adjacent tissues (P<.001). Circ-FBXW7 expression positively associated with glioblastoma patient overall survival (P = .03).Conclusions: Endogenous circRNA encodes a functional protein in human cells, and circ-FBXW7 and FBXW7-185aa have potential prognostic implications in brain cancer.Detection and characterization of ciRS-7: a potential promoter of the development of cancer.Most recently, a study reported ciRS-7 acted as an oncogene in hepatocellular carcinoma through targeting miR-7 expression. This suggest ciRS-7/ miR-7 axis affects oncogenesis, and it provides a new perspective on the mechanisms of decreased miR-7 expression in cancer tissues. This review summarizes the structure and function of circRNAs and provides evidence for the impact of ciRS-7 in promoting the development of cancer by acting as sponge of miR-7.\"}, {\"question\": \"Please list 3 diseases treated with Valtrex(valacyclovir)\", \"ideal_answer\": [\"Valtrex (valacyclovir) is an antiviral medication used to treat infections with: herpes zoster (shingles), herpes simplex genitalis (genital herpes),\\nand herpes labialis (cold sores).\"], \"exact_answer\": [[\"Herpes zoster or Shingles\"], [\"Herpes genitalis, or genital herpes\"], [\"Herpes labialis or cold sores\"]], \"context\": \"Valaciclovir (Valtrex, Zelitrex), the L-valine ester of aciclovir, increases aciclovir bioavailability by 3- to 5-fold over that achievable with oral aciclovir. It addresses many unmet needs of currently available anti-herpesvirus therapieIn controlled clinical trials in herpes zoster, valaciclovir (1000 mg three times daily) is superior to aciclovir in speeding the resolution of zoster-associated pain and post-herpetic neuralgia.alacyclovir is the only antiviral drug approved for a once-daily dose of suppressive therapy for genital herpes, as well as the only antiviral drug US FDA approved for a 3-day regimen of episodic treatment of recurrent genital herpes.valacyclovir is also indicated in the reduction of the sexual transmission of herpes simplex virus infection and for the treatment of herpes labialis. n herpes zoster, valacyclovir is more effective than acyclovir or placebo (and as equally effective as famciclovir) in shortening the length and severity of herpes zoster-associated pain and postherpetic neuralgia. valacyclovir and famciclovir are efficacious and safe for the treatment of the first episode and recurrent genital herpes and are useful as suppressive therapy for individuals with frequent genital herpes recurrences.valacyclovir and famciclovir have been shown to speed the healing of herpes zoster, and data suggests that these agents also decrease associated acute and chronic pain in people of 50 years of age or older.Valaciclovir (Valtrex), the L-valyl ester of acyclovir, is undergoing clinical development for the treatment and suppression of herpesviral diseases.Valacyclovir is frequently prescribed to the elderly to treat diseases such as herpes zoster.The study group comprised 86 immunocompetent adult patients suffering from herpes zoster, who were treated with either famciclovir or valacyclovir for 7 days.Over the past two decades, the treatment of these conditions has been transformed by guanosine nucleoside antivirals such as valacyclovir (Valtrex, a highly bioavailable prodrug of acyclovir (Zovirax, and famciclovir (Famvir), a highly bioavailable prodrug of penciclovir (Denavir).<br><b>OBJECTIVE</b>: We describe the pharmacology, pharmacokinetics, and clinical efficacy of valacyclovir for the treatment of herpes simplex, herpes zoster, and other viral infections.In herpes zoster, valacyclovir is as effective as famciclovir and more effective than either placebo or acyclovir at facilitating cutaneous healing and healing of zoster-associated pain and post-herpetic neuralgia.OBJECTIVE: We describe the pharmacology, pharmacokinetics, and clinical efficacy of valacyclovir for the treatment of herpes simplex, herpes zoster, and other viral infections.Valacyclovir is well tolerated, with convenient dosing frequencies for the treatment of genital herpes or herpes zoster, it also has the option for use as a short course therapy in the episodic treatment of recurrent genital herpes, all of which are important benefits in the management of these conditions.Acyclovir (Zoviraxr, GlaxoSmithKline), valacyclovir (Valtrex, GlaxoSmithKline) or famciclovir (Famvir, Novartis) can be used to treat herpes zoster, and all three have been shown to reduce the duration of the herpetic rash and zoster-associated pain.\"}, {\"question\": \"Which genes are responsible for the high-altitude adaptation of Tibetans?\", \"ideal_answer\": [\"Recent studies have identified genes involved in high-altitude adaptation in Tibetans. Genetic variants/haplotypes within regions containing three of these genes (EPAS1, EGLN1, and PPARA) are associated with relatively decreased hemoglobin levels observed in Tibetans at high altitude, providing corroborative evidence for genetic adaptation to this extreme environment.  A gene (HMOX2) involved in heme catabolism, harbors potentially adaptive variants in Tibetans.\", \"Recent studies have identified genes involved in high-altitude adaptation in Tibetans. Three of these genes, EPAS1, EGLN1 and PPARA, regulate or are regulated by hypoxia inducible factor, a principal controller of erythropoiesis and other organismal functions. Two functional loci in the promoter of EPAS1 gene involved in high-altitude adaptation of Tibetans.\", \"EGLN1 (or HIFPH2, MIM 606425) and EPAS1 (or HIF2A, MIM 603349), both related to hypoxia-inducible factor, were found most differentiated in the two regions, respectively. Genetic variants/haplotypes within regions containing three of these genes (EPAS1, EGLN1, and PPARA) are associated with relatively decreased hemoglobin levels observed in Tibetans at high altitude, providing corroborative evidence for genetic adaptation to this extreme environment. Three of these genes, EPAS1, EGLN1 and PPARA, regulate or are regulated by hypoxia inducible factor, a principal controller of erythropoiesis and other organismal functions.\", \"Genetic variants/haplotypes within regions containing three of these genes (EPAS1, EGLN1, and PPARA) are associated with relatively decreased hemoglobin levels observed in Tibetans at high altitude, providing corroborative evidence for genetic adaptation to this extreme environment. \"], \"exact_answer\": [[\"EGLN1\", \"HIFPH2\"], [\"EPAS1\", \"HIF2A\"], [\"PPARA\"], [\"HMOX2\"]], \"context\": \"We discovered the most differentiated variants between TBN and HAN at chromosome 1q42.2 and 2p21. EGLN1 (or HIFPH2, MIM 606425) and EPAS1 (or HIF2A, MIM 603349), both related to hypoxia-inducible factor, were found most differentiated in the two regions, respectively.All of our population genomic and statistical analyses indicate that EPAS1 and EGLN1 are most likely responsible for HAA of Tibetans.Recent studies have identified genes involved in high-altitude adaptation in Tibetans. Genetic variants/haplotypes within regions containing three of these genes (EPAS1, EGLN1, and PPARA) are associated with relatively decreased hemoglobin levels observed in Tibetans at high altitude, providing corroborative evidence for genetic adaptation to this extreme environment. In combination with the reported data, we identified strong signals of selective sweep in two hypoxia-related genes, EPAS1 and EGLN1. Three of these genes, EPAS1, EGLN1 and PPARA, regulate or are regulated by hypoxia inducible factor, a principal controller of erythropoiesis and other organismal functions.Identification of a Tibetan-specific mutation in the hypoxic gene EGLN1 and its contribution to high-altitude adaptation.Here we conducted resequencing of the entire genomic region (59.4 kb) of the hypoxic gene EGLN1 (one of the top candidates from the genome-wide scans) in Tibetans and identified 185 sequence variations, including 13 novel variations (12 substitutions and 1 insertion or deletion).The estimated selective intensity (0.029 for the C allele of rs186996510) puts EGLN1 among the known genes that have undergone the strongest selection in human populations, and the onset of selection was estimated to have started at the early Neolithic (\\u223c8,400 years ago). Two functional loci in the promoter of EPAS1 gene involved in high-altitude adaptation of Tibetans.EPAS1 involves in the hypoxic response and is suggested to be responsible for the genetic adaptation of high-altitude hypoxia in Tibetans.Three of these genes (EPAS1, EGLN1 and PPARA) are associated with decreased haemoglobin levels compared with non-Tibetans living at altitude. HMOX2 Functions as a Modifier Gene for High-Altitude Adaptation in Tibetans.In this study, we report a downstream gene (HMOX2) involved in heme catabolism, which harbors potentially adaptive variants in Tibetans. Collectively, we propose that HMOX2 contributes to high-altitude adaptation in Tibetans by functioning as a modifier in the regulation of hemoglobin metabolism.Positively selected haplotypes of EGLN1 and PPARA were significantly associated with the decreased hemoglobin phenotype that is unique to this highland population. Studies of Tibetans revealed candidates for high-altitude adaptations in the EGLN1 and EPAS1 genes, associated with lower haemoglobin concentration.Here we analyse genotype data for the Nepalese Sherpa, and find that Tibetans are a mixture of ancestral populations related to the Sherpa and Han Chinese. EGLN1 and EPAS1 genes show a striking enrichment of high-altitude ancestry in the Tibetan genome, indicating that migrants from low altitude acquired adaptive alleles from the highlanders.Genetic variants/haplotypes within regions containing three of these genes (EPAS1, EGLN1, and PPARA) are associated with relatively decreased hemoglobin levels observed in Tibetans at high altitude, providing corroborative evidence for genetic adaptation to this extreme environment.All of the population genomic and statistical analysis indicated that EPAS1 and EGLN1 are mostly likely responsible for high altitude adaptation and closely related to low Hb concentration in Tibetans.Interestingly, one gene previously known to be important in cellular oxygen sensing, EGLN1 (also known as PHD2), shows evidence of positive selection in both Tibetans and Andeans.(or HIFPH2, MIM 606425) and EPAS1 (or HIF2A, MIM 603349), both related to hypoxia-inducible factor, were found most differentiated in the two regions, respectively.It demonstrated that the C allele of rs56721780:G C decreased the binding of IKZF1, leading to the attenuated transcriptional repression of EPAS1 gene.EGLN1 and EPAS1 genes show a striking enrichment of high-altitude ancestry in the Tibetan genome, indicating that migrants from low altitude acquired adaptive alleles from the highlanders.Genetic variants/haplotypes within regions containing three of these genes (EPAS1, EGLN1, and PPARA) are associated with relatively decreased hemoglobin levels observed in Tibetans at high altitude, providing corroborative evidence for genetic adaptation to this extreme environment.All of the population genomic and statistical analysis indicated that EPAS1 and EGLN1 are mostly likely responsible for high altitude adaptation and closely related to low Hb concentration in Tibetans.The Tibetan-specific 200 kb EPAS1 haplotype introgressed from an archaic human population related to Denisovans which underwent evolutionary decay; however, the functional variant(s) responsible for high-altitude adaptation at EPAS1/HIF-2\\u03b1 have not yet been identified.Further functional analysis revealed that lysyl oxidase (LOX) gene, which was reported to be responsible for extracellular matrix protein cross-linking of amnion previously, was a direct target of EPAS1.Two functional loci in the promoter of EPAS1 gene involved in high-altitude adaptation of Tibetans.\"}, {\"question\": \"With which personality traits has the human monoamine oxidase A (MAOA) gene been associated?\", \"ideal_answer\": [\"Association studies suggest that the low activity variant of the monoamine oxidase A (MAOA)-uVNTR polymorphism confers risk for emotional disturbances associated with antisocial traits, particularly in males. These include antisocial and borderline personality disorders and antisocial aggression.\", \"Association of monoamine oxidase-A genetic variants and amygdala morphology in violent offenders with antisocial personality disorder and high psychopathic traits. The interaction of the low activity variant of the monoamine oxidase-A (MAOA-L) gene and early childhood adversity has been shown to predict aggression in clinical and non-clinical populations Monoamine Oxidase-A Genetic Variants and Childhood Abuse Predict Impulsiveness in Borderline Personality Disorder.\", \"Association of monoamine oxidase-A genetic variants and amygdala morphology in violent offenders with antisocial personality disorder and high psychopathic traits. \", \"Association of monoamine oxidase-A genetic variants and amygdala morphology in violent offenders with antisocial personality disorder and high psychopathic traits. Monoamine oxidase a gene is associated with borderline personality disorder. Gene environment interactions with a novel variable Monoamine Oxidase A transcriptional enhancer are associated with antisocial personality disorder.\", \"Monoamine oxidase A (MAOA) gene has an important role in the regulation of neurotransmitter levels and a large number of human behaviors. For example, it has been shown that childhood maltreatment interacts with a monoamine oxidase A (MAOA) gene variant to predict antisocial behavior that is often associated with alcoholism, and an interaction between early life stress and a serotonin transporter promoter variant predicts alcohol abuse in nonhuman primates and depression in humans. The purpose of the present study was to examine whether a cumulative genetic score (CGS) containing the monoamine oxidase A (MAOA) and the human serotonin transporter gene linked polymorphism (5-HTTLPR) was associated with IPV perpetration after accounting for the effects of alcohol problems, drug problems, age, and length of relationship. One approach to this question is examining allelic variation in the X-linked monoamine oxidase A (MAOA) gene, previously associated with impulsive aggression in animals and humans. Allelic variation of the monoamine oxidase A (MAOA) gene has been implicated in conduct disorder and antisocial, aggressive behavior in humans when associated with early adverse experiences.\", \"Association of monoamine oxidase-A genetic variants and amygdala morphology in violent offenders with antisocial personality disorder and high psychopathic traits.\"], \"exact_answer\": [[\"antisocial personality disorder\"], [\"borderline personality disorder\"], [\"antisocial aggression\"]], \"context\": \"Association of monoamine oxidase-A genetic variants and amygdala morphology in violent offenders with antisocial personality disorder and high psychopathic traits.The interaction of the low activity variant of the monoamine oxidase-A (MAOA-L) gene and early childhood adversity has been shown to predict aggression in clinical and non-clinical populationsMonoamine Oxidase-A Genetic Variants and Childhood Abuse Predict Impulsiveness in Borderline Personality Disorder. These preliminary results suggest that MAOA-L may modulate the impact of childhood abuse on impulsivity in BPD. Results additionally indicate that impulsiveness may be expressed differently in BPD and ASPD.Elevated Monoamine Oxidase-A Distribution Volume in Borderline Personality Disorder Is Associated With Severity Across Mood Symptoms, Suicidality, and Cognition.These results suggest that elevated MAO-A VT is associated with multiple indicators of BPD severity, including BPD symptomatology, mood symptoms, suicidality, and neurocognitive impairment.No association between a polymorphism in the promoter region of the MAOA gene with antisocial personality traits in alcoholics.Taken together, these findings suggest that the three-repeat allele of the MAOAuVNTR 30-bp polymorphism is not associated with impulsive and aggressive personality traits.Different allele distribution of a regulatory MAOA gene promoter polymorphism in antisocial and anxious-depressive alcoholicsThe present study tested whether a novel functional polymorphism in the promotor region of the X-chromosomal monoamine oxidase A gene (MAOA) was related to antisocial and anxious-depressive traits in alcoholics.aken together, these findings suggest that the 3-repeat allele of the MAOA polymorphism contributes modestly to the dimension of overand underreactive behaviors as possible antecedents of alcoholism.Association studies suggest that the low activity variant of the monoamine oxidase A (MAOA)-uVNTR polymorphism confers risk for emotional disturbances associated with antisocial traits, particularly in males. A polymorphism of the MAOA gene is associated with emotional brain markers and personality traits on an antisocial indexThe monoamine oxidase A (MAOA) gene has been shown to moderate the impact of maltreatment on antisocial behaviour.Non-linear interactions between the MAOA gene and violence were detected, suggesting that the genetic moderation may come about once a certain level of violence is experiencedThe history of research on the association between platelet monoamine oxidase (MAO) activity and personality traits, such as sensation seeking and impulsiveness, is reviewed. Monoamine oxidase a gene is associated with borderline personality disorder.As for its role in aggression, impulsivity, suicide and mood liability, monoamine oxidase A can be considered a functional candidate in borderline personality disorder.These results show that the monoamine oxidase A gene may play an important role in the etiological development of the borderline personality disorder.Gene environment interactions with a novel variable Monoamine Oxidase A transcriptional enhancer are associated with antisocial personality disorder.However, clinical associations between this VNTR genotype and behavioral states have been inconsistent. Monoamine oxidase A gene promoter methylation and transcriptional downregulation in an offender population with antisocial personality disorder.hese results are consistent with prior literature suggesting MAOA and serotonergic dysregulation in antisocial populations. Our results offer the first evidence suggesting epigenetic mechanisms may contribute to MAOA dysregulation in antisocial offenders.For example, it has been shown that childhood maltreatment interacts with a monoamine oxidase A (MAOA) gene variant to predict antisocial behavior that is often associated with alcoholism, and an interaction between early life stress and a serotonin transporter promoter variant predicts alcohol abuse in nonhuman primates and depression in humans.Allelic variation of the monoamine oxidase A (MAOA) gene has been implicated in conduct disorder and antisocial, aggressive behavior in humans when associated with early adverse experiences.By conferring allele-specific transcriptional activity on the monoamine oxidase A (MAOA) gene in humans, length variation of a repetitive sequence [(variable number of tandem repeat (VNTR)] in the MAOA promoter influences a constellation of personality traits related to aggressive and antisocial behavior and increases the risk of neurodevelopmental and psychiatric disorders.Genotypes with three-repeat alleles were reported to be associated with antisocial as well as impulsive traits.Platelet MAO-B activity has been associated with psychopathy- and aggression-related personality traits, while variants of the MAOA and MAOB genes have been associated with diverse clinical phenotypes, including aggressiveness, antisocial problems and violent delinquency.Association studies suggest that the low activity variant of the monoamine oxidase A (MAOA)-uVNTR polymorphism confers risk for emotional disturbances associated with antisocial traits, particularly in males.CONCLUSION Taken together, these findings suggest that the MAOA-L genotype is to some extent associated with impulsive and antisocial personality traits in alcoholic men.Relationship of monoamine oxidase A binding to adaptive and maladaptive personality traits.Cluster B personality disorders are associated with allelic variation of monoamine oxidase A activity.\"}, {\"question\": \"What are 7 symptoms of yellow fever?\", \"ideal_answer\": [\"Yellow fever is considered to be a hemorrhagic fever and illness is characterized by fever, headache, myalgia, gastrointestinal symptoms, hepatic and renal dysfunction, multi-organ failure,  shock and coagulopathy.\"], \"exact_answer\": [[\"fever\"], [\"multi organ failure\"], [\"hemorrhage\"], [\"headache\"], [\"myalgia\"], [\"hepatic dysfunction\"], [\"renal dysfunction\"]], \"context\": \"Infection with yellow fever virus (YFV), an explosively replicating flavivirus, results in viral hemorrhagic disease characterized by cardiovascular shock and multi-organ failure. YF usually causes fever, muscle pain with prominent backache, headache, shivers, loss of appetite, and nausea or vomiting.The illness was characterized by fever, headache, myalgia, gastrointestinal symptoms, hepatic and renal dysfunction, and (in the fatal case), shock and coagulopathy,\"}, {\"question\": \"What in vivo tau tracers are being used?\", \"ideal_answer\": [\"in-vivo tau PET imaging ligands include [(18)F]THK523, [(18)F]THK5117, [(18)F]THK5105 and [(18)F]THK5351, [(18)F]AV1451(T807) [(11)C]PBB3, (18)F-THK5117, [(18)F]T808, 18F-RO6958948.\"], \"exact_answer\": [[\"[(11)C]PBB3\"], [\"[(18)F]AV1451\", \"T807\"], [\"[(18)F]T808\"], [\"[(18)F]THK523\"], [\"[(18)F]THK5105\"], [\"[(18)F]THK5117\"], [\"[(18)F]THK5351\"], [\"18F-RO6958948\"]], \"context\": \"The recent development of selective in-vivo tau PET imaging ligands including [(18)F]THK523, [(18)F]THK5117, [(18)F]THK5105 and [(18)F]THK5351, [(18)F]AV1451(T807) and [(11)C]PBB3 has provided information about the role of tau in the early phases of neurodegenerative diseases, and provided support for diagnosis, prognosis, and imaging biomarkers to track disease progression.Tau positron emission tomography radiolabelled probes such as T807, THK-5117, and PBB3 can image the pathology of the disease in vivo. The 18F-labeled tau imaging agents 18F-THK-5351, 18F-T807 (18F-AV-1451), and 18F-RO6958948 are presently under evaluation in clinical studies and clinical trials worldwide. \"}, {\"question\": \"List diseases caused by protein glutamine expanded repeats\", \"ideal_answer\": [\"Huntington's Disease,\\ndentatorubral-pallidoluysian atrophy\"], \"exact_answer\": [[\"Huntington's Disease\"], [\"Dentatorubral-pallidoluysian atrophy\"]], \"context\": \"Huntington's Disease (HD) is a fatally inherited neurodegenerative disorder caused by an expanded glutamine repeat in the N-terminal region of the huntingtin (HTT) protein. An increasing number of neurodegenerative disorders have been found to be caused by expanding CAG triplet repeats that code for polyglutamine. Huntington's disease (HD) is the most common of these disorders and dentatorubral-pallidoluysian atrophy (DRPLA) is very similar to HD, but is caused by mutation in a different gene, making them good models to study.Dentatorubral-pallidoluysian atrophy (DRPLA) is caused by expansion of a glutamine repeat in DRPLA protein.The genetic defect in dentatorubral-pallidoluysian atrophy (DRPLA) is expansion of the CAG repeat. The mutant gene is translated into the protein which carries the expanded glutamine repeat.\"}, {\"question\": \"List proteins with HEAT repeats\", \"ideal_answer\": [\"mTOR,\\nTOG5, \\nDNA-PKcs,\\nHEATR1,\\nRif1,\\nB56\\u03b3,\\nPR65/A,\\nSF3b155,\\nPds5B\"], \"exact_answer\": [[\"mTOR\"], [\"TOG5\"], [\"DNA-PKcs\"], [\"HEATR1\"], [\"Rif1\"], [\"B56\\u03b3\"], [\"PR65/A\"], [\"SF3b155\"], [\"Pds5B\"]], \"context\": \"mTOR N-terminal heat repeat domainTOG5-specific N-terminal HEAT repeat DNA-dependent protein kinase catalytic subunit (DNA-PKcs) is central to the regulation of the DNA damage response and repair through nonhomologous end joining. The structure has proved challenging due to its large size and multiple HEAT repeats.HEAT repeat-containing protein 1 (HEATR1) The first 1,300 residues of Tor form a HEAT repeat-containing \\u03b1-solenoid with four distinct segments: a highly curved 800-residue N-terminal 'spiral', followed by a 400-residue low-curvature 'bridge' and an extended 'railing' running along the bridge leading to the 'cap' that links to FAT region.Rif1 proteins are large and characterized by N-terminal HEAT repeats, predicted to form an elongated alpha-helical structureB56\\u03b3 contains 18 \\u03b1-helices that are organized into eight HEAT (Huntington-elongation-A subunit-TOR) repeat motifs. Here, we reveal a remarkable complexity in the unfolding of giant HEAT-repeat protein PR65/A, A HEAT-repeat superhelix within SF3b155 plays a key role in branch site recognition.The HEAT-repeat protein Pds5B\"}, {\"question\": \"What are the 3 antidotes for anticoagulant (anti-blood clotting) drugs, including factor Xa inhibitors, as of November 2017.\", \"ideal_answer\": [\" two potential reversal agents for oral direct factor Xa inhibitors (andexanet alfa and ciraparantag) are at various stages of clinical development. Idarucizumab, a reversal agent for the oral direct thrombin inhibitor dabigatran, was approved by FDA in October 2015.\", \" two potential reversal agents for oral direct factor Xa inhibitors (andexanet alfa and ciraparantag) are at various stages of clinical development. Idarucizumab, a reversal agent for the oral direct thrombin inhibitor dabigatran, was approved by FDA in October 2015. Idarucizumab and andexanet alfa have been reported to produce anticoagulation reversal effects within minutes of administration. \", \"Two potential reversal agents for oral direct factor Xa inhibitors (andexanet alfa and ciraparantag) are at various stages of clinical development. Idarucizumab, a reversal agent for the oral direct thrombin inhibitor dabigatran. Protamine sulfate reverses the effect of unfractionated heparin completely and of low-molecular-weight heparin (LMWH) partially.  Aripazine has shown promising results to reverse the effects of LMWH, fondaparinux, and direct oral anticoagulants but is still in the developmental phase.\"], \"exact_answer\": [[\"Idarucizumab\"], [\"andexanet alfa\"], [\"ciraparantag\"], [\"Protamine sulfate\"], [\"Aripazine\"]], \"context\": \" two potential reversal agents for oral direct factor Xa inhibitors (andexanet alfa and ciraparantag) are at various stages of clinical development. Idarucizumab, a reversal agent for the oral direct thrombin inhibitor dabigatran, was approved by FDA in October 2015.Idarucizumab and andexanet alfa have been reported to produce anticoagulation reversal effects within minutes of administration. Idarucizumab has recently been approved in United States for dabigatran reversal, whereas andexanet alfa is expected to get approved in the near future for reversal of oral factor Xa inhibitors.. Aripazine has shown promising results to reverse the effects of LMWH, fondaparinux, and direct oral anticoagulants but is still in the developmental phase.Protamine sulfate reverses the effect of unfractionated heparin completely and of low-molecular-weight heparin (LMWH) partially. working out pharmaceutical agents (andexanet alpha, idarucizumab, aripazine) being antidotes to direct thrombin inhibitor and the factor Xa inhibitors.\"}, {\"question\": \"Which genes are associated with Epidermolysis Bullosa Simplex?\", \"ideal_answer\": [\"In one family studied, inheritance of EBS is linked to the gene encoding keratin 14\", \"Epidermolysis bullosa simplex (EBS) is mainly caused by mutations in the KRT5 and KRT14 genes.  A compound heterozygous one amino-acid insertion/nonsense mutation in the plectin gene causes epidermolysis bullosa simplex with plectin deficiency.\", \"In one family studied, inheritance of EBS is linked to the gene encoding keratin 14 Homozygous deletion mutations in the plectin gene (PLEC1) in patients with epidermolysis bullosa simplex associated with late-onset muscular dystrophy. Epidermolysis bullosa simplex (EBS) is a rare genodermatosis resulting from multiple gene mutations, including KRT5 and KRT14.\", \"Epidermolysis bullosa simplex (EBS) is a rare genodermatosis resulting from multiple gene mutations, including KRT5 and KRT14.\", \"Keratin 5 and keratin 14 are known to be essential for the basal keratinocyte cytoskeleton and are defective in several forms of epidermolysis bullosa simplex. Homozygous deletion mutations in the plectin gene in patients with epidermolysis bullosa simplex associated with late-onset muscular dystrophy. The KRT5 and KRT14 genes encode the proteins keratin 5 and 14, respectively, which are the primary structural components of the 10-nm intermediate filaments of the mitotic epidermal basal cells. Epidermolysis bullosa simplex is mainly caused by mutations in the KRT5 and KRT14 genes. Verrucous carcinoma in epidermolysis bullosa simplex is possibly associated with a novel mutation in the keratin 5 gene. This is the first report of EBS-generalized intermediate in a newborn with de novo KRT5 gene mutation and KRT14 gene polymorphism, and no familial history of epidermolysis. Study of a family with epidermolysis bullosa simplex resulting from a novel mutation of KRT14 gene. We have systematically scanned genomic sequences of one of these keratins, keratin 14, for mutations in patients from 49 apparently independent kindreds using single-strand conformation polymorphism analysis. \", \"Homozygous deletion mutations in the plectin gene (PLEC1) in patients with epidermolysis bullosa simplex associated with late-onset muscular dystrophy. Epidermolysis bullosa simplex (EBS) is mainly caused by mutations in the KRT5 and KRT14 genes. Verrucous carcinoma in epidermolysis bullosa simplex is possibly associated with a novel mutation in the keratin 5 gene.\", \"Recent advances in molecular biology have enabled the association of epidermolysis bullosa simplex (EBS) with point mutations of keratin 14 and/or keratin 5 genes to be established. \", \"Keratin 14 gene mutations in patients with epidermolysis bullosa simplex. Mutant keratins 5 or 14 are implicated in the etiology of epidermolysis bullosa simplex (EBS).  Keratin 14 gene point mutation in the K\\u00f6bner and Dowling-Meara types of epidermolysis bullosa simplex as detected by the PASA method.\"], \"exact_answer\": [[\"Keratin 14\", \"KRT14\"], [\"Keratin 5\", \"KRT5\"], [\"Plectin\", \"PLEC1\"]], \"context\": \"In one family studied, inheritance of EBS is linked to the gene encoding keratin 14In the first family, linkage was found to chromosome 17 markers flanking the keratin 14 gene (D17S74: Zmax = +2.45, theta = 0.10; COL1A1: Zmax = +0.97, theta = 0.00) and markers near the keratin 5 gene on chromosome 12 were excluded (D12S17: Z less than -2.0, theta = 0.08; COL2A1: Z less than -2.0, theta = 0.13). In contrast, the second family showed linkage to the region containing the keratin 5 gene (D12S17: Zmax = +1.37, theta = 0.08; COL2A1: Zmax = +0.33, theta = 0.15) and was not linked to the keratin 14 gene (D17S74: Z less than -2.0, theta = 0.14). Keratin 14 gene mutations in patients with epidermolysis bullosa simplex.We have systematically scanned genomic sequences of one of these keratins, keratin 14, for mutations in patients from 49 apparently independent kindreds using single-strand conformation polymorphism analysis. Keratin 14 gene point mutation in the K\\u00f6bner and Dowling-Meara types of epidermolysis bullosa simplex as detected by the PASA method.Recent advances in molecular biology have enabled the association of epidermolysis bullosa simplex (EBS) with point mutations of keratin 14 and/or keratin 5 genes to be established. Homozygous deletion mutations in the plectin gene (PLEC1) in patients with epidermolysis bullosa simplex associated with late-onset muscular dystrophy. In this study, we report two patients with EB-MD, each with a homozygous deletion mutation in the plectin gene, PLEC1.A homozygous nonsense mutation in the PLEC1 gene in patients with epidermolysis bullosa simplex with muscular dystrophy.A compound heterozygous one amino-acid insertion/nonsense mutation in the plectin gene causes epidermolysis bullosa simplex with plectin deficiency.The KRT5 and KRT14 genes encode the proteins keratin 5 and 14, respectively, which are the primary structural components of the 10-nm intermediate filaments of the mitotic epidermal basal cells.Keratin 5 and keratin 14 are known to be essential for the basal keratinocyte cytoskeleton and are defective in several forms of epidermolysis bullosa simplex.Severe palmo-plantar hyperkeratosis in Dowling-Meara epidermolysis bullosa simplex caused by a mutation in the keratin 14 gene (KRT14).Mutant keratins 5 or 14 are implicated in the etiology of epidermolysis bullosa simplex (EBS). Epidermolysis bullosa simplex (EBS) is mainly caused by mutations in the KRT5 and KRT14 genes.Verrucous carcinoma in epidermolysis bullosa simplex is possibly associated with a novel mutation in the keratin 5 gene.Epidermolysis bullosa simplex (EBS) is a rare genodermatosis resulting from multiple gene mutations, including KRT5 and KRT14.Study of a family with epidermolysis bullosa simplex resulting from a novel mutation of KRT14 geneThis is the first report of EBS-generalized intermediate in a newborn with de novo KRT5 gene mutation and KRT14 gene polymorphism, and no familial history of epidermolysis. Epidermolysis bullosa simplex (EBS) is an inherited skin disorder caused by mutations in the keratin 5 (KRT5) and keratin 14 (KRT14) genes, with fragility of basal keratinocytes leading to epidermal cytolysis and blistering.In this study, we characterized mutations in KRT5 and KRT14 genes in patients with EBS and investigated their possible structure-function correlations.Mutations were characterized using polymerase chain reaction (PCR) and DNA sequencing.Epidermolysis bullosa simplex (EBS) is a mechanobullous genodermatosis that may be caused by mutations in the genes KRT5 and KRT14 encoding the basal epidermal keratins 5 (K5) and 14 (K14).Basal epidermolysis bullosa simplex (EBS) is a hereditary skin blistering disorder resulting in most cases from missense mutations in the keratin 5 (KRT5) or keratin 14 (KRT14) genes.Epidermolysis bullosa simplex is a hereditary skin blistering disorder caused by mutations in the KRT5 or KRT14 genes.BACKGROUND Basal epidermolysis bullosa simplex (EBS) is a hereditary skin blistering disorder resulting in most cases from missense mutations in the keratin 5 (KRT5) or keratin 14 (KRT14) genes.Epidermolysis bullosa simplex is a group of blistering skin disorders caused by defects in one of the keratin genes, KRT5 and KRT14.Epidermolysis bullosa simplex is an autosomal dominant inherited skin blistering disorder caused by mutations in the genes KRT5 or KRT14 coding for the basal epidermal keratins 5 and 14, respectively.\"}, {\"question\": \"What are the 3 types of ultraviolet (UV) solar radiation?\", \"ideal_answer\": [\"Solar ultraviolet (UV) radiation, an ubiquitous environmental carcinogen, is classified depending on the wavelength, into three regions; short-wave UVC (200-280 nm), mid-wave UVB (280-320 nm), and long-wave UVA (320- 400 nm)\", \"short-wave UVC (200-280 nm), medium-wave UVB (280-320 nm), and long-wave UVA (320-400 nm).\", \"short-wave UVC (200-280 nm), medium-wave UVB (280-320 nm), and long-wave UVA (320-400 nm). \"], \"exact_answer\": [[\"short wave UVC (200-280 nm)\"], [\"mid-wave UVB (280-320 nm)\"], [\"long-wave UVA (320- 400 nm)\"]], \"context\": \"short-wave UVC (200-280 nm), medium-wave UVB (280-320 nm), and long-wave UVA (320-400 nm). short-wave UVC (200-280 nm), mid-wave UVB (280-320 nm), and long-wave UVA (320- 400 nm)solar UV, largely confined to the UVB (290-320 nm) and short-wavelength UVA (320-340 nm) region, there is currently no agreed-upon method to measure broad-spectrum protection against long-wavelength UVA (340-400 nm).Ultraviolet radiation is made up of UV-C, UV-B, and UV-A.\"}, {\"question\": \"Please list 10 conditions which play a role in causing atrial fibrillation.\", \"ideal_answer\": [\"Atrial fibrillation (AF) is the most common sustained arrhythmia and is associated with significant morbidity and mortality. Multiple conditions like hypertension, heart failure, diabetes, sleep apnoea, hyperthyroidism and obesity play a role for the initiation and perpetuation of AF. Other possible causes are alcohol and drug use and atrial ischemia. Risk of AF increases with age.\"], \"exact_answer\": [[\"age\"], [\"heart disease\"], [\"Hypertension\"], [\"diabetes\"], [\"sleep apnea\"], [\"obesity\"], [\"GERD\"], [\"hyperthyroidism\"], [\"atrial ischemia\"], [\"drugs\"]], \"context\": \"Atrial fibrillation (AF) is the most common sustained arrhythmia and is associated with significant morbidity and mortality. Multiple conditions like hypertension, heart failure, diabetes, sleep apnoea, and obesity play a role for the initiation and perpetuation of AF.Atrial fibrillation is the most common arrhythmia worldwide with increasing frequency noted with age. Hyperthyroidism is a well-known cause of atrial fibrillation with a 16%-60% prevalence of atrial fibrillation in patients with known hyperthyroidism Atrial fibrillation continues to be a challenging arrhythmia. There are some conventional, time-tested explanations of atrial fibrillation genesis, however some uncertainty of its complete understanding still exists. We focused on atrial ischemia which, hypothetically, could be responsible for manifestation of the arrhythmia, irrespective of the underlying heart diseaseThe prevalence of atrial fibrillation (AF) increases with age. As the population ages, the burden of AF increases.t is estimated that this arrhythmia's suffers a significant increase throughout the entire life span, and that its highest raise occurs between the ages of 65 and 80. Drug-induced tachyarrhythmias (ventricular tachycardia or atrial tachyarrhythmias with rapid ventricular response) are life-threatening and often cause syncopeAtrial fibrillation is a growing health problem and the most common cardiac arrhythmia, affecting 5% of persons above the age of 65 years.It occurs very often in patients with congestive heart failure and the prevalence increases with the severity of the disease.The prevalence and incidence of atrial fibrillation increase with age.Cardiac arrhythmias are a major problem in elderly persons, because of the high prevalence of underlying heart disease and hypertension,Heavy alcohol use has been suspected to cause acute atrial fibrillation,Heavy alcohol use is an important potential etiology for acute atrial fibrillation; alcohol withdrawal may represent a particular risk for such alcohol-related atrial fibrillation.Atrial fibrillation associated with rheumatic mitral valve disease has the highest stroke risk (about 17 times greater than unaffected controls), but even with nonvalvular heart disease, the risk is increased fivefoldhronic atrial fibrillation is a very common arrhythmia affecting 2 to 4% of the population older than 60 years of ageIn western countries ischemic and hypertensive heart disease (including sick sinus syndrome) and alcohol (holiday heart syndrome) are numerically more important than the classical causes of atrial fibrillation--rheumatic heart disease and thyrotoxicosis-Risk factors for acute atrial fibrillation include increasing age, cardiovascular disease, alcohol, diabetes, and lung disease.  Risk factors for acute atrial fibrillation include increasing age, cardiovascular disease, alcohol, diabetes, and lung diseaseDiabetes mellitus is frequently accompanied by cardiac rhythm disorders. On the other hand, atrial fibrillation is the most frequent cardiac arrhythmia in adult population isk factors for acute atrial fibrillation include increasing age, cardiovascular disease, alcohol, diabetes, and lung disease.  Risk factors for atrial fibrillation include increasing age, male sex, co-existing cardiac and thyroid disease, pyrexial illness, electrolyte imbalance, cancer, and co-existing infection.Risk factors for atrial fibrillation include increasing age, male sex, co-existing cardiac and thyroid disease, pyrexial illness, electrolyte imbalance, cancer, and co-existing infectionRisk factors for atrial fibrillation include increasing age, coexisting cardiac and thyroid disease, pyrexial illness, electrolyte imbalance, cancer, and coexisting infection.Thus, pericarditis might play a role in the development of recent atrial fibrillation during the course of myocardial infarction. Recent atrial fibrillation may be a sign of pericardial effusion which may be otherwise silent.Some patients develop atrial fibrillation after any type of surgery, some patients develop atrial fibrillation only after cardiac surgery, and still other patients never develop postoperative atrial fibrillationHypertension is known to increase the risk of atrial fibrillationOf the long list of various cardiac conditions associated with an increased risk of cardioembolic strokes, non-valvular atrial fibrillation is the most common cause.\"}, {\"question\": \"Please list 7 classes of drugs that interact with Warfarin.\", \"ideal_answer\": [\"The number of drugs reported to interact with warfarin continues to expand. There are reports of interactions with azole antibiotics, macrolides, quinolones, nonsteroidal anti-inflammatory drugs, including selective cyclooxygenase-2 inhibitors, selective serotonin reuptake inhibitors, omeprazole, lipid-lowering agents, protease inhibitors, amiodarone, fluorouracil, psychotropics, and oral corticosteroids.\", \"Seven drugs/drug classes (ibuprofen, morphine, warfarin, bupropion, paroxetine, rosiglitazone, ACE inhibitors) with known ADEs were selected to evaluate the system. This study was aimed to determine the prevalence of drug-related problems (DRPs), identify the most common drugs, and drug classes involved in DRPs as well as associated factors with the occurrence of DRPs.A prospective cross-sectional study was conducted on 225 patients admitted to medical wards of Tikur Anbessa Specialized Hospital, Addis Ababa from March to June 2014. Analysis of the data showed that albumin possesses a single strong binding site for warfarin with an association constant of 154,000 at 3 degrees C and secondary classes of several sites with a much lower affinity. Four medications or medication classes were implicated alone or in combination in 67.0% (95% CI, 60.0 to 74.1) of hospitalizations: warfarin (33.3%), insulins (13.9%), oral antiplatelet agents (13.3%), and oral hypoglycemic agents (10.7%). Drugs such as gentamycin, warfarin, nifedipine, and cimetidine have the highest probability of causing DRP. Drugs with the highest drug risk ratio were gentamycin, warfarin, nifedipine, and cimetidine. The drugs most frequently causing a DRP were angiotensin-converting enzyme inhibitors/angiotensin II receptor blockers, diuretics, warfarin, spironolactone, and \\u03b2-blockers.\"], \"exact_answer\": [[\"SSRI\"], [\"antibiotics\"], [\"NSAIDS\"], [\"antimetabolites\"], [\"oral corticosteroids\"], [\"protease inhibitors\"], [\"psychotropics\"], [\"statins/lipid lowering drugs\"], [\"omeprazole\"], [\"amiodarone\"], [\"carbamazapine\"], [\"narcotics/opiods\"], [\"cardiovascular drugs\"], [\"trazodone\"], [\"antifungals\"], [\"antivirals\"]], \"context\": \"The combined use of warfarin and NSAIDs is generally discouraged because of the increased risk of bleeding in these patients. In patients receiving warfarin who also require NSAIDsThe number of drugs reported to interact with warfarin continues to expand. While most reports are of poor quality and present potentially misleading conclusions, the consistency of reports of interactions with azole antibiotics, macrolides, quinolones, nonsteroidal anti-inflammatory drugs, including selective cyclooxygenase-2 inhibitors, selective serotonin reuptake inhibitors, omeprazole, lipid-lowering agents, amiodarone, and fluorouracil,  potential drug interaction exists between oral corticosteroids and warfarin, but there is limited documentation.Use of oral corticosteroids in patients on long-term warfarin therapy may result in a clinically significant interaction, which requires close INR monitoring and possible warfarin dose reduction.We review the possible mechanisms of this interaction and the reported interactions between warfarin and other protease inhibitors.It is necessary to intensify warfarin monitoring upon initiation or alteration of hormonal contraceptives.To report a single patient case that presented with a probable drug interaction between warfarin and 3 methods of hormonal contraceptives, as assessed by the Horn Interaction Probability Scale.Warfarin-antibiotic interactions in older adults of an outpatient anticoagulation clinicSeveral classes of drugs, such as antibiotics, may interact with warfarin to cause an increase in warfarins anticoagulant activity and the clinical relevance of warfarin-antibiotic interactions in older adults is not clear.As the number of psychotropics on the market expands, the likelihood increases that a patient requiring anticoagulation with warfarin will receive concurrent treatment with a psychotropic drug.The most commonly identified potentially interacting medication pairs were warfarin and non-steroidal anti-inflammatory drugs (6824 cases), theophylline/aminophylline and ciprofloxacin/fluvoxamine (930), and warfarin and barbiturates (567).Among 36 drugs that may interact with warfarin, fluconazole, amiodarone, and omeprazole were associated with the requirement for 45.8, 16.7, and 16.7% lower median warfarin dose (all P<0.05 with a false discovery rate <0.05).The results demonstrate a high prevalence of concomitant drug prescriptions with the potential for clinically relevant DDIs with warfarin, the most frequent being acetylsalicylic acid and amiodaroneWe provide new support for the previous scarce evidence of interactions between warfarin and carbamazepine, bezafibrate, and lactulose.  Noteworthy are the interactions with cardiovascular or antilipidaemic drugs which are often coadministered with coumarins: amiodarone, propafenone and fibrates. Cardiovascular drugs which are obviously devoid or proven to be devoid of an interaction are angiotensin converting enzyme (ACE) inhibitors, calcium antagonists, beta-blockers and cardiac glycosides.Of the antidepressants analyzed, those that showed relevant effects on the interaction with warfarin were, in decreasing order: paroxetine, venlafaxine, fluoxetine, and duloxetine.Isomeric differences in metabolism form an important basis for stereoselective metabolic interactions, especially inhibition; this has been demonstrated with phenylbutazone, metronidazole and co-trimoxazoleMiconazole and nystatin used as topical antifungal drugs interact equally strongly with warfarin.A retrospective case series published in 2012 concluded that miconazole and nystatin used as topical antifungal drugs appear to interact equally strongly with warfarin. trazodone (2.2%) and carbamazepine (1.1%). The most commonly prescribed agents independently associated with increased bleeding risk were cyclooxygenase-2 inhibitors.Metabolic interaction between warfarin and antiretrovirals is likely, particularly if NNRTIs or PIs are included in the antiretroviral regimen.\"}, {\"question\": \"In what states are GDF15 expression increased?\", \"ideal_answer\": [\"Growth differentiation factor 15 (GDF-15) is expressed and secreted in response to inflammation, oxidative stress, hypoxia, telomere erosion, and oncogene activation.\"], \"exact_answer\": [[\"inflammation\"], [\"oxidative stress\"], [\"hypoxia\"], [\"telomere erosion\"], [\"oncogene activation\"], [\"tissue injury\"]], \"context\": \"Macrophage inhibitory cytokine-1 (MIC-1), also known as growth differentiation factor 15 (GDF15), is a stress response cytokine.Growth differentiation factor 15 (GDF-15) is expressed and secreted in response to inflammation, oxidative stress, hypoxia, telomere erosion, and oncogene activation.the tumour-derived macrophage inhibitory cytokine-1 (MIC-1) emerged as a possible mediator of cancer anorexia because lesions of these brainstem areas attenuated the anorectic effect of exogenous MIC-1 in mice.Growth differentiation factor-15, also known as macrophage inhibitory cytokine-1 (MIC-1/GDF15) is a member of the transforming growth factor- \\u03b2 super family and increases during inflammatory states.Growth differentiation factor-15 (GDF-15) is a member of the TGF-\\u03b2cytokine superfamily that is widely expressed and may be induced in response to tissue injury. \"}, {\"question\": \"List four features of the WHIM syndrome.\", \"ideal_answer\": [\"The Warts, Hypogammaglobulinemia, Infections, Myelokathexis (WHIM) syndrome is an immunodeficiency caused by mutations in chemokine receptor CXCR4.\"], \"exact_answer\": [[\"Warts\"], [\"Hypogammaglobulinemia\"], [\"Infections\"], [\"Myelokathexis\"]], \"context\": \"To test whether there may be an advantage of one approach over the other for WHIM (warts, hypogammaglobulinemia, infections, and myelokathexis) syndrome - a primary immunodeficiency disorder caused by gain-of-function autosomal dominant mutations in chemokine receptor CXCR4 - we performed competitive transplantation experiments using both lethally irradiated WT (Cxcr4+/+) and unconditioned WHIM (Cxcr4+/w) recipient mice. WHIM syndrome is a primary autosomal dominant immuno deficiency due to CXCR4 mutations characterized by mucocutaneous warts, hypogammaglobulinemia, recurrent bacterial infections and myelokathesis.WHIM syndrome is a rare congenital immunodeficiency disease, named after its main clinical manifestations: warts, hypogammaglobulinemia, infections, and myelokathexis, which refers to abnormal accumulation of mature neutrophils in the bone marrow.The Warts, Hypogammaglobulinemia, Infections, Myelokathexis (WHIM) syndrome is an immunodeficiency caused by mutations in chemokine receptor CXCR4.WHIM syndrome is a condition in which affected persons have chronic peripheral neutropenia, lymphopenia, abnormal susceptibility to human papilloma virus infection, and myelokathexis.Warts, hypogammaglobulinemia, infections, and myelokathexis (WHIM) syndrome is a genetic disease characterized by neutropenia, lymphopenia, susceptibility to infections, and myelokathexis, which describes degenerative changes of mature neutrophils and hyperplasia of bone marrow myeloid cells. The WHIM syndrome features susceptibility to human Papillomavirus infection-induced warts and carcinomas, hypogammaglobulinemia, recurrent bacterial infections, B and T-cell lymphopenia, and neutropenia associated with retention of senescent neutrophils in the bone marrow (i.e.<b>PURPOSE OF REVIEW</b>: WHIM syndrome (the association of warts, hypogammaglobulinemia, recurrent bacterial infections, and 'myelokathexis') is a rare congenital form of neutropenia associated with an unusual immune disorder involving hypogammaglonulinemia and abnormal susceptibility to warts.\"}, {\"question\": \"Which bacteria are enriched in the gut microbiome of infants following exposure to fury pets?\", \"ideal_answer\": [\"Pre- and postnatal pet exposure enriched the abundance of Oscillospira and/or Ruminococcus in infants.\"], \"exact_answer\": [[\"Oscillospira\"], [\"Ruminococcus\"]], \"context\": \"As a common effect in all birth scenarios, pre- and postnatal pet exposure enriched the abundance of Oscillospira and/or Ruminococcus (P\\u2009<\\u20090.05) with more than a twofold greater likelihood of high abundance.The impact of pet ownership varies under different birth scenarios; however, in common, exposure to pets increased the abundance of two bacteria, Ruminococcus and Oscillospira, which have been negatively associated with childhood atopy and obesity.\"}, {\"question\": \"List the main PPI databases.\", \"ideal_answer\": [\"PPIM,\\nHPRD,\\nSTRING,\\nDAPID,\\nMIPS,\\nINTERACT,\\nBioGRID\"], \"exact_answer\": [[\"PPIM\"], [\"HPRD\"], [\"STRING\"], [\"DAPID\"], [\"MIPS\"], [\"INTERACT\"], [\"BioGRID\"]], \"context\": \"PPIM: A Protein-Protein Interaction Database for Maize. experimentally validated human protein-protein interaction database, HPRD (interactome) STRING Protein-Protein interaction databasehuman protein-protein interaction database HPRD DAPID: a 3D-domain annotated protein-protein interaction database.The MIPS mammalian protein-protein interaction database (MPPI) is a new resource of high-quality experimental protein interaction data in mammals. INTERACT: an object oriented protein-protein interaction database.The BioGRID interaction database\"}, {\"question\": \"List intramembrane rhomboid peptidases\", \"ideal_answer\": [\"PARL\\nPcp1\\nhiGlpG\\necGlpG\\nYqgP\"], \"exact_answer\": [[\"PARL\"], [\"Pcp1\"], [\"hiGlpG\"], [\"ecGlpG\"], [\"YqgP\"]], \"context\": \"rhomboid protease PARL The newly identified Pcp1 protein belongs to the rhomboid-GlpG superfamily of putative intramembrane peptidases.We subjected Haemophilus influenzae, (hiGlpG), Escherichia coli GlpG (ecGlpG) and Bacillus subtilis (YqgP)Crystal structures of H. influenzae (hiGlpG) and E. coli GlpG (ecGlpG) rhomboids have revealed a structure with six transmembrane helices and a Ser-His catalytic dyad buried within the membrane. \"}, {\"question\": \"Which organs are mostly affected in Systemic Lupus Erythematosus (SLE)?\", \"ideal_answer\": [\"In systemic lupus erythematosus (SLE), brain and kidney are the most frequently affected organs. The heart is one of the most frequently affected organs in SLE. Skin is one of the most commonly affected organs in SLE. Other affected organs in SLE-AAC included hematologic system (11, 84.6%), followed by mucocutaneous (seven, 53.8%), musculoskeletal (seven, 53.8%) and neuropsychiatric (two, 15.4%) systems.\", \"In systemic lupus erythematosus (SLE), brain and kidney are the most frequently affected organs. The heart is one of the most frequently affected organs in SLE. Skin is one of the most commonly affected organs in SLE\", \"In systemic lupus erythematosus (SLE), brain and kidney are the most frequently affected organs. The heart is one of the most frequently affected organs in SLE. Any part of the heart can be affected, including the pericardium, myocardium, coronary arteries, valves, and the conduction system\", \"In systemic lupus erythematosus (SLE), brain and kidney are the most frequently affected organs.\"], \"exact_answer\": [[\"kidney\"], [\"brain\"], [\"heart\"], [\"skin\"]], \"context\": \"In systemic lupus erythematosus (SLE), brain and kidney are the most frequently affected organs. The heart is one of the most frequently affected organs in SLE. Any part of the heart can be affected, including the pericardium, myocardium, coronary arteries, valves, and the conduction systemSkin is one of the most commonly affected organs in SLEOther affected organs in SLE-AAC included kidney (11, 84.6%) and hematologic system (11, 84.6%), followed by mucocutaneous (seven, 53.8%), musculoskeletal (seven, 53.8%) and neuropsychiatric (two, 15.4%) systems.Systemic lupus erythematosus (SLE) is a systemic autoimmune disease which affects multiple organs.Cutaneous lupus erythematosus (CLE) includes skin symptoms seen in SLE and cutaneous-limited LE.\"}, {\"question\": \"List 3 diseases for which saRNAs have been evaluated as a potential treatment.\", \"ideal_answer\": [\"saRNAs have been tested for the treatment of breast, bladder, liver cancer and more.\"], \"exact_answer\": [[\"Breast cancer\"], [\"Bladder cancer\"], [\"Liver cancer\"]], \"context\": \"Small activating RNA restores the activity of the tumor suppressor HIC-1 on breast cancer.Upon saRNA treatment, genes upregulated included those involved in immune activation, cell cycle interference, the induction of apoptosis, anti-metastasis, and cell differentiation. Downregulated genes included oncogenes and those that play roles in cell invasion, cell growth, and cell division.We present a novel method for treating bladder cancer with intravesically delivered small activating RNA (saRNA) in an orthotopic xenograft mouse bladder tumor model. Treatment of Liver Cancer by C/EBPA saRNA.In preclinical models of liver disease, treatment with C/EBP\\u03b1 saRNA has shown reduction in tumor volume and improvement in serum markers of essential liver function such as albumin, bilirubin, aspartate aminotransferase (AST), and alanine transaminase (ALT). This saRNA that activates C/EBP\\u03b1 for advanced HCC is the first saRNA therapy to have entered a human clinical trial.\"}, {\"question\": \"Which metabolic pathways have been associated with Systemic Lupus Erythematosus?\", \"ideal_answer\": [\"Genetic polymorphisms of the xenobiotic metabolic pathway involved in estrogen metabolism might contribute towards pathophysiology of systemic lupus erythematosus (SLE). According to the analysis on metabolic pathway, SLE could cause significant changes in unsaturated fatty acid and amino acid metabolism pathway. Results suggest reducing FLI1 in lupus decreases the pathogenicity of T cells by decreasing TCR-specific activation and IL-4 production in part through the modulation of glycosphingolipid metabolism\"], \"exact_answer\": [[\"Estrogen metabolism\"], [\"Unsaturated fatty acid metabolism\"], [\"Amino acid metabolism\"], [\"Glycosphingolipid metabolism\"], [\"One-carbon metabolic pathway\"]], \"context\": \"Measurement of urine neopterin, product of a metabolic pathway controlled by interferon-gamma, has been found useful in many clinical conditions. The objective of this study was to investigate the association between genetic variations in estrogen metabolic pathway genes, including estrogen receptor alpha (ESR1), estrogen receptor beta (ESR2), and aromatase (CYP19A1), and risk of SLE.According to the analysis on metabolic pathway, SLE could cause significant changes in unsaturated fatty acid and amino acid metabolism pathwayTogether, our results suggest reducing FLI1 in lupus decreases the pathogenicity of T cells by decreasing TCR-specific activation and IL-4 production in part through the modulation of glycosphingolipid metabolismgenetic polymorphisms of the xenobiotic metabolic pathway involved in estrogen metabolism might contribute towards pathophysiology of systemic lupus erythematosus (SLE)The current study was conducted to elucidate the effect of genetic variations in one-carbon metabolism on the epigenetic regulation of major histocompatibility complex II transactivator (MHC2TA), reduced folate carrier 1 (RFC1/SLC19A1) and human leukocyte antigen (HLA)-DR in systemic lupus erythematosus (SLE)Putatively functional polymorphisms of one-carbon and xenobiotic metabolic pathways influence susceptibility for wide spectrum of diseases. The current study was aimed to explore gene-gene interactions among these two metabolic pathways in four diseases i.e. breast cancer, systemic lupus erythematosus (SLE), coronary artery disease (CAD) and Parkinson's disease (PD). In this regard, inflammation is not only associated with systemic lupus patients but is also present in patients with metabolic syndrome and insulin resistance.\"}, {\"question\": \"What are the 3 main bacteria found in human milk?\", \"ideal_answer\": [\"Human milk is rich in diverse bacteria, particularly Staphylococcus, Streptococcus and Pseudomonas genera.\"], \"exact_answer\": [[\"Staphylococcus\"], [\"Streptococcus\"], [\"Pseudomonas\"]], \"context\": \"Instead, it provides infants a rich source of diverse bacteria, particularly microbes belonging to the Staphylococcus, Streptococcus, and Pseudomonas genera.\"}, {\"question\": \"In which tissues is the lincRNA Xist expressed?\", \"ideal_answer\": [\"Relative expression of X-linked genes was examined in liver, kidney and brain tissue by real-time PCR in adult XX(Y)* and XY* males and XX females. X-chromosome inactive specific transcript (Xist) was expressed only in female adipose tissue. X-chromosome inactive specific transcript (Xist) was expressed only in female adipose tissue. It was also expressed in duodenum and testis.\"], \"exact_answer\": [[\"duodenum\"], [\"kidney\"], [\"brain\"], [\"liver\"], [\"adipose tissue\"], [\"testis\"]], \"context\": \"Xist (female-only) and Eif2s3y (male-only). The frequency of SDE probesets varied widely between tissues, and was highest in the duodenum (6.2%), whilst less than 0.05% in over half of the surveyed tissues.Embryos positive for Xist expression were classified as female, and Xist negative and equine sex determining region of the Y chromosome positive embryos were classified as male. From 28 embryos tested, 15 (54%) showed positive Xist expression and were thus classified as female.Preattachment embryos at different times after ovulation (Day 8: n = 9; Day 10: n = 12; Day 12: n = 15) were analyzed for Xist RNA expression using quantitative and qualitative reverse transcription-polymerase chain reaction (RT-PCR).Relative expression of X-linked genes was examined in liver, kidney and brain tissue by real-time PCR in adult XX(Y)* and XY* males and XX femalesX-chromosome inactive specific transcript (Xist) was expressed only in female adipose tissue.Reverse transcription-polymerase chain reaction (RT-PCR) revealed the presence of Xist transcripts in all adult female somatic tissues evaluated. In contrast, among the male tissues examined, Xist expression was detected only in testis. We now report the presence of Xist/XIST transcripts in newborn and adult mouse testes, and in human testicular tissue with normal spermatogenesis, but not in the testes of patients who lack germ cells.\"}, {\"question\": \"What forms part of the senescence associated secretory phenotype, or SASP?\", \"ideal_answer\": [\"The diverse arrays of proteins secreted by senescent cells have been described to influence aging and to have both pro-tumorigenic and anti-tumorigenic influences on the surrounding microenvironment. Further characterization of these proteins, known as the senescence-associated secretory phenotype (SASP), and their regulators is required to understand and further manipulate such activities. The senescence-associated secretory phenotype (SASP) is characterized by IL1B, CXCL8, CCL2, TNF, CCL27 and other pro-inflammatory factors including a novel SASP component CLEC11A.\", \"Numerous activities of senescent cells depend on the aptitude of these cells to secrete myriads of bioactive molecules, a behavior termed the senescence-associated secretory phenotype (SASP) The SASP supports cell-autonomous functions like the senescence-associated growth arrest, and mediates paracrine interactions between senescent cells and their surrounding microenvironment. Cultures enriched with not-diploid cells acquired a senescence-associated secretory phenotype (SASP) characterized by IL1B, CXCL8, CCL2, TNF, CCL27 and other pro-inflammatory factors including a novel SASP component CLEC11A.\"], \"exact_answer\": [[\"IL1B\"], [\"CXCL8\"], [\"CCL2\"], [\"TNF\"], [\"CCL27\"]], \"context\": \"SASP mediates chemoresistance and tumor-initiating-activity of mesothelioma cellsNumerous activities of senescent cells depend on the aptitude of these cells to secrete myriads of bioactive molecules, a behavior termed the senescence-associated secretory phenotype (SASP)The SASP supports cell-autonomous functions like the senescence-associated growth arrest, and mediates paracrine interactions between senescent cells and their surrounding microenvironment.Cultures enriched with not-diploid cells acquired a senescence-associated secretory phenotype (SASP) characterized by IL1B, CXCL8, CCL2, TNF, CCL27 and other pro-inflammatory factors including a novel SASP component CLEC11A.Senescent cells secrete cytokines and other factors of the senescence-associated secretory phenotype (SASP) that contribute to tumor suppression by enforcing arrest and recruiting immune cells that remove these damaged or oncogene-expressing cells from organisms.Aim of this study was to evaluate whether and how adalimumab, a monoclonal antibody directed against tumor necrosis factor-\\u03b1 (TNF-\\u03b1), a major SASP component, can prevent the SASPEndothelial cell senescence is characterized by acquisition of senescence-associated secretory phenotype (SASP), able to promote inflammaging and cancer progression.We have shown that the mitochondria are essential for development of senescence and many of the associated phenotypes, including the often detrimental senescence-associated secretory phenotype (SASP)The diverse arrays of proteins secreted by senescent cells have been described to influence aging and to have both pro-tumorigenic and anti-tumorigenic influences on the surrounding microenvironment. Further characterization of these proteins, known as the senescence-associated secretory phenotype (SASP), and their regulators is required to understand and further manipulate such activities. \"}, {\"question\": \"List two medication included in the Juluca pill.\", \"ideal_answer\": [\"Juluca\\u00ae pill includes dolutegravir and rilpivirine. It is the first two-drug single-tablet regimen (STR) to be approved for the treatment of HIV-1 infection in adults.\"], \"exact_answer\": [[\"dolutegravir\"], [\"rilpivirine\"]], \"context\": \"Dolutegravir/rilpivirine (Juluca\\u00ae) is the first two-drug single-tablet regimen (STR) to be approved for the treatment of HIV-1 infection in adults.\"}, {\"question\": \"Which de novo truncating mutations in WASF1 cause intellectual disability?\", \"ideal_answer\": [\"De novo truncating mutations in WAS protein family member 1 (WASF1) were identified in five unrelated individuals with moderate to profound intellectual disability with autistic features and seizures. WASF1, also known as WAVE1, is part of the WAVE complex and acts as a mediator between Rac-GTPase and actin to induce actin polymerization. The three mutations connected by Matchmaker Exchange were c.1516C>T (p.Arg506Ter), which occurs in three unrelated individuals, c.1558C>T (p.Gln520Ter), and c.1482delinsGCCAGG (p.Ile494MetfsTer23). All three variants are predicted to partially or fully disrupt the C-terminal actin-binding WCA domain. Functional studies using fibroblast cells from two affected individuals with the c.1516C>T mutation showed a truncated WASF1 and a defect in actin remodeling.\"], \"exact_answer\": [[\"c.1516C>T\", \"p.Arg506Ter\"], [\"c.1558C>T\", \"p.Gln520Ter\"], [\"c.1482delinsGCCAGG\", \"p.Ile494MetfsTer23\"]], \"context\": \"Here, using exome sequencing and whole-genome sequencing, we identified three de novo truncating mutations in WAS protein family member 1 (WASF1) in five unrelated individuals with moderate to profound intellectual disability with autistic features and seizures. WASF1, also known as WAVE1, is part of the WAVE complex and acts as a mediator between Rac-GTPase and actin to induce actin polymerization. The three mutations connected by Matchmaker Exchange were c.1516C>T (p.Arg506Ter), which occurs in three unrelated individuals, c.1558C>T (p.Gln520Ter), and c.1482delinsGCCAGG (p.Ile494MetfsTer23). All three variants are predicted to partially or fully disrupt the C-terminal actin-binding WCA domain. Functional studies using fibroblast cells from two affected individuals with the c.1516C>T mutation showed a truncated WASF1 and a defect in actin remodeling. This study provides evidence that de\\u00a0novo heterozygous mutations in WASF1 cause a rare form of intellectual disability.\"}, {\"question\": \"Which proteins form the nuclear pore basket in human cells?\", \"ideal_answer\": [\"Nup133 Is Required for Proper Nuclear Pore Basket Assembly and Dynamics in Embryonic Stem Cells. Moreover, the association of TREX-2 with the NPC requires the basket nucleoporins NUP153 and TPR, but is independent of transcription. The nucleoporin Nup153 is required for nuclear pore basket formation, nuclear pore complex anchoring and import of a subset of nuclear proteins\", \"Nup133 Is Required for Proper Nuclear Pore Basket Assembly and Dynamics in Embryonic Stem Cells. Here we show that the NPC basket proteins Nup1 and Nup60 directly induce membrane curvature by amphipathic helix insertion into the lipid bilayer Moreover, the association of TREX-2 with the NPC requires the basket nucleoporins NUP153 and TPR, but is independent of transcription.\", \"Nup133 Is Required for Proper Nuclear Pore Basket Assembly and Dynamics in Embryonic Stem Cells. The nuclear pore complex basket proteins Nup1 and Nup60 directly induce membrane curvature by amphipathic helix insertion into the lipid bilayer. The nucleoporin Nup153 is required for nuclear pore basket formation, nuclear pore complex anchoring and import of a subset of nuclear proteins.\", \"Nup133 Is Required for Proper Nuclear Pore Basket Assembly and Dynamics in Embryonic Stem Cells. Here we show that the NPC basket proteins Nup1 and Nup60 directly induce membrane curvature by amphipathic helix insertion into the lipid bilayer The nucleoporin Nup153 is required for nuclear pore basket formation, nuclear pore complex anchoring and import of a subset of nuclear proteins\"], \"exact_answer\": [[\"Nup133\"], [\"Nup1\"], [\"Nup60\"], [\"Nup153\"]], \"context\": \"Nup133 Is Required for Proper Nuclear Pore Basket Assembly and Dynamics in Embryonic Stem Cells.Here we show that the NPC basket proteins Nup1 and Nup60 directly induce membrane curvature by amphipathic helix insertion into the lipid bilayerMoreover, the association of TREX-2 with the NPC requires the basket nucleoporins NUP153 and TPR, but is independent of transcription.  In esc1Delta cells, Ulp1 and nuclear basket components Nup60 and Mlp1 no longer distribute broadly around the nuclear periphery, but co-localize in a small number of dense-staining perinuclear foci.The nucleoporin Nup153 is required for nuclear pore basket formation, nuclear pore complex anchoring and import of a subset of nuclear proteinsNPCs assembled in the absence of Nup153 lacked several nuclear basket components, were unevenly distributed in the NE and, unlike wild-type NPCs, were mobile within the NEIncorporation of the nuclear pore basket protein nup153 into nuclear pore structures is dependent upon lamina assembly: evidence from cell-free extracts of Xenopus eggs.Using light and EM immunolocalization, we determined that mammalian Tpr is concentrated within the nuclear basket of the pore complex in a distribution similar to Nup153 and Nup98.Immunofluorescence and immunoelectron microscopy localize vertebrate Nup93 at the nuclear basket and at or near the nuclear entry to the gated channel of the pore.Strikingly, Nup133 and Nup107 of the Nup107/160 subcomplex and Nup153 and Nup50 of the nuclear pore basket recruited a near full complement of nucleoporins to the LacO array.Nup153 is a large (153 kD) O-linked glyco-protein which is a component of the basket structure located on the nucleoplasmic face of nuclear pore complexes.Nup153 is a molecular constituent of the nuclear basket of the nuclear pore complex (NPC) that plays a critical role in nuclear export of RNAs and proteins.The nucleoporins Nup60p, Nup2p, and Nup1p form part of the nuclear basket structure of the Saccharomyces cerevisiae nuclear pore complex (NPC).By postembedding immunoelectron microscopy, we have mapped the positions of several human NPC proteins relative to the NPC core and its associated basket, including Nup93, Nup96, Nup98, Nup107, Nup153, Nup205, and the coiled coil-dominated 267-kDa protein Tpr.In esc1Delta cells, Ulp1 and nuclear basket components Nup60 and Mlp1 no longer distribute broadly around the nuclear periphery, but co-localize in a small number of dense-staining perinuclear foci.Targeting information for Nup153 resides in the NH2-terminal domain since this region of the molecule can direct an ordinarily cytoplasmic protein, pyruvate kinase, to the nuclear face of the nuclear pore complex.We show that the assembly and integrity of the nuclear basket is not affected by Nup153 depletion, whereas its integrity is perturbed in cells expressing high concentrations of the zinc-finger domain of Nup153.When full-length human Nup153 is expressed in BHK cells, it accumulates appropriately at the nucleoplasmic face of the nuclear envelope.One such protein, the nucleoporin Nup153, is localized to the nuclear basket of the pore complex and has been shown to be a central component of the nuclear transport machinery.Here, one SLAP is identified as the nuclear pore complex protein Nup153, one member of the F/GXFG motif-containing nucleoporins. Nup133 Is Required for Proper Nuclear Pore Basket Assembly and Dynamics in Embryonic Stem Cells.Nup133 belongs to the Y-complex, a key component of the nuclear pore complex (NPC) scaffold. The nucleoporin Nup153 is required for nuclear pore basket formation, nuclear pore complex anchoring and import of a subset of nuclear proteins.The nuclear pore complex (NPC) is a large proteinaceous structure through which bidirectional transport of macromolecules across the nuclear envelope (NE) takes place. To further assess their contributions to NPC and basket architecture, the genes encoding Nup93, Nup96, Nup107, and Nup205 were posttranscriptionally silenced by RNA interference (RNAi) in HeLa cells, complementing recent RNAi experiments on Nup153 and Tpr.Immunogold-labeling for Nup98 also results in preferential labeling of NPC core regions, whereas Nup153 is shown to bind via its amino-terminal domain to the nuclear coaxial ring linking the NPC core structures and Tpr.Ulp1, whose maintenance at nuclear pores depends on the Nup84 complex, impacts on THO-dependent gene expression, whereas the nuclear basket-associated Mlp1/Pml39 proteins prevent pre-mRNA export at a later stage, contributing to mRNA quality control.\"}, {\"question\": \"What are the effects of STEF depletion?\", \"ideal_answer\": [\"The mechanisms regulating the actin cap are currently poorly understood. STEF/TIAM2, a Rac1 selective guanine nucleotide exchange factor, localises at the nuclear envelope, co-localising with the key perinuclear proteins Nesprin-2G and Non-muscle myosin IIB (NMMIIB), where it regulates perinuclear Rac1 activity. STEF depletion reduces apical perinuclear actin cables (a phenotype rescued by targeting active Rac1 to the nuclear envelope), decreases nuclear stiffness and reduces expression of TAZ-regulated genes, indicating an alteration in mechanosensing pathways as a consequence of disruption of the actin cap.\"], \"exact_answer\": [[\"Reduction of apical perinuclear actin cables\"], [\"Decrease of nuclear stiffness\"], [\"Reduction of expression of TAZ-regulated genes\"]], \"context\": \"The mechanisms regulating the actin cap are currently poorly understood. Here, we demonstrate that STEF/TIAM2, a Rac1 selective guanine nucleotide exchange factor, localises at the nuclear envelope, co-localising with the key perinuclear proteins Nesprin-2G and Non-muscle myosin IIB (NMMIIB), where it regulates perinuclear Rac1 activity. We show that STEF depletion reduces apical perinuclear actin cables (a phenotype rescued by targeting active Rac1 to the nuclear envelope), increases nuclear height and impairs nuclear re-orientation. STEF down-regulation also reduces perinuclear pMLC and decreases myosin-generated tension at the nuclear envelope, suggesting that STEF-mediated Rac1 activity regulates NMMIIB activity to promote stabilisation of the perinuclear actin cap. Finally, STEF depletion decreases nuclear stiffness and reduces expression of TAZ-regulated genes, indicating an alteration in mechanosensing pathways as a consequence of disruption of the actin cap.We show that STEF depletion reduces apical perinuclear actin cables (a phenotype rescued by targeting active Rac1 to the nuclear envelope), increases nuclear height and impairs nuclear re-orientation.Finally, STEF depletion decreases nuclear stiffness and reduces expression of TAZ-regulated genes, indicating an alteration in mechanosensing pathways as a consequence of disruption of the actin cap.STEF depletion drastically reduced dbcAMP-induced neurite outgrowth.STEF depletion and Rac1-N17 expression reduced cAMP-induced TC10 inactivation.Finally, STEF depletion decreases nuclear stiffness and reduces expression of TAZ-regulated genes, indicating an alteration in mechanosensing pathways as a consequence of disruption of the actin cap.<br>\"}, {\"question\": \"Please list symptoms of measles.\", \"ideal_answer\": [\"The leading symptoms of the disease are high fever that presents after an incubation period of 9-10 days and the red rash that begins several days after the fever starts. Beyond specific generalized symptoms, measles may have ocular symptoms. The most commonly occuring satellite symptoms are conjunctivitis, coryza and cough.\"], \"exact_answer\": [[\"red rash on skin\"], [\"high fever\"], [\"conjunctivitis\"], [\"coryza\"], [\"cough\"]], \"context\": \"ever and generalized maculopapular rash, plus \\u22651 of the following symptoms: Coryza, conjunctivitis, or cough.Measles is a highly contagious viral infection associated with clinical symptoms such as fever, cough, conjunctivitis, coryza, eruption and increased serum immunoglobulin M (IgM) antibodies.This patient had typical clinical measles infection symptoms: fever, rash, cough and coryza.\"}, {\"question\": \"Which databases can exchange data using Matchmaker Exchange's API?\", \"ideal_answer\": [\"Matchmaker Exchange (MME) was created to establish a federated network connecting databases of genomic and phenotypic data using a common application programming interface (API). To date, seven databases can exchange data using the API: GeneMatcher, PhenomeCentral, DECIPHER, MyGene2, matchbox, Australian Genomics Health Alliance Patient Archive, and Monarch Initiative; the latter included for model organism matching.\"], \"exact_answer\": [[\"GeneMatcher\"], [\"PhenomeCentral\"], [\"DECIPHER\"], [\"MyGene2\"], [\"matchbox\"], [\"Australian Genomics Health Alliance Patient Archive\"], [\"Monarch Initiative\"]], \"context\": \"To facilitate such communication and improve the search for patients or model organisms with similar phenotypes and variants in specific candidate genes, we have developed the Matchmaker Exchange (MME). MME was created to establish a federated network connecting databases of genomic and phenotypic data using a common application programming interface (API). To date, seven databases can exchange data using the API (GeneMatcher, PhenomeCentral, DECIPHER, MyGene2, matchbox, Australian Genomics Health Alliance Patient Archive, and Monarch Initiative; the latter included for model organism matching).To date, seven databases can exchange data using the API (GeneMatcher, PhenomeCentral, DECIPHER, MyGene2, matchbox, Australian Genomics Health Alliance Patient Archive, and Monarch Initiative; the latter included for model organism matching).To date, seven databases can exchange data using the API (GeneMatcher, PhenomeCentral, DECIPHER, MyGene2, matchbox, Australian Genomics Health Alliance Patient Archive, and Monarch Initiative; the latter included for model organism matching). \"}, {\"question\": \"What membrane proteins constitute TAM family of receptor tyrosine kinases (RTKs)?\", \"ideal_answer\": [\"Tyro3, Axl, and Mer are integral membrane proteins that constitute TAM family of receptor tyrosine kinases (RTKs).\", \"TAM-family RTKs AXL and Mer (MerTK).\"], \"exact_answer\": [[\"Tyro3\"], [\"Axl\"], [\"Mer\"]], \"context\": \"TAM-family RTKs AXL and Mer (MerTK). Deregulation of the TAM (TYRO3, AXL, and MERTKTAM family members AXL or TYRO3 receptors of the Tyro3/Axl/Mer (TAM) family of receptor tyrosine kinases (RTKs)Tyro-3, Axl, and Mer constitute the TAM family of receptor tyrosine kinases (RTKs) characterized by a conserved sequence within the kinase domain and adhesion molecule-like extracellular domains.The TYRO3, AXL (also known as UFO) and MERTK (TAM) family of receptor tyrosine kinases (RTKs) are aberrantly expressed in multiple haematological and epithelial malignancies.Gas6 was cloned and characterized in 1993 and found to be similar to the plasma anticoagulant protein S. Soon after it was recognized as a growth factor-like molecule, as it interacted with receptor tyrosine kinases (RTKs) of the TAM family; Tyro3, Axl, and MerTK.The TAM subfamily of Receptor Tyrosine Kinases (RTKs) contains three human proteins of therapeutical interest, Axl, Mer, and Tyro3.Increasing evidence shows that the TAM (TYRO3, AXL and MERTK) family of RTKs (receptor tyrosine kinases), which is expressed in macrophages, alleviates inflammatory responses through a negative feedback loop.Among the TAM family of RTKs, AXL (AXL receptor tyrosine kinase) only induces autophagy in macrophages after interaction with its ligand, GAS6 (growth arrest specific 6).A series of recent studies now consolidates the view that phosphatidylserine (PtdSer), a common phospholipid constituent of membrane bilayers, is similarly repurposed for use as a signal between cells and that the ligands and receptors of the Tyro3/Axl/Mer (TAM) family of receptor tyrosine kinases (RTKs) are prominent transducers of this signal.The TAM family: phosphatidylserine sensing receptor tyrosine kinases gone awry in cancer.The TYRO3, AXL (also known as UFO) and MERTK (TAM) family of receptor tyrosine kinases (RTKs) are aberrantly expressed in multiple haematological and epithelial malignancies. The TAM family of RTKs, consisting of Tyro3, Axl, and MerTK, is prominently expressed during the development and function of the central nervous system (CNS). The TAM family of RTKs, consisting of Tyro3, Axl, and MerTK, is prominently expressed during the development and function of the central nervous system (CNS).\"}, {\"question\": \"What are the effects of the deletion of all three Pcdh clusters (tricluster deletion) in mice?\", \"ideal_answer\": [\"Multicluster Pcdh diversity is required for mouse olfactory neural circuit assembly. The vertebrate clustered protocadherin (Pcdh) cell surface proteins are encoded by three closely linked gene clusters (Pcdh\\u03b1, Pcdh\\u03b2, and Pcdh\\u03b3). Although deletion of individual Pcdh clusters had subtle phenotypic consequences, the loss of all three clusters (tricluster deletion) led to a severe axonal arborization defect and loss of self-avoidance.\", \"The vertebrate clustered protocadherin (Pcdh) cell surface proteins are encoded by three closely linked gene clusters (Pcdh\\u03b1, Pcdh\\u03b2, and Pcdh\\u03b3). Although deletion of individual Pcdh clusters had subtle phenotypic consequences, the loss of all three clusters (tricluster deletion) led to a severe axonal arborization defect and loss of self-avoidance.\"], \"exact_answer\": [[\"Severe axonal arborization defect\"], [\"Loss of self-avoidance\"]], \"context\": \"Multicluster Pcdh diversity is required for mouse olfactory neural circuit assembly.The vertebrate clustered protocadherin (Pcdh) cell surface proteins are encoded by three closely linked gene clusters (Pcdh\\u03b1, Pcdh\\u03b2, and Pcdh\\u03b3). Here, we show that all three gene clusters functionally cooperate to provide individual mouse olfactory sensory neurons (OSNs) with the cell surface diversity required for their assembly into distinct glomeruli in the olfactory bulb. Although deletion of individual Pcdh clusters had subtle phenotypic consequences, the loss of all three clusters (tricluster deletion) led to a severe axonal arborization defect and loss of self-avoidance.Although deletion of individual Pcdh clusters had subtle phenotypic consequences, the loss of all three clusters (tricluster deletion) led to a severe axonal arborization defect and loss of self-avoidance.Here, we show that all three gene clusters functionally cooperate to provide individual mouse olfactory sensory neurons (OSNs) with the cell surface diversity required for their assembly into distinct glomeruli in the olfactory bulb.Although deletion of individual <i>Pcdh</i> clusters had subtle phenotypic consequences, the loss of all three clusters (tricluster deletion) led to a severe axonal arborization defect and loss of self-avoidance. Here, we show that all three gene clusters functionally cooperate to provide individual mouse olfactory sensory neurons (OSNs) with the cell surface diversity required for their assembly into distinct glomeruli in the olfactory bulb. \"}, {\"question\": \"What micro-RNAs are useful in the diagnosis and prognosis of Heart Failure?\", \"ideal_answer\": [\"In particular, miR-214, miR-423-5p, appear to be promising for the diagnosis, prognosis and management of HF patients.\"], \"exact_answer\": [[\"miR-214\"], [\"miR-423-5p\"]], \"context\": \"n this review, we discuss the role of miR-214 in various cardiac disease conditions, including ischaemic heart diseases, cardiac hypertrophy, pulmonary arterial hypertension (PAH), angiogenesis following vascular injury and heart failure.Likewise, serum levels of c-miRNAs such as miR-423-5p have been evaluated as potential biomarkers in the diagnosis and prognosis of heart failureSerum levels of circulating miRNAs such as miR-423-5p are being evaluated as potential biomarkers in the diagnosis and prognosis of heart failure.Likewise, serum levels of c-miRNAs such as miR-423-5p have been evaluated as potential biomarkers in the diagnosis and prognosis of heart failure.\"}, {\"question\": \"List proteins interacting with Star-PAP\", \"ideal_answer\": [\"Phosphorylation regulates the Star-PAP-PIPKI\\u03b1 interaction and directs specificity toward mRNA targets.\\nStar-PAP directly associated with cleavage and polyadenylation specificity factor (CPSF) 160 and 73 subunits and also the targeted pre-mRNA.\\nwe show that Larp7 interacts with a poly(A) polymerase Star-PAP to maintain Lin28 mRNA stability.\"], \"exact_answer\": [[\"PIPKI\\u03b1\"], [\"CPSF 160\"], [\"CPSF 73\"], [\"Larp7\"]], \"context\": \"Star-PAP and PIPKIalpha function together in a complexPhosphorylation regulates the Star-PAP-PIPKI\\u03b1 interaction and directs specificity toward mRNA targets., we show that Larp7 interacts with a poly(A) polymerase Star-PAP to maintain Lin28 mRNA stability. Star-PAP directly associated with cleavage and polyadenylation specificity factor (CPSF) 160 and 73 subunits and also the targeted pre-mRNA. \"}, {\"question\": \"Name 3 diseases for which lucatumumab is being tested?\", \"ideal_answer\": [\"Lucatumumab is being tested as treatment for malignancies such as chronic lymphatic leukemia (CLL), Multiple Myeloma (MM), and non-Hodgkin's lymphoma (NHL).\"], \"exact_answer\": [[\"chronic lymphatic leukemia\"], [\"multiple myeloma\"], [\"non-Hodgkin's lymphoma\"]], \"context\": \"Here, Dacetuzumab and Lucatumumab are in the most advanced stage and are being tested as treatment for malignancies such as chronic lymphatic leukemia (CLL), Multiple Myeloma (MM), and non-Hodgkin's lymphoma (NHL).\"}, {\"question\": \"Name 4 side effects of enasidenib\", \"ideal_answer\": [\"Enasidenib is found to be associated with certain adverse effects like elevated bilirubin level, diarrhea, differentiation syndrome, decreased potassium and calcium levels etc.\"], \"exact_answer\": [[\"elevated bilirubin levels\"], [\"diarrhea\"], [\"decreased potassium levels\"], [\"decreased calcium levels\"]], \"context\": \"Enasidenib is found to be associated with certain adverse effects like elevated bilirubin level, diarrhea, differentiation syndrome, decreased potassium and calcium levels etc.\"}, {\"question\": \"List MHC-I-associated inflammatory disorders.\", \"ideal_answer\": [\"ankylosing spondylitis\\nbirdshot chorioretinopathy\\nBeh\\u00e7et's disease \\npsoriasis\"], \"exact_answer\": [[\"ankylosing spondylitis\"], [\"birdshot chorioretinopathy\"], [\"Beh\\u00e7et's disease\"], [\"psoriasis\"]], \"context\": \"This review will focus on the four major MHC-I-associated inflammatory disorders: ankylosing spondylitis, birdshot chorioretinopathy, Beh\\u00e7et's disease and psoriasis.Birdshot chorioretinopathy is a rare ocular inflammation whose genetic association with HLA-A*29:02 is the highest between a disease and a major histocompatibility complex (MHC) molecule. It belongs to a group of MHC-I-associated inflammatory disorders, also including ankylosing spondylitis, psoriasis, and Beh\\u00e7et's disease\"}, {\"question\": \"Which two drugs are included in the MAVYRET pill?\", \"ideal_answer\": [\"MAVYRET pill includes glecaprevir and pibrentasvir. It is used for treatment of hepatitis C infection.\"], \"exact_answer\": [[\"glecaprevir\"], [\"pibrentasvir\"]], \"context\": \"A fixed-dose combination tablet of the hepatitis C virus (HCV) NS3/4A protease inhibitor (PI) glecaprevir and the HCV NS5A inhibitor pibrentasvir [glecaprevir/pibrentasvir; MAVIRET\\u2122 (EU); MAVYRET\\u2122 (USA)] has been developed by AbbVie.Aminolevulinic acid hydrochloride (Gleolan) for the visualization of malignant tissue during surgery; delafloxacin (Baxdela) for certain acute bacterial skin infections; and glecaprevir/pibrentasvir (Mavyret) for chronic HCV infection.\"}, {\"question\": \"Which two drugs are included in the Entresto pill?\", \"ideal_answer\": [\"Entresto includes Sacubitril and Valsartan.\"], \"exact_answer\": [[\"Sacubitril\"], [\"Valsartan\"]], \"context\": \"A novel antihypertensive drug, LCZ696 (Entresto\\u00ae), has recently been introduced, which combines the action of an antagonist of the renin-angiotensin-aldosterone system (RAAS), effectively decreasing the blood pressure, with an inhibition of neprilysin, which is responsible for metabolizing natriuretic peptides exerting antihypertensive and antifibrotic effects.METHODS AND RESULTS: The ENTRESTO-SAS trial is a 3-month, multicentric, prospective, open-label real-life cohort study. Patients eligible for sacubitril-valsartan treatment (i.e. adults with left ventricular ejection fraction \\u226435%, who remain symptomatic despite optimal treatment with an angiotensin-converting enzyme inhibitor, a beta-blocker, and a mineralocorticoid receptor antagonist) will be evaluated before and after 3\\u00a0months of treatment (nocturnal ventilatory polygraphy, echocardiography, laboratory testing, and quality-of-life and SDB questionnaires).  We sought to assess in the ENTRESTO-SAS trial whether sacubitril-valsartan could improve the outcome of SDB in CHF patients.This positive change in regulatory perspective coupled with successful commercialization of valsartan-sacubitril co-crystal (Entresto, Novartis) has now brought co-crystals into focus, in both industries as well as academia.We present a case of a 61-year-old man with ischemic cardiomyopathy receiving sacubitril/valsartan (Entresto; Novartis, Cambridge, MA) who developed profound hypotension after HT.Recently, the Food and Drug Administration (FDA) approved a drug with brand name Entresto (Sacubitril/Valsartan or LCZ696), an angiotensin receptor neprilysin inhibitor for the use in Heart Failure with Reduced Ejection Fraction (HFrEF) patients instead of ACEI's and ARBs.A prospective, randomized, double-blind, placebo-controlled pilot study of sacubitril/valsartan (Entresto) in dogs with cardiomegaly secondary to myxomatous mitral valve disease.This review of the literature highlights the new generation of HF drug, sacubitril-valsartan (SV), trade name Entresto (researched as LCZ696, Novartis) which simultaneously blocks RAAS and NP.\\u25bc Sacubitril valsartan (Entresto-Novartis) is a new oral drug licensed for the treatment of symptomatic chronic heart failure in adults with reduced ejection fraction.Sacubitril/valsartan (Entresto) for chronic heart failure; brexpiprazole (Rexulti) for major depressive disorder and schizophrenia; and lumacaftor/ivacaftor (Orkambi) for cystic fibrosis involving specific CFTR mutations.Correction to: Do Limitations in the Design of PARADIGM-HF Justify the Slow Real World Uptake of Sacubitril/Valsartan (Entresto)?The original version of this article unfortunately contained a mistake. \"}, {\"question\": \"For which indications has midostaurin received FDA and EMA approval?\", \"ideal_answer\": [\"Midostaurin was approved by the Food and Drug Administration (FDA) and the European Medical Agency (EMA) for acute myeloid leukemia with activating FLT3 mutations in combination with intensive induction and consolidation therapy as well as aggressive systemic mastocytosis (ASM), systemic mastocytosis with associated hematological neoplasm (SM-AHN) or mast cell leukemia (MCL).\"], \"exact_answer\": [[\"Acute myeloid leukemia with activating FLT3 mutations\"], [\"Aggressive systemic mastocytosis\"], [\"Systemic mastocytosis with associated hematological neoplasm\"], [\"Mast cell leukemia\"]], \"context\": \"Midostaurin: A Multiple Tyrosine Kinases Inhibitor in Acute Myeloid Leukemia and Systemic Mastocytosis.Midostaurin was approved by the Food and Drug Administration (FDA) and the European Medical Agency (EMA) for acute myeloid leukemia with activating FLT3 mutations in combination with intensive induction and consolidation therapy as well as aggressive systemic mastocytosis (ASM), systemic mastocytosis with associated hematological neoplasm (SM-AHN) or mast cell leukemia (MCL).\"}, {\"question\": \"Which drugs are included in the Orkambi pill?\", \"ideal_answer\": [\"Orkambi pill includes lumacaftor combined with ivacaftor. It is approved for treatment of cystic fibrosis with F508del-CFTR mutation.\"], \"exact_answer\": [[\"lumacaftor\"], [\"ivacaftor\"]], \"context\": \"Thus studies were conducted to examine the effects of lumacaftor alone and lumacaftor in combination with ivacaftor (i.e., ORKAMBI) on the ability of human CF ( del508Phe/ del508Phe) monocyte-derived macrophages (MDMs) to phagocytose and kill Pseudomonas aeruginosa.In recent years, two modulators, ivacaftor (Kalydeco) and lumacaftor/ivacaftor (Orkambi), have been approved by the U.S. Food and Drug Administration to treat CF patients with certain CFTR mutations. Recently, ORKAMBI, a combination therapy that includes a corrector of the processing defect of F508del-CFTR (lumacaftor or VX-809) and a potentiator of channel activity (ivacaftor or VX-770), was approved for CF patients homozygous for this mutation. The recent US Food and Drug Administration (FDA) approval of lumacaftor combined with ivacaftor (Orkambi) targets patients with the F508del-CFTR. In July 2015, the US Food and Drug Administration approved Orkambi (lumacaftor/ivacaftor), a combination drug that works on reversing the effects of the defective cystic fibrosis transmembrane conductance regulator protein. Areas covered: In 2016, the Food and Drug Administration and the European Commission approved LUM/IVA (Orkambi), a CFTR modulator that includes both a CFTR corrector and potentiator, for CF patients homozygous for the F508del CFTR mutation. The combination therapy of lumacaftor and ivacaftor (Orkambi\\u00ae) is approved for patients bearing the major cystic fibrosis (CF) mutation: \\u0394F508 It has been predicted that Orkambi\\u00ae could treat patients with rarer mutations of similar \\\"theratype\\\"; however, a standardized approach confirming efficacy in these cohorts has not been reported.ORKAMBI (ivacaftor-lumacaftor [LUMA]) and KALYDECO (ivacaftor; IVA) are two new breakthrough cystic fibrosis (CF) drugs that directly modulate the activity and trafficking of the defective CFTR underlying the CF disease state.Ivacaftor is currently used for the treatment of cystic fibrosis as both monotherapy (Kalydeco; Vertex Pharmaceuticals, Boston, MA) and combination therapy with lumacaftor (Orkambi; Vertex Pharmaceuticals).Lisinopril oral solution (Qbrelis) for the treatment of hypertension, heart failure, and acute myocardial infarction; etanercept-szzs (Erelzi) for multiple autoimmune disorders; and lumacaftor/ivacaftor (Orkambi) for cystic fibrosis.Sacubitril/valsartan (Entresto) for chronic heart failure; brexpiprazole (Rexulti) for major depressive disorder and schizophrenia; and lumacaftor/ivacaftor (Orkambi) for cystic fibrosis involving specific CFTR mutations.The recent US Food and Drug Administration (FDA) approval of lumacaftor combined with ivacaftor (Orkambi) targets patients with the F508del-CFTR.Altering Metabolic Profiles of Drugs by Precision Deuteration 2: Discovery of a Deuterated Analog of Ivacaftor with Differentiated Pharmacokinetics for Clinical Development.Ivacaftor is currently used for the treatment of cystic fibrosis as both monotherapy (Kalydeco; Vertex Pharmaceuticals, Boston, MA) and combination therapy with lumacaftor (Orkambi; Vertex Pharmaceuticals). Development of HPLC and LC-MS/MS methods for the analysis of ivacaftor, its major metabolites and lumacaftor in plasma and sputum of cystic fibrosis patients treated with ORKAMBI or KALYDECO.ORKAMBI (ivacaftor-lumacaftor [LUMA]) and KALYDECO (ivacaftor; IVA) are two new breakthrough cystic fibrosis (CF) drugs that directly modulate the activity and trafficking of the defective CFTR underlying the CF disease state. In July 2015, the US Food and Drug Administration approved Orkambi (lumacaftor/ivacaftor), a combination drug that works on reversing the effects of the defective cystic fibrosis transmembrane conductance regulator protein. The combination therapy of lumacaftor and ivacaftor (Orkambi\\u00ae) is approved for patients bearing the major cystic fibrosis (CF) mutation: \\u0394F508 It has been predicted that Orkambi\\u00ae could treat patients with rarer mutations of similar theratype; however, a standardized approach confirming efficacy in these cohorts has not been reported.\"}, {\"question\": \"List available R packages for processing NanoString data\", \"ideal_answer\": [\"NanoStringNorm and NanoStringNormCNV.\"], \"exact_answer\": [[\"NanoStringNorm\"], [\"NanoStringNormCNV\"]], \"context\": \"NanoStringNorm: an extensible R package for the pre-processing of NanoString mRNA and miRNA data.The NanoString nCounter Platform is a new and promising technology for measuring nucleic acid abundances. It has several advantages over PCR-based techniques, including avoidance of amplification, direct sequence interrogation and digital detection for absolute quantification. These features minimize aspects of experimental error and hold promise for dealing with challenging experimental conditions such as archival formalin-fixed paraffin-embedded samples. However, systematic inter-sample technical artifacts caused by variability in sample preservation, bio-molecular extraction and platform fluctuations must be removed to ensure robust data.RESULTS: To facilitate this process and to address these issues for NanoString datasets, we have written a pre-processing package called NanoStringNorm in the R statistical language.NanoStringNormCNV: pre-processing of NanoString CNV data.The NanoString System is a well-established technology for measuring RNA and DNA abundance. Although it can estimate copy number variation, relatively few tools support analysis of these data. To address this gap, we created NanoStringNormCNV, an R package for pre-processing and copy number variant calling from NanoString data. This package implements algorithms for pre-processing, quality-control, normalization and copy number variation detection. A series of reporting and data visualization methods support exploratory analyses. To demonstrate its utility, we apply it to a new dataset of 96 genes profiled on 41 prostate tumour and 24 matched normal samples.To address this gap, we created NanoStringNormCNV, an R package for pre-processing and copy number variant calling from NanoString data.Summary\\nThe NanoString System is a well-established technology for measuring RNA and DNA abundance.MOTIVATION\\nThe NanoString nCounter Platform is a new and promising technology for measuring nucleic acid abundances.MOTIVATION The NanoString nCounter Platform is a new and promising technology for measuring nucleic acid abundances.Summary The NanoString System is a well-established technology for measuring RNA and DNA abundance.Availability and implementation NanoStringNormCNV is implemented in R and is freely available at http://labs.oicr.on.ca/boutros-lab/software/nanostringnormcnv.NanoStringNorm: an extensible R package for the pre-processing of NanoString mRNA and miRNA data.<AbstractText Label=\\\"MOTIVATION\\\" NlmCategory=\\\"BACKGROUND\\\">The NanoString nCounter Platform is a new and promising technology for measuring nucleic acid abundances. <b>MOTIVATION</b>: The NanoString nCounter Platform is a new and promising technology for measuring nucleic acid abundances.However, systematic inter-sample technical artifacts caused by variability in sample preservation, bio-molecular extraction and platform fluctuations must be removed to ensure robust data.<br><b>RESULTS</b>: To facilitate this process and to address these issues for NanoString datasets, we have written a pre-processing package called NanoStringNorm in the R statistical language.<b>Summary</b>: The NanoString System is a well-established technology for measuring RNA and DNA abundance.To address this gap, we created NanoStringNormCNV, an R package for pre-processing and copy number variant calling from NanoString data.To demonstrate its utility, we apply it to a new dataset of 96 genes profiled on 41 prostate tumour and 24 matched normal samples.<br><b>Availability and implementation</b>: NanoStringNormCNV is implemented in R and is freely available at http://labs.oicr.on.ca/boutros-lab/software/nanostringnormcnv.<br><b>Contact</b>: paul.boutros@oicr.on.ca.<br><b>Supplementary information</b>: Supplementary data are available at Bioinformatics online.<br>\"}, {\"question\": \"List phagosomal markers.\", \"ideal_answer\": [\"Rab7\\nLAMP1\\nCathepsin D\\nRab9\\nV-ATPase \\nCD63\"], \"exact_answer\": [[\"Rab7\"], [\"LAMP1\"], [\"Cathepsin D\"], [\"Rab9\"], [\"V-ATPase\"], [\"CD63\"]], \"context\": \"Monitoring of intracellular trafficking of H. somni with monoclonal antibodies to phagosomal markers indicated that the early phagosomal marker early endosome antigen 1 colocalized with all isolates tested, but only strains that could survive intracellularly did not colocalize with the late lysosomal marker lysosome-associated membrane protein 2 and prevented the acidification of phagosomes. late phagosomal markers viz. Rab7, Lysosomal Associated Membrane Protein 1, Cathepsin D, Rab9, and V-ATPase which indicate phagosome maturation. early phagosomal markers [such as, Ras-related proteins in the brain 5 (Rab5) and Transferrin receptor (TfR)], and a failure to acquire late phagosomal and lysosomal markers (such as Rab7 and LAMP1)late phagosomal markers CD63 and Rab7 late phagosomal markers mannose-6-phosphate receptor (M6PR) and Rab7\"}, {\"question\": \"List bacterial families for which delafloxacin has been shown to be effective.\", \"ideal_answer\": [\"Delafloxacin has been shown to be effective against multiple gram-positive and gram-negative bacteria, including Strepotococcus pneumoniae, Haemophilus influenzae, Moraxella atarrhalis, Staphylococcus aureus, Klebsiella pneumoniae and more.\"], \"exact_answer\": [[\"Streptococcus pneumoniae\"], [\"Haemophilus influenzae\"], [\"Moraxella catarrhalis\"], [\"Staphylococcus aureus\"], [\"Klebsiella pneumoniae\"]], \"context\": \"In this study, 200 Streptococcus pneumoniae (plus 30 levofloxacin-resistant isolates), 200 Haemophilus influenzae, and 100 Moraxella catarrhalis isolates selected primarily from the United States (2014) were tested against delafloxacin and comparator agents. Delafloxacin was the most potent agent tested.Delafloxacin demonstrated in vitro and in vivo potency against a diverse group of pathogens, including those with phenotypic drug resistance to other classes.The goal of the study was to determine the pharmacokinetic/pharmacodynamic (PK/PD) targets in the murine lung infection model for Staphylococcus aureus, Streptococcus pneumoniae, and Klebsiella pneumoniae \"}, {\"question\": \"What type of data does the UK biobank resource contain?\", \"ideal_answer\": [\"The UK Biobank contains deep phenotyping and genomic data. A rich variety of phenotypic and health-related information is available on each participant, including biological measurements, lifestyle indicators, biomarkers in blood and urine, and imaging of the body and brain. Follow-up information is provided by linking health and medical records. Genome-wide genotype data have been collected on all participants, providing many opportunities for the discovery of new genetic associations and the genetic bases of complex traits.\"], \"exact_answer\": [[\"genomic data\"], [\"phenotypic data\"]], \"context\": \"The UK Biobank resource with deep phenotyping and genomic data.The UK Biobank project is a prospective cohort study with deep genetic and phenotypic data collected on approximately 500,000 individuals from across the United Kingdom, aged between 40 and 69 at recruitment. The open resource is unique in its size and scope. A rich variety of phenotypic and health-related information is available on each participant, including biological measurements, lifestyle indicators, biomarkers in blood and urine, and imaging of the body and brain. Follow-up information is provided by linking health and medical records. Genome-wide genotype data have been collected on all participants, providing many opportunities for the discovery of new genetic associations and the genetic bases of complex traits.The UK Biobank resource with deep phenotyping and genomic data.The UK Biobank project is a prospective cohort study with deep genetic and phenotypic data collected on approximately 500,000 individuals from across the United Kingdom, aged between 40 and 69 at recruitment. BACKGROUND\\nThe UK Biobank is a unique resource for biomedical research, with extensive phenotypic and genetic data on half a million adults from the general population.BACKGROUND The UK Biobank is a unique resource for biomedical research, with extensive phenotypic and genetic data on half a million adults from the general population.\"}, {\"question\": \"Which two surgical methods were compared in the RAZOR trial?\", \"ideal_answer\": [\"The RAZOR trial compared open radical cystectomy vs. robot-assisted radical cystectomy in patients with bladder cancer.\"], \"exact_answer\": [[\"open radical cystectomy\"], [\"robot-assisted radical cystectomy\"]], \"context\": \"Robot-assisted radical cystectomy versus open radical cystectomy in patients with bladder cancer (RAZOR): an open-label, randomised, phase 3, non-inferiority trial.METHODS: The RAZOR study is a randomised, open-label, non-inferiority, phase 3 trial done in 15 medical centres in the USA. Eligible participants (aged \\u226518 years) had biopsy-proven clinical stage T1-T4, N0-N1, M0 bladder cancer or refractory carcinoma in situ. Individuals who had previously had open abdominal or pelvic surgery, or who had any pre-existing health conditions that would preclude safe initiation or maintenance of pneumoperitoneum were excluded. Patients were centrally assigned (1:1) via a web-based system, with block randomisation by institution, stratified by type of urinary diversion, clinical T stage, and Eastern Cooperative Oncology Group performance status, to receive robot-assisted radical cystectomy or open radical cystectomy with extracorporeal urinary diversion.Results: Evidence from four single-centre randomised controlled trials and now the multicentre Randomized Trial of Open versus Robotic Cystectomy (RAZOR) trial demonstrate the oncological equivalence of RARC to ORC. The RAZOR (randomized open vs robotic cystectomy) trial: study design and trial update.The purpose of the RAZOR (randomized open vs robotic cystectomy) study is to compare open radical cystectomy (ORC) vs robot-assisted RC (RARC), pelvic lymph node dissection (PLND) and urinary diversion for oncological outcomes, complications and health-related quality of life (HRQL) measures with a primary endpoint of 2-year progression-free survival (PFS). RAZOR is a multi-institutional, randomized, non-inferior, phase III trial that will enrol at least 320 patients with T1-T4, N0-N1, M0 bladder cancer with \\u2248160 patients in both the RARC and ORC arms at 15 participating institutions. AZOR is a multi-institutional, non-inferiority trial evaluating cancer outcomes, surgical complications and HRQL measures of ORC vs RARC with a primary endpoint of 2-year PFS. Full data from the RAZOR trial are not expected until 2016-2017.The purpose of the RAZOR (randomized open vs robotic cystectomy) study is to compare open radical cystectomy (ORC) vs robot-assisted RC (RARC), pelvic lymph node dissection (PLND) and urinary diversion for oncological outcomes, complications and health-related quality of life (HRQL) measures with a primary endpoint of 2-year progression-free survival (PFS).Results\\nEvidence from four single-centre randomised controlled trials and now the multicentre Randomized Trial of Open versus Robotic Cystectomy (RAZOR) trial demonstrate the oncological equivalence of RARC to ORC.Results Evidence from four single-centre randomised controlled trials and now the multicentre Randomized Trial of Open versus Robotic Cystectomy (RAZOR) trial demonstrate the oncological equivalence of RARC to ORC.\"}, {\"question\": \"List potential reasons regarding why potentially important genes are ignored\", \"ideal_answer\": [\"Differences in attention can be explained, to a large extent, exclusively from a small set of identifiable chemical, physical, and biological properties of genes. Together with knowledge about homologous genes from model organisms, these features accurately predict the number of publications on individual human genes, the year of their first report, the levels of funding awarded by the National Institutes of Health (NIH), and the development of drugs against disease-associated genes.\"], \"exact_answer\": [[\"Identifiable chemical properties\"], [\"Identifiable physical properties\"], [\"Identifiable biological properties\"], [\"Knowledge about homologous genes from model organisms\"]], \"context\": \"Here, we demonstrate that these differences in attention can be explained, to a large extent, exclusively from a small set of identifiable chemical, physical, and biological properties of genes. Together with knowledge about homologous genes from model organisms, these features allow us to accurately predict the number of publications on individual human genes, the year of their first report, the levels of funding awarded by the National Institutes of Health (NIH), and the development of drugs against disease-associated genes.Here, we demonstrate that these differences in attention can be explained, to a large extent, exclusively from a small set of identifiable chemical, physical, and biological properties of genes. Here, we demonstrate that these differences in attention can be explained, to a large extent, exclusively from a small set of identifiable chemical, physical, and biological properties of genes.\"}, {\"question\": \"List STING agonists.\", \"ideal_answer\": [\"CDN 3'3'-cGAMP\\ndimethylxanthenone-4-acetic acid \\n\\u03b1-Mangostin\"], \"exact_answer\": [[\"CDN 3'3'-cGAMP\"], [\"dimethylxanthenone-4-acetic acid\"], [\"\\u03b1-Mangostin\"]], \"context\": \" CDN 3'3'-cGAMP (cGAMP) STING agonist, dimethylxanthenone-4-acetic acid (DMXAA) various cyclic dinucleotide (CDN) STING agonistsThe xanthone derivate 5',6'-dimethylxanthenone-4-acetic acid (DMXAA, also known as ASA404 or vadimezan) is a potent agonist of murine STING (stimulator of interferon genes), \\u03b1-Mangostin as an Agonist of Human STING\"}, {\"question\": \"List drugs that were tested in the CheckMate 214 trial.\", \"ideal_answer\": [\"CheckMate 214 clinical trial compared Nivolumab plus Ipilimumab versus Sunitinib in Advanced Renal-Cell Carcinoma.\"], \"exact_answer\": [[\"Nivolumab\"], [\"Ipilimumab\"], [\"Sunitinib\"]], \"context\": \" In 2015, the pivotal phase III study CheckMate 025 led to the Food and Drug Administration approval of nivolumab in patients with aRCC who had received prior anti-angiogenic therapy, and in 2017, the phase III study CheckMate 214 showed that combined immunotherapy with nivolumab plus ipilimumab resulted in greater objective response rate and prolonged progression-free survival when compared with sunitinib in intermediate- and poor-risk patients with previously untreated aRCCNivolumab plus Ipilimumab versus Sunitinib in Advanced Renal-Cell Carcinoma.BACKGROUND: Nivolumab plus ipilimumab produced objective responses in patients with advanced renal-cell carcinoma in a pilot study. This phase 3 trial compared nivolumab plus ipilimumab with sunitinib for previously untreated clear-cell advanced renal-cell carcinoma.METHODS: We randomly assigned adults in a 1:1 ratio to receive either nivolumab (3 mg per kilogram of body weight) plus ipilimumab (1 mg per kilogram) intravenously every 3 weeks for four doses, followed by nivolumab (3 mg per kilogram) every 2 weeks, or sunitinib (50 mg) orally once daily for 4 weeks (6-week cycle). CONCLUSIONS: Overall survival and objective response rates were significantly higher with nivolumab plus ipilimumab than with sunitinib among intermediate- and poor-risk patients with previously untreated advanced renal-cell carcinoma. (Funded by Bristol-Myers Squibb and Ono Pharmaceutical; CheckMate 214 ClinicalTrials.gov number, NCT02231749 .).Out of these CheckMate 214, a randomized phase-3 trial is the first to demonstrate a significant higher objective response rate (42\\u200a% vs. 27\\u200a%, p\\u200a<\\u200a0.0001) and overall survival (Sunitinib 26.0 month, median for Nivo\\u200a+\\u200aIpi has been not yet reached (28.2\\u200a-\\u200aNR); Hazard ratio 0.63) for the combination of Nivolumab and the CTLA-4 antibody Ipilimumab in IMDC intermediate and high risk patients. Furthermore, CheckMate 214 shows better side effect profile and quality of life in patients receiving Nivolumab and Ipilimumab compared with Sunitinib.CheckMate 214, a phase III trial, demonstrated superior overall survival and objective response with combined checkpoint inhibitors compared to sunitinib in Treatment-Na\\u00efve Advanced RCC among intermediate- and poor-risk group (Motzer et al. in N Engl J Med. 378(14):1277-1290, 2018).First, the CheckMate 214 trial demonstrated an objective response rate and overall survival benefit for the combination of nivolumab plus ipilimumab in the intermediate- and poor-risk patients. have recently published in the New England Journal of Medicine the findings of CheckMate 214 trial, using nivolumab and ipilimumab, a cytotoxic T-lymphocyte antigen 4 (CTLA-4) inhibitor, versus sunitinib in previously untreated advanced RCC. CheckMate 214: efficacy and safety of nivolumab plus ipilimumab vs sunitinib for treatment-na\\u00efve advanced or metastatic renal cell carcinoma, including IMDC risk and PD-L1 expression subgroups.The randomised phase III clinical trial Checkmate-214 showed a survival superiority for the combination of ipilimumab and nivolumab when compared with the previous standard of care in first-line metastatic/advanced clear cell renal cell carcinoma (RCC) (Escudier B, Tannir NM, McDermott DF, et al.Out of these CheckMate 214, a randomized phase-3 trial is the first to demonstrate a significant higher objective response rate (42\\u200a% vs. 27\\u200a%, p\\u200a<\\u200a0.0001) and overall survival (Sunitinib 26.0 month, median for Nivo\\u200a+\\u200aIpi has been not yet reached (28.2\\u200a-\\u200aNR); Hazard ratio 0.63) for the combination of Nivolumab and the CTLA-4 antibody Ipilimumab in IMDC intermediate and high risk patients.Furthermore, CheckMate 214 shows better side effect profile and quality of life in patients receiving Nivolumab and Ipilimumab compared with Sunitinib.CheckMate 025, the prospective randomized trial which led to approval of Nivolumab demonstrated improved overall survival (26 month vs. 19.7 month; hazard ratio 0.73; p\\u200a=\\u200a0.0006) and response rate (26\\u200a% vs. 5\\u200a%) as well as a favorable toxicity profile compared with Everolimus.In 2015, the pivotal phase III study CheckMate 025 led to the Food and Drug Administration approval of nivolumab in patients with aRCC who had received prior anti-angiogenic therapy, and in 2017, the phase III study CheckMate 214 showed that combined immunotherapy with nivolumab plus ipilimumab resulted in greater objective response rate and prolonged progression-free survival when compared with sunitinib in intermediate- and poor-risk patients with previously untreated aRCC.have recently published in the New England Journal of Medicine the findings of CheckMate 214 trial, using nivolumab and ipilimumab, a cytotoxic T-lymphocyte antigen 4 (CTLA-4) inhibitor, versus sunitinib in previously untreated advanced RCC.First, the CheckMate 214 trial demonstrated an objective response rate and overall survival benefit for the combination of nivolumab plus ipilimumab in the intermediate- and poor-risk patients.CheckMate 214, a phase III trial, demonstrated superior overall survival and objective response with combined checkpoint inhibitors compared to sunitinib in Treatment-Na\\u00efve Advanced RCC among intermediate- and poor-risk group (Motzer et al.Out of these CheckMate 214, a randomized phase-3 trial is the first to demonstrate a significant higher objective response rate (42\\u200a% vs. 27\\u200a%, p\\u200a&lt;\\u200a0.0001) and overall survival (Sunitinib 26.0 month, median for Nivo\\u200a+\\u200aIpi has been not yet reached (28.2\\u200a-\\u200aNR); Hazard ratio 0.63) for the combination of Nivolumab and the CTLA-4 antibody Ipilimumab in IMDC intermediate and high risk patients.Furthermore, CheckMate 214 shows better side effect profile and quality of life in patients receiving Nivolumab and Ipilimumab compared with Sunitinib.\"}, {\"question\": \"List two indications of Letermovir?\", \"ideal_answer\": [\"Letermovir is approved for the prophylaxis of CMV infection and disease in adult CMV-seropositive recipients of an allogeneic haematopoietic stem cell transplant (HSCT).\"], \"exact_answer\": [[\"prophylaxis of CMV infection\"], [\"CMV-seropositive recipients of an allogeneic haematopoietic stem cell transplant\"]], \"context\": \"Latanoprostene bunod ophthalmic solution (Vyzulta) for the reduction of intraocular pressure; letermovir (Prevymis) for prophylaxis of cytomegalovirus infection and disease after stem cell transplant; benralizumab (Fasenra) for the add-on maintenance treatment of severe asthma of an eosinophilic phenotype.Letermovir has been approved in Canada and the USA for the prophylaxis of CMV infection and disease in adult CMV-seropositive recipients of an allogeneic haematopoietic stem cell transplant (HSCT).This article summarizes the milestones in the development of letermovir leading to its first global approval in Canada as well as the USA for the prophylaxis of CMV infection and disease in adult CMV-seropositive recipients of an allogeneic HSCT.A new terminase inhibitor, letermovir, recently proved effective against HCMV in phase III clinical trials, but the mechanism of action is unclear. Letermovir is being developed for human cytomegalovirus infection treatment and prophylaxis. Letermovir is a human cytomegalovirus terminase inhibitor for cytomegalovirus infection prophylaxis in hematopoietic stem cell transplant recipients. Role of letermovir for prevention of cytomegalovirus infection after allogeneic haematopoietic stem cell transplantation.In this article, the role of the newly approved antiviral compound, letermovir, is reviewed.RECENT FINDINGS: Letermovir inhibits CMV by interfering viral terminase complex. In a phase 3 randomized placebo-controlled clinical study that enrolled 495 CMV-seropositive HSCT recipients, the primary end point of clinically significant CMV infection was significantly reduced by letermovir prophylaxis.SUMMARY: Letermovir is an important addition to the current strategies for CMV prevention after allogeneic HSCT. Letermovir is a human cytomegalovirus (CMV) terminase inhibitor recently approved as prophylaxis in stem cell transplant recipients.Letermovir is a novel antiviral in clinical development for prophylaxis against human cytomegalovirus in immunocompromised transplant recipients.Expert commentary: With the introduction of letermovir, prevention of CMV infection in allo-HCT recipients may shift considerably, from a predominantly preemptive strategy to one that utilizes this novel therapy for prophylaxis.The positive results from the recently published Phase III study of letermovir for prevention of CMV infection in CMV-seropositive allo-HCT recipients led to its approval as a prophylactic agent for CMV in multiple countries.Letermovir has shown promising clinical efficacy and is generally well tolerated, thus providing a favorable new option in the prophylaxis of CMV infection and disease.\"}, {\"question\": \"What are the roles of LEM-3?\", \"ideal_answer\": [\"LEM-3 is a midbody-tethered DNA nuclease that resolves chromatin bridges during late mitosis. The conserved LEM-3/Ankle1 nuclease is involved in the combinatorial regulation of meiotic recombination repair and chromosome segregation in Caenorhabditis elegans. LEM-3 is able to process erroneous recombination intermediates that persist into the second meiotic division.\"], \"exact_answer\": [[\"LEM-3 resolves chromatin bridges during late mitosis\"], [\"LEM-3 is involved in the combinatorial regulation of meiotic recombination repair and chromosome segregation in Caenorhabditis elegans\"], [\"LEM-3 is able to process erroneous recombination intermediates that persist into the second meiotic division.\"]], \"context\": \"LEM-3 is a midbody-tethered DNA nuclease that resolves chromatin bridges during late mitosis.Using C. elegans embryos we show that the LEM-3/Ankle1 nuclease defines a previously undescribed genome integrity mechanism by processing DNA bridges right before cells divide. LEM-3 acts at the midbody, the structure where abscission occurs at the end of cytokinesis. LEM-3 localization depends on factors needed for midbody assembly, and LEM-3 accumulation is increased and prolonged when chromatin bridges are trapped at the cleavage plane. LEM-3 locally processes chromatin bridges that arise from incomplete DNA replication, unresolved recombination intermediates, or the perturbance of chromosome structure. Proper LEM-3 midbody localization and function is regulated by AIR-2/Aurora B kinase. Strikingly, LEM-3 acts cooperatively with the BRC-1/BRCA1 homologous recombination factor to promote genome integrity. These findings provide a molecular basis for the suspected role of the LEM-3 orthologue Ankle1 in human breast cancer.The conserved LEM-3/Ankle1 nuclease is involved in the combinatorial regulation of meiotic recombination repair and chromosome segregation in Caenorhabditis elegans.Here we show that the LEM-3 nuclease, mutation of which by itself does not have an overt meiotic phenotype, genetically interacts with slx-1 and mus-81 mutants, the respective double mutants displaying 100% embryonic lethality. The combined loss of LEM-3 and MUS-81 leads to altered processing of recombination intermediates, a delayed disassembly of foci associated with CO designated sites, and the formation of univalents linked by SPO-11 dependent chromatin bridges (dissociated bivalents). However, LEM-3 foci do not colocalize with ZHP-3, a marker that congresses into CO designated sites. In addition, neither CO frequency nor distribution is altered in lem-3 single mutants or in combination with mus-81 or slx-4 mutations. Finally, we found persistent chromatin bridges during meiotic divisions in lem-3; slx-4 double mutants. Supported by the localization of LEM-3 between dividing meiotic nuclei, this data suggest that LEM-3 is able to process erroneous recombination intermediates that persist into the second meiotic division.LEM-3 - A LEM domain containing nuclease involved in the DNA damage response in C. elegans.Strikingly, LEM-3 acts cooperatively with the BRC-1/BRCA1 homologous recombination factor to promote genome integrity.LEM-3 locally processes chromatin bridges that arise from incomplete DNA replication, unresolved recombination intermediates, or the perturbance of chromosome structure.Using C. elegans embryos we show that the LEM-3/Ankle1 nuclease defines a previously undescribed genome integrity mechanism by processing DNA bridges right before cells divide.LEM-3 acts at the midbody, the structure where abscission occurs at the end of cytokinesis.\"}, {\"question\": \"List uniparental disomy (UPD) detection algorithms\", \"ideal_answer\": [\"UPDtool and AsCNAR are tools for detecting uniparental disomy (UPD). UPDtool is a computational tool for detection and classification of uniparental disomy (UPD) in trio SNP-microarray experiments. AsCNAR (allele-specific copy-number analysis using anonymous references) detects the copy-number neutral LOH, or uniparental disomy (UPD), in a large number of acute leukemia samples.\"], \"exact_answer\": [[\"UPDtool\"], [\"AsCNAR (allele-specific copy-number analysis using anonymous references)\"]], \"context\": \"UPDtool: a tool for detection of iso- and heterodisomy in parent-child trios using SNP microarrays.UPDtool is a computational tool for detection and classification of uniparental disomy (UPD) in trio SNP-microarray experiments.The performance of the new algorithm, called \\\"AsCNAR\\\" (allele-specific copy-number analysis using anonymous references), was demonstrated by detecting the copy-number neutral LOH, or uniparental disomy (UPD), in a large number of acute leukemia samples. We next applied this technique to detection of UPD involving the 9p arm in myeloproliferative disorders (MPDs), which is tightly associated with a homozygous JAK2 mutation. It revealed an unexpectedly high frequency of 9p UPD that otherwise would have been undetected and also disclosed the existence of multiple subpopulations having distinct 9p UPD within the same MPD specimen. In conclusion, AsCNAR should substantially improve our ability to dissect the complexity of cancer genomes and should contribute to our understanding of the genetic basis of human cancers.Our results were compared with both the gold standard, microsatellite analysis and SNPtrio, another program available for UPD detection.Our results were compared with both the gold standard, microsatellite analysis and SNPtrio, another program available for UPD detection.\"}, {\"question\": \"List major features of TEMPI Syndrome.\", \"ideal_answer\": [\"TEMPI syndrome includes telangiectasias, erythrocytosis with elevated erythropoietin, monoclonal gammopathy, perinephric fluid collections, intrapulmonary shunting. It is a newly described clinical entity that is generally considered a plasma cell dyscrasia with multiple system involvement.\"], \"exact_answer\": [[\"telangiectasias\"], [\"erythrocytosis with elevated erythropoietin\"], [\"monoclonal gammopathy\"], [\"perinephric fluid collections\"], [\"intrapulmonary shunting\"]], \"context\": \"TEMPI Syndrome: Erythrocytosis in Plasma Cell Dyscrasia.TEMPI (telangiectasias, erythrocytosis with elevated erythropoietin, monoclonal gammopathy, perinephric fluid collections, intrapulmonary shunting) syndrome is a newly described clinical entity that is generally considered a plasma cell dyscrasia with multiple system involvement. In other MGCS, such as scleromyxedema, Clarkson syndrome, TEMPI syndrome, cutis laxa and the neutrophilic dermatoses, the link between the monoclocal component and the entity is clearly established, but not understood so far.TEMPI syndrome (telangiectasias, elevated erythropoietin level and erythrocytosis, monoclonal gammopathy, perinephric fluid collections, and intrapulmonary shunting) is a recently described syndrome that, owing to erythrocytosis, may be confused with polycythemia vera.In conclusion, TEMPI syndrome should be considered when erythrocytosis and plasma cell dyscrasia coexist.BACKGROUND\\nThe TEMPI syndrome was recently described in 2011, and is characterized by the constellation of five hallmarks: Telangiectasias, Erythrocytosis and elevated Erythropoietin, Monoclonal gammopathy, Perinephric fluids collections, and Intrapulmonary shunting.TEMPI syndrome is a recently described condition defined by teleangiectasias, elevated erythropoietin and erythrocytosis, monoclonal gammopathy, perinephric fluid collections, and intrapulmonary shunting.BACKGROUND The TEMPI syndrome was recently described in 2011, and is characterized by the constellation of five hallmarks: Telangiectasias, Erythrocytosis and elevated Erythropoietin, Monoclonal gammopathy, Perinephric fluids collections, and Intrapulmonary shunting.\"}, {\"question\": \"List the releases of tmVar\", \"ideal_answer\": [\"TmVar is a text mining approach for extracting sequence variants in biomedical literature. TmVar 2.0 integrates genomic variant information from literature with dbSNP and ClinVar for precision medicine.\"], \"exact_answer\": [[\"TmVar\"], [\"TmVar 2.0\"]], \"context\": \"Here, we report tmVar, a text-mining approach based on conditional random field (CRF) for extracting a wide range of sequence variants described at protein, DNA and RNA levels according to a standard nomenclature developed by the Human Genome Variation Society. By doing so, we cover several important types of mutations that were not considered in past studies. Using a novel CRF label model and feature set, our method achieves higher performance than a state-of-the-art method on both our corpus (91.4 versus 78.1% in F-measure) and their own gold standard (93.9 versus 89.4% in F-measure). These results suggest that tmVar is a high-performance method for mutation extraction from biomedical literature.tmVar 2.0: integrating genomic variant information from literature with dbSNP and ClinVar for precision medicine.tmVar: a text mining approach for extracting sequence variants in biomedical literature.To our knowledge, this is the first large-scale study to analyze and integrate text-mined variant data with curated knowledge in existing databases. Our results suggest that databases can be significantly enriched by text mining and that the combined information can greatly assist human efforts in evaluating/prioritizing variants in genomic research.Availability and implementation: The tmVar 2.0 source code and corpus are freely available at https://www.ncbi.nlm.nih.gov/research/bionlp/Tools/tmvar/RESULTS\\nHere, we report tmVar, a text-mining approach based on conditional random field (CRF) for extracting a wide range of sequence variants described at protein, DNA and RNA levels according to a standard nomenclature developed by the Human Genome Variation Society.These results suggest that tmVar is a high-performance method for mutation extraction from biomedical literature.AVAILABILITY\\ntmVar software and its corpus of 500 manually curated abstracts are available for download at http://www.ncbi.nlm.nih.gov/CBBresearch/Lu/pub/tmVarAvailability and implementation\\nThe tmVar 2.0 source code and corpus are freely available at https://www.ncbi.nlm.nih.gov/research/bionlp/Tools/tmvar/.RESULTS Here, we report tmVar, a text-mining approach based on conditional random field (CRF) for extracting a wide range of sequence variants described at protein, DNA and RNA levels according to a standard nomenclature developed by the Human Genome Variation Society.AVAILABILITY tmVar software and its corpus of 500 manually curated abstracts are available for download at http://www.ncbi.nlm.nih.gov/CBBresearch/Lu/pub/tmVarAvailability and implementation The tmVar 2.0 source code and corpus are freely available at https://www.ncbi.nlm.nih.gov/research/bionlp/Tools/tmvar/.As such, new automatic approaches are greatly needed for extracting different kinds of mutations with high accuracy.<br><b>RESULTS</b>: Here, we report tmVar, a text-mining approach based on conditional random field (CRF) for extracting a wide range of sequence variants described at protein, DNA and RNA levels according to a standard nomenclature developed by the Human Genome Variation Society.These results suggest that tmVar is a high-performance method for mutation extraction from biomedical literature.<br><b>AVAILABILITY</b>: tmVar software and its corpus of 500 manually curated abstracts are available for download at http://www.ncbi.nlm.nih.gov/CBBresearch/Lu/pub/tmVar<br>Our results suggest that databases can be significantly enriched by text mining and that the combined information can greatly assist human efforts in evaluating/prioritizing variants in genomic research.<br><b>Availability and implementation</b>: The tmVar 2.0 source code and corpus are freely available at https://www.ncbi.nlm.nih.gov/research/bionlp/Tools/tmvar/.<br><b>Contact</b>: zhiyong.lu@nih.gov.<br>tmVar software and its corpus of 500 manually curated abstracts are available for download at http://www.ncbi.nlm.nih.gov/CBBresearch/Lu/pub/tmVarThe tmVar 2.0 source code and corpus are freely available at https://www.ncbi.nlm.nih.gov/research/bionlp/Tools/tmvar/.These results suggest that tmVar is a high-performance method for mutation extraction from biomedical literature.\"}, {\"question\": \"What 2 biological processes are regulated by STAMP2 in adipocytes?\", \"ideal_answer\": [\"Inflammation, insulin resistance and metabolic response are regulated by STAMP2 in adipocytes.\", \"six-transmembrane protein stamp2 as a critical modulator of this integrated response system of inflammation and metabolism\"], \"exact_answer\": [[\"metabolic response\"], [\"inflammation\"], [\"insulin resistance\"]], \"context\": \"Recently, we found that Stamp2 has a critical role in the integration of inflammatory and metabolic signals in adipose tissue where it is highly expressed and regulated by nutritional and metabolic cues.we identified the six-transmembrane protein STAMP2 as a critical modulator of this integrated response system of inflammation and metabolism in adipocytes. Six transmembrane protein of prostate 2 (STAMP2) is a counterregulator of adipose inflammation and insulin resistance in mice.Recently, we found that Stamp2 has a critical role in the integration of inflammatory and metabolic signals in adipose tissue where it is highly expressed and regulated by nutritional and metabolic cues. These results suggest that Stamp1 and Stamp2 play critical roles in adipogenesis, but through different mechanisms.CONTEXT\\nSix transmembrane protein of prostate 2 (STAMP2) is a counterregulator of adipose inflammation and insulin resistance in mice.These results suggest that STAMP2 is crucial for normal ASC conversion into adipocytes and their metabolic function, as well as their ability to facilitate PCa growth in vivo.We report that Stamp2 is expressed in human and mouse macrophages, is regulated upon differentiation or activation, acts as an anti-inflammatory protein, and regulates foam cell formation.CONTEXT Six-transmembrane protein of prostate 2 (STAMP2) is a counter-regulator of inflammation and insulin resistance according to findings in mice.Taken together, TIARP/STAMP2 is highly upregulated in 3T3-L1 cells and hMSC-Ad by IL-1beta and might, therefore, modulate proinflammatory and insulin resistance-inducing effects of IL-1beta.<br><b>CONTEXT</b>: Six transmembrane protein of prostate 2 (STAMP2) is a counterregulator of adipose inflammation and insulin resistance in mice.Our hypothesis was that STAMP2 could be involved in human obesity and insulin resistance.<br><b>OBJECTIVE</b>: The objective of the study was to elucidate the role of adipose STAMP2 expression in human obesity and insulin resistance.<br><b>DESIGN</b>: The design was to quantify STAMP2 in human abdominal sc and omental white adipose tissue (WAT), isolated adipocytes, and stroma and in vitro differentiated preadipocytes and relate levels of STAMP2 in sc WAT to clinical and adipocyte phenotypes involved in insulin resistance.<br><b>PARTICIPANTS</b>: Nonobese and obese women and men (n = 236) recruited from an obesity clinic or through local advertisement.<br><b>MAIN OUTCOME MEASUREMENT</b>: Clinical measures included body mass index, body fat, total adiponectin, and homeostasis model assessment as measure of overall insulin resistance.Taken together, TIARP/STAMP2 is highly upregulated in 3T3-L1 cells and hMSC-Ad by IL-1beta and might, therefore, modulate proinflammatory and insulin resistance-inducing effects of IL-1beta.\"}, {\"question\": \"Which methods have been developed for extracting sequence variants from the literature?\", \"ideal_answer\": [\"TmVar and nala\"], \"exact_answer\": [[\"tmVar\"], [\"nala\"]], \"context\": \"tmVar: a text mining approach for extracting sequence variants in biomedical literature.Here, we report tmVar, a text-mining approach based on conditional random field (CRF) for extracting a wide range of sequence variants described at protein, DNA and RNA levels according to a standard nomenclature developed by the Human Genome Variation Society. By doing so, we cover several important types of mutations that were not considered in past studies. Using a novel CRF label model and feature set, our method achieves higher performance than a state-of-the-art method on both our corpus (91.4 versus 78.1% in F-measure) and their own gold standard (93.9 versus 89.4% in F-measure). These results suggest that tmVar is a high-performance method for mutation extraction from biomedical literature.nala: text mining natural language mutation mentions. The extraction of sequence variants from the literature remains an important task. Existing methods primarily target standard (ST) mutation mentions (e.g. 'E6V'), leaving relevant mentions natural language (NL) largely untapped (e.g. 'glutamic acid was substituted by valine at residue 6').Results: We introduced three new corpora suggesting named-entity recognition (NER) to be more challenging than anticipated: 28-77% of all articles contained mentions only available in NL. Our new method nala captured NL and ST by combining conditional random fields with word embedding features learned unsupervised from the entire PubMed. In our hands, nala substantially outperformed the state-of-the-art.RESULTS\\nHere, we report tmVar, a text-mining approach based on conditional random field (CRF) for extracting a wide range of sequence variants described at protein, DNA and RNA levels according to a standard nomenclature developed by the Human Genome Variation Society.These results suggest that tmVar is a high-performance method for mutation extraction from biomedical literature.RESULTS Here, we report tmVar, a text-mining approach based on conditional random field (CRF) for extracting a wide range of sequence variants described at protein, DNA and RNA levels according to a standard nomenclature developed by the Human Genome Variation Society.As such, new automatic approaches are greatly needed for extracting different kinds of mutations with high accuracy.<br><b>RESULTS</b>: Here, we report tmVar, a text-mining approach based on conditional random field (CRF) for extracting a wide range of sequence variants described at protein, DNA and RNA levels according to a standard nomenclature developed by the Human Genome Variation Society.These results suggest that tmVar is a high-performance method for mutation extraction from biomedical literature.<br><b>AVAILABILITY</b>: tmVar software and its corpus of 500 manually curated abstracts are available for download at http://www.ncbi.nlm.nih.gov/CBBresearch/Lu/pub/tmVar<br>These results suggest that tmVar is a high-performance method for mutation extraction from biomedical literature.Here, we report tmVar, a text-mining approach based on conditional random field (CRF) for extracting a wide range of sequence variants described at protein, DNA and RNA levels according to a standard nomenclature developed by the Human Genome Variation Society.\"}, {\"question\": \"Which molecules are targeted by defactinib?\", \"ideal_answer\": [\"PURPOSE: VS-6063 (also known as defactinib or PF-04554878) is a second-generation inhibitor of focal adhesion kinase and proline-rich tyrosine kinase-2.\"], \"exact_answer\": [[\"focal adhesion kinase\"], [\"proline-rich tyrosine kinase-2\"]], \"context\": \"LC-ESI-MS/MS determination of defactinib, a novel FAK inhibitor in mice plasma and its application to a pharmacokinetic study in mice.Effect of FAK inhibitor VS-6063 (defactinib) on docetaxel efficacy in prostate cancer.Focal Adhesion Kinase (FAK) activation is a mediator of docetaxel resistance in prostate cancer cells. The aim of this study was to investigate the effect of the second generation FAK inhibitor VS-6063 on docetaxel efficacy in pre-clinical CRPC and CSPC models.Here, we report the development of a selective and potent Fak degrader, PROTAC-3, which outperforms a clinical candidate, defactinib, with respect to Fak activation as well as Fak-mediated cell migration and invasion. Prompted by our observation of a correlation between PTEN loss and FAK phosphorylation in a cohort of patients with stage IV SCC, we evaluated the relevance of PTEN loss in cancer progression as well as the efficacy of a new combined treatment with the pan PI3K inhibitor buparlisip and the FAK inhibitor defactinib.PURPOSE: VS-6063 (also known as defactinib or PF-04554878) is a second-generation inhibitor of focal adhesion kinase and proline-rich tyrosine kinase-2. OBJECTIVE: VS-6063 (also known as defactinib or PF-04554878) is a second-generation inhibitor of focal adhesion kinase (FAK) and proline-rich tyrosine kinase-2 (Pyk2). Accordingly, treatment of C4-2B4 cells with integrin ligands resulted in increased pFAK-Y397 expression and cell survival, whereas targeting integrins with FAK inhibitors PF-562271 or defactinib inhibited FAK phosphorylation and reduced the survival of PC3-mm2 cells. OBJECTIVE\\nVS-6063 (also known as defactinib or PF-04554878) is a second-generation inhibitor of focal adhesion kinase (FAK) and proline-rich tyrosine kinase-2 (Pyk2).PURPOSE\\nVS-6063 (also known as defactinib or PF-04554878) is a second-generation inhibitor of focal adhesion kinase and proline-rich tyrosine kinase-2.OBJECTIVE VS-6063 (also known as defactinib or PF-04554878) is a second-generation inhibitor of focal adhesion kinase (FAK) and proline-rich tyrosine kinase-2 (Pyk2).PURPOSE VS-6063 (also known as defactinib or PF-04554878) is a second-generation inhibitor of focal adhesion kinase and proline-rich tyrosine kinase-2.Accordingly, treatment of C4-2B4 cells with integrin ligands resulted in increased pFAK-Y397 expression and cell survival, whereas targeting integrins with FAK inhibitors PF-562271 or defactinib inhibited FAK phosphorylation and reduced the survival of PC3-mm2 cells.Accordingly, treatment of C4-2B4 cells with integrin ligands resulted in increased pFAK-Y397 expression and cell survival, whereas targeting integrins with FAK inhibitors PF-562271 or defactinib inhibited FAK phosphorylation and reduced the survival of PC3-mm2 cells.\"}, {\"question\": \"Please list the 4 genes involved in Sanfilippo syndrome, also known as mucopolysaccharidosis III (MPS-III).\", \"ideal_answer\": [\"Mucopolysaccharidosis type III (MPS III, Sanfilippo syndrome) is a lysosomal storage disorder, caused by a deficiency in one of the four enzymes involved in the catabolism of glycosaminoglycan heparan sulfate. The genes are SGSH, NAGLU, HGSNAT or GNS.\"], \"exact_answer\": [[\"SGSH\"], [\"NAGLU\"], [\"HGSNAT\"], [\"GNS\"]], \"context\": \"This disease is a complex of four conditions caused by dysfunctions of one of genes coding for lysosomal enzymes involved in degradation of heparan sulfate: SGSH (coding for heparan N-sulfatase) - causing MPS IIIA, NAGLU (coding for alpha-N-acetylglucosaminidase) - causing MPS IIIB, HGSNAT (coding for acetyl CoA alpha-glucosaminide acetyltransferase) - causing MPS IIIC), and GNS (coding for N-acetylglucosamine-6-sulfatase) - causing MPS IIID.MPS III results from a deficiency in one of the four enzymes involved in the heparan sulfate degradation, with sulfamidase (SGSH), \\u03b1-N-acetylglucosaminidase (NAGLU), acetyl-coenzyme A: \\u03b1-glucosaminide N-acetyltransferase (HGSNAT), and N-acetylglucosamine-6-sulfatase (GNS) being deficient respectively in MPS IIIA, MPS IIIB, MPS IIIC and MPS IIID. \"}, {\"question\": \"List clinical disorders or diseases where uc.189 is involved?\", \"ideal_answer\": [\"Univariate and multivariate Cox regression analysis demonstrated that over-expression of uc.189 predicted poor prognosis in Cervical squamous cell carcinomas (CSCC) and Endometrial adenocarcinomas (EAC). Thus, several findings suggested uc.189 might be an evaluating prognosis marker of gynecological tumors. In addition, high expression of uc.189 might reflect poor prognosis of Esophageal squamous cell carcinoma (ESCC) and indicate a potential diagnostic target in ESCC patients. Uc.189 might be considered as a novel molecule involved in ESCC progression, which provides a potential prognostic biomarker and therapeutic target.\"], \"exact_answer\": [[\"Esophageal squamous cell carcinoma\", \"ESCC\"], [\"Cervical squamous cell carcinomas\", \"CSCCs\", \"gynecological tumors\"], [\"Endometrial adenocarcinomas\", \"EACs\", \"gynecological tumors\"]], \"context\": \"Expression of uc.189 and its clinicopathologic significance in gynecological cancers.Until now, the role of uc.189 in human cancers remains undefined and the clinical significance of uc.189 in gynecological cancers remains unknown. This study was to identify the prognostic value of uc.189 expression in gynecological cancers.Tissue microarrays were constructed with 243 samples including 116 cervical squamous cell carcinomas (CSCCs), 98 endometrial adenocarcinomas (EACs), 29 ovarian cystoadenocarcinomas(OCAs), and corresponding normal tissues. In CSCC, uc.189 expression was increased in 78.5% of cases (91/116), decreased in 4.3% (5/116), and unchanged in 17.2% (20/116). In EAC its expression was increased in 74.5% (73/98), decreased in 3.1% (3/98), and unchanged in 22.4% (22/98). Expression of uc.189 was increased in 23, and unchanged/decreased in 6, of 29 cases of ovarian cystoadenocarcinomas. Univariate and multivariate Cox regression analysis demonstrated that over-expression of uc.189 predicted poor prognosis in CSCC and EAC. Thus, these findings suggested uc.189 might be an evaluating prognosis marker of gynecological tumors.Upregulation of uc.189 in patients with esophageal squamous cell carcinoma and its clinicopathologic value.Ultraconserved elements (UCEs) encoding noncoding RNAs serve as important regulators in cancer biology. Until now, the role of the UCE uc.189 in human cancers remains undefined and the clinical significance of uc.189 in esophageal cancers remains unknown. This study was to identify the prognostic value of uc.189 expression in esophageal squamous cell carcinomas (ESCC). Quantitative real-time polymerase chain reaction (qRT-PCR) was used to detect the expression level of uc.189 in matched cancerous tissues and adjacent noncancerous tissues from 152 patients with ESCC. The correlation of uc.189 with clinicopathological features and prognosis were also analyzed. The expression of uc.189 was significantly higher in human ESCC compared with the adjacent noncancerous tissues (122/152, 80.3%, p<0.01), and the high level of uc.189 expression was significantly correlated with invasion of the tumor (p=0.009), advanced clinical stage (p=0.000), lymph node metastasis (p=0.000), and poor prognosis. High expression of uc.189 might reflect poor prognosis of ESCC and indicate a potential diagnostic target in ESCC patients. Uc.189 might be considered as a novel molecule involved in ESCC progression, which provides a potential prognostic biomarker and therapeutic target.Uc.189 might be considered as a novel molecule involved in ESCC progression, which provides a potential prognostic biomarker and therapeutic target.The expression of uc.189 was significantly higher in human ESCC compared with the adjacent noncancerous tissues (122/152, 80.3%, p<0.01), and the high level of uc.189 expression was significantly correlated with invasion of the tumor (p=0.009), advanced clinical stage (p=0.000), lymph node metastasis (p=0.000), and poor prognosis.Until now, the role of the UCE uc.189 in human cancers remains undefined and the clinical significance of uc.189 in esophageal cancers remains unknown.High expression of uc.189 might reflect poor prognosis of ESCC and indicate a potential diagnostic target in ESCC patients.This study was to identify the prognostic value of uc.189 expression in esophageal squamous cell carcinomas (ESCC).Until now, the role of uc.189 in human cancers remains undefined and the clinical significance of uc.189 in gynecological cancers remains unknown.Thus, these findings suggested uc.189 might be an evaluating prognosis marker of gynecological tumors.Univariate and multivariate Cox regression analysis demonstrated that over-expression of uc.189 predicted poor prognosis in CSCC and EAC.This study was to identify the prognostic value of uc.189 expression in gynecological cancers.In CSCC, uc.189 expression was increased in 78.5% of cases (91/116), decreased in 4.3% (5/116), and unchanged in 17.2% (20/116).Uc.189 might be considered as a novel molecule involved in ESCC progression, which provides a potential prognostic biomarker and therapeutic target.The expression of uc.189 was significantly higher in human ESCC compared with the adjacent noncancerous tissues (122/152, 80.3%, p&lt;0.01), and the high level of uc.189 expression was significantly correlated with invasion of the tumor (p=0.009), advanced clinical stage (p=0.000), lymph node metastasis (p=0.000), and poor prognosis.High expression of uc.189 might reflect poor prognosis of ESCC and indicate a potential diagnostic target in ESCC patients.Expression of uc.189 was increased in 23, and unchanged/decreased in 6, of 29 cases of ovarian cystoadenocarcinomas.\"}, {\"question\": \"What is the triad of Melkersson-Rosenthal syndrome?\", \"ideal_answer\": [\"Melkersson-Rosenthal syndrome is an uncommon granulomatous disease characterized by the triad of relapsing facial paralysis, orofacial edema and fissured tongue.\"], \"exact_answer\": [[\"relapsing facial paralysis\"], [\"orofacial edema\"], [\"fissured tongue\"]], \"context\": \"Melkersson-Rosenthal syndrome (MRS) is a rare syndrome of facial nerve palsy, facial edema, and lingua plicata that can be difficult to treat.BACKGROUND AND OBJECTIVE: Melkersson Rosenthal syndrome (MRS) is a rare disorder of unknown etiology and comprises the triad: orofacial edema, recurrent facial paralysis and lingua plicata. Melkersson Rosenthal syndromes (MRS) is a rare autosomal dominantly inherited neurocutaneous syndrome characterised by a triad of facial (seventh cranial) nerve palsy, recurrent orofacial swelling and fissuring of the tongue. Melkersson-Rosenthal Syndrome (MRS) is a rare disorder consisting of a triad of persistent or recurrent orofacial edema, relapsing facial paralysis and fissured tongue. When it presents in a triad encompassing facial nerve palsy, lip swelling, and fissured or furrowed tongue it is called Melkersson-Rosenthal syndrome while monosymptomatic or oligosymptomatic forms are referred to as granulomatous cheilitis.Melkersson-Rosenthal syndrome (MRS) is an uncommon granulomatous disease characterized by the triad of relapsing facial paralysis, orofacial swelling, and fissured tongue. Melkersson Rosenthal Syndrome is a rare neuro-mucocutaneous disorder characterized by the classic triad of facial swelling, recurrent facial nerve palsy and fissured tongue. Melkersson-Rosenthal syndrome: review of the literature and case report of a 10-year misdiagnosis.Melkersson-Rosenthal syndrome is classically described as a triad of orofacial swelling, facial palsy, and fissured tongue. The Melkersson-Rosenthal syndrome: a retrospective study of biopsied cases.Melkersson-Rosenthal syndrome (MRS) is a rare neuromucocutaneous syndrome marked by the triad of recurrent nonpitting orofacial edema, fissured dorsal tongue (lingua plicata), and lower motoneuron facial paralysis. Melkersson-Rosenthal syndrome in the periocular area: a review of the literature and case report.A triad of facial palsy, facial edema, and furrowed tongue characterizes Melkersson-Rosenthal syndrome, a rare, noncaseating granulomatous disease of unknown cause. Melkersson-Rosenthal syndrome: report of three cases.Melkersson-Rosenthal syndrome is a rare disorder consisting of the triad of persistent or recurrent orofacial edema, relapsing facial paralysis and fissured tongue. Melkersson-Rosenthal syndrome is characterised by a triad of recurrent orofacial swelling, relapsing facial paralysis, and fissured tongue.Melkersson-Rosenthal syndrome (MRS) is a rare neuromucocutaneous syndrome marked by the triad of recurrent nonpitting orofacial edema, fissured dorsal tongue (lingua plicata), and lower motoneuron facial paralysis.The patient has a typical for Melkersson-Rosenthal syndrome triad of symptoms: bilateral facial nerve palsy, lingua plicata and facial oedema.RECENT FINDINGS\\nThe Melkersson-Rosenthal syndrome consists of a triad of recurrent lip and/or face swelling, fissured tongue, and intermittent facial palsy.The Melkersson-Rosenthal syndrome is a rare disorder of unknown etiology characterized by a triad of recurrent orofacial swelling, relapsing facial paralysis, and fissured tongue.Melkersson-Rosenthal syndrome is classically described as a triad of orofacial swelling, facial palsy, and fissured tongue.A triad of facial palsy, orofacial edema, and furrowed tongue constitutes an uncommon condition known as Melkersson-Rosenthal syndrome (MRS).Melkersson-Rosenthal syndrome is an uncommon disorder characterized by a triad of facial nerve palsy, orofacial edema, and fissured tongue.Melkersson-Rosenthal syndrome is a rare disorder consisting of the triad of persistent or recurrent orofacial edema, relapsing facial paralysis and fissured tongue.Melkersson-Rosenthal syndrome (MRS) is characterized by the triad of recurrent facial palsy, lingua plicata, and facial edema.Melkersson-Rosenthal syndrome (MRS) is a rare granulomatous inflammatory disease characterised by the triad of orofacial oedema, facial nerve palsy and furrowed tongue.Melkersson Rosenthal Syndrome is a rare neuro-mucocutaneous disorder characterized by the classic triad of facial swelling, recurrent facial nerve palsy and fissured tongue.The Melkersson-Rosenthal syndrome (MRS) is a rare condition characterized by the triad of familial relapsing peripheral facial palsy, facial edema, and lingua plicata.RECENT FINDINGS The Melkersson-Rosenthal syndrome consists of a triad of recurrent lip and/or face swelling, fissured tongue, and intermittent facial palsy.Melkersson-Rosenthal syndrome is an uncommon disorder of uncertain aetiology characterized by orofacial oedema, facial nerve palsy and lingua plicata.Melkersson-Rosenthal syndrome is a rare condition, classically associated with a triad of facial and/or lip edema, fissured tongue, and relapsing facial palsy.The Melkersson-Rosenthal syndrome consists of triad of symptoms: recurrent oedema of lips, recurrent facial nerve paralysis and lingua plicata.\"}, {\"question\": \"Name the algorithms for counting multi-mapping reads\", \"ideal_answer\": [\"RNA-Seq is currently used routinely, and it provides accurate information on gene transcription. However, the method cannot accurately estimate duplicated genes expression. Several strategies have been previously used (drop duplicated genes, distribute uniformly the reads, or estimate expression), but all of them provide biased results. Mmquant is a tool for computing gene expression, including duplicated genes. If a read maps at different positions, the tool detects that the corresponding genes are duplicated; it merges the genes and creates a merged gene. The counts of ambiguous reads is then based on the input genes and the merged genes. Other methods have been developed that use weighted allocation of read counts but these methods treat the different types of multi-reads equivalently. For instance a hierarchical approach was developed for allocation of read counts that first resolves ambiguities among genes, then among isoforms, and lastly between alleles. The model has been implemented in EMASE software (Expectation-Maximization for Allele Specific Expression) to estimate total gene expression, isoform usage and ASE based on this hierarchical allocation.\"], \"exact_answer\": [[\"mmquant\"], [\"EMASE\", \"Expectation-Maximization for Allele Specific Expression\"]], \"context\": \"mmquant: how to count multi-mapping reads?RNA-Seq is currently used routinely, and it provides accurate information on gene transcription. However, the method cannot accurately estimate duplicated genes expression. Several strategies have been previously used (drop duplicated genes, distribute uniformly the reads, or estimate expression), but all of them provide biased results.RESULTS: We provide here a tool, called mmquant, for computing gene expression, included duplicated genes. If a read maps at different positions, the tool detects that the corresponding genes are duplicated; it merges the genes and creates a merged gene. The counts of ambiguous reads is then based on the input genes and the merged genes.CONCLUSION: mmquant is a drop-in replacement of the widely used tools htseq-count and featureCounts that handles multi-mapping reads in an unabiased way.Allele-specific expression (ASE) refers to the differential abundance of the allelic copies of a transcript. RNA sequencing (RNA-seq) can provide quantitative estimates of ASE for genes with transcribed polymorphisms. When short-read sequences are aligned to a diploid transcriptome, read-mapping ambiguities confound our ability to directly count reads. Multi-mapping reads aligning equally well to multiple genomic locations, isoforms or alleles can comprise the majority (>85%) of reads. Discarding them can result in biases and substantial loss of information. Methods have been developed that use weighted allocation of read counts but these methods treat the different types of multi-reads equivalently. We propose a hierarchical approach to allocation of read counts that first resolves ambiguities among genes, then among isoforms, and lastly between alleles. We have implemented our model in EMASE software (Expectation-Maximization for Allele Specific Expression) to estimate total gene expression, isoform usage and ASE based on this hierarchical allocation.mmquant: how to count multi-mapping reads?mmquant is a drop-in replacement of the widely used tools htseq-count and featureCounts that handles multi-mapping reads in an unabiased way.CONCLUSION\\nmmquant is a drop-in replacement of the widely used tools htseq-count and featureCounts that handles multi-mapping reads in an unabiased way.CONCLUSION mmquant is a drop-in replacement of the widely used tools htseq-count and featureCounts that handles multi-mapping reads in an unabiased way.\"}, {\"question\": \"Which integrin genes are activated by the immune system in inflammatory bowel disease?\", \"ideal_answer\": [\"ITGA4, ITGB8, ITGAL and ICAM1. In all four cases, the expression-increasing allele also increases disease risk.\"], \"exact_answer\": [[\"ITGA4\"], [\"ITGB8\"], [\"ITGAL\"], [\"ICAM1\"]], \"context\": \"Genetic association studies have identified 215 risk loci for inflammatory bowel disease, thereby uncovering fundamental aspects of its molecular biology. We performed a genome-wide association study of 25,305 individuals and conducted a meta-analysis with published summary statistics, yielding a total sample size of 59,957 subjects. We identified 25 new susceptibility loci, 3 of which contain integrin genes that encode proteins in pathways that have been identified as important therapeutic targets in inflammatory bowel disease. The associated variants are correlated with expression changes in response to immune stimulus at two of these genes (ITGA4 and ITGB8) and at previously implicated loci (ITGAL and ICAM1). In all four cases, the expression-increasing allele also increases disease risk. We also identified likely causal missense variants in a gene implicated in primary immune deficiency, PLCG2, and a negative regulator of inflammation, SLAMF8. Our results demonstrate that new associations at common variants continue to identify genes relevant to therapeutic target identification and prioritization.The associated variants are correlated with expression changes in response to immune stimulus at two of these genes (ITGA4 and ITGB8) and at previously implicated loci (ITGAL and ICAM1).\"}, {\"question\": \"Which drugs are included in the Lonsurf pill?\", \"ideal_answer\": [\"Lunsurf pill includes trifluridine and tipiracil. It is a novel form of chemotherapy for metastatic colorectal cancer.\"], \"exact_answer\": [[\"trifluridine\"], [\"tipiracil\"]], \"context\": \"The National Institute for Health and Care Excellence (NICE) invited Servier, the company manufacturing trifluridine and tipiracil (T/T; trade name: Lonsurf\\u00ae), to submit evidence for the clinical and cost effectiveness of T/T compared with best supportive care (BSC) for metastatic colorectal cancer (third-line or later). PURPOSE: Trifluridine/tipiracil (FTD/TPI; TAS-102, Lonsurf\\u00ae), a novel form of chemotherapy for metastatic colorectal cancer (mCRC), has shown clinical benefit in the global, phase III RECOURSE trial, regardless of patient age. BACKGROUND: Trifluridine/tipiracil (TAS-102, Lonsurf\\u00ae), a novel oral anti-tumor agent combining an anti-neoplastic thymidine-based nucleoside analogue (trifluridine, FTD) with a thymidine phosphorylase inhibitor (tipiracil hydrochloride, TPI) presents a new treatment option for metastatic colorectal cancer (mCRC) patients refractory or intolerant to standard therapies. These AEs may affect patient adherence, particularly with completely oral regimens, such as trifluridine/tipiracil (TAS-102, Lonsurf\\u00ae), an antimetabolite agent for patients with mCRC refractory or intolerant to standard therapies.\\u2029Trifluridine/tipiracil has recently been approved for the treatment of adult patients with metastatic colorectal cancer (mCRC) who are refractory to or are not considered candidates for, current standard chemotherapy and biological therapy in the EU and USA and in unresectable advanced or recurrent CRC in Japan. TAS-102 (Lonsurf) is an oral fluoropyrimidine that is formed by the combination of 2 active drugs: trifluridine (a nucleoside analog) and tipiracil hydrochloride (a thymidine phosphorylase inhibitor).Evolocumab (Repatha) for patients with hypercholesterolemia whose condition has not been controlled by statins and other therapies; trifluridine/tipiracil (Lonsurf) for metastatic colorectal cancer; and blood coagulation factor VIII (Nuwiq) for adults and children with hemophilia A.In May 2014, tablets containing both trifluridine and tipiracil hydrochloride (Lonsurf\\u00ae tablets) were launched in Japan ahead of other countries, for the treatment of advanced/relapsed unresectable colorectal cancer.Initial Evaluation of the Efficacy and Safety of Tablets Containing Trifluridine and Tipiracil Hydrochloride--Safety Measures Devised by a Multidisciplinary Team Including a Pharmaceutical Outpatient Clinic.\"}, {\"question\": \"Which enzymes are inhibited by Duvelisib?\", \"ideal_answer\": [\"Duvelisib is an oral dual inhibitor of phosphoinositide 3-kinase-\\u03b4 (PI3K-\\u03b4) and PI3K-\\u03b3 in late-stage clinical development for hematologic malignancy treatment.\"], \"exact_answer\": [[\"phosphoinositide 3-kinase-\\u03b4\"], [\"phosphoinositide 3-kinase-\\u03b3\"]], \"context\": \"Duvelisib (IPI-145) is an oral inhibitor of phosphatidylinositol 3-kinase (PI3K)-\\u03b4/\\u03b3 isoforms currently in clinical development. PI3K-\\u03b4/\\u03b3 inhibition may directly inhibit malignant T-cell growth, making duvelisib a promising candidate for patients with peripheral (PTCL) or cutaneous (CTCL) T-cell lymphoma.Activity of the PI3K-\\u03b4,\\u03b3 inhibitor duvelisib in a phase 1 trial and preclinical models of T-cell lymphoma.Duvelisib, a novel oral dual inhibitor of PI3K-\\u03b4,\\u03b3, is clinically active in advanced hematologic malignancies.Duvelisib is an oral dual inhibitor of phosphoinositide 3-kinase-\\u03b4 (PI3K-\\u03b4) and PI3K-\\u03b3 in late-stage clinical development for hematologic malignancy treatment.We observed that pharmaceutical PI3K\\u03b3 inhibition with CZC24832 significantly impaired CLL\\u00a0cell migration, while dual PI3K\\u03b4/\\u03b3 inhibitor duvelisib had a greater impact than single isoform-selective inhibitors. An oral dual inhibitor of PI3K\\u03b3 and PI3K\\u03b4, duvelisib, is in clinical trials for the treatment of lymphoid malignancies.Duvelisib, an oral dual PI3K-\\u03b4, \\u03b3 inhibitor, shows clinical activity in indolent non-Hodgkin lymphoma in a phase 1 study.Duvelisib (IPI-145) is an oral dual inhibitor of phosphoinositide-3-kinase (PI3K)-\\u03b4 and -\\u03b3 in clinical development for the treatment of hematologic malignancies, including indolent non-Hodgkin lymphoma (iNHL). Duvelisib, an oral dual PI3K-\\u03b4,\\u03b3 inhibitor, shows clinical and pharmacodynamic activity in chronic lymphocytic leukemia and small lymphocytic lymphoma in a phase 1 study.Duvelisib (IPI-145), an oral, dual inhibitor of phosphoinositide-3-kinase (PI3K)-\\u03b4 and -\\u03b3, was evaluated in a Phase 1 study in advanced hematologic malignancies, which included expansion cohorts in relapsed/refractory (RR) chronic lymphocytic leukemia (CLL)/small lymphocytic lymphoma (SLL) and treatment-na\\u00efve (TN) CLL.Expert opinion: Duvelisib targets the PI3K \\u03b4 isoform, which is necessary for cell proliferation and survival, and \\u03b3 isoform, which is critical for cytokine signaling and pro-inflammatory responses from the microenvironment. The functions of these individual catalytic enzymes have not been fully resolved in AML, therefore using the PI3K p110\\u03b4 and p110\\u03b3-targeted inhibitor IPI-145 (duvelisib) and specific p110\\u03b4 and p110\\u03b3 shRNA, we analysed the role of these two p110 subunits in human AML blast survival. The phosphoinositide-3 kinase (PI3K)-\\u03b4,\\u03b3 inhibitor, duvelisib shows preclinical synergy with multiple targeted therapies in hematologic malignancies.Duvelisib is an orally active dual inhibitor of PI3K-\\u03b4 and PI3K-\\u03b3 in clinical development in hematologic malignancies (HM).Duvelisib (IPI-145) is an oral dual inhibitor of phosphoinositide-3-kinase (PI3K)-\\u03b4 and -\\u03b3 in clinical development for the treatment of hematologic malignancies, including indolent non-Hodgkin lymphoma (iNHL).Duvelisib (IPI-145) is an oral inhibitor of phosphatidylinositol 3-kinase (PI3K)-\\u03b4/\\u03b3 isoforms currently in clinical development.PI3K-\\u03b4/\\u03b3 inhibition may directly inhibit malignant T-cell growth, making duvelisib a promising candidate for patients with peripheral (PTCL) or cutaneous (CTCL) T-cell lymphoma.The functions of these individual catalytic enzymes have not been fully resolved in AML, therefore using the PI3K p110\\u03b4 and p110\\u03b3-targeted inhibitor IPI-145 (duvelisib) and specific p110\\u03b4 and p110\\u03b3 shRNA, we analysed the role of these two p110 subunits in human AML blast survival.Duvelisib: a phosphoinositide-3 kinase \\u03b4/\\u03b3 inhibitor for chronic lymphocytic leukemia.Duvelisib, a PI3K \\u03b4/\\u03b3 dual isoform specific inhibitor that induces apoptosis and reduces cytokine and chemokine levels in vitro, holds promise for CLL.Duvelisib, an oral dual inhibitor of PI3K-\\u03b4 and PI3K-\\u03b3, is in phase III trials for the treatment of chronic lymphocytic leukemia (CLL) and indolent non-Hodgkin's lymphoma.Expert opinion: Duvelisib targets the PI3K \\u03b4 isoform, which is necessary for cell proliferation and survival, and \\u03b3 isoform, which is critical for cytokine signaling and pro-inflammatory responses from the microenvironment.The purely PI3K-\\u03b4-selective inhibitor idelalisib was compared to copanlisib (BAY 80-6946) and duvelisib (IPI-145), with isoform target profiles that additionally include PI3K-\\u03b1 or PI3K-\\u03b3, respectively.In this study, we evaluated how duvelisib affects the activity of the PI3K/Akt signaling pathway and if it has antitumor effects in EBV-associated lymphoma cell lines.\"}, {\"question\": \"List search engines used in proteomics.\", \"ideal_answer\": [\"Mascot\\nX!Tandem\\nMS-GF\\nMS Amanda \\nMyriMatch\\nComet\\nTide\\nAndromeda\\nOMSSA\"], \"exact_answer\": [[\"Mascot\"], [\"X!Tandem\"], [\"MS-GF\"], [\"MS Amanda\"], [\"MyriMatch\"], [\"Comet\"], [\"Tide\"], [\"Andromeda\"], [\"OMSSA\"]], \"context\": \"We compared the contribution of each of the eight search engines (X!Tandem, MS-GF[Formula: see text], MS Amanda, MyriMatch, Comet, Tide, Andromeda, and OMSSA) integrated in an open-source graphical user interface SearchGUI ( http://searchgui.googlecode.com ) into total result of proteoforms identification and optimized set of engines working simultaneously.  This software is also highly compatible with the identification and quantification results of various frequently used search engines, such as MaxQuant, Proteome Discoverer, or Mascot.  database search engines: Mascot, X!Tandem, and MS-GF+.applied an integrative approach based on the results obtained from Comet, Mascot, OMSSA, and X!Tandem.\"}, {\"question\": \"De novo mutations in which novel genes are involved in systemic lupus erythematosus?\", \"ideal_answer\": [\"DNMT3A, PRKCD, and C1QTNF4.\"], \"exact_answer\": [[\"DNMT3A\"], [\"PRKCD\"], [\"C1QTNF4\"]], \"context\": \"De novo mutations implicate novel genes in systemic lupus erythematosus. Rare variant associations, expected to explain far less of the heritability, may be enriched in core disease genes and thus will be instrumental in the understanding of complex disease pathogenesis and their potential therapeutic targets. Here, using complementary whole-exome sequencing, high-density imputation, and in vitro cellular assays, we identify candidate core genes in the pathogenesis of systemic lupus erythematosus (SLE). Using extreme-phenotype sampling, we sequenced the exomes of 30 SLE parent-affected-offspring trios and identified 14 genes with missense de novo mutations (DNM), none of which are within the\\u2009>80 SLE susceptibility loci implicated through genome-wide association studies. In a follow-up cohort of 10, 995 individuals of matched European ancestry, we imputed genotype data to the density of the combined UK10K-1000 genomes Phase III reference panel across the 14 candidate genes. Gene-level analyses indicate three functional candidates: DNMT3A, PRKCD, and C1QTNF4. We identify a burden of rare variants across PRKCD associated with SLE risk (P\\u2009=\\u20090.0028), and across DNMT3A associated with two severe disease prognosis sub-phenotypes (P\\u2009=\\u20090.0005 and P\\u2009=\\u20090.0033). We further characterise the TNF-dependent functions of the third candidate gene C1QTNF4 on NF-\\u03baB activation and apoptosis, which are inhibited by the p.His198Gln DNM. Our results identify three novel genes in SLE susceptibility and support extreme-phenotype sampling and DNM gene discovery to aid the search for core disease genes implicated through rare variation.We identify a burden of rare variants across PRKCD associated with SLE risk (P\\u2009=\\u20090.0028), and across DNMT3A associated with two severe disease prognosis sub-phenotypes (P\\u2009=\\u20090.0005 and P\\u2009=\\u20090.0033).\"}, {\"question\": \"List drugs included in the TRIUMEQ pill.\", \"ideal_answer\": [\"Triumeq is a single-tablet regimen for patients with HIV infection comprising dolutegravir, abacavir and lamivudine.\"], \"exact_answer\": [[\"dolutegravir\"], [\"abacavir\"], [\"lamivudine\"]], \"context\": \"A patient with human immunodeficiency virus-1 infection presented with sub-acute liver failure, temporally related to commencement of an antiretroviral therapy regimen containing dolutegravir (Triumeq).Newer STRs include dolutegravir (DTG)/lamivudine (3TC)/abacavir (ABC) (Triumeq; Viiv Healthcare, Research Triangle Park, NC), rilpivirine (RPV)/emtricitabine (FTC)/tenofovir alafenamide (TAF) (Odefsey; Gilead, Foster City, CA), RPV/FTC/tenofovir disoproxil fumarate (TDF) (Complera; Gilead), elvitegravir (EVG)/cobicistat (COBI)/FTC/TDF (Stribild; Gilead), and EVG/COBI/FTC/TAF (Genvoya; Gilead). Recently approved FDCs, such as atazanavir (ATV)/COBI (Evotaz; Bristol-Myers Squibb, Princeton, NJ), darunavir (DRV)/COBI (Prezcobix; Janssen Products, Titusville NJ), and FTC/TAF (Descovy; Gilead), are also now available. Intentional overdose of dolutegravir/abacavir/lamivudine (Triumeq) in a 26-year-old man.Triumeq is a single-tablet regimen for patients with HIV infection comprising dolutegravir, abacavir and lamivudine. Overdoses with Triumeq have not been reported previously. Currently, there is no information about crushing of the branded fixed-dose combination of dolutegravir/abacavir/lamivudine (Triumeq\\u00ae, referred to as TRI); therefore, crushing of TRI is not recommended.On running short of medication, he admitted to sharing his partner's treatment (Triumeq; abacavir, lamivudine and dolutegravir). Abacavir/Dolutegravir/Lamivudine (Triumeq)-Induced Liver Toxicity in a Human Immunodeficiency Virus-Infected Patient.Drug-induced liver injury related to Triumeq (abacavir/lamivudine/dolutegravir) has not been reported in clinical trials. Drug-induced liver injury related to Triumeq (abacavir/lamivudine/dolutegravir) has not been reported in clinical trials.Abacavir/Dolutegravir/Lamivudine (Triumeq)-Induced Liver Toxicity in a Human Immunodeficiency Virus-Infected Patient.On running short of medication, he admitted to sharing his partner's treatment (Triumeq; abacavir, lamivudine and dolutegravir).Naltrexone/bupropion (Contrave) for weight management; pembrolizumab (Keytruda) for melanoma; dolutegravir/abacavir/lamivudine (Triumeq) for HIV-1; and immune globulin infusion 10% (human) with recombinant human hyaluronidase (Hyqvia) for primary immunodeficiency.Here we evaluated humanized mice for their susceptibility to HIV-2 infection and tested a single-pill three drug formulation of anti-retrovirals (NRTIs abacavir and lamivudine, integrase inhibitor dolutegravir) (trade name, Triumeq<br>\"}, {\"question\": \"List the four advances integrated into the SHERLOCKv2 platform.\", \"ideal_answer\": [\"SHERLOCKv2 presents with four distinct advances: (i) four-channel single-reaction multiplexing with orthogonal CRISPR enzymes; (ii) quantitative measurement of input as low as 2 attomolar; (iii) 3.5-fold increase in signal sensitivity by combining Cas13 with Csm6, an auxiliary CRISPR-associated enzyme; and (iv) lateral-flow readout.\"], \"exact_answer\": [[\"four-channel single-reaction multiplexing with orthogonal CRISPR enzymes\"], [\"quantitative measurement of input as low as 2 attomolar\"], [\"3.5-fold increase in signal sensitivity by combining Cas13 with Csm6, an auxiliary CRISPR-associated enzyme\"], [\"lateral-flow readout\"]], \"context\": \"Through characterization of CRISPR enzymology and application development, we report here four advances integrated into SHERLOCK version 2 (SHERLOCKv2) (i) four-channel single-reaction multiplexing with orthogonal CRISPR enzymes; (ii) quantitative measurement of input as low as 2 attomolar; (iii) 3.5-fold increase in signal sensitivity by combining Cas13 with Csm6, an auxiliary CRISPR-associated enzyme; and (iv) lateral-flow readout. \"}, {\"question\": \"List clinical symptoms of the MECOM-associated syndrome\", \"ideal_answer\": [\"Heterozygous mutations in MECOM (MDS1 and EVI1 complex locus) have been reported to be causative of a rare association of congenital amegakaryocytic thrombocytopenia and radioulnar synostosis. The clinical picture included radioulnar synostosis, bone marrow failure, clinodactyly, cardiac and renal malformations, B-cell deficiency, and presenile hearing loss. No single clinical manifestation was detected in all patients affected by MECOM mutations. Radioulnar synostosis and B-cell deficiency were observed only in patients with mutations affecting a short region in the C-terminal zinc finger domain of EVI1.\"], \"exact_answer\": [[\"radioulnar synostosis\"], [\"bone marrow failure\"], [\"clinodactyly\"], [\"cardiac malformations\"], [\"renal malformations\"], [\"B-cell deficiency\"], [\"presenile hearing loss\"]], \"context\": \"MECOM-associated syndrome: a heterogeneous inherited bone marrow failure syndrome with amegakaryocytic thrombocytopenia.Heterozygous mutations in MECOM (MDS1 and EVI1 complex locus) have been reported to be causative of a rare association of congenital amegakaryocytic thrombocytopenia and radioulnar synostosis. Here we report on 12 patients with congenital hypomegakaryocytic thrombocytopenia caused by MECOM mutations (including 10 novel mutations). The mutations affected different functional domains of the EVI1 protein. The spectrum of phenotypes was much broader than initially reported for the first 3 patients; we found familial as well as sporadic cases, and the clinical spectrum ranged from isolated radioulnar synostosis with no or mild hematological involvement to severe bone marrow failure without obvious skeletal abnormality. The clinical picture included radioulnar synostosis, bone marrow failure, clinodactyly, cardiac and renal malformations, B-cell deficiency, and presenile hearing loss. No single clinical manifestation was detected in all patients affected by MECOM mutations. Radioulnar synostosis and B-cell deficiency were observed only in patients with mutations affecting a short region in the C-terminal zinc finger domain of EVI1. We propose the term MECOM-associated syndrome for this heterogeneous hereditary disease and inclusion of MECOM sequencing in the diagnostic workup of congenital bone marrow failure.Heterozygous mutations in MECOM (MDS1 and EVI1 complex locus) have been reported to be causative of a rare association of congenital amegakaryocytic thrombocytopenia and radioulnar synostosis.Radioulnar synostosis and B-cell deficiency were observed only in patients with mutations affecting a short region in the C-terminal zinc finger domain of EVI1.The spectrum of phenotypes was much broader than initially reported for the first 3 patients; we found familial as well as sporadic cases, and the clinical spectrum ranged from isolated radioulnar synostosis with no or mild hematological involvement to severe bone marrow failure without obvious skeletal abnormality.The clinical picture included radioulnar synostosis, bone marrow failure, clinodactyly, cardiac and renal malformations, B-cell deficiency, and presenile hearing loss.\"}, {\"question\": \"List features of the Triple A syndrome.\", \"ideal_answer\": [\"Triple A (Allgrove) syndrome is a rare autosomal recessive disorder characterized by cardinal features of adrenal insufficiency, achalasia, and alacrimia. It is frequently associated with neurological manifestations like polyneuropathy.\"], \"exact_answer\": [[\"adrenal insufficiency\"], [\"achalasia\"], [\"alacrimia\"]], \"context\": \"Triple A syndrome (TAS) or Allgrove syndrome (OMIM #231550) is a rare autosomal recessive disorder characterised by adrenocorticotropic hormone-resistant adrenal insufficiency, alacrima, achalasia, and neurological and dermatological abnormalities. Background Triple A syndrome is characterized by achalasia, alacrima and adrenal insufficiency with neurological manifestations occurring later in the course of the disease. Allgrove or triple A syndrome (AS or AAA) is a rare autosomal recessive syndrome with variable phenotype due to mutations in AAAS gene which encodes a protein called ALADIN. Generally, it's characterized by of adrenal insufficiency in consequence of adrenocorticotropic hormone (ACTH) resistance, besides of achalasia, and alacrimia. Triple-A syndrome, also known as Allgrove syndrome, is a rare autosomal recessive disorder. The 3 features of this syndrome are achalasia, adrenal insufficiency, and alacrima. Allgrove (Triple A) syndrome is a rare autosomal recessive disorder characterized by cardinal features of adrenal insufficiency due to adrenocorticotropic hormone (ACTH) resistance, achalasia, and alacrimia. It is frequently associated with neurological manifestations like polyneuropathy.\\\"Triple A\\\" syndrome is a rare, autosomal recessive condition whose main clinical features are alacrima, achalasia, and adrenal failure.The triple A or Allgrove syndrome is an autosomal-recessive disease (MIM*231550) characterized by the triad of achalasia, alacrima and adrenocorticotropic hormone (ACTH)-resistant adrenal insufficiency.The triple A syndrome is a rare autosomal recessive disease that is characterised by the triad of adrenocorticotropin (ACTH)-resistant adrenal insufficiency, achalasia and alacrima.BACKGROUND\\nThe triple A syndrome is characterized by the main features adrenal insufficiency, achalasia and alacrima.Allgrove syndrome (triple A syndrome) is a rare autosomal recessive disorder characterized by achalasia, alacrima, adrenal insufficiency, and--occasionally--autonomic instability.The triple A syndrome (MIM*231550) is a rare autosomal recessive disorder characterized by adrenocorticotropic hormone (ACTH) resistant adrenal failure, achalasia, alacrima and a variety of neurological and dermatological features.The triple A syndrome (MIM#231550) is a rare autosomal recessive disorder characterized by adrenocorticotropic hormone (ACTH) resistant adrenal failure, achalasia, alacrima, and a variety of neurological and dermatological features.Triple A syndrome is an autosomal recessive disorder characterized by the triad of adrenocorticotropic hormone resistant adrenal insufficiency, achalasia, and alacrima.The triple A syndrome is caused by mutations in the AAAS gene, which encodes a protein known as ALADIN (ALacrima Achalasia aDrenal Insufficiency Neurologic disorder).The triple A syndrome or Allgrove syndrome (MIM*231550) is characterized by adrenocorticotropic hormone (ACTH) resistant Adrenal insufficiency, Achalasia of the cardia and Alacrima.Triple A (Allgrove) syndrome is characterized by achalasia, alacrima, adrenal abnormalities and a progressive neurological syndrome.Two months later, achalasia was diagnosed, and in the presence of alacrima, the patient satisfies the diagnostic criteria of triple A syndrome. Clinical and genetic characterization of families with triple A (Allgrove) syndrome.Triple A (Allgrove) syndrome is characterized by achalasia, alacrima, adrenal abnormalities and a progressive neurological syndrome. Those bearing mutations had the classical triple A syndrome of achalasia, alacrima, adrenal abnormalities and a progressive neurological syndrome. Clinical and molecular genetic findings in a 6-year-old Bosnian boy with triple A syndrome.The triple A syndrome is a rare autosomal recessive disease that is characterised by the triad of adrenocorticotropin (ACTH)-resistant adrenal insufficiency, achalasia and alacrima. Triple A syndrome: genotype-phenotype assessment.The triple A or Allgrove syndrome is an autosomal-recessive disease (MIM*231550) characterized by the triad of achalasia, alacrima and adrenocorticotropic hormone (ACTH)-resistant adrenal insufficiency. Neurological features in adult Triple-A (Allgrove) syndrome.Triple-A or Allgrove syndrome is a rare multisystem disease classically associated with esophageal achalasia, adrenal insufficiency and alacrima. Triple A syndrome is a rare, autosomal recessive condition whose main clinical features are alacrima, achalasia, and adrenal failure.\"}, {\"question\": \"List sirtuin inhibitors.\", \"ideal_answer\": [\"Sirtinol\\nnicotinamide (NAM)\\nLC-0296\\nBZD9L1\"], \"exact_answer\": [[\"Sirtinol\"], [\"nicotinamide (NAM)\"], [\"LC-0296\"], [\"BZD9L1\"]], \"context\": \" Sirtinol, a commonly employed sirtuin inhibitorsirtuin inhibitor nicotinamide pan-sirtuin inhibitor nicotinamide (NAM)of a novel sirtuin inhibitor (BZD9L1)  Sirtuin inhibitor nicotinamide.A Novel Sirtuin-3 Inhibitor, LC-0296\"}, {\"question\": \"Which features are evaluated with the CRAFFT screening test?\", \"ideal_answer\": [\"The CRAFFT (Car, Relax, Alone, Forget, Friends, Trouble) was developed as a brief screening instrument for adolescents to measure Alcohol and other substance use disorders.\"], \"exact_answer\": [[\"Car\"], [\"Relax\"], [\"Alone\"], [\"Forget\"], [\"Friends\"], [\"Trouble\"]], \"context\": \" The New Mexico Department of Health school-based health centers (SBHCs) universally administer a validated screen, the CRAFFT (Car, Relax, Alone, Forget, Friends, Trouble), for adolescent substance use concerns; however, quality assurance efforts revealed that SBHC providers needed more information at the point of screening to initiate brief interventions for students with positive CRAFFT screens.We present the results of the reliability and concurrent validity of the CRAFFT (acronym for Car, Relax, Alone, Forget, Friends, and Trouble) substance abuse screening instrument among residents of youth correctional facilities in Lagos, Nigeria. The Problem Oriented Screening Instrument for Teenagers (POSIT) and the Car, Relax, Alone, Forget, Family/Friends, Trouble questionnaire (CRAFFT) are used to this end.The aim of this study was to compare the performance of the Alcohol Use Disorders Identification Test (AUDIT), its short form AUDIT-C, the Substance Module of the Problem Oriented Screening Instrument for Teenagers (POSIT), and CRAFFT (acronym for car, relax, alone, forget, family, and friends).INTRODUCTION: Alcohol (AUD) and other substance use disorders (SUD) are common among adolescents. The CRAFFT (Car, Relax, Alone, Forget, Friends, Trouble, 1999) was developed as a brief screening instrument for adolescents to measure AUD and SUD. The Problem Oriented Screening Instrument for Teenagers (POSIT) and the Car, Relax, Alone, Forget, Family/Friends, Trouble questionnaire (CRAFFT) are used to this end.The CRAFFT (Car, Relax, Alone, Forget, Friends, Trouble, 1999) was developed as a brief screening instrument for adolescents to measure AUD and SUD.We present the results of the reliability and concurrent validity of the CRAFFT (acronym for Car, Relax, Alone, Forget, Friends, and Trouble) substance abuse screening instrument among residents of youth correctional facilities in Lagos, Nigeria.\"}, {\"question\": \"List lymphocytes that are analyzed by a flow cytometer.\", \"ideal_answer\": [\"Quantitation of lymphocyte subsets (B cells, T cells, CD4 and CD8 T cells and NK cells) classically relies on quantitation of lymphocytes and immunophenotyping by flow cytometry.\"], \"exact_answer\": [[\"B cells\"], [\"T cells\"], [\"CD4 and CD8 T cells\"], [\"NK cells\"]], \"context\": \" Quantitation of lymphocyte subsets (B cells, T cells, CD4 and CD8 T cells and NK cells) classically relies on quantitation of lymphocytes and immunophenotyping by flow cytometry. The distributions of Th17 cells, regulatory Treg-cells, CD4+ T-cells, CD8+ T-cells, and CD3+ T-cells were determined by flow cytometer.  main lymphocyte subsets (T, B and NK cell \"}, {\"question\": \"List two drugs that are included in the Akynzeo pill?\", \"ideal_answer\": [\"Akynzeo is an oral fixed combination of netupitant and palonosetron that is available for use in the prevention of acute and delayed chemotherapy-induced nausea and vomiting (CINV).\"], \"exact_answer\": [[\"netupitant\"], [\"palonosetron\"]], \"context\": \"OBJECTIVES: To review the latest antiemetic guideline recommendations and provide an update on the use of NEPA, a fixed combination antiemetic composed of the neurokinin-1 receptor antagonist (RA) netupitant and the 5-hydroxytryptamine-3 RA palonosetron (Akynzeo\\u00ae).An oral fixed combination of netupitant/palonosetron (NEPA; Akynzeo(\\u00ae)) with >85% antiemetic efficacy is available for use in the prevention of acute and delayed chemotherapy-induced nausea and vomiting (CINV).An oral fixed combination of netupitant/palonosetron (NEPA; Akynzeo(\\u00ae)) is available for use in the prevention of acute and delayed chemotherapy-induced nausea and vomiting (CINV).Ledipasvir/sofosbuvir (Harvoni) for hepatitis C virus genotype 1 infection; dulaglutide (Trulicity) for glycemic control in type-2 diabetes; netupitant/palonosetron (Akynzeo) for prevention of nausea and vomiting related to chemotherapy; and naloxegol (Movantik) for opioid-induced constipation in patients with chronic noncancer pain.Netupitant/Palonosetron: A Review in the Prevention of Chemotherapy-Induced Nausea and Vomiting.An oral fixed combination of netupitant/palonosetron (NEPA; Akynzeo(\\u00ae)) is available for use in the prevention of acute and delayed chemotherapy-induced nausea and vomiting (CINV). Pharmaceutical approval update.Ledipasvir/sofosbuvir (Harvoni) for hepatitis C virus genotype 1 infection; dulaglutide (Trulicity) for glycemic control in type-2 diabetes; netupitant/palonosetron (Akynzeo) for prevention of nausea and vomiting related to chemotherapy; and naloxegol (Movantik) for opioid-induced constipation in patients with chronic noncancer pain. \"}, {\"question\": \"List some substances important for proper nervous system function that gut microbes produce.\", \"ideal_answer\": [\"serotonin\\ngamma-aminobutyric acid\\nshort-chain fatty acids\\nneurotransmitters\"], \"exact_answer\": [[\"serotonin\"], [\"gamma-aminobutyric acid\"], [\"short-chain fatty acids\"], [\"neurotransmitters\"]], \"context\": \" Gut microorganisms are capable of producing and delivering neuroactive substances such as serotonin and gamma-aminobutyric acid, which act on the gut-brain axis.Prebiotic compounds are appealing for this purpose as they are generally food-grade substances only degraded by microbes, such as bifidobacteria and lactobacilli, from which beneficial short-chain fatty acids are produced.Moreover, gut microbiota are considerable sources of NTs. \"}, {\"question\": \"List 3 enterotoxins produced by Clostridium difficile.\", \"ideal_answer\": [\"Toxin A (TcdA), toxin B (TcdB), and binary toxin (CDT) produced by Clostridium difficile (CD)\"], \"exact_answer\": [[\"binary toxin (CDT\"], [\"Toxin A (TcdA)\"], [\"Toxin B (TcdB)\"]], \"context\": \"The toxins produced by the C. diff. (toxin A, toxin B, and binary toxin) are the agents that cause injury and disease. Only toxin producing C. diff. Strains will cause disease. Binary toxin by itself is not thought to produce diseaseBinary toxin causes disease in humans when present with toxin A and B producing bacteria,  The main virulence factor is related to the release of two major exotoxins, toxin A (TcdA) and toxin B (TcdB). Recent C. difficile-associated disease (CDAD) outbreaks have been caused by hypervirulent strains which secrete an additional binary toxin (CDTa/CDTb). VThe exotoxins toxin A (TcdA) and toxin B (TcdB) are produced by the bacterial pathogen Clostridium difficile and are responsible for the pathology associated with C. difficile infection (CDI).The pathogenic organism, Clostridium difficile, produces two enterotoxins, toxin A and toxin B, that cause colonic mucosal inflammation.Enterotoxins from Clostridium difficile; diarrhoeogenic potency and morphological effects in the rat intestine.The action of toxins A, B, and C from Clostridium difficile was studied in the small intestine and colon of rats. \"}, {\"question\": \"List adipokines.\", \"ideal_answer\": [\"adiponectin\\nleptin\\nresistin\"], \"exact_answer\": [[\"adiponectin\"], [\"leptin\"], [\"resistin\"]], \"context\": \"adipokines (adiponectin and leptin) number of adipokines, like Resistin (RETN)Plasma adipokine levels were measured before exercise, and at 5 and 90\\u2009min after exercise on the first and the last training days. Adiponectin was higher at 5\\u2009min than 90\\u2009min post-exercise (11.7\\u2009\\u00b1\\u20097.3 and 10.5\\u2009\\u00b1\\u20095.8\\u2009ng/ml; p\\u2009=\\u2009.01) in the first exercise day. Leptin decreased 5\\u2009min after exercise (23.6\\u2009\\u00b1\\u200913.2 vs. baseline 27.8\\u2009\\u00b1\\u200914.4\\u2009ng/ml; p\\u2009<\\u2009.01) and remained depressed following 90\\u2009min (p\\u2009<\\u2009.01) leptin, adiponectin, resistin, \"}, {\"question\": \"List features of the DOOR syndrome.\", \"ideal_answer\": [\"DOOR syndrome is a rare multisystem genetic disorder, consisting of deafness (sensorineural), onychodystrophy, osteodystrophy, and mental retardation.\"], \"exact_answer\": [[\"deafness\"], [\"onychodystrophy\"], [\"osteodystrophy\"], [\"mental retardation\"]], \"context\": \"DOOR syndrome is a rare multisystem genetic disorder, consisting of deafness (sensorineural), onychodystrophy, osteodystrophy, and mental retardation.We report the anaesthetic management of a 48-year-old male patient with Deafness, Onycho-Osteodystrophy and mental Retardation syndrome, epilepsy and cerebral palsy who had two dental procedures under anaesthetic care.We present the case of a 9-year-old boy with DOOR syndrome recognized in the first year of his life because of a delayed development of speech. The diagnosis was based on characteristic abnormalities, including congenital deafness, nail and bone abnormalities, and mild mental retardation.DOOR syndrome (deafness, onychodystrophy, osteodystrophy, and mental retardation) is a rarely described disorder with less than 35 reports in the literature. [DOOR (deafness, onychodystrophy, osteodystrophy, mental retardation) syndrome].DOOR syndrome (deafness, onychodystrophy, osteodystrophy, and mental retardation) is a rarely described disorder with less than 35 reports in the literature.The hallmarks of the syndrome, represented in the DOOR acronym, include sensorineural hearing loss, hypoplastic or absent nails on the hands and feet, small or absent distal phalanges of the hands and feet, and mental retardation.Four patients from three families with the clinical features of DOOR syndrome (onycho-osteodystrophy, dystrophic thumbs, sensorineural deafness, and increased urinary levels of 2-oxoglutarate) are the subjects of this report.The acronym DOOR was first used by Cantwell in 1975 to describe a syndrome comprising sensorineural deafness, osteodystrophy, onychodystrophy, and mental retardation.DOOR syndrome (deafness, onycho-osteodystrophy, and mental retardation): a new patient and delineation of neurologic variability among recessive cases.Congenital heart disease and urinary tract abnormalities in two siblings with DOOR syndrome.The acronym DOOR was first used by Cantwell in 1975 to describe a syndrome comprising sensorineural deafness, osteodystrophy, onychodystrophy, and mental retardation. \"}, {\"question\": \"Which web-based pedigree editors are available?\", \"ideal_answer\": [\"Pedigreejs and Madeline 2.0 Pedigree Drawing Engine (PDE)\"], \"exact_answer\": [[\"Pedigreejs\"], [\"Madeline 2.0 Pedigree Drawing Engine (PDE)\"]], \"context\": \"pedigreejs: a web-based graphical pedigree editor.Several standalone graphical pedigree editors and drawing applications exist but there are no freely available lightweight graphical pedigree editors that can be easily configured and incorporated into web applications.Results: We developed 'pedigreejs', an interactive graphical pedigree editor written in JavaScript, which uses standard pedigree nomenclature. Pedigreejs provides an easily configurable, extensible and lightweight pedigree editor. It makes use of an open-source Javascript library to define a hierarchical layout and to produce images in scalable vector graphics (SVG) format that can be viewed and edited in web browsers.Availability and implementation: The software is freely available under GPL licence (https://ccge-boadicea.github.io/pedigreejs/).Madeline 2.0 PDE: a new program for local and web-based pedigree drawing.The Madeline 2.0 Pedigree Drawing Engine (PDE) is a pedigree drawing program for use in linkage and family-based association studies. The program is designed to handle large and complex pedigrees with an emphasis on readability and aesthetics. For complex pedigrees, we use a hybrid algorithm in which consanguinous loops are drawn as cyclic graphs whenever possible, but we resort to acyclic graphs when matings can no longer be connected without line crossings. A similar hybrid approach is used to avoid line crossings for matings between distant descendants of different founding groups. Written in object-oriented C++ and released under the GNU General Public License (GPL), Madeline 2.0 PDE reads input files specified on the command line and generates pedigree drawings without user interaction. Pedigree output in scalable vector graphics (SVG) format can be viewed in browsers with native SVG rendering support or in vector graphics editors.Madeline 2.0 PDE: a new program for local and web-based pedigree drawing.http://kellogg.umich.edu/madeline.pedigreejs: a web-based graphical pedigree editor.Supplementary data are available at Bioinformatics online.\"}, {\"question\": \"Which curated databases exist for spider-venom toxins?\", \"ideal_answer\": [\"ArachnoServer and its updated version ArachnoServer 2.0 are manually curated databases providing information on the sequence, structure and biological activity of protein toxins from spider venoms.\"], \"exact_answer\": [[\"ArachnoServer\"], [\"ArachnoServer 2.0\"]], \"context\": \"ArachnoServer: a database of protein toxins from spiders.We have developed ArachnoServer, a manually curated database that provides detailed information about proteinaceous toxins from spiders. Key features of ArachnoServer include a new molecular target ontology designed especially for venom toxins, the most up-to-date taxonomic information available, and a powerful advanced search interface. Toxin information can be browsed through dynamic trees, and each toxin has a dedicated page summarising all available information about its sequence, structure, and biological activity. ArachnoServer currently manages 567 protein sequences, 334 nucleic acid sequences, and 51 protein structures.CONCLUSION: ArachnoServer provides a single source of high-quality information about proteinaceous spider toxins that will be an invaluable resource for pharmacologists, neuroscientists, toxinologists, medicinal chemists, ion channel scientists, clinicians, and structural biologists. ArachnoServer is available online at http://www.arachnoserver.org.ArachnoServer 2.0, an updated online resource for spider toxin sequences and structures.ArachnoServer (www.arachnoserver.org) is a manually curated database providing information on the sequence, structure and biological activity of protein toxins from spider venoms. These proteins are of interest to a wide range of biologists due to their diverse applications in medicine, neuroscience, pharmacology, drug discovery and agriculture. ArachnoServer currently manages 1078 protein sequences, 759 nucleic acid sequences and 56 protein structures. Key features of ArachnoServer include a molecular target ontology designed specifically for venom toxins, current and historic taxonomic information and a powerful advanced search interface. The following significant improvements have been implemented in version 2.0: (i) the average and monoisotopic molecular masses of both the reduced and oxidized form of each mature toxin are provided; (ii) the advanced search feature now enables searches on the basis of toxin mass, external database accession numbers and publication date in ArachnoServer; (iii) toxins can now be browsed on the basis of their phyletic specificity; (iv) rapid BLAST searches based on the mature toxin sequence can be performed directly from the toxin card; (v) private silos can be requested from research groups engaged in venoms-based research, enabling them to easily manage and securely store data during the process of toxin discovery; and (vi) a detailed user manual is now available.Summary\\nArachnoServer is a manually curated database that consolidates information on the sequence, structure, function and pharmacology of spider-venom toxins.ArachnoServer is a manually curated database that consolidates information on the sequence, structure, function and pharmacology of spider-venom toxins.\"}, {\"question\": \"Please list 2 antitoxin antibodies approved by the FDA for reducing the recurrence of Clostridium difficile infection\", \"ideal_answer\": [\"The antitoxin antibodies actoxumab and bezlotoxumab bind to and neutralize TcdA and TcdB, respectively. Bezlotoxumab was recently approved by the FDA for reducing the recurrence of CDI.\"], \"exact_answer\": [[\"bezlotoxumab\"], [\"actoxumab\"]], \"context\": \"The exotoxins toxin A (TcdA) and toxin B (TcdB) are produced by the bacterial pathogen Clostridium difficile and are responsible for the pathology associated with C. difficile infection (CDI). The antitoxin antibodies actoxumab and bezlotoxumab bind to and neutralize TcdA and TcdB, respectively. Bezlotoxumab was recently approved by the FDA for reducing the recurrence of CDI.The Monoclonal Antitoxin Antibodies (Actoxumab-Bezlotoxumab) Treatment Facilitates Normalization of the Gut Microbiota of Mice with Clostridium difficile Infection.The antitoxin antibodies actoxumab and bezlotoxumab bind to and neutralize TcdA and TcdB, respectively.Mechanisms of protection against Clostridium difficile infection by the monoclonal antitoxin antibodies actoxumab and bezlotoxumab.Bezlotoxumab: anti-toxin B monoclonal antibody to prevent recurrence of Clostridium difficile infection.Bezlotoxumab (Zinplava\\u2122) is a human monoclonal antibody against Clostridium difficile toxin B developed by Merck & Co.Bezlotoxumab (Zinplava\\u2122) is a fully human monoclonal antibody against Clostridium difficile toxin B indicated for the prevention of C. difficile infection (CDI) recurrence in patients with a high recurrence risk.Bezlotoxumab (Zinplava\\u2122) is a human monoclonal antibody against Clostridium difficile toxin B developed by Merck &amp; Co.\"}, {\"question\": \"List the functions of the protein lactotransferrin.\", \"ideal_answer\": [\"Lactotransferrin has numerous biological roles, including the regulation of iron absorption and modulation of immune responses, and has anti-microbial, anti-viral, antioxidant, anti-cancer, and anti-inflammatory activities.\"], \"exact_answer\": [[\"regulation of iron absorption\"], [\"modulation of immune responses\"], [\"anti-microbial\"], [\"anti-viral\"], [\"antioxidant\"], [\"anti-cancer\"], [\"anti-inflammatory\"]], \"context\": \"It has numerous biological roles, including the regulation of iron absorption and modulation of immune responses, and has anti-microbial, anti-viral, antioxidant, anti-cancer, and anti-inflammatory activities.The tumor suppressor function of lactotransferrin (LTF) has been reported in a variety of tumors, including GC, nasopharyngeal carcinoma (NPC) and prostate cancer. Lactotransferrin (LTF) is a pleiotropic protein, expressed in various body tissues and fluids, which modulates the host immune-inflammatory response and bone metabolism, and might be involved in dental implant osseointegration\"}, {\"question\": \"Name siRNA drugs that have entered phase 2-3 clinical trials (by 2019).\", \"ideal_answer\": [\"siRNAs that have entered phase 2-3 clinical trials by 2019 include PF-04523655, TKM-080301, Atu027, SYL040012, SYL1001, siG12D-LODER (phase 2), QPI-1002, QPI-1007, and patisiran (phase 3).\"], \"exact_answer\": [[\"PF-04523655\"], [\"TKM-080301\"], [\"Atu027\"], [\"SYL040012\"], [\"SYL1001\"], [\"siG12D-LODER\"], [\"QPI-1002\"], [\"QPI-1007\"], [\"patisiran\"]], \"context\": \" This review addresses nine small-interfering RNAs (siRNAs) and one unique microRNA (miRNA) inhibitor, which entered the phase 2-3 clinical trials. The siRNAs in focus are PF-04523655, TKM-080301, Atu027, SYL040012, SYL1001, siG12D-LODER (phase 2), QPI-1002, QPI-1007, and patisiran (phase 3). \"}, {\"question\": \"Mention computational tools that have been developed for alternative polyadenylation (APA) sites analysis\", \"ideal_answer\": [\"TAPAS, PlantAPA and IntMAP\"], \"exact_answer\": [[\"TAPAS\"], [\"PlantAPA\"], [\"IntMAP\"], [\"VAAPA\"]], \"context\": \"TAPAS: tool for alternative polyadenylation site analysis.We propose a new tool, called TAPAS, for detecting novel APA sites from RNA-Seq data. It can deal with more than two APA sites in a gene as well as APA sites that occur before the last exon. The tool is based on an existing method for finding change points in time series data, but some filtration techniques are also adopted to remove change points that are likely false APA sites. It is then extended to identify APA sites that are expressed differently between two biological samples and genes that contain 3' UTRs with shortening/lengthening events. Our extensive experiments on simulated and real RNA-Seq data demonstrate that TAPAS outperforms the existing tools for APA site detection or shortening/lengthening analysis significantly.PlantAPA: A Portal for Visualization and Analysis of Alternative Polyadenylation in Plants.Increasing numbers of poly(A) sites collected in various plant species demand new methods and tools to access and mine these data. We have created an open-access web service called PlantAPA (http://bmi.xmu.edu.cn/plantapa) to visualize and analyze genome-wide poly(A) sites in plants. PlantAPA provides various interactive and dynamic graphics and seamlessly integrates a genome browser that can profile heterogeneous cleavage sites and quantify expression patterns of poly(A) sites across different conditions. Particularly, through PlantAPA, users can analyze poly(A) sites in extended 3' UTR regions, intergenic regions, and ambiguous regions owing to alternative transcription or RNA processing. In addition, it also provides tools for analyzing poly(A) site selections, 3' UTR lengthening or shortening, non-canonical APA site switching, and differential gene expression between conditions, making it more powerful for the study of APA-mediated gene expression regulation. More importantly, PlantAPA offers a bioinformatics pipeline that allows users to upload their own short reads or ESTs for poly(A) site extraction, enabling users to further explore poly(A) site selection using stored PlantAPA poly(A) sites together with their own poly(A) site datasets. To date, PlantAPA hosts the largest database of APA sites in plants, including Oryza sativa, Arabidopsis thaliana, Medicago truncatula, and Chlamydomonas reinhardtii. As a user-friendly web service, PlantAPA will be a valuable addition to the community of biologists studying APA mechanisms and gene expression regulation in plants.In our recent RNA-seq experiments using cells with hyper-activated mammalian target of rapamycin (mTOR), we found that cellular mTOR activation leads to transcriptome-wide alternative polyadenylation (APA), resulting in the activation of multiple cellular pathways. Here, we developed a novel bioinformatics algorithm, IntMAP, which integrates RNA-Seq and PolyA Site (PAS)-Seq data for a comprehensive characterization of APA events. By applying IntMAP to the datasets from cells with hyper-activated mTOR, we identified novel APA events that could otherwise not be identified by either profiling method alone. Several transcription factors including Cebpg (CCAAT/enhancer binding protein gamma) were among the newly discovered APA transcripts, indicating that diverse transcriptional networks may be regulated by mTOR-coordinated APA. The prevention of APA in Cebpg using the CRISPR/cas9-mediated genome editing tool showed that mTOR-driven 3'-UTR shortening in Cebpg is critical in protecting cells from endoplasmic reticulum (ER) stress. Taken together, we present IntMAP as a new bioinformatics algorithm for APA analysis by which we expand our understanding of the physiological role of mTOR-coordinated APA events to ER stress response. IntMAP toolbox is available at http://compbio.cs.umn.edu/IntMAP/.Our extensive experiments on simulated and real RNA-Seq data demonstrate that TAPAS outperforms the existing tools for APA site detection or shortening/lengthening analysis significantly.Results\\nWe propose a new tool, called TAPAS, for detecting novel APA sites from RNA-Seq data.PlantAPA: A Portal for Visualization and Analysis of Alternative Polyadenylation in Plants.Alternative polyadenylation (APA) is an important layer of gene regulation that produces mRNAs that have different 3' ends and/or encode diverse protein isoforms. VAAPA: a web platform for visualization and analysis of alternative polyadenylation.TAPAS: tool for alternative polyadenylation site analysis.Supplementary data are available at Bioinformatics online.Furthermore, the tools are unable to integrate the analysis of shortening/lengthening events with APA site detection.<br><b>Results</b>: We propose a new tool, called TAPAS, for detecting novel APA sites from RNA-Seq data.Our extensive experiments on simulated and real RNA-Seq data demonstrate that TAPAS outperforms the existing tools for APA site detection or shortening/lengthening analysis significantly.<br><b>Availability and implementation</b>: https://github.com/arefeen/TAPAS.<br><b>Supplementary information</b>: Supplementary data are available at Bioinformatics online.<br>Our extensive experiments on simulated and real RNA-Seq data demonstrate that TAPAS outperforms the existing tools for APA site detection or shortening/lengthening analysis significantly.We propose a new tool, called TAPAS, for detecting novel APA sites from RNA-Seq data.Here, a web platform for visualization and analysis of alternative polyadenylation (VAAPA) was developed.\"}, {\"question\": \"Which curated data resources for ChIP-seq data are available?\", \"ideal_answer\": [\"The MGA repository, Cistrome Data Browser, CR Cistrome and GeneProf data.\"], \"exact_answer\": [[\"MGA repository\"], [\"Cistrome Data Browser\"], [\"CR Cistrome\"], [\"GeneProf data\"]], \"context\": \"MGA repository: a curated data resource for ChIP-seq and other genome annotated data.The Mass Genome Annotation (MGA) repository is a resource designed to store published next generation sequencing data and other genome annotation data (such as gene start sites, SNPs, etc.) in a completely standardised format.Cistrome Data Browser: a data portal for ChIP-Seq and chromatin accessibility data in human and mouse.To overcome this challenge, we built the Cistrome database, a collection of ChIP-seq and chromatin accessibility data (DNase-seq and ATAC-seq) published before January 1, 2016, including 13 366 human and 9953 mouse samples. All the data have been carefully curated and processed with a streamlined analysis pipeline and evaluated with comprehensive quality control metrics. We have also created a user-friendly web server for data query, exploration and visualization. The resulting Cistrome DB (Cistrome Data Browser), available online at http://cistrome.org/db, is expected to become a valuable resource for transcriptional and epigenetic regulation studies.CR Cistrome: a ChIP-Seq database for chromatin regulators and histone modification linkages in human and mouse.GeneProf data: a resource of curated, integrated and reusable high-throughput genomics experiments.CR Cistrome: a ChIP-Seq database for chromatin regulators and histone modification linkages in human and mouse.Diversified histone modifications (HMs) are essential epigenetic features. \"}, {\"question\": \"Which diseases are treated with netarsudil?\", \"ideal_answer\": [\"In 2 large, randomized, double-masked trials, once-daily dosing of netarsudil 0.02% was found to be effective and well tolerated for the treatment of patients with ocular hypertension and open-angle glaucoma.\"], \"exact_answer\": [[\"ocular hypertension\"], [\"open-angle glaucoma\"]], \"context\": \"Two Phase 3 Clinical Trials Comparing the Safety and Efficacy of Netarsudil to Timolol in Patients With Elevated Intraocular Pressure: Rho Kinase\\u00a0Elevated IOP Treatment Trial 1 and 2 (ROCKET-1 and ROCKET-2).CONCLUSIONS: In 2 large, randomized, double-masked trials reported here, once-daily dosing of netarsudil 0.02% was found to be effective and well tolerated for the treatment of patients with ocular hypertension and open-angle glaucoma. Netarsudil ophthalmic solution 0.02% (hereafter referred to as netarsudil 0.02%) [Rhopressa\\u00ae] is a Rho-associated protein kinase inhibitor that is thought to lower intraocular pressure (IOP) by increasing aqueous humour outflow through the trabecular meshwork. It has been developed by Aerie Pharmaceuticals and was recently approved in the USA for the reduction of elevated IOP in patients with open-angle glaucoma or ocular hypertension.PURPOSE: Netarsudil, an inhibitor of Rho kinase and a norepinephrine transporter, has been shown to lower elevated intraocular pressure (IOP) in controlled studies of patients with open-angle glaucoma and ocular hypertension, and in healthy volunteers.CONCLUSIONS: Once-daily dosing of netarsudil ophthalmic solution 0.02% lowered IOP through increasing trabecular outflow facility and reducing EVP.Netarsudil ophthalmic solution (Rhopressa) for the reduction of elevated intraocular pressure; ertugliflozin (Steglatro) for adults with type-2 diabetes; and lutetium lu 177 dotatate (Lutathera) for certain gastroenteropancreatic neuroendocrine tumors.Two recently approved multi-pharmacophoric drugs (e.g., rho kinase inhibitor, Netarsudil; a drug conjugate, Latanoprostene Bunod) and novel aqueous humor drainage devices (iStent and CyPass) are also gaining acceptance for treating POAG/ NTG. Ultimately, a search for Rho kinase inhibitors led to the discovery of several molecules of therapeutic interest, leaving us today with 2 new ocular hypotensive agents approved for clinical use: ripasudil in Japan and netarsudil in the United States. Netarsudil ophthalmic solution 0.02% for the treatment of patients with open-angle glaucoma or ocular hypertension.Once-daily (p.m.) netarsudil ophthalmic solution 0.02% (Rhopressa) is approved in the United States for lowering elevated intraocular pressure (IOP) in patients with open-angle glaucoma or ocular hypertension. CONCLUSIONS\\nIn 2 large, randomized, double-masked trials reported here, once-daily dosing of netarsudil 0.02% was found to be effective and well tolerated for the treatment of patients with ocular hypertension and open-angle glaucoma.Discovery and Preclinical Development of Netarsudil, a Novel Ocular Hypotensive Agent for the Treatment of Glaucoma.Netarsudil ophthalmic solution is a novel topical intraocular pressure (IOP)-lowering agent that has recently been approved by the US Food and Drug Administration (FDA) for the treatment of ocular hypertension and open-angle glaucoma.Once-daily (p.m.) netarsudil ophthalmic solution 0.02% (Rhopressa) is approved in the United States for lowering elevated intraocular pressure (IOP) in patients with open-angle glaucoma or ocular hypertension.This article summarizes the milestones in the development of netarsudil 0.02% leading to this first approval for the reduction of elevated IOP in patients with open-angle glaucoma or ocular hypertension.Discovery of the ROCK inhibitor netarsudil for the treatment of open-angle glaucoma.Profile of netarsudil ophthalmic solution and its potential in the treatment of open-angle glaucoma: evidence to date.Netarsudil ophthalmic solution is a novel topical intraocular pressure (IOP)-lowering agent that has recently been approved by the US Food and Drug Administration (FDA) for the treatment of ocular hypertension and open-angle glaucoma. Discovery and Preclinical Development of Netarsudil, a Novel Ocular Hypotensive Agent for the Treatment of Glaucoma.Netarsudil is a novel ROCK/NET inhibitor with high potency in biochemical and cell-based assays, an ability to produce large and durable IOP reductions in animal models, and favorable pharmacokinetic and ocular tolerability profiles.Profile of netarsudil ophthalmic solution and its potential in the treatment of open-angle glaucoma: evidence to date.In 2 large, randomized, double-masked trials reported here, once-daily dosing of netarsudil 0.02% was found to be effective and well tolerated for the treatment of patients with ocular hypertension and open-angle glaucoma.To evaluate the efficacy and ocular and systemic safety of netarsudil 0.02% ophthalmic solution, a rho-kinase inhibitor and norepinephrine transporter inhibitor, in patients with open-angle glaucoma and ocular hypertension.\"}, {\"question\": \"Which malignancies is Keytruda approved for before 2017?\", \"ideal_answer\": [\"Before 2017 Keytruda was approved for the treatment of several types of malignancies, such as metastatic melanoma, metastatic non-small-cell lung cancer, recurrent or metastatic head and neck cancer, refractory Hodgkin lymphoma, and urothelial carcinoma.\"], \"exact_answer\": [[\"metastatic melanoma\"], [\"metastatic non-small cell lung cancer\"], [\"recurrent or metastatic head and neck cancer\"], [\"refractory Hodgkin lymphoma\"], [\"urothelial carcinoma\"]], \"context\": \"FDA previously approved Keytruda for treatment of several types of malignancies, such as metastatic melanoma, metastatic non-small-cell lung cancer, recurrent or metastatic head and neck cancer, refractory Hodgkin lymphoma, and urothelial carcinoma, all of which carry positive programmed death-1/programmed death-ligand 1 biomarkers.\"}, {\"question\": \"Which tools have been developed for computing split-networks?\", \"ideal_answer\": [\"Split-networks are a generalization of phylogenetic trees that have proven to be a powerful tool in phylogenetics. Tools for computing such networks include SPECTRE, FlatNJ and QNet.\"], \"exact_answer\": [[\"SPECTRE\"], [\"FlatNJ\"], [\"QNet\"]], \"context\": \"SPECTRE: a suite of phylogenetic tools for reticulate evolution.Split-networks are a generalization of phylogenetic trees that have proven to be a powerful tool in phylogenetics. Various ways have been developed for computing such networks, including split-decomposition, NeighborNet, QNet and FlatNJ. Some of these approaches are implemented in the user-friendly SplitsTree software package. However, to give the user the option to adjust and extend these approaches and to facilitate their integration into analysis pipelines, there is a need for robust, open-source implementations of associated data structures and algorithms. Here, we present SPECTRE, a readily available, open-source library of data structures written in Java, that comes complete with new implementations of several pre-published algorithms and a basic interactive graphical interface for visualizing planar split networks. SPECTRE also supports the use of longer running algorithms by providing command line interfaces, which can be executed on servers or in High Performance Computing environments.FlatNJ: a novel network-based approach to visualize evolutionary and biogeographical relationships.Split networks are a type of phylogenetic network that allow visualization of conflict in evolutionary data. We present a new method for constructing such networks called FlatNetJoining (FlatNJ).QNet: an agglomerative method for the construction of phylogenetic networks from weighted quartets.We present QNet, a method for constructing split networks from weighted quartet trees. QNet can be viewed as a quartet analogue of the distance-based Neighbor-Net (NNet) method for network construction. Just as NNet, QNet works by agglomeratively computing a collection of circular weighted splits of the taxa set which is subsequently represented by a planar split network.\"}, {\"question\": \"List 3 apoE mimetics.\", \"ideal_answer\": [\"COG133, COG112 and Ac-hE18A-NH(2) are apoE mimetics.\"], \"exact_answer\": [[\"COG133\"], [\"COG112\"], [\"Ac-hE18A-NH(2)\"]], \"context\": \"Using a murine experimental autoimmune encephalomyelitis (EAE) model of human multiple sclerosis, we found that a peptidomimetic of apoE protein, COG133, substantially reduces the clinical symptoms of EAE and promotes remission from the disability when administered before or after onset of disease. Most notably, fusion of COG133 to a protein transduction domain creates COG112, a modified apoE-mimetic peptide with significantly enhanced anti-inflammatory bioactivities in vitro, and improved therapeutic effects on EAE in vivo, which renders a nearly full remission from the disability. Toward this end, we describe the design and studies of a dual-domain apoE mimetic peptide, Ac-hE18A-NH(2). ApoE mimetics such as Ac-hE18A-NH(2) may therefore restore or replace ligands in genetically induced hyperlipidemias to enable reduction in atherogenic lipoproteins via HSPG even in the absence of functional low-density lipoprotein receptors.\"}, {\"question\": \"Which cells secrete lactotransferrin?\", \"ideal_answer\": [\"We conclude that lactotransferrin represents a late stage differentiation marker of neutrophils, macrophages and distinct subtypes of dendritic cells.\"], \"exact_answer\": [[\"neutrophils\"], [\"macrophages\"], [\"dendritic\"]], \"context\": \"We conclude that lactotransferrin represents a late stage differentiation marker of neutrophils, macrophages and distinct subtypes of dendritic cells.\"}, {\"question\": \"List targets of classical analgesics.\", \"ideal_answer\": [\"Patient phenotypes in pharmacological pain treatment varies between individuals, which could be partly assigned to their genotypes regarding the targets of classical analgesics (OPRM1, PTGS2)\"], \"exact_answer\": [[\"OPRM1\"], [\"PTGS2\"]], \"context\": \"Patient phenotypes in pharmacological pain treatment varies between individuals, which could be partly assigned to their genotypes regarding the targets of classical analgesics (OPRM1, PTGS2) Morphine is the current gold standard analgesic acting at MOP receptors but produces a range of variably troublesome side-effects, in particular tolerance.\"}, {\"question\": \"List off label uses for Rituximab.\", \"ideal_answer\": [\"Off label uses for rituximab are for poly- and dermatomyositis, multiple sclerosis, immune thrombocytopenia, systemic lupus erythematosus (SLE), lupus nephritis (LN), and other immune diseases.\"], \"exact_answer\": [[\"poly- and dermatomyositis\"], [\"multiple sclerosis\"], [\"auto immune disease\"], [\"immune thrombocytopenia\"], [\"systemic lupus erythematosus (SLE)\"], [\"lupus nephritis (LN)\"]], \"context\": \"Analysis of the data of the GRAID-2 registry for poly- and dermatomyositis.RESULTS: In 22\\u00a0of the 23\\u00a0patients in the GRAID-2 registry, rituximab (RIX) was administered, while 1\\u00a0patient was given tocilizumab as off-label therapyOff-label use of rituximab to treat MS patients in Sweden is highOff-label use of rituximab in autoimmune disease in the Top End of the Northern Territory, 2008-2016.immune thrombocytopenia who do not adequately respond to first-line therapy, there is no clear consensus on which second-line therapy to initiate and when. This situation leads to suboptimal approaches, including prolonged exposure to treatments that are not intended for long-term use (eg, corticosteroids) and overuse of off-label therapies (eg, rituximab) Conventional treatment of systemic lupus erythematosus (SLE) and lupus nephritis (LN) is associated with damage accrual, hence increased morbidity rate. Off-label use of rituximab (RTX) has shown significant promise in this patient group; however, data are still controversial.Outcomes of rituximab therapy in refractory lupus: A meta-analysis.Rituximab seems to have an interesting benefit-to-risk ratio in Wegener granulomatosis (excepted in granulomatous lesions), HCV-associated symptomatic cryoglobulinemia in patients unresponsive to anti-viral therapy, pemphigus and thrombotic thrombocytopenic purpura.\"}, {\"question\": \"Name the uses of Sideritis scardica in traditional medicine.\", \"ideal_answer\": [\"Sideritis scardica is used in traditional medicine as a loosening agent in bronchitis and bronchial asthma; against the common cold and lung emphysema; in the treatment of inflammation, gastrointestinal disorders and coughs; and as an active constituent of dietary supplements for the prevention of anemia. Sideritis scardica has been attributed a broad range of properties such as antimicrobial, anti-inflammatory, cytotoxic, antioxidant, gastroprotective, antiglioma, and triple monoamine reuptake inhibition.\"], \"exact_answer\": [[\"against inflammation\"], [\"against gastrointestinal disorders\"], [\"against anemia\"], [\"against bronchitis\"]], \"context\": \"It is used in traditional medicine as a loosening agent in bronchitis and bronchial asthma; against the common cold and lung emphysema; in the treatment of inflammation, gastrointestinal disorders and coughs; and as an active constituent of dietary supplements for the prevention of anemia.A broad range of activities of plant extracts and fractions as antimicrobial, anti-inflammatory, cytotoxic, antioxidant, gastroprotective, antiglioma, and triple monoamine reuptake inhibition as well as cultivation of the species as an approach for conservation of the natural habitats and provision of herb with high and permanent quality has also been presented.\"}, {\"question\": \"What are 2 organisms that can cause Human toxocariasis?\", \"ideal_answer\": [\"Human toxocariasis , a worldwide parasitic disease , is caused by the larval stage of intestinal nematodes of dogs and cats , namely Toxocara canis and Toxocara cati\", \"Human toxocariasis, a worldwide parasitic disease, is caused by the larval stage of intestinal nematodes of dogs and cats, namely Toxocara canis and Toxocara cati\", \"human toxocariasis, a worldwide parasitic disease, is caused by the larval stage of intestinal nematodes of dogs and cats, namely toxocara canis and toxocara cati\"], \"exact_answer\": [[\"Toxocara cati\"], [\"Toxocara canis\"]], \"context\": \"Human toxocariasis, a worldwide parasitic disease, is caused by the larval stage of intestinal nematodes of dogs and cats, namely Toxocara canis and Toxocara catiHuman toxocariasis which is caused mainly by the larvae of Toxocara canis and Toxocara cati, is a worldwide zoonotic disease that can be a potentially serious human infection.BACKGROUND\\nToxocariasis is a prevalent zoonosis disease caused by the closely related nematode species Toxocara canis and Toxocara cati which parasitise Canidae and Felidae respectively.Human toxocariasis is a zoonosis caused by infection with larvae of the ascarid nematode Toxocara canis and, less frequently, T. cati.Toxocariasis is a preventable parasitic disease that is caused by the dog and cat roundworms Toxocara cani and T. cati, respectively.Human toxocariasis is a cosmopolitan parasitic zoonosis caused by Toxocara canis and Toxocara cati which are roundworms of dogs and cats.\"}, {\"question\": \"List places in the body where somatostatin is produced.\", \"ideal_answer\": [\"Somatostatin is a cyclic peptide well known for its strong regulatory effects throughout the body. Also known by the name of growth hormone inhibiting hormone, it is produced in many locations, which include the olfactory bulb, hair follicles, pancreas, retina, and central nervous system (CNS).\"], \"exact_answer\": [[\"Hair follicles\"], [\"CNS\"], [\"pancreas\"], [\"olfactory bulb\"], [\"retina\"]], \"context\": \"Somatostatin (SST) is a neuromodulator which is abundant throughout the central nervous system (CNS)Somatostatin (SST) is abundantly produced by the human retina, and the main source is the retinal pigment epithelium (RPE).Somatostatin (Som), one of the most concentrated neuropeptides in the brain, is highly expressed in the olfactory bulb (OB)The production of somatostatin interneurons in the olfactory bulb is regulated by the transcription factor sp8Somatostatin expression in human hair follicles and its potential role in immune privilegeThe kinetics of insulin, glucagon and somatostatin release was studied in human pancreatic islets.Glucose generates coincident insulin and somatostatin pulses and antisynchronous glucagon pulses from human pancreatic islets.\"}, {\"question\": \"What are the 3 types of immunoglobulin heavy chain containing antibodies found in human breast milk?\", \"ideal_answer\": [\"IgA, IgG, AND  IgM can be found in human milk.\"], \"exact_answer\": [[\"IgM\"], [\"IgA\"], [\"IgG\"]], \"context\": \"This is a descriptive cross-sectional study of a convenience sample of 67 donated colostrum samples at different days after delivery, both raw and pasteurized. Antibody profiles were analyzed at different times during breastfeeding, and total and specific antibodies (IgM, IgA, and IgG subclasses)\"}, {\"question\": \"What are 5 key questions in human performance modeling?\", \"ideal_answer\": [\"There are five key questions of human performance modeling: 1) Why we build models of human performance; 2) What the expectations of a good human performance model are; 3) What the procedures and requirements in building and verifying a human performance model are; 4) How we integrate a human performance model with system design; and 5) What the possible future directions of human performance modeling research are.\", \"the five key questions of human performance modeling: 1) why we build models of human performance; 2) what the expectations of a good human performance model are; 3) what the procedures and requirements in building and verifying a human performance model are; 4) how we integrate a human performance model with system design; and 5) what the possible future directions of human performance modeling research are.\", \"the five key questions of human performance modeling: 1) Why we build models of human performance; 2) What the expectations of a good human performance model are; 3) What the procedures and requirements in building and verifying a human performance model are; 4) How we integrate a human performance model with system design; and 5) What the possible future directions of human performance modeling research are.\", \"the five key questions of human performance modeling: 1) Why we build models of human performance; 2) What the expectations of a good human performance model are; 3) What the procedures and requirements in building and verifying a human performance model are; 4) How we integrate a human performance model with system design; and 5) What the possible future directions of human performance modeling research are\"], \"exact_answer\": [[\"Why build models?\"], [\"What are the expectations of a good model?\"], [\"What are the procedures and requirements?\"], [\"How do we integrate a model with system design?\"], [\"What are the future directions of Human performance modeling?\"]], \"context\": \"the five key questions of human performance modeling: 1) Why we build models of human performance; 2) What the expectations of a good human performance model are; 3) What the procedures and requirements in building and verifying a human performance model are; 4) How we integrate a human performance model with system design; and 5) What the possible future directions of human performance modeling research are. This paper describes and summarizes the five key questions of human performance modeling: 1) Why we build models of human performance; 2) What the expectations of a good human performance model are; 3) What the procedures and requirements in building and verifying a human performance model are; 4) How we integrate a human performance model with system design; and 5) What the possible future directions of human performance modeling research are.We conducted a systematic review of the literature to address 5 key questions specific to this population: 1) What are the current national clinical practice recommendations and guidelines for depression screening; 2) What are the prevalence and predictors of screening; 3) How well do screening tools detect depression; 4) Does screening lead to diagnosis, treatment, and improved outcomes; and 5) What are the most effective treatment methods?This paper describes and summarizes the five key questions of human performance modeling: 1) Why we build models of human performance; 2) What the expectations of a good human performance model are; 3) What the procedures and requirements in building and verifying a human performance model are; 4) How we integrate a human performance model with system design; and 5) What the possible future directions of human performance modeling research are. We conducted a systematic review of the literature to address 5 key questions specific to this population: 1) What are the current national clinical practice recommendations and guidelines for depression screening; 2) What are the prevalence and predictors of screening; 3) How well do screening tools detect depression; 4) Does screening lead to diagnosis, treatment, and improved outcomes; and 5) What are the most effective treatment methods?<br><b>METHODS</b>: We searched bibliographic databases for full-length articles published in English between 1990 and 2010 that addressed at least 1 of our key questions.<br><b>RESULTS</b>: We identified 5 clinical practice guidelines pertinent to question 1, and 12 systematic reviews or post-hoc analyses of pooled data that addressed questions 3 through 5.\"}, {\"question\": \"List the ERM proteins.\", \"ideal_answer\": [\"ezrin\\nradixin\\nmoesin\"], \"exact_answer\": [[\"ezrin\"], [\"radixin\"], [\"moesin\"]], \"context\": \"ezrin/radixin/moesin (ERM) proteinsezrin, radixin and moesin (ERM proteins)Ezrin/radixin/moesin (ERM) proteins\"}, {\"question\": \"Name three binding partners of cofilin 2.\", \"ideal_answer\": [\"Cofilin 2 can bind miR-201a, the protein 14-3-3, and ATP/ADP-Pi-actin filaments.\"], \"exact_answer\": [[\"miR-301a\"], [\"14-3-3\"], [\"ATP/ADP-Pi-actin filaments\"]], \"context\": \"Cofilin-2 (Cfl2) was identified as one of the potential targets of miR-301a using prediction databases, which we validated by luciferase assay and mutation of predicted binding sites.In contrast to other isoforms, cofilin-2 efficiently binds and disassembles both ADP- and ATP/ADP-Pi-actin filaments. We mapped surface-exposed cofilin-2-specific residues required for ATP-actin binding and propose that these residues function as an \\\"actin nucleotide-state sensor\\\" among ADF/cofilins.\"}, {\"question\": \"List proteins with RING domain.\", \"ideal_answer\": [\"RING1\\nTRIM proteins\\nTRAF6 \\nUHRF1\\nMARCH7\\nSINA\\nBRCA1\"], \"exact_answer\": [[\"RING1\"], [\"TRIM proteins\"], [\"TRAF6\"], [\"UHRF1\"], [\"MARCH7\"], [\"SINA\"], [\"BRCA1\"]], \"context\": \" The RING domain of RING1 was required for its E3 Ub ligase activity. The TRIM proteins are classically characterized by the presence of an amino-terminal RING domain and a B-box domain followed by a coiled coil domainthe detailed mechanism by which the TRAF6 RING dimer promotes ubiquitin transfer was unknown. RING ubiquitin E3 ligases enclose a RING domain for ubiquitin ligase activity UHRF1 RING domain MARCH7, a RING domain-containing ubiquitin E3 ligase,Seven in absentia (SINA) protein is one subgroup of ubiquitin ligases possessing an N-terminal cysteine-rich really interesting new gene (RING) domain,The BRCA1 RING domain is a ubiquitin ligase\"}, {\"question\": \"List 3 indications for rituximab.\", \"ideal_answer\": [\"Rituximab  is used to treat rheumatoid arthritis as well as poly- and dermatomyositis, chronic lymphocytic leukemia, juvenile idiopathic arthritis, and Pemphigus foliaceus (PF).\"], \"exact_answer\": [[\"Rheumatoid arthritis\"], [\"Chronic Lymphocytic  Leukemia\"], [\"poly- and dermatomyositis\"], [\"juvenile idiopathic arthritis\"], [\"Pemphigus foliaceus\"]], \"context\": \"o assess clinical outcomes including imaging findings on computed tomography (CT), pulmonary function testing (PFT), and glucocorticoid (GC) use in patients with the antisynthetase syndrome (AS) and interstitial lung disease (ILD) treated with rituximab (RTX). RIX is the preferred off-label biologic drug for poly- and dermatomyositis in Germany.Rituximab (RTX) may be a treatment option for children and young people with JIA, although it is not licensed for this indication rituximab in children and young people with juvenile idiopathic arthritischronic lymphocytic leukemia (CLL) patients, including the Bruton's tyrosine kinase (BTK) inhibitor ibrutinib, phosphatidylinositol-3-kinase (PI3K) delta isoform inhibitor idelalisib combined with rituximabrituximab's efficacy has been well-documented in adults with refractory PFRituximab in the management of juvenile pemphigus foliaceusBACKGROUND\\nRituximab is a chimeric, anti-CD20 monoclonal antibody registered for the treatment of B-cell malignancies and refractory rheumatoid arthritis in Australia.Currently, rituximab is only FDA-approved for treatment of follicular and diffuse large B-cell non-Hodgkin's lymphoma, rheumatoid arthritis, chronic lymphocytic leukemia, granulomatosis with polyangiitis (formerly Wegener's granulomatosis) and microscopic polyangiitis.<AbstractText Label=\\\"CONCLUSIONS\\\" NlmCategory=\\\"CONCLUSIONS\\\">In our study, off-label rituximab was mainly used for the treatment of hematological, kidney, and systemic connective tissue disorders, and the response among our patient cohort was variable depending on the specific disease. \"}]}"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Yx3qzLjAT_k5",
        "colab_type": "code",
        "outputId": "736cfcad-b6d7-4cfb-d9a1-a5265d1e8121",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        }
      },
      "source": [
        "questions = dataset['questions']\n",
        "len(dataset['questions'])"
      ],
      "execution_count": 27,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "644"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 27
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "hBuCI06E26mx",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "sample_question = dataset['questions'][0]"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "nlV4WQ9W3A7i",
        "colab_type": "code",
        "outputId": "b86f3ddb-3c1c-44f2-cf2d-11d8009ffbe0",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        }
      },
      "source": [
        "sample_question['question']"
      ],
      "execution_count": 29,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "'List signaling molecules (ligands) that interact with the receptor EGFR?'"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 29
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "w-bN1QHM3D2Q",
        "colab_type": "code",
        "outputId": "934795a0-f7a3-4bbb-fcb1-82001512cf3b",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 54
        }
      },
      "source": [
        "sample_question['context']"
      ],
      "execution_count": 30,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "'the epidermal growth factor receptor (EGFR) ligands, such as epidermal growth factor (EGF) and amphiregulin (AREG) EGFR ligands epidermal growth factor (EGF), amphiregulin (AREG) and transforming growth factor alpha (TGF) EGFR and its ligand EGF Among EGFR ligands, heparin-binding EGF-like growth factor, TGF- and Betacellulin (BTC) are produced in the tumor microenvironment of FDC-S at RNA level. . Plasma amphiregulin (AR), epidermal growth factor (EGF), transforming growth factor-, and heparin binding-EGF were assessed by ELISA in 45 chemorefractory mCRC patientsAmong EGFR ligands, heparin-binding epidermal growth factor (HB-EGF) Of the six known EGFR ligands, transforming growth factor alpha (TGF) was expressed more highly in triple-negative breast tumors than in tumors of other subtypes.the 7 known EGFR ligands (EGF, betacellulin, epiregulin, heparin-binding EGF, transforming growth factor- [TGF-], amphiregulin, and epigen) EGFR ligands based on the two affinity classes: EGF>HB-EGF>TGF->BTC>EPR>EPG>ARIn this article, however, we demonstrate that PEPD directly binds to and activates epidermal growth factor receptor (EGFR),four EGFR ligands (AR, HB-EGF, TGF-, and EREG) Epidermal growth factor (EGF) family peptides are ligands for the EGF receptor (EGFR). oluble amphiregulin (AR), transforming growth factor alpha (TGF), neuregulin 2 beta, and epigen stimulate greater EGFR coupling to cell proliferation and DNA synthesis than do EGF, betacellulin, heparin-binding EGF-like growth factor, and epiregulinHere, we demonstrate that histamine releases 2 EGFR ligands, amphiregulin and heparin-binding epidermal growth factor-like growth factor (HB-EGF), from airway epithelial cells.mammalian EGFR ligands including EGF, TGF- (TGF), amphiregulin (AREG), heparin-binding EGF-like growth factor (HB-EGF), betacellulin, epiregulin, and epigen.'"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 30
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "UDXGFrBJiKmn",
        "colab_type": "text"
      },
      "source": [
        "# Experiment 1: Run each BioASQ question on Elastic Search"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "qvObaQY_brX1",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "d56b057a-34b6-4cf4-d287-f6f38bc27f45"
      },
      "source": [
        "scores = []\n",
        "all_results = []\n",
        "\n",
        "it = tqdm(enumerate(questions), total=len(questions))\n",
        "\n",
        "for qix, q in it:\n",
        "    qin = q['question']\n",
        "    ranking = SEARCH(qin, 'bioasq-8b-baseline', 'context', limit=700)\n",
        "    all_results.append(ranking)\n",
        "    for i, (r, _) in enumerate(ranking):\n",
        "        if r['context'] == q['context']:\n",
        "            scores.append(1./(i+1.)) \n",
        "            break"
      ],
      "execution_count": 31,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "100%|| 644/644 [01:44<00:00,  6.15it/s]\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "NpvKaTDmfRR-",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "9fd32e62-e0d7-49d3-f0fe-07dad1ec9b3f"
      },
      "source": [
        "len(scores)"
      ],
      "execution_count": 32,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "643"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 32
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "8_QO27Jbh0jt",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 108
        },
        "outputId": "4e2c4820-e086-45c8-c252-5086e698e470"
      },
      "source": [
        "mrr = np.average(scores)\n",
        "experiments_results = [{'Experiment': 'Elastic Search', 'MRR': mrr}]\n",
        "pd.DataFrame(experiments_results)"
      ],
      "execution_count": 33,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "application/vnd.google.colaboratory.module+javascript": "\n      import \"https://ssl.gstatic.com/colaboratory/data_table/a6224c040fa35dcf/data_table.js\";\n\n      window.createDataTable({\n        data: [[{\n            'v': 0,\n            'f': \"0\",\n        },\n\"Elastic Search\",\n{\n            'v': 0.9124373431039607,\n            'f': \"0.9124373431039607\",\n        }]],\n        columns: [[\"number\", \"index\"], [\"string\", \"Experiment\"], [\"number\", \"MRR\"]],\n        columnOptions: [{\"width\": \"1px\", \"className\": \"index_column\"}],\n        rowsPerPage: 25,\n        helpUrl: \"https://colab.research.google.com/notebooks/data_table.ipynb\",\n        suppressOutputScrolling: true,\n        minimumWidth: undefined,\n      });\n    ",
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>Experiment</th>\n",
              "      <th>MRR</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>Elastic Search</td>\n",
              "      <td>0.912437</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "       Experiment       MRR\n",
              "0  Elastic Search  0.912437"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 33
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "19FoS_1JiOmF",
        "colab_type": "text"
      },
      "source": [
        "# Experiment 2: Use BERT Embeddings to compute MRR"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "kZZjTAxbhfrF",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 118
        },
        "outputId": "1a1a9202-6ca7-4ecf-913f-c1a7c713dcf1"
      },
      "source": [
        "#@title Make sure you have a GPU enabled\n",
        "print(\"Num GPUs Available: \", len(tf.config.experimental.list_physical_devices('GPU')))\n",
        "# force the GPU load\n",
        "with tf.Session(config=tf.ConfigProto(log_device_placement=True)) as sess:\n",
        "    pass"
      ],
      "execution_count": 34,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "Num GPUs Available:  1\n",
            "Device mapping:\n",
            "/job:localhost/replica:0/task:0/device:XLA_CPU:0 -> device: XLA_CPU device\n",
            "/job:localhost/replica:0/task:0/device:XLA_GPU:0 -> device: XLA_GPU device\n",
            "/job:localhost/replica:0/task:0/device:GPU:0 -> device: 0, name: Tesla P100-PCIE-16GB, pci bus id: 0000:00:04.0, compute capability: 6.0\n",
            "\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "gK4cz-Atn0QJ",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        },
        "outputId": "689e6a76-3ef2-4f8f-fbc4-56b12da34132"
      },
      "source": [
        "#@title Get all BioASQ sentence embeddings\n",
        "with open('./training8b_list.json') as fd:\n",
        "    dataset = json.load(fd)\n",
        "    questions = dataset['questions']\n",
        "\n",
        "# 1. extract context\n",
        "context_list = [ q['context'] for q in questions]\n",
        "query_list = [ q['question'] for q in questions] \n",
        "\n",
        "# 2. compute the embeddings\n",
        "embedding_list = predict_fn(query_list + context_list, estimator, tokenizer)\n",
        "\n",
        "query_embeddings = embedding_list[:len(query_list)] \n",
        "context_embeddings = embedding_list[len(query_list):] "
      ],
      "execution_count": 35,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "WARNING:tensorflow:From /content/bert/extract_features.py:283: The name tf.logging.info is deprecated. Please use tf.compat.v1.logging.info instead.\n",
            "\n",
            "INFO:tensorflow:*** Example ***\n",
            "INFO:tensorflow:unique_id: 0\n",
            "INFO:tensorflow:tokens: [CLS] list signaling molecules ( l ##igan ##ds ) that interact with the receptor e ##g ##f ##r ? [SEP]\n",
            "INFO:tensorflow:input_ids: 101 2190 16085 10799 113 181 10888 3680 114 1115 12254 1114 1103 10814 174 1403 2087 1197 136 102 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\n",
            "INFO:tensorflow:input_mask: 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\n",
            "INFO:tensorflow:input_type_ids: 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\n",
            "INFO:tensorflow:*** Example ***\n",
            "INFO:tensorflow:unique_id: 1\n",
            "INFO:tensorflow:tokens: [CLS] which mi ##rna ##s could be used as potential bio ##mark ##ers for e ##pit ##hel ##ial o ##var ##ian cancer ? [SEP]\n",
            "INFO:tensorflow:input_ids: 101 1134 1940 11782 1116 1180 1129 1215 1112 3209 25128 8519 1468 1111 174 18965 18809 2916 184 8997 1811 4182 136 102 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\n",
            "INFO:tensorflow:input_mask: 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\n",
            "INFO:tensorflow:input_type_ids: 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\n",
            "INFO:tensorflow:*** Example ***\n",
            "INFO:tensorflow:unique_id: 2\n",
            "INFO:tensorflow:tokens: [CLS] which ace ##ty ##l ##cho ##lines ##tera ##se inhibitor ##s are used for treatment of my ##ast ##hen ##ia g ##ra ##vis ? [SEP]\n",
            "INFO:tensorflow:input_ids: 101 1134 20839 2340 1233 8401 10443 17528 2217 27558 1116 1132 1215 1111 3252 1104 1139 12788 10436 1465 176 1611 9356 136 102 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\n",
            "INFO:tensorflow:input_mask: 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\n",
            "INFO:tensorflow:input_type_ids: 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\n",
            "INFO:tensorflow:*** Example ***\n",
            "INFO:tensorflow:unique_id: 3\n",
            "INFO:tensorflow:tokens: [CLS] list the human genes encoding for the dish ##eve ##lled proteins ? [SEP]\n",
            "INFO:tensorflow:input_ids: 101 2190 1103 1769 9077 18922 1111 1103 10478 19907 11572 7865 136 102 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\n",
            "INFO:tensorflow:input_mask: 1 1 1 1 1 1 1 1 1 1 1 1 1 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\n",
            "INFO:tensorflow:input_type_ids: 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\n",
            "INFO:tensorflow:*** Example ***\n",
            "INFO:tensorflow:unique_id: 4\n",
            "INFO:tensorflow:tokens: [CLS] which are the classes of anti - a ##rr ##hy ##th ##mic drugs according to v ##augh ##an - will ##iam ##s classification ? [SEP]\n",
            "INFO:tensorflow:input_ids: 101 1134 1132 1103 3553 1104 2848 118 170 11096 7889 1582 7257 5557 2452 1106 191 23825 1389 118 1209 18331 1116 5393 136 102 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\n",
            "INFO:tensorflow:input_mask: 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\n",
            "INFO:tensorflow:input_type_ids: 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0\n",
            "INFO:tensorflow:Could not find trained model in model_dir: /tmp/tmphc2waln9, running initialization to predict.\n",
            "WARNING:tensorflow:From /tensorflow-1.15.2/python3.6/tensorflow_core/python/ops/resource_variable_ops.py:1630: calling BaseResourceVariable.__init__ (from tensorflow.python.ops.resource_variable_ops) with constraint is deprecated and will be removed in a future version.\n",
            "Instructions for updating:\n",
            "If using Keras pass *_constraint arguments to layers.\n",
            "INFO:tensorflow:Calling model_fn.\n",
            "INFO:tensorflow:Running infer on CPU\n",
            "WARNING:tensorflow:From /content/bert/modeling.py:171: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.\n",
            "\n",
            "WARNING:tensorflow:From /content/bert/modeling.py:409: The name tf.get_variable is deprecated. Please use tf.compat.v1.get_variable instead.\n",
            "\n",
            "WARNING:tensorflow:From /content/bert/modeling.py:490: The name tf.assert_less_equal is deprecated. Please use tf.compat.v1.assert_less_equal instead.\n",
            "\n",
            "WARNING:tensorflow:From /content/bert/modeling.py:671: dense (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.\n",
            "Instructions for updating:\n",
            "Use keras.layers.Dense instead.\n",
            "WARNING:tensorflow:From /tensorflow-1.15.2/python3.6/tensorflow_core/python/layers/core.py:187: Layer.apply (from tensorflow.python.keras.engine.base_layer) is deprecated and will be removed in a future version.\n",
            "Instructions for updating:\n",
            "Please use `layer.__call__` method instead.\n",
            "WARNING:tensorflow:From /content/bert/extract_features.py:171: The name tf.trainable_variables is deprecated. Please use tf.compat.v1.trainable_variables instead.\n",
            "\n",
            "WARNING:tensorflow:From /content/bert/extract_features.py:184: The name tf.train.init_from_checkpoint is deprecated. Please use tf.compat.v1.train.init_from_checkpoint instead.\n",
            "\n",
            "INFO:tensorflow:**** Trainable Variables ****\n",
            "INFO:tensorflow:  name = bert/embeddings/word_embeddings:0, shape = (28996, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/embeddings/token_type_embeddings:0, shape = (2, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/embeddings/position_embeddings:0, shape = (512, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/embeddings/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/embeddings/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_0/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_0/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_0/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_0/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_0/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_0/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_0/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_0/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_0/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_0/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_0/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_0/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_0/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_0/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_0/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_0/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_1/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_1/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_1/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_1/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_1/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_1/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_1/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_1/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_1/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_1/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_1/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_1/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_1/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_1/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_1/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_1/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_2/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_2/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_2/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_2/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_2/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_2/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_2/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_2/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_2/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_2/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_2/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_2/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_2/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_2/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_2/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_2/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_3/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_3/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_3/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_3/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_3/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_3/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_3/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_3/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_3/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_3/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_3/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_3/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_3/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_3/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_3/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_3/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_4/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_4/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_4/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_4/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_4/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_4/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_4/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_4/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_4/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_4/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_4/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_4/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_4/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_4/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_4/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_4/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_5/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_5/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_5/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_5/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_5/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_5/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_5/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_5/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_5/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_5/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_5/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_5/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_5/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_5/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_5/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_5/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_6/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_6/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_6/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_6/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_6/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_6/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_6/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_6/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_6/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_6/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_6/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_6/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_6/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_6/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_6/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_6/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_7/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_7/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_7/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_7/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_7/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_7/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_7/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_7/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_7/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_7/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_7/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_7/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_7/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_7/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_7/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_7/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_8/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_8/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_8/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_8/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_8/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_8/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_8/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_8/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_8/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_8/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_8/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_8/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_8/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_8/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_8/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_8/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_9/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_9/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_9/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_9/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_9/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_9/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_9/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_9/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_9/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_9/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_9/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_9/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_9/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_9/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_9/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_9/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_10/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_10/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_10/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_10/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_10/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_10/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_10/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_10/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_10/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_10/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_10/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_10/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_10/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_10/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_10/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_10/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_11/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_11/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_11/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_11/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_11/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_11/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_11/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_11/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_11/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_11/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_11/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_11/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_11/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_11/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_11/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_11/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_12/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_12/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_12/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_12/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_12/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_12/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_12/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_12/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_12/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_12/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_12/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_12/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_12/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_12/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_12/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_12/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_13/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_13/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_13/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_13/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_13/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_13/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_13/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_13/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_13/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_13/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_13/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_13/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_13/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_13/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_13/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_13/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_14/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_14/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_14/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_14/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_14/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_14/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_14/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_14/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_14/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_14/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_14/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_14/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_14/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_14/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_14/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_14/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_15/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_15/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_15/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_15/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_15/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_15/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_15/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_15/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_15/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_15/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_15/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_15/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_15/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_15/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_15/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_15/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_16/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_16/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_16/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_16/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_16/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_16/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_16/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_16/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_16/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_16/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_16/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_16/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_16/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_16/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_16/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_16/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_17/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_17/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_17/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_17/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_17/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_17/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_17/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_17/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_17/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_17/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_17/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_17/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_17/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_17/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_17/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_17/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_18/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_18/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_18/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_18/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_18/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_18/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_18/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_18/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_18/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_18/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_18/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_18/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_18/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_18/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_18/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_18/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_19/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_19/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_19/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_19/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_19/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_19/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_19/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_19/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_19/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_19/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_19/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_19/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_19/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_19/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_19/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_19/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_20/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_20/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_20/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_20/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_20/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_20/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_20/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_20/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_20/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_20/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_20/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_20/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_20/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_20/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_20/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_20/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_21/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_21/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_21/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_21/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_21/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_21/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_21/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_21/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_21/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_21/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_21/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_21/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_21/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_21/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_21/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_21/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_22/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_22/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_22/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_22/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_22/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_22/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_22/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_22/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_22/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_22/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_22/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_22/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_22/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_22/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_22/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_22/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_23/attention/self/query/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_23/attention/self/query/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_23/attention/self/key/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_23/attention/self/key/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_23/attention/self/value/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_23/attention/self/value/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_23/attention/output/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_23/attention/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_23/attention/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_23/attention/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_23/intermediate/dense/kernel:0, shape = (1024, 4096), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_23/intermediate/dense/bias:0, shape = (4096,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_23/output/dense/kernel:0, shape = (4096, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_23/output/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_23/output/LayerNorm/beta:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/encoder/layer_23/output/LayerNorm/gamma:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/pooler/dense/kernel:0, shape = (1024, 1024), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:  name = bert/pooler/dense/bias:0, shape = (1024,), *INIT_FROM_CKPT*\n",
            "INFO:tensorflow:Done calling model_fn.\n",
            "WARNING:tensorflow:From /tensorflow-1.15.2/python3.6/tensorflow_core/python/ops/array_ops.py:1475: where (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.\n",
            "Instructions for updating:\n",
            "Use tf.where in 2.0, which has the same broadcast rule as np.where\n",
            "INFO:tensorflow:Graph was finalized.\n",
            "INFO:tensorflow:Running local_init_op.\n",
            "INFO:tensorflow:Done running local_init_op.\n",
            "INFO:tensorflow:prediction_loop marked as finished\n",
            "INFO:tensorflow:prediction_loop marked as finished\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "MvifJQ86pac6",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "assert len(query_embeddings) == len(context_embeddings)"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "PQ21r7wipy2V",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "def get_sent_embed(dict_list) :\n",
        "    sent_embed = []\n",
        "    for line in dict_list:\n",
        "        feat_embed = np.array(line['features'][0]['layers'][0]['values'])\n",
        "        # assert feat_embed.sum() > 0\n",
        "        sent_embed.append(feat_embed)\n",
        "    return sent_embed"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "EZ0spGmWn6qy",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "query_embeddings_x = get_sent_embed(query_embeddings)"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "m2EHt8PgpiRE",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "context_embeddings_x = get_sent_embed(context_embeddings)"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "-Cb7VA3pp4kW",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "QE = np.array(query_embeddings_x)"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "mDdbUb60qCNg",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "CE = np.array(context_embeddings_x)"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "EDOf9eAwqEuM",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "f6bd0fc8-d2a2-450c-a337-ee50b45ea759"
      },
      "source": [
        "QE.shape, CE.shape"
      ],
      "execution_count": 43,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "((644, 1024), (644, 1024))"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 43
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "DEFkcl8_qOml",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "from sklearn.metrics.pairwise import cosine_similarity"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "D0iK7qI6oics",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "result = cosine_similarity(QE, CE)"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "u5md82ORqSdy",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "7dfb8b16-4496-43cd-8862-a4bb6c46d8cd"
      },
      "source": [
        "result.shape"
      ],
      "execution_count": 46,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(644, 644)"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 46
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "EoVwXaaUqsgL",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 235
        },
        "outputId": "2ed4f7b0-9804-4410-d963-8326c69cf35f"
      },
      "source": [
        "cosine_similarity(CE, CE)\n"
      ],
      "execution_count": 47,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "array([[1.        , 0.60882387, 0.57962614, ..., 0.67935559, 0.55139151,\n",
              "        0.60941475],\n",
              "       [0.60882387, 1.        , 0.60887039, ..., 0.689746  , 0.58207824,\n",
              "        0.56694001],\n",
              "       [0.57962614, 0.60887039, 1.        , ..., 0.68568682, 0.61297892,\n",
              "        0.69365426],\n",
              "       ...,\n",
              "       [0.67935559, 0.689746  , 0.68568682, ..., 1.        , 0.70755747,\n",
              "        0.63964693],\n",
              "       [0.55139151, 0.58207824, 0.61297892, ..., 0.70755747, 1.        ,\n",
              "        0.56895656],\n",
              "       [0.60941475, 0.56694001, 0.69365426, ..., 0.63964693, 0.56895656,\n",
              "        1.        ]])"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 47
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "CIQqKJTAqaTq",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 235
        },
        "outputId": "b21da756-6114-4407-ee5c-2d89168b02f1"
      },
      "source": [
        "cosine_similarity(QE, QE)"
      ],
      "execution_count": 48,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "array([[1.        , 0.5792667 , 0.57194834, ..., 0.62601613, 0.69520312,\n",
              "        0.58311678],\n",
              "       [0.5792667 , 1.        , 0.55983656, ..., 0.41129551, 0.48551593,\n",
              "        0.45518869],\n",
              "       [0.57194834, 0.55983656, 1.        , ..., 0.48125675, 0.49455673,\n",
              "        0.4804643 ],\n",
              "       ...,\n",
              "       [0.62601613, 0.41129551, 0.48125675, ..., 1.        , 0.57228723,\n",
              "        0.46873744],\n",
              "       [0.69520312, 0.48551593, 0.49455673, ..., 0.57228723, 1.        ,\n",
              "        0.51643087],\n",
              "       [0.58311678, 0.45518869, 0.4804643 , ..., 0.46873744, 0.51643087,\n",
              "        1.        ]])"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 48
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "rnq0crLgrhBl",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 70
        },
        "outputId": "51ba108f-2b18-43e4-cee2-68260a2ac165"
      },
      "source": [
        "qix = 100\n",
        "qe = QE[qix]\n",
        "questions[qix]['question'], questions[qix]['context'] "
      ],
      "execution_count": 49,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "('Which proteins act as factors that promote transcription-coupled repair in bacteria?',\n",
              " 'Transcription-coupled repair (TCR) is a cellular process by which some forms of DNA damage are repaired more rapidly from transcribed strands of active genes than from nontranscribed strands or the overall genome. In humans, the TCR coupling factor, CSB, plays a critical role in restoring transcription following both UV-induced and oxidative DNA damage. It also contributes indirectly to the global repair of some forms of oxidative DNA damage. The Escherichia coli homolog, Mfd, is similarly required for TCR of UV-induced lesions.Transcription coupled nucleotide excision repair (TC-NER) is involved in correcting UV-induced damage and other road-blocks encountered in the transcribed strand. Mutation frequency decline (Mfd) is a transcription repair coupling factor, involved in repair of template strand during transcription.the transcription-repair coupling factor, Mfd, promotes direct restart of the fork following the collision by facilitating displacement of the RNAP.We report observations suggesting that the transcription elongation factor NusA promotes a previously unrecognized class of transcription-coupled repair (TCR) in addition to its previously proposed role in recruiting translesion synthesis (TLS) DNA polymerases to gaps encountered during transcription.NusA participates in an alternative class of TCR involved in the identification and removal of a class of lesion, such as the N(2)-f-dG lesionBacterial transcription-coupled repair is initiated when RNA polymerase stalled at a DNA lesion is removed by Mfd, an ATP-dependent DNA translocase.Transcription-coupled repair, the targeted repair of the transcribed strands of active genes, is defective in bacteria, yeast, and human cells carrying mutations in mfd, RAD26 and ERCC6, respectively.The effect of the bacterial transcription-repair coupling factor, Mfd, at such lesions is not known: it has been suggested that Mfd may promote mutagenesis by increasing the efficiency with which RNA polymerase bypasses non-bulky lesions, but it has also been reported that 8-oxoguanine, a major product of oxidative DNA damage that is efficiently bypassed by RNA polymerase, is subject to Mfd-dependent transcription-coupled repair in Escherichia coli.The transcription-repair coupling factor (TRCF, the product of the mfd gene) is a widely conserved bacterial protein that mediates transcription-coupled DNA repair.Recent structural studies of the bacterial transcription-repair coupling factor, Mfd, have revealed a modular architecture in which an ATP-dependent DNA-based motor is coupled to protein-protein interaction domains that can attach the motor to RNA polymerase and the DNA repair protein UvrA.The bacterial Mfd protein is a transcription-repair coupling factor that performs two key functions during transcription-coupled DNA repair.Mfd remains bound to the DNA in a long-lived complex that could act as a marker for sites of DNA damage, directing assembly of subsequent DNA repair factors.Transcription and DNA repair are coupled in E. coli by the Mfd protein, which dissociates transcription elongation complexes blocked at nonpairing lesions and mediates recruitment of DNA repair proteins.The first is to remove RNA polymerase (RNAP) complexes that have been stalled by a DNA lesion from the site of damage, and the second is to mediate the recruitment of DNA repair proteins.Coupling of transcription and DNA repair in bacteria is mediated by transcription-repair coupling factor (TRCF, the product of the mfd gene), which removes transcription elongation complexes stalled at DNA lesions and recruits the nucleotide excision repair machinery to the site.Mfd remains bound to the DNA in a long-lived complex that could act as a marker for sites of DNA damage, directing assembly of subsequent DNA repair factorsMfd may act through a translocase activity that rewinds upstream DNA, leading either to translocation or to release of RNA polymerase when the enzyme active site cannot continue elongation.Recent structural studies of the bacterial transcription-repair coupling factor, Mfd, have revealed a modular architecture in which an ATP-dependent DNA-based motor is coupled to protein-protein interaction domains that can attach the motor to RNA polymerase and the DNA repair protein UvrAE. coli Transcription repair coupling factor (Mfd protein) rescues arrested complexes by promoting forward translocation.We also find that the transcription-repair coupling factor, Mfd, promotes direct restart of the fork following the collision by facilitating displacement of the RNAPWe also found that the transcription-repair coupling factor Mfd promotes direct restart of the fork after the collision by facilitating displacement of the RNAPWe show that Mfd-dependent TCR in bacteria involves the formation of a damage search complex that can detect lesions downstream of a stalled RNAP, and that the strand specificity of the accelerated repair pathway is independent of the requirement for a lesion to stall RNAPcoli by the Mfd protein, which dissociates transcription elongation complexes blocked at nonpairing lesions and mediates recruitment of DNA repair proteins')"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 49
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "u1i0LyNovGjP",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "b5f352b2-87d7-4395-8ee8-d2338b646eb3"
      },
      "source": [
        "CE.shape"
      ],
      "execution_count": 50,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(644, 1024)"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 50
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "nhnZWKfXvJ2Z",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "CES= CE[:100,]"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "SqWSooj1vO3B",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "1d5c914e-73b9-47ac-f8c4-4efa79998c8d"
      },
      "source": [
        "scores = cosine_similarity(QE, CES)\n",
        "scores.shape"
      ],
      "execution_count": 52,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(644, 100)"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 52
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ZsrWBE8Drl_t",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "491e84fe-d11a-4f68-8242-a461fe8dcc1c"
      },
      "source": [
        "scores = cosine_similarity([qe], CE)\n",
        "scores.shape"
      ],
      "execution_count": 53,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(1, 644)"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 53
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "7k6ASaMirvYd",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "3ba78ace-f465-4c65-c74b-2ad603802aed"
      },
      "source": [
        "ranking = scores[0].argsort() # [::-1]\n",
        "ranking.shape"
      ],
      "execution_count": 54,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(644,)"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 54
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "8L77aFQZwbek",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "5002f01c-5c25-441a-9732-9b7fe0156d3a"
      },
      "source": [
        "qix"
      ],
      "execution_count": 55,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "100"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 55
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "y_-gjelJsByD",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "result_ix = np.where(ranking == qix)"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Ec6DHGQGsOdi",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "rix = result_ix[0][0]"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "BUazbOvpvuGF",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 171
        },
        "outputId": "fae985f5-dbbe-412c-a595-4ef6dcb9e0ab"
      },
      "source": [
        "questions[rix]\n"
      ],
      "execution_count": 58,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "{'context': 'o assess clinical outcomes including imaging findings on computed tomography (CT), pulmonary function testing (PFT), and glucocorticoid (GC) use in patients with the antisynthetase syndrome (AS) and interstitial lung disease (ILD) treated with rituximab (RTX). RIX is the preferred off-label biologic drug for poly- and dermatomyositis in Germany.Rituximab (RTX) may be a treatment option for children and young people with JIA, although it is not licensed for this indication rituximab in children and young people with juvenile idiopathic arthritischronic lymphocytic leukemia (CLL) patients, including the Bruton\\'s tyrosine kinase (BTK) inhibitor ibrutinib, phosphatidylinositol-3-kinase (PI3K) delta isoform inhibitor idelalisib combined with rituximabrituximab\\'s efficacy has been well-documented in adults with refractory PFRituximab in the management of juvenile pemphigus foliaceusBACKGROUND\\nRituximab is a chimeric, anti-CD20 monoclonal antibody registered for the treatment of B-cell malignancies and refractory rheumatoid arthritis in Australia.Currently, rituximab is only FDA-approved for treatment of follicular and diffuse large B-cell non-Hodgkin\\'s lymphoma, rheumatoid arthritis, chronic lymphocytic leukemia, granulomatosis with polyangiitis (formerly Wegener\\'s granulomatosis) and microscopic polyangiitis.<AbstractText Label=\"CONCLUSIONS\" NlmCategory=\"CONCLUSIONS\">In our study, off-label rituximab was mainly used for the treatment of hematological, kidney, and systemic connective tissue disorders, and the response among our patient cohort was variable depending on the specific disease. ',\n",
              " 'exact_answer': [['Rheumatoid arthritis'],\n",
              "  ['Chronic Lymphocytic  Leukemia'],\n",
              "  ['poly- and dermatomyositis'],\n",
              "  ['juvenile idiopathic arthritis'],\n",
              "  ['Pemphigus foliaceus']],\n",
              " 'ideal_answer': ['Rituximab  is used to treat rheumatoid arthritis as well as poly- and dermatomyositis, chronic lymphocytic leukemia, juvenile idiopathic arthritis, and Pemphigus foliaceus (PF).'],\n",
              " 'question': 'List 3 indications for rituximab.'}"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 58
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "vnvzRntcwUNw",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 104
        },
        "outputId": "34f129f1-e703-4366-c0c0-118682c94b17"
      },
      "source": [
        "questions[qix]"
      ],
      "execution_count": 59,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "{'context': 'Transcription-coupled repair (TCR) is a cellular process by which some forms of DNA damage are repaired more rapidly from transcribed strands of active genes than from nontranscribed strands or the overall genome. In humans, the TCR coupling factor, CSB, plays a critical role in restoring transcription following both UV-induced and oxidative DNA damage. It also contributes indirectly to the global repair of some forms of oxidative DNA damage. The Escherichia coli homolog, Mfd, is similarly required for TCR of UV-induced lesions.Transcription coupled nucleotide excision repair (TC-NER) is involved in correcting UV-induced damage and other road-blocks encountered in the transcribed strand. Mutation frequency decline (Mfd) is a transcription repair coupling factor, involved in repair of template strand during transcription.the transcription-repair coupling factor, Mfd, promotes direct restart of the fork following the collision by facilitating displacement of the RNAP.We report observations suggesting that the transcription elongation factor NusA promotes a previously unrecognized class of transcription-coupled repair (TCR) in addition to its previously proposed role in recruiting translesion synthesis (TLS) DNA polymerases to gaps encountered during transcription.NusA participates in an alternative class of TCR involved in the identification and removal of a class of lesion, such as the N(2)-f-dG lesionBacterial transcription-coupled repair is initiated when RNA polymerase stalled at a DNA lesion is removed by Mfd, an ATP-dependent DNA translocase.Transcription-coupled repair, the targeted repair of the transcribed strands of active genes, is defective in bacteria, yeast, and human cells carrying mutations in mfd, RAD26 and ERCC6, respectively.The effect of the bacterial transcription-repair coupling factor, Mfd, at such lesions is not known: it has been suggested that Mfd may promote mutagenesis by increasing the efficiency with which RNA polymerase bypasses non-bulky lesions, but it has also been reported that 8-oxoguanine, a major product of oxidative DNA damage that is efficiently bypassed by RNA polymerase, is subject to Mfd-dependent transcription-coupled repair in Escherichia coli.The transcription-repair coupling factor (TRCF, the product of the mfd gene) is a widely conserved bacterial protein that mediates transcription-coupled DNA repair.Recent structural studies of the bacterial transcription-repair coupling factor, Mfd, have revealed a modular architecture in which an ATP-dependent DNA-based motor is coupled to protein-protein interaction domains that can attach the motor to RNA polymerase and the DNA repair protein UvrA.The bacterial Mfd protein is a transcription-repair coupling factor that performs two key functions during transcription-coupled DNA repair.Mfd remains bound to the DNA in a long-lived complex that could act as a marker for sites of DNA damage, directing assembly of subsequent DNA repair factors.Transcription and DNA repair are coupled in E. coli by the Mfd protein, which dissociates transcription elongation complexes blocked at nonpairing lesions and mediates recruitment of DNA repair proteins.The first is to remove RNA polymerase (RNAP) complexes that have been stalled by a DNA lesion from the site of damage, and the second is to mediate the recruitment of DNA repair proteins.Coupling of transcription and DNA repair in bacteria is mediated by transcription-repair coupling factor (TRCF, the product of the mfd gene), which removes transcription elongation complexes stalled at DNA lesions and recruits the nucleotide excision repair machinery to the site.Mfd remains bound to the DNA in a long-lived complex that could act as a marker for sites of DNA damage, directing assembly of subsequent DNA repair factorsMfd may act through a translocase activity that rewinds upstream DNA, leading either to translocation or to release of RNA polymerase when the enzyme active site cannot continue elongation.Recent structural studies of the bacterial transcription-repair coupling factor, Mfd, have revealed a modular architecture in which an ATP-dependent DNA-based motor is coupled to protein-protein interaction domains that can attach the motor to RNA polymerase and the DNA repair protein UvrAE. coli Transcription repair coupling factor (Mfd protein) rescues arrested complexes by promoting forward translocation.We also find that the transcription-repair coupling factor, Mfd, promotes direct restart of the fork following the collision by facilitating displacement of the RNAPWe also found that the transcription-repair coupling factor Mfd promotes direct restart of the fork after the collision by facilitating displacement of the RNAPWe show that Mfd-dependent TCR in bacteria involves the formation of a damage search complex that can detect lesions downstream of a stalled RNAP, and that the strand specificity of the accelerated repair pathway is independent of the requirement for a lesion to stall RNAPcoli by the Mfd protein, which dissociates transcription elongation complexes blocked at nonpairing lesions and mediates recruitment of DNA repair proteins',\n",
              " 'exact_answer': [['Mfd'], ['NusA']],\n",
              " 'ideal_answer': ['Transcription coupled nucleotide excision repair (TC-NER or TCR) is a cellular process by which UV-induced damage and other road-blocks encountered in the transcribed strand are restored. Bacterial transcription-coupled repair is initiated when RNA polymerase stalled at a DNA lesion is removed by Mfd (Mutation frequency decline), an ATP-dependent DNA translocase. Mfd is the major transcription repair coupling factor in bacteria. Also, the transcription elongation factor NusA, in addition to its role in recruiting translesion synthesis (TLS) DNA polymerases to gaps encountered during transcription,  promotes an alternative class of TCR involved in the identification and removal of a class of lesion, such as the N(2)-f-dG lesion.'],\n",
              " 'question': 'Which proteins act as factors that promote transcription-coupled repair in bacteria?'}"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 59
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "UAKpiIAyrJKx",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "count = 0\n",
        "scores = []\n",
        "for qix, q in enumerate(questions):\n",
        "    qe = QE[qix]\n",
        "    \n",
        "    score = cosine_similarity([qe], CE)\n",
        "    ranking = score[0].argsort()[::-1]\n",
        "    result_ix = np.where(ranking == qix)\n",
        "    rix = result_ix[0][0]\n",
        "    scores.append( 1./(rix + 1) )\n",
        "\n",
        "    assert questions[ranking[rix]]['context'] == questions[qix]['context']"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "9fzsFB4nwr2L",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "501beb2e-6b66-4492-98bd-22785dfb49a2"
      },
      "source": [
        "MRR = np.average(scores)\n",
        "MRR"
      ],
      "execution_count": 61,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "0.5496368744211725"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 61
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "1GftjUzbkm2a",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 128
        },
        "outputId": "7acd9449-9e92-4f6d-f2d9-3dd6040aaf9e"
      },
      "source": [
        "experiments_results.append({ 'Experiment': 'BERT on 700k train',  'MRR':  MRR})\n",
        "pd.DataFrame(experiments_results)"
      ],
      "execution_count": 62,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "application/vnd.google.colaboratory.module+javascript": "\n      import \"https://ssl.gstatic.com/colaboratory/data_table/a6224c040fa35dcf/data_table.js\";\n\n      window.createDataTable({\n        data: [[{\n            'v': 0,\n            'f': \"0\",\n        },\n\"Elastic Search\",\n{\n            'v': 0.9124373431039607,\n            'f': \"0.9124373431039607\",\n        }],\n [{\n            'v': 1,\n            'f': \"1\",\n        },\n\"BERT on 700k train\",\n{\n            'v': 0.5496368744211725,\n            'f': \"0.5496368744211725\",\n        }]],\n        columns: [[\"number\", \"index\"], [\"string\", \"Experiment\"], [\"number\", \"MRR\"]],\n        columnOptions: [{\"width\": \"1px\", \"className\": \"index_column\"}],\n        rowsPerPage: 25,\n        helpUrl: \"https://colab.research.google.com/notebooks/data_table.ipynb\",\n        suppressOutputScrolling: true,\n        minimumWidth: undefined,\n      });\n    ",
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>Experiment</th>\n",
              "      <th>MRR</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>0</th>\n",
              "      <td>Elastic Search</td>\n",
              "      <td>0.912437</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>1</th>\n",
              "      <td>BERT on 700k train</td>\n",
              "      <td>0.549637</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "           Experiment       MRR\n",
              "0      Elastic Search  0.912437\n",
              "1  BERT on 700k train  0.549637"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 62
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "9ZRmRzw1xT5y",
        "colab_type": "text"
      },
      "source": [
        "# Experiment 3: Universal Sentence Encoder Embeddings \n",
        "Use a Sentence Encoder to generete the embeddings.\n",
        "\n",
        "https://tfhub.dev/google/universal-sentence-encoder-lite/2\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ccfqyxTTzE9g",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "%pip install --quiet --user --upgrade sentencepiece\n",
        "import sentencepiece as spm\n",
        "import tensorflow_hub as hub"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "C16qBvHG5j8z",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "def process_to_IDs_in_sparse_format(sp, sentences):\n",
        "  # An utility method that processes sentences with the sentence piece processor\n",
        "  # 'sp' and returns the results in tf.SparseTensor-similar format:\n",
        "  # (values, indices, dense_shape)\n",
        "  ids = [sp.EncodeAsIds(x) for x in sentences]\n",
        "  max_len = max(len(x) for x in ids)\n",
        "  dense_shape=(len(ids), max_len)\n",
        "  values=[item for sublist in ids for item in sublist]\n",
        "  indices=[[row,col] for row in range(len(ids)) for col in range(len(ids[row]))]\n",
        "  return (values, indices, dense_shape)\n"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "I_HGtMoS4rX0",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "def compute_embeddings(sentences):  \n",
        "    with tf.Session() as session:\n",
        "        input_placeholder = tf.sparse_placeholder(tf.int64, shape=[None, None])\n",
        "\n",
        "        module = hub.Module(\"https://tfhub.dev/google/universal-sentence-encoder-lite/2\")\n",
        "        \n",
        "        spm_path = session.run(module(signature=\"spm_path\"))\n",
        "        \n",
        "        sp = spm.SentencePieceProcessor()\n",
        "        sp.Load(spm_path)\n",
        "    \n",
        "        embeddings = module(\n",
        "            inputs=dict(\n",
        "                values=input_placeholder.values,\n",
        "                indices=input_placeholder.indices,\n",
        "                dense_shape=input_placeholder.dense_shape))\n",
        "        \n",
        "        values, indices, dense_shape = process_to_IDs_in_sparse_format(sp, sentences)\n",
        "        # initialize\n",
        "        session.run([tf.global_variables_initializer(), tf.tables_initializer()])        \n",
        "        # compute\n",
        "        message_embeddings = session.run(\n",
        "            embeddings,\n",
        "            feed_dict={input_placeholder.values: values,\n",
        "                        input_placeholder.indices: indices,\n",
        "                        input_placeholder.dense_shape: dense_shape})\n",
        "    return message_embeddings "
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "NAdz0bFL6M8t",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "context_list = [ q['context'] for q in questions]\n",
        "query_list = [ q['question'] for q in questions] "
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ma4NgB8r55J_",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 84
        },
        "outputId": "d8330999-1532-48a0-fec3-4ffe6eddd5cb"
      },
      "source": [
        "CE = compute_embeddings(context_list)"
      ],
      "execution_count": 67,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:tensorflow:Saver not created because there are no variables in the graph to restore\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "stream",
          "text": [
            "INFO:tensorflow:Saver not created because there are no variables in the graph to restore\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "INFO:tensorflow:Saver not created because there are no variables in the graph to restore\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "stream",
          "text": [
            "INFO:tensorflow:Saver not created because there are no variables in the graph to restore\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "6CvYA7OO8RAC",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        },
        "outputId": "6723b167-6b1a-4d1b-b8eb-bc6a7243aa95"
      },
      "source": [
        "CE.shape"
      ],
      "execution_count": 68,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "(644, 512)"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 68
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "VjFVoUDO6R_v",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 84
        },
        "outputId": "d8f92496-1031-45b3-ad2f-ae838228cef5"
      },
      "source": [
        "QE = compute_embeddings(query_list)\n"
      ],
      "execution_count": 69,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "INFO:tensorflow:Saver not created because there are no variables in the graph to restore\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "stream",
          "text": [
            "INFO:tensorflow:Saver not created because there are no variables in the graph to restore\n"
          ],
          "name": "stderr"
        },
        {
          "output_type": "stream",
          "text": [
            "INFO:tensorflow:Saver not created because there are no variables in the graph to restore\n"
          ],
          "name": "stdout"
        },
        {
          "output_type": "stream",
          "text": [
            "INFO:tensorflow:Saver not created because there are no variables in the graph to restore\n"
          ],
          "name": "stderr"
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "RSlmXS3p5_IN",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "count = 0\n",
        "scores = []\n",
        "for qix, q in enumerate(questions):\n",
        "    qe = QE[qix]\n",
        "    \n",
        "    score = cosine_similarity([qe], CE)\n",
        "    ranking = score[0].argsort()[::-1]\n",
        "    result_ix = np.where(ranking == qix)\n",
        "    rix = result_ix[0][0]\n",
        "    scores.append( 1./(rix + 1) )\n",
        "\n",
        "    assert questions[ranking[rix]]['context'] == questions[qix]['context']"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "WC4QghXC8aew",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "MRR = np.average(scores)"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "Oblzw8h6lxGC",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "experiments_results.append( {  'Experiment' : \"Universal Sentence Encoder\", \"MRR\": MRR } )"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "rnu84PBtl-3C",
        "colab_type": "text"
      },
      "source": [
        "# Overall Results"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "ne5WnIl3l98i",
        "colab_type": "code",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 128
        },
        "outputId": "b2541d6d-1d58-450a-e985-262b014a212b"
      },
      "source": [
        "pd.DataFrame(experiments_results).T"
      ],
      "execution_count": 73,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "application/vnd.google.colaboratory.module+javascript": "\n      import \"https://ssl.gstatic.com/colaboratory/data_table/a6224c040fa35dcf/data_table.js\";\n\n      window.createDataTable({\n        data: [[\"Experiment\",\n\"Elastic Search\",\n\"BERT on 700k train\",\n\"Universal Sentence Encoder\"],\n [\"MRR\",\n0.9124373431039607,\n0.5496368744211725,\n0.38342058777179133]],\n        columns: [[\"string\", \"index\"], [\"string\", \"0\"], [\"string\", \"1\"], [\"string\", \"2\"]],\n        columnOptions: [{\"width\": \"1px\", \"className\": \"index_column\"}],\n        rowsPerPage: 25,\n        helpUrl: \"https://colab.research.google.com/notebooks/data_table.ipynb\",\n        suppressOutputScrolling: true,\n        minimumWidth: undefined,\n      });\n    ",
            "text/html": [
              "<div>\n",
              "<style scoped>\n",
              "    .dataframe tbody tr th:only-of-type {\n",
              "        vertical-align: middle;\n",
              "    }\n",
              "\n",
              "    .dataframe tbody tr th {\n",
              "        vertical-align: top;\n",
              "    }\n",
              "\n",
              "    .dataframe thead th {\n",
              "        text-align: right;\n",
              "    }\n",
              "</style>\n",
              "<table border=\"1\" class=\"dataframe\">\n",
              "  <thead>\n",
              "    <tr style=\"text-align: right;\">\n",
              "      <th></th>\n",
              "      <th>0</th>\n",
              "      <th>1</th>\n",
              "      <th>2</th>\n",
              "    </tr>\n",
              "  </thead>\n",
              "  <tbody>\n",
              "    <tr>\n",
              "      <th>Experiment</th>\n",
              "      <td>Elastic Search</td>\n",
              "      <td>BERT on 700k train</td>\n",
              "      <td>Universal Sentence Encoder</td>\n",
              "    </tr>\n",
              "    <tr>\n",
              "      <th>MRR</th>\n",
              "      <td>0.912437</td>\n",
              "      <td>0.549637</td>\n",
              "      <td>0.383421</td>\n",
              "    </tr>\n",
              "  </tbody>\n",
              "</table>\n",
              "</div>"
            ],
            "text/plain": [
              "                         0                   1                           2\n",
              "Experiment  Elastic Search  BERT on 700k train  Universal Sentence Encoder\n",
              "MRR               0.912437            0.549637                    0.383421"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 73
        }
      ]
    }
  ]
}